
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="34" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A24205431220100000000000B67EE" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="33" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="34" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="35" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="34" /></div>
            <div><h2 class="result_count">Results: 3301 to 3400 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3301.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4189466/figure/f5-jcp-18-249/" ref="ordinalpos=3301&amp;ncbi_uid=6739407&amp;link_uid=PMC4189466" href="/pmc/articles/PMC4189466/"><img src="/pmc/articles/instance/4189466/bin/jcp-18-249f5.gif" src-large="/pmc/articles/instance/4189466/bin/jcp-18-249f5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4189466/" ref="ordinalpos=3301&amp;ncbi_uid=6739407&amp;link_uid=PMC4189466" image-link="/pmc/articles/PMC4189466/figure/f5-jcp-18-249/" class="imagepopup">Fig. 5. From: Morin Induces Heme Oxygenase-1 via ERK-Nrf2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A proposed <span class="highlight" style="background-color:">pathway</span> for morin-induced HO-1 via up-regulation of ERK, and Nrf2, explaining the cytoprotective effect against oxidative stress.</div></p></div><div class="aux"><div class="resc">Ji Young Park, et al. J Cancer Prev. 2013 September;18(3):249-256.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3301&amp;ncbi_uid=6739407&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4189466/" ref="ordinalpos=3301&amp;ncbi_uid=6739407&amp;link_uid=PMC4189466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3302.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3118822/figure/Fig5/" ref="ordinalpos=3302&amp;ncbi_uid=3366334&amp;link_uid=PMC3118822" href="/pmc/articles/PMC3118822/"><img src="/pmc/articles/instance/3118822/bin/12192_2011_255_Fig5_HTML.gif" src-large="/pmc/articles/instance/3118822/bin/12192_2011_255_Fig5_HTML.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3118822/" ref="ordinalpos=3302&amp;ncbi_uid=3366334&amp;link_uid=PMC3118822" image-link="/pmc/articles/PMC3118822/figure/Fig5/" class="imagepopup">Fig. 5.  From: Heat-shock protein 90 alpha (HSP90?) modulates <span class="highlight" style="background-color:">signaling</span> pathways towards tolerance of oxidative stress and enhanced survival of hepatocytes of Mugil cephalus. </a></p><div class="supp"><p class="details"><div>Schema flowchart illustrating the role of HSP90α in the mechanisms of cellular survival. HSP90α induction in response to environmental pollutant-mediated oxidative stress promotes the activation of its client proteins such as STATs of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>, ERK1/2 of the Ras/Raf/MAPK <span class="highlight" style="background-color:">pathway</span>, and Akt of the PI3K <span class="highlight" style="background-color:">pathway</span>. STATs upon activation during HSP90α induction favor survival after translocation to the nucleus. ERK1/2 and Akt activations under similar conditions promote cellular survival via their downstream effectors like Elk-1 activations and Bcl-2 upregulation, respectively. Activation of NF-κB by both the survival kinases also provides anti-apoptotic effect facilitating cellular survival. HSP90α-mediated Akt activation at the same time also acts to downregulate ASK1 allowing inhibition of ASK1-mediated JNK1/2 apoptotic <span class="highlight" style="background-color:">signaling</span>. Thus, HSP90α by modulating a series of <span class="highlight" style="background-color:">signaling</span> cascades protects against apoptosis and promotes hepatocyte survival</div></p></div><div class="aux"><div class="resc">Ekambaram Padmini, et al. Cell Stress Chaperones. 2011 July;16(4):411-425.</div><p class="links"><a class="dblinks" href="/pubmed/21274670" ref="ordinalpos=3302&amp;ncbi_uid=3366334&amp;link_uid=21274670">Citation</a><a class="dblinks" href="/pmc/articles/PMC3118822/" ref="ordinalpos=3302&amp;ncbi_uid=3366334&amp;link_uid=PMC3118822">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3303.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3037695/figure/F2/" ref="ordinalpos=3303&amp;ncbi_uid=3627810&amp;link_uid=PMC3037695" href="/pmc/articles/PMC3037695/"><img src="/pmc/articles/instance/3037695/bin/zbc0111151180002.gif" src-large="/pmc/articles/instance/3037695/bin/zbc0111151180002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3037695/" ref="ordinalpos=3303&amp;ncbi_uid=3627810&amp;link_uid=PMC3037695" image-link="/pmc/articles/PMC3037695/figure/F2/" class="imagepopup">FIGURE 2. From: Intracellular Pathogen Sensor NOD2 Programs Macrophages to Trigger Notch1 Activation. </a></p><div class="supp"><p class="details"><div><b>Cross-talk of Notch1 <span class="highlight" style="background-color:">signaling</span> and PI3K <span class="highlight" style="background-color:">pathway</span> during MDP-triggered activation of NOD2 <span class="highlight" style="background-color:">signaling</span>.</b> <i>A</i>, MDP triggers phosphorylation of p85 and 4EBP1 as analyzed by immunoblotting. <i>B</i>, inhibition of the RIP2 kinase by PP2 abolishes MDP-triggered phosphorylation of PI3K <span class="highlight" style="background-color:">pathway</span> members p85 and 4EBP1. <i>C</i>, pharmacological inhibition of Notch1 activation by GSI-I-abrogated MDP-triggered phosphorylation of p85 and 4EBP1. <i>D</i>, transient transfection of AKT dominant-negative (<i>AKT D/N</i>) or p85 dominant-negative (<i>p85 D/N</i>) constructs significantly reduced MDP-induced COX-2 expression in RAW 264.7 macrophages as assessed by immunoblotting. <i>E</i>, pharmacological intervention of PI3K <span class="highlight" style="background-color:">signaling</span> modulates MDP-triggered COX-2 expression. The blots are representative of three independent experiments. <i>Med</i>, medium.</div></p></div><div class="aux"><div class="resc">Kushagra Bansal, et al. J Biol Chem. 2011 February 18;286(7):5823-5835.</div><p class="links"><a class="dblinks" href="/pubmed/21156799" ref="ordinalpos=3303&amp;ncbi_uid=3627810&amp;link_uid=21156799">Citation</a><a class="dblinks" href="/pmc/articles/PMC3037695/" ref="ordinalpos=3303&amp;ncbi_uid=3627810&amp;link_uid=PMC3037695">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3304.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3307061/figure/fig3/" ref="ordinalpos=3304&amp;ncbi_uid=3863381&amp;link_uid=PMC3307061" href="/pmc/articles/PMC3307061/"><img src="/pmc/articles/instance/3307061/bin/onc2011301f3.gif" src-large="/pmc/articles/instance/3307061/bin/onc2011301f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3307061/" ref="ordinalpos=3304&amp;ncbi_uid=3863381&amp;link_uid=PMC3307061" image-link="/pmc/articles/PMC3307061/figure/fig3/" class="imagepopup">Figure 3.  From: A refined molecular taxonomy of breast cancer. </a></p><div class="supp"><p class="details"><div>Molecular subgroups show differential activation of major <span class="highlight" style="background-color:">signaling</span> pathways: correlations between a given <span class="highlight" style="background-color:">pathway</span> and a subgroup are indicated by color boxes. Red boxes show upregulation of the <span class="highlight" style="background-color:">pathway</span>, green downregulation. Up or downregulation was deduced using KEGGanim tool where relative expression measures are projected in the related KEGG <span class="highlight" style="background-color:">pathway</span> interaction graph. Pathways showing no clear direction of regulation were excluded.</div></p></div><div class="aux"><div class="resc">M Guedj, et al. Oncogene. 2012 March 1;31(9):1196-1206.</div><p class="links"><a class="dblinks" href="/pubmed/21785460" ref="ordinalpos=3304&amp;ncbi_uid=3863381&amp;link_uid=21785460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3307061/" ref="ordinalpos=3304&amp;ncbi_uid=3863381&amp;link_uid=PMC3307061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3305.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3981027/figure/f0030/" ref="ordinalpos=3305&amp;ncbi_uid=6012554&amp;link_uid=PMC3981027" href="/pmc/articles/PMC3981027/"><img src="/pmc/articles/instance/3981027/bin/gr6.gif" src-large="/pmc/articles/instance/3981027/bin/gr6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3981027/" ref="ordinalpos=3305&amp;ncbi_uid=6012554&amp;link_uid=PMC3981027" image-link="/pmc/articles/PMC3981027/figure/f0030/" class="imagepopup">Fig. 6.  From: Nitric oxide regulates cardiac intracellular Na+ and Ca2 + by modulating Na/K ATPase via PKC? and phospholemman-dependent mechanism. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Davor Pavlovic, et al. J Mol Cell Cardiol. 2013 August;61(100):164-171.</div><p class="links"><a class="dblinks" href="/pubmed/23612119" ref="ordinalpos=3305&amp;ncbi_uid=6012554&amp;link_uid=23612119">Citation</a><a class="dblinks" href="/pmc/articles/PMC3981027/" ref="ordinalpos=3305&amp;ncbi_uid=6012554&amp;link_uid=PMC3981027">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3306.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3655867/figure/F2/" ref="ordinalpos=3306&amp;ncbi_uid=4902539&amp;link_uid=PMC3655867" href="/pmc/articles/PMC3655867/"><img src="/pmc/articles/instance/3655867/bin/2047-783X-18-12-2.gif" src-large="/pmc/articles/instance/3655867/bin/2047-783X-18-12-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3655867/" ref="ordinalpos=3306&amp;ncbi_uid=4902539&amp;link_uid=PMC3655867" image-link="/pmc/articles/PMC3655867/figure/F2/" class="imagepopup">Figure 2.  From: Leptin, resistin and visfatin: the missing link between endocrine metabolic disorders and immunity. </a></p><div class="supp"><p class="details"><div>Leptin deficiency disturbs the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> JAK2/IRS/PI3K/AKT/AMPK and triggers diabetes.</div></p></div><div class="aux"><div class="resc">Ebtesam A AL-Suhaimi, et al. Eur J Med Res. 2013;18(1):12-12.</div><p class="links"><a class="dblinks" href="/pubmed/23634778" ref="ordinalpos=3306&amp;ncbi_uid=4902539&amp;link_uid=23634778">Citation</a><a class="dblinks" href="/pmc/articles/PMC3655867/" ref="ordinalpos=3306&amp;ncbi_uid=4902539&amp;link_uid=PMC3655867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3307.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3497295/figure/f1-ijms-13-12710/" ref="ordinalpos=3307&amp;ncbi_uid=4399991&amp;link_uid=PMC3497295" href="/pmc/articles/PMC3497295/"><img src="/pmc/articles/instance/3497295/bin/ijms-13-12710f1.gif" src-large="/pmc/articles/instance/3497295/bin/ijms-13-12710f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3497295/" ref="ordinalpos=3307&amp;ncbi_uid=4399991&amp;link_uid=PMC3497295" image-link="/pmc/articles/PMC3497295/figure/f1-ijms-13-12710/" class="imagepopup">Figure 1.  From: Crossing Paths in Human Renal Cell Carcinoma (hRCC). </a></p><div class="supp"><p class="details"><div>Schematic representation of selected <span class="highlight" style="background-color:">signaling</span> pathways in renal cell carcinoma (RCC). Angiogenic and cell proliferating <span class="highlight" style="background-color:">signaling</span> cascades are upregulated in RCC tumor cells. The vascular endothelial growth factor (VEGF) and other related growth factors secreted by tumor cells stimulate angiogenic <span class="highlight" style="background-color:">signaling</span> in the surrounding vascular endothelial cells. In response to growth factor <span class="highlight" style="background-color:">signaling</span> mediated through VEGF, platelet-derived growth factor (PDGF), and KIT receptors, PI3-kinase and Ras effectors activate hypoxia <span class="highlight" style="background-color:">pathway</span> (HIF) transcription factors, which in turn switch on gene expression needed for angiogenesis and cell proliferation in endothelial cells. In addition to the angiogenic <span class="highlight" style="background-color:">pathway</span>, the Wnt <span class="highlight" style="background-color:">pathway</span> is also upregulated in RCC tumor cells.</div></p></div><div class="aux"><div class="resc">Guadalupe Aparicio Gallego, et al. Int J Mol Sci. 2012;13(10):12710-12733.</div><p class="links"><a class="dblinks" href="/pubmed/23202921" ref="ordinalpos=3307&amp;ncbi_uid=4399991&amp;link_uid=23202921">Citation</a><a class="dblinks" href="/pmc/articles/PMC3497295/" ref="ordinalpos=3307&amp;ncbi_uid=4399991&amp;link_uid=PMC3497295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3308.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2064427/figure/fig10/" ref="ordinalpos=3308&amp;ncbi_uid=1194020&amp;link_uid=PMC2064427" href="/pmc/articles/PMC2064427/"><img src="/pmc/articles/instance/2064427/bin/jcb1780107f10.gif" src-large="/pmc/articles/instance/2064427/bin/jcb1780107f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2064427/" ref="ordinalpos=3308&amp;ncbi_uid=1194020&amp;link_uid=PMC2064427" image-link="/pmc/articles/PMC2064427/figure/fig10/" class="imagepopup">Figure 10. From: BMP gradients steer nerve growth cones by a balancing act of LIM kinase and Slingshot phosphatase on ADF/cofilin. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram illustrating the <span class="highlight" style="background-color:">signaling</span> mechanisms that control the bidirectional responses of the growth cone to BMP7 gradients.</b> In 4–8-h neurons, BMP7 induces an attractive response through the LIMK <span class="highlight" style="background-color:">pathway</span>. In overnight neurons, BMP7 repels the growth cone through the CaN–SSH phosphatase <span class="highlight" style="background-color:">pathway</span>, which is activated by Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> through TRP channels. ADF/cofilin acts as the common downstream target of these two <span class="highlight" style="background-color:">signaling</span> pathways to locally regulate the actin cytoskeleton for distinct turning responses. PM, plasma membrane.</div></p></div><div class="aux"><div class="resc">Zhexing Wen, et al. J Cell Biol. 2007 July 2;178(1):107-119.</div><p class="links"><a class="dblinks" href="/pubmed/17606869" ref="ordinalpos=3308&amp;ncbi_uid=1194020&amp;link_uid=17606869">Citation</a><a class="dblinks" href="/pmc/articles/PMC2064427/" ref="ordinalpos=3308&amp;ncbi_uid=1194020&amp;link_uid=PMC2064427">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3309.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891859/figure/F1/" ref="ordinalpos=3309&amp;ncbi_uid=3329614&amp;link_uid=PMC2891859" href="/pmc/articles/PMC2891859/"><img src="/pmc/articles/instance/2891859/bin/nihms167873f1.gif" src-large="/pmc/articles/instance/2891859/bin/nihms167873f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891859/" ref="ordinalpos=3309&amp;ncbi_uid=3329614&amp;link_uid=PMC2891859" image-link="/pmc/articles/PMC2891859/figure/F1/" class="imagepopup">Figure 1.  From: Role of BDNF and GDNF in drug reward and relapse: a review. </a></p><div class="supp"><p class="details"><div>General overview of BDNF and GDNF <span class="highlight" style="background-color:">signaling</span> mechanisms. GDNF binds to GFRα1 and activates Ret tyrosine kinase receptor. Alternatively, GDNF can activate the neural adhesion molecule (NCAM) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. BDNF physiological effects are mediated through the <span class="highlight" style="background-color:">signaling</span> cascades of the trkB and the p75 neurotrophin receptor.</div></p></div><div class="aux"><div class="resc">Udi E. Ghitza, et al. Neurosci Biobehav Rev. ;35(2):157-171.</div><p class="links"><a class="dblinks" href="/pubmed/19914287" ref="ordinalpos=3309&amp;ncbi_uid=3329614&amp;link_uid=19914287">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891859/" ref="ordinalpos=3309&amp;ncbi_uid=3329614&amp;link_uid=PMC2891859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3310.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3976861/figure/fig2/" ref="ordinalpos=3310&amp;ncbi_uid=6468298&amp;link_uid=PMC3976861" href="/pmc/articles/PMC3976861/"><img src="/pmc/articles/instance/3976861/bin/ISRN.TOXICOLOGY2014-976428.002.gif" src-large="/pmc/articles/instance/3976861/bin/ISRN.TOXICOLOGY2014-976428.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3976861/" ref="ordinalpos=3310&amp;ncbi_uid=6468298&amp;link_uid=PMC3976861" image-link="/pmc/articles/PMC3976861/figure/fig2/" class="imagepopup">Figure 2.  From: Cadmium Impairs p53 Activity in HepG2 Cells. </a></p><div class="supp"><p class="details"><div>Representation of the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map from KEGG. The genes identified as regulated in 10 <i>μ</i>m Cd-treated HepG2 cells are colored in the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map. The genes in red are upregulated and those in green are downregulated. Genes identified as regulated have a false discovery rate corrected <i>P</i> value smaller than 0.05 and fold change greater than 2.</div></p></div><div class="aux"><div class="resc">C. Urani, et al. ISRN Toxicol. 2014;2014:976428.</div><p class="links"><a class="dblinks" href="/pubmed/25101185" ref="ordinalpos=3310&amp;ncbi_uid=6468298&amp;link_uid=25101185">Citation</a><a class="dblinks" href="/pmc/articles/PMC3976861/" ref="ordinalpos=3310&amp;ncbi_uid=6468298&amp;link_uid=PMC3976861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3311.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3644607/figure/F2/" ref="ordinalpos=3311&amp;ncbi_uid=6075883&amp;link_uid=PMC3644607" href="/pmc/articles/PMC3644607/"><img src="/pmc/articles/instance/3644607/bin/zeg9991397950002.gif" src-large="/pmc/articles/instance/3644607/bin/zeg9991397950002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3644607/" ref="ordinalpos=3311&amp;ncbi_uid=6075883&amp;link_uid=PMC3644607" image-link="/pmc/articles/PMC3644607/figure/F2/" class="imagepopup">Figure 2. From: Prioritizing Genetic Testing in Patients With Kallmann Syndrome Using Clinical Phenotypes. </a></p><div class="supp"><p class="details"><div>Pubertal severity at presentation in male KS probands. A, Individual values of TV at presentation in male probands within each gene/<span class="highlight" style="background-color:">signaling</span> group. Prepubertal TV less than 4 mL and pubertal TV of 4 mL or greater. KAL1, <i>KAL1</i>; FGF, <i>FGF8/FGFR1</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; PROK, <i>PROK2/PROKR2</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; CHD7, <i>CHD7</i>; Neg, gene–negative group; HS, <i>HS6ST1</i>. B, Comparison of individual TV at presentation between probands harboring <i>KAL1</i> mutations (<i>KAL1</i> group) vs probands who were negative for <i>KAL1</i> mutations (non-<i>KAL1</i> group). *<i>P</i> &lt; .05</div></p></div><div class="aux"><div class="resc">Flavia Amanda Costa-Barbosa, et al. J Clin Endocrinol Metab. 2013 May;98(5):E943-E953.</div><p class="links"><a class="dblinks" href="/pubmed/23533228" ref="ordinalpos=3311&amp;ncbi_uid=6075883&amp;link_uid=23533228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3644607/" ref="ordinalpos=3311&amp;ncbi_uid=6075883&amp;link_uid=PMC3644607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3312.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1573867/figure/fig7/" ref="ordinalpos=3312&amp;ncbi_uid=799895&amp;link_uid=PMC1573867" href="/pmc/articles/PMC1573867/"><img src="/pmc/articles/instance/1573867/bin/139-0705266f7.gif" src-large="/pmc/articles/instance/1573867/bin/139-0705266f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1573867/" ref="ordinalpos=3312&amp;ncbi_uid=799895&amp;link_uid=PMC1573867" image-link="/pmc/articles/PMC1573867/figure/fig7/" class="imagepopup">Figure 7.  From: A3 adenosine and CB1 receptors activate a PKC-sensitive Cl− current in human nonpigmented ciliary epithelial cells via a G?γ-coupled MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of <i>I</i><sub>Cl,Win</sub> is mediated via a G<sub>i/o</sub> protein/G<i>βγ</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Mean <i>I</i><sub>Cl,Win</sub> (agonist-stimulated Cl<sup>−</sup> current minus basal Cl<sup>−</sup> current) following application of 1 <i>μ</i><span style="font-variant: small-caps" class="small-caps">M</span> Win 55,212-2 at –62 and +58 mV in: (a) NPCE cells before (Win, <i>n</i>=5) and after PTX treatment (Win+PTX, <i>n</i>=4), (b) NPCE cells transfected with control vector pIRES2-EGFP (Win+EGFP, <i>n</i>=5) or pIRES2-EGFP-<i>β</i>ARK (Win+EGFP-<i>β</i>ARK, <i>n</i>=4), (c) NPCE cells before (Win, <i>n</i>=7) and after 100 n<span style="font-variant: small-caps" class="small-caps">M</span> wortmannin treatment (Win+Wort, <i>n</i>=6), and (d) NPCE cells before (Win, <i>n</i>=6) and after 5 <i>μ</i><span style="font-variant: small-caps" class="small-caps">M</span> PD98059 treatment (Win+PD, <i>n</i>=6). <sup>*</sup><i>P</i>&lt;0.05, <sup>**</sup><i>P</i>&lt;0.001.</div></p></div><div class="aux"><div class="resc">Chanjuan Shi, et al. Br J Pharmacol. 2003 June;139(3):475-486.</div><p class="links"><a class="dblinks" href="/pubmed/12788807" ref="ordinalpos=3312&amp;ncbi_uid=799895&amp;link_uid=12788807">Citation</a><a class="dblinks" href="/pmc/articles/PMC1573867/" ref="ordinalpos=3312&amp;ncbi_uid=799895&amp;link_uid=PMC1573867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3313.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3706851/figure/F2/" ref="ordinalpos=3313&amp;ncbi_uid=5412490&amp;link_uid=PMC3706851" href="/pmc/articles/PMC3706851/"><img src="/pmc/articles/instance/3706851/bin/alzrt169-2.gif" src-large="/pmc/articles/instance/3706851/bin/alzrt169-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3706851/" ref="ordinalpos=3313&amp;ncbi_uid=5412490&amp;link_uid=PMC3706851" image-link="/pmc/articles/PMC3706851/figure/F2/" class="imagepopup">Figure 2.  From: 5-HT6 receptors and Alzheimer's disease. </a></p><div class="supp"><p class="details"><div><b>Biochemical mechanisms mediating 5-HT<sub>6 </sub>receptor functions</b>. In addition to activating cAMP <span class="highlight" style="background-color:">signaling</span> pathways, 5-HT<sub>6 </sub>receptors activate Ca<sup>2+ </sup><span class="highlight" style="background-color:">signaling</span> and the extracellular signal-regulated kinase 1/2 (ERK1/2) via Fyn-dependent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">María Javier Ramírez. Alzheimers Res Ther. 2013;5(2):15-15.</div><p class="links"><a class="dblinks" href="/pubmed/23607787" ref="ordinalpos=3313&amp;ncbi_uid=5412490&amp;link_uid=23607787">Citation</a><a class="dblinks" href="/pmc/articles/PMC3706851/" ref="ordinalpos=3313&amp;ncbi_uid=5412490&amp;link_uid=PMC3706851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3314.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2974067/figure/F1/" ref="ordinalpos=3314&amp;ncbi_uid=2563453&amp;link_uid=PMC2974067" href="/pmc/articles/PMC2974067/"><img src="/pmc/articles/instance/2974067/bin/cib0305_0409_fig001.gif" src-large="/pmc/articles/instance/2974067/bin/cib0305_0409_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2974067/" ref="ordinalpos=3314&amp;ncbi_uid=2563453&amp;link_uid=PMC2974067" image-link="/pmc/articles/PMC2974067/figure/F1/" class="imagepopup">Figure 1.  From: Model systems to the rescue. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that regulate longevity in <i>C. elegans</i>. The <i>daf-2</i> insulin/IGF-1-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (A) and germline proliferation (B) both regulate the DAF-16/FOXO family transcription factor, which in turn regulates nematode longevity.</div></p></div><div class="aux"><div class="resc">Scott Alper. Commun Integr Biol. 2010 Sep-Oct;3(5):409-414.</div><p class="links"><a class="dblinks" href="/pubmed/21057627" ref="ordinalpos=3314&amp;ncbi_uid=2563453&amp;link_uid=21057627">Citation</a><a class="dblinks" href="/pmc/articles/PMC2974067/" ref="ordinalpos=3314&amp;ncbi_uid=2563453&amp;link_uid=PMC2974067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3315.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3481284/figure/F8/" ref="ordinalpos=3315&amp;ncbi_uid=5417532&amp;link_uid=PMC3481284" href="/pmc/articles/PMC3481284/"><img src="/pmc/articles/instance/3481284/bin/zbc0421225210008.gif" src-large="/pmc/articles/instance/3481284/bin/zbc0421225210008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3481284/" ref="ordinalpos=3315&amp;ncbi_uid=5417532&amp;link_uid=PMC3481284" image-link="/pmc/articles/PMC3481284/figure/F8/" class="imagepopup">FIGURE 8. From: Wnt5a Controls Notch1 <span class="highlight" style="background-color:">Signaling</span> through CaMKII-mediated Degradation of the SMRT Corepressor Protein . </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of CaMKII regulates Notch1 <span class="highlight" style="background-color:">signaling</span> through proteasomal degradation of SMRT.</b> Frizzled (<i>Fz</i>), and Ror1/2 that mediates Wnt5a/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> has not yet been elucidated, however, the formation of a trimeric complex with Fz and Ror1/2 and the PDZ domain-mediated binding with dishevelled (<i>Dvl</i>) could be important. In this study, we evaluated that the mechanism by which Wnt up-regulates Notch1 target genes. We demonstrated that Wnt5a induced CaMKII enhances Notch transcriptional activity. CaMKII physically interacts with SMRT and disrupts the SMRT-RBP-Jk transcription complex through degradation of SMRT. Dissociation of SMRT from RBP-Jk induced by CaMKII activates <i>Hes1</i> and <i>Hes5</i> transcriptional activity. Furthermore, CaMKII can phosphorylate SMRT and activate the Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The kinase activity of CaMKII is necessary for up-regulation of the transcription activity of Notch1 target genes. The results of this study suggest that CaMKII is a crucial regulator for signal cross-talk between Wnt5a and Notch <span class="highlight" style="background-color:">signaling</span> via down-regulation of the SMRT protein level.</div></p></div><div class="aux"><div class="resc">Eun-Jung Ann, et al. J Biol Chem. 2012 October 26;287(44):36814-36829.</div><p class="links"><a class="dblinks" href="/pubmed/22888005" ref="ordinalpos=3315&amp;ncbi_uid=5417532&amp;link_uid=22888005">Citation</a><a class="dblinks" href="/pmc/articles/PMC3481284/" ref="ordinalpos=3315&amp;ncbi_uid=5417532&amp;link_uid=PMC3481284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3316.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3089520/figure/F6/" ref="ordinalpos=3316&amp;ncbi_uid=3396161&amp;link_uid=PMC3089520" href="/pmc/articles/PMC3089520/"><img src="/pmc/articles/instance/3089520/bin/zbc0251162650006.gif" src-large="/pmc/articles/instance/3089520/bin/zbc0251162650006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3089520/" ref="ordinalpos=3316&amp;ncbi_uid=3396161&amp;link_uid=PMC3089520" image-link="/pmc/articles/PMC3089520/figure/F6/" class="imagepopup">FIGURE 6. From: Differential <span class="highlight" style="background-color:">Signaling</span> by Adaptor Molecules LRP1 and ShcA Regulates Adipogenesis by the Insulin-like Growth Factor-1 Receptor. </a></p><div class="supp"><p class="details"><div><b>Activation of the adipocyte differentiation program by IGF-1 in LRP1-deficient cells.</b> <i>A</i>, 3T3-L1 preadipocytes knockdown for LRP1 (−/−), transfected with small interfering control (+/+) and non-transfected (<i>NT</i>) cells induced for adipocyte differentiation with a differentiation mixture containing IGF-1 for 6 days. Intracellular triglycerides (<i>B</i>) and cholesterol (<i>C</i>) quantification in 3T3-L1 preadipocytes knockdown for LRP1 (<i>black</i>), control (<i>gray</i>), and non-transfected cells (<i>white</i>) induced for adipogenesis during the indicated time with a differentiation mixture containing IGF-1. <i>D</i>, the LRP1/ShcA switches IGF-1 <span class="highlight" style="background-color:">signaling</span> from the Akt/mTOR to the Grb2/Ras/ERK1/2 <span class="highlight" style="background-color:">pathway</span>. LRP1 is required for a full activation of the ShcA/Grb2/Ras/ERK1/2 <span class="highlight" style="background-color:">pathway</span> and progression in the cell cycle in response to IGF-1. When LRP1 is lacking, the Akt/mTOR <span class="highlight" style="background-color:">pathway</span> is activated. Results are expressed as mean ± S.E. *, <i>p</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Estelle Woldt, et al. J Biol Chem. 2011 May 13;286(19):16775-16782.</div><p class="links"><a class="dblinks" href="/pubmed/21454706" ref="ordinalpos=3316&amp;ncbi_uid=3396161&amp;link_uid=21454706">Citation</a><a class="dblinks" href="/pmc/articles/PMC3089520/" ref="ordinalpos=3316&amp;ncbi_uid=3396161&amp;link_uid=PMC3089520">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3317.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1420312/figure/F5/" ref="ordinalpos=3317&amp;ncbi_uid=722889&amp;link_uid=PMC1420312" href="/pmc/articles/PMC1420312/"><img src="/pmc/articles/instance/1420312/bin/1471-2407-6-54-5.gif" src-large="/pmc/articles/instance/1420312/bin/1471-2407-6-54-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1420312/" ref="ordinalpos=3317&amp;ncbi_uid=722889&amp;link_uid=PMC1420312" image-link="/pmc/articles/PMC1420312/figure/F5/" class="imagepopup">Figure 5.  From: Cross-platform expression profiling demonstrates that SV40 small tumor antigen activates Notch, Hedgehog, and Wnt <span class="highlight" style="background-color:">signaling</span> in human cells. </a></p><div class="supp"><p class="details"><div>Impact of SV40 ST on the Notch, Wnt, and Hedgehog Pathways. Schematics of the Notch, Wnt, and Hedgehog pathways are shown. Genes with significantly increased mRNA's in HEK-TERST cells compared to HEK-TERV cells are shown in red, while genes significantly decreased are shown in green. Sets of downstream targets of the Wnt <span class="highlight" style="background-color:">pathway</span> are grouped in boxes, as are sets of c-myc downstream targets. Activation of the Hedgehog <span class="highlight" style="background-color:">pathway</span> due to increased HIP, SMO, and GLI2 levels results in increased Wnt ligand expression. Activation of Wnt <span class="highlight" style="background-color:">signaling</span> increases c-myc, activating and repressing multiple downstream target genes. Wnt <span class="highlight" style="background-color:">pathway</span> activation also feeds back to activate expression of Notch ligands such as DLL1.</div></p></div><div class="aux"><div class="resc">Mohamed Ali-Seyed, et al. BMC Cancer. 2006;6:54-54.</div><p class="links"><a class="dblinks" href="/pubmed/16522205" ref="ordinalpos=3317&amp;ncbi_uid=722889&amp;link_uid=16522205">Citation</a><a class="dblinks" href="/pmc/articles/PMC1420312/" ref="ordinalpos=3317&amp;ncbi_uid=722889&amp;link_uid=PMC1420312">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3318.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3180262/figure/F8/" ref="ordinalpos=3318&amp;ncbi_uid=3238866&amp;link_uid=PMC3180262" href="/pmc/articles/PMC3180262/"><img src="/pmc/articles/instance/3180262/bin/1475-2867-11-31-8.gif" src-large="/pmc/articles/instance/3180262/bin/1475-2867-11-31-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3180262/" ref="ordinalpos=3318&amp;ncbi_uid=3238866&amp;link_uid=PMC3180262" image-link="/pmc/articles/PMC3180262/figure/F8/" class="imagepopup">Figure 8.  From: Upstream molecular <span class="highlight" style="background-color:">signaling</span> pathways of p27(Kip1) expression in human breast cancer cells in vitro: differential effects of 4-hydroxytamoxifen and deficiency of either D-(+)-glucose or L-leucine. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram of the hypothetical molecular <span class="highlight" style="background-color:">signaling</span> pathways of the expression of p27 by which moderate increase in the concentration of <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-glucose down-regulates and deficiency of <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-glucose or <span style="font-variant: small-caps" class="small-caps">L</span>-leucine up-regulates the expression of p27</b>. This summary diagram shows the outline of how 4-hydroxytamoxifen uses primarily <span class="highlight" style="background-color:">pathway</span> #1 to up-regulate the expression of p27, arrest the G1-to-S phase transition of cell cycle, and inhibit DNA replication in human breast cancer cells <i>in vitro</i>. 4-Hydroxytamoxifen preferentially phosphorylates 4E-BP1 over S6K1. The diagram also shows the outline of how the deficiency of <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-glucose uses primarily <span class="highlight" style="background-color:">pathway</span> #2 to up-regulate the expression of p27, arrest the G1-to-S phase transition of cell cycle, and inhibit DNA replication in human breast cancer cells <i>in </i><i>vitro</i>. <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-Glucose deficiency preferentially phosphorylates S6K1 over 4E-BP1. The diagram also shows that the deficiency of <span style="font-variant: small-caps" class="small-caps">L</span>-leucine enters the <span class="highlight" style="background-color:">pathway</span> #2 at points different from the deficiency of <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-glucose. Finally, the diagram shows that deficiency of <span style="font-variant: small-caps" class="small-caps">D</span>-(+)-glucose or <span style="font-variant: small-caps" class="small-caps">L</span>-leucine uses <span style="font-variant: small-caps" class="small-caps">L</span>-upstream molecular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> #2 of the expression of p27 to up-regulate the phosphorylation of AMPK and the expression of mitochondrial ATP5A and SIRT3. The mitochondrial SIRT3 exerts anti-aging and other metabolic effects on the cells.</div></p></div><div class="aux"><div class="resc">Isao Eto. Cancer Cell Int. 2011;11:31-31.</div><p class="links"><a class="dblinks" href="/pubmed/21906315" ref="ordinalpos=3318&amp;ncbi_uid=3238866&amp;link_uid=21906315">Citation</a><a class="dblinks" href="/pmc/articles/PMC3180262/" ref="ordinalpos=3318&amp;ncbi_uid=3238866&amp;link_uid=PMC3180262">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3319.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2590753/figure/F7/" ref="ordinalpos=3319&amp;ncbi_uid=1666152&amp;link_uid=PMC2590753" href="/pmc/articles/PMC2590753/"><img src="/pmc/articles/instance/2590753/bin/nihms59069f7.gif" src-large="/pmc/articles/instance/2590753/bin/nihms59069f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2590753/" ref="ordinalpos=3319&amp;ncbi_uid=1666152&amp;link_uid=PMC2590753" image-link="/pmc/articles/PMC2590753/figure/F7/" class="imagepopup">Figure 7. Schematic of proposed mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the SCN.  From: Photic regulation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the suprachiasmatic circadian clock. </a></p><div class="supp"><p class="details"><div>Photic input from the retina induces the release of glutamate and PACAP from RHT terminals. Transmitter binding to postsynaptic receptors on SCN neurons evokes a series of intracellular signal transduction events, including MAPK <span class="highlight" style="background-color:">pathway</span> activation. One downstream target of the MAPK cascade is CREB, which, when phosphorylated on Ser-133, drives clock gene transcription. Light-induced activation of the MAPK cascade also stimulates mTORC1, which in turn targets the translation regulators p70 S6K and 4E-BP1. Activated p70 S6K stimulates S6 phosphorylation, whereas 4E-BP1 phosphorylation has been shown to lead to its dissociation from eIF4E, thereby allowing cap-dependent mRNA translation to occur. U0126 is a specific MEK 1/2 inhibitor and rapamycin inhibits mTORC1.</div></p></div><div class="aux"><div class="resc">Ruifeng Cao, et al. Mol Cell Neurosci. ;38(3):312-324.</div><p class="links"><a class="dblinks" href="/pubmed/18468454" ref="ordinalpos=3319&amp;ncbi_uid=1666152&amp;link_uid=18468454">Citation</a><a class="dblinks" href="/pmc/articles/PMC2590753/" ref="ordinalpos=3319&amp;ncbi_uid=1666152&amp;link_uid=PMC2590753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3320.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2768398/figure/fig6/" ref="ordinalpos=3320&amp;ncbi_uid=2137694&amp;link_uid=PMC2768398" href="/pmc/articles/PMC2768398/"><img src="/pmc/articles/instance/2768398/bin/toxscikfn221f06_3c.gif" src-large="/pmc/articles/instance/2768398/bin/toxscikfn221f06_3c.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2768398/" ref="ordinalpos=3320&amp;ncbi_uid=2137694&amp;link_uid=PMC2768398" image-link="/pmc/articles/PMC2768398/figure/fig6/" class="imagepopup">FIG. 6. From: Cadmium-Induced Differential Toxicogenomic Response in Resistant and Sensitive Mouse Strains Undergoing Neurulation. </a></p><div class="supp"><p class="details"><div>Cd-induced alterations within the ingenuity canonical p53-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in C57BL/6J and SWV embryos. Cd-induced alterations in C57 and SWV embryos were mapped to the p53 <span class="highlight" style="background-color:">Signaling</span> canonical ingenuity <span class="highlight" style="background-color:">pathway</span> (w/modifications). Genes are color coded based on Cd-effect specificity. Shapes of genes indicate known functions.</div></p></div><div class="aux"><div class="resc">Joshua F. Robinson, et al. Toxicol Sci. 2009 January;107(1):206-219.</div><p class="links"><a class="dblinks" href="/pubmed/18974090" ref="ordinalpos=3320&amp;ncbi_uid=2137694&amp;link_uid=18974090">Citation</a><a class="dblinks" href="/pmc/articles/PMC2768398/" ref="ordinalpos=3320&amp;ncbi_uid=2137694&amp;link_uid=PMC2768398">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3321.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4148120/figure/F7/" ref="ordinalpos=3321&amp;ncbi_uid=6546723&amp;link_uid=PMC4148120" href="/pmc/articles/PMC4148120/"><img src="/pmc/articles/instance/4148120/bin/oncotarget-05-5040-g007.gif" src-large="/pmc/articles/instance/4148120/bin/oncotarget-05-5040-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4148120/" ref="ordinalpos=3321&amp;ncbi_uid=6546723&amp;link_uid=PMC4148120" image-link="/pmc/articles/PMC4148120/figure/F7/" class="imagepopup">Figure 7. Scheme of the MEK inhibitor induced crosstalk mechanism.  From: The MAPK <span class="highlight" style="background-color:">pathway</span> as an apoptosis enhancer in melanoma. </a></p><div class="supp"><p class="details"><div><b>A</b>: In absence of MEK inhibitor, BRAF<sup>V600E</sup> activation leads to the ERK1/2-dependent induction of SPRED genes, which block RAS <span class="highlight" style="background-color:">signaling</span> downstream of RTKs in melanoma cells. This results in low PI3K <span class="highlight" style="background-color:">pathway</span> activity. When cells are exposed to genotoxic stress, PUMA is induced and cells undergo apoptosis. <b>B</b>: Application of MEK inhibitor abolishes the expression of <i>SPRED1</i> and <i>SPRED2</i> and thereby relieves the negative feedback on RTK <span class="highlight" style="background-color:">signaling</span>. As a result, the PI3K <span class="highlight" style="background-color:">pathway</span> is activated. This prevents the induction of the apoptosis promoter PUMA in response to genotoxic stress and allows enhanced survival and protection from apoptosis.</div></p></div><div class="aux"><div class="resc">Johannes M. Haydn, et al. Oncotarget. 2014 July;5(13):5040-5053.</div><p class="links"><a class="dblinks" href="/pubmed/24970815" ref="ordinalpos=3321&amp;ncbi_uid=6546723&amp;link_uid=24970815">Citation</a><a class="dblinks" href="/pmc/articles/PMC4148120/" ref="ordinalpos=3321&amp;ncbi_uid=6546723&amp;link_uid=PMC4148120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3322.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2405915/figure/F8/" ref="ordinalpos=3322&amp;ncbi_uid=1435715&amp;link_uid=PMC2405915" href="/pmc/articles/PMC2405915/"><img src="/pmc/articles/instance/2405915/bin/nihms-44878-f0008.gif" src-large="/pmc/articles/instance/2405915/bin/nihms-44878-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2405915/" ref="ordinalpos=3322&amp;ncbi_uid=1435715&amp;link_uid=PMC2405915" image-link="/pmc/articles/PMC2405915/figure/F8/" class="imagepopup">Figure 8.  From: Epidermal Stem Cells of the Skin. </a></p><div class="supp"><p class="details"><div>Summary of the <span class="highlight" style="background-color:">signaling</span> pathways involved in hair follicle morphogenesis and regeneration. This schematic summarizes many of the <span class="highlight" style="background-color:">signaling</span> pathways involved in hair follicle morphogenesis and hair follicle regeneration. Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> acts early in hair follicle specification and quiescent bulge stem cell (SC) activation. The sonic hedgehog (Shh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> acts in the second step to promote embryonic and adult hair germ proliferation. Bone morphogenetic protein (BMP), Notch, and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> pathways act further downstream to allow the normal differentiation of matrix cells into the hair shaft (HS) and its inner root sheath (IRS) envelope.</div></p></div><div class="aux"><div class="resc">Cédric Blanpain, et al. Annu Rev Cell Dev Biol. ;22:339-373.</div><p class="links"><a class="dblinks" href="/pubmed/16824012" ref="ordinalpos=3322&amp;ncbi_uid=1435715&amp;link_uid=16824012">Citation</a><a class="dblinks" href="/pmc/articles/PMC2405915/" ref="ordinalpos=3322&amp;ncbi_uid=1435715&amp;link_uid=PMC2405915">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3323.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3073847/figure/F1/" ref="ordinalpos=3323&amp;ncbi_uid=3567597&amp;link_uid=PMC3073847" href="/pmc/articles/PMC3073847/"><img src="/pmc/articles/instance/3073847/bin/nihms272201f1.gif" src-large="/pmc/articles/instance/3073847/bin/nihms272201f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3073847/" ref="ordinalpos=3323&amp;ncbi_uid=3567597&amp;link_uid=PMC3073847" image-link="/pmc/articles/PMC3073847/figure/F1/" class="imagepopup">Figure 1.  From: Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Patients with Kawasaki Disease. </a></p><div class="supp"><p class="details"><div>TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> TGF-β is secreted in a latent form and activation is mediated by several molecules including furin and emilin. Active TGF-β peptides (TGFB1, 2, 3) bind to the Type II receptor (TGFBR2), which recruits and activates Type I receptors such as TGFBR1 or ACVRL1. The activated Type I receptor phosphorylates receptor-specific SMAD molecules: TGFBR1 phosphorylates SMAD2 and 3, and ACVRL1 phosphorylates SMAD1, 5 and 8. These activated SMADs form a larger complex with Smad4 and translocate to the nucleus where they regulate gene transcription. Accessory receptors such as ENG have a role in the balance of ACVRL1 and TGFBR1 <span class="highlight" style="background-color:">signaling</span> to regulate endothelial cell proliferation. In addition to the classical SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, a SMAD-independent <span class="highlight" style="background-color:">pathway</span> using alternative molecules such as MAP3K7 can also mediate TGF-β <span class="highlight" style="background-color:">signaling</span>. TGFB: Transforming growth factor-β, TGFBR: Transforming growth factor-β receptor, EMILIN1: Elastin microfibril interfacer 1, SMAD: SMAD family member, MAP3K7: Mitogen-activated protein kinase kinase kinase 7, ENG: Endoglin, ACVRL1: Activin A receptor type II-like 1, FURIN: Furin</div></p></div><div class="aux"><div class="resc">Chisato Shimizu, et al. Circ Cardiovasc Genet. ;4(1):16-25.</div><p class="links"><a class="dblinks" href="/pubmed/21127203" ref="ordinalpos=3323&amp;ncbi_uid=3567597&amp;link_uid=21127203">Citation</a><a class="dblinks" href="/pmc/articles/PMC3073847/" ref="ordinalpos=3323&amp;ncbi_uid=3567597&amp;link_uid=PMC3073847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3324.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3902172/figure/F7/" ref="ordinalpos=3324&amp;ncbi_uid=6435696&amp;link_uid=PMC3902172" href="/pmc/articles/PMC3902172/"><img src="/pmc/articles/instance/3902172/bin/nihms-498782-f0007.gif" src-large="/pmc/articles/instance/3902172/bin/nihms-498782-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3902172/" ref="ordinalpos=3324&amp;ncbi_uid=6435696&amp;link_uid=PMC3902172" image-link="/pmc/articles/PMC3902172/figure/F7/" class="imagepopup">Figure 7. “Road map” of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hematopoiesis.  From: In vivo Mapping of Notch <span class="highlight" style="background-color:">Pathway</span> Activity in Normal and Stress Hematopoiesis. </a></p><div class="supp"><p class="details"><div>General overview of level of Notch1 receptor expression (blue gradient), Notch2 receptor expression (red gradient), Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation reported by Hes1<sup>GFP</sup> expression (green gradient) as well as known niches and Notch ligands involved in adult hematopoiesis. LT-HSC: Long Term Hematopoietic Stem Cell; MPP: Multi-Potential Progenitor; CMP: Common Myeloid Progenitor; GMP: Granulocyte/Monocyte Progenitor; Pre-MegE: Pre-Megakaryocyte/Erythrocyte Progenitor; MkP: Megakaryocyte Progenitor; CFU-E: Colony Forming Unit Erythrocyte; ProE: Pro-Erythroblast; LMPP: Lymphocyte-Primed Multi-Potential Progenitor; CLP: Common Lymphocyte Progenitor; ProB: Pro B-cell; FoB: Follicular B-cell; MzB: Marginal zone B-cell; ETP: Early Thymic Progenitor; DN: Double Negative (CD4<sup>−</sup>8<sup>−</sup>) T cell progenitor 3.</div></p></div><div class="aux"><div class="resc">Philmo Oh, et al. Cell Stem Cell. ;13(2):190-204.</div><p class="links"><a class="dblinks" href="/pubmed/23791481" ref="ordinalpos=3324&amp;ncbi_uid=6435696&amp;link_uid=23791481">Citation</a><a class="dblinks" href="/pmc/articles/PMC3902172/" ref="ordinalpos=3324&amp;ncbi_uid=6435696&amp;link_uid=PMC3902172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3325.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4143615/figure/F2/" ref="ordinalpos=3325&amp;ncbi_uid=6596916&amp;link_uid=PMC4143615" href="/pmc/articles/PMC4143615/"><img src="/pmc/articles/instance/4143615/bin/fimmu-05-00396-g002.gif" src-large="/pmc/articles/instance/4143615/bin/fimmu-05-00396-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4143615/" ref="ordinalpos=3325&amp;ncbi_uid=6596916&amp;link_uid=PMC4143615" image-link="/pmc/articles/PMC4143615/figure/F2/" class="imagepopup">Figure 2.  From: RNA-seq Transcriptional Profiling of Peripheral Blood Leukocytes from Cattle Infected with Mycobacterium bovis. </a></p><div class="supp"><p class="details"><div><b>The top-ranked enriched canonical <span class="highlight" style="background-color:">pathway</span> identified using IPA, the <i>T-cell receptor <span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span></b>. Red shading indicates increased expression in <i>M. bovis</i>-infected animals relative to the non-infected control group. Green shading indicates decreased expression in <i>M. bovis</i>-infected animals relative to the non-infected control group.</div></p></div><div class="aux"><div class="resc">Kirsten E. McLoughlin, et al. Front Immunol. 2014;5:396.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3325&amp;ncbi_uid=6596916&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4143615/" ref="ordinalpos=3325&amp;ncbi_uid=6596916&amp;link_uid=PMC4143615">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3326.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3755409/figure/fig6/" ref="ordinalpos=3326&amp;ncbi_uid=5260184&amp;link_uid=PMC3755409" href="/pmc/articles/PMC3755409/"><img src="/pmc/articles/instance/3755409/bin/ECAM2013-305898.006.gif" src-large="/pmc/articles/instance/3755409/bin/ECAM2013-305898.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3755409/" ref="ordinalpos=3326&amp;ncbi_uid=5260184&amp;link_uid=PMC3755409" image-link="/pmc/articles/PMC3755409/figure/fig6/" class="imagepopup">Figure 6.  From: Therapeutic Potential of Andrographolide Isolated from the Leaves of Andrographis paniculata Nees for Treating Lung Adenocarcinomas. </a></p><div class="supp"><p class="details"><div>Scheme of the andrographolide regulatory <span class="highlight" style="background-color:">pathway</span>. The effects of andrographolide inhibition of lung tumors are hypothesized through a cell-cycle <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The diagram shows that andrographolide may inhibit VEGF-A, KDR, Nrp1, ERK2, Cyclin A, and Cyclin B expression.</div></p></div><div class="aux"><div class="resc">Yu-Tang Tung, et al. Evid Based Complement Alternat Med. 2013;2013:305898.</div><p class="links"><a class="dblinks" href="/pubmed/23997793" ref="ordinalpos=3326&amp;ncbi_uid=5260184&amp;link_uid=23997793">Citation</a><a class="dblinks" href="/pmc/articles/PMC3755409/" ref="ordinalpos=3326&amp;ncbi_uid=5260184&amp;link_uid=PMC3755409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3327.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436697/figure/F5/" ref="ordinalpos=3327&amp;ncbi_uid=4221811&amp;link_uid=PMC3436697" href="/pmc/articles/PMC3436697/"><img src="/pmc/articles/instance/3436697/bin/2049-1891-3-23-5.gif" src-large="/pmc/articles/instance/3436697/bin/2049-1891-3-23-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436697/" ref="ordinalpos=3327&amp;ncbi_uid=4221811&amp;link_uid=PMC3436697" image-link="/pmc/articles/PMC3436697/figure/F5/" class="imagepopup">Figure 5.  From: Uterine biology in pigs and sheep. </a></p><div class="supp"><p class="details"><div>The hexosamine <span class="highlight" style="background-color:">pathway</span> allows for both glucose and fructose to be metabolized to glucosamine-6-phosphate that lead to activation of the MTOR cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, as well as synthesis of glycosaminoglycans such as hyaluronic acid, that are critical to growth and development of the conceptus.</div></p></div><div class="aux"><div class="resc">Fuller W Bazer, et al. J Anim Sci Biotechnol. 2012;3(1):23-23.</div><p class="links"><a class="dblinks" href="/pubmed/22958877" ref="ordinalpos=3327&amp;ncbi_uid=4221811&amp;link_uid=22958877">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436697/" ref="ordinalpos=3327&amp;ncbi_uid=4221811&amp;link_uid=PMC3436697">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3328.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4308786/figure/F7/" ref="ordinalpos=3328&amp;ncbi_uid=7127271&amp;link_uid=PMC4308786" href="/pmc/articles/PMC4308786/"><img src="/pmc/articles/instance/4308786/bin/NRR-7-1370-g009.gif" src-large="/pmc/articles/instance/4308786/bin/NRR-7-1370-g009.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4308786/" ref="ordinalpos=3328&amp;ncbi_uid=7127271&amp;link_uid=PMC4308786" image-link="/pmc/articles/PMC4308786/figure/F7/" class="imagepopup">Figure 7.  From: Mitogen activated protein kinase <span class="highlight" style="background-color:">signaling</span> pathways participate in the active principle region of Buyang Huanwu decoction-induced differentiation of bone marrow mesenchymal stem cells☆. </a></p><div class="supp"><p class="details"><div>Reverse transcription-PCR electropherogram of nestin mRNA expression after the mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> was blocked.<br />1: apr-BYHWD group; 2: apr-BYHWD + SB203580 group; 3: apr-BYHWD + PD98059 group; M: Marker; BYHWD: <i>Buyang Huanwu</i> decoction; apr-BYHWD: active principle region of BYHWD; PD98059 and SB203580: cellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocking agents.</div></p></div><div class="aux"><div class="resc">Jinghui Zheng, et al. Neural Regen Res. 2012 June 25;7(18):1370-1377.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3328&amp;ncbi_uid=7127271&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4308786/" ref="ordinalpos=3328&amp;ncbi_uid=7127271&amp;link_uid=PMC4308786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3329.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3627353/figure/F2/" ref="ordinalpos=3329&amp;ncbi_uid=6084058&amp;link_uid=PMC3627353" href="/pmc/articles/PMC3627353/"><img src="/pmc/articles/instance/3627353/bin/nihms434196f2.gif" src-large="/pmc/articles/instance/3627353/bin/nihms434196f2.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3627353/" ref="ordinalpos=3329&amp;ncbi_uid=6084058&amp;link_uid=PMC3627353" image-link="/pmc/articles/PMC3627353/figure/F2/" class="imagepopup">FIGURE 2.  From: Lower urinary tract development and disease. </a></p><div class="supp"><p class="details"><div>Genes and <span class="highlight" style="background-color:">signaling</span> pathways involved in ureteric budding. Scheme of <span class="highlight" style="background-color:">signaling</span> pathways between the ureteric buds (green) and the metanephric mesenchyme (yellow-orange). The most important inducer of UB outgrowth is the Receptor Tyrosine Kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by Gdnf and its receptor Ret. Ret is expressed by the nephric duct (green) and ureteric buds (green). Gdnf is secreted by the metanephric mesenchyme (yellow). The coordination of different <span class="highlight" style="background-color:">signaling</span> pathways in the anterior mesenchyme and the metanephric mesenchyme play a crucial role in the development of a single ureteric bud.</div></p></div><div class="aux"><div class="resc">Hila Milo Rasouly, et al. Wiley Interdiscip Rev Syst Biol Med. 2013 May;5(3):307-342.</div><p class="links"><a class="dblinks" href="/pubmed/23408557" ref="ordinalpos=3329&amp;ncbi_uid=6084058&amp;link_uid=23408557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3627353/" ref="ordinalpos=3329&amp;ncbi_uid=6084058&amp;link_uid=PMC3627353">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3330.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872881/figure/F1/" ref="ordinalpos=3330&amp;ncbi_uid=2374753&amp;link_uid=PMC2872881" href="/pmc/articles/PMC2872881/"><img src="/pmc/articles/instance/2872881/bin/gb-2010-11-2-r21-1.gif" src-large="/pmc/articles/instance/2872881/bin/gb-2010-11-2-r21-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872881/" ref="ordinalpos=3330&amp;ncbi_uid=2374753&amp;link_uid=PMC2872881" image-link="/pmc/articles/PMC2872881/figure/F1/" class="imagepopup">Figure 1.  From: Immunity and other defenses in pea aphids, Acyrthosiphon pisum. </a></p><div class="supp"><p class="details"><div><b>Some key insect recognition, <span class="highlight" style="background-color:">signaling</span> and response genes are missing in the pea aphid</b>. Previously sequenced genomes of other insects (flies, mosquitoes, bees, beetles) have indicated that immune <span class="highlight" style="background-color:">signaling</span> pathways, seen here, are conserved across insects. In aphids, missing IMD <span class="highlight" style="background-color:">pathway</span> members (dashed lines) include those involved in recognition (PGRPs) and <span class="highlight" style="background-color:">signaling</span> (IMD, dFADD, Dredd, REL). Genes encoding antimicrobial peptides common in other insects, including defensins and cecropins, are also missing. In contrast, we found putative homologs for most genes central to the Toll, JNK and JAK/STAT <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Nicole M Gerardo, et al. Genome Biol. 2010;11(2):R21-R21.</div><p class="links"><a class="dblinks" href="/pubmed/20178569" ref="ordinalpos=3330&amp;ncbi_uid=2374753&amp;link_uid=20178569">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872881/" ref="ordinalpos=3330&amp;ncbi_uid=2374753&amp;link_uid=PMC2872881">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3331.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4403800/figure/F3/" ref="ordinalpos=3331&amp;ncbi_uid=7434151&amp;link_uid=PMC4403800" href="/pmc/articles/PMC4403800/"><img src="/pmc/articles/instance/4403800/bin/nihms212162f3.gif" src-large="/pmc/articles/instance/4403800/bin/nihms212162f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4403800/" ref="ordinalpos=3331&amp;ncbi_uid=7434151&amp;link_uid=PMC4403800" image-link="/pmc/articles/PMC4403800/figure/F3/" class="imagepopup">Figure 3.  From: A mathematical model of intercellular <span class="highlight" style="background-color:">signaling</span> during epithelial wound healing. </a></p><div class="supp"><p class="details"><div>Schematic of simplified <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during wound healing: ligands (<i>L</i>) and ROS (<i>S</i>) are free to diffuse and bind to cell surface receptors. This binding forms complexes (<i>C</i><sub>S</sub> and <i>C</i><sub>L</sub>) that activate MAPK <span class="highlight" style="background-color:">signaling</span>. Complexes are lost due to endocytosis or, in the case of ligand-receptor complexes, through ligand unbinding and endocytosis. Intracellular proteases (<i>P</i>) release extracellular ligands (<i>L</i>). Note that receptor are assumed to be in excess. Conversely, ligand-ROS-receptor complexes are negligible.</div></p></div><div class="aux"><div class="resc">Filippo Posta, et al. J Theor Biol. ;266(1):70-78.</div><p class="links"><a class="dblinks" href="/pubmed/20685318" ref="ordinalpos=3331&amp;ncbi_uid=7434151&amp;link_uid=20685318">Citation</a><a class="dblinks" href="/pmc/articles/PMC4403800/" ref="ordinalpos=3331&amp;ncbi_uid=7434151&amp;link_uid=PMC4403800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3332.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634257/figure/F5/" ref="ordinalpos=3332&amp;ncbi_uid=1709748&amp;link_uid=PMC2634257" href="/pmc/articles/PMC2634257/"><img src="/pmc/articles/instance/2634257/bin/org0402_0116_fig005.gif" src-large="/pmc/articles/instance/2634257/bin/org0402_0116_fig005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634257/" ref="ordinalpos=3332&amp;ncbi_uid=1709748&amp;link_uid=PMC2634257" image-link="/pmc/articles/PMC2634257/figure/F5/" class="imagepopup">Figure 5.  From: Wnt <span class="highlight" style="background-color:">signaling</span> in breast organogenesis. </a></p><div class="supp"><p class="details"><div>Proposed interactions between progesterone receptor (PR) and β-catenin <span class="highlight" style="background-color:">signaling</span> during alveologenesis. In the normal virgin mammary gland, PR induces a non-uniform <i>PR</i> expression pattern and competence to respond to β-catenin in alveolar progenitors (blue) during ductal maturation. During early pregnancy, PR-Wnt4 <span class="highlight" style="background-color:">signaling</span> induces expansion of ductal progenitors (yellow) to form side-branches through βcatenin-independent routes. PR and PRL (prolactin) cooperate to induce alveologenesis, a process that is β-catenin-dependent. Later in pregnancy, PR restrains and PRL promote alveolar differentiation. β-catenin is required for alveologenesis and may participate at multiple steps in the secretory differentiation <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Kata Boras-Granic, et al. Organogenesis. 2008 Apr-Jun;4(2):116-122.</div><p class="links"><a class="dblinks" href="/pubmed/19279723" ref="ordinalpos=3332&amp;ncbi_uid=1709748&amp;link_uid=19279723">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634257/" ref="ordinalpos=3332&amp;ncbi_uid=1709748&amp;link_uid=PMC2634257">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3333.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3129154/figure/F9/" ref="ordinalpos=3333&amp;ncbi_uid=3554793&amp;link_uid=PMC3129154" href="/pmc/articles/PMC3129154/"><img src="/pmc/articles/instance/3129154/bin/zbc0331169550009.gif" src-large="/pmc/articles/instance/3129154/bin/zbc0331169550009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3129154/" ref="ordinalpos=3333&amp;ncbi_uid=3554793&amp;link_uid=PMC3129154" image-link="/pmc/articles/PMC3129154/figure/F9/" class="imagepopup">FIGURE 9. From: Polyubiquitination Events Mediate Polymethylmethacrylate (PMMA) Particle Activation of NF-?B <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic model.</b> PMMA particles induce TRAF6-dependent <span class="highlight" style="background-color:">signaling</span> by undefined mechanism. Cellular levels of the E2 UBC13 increase coinciding with TRAF6 poly-ubiquitination. These events facilitate binding of NEMO and TAK1, and activate the latter. IKK2 is recruited to NEMO and activated through its phosphorylation (<i>p</i>) by TAK1 and in turn it phosphorylates IκB leading to activation of NF-κB. Knockdown of TRAF6 or deletion of TAK1 blunts NEMO ubiquitination and hinders activation of NF-κB <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yasuhiro Yamanaka, et al. J Biol Chem. 2011 July 8;286(27):23735-23741.</div><p class="links"><a class="dblinks" href="/pubmed/21566132" ref="ordinalpos=3333&amp;ncbi_uid=3554793&amp;link_uid=21566132">Citation</a><a class="dblinks" href="/pmc/articles/PMC3129154/" ref="ordinalpos=3333&amp;ncbi_uid=3554793&amp;link_uid=PMC3129154">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3334.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3226161/figure/F1/" ref="ordinalpos=3334&amp;ncbi_uid=3684398&amp;link_uid=PMC3226161" href="/pmc/articles/PMC3226161/"><img src="/pmc/articles/instance/3226161/bin/cc3747-1.gif" src-large="/pmc/articles/instance/3226161/bin/cc3747-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3226161/" ref="ordinalpos=3334&amp;ncbi_uid=3684398&amp;link_uid=PMC3226161" image-link="/pmc/articles/PMC3226161/figure/F1/" class="imagepopup">Figure 1.  From: New insights into the protein C <span class="highlight" style="background-color:">pathway</span>: potential implications for the biological activities of drotrecogin alfa (activated). </a></p><div class="supp"><p class="details"><div>The protein C <span class="highlight" style="background-color:">pathway</span>: modulation of thrombin generation and cell <span class="highlight" style="background-color:">signaling</span>. APC, activated protein C; EPCR, endothelial protein C receptor; PAI-1, plasminogen activator inhibitor 1; PAR, protease-activated receptor; Rac, ras-related protein; Rho, ras-homolog; S-1 P-1; sphingosine 1 phosphate 1-receptor.</div></p></div><div class="aux"><div class="resc">William L Macias, et al. Crit Care. 2005;9(Suppl 4):S38-S45.</div><p class="links"><a class="dblinks" href="/pubmed/16168074" ref="ordinalpos=3334&amp;ncbi_uid=3684398&amp;link_uid=16168074">Citation</a><a class="dblinks" href="/pmc/articles/PMC3226161/" ref="ordinalpos=3334&amp;ncbi_uid=3684398&amp;link_uid=PMC3226161">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3335.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3415470/figure/F6/" ref="ordinalpos=3335&amp;ncbi_uid=4506963&amp;link_uid=PMC3415470" href="/pmc/articles/PMC3415470/"><img src="/pmc/articles/instance/3415470/bin/nihms393627f6.gif" src-large="/pmc/articles/instance/3415470/bin/nihms393627f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3415470/" ref="ordinalpos=3335&amp;ncbi_uid=4506963&amp;link_uid=PMC3415470" image-link="/pmc/articles/PMC3415470/figure/F6/" class="imagepopup">Figure 6. Adaptive divergence of duplicated <span class="highlight" style="background-color:">signaling</span> pathways involves the elimination of cross-talk.  From: Adaptive mutations that prevent cross-talk enable the expansion of paralogous <span class="highlight" style="background-color:">signaling</span> protein families. </a></p><div class="supp"><p class="details"><div>Ovals represent the set of response regulators recognized by a histidine kinase, as determined by its specificity residues (see also Figure 5). The NtrB-NtrC <span class="highlight" style="background-color:">pathway</span> is shown duplicating to produce the paralogous system NtrY-NtrX. As these pathways diverged, the specificity of NtrY overlapped that of PhoR, necessitating a change in PhoR specificity to yield the derived state with insulated pathways.</div></p></div><div class="aux"><div class="resc">Emily J. Capra, et al. Cell. ;150(1):222-232.</div><p class="links"><a class="dblinks" href="/pubmed/22770222" ref="ordinalpos=3335&amp;ncbi_uid=4506963&amp;link_uid=22770222">Citation</a><a class="dblinks" href="/pmc/articles/PMC3415470/" ref="ordinalpos=3335&amp;ncbi_uid=4506963&amp;link_uid=PMC3415470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3336.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3094802/figure/F3/" ref="ordinalpos=3336&amp;ncbi_uid=2954009&amp;link_uid=PMC3094802" href="/pmc/articles/PMC3094802/"><img src="/pmc/articles/instance/3094802/bin/nihms290166f3.gif" src-large="/pmc/articles/instance/3094802/bin/nihms290166f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3094802/" ref="ordinalpos=3336&amp;ncbi_uid=2954009&amp;link_uid=PMC3094802" image-link="/pmc/articles/PMC3094802/figure/F3/" class="imagepopup">Fig. 3.  From: Notch <span class="highlight" style="background-color:">Signaling</span> Proteins: Legitimate Targets for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>Diagram of putative therapeutic target in the Notch <span class="highlight" style="background-color:">pathway</span>. Notch <span class="highlight" style="background-color:">signaling</span> could be inhibited theoretically at many different levels. It is possible to (1) interfere with Notch-ligand interactions, (2) inhibit receptor activation, (3) promote Notch ubiquitination and degradation, (4) inhibit its translocation to the nuclear compartment and (5) inhibit NICD nuclear complex formation.</div></p></div><div class="aux"><div class="resc">Zhiwei Wang, et al. Curr Protein Pept Sci. ;11(6):398-408.</div><p class="links"><a class="dblinks" href="/pubmed/20491628" ref="ordinalpos=3336&amp;ncbi_uid=2954009&amp;link_uid=20491628">Citation</a><a class="dblinks" href="/pmc/articles/PMC3094802/" ref="ordinalpos=3336&amp;ncbi_uid=2954009&amp;link_uid=PMC3094802">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3337.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2746254/figure/F1/" ref="ordinalpos=3337&amp;ncbi_uid=2476519&amp;link_uid=PMC2746254" href="/pmc/articles/PMC2746254/"><img src="/pmc/articles/instance/2746254/bin/nihms141503f1.gif" src-large="/pmc/articles/instance/2746254/bin/nihms141503f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2746254/" ref="ordinalpos=3337&amp;ncbi_uid=2476519&amp;link_uid=PMC2746254" image-link="/pmc/articles/PMC2746254/figure/F1/" class="imagepopup">Figure 1.  From: Beyond Thermodynamics: Drug Binding Kinetics Could Influence Epidermal Growth Factor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> model for studying the influence of drug-binding kinetics on EGF <span class="highlight" style="background-color:">signaling</span> in non-small cell lung cancer. A:B represents a complex between A and B. A-P denotes the phosphorylated form of A and A-PP represents the doubly phosphorylated form of A. A* indicates the activated form of A. (Note: The single-direction arrows do not imply all reactions are unidirectional – some represent reversible reactions.)</div></p></div><div class="aux"><div class="resc">Mayank Goyal, et al. J Med Chem. ;52(18):5582-5585.</div><p class="links"><a class="dblinks" href="/pubmed/19702309" ref="ordinalpos=3337&amp;ncbi_uid=2476519&amp;link_uid=19702309">Citation</a><a class="dblinks" href="/pmc/articles/PMC2746254/" ref="ordinalpos=3337&amp;ncbi_uid=2476519&amp;link_uid=PMC2746254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3338.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3880582/figure/F4/" ref="ordinalpos=3338&amp;ncbi_uid=5679281&amp;link_uid=PMC3880582" href="/pmc/articles/PMC3880582/"><img src="/pmc/articles/instance/3880582/bin/nihms537624f4.gif" src-large="/pmc/articles/instance/3880582/bin/nihms537624f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3880582/" ref="ordinalpos=3338&amp;ncbi_uid=5679281&amp;link_uid=PMC3880582" image-link="/pmc/articles/PMC3880582/figure/F4/" class="imagepopup">Figure 4.  From: Hypoxia-induced phrenic long-term facilitation: emergent properties. </a></p><div class="supp"><p class="details"><div>Possible mechanisms of metaplasticity, enhancing pLTF following repettive intermittent hypoxia. <b>A:</b> Decreased S <span class="highlight" style="background-color:">pathway</span> activation, thereby eliminating S to Q inhibition, could explain metaplasticity in pLTF following rAIH. <b>B:</b> Reduced cross-talk inhibition, enabling a positive S <span class="highlight" style="background-color:">pathway</span> contribution could enhance pLTF. <b>C:</b> Increased Q <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> (“hypertrophy”), possibly involving increased 5-HT<sub>2</sub> receptors, ROS production, or BDNF synthesis could explain greater pLTF following rAIH.</div></p></div><div class="aux"><div class="resc">Michael J. Devinney, et al. Ann N Y Acad Sci. ;1279:10.1111/nyas.12085.</div><p class="links"><a class="dblinks" href="/pubmed/23531012" ref="ordinalpos=3338&amp;ncbi_uid=5679281&amp;link_uid=23531012">Citation</a><a class="dblinks" href="/pmc/articles/PMC3880582/" ref="ordinalpos=3338&amp;ncbi_uid=5679281&amp;link_uid=PMC3880582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3339.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC171388/figure/F7/" ref="ordinalpos=3339&amp;ncbi_uid=201755&amp;link_uid=PMC171388" href="/pmc/articles/PMC171388/"><img src="/pmc/articles/instance/171388/bin/JCI0317788.f7.gif" src-large="/pmc/articles/instance/171388/bin/JCI0317788.f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC171388/" ref="ordinalpos=3339&amp;ncbi_uid=201755&amp;link_uid=PMC171388" image-link="/pmc/articles/PMC171388/figure/F7/" class="imagepopup">Figure 7.  From: Antibody blockade of the Cripto CFC domain suppresses tumor cell growth in vivo. </a></p><div class="supp"><p class="details"><div>Model of Cripto’s inhibition of Act B <span class="highlight" style="background-color:">signaling</span> through the Smad2 <span class="highlight" style="background-color:">pathway</span>. Left: Activin (black) <span class="highlight" style="background-color:">signaling</span> via Alk4 (red), ActRII (blue), and Smad2 leads to regulated cell proliferation in many tissues and organs. Right: Cripto-dependent Nodal (yellow) <span class="highlight" style="background-color:">signaling</span> through Smad2 leads to differentiation and patterning in the early embryo. Cripto is represented in green. Center: Cripto inhibits Act B <span class="highlight" style="background-color:">signaling</span> via binding to Alk4 and Act B, resulting in deregulated growth and hyperproliferation. Three possible inhibitory complexes are proposed: Cripto/Act B, Cripto/Alk4, and Cripto/Act B/Alk4. Smad2-p, phospho-Smad2.</div></p></div><div class="aux"><div class="resc">Heather B. Adkins, et al. J Clin Invest. 2003 August 15;112(4):575-587.</div><p class="links"><a class="dblinks" href="/pubmed/12925698" ref="ordinalpos=3339&amp;ncbi_uid=201755&amp;link_uid=12925698">Citation</a><a class="dblinks" href="/pmc/articles/PMC171388/" ref="ordinalpos=3339&amp;ncbi_uid=201755&amp;link_uid=PMC171388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3340.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399380/figure/fig2/" ref="ordinalpos=3340&amp;ncbi_uid=4134290&amp;link_uid=PMC3399380" href="/pmc/articles/PMC3399380/"><img src="/pmc/articles/instance/3399380/bin/CHRP2012-648965.002.gif" src-large="/pmc/articles/instance/3399380/bin/CHRP2012-648965.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399380/" ref="ordinalpos=3340&amp;ncbi_uid=4134290&amp;link_uid=PMC3399380" image-link="/pmc/articles/PMC3399380/figure/fig2/" class="imagepopup">Figure 2.  From: Deciphering the Role of Insulin-Like Growth Factor-I Receptor in Trastuzumab Resistance. </a></p><div class="supp"><p class="details"><div>IGF-IR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Increased expression of IGF-IR, or IGF-IR interaction and crosstalk with HER2 have been reported in models of Herceptin resistance. PI3K <span class="highlight" style="background-color:">signaling</span> has been implicated as a potential mechanism of IGF-IR-mediated Herceptin resistance. Inhibition of IGF-IR has been shown to increase sensitivity to Herceptin in preclinical studies.</div></p></div><div class="aux"><div class="resc">Rita Nahta. Chemother Res Pract. 2012;2012:648965.</div><p class="links"><a class="dblinks" href="/pubmed/22830017" ref="ordinalpos=3340&amp;ncbi_uid=4134290&amp;link_uid=22830017">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399380/" ref="ordinalpos=3340&amp;ncbi_uid=4134290&amp;link_uid=PMC3399380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3341.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3956809/figure/f4-grsb-8-2014-075/" ref="ordinalpos=3341&amp;ncbi_uid=5930771&amp;link_uid=PMC3956809" href="/pmc/articles/PMC3956809/"><img src="/pmc/articles/instance/3956809/bin/grsb-8-2014-075f4.gif" src-large="/pmc/articles/instance/3956809/bin/grsb-8-2014-075f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3956809/" ref="ordinalpos=3341&amp;ncbi_uid=5930771&amp;link_uid=PMC3956809" image-link="/pmc/articles/PMC3956809/figure/f4-grsb-8-2014-075/" class="imagepopup">Figure 4.  From: Tissue-Specific Gene Expression and Regulation in Liver and Muscle Following Chronic Corticosteroid Administration. </a></p><div class="supp"><p class="details"><div>Tissue-specific expression within individual functions—a case of insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The left subfigure shows an abstract of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> where expressed genes in liver and muscle and their corresponding downstream affected functions are included. The right panel displays the tissue-specific regulation of genes within the same functional category; putative TFs are those significantly overrepresented on the promoters of corresponding genes.</div></p></div><div class="aux"><div class="resc">Tung T Nguyen, et al. Gene Regul Syst Bio. 2014;8:75-87.</div><p class="links"><a class="dblinks" href="/pubmed/24653645" ref="ordinalpos=3341&amp;ncbi_uid=5930771&amp;link_uid=24653645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3956809/" ref="ordinalpos=3341&amp;ncbi_uid=5930771&amp;link_uid=PMC3956809">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3342.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012926/figure/fig1/" ref="ordinalpos=3342&amp;ncbi_uid=6628941&amp;link_uid=PMC4012926" href="/pmc/articles/PMC4012926/"><img src="/pmc/articles/instance/4012926/bin/cmi201139f1.gif" src-large="/pmc/articles/instance/4012926/bin/cmi201139f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012926/" ref="ordinalpos=3342&amp;ncbi_uid=6628941&amp;link_uid=PMC4012926" image-link="/pmc/articles/PMC4012926/figure/fig1/" class="imagepopup">Figure 1.  From: SHP gains citizenship of the AMPK kingdom. </a></p><div class="supp"><p class="details"><div>Upstream and downstream AMPK <span class="highlight" style="background-color:">signaling</span> pathways that negatively regulate NF-κB activation and suppress inflammatory responses. AMPK is activated by several hormones, many phytochemicals, certain drugs (for example, metformin) and physical exercise. Obesity and hyperglycemia reduce the expression of AMPK. But Yuk <i>et al.</i><sup>3</sup> showed that LPS, which strongly activates NF-κB <i>via</i> TLR <span class="highlight" style="background-color:">signaling</span>, induced SHP transcription <i>via</i> a CaMKKβ–AMPK–USP1-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and that newly produced SHP inhibited NF-κB in a negative feedback loop. MSP mimicked LPS in that it induced the expression of SHP <i>via</i> an LKB1–AMPK–USP1-dependent <span class="highlight" style="background-color:">pathway</span>, attenuating LPS-induced NF-κB activation. Once AMPK has been activated, it stimulates SIRT1, PGC1α, FoxO and p53, which could attenuate NF-κB activation. Liu <i>et al</i>.<sup>4</sup> showed that AMPK, <i>via</i> the Nrf2/ARE <span class="highlight" style="background-color:">pathway</span>, induced the HO-1/CO system, which reduced cellular stress and inhibited TLR <span class="highlight" style="background-color:">signaling</span> pathways, resulting in the suppression of NF-κB activation. AMPK can ameliorate inflammatory responses <i>via</i> various direct and indirect anti-inflammatory <span class="highlight" style="background-color:">signaling</span> pathways. ↓, activation; ⊥, inhibition. AMPK, AMP-activated kinase; ARE, antioxidant responsive element; CaMKKβ, Ca<sup>2+</sup>/calmodulin-dependent protein kinase kinase-β HO-1/CO, heme oxygenase-1/carbon monoxide; LPS, lipopolysaccharide; MSP, macrophage-stimulating protein; SHP, small heterodimer partner; TLR, Toll-like receptor.</div></p></div><div class="aux"><div class="resc">Hun-Taeg Chung. Cell Mol Immunol. 2011 November;8(6):450-452.</div><p class="links"><a class="dblinks" href="/pubmed/21927016" ref="ordinalpos=3342&amp;ncbi_uid=6628941&amp;link_uid=21927016">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012926/" ref="ordinalpos=3342&amp;ncbi_uid=6628941&amp;link_uid=PMC4012926">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3343.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3788593/figure/F2/" ref="ordinalpos=3343&amp;ncbi_uid=5666573&amp;link_uid=PMC3788593" href="/pmc/articles/PMC3788593/"><img src="/pmc/articles/instance/3788593/bin/nihms-501495-f0002.gif" src-large="/pmc/articles/instance/3788593/bin/nihms-501495-f0002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3788593/" ref="ordinalpos=3343&amp;ncbi_uid=5666573&amp;link_uid=PMC3788593" image-link="/pmc/articles/PMC3788593/figure/F2/" class="imagepopup">FIGURE 2.  From: Androgen Receptor Antagonists in Castration-Resistant Prostate Cancer. </a></p><div class="supp"><p class="details"><div>Therapeutic targets of the androgen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Dana Rathkopf, et al. Cancer J. 2013 Jan-Feb;19(1):43-49.</div><p class="links"><a class="dblinks" href="/pubmed/23337756" ref="ordinalpos=3343&amp;ncbi_uid=5666573&amp;link_uid=23337756">Citation</a><a class="dblinks" href="/pmc/articles/PMC3788593/" ref="ordinalpos=3343&amp;ncbi_uid=5666573&amp;link_uid=PMC3788593">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3344.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3372922/figure/F2/" ref="ordinalpos=3344&amp;ncbi_uid=4055399&amp;link_uid=PMC3372922" href="/pmc/articles/PMC3372922/"><img src="/pmc/articles/instance/3372922/bin/nihms381555f2.gif" src-large="/pmc/articles/instance/3372922/bin/nihms381555f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3372922/" ref="ordinalpos=3344&amp;ncbi_uid=4055399&amp;link_uid=PMC3372922" image-link="/pmc/articles/PMC3372922/figure/F2/" class="imagepopup">Fig. 2.  From: Estrogen-TGF? Cross-Talk in Bone and Other Cell Types: Role of TIEG, Runx2, and Other Transcription Factors. </a></p><div class="supp"><p class="details"><div>Mechanisms of action of TIEG in the TGFβ-Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">J.R. Hawse, et al. J Cell Biochem. ;103(2):383-392.</div><p class="links"><a class="dblinks" href="/pubmed/17541956" ref="ordinalpos=3344&amp;ncbi_uid=4055399&amp;link_uid=17541956">Citation</a><a class="dblinks" href="/pmc/articles/PMC3372922/" ref="ordinalpos=3344&amp;ncbi_uid=4055399&amp;link_uid=PMC3372922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3345.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1877048/figure/f1-22/" ref="ordinalpos=3345&amp;ncbi_uid=936672&amp;link_uid=PMC1877048" href="/pmc/articles/PMC1877048/"><img src="/pmc/articles/instance/1877048/bin/22FF1.gif" src-large="/pmc/articles/instance/1877048/bin/22FF1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1877048/" ref="ordinalpos=3345&amp;ncbi_uid=936672&amp;link_uid=PMC1877048" image-link="/pmc/articles/PMC1877048/figure/f1-22/" class="imagepopup">Figure. From: Nutritional Route Affects ERK Phosphorylation and Cytokine Production in Hepatic Mononuclear Cells. </a></p><div class="supp"><p class="details"><div><b>FIGURE 1.</b> A <span class="highlight" style="background-color:">pathway</span> of lipopolysaccharide (LPS) <span class="highlight" style="background-color:">signaling</span> in Kupffer cells.</div></p></div><div class="aux"><div class="resc">Tomoyuki Moriya, et al. Ann Surg. 2007 April;245(4):642-650.</div><p class="links"><a class="dblinks" href="/pubmed/17414615" ref="ordinalpos=3345&amp;ncbi_uid=936672&amp;link_uid=17414615">Citation</a><a class="dblinks" href="/pmc/articles/PMC1877048/" ref="ordinalpos=3345&amp;ncbi_uid=936672&amp;link_uid=PMC1877048">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3346.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3304298/figure/F1/" ref="ordinalpos=3346&amp;ncbi_uid=3865099&amp;link_uid=PMC3304298" href="/pmc/articles/PMC3304298/"><img src="/pmc/articles/instance/3304298/bin/fnmol-05-00033-g0001.gif" src-large="/pmc/articles/instance/3304298/bin/fnmol-05-00033-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3304298/" ref="ordinalpos=3346&amp;ncbi_uid=3865099&amp;link_uid=PMC3304298" image-link="/pmc/articles/PMC3304298/figure/F1/" class="imagepopup">Figure 1.  From: AKT/GSK3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and schizophrenia. </a></p><div class="supp"><p class="details"><div><b>The AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> AKT is activated by PI3K, which itself is activated by several upstream <span class="highlight" style="background-color:">signaling</span> pathways such as insulin receptors, receptor tyrosine kinases, G protein coupled receptors, cytokine receptors, etc. After activation, it targets several downstream molecules and change their activity by phosphorylation or complex formation. AKT is involved in cell proliferation, glucose metabolism, cell survival, cell cycle, protein synthesis, and in neuronal morphology and plasticity by regulation of several downstream molecules shown in this figure. AKT is involved in cell proliferation through interaction with a number of proteins involved in cell cycle, including Cyclin D1, p21Cip, p27Kip, Myt1, and Wee1. AKT also plays a crucial role in cell survival through interaction with Bad, MDM2 and the subsequent regulation of p53, FoxO1, Bcl-2, and Bax. AKT regulates glucose metabolism through regulation of GSK-3, PFKFB2, PIP5K, and AS160. It could also affect protein synthesis through interaction with S6 kinase, TSC1, and mTOR <span class="highlight" style="background-color:">signaling</span>. In more recent years, AKT's role in neurodegeneration was uncovered, mediated through phosphorylation of Huntingtin and Ataxin-1. AKT also has a role in synaptic plasticity, by phosphorylation of GABA<sub>A</sub> receptor. <span class="highlight" style="background-color:">Pathway</span> diagram reproduced courtesy of Cell <span class="highlight" style="background-color:">Signaling</span> Technology, Inc. (www.cellsignal.com).</div></p></div><div class="aux"><div class="resc">Effat S. Emamian. Front Mol Neurosci. 2012;5:33.</div><p class="links"><a class="dblinks" href="/pubmed/22435049" ref="ordinalpos=3346&amp;ncbi_uid=3865099&amp;link_uid=22435049">Citation</a><a class="dblinks" href="/pmc/articles/PMC3304298/" ref="ordinalpos=3346&amp;ncbi_uid=3865099&amp;link_uid=PMC3304298">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3347.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2846172/figure/F5/" ref="ordinalpos=3347&amp;ncbi_uid=2294775&amp;link_uid=PMC2846172" href="/pmc/articles/PMC2846172/"><img src="/pmc/articles/instance/2846172/bin/nihms160559f5.gif" src-large="/pmc/articles/instance/2846172/bin/nihms160559f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2846172/" ref="ordinalpos=3347&amp;ncbi_uid=2294775&amp;link_uid=PMC2846172" image-link="/pmc/articles/PMC2846172/figure/F5/" class="imagepopup">Figure 5.  From: Mechanisms of Disease: epithelial-mesenchymal transition--does cellular plasticity fuel neoplastic progression?. </a></p><div class="supp"><p class="details"><div>Microenvironmental and spatial regulation of <span class="highlight" style="background-color:">signaling</span> pathways controlling EMT. The RAS–ERK1/ERK2 <span class="highlight" style="background-color:">pathway</span> is an example of a <span class="highlight" style="background-color:">signaling</span> module that is responsive to microenvironmental cues and requires specific subcellular localization to determine the consequences of gene expression and tumor phenotype. A simplified version of this complex process is illustrated in the diagram. ECM components interact with integrin receptors at the cell surface (step 1) and the affinity of this interaction is modified by growth-factor-regulated <span class="highlight" style="background-color:">signaling</span>, in a process known as ‘inside-out <span class="highlight" style="background-color:">signaling</span>’ (steps 1–3). Conversely, the affinity of the integrin–ECM interaction affects growth-factor-regulated <span class="highlight" style="background-color:">signaling</span>, in a process known as ‘outside-in <span class="highlight" style="background-color:">signaling</span>’ (steps 2–3). The collective interactions between integrins and growth factor receptors promote the localization and activation of lipid-modified RAS (step 2) at the inner cell membrane leaflet (step 4). Activated RAS then selectively activates kinases, such as ERK1/ERK2 and other pathways, such as the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. RAS also blocks the tumor-suppressing activity of TGF-β1 by linking this <span class="highlight" style="background-color:">pathway</span> to ERK1/ERK2 <span class="highlight" style="background-color:">signaling</span> pathways (step 5). This linkage promotes the proinvasion properties of TGF-β1. Activated ERK1/ERK2 must translocate to the nucleus or cell-adhesion sites (known as ‘focal contacts’) to have access to target proteins that regulate EMT, motility and/or invasion. ERK1-regulated and ERK2-regulated gene expression (e.g. MMP9) further modifies the microenvironment of the tumor cell, thereby affecting the activation status of the integrin–growth factor receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This downstream consequence of RAS–ERK1–ERK2 activation and the reversal of steps 1–5 have profound effects on the phenotype even if other pathways are mutated. Abbreviations: AKT, protein kinase B; ECM, extracellular matrix; EMT, epithelial–mesenchymal transition; ERK1, extracellular signal-regulated kinase 1; ERK2, extracellular signal-regulated kinase 2; MMP, matrix metalloproteinase; PI3K, phosphatidylinoinositol 3-kinase; TGF, transforming growth factor.</div></p></div><div class="aux"><div class="resc">Eva A Turley, et al. Nat Clin Pract Oncol. ;5(5):280-290.</div><p class="links"><a class="dblinks" href="/pubmed/18349857" ref="ordinalpos=3347&amp;ncbi_uid=2294775&amp;link_uid=18349857">Citation</a><a class="dblinks" href="/pmc/articles/PMC2846172/" ref="ordinalpos=3347&amp;ncbi_uid=2294775&amp;link_uid=PMC2846172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3348.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697172/figure/F7/" ref="ordinalpos=3348&amp;ncbi_uid=1831248&amp;link_uid=PMC2697172" href="/pmc/articles/PMC2697172/"><img src="/pmc/articles/instance/2697172/bin/1471-2164-10-220-7.gif" src-large="/pmc/articles/instance/2697172/bin/1471-2164-10-220-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697172/" ref="ordinalpos=3348&amp;ncbi_uid=1831248&amp;link_uid=PMC2697172" image-link="/pmc/articles/PMC2697172/figure/F7/" class="imagepopup">Figure 7.  From: A network-based integrative approach to prioritize reliable hits from multiple genome-wide RNAi screens in Drosophila. </a></p><div class="supp"><p class="details"><div><b>Distributions of the mutant phenotype similarities between nonhits within each interval of the RR by NePhe score and known regulators (blue bars), between hits and known regulators (orange bar) and among known regulators themselves (red bar) for Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (a) and Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (b)</b>. Only genes with at least one allele phenotype are considered here.</div></p></div><div class="aux"><div class="resc">Li Wang, et al. BMC Genomics. 2009;10:220-220.</div><p class="links"><a class="dblinks" href="/pubmed/19435510" ref="ordinalpos=3348&amp;ncbi_uid=1831248&amp;link_uid=19435510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697172/" ref="ordinalpos=3348&amp;ncbi_uid=1831248&amp;link_uid=PMC2697172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3349.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4366044/figure/pone.0119765.g004/" ref="ordinalpos=3349&amp;ncbi_uid=7315503&amp;link_uid=PMC4366044" href="/pmc/articles/PMC4366044/"><img src="/pmc/articles/instance/4366044/bin/pone.0119765.g004.gif" src-large="/pmc/articles/instance/4366044/bin/pone.0119765.g004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4366044/" ref="ordinalpos=3349&amp;ncbi_uid=7315503&amp;link_uid=PMC4366044" image-link="/pmc/articles/PMC4366044/figure/pone.0119765.g004/" class="imagepopup">Fig 4. Leukocyte Extravasation <span class="highlight" style="background-color:">Signaling</span>: Transcriptomic signatures of kidney cancer showed a significant activation in leukocyte extravasation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Molecular Interaction of a Kinase Inhibitor Midostaurin with Anticancer Drug Targets, S100A8 and EGFR: Transcriptional Profiling and Molecular Docking Study for Kidney Cancer Therapeutics. </a></p><div class="supp"><p class="details"><div>Red represents overexpression and green underexpression.</div></p></div><div class="aux"><div class="resc">Zeenat Mirza, et al. PLoS One. 2015;10(3):e0119765.</div><p class="links"><a class="dblinks" href="/pubmed/25789858" ref="ordinalpos=3349&amp;ncbi_uid=7315503&amp;link_uid=25789858">Citation</a><a class="dblinks" href="/pmc/articles/PMC4366044/" ref="ordinalpos=3349&amp;ncbi_uid=7315503&amp;link_uid=PMC4366044">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3350.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2426945/figure/fig7/" ref="ordinalpos=3350&amp;ncbi_uid=1329582&amp;link_uid=PMC2426945" href="/pmc/articles/PMC2426945/"><img src="/pmc/articles/instance/2426945/bin/jcb1810973f07.gif" src-large="/pmc/articles/instance/2426945/bin/jcb1810973f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2426945/" ref="ordinalpos=3350&amp;ncbi_uid=1329582&amp;link_uid=PMC2426945" image-link="/pmc/articles/PMC2426945/figure/fig7/" class="imagepopup">Figure 7. From: Regulation of neural progenitor cell state by ephrin-B. </a></p><div class="supp"><p class="details"><div><b>A working model of the function of ephrin-B–RGS <span class="highlight" style="background-color:">pathway</span>.</b> Our results indicate that the RGS-mediated ephrin-B reverse <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is required for the maintenance of neural progenitor cells. As the RGS domain acts as an inhibitor of Gα subunit <span class="highlight" style="background-color:">signaling</span>, our data suggest that Gα subunit may function as an activator of progenitor differentiation. Future studies are required to investigate this potential role of Gα subunit. In addition, whether and how EphB functions in the activation of ephrin-B–RGS <span class="highlight" style="background-color:">pathway</span> in neural progenitor cells will also need to be further studied.</div></p></div><div class="aux"><div class="resc">Runxiang Qiu, et al. J Cell Biol. 2008 June 16;181(6):973-983.</div><p class="links"><a class="dblinks" href="/pubmed/18541704" ref="ordinalpos=3350&amp;ncbi_uid=1329582&amp;link_uid=18541704">Citation</a><a class="dblinks" href="/pmc/articles/PMC2426945/" ref="ordinalpos=3350&amp;ncbi_uid=1329582&amp;link_uid=PMC2426945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3351.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997674/figure/f2-ol-07-05-1327/" ref="ordinalpos=3351&amp;ncbi_uid=6053800&amp;link_uid=PMC3997674" href="/pmc/articles/PMC3997674/"><img src="/pmc/articles/instance/3997674/bin/OL-07-05-1327-g01.gif" src-large="/pmc/articles/instance/3997674/bin/OL-07-05-1327-g01.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997674/" ref="ordinalpos=3351&amp;ncbi_uid=6053800&amp;link_uid=PMC3997674" image-link="/pmc/articles/PMC3997674/figure/f2-ol-07-05-1327/" class="imagepopup">Figure 2.  From: TRAIL combinations: The new 'trail' for cancer therapy (Review). </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> of TRAIL/drug combinations. TRAIL, tumor necrosis factor-related apoptosis-inducing ligand.</div></p></div><div class="aux"><div class="resc">ALAA REFAAT, et al. Oncol Lett. 2014 May;7(5):1327-1332.</div><p class="links"><a class="dblinks" href="/pubmed/24765133" ref="ordinalpos=3351&amp;ncbi_uid=6053800&amp;link_uid=24765133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997674/" ref="ordinalpos=3351&amp;ncbi_uid=6053800&amp;link_uid=PMC3997674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3352.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2912609/figure/F2/" ref="ordinalpos=3352&amp;ncbi_uid=2798619&amp;link_uid=PMC2912609" href="/pmc/articles/PMC2912609/"><img src="/pmc/articles/instance/2912609/bin/nihms194918f2.gif" src-large="/pmc/articles/instance/2912609/bin/nihms194918f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2912609/" ref="ordinalpos=3352&amp;ncbi_uid=2798619&amp;link_uid=PMC2912609" image-link="/pmc/articles/PMC2912609/figure/F2/" class="imagepopup">Figure 2.  From: Targeted Therapies in Lung Cancer. </a></p><div class="supp"><p class="details"><div>Currently available therapies targeting EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lucian R. Chirieac, et al. Surg Pathol Clin. ;3(1):71-82.</div><p class="links"><a class="dblinks" href="/pubmed/20680095" ref="ordinalpos=3352&amp;ncbi_uid=2798619&amp;link_uid=20680095">Citation</a><a class="dblinks" href="/pmc/articles/PMC2912609/" ref="ordinalpos=3352&amp;ncbi_uid=2798619&amp;link_uid=PMC2912609">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3353.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2597811/figure/F2/" ref="ordinalpos=3353&amp;ncbi_uid=1551759&amp;link_uid=PMC2597811" href="/pmc/articles/PMC2597811/"><img src="/pmc/articles/instance/2597811/bin/nihms37181f2.gif" src-large="/pmc/articles/instance/2597811/bin/nihms37181f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2597811/" ref="ordinalpos=3353&amp;ncbi_uid=1551759&amp;link_uid=PMC2597811" image-link="/pmc/articles/PMC2597811/figure/F2/" class="imagepopup">Figure 2.  From: Molecular Genetics of Colorectal Cancer: An Overview. </a></p><div class="supp"><p class="details"><div>The Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Readers are referred to the text for a detailed description of the <span class="highlight" style="background-color:">pathway</span>. βCat—β-catenin; CtBP—C-terminal binding protein; Dsh—disheveled; Fz—frizzled; Gro—Groucho; GSK—glycogen synthase kinase 3β; LRP—low-density lipoprotein receptor-related protein; TCF—T-cell factor.</div></p></div><div class="aux"><div class="resc">Irfan M. Hisamuddin, et al. Curr Colorectal Cancer Rep. ;2(2):53-59.</div><p class="links"><a class="dblinks" href="/pubmed/19079560" ref="ordinalpos=3353&amp;ncbi_uid=1551759&amp;link_uid=19079560">Citation</a><a class="dblinks" href="/pmc/articles/PMC2597811/" ref="ordinalpos=3353&amp;ncbi_uid=1551759&amp;link_uid=PMC2597811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3354.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2574929/figure/F6/" ref="ordinalpos=3354&amp;ncbi_uid=1514395&amp;link_uid=PMC2574929" href="/pmc/articles/PMC2574929/"><img src="/pmc/articles/instance/2574929/bin/nihms-64289-f0006.gif" src-large="/pmc/articles/instance/2574929/bin/nihms-64289-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2574929/" ref="ordinalpos=3354&amp;ncbi_uid=1514395&amp;link_uid=PMC2574929" image-link="/pmc/articles/PMC2574929/figure/F6/" class="imagepopup">Figure 6. Selectivity of cisplatin and actinomycin D effects on the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Cisplatin-mediated activation of extracellular signal-regulated kinases 1/2 (ERK1/2) by inhibition of ERK1/2 phosphatases. </a></p><div class="supp"><p class="details"><div><i><b>A</b></i>, Neurons were treated as indicated. Activities of the ERK1/2- or the JNK <span class="highlight" style="background-color:">signaling</span> pathways were determined by western blotting with pThr183/pTyr185 ERK1/2- or pThr183/pTyr185 JNK-specific antibodies (pERK1/2 or pJNK, respectively). The membranes were re-probed to monitor the levels of total ERK1/2 or JNK. In addition, GAPDH levels were determined to provide additional loading control. <i><b>B, C</b></i>, Relative levels of the phospho-MAP-kinases (pMAPK) were determined as fold controls of the pERK/ERK- or pJNK/JNK ratios, respectively. In CPDD-, or ActD-treated neurons, the ERK1/2- but not the JNK <span class="highlight" style="background-color:">pathway</span> is activated. In both cases, the ERK1/2 activation was delayed and long lasting suggesting deficient inactivation of ERK1/2. The graphs depict means of 4 independent experiments ±SEM; *, p&lt;0.05; **, p&lt;0.01.</div></p></div><div class="aux"><div class="resc">Agata Gozdz, et al. J Neurochem. ;106(5):2056-2067.</div><p class="links"><a class="dblinks" href="/pubmed/18665890" ref="ordinalpos=3354&amp;ncbi_uid=1514395&amp;link_uid=18665890">Citation</a><a class="dblinks" href="/pmc/articles/PMC2574929/" ref="ordinalpos=3354&amp;ncbi_uid=1514395&amp;link_uid=PMC2574929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3355.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2080907/figure/fig8/" ref="ordinalpos=3355&amp;ncbi_uid=1083114&amp;link_uid=PMC2080907" href="/pmc/articles/PMC2080907/"><img src="/pmc/articles/instance/2080907/bin/jcb1790717f08.gif" src-large="/pmc/articles/instance/2080907/bin/jcb1790717f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2080907/" ref="ordinalpos=3355&amp;ncbi_uid=1083114&amp;link_uid=PMC2080907" image-link="/pmc/articles/PMC2080907/figure/fig8/" class="imagepopup">Figure 8. From: Myc stimulates B lymphocyte differentiation and amplifies calcium <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model of Myc and Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> events downstream of the pre-BCR/BCR.</b> Engagement of the pre-BCR/BCR results in the recruitment of Src family kinases (Blk, Lyn, and Fyn), phosphorylation of the Igα–Igβ complex, and formation of a Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> module containing SLP-65, PI3K, Btk/Tec, and PLCγ. Full activation of PLCγ by Btk results in IP<sub>3</sub>-mediated Ca<sup>2+</sup> release from the ER, resulting in activation of the Cn–NFAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PKCβ activation by Ca<sup>2+</sup> and DAG leads to the nuclear translocation of NF-κB and the induction of <i>myc</i> expression. Our results implicate Myc in amplifying [Ca<sup>2+</sup>]<sub>i</sub> by decreasing PMCA4-mediated Ca<sup>2+</sup> efflux, thereby promoting optimal NFAT-dependent proliferation, differentiation, and immunogenic B cell responses. Induction of low level Ca<sup>2+</sup> signals or Myc <span class="highlight" style="background-color:">signaling</span> alone results in a tolerogenic and/or apoptotic response, whereas Myc activation together with sustained Ca<sup>2+</sup>/NFAT signals or constitutive Myc <span class="highlight" style="background-color:">signaling</span> alone promotes B cell activation/differentiation or tumorigenesis.</div></p></div><div class="aux"><div class="resc">Tania Habib, et al. J Cell Biol. 2007 November 19;179(4):717-731.</div><p class="links"><a class="dblinks" href="/pubmed/17998397" ref="ordinalpos=3355&amp;ncbi_uid=1083114&amp;link_uid=17998397">Citation</a><a class="dblinks" href="/pmc/articles/PMC2080907/" ref="ordinalpos=3355&amp;ncbi_uid=1083114&amp;link_uid=PMC2080907">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3356.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136221/figure/fig1/" ref="ordinalpos=3356&amp;ncbi_uid=3087618&amp;link_uid=PMC3136221" href="/pmc/articles/PMC3136221/"><img src="/pmc/articles/instance/3136221/bin/JNUME2011-708183.001.gif" src-large="/pmc/articles/instance/3136221/bin/JNUME2011-708183.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136221/" ref="ordinalpos=3356&amp;ncbi_uid=3087618&amp;link_uid=PMC3136221" image-link="/pmc/articles/PMC3136221/figure/fig1/" class="imagepopup">Figure 1.  From: Cancer Risk in Type 2 Diabetes Mellitus: Metabolic Links and Therapeutic Considerations. </a></p><div class="supp"><p class="details"><div>The “big picture” between DM2 and cancer. (a) The shared metabolic factors underlying both DM2 and cancer, including visceral adiposity, inflammation, hyperglycemia, and hyperinsulinemia lead to (b) increased insulin receptor substrate (IRS) stimulating the phosphorylation of Ras <span class="highlight" style="background-color:">signaling</span> proteins and potentially increasing tumor cell growth and proliferation.  IRS-associated PI3K <span class="highlight" style="background-color:">signaling</span> is compromised by insulin resistant states, such as in DM2, and downstream GLUT4 translocation is disrupted.  This disruption drives PI3K <span class="highlight" style="background-color:">signaling</span> towards AKT/mTOR.  AKT and mTOR can affect both the metabolic and mitogenic <span class="highlight" style="background-color:">pathway</span>, but because of the <span class="highlight" style="background-color:">signaling</span> dysfunction, AKT and mTOR are driven towards the mitogenic <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Grace Sun, et al. J Nutr Metab. 2011;2011:708183.</div><p class="links"><a class="dblinks" href="/pubmed/21773024" ref="ordinalpos=3356&amp;ncbi_uid=3087618&amp;link_uid=21773024">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136221/" ref="ordinalpos=3356&amp;ncbi_uid=3087618&amp;link_uid=PMC3136221">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3357.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3490576/figure/fig8/" ref="ordinalpos=3357&amp;ncbi_uid=4352278&amp;link_uid=PMC3490576" href="/pmc/articles/PMC3490576/"><img src="/pmc/articles/instance/3490576/bin/PP_202358_gs_f8.gif" src-large="/pmc/articles/instance/3490576/bin/PP_202358_gs_f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3490576/" ref="ordinalpos=3357&amp;ncbi_uid=4352278&amp;link_uid=PMC3490576" image-link="/pmc/articles/PMC3490576/figure/fig8/" class="imagepopup">Figure 8. From: Strigolactones Are Involved in Root Response to Low Phosphate Conditions in Arabidopsis[W][OA]. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the hormonal pathways activated by SLs in response to low Pi growth conditions. Under conditions of Pi depletion, and in response to endogenous P concentrations, the SL <span class="highlight" style="background-color:">pathway</span>, through MAX2, is suggested to activate <i>TIR1</i> expression and the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and thereby regulate root development to suit those growth conditions. Another possibility is that there could also be a parallel <span class="highlight" style="background-color:">pathway</span> to TIR1 that involves IAA, in a MAX2-independent <span class="highlight" style="background-color:">pathway</span>, which leads to partial low Pi response at early stages. However, evidence is found to the existence of other pathways that are MAX2 independent, which may dominate low Pi response during later stages of development.</div></p></div><div class="aux"><div class="resc">Einav Mayzlish-Gati, et al. Plant Physiol. 2012 November;160(3):1329-1341.</div><p class="links"><a class="dblinks" href="/pubmed/22968830" ref="ordinalpos=3357&amp;ncbi_uid=4352278&amp;link_uid=22968830">Citation</a><a class="dblinks" href="/pmc/articles/PMC3490576/" ref="ordinalpos=3357&amp;ncbi_uid=4352278&amp;link_uid=PMC3490576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3358.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266783/figure/F2/" ref="ordinalpos=3358&amp;ncbi_uid=4487769&amp;link_uid=PMC3266783" href="/pmc/articles/PMC3266783/"><img src="/pmc/articles/instance/3266783/bin/JCI58779.f2.gif" src-large="/pmc/articles/instance/3266783/bin/JCI58779.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266783/" ref="ordinalpos=3358&amp;ncbi_uid=4487769&amp;link_uid=PMC3266783" image-link="/pmc/articles/PMC3266783/figure/F2/" class="imagepopup">Figure 2. The Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> — a simplified model..  From: Basal cell carcinoma -- molecular biology and potential new therapies. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In its “off” state, Ptch1 represses Smo activity. Gli2 and Gli3, effectors of the Hh <span class="highlight" style="background-color:">pathway</span>, are phosphorylated by a kinase cascade, which includes PKA, CK1, and GSK3β, and are directed to the proteasomal degradation <span class="highlight" style="background-color:">pathway</span> via the SPOP complex. A fraction of the Gli2/3 protein is processed into a repressor form, Gli-R, which inhibits Hh target gene transcription. (<b>B</b>) Hh ligand binding to Ptch1 abrogates its inhibitory effect on Smo, allowing Smo to translocate into the primary cilium and induce accumulation of the Gli-Sufu complex at the tip of the primary cilium. Activation of the Hh <span class="highlight" style="background-color:">pathway</span> results in accumulation of Gli-A and initiation of the transcription of Hh target genes such as <i>PTCH1</i>, <i>GLI1</i>, and <i>HHIP</i>.    </div></p></div><div class="aux"><div class="resc">Maria Kasper, et al. J Clin Invest. 2012 February 1;122(2):455-463.</div><p class="links"><a class="dblinks" href="/pubmed/22293184" ref="ordinalpos=3358&amp;ncbi_uid=4487769&amp;link_uid=22293184">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266783/" ref="ordinalpos=3358&amp;ncbi_uid=4487769&amp;link_uid=PMC3266783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3359.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3221684/figure/pone-0028067-g008/" ref="ordinalpos=3359&amp;ncbi_uid=3690467&amp;link_uid=PMC3221684" href="/pmc/articles/PMC3221684/"><img src="/pmc/articles/instance/3221684/bin/pone.0028067.g008.gif" src-large="/pmc/articles/instance/3221684/bin/pone.0028067.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3221684/" ref="ordinalpos=3359&amp;ncbi_uid=3690467&amp;link_uid=PMC3221684" image-link="/pmc/articles/PMC3221684/figure/pone-0028067-g008/" class="imagepopup">Figure 8. Sl modes of action in growth and differentiation..  From: Insulin Receptor-Mediated <span class="highlight" style="background-color:">Signaling</span> via Phospholipase C-? Regulates Growth and Differentiation in Drosophila. </a></p><div class="supp"><p class="details"><div>Panel (A) shows Sl, activated by the insulin <span class="highlight" style="background-color:">pathway</span>, acting as a liaison regulating MAPK <span class="highlight" style="background-color:">pathway</span> ligand processing, to foster MAPK activation to a level promoting growth (red inhibitory interaction). (B) Conversely, for differentiation, reduced insulin receptor <span class="highlight" style="background-color:">signaling</span> leads to lower levels of Sl activation and augmented Spi processing (different from A, gray inhibitory interaction; possibly other targets from those in A), and this, in turn, allows MAPK activation in a manner consistent with promotion of differentiation.</div></p></div><div class="aux"><div class="resc">Juan M. Murillo-Maldonado, et al. PLoS One. 2011;6(11):e28067.</div><p class="links"><a class="dblinks" href="/pubmed/22132213" ref="ordinalpos=3359&amp;ncbi_uid=3690467&amp;link_uid=22132213">Citation</a><a class="dblinks" href="/pmc/articles/PMC3221684/" ref="ordinalpos=3359&amp;ncbi_uid=3690467&amp;link_uid=PMC3221684">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3360.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407377/figure/F3/" ref="ordinalpos=3360&amp;ncbi_uid=4842846&amp;link_uid=PMC3407377" href="/pmc/articles/PMC3407377/"><img src="/pmc/articles/instance/3407377/bin/nihms388183f3.gif" src-large="/pmc/articles/instance/3407377/bin/nihms388183f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407377/" ref="ordinalpos=3360&amp;ncbi_uid=4842846&amp;link_uid=PMC3407377" image-link="/pmc/articles/PMC3407377/figure/F3/" class="imagepopup">Fig. 3. TGF-β-induced TAK1 <span class="highlight" style="background-color:">signaling</span> and profibrotic response.  From: TGF-? <span class="highlight" style="background-color:">signaling</span> via TAK1 <span class="highlight" style="background-color:">pathway</span>: Role in kidney fibrosis. </a></p><div class="supp"><p class="details"><div>TGF-β1-induced TAK1 activation triggers downstream <span class="highlight" style="background-color:">signaling</span> pathways such as the MKK4-JNK, or the MKK3-p38 cascade, and promotes degradation of Ik-B, which in turn lead to the activation of transcription factors ATF-2, AP-1 and NF-kB, respectively, to regulate the expression of extracellular matrix proteins, including collagens and fibronectin, and inflammatory cytokines. In addition, TGF-β1-induced Smad activation is also indispensible for TGF-β1-induced type I collagen expression through the crosstalk with TAK1-MKK3-p38 <span class="highlight" style="background-color:">signaling</span> axis.</div></p></div><div class="aux"><div class="resc">Mary E. Choi, et al. Semin Nephrol. ;32(3):244-252.</div><p class="links"><a class="dblinks" href="/pubmed/22835455" ref="ordinalpos=3360&amp;ncbi_uid=4842846&amp;link_uid=22835455">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407377/" ref="ordinalpos=3360&amp;ncbi_uid=4842846&amp;link_uid=PMC3407377">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3361.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3497822/figure/AS-fig01/" ref="ordinalpos=3361&amp;ncbi_uid=4836891&amp;link_uid=PMC3497822" href="/pmc/articles/PMC3497822/"><img src="/pmc/articles/instance/3497822/bin/pnas.1205661S109fig01.gif" src-large="/pmc/articles/instance/3497822/bin/pnas.1205661S109fig01.jpg" alt="Fig. P1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3497822/" ref="ordinalpos=3361&amp;ncbi_uid=4836891&amp;link_uid=PMC3497822" image-link="/pmc/articles/PMC3497822/figure/AS-fig01/" class="imagepopup">Fig. P1. From: PKA and GAB2 play central roles in the FSH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to PI3K and AKT in ovarian granulosa cells. </a></p><div class="supp"><p class="details"><div>Model of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which FSH promotes AKT activation in GCs. FSH binding to and activation of the FSH G protein-coupled receptor leads to activation of the stimulatory G protein (G<sub>s</sub>), activation of adenylyl cyclase (AC), generation of cAMP, and activation of type I PKA (R<sup>I</sup><sub>2</sub>C<sub>2</sub>). PKA directly (unbroken line) or indirectly (broken line) promotes dephosphorylation (slashed P) of GAB2(Y<sup>452</sup>) and phosphorylation (P) of GAB2(Ser<sup>159</sup>) and IRS-1(Tyr<sup>989</sup>). Phosphorylation of IRS-1 on Tyr<sup>989</sup> provides the binding site for the p85 R-subunit of PI3K to allosterically activate the p110 catalytic subunit, resulting in phosphorylation-dependent activation of AKT(Thr<sup>308</sup>)(Ser<sup>473</sup>) that promotes GC proliferation and differentiation.</div></p></div><div class="aux"><div class="resc">Mary E. Hunzicker-Dunn, et al. Proc Natl Acad Sci U S A. 2012 October 30;109(44):E2979-E2988.</div><p class="links"><a class="dblinks" href="/pubmed/23045700" ref="ordinalpos=3361&amp;ncbi_uid=4836891&amp;link_uid=23045700">Citation</a><a class="dblinks" href="/pmc/articles/PMC3497822/" ref="ordinalpos=3361&amp;ncbi_uid=4836891&amp;link_uid=PMC3497822">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3362.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4373890/figure/pone.0122192.g002/" ref="ordinalpos=3362&amp;ncbi_uid=7339042&amp;link_uid=PMC4373890" href="/pmc/articles/PMC4373890/"><img src="/pmc/articles/instance/4373890/bin/pone.0122192.g002.gif" src-large="/pmc/articles/instance/4373890/bin/pone.0122192.g002.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4373890/" ref="ordinalpos=3362&amp;ncbi_uid=7339042&amp;link_uid=PMC4373890" image-link="/pmc/articles/PMC4373890/figure/pone.0122192.g002/" class="imagepopup">Fig 2. <span class="highlight" style="background-color:">Signaling</span> networks instantiated in SEGMEnT..  From: Towards Anatomic Scale Agent-Based Modeling with a Massively Parallel Spatially Explicit General-Purpose Model of Enteric Tissue (SEGMEnT_HPC). </a></p><div class="supp"><p class="details"><div>Morphogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components are shaded in blue; inflammatory <span class="highlight" style="background-color:">signaling</span> components are shaded in orange. Stimulation/production relationships are depicted by green connectors; inhibitory relationships are seen as red connectors. The <span class="highlight" style="background-color:">signaling</span> network comprises the Wingless-related integration site (Wnt), Bone Morphogenetic Protein (BMP), Phosphotase and tensin homolog/phosphoinositide 3-kinase (PTEN/PI3K), Sonic Hedgehog Homolog (Hh), Tumor Necrosis Factor (TNF)-α, Interferon (IFN)-γ, RIP Kinase, nuclear factor kappa-light-chain-enhancer of activated B cells (NF-κB), Janus Kinase (JAK), Signal transducer and activator of transcription 3 (Stat3), and reactive oxygen species (ROSs), and Interleukin (IL) 6,10,13, and 15 <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Robert Chase Cockrell, et al. PLoS One. 2015;10(3):e0122192.</div><p class="links"><a class="dblinks" href="/pubmed/25806784" ref="ordinalpos=3362&amp;ncbi_uid=7339042&amp;link_uid=25806784">Citation</a><a class="dblinks" href="/pmc/articles/PMC4373890/" ref="ordinalpos=3362&amp;ncbi_uid=7339042&amp;link_uid=PMC4373890">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3363.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3825194/figure/F004/" ref="ordinalpos=3363&amp;ncbi_uid=5493457&amp;link_uid=PMC3825194" href="/pmc/articles/PMC3825194/"><img src="/pmc/articles/instance/3825194/bin/ooomb4-21-323-g004.gif" src-large="/pmc/articles/instance/3825194/bin/ooomb4-21-323-g004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3825194/" ref="ordinalpos=3363&amp;ncbi_uid=5493457&amp;link_uid=PMC3825194" image-link="/pmc/articles/PMC3825194/figure/F004/" class="imagepopup">Fig. 4.TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors under development for potential cancer therapy. (blue line box) Antisense molecules inhibit TGF-β synthesis; (black line box) monoclonal antibodies, soluble receptors and anti-receptor monoclonal antibodies inhibit ligand-receptor interaction; (red line box) receptor kinase inhibitors and peptide aptamers (purple line box) inhibit signal transduction..  From: Targeting the Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> in Cancer Therapy. </a></p><div class="aux"><div class="resc">Yhun Yhong Sheen, et al. Biomol Ther (Seoul). 2013 September 30;21(5):323-331.</div><p class="links"><a class="dblinks" href="/pubmed/24244818" ref="ordinalpos=3363&amp;ncbi_uid=5493457&amp;link_uid=24244818">Citation</a><a class="dblinks" href="/pmc/articles/PMC3825194/" ref="ordinalpos=3363&amp;ncbi_uid=5493457&amp;link_uid=PMC3825194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3364.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2724047/figure/fig7/" ref="ordinalpos=3364&amp;ncbi_uid=1908489&amp;link_uid=PMC2724047" href="/pmc/articles/PMC2724047/"><img src="/pmc/articles/instance/2724047/bin/1509fig7.gif" src-large="/pmc/articles/instance/2724047/bin/1509fig7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2724047/" ref="ordinalpos=3364&amp;ncbi_uid=1908489&amp;link_uid=PMC2724047" image-link="/pmc/articles/PMC2724047/figure/fig7/" class="imagepopup">Fig. 7. From: Bile acids as regulatory molecules. </a></p><div class="supp"><p class="details"><div>Regulation of hepatic glucose metabolism and cholesterol and fatty acid biosynthetic pathways by bile acids. Bile acids regulate glucose metabolism in hepatocytes by activating the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and by induction of the gene encoding SHP. Conjugated bile acids activate the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (pAKT) via surface G-protein-coupled receptors (GPCRs), which activates glycogen synthase activity and inhibits gluconeogenesis by phosphorylation of FOX01. SHP is known to interact with FOX01, a transcription factor known to upregulate gluconeogenic genes. Phosphoinositide-dependent protein kinase 1 (PDK-1), a kinase in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, is known to phosphorylate and activate protein kinase Cζ (PCKζ). PKCζ is proposed to phosphorylate FXR, enhancing the ability of conjugated bile acids to induce the gene encoding SHP. SHP can also downregulate the gene encoding SREBP1-c, which controls the rate of FA synthesis. Bile acids can activate JNK 1/2 in hepatocyte cultures by stimulating ceramide synthesis. In vivo, bile acids can activate the JNK 1/2 <span class="highlight" style="background-color:">pathway</span> by stimulating the synthesis of fibroblast growth factor 15/19 (FGF-15/19) in the intestines. Finally, TNF-α and IL-1 can also activate the JNK 1/2 <span class="highlight" style="background-color:">pathway</span> downregulating CYP7A1. Bold lines indicate pathways activated by bile acids.</div></p></div><div class="aux"><div class="resc">Phillip B. Hylemon, et al. J Lipid Res. 2009 August;50(8):1509-1520.</div><p class="links"><a class="dblinks" href="/pubmed/19346331" ref="ordinalpos=3364&amp;ncbi_uid=1908489&amp;link_uid=19346331">Citation</a><a class="dblinks" href="/pmc/articles/PMC2724047/" ref="ordinalpos=3364&amp;ncbi_uid=1908489&amp;link_uid=PMC2724047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3365.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727475/figure/F1/" ref="ordinalpos=3365&amp;ncbi_uid=1947267&amp;link_uid=PMC2727475" href="/pmc/articles/PMC2727475/"><img src="/pmc/articles/instance/2727475/bin/nihms119570f1.gif" src-large="/pmc/articles/instance/2727475/bin/nihms119570f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727475/" ref="ordinalpos=3365&amp;ncbi_uid=1947267&amp;link_uid=PMC2727475" image-link="/pmc/articles/PMC2727475/figure/F1/" class="imagepopup">Figure 1. The Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Acts at Multiple Developmental Stages to Promote Mammalian Cardiogenesis. </a></p><div class="supp"><p class="details"><div>Canonical Wnt <span class="highlight" style="background-color:">signaling</span> is initiated when Wnts bind to their cognate receptor complex of the transmembrane proteins Frizzled and Lrp. This interaction activates the cytoplasmic protein Dishevelled, which stabilizes β-catenin by inhibiting the kinase activity of the destruction complex of adenomatous polyposis coli (APC), Axin, and glycogen synthase kinase-3β (GSK3β). Unphosphorylated β-catenin on its N-terminus translocates into the nucleus, where it interacts with the TCF/lymphoid enhancer factor (LEF) transcription factors to activate Wnt target genes. In the absence of Wnt, the destruction complex phosphorylates β-catenin, targeting it for degradation.</div></p></div><div class="aux"><div class="resc">Chulan Kwon, et al. Cell Cycle. ;7(24):3815-3818.</div><p class="links"><a class="dblinks" href="/pubmed/19066459" ref="ordinalpos=3365&amp;ncbi_uid=1947267&amp;link_uid=19066459">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727475/" ref="ordinalpos=3365&amp;ncbi_uid=1947267&amp;link_uid=PMC2727475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3366.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3318714/figure/F3/" ref="ordinalpos=3366&amp;ncbi_uid=4693188&amp;link_uid=PMC3318714" href="/pmc/articles/PMC3318714/"><img src="/pmc/articles/instance/3318714/bin/zbc0131200990003.gif" src-large="/pmc/articles/instance/3318714/bin/zbc0131200990003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3318714/" ref="ordinalpos=3366&amp;ncbi_uid=4693188&amp;link_uid=PMC3318714" image-link="/pmc/articles/PMC3318714/figure/F3/" class="imagepopup">FIGURE 3. From: Connections between Sphingosine Kinase and Phospholipase D in the Abscisic Acid <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Arabidopsis. </a></p><div class="supp"><p class="details"><div><b>PLDα1 and PA mediate the phyto-S1P effect on the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ABA-mediated stomatal closure.</b> <i>A</i>, effect of phyto-S1P on stomatal closure in WT and mutants. The epidermal peels were incubated in stomatal incubation buffer containing 10 μ<span style="font-variant: small-caps" class="small-caps">m</span> phyto-S1P or 10 μ<span style="font-variant: small-caps" class="small-caps">m</span> phyto-S1P plus 0.1% 1-butanol. <i>Asterisks</i> indicate that the mean value is significantly different from that of the samples treated with phyto-S1P at <i>p</i> &lt; 0.05 based on Student's <i>t</i> test. <i>B</i>, PA (18:1/18:1) induces stomatal closure in WT and mutants. Epidermal peels were treated with 50 μ<span style="font-variant: small-caps" class="small-caps">m</span> PA. All values are mean ± S.E. (<i>n</i> = 50) in the stomatal assays.</div></p></div><div class="aux"><div class="resc">Liang Guo, et al. J Biol Chem. 2012 March 9;287(11):8286-8296.</div><p class="links"><a class="dblinks" href="/pubmed/22275366" ref="ordinalpos=3366&amp;ncbi_uid=4693188&amp;link_uid=22275366">Citation</a><a class="dblinks" href="/pmc/articles/PMC3318714/" ref="ordinalpos=3366&amp;ncbi_uid=4693188&amp;link_uid=PMC3318714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3367.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2946515/figure/F3/" ref="ordinalpos=3367&amp;ncbi_uid=3257686&amp;link_uid=PMC2946515" href="/pmc/articles/PMC2946515/"><img src="/pmc/articles/instance/2946515/bin/nihms165144f3.gif" src-large="/pmc/articles/instance/2946515/bin/nihms165144f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2946515/" ref="ordinalpos=3367&amp;ncbi_uid=3257686&amp;link_uid=PMC2946515" image-link="/pmc/articles/PMC2946515/figure/F3/" class="imagepopup">Figure 3.  From: Epidermal Growth Factor Receptor Targeted Therapy of Squamous Cell Carcinoma of the Head and Neck. </a></p><div class="supp"><p class="details"><div>The mutant epidermal growth factor receptor, EGFRvIII, promotes cell survival and proliferation by preferential <span class="highlight" style="background-color:">signaling</span> through the phosphotidyinositol-3-kinase (PI3K) <span class="highlight" style="background-color:">pathway</span>. The wild-type receptor utilizes both the PI3K <span class="highlight" style="background-color:">pathway</span> and the Ras-Raf-MEK <span class="highlight" style="background-color:">pathway</span>. Downstream effectors of both pathways further promote cell survival via inhibition of apoptosis.</div></p></div><div class="aux"><div class="resc">Mark E. Sharafinski, et al. Head Neck. ;32(10):1412-1421.</div><p class="links"><a class="dblinks" href="/pubmed/20848399" ref="ordinalpos=3367&amp;ncbi_uid=3257686&amp;link_uid=20848399">Citation</a><a class="dblinks" href="/pmc/articles/PMC2946515/" ref="ordinalpos=3367&amp;ncbi_uid=3257686&amp;link_uid=PMC2946515">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3368.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3154417/figure/F2/" ref="ordinalpos=3368&amp;ncbi_uid=3633206&amp;link_uid=PMC3154417" href="/pmc/articles/PMC3154417/"><img src="/pmc/articles/instance/3154417/bin/cc1012_1940_fig002.gif" src-large="/pmc/articles/instance/3154417/bin/cc1012_1940_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3154417/" ref="ordinalpos=3368&amp;ncbi_uid=3633206&amp;link_uid=PMC3154417" image-link="/pmc/articles/PMC3154417/figure/F2/" class="imagepopup">Figure 2.  From: mTOR <span class="highlight" style="background-color:">signaling</span> in protein homeostasis. </a></p><div class="supp"><p class="details"><div>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. mTORC1 and mTORC2 receive distinct upstream signals from both extracellular and intracellular stimuli and have different downstream targets. Notably, interconnection between mTORC1 and mTORC2 occurs at several levels. For example, one of the mTORC2 downstream targets, Akt, serves as an upstream regulator of mTORC1, whereas one important mTORC1 target ribosome acts as an activator of mTORC2. Most of the <span class="highlight" style="background-color:">signaling</span> components are involved in growth and aging, and lots of mutations have been found in human diseases such as cancer. For simplicity, only typical mTOR components are shown. See the main text for details.</div></p></div><div class="aux"><div class="resc">Crystal S Conn, et al. Cell Cycle. 2011 June 15;10(12):1940-1947.</div><p class="links"><a class="dblinks" href="/pubmed/21555915" ref="ordinalpos=3368&amp;ncbi_uid=3633206&amp;link_uid=21555915">Citation</a><a class="dblinks" href="/pmc/articles/PMC3154417/" ref="ordinalpos=3368&amp;ncbi_uid=3633206&amp;link_uid=PMC3154417">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3369.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634135/figure/F1/" ref="ordinalpos=3369&amp;ncbi_uid=1708840&amp;link_uid=PMC2634135" href="/pmc/articles/PMC2634135/"><img src="/pmc/articles/instance/2634135/bin/psb0204_0245_fig001.gif" src-large="/pmc/articles/instance/2634135/bin/psb0204_0245_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634135/" ref="ordinalpos=3369&amp;ncbi_uid=1708840&amp;link_uid=PMC2634135" image-link="/pmc/articles/PMC2634135/figure/F1/" class="imagepopup">Figure 1.  From: The Protein Phosphatases POL and PLL1 are <span class="highlight" style="background-color:">Signaling</span> Intermediates for Multiple Pathways in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Clavata <span class="highlight" style="background-color:">pathway</span> model. The receptor-like kinase protein, CLV1, and the receptor-like protein, CLV2, are localized to the plasma membrane. The putative ligand, CLV3 is produced as a propeptide and then cleaved to release the mature <span class="highlight" style="background-color:">signaling</span> molecule. Upon activation, the CLV1/2 complex blocks the activity of the PP2C proteins, POL and PLL1 via an unknown mechanism. When CLV3 is not present, POL and PLL1 are active and through unidentified <span class="highlight" style="background-color:">signaling</span> intermediates upregulate the homeodomain transcription factor, WUS, which is required for the maintenance and specification of the stem cells.</div></p></div><div class="aux"><div class="resc">Jennifer M Gagne, et al. Plant Signal Behav. 2007 Jul-Aug;2(4):245-246.</div><p class="links"><a class="dblinks" href="/pubmed/19704666" ref="ordinalpos=3369&amp;ncbi_uid=1708840&amp;link_uid=19704666">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634135/" ref="ordinalpos=3369&amp;ncbi_uid=1708840&amp;link_uid=PMC2634135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3370.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC548942/figure/fig4/" ref="ordinalpos=3370&amp;ncbi_uid=484419&amp;link_uid=PMC548942" href="/pmc/articles/PMC548942/"><img src="/pmc/articles/instance/548942/bin/E3267Bf4_4t.gif" src-large="/pmc/articles/instance/548942/bin/E3267Bf4_4t.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC548942/" ref="ordinalpos=3370&amp;ncbi_uid=484419&amp;link_uid=PMC548942" image-link="/pmc/articles/PMC548942/figure/fig4/" class="imagepopup">Figure 4. From: Frat is dispensable for canonical Wnt <span class="highlight" style="background-color:">signaling</span> in mammals. </a></p><div class="supp"><p class="details"><div>Model for the physiological role of Frat. (<i>A</i>) Current model of canonical Wnt-signal transduction. See text for details. (<i>B</i>) Several models could explain the observation that Frat is not critically required for canonical Wnt <span class="highlight" style="background-color:">signaling</span> in mammals. (<i>Left</i>) Frat might be induced under specific conditions or it might feed into the canonical Wnt <span class="highlight" style="background-color:">pathway</span> from a parallel route independent from upstream Wnt signals. (<i>Right</i>) Alternatively, Frat might be involved in any of the numerous other GSK3-dependent cellular activities, such as the NFκB and NFAT <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Renée van Amerongen, et al. Genes Dev. 2005 February 15;19(4):425-430.</div><p class="links"><a class="dblinks" href="/pubmed/15681612" ref="ordinalpos=3370&amp;ncbi_uid=484419&amp;link_uid=15681612">Citation</a><a class="dblinks" href="/pmc/articles/PMC548942/" ref="ordinalpos=3370&amp;ncbi_uid=484419&amp;link_uid=PMC548942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3371.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945784/figure/F2/" ref="ordinalpos=3371&amp;ncbi_uid=2479607&amp;link_uid=PMC2945784" href="/pmc/articles/PMC2945784/"><img src="/pmc/articles/instance/2945784/bin/gb-2010-11-8-r82-2.gif" src-large="/pmc/articles/instance/2945784/bin/gb-2010-11-8-r82-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945784/" ref="ordinalpos=3371&amp;ncbi_uid=2479607&amp;link_uid=PMC2945784" image-link="/pmc/articles/PMC2945784/figure/F2/" class="imagepopup">Figure 2.  From: Evolution of an adenocarcinoma in response to selection by targeted kinase inhibitors. </a></p><div class="supp"><p class="details"><div><b>Cancer <span class="highlight" style="background-color:">signaling</span> pathways affected within the tumor</b>. <b>(a) </b>Pre-treatment: overall, the down-regulation of <i>PTEN </i>and up-regulation of the <i>RET </i><span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> appear to be driving tumor proliferation. Increased <span class="highlight" style="background-color:">signaling</span> independent of EGFR is consistent with the observed erlotinib insensitivity of the tumor. <b>(b) </b>Post-versus pre-treatment: after treatment with the RET inhibitors sunitinib and sorafenib, there is a marked increase in the <span class="highlight" style="background-color:">signaling</span> of <span class="highlight" style="background-color:">pathway</span> constituents, increasing tumor proliferation. Black and red <span class="highlight" style="background-color:">pathway</span> arrows represent activation and inhibition, respectively. Dotted arrows represent indirect interactions. The number of arrows denoting significantly over- or under-expressed genes are quantified using fold change of tumor versus compendium in (a), and primary tumor versus the tumor recurrence in (b): 1 arrow is FC ≥2; 2 arrows is FC ≥10; and 3 arrows is FC ≥50. CNV, copy number variation.</div></p></div><div class="aux"><div class="resc">Steven JM Jones, et al. Genome Biol. 2010;11(8):R82-R82.</div><p class="links"><a class="dblinks" href="/pubmed/20696054" ref="ordinalpos=3371&amp;ncbi_uid=2479607&amp;link_uid=20696054">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945784/" ref="ordinalpos=3371&amp;ncbi_uid=2479607&amp;link_uid=PMC2945784">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3372.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3538028/figure/fig7/" ref="ordinalpos=3372&amp;ncbi_uid=4679926&amp;link_uid=PMC3538028" href="/pmc/articles/PMC3538028/"><img src="/pmc/articles/instance/3538028/bin/gr7.gif" src-large="/pmc/articles/instance/3538028/bin/gr7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3538028/" ref="ordinalpos=3372&amp;ncbi_uid=4679926&amp;link_uid=PMC3538028" image-link="/pmc/articles/PMC3538028/figure/fig7/" class="imagepopup">Figure 7.  From: Transforming Growth Factor ?-1 Stimulates Profibrotic Epithelial <span class="highlight" style="background-color:">Signaling</span> to Activate Pericyte-Myofibroblast Transition in Obstructive Kidney Fibrosis. </a></p><div class="supp"><p class="details"><div>Schematic of TGF-β1 stimulated profibrotic epithelial <span class="highlight" style="background-color:">signaling</span> to pericytes during fibrotic kidney injury. Fibrotic injury induced TGF-β1 production of tubular epithelial cells. TGF-β1 then induced G2/M cell cycle arrest and profibrotic phenotype through up-regulation of p21 and activation of the JNK <span class="highlight" style="background-color:">pathway</span>, respectively. TGF-β1 and PDGF subsequently stimulated pericyte-myofibroblast transition through differentiation and proliferation, respectively.</div></p></div><div class="aux"><div class="resc">Ching-Fang Wu, et al. Am J Pathol. 2013 January;182(1):118-131.</div><p class="links"><a class="dblinks" href="/pubmed/23142380" ref="ordinalpos=3372&amp;ncbi_uid=4679926&amp;link_uid=23142380">Citation</a><a class="dblinks" href="/pmc/articles/PMC3538028/" ref="ordinalpos=3372&amp;ncbi_uid=4679926&amp;link_uid=PMC3538028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3373.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2897923/figure/F1/" ref="ordinalpos=3373&amp;ncbi_uid=3186733&amp;link_uid=PMC2897923" href="/pmc/articles/PMC2897923/"><img src="/pmc/articles/instance/2897923/bin/nihms188775f1.gif" src-large="/pmc/articles/instance/2897923/bin/nihms188775f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2897923/" ref="ordinalpos=3373&amp;ncbi_uid=3186733&amp;link_uid=PMC2897923" image-link="/pmc/articles/PMC2897923/figure/F1/" class="imagepopup">Figure 1. The Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Points of Intervention.  From: Striking the Target in Wnt-y Conditions: Intervening in Wnt <span class="highlight" style="background-color:">Signaling</span> During Cancer Progression. </a></p><div class="supp"><p class="details"><div>Receptor activation upon binding of Wnt 1 or Wnt3A leads to stabilization of β-catenin which localizes to the nucleus where it associates with transcription factors and activates transcription of genes involved in proliferation and tumor progression. In the absence of Wnt, transcription of these genes is prevented by phosphorylation of β-catenin by GSK3β within a destruction complex and its subsequent degradation via the proteasome.</div></p></div><div class="aux"><div class="resc">Tura C. Camilli, et al. Biochem Pharmacol. ;80(5):702-711.</div><p class="links"><a class="dblinks" href="/pubmed/20211149" ref="ordinalpos=3373&amp;ncbi_uid=3186733&amp;link_uid=20211149">Citation</a><a class="dblinks" href="/pmc/articles/PMC2897923/" ref="ordinalpos=3373&amp;ncbi_uid=3186733&amp;link_uid=PMC2897923">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3374.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2735497/figure/F4/" ref="ordinalpos=3374&amp;ncbi_uid=2034047&amp;link_uid=PMC2735497" href="/pmc/articles/PMC2735497/"><img src="/pmc/articles/instance/2735497/bin/nihms114984f4.gif" src-large="/pmc/articles/instance/2735497/bin/nihms114984f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2735497/" ref="ordinalpos=3374&amp;ncbi_uid=2034047&amp;link_uid=PMC2735497" image-link="/pmc/articles/PMC2735497/figure/F4/" class="imagepopup">Figure 4. Hsp90 does not repress Ras1-PKA <span class="highlight" style="background-color:">signaling</span> via negative regulators of the <span class="highlight" style="background-color:">pathway</span>.  From: Hsp90 Orchestrates Temperature-Dependent Candida albicans Morphogenesis Via Ras1-PKA <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Loss of function of negative regulators of the Ras1-PKA cascade did not phenocopy Hsp90 inhibition while inhibition of Hsp90 induced filamentation. Cells were incubated in YPD at 30°C for 6 hours with GdA, as indicated. Scale bar, 10 μm.</div></p></div><div class="aux"><div class="resc">R. S. Shapiro, et al. Curr Biol. ;19(8):621-629.</div><p class="links"><a class="dblinks" href="/pubmed/19327993" ref="ordinalpos=3374&amp;ncbi_uid=2034047&amp;link_uid=19327993">Citation</a><a class="dblinks" href="/pmc/articles/PMC2735497/" ref="ordinalpos=3374&amp;ncbi_uid=2034047&amp;link_uid=PMC2735497">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3375.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997853/figure/fig7/" ref="ordinalpos=3375&amp;ncbi_uid=6109213&amp;link_uid=PMC3997853" href="/pmc/articles/PMC3997853/"><img src="/pmc/articles/instance/3997853/bin/IJG2014-426025.007.gif" src-large="/pmc/articles/instance/3997853/bin/IJG2014-426025.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997853/" ref="ordinalpos=3375&amp;ncbi_uid=6109213&amp;link_uid=PMC3997853" image-link="/pmc/articles/PMC3997853/figure/fig7/" class="imagepopup">Figure 7.  From: Microarray Analysis of the Juvenile Hormone Response in Larval Integument of the Silkworm, Bombyx mori. </a></p><div class="supp"><p class="details"><div>The IIS <span class="highlight" style="background-color:">pathway</span> is modulated by JHA. (a) Hierarchical clustering of JHA-modulated genes related to the IIS <span class="highlight" style="background-color:">pathway</span> and 20E biosynthesis. The probe ID, KEGG ID, and expression intensity of each JHA-modulated gene are listed in Tables S5 and S8. Hat: hours after treatment. (b) Real-time RT-PCR analysis of JHA modulation of the expression levels of key regulators involved in the IIS <span class="highlight" style="background-color:">pathway</span> and 20E <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Daojun Cheng, et al. Int J Genomics. 2014;2014:426025.</div><p class="links"><a class="dblinks" href="/pubmed/24809046" ref="ordinalpos=3375&amp;ncbi_uid=6109213&amp;link_uid=24809046">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997853/" ref="ordinalpos=3375&amp;ncbi_uid=6109213&amp;link_uid=PMC3997853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3376.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3747576/figure/fig6/" ref="ordinalpos=3376&amp;ncbi_uid=5219468&amp;link_uid=PMC3747576" href="/pmc/articles/PMC3747576/"><img src="/pmc/articles/instance/3747576/bin/mbo0041315780006.gif" src-large="/pmc/articles/instance/3747576/bin/mbo0041315780006.jpg" alt="FIG 6 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3747576/" ref="ordinalpos=3376&amp;ncbi_uid=5219468&amp;link_uid=PMC3747576" image-link="/pmc/articles/PMC3747576/figure/fig6/" class="imagepopup">FIG 6 .  From: Mechanisms of Severe Acute Respiratory Syndrome Coronavirus-Induced Acute Lung Injury. </a></p><div class="supp"><p class="details"><div>Urokinase <span class="highlight" style="background-color:">pathway</span> model. (A) Representation of the unperturbed urokinase <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) Without the presence of Serpine1, an inhibitor of both PLAU/urokinase and PLAT/tPA, there is increased cleavage of plasminogen into the active plasmin and thus increased breakdown of fibrin clots and hemorrhage compared to an unperturbed system. Red T shapes indicate inhibition, and blue arrows indicate activation.</div></p></div><div class="aux"><div class="resc">Lisa E. Gralinski, et al. mBio. 2013 Jul-Aug;4(4):e00271-13.</div><p class="links"><a class="dblinks" href="/pubmed/23919993" ref="ordinalpos=3376&amp;ncbi_uid=5219468&amp;link_uid=23919993">Citation</a><a class="dblinks" href="/pmc/articles/PMC3747576/" ref="ordinalpos=3376&amp;ncbi_uid=5219468&amp;link_uid=PMC3747576">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3377.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4009444/figure/F2/" ref="ordinalpos=3377&amp;ncbi_uid=6107664&amp;link_uid=PMC4009444" href="/pmc/articles/PMC4009444/"><img src="/pmc/articles/instance/4009444/bin/fphar-05-00086-g002.gif" src-large="/pmc/articles/instance/4009444/bin/fphar-05-00086-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4009444/" ref="ordinalpos=3377&amp;ncbi_uid=6107664&amp;link_uid=PMC4009444" image-link="/pmc/articles/PMC4009444/figure/F2/" class="imagepopup">FIGURE 2.  From: Hepcidin antagonists for potential treatments of disorders with hepcidin excess. </a></p><div class="supp"><p class="details"><div><b>Intracellular <span class="highlight" style="background-color:">signaling</span> pathways for hepcidin expression. (A)</b> Iron status stimulates hepcidin expression through BMP6 and Holo-Tf. This <span class="highlight" style="background-color:">signaling</span> involves BMP receptor, HJV (BMPco-receptor), TMPRSS6 (MT2, a negative regulator), Neogenin, TfR2, and HFE (sensors of Holo-Tf saturation). Intracellularly this multiprotein complex triggers the activation of SMAD1/5/8 and SMAD4 components and possibly ERK1/2. <b>(B)</b> The inflammatory status triggers the hepcidin expression through IL6/STAT3 <span class="highlight" style="background-color:">pathway</span> and ActivinB that activates SMAD1/5/8 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Maura Poli, et al. Front Pharmacol. 2014;5:86.</div><p class="links"><a class="dblinks" href="/pubmed/24808863" ref="ordinalpos=3377&amp;ncbi_uid=6107664&amp;link_uid=24808863">Citation</a><a class="dblinks" href="/pmc/articles/PMC4009444/" ref="ordinalpos=3377&amp;ncbi_uid=6107664&amp;link_uid=PMC4009444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3378.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3065164/figure/F4/" ref="ordinalpos=3378&amp;ncbi_uid=2878086&amp;link_uid=PMC3065164" href="/pmc/articles/PMC3065164/"><img src="/pmc/articles/instance/3065164/bin/fnbeh-05-00015-g004.gif" src-large="/pmc/articles/instance/3065164/bin/fnbeh-05-00015-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3065164/" ref="ordinalpos=3378&amp;ncbi_uid=2878086&amp;link_uid=PMC3065164" image-link="/pmc/articles/PMC3065164/figure/F4/" class="imagepopup">Figure 4.  From: Dopamine-Mediated Learning and Switching in Cortico-Striatal Circuit Explain Behavioral Changes in Reinforcement Learning. </a></p><div class="supp"><p class="details"><div><b>Simulated neural components of the model performing reward and no-reward trials of 1DR task</b>. In reward trials <b>(A)</b> the reward-category unit (REW category) ramps up its activity shortly after the presentation of the fixation point. The activity shuts off in response to the burst activity of DA unit (DA) <span class="highlight" style="background-color:">signaling</span> the reward value of the target. The FEF unit combines the tonic reward-category activity and the phasic target signal. In the BG, both the direct <span class="highlight" style="background-color:">pathway</span> MSN unit (D1) and the indirect <span class="highlight" style="background-color:">pathway</span> MSN unit (D2) receive an input from the FEF. The direct <span class="highlight" style="background-color:">pathway</span> MSN unit (D1), in addition, receives an input directly from the reward-category unit and therefore shows larger ramping activity than the indirect <span class="highlight" style="background-color:">pathway</span> MSN unit (D2). The activity of the direct <span class="highlight" style="background-color:">pathway</span> MSN unit (D1) is further enhanced by DA-dependent LTP, which is triggered by the DA burst, and mediated by D1 receptors. This results in a stronger disinhibition of the SC by the SNr leading to a stronger activity in the SC. In contrast, the activity of indirect <span class="highlight" style="background-color:">pathway</span> MSN unit (D2) is further depressed by DA-dependent LTD, which is triggered by the DA burst, and mediated by D2 receptors. This results in the suppression of the excitatory input from the STN to the SNr, further enhancing the SC activity. The combined effects from the direct and indirect pathways lead to a shorter latency saccade (see the arrow head on top, indicating the time of saccade initiation). In no-reward trials <b>(B)</b> the activity of the reward-category unit is much weaker, thus lowering the activity of the FEF unit and the direct <span class="highlight" style="background-color:">pathway</span> MSN unit (D1). The activity of the direct <span class="highlight" style="background-color:">pathway</span> MSN unit (D1) is further depressed by DA-independent LTD. In contrast, the activity of D2 MSN increases because DA-dependent LTD is attenuated due to the “pause” of DA activity (DA) and thus is dominated by DA-independent LTP. The combined effects from the direct and indirect pathways lead to a weaker activation of the SC unit and hence a longer latency saccade. The scale of all the ordinate axes is from 0 to 1.</div></p></div><div class="aux"><div class="resc">Simon Hong, et al. Front Behav Neurosci. 2011;5:15.</div><p class="links"><a class="dblinks" href="/pubmed/21472026" ref="ordinalpos=3378&amp;ncbi_uid=2878086&amp;link_uid=21472026">Citation</a><a class="dblinks" href="/pmc/articles/PMC3065164/" ref="ordinalpos=3378&amp;ncbi_uid=2878086&amp;link_uid=PMC3065164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3379.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4004623/figure/fig5/" ref="ordinalpos=3379&amp;ncbi_uid=6075188&amp;link_uid=PMC4004623" href="/pmc/articles/PMC4004623/"><img src="/pmc/articles/instance/4004623/bin/ar-2013-00286b_0006.gif" src-large="/pmc/articles/instance/4004623/bin/ar-2013-00286b_0006.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4004623/" ref="ordinalpos=3379&amp;ncbi_uid=6075188&amp;link_uid=PMC4004623" image-link="/pmc/articles/PMC4004623/figure/fig5/" class="imagepopup">Figure 5.  From: Catalytic Enantioselective 1,3-Dipolar Cycloadditions of Azomethine Ylides for Biology-Oriented Synthesis. </a></p><div class="supp"><p class="details"><div>Representative examples of the Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition.</div></p></div><div class="aux"><div class="resc">Rishikesh Narayan, et al. Acc Chem Res. 2014 April 15;47(4):1296-1310.</div><p class="links"><a class="dblinks" href="/pubmed/24730692" ref="ordinalpos=3379&amp;ncbi_uid=6075188&amp;link_uid=24730692">Citation</a><a class="dblinks" href="/pmc/articles/PMC4004623/" ref="ordinalpos=3379&amp;ncbi_uid=6075188&amp;link_uid=PMC4004623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3380.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507315/figure/fig9/" ref="ordinalpos=3380&amp;ncbi_uid=4461468&amp;link_uid=PMC3507315" href="/pmc/articles/PMC3507315/"><img src="/pmc/articles/instance/3507315/bin/MI2012-563709.009.gif" src-large="/pmc/articles/instance/3507315/bin/MI2012-563709.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507315/" ref="ordinalpos=3380&amp;ncbi_uid=4461468&amp;link_uid=PMC3507315" image-link="/pmc/articles/PMC3507315/figure/fig9/" class="imagepopup">Figure 9.  From: Clinical Application of Development of Nonantibiotic Macrolides That Correct Inflammation-Driven Immune Dysfunction in Inflammatory Skin Diseases. </a></p><div class="supp"><p class="details"><div>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> extracted from KEGG database [8].</div></p></div><div class="aux"><div class="resc">Carmen Rodriguez-Cerdeira, et al. Mediators Inflamm. 2012;2012:563709.</div><p class="links"><a class="dblinks" href="/pubmed/23258954" ref="ordinalpos=3380&amp;ncbi_uid=4461468&amp;link_uid=23258954">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507315/" ref="ordinalpos=3380&amp;ncbi_uid=4461468&amp;link_uid=PMC3507315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3381.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3391993/figure/F1/" ref="ordinalpos=3381&amp;ncbi_uid=4109106&amp;link_uid=PMC3391993" href="/pmc/articles/PMC3391993/"><img src="/pmc/articles/instance/3391993/bin/1479-5876-10-85-1.gif" src-large="/pmc/articles/instance/3391993/bin/1479-5876-10-85-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3391993/" ref="ordinalpos=3381&amp;ncbi_uid=4109106&amp;link_uid=PMC3391993" image-link="/pmc/articles/PMC3391993/figure/F1/" class="imagepopup">Figure 1.  From: The role of BRAF V600 mutation in melanoma. </a></p><div class="supp"><p class="details"><div>Oncogenic BRAF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. RTK: receptor tyrosine kinase.</div></p></div><div class="aux"><div class="resc">Paolo A Ascierto, et al. J Transl Med. 2012;10:85-85.</div><p class="links"><a class="dblinks" href="/pubmed/22554099" ref="ordinalpos=3381&amp;ncbi_uid=4109106&amp;link_uid=22554099">Citation</a><a class="dblinks" href="/pmc/articles/PMC3391993/" ref="ordinalpos=3381&amp;ncbi_uid=4109106&amp;link_uid=PMC3391993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3382.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3315579/figure/pone-0033684-g004/" ref="ordinalpos=3382&amp;ncbi_uid=3888654&amp;link_uid=PMC3315579" href="/pmc/articles/PMC3315579/"><img src="/pmc/articles/instance/3315579/bin/pone.0033684.g004.gif" src-large="/pmc/articles/instance/3315579/bin/pone.0033684.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3315579/" ref="ordinalpos=3382&amp;ncbi_uid=3888654&amp;link_uid=PMC3315579" image-link="/pmc/articles/PMC3315579/figure/pone-0033684-g004/" class="imagepopup">Figure 4. Role of PTEN in the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Loss of PTEN Is Not Associated with Poor Survival in Newly Diagnosed Glioblastoma Patients of the Temozolomide Era. </a></p><div class="aux"><div class="resc">Christine Carico, et al. PLoS One. 2012;7(3):e33684.</div><p class="links"><a class="dblinks" href="/pubmed/22479427" ref="ordinalpos=3382&amp;ncbi_uid=3888654&amp;link_uid=22479427">Citation</a><a class="dblinks" href="/pmc/articles/PMC3315579/" ref="ordinalpos=3382&amp;ncbi_uid=3888654&amp;link_uid=PMC3315579">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3383.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2945552/figure/F11/" ref="ordinalpos=3383&amp;ncbi_uid=3257039&amp;link_uid=PMC2945552" href="/pmc/articles/PMC2945552/"><img src="/pmc/articles/instance/2945552/bin/zbc0431033770011.gif" src-large="/pmc/articles/instance/2945552/bin/zbc0431033770011.jpg" alt="FIGURE 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2945552/" ref="ordinalpos=3383&amp;ncbi_uid=3257039&amp;link_uid=PMC2945552" image-link="/pmc/articles/PMC2945552/figure/F11/" class="imagepopup">FIGURE 11. From: Tilapia Hepcidin 2-3 Peptide Modulates Lipopolysaccharide-induced Cytokines and Inhibits Tumor Necrosis Factor-? through Cyclooxygenase-2 and Phosphodiesterase 4D. </a></p><div class="supp"><p class="details"><div><b>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulated by TH2-3 in LPS-induced RAW264.7 cells.</b></div></p></div><div class="aux"><div class="resc">Venugopal Rajanbabu, et al. J Biol Chem. 2010 October 1;285(40):30577-30586.</div><p class="links"><a class="dblinks" href="/pubmed/20675368" ref="ordinalpos=3383&amp;ncbi_uid=3257039&amp;link_uid=20675368">Citation</a><a class="dblinks" href="/pmc/articles/PMC2945552/" ref="ordinalpos=3383&amp;ncbi_uid=3257039&amp;link_uid=PMC2945552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3384.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2909145/figure/pone-0011568-g001/" ref="ordinalpos=3384&amp;ncbi_uid=2380117&amp;link_uid=PMC2909145" href="/pmc/articles/PMC2909145/"><img src="/pmc/articles/instance/2909145/bin/pone.0011568.g001.gif" src-large="/pmc/articles/instance/2909145/bin/pone.0011568.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2909145/" ref="ordinalpos=3384&amp;ncbi_uid=2380117&amp;link_uid=PMC2909145" image-link="/pmc/articles/PMC2909145/figure/pone-0011568-g001/" class="imagepopup">Figure 1. Cartoon representation of the canonical MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Signal Response Sensitivity in the Yeast Mitogen-Activated Protein Kinase Cascade. </a></p><div class="aux"><div class="resc">Craig J. Thalhauser, et al. PLoS One. 2010;5(7):e11568.</div><p class="links"><a class="dblinks" href="/pubmed/20668519" ref="ordinalpos=3384&amp;ncbi_uid=2380117&amp;link_uid=20668519">Citation</a><a class="dblinks" href="/pmc/articles/PMC2909145/" ref="ordinalpos=3384&amp;ncbi_uid=2380117&amp;link_uid=PMC2909145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3385.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2890228/figure/F1/" ref="ordinalpos=3385&amp;ncbi_uid=2344591&amp;link_uid=PMC2890228" href="/pmc/articles/PMC2890228/"><img src="/pmc/articles/instance/2890228/bin/nihms37993f1.gif" src-large="/pmc/articles/instance/2890228/bin/nihms37993f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2890228/" ref="ordinalpos=3385&amp;ncbi_uid=2344591&amp;link_uid=PMC2890228" image-link="/pmc/articles/PMC2890228/figure/F1/" class="imagepopup">Figure 1. An insulin-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates dauer arrest in C. elegans.  From: Dauer. </a></p><div class="supp"><p class="details"><div>See text for details.</div></p></div><div class="aux"><div class="resc">Patrick J. Hu. WormBook. :1-19.</div><p class="links"><a class="dblinks" href="/pubmed/17988074" ref="ordinalpos=3385&amp;ncbi_uid=2344591&amp;link_uid=17988074">Citation</a><a class="dblinks" href="/pmc/articles/PMC2890228/" ref="ordinalpos=3385&amp;ncbi_uid=2344591&amp;link_uid=PMC2890228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3386.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1351030/figure/F8/" ref="ordinalpos=3386&amp;ncbi_uid=571570&amp;link_uid=PMC1351030" href="/pmc/articles/PMC1351030/"><img src="/pmc/articles/instance/1351030/bin/nihms2404f8.gif" src-large="/pmc/articles/instance/1351030/bin/nihms2404f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1351030/" ref="ordinalpos=3386&amp;ncbi_uid=571570&amp;link_uid=PMC1351030" image-link="/pmc/articles/PMC1351030/figure/F8/" class="imagepopup">Figure 8.  From: Kr?ppel-like factor 5 mediates the transforming activity of oncogenic H-Ras. </a></p><div class="supp"><p class="details"><div>A model of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from oncogenic H-Ras to increased cell proliferation and transformation</div></p></div><div class="aux"><div class="resc">Mandayam O Nandan, et al. Oncogene. ;23(19):3404-3413.</div><p class="links"><a class="dblinks" href="/pubmed/15077182" ref="ordinalpos=3386&amp;ncbi_uid=571570&amp;link_uid=15077182">Citation</a><a class="dblinks" href="/pmc/articles/PMC1351030/" ref="ordinalpos=3386&amp;ncbi_uid=571570&amp;link_uid=PMC1351030">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3387.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3122035/figure/F2/" ref="ordinalpos=3387&amp;ncbi_uid=3038730&amp;link_uid=PMC3122035" href="/pmc/articles/PMC3122035/"><img src="/pmc/articles/instance/3122035/bin/psb0601_0164_fig002.gif" src-large="/pmc/articles/instance/3122035/bin/psb0601_0164_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3122035/" ref="ordinalpos=3387&amp;ncbi_uid=3038730&amp;link_uid=PMC3122035" image-link="/pmc/articles/PMC3122035/figure/F2/" class="imagepopup">Figure 2.  From: New paradigm in ethylene <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model of complex formation at the ER membrane and downstream signal transfer in ethylene <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Melanie MA Bisson, et al. Plant Signal Behav. 2011 January;6(1):164-166.</div><p class="links"><a class="dblinks" href="/pubmed/21242723" ref="ordinalpos=3387&amp;ncbi_uid=3038730&amp;link_uid=21242723">Citation</a><a class="dblinks" href="/pmc/articles/PMC3122035/" ref="ordinalpos=3387&amp;ncbi_uid=3038730&amp;link_uid=PMC3122035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3388.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711318/figure/F10/" ref="ordinalpos=3388&amp;ncbi_uid=6366265&amp;link_uid=PMC3711318" href="/pmc/articles/PMC3711318/"><img src="/pmc/articles/instance/3711318/bin/zbc0331355680010.gif" src-large="/pmc/articles/instance/3711318/bin/zbc0331355680010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711318/" ref="ordinalpos=3388&amp;ncbi_uid=6366265&amp;link_uid=PMC3711318" image-link="/pmc/articles/PMC3711318/figure/F10/" class="imagepopup">FIGURE 10. From: 4-Hydroxynonenal Induces G2/M Phase Cell Cycle Arrest by Activation of the Ataxia Telangiectasia Mutated and Rad3-related Protein (ATR)/Checkpoint Kinase 1 (Chk1) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Proposed model for activation of the ATR/Chk1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and G<sub>2</sub>/M phase cell cycle arrest induced by HNE.</b> HNE treatment stimulates a DNA damage <span class="highlight" style="background-color:">signaling</span> that is mediated by sensor protein ATR. Upon activation, ATR activates the phosphorylation of checkpoint kinase, Chk1, which phosphorylates/inactivates Cdc25C. Chk1-mediated phosphorylation/inactivation of Cdc25C and p53-independent activation of p21 both partially play a major role in deactivation of CDK1 kinase activity and dissociation of CDK1-cyclin B1 complex, leading to G<sub>2</sub>/M phase arrest by HNE. Simultaneously, HNE also stimulates the phosphorylation of H2A.X at Ser-139 and initiates the signal for the recruitment of DNA repair machinery.</div></p></div><div class="aux"><div class="resc">Pankaj Chaudhary, et al. J Biol Chem. 2013 July 12;288(28):20532-20546.</div><p class="links"><a class="dblinks" href="/pubmed/23733185" ref="ordinalpos=3388&amp;ncbi_uid=6366265&amp;link_uid=23733185">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711318/" ref="ordinalpos=3388&amp;ncbi_uid=6366265&amp;link_uid=PMC3711318">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3389.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3429047/figure/F5/" ref="ordinalpos=3389&amp;ncbi_uid=4200498&amp;link_uid=PMC3429047" href="/pmc/articles/PMC3429047/"><img src="/pmc/articles/instance/3429047/bin/fphys-03-00273-g0005.gif" src-large="/pmc/articles/instance/3429047/bin/fphys-03-00273-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3429047/" ref="ordinalpos=3389&amp;ncbi_uid=4200498&amp;link_uid=PMC3429047" image-link="/pmc/articles/PMC3429047/figure/F5/" class="imagepopup">Figure 5.  From: The role of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in incretin hormone production and function. </a></p><div class="supp"><p class="details"><div><b>A simplified illustration of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> In the absence of Wnt ligand stimulation, β-cat is located within the “destruction complex”, phosphorylated by GSK-3 and CK-1α at Ser33 and adjacent serine positions, which leads to proteasome-mediated degradation process. <b>(B)</b> Following Wnt ligand stimulation, the destruction complex disassembles, resulting in free β-cat accumulation. It then enters the nucleus and forms the bipartite transcription factor β-cat/TCF, which leads to the stimulation of Wnt target gene expression. Lithium is an inhibitor of GSK-3, while cAMP <span class="highlight" style="background-color:">signaling</span> and insulin are able to stimulate β-cat Ser675 phosphorylation. Thus, in certain cell lineages, β-cat/TCF can also mediate the effect of lithium, cAMP, and insulin on Wnt target gene expression.</div></p></div><div class="aux"><div class="resc">Yu-ting A. Chiang, et al. Front Physiol. 2012;3:273.</div><p class="links"><a class="dblinks" href="/pubmed/22934027" ref="ordinalpos=3389&amp;ncbi_uid=4200498&amp;link_uid=22934027">Citation</a><a class="dblinks" href="/pmc/articles/PMC3429047/" ref="ordinalpos=3389&amp;ncbi_uid=4200498&amp;link_uid=PMC3429047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3390.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3903228/figure/fig07/" ref="ordinalpos=3390&amp;ncbi_uid=6417002&amp;link_uid=PMC3903228" href="/pmc/articles/PMC3903228/"><img src="/pmc/articles/instance/3903228/bin/pnas.1321510111fig07.gif" src-large="/pmc/articles/instance/3903228/bin/pnas.1321510111fig07.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3903228/" ref="ordinalpos=3390&amp;ncbi_uid=6417002&amp;link_uid=PMC3903228" image-link="/pmc/articles/PMC3903228/figure/fig07/" class="imagepopup">Fig. 7. From: Hypoxia-inducible factors mediate coordinated RhoA-ROCK1 expression and <span class="highlight" style="background-color:">signaling</span> in breast cancer cells. </a></p><div class="supp"><p class="details"><div>Consequences of coordinate activation of <i>RHOA</i> and <i>ROCK1</i> by HIFs. Schematic diagram of hypoxia-induced and HIF-regulated RhoA/ROCK1 expression, which inhibits MYPT activity and increases MLC phosphorylation, leading to actin polymerization, cell motility, actin-myosin contractility, and the formation of focal adhesions which promote FAK activation. The pathways converge to promote cell motility, an essential step required for invasion and metastasis. Results from this study show that hypoxia promotes motility through the RhoA → ROCK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Blue: transcriptional regulation. Black: <span class="highlight" style="background-color:">signaling</span> mechanisms.</div></p></div><div class="aux"><div class="resc">Daniele M. Gilkes, et al. Proc Natl Acad Sci U S A. 2014 January 21;111(3):E384-E393.</div><p class="links"><a class="dblinks" href="/pubmed/24324133" ref="ordinalpos=3390&amp;ncbi_uid=6417002&amp;link_uid=24324133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3903228/" ref="ordinalpos=3390&amp;ncbi_uid=6417002&amp;link_uid=PMC3903228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3391.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109034/figure/F6/" ref="ordinalpos=3391&amp;ncbi_uid=3318716&amp;link_uid=PMC3109034" href="/pmc/articles/PMC3109034/"><img src="/pmc/articles/instance/3109034/bin/z7g9991114760006.gif" src-large="/pmc/articles/instance/3109034/bin/z7g9991114760006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109034/" ref="ordinalpos=3391&amp;ncbi_uid=3318716&amp;link_uid=PMC3109034" image-link="/pmc/articles/PMC3109034/figure/F6/" class="imagepopup">Figure 6. From: Coordinated Regulation of Extracellular Matrix Synthesis by the MicroRNA-29 Family in the Trabecular Meshwork. </a></p><div class="supp"><p class="details"><div>Working model for the role of the miR-29 family in TGF-β2–mediated <span class="highlight" style="background-color:">signaling</span>. TGF-β2 stimulation activates SMAD3 <span class="highlight" style="background-color:">signaling</span>, which is necessary for both miR-29b expression and ECM induction by TGF-β2. Concurrently, TGF-β2 suppresses miR-29b levels by an unidentified molecular <span class="highlight" style="background-color:">pathway</span> dominant over that mediated by SMAD3 <span class="highlight" style="background-color:">signaling</span>. TGF-β2 stimulation also results in the induction of miR-29a expression. Together, the miR-29 family functions as an important modulator of ECM synthesis by TGF-β2.</div></p></div><div class="aux"><div class="resc">Guadalupe Villarreal, et al. Invest Ophthalmol Vis Sci. 2011 May;52(6):3391-3397.</div><p class="links"><a class="dblinks" href="/pubmed/21330653" ref="ordinalpos=3391&amp;ncbi_uid=3318716&amp;link_uid=21330653">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109034/" ref="ordinalpos=3391&amp;ncbi_uid=3318716&amp;link_uid=PMC3109034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3392.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469622/figure/pone-0046725-g001/" ref="ordinalpos=3392&amp;ncbi_uid=4297716&amp;link_uid=PMC3469622" href="/pmc/articles/PMC3469622/"><img src="/pmc/articles/instance/3469622/bin/pone.0046725.g001.gif" src-large="/pmc/articles/instance/3469622/bin/pone.0046725.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469622/" ref="ordinalpos=3392&amp;ncbi_uid=4297716&amp;link_uid=PMC3469622" image-link="/pmc/articles/PMC3469622/figure/pone-0046725-g001/" class="imagepopup">Figure 1. Canonical TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Quantitative Phosphoproteomics Reveals SLP-76 Dependent Regulation of PAG and Src Family Kinases in T Cells. </a></p><div class="supp"><p class="details"><div>Established <span class="highlight" style="background-color:">signaling</span> cascades in activated T cells. Proteins identified in our quantitative phosphoproteomic analysis are highlighted in red and the detailed data regarding these proteins, their phosphorylation kinetics in both SLP-76 reconstituted and deficient Jurkat T cells upon TCR stimulation, were presented later in Figure 3 and Figure S3.</div></p></div><div class="aux"><div class="resc">Lulu Cao, et al. PLoS One. 2012;7(10):e46725.</div><p class="links"><a class="dblinks" href="/pubmed/23071622" ref="ordinalpos=3392&amp;ncbi_uid=4297716&amp;link_uid=23071622">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469622/" ref="ordinalpos=3392&amp;ncbi_uid=4297716&amp;link_uid=PMC3469622">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3393.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355939/figure/F4/" ref="ordinalpos=3393&amp;ncbi_uid=4026118&amp;link_uid=PMC3355939" href="/pmc/articles/PMC3355939/"><img src="/pmc/articles/instance/3355939/bin/fendo-02-00030-g004.gif" src-large="/pmc/articles/instance/3355939/bin/fendo-02-00030-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355939/" ref="ordinalpos=3393&amp;ncbi_uid=4026118&amp;link_uid=PMC3355939" image-link="/pmc/articles/PMC3355939/figure/F4/" class="imagepopup">Figure 4.  From: ERK, Akt, and STAT5 are Differentially Activated by the Two Growth Hormone Receptor Subtypes of a Teleost Fish (Oncorhynchus Mykiss). </a></p><div class="supp"><p class="details"><div><b>Model of differential activation of <span class="highlight" style="background-color:">signaling</span> pathways by growth hormone receptor (GHR) subtypes</b>. JAK2 activation is essential for propagation of <span class="highlight" style="background-color:">signaling</span> from both GHR1 and GHR2 to the ERK, PI3K/Akt, and STAT5 pathways (details of <span class="highlight" style="background-color:">pathway</span> elements are omitted for simplicity). Cross-talk occurs between the ERK and PI3K/Akt pathways, possibly through Akt activation of c-Raf in the ERK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jeffrey D. Kittilson, et al. Front Endocrinol (Lausanne). 2011;2:30.</div><p class="links"><a class="dblinks" href="/pubmed/22649371" ref="ordinalpos=3393&amp;ncbi_uid=4026118&amp;link_uid=22649371">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355939/" ref="ordinalpos=3393&amp;ncbi_uid=4026118&amp;link_uid=PMC3355939">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3394.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3972037/figure/F4/" ref="ordinalpos=3394&amp;ncbi_uid=5964245&amp;link_uid=PMC3972037" href="/pmc/articles/PMC3972037/"><img src="/pmc/articles/instance/3972037/bin/nihms563728f4.gif" src-large="/pmc/articles/instance/3972037/bin/nihms563728f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3972037/" ref="ordinalpos=3394&amp;ncbi_uid=5964245&amp;link_uid=PMC3972037" image-link="/pmc/articles/PMC3972037/figure/F4/" class="imagepopup">Fig. 4.  From: PTEN Loss Does Not Predict for Response to RAD001 (Everolimus) in a Glioblastoma Orthotopic Xenograft Test Panel. </a></p><div class="supp"><p class="details"><div>Western blot analysis of mTOR <span class="highlight" style="background-color:">signaling</span>. Flank tumor specimens were evaluated for activation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Western blotting with phosphorylation-specific and nonphosphorylation antibodies as indicated.</div></p></div><div class="aux"><div class="resc">Lin Yang, et al. Clin Cancer Res. 2008 June 15;14(12):3993-4001.</div><p class="links"><a class="dblinks" href="/pubmed/18559622" ref="ordinalpos=3394&amp;ncbi_uid=5964245&amp;link_uid=18559622">Citation</a><a class="dblinks" href="/pmc/articles/PMC3972037/" ref="ordinalpos=3394&amp;ncbi_uid=5964245&amp;link_uid=PMC3972037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3395.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1934587/figure/F11/" ref="ordinalpos=3395&amp;ncbi_uid=1007728&amp;link_uid=PMC1934587" href="/pmc/articles/PMC1934587/"><img src="/pmc/articles/instance/1934587/bin/JCI0731807.f11.gif" src-large="/pmc/articles/instance/1934587/bin/JCI0731807.f11.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1934587/" ref="ordinalpos=3395&amp;ncbi_uid=1007728&amp;link_uid=PMC1934587" image-link="/pmc/articles/PMC1934587/figure/F11/" class="imagepopup">Figure 11. Model of the regulation of Foxo3 subcellular localization in erythropoiesis..  From: Foxo3 is required for the regulation of oxidative stress in erythropoiesis. </a></p><div class="supp"><p class="details"><div>Inhibition of EpoR <span class="highlight" style="background-color:">signaling</span>, which includes inhibition of the PI3-kinase/AKT <span class="highlight" style="background-color:">pathway</span>, as a result of downregulation of EpoR on erythroblasts, coupled with accumulation of hemoglobin, may result in ROS <span class="highlight" style="background-color:">signaling</span>, leading to Foxo3 nuclear localization and activation.</div></p></div><div class="aux"><div class="resc">Dragan Marinkovic, et al. J Clin Invest. 2007 August 1;117(8):2133-2144.</div><p class="links"><a class="dblinks" href="/pubmed/17671650" ref="ordinalpos=3395&amp;ncbi_uid=1007728&amp;link_uid=17671650">Citation</a><a class="dblinks" href="/pmc/articles/PMC1934587/" ref="ordinalpos=3395&amp;ncbi_uid=1007728&amp;link_uid=PMC1934587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3396.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2799114/figure/F6/" ref="ordinalpos=3396&amp;ncbi_uid=2136038&amp;link_uid=PMC2799114" href="/pmc/articles/PMC2799114/"><img src="/pmc/articles/instance/2799114/bin/nihms129394f6.gif" src-large="/pmc/articles/instance/2799114/bin/nihms129394f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2799114/" ref="ordinalpos=3396&amp;ncbi_uid=2136038&amp;link_uid=PMC2799114" image-link="/pmc/articles/PMC2799114/figure/F6/" class="imagepopup">Figure 6. Many Wnt <span class="highlight" style="background-color:">pathway</span> genes are expressed in the developing pituitary.  From: WNT <span class="highlight" style="background-color:">signaling</span> affects gene expression in the ventral diencephalon and pituitary gland growth. </a></p><div class="supp"><p class="details"><div>RT-PCR was used to detect expression of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members and regulators during various times of pituitary gland development. RNA from the specified ages was analyzed using intron-spanning primers specific to each <i>Wnt</i> and <i>Frizzled</i> gene. For positive controls, cDNAs generated from adult and embryonic tissues were chosen that had previously been reported to express each <i>Wnt</i> or <i>Fzd</i> gene. Water was used as a negative control. HPRT PCR products showing +RT and −RT reactions confirm no genomic contamination in the e12.5, e14.5 or adult pituitary cDNA. PCR products were sequenced to confirm the identity of each gene.</div></p></div><div class="aux"><div class="resc">Mary Anne Potok, et al. Dev Dyn. ;237(4):1006-1020.</div><p class="links"><a class="dblinks" href="/pubmed/18351662" ref="ordinalpos=3396&amp;ncbi_uid=2136038&amp;link_uid=18351662">Citation</a><a class="dblinks" href="/pmc/articles/PMC2799114/" ref="ordinalpos=3396&amp;ncbi_uid=2136038&amp;link_uid=PMC2799114">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3397.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676112/figure/F7/" ref="ordinalpos=3397&amp;ncbi_uid=5547661&amp;link_uid=PMC3676112" href="/pmc/articles/PMC3676112/"><img src="/pmc/articles/instance/3676112/bin/zjv9990976970007.gif" src-large="/pmc/articles/instance/3676112/bin/zjv9990976970007.jpg" alt="Fig 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676112/" ref="ordinalpos=3397&amp;ncbi_uid=5547661&amp;link_uid=PMC3676112" image-link="/pmc/articles/PMC3676112/figure/F7/" class="imagepopup">Fig 7.  From: Genome-Wide Analysis of Host mRNA Translation during Hepatitis C Virus Infection. </a></p><div class="supp"><p class="details"><div>Translation regulation of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and translation process. (A) Representation of the mTOR <span class="highlight" style="background-color:">signaling</span> KEGG <span class="highlight" style="background-color:">pathway</span>, significantly associated with translationally regulated genes. The translationally upregulated genes are shown in red and the translationally downregulated genes in green. The direction of transcriptional regulation (<i>P</i> &lt; 0.05, fold change &gt; 2) is indicated by arrows beside the gene names. (B) Representation of the network of interactions between the translationally regulated proteins related to translation (obtained with Cytoscape and the MiMI plugin). As for the KEGG <span class="highlight" style="background-color:">pathway</span>, the translationally upregulated genes are shown in red and the translationally downregulated genes in green, and potential transcriptional regulations are indicated by arrows beside the circles.</div></p></div><div class="aux"><div class="resc">Hélène Colman, et al. J Virol. 2013 June;87(12):6668-6677.</div><p class="links"><a class="dblinks" href="/pubmed/23552407" ref="ordinalpos=3397&amp;ncbi_uid=5547661&amp;link_uid=23552407">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676112/" ref="ordinalpos=3397&amp;ncbi_uid=5547661&amp;link_uid=PMC3676112">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3398.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2758290/figure/F2/" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=PMC2758290" href="/pmc/articles/PMC2758290/"><img src="/pmc/articles/instance/2758290/bin/nihms137649f2a.gif" src-large="/pmc/articles/instance/2758290/bin/nihms137649f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2758290/figure/F2/" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=PMC2758290" href="/pmc/articles/PMC2758290/"><img src="/pmc/articles/instance/2758290/bin/nihms137649f2b.gif" src-large="/pmc/articles/instance/2758290/bin/nihms137649f2b.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2758290/figure/F2/" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=PMC2758290" href="/pmc/articles/PMC2758290/"><img src="/pmc/articles/instance/2758290/bin/nihms137649f2c.gif" src-large="/pmc/articles/instance/2758290/bin/nihms137649f2c.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758290/" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=PMC2758290" image-link="/pmc/articles/PMC2758290/figure/F2/" class="imagepopup">Figure 2.  From: Esterases and autoimmunity: the Sialic acid acetylesterase <span class="highlight" style="background-color:">pathway</span> and the regulation of peripheral B cell tolerance. </a></p><div class="supp"><p class="details"><div>Possible mechanisms by which the SIAE-Siglec <span class="highlight" style="background-color:">pathway</span> could mediate tolerance at the B cell level. (A) Within the B cell follicles of lymph nodes, inhibitory signals prevent weakly autoreactive B cells from migrating to the T cell zone and receiving T cell help in the form of activating signals. In contrast defective inhibitory <span class="highlight" style="background-color:">signaling</span> via the SIAE-Siglec <span class="highlight" style="background-color:">pathway</span> allows autoreactive B cells to migrate to the T cell zone where they receive T cell help, undergo somatic mutation and secrete high affinity autoantibodies. (B) Within the germinal center, inhibitory <span class="highlight" style="background-color:">signaling</span> prevents autoreactive B cells from being positively selected in the light zone. Attenuation of SIAE-Siglec inhibition increases the probability of selection of low affinity autoreactive B cell clones. (C) Inhibitory signals mediated by the SIAE-Siglec <span class="highlight" style="background-color:">pathway</span> normally prevent B cells from serving as efficient auto-antigen presenting cells however disruption of this <span class="highlight" style="background-color:">pathway</span> allows B cell presentation to autoreactive T cells.</div></p></div><div class="aux"><div class="resc">Shiv Pillai, et al. Trends Immunol. ;30(10):488-493.</div><p class="links"><a class="dblinks" href="/pubmed/19766537" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=19766537">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758290/" ref="ordinalpos=3398&amp;ncbi_uid=2495905&amp;link_uid=PMC2758290">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3399.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4316815/figure/F2/" ref="ordinalpos=3399&amp;ncbi_uid=7125520&amp;link_uid=PMC4316815" href="/pmc/articles/PMC4316815/"><img src="/pmc/articles/instance/4316815/bin/nihms602501f2.gif" src-large="/pmc/articles/instance/4316815/bin/nihms602501f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4316815/" ref="ordinalpos=3399&amp;ncbi_uid=7125520&amp;link_uid=PMC4316815" image-link="/pmc/articles/PMC4316815/figure/F2/" class="imagepopup">Fig. 2.  From: Zinc and the ERK Kinases in the Developing Brain. </a></p><div class="supp"><p class="details"><div>Regulation by zinc of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> cascade. Zinc could activate the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at various levels: <i>1</i> zinc inhibits phosphatases that are involved in ERK1/2 dephosphorylation, and zinc inhibits kinases that indirectly regulate the phosphorylation levels of receptors that subsequently activate the ERK cascade; <i>2</i> zinc can directly bind to and activate receptors (e.g., GPR39) which have ERK1/2 as a downstream <span class="highlight" style="background-color:">signaling</span> cascade; and <i>3</i> zinc can activate MMPs, which subsequently cleave and activate substrates (e.g., pro-BDNF) releasing ligands (e.g. BDNF) that activate receptors upstream from the ERK1/2 cascade</div></p></div><div class="aux"><div class="resc">J. R. Nuttall, et al. Neurotox Res. ;21(1):128-141.</div><p class="links"><a class="dblinks" href="/pubmed/22095091" ref="ordinalpos=3399&amp;ncbi_uid=7125520&amp;link_uid=22095091">Citation</a><a class="dblinks" href="/pmc/articles/PMC4316815/" ref="ordinalpos=3399&amp;ncbi_uid=7125520&amp;link_uid=PMC4316815">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3400.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3063194/figure/F1/" ref="ordinalpos=3400&amp;ncbi_uid=2844022&amp;link_uid=PMC3063194" href="/pmc/articles/PMC3063194/"><img src="/pmc/articles/instance/3063194/bin/2042-4280-2-1-1.gif" src-large="/pmc/articles/instance/3063194/bin/2042-4280-2-1-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3063194/" ref="ordinalpos=3400&amp;ncbi_uid=2844022&amp;link_uid=PMC3063194" image-link="/pmc/articles/PMC3063194/figure/F1/" class="imagepopup">Figure 1.  From: Innate immune modulation in EBV infection. </a></p><div class="supp"><p class="details"><div><b>Elicitation of innate immune response in EBV infection</b>. EBV dUTPase and GP350 act as ligands for TLR2. EBV EBERs can mount innate immune responses via both TLR3 and RIG-I <span class="highlight" style="background-color:">signaling</span> pathways. Infection of EBV also activates TLR9 <span class="highlight" style="background-color:">signaling</span> leading to IFNα production in pDCs. In latency, EBV manipulates the TLR7/IRF5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which promotes cell proliferation. EBV products are indicated in red fonts.</div></p></div><div class="aux"><div class="resc">Shunbin Ning. Herpesviridae. 2011;2:1-1.</div><p class="links"><a class="dblinks" href="/pubmed/21429244" ref="ordinalpos=3400&amp;ncbi_uid=2844022&amp;link_uid=21429244">Citation</a><a class="dblinks" href="/pmc/articles/PMC3063194/" ref="ordinalpos=3400&amp;ncbi_uid=2844022&amp;link_uid=PMC3063194">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="33" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="34" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="35" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:42:03-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A24205431220100000000000B67EE;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A24205431220100000000000B67EE" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>