
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="32" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A7BD7543116010000000000B2A8FF" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="31" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="32" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="33" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="32" /></div>
            <div><h2 class="result_count">Results: 3101 to 3200 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3101.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3852000/figure/F4/" ref="ordinalpos=3101&amp;ncbi_uid=5567064&amp;link_uid=PMC3852000" href="/pmc/articles/PMC3852000/"><img src="/pmc/articles/instance/3852000/bin/1471-2407-13-471-4.gif" src-large="/pmc/articles/instance/3852000/bin/1471-2407-13-471-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3852000/" ref="ordinalpos=3101&amp;ncbi_uid=5567064&amp;link_uid=PMC3852000" image-link="/pmc/articles/PMC3852000/figure/F4/" class="imagepopup">Figure 4.  From: Prognostic significance and therapeutic potential of the activation of anaplastic lymphoma kinase/protein kinase B/mammalian target of rapamycin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in anaplastic large cell lymphoma. </a></p><div class="supp"><p class="details"><div><b>Overexpression of NPM-ALK activated the AKT/mTOR <span class="highlight" style="background-color:">pathway</span>.</b> Overexpression of NPM-ALK in BaF3 cells induced hyper-activation of the AKT/mTOR <span class="highlight" style="background-color:">pathway</span>, as demonstrated by hyperphosphorylation of AKT and downstream molecules of mTOR: 4E-BP1 and p70SK6.</div></p></div><div class="aux"><div class="resc">Ju Gao, et al. BMC Cancer. 2013;13:471-471.</div><p class="links"><a class="dblinks" href="/pubmed/24112608" ref="ordinalpos=3101&amp;ncbi_uid=5567064&amp;link_uid=24112608">Citation</a><a class="dblinks" href="/pmc/articles/PMC3852000/" ref="ordinalpos=3101&amp;ncbi_uid=5567064&amp;link_uid=PMC3852000">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3102.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC130620/figure/F4/" ref="ordinalpos=3102&amp;ncbi_uid=252090&amp;link_uid=PMC130620" href="/pmc/articles/PMC130620/"><img src="/pmc/articles/instance/130620/bin/pq2023557004.gif" src-large="/pmc/articles/instance/130620/bin/pq2023557004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC130620/" ref="ordinalpos=3102&amp;ncbi_uid=252090&amp;link_uid=PMC130620" image-link="/pmc/articles/PMC130620/figure/F4/" class="imagepopup">Figure 4.  From: Calpain as an effector of the Gq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for inhibition of Wnt/?-catenin-regulated cell proliferation. </a></p><div class="supp"><p class="details"><div>Constitutively active TCF3 rescues effect of Gq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on TCF/LEF-1-mediated transcriptional activity and cell proliferation in SW480 cells. (<i>a</i>) A constitutively activated form of TCF3, ΔN-XTCF3-VP16, rescues the effect of Gq <span class="highlight" style="background-color:">signaling</span> on TCF/LEF-1-mediated transcriptional activity in SW480 cells. SW480 cells were transfected with empty plasmid as a control or indicated expression plasmids. Relative TCF/LEF-1 transcriptional activity was assayed as described. (<i>b</i>) ΔN-XTCF3-VP16 rescues the effect of Gq <span class="highlight" style="background-color:">signaling</span> on cyclin D1 expression in SW480 cells. SW480 cells stably expressing M3R alone or together with ΔN-XTCF3-VP16 were treated without or with carbachol (1 mM) for 12 h. Total lysates were prepared and resolved by SDS/PAGE. Immunoblotting was performed with antibody against cyclin D1. The untreated M3R-expressing cells were used as control and β-actin was the loading control. WB, Western blot. (<i>c</i>) ΔN-XTCF3-VP16 rescues the effect of Gq <span class="highlight" style="background-color:">signaling</span> on SW480 cell proliferation. A total of 2.5 × 10<sup>3</sup> cells stably expressing M3R alone or together with ΔN-XTCF3-VP16 were plated on 96-well plates. Cells were treated with carbachol (1 mM) for 12 h as indicated. Cell proliferation was assessed colorimeterically. The untreated M3R-expressing cells were used as control. Data shown are mean ± SD.</div></p></div><div class="aux"><div class="resc">Guangnan Li, et al. Proc Natl Acad Sci U S A. 2002 October 1;99(20):13254-13259.</div><p class="links"><a class="dblinks" href="/pubmed/12239346" ref="ordinalpos=3102&amp;ncbi_uid=252090&amp;link_uid=12239346">Citation</a><a class="dblinks" href="/pmc/articles/PMC130620/" ref="ordinalpos=3102&amp;ncbi_uid=252090&amp;link_uid=PMC130620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3103.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384726/figure/F2/" ref="ordinalpos=3103&amp;ncbi_uid=4405182&amp;link_uid=PMC3384726" href="/pmc/articles/PMC3384726/"><img src="/pmc/articles/instance/3384726/bin/nihms383040f2.gif" src-large="/pmc/articles/instance/3384726/bin/nihms383040f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384726/" ref="ordinalpos=3103&amp;ncbi_uid=4405182&amp;link_uid=PMC3384726" image-link="/pmc/articles/PMC3384726/figure/F2/" class="imagepopup">Fig. 2. MPS-1 reduced FGFR3 protein expression and blocked MAPK/ErK <span class="highlight" style="background-color:">signaling</span>.  From: Ribosomal protein metallopanstimulin-1 impairs multiple myeloma CAG cells growth and inhibits fibroblast growth factor receptor 3. </a></p><div class="supp"><p class="details"><div>(a) Cell lysates were analyzed with Western blotting, which showed lower levels of FGFR3 protein in MPS-1–transfected cells (M) than in control cells (C). (b) When cultured in standard rich medium, CAG/MPS-1 cells (M) had diminished pp44/42, which indicates MAPK/ErK <span class="highlight" style="background-color:">signaling</span> activity, while control cells (C) had activated <span class="highlight" style="background-color:">signaling</span>. (c) Serum-starved cells were treated with exogenous recombinant FGF2 at low concentration (5 ng/ml) and high concentration (100 ng/ml). The MAPK/ErK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, indicated by pp44/42 level, was not activated in CAG/MPS-1 cells (M) but was activated in control cells (C). (pp44/42: phosphorylated ErK 44/42 protein; p44/42: total ErK 44/42 protein).</div></p></div><div class="aux"><div class="resc">Yuemeng Dai, et al. Clin Lymphoma Myeloma Leuk. ;11(6):490-497.</div><p class="links"><a class="dblinks" href="/pubmed/21889435" ref="ordinalpos=3103&amp;ncbi_uid=4405182&amp;link_uid=21889435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384726/" ref="ordinalpos=3103&amp;ncbi_uid=4405182&amp;link_uid=PMC3384726">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3104.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679353/figure/F4/" ref="ordinalpos=3104&amp;ncbi_uid=6331202&amp;link_uid=PMC3679353" href="/pmc/articles/PMC3679353/"><img src="/pmc/articles/instance/3679353/bin/nihms447063f4.gif" src-large="/pmc/articles/instance/3679353/bin/nihms447063f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679353/" ref="ordinalpos=3104&amp;ncbi_uid=6331202&amp;link_uid=PMC3679353" image-link="/pmc/articles/PMC3679353/figure/F4/" class="imagepopup">Figure 4.  From: The ASC/Caspase-1/IL-1? <span class="highlight" style="background-color:">signaling</span> triggers inflammatory responses by promoting HMGB1 induction in liver ischemia-reperfusion injury. </a></p><div class="supp"><p class="details"><div>ASC-mediated caspase-1 <span class="highlight" style="background-color:">signaling</span> induces HMGB1 via MAPKs <span class="highlight" style="background-color:">pathway</span> and NF-κB activation in LPS-stimulated BMMs in vitro.<br />(A) Western blot-assisted analysis of HMGB1, NF-κB, TLR4, and phospho-p38. β-actin was used as an internal control. Representative of three experiments.<br />(B) Quantitative RT-PCR-assisted detection of HMGB1. Data normalized to β-actin gene expression. *p&lt;0.05, n=3/group. Mean±SD are shown.<br />(C) Caspase-1 activity (U). Mean±SD; n=3/group. *p&lt;0.005.<br />(D) ELISA-assisted detection of IL-1β. Mean±SD; n=3/group. *p&lt;0.005.<br />(E) Quantitative RT-PCR-assisted detection of IL-1β and IL-18. Data were normalized to β-actin gene expression. *p&lt;0.005, n=3/group. Mean±SD are shown.</div></p></div><div class="aux"><div class="resc">Naoko Kamo, et al. Hepatology. ;58(1):351-362.</div><p class="links"><a class="dblinks" href="/pubmed/23408710" ref="ordinalpos=3104&amp;ncbi_uid=6331202&amp;link_uid=23408710">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679353/" ref="ordinalpos=3104&amp;ncbi_uid=6331202&amp;link_uid=PMC3679353">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3105.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC26952/figure/F7/" ref="ordinalpos=3105&amp;ncbi_uid=13044&amp;link_uid=PMC26952" href="/pmc/articles/PMC26952/"><img src="/pmc/articles/instance/26952/bin/pq1501526007.gif" src-large="/pmc/articles/instance/26952/bin/pq1501526007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC26952/" ref="ordinalpos=3105&amp;ncbi_uid=13044&amp;link_uid=PMC26952" image-link="/pmc/articles/PMC26952/figure/F7/" class="imagepopup">Figure 7.  From: Association of Smads with lymphoid enhancer binding factor 1/T cell-specific factor mediates cooperative <span class="highlight" style="background-color:">signaling</span> by the transforming growth factor-? and Wnt pathways. </a></p><div class="supp"><p class="details"><div>A model for activation of specific target genes by TGFβ and Wnt pathways. Promoters with SBEs adjacent to the LEF1/TCF-binding sites can be activated by the TGFβ <span class="highlight" style="background-color:">pathway</span> in the absence of β-catenin (<i>A</i>), by Wnt <span class="highlight" style="background-color:">signaling</span> alone (<i>B</i>), or synergistically in the presence of both TGFβ and Wnt signals (<i>C</i>).</div></p></div><div class="aux"><div class="resc">Etienne Labbé, et al. Proc Natl Acad Sci U S A. 2000 July 18;97(15):8358-8363.</div><p class="links"><a class="dblinks" href="/pubmed/10890911" ref="ordinalpos=3105&amp;ncbi_uid=13044&amp;link_uid=10890911">Citation</a><a class="dblinks" href="/pmc/articles/PMC26952/" ref="ordinalpos=3105&amp;ncbi_uid=13044&amp;link_uid=PMC26952">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3106.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4049685/figure/f2-ol-07-06-1761/" ref="ordinalpos=3106&amp;ncbi_uid=6249976&amp;link_uid=PMC4049685" href="/pmc/articles/PMC4049685/"><img src="/pmc/articles/instance/4049685/bin/OL-07-06-1761-g01.gif" src-large="/pmc/articles/instance/4049685/bin/OL-07-06-1761-g01.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4049685/" ref="ordinalpos=3106&amp;ncbi_uid=6249976&amp;link_uid=PMC4049685" image-link="/pmc/articles/PMC4049685/figure/f2-ol-07-06-1761/" class="imagepopup">Figure 2.  From: 'Druggable' alterations detected by Ion Torrent in metastatic colorectal cancer patients. </a></p><div class="supp"><p class="details"><div>Colorectal cancer metastasis <span class="highlight" style="background-color:">signaling</span> pattern. Colorectal cancer metastasis <span class="highlight" style="background-color:">signaling</span> is the the most commonly mutated canonical <span class="highlight" style="background-color:">pathway</span>, and includes Wnt, phosphoinositide 3-kinase (PI3K)/AKT and transforming growth factor (TGF)-β/SMAD <span class="highlight" style="background-color:">signaling</span>. The key mutated genes detected in the present study are highlighted in red.</div></p></div><div class="aux"><div class="resc">WEIJIA FANG, et al. Oncol Lett. 2014 June;7(6):1761-1766.</div><p class="links"><a class="dblinks" href="/pubmed/24932229" ref="ordinalpos=3106&amp;ncbi_uid=6249976&amp;link_uid=24932229">Citation</a><a class="dblinks" href="/pmc/articles/PMC4049685/" ref="ordinalpos=3106&amp;ncbi_uid=6249976&amp;link_uid=PMC4049685">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3107.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2899694/figure/F1/" ref="ordinalpos=3107&amp;ncbi_uid=2322553&amp;link_uid=PMC2899694" href="/pmc/articles/PMC2899694/"><img src="/pmc/articles/instance/2899694/bin/nihms213803f1.gif" src-large="/pmc/articles/instance/2899694/bin/nihms213803f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2899694/" ref="ordinalpos=3107&amp;ncbi_uid=2322553&amp;link_uid=PMC2899694" image-link="/pmc/articles/PMC2899694/figure/F1/" class="imagepopup">Figure 1.  From: THE TWO-COMPONENT <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span> OF BACTERIAL CHEMOTAXIS: A Molecular View of Signal Transduction by Receptors, Kinases, and Adaptation Enzymes. </a></p><div class="supp"><p class="details"><div>Information flow through a two-component <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Shown are the standard prokaryotic and eukaryotic <span class="highlight" style="background-color:">signaling</span> components including (<i>a</i>) the sensor module, typically a transmembrane receptor with two putative membrane-spanning helices; (<i>b</i>) a transmitter histidine kinase that is regulated by the receptor and catalyzes autophosphorylation on histidine; and (<i>c</i>) a receiver or response regulator whose active site catalyzes phosphotransfer from the transmitter, thereby yielding autophosphorylation on aspartate. The response regulator can catalyze its own dephosphorylation, but some pathways require a separate phosphatase to generate more rapid dephosphorylation, or to provide additional <span class="highlight" style="background-color:">pathway</span> regulation. Different pathways display highly specialized assemblages of the modular elements; e.g. the sensor, transmitter, and response regulator modules can be separate proteins or can be fused together in various combinations (see text for references).</div></p></div><div class="aux"><div class="resc">Joseph J. Falke, et al. Annu Rev Cell Dev Biol. ;13:457-512.</div><p class="links"><a class="dblinks" href="/pubmed/9442881" ref="ordinalpos=3107&amp;ncbi_uid=2322553&amp;link_uid=9442881">Citation</a><a class="dblinks" href="/pmc/articles/PMC2899694/" ref="ordinalpos=3107&amp;ncbi_uid=2322553&amp;link_uid=PMC2899694">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3108.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404301/figure/F3/" ref="ordinalpos=3108&amp;ncbi_uid=5145804&amp;link_uid=PMC3404301" href="/pmc/articles/PMC3404301/"><img src="/pmc/articles/instance/3404301/bin/zmg0081250450003.gif" src-large="/pmc/articles/instance/3404301/bin/zmg0081250450003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404301/" ref="ordinalpos=3108&amp;ncbi_uid=5145804&amp;link_uid=PMC3404301" image-link="/pmc/articles/PMC3404301/figure/F3/" class="imagepopup">Fig. 3. From: COUP-TFII Is a Major Regulator of Cell Cycle and Notch <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Up-regulation of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes in the COUP-TFII knockdown of HUVEC cells. A, Heat map of microarray analysis of members of the Notch <span class="highlight" style="background-color:">pathway</span> in HUVEC cells treated with COUP-TFII-specific siRNA or scrambled control siRNA. B, qRT-PCR analysis of <i>Notch4, Foxc1, Hey2, Hes1, Hes2, PSENEN, NCSTN, EphrinB2, EphB4, CD44, Np-1, Np-2</i>, and <i>COUP-TFII</i> expression in control and COUP-TFII-depleted HUVEC cells. <i>Error bars</i> indicate <span style="font-variant: small-caps" class="small-caps">sd</span>. *, <i>P</i> &lt; 0.05; **, <i>P</i> &lt;0.005; ***, <i>P</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Xinpu Chen, et al. Mol Endocrinol. 2012 August;26(8):1268-1277.</div><p class="links"><a class="dblinks" href="/pubmed/22734039" ref="ordinalpos=3108&amp;ncbi_uid=5145804&amp;link_uid=22734039">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404301/" ref="ordinalpos=3108&amp;ncbi_uid=5145804&amp;link_uid=PMC3404301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3109.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3610942/figure/pgen-1003414-g006/" ref="ordinalpos=3109&amp;ncbi_uid=4777408&amp;link_uid=PMC3610942" href="/pmc/articles/PMC3610942/"><img src="/pmc/articles/instance/3610942/bin/pgen.1003414.g006.gif" src-large="/pmc/articles/instance/3610942/bin/pgen.1003414.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3610942/" ref="ordinalpos=3109&amp;ncbi_uid=4777408&amp;link_uid=PMC3610942" image-link="/pmc/articles/PMC3610942/figure/pgen-1003414-g006/" class="imagepopup">Figure 6. Ethanol modulates aging-related modules through oxidative stress response <span class="highlight" style="background-color:">pathway</span>..  From: Statistical Analysis Reveals Co-Expression Patterns of Many Pairs of Genes in Yeast Are Jointly Regulated by Interacting Loci. </a></p><div class="supp"><p class="details"><div>(A) One module contains two silencing genes, <i>UBP10</i> and <i>HMLALPHA1</i>. The regulatory loci contain two oxidative stress response genes, Gre2 and Hog1.; (B) One module contains two silencing genes, <i>SPT23</i> and <i>RPD3</i>. The regulatory loci contain two oxidative stress response genes, Ste11 and Vms1; (C) A diagram showing the module A related <span class="highlight" style="background-color:">pathway</span>, which indicates a potential regulatory relationship between oxidative stress response and Ubp10 induced silencing process; (D) A diagram showing the module B related <span class="highlight" style="background-color:">pathway</span>. The epistatic interaction between the two candidates Vms1 and Ste11 indicates the potential crosstalk between caloric restriction response and oxidative stress. Moreover, Ste11 may also regulate Pbs2-Hog1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> under oxidative stress besides osmotic stress response.</div></p></div><div class="aux"><div class="resc">Lin Wang, et al. PLoS Genet. 2013 March;9(3):e1003414.</div><p class="links"><a class="dblinks" href="/pubmed/23555313" ref="ordinalpos=3109&amp;ncbi_uid=4777408&amp;link_uid=23555313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3610942/" ref="ordinalpos=3109&amp;ncbi_uid=4777408&amp;link_uid=PMC3610942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3110.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g004/" ref="ordinalpos=3110&amp;ncbi_uid=7007975&amp;link_uid=PMC4275273" href="/pmc/articles/PMC4275273/"><img src="/pmc/articles/instance/4275273/bin/pone.0115985.g004.gif" src-large="/pmc/articles/instance/4275273/bin/pone.0115985.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4275273/" ref="ordinalpos=3110&amp;ncbi_uid=7007975&amp;link_uid=PMC4275273" image-link="/pmc/articles/PMC4275273/figure/pone-0115985-g004/" class="imagepopup">Figure 4. Roles of Gαq <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in regulating behavioral choice between conflicting alternatives..  From: Dopamine Receptors Antagonistically Regulate Behavioral Choice between Conflicting Alternatives in C. elegans. </a></p><div class="supp"><p class="details"><div>(A) Phenotypes of Gα<sub>q</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutants in the interaction assay under the well-fed or starved condition. In the assay system, the Cu<sup>2+</sup> ion concentration was 100 mM, and the diacetyl concentration was 10<sup>−2</sup>. (B) Dose-response curves of wild-type N2 and mutants to diacetyl with (+) or without (-) 100 mM of Cu<sup>2+</sup> ion. Differences between groups were determined using two-way ANOVA. (C) Dose-response curves of wild-type N2 and mutants to Cu<sup>2+</sup> ion with (+) or without (-) 10<sup>−2</sup> of diacetyl. Differences between groups were determined using two-way ANOVA. (D) Locomotion behavior of wild-type N2 and mutants in the absence (-) of food under the well-fed or starved condition. Locomotion behavior was assessed by the body bend. Bars represent mean ± S.E.M. **<i>P</i>&lt;0.01 <i>vs</i> N2 (if not specially indicated).</div></p></div><div class="aux"><div class="resc">Daoyong Wang, et al. PLoS One. 2014;9(12):e115985.</div><p class="links"><a class="dblinks" href="/pubmed/25536037" ref="ordinalpos=3110&amp;ncbi_uid=7007975&amp;link_uid=25536037">Citation</a><a class="dblinks" href="/pmc/articles/PMC4275273/" ref="ordinalpos=3110&amp;ncbi_uid=7007975&amp;link_uid=PMC4275273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3111.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3565315/figure/f2-ijms-14-01179/" ref="ordinalpos=3111&amp;ncbi_uid=4694570&amp;link_uid=PMC3565315" href="/pmc/articles/PMC3565315/"><img src="/pmc/articles/instance/3565315/bin/ijms-14-01179f2.gif" src-large="/pmc/articles/instance/3565315/bin/ijms-14-01179f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3565315/" ref="ordinalpos=3111&amp;ncbi_uid=4694570&amp;link_uid=PMC3565315" image-link="/pmc/articles/PMC3565315/figure/f2-ijms-14-01179/" class="imagepopup">Figure 2.  From: Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>Overview about currently tested Hedgehog <span class="highlight" style="background-color:">pathway</span> inhibitors and their mode of action.</div></p></div><div class="aux"><div class="resc">Joanna Szkandera, et al. Int J Mol Sci. 2013 January;14(1):1179-1196.</div><p class="links"><a class="dblinks" href="/pubmed/23303278" ref="ordinalpos=3111&amp;ncbi_uid=4694570&amp;link_uid=23303278">Citation</a><a class="dblinks" href="/pmc/articles/PMC3565315/" ref="ordinalpos=3111&amp;ncbi_uid=4694570&amp;link_uid=PMC3565315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3112.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4064951/figure/f1-dddt-8-775/" ref="ordinalpos=3112&amp;ncbi_uid=6294216&amp;link_uid=PMC4064951" href="/pmc/articles/PMC4064951/"><img src="/pmc/articles/instance/4064951/bin/dddt-8-775Fig1.gif" src-large="/pmc/articles/instance/4064951/bin/dddt-8-775Fig1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4064951/" ref="ordinalpos=3112&amp;ncbi_uid=6294216&amp;link_uid=PMC4064951" image-link="/pmc/articles/PMC4064951/figure/f1-dddt-8-775/" class="imagepopup">Figure 1.  From: Vemurafenib: an evidence-based review of its clinical utility in the treatment of metastatic melanoma. </a></p><div class="supp"><p class="details"><div>Activation of the MAPK <span class="highlight" style="background-color:">pathway</span> through a <i>BRAF</i>V600E mutation.<br /><b>Notes:</b> (<b>A</b>) Normal <span class="highlight" style="background-color:">signaling</span> of the BRAF <span class="highlight" style="background-color:">pathway</span>. The BRAF <span class="highlight" style="background-color:">pathway</span> initiates <span class="highlight" style="background-color:">signaling</span> through extracellular growth factor domains. RAS family members are subsequently activated through receptor tyrosine kinases domains, and activation of RAS proteins bind RAF isoforms leading to activation of RAF, followed by MEK, and the final step of phosphorylation of ERK. The net effect leads toward increased cell survival and decreased apoptosis. (<b>B</b>) BRAF mutant <span class="highlight" style="background-color:">pathway</span>. In the presence of an activating <i>BRAF</i>V600E mutation, BRAF no longer requires dimerization with RAS, and therefore remains constitutively active. (<b>C</b>) Inhibition of BRAF mutant <span class="highlight" style="background-color:">pathway</span>. The addition of vemurafenib inhibits mutant BRAF, thereby stopping downstream activation of the MAPK <span class="highlight" style="background-color:">pathway</span>, and thereby decreasing cellular proliferation and inducing apoptosis.<br /><b>Abbreviations:</b> ERK, extracellular signal regulated kinase; MAPK, mitogen-activated protein kinase; RAF, rapidly accelerated fibrosarcoma; RAS, rat sarcoma virus monologue; RTK, receptor tyrosine kinases.</div></p></div><div class="aux"><div class="resc">Abhisek Swaika, et al. Drug Des Devel Ther. 2014;8:775-787.</div><p class="links"><a class="dblinks" href="/pubmed/24966667" ref="ordinalpos=3112&amp;ncbi_uid=6294216&amp;link_uid=24966667">Citation</a><a class="dblinks" href="/pmc/articles/PMC4064951/" ref="ordinalpos=3112&amp;ncbi_uid=6294216&amp;link_uid=PMC4064951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3113.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4218725/figure/pone-0111408-g006/" ref="ordinalpos=3113&amp;ncbi_uid=6803136&amp;link_uid=PMC4218725" href="/pmc/articles/PMC4218725/"><img src="/pmc/articles/instance/4218725/bin/pone.0111408.g006.gif" src-large="/pmc/articles/instance/4218725/bin/pone.0111408.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4218725/" ref="ordinalpos=3113&amp;ncbi_uid=6803136&amp;link_uid=PMC4218725" image-link="/pmc/articles/PMC4218725/figure/pone-0111408-g006/" class="imagepopup">Figure 6. Schematic representation of mucin gene induction by Shigella..  From: Shigella dysenteriae Modulates BMP <span class="highlight" style="background-color:">Pathway</span> to Induce Mucin Gene Expression In Vivo and In Vitro. </a></p><div class="supp"><p class="details"><div>As indicated, BMP-SMAD <span class="highlight" style="background-color:">pathway</span> induces MUC2 gene expression through the CDX2 transcription factor whereas SMAD4 directly influences MUC5AC gene. Our data thereby unveil a complex <span class="highlight" style="background-color:">signaling</span> mechanism underlying mucin gene induction by <i>Shigella</i>.</div></p></div><div class="aux"><div class="resc">Ashidha Gopal, et al. PLoS One. 2014;9(11):e111408.</div><p class="links"><a class="dblinks" href="/pubmed/25365201" ref="ordinalpos=3113&amp;ncbi_uid=6803136&amp;link_uid=25365201">Citation</a><a class="dblinks" href="/pmc/articles/PMC4218725/" ref="ordinalpos=3113&amp;ncbi_uid=6803136&amp;link_uid=PMC4218725">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3114.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4009183/figure/fig7/" ref="ordinalpos=3114&amp;ncbi_uid=6131489&amp;link_uid=PMC4009183" href="/pmc/articles/PMC4009183/"><img src="/pmc/articles/instance/4009183/bin/MI2014-451826.007.gif" src-large="/pmc/articles/instance/4009183/bin/MI2014-451826.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4009183/" ref="ordinalpos=3114&amp;ncbi_uid=6131489&amp;link_uid=PMC4009183" image-link="/pmc/articles/PMC4009183/figure/fig7/" class="imagepopup">Figure 7.  From: Protective Effects of Pretreatment with Oleanolic Acid in Rats in the Acute Phase of Hepatic Ischemia-Reperfusion Injury: Role of the PI3K/Akt <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed schematic flow of OA's role in attenuating hepatic IR injury. IR procedure induces activation of PI3K cascades and then results in increases in IL-1<i>β</i> and ALT in the serum. OA can further activate these molecular changes at the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and results in the decreased serum ALT activity and IL-1<i>β</i> concentration.</div></p></div><div class="aux"><div class="resc">Bo Gui, et al. Mediators Inflamm. 2014;2014:451826.</div><p class="links"><a class="dblinks" href="/pubmed/24829521" ref="ordinalpos=3114&amp;ncbi_uid=6131489&amp;link_uid=24829521">Citation</a><a class="dblinks" href="/pmc/articles/PMC4009183/" ref="ordinalpos=3114&amp;ncbi_uid=6131489&amp;link_uid=PMC4009183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3115.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3777916/figure/pcbi-1003194-g002/" ref="ordinalpos=3115&amp;ncbi_uid=5329652&amp;link_uid=PMC3777916" href="/pmc/articles/PMC3777916/"><img src="/pmc/articles/instance/3777916/bin/pcbi.1003194.g002.gif" src-large="/pmc/articles/instance/3777916/bin/pcbi.1003194.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3777916/" ref="ordinalpos=3115&amp;ncbi_uid=5329652&amp;link_uid=PMC3777916" image-link="/pmc/articles/PMC3777916/figure/pcbi-1003194-g002/" class="imagepopup">Figure 2. A plausible <span class="highlight" style="background-color:">signaling</span> path (yellow) between the androgen receptor (AR, top) and the TP53 protein (p53, right) is found by a Paxtools “paths-between” query and visualized in the Chisio BioPAX Editor..  From: Using Biological <span class="highlight" style="background-color:">Pathway</span> Data with Paxtools. </a></p><div class="supp"><p class="details"><div>These cross-talks are difficult to find manually without a graph search because of the large number of connections to/from AR and p53 and because of fragmentation of data across multiple <span class="highlight" style="background-color:">pathway</span> data sources.</div></p></div><div class="aux"><div class="resc">Emek Demir, et al. PLoS Comput Biol. 2013 September;9(9):e1003194.</div><p class="links"><a class="dblinks" href="/pubmed/24068901" ref="ordinalpos=3115&amp;ncbi_uid=5329652&amp;link_uid=24068901">Citation</a><a class="dblinks" href="/pmc/articles/PMC3777916/" ref="ordinalpos=3115&amp;ncbi_uid=5329652&amp;link_uid=PMC3777916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3116.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3824607/figure/F3/" ref="ordinalpos=3116&amp;ncbi_uid=6435200&amp;link_uid=PMC3824607" href="/pmc/articles/PMC3824607/"><img src="/pmc/articles/instance/3824607/bin/nihms-471006-f0003.gif" src-large="/pmc/articles/instance/3824607/bin/nihms-471006-f0003.jpg" alt="Fig.3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3824607/" ref="ordinalpos=3116&amp;ncbi_uid=6435200&amp;link_uid=PMC3824607" image-link="/pmc/articles/PMC3824607/figure/F3/" class="imagepopup">Fig.3.  From: TNFR1 <span class="highlight" style="background-color:">Signaling</span> Kinetics: Spatiotemporal Control of Three Phases of IKK activation by Posttranslational Modification. </a></p><div class="supp"><p class="details"><div>Crosstalk in TNFR1 <span class="highlight" style="background-color:">signaling</span>. Ligation of TNFR1 leads to activation of the pro-inflammatory NF-κB and JNK, and pro-apoptotic caspase-8 and ROS pathways. However, in normal cells, NF-κB target gene products suppress pro-apoptotic pathways. For instance, cIAP1/2 and cFLIP inhibit caspase-8 activation, XIAP and GADD45β suppress prolonged JNK activation, and FHC and Mn-SOD inhibit ROS accumulation. Therefore, unless NF-κB <span class="highlight" style="background-color:">pathway</span> is blocked, TNFα is insufficient to induce cell death.</div></p></div><div class="aux"><div class="resc">Lauren M. Workman, et al. Cell Signal. 2013 August;25(8):10.1016/j.cellsig.2013.04.005.</div><p class="links"><a class="dblinks" href="/pubmed/23612498" ref="ordinalpos=3116&amp;ncbi_uid=6435200&amp;link_uid=23612498">Citation</a><a class="dblinks" href="/pmc/articles/PMC3824607/" ref="ordinalpos=3116&amp;ncbi_uid=6435200&amp;link_uid=PMC3824607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3117.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3078731/figure/F2/" ref="ordinalpos=3117&amp;ncbi_uid=3390680&amp;link_uid=PMC3078731" href="/pmc/articles/PMC3078731/"><img src="/pmc/articles/instance/3078731/bin/809fig2.gif" src-large="/pmc/articles/instance/3078731/bin/809fig2.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3078731/" ref="ordinalpos=3117&amp;ncbi_uid=3390680&amp;link_uid=PMC3078731" image-link="/pmc/articles/PMC3078731/figure/F2/" class="imagepopup">FIGURE 2. From: miTALOS: Analyzing the tissue-specific regulation of <span class="highlight" style="background-color:">signaling</span> pathways by human and mouse microRNAs. </a></p><div class="supp"><p class="details"><div>Proximity vs. enrichment in <span class="highlight" style="background-color:">signaling</span> pathways. Density of the number of targets in miRNA-<span class="highlight" style="background-color:">pathway</span> associations (solid line). Although the number of targets in a significantly enriched (FDR &lt; 0.01) <span class="highlight" style="background-color:">pathway</span> is mainly between 10 and 20 (dotted line), significantly proximal (FDR &lt; 0.01) targeted pathways have three to seven targets (dashed line). Thus proximity and enrichment score identify two alternative forms of miRNA control.</div></p></div><div class="aux"><div class="resc">Andreas Kowarsch, et al. RNA. 2011 May;17(5):809-819.</div><p class="links"><a class="dblinks" href="/pubmed/21441347" ref="ordinalpos=3117&amp;ncbi_uid=3390680&amp;link_uid=21441347">Citation</a><a class="dblinks" href="/pmc/articles/PMC3078731/" ref="ordinalpos=3117&amp;ncbi_uid=3390680&amp;link_uid=PMC3078731">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3118.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2848118/figure/F9/" ref="ordinalpos=3118&amp;ncbi_uid=2313545&amp;link_uid=PMC2848118" href="/pmc/articles/PMC2848118/"><img src="/pmc/articles/instance/2848118/bin/JCS056176F9.gif" src-large="/pmc/articles/instance/2848118/bin/JCS056176F9.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2848118/" ref="ordinalpos=3118&amp;ncbi_uid=2313545&amp;link_uid=PMC2848118" image-link="/pmc/articles/PMC2848118/figure/F9/" class="imagepopup">Fig. 9. From: Dishevelled-KSRP complex regulates Wnt <span class="highlight" style="background-color:">signaling</span> through post-transcriptional stabilization of ?-catenin mRNA. </a></p><div class="supp"><p class="details"><div><b>Regulation of β-catenin levels in canonical Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Basal levels of β-catenin are regulated at the mRNA level by Dvl3-KSRP complex that promotes de-stabilization of <i>CTNNB1</i> mRNA, and at the protein level by GSK3β-mediated phosphorylation that promotes degradation of β-catenin. Wnt stimulation contributes to rapid accumulation of β-catenin by opposing the ability of the Dvl3-KSRP complex to promote destabilization of <i>CTNNB1</i> mRNA and the degradation of β-catenin mediated by GSK3β phosphorylation.</div></p></div><div class="aux"><div class="resc">Rama Kamesh Bikkavilli, et al. J Cell Sci. 2010 April 15;123(8):1352-1362.</div><p class="links"><a class="dblinks" href="/pubmed/20332102" ref="ordinalpos=3118&amp;ncbi_uid=2313545&amp;link_uid=20332102">Citation</a><a class="dblinks" href="/pmc/articles/PMC2848118/" ref="ordinalpos=3118&amp;ncbi_uid=2313545&amp;link_uid=PMC2848118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3119.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC207628/figure/f9/" ref="ordinalpos=3119&amp;ncbi_uid=261364&amp;link_uid=PMC207628" href="/pmc/articles/PMC207628/"><img src="/pmc/articles/instance/207628/bin/mb2130730009.gif" src-large="/pmc/articles/instance/207628/bin/mb2130730009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC207628/" ref="ordinalpos=3119&amp;ncbi_uid=261364&amp;link_uid=PMC207628" image-link="/pmc/articles/PMC207628/figure/f9/" class="imagepopup">FIG. 9. From: Molecular Mechanism for a Role of SHP2 in Epidermal Growth Factor Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic representation of the molecular mechanism of SHP2 in the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The five Y's represent autophosphorylation sites on the C-terminal region of the EGFR. SHP2 PTP activity dephosphorylates the most proximal pY, which corresponds to pY992, a binding site of RasGAP shown as GAP SH2.</div></p></div><div class="aux"><div class="resc">Yehenew M. Agazie, et al. Mol Cell Biol. 2003 November;23(21):7875-7886.</div><p class="links"><a class="dblinks" href="/pubmed/14560030" ref="ordinalpos=3119&amp;ncbi_uid=261364&amp;link_uid=14560030">Citation</a><a class="dblinks" href="/pmc/articles/PMC207628/" ref="ordinalpos=3119&amp;ncbi_uid=261364&amp;link_uid=PMC207628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3120.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC153558/figure/F3/" ref="ordinalpos=3120&amp;ncbi_uid=284617&amp;link_uid=PMC153558" href="/pmc/articles/PMC153558/"><img src="/pmc/articles/instance/153558/bin/pq0931010003.gif" src-large="/pmc/articles/instance/153558/bin/pq0931010003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC153558/" ref="ordinalpos=3120&amp;ncbi_uid=284617&amp;link_uid=PMC153558" image-link="/pmc/articles/PMC153558/figure/F3/" class="imagepopup">Figure 3.  From: Radar breaks the fog: Insights into dorsoventral patterning in zebrafish. </a></p><div class="supp"><p class="details"><div>Model of the maternal Radar <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediating early steps of dorsoventral axis formation. Radar acts through the TGF-β type I receptor Alk8 and causes transcriptional activation of BMP2b and BMP4. The intracellular signal transducer, believed to be a member of the Smad1/5/8 protein family is not known (indicated in gray), although Smad5 may be excluded. Maternal Smad5 may transduce the signal of a hypothesized distinct maternal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> acting in parallel to Radar. Alk8 is also the receptor for zygotic BMP2b/7 <span class="highlight" style="background-color:">signaling</span>, and Smad5 transduces that signal, leading to the autoregulatory transcriptional activation of BMP2b/4/7.</div></p></div><div class="aux"><div class="resc">Thomas P. Wilm, et al. Proc Natl Acad Sci U S A. 2003 April 15;100(8):4363-4365.</div><p class="links"><a class="dblinks" href="/pubmed/12682283" ref="ordinalpos=3120&amp;ncbi_uid=284617&amp;link_uid=12682283">Citation</a><a class="dblinks" href="/pmc/articles/PMC153558/" ref="ordinalpos=3120&amp;ncbi_uid=284617&amp;link_uid=PMC153558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3121.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1501083/figure/F5/" ref="ordinalpos=3121&amp;ncbi_uid=700877&amp;link_uid=PMC1501083" href="/pmc/articles/PMC1501083/"><img src="/pmc/articles/instance/1501083/bin/nihms10559f5.gif" src-large="/pmc/articles/instance/1501083/bin/nihms10559f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1501083/" ref="ordinalpos=3121&amp;ncbi_uid=700877&amp;link_uid=PMC1501083" image-link="/pmc/articles/PMC1501083/figure/F5/" class="imagepopup">Figure 5.  From: Parallel analysis of transcript and translation profiles: identification of metastasis-related signal pathways differentially regulated by drug and genetic modifications. </a></p><div class="supp"><p class="details"><div>Unsupervised clustering analysis of response profiles to caffeine treatment and genetic variation based on the three pathways examined, including the Akt <span class="highlight" style="background-color:">pathway</span>, caspase-dependent apoptosis <span class="highlight" style="background-color:">pathway</span> and the Stats <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. CAFF21: caffeine treatment starting from weaning; CAFF60: Caffeine treatment starting from age of 60 days.</div></p></div><div class="aux"><div class="resc">Haiyan Yang, et al. J Proteome Res. ;5(7):1555-1567.</div><p class="links"><a class="dblinks" href="/pubmed/16823962" ref="ordinalpos=3121&amp;ncbi_uid=700877&amp;link_uid=16823962">Citation</a><a class="dblinks" href="/pmc/articles/PMC1501083/" ref="ordinalpos=3121&amp;ncbi_uid=700877&amp;link_uid=PMC1501083">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3122.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4290028/figure/fig5/" ref="ordinalpos=3122&amp;ncbi_uid=7089345&amp;link_uid=PMC4290028" href="/pmc/articles/PMC4290028/"><img src="/pmc/articles/instance/4290028/bin/SCI2014-917168.005.gif" src-large="/pmc/articles/instance/4290028/bin/SCI2014-917168.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4290028/" ref="ordinalpos=3122&amp;ncbi_uid=7089345&amp;link_uid=PMC4290028" image-link="/pmc/articles/PMC4290028/figure/fig5/" class="imagepopup">Figure 5.  From: Effects of Toll-Like Receptors 3 and 4 in the Osteogenesis of Stem Cells. </a></p><div class="supp"><p class="details"><div>Expression of <i>β</i>-catenin. Western blotting analysis was used to access the expression of total <i>β</i>-catenin and nuclear <i>β</i>-catenin in OBs stimulated by TLR-3 ligands (<i>n</i> = 12) and TLR-4 ligands (<i>n</i> = 12). Untreated OBs were used as negative control (<i>n</i> = 12). More expression of <i>β</i>-catenin indicated better activation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TLR-4 ligands inhibited Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while TLR-3 ligands did not affect that mechanism.</div></p></div><div class="aux"><div class="resc">Chen Qi, et al. Stem Cells Int. 2014;2014:917168.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3122&amp;ncbi_uid=7089345&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4290028/" ref="ordinalpos=3122&amp;ncbi_uid=7089345&amp;link_uid=PMC4290028">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3123.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2958340/figure/fig3/" ref="ordinalpos=3123&amp;ncbi_uid=2534177&amp;link_uid=PMC2958340" href="/pmc/articles/PMC2958340/"><img src="/pmc/articles/instance/2958340/bin/JO2011-941876.003.gif" src-large="/pmc/articles/instance/2958340/bin/JO2011-941876.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2958340/" ref="ordinalpos=3123&amp;ncbi_uid=2534177&amp;link_uid=PMC2958340" image-link="/pmc/articles/PMC2958340/figure/fig3/" class="imagepopup">Figure 3.  From: Targeting the Mechanisms of Resistance to Chemotherapy and  Radiotherapy with the Cancer Stem Cell Hypothesis. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the canonical Wnt/<i>β</i>-catenin <span class="highlight" style="background-color:">signaling</span>. Wnt/<i>β</i>-catenin <span class="highlight" style="background-color:">pathway</span> may promote genomic instability after irradiation, thus allowing tumor cells to both survive after irradiation and develop additional adaptive mutations. ICG-001, PKF115-854, and CGP049040 are anticancer drugs in development that target the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ryan Morrison, et al. J Oncol. 2011;2011:941876.</div><p class="links"><a class="dblinks" href="/pubmed/20981352" ref="ordinalpos=3123&amp;ncbi_uid=2534177&amp;link_uid=20981352">Citation</a><a class="dblinks" href="/pmc/articles/PMC2958340/" ref="ordinalpos=3123&amp;ncbi_uid=2534177&amp;link_uid=PMC2958340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3124.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4192540/figure/pone-0109875-g006/" ref="ordinalpos=3124&amp;ncbi_uid=6720435&amp;link_uid=PMC4192540" href="/pmc/articles/PMC4192540/"><img src="/pmc/articles/instance/4192540/bin/pone.0109875.g006.gif" src-large="/pmc/articles/instance/4192540/bin/pone.0109875.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4192540/" ref="ordinalpos=3124&amp;ncbi_uid=6720435&amp;link_uid=PMC4192540" image-link="/pmc/articles/PMC4192540/figure/pone-0109875-g006/" class="imagepopup">Figure 6. Specific down-regulation of the cytokinin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in A. thaliana mesophyll protoplasts after ectopic expression of scFv hB7A..  From: Targeted In Vivo Inhibition of Specific Protein-Protein Interactions Using Recombinant Antibodies. </a></p><div class="supp"><p class="details"><div>Levels of the TCS:LUC induced luciferase activity correspond to the activity of the cytokinin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated with 100 nM trans-Zeatin (tZ) and normalized to the transfection control 35S:Renilla Luciferase. The ectopic expression of scFv hB7A is sorted by the amount of total (20 µg) co-transfected DNA (expressed in percent). The Col-0 wild type plants (white) and ahp 1,3,4,5 mutant plants with solitary active AHP2 (gray) show no significant effect. The ahp 1,2,4,5 mutant plants (black) with solitary active AHP3 were severely affected</div></p></div><div class="aux"><div class="resc">Matej Zábrady, et al. PLoS One. 2014;9(10):e109875.</div><p class="links"><a class="dblinks" href="/pubmed/25299686" ref="ordinalpos=3124&amp;ncbi_uid=6720435&amp;link_uid=25299686">Citation</a><a class="dblinks" href="/pmc/articles/PMC4192540/" ref="ordinalpos=3124&amp;ncbi_uid=6720435&amp;link_uid=PMC4192540">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3125.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4294461/figure/fig1/" ref="ordinalpos=3125&amp;ncbi_uid=7085386&amp;link_uid=PMC4294461" href="/pmc/articles/PMC4294461/"><img src="/pmc/articles/instance/4294461/bin/JAR2014-153127.001.gif" src-large="/pmc/articles/instance/4294461/bin/JAR2014-153127.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4294461/" ref="ordinalpos=3125&amp;ncbi_uid=7085386&amp;link_uid=PMC4294461" image-link="/pmc/articles/PMC4294461/figure/fig1/" class="imagepopup">Figure 1.  From: Efficacy of Female Rat Models in Translational Cardiovascular Aging Research. </a></p><div class="supp"><p class="details"><div>The NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (see text for details).</div></p></div><div class="aux"><div class="resc">K. M. Rice, et al. J Aging Res. 2014;2014:153127.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3125&amp;ncbi_uid=7085386&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4294461/" ref="ordinalpos=3125&amp;ncbi_uid=7085386&amp;link_uid=PMC4294461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3126.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2722148/figure/fig1/" ref="ordinalpos=3126&amp;ncbi_uid=2057039&amp;link_uid=PMC2722148" href="/pmc/articles/PMC2722148/"><img src="/pmc/articles/instance/2722148/bin/carcinbgn247f01_lw.gif" src-large="/pmc/articles/instance/2722148/bin/carcinbgn247f01_lw.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2722148/" ref="ordinalpos=3126&amp;ncbi_uid=2057039&amp;link_uid=PMC2722148" image-link="/pmc/articles/PMC2722148/figure/fig1/" class="imagepopup">Fig. 1. From: Hormone-dependent effects of FGFR2 and MAP3K1 in breast cancer susceptibility in a population-based sample of post-menopausal African-American and European-American women. </a></p><div class="supp"><p class="details"><div>The MAPK transduction <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Timothy R. Rebbeck, et al. Carcinogenesis. 2009 February;30(2):269-274.</div><p class="links"><a class="dblinks" href="/pubmed/19028704" ref="ordinalpos=3126&amp;ncbi_uid=2057039&amp;link_uid=19028704">Citation</a><a class="dblinks" href="/pmc/articles/PMC2722148/" ref="ordinalpos=3126&amp;ncbi_uid=2057039&amp;link_uid=PMC2722148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3127.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2481274/figure/pone-0002737-g002/" ref="ordinalpos=3127&amp;ncbi_uid=1482244&amp;link_uid=PMC2481274" href="/pmc/articles/PMC2481274/"><img src="/pmc/articles/instance/2481274/bin/pone.0002737.g002.gif" src-large="/pmc/articles/instance/2481274/bin/pone.0002737.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2481274/" ref="ordinalpos=3127&amp;ncbi_uid=1482244&amp;link_uid=PMC2481274" image-link="/pmc/articles/PMC2481274/figure/pone-0002737-g002/" class="imagepopup">Figure 2. Type I IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated in lesional skin of psoriatic patients..  From: Type I Interferon: Potential Therapeutic Target for Psoriasis?. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> image was generated with GeneGo's MetaCore integrated software suite. Individual symbols within the image represent well–characterized proteins or protein complexes. Arrows linking the proteins represent the stimulatory, inhibitory, or interactive effect of the protein on the target protein. Thermometers adjacent to the individual symbols represent relative expression levels (red indicates overexpression; blue indicates underexpression) of transcripts that encode the protein (or protein complex) within the particular <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yihong Yao, et al. PLoS ONE. 2008;3(7):e2737.</div><p class="links"><a class="dblinks" href="/pubmed/18648529" ref="ordinalpos=3127&amp;ncbi_uid=1482244&amp;link_uid=18648529">Citation</a><a class="dblinks" href="/pmc/articles/PMC2481274/" ref="ordinalpos=3127&amp;ncbi_uid=1482244&amp;link_uid=PMC2481274">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3128.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861677/figure/F3/" ref="ordinalpos=3128&amp;ncbi_uid=2308440&amp;link_uid=PMC2861677" href="/pmc/articles/PMC2861677/"><img src="/pmc/articles/instance/2861677/bin/1471-2164-11-197-3.gif" src-large="/pmc/articles/instance/2861677/bin/1471-2164-11-197-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861677/" ref="ordinalpos=3128&amp;ncbi_uid=2308440&amp;link_uid=PMC2861677" image-link="/pmc/articles/PMC2861677/figure/F3/" class="imagepopup">Figure 3.  From: Reciprocal regulation of metabolic and <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b>Correlation of KEGG <span class="highlight" style="background-color:">pathway</span> gene expression</b>. There is a positive correlation between OXPHOS and TCA ('citrate') cycle <b>(A) </b>and between the KEGG pathways of OXPHOS and proteasome <b>(B)</b>. OXPHOS and MAPK <span class="highlight" style="background-color:">signaling</span> pathways are negatively correlated <b>(C)</b>, while <span class="highlight" style="background-color:">signaling</span> pathways (e.g. JAK-STAT and MAPK) are positively correlated to each other <b>(D)</b>. Plots represent net direction of regulation of a KEGG <span class="highlight" style="background-color:">pathway</span>, i.e. number of up- <i>minus </i>down-regulated genes in relation to the total number of regulated genes within a study. Correlation plots include all 64 animal and human myocardial microarray studies listed in Additional File 2.</div></p></div><div class="aux"><div class="resc">Andreas S Barth, et al. BMC Genomics. 2010;11:197-197.</div><p class="links"><a class="dblinks" href="/pubmed/20334672" ref="ordinalpos=3128&amp;ncbi_uid=2308440&amp;link_uid=20334672">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861677/" ref="ordinalpos=3128&amp;ncbi_uid=2308440&amp;link_uid=PMC2861677">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3129.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3353202/figure/F5/" ref="ordinalpos=3129&amp;ncbi_uid=4007052&amp;link_uid=PMC3353202" href="/pmc/articles/PMC3353202/"><img src="/pmc/articles/instance/3353202/bin/1471-2164-13-153-5.gif" src-large="/pmc/articles/instance/3353202/bin/1471-2164-13-153-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3353202/" ref="ordinalpos=3129&amp;ncbi_uid=4007052&amp;link_uid=PMC3353202" image-link="/pmc/articles/PMC3353202/figure/F5/" class="imagepopup">Figure 5.  From: Characterization of the transcriptome profiles related to globin gene switching during in vitro erythroid maturation. </a></p><div class="supp"><p class="details"><div><b>Pathways identified by IPA</b>. Profile-1 genes were used for IPA. The A) Cdc42 and B) p38 MAPK <span class="highlight" style="background-color:">signaling</span> pathways identified for profile-1 genes are shown. The shaded symbols represent gene hits in the <span class="highlight" style="background-color:">pathway</span> and clear symbols representing other known genes in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (squares = cytokines, oval shape = transcription factors and triangle shape = kinases).</div></p></div><div class="aux"><div class="resc">Biaoru Li, et al. BMC Genomics. 2012;13:153-153.</div><p class="links"><a class="dblinks" href="/pubmed/22537182" ref="ordinalpos=3129&amp;ncbi_uid=4007052&amp;link_uid=22537182">Citation</a><a class="dblinks" href="/pmc/articles/PMC3353202/" ref="ordinalpos=3129&amp;ncbi_uid=4007052&amp;link_uid=PMC3353202">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3130.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2556276/figure/F1/" ref="ordinalpos=3130&amp;ncbi_uid=1526761&amp;link_uid=PMC2556276" href="/pmc/articles/PMC2556276/"><img src="/pmc/articles/instance/2556276/bin/nihms-49618-f0001.gif" src-large="/pmc/articles/instance/2556276/bin/nihms-49618-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2556276/" ref="ordinalpos=3130&amp;ncbi_uid=1526761&amp;link_uid=PMC2556276" image-link="/pmc/articles/PMC2556276/figure/F1/" class="imagepopup">Figure 1.  From: Signature pathways identified from gene expression profiles in the human uterine cervix before and after spontaneous term parturition. </a></p><div class="supp"><p class="details"><div>Display of differentially expressed genes in the uterine cervix after term spontaneous labor mapped on the <span style="text-decoration: underline" class="underline">PLAU <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></span>. ‘Thermometer’ represents differentially expressed gene.<br />Connectivity Analysis results: Several nodes in the PLAU <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were found to be significant in providing connectivity among the differentially expressed genes. Significant nodes are encircled in red.</div></p></div><div class="aux"><div class="resc">Sonia S. HASSAN, et al. Am J Obstet Gynecol. ;197(3):250.e1-250.e7.</div><p class="links"><a class="dblinks" href="/pubmed/17826407" ref="ordinalpos=3130&amp;ncbi_uid=1526761&amp;link_uid=17826407">Citation</a><a class="dblinks" href="/pmc/articles/PMC2556276/" ref="ordinalpos=3130&amp;ncbi_uid=1526761&amp;link_uid=PMC2556276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3131.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676795/figure/f2-ijms-14-09487/" ref="ordinalpos=3131&amp;ncbi_uid=5060381&amp;link_uid=PMC3676795" href="/pmc/articles/PMC3676795/"><img src="/pmc/articles/instance/3676795/bin/ijms-14-09487f2.gif" src-large="/pmc/articles/instance/3676795/bin/ijms-14-09487f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676795/" ref="ordinalpos=3131&amp;ncbi_uid=5060381&amp;link_uid=PMC3676795" image-link="/pmc/articles/PMC3676795/figure/f2-ijms-14-09487/" class="imagepopup">Figure 2.  From: The Role of microRNA in Gastric Malignancy. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> cascades that regulate the intrinsic and extrinsic pathways of apoptosis. Receptor tyrosine kinase detects survival stimuli such as growth factors and induces phosphoinositide 3-kinase (PI3K)/Akt <span class="highlight" style="background-color:">signaling</span> cascades that ultimately result in the inhibition of the pro-apoptotic protein, Bad. Pro-apoptotic and anti-apoptotic proteins govern the intrinsic cell death <span class="highlight" style="background-color:">pathway</span>, which results in the release of cytochrome <i>c</i> from the mitochondria and induction of the caspase cascade. <span class="highlight" style="background-color:">Signaling</span> through death receptors initiates the extrinsic <span class="highlight" style="background-color:">pathway</span> of apoptosis.</div></p></div><div class="aux"><div class="resc">Toshihiro Nishizawa, et al. Int J Mol Sci. 2013 May;14(5):9487-9496.</div><p class="links"><a class="dblinks" href="/pubmed/23629677" ref="ordinalpos=3131&amp;ncbi_uid=5060381&amp;link_uid=23629677">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676795/" ref="ordinalpos=3131&amp;ncbi_uid=5060381&amp;link_uid=PMC3676795">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3132.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4222471/figure/f2-0991653/" ref="ordinalpos=3132&amp;ncbi_uid=6832741&amp;link_uid=PMC4222471" href="/pmc/articles/PMC4222471/"><img src="/pmc/articles/instance/4222471/bin/991653.fig2.gif" src-large="/pmc/articles/instance/4222471/bin/991653.fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4222471/" ref="ordinalpos=3132&amp;ncbi_uid=6832741&amp;link_uid=PMC4222471" image-link="/pmc/articles/PMC4222471/figure/f2-0991653/" class="imagepopup">Figure 2. From: RAS diseases in children. </a></p><div class="supp"><p class="details"><div>The RAS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and genes mutated in juvenile myelomonocytic leukemia (in black).</div></p></div><div class="aux"><div class="resc">Charlotte M. Niemeyer. Haematologica. 2014 November;99(11):1653-1662.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3132&amp;ncbi_uid=6832741&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4222471/" ref="ordinalpos=3132&amp;ncbi_uid=6832741&amp;link_uid=PMC4222471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3133.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3675375/figure/F1/" ref="ordinalpos=3133&amp;ncbi_uid=4977078&amp;link_uid=PMC3675375" href="/pmc/articles/PMC3675375/"><img src="/pmc/articles/instance/3675375/bin/1471-2156-14-28-1.gif" src-large="/pmc/articles/instance/3675375/bin/1471-2156-14-28-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3675375/" ref="ordinalpos=3133&amp;ncbi_uid=4977078&amp;link_uid=PMC3675375" image-link="/pmc/articles/PMC3675375/figure/F1/" class="imagepopup">Figure 1.  From: Carriers of a novel frame-shift insertion in WNT16a possess elevated pancreatic expression of TCF7L2. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in diabetes mellitus.</div></p></div><div class="aux"><div class="resc">Eric W Howard, et al. BMC Genet. 2013;14:28-28.</div><p class="links"><a class="dblinks" href="/pubmed/23617586" ref="ordinalpos=3133&amp;ncbi_uid=4977078&amp;link_uid=23617586">Citation</a><a class="dblinks" href="/pmc/articles/PMC3675375/" ref="ordinalpos=3133&amp;ncbi_uid=4977078&amp;link_uid=PMC3675375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3134.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3116336/figure/F1/" ref="ordinalpos=3134&amp;ncbi_uid=3022883&amp;link_uid=PMC3116336" href="/pmc/articles/PMC3116336/"><img src="/pmc/articles/instance/3116336/bin/zsd0011127910001.gif" src-large="/pmc/articles/instance/3116336/bin/zsd0011127910001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3116336/" ref="ordinalpos=3134&amp;ncbi_uid=3022883&amp;link_uid=PMC3116336" image-link="/pmc/articles/PMC3116336/figure/F1/" class="imagepopup">Fig. 1.  From: Adenosine Receptor Subtypes and the Heart Failure Phenotype: Translating Lessons from Mice to Man. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> for adenosine receptors.</div></p></div><div class="aux"><div class="resc">Arthur M. Feldman, et al. Trans Am Clin Climatol Assoc. 2011;122:198-214.</div><p class="links"><a class="dblinks" href="/pubmed/21686225" ref="ordinalpos=3134&amp;ncbi_uid=3022883&amp;link_uid=21686225">Citation</a><a class="dblinks" href="/pmc/articles/PMC3116336/" ref="ordinalpos=3134&amp;ncbi_uid=3022883&amp;link_uid=PMC3116336">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3135.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4033955/figure/pharmaceuticals-03-01842-f003/" ref="ordinalpos=3135&amp;ncbi_uid=6170362&amp;link_uid=PMC4033955" href="/pmc/articles/PMC4033955/"><img src="/pmc/articles/instance/4033955/bin/pharmaceuticals-03-01842-g003.gif" src-large="/pmc/articles/instance/4033955/bin/pharmaceuticals-03-01842-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4033955/" ref="ordinalpos=3135&amp;ncbi_uid=6170362&amp;link_uid=PMC4033955" image-link="/pmc/articles/PMC4033955/figure/pharmaceuticals-03-01842-f003/" class="imagepopup">Figure 3.  From: Use of p38 MAPK Inhibitors for the Treatment of Werner Syndrome. </a></p><div class="supp"><p class="details"><div>The p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mark C. Bagley, et al. Pharmaceuticals (Basel). 2010 June;3(6):1842-1872.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3135&amp;ncbi_uid=6170362&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4033955/" ref="ordinalpos=3135&amp;ncbi_uid=6170362&amp;link_uid=PMC4033955">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3136.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2782741/figure/F6/" ref="ordinalpos=3136&amp;ncbi_uid=2165351&amp;link_uid=PMC2782741" href="/pmc/articles/PMC2782741/"><img src="/pmc/articles/instance/2782741/bin/nihms92825f6.gif" src-large="/pmc/articles/instance/2782741/bin/nihms92825f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2782741/" ref="ordinalpos=3136&amp;ncbi_uid=2165351&amp;link_uid=PMC2782741" image-link="/pmc/articles/PMC2782741/figure/F6/" class="imagepopup">Fig. 6.  From: Invasive Prostate Cancer Cells Are Tumor Initiating Cells That Have A Stem Cell-Like Genomic Signature. </a></p><div class="supp"><p class="details"><div>Cell <span class="highlight" style="background-color:">Signaling</span> Pathways Active in the Invasive Subpopulation. Data extracted from the microarrays to examine key genes in the A. Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; B. Hedgehog <span class="highlight" style="background-color:">pathway</span>; C. Notch <span class="highlight" style="background-color:">pathway</span>; and D. Nanog/Sox2/Oct3 network. Fold changes in expression are depicted for the invasive cells relative to the non-invasive cells. The asterisk indicates data averaged from multiple oligonucleotides that bind different regions of the same gene.</div></p></div><div class="aux"><div class="resc">George J. Klarmann, et al. Clin Exp Metastasis. ;26(5):433-446.</div><p class="links"><a class="dblinks" href="/pubmed/19221883" ref="ordinalpos=3136&amp;ncbi_uid=2165351&amp;link_uid=19221883">Citation</a><a class="dblinks" href="/pmc/articles/PMC2782741/" ref="ordinalpos=3136&amp;ncbi_uid=2165351&amp;link_uid=PMC2782741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3137.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281649/figure/F1/" ref="ordinalpos=3137&amp;ncbi_uid=4610094&amp;link_uid=PMC3281649" href="/pmc/articles/PMC3281649/"><img src="/pmc/articles/instance/3281649/bin/zbc0091297680001.gif" src-large="/pmc/articles/instance/3281649/bin/zbc0091297680001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281649/" ref="ordinalpos=3137&amp;ncbi_uid=4610094&amp;link_uid=PMC3281649" image-link="/pmc/articles/PMC3281649/figure/F1/" class="imagepopup">FIGURE 1. From: Dual Regulation of Notch1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Adaptor Protein Fe65. </a></p><div class="supp"><p class="details"><div><b>Fe65 suppresses the Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, NIH3T3 cells were transfected with expression vectors for 4XCSL-Luc, Hes1-Luc, Hes5-Luc, and β-galactosidase, along with ΔEN1 and Fe65, as indicated. <i>B</i>, HEK293 cells were transfected with expression vectors for 4XCSL-Luc, Hes1-Luc, Hes5-Luc, and β-galactosidase, along with ΔEN1 and Fe65, as indicated. <i>A</i> and <i>B</i>, after 48 h of transfection, the cells were lysed, and the luciferase activity was determined. The data were normalized with β-galactosidase. These results represent the mean ± S.D. of three independent experiments. <i>R.L.U.</i>, relative luciferase units. The data we evaluated for significant difference by Student's <i>t</i> test. *, <i>p</i> &lt; 0.01 (analysis of variance).</div></p></div><div class="aux"><div class="resc">Mi-Yeon Kim, et al. J Biol Chem. 2012 February 10;287(7):4690-4701.</div><p class="links"><a class="dblinks" href="/pubmed/22199353" ref="ordinalpos=3137&amp;ncbi_uid=4610094&amp;link_uid=22199353">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281649/" ref="ordinalpos=3137&amp;ncbi_uid=4610094&amp;link_uid=PMC3281649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3138.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2803450/figure/F4/" ref="ordinalpos=3138&amp;ncbi_uid=2175740&amp;link_uid=PMC2803450" href="/pmc/articles/PMC2803450/"><img src="/pmc/articles/instance/2803450/bin/1476-4598-8-123-4.gif" src-large="/pmc/articles/instance/2803450/bin/1476-4598-8-123-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2803450/" ref="ordinalpos=3138&amp;ncbi_uid=2175740&amp;link_uid=PMC2803450" image-link="/pmc/articles/PMC2803450/figure/F4/" class="imagepopup">Figure 4.  From: The sonic hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is reactivated in human renal cell carcinoma and plays orchestral role in tumor growth. </a></p><div class="supp"><p class="details"><div><b>The inhibition of the SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces human CRCC cells apoptosis</b>. FACS analysis of 786-0 treated in control (Ctl) or with cyclopamine (Cyclopamine) at 20 μM. (<i>A</i>) Exemples of FACS analysis (day 4). No evidence of necrosis was observed in any cases. (<i>B</i>) Quantitative analysis of apoptotic cells as a function of the time of treatments. Results are shown as mean ± SEM, n = 6 *, <i>P </i>&lt; 0.05 and **, <i>P </i>&lt; 0.01 from Ctl apoptosis. Bars, 5 μm.</div></p></div><div class="aux"><div class="resc">Valérian Dormoy, et al. Mol Cancer. 2009;8:123-123.</div><p class="links"><a class="dblinks" href="/pubmed/20015350" ref="ordinalpos=3138&amp;ncbi_uid=2175740&amp;link_uid=20015350">Citation</a><a class="dblinks" href="/pmc/articles/PMC2803450/" ref="ordinalpos=3138&amp;ncbi_uid=2175740&amp;link_uid=PMC2803450">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3139.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1253564/figure/fig3/" ref="ordinalpos=3139&amp;ncbi_uid=673322&amp;link_uid=PMC1253564" href="/pmc/articles/PMC1253564/"><img src="/pmc/articles/instance/1253564/bin/zpq0410597730003.gif" src-large="/pmc/articles/instance/1253564/bin/zpq0410597730003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1253564/" ref="ordinalpos=3139&amp;ncbi_uid=673322&amp;link_uid=PMC1253564" image-link="/pmc/articles/PMC1253564/figure/fig3/" class="imagepopup">Fig. 3. From: Mechanism of radiation-induced bystander effect: Role of the cyclooxygenase-2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Identification of differentially expressed genes in radiation-induced bystander cells. (<i>A</i>) Differentially expressed <span class="highlight" style="background-color:">signaling</span> genes in bystander and control normal human lung fibroblasts using the cDNA signal transduction <span class="highlight" style="background-color:">pathway</span> finder array (SuperArray). (<i>B</i>) RT-PCR analysis and confirmation of the array data showing down-regulated <i>IGFBP-3</i> and up-regulated <i>COX-2</i>.(<i>C</i>) Western blotting of COX-2 protein expression (72 kD) in bystander and control cells with or without treatment with NS 398 (50 μM), a specific COX-2 inhibitor.</div></p></div><div class="aux"><div class="resc">Hongning Zhou, et al. Proc Natl Acad Sci U S A. 2005 October 11;102(41):14641-14646.</div><p class="links"><a class="dblinks" href="/pubmed/16203985" ref="ordinalpos=3139&amp;ncbi_uid=673322&amp;link_uid=16203985">Citation</a><a class="dblinks" href="/pmc/articles/PMC1253564/" ref="ordinalpos=3139&amp;ncbi_uid=673322&amp;link_uid=PMC1253564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3140.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3548572/figure/F2/" ref="ordinalpos=3140&amp;ncbi_uid=4541804&amp;link_uid=PMC3548572" href="/pmc/articles/PMC3548572/"><img src="/pmc/articles/instance/3548572/bin/nihms331373f2.gif" src-large="/pmc/articles/instance/3548572/bin/nihms331373f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3548572/" ref="ordinalpos=3140&amp;ncbi_uid=4541804&amp;link_uid=PMC3548572" image-link="/pmc/articles/PMC3548572/figure/F2/" class="imagepopup">Figure 2.  From: Bone Morphogenetic Protein Functions as a Context-Dependent Angiogenic Cue in Vertebrates. </a></p><div class="supp"><p class="details"><div>VEGFA and BMP pathways have distinct as well as overlapping intracellular targets. The VEGFA <span class="highlight" style="background-color:">pathway</span> is known to phosphorylate and activate, among other factors, MAPK <span class="highlight" style="background-color:">signaling</span> cascades, which includes both p38 and ERK. Canonical BMP <span class="highlight" style="background-color:">signaling</span> activates Smads however BMP stimulation has been shown to effectively activate MAPK <span class="highlight" style="background-color:">signaling</span> cascades as well. During angiogenesis, BMP requires activation of ERK and not p38 [47].</div></p></div><div class="aux"><div class="resc">David M. Wiley, et al. Semin Cell Dev Biol. ;22(9):1012-1018.</div><p class="links"><a class="dblinks" href="/pubmed/22008724" ref="ordinalpos=3140&amp;ncbi_uid=4541804&amp;link_uid=22008724">Citation</a><a class="dblinks" href="/pmc/articles/PMC3548572/" ref="ordinalpos=3140&amp;ncbi_uid=4541804&amp;link_uid=PMC3548572">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3141.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783519/figure/F5/" ref="ordinalpos=3141&amp;ncbi_uid=2546069&amp;link_uid=PMC2783519" href="/pmc/articles/PMC2783519/"><img src="/pmc/articles/instance/2783519/bin/nihms149288f5.gif" src-large="/pmc/articles/instance/2783519/bin/nihms149288f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783519/" ref="ordinalpos=3141&amp;ncbi_uid=2546069&amp;link_uid=PMC2783519" image-link="/pmc/articles/PMC2783519/figure/F5/" class="imagepopup">Figure 5.  From: Mouse-Specific Tandem IgY7-SuperMix Immunoaffinity Separations for Improved LC-MS/MS Coverage of the Plasma Proteome. </a></p><div class="supp"><p class="details"><div>Improved coverage of canonical pathways by tandem IgY7-SuperMix separation. Blue bars, IgY7 flow-through; Red bars, SuperMix flow-through and SuperMix bound combined. <i>APRS</i>, Acute Phase Response <span class="highlight" style="background-color:">Signaling</span>; <i>ComS</i>, Complement System; <i>CoaS</i>, Coagulation System; <i>HepF</i>, Hepatic Fibrosis / Hepatic Stellate Cell Activation; <i>LMIRF</i>, LPS/IL-1 Mediated Inhibition of RXR Function; <i>PUP</i>, Protein Ubiquitination <span class="highlight" style="background-color:">Pathway</span>; <i>ACS</i>, Actin Cytoskeleton <span class="highlight" style="background-color:">Signaling</span>; <i>F/RA</i>, FXR/RXR Activation; <i>L/RA</i>, LXR/RXR Activation; <i>Glyc</i>, Glycolysis/Gluconeogenesis; <i>CME</i>, Caveolar-mediated Endocytosis; <i>GRS</i>, Glucocorticoid Receptor <span class="highlight" style="background-color:">Signaling</span>; <i>NMOSR</i>, NRF2-mediated Oxidative Stress Response; <i>CME</i>, Clathrin-mediated Endocytosis; <i>FRMP</i>, Fcγ Receptor-mediated Phagocytosis in Macrophages and Monocytes; <i>IntS</i>, Integrin <span class="highlight" style="background-color:">Signaling</span>; <i>PyrM</i>, Pyruvate Metabolism; <i>RAMR</i>, Regulation of Actin-based Motility by Rho; <i>GluM,</i> Glutathione Metabolism; <i>XMS</i>, Xenobiotic Metabolism <span class="highlight" style="background-color:">Signaling</span>; <i>AHRS</i>, Aryl Hydrocarbon Receptor <span class="highlight" style="background-color:">Signaling</span>; <i>PurM</i>, Purine Metabolism; <i>PI3K</i>, PI3K/AKT <span class="highlight" style="background-color:">Signaling</span>.</div></p></div><div class="aux"><div class="resc">Jian-Ying Zhou, et al. J Proteome Res. ;8(11):5387-5395.</div><p class="links"><a class="dblinks" href="/pubmed/19722698" ref="ordinalpos=3141&amp;ncbi_uid=2546069&amp;link_uid=19722698">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783519/" ref="ordinalpos=3141&amp;ncbi_uid=2546069&amp;link_uid=PMC2783519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3142.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047895/figure/fig1/" ref="ordinalpos=3142&amp;ncbi_uid=6244209&amp;link_uid=PMC4047895" href="/pmc/articles/PMC4047895/"><img src="/pmc/articles/instance/4047895/bin/cddis2014215f1.gif" src-large="/pmc/articles/instance/4047895/bin/cddis2014215f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047895/" ref="ordinalpos=3142&amp;ncbi_uid=6244209&amp;link_uid=PMC4047895" image-link="/pmc/articles/PMC4047895/figure/fig1/" class="imagepopup">Figure 1.  From: The pleiotropic roles of sphingolipid <span class="highlight" style="background-color:">signaling</span> in autophagy. </a></p><div class="supp"><p class="details"><div>Cer formation through the <i>de novo</i>, turnover, or salvage <span class="highlight" style="background-color:">pathway</span>. Red arrows: <i>de novo</i> <span class="highlight" style="background-color:">pathway</span>; green arrows: turnover <span class="highlight" style="background-color:">pathway</span>; blue arrows: salvage <span class="highlight" style="background-color:">pathway</span>. PM, plasma membrane</div></p></div><div class="aux"><div class="resc">Y Li, et al. Cell Death Dis. 2014 May;5(5):e1245.</div><p class="links"><a class="dblinks" href="/pubmed/24853423" ref="ordinalpos=3142&amp;ncbi_uid=6244209&amp;link_uid=24853423">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047895/" ref="ordinalpos=3142&amp;ncbi_uid=6244209&amp;link_uid=PMC4047895">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3143.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2832945/figure/F7/" ref="ordinalpos=3143&amp;ncbi_uid=2823813&amp;link_uid=PMC2832945" href="/pmc/articles/PMC2832945/"><img src="/pmc/articles/instance/2832945/bin/zbc0141009460007.gif" src-large="/pmc/articles/instance/2832945/bin/zbc0141009460007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2832945/" ref="ordinalpos=3143&amp;ncbi_uid=2823813&amp;link_uid=PMC2832945" image-link="/pmc/articles/PMC2832945/figure/F7/" class="imagepopup">FIGURE 7. From: Androgen Receptor Survival <span class="highlight" style="background-color:">Signaling</span> Is Blocked by Anti-?2-microglobulin Monoclonal Antibody via a MAPK/Lipogenic <span class="highlight" style="background-color:">Pathway</span> in Human Prostate Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Proposed mechanism for β2M mAb inhibition of AR and cell proliferation, survival, and progression through a MAPK/SREBP-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells.</b> β2M mAb inhibited p-MAPK and p-AKT expression and decreased SREBP-1 expression in LNCaP and C4-2B cells. By the addition of U0126, a MAPK selective inhibitor, we observed down-regulation of SREBP-1 expression (see Fig. 6<i>B</i>). In addition, β2M mAb inhibited AR and FAS expression; the latter is known to regulate fatty acid biosynthesis, cell membrane integrity, energy metabolism, and lipid raft-regulated cell <span class="highlight" style="background-color:">signaling</span> (31, 32, 46), which ultimately control prostate cancer cell proliferation, survival, and progression.</div></p></div><div class="aux"><div class="resc">Wen-Chin Huang, et al. J Biol Chem. 2010 March 12;285(11):7947-7956.</div><p class="links"><a class="dblinks" href="/pubmed/20071336" ref="ordinalpos=3143&amp;ncbi_uid=2823813&amp;link_uid=20071336">Citation</a><a class="dblinks" href="/pmc/articles/PMC2832945/" ref="ordinalpos=3143&amp;ncbi_uid=2823813&amp;link_uid=PMC2832945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3144.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3146023/figure/F3/" ref="ordinalpos=3144&amp;ncbi_uid=3476677&amp;link_uid=PMC3146023" href="/pmc/articles/PMC3146023/"><img src="/pmc/articles/instance/3146023/bin/nihms295935f3.gif" src-large="/pmc/articles/instance/3146023/bin/nihms295935f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3146023/" ref="ordinalpos=3144&amp;ncbi_uid=3476677&amp;link_uid=PMC3146023" image-link="/pmc/articles/PMC3146023/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> pathways triggered by GDNF in spermatogonial stem cells.  From: Human spermatogonial stem cells: a possible origin for spermatocytic seminoma. </a></p><div class="supp"><p class="details"><div>GDNF dimerizes and binds to the GFR<b>α</b>-1/RET receptor complex. <b>A:</b> Binding of GDNF activates RET, which triggers SRC kinase phosphorylation and the downstream activation of PI3K/AKT. Ultimately the transcription factor MYCN is up-regulated. <b>B</b>: Binding of GDNF also can activate the RAS-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which triggers ERK1/2 phosphorylation and up-regulation of the transcription factor FOS.</div></p></div><div class="aux"><div class="resc">Reham Waheeb, et al. Int J Androl. ;34(4 Pt 2):e296-e305.</div><p class="links"><a class="dblinks" href="/pubmed/21790653" ref="ordinalpos=3144&amp;ncbi_uid=3476677&amp;link_uid=21790653">Citation</a><a class="dblinks" href="/pmc/articles/PMC3146023/" ref="ordinalpos=3144&amp;ncbi_uid=3476677&amp;link_uid=PMC3146023">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3145.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766869/figure/F3/" ref="ordinalpos=3145&amp;ncbi_uid=2417310&amp;link_uid=PMC2766869" href="/pmc/articles/PMC2766869/"><img src="/pmc/articles/instance/2766869/bin/nihms141387f3.gif" src-large="/pmc/articles/instance/2766869/bin/nihms141387f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766869/" ref="ordinalpos=3145&amp;ncbi_uid=2417310&amp;link_uid=PMC2766869" image-link="/pmc/articles/PMC2766869/figure/F3/" class="imagepopup">Figure 3. Pyoverdine “ferrimone” <span class="highlight" style="background-color:">signaling</span>.  From: Microbial Telesensing: Probing the environment for friends, foes and food. </a></p><div class="supp"><p class="details"><div>Ferri-pyoverdine, having extracted a ferric ion from the environment, complexes with a specific outer-membrane receptor inducing a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> ultimately sensed by the PvdS alternative sigma factor. Activated PvdS recruits RNA polymerase to target promoters leading to activation of gene expression including genes involved in pyoverdine production, <i>prpL</i> encoding a protease and indirectly <i>toxA</i> coding for the exotoxin A.</div></p></div><div class="aux"><div class="resc">Agnès Roux, et al. Cell Host Microbe. ;6(2):115-124.</div><p class="links"><a class="dblinks" href="/pubmed/19683678" ref="ordinalpos=3145&amp;ncbi_uid=2417310&amp;link_uid=19683678">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766869/" ref="ordinalpos=3145&amp;ncbi_uid=2417310&amp;link_uid=PMC2766869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3146.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4299277/figure/DEV112797F1/" ref="ordinalpos=3146&amp;ncbi_uid=7111264&amp;link_uid=PMC4299277" href="/pmc/articles/PMC4299277/"><img src="/pmc/articles/instance/4299277/bin/develop-141-112797-g1.gif" src-large="/pmc/articles/instance/4299277/bin/develop-141-112797-g1.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4299277/" ref="ordinalpos=3146&amp;ncbi_uid=7111264&amp;link_uid=PMC4299277" image-link="/pmc/articles/PMC4299277/figure/DEV112797F1/" class="imagepopup">Fig. 1. From: Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> integrates patterning and metabolism of the insect growth zone. </a></p><div class="supp"><p class="details"><div><b>Complementary interactions of Wnt and Hh <span class="highlight" style="background-color:">signaling</span> in the head and GZ.</b> (A-C) Expression of <i>Tc-wg</i> (rows 1 and 2) and <i>Tc-hh</i> (rows 3 and 4) in wild-type (Aa-d) and RNAi-treated embryos with interrupted Wnt (Ba-d) or Hh (Ca-d) <span class="highlight" style="background-color:">signaling</span> in germ rudiments (rows 1 and 3) and elongating germ bands (rows 2 and 4). Anterior is oriented to the left. (Ba-d) When the Wnt <span class="highlight" style="background-color:">pathway</span> was disrupted, both <i>Tc-wg</i> and <i>Tc-hh</i> expression was abolished in <i>Tc-</i><i>arr</i><i><sup>RNAi</sup></i> embryos in the GZ (black arrows) and at the ocular parasegment boundary (black arrowheads). (Ca-d) When the Hh <span class="highlight" style="background-color:">pathway</span> was disrupted, <i>Tc-wg</i> expression was missing in the anterior head of <i>Tc-hh<sup>RNAi</sup></i> embryos (Ca,b, white arrows), and <i>Tc-hh</i> expression was largely present in <i>Tc-smo<sup>RNAi</sup></i> but the anterior head domains appeared enlarged (Cc,d, white arrows). (D) Genetic interactions in the anterior head: Hh <span class="highlight" style="background-color:">signaling</span> acts upstream of <i>Tc-wg</i>. (E) Genetic interactions in the GZ: Wnt/β-catenin acts upstream of <i>Tc-hh</i> and shows autoregulation. Arrows indicate activation. an, antennal stripe; gz, growth zone; oc, ocular stripe.</div></p></div><div class="aux"><div class="resc">Georg Oberhofer, et al. Development. 2014 December 15;141(24):4740-4750.</div><p class="links"><a class="dblinks" href="/pubmed/25395458" ref="ordinalpos=3146&amp;ncbi_uid=7111264&amp;link_uid=25395458">Citation</a><a class="dblinks" href="/pmc/articles/PMC4299277/" ref="ordinalpos=3146&amp;ncbi_uid=7111264&amp;link_uid=PMC4299277">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3147.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3547545/figure/f2-ebo-9-2013-001/" ref="ordinalpos=3147&amp;ncbi_uid=4564327&amp;link_uid=PMC3547545" href="/pmc/articles/PMC3547545/"><img src="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f2.gif" src-large="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3547545/" ref="ordinalpos=3147&amp;ncbi_uid=4564327&amp;link_uid=PMC3547545" image-link="/pmc/articles/PMC3547545/figure/f2-ebo-9-2013-001/" class="imagepopup">Figure 2.  From: The Molecular Evolutionary Patterns of the Insulin/FOXO <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Survey of the components of IIF <span class="highlight" style="background-color:">pathway</span> in selected metazoans.<br /><b>Notes:</b> Each square indicates a single <span class="highlight" style="background-color:">pathway</span> gene and squares with a question mark indicating uncertainty of the orthology, while with letter A indicating apparent orthology. Letter X’s represent absence of members.</div></p></div><div class="aux"><div class="resc">Minghui Wang, et al. Evol Bioinform Online. 2013;9:1-16.</div><p class="links"><a class="dblinks" href="/pubmed/23362368" ref="ordinalpos=3147&amp;ncbi_uid=4564327&amp;link_uid=23362368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3547545/" ref="ordinalpos=3147&amp;ncbi_uid=4564327&amp;link_uid=PMC3547545">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3148.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4310282/figure/F1/" ref="ordinalpos=3148&amp;ncbi_uid=7182282&amp;link_uid=PMC4310282" href="/pmc/articles/PMC4310282/"><img src="/pmc/articles/instance/4310282/bin/fonc-05-00005-g001.gif" src-large="/pmc/articles/instance/4310282/bin/fonc-05-00005-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4310282/" ref="ordinalpos=3148&amp;ncbi_uid=7182282&amp;link_uid=PMC4310282" image-link="/pmc/articles/PMC4310282/figure/F1/" class="imagepopup">Figure 1.  From: Current Therapeutic Advances Targeting EGFR and EGFRvIII in Glioblastoma. </a></p><div class="supp"><p class="details"><div><b>EGFR <span class="highlight" style="background-color:">signaling</span> and targeted therapies</b>. There are three key <span class="highlight" style="background-color:">signaling</span> pathways in GBM. The RTK/RAS/PI(3)K <span class="highlight" style="background-color:">pathway</span> is involved in cell growth, apoptosis resistance, invasion, and migration (pictured above with targeted therapies). The other critically important pathways that regulate cell proliferation and survival are p53 and RB <span class="highlight" style="background-color:">signaling</span> (5) [adapted from Ref. (14)].</div></p></div><div class="aux"><div class="resc">Emily Padfield, et al. Front Oncol. 2015;5:5.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3148&amp;ncbi_uid=7182282&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4310282/" ref="ordinalpos=3148&amp;ncbi_uid=7182282&amp;link_uid=PMC4310282">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3149.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3192283/figure/Fig1/" ref="ordinalpos=3149&amp;ncbi_uid=3300513&amp;link_uid=PMC3192283" href="/pmc/articles/PMC3192283/"><img src="/pmc/articles/instance/3192283/bin/18_2011_797_Fig1_HTML.gif" src-large="/pmc/articles/instance/3192283/bin/18_2011_797_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3192283/" ref="ordinalpos=3149&amp;ncbi_uid=3300513&amp;link_uid=PMC3192283" image-link="/pmc/articles/PMC3192283/figure/Fig1/" class="imagepopup">Fig. 1.  From: Regulation of immune cell function and differentiation by the NKG2D receptor. </a></p><div class="supp"><p class="details"><div>Coupling of the IL-15R and NKG2D <span class="highlight" style="background-color:">signaling</span> components in NK cells. The figure roughly summarizes interactions between <span class="highlight" style="background-color:">signaling</span> components of the IL-15R and NKG2D, which was proposed by Horng et al. [129]. Jak3, as a part of canonical IL-15R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, is essential for the phosphorylation of DAP10. Activated DAP10 recruits PI3K and Grb2, which control proliferation, survival, and cytotoxicity of NK cells. DAP10 also associates with IL-15Rβ and -γ chains</div></p></div><div class="aux"><div class="resc">Biljana Zafirova, et al. Cell Mol Life Sci. 2011 November;68(21):3519-3529.</div><p class="links"><a class="dblinks" href="/pubmed/21898152" ref="ordinalpos=3149&amp;ncbi_uid=3300513&amp;link_uid=21898152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3192283/" ref="ordinalpos=3149&amp;ncbi_uid=3300513&amp;link_uid=PMC3192283">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3150.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4048185/figure/pone-0097788-g009/" ref="ordinalpos=3150&amp;ncbi_uid=6231221&amp;link_uid=PMC4048185" href="/pmc/articles/PMC4048185/"><img src="/pmc/articles/instance/4048185/bin/pone.0097788.g009.gif" src-large="/pmc/articles/instance/4048185/bin/pone.0097788.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4048185/" ref="ordinalpos=3150&amp;ncbi_uid=6231221&amp;link_uid=PMC4048185" image-link="/pmc/articles/PMC4048185/figure/pone-0097788-g009/" class="imagepopup">Figure 9. Schematic of the β2-AR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in CS-induced upregulation of MUC5AC production in NCI-H292 cells, a mucin-expressin cell line from human airway epithelia..  From: ?2-Adrenoceptor Involved in Smoking-Induced Airway Mucus Hypersecretion through ?-Arrestin-Dependent <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Wtih CSE stimulation, <i>β<sub>2</sub>-AR</i> is activated, which increases cAMP and recruits many <i>β</i>-arrestins. Increased cAMP binds and activates PKA but does not stimulate MUC5AC transcription and production. <i>β</i>-arrestin2 phosphorylates ERK1/2 and p38MAPK, the key players, which activate transcription factors to stimulate MUC5AC transcription and production. β-AR antagonists, kinase inhibitors and molecular biological methods of interfering with <span class="highlight" style="background-color:">signaling</span> molecules are also shown.</div></p></div><div class="aux"><div class="resc">Yujiao Zhou, et al. PLoS One. 2014;9(6):e97788.</div><p class="links"><a class="dblinks" href="/pubmed/24905583" ref="ordinalpos=3150&amp;ncbi_uid=6231221&amp;link_uid=24905583">Citation</a><a class="dblinks" href="/pmc/articles/PMC4048185/" ref="ordinalpos=3150&amp;ncbi_uid=6231221&amp;link_uid=PMC4048185">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3151.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3887879/figure/f1-molcell-35-6-463-1/" ref="ordinalpos=3151&amp;ncbi_uid=6342431&amp;link_uid=PMC3887879" href="/pmc/articles/PMC3887879/"><img src="/pmc/articles/instance/3887879/bin/molcell-35-6-463-1f1.gif" src-large="/pmc/articles/instance/3887879/bin/molcell-35-6-463-1f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3887879/" ref="ordinalpos=3151&amp;ncbi_uid=6342431&amp;link_uid=PMC3887879" image-link="/pmc/articles/PMC3887879/figure/f1-molcell-35-6-463-1/" class="imagepopup">Fig. 1.  From: Nutrient Regulation of the mTOR Complex 1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The key <span class="highlight" style="background-color:">signaling</span> pathways that regulate mTORC1 and mTORC2 and the composition of each mTOR complex are depicted. Multiple inputs from growth factors, amino acids, cellular energy status, and stress are integrated into mTORC1 through the mechanisms shown. When mTORC1 is active, it plays a major role in promoting cell growth and proliferation by stimulating various anabolic processes such as protein, lipid, and nucleotide synthesis and ribosome biogenesis, and by inhibiting catabolic processes such as autophagy. mTORC2 is regulated by growth factors through a poorly identified mechanism, but unlike mTORC1, it does not respond to other upstream signals derived from nutrients or stress.</div></p></div><div class="aux"><div class="resc">Sang Gyun Kim, et al. Mol Cells. 2013 June 30;35(6):463-473.</div><p class="links"><a class="dblinks" href="/pubmed/23694989" ref="ordinalpos=3151&amp;ncbi_uid=6342431&amp;link_uid=23694989">Citation</a><a class="dblinks" href="/pmc/articles/PMC3887879/" ref="ordinalpos=3151&amp;ncbi_uid=6342431&amp;link_uid=PMC3887879">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3152.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2810550/figure/F2/" ref="ordinalpos=3152&amp;ncbi_uid=2141045&amp;link_uid=PMC2810550" href="/pmc/articles/PMC2810550/"><img src="/pmc/articles/instance/2810550/bin/nihms165753f2.gif" src-large="/pmc/articles/instance/2810550/bin/nihms165753f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2810550/" ref="ordinalpos=3152&amp;ncbi_uid=2141045&amp;link_uid=PMC2810550" image-link="/pmc/articles/PMC2810550/figure/F2/" class="imagepopup">Figure 2.  From: A common biological mechanism in cancer and Alzheimer's disease?. </a></p><div class="supp"><p class="details"><div>The wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involvement in cancer and neurodegeneration. When wnt binds to the LRP-frizzled receptor in the surface of the cell,β-catenin is stabilized promoting expression of wnt target genes and proliferation. Subtle disequilibrium in any step of the <span class="highlight" style="background-color:">pathway</span> in a manner that determines activation of the <span class="highlight" style="background-color:">pathway</span>, such as increased expression or polimorphisms that induce activation of wnt or β-catenin would favor cancer development, preventing neurodegeneration. On the contrary, conditions that induce inactivation of the <span class="highlight" style="background-color:">pathway</span> would favor the development of Alzheimer’s disease or other degenerative disorder, and as a consequence protect from cancer development.</div></p></div><div class="aux"><div class="resc">Maria I Behrens, et al. Curr Alzheimer Res. ;6(3):196-204.</div><p class="links"><a class="dblinks" href="/pubmed/19519301" ref="ordinalpos=3152&amp;ncbi_uid=2141045&amp;link_uid=19519301">Citation</a><a class="dblinks" href="/pmc/articles/PMC2810550/" ref="ordinalpos=3152&amp;ncbi_uid=2141045&amp;link_uid=PMC2810550">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3153.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3288246/figure/F8/" ref="ordinalpos=3153&amp;ncbi_uid=4710778&amp;link_uid=PMC3288246" href="/pmc/articles/PMC3288246/"><img src="/pmc/articles/instance/3288246/bin/nihms-341571-f0008.gif" src-large="/pmc/articles/instance/3288246/bin/nihms-341571-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3288246/" ref="ordinalpos=3153&amp;ncbi_uid=4710778&amp;link_uid=PMC3288246" image-link="/pmc/articles/PMC3288246/figure/F8/" class="imagepopup">Fig. 8.  From: ERK/GSK3ss/Snail <span class="highlight" style="background-color:">signaling</span> mediates radiation-induced alveolar epithelial-to-mesenchymal transition. </a></p><div class="supp"><p class="details"><div>Schematic representation of the proposed mechanism for radiation-induced EMT in alveolar type II epithelial cells. Radiation-induced generation of ROS leads to activation of the MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activated ERK1/2 causes the inactivation of GSK3ß by phosphorylation of the serine 9 active site, which leads to the disassociation of Snail from GSK3ß. Snail then translocates to the nucleus where it functions as a repressor of E-cadherin and initiate EMT in AE2 cells.</div></p></div><div class="aux"><div class="resc">Devipriya Nagarajan, et al. Free Radic Biol Med. ;52(6):983-992.</div><p class="links"><a class="dblinks" href="/pubmed/22198183" ref="ordinalpos=3153&amp;ncbi_uid=4710778&amp;link_uid=22198183">Citation</a><a class="dblinks" href="/pmc/articles/PMC3288246/" ref="ordinalpos=3153&amp;ncbi_uid=4710778&amp;link_uid=PMC3288246">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3154.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872116/figure/F2/" ref="ordinalpos=3154&amp;ncbi_uid=2370044&amp;link_uid=PMC2872116" href="/pmc/articles/PMC2872116/"><img src="/pmc/articles/instance/2872116/bin/nihms193429f2.gif" src-large="/pmc/articles/instance/2872116/bin/nihms193429f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872116/" ref="ordinalpos=3154&amp;ncbi_uid=2370044&amp;link_uid=PMC2872116" image-link="/pmc/articles/PMC2872116/figure/F2/" class="imagepopup">Fig. 2.  From: Visual integration of quantitative proteomic data, pathways and protein interactions. </a></p><div class="supp"><p class="details"><div>Structuring protein interactions around familiar canonical pathways provides intuitive visualizations. A canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> representation (top) can be imported into the system in two ways: on the lower left the <span class="highlight" style="background-color:">pathway</span> image itself is loaded into the system and preprocessed by circling proteins and drawing over interactions; the <span class="highlight" style="background-color:">pathway</span> features are then inferred from the user strokes and image features and shown here in black; or, on the lower right, protein and interaction icons are placed and dragged on an empty canvas to create a new <span class="highlight" style="background-color:">pathway</span> model. After positional assignment of each protein, the software aids in associating interaction database accession numbers to each of the newly defined canonical <span class="highlight" style="background-color:">pathway</span> proteins.</div></p></div><div class="aux"><div class="resc">Radu Jianu, et al. IEEE Trans Vis Comput Graph. ;16(4):609-620.</div><p class="links"><a class="dblinks" href="/pubmed/20467059" ref="ordinalpos=3154&amp;ncbi_uid=2370044&amp;link_uid=20467059">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872116/" ref="ordinalpos=3154&amp;ncbi_uid=2370044&amp;link_uid=PMC2872116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3155.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2877705/figure/pone-0010859-g007/" ref="ordinalpos=3155&amp;ncbi_uid=2249122&amp;link_uid=PMC2877705" href="/pmc/articles/PMC2877705/"><img src="/pmc/articles/instance/2877705/bin/pone.0010859.g007.gif" src-large="/pmc/articles/instance/2877705/bin/pone.0010859.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2877705/" ref="ordinalpos=3155&amp;ncbi_uid=2249122&amp;link_uid=PMC2877705" image-link="/pmc/articles/PMC2877705/figure/pone-0010859-g007/" class="imagepopup">Figure 7. A model showing PTEN/PI3K/AKT <span class="highlight" style="background-color:">pathway</span> with miR-22 and the regulatory loop exerted by miR-22..  From: miR-22 Forms a Regulatory Loop in PTEN/AKT <span class="highlight" style="background-color:">Pathway</span> and Modulates <span class="highlight" style="background-color:">Signaling</span> Kinetics. </a></p><div class="aux"><div class="resc">Nadav Bar, et al. PLoS One. 2010;5(5):e10859.</div><p class="links"><a class="dblinks" href="/pubmed/20523723" ref="ordinalpos=3155&amp;ncbi_uid=2249122&amp;link_uid=20523723">Citation</a><a class="dblinks" href="/pmc/articles/PMC2877705/" ref="ordinalpos=3155&amp;ncbi_uid=2249122&amp;link_uid=PMC2877705">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3156.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3111669/figure/Fig2/" ref="ordinalpos=3156&amp;ncbi_uid=3092153&amp;link_uid=PMC3111669" href="/pmc/articles/PMC3111669/"><img src="/pmc/articles/instance/3111669/bin/11064_2011_422_Fig2_HTML.gif" src-large="/pmc/articles/instance/3111669/bin/11064_2011_422_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3111669/" ref="ordinalpos=3156&amp;ncbi_uid=3092153&amp;link_uid=PMC3111669" image-link="/pmc/articles/PMC3111669/figure/Fig2/" class="imagepopup">Fig. 2.  From: Role of BMP, FGF, Calcium <span class="highlight" style="background-color:">Signaling</span>, and Zic Proteins in Vertebrate Neuroectodermal Differentiation. </a></p><div class="supp"><p class="details"><div>BMP <span class="highlight" style="background-color:">signaling</span> and its blocking. In the course of gastrulation, natural BMP-signal blocking occurs in BCNE through the physical interaction between BMP2/BMP4 and noggin/chordin. Three ways of experimentally blocking BMP signal in Xenopus embryos are shown: (1) dominant-negative type BMP receptor; (2) Smad6 (inhibitory Smad); and (3) Smad5-sbn (somitabun), which is thought to be deficient in binding to Smad4 and to form an inactive heteromeric complex with intact Smad5 and Smad1, resulting in efficient inhibition of the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Phosphorylation of Smad1 by FGF/MAPK <span class="highlight" style="background-color:">signaling</span> (<i>P</i> on <i>black background</i>) occurs at a different amino acid position from that of BMP receptor-mediated phosphorylation (<i>P</i> on <i>white background</i>), and inhibits the transcriptional activity of Smad1. Natural inhibitory factors for <i>BMP</i> <span class="highlight" style="background-color:">signaling</span> are <i>highlighted with gray</i></div></p></div><div class="aux"><div class="resc">Jun Aruga, et al. Neurochem Res. 2011 July;36(7):1286-1292.</div><p class="links"><a class="dblinks" href="/pubmed/21336820" ref="ordinalpos=3156&amp;ncbi_uid=3092153&amp;link_uid=21336820">Citation</a><a class="dblinks" href="/pmc/articles/PMC3111669/" ref="ordinalpos=3156&amp;ncbi_uid=3092153&amp;link_uid=PMC3111669">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3157.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4381104/figure/fig1/" ref="ordinalpos=3157&amp;ncbi_uid=7391486&amp;link_uid=PMC4381104" href="/pmc/articles/PMC4381104/"><img src="/pmc/articles/instance/4381104/bin/tpj201445f1.gif" src-large="/pmc/articles/instance/4381104/bin/tpj201445f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4381104/" ref="ordinalpos=3157&amp;ncbi_uid=7391486&amp;link_uid=PMC4381104" image-link="/pmc/articles/PMC4381104/figure/fig1/" class="imagepopup">Figure 1.  From: Gene signatures of drug resistance predict patient survival in colorectal cancer. </a></p><div class="supp"><p class="details"><div>(<b>a1</b>) The differentially expressed genes between 5-fluorouracil (5-FU)-resistant and control HCT116 cells were subjected to oncology analysis. These genes can be categorized based on their functions into metabolic process (53.6%), growth (27.7%), pigmentation (12%), viral reproduction (3.8%) and cell proliferation (2.3%). (<b>a2</b>) The differentially expressed genes between 5-FU-resistant and control HCT116 cells were subjected to ingenuity <span class="highlight" style="background-color:">pathway</span> analysis and the top five enriched <span class="highlight" style="background-color:">signaling</span> networks were obtained. (<b>b1</b>) Gene Ontology analysis indicated that these differentially expressed genes between oxaliplatin-resistant and control HCT116 cells were enriched in functions involving rhythmic process (31.7%), viral reproduction (27.2%), cellular process (26.2%), cell proliferation (7.7%) and metabolic process (6.1%). (<b>b2</b>) The differentially expressed genes between oxaliplatin-resistant and control HCT116 cells were subjected to ingenuity <span class="highlight" style="background-color:">pathway</span> analysis and the top five enriched <span class="highlight" style="background-color:">signaling</span> networks were obtained. (<b>c1</b>) Gene Ontology analysis indicated that the differentially expressed genes between SN38-resistant and control HCT116 cells were enriched in functions involving cell proliferation (39.7%), response to stimulus (16.1%), growth (15.9%), rhythmic process (11%), viral reproduction (8.6%) and cellular process (8.4%). (<b>c2</b>) The differentially expressed genes between SN38-resistant and control HCT116 cells were subjected to ingenuity <span class="highlight" style="background-color:">pathway</span> analysis and the top five enriched <span class="highlight" style="background-color:">signaling</span> networks were obtained.</div></p></div><div class="aux"><div class="resc">Y Zheng, et al. Pharmacogenomics J. 2015 April;15(2):135-143.</div><p class="links"><a class="dblinks" href="/pubmed/25179828" ref="ordinalpos=3157&amp;ncbi_uid=7391486&amp;link_uid=25179828">Citation</a><a class="dblinks" href="/pmc/articles/PMC4381104/" ref="ordinalpos=3157&amp;ncbi_uid=7391486&amp;link_uid=PMC4381104">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3158.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3708179/figure/F8/" ref="ordinalpos=3158&amp;ncbi_uid=6326238&amp;link_uid=PMC3708179" href="/pmc/articles/PMC3708179/"><img src="/pmc/articles/instance/3708179/bin/zjw0071344920008.gif" src-large="/pmc/articles/instance/3708179/bin/zjw0071344920008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3708179/" ref="ordinalpos=3158&amp;ncbi_uid=6326238&amp;link_uid=PMC3708179" image-link="/pmc/articles/PMC3708179/figure/F8/" class="imagepopup">Fig. 8. From: Protein Sets Define Disease States and Predict In Vivo Effects of Drug Treatment. </a></p><div class="supp"><p class="details"><div><b>Enrichment of hyper- and hypophosphorylated peptides in the ERK/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the phosphoproteome of murine liver on treatment with high-fat <i>versus</i> low-fat diet.</b> Proteins with phosphopeptide ratios ≤0.75 or ≥1.33 were marked in gray. The ERK/MAPK <span class="highlight" style="background-color:">pathway</span> includes the apoptosis initiating BAD protein (Fig. 6<i>D</i>). Enrichment analysis and visualization were performed with Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA).</div></p></div><div class="aux"><div class="resc">David Meierhofer, et al. Mol Cell Proteomics. 2013 July;12(7):1965-1979.</div><p class="links"><a class="dblinks" href="/pubmed/23579186" ref="ordinalpos=3158&amp;ncbi_uid=6326238&amp;link_uid=23579186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3708179/" ref="ordinalpos=3158&amp;ncbi_uid=6326238&amp;link_uid=PMC3708179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3159.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3385931/figure/F2/" ref="ordinalpos=3159&amp;ncbi_uid=4083623&amp;link_uid=PMC3385931" href="/pmc/articles/PMC3385931/"><img src="/pmc/articles/instance/3385931/bin/nihms378349f2.gif" src-large="/pmc/articles/instance/3385931/bin/nihms378349f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3385931/" ref="ordinalpos=3159&amp;ncbi_uid=4083623&amp;link_uid=PMC3385931" image-link="/pmc/articles/PMC3385931/figure/F2/" class="imagepopup">Fig. 2.  From: Dynamic Single-Cell Network Profiles in Acute Myelogenous Leukemia Are Associated with Patient Response to Standard Induction Therapy. </a></p><div class="supp"><p class="details"><div>Role of each metric in assessing different aspects of <span class="highlight" style="background-color:">signaling</span> biology. Summary schema of all metrics used in the two studies and the role each has in assessing different aspects of <span class="highlight" style="background-color:">signaling</span> and <span class="highlight" style="background-color:">pathway</span> biology.</div></p></div><div class="aux"><div class="resc">Steven M. Kornblau, et al. Clin Cancer Res. ;16(14):3721-3733.</div><p class="links"><a class="dblinks" href="/pubmed/20525753" ref="ordinalpos=3159&amp;ncbi_uid=4083623&amp;link_uid=20525753">Citation</a><a class="dblinks" href="/pmc/articles/PMC3385931/" ref="ordinalpos=3159&amp;ncbi_uid=4083623&amp;link_uid=PMC3385931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3160.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4198691/figure/f0030/" ref="ordinalpos=3160&amp;ncbi_uid=6744790&amp;link_uid=PMC4198691" href="/pmc/articles/PMC4198691/"><img src="/pmc/articles/instance/4198691/bin/gr6.gif" src-large="/pmc/articles/instance/4198691/bin/gr6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4198691/" ref="ordinalpos=3160&amp;ncbi_uid=6744790&amp;link_uid=PMC4198691" image-link="/pmc/articles/PMC4198691/figure/f0030/" class="imagepopup">Figure 6.  From: Essential Function for PDLIM2 in Cell Polarization in Three-Dimensional Cultures by Feedback Regulation of the ?1-Integrin-RhoA <span class="highlight" style="background-color:">Signaling</span> Axis. </a></p><div class="supp"><p class="details"><div>PDLIM2 has an essential function in maintaining breast epithelial cell polarity at the level of feedback <span class="highlight" style="background-color:">signaling</span> through the β1-integrin–FAK–RhoA axis. Suppression of PDLIM2 expression prevents negative feedback control of FAK-ROCK-RhoA <span class="highlight" style="background-color:">signaling</span> downstream of β1-integrin resulting in hyperactivation of the <span class="highlight" style="background-color:">pathway</span> causing dysregulation of polarization, proliferation/apoptosis, cell-cell and cell-matrix adhesion, and loss of normal phenotype of breast epithelial cells.</div></p></div><div class="aux"><div class="resc">Ravi Kiran Deevi, et al. Neoplasia. 2014 May;16(5):422-431.</div><p class="links"><a class="dblinks" href="/pubmed/24863845" ref="ordinalpos=3160&amp;ncbi_uid=6744790&amp;link_uid=24863845">Citation</a><a class="dblinks" href="/pmc/articles/PMC4198691/" ref="ordinalpos=3160&amp;ncbi_uid=6744790&amp;link_uid=PMC4198691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3161.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3602389/figure/F2/" ref="ordinalpos=3161&amp;ncbi_uid=5859188&amp;link_uid=PMC3602389" href="/pmc/articles/PMC3602389/"><img src="/pmc/articles/instance/3602389/bin/nihms-440010-f0002.gif" src-large="/pmc/articles/instance/3602389/bin/nihms-440010-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3602389/" ref="ordinalpos=3161&amp;ncbi_uid=5859188&amp;link_uid=PMC3602389" image-link="/pmc/articles/PMC3602389/figure/F2/" class="imagepopup">Figure 2.  From: The effects of ingested mammalian blood factors on vector arthropod immunity and physiology. </a></p><div class="supp"><p class="details"><div>Model of TGF-β1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vector arthropods.</div></p></div><div class="aux"><div class="resc">Nazzy Pakpour, et al. Microbes Infect. 2013 March;15(3):243-254.</div><p class="links"><a class="dblinks" href="/pubmed/23370408" ref="ordinalpos=3161&amp;ncbi_uid=5859188&amp;link_uid=23370408">Citation</a><a class="dblinks" href="/pmc/articles/PMC3602389/" ref="ordinalpos=3161&amp;ncbi_uid=5859188&amp;link_uid=PMC3602389">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3162.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679989/figure/F1/" ref="ordinalpos=3162&amp;ncbi_uid=1848899&amp;link_uid=PMC2679989" href="/pmc/articles/PMC2679989/"><img src="/pmc/articles/instance/2679989/bin/nihms100433f1.gif" src-large="/pmc/articles/instance/2679989/bin/nihms100433f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679989/" ref="ordinalpos=3162&amp;ncbi_uid=1848899&amp;link_uid=PMC2679989" image-link="/pmc/articles/PMC2679989/figure/F1/" class="imagepopup">Fig. 1. Schematic representation of the Nod2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Function of Nod-like Receptors in Microbial Recognition and Host Defense. </a></p><div class="supp"><p class="details"><div>The NLR Nod2 recognizes bacterial PGN fragments such as MDP and activates NF-κB and MAPK <span class="highlight" style="background-color:">signaling</span> via the indicated proteins leading to the transcriptional upregulation of a variety of proinflammatory cytokines and antimicrobial peptides. Negative feedback regulation occurs upon induction of the deubiquitanase A20. The posttranslational regulation of various components of the <span class="highlight" style="background-color:">pathway</span> is indicated by green circles (phosphorylation) and red or blue circles (ubiquitination). In the case of RICK, K63-mediated polyubiquitination at K209 is essential for recruitment of Tak1. For NEMO, K48-mediated ubiquitination at residue K399 targets it for proteasome-dependent degradation. K63-linked polyubiquitination at amino acid K285 is important for complex formation with RICK (48).</div></p></div><div class="aux"><div class="resc">Luigi Franchi, et al. Immunol Rev. ;227(1):106-128.</div><p class="links"><a class="dblinks" href="/pubmed/19120480" ref="ordinalpos=3162&amp;ncbi_uid=1848899&amp;link_uid=19120480">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679989/" ref="ordinalpos=3162&amp;ncbi_uid=1848899&amp;link_uid=PMC2679989">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3163.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2685917/figure/F9/" ref="ordinalpos=3163&amp;ncbi_uid=1829254&amp;link_uid=PMC2685917" href="/pmc/articles/PMC2685917/"><img src="/pmc/articles/instance/2685917/bin/halms340599f9.gif" src-large="/pmc/articles/instance/2685917/bin/halms340599f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2685917/" ref="ordinalpos=3163&amp;ncbi_uid=1829254&amp;link_uid=PMC2685917" image-link="/pmc/articles/PMC2685917/figure/F9/" class="imagepopup">Figure 9. Schematic diagram showing the intracellular pathways for IGF1, AR or combination of the both growth factors involved in serum deprivation-induced apoptosis in NSCLC cell lines.  From: Cooperation of amphiregulin and insulin-like growth factor-1 inhibits Bax- and Bad-mediated apoptosis via a protein kinase C-dependent <span class="highlight" style="background-color:">pathway</span> in non-small cell lung cancer cells. </a></p><div class="supp"><p class="details"><div><b>A:</b> <span class="highlight" style="background-color:">Signaling</span> pathways activated by IGF1 or AR growth factors used as single agent. IGF1 activates the IGF1 receptor and the PI<sub>3</sub>K/Akt <span class="highlight" style="background-color:">pathway</span>. AR activates the EGF receptor which is linked to the MAPK/ERK and the PI<sub>3</sub>K/Akt pathways. Both IGF1 and EGF receptors converge to the inactivation of the pro-apoptotic protein Bad. <b>B:</b> Survival <span class="highlight" style="background-color:">pathway</span> activated by the cooperation of IGF1 and AR growth factors. AR and IGF1, present in the H358 CM, activate both IGF1 and EGF receptors (box) (11). This cooperation induces a PKC-dependent survival <span class="highlight" style="background-color:">pathway</span>, independent of MAPK/ERK and PI<sub>3</sub>K/Akt pathways. This anti-apoptotic <span class="highlight" style="background-color:">pathway</span> leads to Bad phosphorylation on Ser<sup>112</sup> and inhibition of conformational change of the pro-apoptotic protein Bax. Inhibitors used are mentioned in blue.</div></p></div><div class="aux"><div class="resc">Amandine Hurbin, et al. J Biol Chem. ;280(20):19757-19767.</div><p class="links"><a class="dblinks" href="/pubmed/15767261" ref="ordinalpos=3163&amp;ncbi_uid=1829254&amp;link_uid=15767261">Citation</a><a class="dblinks" href="/pmc/articles/PMC2685917/" ref="ordinalpos=3163&amp;ncbi_uid=1829254&amp;link_uid=PMC2685917">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3164.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469159/figure/F3/" ref="ordinalpos=3164&amp;ncbi_uid=4287959&amp;link_uid=PMC3469159" href="/pmc/articles/PMC3469159/"><img src="/pmc/articles/instance/3469159/bin/nihms-411756-f0003.gif" src-large="/pmc/articles/instance/3469159/bin/nihms-411756-f0003.jpg" alt="Fig. 10.3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469159/" ref="ordinalpos=3164&amp;ncbi_uid=4287959&amp;link_uid=PMC3469159" image-link="/pmc/articles/PMC3469159/figure/F3/" class="imagepopup">Fig. 10.3. Lineage allocation in the ICM depends on FGF/MAPK <span class="highlight" style="background-color:">signaling</span>.  From: Cell Lineage Allocation Within the Inner Cell Mass of the Mouse Blastocyst. </a></p><div class="supp"><p class="details"><div>At around the 32-cell stage, when EPI- and PrE-specific transcription factors are expressed homogeneously in all ICM cells, a differential expression of the ligand Fgf4 and the receptor Fgfr2 starts to become evident. EPI precursor cells express and secrete Fgf4, whereas PrE precursor cells express Fgfr2. Binding of Fgf4 to Fgfr2 activates the FGF/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is required for the induction of PrE-specific transcription factors, thus promoting PrE formation. When the <span class="highlight" style="background-color:">signaling</span> is not active, such as in <i>Fgf4</i> mutants or by inhibiting the receptor Fgfr2, PrE is not formed and all ICM cells contribute to EPI. On the contrary, in the presence of excess Fgf4, all ICM cells contribute to PrE</div></p></div><div class="aux"><div class="resc">Panagiotis Xenopoulos, et al. Results Probl Cell Differ. 2012;55:185-202.</div><p class="links"><a class="dblinks" href="/pubmed/22918807" ref="ordinalpos=3164&amp;ncbi_uid=4287959&amp;link_uid=22918807">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469159/" ref="ordinalpos=3164&amp;ncbi_uid=4287959&amp;link_uid=PMC3469159">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3165.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3954580/figure/F5/" ref="ordinalpos=3165&amp;ncbi_uid=5918736&amp;link_uid=PMC3954580" href="/pmc/articles/PMC3954580/"><img src="/pmc/articles/instance/3954580/bin/nihms337812f5.gif" src-large="/pmc/articles/instance/3954580/bin/nihms337812f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3954580/" ref="ordinalpos=3165&amp;ncbi_uid=5918736&amp;link_uid=PMC3954580" image-link="/pmc/articles/PMC3954580/figure/F5/" class="imagepopup">Fig. 5.  From: MEK5/ERK5 <span class="highlight" style="background-color:">Pathway</span>: The first fifteen years. </a></p><div class="supp"><p class="details"><div>Proposed MEK5 regulation in breast cancer: Similarly to MEK5/ERK5 <span class="highlight" style="background-color:">pathway</span> activation in physiologically healthy tissue, activation may occur through extracellular stressors, mitogens or cytokine activation to promote survival, anti-apoptotic <span class="highlight" style="background-color:">signaling</span> and proliferation of cancer cells. Some of the microarray analysis of breast cancer tissue samples as well as experiments completed by our laboratory suggests that activation of this <span class="highlight" style="background-color:">pathway</span> may lead to epithelial mesenchymal transitions (EMT), metastasis and invasion especially in more aggressive phenotypes of breast cancer.</div></p></div><div class="aux"><div class="resc">Barbara A. Drew, et al. Biochim Biophys Acta. 2012 January;1825(1):37-48.</div><p class="links"><a class="dblinks" href="/pubmed/22020294" ref="ordinalpos=3165&amp;ncbi_uid=5918736&amp;link_uid=22020294">Citation</a><a class="dblinks" href="/pmc/articles/PMC3954580/" ref="ordinalpos=3165&amp;ncbi_uid=5918736&amp;link_uid=PMC3954580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3166.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726964/figure/fig03/" ref="ordinalpos=3166&amp;ncbi_uid=5171709&amp;link_uid=PMC3726964" href="/pmc/articles/PMC3726964/"><img src="/pmc/articles/instance/3726964/bin/ijcep0006-1493-f3.gif" src-large="/pmc/articles/instance/3726964/bin/ijcep0006-1493-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726964/" ref="ordinalpos=3166&amp;ncbi_uid=5171709&amp;link_uid=PMC3726964" image-link="/pmc/articles/PMC3726964/figure/fig03/" class="imagepopup">Figure 3.  From: RAF1-MEK1-ERK/AKT axis may confer NSCLC cell lines resistance to erlotinib. </a></p><div class="supp"><p class="details"><div>Regulation of the EMT <span class="highlight" style="background-color:">Pathway</span> and Tec kinase <span class="highlight" style="background-color:">signaling</span> in Calu-6 compared to U1752. This figure was derived from IPA. The colored molecules were genes whose expression altered significantly in Calu-6 compared to U1752. The red color represented up-regulated genes in Calu-6, while the green color represented down-regulated genes in Calu-6. A: Regulation of the EMT <span class="highlight" style="background-color:">Pathway</span> in Calu-6 compared to U1752. TGF-β-EMT was shown, while Wnt-EMT, Notch-EMT and Receptor tyrosine kinase-EMT were not shown. B: Tec kinase <span class="highlight" style="background-color:">signaling</span> in Calu-6 compared to U1752.</div></p></div><div class="aux"><div class="resc">Zhi-Hong Xu, et al. Int J Clin Exp Pathol. 2013;6(8):1493-1504.</div><p class="links"><a class="dblinks" href="/pubmed/23923067" ref="ordinalpos=3166&amp;ncbi_uid=5171709&amp;link_uid=23923067">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726964/" ref="ordinalpos=3166&amp;ncbi_uid=5171709&amp;link_uid=PMC3726964">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3167.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3934076/figure/F0001/" ref="ordinalpos=3167&amp;ncbi_uid=5883120&amp;link_uid=PMC3934076" href="/pmc/articles/PMC3934076/"><img src="/pmc/articles/instance/3934076/bin/WO-17-20946-g001.gif" src-large="/pmc/articles/instance/3934076/bin/WO-17-20946-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3934076/" ref="ordinalpos=3167&amp;ncbi_uid=5883120&amp;link_uid=PMC3934076" image-link="/pmc/articles/PMC3934076/figure/F0001/" class="imagepopup">Fig. 1.  From: The Notch ligand Delta-like 4 (DLL4) as a target in angiogenesis-based cancer therapy?. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> – details in the text</div></p></div><div class="aux"><div class="resc">Marlena Brzozowa, et al. Contemp Oncol (Pozn). 2013;17(3):234-237.</div><p class="links"><a class="dblinks" href="/pubmed/24596507" ref="ordinalpos=3167&amp;ncbi_uid=5883120&amp;link_uid=24596507">Citation</a><a class="dblinks" href="/pmc/articles/PMC3934076/" ref="ordinalpos=3167&amp;ncbi_uid=5883120&amp;link_uid=PMC3934076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3168.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035650/figure/pone-0016653-g007/" ref="ordinalpos=3168&amp;ncbi_uid=2778479&amp;link_uid=PMC3035650" href="/pmc/articles/PMC3035650/"><img src="/pmc/articles/instance/3035650/bin/pone.0016653.g007.gif" src-large="/pmc/articles/instance/3035650/bin/pone.0016653.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035650/" ref="ordinalpos=3168&amp;ncbi_uid=2778479&amp;link_uid=PMC3035650" image-link="/pmc/articles/PMC3035650/figure/pone-0016653-g007/" class="imagepopup">Figure 7. Simplified schematic diagram of a MCP-1/MMP-2/TGF-β1 loop <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: A Local Proinflammatory Signalling Loop Facilitates Adverse Age-Associated Arterial Remodeling. </a></p><div class="aux"><div class="resc">Mingyi Wang, et al. PLoS One. 2011;6(2):e16653.</div><p class="links"><a class="dblinks" href="/pubmed/21347430" ref="ordinalpos=3168&amp;ncbi_uid=2778479&amp;link_uid=21347430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035650/" ref="ordinalpos=3168&amp;ncbi_uid=2778479&amp;link_uid=PMC3035650">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3169.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2292640/figure/F1/" ref="ordinalpos=3169&amp;ncbi_uid=1312948&amp;link_uid=PMC2292640" href="/pmc/articles/PMC2292640/"><img src="/pmc/articles/instance/2292640/bin/nihms44316f1.gif" src-large="/pmc/articles/instance/2292640/bin/nihms44316f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2292640/" ref="ordinalpos=3169&amp;ncbi_uid=1312948&amp;link_uid=PMC2292640" image-link="/pmc/articles/PMC2292640/figure/F1/" class="imagepopup">Figure 1. IFN sensing and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: SARS CORONAVIRUS AND INNATE IMMUNITY. </a></p><div class="supp"><p class="details"><div>RNA viruses are internalized through several mechanisms, either fusion with the plasma membrane or binding to a surface receptor (ACE2 for SARS-CoV, carcinoembryonic antigen-related cell adhesion molecule 1 (CEACAM1) for MHV). That internalization exposes the genomic RNA to the dsRNA sensing machinery in the cell; TLR3, RIGI and MDA5. These proteins signal the IRF3 cascade leading to induction of IFNb and production of secreted IFNβ protein. That IFNβ protein can then bind IFNα/β receptors (INFAR1) on the surface of the same cell or surrounding cells. This activates the Stat1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to activate the many anti-viral genes found with ISRE promoter elements.</div></p></div><div class="aux"><div class="resc">Matthew Frieman, et al. Virus Res. ;133(1):101-112.</div><p class="links"><a class="dblinks" href="/pubmed/17451827" ref="ordinalpos=3169&amp;ncbi_uid=1312948&amp;link_uid=17451827">Citation</a><a class="dblinks" href="/pmc/articles/PMC2292640/" ref="ordinalpos=3169&amp;ncbi_uid=1312948&amp;link_uid=PMC2292640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3170.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2910980/figure/fig12/" ref="ordinalpos=3170&amp;ncbi_uid=2985891&amp;link_uid=PMC2910980" href="/pmc/articles/PMC2910980/"><img src="/pmc/articles/instance/2910980/bin/PC.073874.wc.f12.gif" src-large="/pmc/articles/instance/2910980/bin/PC.073874.wc.f12.jpg" alt="Figure 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2910980/" ref="ordinalpos=3170&amp;ncbi_uid=2985891&amp;link_uid=PMC2910980" image-link="/pmc/articles/PMC2910980/figure/fig12/" class="imagepopup">Figure 12. From: The Mg-Chelatase H Subunit of Arabidopsis Antagonizes a Group of WRKY Transcription Repressors to Relieve ABA-Responsive Genes of Inhibition[W][OA]. </a></p><div class="supp"><p class="details"><div>Proposition of an ABAR-Mediated <span class="highlight" style="background-color:">Signaling</span> Model.<br /><b>(A)</b> A model of functional domains in ABAR molecule. Numbers of the amino acid residues are shown in the molecule. See text for the model.<br /><b>(B)</b> A model of ABAR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The symbol &lt;?&gt; indicates an unknown factor or <span class="highlight" style="background-color:">signaling</span> cascade that may repress the <i>WRKY40</i> gene expression. Note that ABAR interacts with WRKY40 in the absence or low levels of ABA, and ABA at high levels promotes this bimolecular interaction and triggers downstream <span class="highlight" style="background-color:">signaling</span> cascade. See text for this model in detail.</div></p></div><div class="aux"><div class="resc">Yi Shang, et al. Plant Cell. 2010 June;22(6):1909-1935.</div><p class="links"><a class="dblinks" href="/pubmed/20543028" ref="ordinalpos=3170&amp;ncbi_uid=2985891&amp;link_uid=20543028">Citation</a><a class="dblinks" href="/pmc/articles/PMC2910980/" ref="ordinalpos=3170&amp;ncbi_uid=2985891&amp;link_uid=PMC2910980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3171.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2860273/figure/F1/" ref="ordinalpos=3171&amp;ncbi_uid=2294468&amp;link_uid=PMC2860273" href="/pmc/articles/PMC2860273/"><img src="/pmc/articles/instance/2860273/bin/nihms170797f1.gif" src-large="/pmc/articles/instance/2860273/bin/nihms170797f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2860273/" ref="ordinalpos=3171&amp;ncbi_uid=2294468&amp;link_uid=PMC2860273" image-link="/pmc/articles/PMC2860273/figure/F1/" class="imagepopup">FIGURE 1.  From: The Ubiquitin Ligase Siah2 and the Hypoxia Response. </a></p><div class="supp"><p class="details"><div>Hypoxic <span class="highlight" style="background-color:">signaling</span> connected to Siah. Under the hypoxia condition, hypoxia-activated p38 and Akt pathways positively regulate Siah2 activity through phosphorylation and induction, respectively. Siah2 is phosphorylated by p38 on its serine and threonine residues, which will enhance its activity. Akt <span class="highlight" style="background-color:">pathway</span> increases the abundance of Siah2 by induction of its mRNA through <span class="highlight" style="background-color:">signaling</span> pathways yet to be clarified. Induced and activated Siah2 modulates the hypoxic <span class="highlight" style="background-color:">signaling</span> through PHD3 degradation and HIF-1α stabilization on one hand. Although, it is not yet identified, other Siah2 substrates would also play roles on hypoxic <span class="highlight" style="background-color:">signaling</span> by both HIF-1α-dependent and -independent mechanism.</div></p></div><div class="aux"><div class="resc">Koh Nakayama, et al. Mol Cancer Res. ;7(4):443-451.</div><p class="links"><a class="dblinks" href="/pubmed/19372575" ref="ordinalpos=3171&amp;ncbi_uid=2294468&amp;link_uid=19372575">Citation</a><a class="dblinks" href="/pmc/articles/PMC2860273/" ref="ordinalpos=3171&amp;ncbi_uid=2294468&amp;link_uid=PMC2860273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3172.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3986667/figure/F3/" ref="ordinalpos=3172&amp;ncbi_uid=6095155&amp;link_uid=PMC3986667" href="/pmc/articles/PMC3986667/"><img src="/pmc/articles/instance/3986667/bin/1752-0509-8-39-3.gif" src-large="/pmc/articles/instance/3986667/bin/1752-0509-8-39-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3986667/" ref="ordinalpos=3172&amp;ncbi_uid=6095155&amp;link_uid=PMC3986667" image-link="/pmc/articles/PMC3986667/figure/F3/" class="imagepopup">Figure 3.  From: The protein-protein interaction network of eyestalk, Y-organ and hepatopancreas in Chinese mitten crab Eriocheir sinensis. </a></p><div class="supp"><p class="details"><div><b>The <span class="highlight" style="background-color:">signaling</span> sub-network of </b><b><i>E. sinensis. </i></b><b>A</b>: The 68 proteins in the 7 known pathways. <b>B</b>: Hippo signal transduction <span class="highlight" style="background-color:">pathway</span>. <b>C</b>: Jak-STAT signal transduction <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tong Hao, et al. BMC Syst Biol. 2014;8:39-39.</div><p class="links"><a class="dblinks" href="/pubmed/24674293" ref="ordinalpos=3172&amp;ncbi_uid=6095155&amp;link_uid=24674293">Citation</a><a class="dblinks" href="/pmc/articles/PMC3986667/" ref="ordinalpos=3172&amp;ncbi_uid=6095155&amp;link_uid=PMC3986667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3173.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3545399/figure/F6/" ref="ordinalpos=3173&amp;ncbi_uid=4532822&amp;link_uid=PMC3545399" href="/pmc/articles/PMC3545399/"><img src="/pmc/articles/instance/3545399/bin/nihms424897f6.gif" src-large="/pmc/articles/instance/3545399/bin/nihms424897f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3545399/" ref="ordinalpos=3173&amp;ncbi_uid=4532822&amp;link_uid=PMC3545399" image-link="/pmc/articles/PMC3545399/figure/F6/" class="imagepopup">Figure 6.  From: PI3K Synergizes With Loss of PKC to Elicit Apoptosis via the UPR. </a></p><div class="supp"><p class="details"><div>Stochastic <span class="highlight" style="background-color:">signaling</span> pathways regulated by oncogenic Ras in the induction of apoptosis. The <span class="highlight" style="background-color:">pathway</span> governed by JNK in grey color was reported previously. The signal <span class="highlight" style="background-color:">pathway</span> regulated by PI3K/Akt in dark color is presented in this study.</div></p></div><div class="aux"><div class="resc">Jinjin Guo, et al. J Cell Biochem. ;107(1):76-85.</div><p class="links"><a class="dblinks" href="/pubmed/19241442" ref="ordinalpos=3173&amp;ncbi_uid=4532822&amp;link_uid=19241442">Citation</a><a class="dblinks" href="/pmc/articles/PMC3545399/" ref="ordinalpos=3173&amp;ncbi_uid=4532822&amp;link_uid=PMC3545399">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3174.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847551/figure/F1/" ref="ordinalpos=3174&amp;ncbi_uid=5610457&amp;link_uid=PMC3847551" href="/pmc/articles/PMC3847551/"><img src="/pmc/articles/instance/3847551/bin/fphys-04-00353-g0001.gif" src-large="/pmc/articles/instance/3847551/bin/fphys-04-00353-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847551/" ref="ordinalpos=3174&amp;ncbi_uid=5610457&amp;link_uid=PMC3847551" image-link="/pmc/articles/PMC3847551/figure/F1/" class="imagepopup">Figure 1.  From: Regulation of insect behavior via the insulin-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Regulation of behavior by the Insulin-Producing Cells (IPCs) in Drosophila</b>. The IPCs modulate feeding and locomotor (shaded) behavior through the insulin <span class="highlight" style="background-color:">pathway</span>. Decreased insulin production/release from the brain IPCs results in decreased downstream insulin <span class="highlight" style="background-color:">signaling</span> and leads to increased ethanol (EtOH) sensitivity and motivated foraging. Decreased insulin <span class="highlight" style="background-color:">signaling</span> in sNPFR (short Neuropeptide F Receptor) expressing odor receptor neurons (ORNs) and in NPFR (Neuropeptide F Receptor) expressing neurons, increases the sensitivity of ORNs to odors and increases the attraction of flies towards normally aversive or noxious food sources, respectively. With respect to locomotion, octopaminergic neurons signal through the IPCs to promote wake; however, this effect is independent of insulin <span class="highlight" style="background-color:">signaling</span>. Insulin <span class="highlight" style="background-color:">signaling</span> in the corpus allatum (CA), a non-neuronal endocrine gland (indicated by square), drives sexual dimorphism of locomotor patterns. Lastly, insulin <span class="highlight" style="background-color:">signaling</span> in the circadian small ventrolateral neurons (sLNVs) modifies the sensitivity of rest: Lactivity rhythms to oxidative stress.</div></p></div><div class="aux"><div class="resc">Renske Erion, et al. Front Physiol. 2013;4:353.</div><p class="links"><a class="dblinks" href="/pubmed/24348428" ref="ordinalpos=3174&amp;ncbi_uid=5610457&amp;link_uid=24348428">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847551/" ref="ordinalpos=3174&amp;ncbi_uid=5610457&amp;link_uid=PMC3847551">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3175.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1728094/figure/F1/" ref="ordinalpos=3175&amp;ncbi_uid=880298&amp;link_uid=PMC1728094" href="/pmc/articles/PMC1728094/"><img src="/pmc/articles/instance/1728094/bin/00840.f1.gif" src-large="/pmc/articles/instance/1728094/bin/00840.f1.jpg" alt="Figure 1  " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1728094/" ref="ordinalpos=3175&amp;ncbi_uid=880298&amp;link_uid=PMC1728094" image-link="/pmc/articles/PMC1728094/figure/F1/" class="imagepopup">Figure 1  .  From: Specific targeting of IL-6 signalling <span class="highlight" style="background-color:">pathway</span>: a new way to treat IBD?. </a></p><div class="supp"><p class="details"><div> The receptor for IL-6 consists of the ligand binding component (IL-6R) and the signal-transducing gp130. The soluble form of the IL-6R may form a complex with IL-6 having the ability to bind gp130 (trans <span class="highlight" style="background-color:">signaling</span>) expressed by T cells and to activate the anti-apoptotic STAT-3 dependent genes bcl-2 and bcl-xl. In inflammatory diseases, targeting of IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulates inflammation at least in part through the regulation of T cell expansion and the expression of adhesion molecules. </div></p></div><div class="aux"><div class="resc">P DESREUMAUX. Gut. 2000 October;47(4):465-466.</div><p class="links"><a class="dblinks" href="/pubmed/10986204" ref="ordinalpos=3175&amp;ncbi_uid=880298&amp;link_uid=10986204">Citation</a><a class="dblinks" href="/pmc/articles/PMC1728094/" ref="ordinalpos=3175&amp;ncbi_uid=880298&amp;link_uid=PMC1728094">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3176.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2567390/figure/fig6/" ref="ordinalpos=3176&amp;ncbi_uid=1596165&amp;link_uid=PMC2567390" href="/pmc/articles/PMC2567390/"><img src="/pmc/articles/instance/2567390/bin/GEN1802905f6.gif" src-large="/pmc/articles/instance/2567390/bin/GEN1802905f6.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  6.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2567390/" ref="ordinalpos=3176&amp;ncbi_uid=1596165&amp;link_uid=PMC2567390" image-link="/pmc/articles/PMC2567390/figure/fig6/" class="imagepopup">F igure  6.—.  From: Proteasomal Regulation of the Proliferation vs. Meiotic Entry Decision in the Caenorhabditis elegans Germ Line. </a></p><div class="supp"><p class="details"><div>Model of proteasome activity in regulating the proliferation <i>vs.</i> meiotic entry decision. In this model, proliferative cells in the mitotic zone (green) have active GLP-1/Notch <span class="highlight" style="background-color:">signaling</span>. This <span class="highlight" style="background-color:">signaling</span> inhibits the activity of GLD-1 by limiting its expression in the distal end (Hansen and Schedl 2006), which allows for specific mRNAs to be translated and produce proteins necessary for proliferation (blue). As part of one model, an unknown factor (X) may also inhibit GLD-1 activity in a manner independent of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> (see <span style="font-variant: small-caps" class="small-caps">discussion</span>). As cells move proximally, GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> is reduced due to the distance from the LAG-2 ligand expressed in the DTC (yellow), and cells enter meiotic prophase (red). The proteasome also degrades a core component(s) or positive regulator of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span>, reducing the level of <span class="highlight" style="background-color:">signaling</span> further. Reduced GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> allows GLD-1 to bind target mRNAs, preventing further production of proliferation proteins. The proteasome could interact with the GLD-1 <span class="highlight" style="background-color:">pathway</span> by degrading a negative regulator of GLD-1 activity (factor X) and/or by degrading proliferation proteins that were translated while cells were still in mitosis, but persist as cells enter meiotic prophase. In either situation, proliferation protein levels are reduced, which allows cells to enter meiotic prophase (red). Dashed lines represent reduced activity.</div></p></div><div class="aux"><div class="resc">Lindsay D. MacDonald, et al. Genetics. 2008 October;180(2):905-920.</div><p class="links"><a class="dblinks" href="/pubmed/18791239" ref="ordinalpos=3176&amp;ncbi_uid=1596165&amp;link_uid=18791239">Citation</a><a class="dblinks" href="/pmc/articles/PMC2567390/" ref="ordinalpos=3176&amp;ncbi_uid=1596165&amp;link_uid=PMC2567390">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3177.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2990476/figure/F3/" ref="ordinalpos=3177&amp;ncbi_uid=2587054&amp;link_uid=PMC2990476" href="/pmc/articles/PMC2990476/"><img src="/pmc/articles/instance/2990476/bin/nihms211358f3.gif" src-large="/pmc/articles/instance/2990476/bin/nihms211358f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2990476/" ref="ordinalpos=3177&amp;ncbi_uid=2587054&amp;link_uid=PMC2990476" image-link="/pmc/articles/PMC2990476/figure/F3/" class="imagepopup">Figure 3.  From: Genetic Associations of Brain Structural Networks in Schizophrenia: A Preliminary Study. </a></p><div class="supp"><p class="details"><div>Functional network identified by genes in the single nucleotide polymorphism (SNP) component using independent component analysis. Nodes in the red represent genes in the SNP component. The other molecules found in the network are possible connections and interactions between molecules, based onIngenuity <span class="highlight" style="background-color:">Pathway</span> Analysis knowledge base. Axonal Guidance <span class="highlight" style="background-color:">signaling</span>, G-protein-coupled receptor (GPCR) <span class="highlight" style="background-color:">signaling</span>, and PI3K/AKT <span class="highlight" style="background-color:">signaling</span> were the main <span class="highlight" style="background-color:">signaling</span> networks found. The genes with the colored circles in the network diagram are relevantly expressed in their respective pathways. The shapes of each gene symbol in the network denote the class of that gene as defined by the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis tool.</div></p></div><div class="aux"><div class="resc">Kanchana Jagannathan, et al. Biol Psychiatry. ;68(7):657-666.</div><p class="links"><a class="dblinks" href="/pubmed/20691427" ref="ordinalpos=3177&amp;ncbi_uid=2587054&amp;link_uid=20691427">Citation</a><a class="dblinks" href="/pmc/articles/PMC2990476/" ref="ordinalpos=3177&amp;ncbi_uid=2587054&amp;link_uid=PMC2990476">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3178.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3467413/figure/F5/" ref="ordinalpos=3178&amp;ncbi_uid=4310050&amp;link_uid=PMC3467413" href="/pmc/articles/PMC3467413/"><img src="/pmc/articles/instance/3467413/bin/in-12-148-g005.gif" src-large="/pmc/articles/instance/3467413/bin/in-12-148-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3467413/" ref="ordinalpos=3178&amp;ncbi_uid=4310050&amp;link_uid=PMC3467413" image-link="/pmc/articles/PMC3467413/figure/F5/" class="imagepopup">Figure 5.  From: Calcium/Calmodulin-Dependent Protein Kinase is Involved in the Release of High Mobility Group Box 1 Via the Interferon-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The release of IFN-β depends on CaMK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <i>in vivo</i>. (A and B) Serum levels of IFN-β (A) and TNF (B) in B6 mice treated with LPS (50 mg/kg) plus various doses of STO609 or vehicle (DMSO) were measured by ELISA (n=4). B6 mice were treated various doses of STO609 twice (18 and 0.5 hours) before LPS infusion. Mice were sacrificed 2 hours after LPS administration and sera were collected from each mouse for ELISA. Data are presented as mean±SEM. <sup>*</sup>p&lt;0.05 compared with LPS plus vehicle-treated mice.</div></p></div><div class="aux"><div class="resc">Lijuan Ma, et al. Immune Netw. 2012 August;12(4):148-154.</div><p class="links"><a class="dblinks" href="/pubmed/23091438" ref="ordinalpos=3178&amp;ncbi_uid=4310050&amp;link_uid=23091438">Citation</a><a class="dblinks" href="/pmc/articles/PMC3467413/" ref="ordinalpos=3178&amp;ncbi_uid=4310050&amp;link_uid=PMC3467413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3179.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2675004/figure/F3/" ref="ordinalpos=3179&amp;ncbi_uid=1780059&amp;link_uid=PMC2675004" href="/pmc/articles/PMC2675004/"><img src="/pmc/articles/instance/2675004/bin/nihms-107317-f0005.gif" src-large="/pmc/articles/instance/2675004/bin/nihms-107317-f0005.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2675004/" ref="ordinalpos=3179&amp;ncbi_uid=1780059&amp;link_uid=PMC2675004" image-link="/pmc/articles/PMC2675004/figure/F3/" class="imagepopup">Fig. 3. The <span class="highlight" style="background-color:">signaling</span> cross-talk model in physiology and in disease.  From: Crosstalk via the NF-?B <span class="highlight" style="background-color:">Signaling</span> System. </a></p><div class="supp"><p class="details"><div><b>(A)</b> A schematic representing normal healthy cells that receive both inflammatory (blue line) and developmental (green line) signals. Inflammatory stimuli, generally associated with pathogen responses, also boost developmental responses <i>via</i> <span class="highlight" style="background-color:">signaling</span> crosstalk within the NF-κB <span class="highlight" style="background-color:">signaling</span> system (black arrow). <b>(B)</b> A schematic representing a cancer cell with a cell-autonomous defect (red cross) that results in an elevated IKK activity. Such heightened IKK activity degrades theee canonical IκB isoforms, but IκBδ, whose expression is NF-κB inducible but whose degradation is IKK2-insensitive, provides a brake on chronic NF-κB activity. <b>(C)</b> Elevated IκBδ levels result in hyper-responsiveness of such cancerous cells to tonic developmental cues, that may contribute to pathological outcomes. <b>(D)</b> Elevated inflammation that is often associated with cancer may generate similar cellular hyper-responsiveness towards developmental cues <i>via</i> the <span class="highlight" style="background-color:">signaling</span> crosstalk, thus implicating a role of the developmental <span class="highlight" style="background-color:">pathway</span> in pathogenesis.</div></p></div><div class="aux"><div class="resc">Soumen Basak, et al. Cytokine Growth Factor Rev. ;19(3-4):187-197.</div><p class="links"><a class="dblinks" href="/pubmed/18515173" ref="ordinalpos=3179&amp;ncbi_uid=1780059&amp;link_uid=18515173">Citation</a><a class="dblinks" href="/pmc/articles/PMC2675004/" ref="ordinalpos=3179&amp;ncbi_uid=1780059&amp;link_uid=PMC2675004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3180.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2674646/figure/F1/" ref="ordinalpos=3180&amp;ncbi_uid=1773242&amp;link_uid=PMC2674646" href="/pmc/articles/PMC2674646/"><img src="/pmc/articles/instance/2674646/bin/nihms88867f1.gif" src-large="/pmc/articles/instance/2674646/bin/nihms88867f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2674646/" ref="ordinalpos=3180&amp;ncbi_uid=1773242&amp;link_uid=PMC2674646" image-link="/pmc/articles/PMC2674646/figure/F1/" class="imagepopup">Figure 1.  From: ADAM Proteases: Ligand Processing and Modulation of the Notch <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Different modes of ADAM modulation of the Notch <span class="highlight" style="background-color:">pathway</span>. A, Ligand binding induces Notch cleavage by an ADAM protease at the S2 site, followed by γ-secretase cleavage at the S3/S4 site and release of the NICD from the membrane in a signal-receiving cell. B, Ligand shedding by an ADAM in a signal-sending cell down-regulates Notch <span class="highlight" style="background-color:">signaling</span> in a signal-receiving cell. C, Ligand shedding by an ADAM relieves <i>cis</i>-inhibition of Notch and makes Notch available for interactions with ligands <i>in trans</i>. D, After ADAM cleavage, the transmembrane fragment of the ligand is processed by γ-secretase, and the intracellular domain released from the membrane may act as a <span class="highlight" style="background-color:">signaling</span> molecule.</div></p></div><div class="aux"><div class="resc">Anna Zolkiewska. Cell Mol Life Sci. ;65(13):2056-2068.</div><p class="links"><a class="dblinks" href="/pubmed/18344021" ref="ordinalpos=3180&amp;ncbi_uid=1773242&amp;link_uid=18344021">Citation</a><a class="dblinks" href="/pmc/articles/PMC2674646/" ref="ordinalpos=3180&amp;ncbi_uid=1773242&amp;link_uid=PMC2674646">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3181.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3975884/figure/F1/" ref="ordinalpos=3181&amp;ncbi_uid=5989178&amp;link_uid=PMC3975884" href="/pmc/articles/PMC3975884/"><img src="/pmc/articles/instance/3975884/bin/1746-1596-9-48-1.gif" src-large="/pmc/articles/instance/3975884/bin/1746-1596-9-48-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3975884/" ref="ordinalpos=3181&amp;ncbi_uid=5989178&amp;link_uid=PMC3975884" image-link="/pmc/articles/PMC3975884/figure/F1/" class="imagepopup">Figure 1.  From: Tuberous-sclerosis complex-related cell <span class="highlight" style="background-color:">signaling</span> in the pathogenesis of lung cancer. </a></p><div class="supp"><p class="details"><div><b>EGFR associated <span class="highlight" style="background-color:">signaling</span> pathways in lung cancer.</b> Phosphorylation, i.e. activation of the Epidermal Growth Factor Receptor (EGFR) results in various downstream <span class="highlight" style="background-color:">signaling</span> pathways including the JAK-STAT-<span class="highlight" style="background-color:">signaling</span>, KRAS, MAPK and the phosphatidylinositol 3-kinase (PI3K) related <span class="highlight" style="background-color:">pathway</span>. Extrinsic binding of the Endothelian Growth Factor (EGF) to the corresponding Endothelian Growth Factor Receptor (EGFR) leads to phosphorylation of EGFR and subsequent activation of phosphatidylinositol 3-kinase (PI3K). PI3K <span class="highlight" style="background-color:">signaling</span> results in an activation of Akt and TSC1/TSC2, diminishing its inhibitory effect on Rheb (Ras homolog enriched in brain), a major activator of mTOR <span class="highlight" style="background-color:">signaling</span>. Disruption of the TSC-complex leads to activation (disinhibition) of mTOR and may play a putative pathogenic role in lung cancer pathogenesis. <i>Arrows represent activation; bars represent inhibitory effects.</i></div></p></div><div class="aux"><div class="resc">Angela Fuchs, et al. Diagn Pathol. 2014;9:48-48.</div><p class="links"><a class="dblinks" href="/pubmed/24593867" ref="ordinalpos=3181&amp;ncbi_uid=5989178&amp;link_uid=24593867">Citation</a><a class="dblinks" href="/pmc/articles/PMC3975884/" ref="ordinalpos=3181&amp;ncbi_uid=5989178&amp;link_uid=PMC3975884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3182.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3334869/figure/F7/" ref="ordinalpos=3182&amp;ncbi_uid=4815505&amp;link_uid=PMC3334869" href="/pmc/articles/PMC3334869/"><img src="/pmc/articles/instance/3334869/bin/nihms364799f7.gif" src-large="/pmc/articles/instance/3334869/bin/nihms364799f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3334869/" ref="ordinalpos=3182&amp;ncbi_uid=4815505&amp;link_uid=PMC3334869" image-link="/pmc/articles/PMC3334869/figure/F7/" class="imagepopup">Figure 7. Model for induction of C. elegans zip-2/irg-1 surveillance <span class="highlight" style="background-color:">pathway</span>.  From: C. elegans detects pathogen-induced translational inhibition to activate immune <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) Endocytosis of <i>P. aeruginosa</i> toxins causes translational inhibition, which activates <i>zip-2</i> immune response <span class="highlight" style="background-color:">pathway</span>. (B) <i>zip-2/irg-1</i> <span class="highlight" style="background-color:">pathway</span> is activated by disruption of several core host processes. See Discussion for details.</div></p></div><div class="aux"><div class="resc">Tiffany L. Dunbar, et al. Cell Host Microbe. ;11(4):375-386.</div><p class="links"><a class="dblinks" href="/pubmed/22520465" ref="ordinalpos=3182&amp;ncbi_uid=4815505&amp;link_uid=22520465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3334869/" ref="ordinalpos=3182&amp;ncbi_uid=4815505&amp;link_uid=PMC3334869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3183.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3575446/figure/F1/" ref="ordinalpos=3183&amp;ncbi_uid=4639749&amp;link_uid=PMC3575446" href="/pmc/articles/PMC3575446/"><img src="/pmc/articles/instance/3575446/bin/cc-12-202-g1.gif" src-large="/pmc/articles/instance/3575446/bin/cc-12-202-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3575446/" ref="ordinalpos=3183&amp;ncbi_uid=4639749&amp;link_uid=PMC3575446" image-link="/pmc/articles/PMC3575446/figure/F1/" class="imagepopup">Figure. From: Linking JNK-STAT3-Akt <span class="highlight" style="background-color:">signaling</span> axis to EZH2 phosphorylation. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> JNK-STAT3-Akt <span class="highlight" style="background-color:">signaling</span> axis (red arrows) in carcinogenesis.</div></p></div><div class="aux"><div class="resc">Yon Rojanasakul. Cell Cycle. 2013 January 15;12(2):202-202.</div><p class="links"><a class="dblinks" href="/pubmed/23287465" ref="ordinalpos=3183&amp;ncbi_uid=4639749&amp;link_uid=23287465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3575446/" ref="ordinalpos=3183&amp;ncbi_uid=4639749&amp;link_uid=PMC3575446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3184.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4227623/figure/f8-ijn-9-5131/" ref="ordinalpos=3184&amp;ncbi_uid=6831275&amp;link_uid=PMC4227623" href="/pmc/articles/PMC4227623/"><img src="/pmc/articles/instance/4227623/bin/ijn-9-5131Fig8.gif" src-large="/pmc/articles/instance/4227623/bin/ijn-9-5131Fig8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4227623/" ref="ordinalpos=3184&amp;ncbi_uid=6831275&amp;link_uid=PMC4227623" image-link="/pmc/articles/PMC4227623/figure/f8-ijn-9-5131/" class="imagepopup">Figure 8.  From: Silica nanoparticles induce autophagy and endothelial dysfunction via the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of Nano-SiO<sub>2</sub> on the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.<br /><b>Notes:</b> (<b>A</b>) The expression of p-PI3K, PI3K, p-Akt, Akt, p-mTOR, and mTOR were measured by Western blot assay. (<b>B</b>) Relative densitometric analysis of the protein bands was performed and presented. Nano-SiO<sub>2</sub> inhibited the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ANOVA followed by the LSD method are used for the data analysis. Data are expressed as the mean ± standard deviation from three independent experiments (*<i>P</i>&lt;0.05).<br /><b>Abbreviations:</b> PI3K, phosphoinositide 3-kinase; Akt, protein kinase B; mTOR, mammalian target of rapamycin; GAPDH, glyceraldehyde 3-phosphate dehydrogenase; Nano-SiO<sub>2</sub>, silica nanoparticles; ANOVA, one-way analysis of variance; LSD, least significant difference.</div></p></div><div class="aux"><div class="resc">Junchao Duan, et al. Int J Nanomedicine. 2014;9:5131-5141.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3184&amp;ncbi_uid=6831275&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4227623/" ref="ordinalpos=3184&amp;ncbi_uid=6831275&amp;link_uid=PMC4227623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3185.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3712008/figure/F3/" ref="ordinalpos=3185&amp;ncbi_uid=5100883&amp;link_uid=PMC3712008" href="/pmc/articles/PMC3712008/"><img src="/pmc/articles/instance/3712008/bin/1475-2867-13-66-3.gif" src-large="/pmc/articles/instance/3712008/bin/1475-2867-13-66-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3712008/" ref="ordinalpos=3185&amp;ncbi_uid=5100883&amp;link_uid=PMC3712008" image-link="/pmc/articles/PMC3712008/figure/F3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> switch in breast cancer. </a></p><div class="supp"><p class="details"><div><b>Hypothetical representation of opposite downstream effects of two <span class="highlight" style="background-color:">signaling</span> pathways on the G1 phase of the cell cycle in a breast tumor</b>-<b>initiating cell.</b> During the early G1 subphase the cell integrates signals that may lead it to several fates (differentiation, apoptosis, senescence, quiescence, pause for DNA repair, progression to S phase) [8]. Inhibition of the JUN/MAPK <span class="highlight" style="background-color:">pathway</span> shortens the early G1 subphase rendering the cell independent of differentiation and apoptotic factors. In the same time, activation of the PI3K/AKT <span class="highlight" style="background-color:">pathway</span> induces the cell to progress to late G1 subphase beyond the restriction (R) point; the cell may then undergo symmetric (proliferation) or asymmetric (self-renewal) division. In green: proteins with repressive effect, in red: proteins with activating effect.</div></p></div><div class="aux"><div class="resc">Arnaud Guille, et al. Cancer Cell Int. 2013;13:66-66.</div><p class="links"><a class="dblinks" href="/pubmed/23805779" ref="ordinalpos=3185&amp;ncbi_uid=5100883&amp;link_uid=23805779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3712008/" ref="ordinalpos=3185&amp;ncbi_uid=5100883&amp;link_uid=PMC3712008">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3186.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3814130/figure/F2/" ref="ordinalpos=3186&amp;ncbi_uid=5447468&amp;link_uid=PMC3814130" href="/pmc/articles/PMC3814130/"><img src="/pmc/articles/instance/3814130/bin/nihms93534f2.gif" src-large="/pmc/articles/instance/3814130/bin/nihms93534f2.jpg" alt="Fig.(2)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3814130/" ref="ordinalpos=3186&amp;ncbi_uid=5447468&amp;link_uid=PMC3814130" image-link="/pmc/articles/PMC3814130/figure/F2/" class="imagepopup">Fig.(2). Mechanism of activation and effects of activated phospho-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by B-RafV600E oncogene.  From: Role of B-RafV600E in differentiated thyroid cancer and preclinical validation of compounds against B-RafV600E. </a></p><div class="supp"><p class="details"><div>When the B-Raf<sup>V600E</sup> oncogene is generated through a point mutation, activation of the MAP kinase <span class="highlight" style="background-color:">pathway</span> becomes constitutive, inducing cells to proliferate indefinitely and initiate tumor formation. B-Raf<sup>V600E</sup> shows a dramatically increased basal kinase activity (increased phosphorylation, PO<sub>4</sub><sup>−</sup>), ~480-folds higher than wild-type B-Raf activity. Response to B-Raf<sup>V600E</sup> is independent of extra-cellular mitogens and Ras <span class="highlight" style="background-color:">signaling</span>, creating a deregulation of transcription mechanisms. The B-Raf<sup>V600E</sup>-activated phospho-ERK1/2 <span class="highlight" style="background-color:">pathway</span> transforms human normal cells both <i>in vitro</i> and <i>in vivo</i>. In addition, B-Raf<sup><i>V600E</i></sup> induces senescence in normal human foreskin fibroblasts by stimulation of the cell cycle inhibitor p16<sup>INK4a</sup>.</div></p></div><div class="aux"><div class="resc">Carmelo Nucera, et al. Biochim Biophys Acta. 2009 April;1795(2):10.1016/j.bbcan.2009.01.003.</div><p class="links"><a class="dblinks" href="/pubmed/19356676" ref="ordinalpos=3186&amp;ncbi_uid=5447468&amp;link_uid=19356676">Citation</a><a class="dblinks" href="/pmc/articles/PMC3814130/" ref="ordinalpos=3186&amp;ncbi_uid=5447468&amp;link_uid=PMC3814130">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3187.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2720726/figure/F1/" ref="ordinalpos=3187&amp;ncbi_uid=2041916&amp;link_uid=PMC2720726" href="/pmc/articles/PMC2720726/"><img src="/pmc/articles/instance/2720726/bin/btn400f1.gif" src-large="/pmc/articles/instance/2720726/bin/btn400f1.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2720726/" ref="ordinalpos=3187&amp;ncbi_uid=2041916&amp;link_uid=PMC2720726" image-link="/pmc/articles/PMC2720726/figure/F1/" class="imagepopup">Fig. 1. From: Model-based analysis of interferon-? induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Diagram of the IFN-β activated <span class="highlight" style="background-color:">pathway</span>. Postulated control mechanisms are shown using thick lines.</div></p></div><div class="aux"><div class="resc">Jaroslaw Smieja, et al. Bioinformatics. 2008 October 15;24(20):2363-2369.</div><p class="links"><a class="dblinks" href="/pubmed/18713791" ref="ordinalpos=3187&amp;ncbi_uid=2041916&amp;link_uid=18713791">Citation</a><a class="dblinks" href="/pmc/articles/PMC2720726/" ref="ordinalpos=3187&amp;ncbi_uid=2041916&amp;link_uid=PMC2720726">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3188.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3280589/figure/F2/" ref="ordinalpos=3188&amp;ncbi_uid=3803922&amp;link_uid=PMC3280589" href="/pmc/articles/PMC3280589/"><img src="/pmc/articles/instance/3280589/bin/nihms205955f2.gif" src-large="/pmc/articles/instance/3280589/bin/nihms205955f2.jpg" alt="FIG. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3280589/" ref="ordinalpos=3188&amp;ncbi_uid=3803922&amp;link_uid=PMC3280589" image-link="/pmc/articles/PMC3280589/figure/F2/" class="imagepopup">FIG. 2.  From: Minireview: Rapid Glucocorticoid <span class="highlight" style="background-color:">Signaling</span> via Membrane-Associated Receptors. </a></p><div class="supp"><p class="details"><div>Model of rapid glucocorticoid <span class="highlight" style="background-color:">signaling</span> in PVN parvocellular neuroendocrine cells. A, Glucocorticoids (CORT) bind to a membrane-associated receptor (mbGR) and activate a G protein-dependent intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that leads to endocannabinoid (CB) synthesis and retrograde release. Endocannabinoids bind to presynaptic CB1 cannabinoid receptors and suppress glutamate release from glutamate synapses onto the PVN neuron. B, Proposed membrane glucocorticoid receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In this model, the membrane glucocorticoid receptor (mbGR) is a stimulatory G protein-coupled receptor (Gs) that, upon binding corticosterone (CORT), triggers the activation of adenylyl cyclase and the production of cAMP, resulting in increased PKA activity, which leads downstream to endocannabinoid synthesis from membrane lipid precursors followed by release.</div></p></div><div class="aux"><div class="resc">Jeffrey G. Tasker, et al. Endocrinology. ;147(12):5549-5556.</div><p class="links"><a class="dblinks" href="/pubmed/16946006" ref="ordinalpos=3188&amp;ncbi_uid=3803922&amp;link_uid=16946006">Citation</a><a class="dblinks" href="/pmc/articles/PMC3280589/" ref="ordinalpos=3188&amp;ncbi_uid=3803922&amp;link_uid=PMC3280589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3189.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4029773/figure/F2/" ref="ordinalpos=3189&amp;ncbi_uid=6153421&amp;link_uid=PMC4029773" href="/pmc/articles/PMC4029773/"><img src="/pmc/articles/instance/4029773/bin/nihms577662f2.gif" src-large="/pmc/articles/instance/4029773/bin/nihms577662f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4029773/" ref="ordinalpos=3189&amp;ncbi_uid=6153421&amp;link_uid=PMC4029773" image-link="/pmc/articles/PMC4029773/figure/F2/" class="imagepopup">Figure 2. JAK-STAT <span class="highlight" style="background-color:">signaling</span> in white and brown adipose tissue.  From: The role of JAK-STAT <span class="highlight" style="background-color:">signaling</span> in adipose tissue function. </a></p><div class="supp"><p class="details"><div>Different transcriptional programs control the adipogenesis of white and brown adipocytes, and different JAK and STAT members have been implicated in these programs. Cytokine and hormone activators of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> are produced and utilized by various cell types present within adipose tissue to modulate the physiological function of WAT and BAT.</div></p></div><div class="aux"><div class="resc">Allison J. Richard, et al. Biochim Biophys Acta. ;1842(3):431-439.</div><p class="links"><a class="dblinks" href="/pubmed/23735217" ref="ordinalpos=3189&amp;ncbi_uid=6153421&amp;link_uid=23735217">Citation</a><a class="dblinks" href="/pmc/articles/PMC4029773/" ref="ordinalpos=3189&amp;ncbi_uid=6153421&amp;link_uid=PMC4029773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3190.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634503/figure/f1-ijms-14-05312/" ref="ordinalpos=3190&amp;ncbi_uid=4863276&amp;link_uid=PMC3634503" href="/pmc/articles/PMC3634503/"><img src="/pmc/articles/instance/3634503/bin/ijms-14-05312f1.gif" src-large="/pmc/articles/instance/3634503/bin/ijms-14-05312f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634503/" ref="ordinalpos=3190&amp;ncbi_uid=4863276&amp;link_uid=PMC3634503" image-link="/pmc/articles/PMC3634503/figure/f1-ijms-14-05312/" class="imagepopup">Figure 1.  From: Cold <span class="highlight" style="background-color:">Signaling</span> and Cold Response in Plants. </a></p><div class="supp"><p class="details"><div>The cold <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which involves ICE1 and CBF3/DREB1A. ICE1 is a MYC-type transcription factor and binds to <i>cis-</i>elements in the promoter of <i>CBF3/DREB1A</i> to induce its expression. CBF3/DREB1A is an AP2-type transcription factor to regulate expression of <i>COR</i> (cold-regulated genes) and cold tolerance. Ubiquitylation of ICE1 is mediated by HOS1, an ubiquitin E3 ligase for proteasome-dependent degradation. SIZ1, a SUMO E3 ligase, mediates sumoylation (SUMO conjugation) of ICE1, probably leading to blocking of ubiquitylation and stabilization of ICE1.</div></p></div><div class="aux"><div class="resc">Kenji Miura, et al. Int J Mol Sci. 2013 March;14(3):5312-5337.</div><p class="links"><a class="dblinks" href="/pubmed/23466881" ref="ordinalpos=3190&amp;ncbi_uid=4863276&amp;link_uid=23466881">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634503/" ref="ordinalpos=3190&amp;ncbi_uid=4863276&amp;link_uid=PMC3634503">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3191.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1847715/figure/F8/" ref="ordinalpos=3191&amp;ncbi_uid=899377&amp;link_uid=PMC1847715" href="/pmc/articles/PMC1847715/"><img src="/pmc/articles/instance/1847715/bin/984fig8.gif" src-large="/pmc/articles/instance/1847715/bin/984fig8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1847715/" ref="ordinalpos=3191&amp;ncbi_uid=899377&amp;link_uid=PMC1847715" image-link="/pmc/articles/PMC1847715/figure/F8/" class="imagepopup">Figure 8. From: CARMA3 deficiency abrogates G protein-coupled receptor-induced NF-?B activation. </a></p><div class="supp"><p class="details"><div>The working model for CARMA3-mediated NF-κB activation in GPCR signal transduction <span class="highlight" style="background-color:">pathway</span>. GPCR-induced NF-κB activation involves a <span class="highlight" style="background-color:">signaling</span> cascade of PKC–CARMA3–BCL10–MALT1–TRAF6 leading to polyubiquitination of the IKK complex, whereas a CARMA3-independent and PKC-dependent signal induces IKKβ phosphorylation by an unknown kinase in the GPCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of CARMA3, GPCR-induced polyubiquitination of the IKK complex is defective, thereby resulting in a defective IKK complex.</div></p></div><div class="aux"><div class="resc">Brian C. Grabiner, et al. Genes Dev. 2007 April 15;21(8):984-996.</div><p class="links"><a class="dblinks" href="/pubmed/17438001" ref="ordinalpos=3191&amp;ncbi_uid=899377&amp;link_uid=17438001">Citation</a><a class="dblinks" href="/pmc/articles/PMC1847715/" ref="ordinalpos=3191&amp;ncbi_uid=899377&amp;link_uid=PMC1847715">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3192.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4108439/figure/pone-0102948-g008/" ref="ordinalpos=3192&amp;ncbi_uid=6414949&amp;link_uid=PMC4108439" href="/pmc/articles/PMC4108439/"><img src="/pmc/articles/instance/4108439/bin/pone.0102948.g008.gif" src-large="/pmc/articles/instance/4108439/bin/pone.0102948.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4108439/" ref="ordinalpos=3192&amp;ncbi_uid=6414949&amp;link_uid=PMC4108439" image-link="/pmc/articles/PMC4108439/figure/pone-0102948-g008/" class="imagepopup">Figure 8. Schematic depicting of EP3 induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> contribution to apoptosis during cytokine withdrawal..  From: PGE2 Promotes Apoptosis Induced by Cytokine Deprivation through EP3 Receptor and Induces Bim in Mouse Mast Cells. </a></p><div class="supp"><p class="details"><div>Withdrawal of cytokines results in activation of intrinsic apoptotic <span class="highlight" style="background-color:">pathway</span> due to imbalance between Bcl-2 anti-apoptotic protein family members and BH3 only pro-apoptotic proteins. Intrinsic apoptotic <span class="highlight" style="background-color:">pathway</span> in mast cells is upregulated by PGE<sub>2</sub> via EP3 receptor and Ca<sup>2+</sup>-CaMII-ERK1/2 <span class="highlight" style="background-color:">pathway</span>. While Erk 1/2 mediated phosphorylation of Bim<sub>EL</sub> and Bim <sub>L</sub> immediately after PGE<sub>2</sub> stimulation and this leads to ubiquination and degradation of these isoforms, Erk also induces expression of c-Jun, a potent upregulator of Bim expression. Final outcome of Erk regulation of Bim is a balance between Bim degradation after its phosphorylation and increase in its expression due to c-Jun. Data presented here support view that increase expression of Bim and proapoptotic action of EP3-Ca<sup>2+</sup>-CamKII –Erk1/2 <span class="highlight" style="background-color:">pathway</span> is prevalent in mast cells in presence of PGE<sub>2</sub> and during cytokine withdrawal.</div></p></div><div class="aux"><div class="resc">Martina Kovarova, et al. PLoS One. 2014;9(7):e102948.</div><p class="links"><a class="dblinks" href="/pubmed/25054560" ref="ordinalpos=3192&amp;ncbi_uid=6414949&amp;link_uid=25054560">Citation</a><a class="dblinks" href="/pmc/articles/PMC4108439/" ref="ordinalpos=3192&amp;ncbi_uid=6414949&amp;link_uid=PMC4108439">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3193.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4354340/figure/fig8/" ref="ordinalpos=3193&amp;ncbi_uid=7293967&amp;link_uid=PMC4354340" href="/pmc/articles/PMC4354340/"><img src="/pmc/articles/instance/4354340/bin/mtna20155f8.gif" src-large="/pmc/articles/instance/4354340/bin/mtna20155f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4354340/" ref="ordinalpos=3193&amp;ncbi_uid=7293967&amp;link_uid=PMC4354340" image-link="/pmc/articles/PMC4354340/figure/fig8/" class="imagepopup">Figure 8.  From: Concurrent Targeting of KRAS and AKT by MiR-4689 Is a Novel Treatment Against Mutant KRAS Colorectal Cancer. </a></p><div class="supp"><p class="details"><div><b>miR-4689 regulates EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> A simplified scheme illustrating that exogenous miR-4689 can inhibit EGFR <span class="highlight" style="background-color:">signaling</span> through direct inhibition of both KRAS and AKT1. Endogenous miR-4689 expression is normally suppressed in mutated <i>KRAS</i> CRC.</div></p></div><div class="aux"><div class="resc">Masayuki Hiraki, et al. Mol Ther Nucleic Acids. 2015 March;4(3):e231.</div><p class="links"><a class="dblinks" href="/pubmed/25756961" ref="ordinalpos=3193&amp;ncbi_uid=7293967&amp;link_uid=25756961">Citation</a><a class="dblinks" href="/pmc/articles/PMC4354340/" ref="ordinalpos=3193&amp;ncbi_uid=7293967&amp;link_uid=PMC4354340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3194.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3158866/figure/F1/" ref="ordinalpos=3194&amp;ncbi_uid=3215587&amp;link_uid=PMC3158866" href="/pmc/articles/PMC3158866/"><img src="/pmc/articles/instance/3158866/bin/WJBC-2-167-g001.gif" src-large="/pmc/articles/instance/3158866/bin/WJBC-2-167-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3158866/" ref="ordinalpos=3194&amp;ncbi_uid=3215587&amp;link_uid=PMC3158866" image-link="/pmc/articles/PMC3158866/figure/F1/" class="imagepopup">Figure 1.  From: Toll-like receptors are potential therapeutic targets in rheumatoid arthritis. </a></p><div class="supp"><p class="details"><div>The Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The ligands of Toll-like receptors, their <span class="highlight" style="background-color:">signaling</span> adaptors and downstream mediators are depicted. LPS: Lipopolysaccharide; HSP: Heat shock protein; TLR: Toll-like receptor; IRAK: Interleukin-1 receptor-associated kinase.</div></p></div><div class="aux"><div class="resc">Siamak Sandoghchian Shotorbani, et al. World J Biol Chem. 2011 July 26;2(7):167-172.</div><p class="links"><a class="dblinks" href="/pubmed/21912729" ref="ordinalpos=3194&amp;ncbi_uid=3215587&amp;link_uid=21912729">Citation</a><a class="dblinks" href="/pmc/articles/PMC3158866/" ref="ordinalpos=3194&amp;ncbi_uid=3215587&amp;link_uid=PMC3158866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3195.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3240668/figure/F6/" ref="ordinalpos=3195&amp;ncbi_uid=3715537&amp;link_uid=PMC3240668" href="/pmc/articles/PMC3240668/"><img src="/pmc/articles/instance/3240668/bin/1471-2105-12-80-6.gif" src-large="/pmc/articles/instance/3240668/bin/1471-2105-12-80-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3240668/" ref="ordinalpos=3195&amp;ncbi_uid=3715537&amp;link_uid=PMC3240668" image-link="/pmc/articles/PMC3240668/figure/F6/" class="imagepopup">Figure 6.  From: Network based transcription factor analysis of regenerating axolotl limbs. </a></p><div class="supp"><p class="details"><div><b>Fibronectin - c-Myc network</b>. This network shows that multiple <span class="highlight" style="background-color:">signaling</span> pathways from FN can lead to the activation of c-Myc. The <span class="highlight" style="background-color:">pathway</span> highlighted in pink includes proteins involved in canonical Wnt <span class="highlight" style="background-color:">signaling</span> (GSK3beta, beta-catenin, and Tcf(Lef)).</div></p></div><div class="aux"><div class="resc">Deepali Jhamb, et al. BMC Bioinformatics. 2011;12:80-80.</div><p class="links"><a class="dblinks" href="/pubmed/21418574" ref="ordinalpos=3195&amp;ncbi_uid=3715537&amp;link_uid=21418574">Citation</a><a class="dblinks" href="/pmc/articles/PMC3240668/" ref="ordinalpos=3195&amp;ncbi_uid=3715537&amp;link_uid=PMC3240668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3196.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4143617/figure/F1/" ref="ordinalpos=3196&amp;ncbi_uid=6596697&amp;link_uid=PMC4143617" href="/pmc/articles/PMC4143617/"><img src="/pmc/articles/instance/4143617/bin/fpls-05-00428-g001.gif" src-large="/pmc/articles/instance/4143617/bin/fpls-05-00428-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4143617/" ref="ordinalpos=3196&amp;ncbi_uid=6596697&amp;link_uid=PMC4143617" image-link="/pmc/articles/PMC4143617/figure/F1/" class="imagepopup">FIGURE 1.  From: Emerging roles of small GTPases in secondary cell wall development. </a></p><div class="supp"><p class="details"><div><b>A multi-Rho of plants (ROP) <span class="highlight" style="background-color:">signaling</span> model for secondary cell wall development.</b> ROPGEF4 and ROPGAP3 establish the ROP11/MIDD1/kinesin-13A <span class="highlight" style="background-color:">pathway</span> to generate a cortical microtubule-absent area by promoting microtubule depolymerization, which results in pits of secondary cell walls in metaxylem vessels. Cortical microtubules restrict <span class="highlight" style="background-color:">signaling</span> from the ROP11/MIDD1/kinesin-13A <span class="highlight" style="background-color:">pathway</span> and promote local secondary cell wall deposition via an unknown ROP <span class="highlight" style="background-color:">pathway</span> and/or an ROP-independent <span class="highlight" style="background-color:">pathway</span> that controls targeting of exocytotic vesicles through EXO70A1. An adaptor protein such as RIP1/ICR1 might mediate interactions between the putative ROP, exocysts, and cortical microtubules. Vesicles from the <i>trans</i>-Golgi network are directly targeted to cortical microtubules, and subsequently supply cellulose synthase complexes to the cellular sites where microtubules are present.</div></p></div><div class="aux"><div class="resc">Yoshihisa Oda, et al. Front Plant Sci. 2014;5:428.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3196&amp;ncbi_uid=6596697&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4143617/" ref="ordinalpos=3196&amp;ncbi_uid=6596697&amp;link_uid=PMC4143617">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3197.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4086483/figure/F1/" ref="ordinalpos=3197&amp;ncbi_uid=6351964&amp;link_uid=PMC4086483" href="/pmc/articles/PMC4086483/"><img src="/pmc/articles/instance/4086483/bin/nihms-604351-f0001.gif" src-large="/pmc/articles/instance/4086483/bin/nihms-604351-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4086483/" ref="ordinalpos=3197&amp;ncbi_uid=6351964&amp;link_uid=PMC4086483" image-link="/pmc/articles/PMC4086483/figure/F1/" class="imagepopup">Fig. 1.  From: Adipose tissue macrophages in the Development of Obesity-induced Inflammation, Insulin Resistance and Type 2 Diabetes. </a></p><div class="supp"><p class="details"><div>NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Jongsoon Lee. Arch Pharm Res. ;36(2):208-222.</div><p class="links"><a class="dblinks" href="/pubmed/23397293" ref="ordinalpos=3197&amp;ncbi_uid=6351964&amp;link_uid=23397293">Citation</a><a class="dblinks" href="/pmc/articles/PMC4086483/" ref="ordinalpos=3197&amp;ncbi_uid=6351964&amp;link_uid=PMC4086483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3198.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3562679/figure/fig1/" ref="ordinalpos=3198&amp;ncbi_uid=4614609&amp;link_uid=PMC3562679" href="/pmc/articles/PMC3562679/"><img src="/pmc/articles/instance/3562679/bin/IJN2013-954956.001.gif" src-large="/pmc/articles/instance/3562679/bin/IJN2013-954956.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3562679/" ref="ordinalpos=3198&amp;ncbi_uid=4614609&amp;link_uid=PMC3562679" image-link="/pmc/articles/PMC3562679/figure/fig1/" class="imagepopup">Figure 1.  From: Endotoxin Binding by Sevelamer: Potential Impact on Nutritional Status. </a></p><div class="supp"><p class="details"><div>Lipopolysaccharide (endotoxin) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Used with permission from [22].)</div></p></div><div class="aux"><div class="resc">Natsuki Kubotera, et al. Int J Nephrol. 2013;2013:954956.</div><p class="links"><a class="dblinks" href="/pubmed/23401772" ref="ordinalpos=3198&amp;ncbi_uid=4614609&amp;link_uid=23401772">Citation</a><a class="dblinks" href="/pmc/articles/PMC3562679/" ref="ordinalpos=3198&amp;ncbi_uid=4614609&amp;link_uid=PMC3562679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3199.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3337681/figure/fig7/" ref="ordinalpos=3199&amp;ncbi_uid=3996356&amp;link_uid=PMC3337681" href="/pmc/articles/PMC3337681/"><img src="/pmc/articles/instance/3337681/bin/MI2012-781375.007.gif" src-large="/pmc/articles/instance/3337681/bin/MI2012-781375.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3337681/" ref="ordinalpos=3199&amp;ncbi_uid=3996356&amp;link_uid=PMC3337681" image-link="/pmc/articles/PMC3337681/figure/fig7/" class="imagepopup">Figure 7.  From: Syk/Src <span class="highlight" style="background-color:">Pathway</span>-Targeted Inhibition of Skin Inflammatory Responses by Carnosic Acid. </a></p><div class="supp"><p class="details"><div>Putative inhibitory <span class="highlight" style="background-color:">pathway</span> of inflammatory <span class="highlight" style="background-color:">signaling</span> events by CA.</div></p></div><div class="aux"><div class="resc">Jueun Oh, et al. Mediators Inflamm. 2012;2012:781375.</div><p class="links"><a class="dblinks" href="/pubmed/22577255" ref="ordinalpos=3199&amp;ncbi_uid=3996356&amp;link_uid=22577255">Citation</a><a class="dblinks" href="/pmc/articles/PMC3337681/" ref="ordinalpos=3199&amp;ncbi_uid=3996356&amp;link_uid=PMC3337681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3200.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2994381/figure/F5/" ref="ordinalpos=3200&amp;ncbi_uid=2615766&amp;link_uid=PMC2994381" href="/pmc/articles/PMC2994381/"><img src="/pmc/articles/instance/2994381/bin/1423-0127-17-S1-S28-5.gif" src-large="/pmc/articles/instance/2994381/bin/1423-0127-17-S1-S28-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2994381/" ref="ordinalpos=3200&amp;ncbi_uid=2615766&amp;link_uid=PMC2994381" image-link="/pmc/articles/PMC2994381/figure/F5/" class="imagepopup">Figure 5.  From: Stress-responsive gene TauT and acute kidney injury. </a></p><div class="supp"><p class="details"><div>Stress-induced regulation of <i>TauT</i> by p53 and c-Jun via JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Xiaobin Han, et al. J Biomed Sci. 2010;17(Suppl 1):S28-S28.</div><p class="links"><a class="dblinks" href="/pubmed/20804603" ref="ordinalpos=3200&amp;ncbi_uid=2615766&amp;link_uid=20804603">Citation</a><a class="dblinks" href="/pmc/articles/PMC2994381/" ref="ordinalpos=3200&amp;ncbi_uid=2615766&amp;link_uid=PMC2994381">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="31" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="32" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="33" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:41:06-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A7BD7543116010000000000B2A8FF;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A7BD7543116010000000000B2A8FF" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>