
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="39" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB6B4354311F510000000000DCC0E4" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="38" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="39" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="40" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="39" /></div>
            <div><h2 class="result_count">Results: 3801 to 3900 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3801.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119700/figure/pone-0021370-g004/" ref="ordinalpos=3801&amp;ncbi_uid=3041654&amp;link_uid=PMC3119700" href="/pmc/articles/PMC3119700/"><img src="/pmc/articles/instance/3119700/bin/pone.0021370.g004.gif" src-large="/pmc/articles/instance/3119700/bin/pone.0021370.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119700/" ref="ordinalpos=3801&amp;ncbi_uid=3041654&amp;link_uid=PMC3119700" image-link="/pmc/articles/PMC3119700/figure/pone-0021370-g004/" class="imagepopup">Figure 4. Cambogin triggers cell death via preferentially PDGFRs <span class="highlight" style="background-color:">pathway</span>..  From: Cambogin Is Preferentially Cytotoxic to Cells Expressing PDGFR. </a></p><div class="supp"><p class="details"><div>A. Cambogin down regulated PDGFR <span class="highlight" style="background-color:">signaling</span> in Daoy cells using Western blotting. B and C. Cambogin induced significant cell death in MEF cell lines with PDGFR <span class="highlight" style="background-color:">signaling</span> compared with PDGFR <span class="highlight" style="background-color:">signaling</span> null cells. The cell growth was measured by MTT and photograph was taken under the microscope. D. Cambogin down regulates PDGFR <span class="highlight" style="background-color:">signaling</span> (PDGFRα, PDGFRβ, erk, p-erk, and PARP) in genetically defined MEF cell lines using Western blotting (β-actin was used as internal loading control). Four cell lines (PDGFRα <sup>+/+</sup>; PDGFRβ <sup>+/+</sup>, PDGFRα <sup>+/+</sup> β<sup>+/+</sup> and PDGFR α<sup>−/−</sup>β<sup>−/−</sup> cells) were used as indicated.</div></p></div><div class="aux"><div class="resc">Ze Tian, et al. PLoS One. 2011;6(6):e21370.</div><p class="links"><a class="dblinks" href="/pubmed/21712951" ref="ordinalpos=3801&amp;ncbi_uid=3041654&amp;link_uid=21712951">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119700/" ref="ordinalpos=3801&amp;ncbi_uid=3041654&amp;link_uid=PMC3119700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3802.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2834616/figure/F4/" ref="ordinalpos=3802&amp;ncbi_uid=2204805&amp;link_uid=PMC2834616" href="/pmc/articles/PMC2834616/"><img src="/pmc/articles/instance/2834616/bin/1752-0509-4-10-4.gif" src-large="/pmc/articles/instance/2834616/bin/1752-0509-4-10-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2834616/" ref="ordinalpos=3802&amp;ncbi_uid=2204805&amp;link_uid=PMC2834616" image-link="/pmc/articles/PMC2834616/figure/F4/" class="imagepopup">Figure 4.  From: A Novel microRNA and transcription factor mediated regulatory network in schizophrenia. </a></p><div class="supp"><p class="details"><div><b>Model of EGR3 and miRNAs involving in <span class="highlight" style="background-color:">signaling</span> pathways and regulatory networks in schizophrenia and nervous system</b>. Binding of neurotrophins (BDNF or NGF) to Trk receptors activates multiple <span class="highlight" style="background-color:">signaling</span> pathways, including Ras/MAPK cascade, PKC, PI3K/AKT, and IP3/calcium <span class="highlight" style="background-color:">signaling</span> [72,73]. Stimulation of NRG1 activates MAPK and PI3K/AKT pathways [74]. These signal transductions trigger activation of several TFs such as CREB, CBP, EGR1 and EGR3, which subsequently activate their target genes (e.g., <i>EGR3</i>, <i>BCL2 </i>and <i>NPY</i>). Calcium influx through NMDARs activates calcineurin/NFAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, then activates target genes such as <i>EGR3 </i>[46,49]. EGR3 regulates its target genes (e.g., <i>GABRA4</i>, <i>ARC </i>and <i>NGFR</i>) and miRNAs (e.g., hsa-miR-195 and hsa-miR-20b).</div></p></div><div class="aux"><div class="resc">An-Yuan Guo, et al. BMC Syst Biol. 2010;4:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/20156358" ref="ordinalpos=3802&amp;ncbi_uid=2204805&amp;link_uid=20156358">Citation</a><a class="dblinks" href="/pmc/articles/PMC2834616/" ref="ordinalpos=3802&amp;ncbi_uid=2204805&amp;link_uid=PMC2834616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3803.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4216012/figure/pone-0111350-g001/" ref="ordinalpos=3803&amp;ncbi_uid=6805117&amp;link_uid=PMC4216012" href="/pmc/articles/PMC4216012/"><img src="/pmc/articles/instance/4216012/bin/pone.0111350.g001.gif" src-large="/pmc/articles/instance/4216012/bin/pone.0111350.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4216012/" ref="ordinalpos=3803&amp;ncbi_uid=6805117&amp;link_uid=PMC4216012" image-link="/pmc/articles/PMC4216012/figure/pone-0111350-g001/" class="imagepopup">Figure 1. Schematic representation of the canonical (Wnt/β-catenin) and non-canonical (Wnt/Ca2+ and Wnt/PCP) <span class="highlight" style="background-color:">signaling</span> pathways..  From: Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Modulates Human Airway Sensitization Induced by ?2-Adrenoceptor Stimulation. </a></p><div class="supp"><p class="details"><div>Drugs used in the experiments and their action on Wnt <span class="highlight" style="background-color:">signaling</span> pathways are figured in black. Activation and inhibition are symbolized by <img src="/pmc/articles/instance/4216012/bin/pone.0111350.e001.jpg" class="inline-graphic" style="border-style: none;" /> and <img src="/pmc/articles/instance/4216012/bin/pone.0111350.e002.jpg" class="inline-graphic" style="border-style: none;" />, respectively. In the presence of Wnt ligand, Fzd and LRP5/6 form a receptor complex leading to the recruitment of cytosolic proteins Dsh and LRP5/6 phosphorylation. The phosphorylation and partial internalization of LRP5/6 initiate the disruption of the destruction complex (axin, APC, CK-1, and GSK-3β), allowing cytosolic β-catenin accumulation and then translocation into the nucleus, where β-catenin serves as a coactivator of TCF/LEF transcription factors to control gene transcription. Depending on the cellular context, the non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is stimulated by the binding of Fzd and ROR2/RYK coreceptors. The Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> induces PKC activation via the intracellular Ca<sup>2+</sup> increase. The Wnt/PCP <span class="highlight" style="background-color:">pathway</span> activates the small G proteins Rho and the MAPK cascade via the recruitment of the cytosolic proteins Dsh and DAAM1, or, alternatively, triggers activation of JNK leading to the transcription of target genes through AP-1 activation.</div></p></div><div class="aux"><div class="resc">Christophe Faisy, et al. PLoS One. 2014;9(10):e111350.</div><p class="links"><a class="dblinks" href="/pubmed/25360795" ref="ordinalpos=3803&amp;ncbi_uid=6805117&amp;link_uid=25360795">Citation</a><a class="dblinks" href="/pmc/articles/PMC4216012/" ref="ordinalpos=3803&amp;ncbi_uid=6805117&amp;link_uid=PMC4216012">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3804.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2867676/figure/F4/" ref="ordinalpos=3804&amp;ncbi_uid=2523165&amp;link_uid=PMC2867676" href="/pmc/articles/PMC2867676/"><img src="/pmc/articles/instance/2867676/bin/pnas.1000302107fig4.gif" src-large="/pmc/articles/instance/2867676/bin/pnas.1000302107fig4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2867676/" ref="ordinalpos=3804&amp;ncbi_uid=2523165&amp;link_uid=PMC2867676" image-link="/pmc/articles/PMC2867676/figure/F4/" class="imagepopup">Fig. 4. From: Recapitulation of endochondral bone formation using human adult mesenchymal stem cells as a paradigm for developmental engineering. </a></p><div class="supp"><p class="details"><div>Activation of <span class="highlight" style="background-color:">signaling</span> pathways involved in endochondral bone formation in embryos. <span class="highlight" style="background-color:">Signaling</span> pathways typically involved in endochondral ossification were activated in the engineered samples. (<i>A</i>) Real-time RT-PCR analysis indicated that MSC cultured under hypertrophic conditions up-regulated the expression of genes in the IHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (involving IHH, GLI1, and PTCH1), BMPs and parathyroid hormone-related protein <span class="highlight" style="background-color:">signaling</span> (PTHLH, PTHR1). Note that all fold changes in transcript levels are shown in logarithmic scale. (<i>B</i>–<i>D</i>) Four weeks after implantation, the expression of representative genes was assessed by ISH (IHH, GLI1, and BMP7). (<i>E</i>–<i>G</i>) Functional inhibition of the IHH <span class="highlight" style="background-color:">pathway</span> by cyclopamine treatment significantly reduced the expression of genes involved in IHH <span class="highlight" style="background-color:">signaling</span> (IHH, GLI1, PTCH1), PTH <span class="highlight" style="background-color:">signaling</span> (PTHLH, PTHR1), as well as chondrogenic/hypertrophic genes (Col II, VEGF), and osteogenic genes (Cbfa-1, Col I, BSP). Cyclopamine also blocked the differentiation and maturation of the cartilaginous templates in vitro, as assessed by Safranin-O stain. [Scale bar: 200 μm (<i>B</i>–<i>D</i>).] [Scale bar: 400 μm (<i>E</i> and <i>F</i>)].</div></p></div><div class="aux"><div class="resc">Celeste Scotti, et al. Proc Natl Acad Sci U S A. 2010 April 20;107(16):7251-7256.</div><p class="links"><a class="dblinks" href="/pubmed/20406908" ref="ordinalpos=3804&amp;ncbi_uid=2523165&amp;link_uid=20406908">Citation</a><a class="dblinks" href="/pmc/articles/PMC2867676/" ref="ordinalpos=3804&amp;ncbi_uid=2523165&amp;link_uid=PMC2867676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3805.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3666437/figure/fig5/" ref="ordinalpos=3805&amp;ncbi_uid=4994624&amp;link_uid=PMC3666437" href="/pmc/articles/PMC3666437/"><img src="/pmc/articles/instance/3666437/bin/ECAM2013-510208.005.gif" src-large="/pmc/articles/instance/3666437/bin/ECAM2013-510208.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3666437/" ref="ordinalpos=3805&amp;ncbi_uid=4994624&amp;link_uid=PMC3666437" image-link="/pmc/articles/PMC3666437/figure/fig5/" class="imagepopup">Figure 5.  From: A Systems Biology Approach to Characterize Biomarkers for Blood Stasis Syndrome of Unstable Angina Patients by Integrating MicroRNA and Messenger RNA Expression Profiling. </a></p><div class="supp"><p class="details"><div>The regulation of upregulated miRNAs on the downregulated mRNAs in antigen processing and presentation <span class="highlight" style="background-color:">pathway</span> and p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upregulated miRNAs were depicted in red and downregulated mRNAs were depicted in green on the maps.</div></p></div><div class="aux"><div class="resc">Jie Wang, et al. Evid Based Complement Alternat Med. 2013;2013:510208.</div><p class="links"><a class="dblinks" href="/pubmed/23762142" ref="ordinalpos=3805&amp;ncbi_uid=4994624&amp;link_uid=23762142">Citation</a><a class="dblinks" href="/pmc/articles/PMC3666437/" ref="ordinalpos=3805&amp;ncbi_uid=4994624&amp;link_uid=PMC3666437">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3806.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508514/figure/F2/" ref="ordinalpos=3806&amp;ncbi_uid=4393067&amp;link_uid=PMC3508514" href="/pmc/articles/PMC3508514/"><img src="/pmc/articles/instance/3508514/bin/nihms421465f2.gif" src-large="/pmc/articles/instance/3508514/bin/nihms421465f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508514/" ref="ordinalpos=3806&amp;ncbi_uid=4393067&amp;link_uid=PMC3508514" image-link="/pmc/articles/PMC3508514/figure/F2/" class="imagepopup">Fig. 2.  From: Cooperativity of adaptive and innate immunity: implications for cancer therapy. </a></p><div class="supp"><p class="details"><div>The classical STAT-dependent <span class="highlight" style="background-color:">pathway</span> of cytokine receptor cross-talk. The schematic illustration depicts the classical <span class="highlight" style="background-color:">pathway</span> of Jak-STAT-dependent cytokine-mediated regulation of heterologous receptor functions. Cytokine <span class="highlight" style="background-color:">signaling</span> initiates a unique transcriptional program leading to the cellular expression of activating or inhibitory heterologous receptors</div></p></div><div class="aux"><div class="resc">Anil Shanker, et al. Cancer Immunol Immunother. ;60(8):1061-1074.</div><p class="links"><a class="dblinks" href="/pubmed/21656157" ref="ordinalpos=3806&amp;ncbi_uid=4393067&amp;link_uid=21656157">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508514/" ref="ordinalpos=3806&amp;ncbi_uid=4393067&amp;link_uid=PMC3508514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3807.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3880986/figure/F4/" ref="ordinalpos=3807&amp;ncbi_uid=5681666&amp;link_uid=PMC3880986" href="/pmc/articles/PMC3880986/"><img src="/pmc/articles/instance/3880986/bin/ijmsv11p0007g006.gif" src-large="/pmc/articles/instance/3880986/bin/ijmsv11p0007g006.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3880986/figure/F4/" ref="ordinalpos=3807&amp;ncbi_uid=5681666&amp;link_uid=PMC3880986" href="/pmc/articles/PMC3880986/"><img src="/pmc/articles/instance/3880986/bin/ijmsv11p0007g007.gif" src-large="/pmc/articles/instance/3880986/bin/ijmsv11p0007g007.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3880986/" ref="ordinalpos=3807&amp;ncbi_uid=5681666&amp;link_uid=PMC3880986" image-link="/pmc/articles/PMC3880986/figure/F4/" class="imagepopup">Fig 4.  From: 17?-Estradiol Inhibits Mesenchymal Stem Cells-Induced Human AGS Gastric Cancer Cell Mobility via Suppression of CCL5- Src/Cas/Paxillin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b> CCL-5 induces AGS cell mobility via Src <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> AGS cells were pretreated with vehicle, LY294002 (Akt activation inhibitor, 1μM), U0126 (ERK1/2 activation inhibitor, 1μM), SB203580 (p38 MAPK inhibitor, 1μM), SP600125 (JNK1/2 inhibitor, 1μM) or PP2 (Src inhibitor, 1μM) for 1h, and followed by recombinant CCL-5 (10 ng/ml) administration for 24h.<b> (B)</b> AGS cells were treated with these inhibitors, respectively. AGS cells were harvested and measured for cellular mobility capacity. **, <i>p</i>&lt;0.01 <i>versus</i> control (line 1); ##, <i>p</i>&lt;0.01 <i>versus</i> CCL-5 treatment (line 2) (mean ± SD, n = 3).</div></p></div><div class="aux"><div class="resc">Chao-Hung Kuo, et al. Int J Med Sci. 2014;11(1):7-16.</div><p class="links"><a class="dblinks" href="/pubmed/24396281" ref="ordinalpos=3807&amp;ncbi_uid=5681666&amp;link_uid=24396281">Citation</a><a class="dblinks" href="/pmc/articles/PMC3880986/" ref="ordinalpos=3807&amp;ncbi_uid=5681666&amp;link_uid=PMC3880986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3808.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590365/figure/Fig2/" ref="ordinalpos=3808&amp;ncbi_uid=4708021&amp;link_uid=PMC3590365" href="/pmc/articles/PMC3590365/"><img src="/pmc/articles/instance/3590365/bin/12079_2012_183_Fig2_HTML.gif" src-large="/pmc/articles/instance/3590365/bin/12079_2012_183_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590365/" ref="ordinalpos=3808&amp;ncbi_uid=4708021&amp;link_uid=PMC3590365" image-link="/pmc/articles/PMC3590365/figure/Fig2/" class="imagepopup">Fig. 2.  From: CCN2 exerts direct cytoprotective actions in adult cardiac myocytes by activation of the PI3-kinase/Akt/GSK-3? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of Akt/GSK-3β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by CCN2. <b>a</b> Western blot analysis of phospho-Akt (Ser473) and <b>b</b> phospho-GSK-3β (Ser9) contents in cardiac myocytes stimulated with increasing concentrations of rhCCN2 for 30 min. <b>c</b> Western blot analysis of phospho-glycogen synthase (Ser641) levels in cardiac myocytes stimulated with 250 nmol/L rhCCN2 for 24 and 40 h. The histograms demonstrate densitometric analysis of phospho-protein levels relative to respective total protein contents. Data are mean ± SEM of <i>n</i> = 3 in each group, *<i>p</i> &lt; 0.05 vs. control. The figure is representative of three independent experiments</div></p></div><div class="aux"><div class="resc">Ingvild Tronstad Moe, et al. J Cell Commun Signal. 2013 March;7(1):31-47.</div><p class="links"><a class="dblinks" href="/pubmed/23208610" ref="ordinalpos=3808&amp;ncbi_uid=4708021&amp;link_uid=23208610">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590365/" ref="ordinalpos=3808&amp;ncbi_uid=4708021&amp;link_uid=PMC3590365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3809.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4252254/figure/pone-0113257-g010/" ref="ordinalpos=3809&amp;ncbi_uid=6918143&amp;link_uid=PMC4252254" href="/pmc/articles/PMC4252254/"><img src="/pmc/articles/instance/4252254/bin/pone.0113257.g010.gif" src-large="/pmc/articles/instance/4252254/bin/pone.0113257.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4252254/" ref="ordinalpos=3809&amp;ncbi_uid=6918143&amp;link_uid=PMC4252254" image-link="/pmc/articles/PMC4252254/figure/pone-0113257-g010/" class="imagepopup">Figure 10. Proposed model of p,p′-DDT-induced <span class="highlight" style="background-color:">signaling</span> pathways leading to apoptosis..  From: Protective Efficacy of Vitamins C and E on p,p′-DDT-Induced Cytotoxicity via the ROS-Mediated Mitochondrial <span class="highlight" style="background-color:">Pathway</span> and NF-κB/FasL <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>ROS generation might play a critical role in the initiation of <i>p,p′</i>-DDT-induced apoptosis of human liver cells through two mechanisms, one was the mitochondria-mediated <span class="highlight" style="background-color:">pathway</span> including elevation of ROS, decrease in ΔΨm along with the cytochrome c release from mitochondria into the cytosol, and activation of the caspase 9 and 3 in our previous study; and the other was the elevation of ROS, which resulted in the activation of NF-κB and expression of FasL, then triggered FasL-dependent <span class="highlight" style="background-color:">pathway</span> in the present study.</div></p></div><div class="aux"><div class="resc">Xiaoting Jin, et al. PLoS One. 2014;9(12):e113257.</div><p class="links"><a class="dblinks" href="/pubmed/25464339" ref="ordinalpos=3809&amp;ncbi_uid=6918143&amp;link_uid=25464339">Citation</a><a class="dblinks" href="/pmc/articles/PMC4252254/" ref="ordinalpos=3809&amp;ncbi_uid=6918143&amp;link_uid=PMC4252254">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3810.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2639554/figure/F1/" ref="ordinalpos=3810&amp;ncbi_uid=1774400&amp;link_uid=PMC2639554" href="/pmc/articles/PMC2639554/"><img src="/pmc/articles/instance/2639554/bin/nihms82409f1.gif" src-large="/pmc/articles/instance/2639554/bin/nihms82409f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2639554/" ref="ordinalpos=3810&amp;ncbi_uid=1774400&amp;link_uid=PMC2639554" image-link="/pmc/articles/PMC2639554/figure/F1/" class="imagepopup">Fig. 1. Raf-1 activation decreases the levels of cell–cell contact molecules in MTC cells.  From: Regulation of cell-cell contact molecules and the metastatic phenotype of medullary thyroid carcinoma by the Raf-1/MEK/ERK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>TT and TT-raf cells were treated with ethanol (<i>C</i>) and estradiol (<i>E2</i>) for 48 hours, and the total cellular extracts were analyzed by Western blot to examine activation of the Raf-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Total ERK1/2 and G3PDH were used to confirm equal protein loading. (<i>A</i>) Treatment of TT-raf cells with E<sub>2</sub> led to high levels of phospho-MEK1/2 and phospho-ERK1/2, confirming the Raf-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. (<i>B</i>) Treatment of TT-raf cells with E<sub>2</sub> resulted in a significant decrease in the expression of E-cadherin, β-catenin, and occluding.</div></p></div><div class="aux"><div class="resc">Li Ning, et al. Surgery. ;144(6):920-925.</div><p class="links"><a class="dblinks" href="/pubmed/19040998" ref="ordinalpos=3810&amp;ncbi_uid=1774400&amp;link_uid=19040998">Citation</a><a class="dblinks" href="/pmc/articles/PMC2639554/" ref="ordinalpos=3810&amp;ncbi_uid=1774400&amp;link_uid=PMC2639554">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3811.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f001/" ref="ordinalpos=3811&amp;ncbi_uid=5948548&amp;link_uid=PMC3967167" href="/pmc/articles/PMC3967167/"><img src="/pmc/articles/instance/3967167/bin/viruses-05-03192-g001.gif" src-large="/pmc/articles/instance/3967167/bin/viruses-05-03192-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3967167/" ref="ordinalpos=3811&amp;ncbi_uid=5948548&amp;link_uid=PMC3967167" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f001/" class="imagepopup">Figure 1.  From: Make Yourself at Home: Viral Hijacking of the PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic drawing of the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The phosphatidylinositol 3-kinase (PI3K) is activated through receptor-binding (receptor tyrosine kinases (RTKs) or G protein-coupled receptors (GPCR), sidled blue line) by growth factors (GF) or cytokines (CY) resulting in phosphorylation of PIP<sub>2</sub>. PIP<sub>3</sub> subsequently serves as a second messenger allowing the binding of pleckstrin homology domain-containing proteins like Akt. Thereby the latter undergoes conformational changes leading to its phosphorylation and activation by PDK1 and mTORC2. Akt finally participates in the regulation of cellular processes like translation, cell growth and apoptosis by phosphorylating further proteins. Termination of the <span class="highlight" style="background-color:">signaling</span> cascade can either occur through the dephosphorylation of PIP<sub>3</sub> by the phosphatase PTEN or SHIP, or further downstream through the dephosphorylation of Akt by PHLPP or PP2A.</div></p></div><div class="aux"><div class="resc">Nora Diehl, et al. Viruses. 2013 December;5(12):3192-3212.</div><p class="links"><a class="dblinks" href="/pubmed/24351799" ref="ordinalpos=3811&amp;ncbi_uid=5948548&amp;link_uid=24351799">Citation</a><a class="dblinks" href="/pmc/articles/PMC3967167/" ref="ordinalpos=3811&amp;ncbi_uid=5948548&amp;link_uid=PMC3967167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3812.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2075101/figure/F1/" ref="ordinalpos=3812&amp;ncbi_uid=1213593&amp;link_uid=PMC2075101" href="/pmc/articles/PMC2075101/"><img src="/pmc/articles/instance/2075101/bin/nihms-30394-f0001.gif" src-large="/pmc/articles/instance/2075101/bin/nihms-30394-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2075101/" ref="ordinalpos=3812&amp;ncbi_uid=1213593&amp;link_uid=PMC2075101" image-link="/pmc/articles/PMC2075101/figure/F1/" class="imagepopup">Figure 1.  From: Chronic Olanzapine Activates the Stat3 Signal Transduction <span class="highlight" style="background-color:">Pathway</span> and Alters Expression of Components of the 5-HT2A Receptor <span class="highlight" style="background-color:">Signaling</span> System in Rat Frontal Cortex. </a></p><div class="supp"><p class="details"><div>5-HT<sub>2A</sub> receptor agonists cause activation of G<sub>αq/11</sub> proteins which in turn activate the second messenger enzyme PLC. Chronic treatment with olanzapine causes desensitization of this <span class="highlight" style="background-color:">pathway</span> as measured by either production of inositol phosphate in cells or PLC activity in brain tissue. We hypothesize that 5-HT<sub>2A</sub> receptor antagonism (induced by olanzapine) activates the Jak2/Stat3 <span class="highlight" style="background-color:">pathway</span> causing phosphorylated Stat3 to dimerize and translocate to the nucleus, stimulate immediate early genes and subsequently increase RGS7 transcription and RGS7 protein levels in the membrane. The increased membrane-associated RGS7 protein can then increase hydrolysis of activated G<sub>αq/11</sub> and result in desensitization of 5-HT<sub>2A</sub> receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">NA Muma, et al. Neuropharmacology. ;53(4):552-562.</div><p class="links"><a class="dblinks" href="/pubmed/17675105" ref="ordinalpos=3812&amp;ncbi_uid=1213593&amp;link_uid=17675105">Citation</a><a class="dblinks" href="/pmc/articles/PMC2075101/" ref="ordinalpos=3812&amp;ncbi_uid=1213593&amp;link_uid=PMC2075101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3813.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2709856/figure/F1/" ref="ordinalpos=3813&amp;ncbi_uid=1882885&amp;link_uid=PMC2709856" href="/pmc/articles/PMC2709856/"><img src="/pmc/articles/instance/2709856/bin/nihms-110725-f0001.gif" src-large="/pmc/articles/instance/2709856/bin/nihms-110725-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2709856/" ref="ordinalpos=3813&amp;ncbi_uid=1882885&amp;link_uid=PMC2709856" image-link="/pmc/articles/PMC2709856/figure/F1/" class="imagepopup">Figure 1. Preconditioning by redirecting Toll-like receptor <span class="highlight" style="background-color:">signaling</span> pathways.  From: It's all in the family: multiple Toll-like receptors offer promise as novel therapeutic targets for stroke neuroprotection. </a></p><div class="supp"><p class="details"><div>Using TLR4, which can signal through both the MyD88 and TRIF pathways, as an example, this figure depicts how the shift from a proinflammatory to an anti-inflammatory response may be achieved to provide neuroprotection. <span class="highlight" style="background-color:">Signaling</span> through the MyD88 <span class="highlight" style="background-color:">pathway</span> leads to the production of proinflammatory mediators, whereas <span class="highlight" style="background-color:">signaling</span> through the TRIF <span class="highlight" style="background-color:">pathway</span> both activates NF-κB and induces Type I IFNs and IFN-inducible genes. Experiments using MyD88- and TRIF-deficient mice that have been preconditioned with lipopolysaccharide (LPS) have led to the hypothesis that TRIF is the key adaptor protein for the TLR4 response to LPS preconditioning, since MyD88-deficient mice can be preconditioned whereas TRIF-deficient mice cannot. In turn, <span class="highlight" style="background-color:">signaling</span> down the TRIF <span class="highlight" style="background-color:">pathway</span> leads to the activation of NF-κB for proinflammatory responses and IRFs for anti-inflammatory responses. The initial TLR4 stimulation by LPS (solid line) may reprogram the subsequent TLR4 response to ischemia to one that activates IRFs and IFN production (dotted line). This reprogramming may be mediated by several <span class="highlight" style="background-color:">pathway</span> inhibitors (e.g., TRIM30α), which are upregulated after LPS preconditioning as well as ischemic insult, and may act to destabilize the TAK1 complex, which in turn blocks NF-κB activation.<br />IFN: Interferon; IRF: IFN-regulatory factors; TAB: Transforming growth factor-B binding protein; TIR: Toll—interleukin 1 receptor; TIRAP: TIR domain-containing adaptor protein; TLR: Toll-like receptor; TRAF: TNF-receptor-associated factor; TRIF: TIR domain-containing adaptor inducing IFN.</div></p></div><div class="aux"><div class="resc">Philberta Y Leung, et al. Future Neurol. ;4(2):201-208.</div><p class="links"><a class="dblinks" href="/pubmed/19885374" ref="ordinalpos=3813&amp;ncbi_uid=1882885&amp;link_uid=19885374">Citation</a><a class="dblinks" href="/pmc/articles/PMC2709856/" ref="ordinalpos=3813&amp;ncbi_uid=1882885&amp;link_uid=PMC2709856">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3814.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3580779/figure/F3/" ref="ordinalpos=3814&amp;ncbi_uid=5261800&amp;link_uid=PMC3580779" href="/pmc/articles/PMC3580779/"><img src="/pmc/articles/instance/3580779/bin/CG-14-49_F3.gif" src-large="/pmc/articles/instance/3580779/bin/CG-14-49_F3.jpg" alt="Fig. (3)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3580779/" ref="ordinalpos=3814&amp;ncbi_uid=5261800&amp;link_uid=PMC3580779" image-link="/pmc/articles/PMC3580779/figure/F3/" class="imagepopup">Fig. (3).  From:  Mechano-Regulation of Alternative Splicing. </a></p><div class="supp"><p class="details"><div><b>Phosphorylation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of alternative splicing regulated by mechanical stress.</b> In response to mechanical stimulation, Ca<sup>2+</sup> influx is induced by Ca<sup>2+</sup> channel. The pathways of CaMKII-TAK1-SAPKs and NF-κB-IL-6 and PKC are activated by Ca<sup>2+</sup> influx, then regulate the alternative splicing of pre-mRNA. Integrins which are sensitive to mechanical stimulation, transmit mechanical signals into cells by ECM and activate FAK/ERK/MAPK path way which regulate alternative splicing directly or under help of RhoA/Rho-ROCK <span class="highlight" style="background-color:">signaling</span>. PI3K is another possible <span class="highlight" style="background-color:">signaling</span> that modulates alternative splicing in cells exposed to mechanical stress.</div></p></div><div class="aux"><div class="resc">Huan Liu, et al. Curr Genomics. 2013 March;14(1):49-55.</div><p class="links"><a class="dblinks" href="/pubmed/23997650" ref="ordinalpos=3814&amp;ncbi_uid=5261800&amp;link_uid=23997650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3580779/" ref="ordinalpos=3814&amp;ncbi_uid=5261800&amp;link_uid=PMC3580779">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3815.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3543343/figure/F5/" ref="ordinalpos=3815&amp;ncbi_uid=4531270&amp;link_uid=PMC3543343" href="/pmc/articles/PMC3543343/"><img src="/pmc/articles/instance/3543343/bin/1479-5876-10-230-5.gif" src-large="/pmc/articles/instance/3543343/bin/1479-5876-10-230-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3543343/" ref="ordinalpos=3815&amp;ncbi_uid=4531270&amp;link_uid=PMC3543343" image-link="/pmc/articles/PMC3543343/figure/F5/" class="imagepopup">Figure 5.  From: After insufficient radiofrequency ablation, tumor-associated endothelial cells exhibit enhanced angiogenesis and promote invasiveness of residual hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div><b>Enhanced activity of ERK1/2, NF-κB and Akt <span class="highlight" style="background-color:">signaling</span> pathways and inhibition of the STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after insufficient RFA.</b> The changes in <span class="highlight" style="background-color:">signaling</span> pathways involving TAECs after insufficient RFA were detected using western blot. Data are the representative results of three independent experiments with five samples per group.</div></p></div><div class="aux"><div class="resc">Jian Kong, et al. J Transl Med. 2012;10:230-230.</div><p class="links"><a class="dblinks" href="/pubmed/23171368" ref="ordinalpos=3815&amp;ncbi_uid=4531270&amp;link_uid=23171368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3543343/" ref="ordinalpos=3815&amp;ncbi_uid=4531270&amp;link_uid=PMC3543343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3816.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3874981/figure/fig4/" ref="ordinalpos=3816&amp;ncbi_uid=5699709&amp;link_uid=PMC3874981" href="/pmc/articles/PMC3874981/"><img src="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.004.gif" src-large="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3874981/" ref="ordinalpos=3816&amp;ncbi_uid=5699709&amp;link_uid=PMC3874981" image-link="/pmc/articles/PMC3874981/figure/fig4/" class="imagepopup">Figure 4.  From: Review of <span class="highlight" style="background-color:">Signaling</span> Pathways Governing MSC Osteogenic and Adipogenic Differentiation. </a></p><div class="supp"><p class="details"><div>Schematic of Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The initially insoluble Hedgehog (HH) ligand precursor undergoes a series of intracellular modifications before reaching an active, multimeric form. Following release from the membrane by Dispatched (DISP), the morphogen binds to Patched (PTCH), which releases Smoothened (SMO) from constitutive inhibition by PTCH. This activates the Gli2/3 complex, which goes on to promote gene expression of Gli1, while repressing the transcriptional repressor Gli3.</div></p></div><div class="aux"><div class="resc">Aaron W. James. Scientifica (Cairo). 2013;2013:684736.</div><p class="links"><a class="dblinks" href="/pubmed/24416618" ref="ordinalpos=3816&amp;ncbi_uid=5699709&amp;link_uid=24416618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3874981/" ref="ordinalpos=3816&amp;ncbi_uid=5699709&amp;link_uid=PMC3874981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3817.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2821661/figure/F5/" ref="ordinalpos=3817&amp;ncbi_uid=2734856&amp;link_uid=PMC2821661" href="/pmc/articles/PMC2821661/"><img src="/pmc/articles/instance/2821661/bin/nihms150364f5.gif" src-large="/pmc/articles/instance/2821661/bin/nihms150364f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2821661/" ref="ordinalpos=3817&amp;ncbi_uid=2734856&amp;link_uid=PMC2821661" image-link="/pmc/articles/PMC2821661/figure/F5/" class="imagepopup">Fig. 5.  From: A direct cross-talk between interferon-? and sonic hedgehog <span class="highlight" style="background-color:">signaling</span> that leads to the proliferation of neuronal precursor cells. </a></p><div class="supp"><p class="details"><div>Attenuation of IFN-γ-induced GNP proliferation by inhibition of Shh or Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The primary GNP cultures were treated with cyclopamine (CPM) (A) or anti-Shh antibodies (B) at the indicated concentrations before the addition of IFN-γ (200 IU/ml), and MTT proliferation assay was performed after 72-hour treatment. (<i>n</i> = 3-4; *: <i>p</i> &lt; 0.05; **: <i>p</i> &lt; 0.01 (one-way ANOVA followed by Dunnett's t-test)).</div></p></div><div class="aux"><div class="resc">Li Sun, et al. Brain Behav Immun. ;24(2):220.</div><p class="links"><a class="dblinks" href="/pubmed/19800966" ref="ordinalpos=3817&amp;ncbi_uid=2734856&amp;link_uid=19800966">Citation</a><a class="dblinks" href="/pmc/articles/PMC2821661/" ref="ordinalpos=3817&amp;ncbi_uid=2734856&amp;link_uid=PMC2821661">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3818.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2883790/figure/F2/" ref="ordinalpos=3818&amp;ncbi_uid=2681184&amp;link_uid=PMC2883790" href="/pmc/articles/PMC2883790/"><img src="/pmc/articles/instance/2883790/bin/nihms183051f2.gif" src-large="/pmc/articles/instance/2883790/bin/nihms183051f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2883790/" ref="ordinalpos=3818&amp;ncbi_uid=2681184&amp;link_uid=PMC2883790" image-link="/pmc/articles/PMC2883790/figure/F2/" class="imagepopup">Fig. 2.  From: mTOR <span class="highlight" style="background-color:">SIGNALING</span> IN CANCER CELL MOTILITY AND TUMOR METASTASIS. </a></p><div class="supp"><p class="details"><div>Schematic of IGF-I activation of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to mTOR. mTOR protein exists in two distinct complexes, mTORC1 and mTORC2. mTORC1, which receives input from several upstream pathways, including PI3K/Akt, TSC1/TSC2, and AMPK, phosphorylates downstream effectors S6K and 4EBP1 and regulates ribosome biogenesis, cell growth, autophagy, cell motility and metabolism. mTORC2 phosphorylates Akt at Ser473 and regulates the actin cytoskeleton and cell motility. Arrows represent activation, whereas bars represent inhibition.</div></p></div><div class="aux"><div class="resc">Hongyu Zhou, et al. Crit Rev Eukaryot Gene Expr. ;20(1):1-16.</div><p class="links"><a class="dblinks" href="/pubmed/20528734" ref="ordinalpos=3818&amp;ncbi_uid=2681184&amp;link_uid=20528734">Citation</a><a class="dblinks" href="/pmc/articles/PMC2883790/" ref="ordinalpos=3818&amp;ncbi_uid=2681184&amp;link_uid=PMC2883790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3819.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3065765/figure/fig3/" ref="ordinalpos=3819&amp;ncbi_uid=3645915&amp;link_uid=PMC3065765" href="/pmc/articles/PMC3065765/"><img src="/pmc/articles/instance/3065765/bin/112fig3.gif" src-large="/pmc/articles/instance/3065765/bin/112fig3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3065765/" ref="ordinalpos=3819&amp;ncbi_uid=3645915&amp;link_uid=PMC3065765" image-link="/pmc/articles/PMC3065765/figure/fig3/" class="imagepopup">Figure 3.  From: The Role of Iron in Learning and Memory. </a></p><div class="supp"><p class="details"><div>The role of iron in regulating the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span><i>.</i> Note potential entry points where iron can directly and indirectly alter mTOR <span class="highlight" style="background-color:">signaling</span>. Figure adapted with permission from Wullschleger et al. (80). Citations supporting the evidence for iron regulation are noted in parentheses.</div></p></div><div class="aux"><div class="resc">Stephanie J. B. Fretham, et al. Adv Nutr. 2011 March;2(2):112-121.</div><p class="links"><a class="dblinks" href="/pubmed/22332040" ref="ordinalpos=3819&amp;ncbi_uid=3645915&amp;link_uid=22332040">Citation</a><a class="dblinks" href="/pmc/articles/PMC3065765/" ref="ordinalpos=3819&amp;ncbi_uid=3645915&amp;link_uid=PMC3065765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3820.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2422835/figure/F2/" ref="ordinalpos=3820&amp;ncbi_uid=1471978&amp;link_uid=PMC2422835" href="/pmc/articles/PMC2422835/"><img src="/pmc/articles/instance/2422835/bin/nihms-50556-f0002.gif" src-large="/pmc/articles/instance/2422835/bin/nihms-50556-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2422835/" ref="ordinalpos=3820&amp;ncbi_uid=1471978&amp;link_uid=PMC2422835" image-link="/pmc/articles/PMC2422835/figure/F2/" class="imagepopup">Figure 2. Regulation of the salvage <span class="highlight" style="background-color:">pathway</span> and function of ceramide signals.  From: The sphingolipid salvage <span class="highlight" style="background-color:">pathway</span> in ceramide metabolism and <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>PKC, protein kinase C; CAPP, ceramide-activated protein phosphatase</div></p></div><div class="aux"><div class="resc">Kazuyuki Kitatani, et al. Cell Signal. ;20(6):1010-1018.</div><p class="links"><a class="dblinks" href="/pubmed/18191382" ref="ordinalpos=3820&amp;ncbi_uid=1471978&amp;link_uid=18191382">Citation</a><a class="dblinks" href="/pmc/articles/PMC2422835/" ref="ordinalpos=3820&amp;ncbi_uid=1471978&amp;link_uid=PMC2422835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3821.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3383846/figure/fig2-0022034512437372/" ref="ordinalpos=3821&amp;ncbi_uid=5033420&amp;link_uid=PMC3383846" href="/pmc/articles/PMC3383846/"><img src="/pmc/articles/instance/3383846/bin/10.1177_0022034512437372-fig2.gif" src-large="/pmc/articles/instance/3383846/bin/10.1177_0022034512437372-fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3383846/" ref="ordinalpos=3821&amp;ncbi_uid=5033420&amp;link_uid=PMC3383846" image-link="/pmc/articles/PMC3383846/figure/fig2-0022034512437372/" class="imagepopup">Figure 2. From: Control of Cytokine mRNA Expression by RNA-binding Proteins and microRNAs. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways that regulate cytokine AREs post-transcriptionally. The p38 <span class="highlight" style="background-color:">pathway</span> is the best-described <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that regulates AU-rich element (ARE)-mediated cytokine degradation. Activation of IL-1β receptors and the Toll-like receptor (TLR) family stimulates p38 MAPK <span class="highlight" style="background-color:">signaling</span> <i>via</i> <span class="highlight" style="background-color:">signaling</span> intermediates, including MKK3/6. Active p38 dictates mRNA stability through activation of MK2, which subsequently phosphorylates and inactivates TTP. Once inactivated, phosphorylated TTP dissociates from the ARE region of cytokine transcripts to enhance mRNA stability and translation, thereby causing an increase in cytokine secretion that can promote chronic inflammation and inflammatory disease progression if not properly regulated. In addition, the phosphatidylinositol-3 kinase (PI3K) <span class="highlight" style="background-color:">pathway</span> can also post-transcriptionally regulate mRNAs through AKT phosphorylation of RBPs. This phosphorylation alleviates the mRNAs of RBPs that normally destabilize the mRNA and results in increased mRNA stability and thus increased cytokine protein production and secretion, which, if not properly controlled, can induce chronic inflammation.</div></p></div><div class="aux"><div class="resc">V. Palanisamy, et al. J Dent Res. 2012 July;91(7):651-658.</div><p class="links"><a class="dblinks" href="/pubmed/22302144" ref="ordinalpos=3821&amp;ncbi_uid=5033420&amp;link_uid=22302144">Citation</a><a class="dblinks" href="/pmc/articles/PMC3383846/" ref="ordinalpos=3821&amp;ncbi_uid=5033420&amp;link_uid=PMC3383846">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3822.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1976333/figure/pgen-0030154-g003/" ref="ordinalpos=3822&amp;ncbi_uid=1059270&amp;link_uid=PMC1976333" href="/pmc/articles/PMC1976333/"><img src="/pmc/articles/instance/1994700/bin/pgen.0030154.g003.gif" src-large="/pmc/articles/instance/1994700/bin/pgen.0030154.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1976333/" ref="ordinalpos=3822&amp;ncbi_uid=1059270&amp;link_uid=PMC1976333" image-link="/pmc/articles/PMC1976333/figure/pgen-0030154-g003/" class="imagepopup">Figure 3. Systematic Identification and Analysis of Nonswarming Mutants.  From: A Comprehensive Genetic Characterization of Bacterial Motility. </a></p><div class="supp"><p class="details"><div>(A) Nonswarming mutants were enriched through twenty rounds of daily successive transfers. Plates imaged after each stage of enrichment show the gradual accumulation of nonswarming mutants at the center of the plates.<br />(B) Distribution of <i>z</i>-scores after enrichment for nonswarming mutants. Not shown is <i>flgD</i>, which has a <i>z</i>-score of 1,350.<br />(C) Images of the wild-type parent strain and chemotaxis mutants on swarm plates.<br />(D) Swarming motility of Δ<i>hns</i> and Δ<i>yfiR</i> mutants.<br />(E) Images of swarm plates with LPS and OPG mutants.<br />(F) Images of otherwise nonswarming LPS and OPG mutants with a Δ<i>rcsF</i> mutation.<br />(G) A model summarizing the involvement of Rcs <span class="highlight" style="background-color:">signaling</span> with swimming, swarming, and χ-phage infection. Components of the Rcs <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in yellow. Mutations in the cellular components highlighted in red were found to impair swimming, swarming, and χ-phage infection by <span class="highlight" style="background-color:">signaling</span> through the Rcs phosphorelay <span class="highlight" style="background-color:">pathway</span> and repressing the expression of the <i>flhDC</i> operon. Black solid arrows denote precursor synthesis or gene expression. Blue dotted arrows denote <span class="highlight" style="background-color:">signaling</span> through the Rcs <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Hany S Girgis, et al. PLoS Genet. 2007 September;3(9):e154.</div><p class="links"><a class="dblinks" href="/pubmed/17941710" ref="ordinalpos=3822&amp;ncbi_uid=1059270&amp;link_uid=17941710">Citation</a><a class="dblinks" href="/pmc/articles/PMC1976333/" ref="ordinalpos=3822&amp;ncbi_uid=1059270&amp;link_uid=PMC1976333">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3823.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4365493/figure/f08/" ref="ordinalpos=3823&amp;ncbi_uid=7351657&amp;link_uid=PMC4365493" href="/pmc/articles/PMC4365493/"><img src="/pmc/articles/instance/4365493/bin/bio-04-02-244-f08.gif" src-large="/pmc/articles/instance/4365493/bin/bio-04-02-244-f08.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4365493/" ref="ordinalpos=3823&amp;ncbi_uid=7351657&amp;link_uid=PMC4365493" image-link="/pmc/articles/PMC4365493/figure/f08/" class="imagepopup">Fig. 8.Hypothetical model of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through which MIF increases vascular permeability..  From: Macrophage migration inhibitory factor induces vascular leakage via autophagy. </a></p><div class="supp"><p class="details"><div>Extracellular MIF binds to the cell surface receptor complex, which consists of CXCR2/CXCR4 and CD74. CD74 interacts with CD44, which mediates signal transduction through MAPK/ERK. ERK has been reported to repress the mTOR complex via the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> to induce the formation of autophagosomes, which later fuse with lysosomes to become autophagolysosomes and degrade proteins. Both MIF-induced junction protein translocation and degradation could increase vascular permeability.</div></p></div><div class="aux"><div class="resc">Hong-Ru Chen, et al. Biol Open. 2015 February 15;4(2):244-252.</div><p class="links"><a class="dblinks" href="/pubmed/25617421" ref="ordinalpos=3823&amp;ncbi_uid=7351657&amp;link_uid=25617421">Citation</a><a class="dblinks" href="/pmc/articles/PMC4365493/" ref="ordinalpos=3823&amp;ncbi_uid=7351657&amp;link_uid=PMC4365493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3824.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3548178/figure/F7/" ref="ordinalpos=3824&amp;ncbi_uid=5760714&amp;link_uid=PMC3548178" href="/pmc/articles/PMC3548178/"><img src="/pmc/articles/instance/3548178/bin/zee0021368930007.gif" src-large="/pmc/articles/instance/3548178/bin/zee0021368930007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3548178/" ref="ordinalpos=3824&amp;ncbi_uid=5760714&amp;link_uid=PMC3548178" image-link="/pmc/articles/PMC3548178/figure/F7/" class="imagepopup">Figure 7. From: Gastrin Induces Sodium-Hydrogen Exchanger 3 Phosphorylation and mTOR Activation via a Phosphoinositide 3-Kinase-/Protein Kinase C-Dependent but AKT-Independent <span class="highlight" style="background-color:">Pathway</span> in Renal Proximal Tubule Cells Derived From a Normotensive Male Human. </a></p><div class="supp"><p class="details"><div>Hypothetical gastrin-CCKBR signal transduction <span class="highlight" style="background-color:">pathway</span>. Gastrin <span class="highlight" style="background-color:">signaling</span> through CCKBR causes PI3 kinase activation and the production of phosphatidylinositol 4,5-trisphosphate (PIP3), which in turn increases the binding of PLC to the plasma membrane. PLC then activates PKCα through both diacylglycerol (DAG) and calcium, and PKCα, but not AKT, causes the phosphorylation of NHE3 and S6.</div></p></div><div class="aux"><div class="resc">Tianbing Liu, et al. Endocrinology. 2013 February;154(2):865-875.</div><p class="links"><a class="dblinks" href="/pubmed/23275470" ref="ordinalpos=3824&amp;ncbi_uid=5760714&amp;link_uid=23275470">Citation</a><a class="dblinks" href="/pmc/articles/PMC3548178/" ref="ordinalpos=3824&amp;ncbi_uid=5760714&amp;link_uid=PMC3548178">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3825.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3166863/figure/fig5/" ref="ordinalpos=3825&amp;ncbi_uid=3406055&amp;link_uid=PMC3166863" href="/pmc/articles/PMC3166863/"><img src="/pmc/articles/instance/3166863/bin/JCB_201009069_RGB_Fig5.gif" src-large="/pmc/articles/instance/3166863/bin/JCB_201009069_RGB_Fig5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3166863/" ref="ordinalpos=3825&amp;ncbi_uid=3406055&amp;link_uid=PMC3166863" image-link="/pmc/articles/PMC3166863/figure/fig5/" class="imagepopup">Figure 5. From: NPHP4, a cilia-associated protein, negatively regulates the Hippo <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>NPHP4 antagonizes the Hippo <span class="highlight" style="background-color:">pathway</span>.</b> Active Hippo <span class="highlight" style="background-color:">signaling</span> leads to phosphorylation (P) and activation of the large tumor suppressor (LATS) kinase, which in turn phosphorylates YAP at Ser-127 and TAZ at Ser-89. This creates phosphoepitopes that are rapidly engaged by 14-3-3, leading to cytoplasmic retention of YAP and TAZ and the inhibition of their transcriptional activation potential. NPHP4 abrogates the interaction of large tumor suppressor with its substrates YAP and TAZ, which abolishes 14-3-3 binding and allows for nuclear translocation of YAP and TAZ.</div></p></div><div class="aux"><div class="resc">Sandra Habbig, et al. J Cell Biol. 2011 May 16;193(4):633-642.</div><p class="links"><a class="dblinks" href="/pubmed/21555462" ref="ordinalpos=3825&amp;ncbi_uid=3406055&amp;link_uid=21555462">Citation</a><a class="dblinks" href="/pmc/articles/PMC3166863/" ref="ordinalpos=3825&amp;ncbi_uid=3406055&amp;link_uid=PMC3166863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3826.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2423172/figure/f7/" ref="ordinalpos=3826&amp;ncbi_uid=1476490&amp;link_uid=PMC2423172" href="/pmc/articles/PMC2423172/"><img src="/pmc/articles/instance/2423172/bin/zmb0100875060007.gif" src-large="/pmc/articles/instance/2423172/bin/zmb0100875060007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2423172/" ref="ordinalpos=3826&amp;ncbi_uid=1476490&amp;link_uid=PMC2423172" image-link="/pmc/articles/PMC2423172/figure/f7/" class="imagepopup">FIG. 7. From: MURC, a Muscle-Restricted Coiled-Coil Protein That Modulates the Rho/ROCK <span class="highlight" style="background-color:">Pathway</span>, Induces Cardiac Dysfunction and Conduction Disturbance . </a></p><div class="supp"><p class="details"><div>Induction of myofibrillar organization by MURC through the Rho/ROCK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Cardiomyocytes were infected with Ad-LacZ or Ad-MURC at an MOI of 10. After infection, cells were treated with vehicle or 10 μM Y-27632. Ninety-six hours later, cells were stained with fluorescein isothiocyanate-conjugated phalloidin. (B) Cardiomyocytes were infected with Ad-Luc shRNA or Ad-rMURC shRNA at an MOI of 20. After 48 h of serum deprivation, cells were treated with vehicle or 100 μM PE. Forty-eight hours later, cells were stained with phalloidin.</div></p></div><div class="aux"><div class="resc">Takehiro Ogata, et al. Mol Cell Biol. 2008 May;28(10):3424-3436.</div><p class="links"><a class="dblinks" href="/pubmed/18332105" ref="ordinalpos=3826&amp;ncbi_uid=1476490&amp;link_uid=18332105">Citation</a><a class="dblinks" href="/pmc/articles/PMC2423172/" ref="ordinalpos=3826&amp;ncbi_uid=1476490&amp;link_uid=PMC2423172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3827.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2789488/figure/F3/" ref="ordinalpos=3827&amp;ncbi_uid=2223019&amp;link_uid=PMC2789488" href="/pmc/articles/PMC2789488/"><img src="/pmc/articles/instance/2789488/bin/nihms-135694-f0003.gif" src-large="/pmc/articles/instance/2789488/bin/nihms-135694-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2789488/" ref="ordinalpos=3827&amp;ncbi_uid=2223019&amp;link_uid=PMC2789488" image-link="/pmc/articles/PMC2789488/figure/F3/" class="imagepopup">Figure 3.  From: Cell and molecular biology of kidney development. </a></p><div class="supp"><p class="details"><div>Schema of major <span class="highlight" style="background-color:">signaling</span> pathways involved in renal development (for space and clarity, not all factors in each <span class="highlight" style="background-color:">pathway</span> shown)</div></p></div><div class="aux"><div class="resc">Kimberly J. Reidy, et al. Semin Nephrol. ;29(4):321-337.</div><p class="links"><a class="dblinks" href="/pubmed/19615554" ref="ordinalpos=3827&amp;ncbi_uid=2223019&amp;link_uid=19615554">Citation</a><a class="dblinks" href="/pmc/articles/PMC2789488/" ref="ordinalpos=3827&amp;ncbi_uid=2223019&amp;link_uid=PMC2789488">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3828.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4320626/figure/Fig7/" ref="ordinalpos=3828&amp;ncbi_uid=7157448&amp;link_uid=PMC4320626" href="/pmc/articles/PMC4320626/"><img src="/pmc/articles/instance/4320626/bin/12943_2014_274_Fig7_HTML.gif" src-large="/pmc/articles/instance/4320626/bin/12943_2014_274_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4320626/" ref="ordinalpos=3828&amp;ncbi_uid=7157448&amp;link_uid=PMC4320626" image-link="/pmc/articles/PMC4320626/figure/Fig7/" class="imagepopup">Figure 7.  From: Activation of the NF-κB <span class="highlight" style="background-color:">pathway</span> as a mechanism of alcohol enhanced progression and metastasis of human hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div> <b>Proposed working model for the role of the NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the regulation of ethanol-enhanced the progression of HCC.</b> (1) ROS induced by alcohol mediated activation of the NF-kB <span class="highlight" style="background-color:">pathway</span>. This effect can be abolished by C3G or PDTC. (2) Nuclear translocation of NF-kB via phosphorylation by the IkB kinase, which then binds target DNA that regulates VEGF and MCP-1 gene expression. Ultimately, the changes stimulated by alcohol enhance angiogenesis, tumor growth and metastasis.</div></p></div><div class="aux"><div class="resc">Fei Wang, et al. Mol Cancer. 2015;14(1):10.</div><p class="links"><a class="dblinks" href="/pubmed/25622857" ref="ordinalpos=3828&amp;ncbi_uid=7157448&amp;link_uid=25622857">Citation</a><a class="dblinks" href="/pmc/articles/PMC4320626/" ref="ordinalpos=3828&amp;ncbi_uid=7157448&amp;link_uid=PMC4320626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3829.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1839891/figure/F1/" ref="ordinalpos=3829&amp;ncbi_uid=889548&amp;link_uid=PMC1839891" href="/pmc/articles/PMC1839891/"><img src="/pmc/articles/instance/1839891/bin/1752-0509-1-13-1.gif" src-large="/pmc/articles/instance/1839891/bin/1752-0509-1-13-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1839891/" ref="ordinalpos=3829&amp;ncbi_uid=889548&amp;link_uid=PMC1839891" image-link="/pmc/articles/PMC1839891/figure/F1/" class="imagepopup">Figure 1.  From: Systems biology by the rules: hybrid intelligent systems for <span class="highlight" style="background-color:">pathway</span> modeling and discovery. </a></p><div class="supp"><p class="details"><div>a. A schematic diagram of hedgehog regulation of tumor growth in the cerebellum is shown. These proteins are key components of the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, though many intermediate enzymes may mediate some steps. The "transforming event" is hypothesized to be required for tumor growth, but details about its function are completely lacking [61]. Tumor growth is a macroscopically defined process that is connected to molecular dynamics using rules. b. The rules for this model are given in linguistic form, much as they might be described in journal articles that discuss this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">William J Bosl. BMC Syst Biol. 2007;1:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/17408503" ref="ordinalpos=3829&amp;ncbi_uid=889548&amp;link_uid=17408503">Citation</a><a class="dblinks" href="/pmc/articles/PMC1839891/" ref="ordinalpos=3829&amp;ncbi_uid=889548&amp;link_uid=PMC1839891">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3830.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3484676/figure/fig3/" ref="ordinalpos=3830&amp;ncbi_uid=4382550&amp;link_uid=PMC3484676" href="/pmc/articles/PMC3484676/"><img src="/pmc/articles/instance/3484676/bin/1459f3.gif" src-large="/pmc/articles/instance/3484676/bin/1459f3.jpg" alt="Figure 3 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3484676/" ref="ordinalpos=3830&amp;ncbi_uid=4382550&amp;link_uid=PMC3484676" image-link="/pmc/articles/PMC3484676/figure/fig3/" class="imagepopup">Figure 3 .  From: Evidence for Autoregulation and Cell <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulation From Genome-Wide Binding of the Drosophila Retinoblastoma Protein. </a></p><div class="supp"><p class="details"><div>Rbf1 occupies multiple nodes in conserved <span class="highlight" style="background-color:">signaling</span> pathways. Genes in the insulin (A), JAK/STAT (B), Notch (C), and Hippo/Warts/Yorkie (D) <span class="highlight" style="background-color:">signaling</span> pathways are targeted at multiple levels by Rbf1. Proteins of genes targeted by Rbf1 are indicated in red.</div></p></div><div class="aux"><div class="resc">Pankaj Acharya, et al. G3 (Bethesda). 2012 November;2(11):1459-1472.</div><p class="links"><a class="dblinks" href="/pubmed/23173097" ref="ordinalpos=3830&amp;ncbi_uid=4382550&amp;link_uid=23173097">Citation</a><a class="dblinks" href="/pmc/articles/PMC3484676/" ref="ordinalpos=3830&amp;ncbi_uid=4382550&amp;link_uid=PMC3484676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3831.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694616/figure/F6/" ref="ordinalpos=3831&amp;ncbi_uid=1936749&amp;link_uid=PMC2694616" href="/pmc/articles/PMC2694616/"><img src="/pmc/articles/instance/2694616/bin/nihms114667f6.gif" src-large="/pmc/articles/instance/2694616/bin/nihms114667f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694616/" ref="ordinalpos=3831&amp;ncbi_uid=1936749&amp;link_uid=PMC2694616" image-link="/pmc/articles/PMC2694616/figure/F6/" class="imagepopup">Figure 6. Activity of <span class="highlight" style="background-color:">pathway</span> module signatures in lung cancer.  From: A Genomic Strategy to Elucidate Modules of Oncogenic <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>A. Predicted Ras and E2F3 <span class="highlight" style="background-color:">pathway</span> activities across a set of 74 lung tumors. Ras activity is largely absent in the squamous cell carcinomas, and present in nearly all adenocarcinomas. Conversely, E2F3 activity is high in squamous cell carcinoma but low in adenocarcinomas. The main heat maps shows the scores of the 8 E2F <span class="highlight" style="background-color:">pathway</span> and 20 Ras <span class="highlight" style="background-color:">pathway</span> signatures (rows) projected onto the subset of 36 squamous cell carcinomas and 38 adenocarcinomas (columns), respectively.<br />B. Kaplan-Meier survival curves for E2F signature 6 and Ras signature 9 across their respective diseases. The dotted black line shows the overall survival of all patients in the data set. The red line shows only the patients with high signature scores, and the blue line shows those with low scores.</div></p></div><div class="aux"><div class="resc">Jeffrey T. Chang, et al. Mol Cell. ;34(1):104-114.</div><p class="links"><a class="dblinks" href="/pubmed/19362539" ref="ordinalpos=3831&amp;ncbi_uid=1936749&amp;link_uid=19362539">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694616/" ref="ordinalpos=3831&amp;ncbi_uid=1936749&amp;link_uid=PMC2694616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3832.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3303591/figure/fig1/" ref="ordinalpos=3832&amp;ncbi_uid=3899813&amp;link_uid=PMC3303591" href="/pmc/articles/PMC3303591/"><img src="/pmc/articles/instance/3303591/bin/JAR2012-384017.001.gif" src-large="/pmc/articles/instance/3303591/bin/JAR2012-384017.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3303591/" ref="ordinalpos=3832&amp;ncbi_uid=3899813&amp;link_uid=PMC3303591" image-link="/pmc/articles/PMC3303591/figure/fig1/" class="imagepopup">Figure 1.  From: Insulin in Central Nervous System: More than Just a Peripheral Hormone. </a></p><div class="supp"><p class="details"><div>Stimulation of PI-3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after activation of IR/IGF-1R mediates insulin neuroprotection against damaging conditions. For example, insulin administration under oxidative stress phosphorylates IR and/or IGF-1R, which in turn activate the PI-3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, regulating the expression of “candidate” proteins, namely, glutathione peroxidase-1 (GPx-1), hexokinase-II (Hxk-II), and also the antiapoptotic Bcl-2 and the proapoptotic caspase-3. Thus, oxidative stress, impaired glucose metabolism and, neuronal apoptosis are counterbalanced. Insulin also interferes with GSK-3<i>β</i> <span class="highlight" style="background-color:">signaling</span>, decreasing its activated form and inhibiting apoptotic neuronal death under oxidized conditions.</div></p></div><div class="aux"><div class="resc">Ana I. Duarte, et al. J Aging Res. 2012;2012:384017.</div><p class="links"><a class="dblinks" href="/pubmed/22500228" ref="ordinalpos=3832&amp;ncbi_uid=3899813&amp;link_uid=22500228">Citation</a><a class="dblinks" href="/pmc/articles/PMC3303591/" ref="ordinalpos=3832&amp;ncbi_uid=3899813&amp;link_uid=PMC3303591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3833.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889256/figure/Fig7/" ref="ordinalpos=3833&amp;ncbi_uid=5703548&amp;link_uid=PMC3889256" href="/pmc/articles/PMC3889256/"><img src="/pmc/articles/instance/3889256/bin/12079_2013_207_Fig7_HTML.gif" src-large="/pmc/articles/instance/3889256/bin/12079_2013_207_Fig7_HTML.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889256/" ref="ordinalpos=3833&amp;ncbi_uid=5703548&amp;link_uid=PMC3889256" image-link="/pmc/articles/PMC3889256/figure/Fig7/" class="imagepopup">Fig. 7.  From: Rsu1 contributes to regulation of cell adhesion and spreading by PINCH1-dependent and - independent mechanisms. </a></p><div class="supp"><p class="details"><div>Rsu1 is a critical component for regulation of p38 Map kinase <span class="highlight" style="background-color:">signaling</span> from the IPP complex. Rsu1 is required for the constitutive and EGF-induced p38 <span class="highlight" style="background-color:">signaling</span>. The elimination of Rsu1 reduces both constitutive and EGF-induced p38 phosphorylation and the expression of N92D-Rsu1 activates constitutive p38 <span class="highlight" style="background-color:">signaling</span>. p38 Map kinase can be activated by detachment and this may occur via the αPix-induced Rac <span class="highlight" style="background-color:">pathway</span>. Dashed and dotted lines represent incompletely characterized pathways</div></p></div><div class="aux"><div class="resc">Reyda Gonzalez-Nieves, et al. J Cell Commun Signal. 2013 December;7(4):279-293.</div><p class="links"><a class="dblinks" href="/pubmed/23765260" ref="ordinalpos=3833&amp;ncbi_uid=5703548&amp;link_uid=23765260">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889256/" ref="ordinalpos=3833&amp;ncbi_uid=5703548&amp;link_uid=PMC3889256">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3834.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2730119/figure/F6/" ref="ordinalpos=3834&amp;ncbi_uid=1980775&amp;link_uid=PMC2730119" href="/pmc/articles/PMC2730119/"><img src="/pmc/articles/instance/2730119/bin/nihms122211f6.gif" src-large="/pmc/articles/instance/2730119/bin/nihms122211f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2730119/" ref="ordinalpos=3834&amp;ncbi_uid=1980775&amp;link_uid=PMC2730119" image-link="/pmc/articles/PMC2730119/figure/F6/" class="imagepopup">Fig. 6.  From: Regulation of potassium (K) handling in the renal collecting duct. </a></p><div class="supp"><p class="details"><div>A scheme showing the role of different <span class="highlight" style="background-color:">signaling</span> pathways in mediating the effect of low K intake on ROMK and BK channels in the CCD. The <i>circle with x</i> indicates the inhibition of a particular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Wen-Hui Wang, et al. Pflugers Arch. ;458(1):157-168.</div><p class="links"><a class="dblinks" href="/pubmed/18839206" ref="ordinalpos=3834&amp;ncbi_uid=1980775&amp;link_uid=18839206">Citation</a><a class="dblinks" href="/pmc/articles/PMC2730119/" ref="ordinalpos=3834&amp;ncbi_uid=1980775&amp;link_uid=PMC2730119">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3835.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3667651/figure/F3/" ref="ordinalpos=3835&amp;ncbi_uid=5562964&amp;link_uid=PMC3667651" href="/pmc/articles/PMC3667651/"><img src="/pmc/articles/instance/3667651/bin/nihms424679f3.gif" src-large="/pmc/articles/instance/3667651/bin/nihms424679f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3667651/" ref="ordinalpos=3835&amp;ncbi_uid=5562964&amp;link_uid=PMC3667651" image-link="/pmc/articles/PMC3667651/figure/F3/" class="imagepopup">Figure 3.  From: Cancer Therapy-Induced Cardiac Toxicity in Early Breast Cancer: Addressing the Unresolved Issues. </a></p><div class="supp"><p class="details"><div>Mechanisms underlying molecularly targeted therapeutics-induced cardiac toxicity. Inhibition of Nrg1/ErbB receptors with HER2-directed therapies impacts numerous <span class="highlight" style="background-color:">signaling</span> pathways resulting in 1) suppression of myofilament protein synthesis via the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>, 2) suppression of protein hypertrophy via the MAPK <span class="highlight" style="background-color:">pathway</span>, 3) suppression of cell survival via Src/Fak <span class="highlight" style="background-color:">pathway</span>, 4) upregulation of protein degradation via FOXO <span class="highlight" style="background-color:">signaling</span>. Fak indicates focal adhesion kinase; MAPK, mitogen-activated protein kinase; NO, nitric oxide; Nrg1, neuregulin-1β; PI3K, phosphatidylinositol 3-kinase.</div></p></div><div class="aux"><div class="resc">Michel G. Khouri, et al. Circulation. ;126(23):2749-2763.</div><p class="links"><a class="dblinks" href="/pubmed/23212997" ref="ordinalpos=3835&amp;ncbi_uid=5562964&amp;link_uid=23212997">Citation</a><a class="dblinks" href="/pmc/articles/PMC3667651/" ref="ordinalpos=3835&amp;ncbi_uid=5562964&amp;link_uid=PMC3667651">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3836.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4125180/figure/pone-0104127-g003/" ref="ordinalpos=3836&amp;ncbi_uid=6486667&amp;link_uid=PMC4125180" href="/pmc/articles/PMC4125180/"><img src="/pmc/articles/instance/4125180/bin/pone.0104127.g003.gif" src-large="/pmc/articles/instance/4125180/bin/pone.0104127.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4125180/" ref="ordinalpos=3836&amp;ncbi_uid=6486667&amp;link_uid=PMC4125180" image-link="/pmc/articles/PMC4125180/figure/pone-0104127-g003/" class="imagepopup">Figure 3. Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) of differentially expressed genes..  From: Profiling of Luteal Transcriptome during Prostaglandin F2-Alpha Treatment in Buffalo Cows: Analysis of <span class="highlight" style="background-color:">Signaling</span> Pathways Associated with Luteolysis. </a></p><div class="supp"><p class="details"><div>(A) The <span class="highlight" style="background-color:">pathway</span> analysis indicated that a large number of differentially expressed genes belong to canonical pathways such as IGF-1 <span class="highlight" style="background-color:">signaling</span>, steroidogenesis, chemokine <span class="highlight" style="background-color:">signaling</span>, prolactin <span class="highlight" style="background-color:">signaling</span>, cellular growth and proliferation, extracellular matrix modulation and apoptosis. The orange line represents score for the likelihood [-log (B-H p&lt;0.05)] that genes belonging to a specific canonical <span class="highlight" style="background-color:">pathway</span> category affected at 3 h post PGF<sub>2α</sub> administration. The stacked bars indicate the percentage of genes distributed according to regulation, i.e., green (down), red (up) and open bars (no overlap with dataset) in each canonical <span class="highlight" style="background-color:">pathway</span>. (B) Network 0 vs. 3 h: Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis of the differentially regulated genes 3 h post PGF<sub>2α</sub> administration shows a network of 21 focus molecules with a score of 48, with top biological functions of cell death, cellular growth and proliferation (immune cells), and cell-to-cell <span class="highlight" style="background-color:">signaling</span>. The network is displayed graphically as nodes (genes/gene products) and edges (biological relationship between nodes). The node color intensity indicates the fold change expression of genes; with red representing up-regulation, and green representing down-regulation of genes between 0 vs. 3 h post PGF<sub>2α</sub> administration. The fold change value for individual gene is indicated under each node. The shapes of nodes indicate the functional class of the gene product and the lines indicate the type of interaction.</div></p></div><div class="aux"><div class="resc">Kunal B. Shah, et al. PLoS One. 2014;9(8):e104127.</div><p class="links"><a class="dblinks" href="/pubmed/25102061" ref="ordinalpos=3836&amp;ncbi_uid=6486667&amp;link_uid=25102061">Citation</a><a class="dblinks" href="/pmc/articles/PMC4125180/" ref="ordinalpos=3836&amp;ncbi_uid=6486667&amp;link_uid=PMC4125180">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3837.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885163/figure/Fig6/" ref="ordinalpos=3837&amp;ncbi_uid=2307423&amp;link_uid=PMC2885163" href="/pmc/articles/PMC2885163/"><img src="/pmc/articles/instance/2885163/bin/12263_2009_158_Fig6_HTML.gif" src-large="/pmc/articles/instance/2885163/bin/12263_2009_158_Fig6_HTML.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885163/" ref="ordinalpos=3837&amp;ncbi_uid=2307423&amp;link_uid=PMC2885163" image-link="/pmc/articles/PMC2885163/figure/Fig6/" class="imagepopup">Fig. 6.  From: Gene expression modifications in the liver caused by binge drinking and S-adenosylmethionine feeding. The role of epigenetic changes. </a></p><div class="supp"><p class="details"><div>KEGG functional pathways 12 h post ethanol bolus where SAMe feeding with ethanol was compared to ethanol fed alone (<i>n</i> = 3). The PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were mainly up regulated (<i>red</i>). The results are quite different from 3 h post ethanol bolus where SAMe down regulated most of the pathways that were up regulated by ethanol (see Fig. 3)</div></p></div><div class="aux"><div class="resc">Jun Li, et al. Genes Nutr. 2010 June;5(2):169-179.</div><p class="links"><a class="dblinks" href="/pubmed/19960281" ref="ordinalpos=3837&amp;ncbi_uid=2307423&amp;link_uid=19960281">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885163/" ref="ordinalpos=3837&amp;ncbi_uid=2307423&amp;link_uid=PMC2885163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3838.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3650338/figure/F3/" ref="ordinalpos=3838&amp;ncbi_uid=6076303&amp;link_uid=PMC3650338" href="/pmc/articles/PMC3650338/"><img src="/pmc/articles/instance/3650338/bin/zjw0051344350003.gif" src-large="/pmc/articles/instance/3650338/bin/zjw0051344350003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3650338/" ref="ordinalpos=3838&amp;ncbi_uid=6076303&amp;link_uid=PMC3650338" image-link="/pmc/articles/PMC3650338/figure/F3/" class="imagepopup">Fig. 3. From: Platelet Proteome Analysis Reveals Integrin-dependent Aggregation Defects in Patients with Myelodysplastic Syndromes. </a></p><div class="supp"><p class="details"><div><b>Ingenuity network and <span class="highlight" style="background-color:">pathway</span> analysis.</b> <i>A</i>, ingenuity network analysis using all identified proteins from our two-dimensional difference gel electrophoresis analysis. <i>B</i>, ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of integrin <span class="highlight" style="background-color:">signaling</span>. Both show the up-regulated proteins in the MDS platelet in <i>red</i> and the down-regulated in <i>green</i>.</div></p></div><div class="aux"><div class="resc">Julia Fröbel, et al. Mol Cell Proteomics. 2013 May;12(5):1272-1280.</div><p class="links"><a class="dblinks" href="/pubmed/23382103" ref="ordinalpos=3838&amp;ncbi_uid=6076303&amp;link_uid=23382103">Citation</a><a class="dblinks" href="/pmc/articles/PMC3650338/" ref="ordinalpos=3838&amp;ncbi_uid=6076303&amp;link_uid=PMC3650338">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3839.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC25360/figure/F9/" ref="ordinalpos=3839&amp;ncbi_uid=180367&amp;link_uid=PMC25360" href="/pmc/articles/PMC25360/"><img src="/pmc/articles/instance/25360/bin/mk0690891009.gif" src-large="/pmc/articles/instance/25360/bin/mk0690891009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC25360/" ref="ordinalpos=3839&amp;ncbi_uid=180367&amp;link_uid=PMC25360" image-link="/pmc/articles/PMC25360/figure/F9/" class="imagepopup">Figure 9.  From: Transformation by v-Src: Ras-MAPK and PI3K-mTOR Mediate Parallel Pathways. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways required for transformation by v-Src. v-Src activates both the Ras-MAPK <span class="highlight" style="background-color:">pathway</span>, which can be inhibited by coexpression of N17Ras or by PD98059, and, in parallel, the PI3K-Akt-mTOR <span class="highlight" style="background-color:">pathway</span>, which can be inhibited by LY294002 or rapamycin. Inhibition of both pathways is required to block transformation. Our data do not exclude the possibility that Src, Ras, and PI3K activate other pathways that are also necessary for transformation.</div></p></div><div class="aux"><div class="resc">Elicia Penuel, et al. Mol Biol Cell. 1999 June;10(6):1693-1703.</div><p class="links"><a class="dblinks" href="/pubmed/10359590" ref="ordinalpos=3839&amp;ncbi_uid=180367&amp;link_uid=10359590">Citation</a><a class="dblinks" href="/pmc/articles/PMC25360/" ref="ordinalpos=3839&amp;ncbi_uid=180367&amp;link_uid=PMC25360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3840.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775648/figure/F3/" ref="ordinalpos=3840&amp;ncbi_uid=6449587&amp;link_uid=PMC3775648" href="/pmc/articles/PMC3775648/"><img src="/pmc/articles/instance/3775648/bin/nihms495103f3.gif" src-large="/pmc/articles/instance/3775648/bin/nihms495103f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775648/" ref="ordinalpos=3840&amp;ncbi_uid=6449587&amp;link_uid=PMC3775648" image-link="/pmc/articles/PMC3775648/figure/F3/" class="imagepopup">Figure 3.  From: Next Generation Sequencing Analysis of Gene Regulation in the Rat Model of Retinopathy of Prematurity. </a></p><div class="supp"><p class="details"><div>Major elements of the noncanonical Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Symbology as in figure 1.</div></p></div><div class="aux"><div class="resc">Rachel M. Griffith, et al. Doc Ophthalmol. 2013 August;127(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/23775346" ref="ordinalpos=3840&amp;ncbi_uid=6449587&amp;link_uid=23775346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775648/" ref="ordinalpos=3840&amp;ncbi_uid=6449587&amp;link_uid=PMC3775648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3841.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775648/figure/F4/" ref="ordinalpos=3841&amp;ncbi_uid=6449581&amp;link_uid=PMC3775648" href="/pmc/articles/PMC3775648/"><img src="/pmc/articles/instance/3775648/bin/nihms495103f4.gif" src-large="/pmc/articles/instance/3775648/bin/nihms495103f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775648/" ref="ordinalpos=3841&amp;ncbi_uid=6449581&amp;link_uid=PMC3775648" image-link="/pmc/articles/PMC3775648/figure/F4/" class="imagepopup">Figure 4.  From: Next Generation Sequencing Analysis of Gene Regulation in the Rat Model of Retinopathy of Prematurity. </a></p><div class="supp"><p class="details"><div>Major elements of the Rho <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Symbology as in figure 1.</div></p></div><div class="aux"><div class="resc">Rachel M. Griffith, et al. Doc Ophthalmol. 2013 August;127(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/23775346" ref="ordinalpos=3841&amp;ncbi_uid=6449581&amp;link_uid=23775346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775648/" ref="ordinalpos=3841&amp;ncbi_uid=6449581&amp;link_uid=PMC3775648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3842.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775648/figure/F5/" ref="ordinalpos=3842&amp;ncbi_uid=6449579&amp;link_uid=PMC3775648" href="/pmc/articles/PMC3775648/"><img src="/pmc/articles/instance/3775648/bin/nihms495103f5.gif" src-large="/pmc/articles/instance/3775648/bin/nihms495103f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775648/" ref="ordinalpos=3842&amp;ncbi_uid=6449579&amp;link_uid=PMC3775648" image-link="/pmc/articles/PMC3775648/figure/F5/" class="imagepopup">Figure 5.  From: Next Generation Sequencing Analysis of Gene Regulation in the Rat Model of Retinopathy of Prematurity. </a></p><div class="supp"><p class="details"><div>Major elements of the Cdc42 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Symbology as in figure 1.</div></p></div><div class="aux"><div class="resc">Rachel M. Griffith, et al. Doc Ophthalmol. 2013 August;127(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/23775346" ref="ordinalpos=3842&amp;ncbi_uid=6449579&amp;link_uid=23775346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775648/" ref="ordinalpos=3842&amp;ncbi_uid=6449579&amp;link_uid=PMC3775648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3843.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1599931/figure/f1/" ref="ordinalpos=3843&amp;ncbi_uid=865043&amp;link_uid=PMC1599931" href="/pmc/articles/PMC1599931/"><img src="/pmc/articles/instance/1599931/bin/zpq0340632620001.gif" src-large="/pmc/articles/instance/1599931/bin/zpq0340632620001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1599931/" ref="ordinalpos=3843&amp;ncbi_uid=865043&amp;link_uid=PMC1599931" image-link="/pmc/articles/PMC1599931/figure/f1/" class="imagepopup">Fig. 1. From: Human cytomegalovirus infection alters the substrate specificities and rapamycin sensitivities of raptor- and rictor-containing complexes. </a></p><div class="supp"><p class="details"><div>The phosphatidylinositide 3′-OH kinase/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activities of both the raptor (mTORC1) and rictor (mTORC2) complexes are shown.</div></p></div><div class="aux"><div class="resc">Sagar B. Kudchodkar, et al. Proc Natl Acad Sci U S A. 2006 September 19;103(38):14182-14187.</div><p class="links"><a class="dblinks" href="/pubmed/16959881" ref="ordinalpos=3843&amp;ncbi_uid=865043&amp;link_uid=16959881">Citation</a><a class="dblinks" href="/pmc/articles/PMC1599931/" ref="ordinalpos=3843&amp;ncbi_uid=865043&amp;link_uid=PMC1599931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3844.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2904107/figure/F12/" ref="ordinalpos=3844&amp;ncbi_uid=3053542&amp;link_uid=PMC2904107" href="/pmc/articles/PMC2904107/"><img src="/pmc/articles/instance/2904107/bin/zh30071056830012.gif" src-large="/pmc/articles/instance/2904107/bin/zh30071056830012.jpg" alt="Fig. 12." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2904107/" ref="ordinalpos=3844&amp;ncbi_uid=3053542&amp;link_uid=PMC2904107" image-link="/pmc/articles/PMC2904107/figure/F12/" class="imagepopup">Fig. 12. From: ?-Adrenergic activation of electrogenic K+ and Cl− secretion in guinea pig distal colonic epithelium proceeds via separate cAMP <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways for epi activation of secretion. The β<sub>1</sub>- and β<sub>2</sub>-adrenergic receptors (β<sub>1</sub>-AdrR, β<sub>2</sub>-AdrR) and Y2-neuropeptide receptor (Y2-NpR) are shown interacting with transmembrane adenylyl cyclase (tmAC) and sAC. Phosphodiesterases (PDEs) limit buildup of the cAMP, leading to activation of secretion. cAMP from other ACs in the cell also is shown leading to PKA activation as well as to Epac activation of Rap1 and Rap2. Activated Rap2 may enter the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for K<sup>+</sup> secretion, or possibly that for Cl<sup>−</sup> secretion. cAMP production via the sAC-dependent <span class="highlight" style="background-color:">pathway</span> would be maximally HCO<sub>3</sub><sup>−</sup> sensitive, since the physiological intracellular HCO<sub>3</sub><sup>−</sup> concentration is near the EC<sub>50</sub> for sAC (25, 48). Only with severe hypercapnia would CO<sub>2</sub> rise sufficiently to stimulate a large increase in cAMP production via the tmAC <span class="highlight" style="background-color:">pathway</span> (66).</div></p></div><div class="aux"><div class="resc">Susan T. Halm, et al. Am J Physiol Gastrointest Liver Physiol. 2010 July;299(1):G81-G95.</div><p class="links"><a class="dblinks" href="/pubmed/20413718" ref="ordinalpos=3844&amp;ncbi_uid=3053542&amp;link_uid=20413718">Citation</a><a class="dblinks" href="/pmc/articles/PMC2904107/" ref="ordinalpos=3844&amp;ncbi_uid=3053542&amp;link_uid=PMC2904107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3845.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316720/figure/F6/" ref="ordinalpos=3845&amp;ncbi_uid=403387&amp;link_uid=PMC316720" href="/pmc/articles/PMC316720/"><img src="/pmc/articles/instance/316720/bin/gad.6f6.gif" src-large="/pmc/articles/instance/316720/bin/gad.6f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316720/" ref="ordinalpos=3845&amp;ncbi_uid=403387&amp;link_uid=PMC316720" image-link="/pmc/articles/PMC316720/figure/F6/" class="imagepopup">Figure 6.  From: Down-regulation of transcription factor CF2 by Drosophila Ras/MAP kinase <span class="highlight" style="background-color:">signaling</span> in oogenesis: cytoplasmic retention and degradation. </a></p><div class="supp"><p class="details"><div> A model for the Egfr <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in dorsal follicle cell fate determination. In mid-oogenesis, <i>grk</i> gene product is sequestered to the anterodorsal region of the oocyte, aided by gene functions such as <i>fs(1)K10, sqd, spir, capu, orb,</i> etc. Grk acts across the oocyte membrane and activates the Egfr on the neighboring follicle cells. See text for other details on the resulting <span class="highlight" style="background-color:">signaling</span> events.</div></p></div><div class="aux"><div class="resc">Ekaterina Yu. Mantrova, et al. Genes Dev. 1998 April 15;12(8):1166-1175.</div><p class="links"><a class="dblinks" href="/pubmed/9553046" ref="ordinalpos=3845&amp;ncbi_uid=403387&amp;link_uid=9553046">Citation</a><a class="dblinks" href="/pmc/articles/PMC316720/" ref="ordinalpos=3845&amp;ncbi_uid=403387&amp;link_uid=PMC316720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3846.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3295063/figure/f2-ad-2-6-449/" ref="ordinalpos=3846&amp;ncbi_uid=3843990&amp;link_uid=PMC3295063" href="/pmc/articles/PMC3295063/"><img src="/pmc/articles/instance/3295063/bin/AD-2-6-449-Figure3.gif" src-large="/pmc/articles/instance/3295063/bin/AD-2-6-449-Figure3.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3295063/" ref="ordinalpos=3846&amp;ncbi_uid=3843990&amp;link_uid=PMC3295063" image-link="/pmc/articles/PMC3295063/figure/f2-ad-2-6-449/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Signaling</span> via the IKK/NF-κB Classical <span class="highlight" style="background-color:">Pathway</span>.  From: NF-?B in Aging and Disease. </a></p><div class="supp"><p class="details"><div>The IKK complex (NEMO, IKK1 (IKKα), and IKK2 (IKKβ)) can be activated by numerous stimuli and via shared <span class="highlight" style="background-color:">signaling</span> components. Extracellular receptors bind to their ligands and signal via TRAF/RIP/NIK molecules leading to phosphorylation of IKK subunits, which subsequently phosphorylate IκBα and lead to its ubiquitination and proteosomal degredation. This then releases NF-κB into the nucleus where it acts as a transcription factor. In addition, ATM responds to DNA damage and can also activate the IKK complex. (Figure adapted from [8, 146, 147])</div></p></div><div class="aux"><div class="resc">Jeremy S. Tilstra, et al. Aging Dis. 2011 December;2(6):449-465.</div><p class="links"><a class="dblinks" href="/pubmed/22396894" ref="ordinalpos=3846&amp;ncbi_uid=3843990&amp;link_uid=22396894">Citation</a><a class="dblinks" href="/pmc/articles/PMC3295063/" ref="ordinalpos=3846&amp;ncbi_uid=3843990&amp;link_uid=PMC3295063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3847.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4309199/figure/F2/" ref="ordinalpos=3847&amp;ncbi_uid=7167482&amp;link_uid=PMC4309199" href="/pmc/articles/PMC4309199/"><img src="/pmc/articles/instance/4309199/bin/fimmu-06-00014-g002.gif" src-large="/pmc/articles/instance/4309199/bin/fimmu-06-00014-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4309199/" ref="ordinalpos=3847&amp;ncbi_uid=7167482&amp;link_uid=PMC4309199" image-link="/pmc/articles/PMC4309199/figure/F2/" class="imagepopup">Figure 2.  From: Various Forms of Tissue Damage and Danger Signals Following Hematopoietic Stem-Cell Transplantation. </a></p><div class="supp"><p class="details"><div><b>Toll-like receptors and IL-1 receptor family <span class="highlight" style="background-color:">signaling</span> pathways</b>. ST2L (IL-33r) and IL-1r signal through the MyD88, IRAK4, and TRAF6 <span class="highlight" style="background-color:">pathway</span>. ST2L and IL-1r share this <span class="highlight" style="background-color:">pathway</span> with most TLRs. Binding of ST2L, IL-1r, and TLRs activates NF-κb, resulting in the release of inflammatory cytokines. Most TLRs signal through MyD88 expect for TLR3, which signals through the TRIF <span class="highlight" style="background-color:">pathway</span>. TLR4 can signal through both MyD88 and TRIF. TLR3, TLR7/TLR8, and TLR9 are expressed in the endosome while other TLRs are expressed on the cell surface. TLR1 and TLR6 recognize their ligand with TLR2 heterodimers.</div></p></div><div class="aux"><div class="resc">Abdulraouf Ramadan, et al. Front Immunol. 2015;6:14.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3847&amp;ncbi_uid=7167482&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4309199/" ref="ordinalpos=3847&amp;ncbi_uid=7167482&amp;link_uid=PMC4309199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3848.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204250/figure/F2/" ref="ordinalpos=3848&amp;ncbi_uid=3313892&amp;link_uid=PMC3204250" href="/pmc/articles/PMC3204250/"><img src="/pmc/articles/instance/3204250/bin/1756-0500-4-408-2.gif" src-large="/pmc/articles/instance/3204250/bin/1756-0500-4-408-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204250/" ref="ordinalpos=3848&amp;ncbi_uid=3313892&amp;link_uid=PMC3204250" image-link="/pmc/articles/PMC3204250/figure/F2/" class="imagepopup">Figure 2.  From: A comprehensive curated resource for follicle stimulating hormone <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>FSH <span class="highlight" style="background-color:">pathway</span> reaction map generated using CellDesigner</b>. The reactions in the <span class="highlight" style="background-color:">pathway</span> resource were used to obtain a descriptive network of reactions using CellDesigner. The legend for the <span class="highlight" style="background-color:">pathway</span> map is provided in the box to the right. The reactions are broadly arranged according to the cellular component they occur in and are depicted here on the plasma membrane (yellow thick line at the top) and the nucleus (the rectangle with the blue border at the bottom) and the cytosol (the space between the yellow line and the blue rectangle). The small molecules such as cyclic AMP (cAMP), calcium (Ca++), inositol 3-phosphate (IP3) and diacylglycerol (DAG) which are important mediators of FSH <span class="highlight" style="background-color:">signaling</span> are also represented in the map.</div></p></div><div class="aux"><div class="resc">Deepthi Telikicherla, et al. BMC Res Notes. 2011;4:408-408.</div><p class="links"><a class="dblinks" href="/pubmed/21996254" ref="ordinalpos=3848&amp;ncbi_uid=3313892&amp;link_uid=21996254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204250/" ref="ordinalpos=3848&amp;ncbi_uid=3313892&amp;link_uid=PMC3204250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3849.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3988951/figure/fig6/" ref="ordinalpos=3849&amp;ncbi_uid=6030156&amp;link_uid=PMC3988951" href="/pmc/articles/PMC3988951/"><img src="/pmc/articles/instance/3988951/bin/gr6.gif" src-large="/pmc/articles/instance/3988951/bin/gr6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3988951/" ref="ordinalpos=3849&amp;ncbi_uid=6030156&amp;link_uid=PMC3988951" image-link="/pmc/articles/PMC3988951/figure/fig6/" class="imagepopup">Fig. 6.  From: Insights into TREM2 biology by network analysis of human brain gene expression data. </a></p><div class="supp"><p class="details"><div>Insights into <i>TREM2</i>-related <span class="highlight" style="background-color:">signaling</span> using data-driven network constructions and information provided by Kyoto Encyclopedia of Genes and Genomes (KEGG) <span class="highlight" style="background-color:">pathway</span> analysis. (A) Network depiction of the <i>TREM2</i>-containing module in HIPP, showing all genes connected with a topological overlap measure (TOM) &gt;0.10. <i>TREM2</i> is highlighted in red, together with all of the genes that are directly connected to it, based on the TOM cut-off used. Genes that are featured in our proposed <i>TREM2</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are boxed in blue. (B) Network depiction of the <i>TREM2</i>-containing module in CRBL, showing all genes connected with a TOM &gt;0.10. As above, <i>TREM2</i> is highlighted in red, together with the genes that are directly connected to it, based on the TOM cut-off used. Genes that are featured in our proposed <i>TREM2</i> signalling <span class="highlight" style="background-color:">pathway</span> are boxed in blue. (C) Proposed <i>TREM2</i>-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in human brain adapted from the KEGG pathways for “Fc gamma R-mediated phagocytosis” and “Systemic lupus erythematosus.” Genes/complexes highlighted by data-driven network analyses are boxed in blue.</div></p></div><div class="aux"><div class="resc">Paola Forabosco, et al. Neurobiol Aging. 2013 December;34(12):2699-2714.</div><p class="links"><a class="dblinks" href="/pubmed/23855984" ref="ordinalpos=3849&amp;ncbi_uid=6030156&amp;link_uid=23855984">Citation</a><a class="dblinks" href="/pmc/articles/PMC3988951/" ref="ordinalpos=3849&amp;ncbi_uid=6030156&amp;link_uid=PMC3988951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3850.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3797300/figure/f4-etm-06-04-0899/" ref="ordinalpos=3850&amp;ncbi_uid=5393157&amp;link_uid=PMC3797300" href="/pmc/articles/PMC3797300/"><img src="/pmc/articles/instance/3797300/bin/ETM-06-04-0899-g03.gif" src-large="/pmc/articles/instance/3797300/bin/ETM-06-04-0899-g03.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3797300/" ref="ordinalpos=3850&amp;ncbi_uid=5393157&amp;link_uid=PMC3797300" image-link="/pmc/articles/PMC3797300/figure/f4-etm-06-04-0899/" class="imagepopup">Figure 4. From: Rosuvastatin suppresses platelet-derived growth factor-BB-induced vascular smooth muscle cell proliferation and migration via the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Rosuvastatin inhibited the mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by platelet-derived growth factor-BB (PDGF-BB) in vascular smooth muscle cells (VSMCs). Con, VSMCs were cultured without any treatment; NC, VSMCs were treated only with PDGF-BB (20 ng/ml) for 48 h; rosuvastatin, VSMCs were treated with rosuvastatin (10 <i>μ</i>M) and PDGF-BB (20 ng/ml) for 48 h. Western blot analysis was used to determine the protein expression of phospho-extracellular signal-regulated kinase 1/2 (ERK1/2), ERK, phospho-p38, p38, phospho-c-Jun N-terminal kinase (JNK) and JNK. Glyceraldehyde 3-phosphate dehydrogenase (GAPDH) was used as an internal reference.</div></p></div><div class="aux"><div class="resc">JIANTING GAN, et al. Exp Ther Med. 2013 October;6(4):899-903.</div><p class="links"><a class="dblinks" href="/pubmed/24137286" ref="ordinalpos=3850&amp;ncbi_uid=5393157&amp;link_uid=24137286">Citation</a><a class="dblinks" href="/pmc/articles/PMC3797300/" ref="ordinalpos=3850&amp;ncbi_uid=5393157&amp;link_uid=PMC3797300">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3851.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2568960/figure/ppat-1000175-g007/" ref="ordinalpos=3851&amp;ncbi_uid=1605923&amp;link_uid=PMC2568960" href="/pmc/articles/PMC2568960/"><img src="/pmc/articles/instance/2568960/bin/ppat.1000175.g007.gif" src-large="/pmc/articles/instance/2568960/bin/ppat.1000175.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2568960/" ref="ordinalpos=3851&amp;ncbi_uid=1605923&amp;link_uid=PMC2568960" image-link="/pmc/articles/PMC2568960/figure/ppat-1000175-g007/" class="imagepopup">Figure 7. Model of P. aeruginosa activation of insulin-like <span class="highlight" style="background-color:">signaling</span>..  From: Pseudomonas aeruginosa Suppresses Host Immunity by Activating the DAF-2 Insulin-Like <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>PA14 infection causes an insulin agonist-mediated activation of DAF-2, which likely involves expression of <i>ins-7</i>. Upon DAF-2 activation, several serine-threonine kinases phorphorylate DAF-16, causing the transcription factor to be retained in the cytoplasm and inhibiting its transcriptional-regulatory activity. Consequently, DAF-2–regulated gene expression is affected, causing the downregulation of immune defense genes such as <i>lys-7</i> and <i>thn-2</i>. Germline <span class="highlight" style="background-color:">signaling</span> and heat shock regulate DAF-16 in parallel to the effect of PA14 and DAF-2 activity. DAF-2 and DAF-16 may function in a network with other <span class="highlight" style="background-color:">signaling</span> components to affect the expression of DAF-2-regulated genes including <i>spp-1</i>.</div></p></div><div class="aux"><div class="resc">Eric A. Evans, et al. PLoS Pathog. 2008 October;4(10):e1000175.</div><p class="links"><a class="dblinks" href="/pubmed/18927620" ref="ordinalpos=3851&amp;ncbi_uid=1605923&amp;link_uid=18927620">Citation</a><a class="dblinks" href="/pmc/articles/PMC2568960/" ref="ordinalpos=3851&amp;ncbi_uid=1605923&amp;link_uid=PMC2568960">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3852.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1356553/figure/fig10/" ref="ordinalpos=3852&amp;ncbi_uid=588975&amp;link_uid=PMC1356553" href="/pmc/articles/PMC1356553/"><img src="/pmc/articles/instance/1356553/bin/tpc1800477f10.gif" src-large="/pmc/articles/instance/1356553/bin/tpc1800477f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1356553/" ref="ordinalpos=3852&amp;ncbi_uid=588975&amp;link_uid=PMC1356553" image-link="/pmc/articles/PMC1356553/figure/fig10/" class="imagepopup">Figure 10. From: Arabidopsis TARGET OF RAPAMYCIN Interacts with RAPTOR, Which Regulates the Activity of S6 Kinase in Response to Osmotic Stress Signals. </a></p><div class="supp"><p class="details"><div>TOR Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Plants.<br />Data from this study support the existence of the plant TOR <span class="highlight" style="background-color:">pathway</span> modulating plant cell growth in response to stress. The basic skeleton of the <span class="highlight" style="background-color:">pathway</span>, as illustrated above, is conserved as in other eukaryotes. It remains to be answered whether the activity of plant TOR also requires the G-β-like (GBL) protein.</div></p></div><div class="aux"><div class="resc">Magdy M. Mahfouz, et al. Plant Cell. 2006 February;18(2):477-490.</div><p class="links"><a class="dblinks" href="/pubmed/16377759" ref="ordinalpos=3852&amp;ncbi_uid=588975&amp;link_uid=16377759">Citation</a><a class="dblinks" href="/pmc/articles/PMC1356553/" ref="ordinalpos=3852&amp;ncbi_uid=588975&amp;link_uid=PMC1356553">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3853.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC526469/figure/f6/" ref="ordinalpos=3853&amp;ncbi_uid=389141&amp;link_uid=PMC526469" href="/pmc/articles/PMC526469/"><img src="/pmc/articles/instance/526469/bin/7600454f6.gif" src-large="/pmc/articles/instance/526469/bin/7600454f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC526469/" ref="ordinalpos=3853&amp;ncbi_uid=389141&amp;link_uid=PMC526469" image-link="/pmc/articles/PMC526469/figure/f6/" class="imagepopup">Figure 6.  From: Small proline-rich protein 1A is a gp130 <span class="highlight" style="background-color:">pathway</span>- and stress-inducible cardioprotective protein. </a></p><div class="supp"><p class="details"><div>Working model for SPRR1A activation in the myocardium. In response to myocardial injuries, IL-6 cytokines are released and act on the surrounding cells (left panel). The binding of IL-6 cytokines (CT-1, LIF) to gp130/LIFR receptor complex activates STAT3, PI-3K, and MAPK <span class="highlight" style="background-color:">signaling</span> pathways (right panel). A MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates the transcription factors C/EBPβ and AP-1. In turn, they bind to SPRR1A promoter and induce SPRR1A gene transcription. SPRR1A confers a protective effect to the cardiomyocytes, possibly by stabilizing the actin cytoskeleton.</div></p></div><div class="aux"><div class="resc">Sylvain Pradervand, et al. EMBO J. 2004 November 10;23(22):4517-4525.</div><p class="links"><a class="dblinks" href="/pubmed/15510217" ref="ordinalpos=3853&amp;ncbi_uid=389141&amp;link_uid=15510217">Citation</a><a class="dblinks" href="/pmc/articles/PMC526469/" ref="ordinalpos=3853&amp;ncbi_uid=389141&amp;link_uid=PMC526469">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3854.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2097959/figure/F2/" ref="ordinalpos=3854&amp;ncbi_uid=1112144&amp;link_uid=PMC2097959" href="/pmc/articles/PMC2097959/"><img src="/pmc/articles/instance/2097959/bin/nihms33845f2.gif" src-large="/pmc/articles/instance/2097959/bin/nihms33845f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2097959/" ref="ordinalpos=3854&amp;ncbi_uid=1112144&amp;link_uid=PMC2097959" image-link="/pmc/articles/PMC2097959/figure/F2/" class="imagepopup">Figure 2. EGFR activation and downstream <span class="highlight" style="background-color:">signaling</span>.  From: INTEGRATING RADIOTHERAPY WITH EGFR ANTAGONISTS AND OTHER MOLECULAR THERAPEUTICS FOR THE TREATMENT OF HEAD AND NECK CANCER. </a></p><div class="supp"><p class="details"><div>Upper panel illustrated ligand-induced conformational change followed by dimerization progressing from Panels A to C (see Jorissen <i>et al.</i> for complete review.73). This process triggers downstream <span class="highlight" style="background-color:">signaling</span> through four major cytoplasmic pathways depicted schematically. Nuclear influx of EGFR initiates interaction with transcription factors and proteins participating in DNA double-strand break repair. Endosomal degradation or recycling regulates EGFR <span class="highlight" style="background-color:">signaling</span>. A mitochondrial <span class="highlight" style="background-color:">pathway</span> has recently been described and its functions is under investigation.</div></p></div><div class="aux"><div class="resc">Juliette Thariat, et al. Int J Radiat Oncol Biol Phys. ;69(4):974-984.</div><p class="links"><a class="dblinks" href="/pubmed/17967298" ref="ordinalpos=3854&amp;ncbi_uid=1112144&amp;link_uid=17967298">Citation</a><a class="dblinks" href="/pmc/articles/PMC2097959/" ref="ordinalpos=3854&amp;ncbi_uid=1112144&amp;link_uid=PMC2097959">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3855.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411626/figure/pone-0042266-g006/" ref="ordinalpos=3855&amp;ncbi_uid=4161741&amp;link_uid=PMC3411626" href="/pmc/articles/PMC3411626/"><img src="/pmc/articles/instance/3411626/bin/pone.0042266.g006.gif" src-large="/pmc/articles/instance/3411626/bin/pone.0042266.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411626/" ref="ordinalpos=3855&amp;ncbi_uid=4161741&amp;link_uid=PMC3411626" image-link="/pmc/articles/PMC3411626/figure/pone-0042266-g006/" class="imagepopup">Figure 6. JNK and NF-κB pathways mediate the production of IL-1β, IL-6 and TNF-α under induction of the rL-hemolysins for 24 h..  From: Leptospiral Hemolysins Induce Proinflammatory Cytokines through Toll-Like Receptor 2-and 4-Mediated JNK and NF-?B <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Bars show the mean ± SD of three independent experiments. E-LPS indicates the LPS of <i>E. coli</i> serotype O111:B4. PK-H indicates that the rL-hemolysins or E-LPS were pretreated with proteinase K digestion plus heat-inactivation while PMB indicates that the rL-hemolysins or E-LPS were pretreated with polymyxin B blockade, and were used to monitor possible contamination with <i>E. coli</i> LPS in the rL-hemolysin proteins. The concentration of each of the rL-hemolysins or E-LPS tesed was 1 µg. Control indicates the IL-1β, IL-6 and TNF-α levels in the human THP-1 and mouse J774A.1 macrophages before treatment with any rL-hemolysins or E-LPS. <sup>#</sup><i>P</i>&lt;0.05 <i>vs</i> IL-1β, IL-6 and TNF-α levels in the THP-1 or J774A.1 macrophages before treatment with any rL-hemolysins or E-LPS (control). *<i>P</i>&lt;0.05 <i>vs</i> IL-1β, IL-6 and TNF-α levels in the THP-1 or J774A.1 macrophages that had not been treated with different <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors (SN50 for the NF-κB <span class="highlight" style="background-color:">pathway</span>, SP600125 for the JNK <span class="highlight" style="background-color:">pathway</span>, and SB203580 for the p38MAPK <span class="highlight" style="background-color:">pathway</span>).</div></p></div><div class="aux"><div class="resc">Huan Wang, et al. PLoS One. 2012;7(8):e42266.</div><p class="links"><a class="dblinks" href="/pubmed/22870312" ref="ordinalpos=3855&amp;ncbi_uid=4161741&amp;link_uid=22870312">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411626/" ref="ordinalpos=3855&amp;ncbi_uid=4161741&amp;link_uid=PMC3411626">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3856.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229251/figure/F9/" ref="ordinalpos=3856&amp;ncbi_uid=6823719&amp;link_uid=PMC4229251" href="/pmc/articles/PMC4229251/"><img src="/pmc/articles/instance/4229251/bin/nihms525600f9.gif" src-large="/pmc/articles/instance/4229251/bin/nihms525600f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229251/" ref="ordinalpos=3856&amp;ncbi_uid=6823719&amp;link_uid=PMC4229251" image-link="/pmc/articles/PMC4229251/figure/F9/" class="imagepopup">Figure 9. Regulation of microtubule assembly in promoting axon branching and growth by CNP <span class="highlight" style="background-color:">signaling</span>.  From: CNP/cGMP <span class="highlight" style="background-color:">Signaling</span> Regulates Axon Branching and Growth by Modulating Microtubule Polymerization. </a></p><div class="supp"><p class="details"><div>A) Summary of the hypothesis and the three lines of evidence that support it. Here, activated CNP/cGMP <span class="highlight" style="background-color:">signaling</span> leads to decreased phosphorylation of CRMP2 (p-CRMP2), which activates microtubule assembly and axon branching/growth. The supporting evidence is based on correlative observations from biochemical, cell biological, and pharmacological studies. NOC: nocodazole.<br />B) The proposed molecular <span class="highlight" style="background-color:">pathway</span> connecting the extracellular cue CNP to dynamic microtubule polymerization. Here, CNP binds to its cell surface receptor Npr2, which catalyzes the production of cGMP. cGMP-activated PrkG1 can phosphorylate GSK3 (p-GSK3) and prevent it from phosphorylating CRMP2. Following that in A), decreased CRMP2 phosphorylation (p-CRMP2) then leads to stimulate microtubule polymerization that is needed for axon branching and growth.</div></p></div><div class="aux"><div class="resc">Caihong Xia, et al. Dev Neurobiol. ;73(9):673-687.</div><p class="links"><a class="dblinks" href="/pubmed/23420620" ref="ordinalpos=3856&amp;ncbi_uid=6823719&amp;link_uid=23420620">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229251/" ref="ordinalpos=3856&amp;ncbi_uid=6823719&amp;link_uid=PMC4229251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3857.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508405/figure/F1/" ref="ordinalpos=3857&amp;ncbi_uid=4430099&amp;link_uid=PMC3508405" href="/pmc/articles/PMC3508405/"><img src="/pmc/articles/instance/3508405/bin/fphys-03-00452-g0001.gif" src-large="/pmc/articles/instance/3508405/bin/fphys-03-00452-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508405/" ref="ordinalpos=3857&amp;ncbi_uid=4430099&amp;link_uid=PMC3508405" image-link="/pmc/articles/PMC3508405/figure/F1/" class="imagepopup">Figure 1.  From: The other side of cardiac Ca2+ <span class="highlight" style="background-color:">signaling</span>: transcriptional control. </a></p><div class="supp"><p class="details"><div><b>Scheme for the TRPC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hypertrophy.</b> Stimulation of Gq-protein coupled receptors (GPCR) and subsequent activation of PLC leads to IP<sub>3</sub> and DAG generation. DAG directly activates TRPCs and induces receptor-operated Ca<sup>2+</sup> entry (ROCE), causing membrane depolarization and secondarily activating LTCCs. IP<sub>3</sub> activates IP<sub>3</sub>Rs, which induce depletion of stores and activation of STIM. STIM1 provokes store-operated Ca<sup>2+</sup> entry (SOCE) through Orai and/or TRPCs. TRPC channels can also be activated by stretch (SAC). The increase in intracellular Ca<sup>2+</sup> following TRPC activation is involved in hypertrophy development via activation of the calcineurin-NFAT <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Alejandro Domínguez-Rodríguez, et al. Front Physiol. 2012;3:452.</div><p class="links"><a class="dblinks" href="/pubmed/23226134" ref="ordinalpos=3857&amp;ncbi_uid=4430099&amp;link_uid=23226134">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508405/" ref="ordinalpos=3857&amp;ncbi_uid=4430099&amp;link_uid=PMC3508405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3858.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2667359/figure/f6/" ref="ordinalpos=3858&amp;ncbi_uid=1835749&amp;link_uid=PMC2667359" href="/pmc/articles/PMC2667359/"><img src="/pmc/articles/instance/2667359/bin/zjw0030933380006.gif" src-large="/pmc/articles/instance/2667359/bin/zjw0030933380006.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2667359/" ref="ordinalpos=3858&amp;ncbi_uid=1835749&amp;link_uid=PMC2667359" image-link="/pmc/articles/PMC2667359/figure/f6/" class="imagepopup">F ig . 6. From: Identification of Replication-competent HSV-1 Cgal+ Strain <span class="highlight" style="background-color:">Signaling</span> Targets in Human Hepatoma Cells by Functional Organelle Proteomics. </a></p><div class="supp"><p class="details"><div><b>Impairment of <span class="highlight" style="background-color:">signaling</span> intermediates in HSV-infected Huh7 human hepatoma cells.</b> Levels and residue-specific phosphorylation of PP2A scaffold and catalytic subunits (<i>A</i>), Raf/MAP kinase <span class="highlight" style="background-color:">pathway</span> (<i>B</i>), and FAK/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (<i>C</i>) intermediates were determined by Western blot in HSV-Cgal<sup>+</sup>-infected Huh7 cells. Differential phosphorylation of intermediates was confirmed by normalization with the total amount of the corresponding protein. Equal protein loading was demonstrated using an antibody against β-actin. Experiments were performed in triplicate. Representative blots are shown.</div></p></div><div class="aux"><div class="resc">Enrique Santamaría, et al. Mol Cell Proteomics. 2009 April;8(4):805-815.</div><p class="links"><a class="dblinks" href="/pubmed/19098277" ref="ordinalpos=3858&amp;ncbi_uid=1835749&amp;link_uid=19098277">Citation</a><a class="dblinks" href="/pmc/articles/PMC2667359/" ref="ordinalpos=3858&amp;ncbi_uid=1835749&amp;link_uid=PMC2667359">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3859.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2720918/figure/fig6/" ref="ordinalpos=3859&amp;ncbi_uid=2043949&amp;link_uid=PMC2720918" href="/pmc/articles/PMC2720918/"><img src="/pmc/articles/instance/2720918/bin/JCS028621F6.gif" src-large="/pmc/articles/instance/2720918/bin/JCS028621F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2720918/" ref="ordinalpos=3859&amp;ncbi_uid=2043949&amp;link_uid=PMC2720918" image-link="/pmc/articles/PMC2720918/figure/fig6/" class="imagepopup">Fig. 6. From: Functional interactions between the ciliopathy-associated Meckel syndrome  1 (MKS1) protein and two novel MKS1-related (MKSR) proteins. </a></p><div class="supp"><p class="details"><div>MKS-1, MKSR-1 and MKSR-2 proteins appear to function upstream of DAF-2 and  DAF-16 in the insulin–IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to regulate lifespan. (A)  The lifespan of <i>mksr-1;mksr-2;daf-2</i> triple mutant animals is the same  as that of the long-lived <i>daf-2</i> mutants, supporting the notion that  B9-domain-containing proteins function upstream of the DAF-2  insulin–IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <sup>*</sup><i>P</i>&lt;0.0001  compared with <i>daf-2</i>. (B) The lifespan of all <i>mks/mksr</i> double  mutants is shortened when introduced into a <i>daf-16</i> mutant background,  suggesting that the MKS-1, MKSR-1 and MKSR-2 proteins function upstream of the  DAF-16 FOXO transcription factor in the insulin–IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  <sup>*</sup><i>P</i>&lt;0.0001 compared with <i>daf-16</i>. (C) The  <i>mks/mksr</i> double mutants have increased levels of DAF-16::GFP in the  nucleus, consistent with their increased lifespan compared with wild-type (N2)  animals and their corresponding single mutants (<i>mks-1</i> and  <i>mksr-1</i>). The <i>osm-5</i> ciliary mutant positive control is  impaired in DAF-2 <span class="highlight" style="background-color:">signaling</span> and has an increased level of nuclear DAF-16::GFP.  Animals were categorized as having mainly cytoplasmic DAF-16::GFP (white  bars), intermediate localization between the cytoplasm and nucleus (light gray  bars), or mainly nuclear localization (dark gray bars); examples of these  three localization patterns are shown on the right.  <sup>*</sup><i>P</i>&lt;0.05 compared with N2.</div></p></div><div class="aux"><div class="resc">Nathan J. Bialas, et al. J Cell Sci. 2009 March 1;122(5):611-624.</div><p class="links"><a class="dblinks" href="/pubmed/19208769" ref="ordinalpos=3859&amp;ncbi_uid=2043949&amp;link_uid=19208769">Citation</a><a class="dblinks" href="/pmc/articles/PMC2720918/" ref="ordinalpos=3859&amp;ncbi_uid=2043949&amp;link_uid=PMC2720918">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3860.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2234575/figure/F2/" ref="ordinalpos=3860&amp;ncbi_uid=1261946&amp;link_uid=PMC2234575" href="/pmc/articles/PMC2234575/"><img src="/pmc/articles/instance/2234575/bin/nihms-38868-f0002.gif" src-large="/pmc/articles/instance/2234575/bin/nihms-38868-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2234575/" ref="ordinalpos=3860&amp;ncbi_uid=1261946&amp;link_uid=PMC2234575" image-link="/pmc/articles/PMC2234575/figure/F2/" class="imagepopup">Figure 2. Common Mechanisms Promoting PAH May Rely upon the Intersecting BMPR-II and Serotoninergic Pathways.  From: Pathogenic Mechanisms of Pulmonary Arterial Hypertension. </a></p><div class="supp"><p class="details"><div>In the serotoninergic <span class="highlight" style="background-color:">pathway</span>, both genetic predisposition (i.e., platelet storage pool disease, perhaps <i>5-HT2B</i> mutations, or perhaps the <i>L</i>-allele variant of <i>5-HTT</i>) and specific exogenous stimuli (i.e., hypoxia, anorexigens) lead to vessel remodeling via increased serotonin levels, increased 5-HTT activity, and/or increased serotonin receptor <span class="highlight" style="background-color:">signaling</span>. In the BMPR-II <span class="highlight" style="background-color:">pathway</span>, genetic predisposition (such as heterozygous loss-of-function +/− <i>BMPR2</i> mutations) and, perhaps, exogenous stimuli lead to decreased receptor expression, alteration of downstream <span class="highlight" style="background-color:">signaling</span>, and resulting vessel remodeling. Additional pathogenic mechanisms may upregulate the angiopoietin-1/Tie2 receptor <span class="highlight" style="background-color:">pathway</span> with resulting repression of BMPR-1A function and alteration of BMPR-II <span class="highlight" style="background-color:">signaling</span>. Interestingly, serotonin can directly upregulate angiopoietin-1 and can modulate downstream Smad function, thereby influencing the BMP <span class="highlight" style="background-color:">signaling</span> cascade. This functional intersection has yet to be fully described, but may represent a common, unifying mechanism of pathogenesis.</div></p></div><div class="aux"><div class="resc">Stephen Y. Chan, et al. J Mol Cell Cardiol. ;44(1):14-30.</div><p class="links"><a class="dblinks" href="/pubmed/17950310" ref="ordinalpos=3860&amp;ncbi_uid=1261946&amp;link_uid=17950310">Citation</a><a class="dblinks" href="/pmc/articles/PMC2234575/" ref="ordinalpos=3860&amp;ncbi_uid=1261946&amp;link_uid=PMC2234575">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3861.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3647886/figure/F2/" ref="ordinalpos=3861&amp;ncbi_uid=5021034&amp;link_uid=PMC3647886" href="/pmc/articles/PMC3647886/"><img src="/pmc/articles/instance/3647886/bin/nihms454345f2.gif" src-large="/pmc/articles/instance/3647886/bin/nihms454345f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3647886/" ref="ordinalpos=3861&amp;ncbi_uid=5021034&amp;link_uid=PMC3647886" image-link="/pmc/articles/PMC3647886/figure/F2/" class="imagepopup">Figure 2.  From: Progranulin: An emerging target for FTLD therapies. </a></p><div class="supp"><p class="details"><div><b>A.</b> The C-terminal end of PGRN binds to the Beta-propeller region of SORT1. Upon binding, uptake of PGRN via SORT1 traffics PGRN to the lysosome. Endocytosed SORT1 is then recycled back to the cell membrane. <b>B.</b> PGRN binding to TNFR acts as an antagonist of TNFα. TNFα binding to TNFR activates pro-inflammatory <span class="highlight" style="background-color:">signaling</span> to induce inflammation. PGRN blocks this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> when bound to TNFR therefore inhibiting inflammation.</div></p></div><div class="aux"><div class="resc">Jennifer Gass, et al. Brain Res. 2012 June 26;1462:118-128.</div><p class="links"><a class="dblinks" href="/pubmed/22338605" ref="ordinalpos=3861&amp;ncbi_uid=5021034&amp;link_uid=22338605">Citation</a><a class="dblinks" href="/pmc/articles/PMC3647886/" ref="ordinalpos=3861&amp;ncbi_uid=5021034&amp;link_uid=PMC3647886">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3862.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137124/figure/F5/" ref="ordinalpos=3862&amp;ncbi_uid=3505223&amp;link_uid=PMC3137124" href="/pmc/articles/PMC3137124/"><img src="/pmc/articles/instance/3137124/bin/nihms306276f5.gif" src-large="/pmc/articles/instance/3137124/bin/nihms306276f5.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137124/" ref="ordinalpos=3862&amp;ncbi_uid=3505223&amp;link_uid=PMC3137124" image-link="/pmc/articles/PMC3137124/figure/F5/" class="imagepopup">FIG. 5.  From: Probing the local electronic and geometric properties of the heme iron center in a H-NOX domain. </a></p><div class="supp"><p class="details"><div>Insight into the molecular basis of signal initiation of H-NOX domains based on XANES spectroscopy. The synergism between local structural rearrangement and electronic environment change leads to the <span class="highlight" style="background-color:">signaling</span> initiation step of the NO/sGC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Zhou Dai, et al. J Inorg Biochem. ;105(6):784-792.</div><p class="links"><a class="dblinks" href="/pubmed/21497576" ref="ordinalpos=3862&amp;ncbi_uid=3505223&amp;link_uid=21497576">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137124/" ref="ordinalpos=3862&amp;ncbi_uid=3505223&amp;link_uid=PMC3137124">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3863.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2710557/figure/F1/" ref="ordinalpos=3863&amp;ncbi_uid=1894195&amp;link_uid=PMC2710557" href="/pmc/articles/PMC2710557/"><img src="/pmc/articles/instance/2710557/bin/psb0407_0625_fig001.gif" src-large="/pmc/articles/instance/2710557/bin/psb0407_0625_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2710557/" ref="ordinalpos=3863&amp;ncbi_uid=1894195&amp;link_uid=PMC2710557" image-link="/pmc/articles/PMC2710557/figure/F1/" class="imagepopup">Figure 1.  From: Lipid determinants of cell death. </a></p><div class="supp"><p class="details"><div>In vivo model system with which to study the effects of fatty acid elongation on cell death. (A) Phenotypes of wild type and <i>FDH:FAE1</i> before and after spraying with chemical inhibitors. (B) One possible <span class="highlight" style="background-color:">pathway</span>, leading to a lipoapoptosis-like cell death, involves PLA2 that is required for the acyl remodeling of phospholipids, e.g., with VLCFAs. This stimulates the production of a lysosphingophospholipid by a secretory lipase and cell death <span class="highlight" style="background-color:">signaling</span> through the canonical 7TMR-mediated <span class="highlight" style="background-color:">pathway</span>. The involvement of the G-protein <span class="highlight" style="background-color:">signaling</span> in the trichome death could now be tested by crossing knock-out Gα and Gβ mutants with the <i>FDH:FAE1</i> plants. The non-canonical 7TMR-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be probed, e.g., with chemical inhibitors of calmodulin or protein kinases.</div></p></div><div class="aux"><div class="resc">José J Reina-Pinto, et al. Plant Signal Behav. 2009 July;4(7):625-628.</div><p class="links"><a class="dblinks" href="/pubmed/19820339" ref="ordinalpos=3863&amp;ncbi_uid=1894195&amp;link_uid=19820339">Citation</a><a class="dblinks" href="/pmc/articles/PMC2710557/" ref="ordinalpos=3863&amp;ncbi_uid=1894195&amp;link_uid=PMC2710557">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3864.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3434281/figure/F6/" ref="ordinalpos=3864&amp;ncbi_uid=5320076&amp;link_uid=PMC3434281" href="/pmc/articles/PMC3434281/"><img src="/pmc/articles/instance/3434281/bin/nihms369481f6.gif" src-large="/pmc/articles/instance/3434281/bin/nihms369481f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3434281/" ref="ordinalpos=3864&amp;ncbi_uid=5320076&amp;link_uid=PMC3434281" image-link="/pmc/articles/PMC3434281/figure/F6/" class="imagepopup">Figure 6.  From: TGF-? and NF-?B signal <span class="highlight" style="background-color:">pathway</span> cross-talk is mediated through TAK1 and SMAD7 in a subset of head and neck cancers. </a></p><div class="supp"><p class="details"><div>Proposed model for attenuation of canonical TGF-β SMAD and activation of non-canonical TGF-β-NF-κB <span class="highlight" style="background-color:">signaling</span> pathways in HNSCC. TGF-β activates NF-κB <span class="highlight" style="background-color:">signaling</span> through a sequential regulation of TAK1 and IKK kinases leading to phosphorylation of IκBα, nuclear translocation and phosphorylation of NF-κB subunit p65 and activation of NF-κB downstream targets. Both TGF-β and NF-κB <span class="highlight" style="background-color:">signaling</span> induce expression of SMAD7, which in turn preferentially suppresses TGF-β induced canonical and NF-κB <span class="highlight" style="background-color:">signaling</span>, relative to TNF-α-induced NF-κB <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Christian Freudlsperger, et al. Oncogene. 2013 March 21;32(12):1549-1559.</div><p class="links"><a class="dblinks" href="/pubmed/22641218" ref="ordinalpos=3864&amp;ncbi_uid=5320076&amp;link_uid=22641218">Citation</a><a class="dblinks" href="/pmc/articles/PMC3434281/" ref="ordinalpos=3864&amp;ncbi_uid=5320076&amp;link_uid=PMC3434281">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3865.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3983720/figure/F8/" ref="ordinalpos=3865&amp;ncbi_uid=6019642&amp;link_uid=PMC3983720" href="/pmc/articles/PMC3983720/"><img src="/pmc/articles/instance/3983720/bin/nihms567949f8.gif" src-large="/pmc/articles/instance/3983720/bin/nihms567949f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3983720/" ref="ordinalpos=3865&amp;ncbi_uid=6019642&amp;link_uid=PMC3983720" image-link="/pmc/articles/PMC3983720/figure/F8/" class="imagepopup">Fig. 8.  From: BMP and TGF-? <span class="highlight" style="background-color:">pathway</span> mediators are critical upstream regulators of Wnt <span class="highlight" style="background-color:">signaling</span> during midbrain dopamine differentiation in human pluripotent stem cells. </a></p><div class="supp"><p class="details"><div>Diagram of putative mDA differentiation <span class="highlight" style="background-color:">pathway</span> based on this and other studies.</div></p></div><div class="aux"><div class="resc">Jingli Cai, et al. Dev Biol. 2013 April 1;376(1):62-73.</div><p class="links"><a class="dblinks" href="/pubmed/23352789" ref="ordinalpos=3865&amp;ncbi_uid=6019642&amp;link_uid=23352789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3983720/" ref="ordinalpos=3865&amp;ncbi_uid=6019642&amp;link_uid=PMC3983720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3866.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2630276/figure/F7/" ref="ordinalpos=3866&amp;ncbi_uid=1659281&amp;link_uid=PMC2630276" href="/pmc/articles/PMC2630276/"><img src="/pmc/articles/instance/2630276/bin/zh80020928920007.gif" src-large="/pmc/articles/instance/2630276/bin/zh80020928920007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2630276/" ref="ordinalpos=3866&amp;ncbi_uid=1659281&amp;link_uid=PMC2630276" image-link="/pmc/articles/PMC2630276/figure/F7/" class="imagepopup">Figure 7.  From: Maintenance of the BMP4-dependent stress erythropoiesis <span class="highlight" style="background-color:">pathway</span> in the murine spleen requires hedgehog <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Model for the role of Hedgehog in the recovery of the BMP4-dependent stress erythropoiesis <span class="highlight" style="background-color:">pathway</span> in the spleen</b>.</div></p></div><div class="aux"><div class="resc">John M. Perry, et al. Blood. 2009 January 22;113(4):911-918.</div><p class="links"><a class="dblinks" href="/pubmed/18927434" ref="ordinalpos=3866&amp;ncbi_uid=1659281&amp;link_uid=18927434">Citation</a><a class="dblinks" href="/pmc/articles/PMC2630276/" ref="ordinalpos=3866&amp;ncbi_uid=1659281&amp;link_uid=PMC2630276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3867.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3773983/figure/F2/" ref="ordinalpos=3867&amp;ncbi_uid=5308681&amp;link_uid=PMC3773983" href="/pmc/articles/PMC3773983/"><img src="/pmc/articles/instance/3773983/bin/nihms-484044-f0002.gif" src-large="/pmc/articles/instance/3773983/bin/nihms-484044-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3773983/" ref="ordinalpos=3867&amp;ncbi_uid=5308681&amp;link_uid=PMC3773983" image-link="/pmc/articles/PMC3773983/figure/F2/" class="imagepopup">Fig. 2.  From: Complexity in Biological <span class="highlight" style="background-color:">Signaling</span> Systems. </a></p><div class="supp"><p class="details"><div>Four major <span class="highlight" style="background-color:">signaling</span> pathways in the postsynaptic region of a neuron that combine to form a local <span class="highlight" style="background-color:">signaling</span> network. The major linear routes of signal flow are depicted by the thick arrows of four different colors: orange [phospholipase C (PLC) <span class="highlight" style="background-color:">pathway</span>], pink (Ras <span class="highlight" style="background-color:">pathway</span>), green (adenylyl cyclase <span class="highlight" style="background-color:">pathway</span>), and blue [Ca<sup>2+</sup>/calmodulin (CaM) <span class="highlight" style="background-color:">pathway</span>]. The interactions between different pathways are represented by black lines with arrow (representing activation) or a dot (representing inhibition). Although most major interactions in the network are shown, these connections are not meant to be all-inclusive; additional connections could exist. The three-colored background represents three different cell compartments: the plasma membrane (light yellow), cytosol (light blue), and nucleus (light green). Some of the <span class="highlight" style="background-color:">signaling</span> proteins that translocate between different compartments are shown in both compartments. Examples include MAP kinase, which when activated translocates from the cytoplasm to the nucleus to phosphorylate and activate transcription factors; and the transcription factor CREB, which upon phosphorylation by protein kinase A (PKA) translocates to the nucleus.</div></p></div><div class="aux"><div class="resc">Gezhi Weng, et al. Science. 1999 April 2;284(5411):92-96.</div><p class="links"><a class="dblinks" href="/pubmed/10102825" ref="ordinalpos=3867&amp;ncbi_uid=5308681&amp;link_uid=10102825">Citation</a><a class="dblinks" href="/pmc/articles/PMC3773983/" ref="ordinalpos=3867&amp;ncbi_uid=5308681&amp;link_uid=PMC3773983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3868.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3742228/figure/f2-ijms-14-13979/" ref="ordinalpos=3868&amp;ncbi_uid=5191445&amp;link_uid=PMC3742228" href="/pmc/articles/PMC3742228/"><img src="/pmc/articles/instance/3742228/bin/ijms-14-13979f2.gif" src-large="/pmc/articles/instance/3742228/bin/ijms-14-13979f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3742228/" ref="ordinalpos=3868&amp;ncbi_uid=5191445&amp;link_uid=PMC3742228" image-link="/pmc/articles/PMC3742228/figure/f2-ijms-14-13979/" class="imagepopup">Figure 2.  From: Hedgehog <span class="highlight" style="background-color:">Signaling</span> in Prostate Cancer and Its Therapeutic Implication. </a></p><div class="supp"><p class="details"><div>Schematic overview of Hedgehog <span class="highlight" style="background-color:">signaling</span> and rationale for combination therapy with (chemo)radiotherapy. Upon Sonic Hedgehog (Shh) ligand binding to its receptor Patched (Ptch1) 1, the repression of Smoothened (Smo) is relieved, resulting in the movement of Smo from the intracellular vesicles to the primary cilium. Smo becomes activated and promotes the activation of the Gli proteins (Gli1/2) that enter the nucleus and promote transcription of the target genes (canonical <span class="highlight" style="background-color:">pathway</span> activation). The Gli transcription factors can also become activated by means of non-canonical <span class="highlight" style="background-color:">pathway</span> activation due to significant crosstalk with other important pathways such as the PI3K-Akt, KRAS, PKC-δ and TGFβ pathways. The Hh <span class="highlight" style="background-color:">signaling</span> also has important interactions with Wnt <span class="highlight" style="background-color:">pathway</span> and P53. The response to radiation therapy is determined by the four R’s of radiobiology: repopulation, repair of sublethal DNA damage, redistribution and reoxygenation. Hh <span class="highlight" style="background-color:">signaling</span> can potentially interfere with all these processes and targeting Hh <span class="highlight" style="background-color:">signaling</span> could therefore increase radiosensitivity of tumor cells. Moreover, inhibition of Hh <span class="highlight" style="background-color:">signaling</span> could also improve the response to chemotherapy by targeting multidrug resistance and cancer stems cells in addition to its effects on tumor vasculature. Abbreviations: PC, primary cilia; MDR, multidrug resistance; CSCs, cancer stem cells.</div></p></div><div class="aux"><div class="resc">Annelies Gonnissen, et al. Int J Mol Sci. 2013 July;14(7):13979-14007.</div><p class="links"><a class="dblinks" href="/pubmed/23880852" ref="ordinalpos=3868&amp;ncbi_uid=5191445&amp;link_uid=23880852">Citation</a><a class="dblinks" href="/pmc/articles/PMC3742228/" ref="ordinalpos=3868&amp;ncbi_uid=5191445&amp;link_uid=PMC3742228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3869.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3516847/figure/F1/" ref="ordinalpos=3869&amp;ncbi_uid=4436703&amp;link_uid=PMC3516847" href="/pmc/articles/PMC3516847/"><img src="/pmc/articles/instance/3516847/bin/nihms419803f1.gif" src-large="/pmc/articles/instance/3516847/bin/nihms419803f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3516847/" ref="ordinalpos=3869&amp;ncbi_uid=4436703&amp;link_uid=PMC3516847" image-link="/pmc/articles/PMC3516847/figure/F1/" class="imagepopup">FIGURE 1.  From: Phosphatidylenthanolamine Binding Protein aka Raf Kinase Inhibitor Protein: A Brief History of Its Discovery and the Remarkable Diversity of Biological Functions. </a></p><div class="supp"><p class="details"><div>Roles of RKIP in intracellular <span class="highlight" style="background-color:">signaling</span>. (<b>A</b>) Downregulation of <span class="highlight" style="background-color:">signaling</span> mediated by inhibitory RKIP actions. (<b>B</b>) Activation of <span class="highlight" style="background-color:">signaling</span> elicited by removal of RKIP. Left: the ERK MAPK <span class="highlight" style="background-color:">pathway</span>. RKIP binds to activated RAF1 and inhibits its interaction with MEK, thus antagonizing MEK phosphorylation and activation. Phosphorylation of RKIP by protein kinase C (PKC) releases RKIP from RAF1. Center: the NF-κB <span class="highlight" style="background-color:">pathway</span>. RKIP interacts with the activating kinases of the NF-κB <span class="highlight" style="background-color:">pathway</span>: transforming growth factor β-activated kinase (TAK), IκB kinase-α (IKK-α), IκB kinase-β (IKK-β), and NF-κB-inducing kinase (NIK). Release of RKIP from TAK, IKKs, and NIK (by an unknown mechanism) results in the phosphorylation and degradation of IκB, which in turn releases NF-κB. Right: G protein–coupled receptor (GPCR) pathways. GPCR kinase-2 (GRK2) antagonizes the activity of GPCRs. Phosphorylation of RKIP by PKC activates it for binding to GRK2, resulting in the inhibition of GRK2 activity. This allows GPCRs to interact with their cognate G-proteins (G), thus promoting GPCR-mediated <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">John M. Sedivy. For Immunopathol Dis Therap. ;2(1):1-12.</div><p class="links"><a class="dblinks" href="/pubmed/23227430" ref="ordinalpos=3869&amp;ncbi_uid=4436703&amp;link_uid=23227430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3516847/" ref="ordinalpos=3869&amp;ncbi_uid=4436703&amp;link_uid=PMC3516847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3870.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4334899/figure/pone.0118168.g008/" ref="ordinalpos=3870&amp;ncbi_uid=7210846&amp;link_uid=PMC4334899" href="/pmc/articles/PMC4334899/"><img src="/pmc/articles/instance/4334899/bin/pone.0118168.g008.gif" src-large="/pmc/articles/instance/4334899/bin/pone.0118168.g008.jpg" alt="Fig 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4334899/" ref="ordinalpos=3870&amp;ncbi_uid=7210846&amp;link_uid=PMC4334899" image-link="/pmc/articles/PMC4334899/figure/pone.0118168.g008/" class="imagepopup">Fig 8. Both Notch <span class="highlight" style="background-color:">signaling</span> and proteasome <span class="highlight" style="background-color:">pathway</span> contributes to the osteogenesis of hUC-MSCs..  From: The Notch <span class="highlight" style="background-color:">Signaling</span> Regulates CD105 Expression, Osteogenic Differentiation and Immunomodulation of Human Umbilical Cord Mesenchymal Stem Cells. </a></p><div class="supp"><p class="details"><div>The semi-quantitative RT-PCR shows that the treatment with 2.5 μM GSI-I significantly reduces RGC32 transcription at day 1 and day 3 after osteogenic induction (A). However, whereas 5 nM Bortezomib alone reduces RGC32 transcription at day 3, not day 1, of osteogenic induction, and siNotch transfection alone slightly reduces RGC32 transcription at both day 1 and day 3, the combination of both siNotch1 transfection and Bortezomib treatment significantly reduces the transcription at both day 1 and day 3 after osteogenic induction (B).</div></p></div><div class="aux"><div class="resc">Tao Na, et al. PLoS One. 2015;10(2):e0118168.</div><p class="links"><a class="dblinks" href="/pubmed/25692676" ref="ordinalpos=3870&amp;ncbi_uid=7210846&amp;link_uid=25692676">Citation</a><a class="dblinks" href="/pmc/articles/PMC4334899/" ref="ordinalpos=3870&amp;ncbi_uid=7210846&amp;link_uid=PMC4334899">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3871.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4304700/figure/F3/" ref="ordinalpos=3871&amp;ncbi_uid=7095863&amp;link_uid=PMC4304700" href="/pmc/articles/PMC4304700/"><img src="/pmc/articles/instance/4304700/bin/nihms656114f3.gif" src-large="/pmc/articles/instance/4304700/bin/nihms656114f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4304700/" ref="ordinalpos=3871&amp;ncbi_uid=7095863&amp;link_uid=PMC4304700" image-link="/pmc/articles/PMC4304700/figure/F3/" class="imagepopup">Figure 3.  From: Cell <span class="highlight" style="background-color:">Signaling</span> Pathways in Vertebrate Lens Regeneration. Published in “New Perspectives in Regeneration” (E. Heber-Katz and D. L. Stocum, eds). </a></p><div class="supp"><p class="details"><div>Highly simplified summary diagrams of various <span class="highlight" style="background-color:">signaling</span> pathways known to play roles during vertebrate lens regeneration. Various pathways and <span class="highlight" style="background-color:">pathway</span> members are as labeled. Not all known elements are included here for the sake of simplicity. Specific activators and inhibitors used to manipulate these pathways are also included, and specific synthetic agents are enclosed in ovals. See text for further details.</div></p></div><div class="aux"><div class="resc">Jonathan J. Henry, et al. Curr Top Microbiol Immunol. ;367:75-98.</div><p class="links"><a class="dblinks" href="/pubmed/23224710" ref="ordinalpos=3871&amp;ncbi_uid=7095863&amp;link_uid=23224710">Citation</a><a class="dblinks" href="/pmc/articles/PMC4304700/" ref="ordinalpos=3871&amp;ncbi_uid=7095863&amp;link_uid=PMC4304700">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3872.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3956502/figure/F4/" ref="ordinalpos=3872&amp;ncbi_uid=5940797&amp;link_uid=PMC3956502" href="/pmc/articles/PMC3956502/"><img src="/pmc/articles/instance/3956502/bin/viru-5-270-g4.gif" src-large="/pmc/articles/instance/3956502/bin/viru-5-270-g4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3956502/" ref="ordinalpos=3872&amp;ncbi_uid=5940797&amp;link_uid=PMC3956502" image-link="/pmc/articles/PMC3956502/figure/F4/" class="imagepopup">Figure. From: Mechanisms underlying the inhibition of interferon <span class="highlight" style="background-color:">signaling</span> by viruses. </a></p><div class="supp"><p class="details"><div><b>Figure 4.</b> A summary of viral mechanisms that target the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. NS5 and V proteins prevent the activation of TYK2 and JAK1 respectively; V proteins are also involved in STAT sequestration. NS5 and SV5 degrade STAT proteins; C proteins prevent the formation of activated STAT dimers and HCV core proteins induce SOCS expression. ISRE, interferon stimulated response element; GAS, gamma interferon activation site.</div></p></div><div class="aux"><div class="resc">Anand S Devasthanam. Virulence. 2014 February 15;5(2):270-277.</div><p class="links"><a class="dblinks" href="/pubmed/24504013" ref="ordinalpos=3872&amp;ncbi_uid=5940797&amp;link_uid=24504013">Citation</a><a class="dblinks" href="/pmc/articles/PMC3956502/" ref="ordinalpos=3872&amp;ncbi_uid=5940797&amp;link_uid=PMC3956502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3873.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397482/figure/f3-ijms-13-06561/" ref="ordinalpos=3873&amp;ncbi_uid=4140514&amp;link_uid=PMC3397482" href="/pmc/articles/PMC3397482/"><img src="/pmc/articles/instance/3397482/bin/ijms-13-06561f3.gif" src-large="/pmc/articles/instance/3397482/bin/ijms-13-06561f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397482/" ref="ordinalpos=3873&amp;ncbi_uid=4140514&amp;link_uid=PMC3397482" image-link="/pmc/articles/PMC3397482/figure/f3-ijms-13-06561/" class="imagepopup">Figure 3.  From: State of the Art in Silico Tools for the Study of <span class="highlight" style="background-color:">Signaling</span> Pathways in Cancer. </a></p><div class="supp"><p class="details"><div>An illustration of the multi-scale visualization of biological interactions using VisANT’s metagraph capability. VisANT [54] is an integrative software platform for the visualization, mining, analysis and modeling of biological networks. This software extends the applications of GO for network visualization, analysis and inference. The image shows the biological networks between the molecules that belong to the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (1) VEGF, (2) VEGFR2, (3) COX2 and (4) p33.</div></p></div><div class="aux"><div class="resc">Vanessa Medina Villaamil, et al. Int J Mol Sci. 2012;13(6):6561-6581.</div><p class="links"><a class="dblinks" href="/pubmed/22837650" ref="ordinalpos=3873&amp;ncbi_uid=4140514&amp;link_uid=22837650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397482/" ref="ordinalpos=3873&amp;ncbi_uid=4140514&amp;link_uid=PMC3397482">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3874.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1142215/figure/F2/" ref="ordinalpos=3874&amp;ncbi_uid=488727&amp;link_uid=PMC1142215" href="/pmc/articles/PMC1142215/"><img src="/pmc/articles/instance/1142215/bin/ijbsv01p0080g02.gif" src-large="/pmc/articles/instance/1142215/bin/ijbsv01p0080g02.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1142215/" ref="ordinalpos=3874&amp;ncbi_uid=488727&amp;link_uid=PMC1142215" image-link="/pmc/articles/PMC1142215/figure/F2/" class="imagepopup">Figure 2.  From: Dissecting Nck/Dock <span class="highlight" style="background-color:">Signaling</span> Pathways in Drosophila Visual System. </a></p><div class="supp"><p class="details"><div>Models for the action of the Nck/Dock <span class="highlight" style="background-color:">signaling</span> pathways in R-cell growth cones. During R-cell axon guidance and growth-cone expansion, Pak is activated by the combined action of an InR-Dock-linked signal and a Trio-Rac-linked signal, thus allowing Pak to modulate cytoskeletal changes. Dock may also link the guidance signal to the actin cytoskeleton via the Sra-1-Kette-SCAR <span class="highlight" style="background-color:">pathway</span>. During R1-R6 targeting, a Dock-linked stop signal activates Msn, which in turn phosphorylates Bif. Bif then restructures the actin cytoskeleton.</div></p></div><div class="aux"><div class="resc">Yong Rao. Int J Biol Sci. 2005;1(2):80-86.</div><p class="links"><a class="dblinks" href="/pubmed/15951852" ref="ordinalpos=3874&amp;ncbi_uid=488727&amp;link_uid=15951852">Citation</a><a class="dblinks" href="/pmc/articles/PMC1142215/" ref="ordinalpos=3874&amp;ncbi_uid=488727&amp;link_uid=PMC1142215">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3875.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3103323/figure/F7/" ref="ordinalpos=3875&amp;ncbi_uid=3371282&amp;link_uid=PMC3103323" href="/pmc/articles/PMC3103323/"><img src="/pmc/articles/instance/3103323/bin/zbc0281165610007.gif" src-large="/pmc/articles/instance/3103323/bin/zbc0281165610007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3103323/" ref="ordinalpos=3875&amp;ncbi_uid=3371282&amp;link_uid=PMC3103323" image-link="/pmc/articles/PMC3103323/figure/F7/" class="imagepopup">FIGURE 7. From: cAMP Activates TRPC6 Channels via the Phosphatidylinositol 3-Kinase (PI3K)-Protein Kinase B (PKB)-Mitogen-activated Protein Kinase Kinase (MEK)-ERK1/2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic showing the involving cAMP-PI3K-PKB-MEK-ERK1/2-TRPC6 signal transduction <span class="highlight" style="background-color:">pathway</span>.</b> <i>PDEs</i>, phosphodiesterases; <i>8-pCPT-2′-O-Me-cAMP</i>, 8-(4-chlorophenylthio)-2′-<i>O</i>-Me-cAMP.</div></p></div><div class="aux"><div class="resc">Bing Shen, et al. J Biol Chem. 2011 June 3;286(22):19439-19445.</div><p class="links"><a class="dblinks" href="/pubmed/21487005" ref="ordinalpos=3875&amp;ncbi_uid=3371282&amp;link_uid=21487005">Citation</a><a class="dblinks" href="/pmc/articles/PMC3103323/" ref="ordinalpos=3875&amp;ncbi_uid=3371282&amp;link_uid=PMC3103323">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3876.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3753504/figure/Fig39/" ref="ordinalpos=3876&amp;ncbi_uid=5269584&amp;link_uid=PMC3753504" href="/pmc/articles/PMC3753504/"><img src="/pmc/articles/instance/3753504/bin/204_2013_1078_Fig39_HTML.gif" src-large="/pmc/articles/instance/3753504/bin/204_2013_1078_Fig39_HTML.jpg" alt="Fig. 39" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3753504/" ref="ordinalpos=3876&amp;ncbi_uid=5269584&amp;link_uid=PMC3753504" image-link="/pmc/articles/PMC3753504/figure/Fig39/" class="imagepopup">Fig. 39.  From: Recent advances in 2D and 3D in vitro systems using primary hepatocytes, alternative hepatocyte sources and non-parenchymal liver cells and their use in investigating mechanisms of hepatotoxicity, cell <span class="highlight" style="background-color:">signaling</span> and ADME. </a></p><div class="supp"><p class="details"><div>Activation of different death receptor <span class="highlight" style="background-color:">signaling</span> pathways depending on the culture conditions of hepatocytes. In the mouse liver and when kept in suspension right after isolation, hepatocytes undergo apoptosis in response to FasL in a manner dependent on the BH3-only protein Bid and mitochondrial outer membrane permeabilization (cytochrome c release) (so-called type II <span class="highlight" style="background-color:">signaling</span>). However, when plated on a stiff collagen monolayer or embedded into a soft collagen sandwich, the FasL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> switches to a more direct manner that bypasses mitochondria and cleaves and activates effector caspase-3 directly by death receptor-bound caspase-8 (type I <span class="highlight" style="background-color:">signaling</span>). Interestingly, TNFa <span class="highlight" style="background-color:">signaling</span> does not switch in collagen-cultured hepatocytes, as sensitization of FasL-induced apoptosis by a pretreatment with TNFa remains Bid/mitochondria (type II)-dependent</div></p></div><div class="aux"><div class="resc">Patricio Godoy, et al. Arch Toxicol. 2013;87(8):1315-1530.</div><p class="links"><a class="dblinks" href="/pubmed/23974980" ref="ordinalpos=3876&amp;ncbi_uid=5269584&amp;link_uid=23974980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3753504/" ref="ordinalpos=3876&amp;ncbi_uid=5269584&amp;link_uid=PMC3753504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3877.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3311240/figure/F11/" ref="ordinalpos=3877&amp;ncbi_uid=4709869&amp;link_uid=PMC3311240" href="/pmc/articles/PMC3311240/"><img src="/pmc/articles/instance/3311240/bin/zh00031268800011.gif" src-large="/pmc/articles/instance/3311240/bin/zh00031268800011.jpg" alt="Fig. 11." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3311240/" ref="ordinalpos=3877&amp;ncbi_uid=4709869&amp;link_uid=PMC3311240" image-link="/pmc/articles/PMC3311240/figure/F11/" class="imagepopup">Fig. 11. From: Laminin-?1 LG4-5 domain binding to dystroglycan mediates muscle cell survival, growth, and the AP-1 and NF-?B transcription factors but also has adverse effects. </a></p><div class="supp"><p class="details"><div>A model of muscle cell <span class="highlight" style="background-color:">signaling</span> in response to 1E3. Intact laminin structure is depicted. <i>Inset</i>: 1E3 LG4–5 region. 1E3 binds to the DGC via α-DG. Other components of the DGC are shown, along with associated <span class="highlight" style="background-color:">signaling</span> below. <i>Left</i>: Rac1/JNK/c-jun <span class="highlight" style="background-color:">pathway</span> previously characterized (31, 47), which shows stimulation by 1E3 at all concentrations. <i>Right</i>: PI3K/Akt/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (9, 45), which shows a biphasic response to 1E3. NF-κB has been closely associated with atrophy (4, 42) while Akt <span class="highlight" style="background-color:">signaling</span> is associated with resisting muscle atrophy (26). The p21 small G-protein CDC42 has been shown to be important to myofiber diameter and increases upon atrophy (5) but was not included in the studies here.</div></p></div><div class="aux"><div class="resc">Yan Wen Zhou, et al. Am J Physiol Cell Physiol. 2012 March 15;302(6):C902-C914.</div><p class="links"><a class="dblinks" href="/pubmed/22159078" ref="ordinalpos=3877&amp;ncbi_uid=4709869&amp;link_uid=22159078">Citation</a><a class="dblinks" href="/pmc/articles/PMC3311240/" ref="ordinalpos=3877&amp;ncbi_uid=4709869&amp;link_uid=PMC3311240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3878.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3830884/figure/fig4/" ref="ordinalpos=3878&amp;ncbi_uid=5527321&amp;link_uid=PMC3830884" href="/pmc/articles/PMC3830884/"><img src="/pmc/articles/instance/3830884/bin/BMRI2013-298920.004.gif" src-large="/pmc/articles/instance/3830884/bin/BMRI2013-298920.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3830884/" ref="ordinalpos=3878&amp;ncbi_uid=5527321&amp;link_uid=PMC3830884" image-link="/pmc/articles/PMC3830884/figure/fig4/" class="imagepopup">Figure 4.  From: Transcriptome Profile Analysis of Sugarcane Responses to Sporisorium scitaminea Infection Using Solexa Sequencing Technology. </a></p><div class="supp"><p class="details"><div>MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of sugarcane infected by <i>S. scitaminea</i>.</div></p></div><div class="aux"><div class="resc">Qibin Wu, et al. Biomed Res Int. 2013;2013:298920.</div><p class="links"><a class="dblinks" href="/pubmed/24288673" ref="ordinalpos=3878&amp;ncbi_uid=5527321&amp;link_uid=24288673">Citation</a><a class="dblinks" href="/pmc/articles/PMC3830884/" ref="ordinalpos=3878&amp;ncbi_uid=5527321&amp;link_uid=PMC3830884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3879.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241618/figure/F2/" ref="ordinalpos=3879&amp;ncbi_uid=3717457&amp;link_uid=PMC3241618" href="/pmc/articles/PMC3241618/"><img src="/pmc/articles/instance/3241618/bin/1471-2407-11-506-2.gif" src-large="/pmc/articles/instance/3241618/bin/1471-2407-11-506-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241618/" ref="ordinalpos=3879&amp;ncbi_uid=3717457&amp;link_uid=PMC3241618" image-link="/pmc/articles/PMC3241618/figure/F2/" class="imagepopup">Figure 2.  From: STAT3 activation in monocytes accelerates liver cancer progression. </a></p><div class="supp"><p class="details"><div><b>Monocytes promoted HCC cell proliferation in vitro via IL-6/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Wen-Yong Wu, et al. BMC Cancer. 2011;11:506-506.</div><p class="links"><a class="dblinks" href="/pubmed/22136659" ref="ordinalpos=3879&amp;ncbi_uid=3717457&amp;link_uid=22136659">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241618/" ref="ordinalpos=3879&amp;ncbi_uid=3717457&amp;link_uid=PMC3241618">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3880.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124798/figure/F1/" ref="ordinalpos=3880&amp;ncbi_uid=3077012&amp;link_uid=PMC3124798" href="/pmc/articles/PMC3124798/"><img src="/pmc/articles/instance/3124798/bin/97320630006176F1.gif" src-large="/pmc/articles/instance/3124798/bin/97320630006176F1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124798/" ref="ordinalpos=3880&amp;ncbi_uid=3077012&amp;link_uid=PMC3124798" image-link="/pmc/articles/PMC3124798/figure/F1/" class="imagepopup">Figure 1.  From: Identification of EPAC (Exchange Protein Activated by cAMP) bioinformatically as a potential signalling biomarker in Cardiovascular Disease (CVD) and its molecular docking by a lead molecule. </a></p><div class="supp"><p class="details"><div>Data curation protocol for cataloguing the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules</div></p></div><div class="aux"><div class="resc">Saranya Bala, et al. Bioinformation. 2011;6(5):176-178.</div><p class="links"><a class="dblinks" href="/pubmed/21738308" ref="ordinalpos=3880&amp;ncbi_uid=3077012&amp;link_uid=21738308">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124798/" ref="ordinalpos=3880&amp;ncbi_uid=3077012&amp;link_uid=PMC3124798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3881.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3022486/figure/F9/" ref="ordinalpos=3881&amp;ncbi_uid=2714799&amp;link_uid=PMC3022486" href="/pmc/articles/PMC3022486/"><img src="/pmc/articles/instance/3022486/bin/nihms261828f9.gif" src-large="/pmc/articles/instance/3022486/bin/nihms261828f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3022486/" ref="ordinalpos=3881&amp;ncbi_uid=2714799&amp;link_uid=PMC3022486" image-link="/pmc/articles/PMC3022486/figure/F9/" class="imagepopup">Fig. 9.  From: Alcohol-Induced Interactive Phosphorylation of Src and Toll-like Receptor Regulates the Secretion of Inflammatory Mediators by Human Astrocytes. </a></p><div class="supp"><p class="details"><div>Alcohol-induced TLR4 protein recruitment mediates the activation of Src kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Nicholas A. Floreani, et al. J Neuroimmune Pharmacol. ;5(4):533-545.</div><p class="links"><a class="dblinks" href="/pubmed/20379791" ref="ordinalpos=3881&amp;ncbi_uid=2714799&amp;link_uid=20379791">Citation</a><a class="dblinks" href="/pmc/articles/PMC3022486/" ref="ordinalpos=3881&amp;ncbi_uid=2714799&amp;link_uid=PMC3022486">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3882.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3899443/figure/F4/" ref="ordinalpos=3882&amp;ncbi_uid=5730466&amp;link_uid=PMC3899443" href="/pmc/articles/PMC3899443/"><img src="/pmc/articles/instance/3899443/bin/1479-5876-12-16-4.gif" src-large="/pmc/articles/instance/3899443/bin/1479-5876-12-16-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3899443/" ref="ordinalpos=3882&amp;ncbi_uid=5730466&amp;link_uid=PMC3899443" image-link="/pmc/articles/PMC3899443/figure/F4/" class="imagepopup">Figure 4.  From: The therapeutic target of estrogen receptor-alpha36 in estrogen-dependent tumors. </a></p><div class="supp"><p class="details"><div><b>Interactions between ER-α36 and EGFR in mitogenic <span class="highlight" style="background-color:">signaling</span>.</b> After treatment with estrogen, ER-α36 is dissociated from epidermal growth factor receptor (EGFR) and is associated with Src and Shc. Furthermore, E2 induces auto-phosphorylation of Src and Src-dependent phosphorylation of EGFR. EGFR activates signal transducer and activator of transcription 5(STAT5), which leads to transcription of target genes such as Cyclin D1<i>.</i> On the other hand, ER-α36 and EGFR complex mediated non-genomic <span class="highlight" style="background-color:">signaling</span> like MAPK/ERK <span class="highlight" style="background-color:">pathway</span> is also involved in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> contributing to cell proliferation.</div></p></div><div class="aux"><div class="resc">Yu Gu, et al. J Transl Med. 2014;12:16-16.</div><p class="links"><a class="dblinks" href="/pubmed/24447535" ref="ordinalpos=3882&amp;ncbi_uid=5730466&amp;link_uid=24447535">Citation</a><a class="dblinks" href="/pmc/articles/PMC3899443/" ref="ordinalpos=3882&amp;ncbi_uid=5730466&amp;link_uid=PMC3899443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3883.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC16561/figure/F6/" ref="ordinalpos=3883&amp;ncbi_uid=32947&amp;link_uid=PMC16561" href="/pmc/articles/PMC16561/"><img src="/pmc/articles/instance/16561/bin/pq1201750006.gif" src-large="/pmc/articles/instance/16561/bin/pq1201750006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC16561/" ref="ordinalpos=3883&amp;ncbi_uid=32947&amp;link_uid=PMC16561" image-link="/pmc/articles/PMC16561/figure/F6/" class="imagepopup">Figure 6.  From: Regulation of protein kinase C?I by two protein-tyrosine kinases, Btk and Syk. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the upstream protein-tyrosine kinases, Lyn/Syk/Btk, to cytokine gene expression through intermediate <span class="highlight" style="background-color:">signaling</span> proteins, PKCβI/JNK. For the sake of simplicity, potential contribution of PKCα and PKCβII to JNK activation is omitted here. <span class="highlight" style="background-color:">Signaling</span> proteins are depicted in bold letters and nonprotein messengers in plain letters. Direct signal transmission by catalytic function is indicated by solid lines and indirect transmission by dashed lines.</div></p></div><div class="aux"><div class="resc">Yuko Kawakami, et al. Proc Natl Acad Sci U S A. 2000 June 20;97(13):7423-7428.</div><p class="links"><a class="dblinks" href="/pubmed/10852954" ref="ordinalpos=3883&amp;ncbi_uid=32947&amp;link_uid=10852954">Citation</a><a class="dblinks" href="/pmc/articles/PMC16561/" ref="ordinalpos=3883&amp;ncbi_uid=32947&amp;link_uid=PMC16561">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3884.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3136902/figure/F1/" ref="ordinalpos=3884&amp;ncbi_uid=3090590&amp;link_uid=PMC3136902" href="/pmc/articles/PMC3136902/"><img src="/pmc/articles/instance/3136902/bin/self0201_0035_fig001.gif" src-large="/pmc/articles/instance/3136902/bin/self0201_0035_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3136902/" ref="ordinalpos=3884&amp;ncbi_uid=3090590&amp;link_uid=PMC3136902" image-link="/pmc/articles/PMC3136902/figure/F1/" class="imagepopup">Figure 1.  From: A role of IL-1R1 <span class="highlight" style="background-color:">signaling</span> in the differentiation of Th17 cells and the development of autoimmune diseases. </a></p><div class="supp"><p class="details"><div>IL-1RI <span class="highlight" style="background-color:">signaling</span>. Upon binding of IL-1 to IL-1RI, an IL-1RAcP is recruited to form a high affinity IL-1R1-IL-1RAcP heterodimeric receptor, which initiates the downstream <span class="highlight" style="background-color:">signaling</span> cascade. The trimeric complex rapidly assembles MyD88 and IRAK4 and forms a stable IL-1-induced first <span class="highlight" style="background-color:">signaling</span> module, which subsequently phosphorylates IRAK1 and IRAK2, and recruits TRAF6. Complexes of IRAK1, IRAK2 and TRAF6 dissociate from the initial receptor complex and promote TGFβ-activated protein kinase (TAK)1/TAK-binding protein (TAB)1 association, which is followed by the activation of NFκB, JNK and p38 MAPK pathways. Activation of the IKK complex by IL-1 promotes IκBα ubiquitination. The nuclear translocation of NFκB with the nuclear translocation of c-JUN, induced by the activation of JNK and p38 MAPK, modulates the gene expression of IL-6, TNFα, IL-1β, IFNα, IFNβ and TGFβ. Multiple negative regulators of this <span class="highlight" style="background-color:">pathway</span>, including inhibitory IL-RII, secreted soluble (s)IL-1RI and sIL-RII, regulatory IL-1R1a and SIGIRR provide a negative feedback control of this <span class="highlight" style="background-color:">pathway</span> and suppress excessive IL-1 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Yonggang Sha, et al. Self Nonself. 2011 Jan-Mar;2(1):35-42.</div><p class="links"><a class="dblinks" href="/pubmed/21776333" ref="ordinalpos=3884&amp;ncbi_uid=3090590&amp;link_uid=21776333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3136902/" ref="ordinalpos=3884&amp;ncbi_uid=3090590&amp;link_uid=PMC3136902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3885.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464112/figure/fig4/" ref="ordinalpos=3885&amp;ncbi_uid=4280900&amp;link_uid=PMC3464112" href="/pmc/articles/PMC3464112/"><img src="/pmc/articles/instance/3464112/bin/1197f4.gif" src-large="/pmc/articles/instance/3464112/bin/1197f4.jpg" alt="Figure 4 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464112/" ref="ordinalpos=3885&amp;ncbi_uid=4280900&amp;link_uid=PMC3464112" image-link="/pmc/articles/PMC3464112/figure/fig4/" class="imagepopup">Figure 4 .  From: PUF-8, a Pumilio Homolog, Inhibits the Proliferative Fate in the Caenorhabditis elegans Germline. </a></p><div class="supp"><p class="details"><div>Models of PUF-8 function in gamete development. (A) PUF-8 functions at multiple stages in the development of gametes. These functions include promoting stem cell maintenance (Ariz et al. 2009; Bachorik and Kimble 2005), inhibition of the proliferative fate in GSCs (this work), inhibition of the male sexual fate (Bachorik and Kimble 2005), and inhibition of dedifferentiation of male germ cells progressing through meiosis (Subramaniam and Seydoux 2003). (B and C) Models of PUF-8 function in regulating the balance between the proliferative fate and meiotic entry. Illustration of distal end of gonad showing the DTC (yellow), mitotic cells (green) and meiotic cells (red). (B) PUF-8 may inhibit the activity of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span>. GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> levels are thought to be high at the very distal end, and lower as cells progress proximally. In this model, PUF-8 contributes to the lowering of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> levels, allowing cells to enter meiotic prophase. The inhibition of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> may not be direct, but rather PUF-8 may inhibit the activity of a positive regulator of GLP-1/Notch <span class="highlight" style="background-color:">signaling</span>. (C) PUF-8 may inhibit the proliferative fate in a <span class="highlight" style="background-color:">pathway</span> that is parallel to the GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In both models (B and C) PUF-8 is shown to be active throughout the distal end of the gonad; however, it is possible that another factor or post-translational modification causes PUF-8 not to be active in some regions where it is expressed.</div></p></div><div class="aux"><div class="resc">Hilary Racher, et al. G3 (Bethesda). 2012 October;2(10):1197-1205.</div><p class="links"><a class="dblinks" href="/pubmed/23050230" ref="ordinalpos=3885&amp;ncbi_uid=4280900&amp;link_uid=23050230">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464112/" ref="ordinalpos=3885&amp;ncbi_uid=4280900&amp;link_uid=PMC3464112">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3886.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC55373/figure/F3/" ref="ordinalpos=3886&amp;ncbi_uid=94989&amp;link_uid=PMC55373" href="/pmc/articles/PMC55373/"><img src="/pmc/articles/instance/55373/bin/pq1111535003.gif" src-large="/pmc/articles/instance/55373/bin/pq1111535003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC55373/" ref="ordinalpos=3886&amp;ncbi_uid=94989&amp;link_uid=PMC55373" image-link="/pmc/articles/PMC55373/figure/F3/" class="imagepopup">Figure 3.  From: Folding and <span class="highlight" style="background-color:">signaling</span> share the same <span class="highlight" style="background-color:">pathway</span> in a photoreceptor. </a></p><div class="supp"><p class="details"><div>Integrating folding and <span class="highlight" style="background-color:">signaling</span> in PYP. The folding and photocycle transitions in PYP are depicted schematically. Note that pU<sup>cis</sup> is an off-<span class="highlight" style="background-color:">pathway</span> intermediate for the photocycle, whereas pB<sup>cis</sup> is an on-<span class="highlight" style="background-color:">pathway</span> intermediate for pU<sup>cis</sup> refolding.</div></p></div><div class="aux"><div class="resc">Byoung-Chul Lee, et al. Proc Natl Acad Sci U S A. 2001 July 31;98(16):9062-9067.</div><p class="links"><a class="dblinks" href="/pubmed/11470891" ref="ordinalpos=3886&amp;ncbi_uid=94989&amp;link_uid=11470891">Citation</a><a class="dblinks" href="/pmc/articles/PMC55373/" ref="ordinalpos=3886&amp;ncbi_uid=94989&amp;link_uid=PMC55373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3887.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3787041/figure/pone-0075556-g005/" ref="ordinalpos=3887&amp;ncbi_uid=5369708&amp;link_uid=PMC3787041" href="/pmc/articles/PMC3787041/"><img src="/pmc/articles/instance/3787041/bin/pone.0075556.g005.gif" src-large="/pmc/articles/instance/3787041/bin/pone.0075556.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3787041/" ref="ordinalpos=3887&amp;ncbi_uid=5369708&amp;link_uid=PMC3787041" image-link="/pmc/articles/PMC3787041/figure/pone-0075556-g005/" class="imagepopup">Figure 5. Network <span class="highlight" style="background-color:">pathway</span> analysis..  From: Live Attenuated Rev-Independent Nef?SIV Enhances Acquisition of Heterologous SIVsmE660 in Acutely Vaccinated Rhesus Macaques. </a></p><div class="supp"><p class="details"><div>Network <span class="highlight" style="background-color:">pathway</span> analyses were performed using the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis Network tool. Node colors show the average gene expression levels of RMs vaccinated with Rev-Ind Nef¯SIV relative to the average before vaccination (red, relative increase in expression; green, relative decrease in expression). CCR5, CCL3 and other <span class="highlight" style="background-color:">signaling</span> molecules were upregulated and were directly and indirectly involved with other immune system pathways. The numbers underneath the nodes are 1) log<sub>2</sub>-fold change, and 2) FDR-adjusted <i>P</i> values.</div></p></div><div class="aux"><div class="resc">Siddappa N. Byrareddy, et al. PLoS One. 2013;8(9):e75556.</div><p class="links"><a class="dblinks" href="/pubmed/24098702" ref="ordinalpos=3887&amp;ncbi_uid=5369708&amp;link_uid=24098702">Citation</a><a class="dblinks" href="/pmc/articles/PMC3787041/" ref="ordinalpos=3887&amp;ncbi_uid=5369708&amp;link_uid=PMC3787041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3888.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1143566/figure/F4/" ref="ordinalpos=3888&amp;ncbi_uid=494857&amp;link_uid=PMC1143566" href="/pmc/articles/PMC1143566/"><img src="/pmc/articles/instance/1143566/bin/bcr1021-4.gif" src-large="/pmc/articles/instance/1143566/bin/bcr1021-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1143566/" ref="ordinalpos=3888&amp;ncbi_uid=494857&amp;link_uid=PMC1143566" image-link="/pmc/articles/PMC1143566/figure/F4/" class="imagepopup">Figure 4.  From: Mammary stem cells, self-renewal pathways, and carcinogenesis. </a></p><div class="supp"><p class="details"><div>A schematic diagram for the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(a) </b>The canonical Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span>. Canonical Wnt <span class="highlight" style="background-color:">signaling</span> requires the Frizzled (Fz) and low-density lipoprotein receptor-related proteins 5 and 6 (LRP5/6) co-receptors to activate Dishevelled (Dsh). Then Dsh inhibits the activity of the β-catenin destruction complex (adenomatous polyposis coli (APC), axin, and glycogen synthase kinase-3 (GSK-3)), which phosphorylates β-catenin in the absence of the ligands. β-Catenin is stabilized and translocated to the nucleus, where it recruits transactivators to high mobility group (HMG)-box DNA-binding proteins of the lymphoid enhancer factor/T cell factor (LEF/TCF) family. <b>(b) </b>The noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> requires Frizzled receptors and the proteoglycan co-receptor Knypek. In this <span class="highlight" style="background-color:">pathway</span>, Dsh localizes to the cell membrane through its DEP domain. A main branch downstream of Dsh involves the small GTPases of the Rho family. Dsh activation of Rho requires the bridging molecule Daam1. Dsh can also stimulate calcium flux and the activation of the calcium-sensitive kinases protein kinase C (PKC) and calmodulin-dependent protein kinase II (CanKII). At the end, the activation of this <span class="highlight" style="background-color:">pathway</span> induces the complex and dynamic cellular response.</div></p></div><div class="aux"><div class="resc">Suling Liu, et al. Breast Cancer Res. 2005;7(3):86-95.</div><p class="links"><a class="dblinks" href="/pubmed/15987436" ref="ordinalpos=3888&amp;ncbi_uid=494857&amp;link_uid=15987436">Citation</a><a class="dblinks" href="/pmc/articles/PMC1143566/" ref="ordinalpos=3888&amp;ncbi_uid=494857&amp;link_uid=PMC1143566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3889.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3931292/figure/F6/" ref="ordinalpos=3889&amp;ncbi_uid=5835844&amp;link_uid=PMC3931292" href="/pmc/articles/PMC3931292/"><img src="/pmc/articles/instance/3931292/bin/1471-2407-14-89-6.gif" src-large="/pmc/articles/instance/3931292/bin/1471-2407-14-89-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3931292/" ref="ordinalpos=3889&amp;ncbi_uid=5835844&amp;link_uid=PMC3931292" image-link="/pmc/articles/PMC3931292/figure/F6/" class="imagepopup">Figure 6.  From: The effect of ROCK-1 activity change on the adhesive and invasive ability of Y79 retinoblastoma cells. </a></p><div class="supp"><p class="details"><div><b>Y-27632 or ROCK-1 siRNA inhibits LIMK2/cofilin <span class="highlight" style="background-color:">pathway</span>.</b> Effects of Y-27632 and ROCK siRNA on the ROCK/LIMK2/cofilin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Western blot analysis, Y-27632 or ROCK-1 siRNA could inhibit the levels of LIMK2 and cofilin phosphorylation in Y79 cells (n = 3).</div></p></div><div class="aux"><div class="resc">Jing Wang, et al. BMC Cancer. 2014;14:89-89.</div><p class="links"><a class="dblinks" href="/pubmed/24528629" ref="ordinalpos=3889&amp;ncbi_uid=5835844&amp;link_uid=24528629">Citation</a><a class="dblinks" href="/pmc/articles/PMC3931292/" ref="ordinalpos=3889&amp;ncbi_uid=5835844&amp;link_uid=PMC3931292">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3890.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3009957/figure/F9/" ref="ordinalpos=3890&amp;ncbi_uid=2648404&amp;link_uid=PMC3009957" href="/pmc/articles/PMC3009957/"><img src="/pmc/articles/instance/3009957/bin/1750-2187-5-20-9.gif" src-large="/pmc/articles/instance/3009957/bin/1750-2187-5-20-9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3009957/" ref="ordinalpos=3890&amp;ncbi_uid=2648404&amp;link_uid=PMC3009957" image-link="/pmc/articles/PMC3009957/figure/F9/" class="imagepopup">Figure 9.  From: Peroxisome proliferator-activated receptor gamma ligand-mediated apoptosis of hepatocellular carcinoma cells depends upon modulation of PI3Kinase <span class="highlight" style="background-color:">pathway</span> independent of Akt. </a></p><div class="supp"><p class="details"><div><b>Model representing the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of TRG-induced cellular effects in HCC cells </b>. Incubation of the HCC cells with TRG in serum-containing media leads to a decrease in the expression of cyclin D1 resulting in cell growth arrest. TRG-induced reduction of cyclin D1 was shown to involve inhibition of CREB <span class="highlight" style="background-color:">pathway</span> in our earlier studies [32]. However, incubation with TRG under these conditions shows no apoptosis and leads to an increase in Akt<sup>Ser473 </sup>and FoxO1<sup>Thr24</sup>/3a<sup>Thr32 </sup>phosphorylation involving PI3K and Pak pathways, which might lead to tumor progression. Inhibition of PI3K <span class="highlight" style="background-color:">pathway</span> but not Pak or Akt pathways sensitizes cells towards apoptosis. In addition, incubation with TRG in serum-deficient media antagonizes Akt<sup>Ser473 </sup>phosphorylation and leads to potent apoptosis.</div></p></div><div class="aux"><div class="resc">Prajna Mishra, et al. J Mol Signal. 2010;5:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/21144036" ref="ordinalpos=3890&amp;ncbi_uid=2648404&amp;link_uid=21144036">Citation</a><a class="dblinks" href="/pmc/articles/PMC3009957/" ref="ordinalpos=3890&amp;ncbi_uid=2648404&amp;link_uid=PMC3009957">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3891.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3810594/figure/F1/" ref="ordinalpos=3891&amp;ncbi_uid=5456117&amp;link_uid=PMC3810594" href="/pmc/articles/PMC3810594/"><img src="/pmc/articles/instance/3810594/bin/fendo-04-00161-g001.gif" src-large="/pmc/articles/instance/3810594/bin/fendo-04-00161-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3810594/" ref="ordinalpos=3891&amp;ncbi_uid=5456117&amp;link_uid=PMC3810594" image-link="/pmc/articles/PMC3810594/figure/F1/" class="imagepopup">Figure 1.  From: The Role of Tissue Renin-Angiotensin-Aldosterone System in the Development of Endothelial Dysfunction and Arterial Stiffness. </a></p><div class="supp"><p class="details"><div><b>Renin-angiotensin-aldosterone system and regulation of insulin <span class="highlight" style="background-color:">signaling</span> phosphorylation of docking protein insulin receptor substrate -1 (IRS-1) is the major converging point in insulin <span class="highlight" style="background-color:">signaling</span></b>. The phosphorylation of serine residues of IRS-1 by mammalian target of ribosomal p70 S6 kinase (S6K1) acts as a convergence point for the regulation of IRS-1 phosphorylation by nutrients, hormones, and cytokines. Activation of RAAS in endothelial cells and vascular smooth muscle cells leads to inhibition of insulin <span class="highlight" style="background-color:">signaling</span> though phosphorylation of serine residues of IRS-1. This results in impaired <span class="highlight" style="background-color:">signaling</span> though attenuation of phosphatidylinsositol-3 kinase (PI3- kinase)/protein kinase B (Akt) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> linked to metabolic insulin <span class="highlight" style="background-color:">signaling</span>. This leads to reduced production of nitric oxide and endothelial dysfunction and altered vascular smooth muscle function.</div></p></div><div class="aux"><div class="resc">Annayya R. Aroor, et al. Front Endocrinol (Lausanne). 2013;4:161.</div><p class="links"><a class="dblinks" href="/pubmed/24194732" ref="ordinalpos=3891&amp;ncbi_uid=5456117&amp;link_uid=24194732">Citation</a><a class="dblinks" href="/pmc/articles/PMC3810594/" ref="ordinalpos=3891&amp;ncbi_uid=5456117&amp;link_uid=PMC3810594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3892.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3760459/figure/F1/" ref="ordinalpos=3892&amp;ncbi_uid=5415864&amp;link_uid=PMC3760459" href="/pmc/articles/PMC3760459/"><img src="/pmc/articles/instance/3760459/bin/lic-0001-0247-g01.gif" src-large="/pmc/articles/instance/3760459/bin/lic-0001-0247-g01.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3760459/" ref="ordinalpos=3892&amp;ncbi_uid=5415864&amp;link_uid=PMC3760459" image-link="/pmc/articles/PMC3760459/figure/F1/" class="imagepopup">Fig. 1.  From: Current and Future Treatment Strategies for Patients with Advanced Hepatocellular Carcinoma: Role of mTOR Inhibition. </a></p><div class="supp"><p class="details"><div>The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Reproduced with permission from Al-Batran et al. [50].</div></p></div><div class="aux"><div class="resc">Richard S. Finn. Liver Cancer. 2012 November;1(3-4):247-256.</div><p class="links"><a class="dblinks" href="/pubmed/24159589" ref="ordinalpos=3892&amp;ncbi_uid=5415864&amp;link_uid=24159589">Citation</a><a class="dblinks" href="/pmc/articles/PMC3760459/" ref="ordinalpos=3892&amp;ncbi_uid=5415864&amp;link_uid=PMC3760459">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3893.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2544441/figure/f3/" ref="ordinalpos=3893&amp;ncbi_uid=1480448&amp;link_uid=PMC2544441" href="/pmc/articles/PMC2544441/"><img src="/pmc/articles/instance/2544441/bin/zh00090857030003.gif" src-large="/pmc/articles/instance/2544441/bin/zh00090857030003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2544441/" ref="ordinalpos=3893&amp;ncbi_uid=1480448&amp;link_uid=PMC2544441" image-link="/pmc/articles/PMC2544441/figure/f3/" class="imagepopup">Fig. 3. From: Actin cytoskeletal dynamics in smooth muscle: a new paradigm for the regulation of smooth muscle contraction. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> for the regulation of actin polymerization in smooth muscle. ECM, extracellular matrix.</div></p></div><div class="aux"><div class="resc">Susan J. Gunst, et al. Am J Physiol Cell Physiol. 2008 September;295(3):C576-C587.</div><p class="links"><a class="dblinks" href="/pubmed/18596210" ref="ordinalpos=3893&amp;ncbi_uid=1480448&amp;link_uid=18596210">Citation</a><a class="dblinks" href="/pmc/articles/PMC2544441/" ref="ordinalpos=3893&amp;ncbi_uid=1480448&amp;link_uid=PMC2544441">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3894.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3529343/figure/f8-0060106/" ref="ordinalpos=3894&amp;ncbi_uid=4515540&amp;link_uid=PMC3529343" href="/pmc/articles/PMC3529343/"><img src="/pmc/articles/instance/3529343/bin/DMM008821F8.gif" src-large="/pmc/articles/instance/3529343/bin/DMM008821F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3529343/" ref="ordinalpos=3894&amp;ncbi_uid=4515540&amp;link_uid=PMC3529343" image-link="/pmc/articles/PMC3529343/figure/f8-0060106/" class="imagepopup">Fig. 8. From: Timing and expression of the angiopoietin-1-Tie-2 <span class="highlight" style="background-color:">pathway</span> in murine lung development and congenital diaphragmatic hernia. </a></p><div class="supp"><p class="details"><div><b>Model of the Ang-1 <span class="highlight" style="background-color:">pathway</span> in early lung development.</b> Proposed model of the Ang-1 <span class="highlight" style="background-color:">pathway</span> in early lung development (E12.5 to E15.5) hypothesizes that Ang-1 (red) secreted by the distal lung bud acts in a trophic fashion on progenitor vascular endothelial cells expressing receptor Tie-2 (green) in the mesenchyme to induce downstream <span class="highlight" style="background-color:">signaling</span> and stabilization of the nascent vasculature. In later development (E18.5 to postnatal), the relationship between Ang-1 and Tie-2 contributes to the stabilization of the primary blood vessels associated with the central airways in the maturing lungs. PN, post-natal.</div></p></div><div class="aux"><div class="resc">Adrienne Grzenda, et al. Dis Model Mech. 2013 January;6(1):106-114.</div><p class="links"><a class="dblinks" href="/pubmed/22917924" ref="ordinalpos=3894&amp;ncbi_uid=4515540&amp;link_uid=22917924">Citation</a><a class="dblinks" href="/pmc/articles/PMC3529343/" ref="ordinalpos=3894&amp;ncbi_uid=4515540&amp;link_uid=PMC3529343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3895.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2806747/figure/fig01/" ref="ordinalpos=3895&amp;ncbi_uid=2209446&amp;link_uid=PMC2806747" href="/pmc/articles/PMC2806747/"><img src="/pmc/articles/instance/2806747/bin/pnas.0912937107fig01.gif" src-large="/pmc/articles/instance/2806747/bin/pnas.0912937107fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2806747/" ref="ordinalpos=3895&amp;ncbi_uid=2209446&amp;link_uid=PMC2806747" image-link="/pmc/articles/PMC2806747/figure/fig01/" class="imagepopup">Fig. 1. From: Involvement of the Toll-like receptor 4 <span class="highlight" style="background-color:">pathway</span> and use of TNF-? antagonists for treatment of the mucopolysaccharidoses. </a></p><div class="supp"><p class="details"><div>(<i>A</i>) TLR4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be activated via GAG fragments requiring CD44 and MyD88 or through LPS, which requires MyD88, LBP, and CD14. (<i>B</i>) CD44 adhesion receptor (55 kDa) is a hyaluronan-binding cell surface glycoprotein involved in the initiation of GAG-induced inflammation via TLR4. Elevated CD44 expression was detected in 6-month-old MPS VI rat FLSs and articular chondrocytes, confirming the activation of TLR4 via the GAG <span class="highlight" style="background-color:">pathway</span> in MPS. Protein kinase C-α (PKC-α; 82 kDa) was used as a loading control.</div></p></div><div class="aux"><div class="resc">Calogera M. Simonaro, et al. Proc Natl Acad Sci U S A. 2010 January 5;107(1):222-227.</div><p class="links"><a class="dblinks" href="/pubmed/20018674" ref="ordinalpos=3895&amp;ncbi_uid=2209446&amp;link_uid=20018674">Citation</a><a class="dblinks" href="/pmc/articles/PMC2806747/" ref="ordinalpos=3895&amp;ncbi_uid=2209446&amp;link_uid=PMC2806747">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3896.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2738275/figure/f9/" ref="ordinalpos=3896&amp;ncbi_uid=1955376&amp;link_uid=PMC2738275" href="/pmc/articles/PMC2738275/"><img src="/pmc/articles/instance/2738275/bin/zmb0180982600009.gif" src-large="/pmc/articles/instance/2738275/bin/zmb0180982600009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2738275/" ref="ordinalpos=3896&amp;ncbi_uid=1955376&amp;link_uid=PMC2738275" image-link="/pmc/articles/PMC2738275/figure/f9/" class="imagepopup">FIG. 9. From: Time-Dependent Activation of Phox2a by the Cyclic AMP <span class="highlight" style="background-color:">Pathway</span> Modulates Onset and Duration of p27Kip1 Transcription . </a></p><div class="supp"><p class="details"><div>Mechanism of the time-dependent activation of Phox2a by cAMP <span class="highlight" style="background-color:">signaling</span>. Phosphorylation of Ser206 (the major phosphorylation site, depicted by filled circle), Ser202, and Ser208 indicates the inactive state of Phox2a. Activation of the cAMP <span class="highlight" style="background-color:">pathway</span> mediates the dephosphorylation of Ser206 via an OA-sensitive phosphatase. Active Phox2a binds to HD site 3 of the p27<sup>Kip1</sup> promoter and induces p27<sup>Kip1</sup> transcription. Upon complete dephosphorylation of Ser202, Ser206, and Ser208, Phox2a-mediated transcription is terminated via a PKA-mediated phosphorylation of Phox2a Ser153. This represents a novel mechanism of transcription factor activation by the cAMP <span class="highlight" style="background-color:">pathway</span>. The same signal determines the onset and duration of Phox2a-mediated transcription with a built-in time delay.</div></p></div><div class="aux"><div class="resc">Min Hwa Shin, et al. Mol Cell Biol. 2009 September;29(18):4878-4890.</div><p class="links"><a class="dblinks" href="/pubmed/19564421" ref="ordinalpos=3896&amp;ncbi_uid=1955376&amp;link_uid=19564421">Citation</a><a class="dblinks" href="/pmc/articles/PMC2738275/" ref="ordinalpos=3896&amp;ncbi_uid=1955376&amp;link_uid=PMC2738275">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3897.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316827/figure/F9/" ref="ordinalpos=3897&amp;ncbi_uid=406015&amp;link_uid=PMC316827" href="/pmc/articles/PMC316827/"><img src="/pmc/articles/instance/316827/bin/gad.9f9.gif" src-large="/pmc/articles/instance/316827/bin/gad.9f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316827/" ref="ordinalpos=3897&amp;ncbi_uid=406015&amp;link_uid=PMC316827" image-link="/pmc/articles/PMC316827/figure/F9/" class="imagepopup">Figure 9.  From: Nuclear hormone receptor antagonism with AP-1 by inhibition of the JNK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> Model for the antagonistic action of hormone-activated nuclear receptors on AP-1 activity through the inhibition of the JNK signal transduction <span class="highlight" style="background-color:">pathway</span>. Activation of gene transcription by either nuclear hormone receptors or AP-1 relays on an activation step: hormone (▴) binding for nuclear receptors and c-Jun phosphorylation on Ser-63/73 (▪) for AP-1. According to our data, upon hormone activation nuclear receptors block the induction of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and, in consequence, c-Jun amino-terminal phosphorylation. By this mechanism, steroid/thyroid hormones and RARs may antagonize AP-1 activity and inhibit expression of AP-1 target genes.</div></p></div><div class="aux"><div class="resc">Carme Caelles, et al. Genes Dev. 1997 December 15;11(24):3351-3364.</div><p class="links"><a class="dblinks" href="/pubmed/9407028" ref="ordinalpos=3897&amp;ncbi_uid=406015&amp;link_uid=9407028">Citation</a><a class="dblinks" href="/pmc/articles/PMC316827/" ref="ordinalpos=3897&amp;ncbi_uid=406015&amp;link_uid=PMC316827">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3898.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3437822/figure/fig04/" ref="ordinalpos=3898&amp;ncbi_uid=4678519&amp;link_uid=PMC3437822" href="/pmc/articles/PMC3437822/"><img src="/pmc/articles/instance/3437822/bin/pnas.1212547109fig04.gif" src-large="/pmc/articles/instance/3437822/bin/pnas.1212547109fig04.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3437822/" ref="ordinalpos=3898&amp;ncbi_uid=4678519&amp;link_uid=PMC3437822" image-link="/pmc/articles/PMC3437822/figure/fig04/" class="imagepopup">Fig. 4. From: FERONIA receptor kinase <span class="highlight" style="background-color:">pathway</span> suppresses abscisic acid <span class="highlight" style="background-color:">signaling</span> in Arabidopsis by activating ABI2 phosphatase. </a></p><div class="supp"><p class="details"><div>ABA response of <i>gef</i> mutants and a working model for the FER <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the regulation of ABA responses. (<i>A</i>) ABA-induced growth inhibition in roots. The WT, single mutants, <i>gef1/4</i> double mutant, and <i>gef1/4/10</i> triple mutant were grown on MS medium for 5 d before transfer to MS containing 12 μM ABA. Root length was then measured 3 d after transfer. Each value is the average ± SE of three replicates of 30 plants. Three separate experiments were conducted. (<i>B</i>) ABA-induced stomatal closure in WT and <i>gef</i> mutants. Leaves were treated with 5 μM ABA for 1 h before stomatal apertures were measured. Each value is the average ± SE of 3 independent experiments with 10 apertures each. (<i>C</i>) Working model for FER <span class="highlight" style="background-color:">pathway</span> in regulating response to ABA. The FER–GEF–ROP/ARAC <span class="highlight" style="background-color:">pathway</span> is an essential module for the auxin response in cell growth. ABI2 is a major component of ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The ROP11/ARAC10–ABI2 interaction mediates cross-talk between the auxin and ABA signals. Arrows denote activation, and bars indicate repression. Details are described in the text.</div></p></div><div class="aux"><div class="resc">Feng Yu, et al. Proc Natl Acad Sci U S A. 2012 September 4;109(36):14693-14698.</div><p class="links"><a class="dblinks" href="/pubmed/22908257" ref="ordinalpos=3898&amp;ncbi_uid=4678519&amp;link_uid=22908257">Citation</a><a class="dblinks" href="/pmc/articles/PMC3437822/" ref="ordinalpos=3898&amp;ncbi_uid=4678519&amp;link_uid=PMC3437822">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3899.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399969/figure/F5/" ref="ordinalpos=3899&amp;ncbi_uid=4474114&amp;link_uid=PMC3399969" href="/pmc/articles/PMC3399969/"><img src="/pmc/articles/instance/3399969/bin/nihms380694f5.gif" src-large="/pmc/articles/instance/3399969/bin/nihms380694f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399969/" ref="ordinalpos=3899&amp;ncbi_uid=4474114&amp;link_uid=PMC3399969" image-link="/pmc/articles/PMC3399969/figure/F5/" class="imagepopup">Figure 5. Regulation of mTORC1.  From: State-of-the-science: An update on renal cell carcinoma. </a></p><div class="supp"><p class="details"><div>HIF regulation and mTOR <span class="highlight" style="background-color:">pathway</span> connections. Hypoxia blocks HIF expression in a TSC1/2 and REDD dependent <span class="highlight" style="background-color:">pathway</span> [155]. HIF1α appears to be both TORC1 and TORC2 dependent, whereas HIF2α is only TORC2 dependent [275]. <span class="highlight" style="background-color:">Signaling</span> via TORC2 appears to upregulate HIF2α in an AKT dependent manner [69].</div></p></div><div class="aux"><div class="resc">Eric Jonasch, et al. Mol Cancer Res. ;10(7):859-880.</div><p class="links"><a class="dblinks" href="/pubmed/22638109" ref="ordinalpos=3899&amp;ncbi_uid=4474114&amp;link_uid=22638109">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399969/" ref="ordinalpos=3899&amp;ncbi_uid=4474114&amp;link_uid=PMC3399969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3900.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3253886/figure/F4/" ref="ordinalpos=3900&amp;ncbi_uid=4566784&amp;link_uid=PMC3253886" href="/pmc/articles/PMC3253886/"><img src="/pmc/articles/instance/3253886/bin/nihms330453f4.gif" src-large="/pmc/articles/instance/3253886/bin/nihms330453f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3253886/" ref="ordinalpos=3900&amp;ncbi_uid=4566784&amp;link_uid=PMC3253886" image-link="/pmc/articles/PMC3253886/figure/F4/" class="imagepopup">Figure 4.  From: A Quantitative Proteomic Approach for Detecting Protein Profiles of Activated Human Myeloid Dendritic Cells. </a></p><div class="supp"><p class="details"><div>IRF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that was generated from <span class="highlight" style="background-color:">pathway</span> analysis of proteins with significant fold changes across both studies. The proteins highlighted in red were detected in our analysis and had a significant increase in abundance upon stimulation.</div></p></div><div class="aux"><div class="resc">Daniela M Schlatzer, et al. J Immunol Methods. ;375(1-2):39-45.</div><p class="links"><a class="dblinks" href="/pubmed/21945394" ref="ordinalpos=3900&amp;ncbi_uid=4566784&amp;link_uid=21945394">Citation</a><a class="dblinks" href="/pmc/articles/PMC3253886/" ref="ordinalpos=3900&amp;ncbi_uid=4566784&amp;link_uid=PMC3253886">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="38" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="39" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="40" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:44:17-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB6B4354311F510000000000DCC0E4;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB6B4354311F510000000000DCC0E4" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>