
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="37" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A218F5431210100000000007BB942" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="36" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="37" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="38" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="37" /></div>
            <div><h2 class="result_count">Results: 3601 to 3700 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3601.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4407080/figure/F1/" ref="ordinalpos=3601&amp;ncbi_uid=7467660&amp;link_uid=PMC4407080" href="/pmc/articles/PMC4407080/"><img src="/pmc/articles/instance/4407080/bin/1752-0509-9-S2-S3-1.gif" src-large="/pmc/articles/instance/4407080/bin/1752-0509-9-S2-S3-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4407080/" ref="ordinalpos=3601&amp;ncbi_uid=7467660&amp;link_uid=PMC4407080" image-link="/pmc/articles/PMC4407080/figure/F1/" class="imagepopup">Figure 1.  From: KPP: KEGG <span class="highlight" style="background-color:">Pathway</span> Painter. </a></p><div class="supp"><p class="details"><div><b>Image of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> painted by KPP according to the imported list of genes differentially expressed in the prostatic carcinoma as compared to normal prostate</b>. Red and blue boxes represent up- and down- regulated genes, respectively. The genes in green background represent the species specific genes (<i>Homo sapiens</i>, in this case).</div></p></div><div class="aux"><div class="resc">Ganiraju Manyam, et al. BMC Syst Biol. 2015;9(Suppl 2):S3-S3.</div><p class="links"><a class="dblinks" href="/pubmed/25879163" ref="ordinalpos=3601&amp;ncbi_uid=7467660&amp;link_uid=25879163">Citation</a><a class="dblinks" href="/pmc/articles/PMC4407080/" ref="ordinalpos=3601&amp;ncbi_uid=7467660&amp;link_uid=PMC4407080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3602.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3958489/figure/pone-0092250-g005/" ref="ordinalpos=3602&amp;ncbi_uid=5941147&amp;link_uid=PMC3958489" href="/pmc/articles/PMC3958489/"><img src="/pmc/articles/instance/3958489/bin/pone.0092250.g005.gif" src-large="/pmc/articles/instance/3958489/bin/pone.0092250.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3958489/" ref="ordinalpos=3602&amp;ncbi_uid=5941147&amp;link_uid=PMC3958489" image-link="/pmc/articles/PMC3958489/figure/pone-0092250-g005/" class="imagepopup">Figure 5. The JNK <span class="highlight" style="background-color:">pathway</span> is activated in the usp5 mutant clone..  From: Drosophila USP5 Controls the Activation of Apoptosis and the Jun N-Terminal Kinase <span class="highlight" style="background-color:">Pathway</span> during Eye Development. </a></p><div class="supp"><p class="details"><div>A′ Eye imaginal discs from third instar larvae are shown. JNK phosphorylation was used to test the activation of JNK <span class="highlight" style="background-color:">signaling</span>; pJNK is increased in <i>usp5</i> mutant clones. B′ <i>puc</i> transcription level is labeled using the <i>puc</i>-<i>lacZ</i> reporter (β-GAL labeling); <i>puc</i> level is increased in the dorsal eye imaginal disc where USP5 is knocked-down as driven by <i>mirr</i>-<i>Gal4</i>.</div></p></div><div class="aux"><div class="resc">Xiaolan Fan, et al. PLoS One. 2014;9(3):e92250.</div><p class="links"><a class="dblinks" href="/pubmed/24643212" ref="ordinalpos=3602&amp;ncbi_uid=5941147&amp;link_uid=24643212">Citation</a><a class="dblinks" href="/pmc/articles/PMC3958489/" ref="ordinalpos=3602&amp;ncbi_uid=5941147&amp;link_uid=PMC3958489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3603.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3542123/figure/F5/" ref="ordinalpos=3603&amp;ncbi_uid=4526034&amp;link_uid=PMC3542123" href="/pmc/articles/PMC3542123/"><img src="/pmc/articles/instance/3542123/bin/1476-4598-11-64-5.gif" src-large="/pmc/articles/instance/3542123/bin/1476-4598-11-64-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3542123/" ref="ordinalpos=3603&amp;ncbi_uid=4526034&amp;link_uid=PMC3542123" image-link="/pmc/articles/PMC3542123/figure/F5/" class="imagepopup">Figure 5.  From: Cooperative interaction of MUC1 with the HGF/c-Met <span class="highlight" style="background-color:">pathway</span> during hepatocarcinogenesis. </a></p><div class="supp"><p class="details"><div><b>Activation of c-Met <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by HGF administration induced p42/44-MAPK and β-catenin phosphorylation.</b> Overnight starved Mahlavu (<b>A, B</b>) and SNU-449 (<b>C, D</b>) cells were stimulated with medium alone and with HGF for 15, 30, and 60 min. Total cell lysates were then analyzed with Western blotting. Blots were probed with anti-p-Met, anti-c-Met, anti-MUC1, p-p42/44-MAPK, p42/44-MAPK, anti-p-β-catenin, anti-β-catenin, anti-c-Myc, and anti-calnexin antibodies.</div></p></div><div class="aux"><div class="resc">Giray Bozkaya, et al. Mol Cancer. 2012;11:64-64.</div><p class="links"><a class="dblinks" href="/pubmed/22962849" ref="ordinalpos=3603&amp;ncbi_uid=4526034&amp;link_uid=22962849">Citation</a><a class="dblinks" href="/pmc/articles/PMC3542123/" ref="ordinalpos=3603&amp;ncbi_uid=4526034&amp;link_uid=PMC3542123">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3604.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2662295/figure/fig5/" ref="ordinalpos=3604&amp;ncbi_uid=1759031&amp;link_uid=PMC2662295" href="/pmc/articles/PMC2662295/"><img src="/pmc/articles/instance/2662295/bin/zbc0050962250005.gif" src-large="/pmc/articles/instance/2662295/bin/zbc0050962250005.jpg" alt="FIGURE 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2662295/" ref="ordinalpos=3604&amp;ncbi_uid=1759031&amp;link_uid=PMC2662295" image-link="/pmc/articles/PMC2662295/figure/fig5/" class="imagepopup">FIGURE 5. From: TRAF6 and MEKK1 Play a Pivotal Role in the RIG-I-like Helicase Antiviral  <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A model for TRAF6- and MEKK1-mediated signal for IFN-β production  in the RLH/IPS-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Upon recognition of intracellular  dsRNA by RIG-I or MDA5, RLHs recruit IPS-1 through CARD domains, which induces  interaction between IPS-1 and TRAF3, TRAF6, FADD, and RIP1. TRAF3 activates  TBK1/IKK-<i>i</i>, leading to the activation of IRF3, whereas TRAF6 may  interact with MEKK1, which in turn contributes to the activation of IKK and  MAPKK, subsequently activates NF-κB and AP-1, respectively. These  transcription factors cooperatively activate the IFN-β promoter.</div></p></div><div class="aux"><div class="resc">Ryoko Yoshida, et al. J Biol Chem. 2008 December 26;283(52):36211-36220.</div><p class="links"><a class="dblinks" href="/pubmed/18984593" ref="ordinalpos=3604&amp;ncbi_uid=1759031&amp;link_uid=18984593">Citation</a><a class="dblinks" href="/pmc/articles/PMC2662295/" ref="ordinalpos=3604&amp;ncbi_uid=1759031&amp;link_uid=PMC2662295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3605.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3881586/figure/fig2/" ref="ordinalpos=3605&amp;ncbi_uid=5721381&amp;link_uid=PMC3881586" href="/pmc/articles/PMC3881586/"><img src="/pmc/articles/instance/3881586/bin/ECAM2013-825814.002.gif" src-large="/pmc/articles/instance/3881586/bin/ECAM2013-825814.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3881586/" ref="ordinalpos=3605&amp;ncbi_uid=5721381&amp;link_uid=PMC3881586" image-link="/pmc/articles/PMC3881586/figure/fig2/" class="imagepopup">Figure 2.  From: Wen-pi-tang-Hab-Wu-ling-san, a Polyherbal Medicine, Attenuates ER Stress in 3T3-L1 Preadipocytes by Promoting the Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of WHW extract on insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 3T3-L1 cells. The cells were treated with WHW extract at concentrations of 500 and 1000 <i>μ</i>g/mL for 9 days. The expression of JNK, IRS-1, or PI3K was measured by western blot analysis. Relative density was calculated as the ratio of p-JNK expression to JNK, p-IRS-1 expression to IRS-1 expression, or PI3K expression to actin expression, respectively. Con: differentiated adipocytes as a control; Tu: 2 <i>μ</i>g/mL tunicamycin; 500: 2 <i>μ</i>g/mL tunicamycin + WHW 500 <i>μ</i>g/mL; 1000: 2 <i>μ</i>g/mL tunicamycin + WHW 1000 <i>μ</i>g/mL. These data are presented as the mean ± SD (<i>n</i> = 3). *Significantly different (<i>P</i> &lt; 0.05) from tunicamycin treatment.</div></p></div><div class="aux"><div class="resc">Yunkyung Han, et al. Evid Based Complement Alternat Med. 2013;2013:825814.</div><p class="links"><a class="dblinks" href="/pubmed/24454515" ref="ordinalpos=3605&amp;ncbi_uid=5721381&amp;link_uid=24454515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3881586/" ref="ordinalpos=3605&amp;ncbi_uid=5721381&amp;link_uid=PMC3881586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3606.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440011/figure/f1/" ref="ordinalpos=3606&amp;ncbi_uid=5249431&amp;link_uid=PMC3440011" href="/pmc/articles/PMC3440011/"><img src="/pmc/articles/instance/3440011/bin/fig-1.gif" src-large="/pmc/articles/instance/3440011/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440011/" ref="ordinalpos=3606&amp;ncbi_uid=5249431&amp;link_uid=PMC3440011" image-link="/pmc/articles/PMC3440011/figure/f1/" class="imagepopup">FIG. 1. From: Narratives in the Network: Interactive Methods for Mining Cell <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>Two pathways, Ga13 <span class="highlight" style="background-color:">pathway</span> and Gai <span class="highlight" style="background-color:">pathway</span>, are connected via Gaq <span class="highlight" style="background-color:">pathway</span>. The figure shows how our storytelling framework connects two consecutive pathways based on subgraph overlap. Note that Ga13 and Gai pathways do not have significant overlap between them, but both of them have significant subgraph overlap with the intermediate Gaq <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">M. Shahriar Hossain, et al. J Comput Biol. 2012 September;19(9):1043-1059.</div><p class="links"><a class="dblinks" href="/pubmed/22897227" ref="ordinalpos=3606&amp;ncbi_uid=5249431&amp;link_uid=22897227">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440011/" ref="ordinalpos=3606&amp;ncbi_uid=5249431&amp;link_uid=PMC3440011">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3607.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2042176/figure/F4/" ref="ordinalpos=3607&amp;ncbi_uid=1142102&amp;link_uid=PMC2042176" href="/pmc/articles/PMC2042176/"><img src="/pmc/articles/instance/2042176/bin/zpq0410779130004.gif" src-large="/pmc/articles/instance/2042176/bin/zpq0410779130004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2042176/" ref="ordinalpos=3607&amp;ncbi_uid=1142102&amp;link_uid=PMC2042176" image-link="/pmc/articles/PMC2042176/figure/F4/" class="imagepopup">Fig. 4. From: Kinetic insulation as an effective mechanism for achieving <span class="highlight" style="background-color:">pathway</span> specificity in intracellular <span class="highlight" style="background-color:">signaling</span> networks. </a></p><div class="supp"><p class="details"><div>A model for kinetic insulation. The upstream components of <span class="highlight" style="background-color:">pathway</span> A transform diverse SA input signals into a slowly varying output signal that activates kinase KA but not KB. The upstream components of <span class="highlight" style="background-color:">pathway</span> B transform diverse SB input signals into a transient output signal, causing the activation of kinase KB but not KA.</div></p></div><div class="aux"><div class="resc">Marcelo Behar, et al. Proc Natl Acad Sci U S A. 2007 October 9;104(41):16146-16151.</div><p class="links"><a class="dblinks" href="/pubmed/17913886" ref="ordinalpos=3607&amp;ncbi_uid=1142102&amp;link_uid=17913886">Citation</a><a class="dblinks" href="/pmc/articles/PMC2042176/" ref="ordinalpos=3607&amp;ncbi_uid=1142102&amp;link_uid=PMC2042176">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3608.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g003/" ref="ordinalpos=3608&amp;ncbi_uid=5168260&amp;link_uid=PMC3726753" href="/pmc/articles/PMC3726753/"><img src="/pmc/articles/instance/3726753/bin/pone.0068985.g003.gif" src-large="/pmc/articles/instance/3726753/bin/pone.0068985.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726753/" ref="ordinalpos=3608&amp;ncbi_uid=5168260&amp;link_uid=PMC3726753" image-link="/pmc/articles/PMC3726753/figure/pone-0068985-g003/" class="imagepopup">Figure 3. Morphoproteomic analysis of insulin-like growth factor(IGF) <span class="highlight" style="background-color:">pathway</span>..  From: Morphoproteomic Profiling of the Mammalian Target of Rapamycin (mTOR) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Desmoplastic Small Round Cell Tumor (EWS/WT1), Ewing's Sarcoma (EWS/FLI1) and Wilms' Tumor(WT1). </a></p><div class="supp"><p class="details"><div>Morphoproteomic analysis of insulin-like growth factor(IGF) <span class="highlight" style="background-color:">pathway</span> reveals constitutive activation of IGF-1receptor as evidenced by the expression of phosphorylated (p)-IGF-1R (Tyr1165/1166) on the plasmalemmal aspect and in cytoplasmic compartments of the tumor cells in DSCRT (Patient #1). Note overexpression vis-à-vis the tumoral stroma (original magnification ×400).</div></p></div><div class="aux"><div class="resc">Vivek Subbiah, et al. PLoS One. 2013;8(7):e68985.</div><p class="links"><a class="dblinks" href="/pubmed/23922674" ref="ordinalpos=3608&amp;ncbi_uid=5168260&amp;link_uid=23922674">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726753/" ref="ordinalpos=3608&amp;ncbi_uid=5168260&amp;link_uid=PMC3726753">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3609.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355968/figure/F2/" ref="ordinalpos=3609&amp;ncbi_uid=4027587&amp;link_uid=PMC3355968" href="/pmc/articles/PMC3355968/"><img src="/pmc/articles/instance/3355968/bin/fonc-02-00015-g002.gif" src-large="/pmc/articles/instance/3355968/bin/fonc-02-00015-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355968/" ref="ordinalpos=3609&amp;ncbi_uid=4027587&amp;link_uid=PMC3355968" image-link="/pmc/articles/PMC3355968/figure/F2/" class="imagepopup">Figure 2.  From: Regulation of Ubiquitination-Mediated Protein Degradation by Survival Kinases in Cancer. </a></p><div class="supp"><p class="details"><div><b>Regulation of Mcl-1 and Snail by AKT, ERK, and IKK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. <b>(A)</b> AKT and ERK phosphorylates GSK3β and inhibit it. GSK3β phosphorylates Mcl-1 and induces its ubiquitination and subsequent degradation via β-TrCP, Mule, and/or FBW7. ERK also directly phosphorylates Mcl-1, and phosphorylated Mcl-1 interacts with Pin1, resulting in its stabilization. The IKK–NF-κB <span class="highlight" style="background-color:">pathway</span> also contributes to Mcl-1 upregulation but the mechanisms are uncertain. <b>(B)</b> AKT and ERK phosphorylates GSK3β and inhibit it. GSK3β phosphorylates Snail and induces its nuclear exclusion, ubiquitination and subsequent degradation via β-TrCP. ERK also upregulates Snail transcription via AP-1 transcription factor. The IKK–NF-κB <span class="highlight" style="background-color:">pathway</span> inhibits Snail via upregulation of CSN2, which interferes with the GSK3–Snail interaction and ubiquitination of Snail.</div></p></div><div class="aux"><div class="resc">Hirohito Yamaguchi, et al. Front Oncol. 2012;2:15.</div><p class="links"><a class="dblinks" href="/pubmed/22649777" ref="ordinalpos=3609&amp;ncbi_uid=4027587&amp;link_uid=22649777">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355968/" ref="ordinalpos=3609&amp;ncbi_uid=4027587&amp;link_uid=PMC3355968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3610.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1681468/figure/f5/" ref="ordinalpos=3610&amp;ncbi_uid=809193&amp;link_uid=PMC1681468" href="/pmc/articles/PMC1681468/"><img src="/pmc/articles/instance/1681468/bin/msb4100014-f5.gif" src-large="/pmc/articles/instance/1681468/bin/msb4100014-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1681468/" ref="ordinalpos=3610&amp;ncbi_uid=809193&amp;link_uid=PMC1681468" image-link="/pmc/articles/PMC1681468/figure/f5/" class="imagepopup">Figure 5.  From: A comprehensive <span class="highlight" style="background-color:">pathway</span> map of epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Ellipsis in drawing association states of proteins using an 'address'. (<b>A</b>) Precise association states between EGFR and adaptors. Three adaptor proteins, Shc, Grb2, and Gab1, bind to the activated EGFR via its autophosphorylated tyrosine residues. Shc binds to activated EGFR and is phosphorylated on its tyrosine 317. Grb2 binds to activated EGFR either directly or via Shc bound to activated EGFR. Gab1 also binds to activated EGFR either directly or via Grb2 bound to activated EGFR, and is phosphorylated on its tyrosine 446, 472, and 589. (<b>B</b>) The same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as in panel A using an 'address' such as 'Grb2@EGFR.Y1068/1086P', thereby achieving a presentation of the <span class="highlight" style="background-color:">pathway</span> details. EGF, epidermal growth factor; EGFR, epidermal growth factor receptor; Gab, GRB2-associated binding protein; Grb, growth factor receptor-bound protein; Shc, Src homology 2 domain containing transforming protein.</div></p></div><div class="aux"><div class="resc">Kanae Oda, et al. Mol Syst Biol. 2005;1:2005.0010-2005.0010.</div><p class="links"><a class="dblinks" href="/pubmed/16729045" ref="ordinalpos=3610&amp;ncbi_uid=809193&amp;link_uid=16729045">Citation</a><a class="dblinks" href="/pmc/articles/PMC1681468/" ref="ordinalpos=3610&amp;ncbi_uid=809193&amp;link_uid=PMC1681468">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3611.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2925531/figure/f7/" ref="ordinalpos=3611&amp;ncbi_uid=3983848&amp;link_uid=PMC2925531" href="/pmc/articles/PMC2925531/"><img src="/pmc/articles/instance/2925531/bin/msb201052-f7.gif" src-large="/pmc/articles/instance/2925531/bin/msb201052-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2925531/" ref="ordinalpos=3611&amp;ncbi_uid=3983848&amp;link_uid=PMC2925531" image-link="/pmc/articles/PMC2925531/figure/f7/" class="imagepopup">Figure 7.  From: Kinase/phosphatase overexpression reveals pathways regulating hippocampal neuron morphology. </a></p><div class="supp"><p class="details"><div>Neural process development is affected by known pathways, including cancer pathways. <span class="highlight" style="background-color:">Pathway</span> annotations from the KEGG grouped genes into 79 pathways. (<b>A</b>) KEGG categories. (<b>B</b>) KEGG <span class="highlight" style="background-color:">pathway</span> names. (<b>C</b>) Heat map for <span class="highlight" style="background-color:">pathway</span>'s effect on neurite average length as indicated by color (red, decrease; white, no change from control; green, increase). (<b>D</b>) The 24 <span class="highlight" style="background-color:">pathway</span> subcategories, with heat map representing average effects of all genes in the category. (<b>E</b>) Names of <span class="highlight" style="background-color:">pathway</span> subcategories. (<b>F</b>) Legend for heat map. Symbols atop the heat map indicated significance from non-parametric statistics, where diamond is <i>P</i>⩽0.05 by spiked bootstrap, and square is <i>P</i>⩽0.05 by bootstrap. (<b>G</b>) After the analysis was repeated for four of the other parameters, the most significant pathways were compiled. They included three cancer pathways, the closely related ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and dorsal-ventral axis formation <span class="highlight" style="background-color:">pathway</span>. Source data are available for this figure at www.nature.com/msb.</div></p></div><div class="aux"><div class="resc">William J Buchser, et al. Mol Syst Biol. 2010;6:391-391.</div><p class="links"><a class="dblinks" href="/pubmed/20664637" ref="ordinalpos=3611&amp;ncbi_uid=3983848&amp;link_uid=20664637">Citation</a><a class="dblinks" href="/pmc/articles/PMC2925531/" ref="ordinalpos=3611&amp;ncbi_uid=3983848&amp;link_uid=PMC2925531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3612.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3266301/figure/pone-0030725-g006/" ref="ordinalpos=3612&amp;ncbi_uid=3782135&amp;link_uid=PMC3266301" href="/pmc/articles/PMC3266301/"><img src="/pmc/articles/instance/3266301/bin/pone.0030725.g006.gif" src-large="/pmc/articles/instance/3266301/bin/pone.0030725.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3266301/" ref="ordinalpos=3612&amp;ncbi_uid=3782135&amp;link_uid=PMC3266301" image-link="/pmc/articles/PMC3266301/figure/pone-0030725-g006/" class="imagepopup">Figure 6. p-ERK1/2 expression of in IBC tissues..  From: Alternate Estrogen Receptors Promote Invasion of Inflammatory Breast Cancer Cells via Non-Genomic <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) The expression of p-ERK1/2 was tested in tumor tissue samples from patients with IBC using the rabbit anti-p-ERK1/2 antibody. (B) Proposed working model for the stimulation of estrogen receptor ligand-induced non-genomic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via p-ERK1/2 activation in IBC cells. Estrogen receptor ligand rapidly activates p-ERK1/2 phosphorylation, leading to the activation of estrogen non-genomic <span class="highlight" style="background-color:">signaling</span> including cell migration and invasiveness in inflammatory breast cancer.</div></p></div><div class="aux"><div class="resc">Kazufumi Ohshiro, et al. PLoS One. 2012;7(1):e30725.</div><p class="links"><a class="dblinks" href="/pubmed/22295107" ref="ordinalpos=3612&amp;ncbi_uid=3782135&amp;link_uid=22295107">Citation</a><a class="dblinks" href="/pmc/articles/PMC3266301/" ref="ordinalpos=3612&amp;ncbi_uid=3782135&amp;link_uid=PMC3266301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3613.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3998841/figure/F2/" ref="ordinalpos=3613&amp;ncbi_uid=6055786&amp;link_uid=PMC3998841" href="/pmc/articles/PMC3998841/"><img src="/pmc/articles/instance/3998841/bin/nihms570576f2.gif" src-large="/pmc/articles/instance/3998841/bin/nihms570576f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3998841/" ref="ordinalpos=3613&amp;ncbi_uid=6055786&amp;link_uid=PMC3998841" image-link="/pmc/articles/PMC3998841/figure/F2/" class="imagepopup">Fig. 2.  From: A multipathway phosphoproteomic <span class="highlight" style="background-color:">signaling</span> network model of idiosyncratic drug- and inflammatory cytokine-induced toxicity in human hepatocytes. </a></p><div class="supp"><p class="details"><div>Inflammatory cytokine-associated idiosyncratic drug hepatotoxicity as a “network toxicity”. The multipathway modeling approach presented here suggests that an integration of multiple intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> -- namely the MEK–ERK, mTOR–p70 S6K, Akt, and p38–HSP27 pathways -- activities is necessary for hepatocytes to specify death responses to hepatotoxic drug/cytokine co-treatment conditions. This provides motivation of the network-level consideration of multiple survival, stress, and apoptosis <span class="highlight" style="background-color:">signaling</span> pathways in evaluating the hepatotoxicity mechanisms of context-dependent hepatotoxic drugs. Schematic reproduced from [10].</div></p></div><div class="aux"><div class="resc">Benjamin D. Cosgrove, et al. Conf Proc IEEE Eng Med Biol Soc. 2009;2009:5452-5455.</div><p class="links"><a class="dblinks" href="/pubmed/19964679" ref="ordinalpos=3613&amp;ncbi_uid=6055786&amp;link_uid=19964679">Citation</a><a class="dblinks" href="/pmc/articles/PMC3998841/" ref="ordinalpos=3613&amp;ncbi_uid=6055786&amp;link_uid=PMC3998841">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3614.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3636053/figure/F3/" ref="ordinalpos=3614&amp;ncbi_uid=4832214&amp;link_uid=PMC3636053" href="/pmc/articles/PMC3636053/"><img src="/pmc/articles/instance/3636053/bin/1471-2164-14-35-3.gif" src-large="/pmc/articles/instance/3636053/bin/1471-2164-14-35-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3636053/" ref="ordinalpos=3614&amp;ncbi_uid=4832214&amp;link_uid=PMC3636053" image-link="/pmc/articles/PMC3636053/figure/F3/" class="imagepopup">Figure 3.  From: Enhancing the utility of Proteomics Signature Profiling (PSP) with <span class="highlight" style="background-color:">Pathway</span> Derived Subnets (PDSs), performance analysis and specialised ontologies. </a></p><div class="supp"><p class="details"><div><b>False positive distribution for PSP (A) and PDS (B).</b> Co-localisation and expression profile of PDSs and complexes (DNA synthesome and TNF-alpha/NF-kappa B <span class="highlight" style="background-color:">signaling</span> complex 5) on the purine metabolism <span class="highlight" style="background-color:">pathway</span>. <b>A</b>: The purine metabolic <span class="highlight" style="background-color:">pathway</span> is shown as an undirected graph. Significant PSP clusters are highlighted in green while significant PDS is shown in purple. <b>B</b> and <b>C</b>: Regression plots for expression scores of protein complexes and PDSs co-located on the same biological <span class="highlight" style="background-color:">pathway</span> in mod and poor stage respectively. Enveloped in a checked circle are two complex outliers that are low expressing in the mod stage relative to the co-locating PDS but swung to overexpression in the poor stage</div></p></div><div class="aux"><div class="resc">Wilson Wen Bin Goh, et al. BMC Genomics. 2013;14:35-35.</div><p class="links"><a class="dblinks" href="/pubmed/23324392" ref="ordinalpos=3614&amp;ncbi_uid=4832214&amp;link_uid=23324392">Citation</a><a class="dblinks" href="/pmc/articles/PMC3636053/" ref="ordinalpos=3614&amp;ncbi_uid=4832214&amp;link_uid=PMC3636053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3615.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3497822/figure/fig07/" ref="ordinalpos=3615&amp;ncbi_uid=4836892&amp;link_uid=PMC3497822" href="/pmc/articles/PMC3497822/"><img src="/pmc/articles/instance/3497822/bin/pnas.1205661109fig07.gif" src-large="/pmc/articles/instance/3497822/bin/pnas.1205661109fig07.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3497822/" ref="ordinalpos=3615&amp;ncbi_uid=4836892&amp;link_uid=PMC3497822" image-link="/pmc/articles/PMC3497822/figure/fig07/" class="imagepopup">Fig. 7. From: PKA and GAB2 play central roles in the FSH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to PI3K and AKT in ovarian granulosa cells. </a></p><div class="supp"><p class="details"><div>Model of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which FSH promotes AKT activation in GCs. FSH binding to and activation of the FSH GPCR leads to activation of the stimulatory G protein (G<sub>s</sub>), activation of adenylyl cyclase (AC), generation of cAMP, and activation of the type I PKA holoenzyme (R<sup>I</sup><sub>2</sub>C<sub>2</sub>). PKA directly (unbroken line) or indirectly (broken line) promotes dephosphorylation (slash-P) of GAB2(Tyr<sup>452</sup>) and phosphorylation (P) of GAB2(Ser<sup>159</sup>) and IRS-1(Tyr<sup>989</sup>). Phosphorylation of IRS-1 on Tyr<sup>989</sup> provides the binding site for the p85 R-subunit of PI3K to allosterically activate the p110 catalytic subunit, resulting in phosphorylation-dependent activation of AKT(Thr<sup>308</sup>)(Ser<sup>473</sup>) that promotes GC proliferation and differentiation.</div></p></div><div class="aux"><div class="resc">Mary E. Hunzicker-Dunn, et al. Proc Natl Acad Sci U S A. 2012 October 30;109(44):E2979-E2988.</div><p class="links"><a class="dblinks" href="/pubmed/23045700" ref="ordinalpos=3615&amp;ncbi_uid=4836892&amp;link_uid=23045700">Citation</a><a class="dblinks" href="/pmc/articles/PMC3497822/" ref="ordinalpos=3615&amp;ncbi_uid=4836892&amp;link_uid=PMC3497822">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3616.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4048316/figure/pone-0099378-g006/" ref="ordinalpos=3616&amp;ncbi_uid=6232398&amp;link_uid=PMC4048316" href="/pmc/articles/PMC4048316/"><img src="/pmc/articles/instance/4048316/bin/pone.0099378.g006.gif" src-large="/pmc/articles/instance/4048316/bin/pone.0099378.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4048316/" ref="ordinalpos=3616&amp;ncbi_uid=6232398&amp;link_uid=PMC4048316" image-link="/pmc/articles/PMC4048316/figure/pone-0099378-g006/" class="imagepopup">Figure 6. The ATF4/p75NTR/IL-8 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was involved in the induction of EndoMT by SFO..  From: Finding ATF4/p75NTR/IL-8 Signal <span class="highlight" style="background-color:">Pathway</span> in Endothelial-Mesenchymal Transition by Safrole Oxide. </a></p><div class="supp"><p class="details"><div>(A and B) Cells were preincubated with ATF4 siRNA for 48 h, then 50 µg/ml SFO for 3 h for qPCR assay and 6 h for western blot assay. Effect of ATF4 siRNA on SFO-induced p75NTR expression in HUVECs was detected by qPCR assay (A) and western blot assay (B). Data below gels were used to quantify the density of the bands. (C to F) Cells were preincubated with p75NTR siRNA for 48 h, then 50 µg/ml of SFO for 3 h for qPCR assay and 6 h for western blot or ELISA assay. Effect of p75NTR siRNA on SFO-induced ATF4 expression in HUVECs was detected by qPCR assay (C) and western blot assay (D). Effect of p75NTR siRNA on SFO-induced IL-8 expression and release in HUVECs was detected by qPCR assay (E) and ELISA (F). G. A schematic of the <span class="highlight" style="background-color:">signaling</span> pathways of SFO in HUVEC transdifferentiation. (**p&lt;0.01; &amp; p&gt;0.05, n = 3).</div></p></div><div class="aux"><div class="resc">Di Ge, et al. PLoS One. 2014;9(6):e99378.</div><p class="links"><a class="dblinks" href="/pubmed/24905361" ref="ordinalpos=3616&amp;ncbi_uid=6232398&amp;link_uid=24905361">Citation</a><a class="dblinks" href="/pmc/articles/PMC4048316/" ref="ordinalpos=3616&amp;ncbi_uid=6232398&amp;link_uid=PMC4048316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3617.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3114396/figure/F3/" ref="ordinalpos=3617&amp;ncbi_uid=3375249&amp;link_uid=PMC3114396" href="/pmc/articles/PMC3114396/"><img src="/pmc/articles/instance/3114396/bin/1668fig3.gif" src-large="/pmc/articles/instance/3114396/bin/1668fig3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3114396/" ref="ordinalpos=3617&amp;ncbi_uid=3375249&amp;link_uid=PMC3114396" image-link="/pmc/articles/PMC3114396/figure/F3/" class="imagepopup">FIG. 3. From: Cross-Talk Between Interferon-? and Hedgehog <span class="highlight" style="background-color:">Signaling</span> Regulates Adipogenesis. </a></p><div class="supp"><p class="details"><div>IFN-γ prevents SAG-induced inhibition of adipocyte differentiation via Jak-Stat <span class="highlight" style="background-color:">signaling</span>. <i>A</i>: Shh-LIGHT2 cells were treated with 200 nmol/L SAG and/or 25 ng/µL IFN-γ in the absence or presence of JakInh. Simultaneous treatment of cells with SAG and JakInh resulted in an additional increase in relative Gli1 luciferase activity compared with SAG only–treated cells. <i>B–D</i>: We also tested whether effects of IFN-γ on <i>Gli1</i> expression could persist in MEFs from Ifngr1<sup>−/−</sup> and Stat1<sup>−/−</sup> mice. The inhibitory effect of IFN-γ on Shh <span class="highlight" style="background-color:">signaling</span> was cancelled in Ifngr1<sup>−/−</sup> and Stat1<sup>−/−</sup> MEFs. These experiments demonstrate that IFN-γ acts by means of the Jak-Stat <span class="highlight" style="background-color:">pathway</span> to abrogate the effects of Shh <span class="highlight" style="background-color:">signaling</span>. Data represent means ± SEM of triplicate samples. *<i>P</i> &lt; 0.01; **<i>P</i> &lt; 0.001; NS, not significant.</div></p></div><div class="aux"><div class="resc">Jelena Todoric, et al. Diabetes. 2011 June;60(6):1668-1676.</div><p class="links"><a class="dblinks" href="/pubmed/21536945" ref="ordinalpos=3617&amp;ncbi_uid=3375249&amp;link_uid=21536945">Citation</a><a class="dblinks" href="/pmc/articles/PMC3114396/" ref="ordinalpos=3617&amp;ncbi_uid=3375249&amp;link_uid=PMC3114396">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3618.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2763610/figure/fig5/" ref="ordinalpos=3618&amp;ncbi_uid=2306945&amp;link_uid=PMC2763610" href="/pmc/articles/PMC2763610/"><img src="/pmc/articles/instance/2830121/bin/zpg0010922590005.gif" src-large="/pmc/articles/instance/2830121/bin/zpg0010922590005.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2763610/" ref="ordinalpos=3618&amp;ncbi_uid=2306945&amp;link_uid=PMC2763610" image-link="/pmc/articles/PMC2763610/figure/fig5/" class="imagepopup">F ig . 5. From: Endogenous Cardiotonic Steroids: Physiology, Pharmacology, and Novel Therapeutic Targets. </a></p><div class="supp"><p class="details"><div>Schematic diagram contrasting “classic” (“ionic”) versus “<span class="highlight" style="background-color:">signaling</span>” pathways for CTS effects. In the classic <span class="highlight" style="background-color:">pathway</span>, any <span class="highlight" style="background-color:">signaling</span> through the Na<sup>+</sup>/K<sup>+</sup>-ATPase requires inhibition of the Na<sup>+</sup>/K<sup>+</sup>-ATPase pumping activity, which in turn is accompanied by changes in cytosolic [Na<sup>+</sup>] and [K<sup>+</sup>]. As discussed in the text, some authors feel that the caveolar Na<sup>+</sup>/K<sup>+</sup>-ATPase may be more sensitive to CTS in terms of enzymatic function. The increases in [Na<sup>+</sup>] and decreases in [K<sup>+</sup>] then induce an increase in cytosolic [Ca<sup>2+</sup>], which, in turn, activates a variety of pathways that have combinations of genomic and nongenomic effects. In contrast, the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves the association of Src with the Na<sup>+</sup>/K<sup>+</sup>-ATPase in a caveolar domain. Binding of the CTS to the Na<sup>+</sup>/K<sup>+</sup>-ATPase activates Src, which, in turn, transactivates the EGFR and phospholipase C (PLC). This leads to a cascade that involves generation of ROS, activation of mitogen-activated protein kinase (ERK) through activation of its mitogen-activated protein kinase kinase (MEK), activation of PI(3)K, stimulation of endocytosis and activation of Akt as well as activation of protein kinase C. This <span class="highlight" style="background-color:">signaling</span> construct proposes that these steps induce increases in cytosolic [Ca<sup>2+</sup>] and induce the combinations of genomic and nongenomic effects. Note that whereas both the classic and <span class="highlight" style="background-color:">signaling</span> pathways allow for intervention at the level of CTS binding to the Na<sup>+</sup>/K<sup>+</sup>-ATPase [(a), immunoneutralization or pharmacological neutralization], the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> presents a number of potential targets such as (b) interference with Src activation and EGFR transactivation, (c) PLC activation, (d) MEK activation, (e) ROS generation or scavenging, or (f) PI(3)K activation. Modulation of the <span class="highlight" style="background-color:">signaling</span> pathways at the level of PKC (g), ERK, and Akt might also be possible.</div></p></div><div class="aux"><div class="resc">Alexei Y. Bagrov, et al. Pharmacol Rev. 2009 March;61(1):9-38.</div><p class="links"><a class="dblinks" href="/pubmed/19325075" ref="ordinalpos=3618&amp;ncbi_uid=2306945&amp;link_uid=19325075">Citation</a><a class="dblinks" href="/pmc/articles/PMC2763610/" ref="ordinalpos=3618&amp;ncbi_uid=2306945&amp;link_uid=PMC2763610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3619.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2602983/figure/pone-0003917-g002/" ref="ordinalpos=3619&amp;ncbi_uid=1580611&amp;link_uid=PMC2602983" href="/pmc/articles/PMC2602983/"><img src="/pmc/articles/instance/2602983/bin/pone.0003917.g002.gif" src-large="/pmc/articles/instance/2602983/bin/pone.0003917.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2602983/" ref="ordinalpos=3619&amp;ncbi_uid=1580611&amp;link_uid=PMC2602983" image-link="/pmc/articles/PMC2602983/figure/pone-0003917-g002/" class="imagepopup">Figure 2. Downregulation of Wnt <span class="highlight" style="background-color:">pathway</span> in F3.Olig2..  From: Olig2-Induced Neural Stem Cell Differentiation Involves Downregulation of Wnt <span class="highlight" style="background-color:">Signaling</span> and Induction of Dickkopf-1 Expression. </a></p><div class="supp"><p class="details"><div>GSEA shows that Wnt <span class="highlight" style="background-color:">pathway</span> genes (A), Wnt-responsive genes (B), and Wnt <span class="highlight" style="background-color:">pathway</span> target genes (C) are enriched in HB1.F3. TOPFLASH/FOPflash reporter analysis (D) confirms that Wnt <span class="highlight" style="background-color:">pathway</span> activity in HB1.F3 is significantly higher than that in F3.Olig2. The reporter activity in HB1.F3 was inhibited to the level of F3.Olig2 when HB1.F3 was co-transfected with ‘dominant negative’ (DN) TCF constructs. Data are means±S.D. of triplicate samples. A.U., Activity unit</div></p></div><div class="aux"><div class="resc">Sung-Min Ahn, et al. PLoS ONE. 2008;3(12):e3917.</div><p class="links"><a class="dblinks" href="/pubmed/19093005" ref="ordinalpos=3619&amp;ncbi_uid=1580611&amp;link_uid=19093005">Citation</a><a class="dblinks" href="/pmc/articles/PMC2602983/" ref="ordinalpos=3619&amp;ncbi_uid=1580611&amp;link_uid=PMC2602983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3620.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3751776/figure/F2/" ref="ordinalpos=3620&amp;ncbi_uid=5220562&amp;link_uid=PMC3751776" href="/pmc/articles/PMC3751776/"><img src="/pmc/articles/instance/3751776/bin/1756-8722-6-59-2.gif" src-large="/pmc/articles/instance/3751776/bin/1756-8722-6-59-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3751776/" ref="ordinalpos=3620&amp;ncbi_uid=5220562&amp;link_uid=PMC3751776" image-link="/pmc/articles/PMC3751776/figure/F2/" class="imagepopup">Figure 2.  From: Ibrutinib and novel BTK inhibitors in clinical development. </a></p><div class="supp"><p class="details"><div><b>BTK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> BTK translocates to the plasma membrane by interacting with PIP3 to become membrane-bound where it undergoes sequential activation through trans-phosphorylation by Lyn and Syk kinases, followed by autophosphorylation. The downstream substrates of activated BTK and their associated <span class="highlight" style="background-color:">signaling</span> cascades are indicated.</div></p></div><div class="aux"><div class="resc">Akintunde Akinleye, et al. J Hematol Oncol. 2013;6:59-59.</div><p class="links"><a class="dblinks" href="/pubmed/23958373" ref="ordinalpos=3620&amp;ncbi_uid=5220562&amp;link_uid=23958373">Citation</a><a class="dblinks" href="/pmc/articles/PMC3751776/" ref="ordinalpos=3620&amp;ncbi_uid=5220562&amp;link_uid=PMC3751776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3621.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3976307/figure/pone-0093818-g005/" ref="ordinalpos=3621&amp;ncbi_uid=6002912&amp;link_uid=PMC3976307" href="/pmc/articles/PMC3976307/"><img src="/pmc/articles/instance/3976307/bin/pone.0093818.g005.gif" src-large="/pmc/articles/instance/3976307/bin/pone.0093818.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3976307/" ref="ordinalpos=3621&amp;ncbi_uid=6002912&amp;link_uid=PMC3976307" image-link="/pmc/articles/PMC3976307/figure/pone-0093818-g005/" class="imagepopup">Figure 5. FRLFE suppresses iNOS induction by the IL-1β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: The Anti-Inflammatory Effects of Flavanol-Rich Lychee Fruit Extract in Rat Hepatocytes. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">pathway</span> to activate the <i>iNOS</i> gene and the action of FRLFE are schematically depicted. The bold arrows indicate the decreases caused by FRLFE in this study. The proinflammatory cytokine IL-1β binds to its receptor (type I IL-1 receptor, IL1R1) to activate NF-κB through the IκB kinase (IKK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [23], [25]. Activated IKK phosphorylates IκB-α, resulting in the degradation of IκB-α. A circled P denotes protein phosphorylation. Active NF-κB enters into the nucleus, binds to the <i>iNOS</i> gene promoter (κB sites), and activates transcription. FRLFE inhibits the phosphorylation of IκB-α and the nuclear translocation of NF-κB, resulting in a decrease in the nuclear levels of NF-κB. Because NF-κB also regulates the transcription of the iNOS asRNA, FRLFE reduces the level of iNOS asRNA, which interacts with and stabilizes the iNOS mRNA [17]. Furthermore, FRLFE may interfere with the iNOS mRNA–asRNA interaction at a posttranscriptional level. Therefore, FRLFE significantly decreases the level of iNOS mRNA.</div></p></div><div class="aux"><div class="resc">Ryota Yamanishi, et al. PLoS One. 2014;9(4):e93818.</div><p class="links"><a class="dblinks" href="/pubmed/24705335" ref="ordinalpos=3621&amp;ncbi_uid=6002912&amp;link_uid=24705335">Citation</a><a class="dblinks" href="/pmc/articles/PMC3976307/" ref="ordinalpos=3621&amp;ncbi_uid=6002912&amp;link_uid=PMC3976307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3622.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4210072/figure/jdi12225-fig-0001/" ref="ordinalpos=3622&amp;ncbi_uid=6855008&amp;link_uid=PMC4210072" href="/pmc/articles/PMC4210072/"><img src="/pmc/articles/instance/4210072/bin/jdi-5-347-g1.gif" src-large="/pmc/articles/instance/4210072/bin/jdi-5-347-g1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4210072/" ref="ordinalpos=3622&amp;ncbi_uid=6855008&amp;link_uid=PMC4210072" image-link="/pmc/articles/PMC4210072/figure/jdi12225-fig-0001/" class="imagepopup">Figure 1.  From: Are both protein kinase A‐ and protein kinase C‐dependent pathways involved in glucagon‐like peptide‐1 action on pancreatic insulin secretion?. </a></p><div class="supp"><p class="details"><div>New concept of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of Glucagon‐like peptide‐1 (GLP‐1). Nanomolar GLP‐1 bind to GLP‐1 receptor,and then activate the cyclic adenosine monophosphate–protein kinase A (PKA) <span class="highlight" style="background-color:">pathway</span> and exchange protein directly activated by cyclic adenosine monophosphate 2 (Epac 2), which requires elevation of cyclic adenosine monophosphate (classical <span class="highlight" style="background-color:">pathway</span>). Picomolar GLP‐1 activates protein kinase C (PKC) through the same GLP‐1 receptor (high affinity site) and opens the L‐type calcium channel (novel <span class="highlight" style="background-color:">pathway</span>; blue). AC, adenylate cyclase; PLC, phospholipase C.</div></p></div><div class="aux"><div class="resc">Makoto Shigeto, et al. J Diabetes Investig. 2014 July;5(4):347-348.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3622&amp;ncbi_uid=6855008&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4210072/" ref="ordinalpos=3622&amp;ncbi_uid=6855008&amp;link_uid=PMC4210072">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3623.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4324858/figure/Fig7/" ref="ordinalpos=3623&amp;ncbi_uid=7168156&amp;link_uid=PMC4324858" href="/pmc/articles/PMC4324858/"><img src="/pmc/articles/instance/4324858/bin/40001_2015_84_Fig7_HTML.gif" src-large="/pmc/articles/instance/4324858/bin/40001_2015_84_Fig7_HTML.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4324858/" ref="ordinalpos=3623&amp;ncbi_uid=7168156&amp;link_uid=PMC4324858" image-link="/pmc/articles/PMC4324858/figure/Fig7/" class="imagepopup">Figure 7.  From: Hypoxia-induced MTA1 promotes MC3T3 osteoblast growth but suppresses MC3T3 osteoblast differentiation. </a></p><div class="supp"><p class="details"><div> <b>A schematic presentation showing the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from fracture-induced hypoxia to MTA1 upregulation, suppressing osteoblast differentiation.</b> </div></p></div><div class="aux"><div class="resc">Tielong Liu, et al. Eur J Med Res. 2015;20(1):10.</div><p class="links"><a class="dblinks" href="/pubmed/25644400" ref="ordinalpos=3623&amp;ncbi_uid=7168156&amp;link_uid=25644400">Citation</a><a class="dblinks" href="/pmc/articles/PMC4324858/" ref="ordinalpos=3623&amp;ncbi_uid=7168156&amp;link_uid=PMC4324858">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3624.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3223690/figure/fig4/" ref="ordinalpos=3624&amp;ncbi_uid=3712302&amp;link_uid=PMC3223690" href="/pmc/articles/PMC3223690/"><img src="/pmc/articles/instance/3223690/bin/cddis2011103f4.gif" src-large="/pmc/articles/instance/3223690/bin/cddis2011103f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3223690/" ref="ordinalpos=3624&amp;ncbi_uid=3712302&amp;link_uid=PMC3223690" image-link="/pmc/articles/PMC3223690/figure/fig4/" class="imagepopup">Figure 4.  From: TRAIL promotes caspase-dependent pro-inflammatory responses via PKCδ activation by vascular smooth muscle cells. </a></p><div class="supp"><p class="details"><div>Proposed model of TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> responsible for ICAM-1 expression</div></p></div><div class="aux"><div class="resc">S Song, et al. Cell Death Dis. 2011 November;2(11):e223.</div><p class="links"><a class="dblinks" href="/pubmed/22048166" ref="ordinalpos=3624&amp;ncbi_uid=3712302&amp;link_uid=22048166">Citation</a><a class="dblinks" href="/pmc/articles/PMC3223690/" ref="ordinalpos=3624&amp;ncbi_uid=3712302&amp;link_uid=PMC3223690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3625.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151704/figure/f9/" ref="ordinalpos=3625&amp;ncbi_uid=255899&amp;link_uid=PMC151704" href="/pmc/articles/PMC151704/"><img src="/pmc/articles/instance/151704/bin/mb0531267009.gif" src-large="/pmc/articles/instance/151704/bin/mb0531267009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151704/" ref="ordinalpos=3625&amp;ncbi_uid=255899&amp;link_uid=PMC151704" image-link="/pmc/articles/PMC151704/figure/f9/" class="imagepopup">FIG. 9. From: Karyopherin-Mediated Nuclear Import of the Homing Endonuclease VMA1-Derived Endonuclease Is Required for Self-Propagation of the Coding Region. </a></p><div class="supp"><p class="details"><div>Model of nucleocytoplasmic transport of VDE mediated by the TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and karyopherins.</div></p></div><div class="aux"><div class="resc">Yuri Nagai, et al. Mol Cell Biol. 2003 March;23(5):1726-1736.</div><p class="links"><a class="dblinks" href="/pubmed/12588991" ref="ordinalpos=3625&amp;ncbi_uid=255899&amp;link_uid=12588991">Citation</a><a class="dblinks" href="/pmc/articles/PMC151704/" ref="ordinalpos=3625&amp;ncbi_uid=255899&amp;link_uid=PMC151704">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3626.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2690692/figure/pone-0005880-g001/" ref="ordinalpos=3626&amp;ncbi_uid=1883491&amp;link_uid=PMC2690692" href="/pmc/articles/PMC2690692/"><img src="/pmc/articles/instance/2690692/bin/pone.0005880.g001.gif" src-large="/pmc/articles/instance/2690692/bin/pone.0005880.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2690692/" ref="ordinalpos=3626&amp;ncbi_uid=1883491&amp;link_uid=PMC2690692" image-link="/pmc/articles/PMC2690692/figure/pone-0005880-g001/" class="imagepopup">Figure 1. Hyperactivation of Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> leads to late somite-loss..  From: Regulation of Slow and Fast Muscle Myofibrillogenesis by Wnt/?-Catenin and Myostatin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) TOPdGFP transgenic embryos report activated Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, i.e. TOPdGFP transcripts. Cartoon depicts the level of vibratome sectioning i.e. left panel at the yolk extension and right panel caudal to the yolk extension. Scale bar, 50 µm. (B) Hyperactivation of a target of Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> <i>lef1</i>, as shown with WISH, in <i>axin1/apc1</i> mutants matches the expression of TOPdGFP. Scale bar, 250 µm. (C) The <i>axin1/apc1</i> embryos are slightly shorter compared to wild-type embryos at 36 hpf, top panels. At 54 hpf, the difference becomes striking, bottom panels. Scale bar, 500 µm. (D) Somite counts at 36 hpf and 54 hpf, corresponding to embryos depicted in (C) with error bars showing the standard deviation.</div></p></div><div class="aux"><div class="resc">Jin-Ming Tee, et al. PLoS ONE. 2009;4(6):e5880.</div><p class="links"><a class="dblinks" href="/pubmed/19517013" ref="ordinalpos=3626&amp;ncbi_uid=1883491&amp;link_uid=19517013">Citation</a><a class="dblinks" href="/pmc/articles/PMC2690692/" ref="ordinalpos=3626&amp;ncbi_uid=1883491&amp;link_uid=PMC2690692">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3627.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99013/figure/F5/" ref="ordinalpos=3627&amp;ncbi_uid=52578&amp;link_uid=PMC99013" href="/pmc/articles/PMC99013/"><img src="/pmc/articles/instance/99013/bin/mr0400030005.gif" src-large="/pmc/articles/instance/99013/bin/mr0400030005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99013/" ref="ordinalpos=3627&amp;ncbi_uid=52578&amp;link_uid=PMC99013" image-link="/pmc/articles/PMC99013/figure/F5/" class="imagepopup">FIG. 5.  From: Signal Transduction Cascades Regulating Fungal  Development and Virulence. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways regulating mating of <i>S. pombe</i>. Similar to <i>S. cerevisiae</i> diploid pseudohyphal growth, mating in <i>S. pombe</i> is regulated by two parallel <span class="highlight" style="background-color:">signaling</span> pathways. One responds to pheromones and involves a conserved MAP kinase (MAPK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The second is a nutrient-sensing <span class="highlight" style="background-color:">pathway</span> that, in the presence of abundant nutrients, stimulates cAMP production to repress <i>ste11</i> expression and inhibit mating. CAP, cyclase-associated protein.</div></p></div><div class="aux"><div class="resc">Klaus B. Lengeler, et al. Microbiol Mol Biol Rev. 2000 December;64(4):746-785.</div><p class="links"><a class="dblinks" href="/pubmed/11104818" ref="ordinalpos=3627&amp;ncbi_uid=52578&amp;link_uid=11104818">Citation</a><a class="dblinks" href="/pmc/articles/PMC99013/" ref="ordinalpos=3627&amp;ncbi_uid=52578&amp;link_uid=PMC99013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3628.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3275167/figure/F7/" ref="ordinalpos=3628&amp;ncbi_uid=4569201&amp;link_uid=PMC3275167" href="/pmc/articles/PMC3275167/"><img src="/pmc/articles/instance/3275167/bin/zmg0021249480007.gif" src-large="/pmc/articles/instance/3275167/bin/zmg0021249480007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3275167/" ref="ordinalpos=3628&amp;ncbi_uid=4569201&amp;link_uid=PMC3275167" image-link="/pmc/articles/PMC3275167/figure/F7/" class="imagepopup">Fig. 7. From: Complex Human Glucocorticoid Receptor dim Mutations Define Glucocorticoid Induced Apoptotic Resistance in Bone Cells. </a></p><div class="supp"><p class="details"><div>Regulation of genes in the apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A, Using Ingenuity Pathways Analysis, genes involved in apoptosis <span class="highlight" style="background-color:">signaling</span> were shown to be regulated by hGRα+DEX, including Bcl-xL (down-regulated) and CASP6 (up-regulated) (n = 3; *, <i>P</i> &lt; 0.01). B, Ingenuity Pathways Analysis of genes involved in apoptosis <span class="highlight" style="background-color:">signaling</span> and regulated by hGR<sup>dim4</sup>+DEX. These included cIAP2 (up-regulated) and Gas2 (up-regulated) (n = 3; *, <i>P</i> &lt; 0.01). C, Total RNA was isolated from hGRα, hGR<sup>dim</sup>, and hGR<sup>dim4</sup> cells treated with 100 n<span style="font-variant: small-caps" class="small-caps">m</span> DEX for 6 h and gene regulation analyzed using quantitative RT-PCR. Four genes involved in apoptosis <span class="highlight" style="background-color:">signaling</span> were shown to be differentially regulated by hGRα (Bcl-xL and CASP6) or hGR<sup>dim</sup> and hGR<sup>dim4</sup> (cIAP2 and Gas2) (n = 3; *, <i>P</i> &lt; 0.01). CON, Control.</div></p></div><div class="aux"><div class="resc">Christine M. Jewell, et al. Mol Endocrinol. 2012 February;26(2):244-256.</div><p class="links"><a class="dblinks" href="/pubmed/22174376" ref="ordinalpos=3628&amp;ncbi_uid=4569201&amp;link_uid=22174376">Citation</a><a class="dblinks" href="/pmc/articles/PMC3275167/" ref="ordinalpos=3628&amp;ncbi_uid=4569201&amp;link_uid=PMC3275167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3629.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4133022/figure/Fig8/" ref="ordinalpos=3629&amp;ncbi_uid=6523001&amp;link_uid=PMC4133022" href="/pmc/articles/PMC4133022/"><img src="/pmc/articles/instance/4133022/bin/11103_2014_221_Fig8_HTML.gif" src-large="/pmc/articles/instance/4133022/bin/11103_2014_221_Fig8_HTML.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4133022/" ref="ordinalpos=3629&amp;ncbi_uid=6523001&amp;link_uid=PMC4133022" image-link="/pmc/articles/PMC4133022/figure/Fig8/" class="imagepopup">Fig. 8.  From: WRKY45-dependent priming of diterpenoid phytoalexin biosynthesis in rice and the role of cytokinin in triggering the reaction. </a></p><div class="supp"><p class="details"><div>Proposed model for WRKY45-dependent regulation of DP biosynthesis. BTH acts on the rice SA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is split into WRKY45- and OsNPR1 subpathways. BTH primes the expression of DP biosynthetic genes via WRKY45. <i>M. oryzae</i> infection triggers early accumulation of CKs, which in turn act synergistically with the WRKY45-mediated SA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to activate expression of DP biosynthetic genes, leading to accumulation of DPs. MKK4-dependent pathways are also shown</div></p></div><div class="aux"><div class="resc">Aya Akagi, et al. Plant Mol Biol. 2014;86:171-183.</div><p class="links"><a class="dblinks" href="/pubmed/25033935" ref="ordinalpos=3629&amp;ncbi_uid=6523001&amp;link_uid=25033935">Citation</a><a class="dblinks" href="/pmc/articles/PMC4133022/" ref="ordinalpos=3629&amp;ncbi_uid=6523001&amp;link_uid=PMC4133022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3630.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1375233/figure/fig5/" ref="ordinalpos=3630&amp;ncbi_uid=652362&amp;link_uid=PMC1375233" href="/pmc/articles/PMC1375233/"><img src="/pmc/articles/instance/1375233/bin/nek010f5.gif" src-large="/pmc/articles/instance/1375233/bin/nek010f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1375233/" ref="ordinalpos=3630&amp;ncbi_uid=652362&amp;link_uid=PMC1375233" image-link="/pmc/articles/PMC1375233/figure/fig5/" class="imagepopup">Figure 5.  From: Role of Toll-like Receptors in Adjuvant-Augmented Immune Therapies. </a></p><div class="supp"><p class="details"><div>The two <span class="highlight" style="background-color:">signaling</span> pathways of TLR in myeloid DCs. MyD88-dependent <span class="highlight" style="background-color:">pathway</span> are indicated to the left and TICAM-1 <span class="highlight" style="background-color:">pathway</span> are shown to the right in blue. Many <span class="highlight" style="background-color:">signaling</span> molecules are involved in the two pathways leading to different outputs. NK and CTL are generated by the resultant mDC activation. In particular, cross-priming must occur in mDC to induce CTL, the molecular mechanism of which are largely unknown.</div></p></div><div class="aux"><div class="resc">Tsukasa Seya, et al. Evid Based Complement Alternat Med. 2006 March;3(1):31-38.</div><p class="links"><a class="dblinks" href="/pubmed/16550221" ref="ordinalpos=3630&amp;ncbi_uid=652362&amp;link_uid=16550221">Citation</a><a class="dblinks" href="/pmc/articles/PMC1375233/" ref="ordinalpos=3630&amp;ncbi_uid=652362&amp;link_uid=PMC1375233">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3631.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854134/figure/F6/" ref="ordinalpos=3631&amp;ncbi_uid=2795532&amp;link_uid=PMC2854134" href="/pmc/articles/PMC2854134/"><img src="/pmc/articles/instance/2854134/bin/nihms-169863-f0006.gif" src-large="/pmc/articles/instance/2854134/bin/nihms-169863-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854134/" ref="ordinalpos=3631&amp;ncbi_uid=2795532&amp;link_uid=PMC2854134" image-link="/pmc/articles/PMC2854134/figure/F6/" class="imagepopup">Figure 6. Celastrol induces cell apoptosis and inhibits AKT/mTOR/P70S6K <span class="highlight" style="background-color:">pathway</span> in PC-3 cancer cells.  From: Celastrol Suppresses Angiogenesis-Mediated Tumor Growth through Inhibition of AKT/Mammalian Target of Rapamycin <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><i>A</i>. Celastrol inhibited cell viability of prostrate PC-3 cancer cells. Cell viability was quantified by MTS assay. <i>Columns</i>, mean from three different experiments; <i>bars</i>, standard deviation, **, <i>P</i> &lt; 0.01 <i>vs</i>. VEGF alone. <i>B</i>. Celastrol induced PC-3 cancer cell apoptosis in concentration-dependent and time-dependent manners by the cleaved-PARP analysis. <i>C</i>, Celastrol suppressed the phosphorylation of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> kinases in PC-3 prostate cancer cell. To examine mTOR <span class="highlight" style="background-color:">pathway</span> in prostate tumor cells, normal cultured PC-3 cells (non-starving) were directly treated with indicated dilutions of Celastrol for 4 h. Proteins from different treatments were applied to Western blotting and probed with specific antibodies. Similar data were obtained from three independent experiments.</div></p></div><div class="aux"><div class="resc">Xiufeng Pang, et al. Cancer Res. ;70(5):1951-1959.</div><p class="links"><a class="dblinks" href="/pubmed/20160026" ref="ordinalpos=3631&amp;ncbi_uid=2795532&amp;link_uid=20160026">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854134/" ref="ordinalpos=3631&amp;ncbi_uid=2795532&amp;link_uid=PMC2854134">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3632.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3096550/figure/F6/" ref="ordinalpos=3632&amp;ncbi_uid=2955476&amp;link_uid=PMC3096550" href="/pmc/articles/PMC3096550/"><img src="/pmc/articles/instance/3096550/bin/nihms287625f6.gif" src-large="/pmc/articles/instance/3096550/bin/nihms287625f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3096550/" ref="ordinalpos=3632&amp;ncbi_uid=2955476&amp;link_uid=PMC3096550" image-link="/pmc/articles/PMC3096550/figure/F6/" class="imagepopup">Fig. 6.  From: Arginine Deprivation, Autophagy, Apoptosis (AAA) for the Treatment of Melanoma. </a></p><div class="supp"><p class="details"><div>Growth <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affected by arginine deprivation in ASS(−) cells which leads to autophagy.</div></p></div><div class="aux"><div class="resc">N. Savaraj, et al. Curr Mol Med. ;10(4):405-412.</div><p class="links"><a class="dblinks" href="/pubmed/20459375" ref="ordinalpos=3632&amp;ncbi_uid=2955476&amp;link_uid=20459375">Citation</a><a class="dblinks" href="/pmc/articles/PMC3096550/" ref="ordinalpos=3632&amp;ncbi_uid=2955476&amp;link_uid=PMC3096550">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3633.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2769160/figure/f3-ijms-10-03793/" ref="ordinalpos=3633&amp;ncbi_uid=2146796&amp;link_uid=PMC2769160" href="/pmc/articles/PMC2769160/"><img src="/pmc/articles/instance/2769160/bin/ijms-10-03793f3.gif" src-large="/pmc/articles/instance/2769160/bin/ijms-10-03793f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2769160/" ref="ordinalpos=3633&amp;ncbi_uid=2146796&amp;link_uid=PMC2769160" image-link="/pmc/articles/PMC2769160/figure/f3-ijms-10-03793/" class="imagepopup">Figure 3. From: Trehalose Metabolism: From Osmoprotection to <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The role of trehalose <span class="highlight" style="background-color:">pathway</span> in prokaryotes and eukaryotes. Trehalose in prokaryotes dramatically accumulates in osmotic or thermal stress conditions; it can signal from the bacteria to the plant cell stress tolerance and nitrogen and carbon metabolism. In plants, trehalose 6-phosphate (T6P) plays a central role regulating sugar metabolism and plant development. Glucose and trehalose are also important keys to several <span class="highlight" style="background-color:">signaling</span> and regulatory pathways and integrate external cues to adapt cells to abiotic stress, growth and development. It seems that TPS1 and ABI4 are part of the HXK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Other molecular actors in this network are the 14-3-3 proteins, which are known to interact with phosphoserine in diverse proteins including TPS; and SnRK1, which signals catabolism in starvation conditions, and is countered by T6P to induce anabolism. Thus, an important role of the trehalose biosynthesis <span class="highlight" style="background-color:">pathway</span> in higher plants would be the synthesis of small amounts of T6P and/or trehalose <span class="highlight" style="background-color:">signaling</span> molecules rather than accumulation of this latter as an osmoprotective compound.</div></p></div><div class="aux"><div class="resc">Gabriel Iturriaga, et al. Int J Mol Sci. 2009 September;10(9):3793-3810.</div><p class="links"><a class="dblinks" href="/pubmed/19865519" ref="ordinalpos=3633&amp;ncbi_uid=2146796&amp;link_uid=19865519">Citation</a><a class="dblinks" href="/pmc/articles/PMC2769160/" ref="ordinalpos=3633&amp;ncbi_uid=2146796&amp;link_uid=PMC2769160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3634.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878570/figure/F1/" ref="ordinalpos=3634&amp;ncbi_uid=2999419&amp;link_uid=PMC2878570" href="/pmc/articles/PMC2878570/"><img src="/pmc/articles/instance/2878570/bin/zbc0261019240001.gif" src-large="/pmc/articles/instance/2878570/bin/zbc0261019240001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878570/" ref="ordinalpos=3634&amp;ncbi_uid=2999419&amp;link_uid=PMC2878570" image-link="/pmc/articles/PMC2878570/figure/F1/" class="imagepopup">FIGURE 1. From: Receptor-independent Ambient pH <span class="highlight" style="background-color:">Signaling</span> by Ubiquitin Attachment to Fungal Arrestin-like PalF. </a></p><div class="supp"><p class="details"><div><b>PalH dependence of PacC processing.</b> <i>A</i>, sketch of the <i>A. nidulans</i> pH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Upon alkaline pH stimulation, the <i>pal</i> <span class="highlight" style="background-color:">pathway</span> (PalH, PalF, PalI, PalC, PalA, and PalB) mediates conversion of PacC<sup>72</sup> (<i>i.e.</i> 72 kDa) into the committed 53-kDa intermediate PacC<sup>53</sup>, which is further processed to 27-kDa PacC<sup>27</sup>. <i>B</i>, Western analysis of Myc3-PacC processing in wild-type and <i>palH</i><sup>−</sup> strains. Cells were cultured at pH 4.2 (H<sup>+</sup>) and shifted to pH 8.2 (OH<sup>−</sup>) for the indicated times. Actin levels were used as loading control.</div></p></div><div class="aux"><div class="resc">América Hervás-Aguilar, et al. J Biol Chem. 2010 June 4;285(23):18095-18102.</div><p class="links"><a class="dblinks" href="/pubmed/20368671" ref="ordinalpos=3634&amp;ncbi_uid=2999419&amp;link_uid=20368671">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878570/" ref="ordinalpos=3634&amp;ncbi_uid=2999419&amp;link_uid=PMC2878570">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3635.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3744159/figure/F1/" ref="ordinalpos=3635&amp;ncbi_uid=5198910&amp;link_uid=PMC3744159" href="/pmc/articles/PMC3744159/"><img src="/pmc/articles/instance/3744159/bin/nihms445512f1.gif" src-large="/pmc/articles/instance/3744159/bin/nihms445512f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3744159/" ref="ordinalpos=3635&amp;ncbi_uid=5198910&amp;link_uid=PMC3744159" image-link="/pmc/articles/PMC3744159/figure/F1/" class="imagepopup">Figure 1.  From: Satellite <span class="highlight" style="background-color:">signaling</span> at synapses. </a></p><div class="supp"><p class="details"><div>Model of Nervous Wreck function. Nwk binds Dap160, dynamin and Thickveins to negatively regulate the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Because Nwk specifically localizes to Rab11-positive endocytic compartments, we hypothesize that Nwk regulates receptor recycling through an RE-dependent <span class="highlight" style="background-color:">pathway</span>. Shunting Thickveins away from the more rapid RE-independent recycling route would have the effect of reducing plasma membrane receptor concentrations and, thus, neuronal responsiveness to the retrograde BMP signal. Early endosome (EE); Recycling endosome (RE); Wishful Thinking Type II BMP receptor (Wit).</div></p></div><div class="aux"><div class="resc">Kate M. O'Connor-Giles, et al. Fly (Austin). ;2(5):259-261.</div><p class="links"><a class="dblinks" href="/pubmed/20798607" ref="ordinalpos=3635&amp;ncbi_uid=5198910&amp;link_uid=20798607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3744159/" ref="ordinalpos=3635&amp;ncbi_uid=5198910&amp;link_uid=PMC3744159">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3636.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3544245/figure/fig05/" ref="ordinalpos=3636&amp;ncbi_uid=4539585&amp;link_uid=PMC3544245" href="/pmc/articles/PMC3544245/"><img src="/pmc/articles/instance/3544245/bin/ijcep0006-0168-f5.gif" src-large="/pmc/articles/instance/3544245/bin/ijcep0006-0168-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3544245/" ref="ordinalpos=3636&amp;ncbi_uid=4539585&amp;link_uid=PMC3544245" image-link="/pmc/articles/PMC3544245/figure/fig05/" class="imagepopup">Figure 5.  From: ?-Naphthoflavone inhibits 3T3-L1 pre-adipocytes differentiation via modulating p38MAPK <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>α-NF inhibits 3T3-L1 pre-adipocyte differentiation involving p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A and B. The mRNA expressions of both p38α and p38β isoforms were down-regulated by α-NF during adipogenesis; C. α-NF slightly suppressed total p38 protein expression but not reached significantly level; D. After 20 and 40 minutes of hormone cocktail administration, phosphorylation of p-p38(thr180/tyr182) was significantly induced and was substainly inhibited by α-NF.</div></p></div><div class="aux"><div class="resc">Qiqiang He, et al. Int J Clin Exp Pathol. 2013;6(2):168-178.</div><p class="links"><a class="dblinks" href="/pubmed/23330002" ref="ordinalpos=3636&amp;ncbi_uid=4539585&amp;link_uid=23330002">Citation</a><a class="dblinks" href="/pmc/articles/PMC3544245/" ref="ordinalpos=3636&amp;ncbi_uid=4539585&amp;link_uid=PMC3544245">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3637.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3325758/figure/F2/" ref="ordinalpos=3637&amp;ncbi_uid=4036824&amp;link_uid=PMC3325758" href="/pmc/articles/PMC3325758/"><img src="/pmc/articles/instance/3325758/bin/nihms-367471-f0002.gif" src-large="/pmc/articles/instance/3325758/bin/nihms-367471-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3325758/" ref="ordinalpos=3637&amp;ncbi_uid=4036824&amp;link_uid=PMC3325758" image-link="/pmc/articles/PMC3325758/figure/F2/" class="imagepopup">Fig. 2.  From: Role of phospholipase C? in physiological phosphoinositide <span class="highlight" style="background-color:">signaling</span> networks. </a></p><div class="supp"><p class="details"><div>Common mechanisms for PLCε regulation of CICR in cardiac myocytes and pancreatic β cells downstream of Gs-coupled receptors. A) Common upstream components regulate of PLCε by Gs coupled receptors in adult cardiac myocytes and isolated pancreatic β cells [19,54]. B) <span class="highlight" style="background-color:">Signaling</span> downstream of PLCε. In red are components found to be involved in regulation of CICR in both cardiac myocytes and pancreatic β cells [19,54]. An alternative <span class="highlight" style="background-color:">pathway</span> has also been proposed and is depicted in black and may operate in β cells as well [57].</div></p></div><div class="aux"><div class="resc">Alan V. Smrcka, et al. Cell Signal. ;24(6):1333-1343.</div><p class="links"><a class="dblinks" href="/pubmed/22286105" ref="ordinalpos=3637&amp;ncbi_uid=4036824&amp;link_uid=22286105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3325758/" ref="ordinalpos=3637&amp;ncbi_uid=4036824&amp;link_uid=PMC3325758">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3638.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3078043/figure/F5/" ref="ordinalpos=3638&amp;ncbi_uid=3390809&amp;link_uid=PMC3078043" href="/pmc/articles/PMC3078043/"><img src="/pmc/articles/instance/3078043/bin/nihms-257938-f0005.gif" src-large="/pmc/articles/instance/3078043/bin/nihms-257938-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3078043/" ref="ordinalpos=3638&amp;ncbi_uid=3390809&amp;link_uid=PMC3078043" image-link="/pmc/articles/PMC3078043/figure/F5/" class="imagepopup">Figure 5.  From: ?1-adrenergic receptors activate two distinct <span class="highlight" style="background-color:">signaling</span> pathways in striatal neurons. </a></p><div class="supp"><p class="details"><div>β1-adrenergic receptors activate multiple pathways in striatal neurons. β1-adrenergic receptor stimulation induces canonical cAMP production and the activation of a receptor tyrosine kinase. Receptor tyrosine kinase activation leads to the stimulation of the Ras/Raf/MEK/MAPK/MSK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, ultimately leading to rapid CREB phosphorylation. We note that receptor tyrosine kinases can be present on both the cellular and internal membranes.</div></p></div><div class="aux"><div class="resc">John Meitzen, et al. J Neurochem. ;116(6):984-995.</div><p class="links"><a class="dblinks" href="/pubmed/21143600" ref="ordinalpos=3638&amp;ncbi_uid=3390809&amp;link_uid=21143600">Citation</a><a class="dblinks" href="/pmc/articles/PMC3078043/" ref="ordinalpos=3638&amp;ncbi_uid=3390809&amp;link_uid=PMC3078043">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3639.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2676770/figure/F1/" ref="ordinalpos=3639&amp;ncbi_uid=1802734&amp;link_uid=PMC2676770" href="/pmc/articles/PMC2676770/"><img src="/pmc/articles/instance/2676770/bin/psb0405_0470_fig001.gif" src-large="/pmc/articles/instance/2676770/bin/psb0405_0470_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2676770/" ref="ordinalpos=3639&amp;ncbi_uid=1802734&amp;link_uid=PMC2676770" image-link="/pmc/articles/PMC2676770/figure/F1/" class="imagepopup">Figure 1.  From: DOR. </a></p><div class="supp"><p class="details"><div>Model for DOR in mediating ABA-induced stomatal closure under drought stress. Arrows indicate the ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for inducing stomatal closure in response to drought stress. <i>DOR</i> functions as a negative regulator of ABA biosynthesis and ABA further reduces the <i>DOR</i> expression. <i>ABI1</i> acts as a negative regulator for both ABA-induced stomatal closure and <i>DOR</i> expression. <i>DOR</i> likely forms two negative feedback regulatory loops for ABA-induced stomatal closure.</div></p></div><div class="aux"><div class="resc">Yu'e Zhang, et al. Plant Signal Behav. 2009 May;4(5):470-471.</div><p class="links"><a class="dblinks" href="/pubmed/19816119" ref="ordinalpos=3639&amp;ncbi_uid=1802734&amp;link_uid=19816119">Citation</a><a class="dblinks" href="/pmc/articles/PMC2676770/" ref="ordinalpos=3639&amp;ncbi_uid=1802734&amp;link_uid=PMC2676770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3640.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711107/figure/F7/" ref="ordinalpos=3640&amp;ncbi_uid=5102705&amp;link_uid=PMC3711107" href="/pmc/articles/PMC3711107/"><img src="/pmc/articles/instance/3711107/bin/nihms484331f7.gif" src-large="/pmc/articles/instance/3711107/bin/nihms484331f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711107/" ref="ordinalpos=3640&amp;ncbi_uid=5102705&amp;link_uid=PMC3711107" image-link="/pmc/articles/PMC3711107/figure/F7/" class="imagepopup">Figure 7. The Effect of IGF-1R siRNA on mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> is Reversed by AICAR.  From: CROSSTALK BETWEEN THE AMP-ACTIVATED KINASE AND INSULIN <span class="highlight" style="background-color:">SIGNALING</span> PATHWAYS RESCUES MURINE BLASTOCYST CELLS FROM INSULIN RESISTANCE. </a></p><div class="supp"><p class="details"><div>TS IGF-1R knockdown cells were incubated 2h with AICAR (250μM). Lysates were harvested and analyzed by Western blotting using antibodies specific for (A) p-mTOR, (B) p-70S6K, and (C) p-riboS6K. The p-mTOR Western blot was quantitated using densitometry and NIH image software, and normalized to β-Actin in the corresponding graph.</div></p></div><div class="aux"><div class="resc">Erica Louden, et al. Reproduction. ;136(3):335-344.</div><p class="links"><a class="dblinks" href="/pubmed/18577554" ref="ordinalpos=3640&amp;ncbi_uid=5102705&amp;link_uid=18577554">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711107/" ref="ordinalpos=3640&amp;ncbi_uid=5102705&amp;link_uid=PMC3711107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3641.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2373331/figure/F5/" ref="ordinalpos=3641&amp;ncbi_uid=1313615&amp;link_uid=PMC2373331" href="/pmc/articles/PMC2373331/"><img src="/pmc/articles/instance/2373331/bin/zpq0150801420005.gif" src-large="/pmc/articles/instance/2373331/bin/zpq0150801420005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2373331/" ref="ordinalpos=3641&amp;ncbi_uid=1313615&amp;link_uid=PMC2373331" image-link="/pmc/articles/PMC2373331/figure/F5/" class="imagepopup">Fig. 5. From: Insulin <span class="highlight" style="background-color:">signaling</span> and FOXO regulate the overwintering diapause of the mosquito Culex pipiens. </a></p><div class="supp"><p class="details"><div>Model for diapause regulation in the mosquito <i>C. pipiens</i>. Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during the nondiapausing stage, resulting in ovarian development (<i>A</i>), and during diapause, resulting in arrested ovarian development, fat hypertrophy, and resistance to overwintering stress (<i>B</i>) is shown. Arrows indicate stimulation and T-bars indicate suppression. Black and gray indicate the ON and OFF activity states of the genes, respectively.</div></p></div><div class="aux"><div class="resc">Cheolho Sim, et al. Proc Natl Acad Sci U S A. 2008 May 6;105(18):6777-6781.</div><p class="links"><a class="dblinks" href="/pubmed/18448677" ref="ordinalpos=3641&amp;ncbi_uid=1313615&amp;link_uid=18448677">Citation</a><a class="dblinks" href="/pmc/articles/PMC2373331/" ref="ordinalpos=3641&amp;ncbi_uid=1313615&amp;link_uid=PMC2373331">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3642.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3883963/figure/f3-13_099_chen/" ref="ordinalpos=3642&amp;ncbi_uid=5686371&amp;link_uid=PMC3883963" href="/pmc/articles/PMC3883963/"><img src="/pmc/articles/instance/3883963/bin/13_099_chenf3.gif" src-large="/pmc/articles/instance/3883963/bin/13_099_chenf3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3883963/" ref="ordinalpos=3642&amp;ncbi_uid=5686371&amp;link_uid=PMC3883963" image-link="/pmc/articles/PMC3883963/figure/f3-13_099_chen/" class="imagepopup">Figure 3.  From: Emerging Role of High-Mobility Group Box 1 (HMGB1) in Liver Diseases. </a></p><div class="supp"><p class="details"><div>HMGB1-TLR4 <span class="highlight" style="background-color:">signaling</span> mediates liver I/R injury. HMGB1 is an early mediator of injury and inflammation in liver I/R, and TLR4 is the major receptor that is involved in the process by which the JNK MAPK/NF-κB <span class="highlight" style="background-color:">pathway</span> is activated. The release of HMGB1 in liver I/R requires TLR4 but not CD14. Different strategies of HMGB1-TLR4 <span class="highlight" style="background-color:">pathway</span> inhibition as indicated have been shown to reduce liver I/R injury.</div></p></div><div class="aux"><div class="resc">Ruochan Chen, et al. Mol Med. 2013;19(1):357-366.</div><p class="links"><a class="dblinks" href="/pubmed/24306421" ref="ordinalpos=3642&amp;ncbi_uid=5686371&amp;link_uid=24306421">Citation</a><a class="dblinks" href="/pmc/articles/PMC3883963/" ref="ordinalpos=3642&amp;ncbi_uid=5686371&amp;link_uid=PMC3883963">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3643.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3122216/figure/F8/" ref="ordinalpos=3643&amp;ncbi_uid=3580477&amp;link_uid=PMC3122216" href="/pmc/articles/PMC3122216/"><img src="/pmc/articles/instance/3122216/bin/zbc0291166720008.gif" src-large="/pmc/articles/instance/3122216/bin/zbc0291166720008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3122216/" ref="ordinalpos=3643&amp;ncbi_uid=3580477&amp;link_uid=PMC3122216" image-link="/pmc/articles/PMC3122216/figure/F8/" class="imagepopup">FIGURE 8. From: Casein Kinase 2 Regulates the mRNA-destabilizing Activity of Tristetraprolin. </a></p><div class="supp"><p class="details"><div><b>Model of regulation of TTP expression by CK2.</b> The p38 MAPK/MK2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can induce phosphorylation (<i>P</i>) of TTP, which leads to proteasomal degradation of TTP protein. However, when CK2 is activated by TGF-β stimulation, it can activate MKP-1, which in turn inhibits the p38 MAPK/MK2 <span class="highlight" style="background-color:">pathway</span> by dephosphorylation of p38 MAPK, thereby protecting the TTP protein from proteasomal degradation.</div></p></div><div class="aux"><div class="resc">Won Hyeok Lee, et al. J Biol Chem. 2011 June 17;286(24):21577-21587.</div><p class="links"><a class="dblinks" href="/pubmed/21507959" ref="ordinalpos=3643&amp;ncbi_uid=3580477&amp;link_uid=21507959">Citation</a><a class="dblinks" href="/pmc/articles/PMC3122216/" ref="ordinalpos=3643&amp;ncbi_uid=3580477&amp;link_uid=PMC3122216">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3644.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676867/figure/F2/" ref="ordinalpos=3644&amp;ncbi_uid=4977598&amp;link_uid=PMC3676867" href="/pmc/articles/PMC3676867/"><img src="/pmc/articles/instance/3676867/bin/nihms195252f2.gif" src-large="/pmc/articles/instance/3676867/bin/nihms195252f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676867/" ref="ordinalpos=3644&amp;ncbi_uid=4977598&amp;link_uid=PMC3676867" image-link="/pmc/articles/PMC3676867/figure/F2/" class="imagepopup">Fig. 2.  From: GRP94 in ER Quality Control and Stress Responses. </a></p><div class="supp"><p class="details"><div>IGF is both a causative agent and a target of UPR.<br />The scheme illustrates that ER stress activates the UPR, leading either to resolution of the stress (recovery) or to apoptosis. In addition to the ‘conventional’ UPR target genes, (BiP, PDI, CHOP, etc.) IGF, IGFBP1 and possibly other members of the insulin/IGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are UPR targets. GRP94 and IGFs play a specific role in alleviating ER-stress: GRP94 is essential for production of IGF. Autocrine/paracrine IGF <span class="highlight" style="background-color:">signaling</span> can delivers both anti-apoptotic and pro-recovery signals. In addition, IGF <span class="highlight" style="background-color:">signaling</span> also potentiates an existing UPR, which in turn can control IGF <span class="highlight" style="background-color:">signaling</span> either positively or negatively.</div></p></div><div class="aux"><div class="resc">Davide Eletto, et al. Semin Cell Dev Biol. ;21(5):479-485.</div><p class="links"><a class="dblinks" href="/pubmed/20223290" ref="ordinalpos=3644&amp;ncbi_uid=4977598&amp;link_uid=20223290">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676867/" ref="ordinalpos=3644&amp;ncbi_uid=4977598&amp;link_uid=PMC3676867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3645.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3051915/figure/F6/" ref="ordinalpos=3645&amp;ncbi_uid=2822957&amp;link_uid=PMC3051915" href="/pmc/articles/PMC3051915/"><img src="/pmc/articles/instance/3051915/bin/1471-213X-11-12-6.gif" src-large="/pmc/articles/instance/3051915/bin/1471-213X-11-12-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3051915/" ref="ordinalpos=3645&amp;ncbi_uid=2822957&amp;link_uid=PMC3051915" image-link="/pmc/articles/PMC3051915/figure/F6/" class="imagepopup">Figure 6.  From: Notch <span class="highlight" style="background-color:">signaling</span> regulates remodeling and vessel diameter in the extraembryonic yolk sac. </a></p><div class="supp"><p class="details"><div><b>Notch regulates the expression of key genes and the remodeling of the yolk sac vasculature</b>. (A) Proposed model depicts the Notch <span class="highlight" style="background-color:">pathway</span> as a key component in the regulation of the remodeling of the vasculature. In wild type yolk sac Notch acts in concert with VEGF and other <span class="highlight" style="background-color:">signaling</span> families to direct the integration of the endothelial cells into both large and small caliber vessels. When Notch is activated this integration is increased leading to very large caliber vessels with limited intra-vascular space. When Notch activity is abrogated the integration is limited and the vascular retains the simple unremodeled appearance of the early vascular plexus. (B) Proposed model of <span class="highlight" style="background-color:">signaling</span> in the endodermal and endothelial cells of the early yolk sac. <span class="highlight" style="background-color:">Signaling</span> from the endoderm and regulation from blood flow activates Notch <span class="highlight" style="background-color:">signaling</span> in select endothelial cells. Notch activates the expression of select secreted genes that act in both a paracrine and juxtacrine manner to direct endothelial cell migration and integration within the developing vasculature.</div></p></div><div class="aux"><div class="resc">Jessica N Copeland, et al. BMC Dev Biol. 2011;11:12-12.</div><p class="links"><a class="dblinks" href="/pubmed/21352545" ref="ordinalpos=3645&amp;ncbi_uid=2822957&amp;link_uid=21352545">Citation</a><a class="dblinks" href="/pmc/articles/PMC3051915/" ref="ordinalpos=3645&amp;ncbi_uid=2822957&amp;link_uid=PMC3051915">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3646.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3125286/figure/F1/" ref="ordinalpos=3646&amp;ncbi_uid=3044550&amp;link_uid=PMC3125286" href="/pmc/articles/PMC3125286/"><img src="/pmc/articles/instance/3125286/bin/1471-2172-12-28-1.gif" src-large="/pmc/articles/instance/3125286/bin/1471-2172-12-28-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3125286/" ref="ordinalpos=3646&amp;ncbi_uid=3044550&amp;link_uid=PMC3125286" image-link="/pmc/articles/PMC3125286/figure/F1/" class="imagepopup">Figure 1.  From: Nitric oxide/cGMP <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> actively down-regulates ?4?1-integrin affinity: an unexpected mechanism for inducing cell de-adhesion. </a></p><div class="supp"><p class="details"><div><b>NO/cGMP <span class="highlight" style="background-color:">signaling</span> cascade and small molecules that modulate this <span class="highlight" style="background-color:">pathway</span>, used in the current study</b>. Nitric oxide, generated by nitric oxide synthase, diffuses across the plasma membrane and through the cytoplasm. In leukocytes NO reacts with the active site of guanylyl cyclase (guanylate GC), and stimulates the production of the intracellular mediator cyclic GMP (cGMP). Next, cGMP interacts with the cGMP-dependent protein kinase (PKG), which phosphorylates multiple substrates, and participates in signal propagation. Cyclic nucleotide phosphodiesterases (PDEs, not shown) can rapidly hydrolyze cGMP and terminate signal propagation. The NO/cGMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be specifically targeted using small molecules. The nitric oxide donor provides an exogenous source of NO. The activator of soluble guanylyl cyclase binds to GC, and induces enzyme activation in the absence of NO. The cell permeable analog of cGMP diffuses across the plasma membrane, and thus, activates cGMP-dependent <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Alexandre Chigaev, et al. BMC Immunol. 2011;12:28-28.</div><p class="links"><a class="dblinks" href="/pubmed/21586157" ref="ordinalpos=3646&amp;ncbi_uid=3044550&amp;link_uid=21586157">Citation</a><a class="dblinks" href="/pmc/articles/PMC3125286/" ref="ordinalpos=3646&amp;ncbi_uid=3044550&amp;link_uid=PMC3125286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3647.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4229029/figure/F1/" ref="ordinalpos=3647&amp;ncbi_uid=6829018&amp;link_uid=PMC4229029" href="/pmc/articles/PMC4229029/"><img src="/pmc/articles/instance/4229029/bin/nihms352986f1.gif" src-large="/pmc/articles/instance/4229029/bin/nihms352986f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4229029/" ref="ordinalpos=3647&amp;ncbi_uid=6829018&amp;link_uid=PMC4229029" image-link="/pmc/articles/PMC4229029/figure/F1/" class="imagepopup">Fig. 1. Potential mechanism for resistance to molecularly targeted therapies.  From: Resistance to Chemotherapy and Molecularly Targeted Therapies: Rationale for Combination Therapy in Malignant Melanoma. </a></p><div class="supp"><p class="details"><div><b>A</b>. Extracellular growth factors bind with cell receptors, activating a series of intricately regulated <span class="highlight" style="background-color:">signaling</span>. Tumor cells retain some levels of feedback inhibition, activation of one <span class="highlight" style="background-color:">pathway</span> might lead to inhibition of upstream substrates within the <span class="highlight" style="background-color:">pathway</span>, and/or the inhibition of compensatory pathways. Tumor cells depend on certain oncogenes for growth (“oncogene addiction”), these oncogene products provide good targets for molecularly targeted therapies. <b>B</b>. Molecular inhibitors suppress the overactivated growth/survival <span class="highlight" style="background-color:">pathway</span>, therefore tumors shrink. <b>C</b>. After prolonged treatment, induction of growth/survival signals of the same <span class="highlight" style="background-color:">pathway</span> and/or compensatory pathways leads to resistance against molecular inhibitors, which might be due to the disrupted feedback inhibitions. +++, overactivated <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">S. Wu, et al. Curr Mol Med. ;11(7):553-563.</div><p class="links"><a class="dblinks" href="/pubmed/21707515" ref="ordinalpos=3647&amp;ncbi_uid=6829018&amp;link_uid=21707515">Citation</a><a class="dblinks" href="/pmc/articles/PMC4229029/" ref="ordinalpos=3647&amp;ncbi_uid=6829018&amp;link_uid=PMC4229029">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3648.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553141/figure/fig3/" ref="ordinalpos=3648&amp;ncbi_uid=1509441&amp;link_uid=PMC2553141" href="/pmc/articles/PMC2553141/"><img src="/pmc/articles/instance/2553141/bin/BIO.135483.gs.f3.gif" src-large="/pmc/articles/instance/2553141/bin/BIO.135483.gs.f3.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553141/" ref="ordinalpos=3648&amp;ncbi_uid=1509441&amp;link_uid=PMC2553141" image-link="/pmc/articles/PMC2553141/figure/fig3/" class="imagepopup">FIGURE 3.  From: Monte Carlo Simulation of Cell Death <span class="highlight" style="background-color:">Signaling</span> Predicts Large Cell-to-Cell Stochastic Fluctuations through the Type 2 <span class="highlight" style="background-color:">Pathway</span> of Apoptosis. </a></p><div class="supp"><p class="details"><div>Distribution of activated caspase-3 molecules for various time points. (<i>Left</i>) Gradual increase in caspase-3 for the type 1 <span class="highlight" style="background-color:">pathway</span> and (<i>right</i>) bimodal distribution for the type 2 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Subhadip Raychaudhuri, et al. Biophys J. 2008 October 15;95(8):3559-3562.</div><p class="links"><a class="dblinks" href="/pubmed/18641073" ref="ordinalpos=3648&amp;ncbi_uid=1509441&amp;link_uid=18641073">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553141/" ref="ordinalpos=3648&amp;ncbi_uid=1509441&amp;link_uid=PMC2553141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3649.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4158574/figure/f3/" ref="ordinalpos=3649&amp;ncbi_uid=6603580&amp;link_uid=PMC4158574" href="/pmc/articles/PMC4158574/"><img src="/pmc/articles/instance/4158574/bin/srep06304-f3.gif" src-large="/pmc/articles/instance/4158574/bin/srep06304-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4158574/" ref="ordinalpos=3649&amp;ncbi_uid=6603580&amp;link_uid=PMC4158574" image-link="/pmc/articles/PMC4158574/figure/f3/" class="imagepopup">Figure 3. Hypoxia activates p38/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Hypoxia regulates CD9-mediated keratinocyte migration via the P38/MAPK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) Representative cropped blots and data summary of phospho-p38 (p-p38) and p38 in HaCaT cells and MKs under normoxic and hypoxic conditions. Representative western blots are shown for the two groups. (B) The graph represents the mean ± SD (n = 3) of the relative integrated signals. N, normoxia; *P &lt; 0.05 versus N group.</div></p></div><div class="aux"><div class="resc">Xupin Jiang, et al. Sci Rep. 2014;4:6304.</div><p class="links"><a class="dblinks" href="/pubmed/25200404" ref="ordinalpos=3649&amp;ncbi_uid=6603580&amp;link_uid=25200404">Citation</a><a class="dblinks" href="/pmc/articles/PMC4158574/" ref="ordinalpos=3649&amp;ncbi_uid=6603580&amp;link_uid=PMC4158574">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3650.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2889938/figure/F4/" ref="ordinalpos=3650&amp;ncbi_uid=2342704&amp;link_uid=PMC2889938" href="/pmc/articles/PMC2889938/"><img src="/pmc/articles/instance/2889938/bin/1471-2105-11-308-4.gif" src-large="/pmc/articles/instance/2889938/bin/1471-2105-11-308-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2889938/" ref="ordinalpos=3650&amp;ncbi_uid=2342704&amp;link_uid=PMC2889938" image-link="/pmc/articles/PMC2889938/figure/F4/" class="imagepopup">Figure 4.  From: Computational modeling with forward and reverse engineering links <span class="highlight" style="background-color:">signaling</span> network and genomic regulatory responses: NF-?B <span class="highlight" style="background-color:">signaling</span>-induced gene expression responses in inflammation. </a></p><div class="supp"><p class="details"><div><b>Three examples of NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> simulation</b>. The NF-κB profile can be obtained by using various IKK profiles with different intensities and stimulation durations in the IKK-NF-κB <span class="highlight" style="background-color:">signaling</span> simulator. Three examples of IKK stimuli and the corresponding NF-κB profiles are shown. (A) The acute stimulus of IKK that has a peak at around two hours and a positive feedback control after ten hours. (B) A weak stimulus of IKK. (C) The IKK stimulus is not an impulse stimulation. It rises steeply during the first two hours and then increases slowly until ten hours.</div></p></div><div class="aux"><div class="resc">Shih Chi Peng, et al. BMC Bioinformatics. 2010;11:308-308.</div><p class="links"><a class="dblinks" href="/pubmed/20529327" ref="ordinalpos=3650&amp;ncbi_uid=2342704&amp;link_uid=20529327">Citation</a><a class="dblinks" href="/pmc/articles/PMC2889938/" ref="ordinalpos=3650&amp;ncbi_uid=2342704&amp;link_uid=PMC2889938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3651.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776707/figure/F7/" ref="ordinalpos=3651&amp;ncbi_uid=2432313&amp;link_uid=PMC2776707" href="/pmc/articles/PMC2776707/"><img src="/pmc/articles/instance/2776707/bin/nihms134503f7.gif" src-large="/pmc/articles/instance/2776707/bin/nihms134503f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776707/" ref="ordinalpos=3651&amp;ncbi_uid=2432313&amp;link_uid=PMC2776707" image-link="/pmc/articles/PMC2776707/figure/F7/" class="imagepopup">Fig. 7. A model for SKN-1 functions under normal conditions.  From: Condition-adapted stress and longevity gene regulation by Caenorhabditis elegans SKN-1/Nrf. </a></p><div class="supp"><p class="details"><div>A positive feedback interaction with <i>ins-7</i> and the DAF-2 <span class="highlight" style="background-color:">pathway</span> is featured. SKN-1 upregulates many genes that promote detoxification and stress resistance, and also downregulates genes that decrease stress resistance, lifespan, or both. Among the SKN-1-downregulated genes are both <i>ins-7</i> and <i>pdk-1</i> (not shown), each of which promotes DAF-2 <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> (see text). The DAF-2 <span class="highlight" style="background-color:">pathway</span> in turn inhibits SKN-1 (Tullet et al. 2008).</div></p></div><div class="aux"><div class="resc">Riva P. Oliveira, et al. Aging Cell. ;8(5):524-541.</div><p class="links"><a class="dblinks" href="/pubmed/19575768" ref="ordinalpos=3651&amp;ncbi_uid=2432313&amp;link_uid=19575768">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776707/" ref="ordinalpos=3651&amp;ncbi_uid=2432313&amp;link_uid=PMC2776707">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3652.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3003706/figure/F7/" ref="ordinalpos=3652&amp;ncbi_uid=2639418&amp;link_uid=PMC3003706" href="/pmc/articles/PMC3003706/"><img src="/pmc/articles/instance/3003706/bin/nihms-250984-f0007.gif" src-large="/pmc/articles/instance/3003706/bin/nihms-250984-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3003706/" ref="ordinalpos=3652&amp;ncbi_uid=2639418&amp;link_uid=PMC3003706" image-link="/pmc/articles/PMC3003706/figure/F7/" class="imagepopup">Figure 7. Schematic diagram showing that β1- and β3-mediated CLAN formation is the result of distinct <span class="highlight" style="background-color:">signaling</span> pathways.  From: Distinct ?1 and ?3 integrin pathways converge to regulate cross-linked actin network (CLAN) formation in human trabecular meshwork (HTM) cells. </a></p><div class="supp"><p class="details"><div>Both pathways are dependent upon a SFK, possibly Src. The β1 integrin-activated <span class="highlight" style="background-color:">pathway</span> involves PI3-K while the αvβ3-activated <span class="highlight" style="background-color:">pathway</span> involves CD47 and Rac1 <span class="highlight" style="background-color:">signaling</span> mediated by the GEF Trio. CD47 activation by TSP1 containing the carboxy-terminal sequence found in peptide 4N1K (RFYVVMWK) may be an important regulator of CLAN formation. Convergence of the two pathways could potentially occur at two points. The β1 integrin <span class="highlight" style="background-color:">pathway</span> might also signal through Rac1, but via activation of a Rac1 GEF distinct from Trio. Alternatively, PI-3K could bypass Rac1 completely and the two pathways would converge at a different downstream effector to promote CLAN formation.</div></p></div><div class="aux"><div class="resc">Mark S. Filla, et al. Invest Ophthalmol Vis Sci. ;50(12):5723-5731.</div><p class="links"><a class="dblinks" href="/pubmed/19643963" ref="ordinalpos=3652&amp;ncbi_uid=2639418&amp;link_uid=19643963">Citation</a><a class="dblinks" href="/pmc/articles/PMC3003706/" ref="ordinalpos=3652&amp;ncbi_uid=2639418&amp;link_uid=PMC3003706">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3653.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2692661/figure/F8/" ref="ordinalpos=3653&amp;ncbi_uid=1907121&amp;link_uid=PMC2692661" href="/pmc/articles/PMC2692661/"><img src="/pmc/articles/instance/2692661/bin/nihms14835f8.gif" src-large="/pmc/articles/instance/2692661/bin/nihms14835f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2692661/" ref="ordinalpos=3653&amp;ncbi_uid=1907121&amp;link_uid=PMC2692661" image-link="/pmc/articles/PMC2692661/figure/F8/" class="imagepopup">Figure 8.  From: A negative feedback <span class="highlight" style="background-color:">signaling</span> network underlies oncogene-induced senescence. </a></p><div class="supp"><p class="details"><div>Revised model of oncogene-induced senescence<br /><b>A:</b> In cells that are insensitive to oncogene-induced senescence, activation of the Ras <span class="highlight" style="background-color:">pathway</span> leads to hyper-activation of well known effector pathways which function to promote tumorigenesis.<br /><b>B:</b> In sensitive cells the ultimate response to the aberrant activation of the Ras <span class="highlight" style="background-color:">pathway</span> is the initiation of a multi-faceted negative feedback <span class="highlight" style="background-color:">signaling</span> network designed to terminate the oncogenic signal. These signals are triggered by the Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span> and involve numerous transcriptional and post-translational events, including the suppression of Ras exchange factors, and the up-regulation of Sprouty proteins and RasGAPs, among others. These inhibitory signals, via the consequential suppression of PI3K, can activate Rb and p53 through multiple signals. Thus, we propose that oncogene-induced senescence is mediated by this negative feedback <span class="highlight" style="background-color:">signaling</span> program, that functions along with other known senescence regulators, such as p16 and ARF, to achieve a threshold senescence signal.</div></p></div><div class="aux"><div class="resc">Stéphanie Courtois-Cox, et al. Cancer Cell. ;10(6):459-472.</div><p class="links"><a class="dblinks" href="/pubmed/17157787" ref="ordinalpos=3653&amp;ncbi_uid=1907121&amp;link_uid=17157787">Citation</a><a class="dblinks" href="/pmc/articles/PMC2692661/" ref="ordinalpos=3653&amp;ncbi_uid=1907121&amp;link_uid=PMC2692661">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3654.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4171604/figure/F7/" ref="ordinalpos=3654&amp;ncbi_uid=6643857&amp;link_uid=PMC4171604" href="/pmc/articles/PMC4171604/"><img src="/pmc/articles/instance/4171604/bin/oncotarget-05-5950-g007.gif" src-large="/pmc/articles/instance/4171604/bin/oncotarget-05-5950-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4171604/" ref="ordinalpos=3654&amp;ncbi_uid=6643857&amp;link_uid=PMC4171604" image-link="/pmc/articles/PMC4171604/figure/F7/" class="imagepopup">Figure 7. EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> deregulation in DU145 cells after hydralazine exposure.  From: Anti-neoplastic properties of hydralazine in prostate cancer. </a></p><div class="supp"><p class="details"><div>(A) Protein expression heat map of DU145exposed to 40 μM hydralazine (red: protein overexpression; green: protein reduced expression. Experiments were performed in triplicate. (B) Altered cellular pathways identified by DAVID software. EGF receptor <span class="highlight" style="background-color:">pathway</span> showed the lowest p value (10<sup>−8</sup>). (C) Western Blot validation of disrupted targets of EGF receptor <span class="highlight" style="background-color:">pathway</span> in DU145 cell line upon hydralazine exposure. β-Actin was used as loading control</div></p></div><div class="aux"><div class="resc">Inês Graça, et al. Oncotarget. 2014 August;5(15):5950-5964.</div><p class="links"><a class="dblinks" href="/pubmed/24797896" ref="ordinalpos=3654&amp;ncbi_uid=6643857&amp;link_uid=24797896">Citation</a><a class="dblinks" href="/pmc/articles/PMC4171604/" ref="ordinalpos=3654&amp;ncbi_uid=6643857&amp;link_uid=PMC4171604">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3655.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3719407/figure/F4/" ref="ordinalpos=3655&amp;ncbi_uid=5120781&amp;link_uid=PMC3719407" href="/pmc/articles/PMC3719407/"><img src="/pmc/articles/instance/3719407/bin/nihms480818f4.gif" src-large="/pmc/articles/instance/3719407/bin/nihms480818f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3719407/" ref="ordinalpos=3655&amp;ncbi_uid=5120781&amp;link_uid=PMC3719407" image-link="/pmc/articles/PMC3719407/figure/F4/" class="imagepopup">Fig. 4.  From: Interaction between gonadotropin-releasing hormone and bone morphogenetic protein-6 and -7 <span class="highlight" style="background-color:">signaling</span> in L?T2 gonadotrope cells. </a></p><div class="supp"><p class="details"><div>A functional link between the BMP system and GnRH-induced FSH transcription. BMP-6 and GnRH mutually augment FSHβ transcription in LβT2 cells. GnRH activates FSHβ transcription by activating the ERK <span class="highlight" style="background-color:">pathway</span>, which is negatively regulated by BMP-7. GnRH also provides a feedback inhibition of BMP-Smad <span class="highlight" style="background-color:">signaling</span> by suppressing Smad1/5/8, in part via increasing Smad6/7 expression, and reducing endogenous BMP-7 expression. There is a functional link between GnRH action and the BMP system for regulating FSHβ transcription in gonadotrope cells.</div></p></div><div class="aux"><div class="resc">Masaya Takeda, et al. Mol Cell Endocrinol. ;348(1):147-154.</div><p class="links"><a class="dblinks" href="/pubmed/21846488" ref="ordinalpos=3655&amp;ncbi_uid=5120781&amp;link_uid=21846488">Citation</a><a class="dblinks" href="/pmc/articles/PMC3719407/" ref="ordinalpos=3655&amp;ncbi_uid=5120781&amp;link_uid=PMC3719407">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3656.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3142412/figure/F1/" ref="ordinalpos=3656&amp;ncbi_uid=3176548&amp;link_uid=PMC3142412" href="/pmc/articles/PMC3142412/"><img src="/pmc/articles/instance/3142412/bin/psb0603_0339_fig001.gif" src-large="/pmc/articles/instance/3142412/bin/psb0603_0339_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3142412/" ref="ordinalpos=3656&amp;ncbi_uid=3176548&amp;link_uid=PMC3142412" image-link="/pmc/articles/PMC3142412/figure/F1/" class="imagepopup">Figure 1.  From: Information flow in plant <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>The ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is composed of two modules which are mutually exclusive in their effects on the phenotype of the root cell. The MAPK module promotes an auxin dependent phenotype due to the positive action of the CTR1 kinase on <i>POLARIS</i> expression. The ethylene module promotes an ethylene dependent phenotype due to the activation of <i>ERF1</i>. When the ethylene receptor is switched on (absence of ethylene) the MAPK module is activated and the ethylene module is inactivated. When the ethylene receptor is switched <i>off</i> by ethylene, the ethylene module is activated and the MAPK module is inactivated.</div></p></div><div class="aux"><div class="resc">José Díaz. Plant Signal Behav. 2011 March;6(3):339-343.</div><p class="links"><a class="dblinks" href="/pubmed/21368577" ref="ordinalpos=3656&amp;ncbi_uid=3176548&amp;link_uid=21368577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3142412/" ref="ordinalpos=3656&amp;ncbi_uid=3176548&amp;link_uid=PMC3142412">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3657.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1350949/figure/F7/" ref="ordinalpos=3657&amp;ncbi_uid=570309&amp;link_uid=PMC1350949" href="/pmc/articles/PMC1350949/"><img src="/pmc/articles/instance/1350949/bin/nihms5852f7.gif" src-large="/pmc/articles/instance/1350949/bin/nihms5852f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1350949/" ref="ordinalpos=3657&amp;ncbi_uid=570309&amp;link_uid=PMC1350949" image-link="/pmc/articles/PMC1350949/figure/F7/" class="imagepopup">Figure 7.  From: Human Cytomegalovirus Causes Endothelial Injury Through the Ataxia Telangiectasia Mutant and p53 DNA Damage <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Proposed model of the HCMV-induced apoptotic <span class="highlight" style="background-color:">signaling</span> pathways. HCMV infection causes genotoxic stress, which activates ATM kinase. The activated ATM can activate p53 either directly or through activation of Chk2. The activation of p53 leads to the activation of mitochondrial <span class="highlight" style="background-color:">pathway</span>, ie, Bax and Bak, hence the activation of caspase-9 and caspase-3 resulting in cell apoptosis.</div></p></div><div class="aux"><div class="resc">Y.H. Shen, et al. Circ Res. ;94(10):1310-1317.</div><p class="links"><a class="dblinks" href="/pubmed/15105295" ref="ordinalpos=3657&amp;ncbi_uid=570309&amp;link_uid=15105295">Citation</a><a class="dblinks" href="/pmc/articles/PMC1350949/" ref="ordinalpos=3657&amp;ncbi_uid=570309&amp;link_uid=PMC1350949">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3658.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3854019/figure/pharmaceuticals-06-01429-f001/" ref="ordinalpos=3658&amp;ncbi_uid=5605112&amp;link_uid=PMC3854019" href="/pmc/articles/PMC3854019/"><img src="/pmc/articles/instance/3854019/bin/pharmaceuticals-06-01429-g001.gif" src-large="/pmc/articles/instance/3854019/bin/pharmaceuticals-06-01429-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3854019/" ref="ordinalpos=3658&amp;ncbi_uid=5605112&amp;link_uid=PMC3854019" image-link="/pmc/articles/PMC3854019/figure/pharmaceuticals-06-01429-f001/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">Pathway</span> Blockade Inhibits Melanoma Cell Growth in Vitro and in Vivo. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">pathway</span> members are upregulated in a subset of melanoma cell lines as measured via affymetrix microarray analysis. SMO, GLI2, PTCH1, and WNT5a are upregulated, while negative regulators of hedgehog <span class="highlight" style="background-color:">signaling</span> such as protein kinase A (PRKA) and dual specificity tyrosine-phosphorylation-regulated kinase 2 (DYRK2) are downregulated as compared to melanocytes.</div></p></div><div class="aux"><div class="resc">Kathryn E. O’Reilly, et al. Pharmaceuticals (Basel). 2013 November;6(11):1429-1450.</div><p class="links"><a class="dblinks" href="/pubmed/24287465" ref="ordinalpos=3658&amp;ncbi_uid=5605112&amp;link_uid=24287465">Citation</a><a class="dblinks" href="/pmc/articles/PMC3854019/" ref="ordinalpos=3658&amp;ncbi_uid=5605112&amp;link_uid=PMC3854019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3659.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3943199/figure/fig4/" ref="ordinalpos=3659&amp;ncbi_uid=5958731&amp;link_uid=PMC3943199" href="/pmc/articles/PMC3943199/"><img src="/pmc/articles/instance/3943199/bin/ECAM2014-982165.004.gif" src-large="/pmc/articles/instance/3943199/bin/ECAM2014-982165.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3943199/" ref="ordinalpos=3659&amp;ncbi_uid=5958731&amp;link_uid=PMC3943199" image-link="/pmc/articles/PMC3943199/figure/fig4/" class="imagepopup">Figure 4.  From: Hexane Fractions of Bupleurum falcatum L. Stimulates Glucagon-Like Peptide-1 Secretion through Gβγ-Mediated <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>HFBF stimulates GLP-1 secretion through G<sub><i>βγ</i></sub> <span class="highlight" style="background-color:">pathway</span>. Treatment of BF hexane fraction at 100 seconds into the medium increased the concentration of Ca<sup>2+</sup> in cytosol. With BF hexane fraction, inhibitors were treated. (a) Gallein which is an inhibitor of G<sub><i>βγ</i></sub> subunit-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reduced GLP-1 secretion dose dependently. (b) 2APB inhibits the IP<sub>3</sub>-induced Ca<sup>2+</sup> release. 2APB also reduced GLP-1 secretion dose dependently. (c) U73122 is an inhibitor of PLC-dependent processes. With this inhibitor, concentration of Ca<sup>2+</sup> showed less change despite treatment of BF hexane fraction. Data are means of intensity of the NCI-H716 cells (<i>n</i> = 10). ***<i>P</i> &lt; 0.0001 versus BF hexane fraction 100 <i>μ</i>g/mL.</div></p></div><div class="aux"><div class="resc">Min-Hee Shin, et al. Evid Based Complement Alternat Med. 2014;2014:982165.</div><p class="links"><a class="dblinks" href="/pubmed/24688594" ref="ordinalpos=3659&amp;ncbi_uid=5958731&amp;link_uid=24688594">Citation</a><a class="dblinks" href="/pmc/articles/PMC3943199/" ref="ordinalpos=3659&amp;ncbi_uid=5958731&amp;link_uid=PMC3943199">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3660.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC20445/figure/F6/" ref="ordinalpos=3660&amp;ncbi_uid=91061&amp;link_uid=PMC20445" href="/pmc/articles/PMC20445/"><img src="/pmc/articles/instance/20445/bin/pq0980676006.gif" src-large="/pmc/articles/instance/20445/bin/pq0980676006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC20445/" ref="ordinalpos=3660&amp;ncbi_uid=91061&amp;link_uid=PMC20445" image-link="/pmc/articles/PMC20445/figure/F6/" class="imagepopup">Figure 6.  From: Activation of NF-?B via a Src-dependent Ras-MAPK-pp90rsk <span class="highlight" style="background-color:">pathway</span> is required for Pseudomonas aeruginosa-induced mucin overproduction in epithelial cells. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing steps in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which <i>P. aeruginosa</i> up-regulates human <i>MUC2</i> mucin gene transcription. As indicated, <i>P. aeruginosa</i> releases LPS, thereby activating a c-Src-Ras-Raf-1-MEK1/2-MAPK (ERK1/2)-pp90rsk <span class="highlight" style="background-color:">pathway</span>, which in turn leads to the activation of NF-κB and triggers <i>MUC2</i> mucin transcription. LBP, LPS-binding protein. The overproduced mucin, in concert with abnormal airway lining fluid secondary to CFTR mutation, leads to airway mucus obstruction and lung failure.</div></p></div><div class="aux"><div class="resc">Jian-Dong Li, et al. Proc Natl Acad Sci U S A. 1998 May 12;95(10):5718-5723.</div><p class="links"><a class="dblinks" href="/pubmed/9576950" ref="ordinalpos=3660&amp;ncbi_uid=91061&amp;link_uid=9576950">Citation</a><a class="dblinks" href="/pmc/articles/PMC20445/" ref="ordinalpos=3660&amp;ncbi_uid=91061&amp;link_uid=PMC20445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3661.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355926/figure/F1/" ref="ordinalpos=3661&amp;ncbi_uid=4027555&amp;link_uid=PMC3355926" href="/pmc/articles/PMC3355926/"><img src="/pmc/articles/instance/3355926/bin/fendo-02-00054-g001.gif" src-large="/pmc/articles/instance/3355926/bin/fendo-02-00054-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355926/" ref="ordinalpos=3661&amp;ncbi_uid=4027555&amp;link_uid=PMC3355926" image-link="/pmc/articles/PMC3355926/figure/F1/" class="imagepopup">Figure 1.  From: Insulin <span class="highlight" style="background-color:">Signaling</span> and Autism. </a></p><div class="supp"><p class="details"><div><b>Activation of the Pi3K/Tor <span class="highlight" style="background-color:">pathway</span> by insulin and glutamate</b>. The relationships among the genes described in the text are shown. Proteins shown in red are PI3K/Tor <span class="highlight" style="background-color:">pathway</span> inhibitors, and autism is associated with loss of function mutation in these genes, whereas the proteins shown in green are PI3K/Tor <span class="highlight" style="background-color:">pathway</span> activators and autism is associated with gain of function alterations in these genes. Proteins shown in blue represent other <span class="highlight" style="background-color:">pathway</span> activators. For clarity, several molecular intermediates are omitted. See Hay and Sonenberg (2004) for a more complete molecular description of this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Michael Stern. Front Endocrinol (Lausanne). 2011;2:54.</div><p class="links"><a class="dblinks" href="/pubmed/22649376" ref="ordinalpos=3661&amp;ncbi_uid=4027555&amp;link_uid=22649376">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355926/" ref="ordinalpos=3661&amp;ncbi_uid=4027555&amp;link_uid=PMC3355926">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3662.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281614/figure/F6/" ref="ordinalpos=3662&amp;ncbi_uid=4608377&amp;link_uid=PMC3281614" href="/pmc/articles/PMC3281614/"><img src="/pmc/articles/instance/3281614/bin/zbc0071296390006.gif" src-large="/pmc/articles/instance/3281614/bin/zbc0071296390006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281614/" ref="ordinalpos=3662&amp;ncbi_uid=4608377&amp;link_uid=PMC3281614" image-link="/pmc/articles/PMC3281614/figure/F6/" class="imagepopup">FIGURE 6. From: Structural Conservation of Ligand Binding Reveals a Bile Acid-like <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Nematodes. </a></p><div class="supp"><p class="details"><div><b>Model of conserved bile acid-like <span class="highlight" style="background-color:">signaling</span> pathways in nematodes and mammals.</b> Bile acid-like <span class="highlight" style="background-color:">signaling</span> pathways in mammals and nematodes are compared and illustrated. Free-living nematodes (<i>e.g. C. elegans</i>) produce their own DAF-12 ligands to regulate the reproductive development. Parasitic nematode (<i>e.g.</i> hookworms) might use the host bile acid(-like) metabolites to activate their DAF-12 to regulate the reproductive development.</div></p></div><div class="aux"><div class="resc">Xiaoyong Zhi, et al. J Biol Chem. 2012 February 10;287(7):4894-4903.</div><p class="links"><a class="dblinks" href="/pubmed/22170062" ref="ordinalpos=3662&amp;ncbi_uid=4608377&amp;link_uid=22170062">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281614/" ref="ordinalpos=3662&amp;ncbi_uid=4608377&amp;link_uid=PMC3281614">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3663.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1851854/figure/F7-7129/" ref="ordinalpos=3663&amp;ncbi_uid=932830&amp;link_uid=PMC1851854" href="/pmc/articles/PMC1851854/"><img src="/pmc/articles/instance/1851854/bin/zjh0020771290007.gif" src-large="/pmc/articles/instance/1851854/bin/zjh0020771290007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1851854/" ref="ordinalpos=3663&amp;ncbi_uid=932830&amp;link_uid=PMC1851854" image-link="/pmc/articles/PMC1851854/figure/F7-7129/" class="imagepopup">Figure 7.  From: Repression of Repulsive Guidance Molecule C during Inflammation Is Independent of Hfe and Involves Tumor Necrosis Factor-?. </a></p><div class="supp"><p class="details"><div>Model showing the iron-sensing and inflammatory pathways for Rgmc and hepcidin regulation. Iron-sensing <span class="highlight" style="background-color:">pathway</span>: Hfe/β2m and Rgmc regulate basal hepcidin expression independently. Iron-sensing by Rgmc does not involve modulation of <i>Rgmc</i> mRNA expression levels, but Rgmc is necessary for the regulation of hepcidin levels through the iron sensing-<span class="highlight" style="background-color:">pathway</span>.22 Inflammatory <span class="highlight" style="background-color:">pathway</span>: mRNA <i>Rgmc</i> down-regulation by LPS involves <span class="highlight" style="background-color:">signaling</span> through Tlr4 and consequent production of TNF-α. Down-regulation of <i>Rgmc</i> suppresses hepcidin control through the iron-sensing <span class="highlight" style="background-color:">pathway</span> and allows hepcidin induction by IL-6.</div></p></div><div class="aux"><div class="resc">Marco Constante, et al. Am J Pathol. 2007 February;170(2):497-504.</div><p class="links"><a class="dblinks" href="/pubmed/17255318" ref="ordinalpos=3663&amp;ncbi_uid=932830&amp;link_uid=17255318">Citation</a><a class="dblinks" href="/pmc/articles/PMC1851854/" ref="ordinalpos=3663&amp;ncbi_uid=932830&amp;link_uid=PMC1851854">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3664.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3444627/figure/F5/" ref="ordinalpos=3664&amp;ncbi_uid=5441054&amp;link_uid=PMC3444627" href="/pmc/articles/PMC3444627/"><img src="/pmc/articles/instance/3444627/bin/nihms-402572-f0005.gif" src-large="/pmc/articles/instance/3444627/bin/nihms-402572-f0005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3444627/" ref="ordinalpos=3664&amp;ncbi_uid=5441054&amp;link_uid=PMC3444627" image-link="/pmc/articles/PMC3444627/figure/F5/" class="imagepopup">Fig. 5. Parameter sensitivity analysis of the intracellular <span class="highlight" style="background-color:">pathway</span> model.  From: Cytokine Combination Therapy Prediction for Bone Remodeling in Tissue Engineering Based on the Intracellular <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Above subfigure shows that the expression of Runx2 is more sensitivity to <i>V<sub>6</sub>, K<sub>6</sub></i> and <i>d<sub>4</sub></i>. Below subfigure shows that the expression of Osx is more sensitivity to <i>V<sub>8</sub>, K<sub>8</sub></i> and <i>d<sub>8</sub></i>, as well as to <i>V<sub>1</sub>, K<sub>1</sub>, d<sub>1</sub></i> and <i>V<sub>7</sub>, K<sub>7</sub></i>. These results show us which kinetic parameters and molecules are more important in the system and also demonstrate that the developed intracellular <span class="highlight" style="background-color:">pathway</span> system is robust.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Sun, et al. Biomaterials. 2012 November;33(33):8265-8276.</div><p class="links"><a class="dblinks" href="/pubmed/22910219" ref="ordinalpos=3664&amp;ncbi_uid=5441054&amp;link_uid=22910219">Citation</a><a class="dblinks" href="/pmc/articles/PMC3444627/" ref="ordinalpos=3664&amp;ncbi_uid=5441054&amp;link_uid=PMC3444627">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3665.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404563/figure/F3/" ref="ordinalpos=3665&amp;ncbi_uid=5041951&amp;link_uid=PMC3404563" href="/pmc/articles/PMC3404563/"><img src="/pmc/articles/instance/3404563/bin/zh10131265870003.gif" src-large="/pmc/articles/instance/3404563/bin/zh10131265870003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404563/" ref="ordinalpos=3665&amp;ncbi_uid=5041951&amp;link_uid=PMC3404563" image-link="/pmc/articles/PMC3404563/figure/F3/" class="imagepopup">Fig. 3. From: Nodal induces apoptosis through activation of the ALK7 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pancreatic INS-1 ?-cells. </a></p><div class="supp"><p class="details"><div>Pathophysiological stimulus-induced apoptosis is associated with upregulation of Nodal expression and activation of activin receptor-like kinase-7 (ALK7)-Smad3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in INS-1 cells. <i>A</i>: Western blotting using protein extracts (25 μg) from INS-1 cells treated with 30 mM HG or 0.4 mM Pal or Cyto cocktail (IL-1β 10 ng/ml, TNF-α 50 ng/ml, and IFN-γ 50 ng/ml) or STZ (0.5 mg/ml) for 24 h and probed for Nodal, ALK7, phosphorylated Smad3, active caspase-3, or GAPDH as loading control. <i>B</i>–<i>E</i>: bar graphs represent densitometry analysis. Data are means ± SE; <i>n</i> = 3–5. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Fang Zhao, et al. Am J Physiol Endocrinol Metab. 2012 July 1;303(1):E132-E143.</div><p class="links"><a class="dblinks" href="/pubmed/22550067" ref="ordinalpos=3665&amp;ncbi_uid=5041951&amp;link_uid=22550067">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404563/" ref="ordinalpos=3665&amp;ncbi_uid=5041951&amp;link_uid=PMC3404563">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3666.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3618283/figure/F1/" ref="ordinalpos=3666&amp;ncbi_uid=4785377&amp;link_uid=PMC3618283" href="/pmc/articles/PMC3618283/"><img src="/pmc/articles/instance/3618283/bin/nihms451269f1.gif" src-large="/pmc/articles/instance/3618283/bin/nihms451269f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3618283/" ref="ordinalpos=3666&amp;ncbi_uid=4785377&amp;link_uid=PMC3618283" image-link="/pmc/articles/PMC3618283/figure/F1/" class="imagepopup">Figure 1.  From: Antipsychotic drug mechanisms: links between therapeutic effects, metabolic side effects and the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A simplified version of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Insulin or insulin-like growth factor (IGF) binds to the insulin/IGF receptor causing autophosphorylation and activation. This leads to activation of phosphoinositide-3-kinase (PI3kinase), and the end result is to phosphorylate and activate Akt and phosphorylate and deactivate glycogen synthase kinase-3 (GSK-3). Active GSK-3 deactivates glycogen synthase and eukaryotic initiation factor 2B (eIF2B) and activates insulin receptor substrate-1 (IRS-1; an inhibitor of the insulin receptor). PIP3, phosphoinositide 3,4,5-tris-phosphate; PKC, protein kinase C (adapted and modified with permission from Gould and Manji<sup>49</sup>).</div></p></div><div class="aux"><div class="resc">RR Girgis, et al. Mol Psychiatry. 2008 October;13(10):918-929.</div><p class="links"><a class="dblinks" href="/pubmed/18414407" ref="ordinalpos=3666&amp;ncbi_uid=4785377&amp;link_uid=18414407">Citation</a><a class="dblinks" href="/pmc/articles/PMC3618283/" ref="ordinalpos=3666&amp;ncbi_uid=4785377&amp;link_uid=PMC3618283">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3667.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2932783/figure/F1/" ref="ordinalpos=3667&amp;ncbi_uid=3185419&amp;link_uid=PMC2932783" href="/pmc/articles/PMC2932783/"><img src="/pmc/articles/instance/2932783/bin/nihms218170f1.gif" src-large="/pmc/articles/instance/2932783/bin/nihms218170f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2932783/" ref="ordinalpos=3667&amp;ncbi_uid=3185419&amp;link_uid=PMC2932783" image-link="/pmc/articles/PMC2932783/figure/F1/" class="imagepopup">Figure 1.  From: Arsenic Trioxide and the PI3K/AKT <span class="highlight" style="background-color:">Pathway</span> in Chronic Lymphocytic Leukemia. </a></p><div class="supp"><p class="details"><div>Targeting of <span class="highlight" style="background-color:">signaling</span> pathways by arsenic trioxide in chronic lymphocytic leukemia cells. Activation of the JNK MAPK <span class="highlight" style="background-color:">pathway</span> is required for induction of apoptosis of CLL cells. Arsenic trioxide treatment of leukemic cells also results in inhibition of the PI 3’K/AKT <span class="highlight" style="background-color:">pathway</span>; and pharmacological targeting of this <span class="highlight" style="background-color:">pathway</span> enhance the antileukemic effects of arsenic trioxide. The potential involvement of other MAPK pathways, such as the p38 MAPK and MEK/ERK pathways which play important roles in control of growth and survival of other types of leukemic cells, in the regulation ATO-dependent responses in CLL cells remains to be defined.</div></p></div><div class="aux"><div class="resc">Dennis J. Goussetis, et al. Clin Cancer Res. ;16(17):4311-4312.</div><p class="links"><a class="dblinks" href="/pubmed/20622048" ref="ordinalpos=3667&amp;ncbi_uid=3185419&amp;link_uid=20622048">Citation</a><a class="dblinks" href="/pmc/articles/PMC2932783/" ref="ordinalpos=3667&amp;ncbi_uid=3185419&amp;link_uid=PMC2932783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3668.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3813785/figure/f4-ol-06-05-1201/" ref="ordinalpos=3668&amp;ncbi_uid=5442757&amp;link_uid=PMC3813785" href="/pmc/articles/PMC3813785/"><img src="/pmc/articles/instance/3813785/bin/OL-06-05-1201-g03.gif" src-large="/pmc/articles/instance/3813785/bin/OL-06-05-1201-g03.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3813785/" ref="ordinalpos=3668&amp;ncbi_uid=5442757&amp;link_uid=PMC3813785" image-link="/pmc/articles/PMC3813785/figure/f4-ol-06-05-1201/" class="imagepopup">Figure 4.  From: Notch <span class="highlight" style="background-color:">signaling</span> induces EMT in OSCC cell lines in a hypoxic environment. </a></p><div class="supp"><p class="details"><div>(A) Following culture under hypoxia or normoxia, invasive cells were stained using Diff-Quik for counting. (B) The ratio of invaded cells was significantly increased by hypoxia compared with that under normoxia in HSC-2, HSC-4 and Ca9-22 cells (<sup>*</sup>P&lt;0.05). (C) To analyze whether hypoxia enhanced cell invasion through the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, cells were cultured with or without 5 μM GSI under hypoxia for 48 h. (D) GSI prevented cell invasion significantly in OSCC cell lines (<sup>*</sup>P&lt;0.05 and <sup>**</sup>P&lt;0.01). These findings suggested that hypoxia promoted the invasiveness of OSCC cell lines through the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. GSI, γ-secretase inhibitor; OSCC, oral squamous cell carcinoma.</div></p></div><div class="aux"><div class="resc">TAKAYUKI ISHIDA, et al. Oncol Lett. 2013 November;6(5):1201-1206.</div><p class="links"><a class="dblinks" href="/pubmed/24179495" ref="ordinalpos=3668&amp;ncbi_uid=5442757&amp;link_uid=24179495">Citation</a><a class="dblinks" href="/pmc/articles/PMC3813785/" ref="ordinalpos=3668&amp;ncbi_uid=5442757&amp;link_uid=PMC3813785">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3669.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3571113/figure/F1/" ref="ordinalpos=3669&amp;ncbi_uid=4634611&amp;link_uid=PMC3571113" href="/pmc/articles/PMC3571113/"><img src="/pmc/articles/instance/3571113/bin/nihms-435869-f0001.gif" src-large="/pmc/articles/instance/3571113/bin/nihms-435869-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3571113/" ref="ordinalpos=3669&amp;ncbi_uid=4634611&amp;link_uid=PMC3571113" image-link="/pmc/articles/PMC3571113/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> and transcriptional regulation in osteoblast commitment and differentiation. </a></p><div class="supp"><p class="details"><div>Six important <span class="highlight" style="background-color:">signaling</span> networks of osteoblast differentiation. Binding of Wnt to the FZD receptor induces ß-catenin accumulation, which translocates to the nucleus to activate target gene transcription. Several transcription factors have been found crucial for osteoblast differentiation downstream of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, such as Runx2, Osterix, and ATF4. They are essential for differentiation of mesenchymal stem cells into differentiated osteoblasts and also function in the transcription of osteoclast-specific genes. The dotted lines indicate that the physiological function or the stage at which the factor mainly works remains to be proven.</div></p></div><div class="aux"><div class="resc">Wei Huang, et al. Front Biosci. ;12:3068-3092.</div><p class="links"><a class="dblinks" href="/pubmed/17485283" ref="ordinalpos=3669&amp;ncbi_uid=4634611&amp;link_uid=17485283">Citation</a><a class="dblinks" href="/pmc/articles/PMC3571113/" ref="ordinalpos=3669&amp;ncbi_uid=4634611&amp;link_uid=PMC3571113">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3670.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2918404/figure/F8/" ref="ordinalpos=3670&amp;ncbi_uid=2405092&amp;link_uid=PMC2918404" href="/pmc/articles/PMC2918404/"><img src="/pmc/articles/instance/2918404/bin/nihms215799f8.gif" src-large="/pmc/articles/instance/2918404/bin/nihms215799f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2918404/" ref="ordinalpos=3670&amp;ncbi_uid=2405092&amp;link_uid=PMC2918404" image-link="/pmc/articles/PMC2918404/figure/F8/" class="imagepopup">Figure 8. Cross-talk between the Runx2 and FGF2 <span class="highlight" style="background-color:">signaling</span> axes during osteoblastic lineage progression.  From: The Osteogenic Transcription Factor Runx2 Regulates Components of the Fibroblast Growth Factor/Proteoglycan <span class="highlight" style="background-color:">Signaling</span> Axis in Osteoblasts. </a></p><div class="supp"><p class="details"><div>The model depicts several well-known aspects of the FGF <span class="highlight" style="background-color:">signaling</span> cascade including the synergy between FGF <span class="highlight" style="background-color:">signaling</span> and proteoglycans, as well as downstream effects on FGF <span class="highlight" style="background-color:">signaling</span> on MAPK and CDK related pathways. We propose that Runx2 may participate in two distinct feed-back loops. In actively dividing cells, FGF2 is mitogenic and activates MAPKs and CDKs. This activation may both promote and attenuate Runx2 activity to generate short-term changes in proteoglycan expression that transiently modulate responsiveness to FGF2. In post-proliferative cells, FGF2 functions anabolically and CDK effects on Runx2 are blocked by CDK inhibitors (CKIs). Consequently, the FGF/MAPK/Runx2 <span class="highlight" style="background-color:">pathway</span> may generate a long-term sustained response in which Runx2 modulates a program of proteoglycan expression to promote osteoblast maturation. The ideas presented in the model are consistent with references presented in the main text, but remain to be experimentally tested.</div></p></div><div class="aux"><div class="resc">Nadiya M. Teplyuk, et al. J Cell Biochem. ;107(1):144-154.</div><p class="links"><a class="dblinks" href="/pubmed/19259985" ref="ordinalpos=3670&amp;ncbi_uid=2405092&amp;link_uid=19259985">Citation</a><a class="dblinks" href="/pmc/articles/PMC2918404/" ref="ordinalpos=3670&amp;ncbi_uid=2405092&amp;link_uid=PMC2918404">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3671.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634256/figure/F1/" ref="ordinalpos=3671&amp;ncbi_uid=1709728&amp;link_uid=PMC2634256" href="/pmc/articles/PMC2634256/"><img src="/pmc/articles/instance/2634256/bin/org0402_0109_fig001.gif" src-large="/pmc/articles/instance/2634256/bin/org0402_0109_fig001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634256/" ref="ordinalpos=3671&amp;ncbi_uid=1709728&amp;link_uid=PMC2634256" image-link="/pmc/articles/PMC2634256/figure/F1/" class="imagepopup">Fig 1.  From: Wnt <span class="highlight" style="background-color:">signaling</span> in limb organogenesis. </a></p><div class="supp"><p class="details"><div>Three Wnt <span class="highlight" style="background-color:">signaling</span> pathways (modified from Huelsken and Behrens, 2002). (A) The Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span>. Wnt signal is transduced through the frizzled receptor (Fz) via dishevelled (Dsh), which represses the axin/glycogen synthase kinase-3β (GSK3β) complex, which induces the degradation of β-catenin. Accumulated Cytoplasmic β-catenin is translocated to the nucleus where it binds with Tcf/Lef, thus activating the transcription of its target gene. (B) The planar cell polarity (PCP) <span class="highlight" style="background-color:">pathway</span> activated in response to Wnt <span class="highlight" style="background-color:">signaling</span> signals via the small GTPases Rho and Cdc42 to c-Jun N-terminal kinase (JNK), acting primarily on the cytoskeleton (C) An alternative <span class="highlight" style="background-color:">pathway</span> stimulates the release of intracellular Ca2+, activating protein kinase C (PKC) and Ca2+/calmodulin-dependent kinase II (CamKII).</div></p></div><div class="aux"><div class="resc">Poongodi Geetha-Loganathan, et al. Organogenesis. 2008 Apr-Jun;4(2):109-115.</div><p class="links"><a class="dblinks" href="/pubmed/19279722" ref="ordinalpos=3671&amp;ncbi_uid=1709728&amp;link_uid=19279722">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634256/" ref="ordinalpos=3671&amp;ncbi_uid=1709728&amp;link_uid=PMC2634256">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3672.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3030489/figure/fig5/" ref="ordinalpos=3672&amp;ncbi_uid=3616110&amp;link_uid=PMC3030489" href="/pmc/articles/PMC3030489/"><img src="/pmc/articles/instance/3030489/bin/GEN1872455f5.gif" src-large="/pmc/articles/instance/3030489/bin/GEN1872455f5.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  5.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3030489/" ref="ordinalpos=3672&amp;ncbi_uid=3616110&amp;link_uid=PMC3030489" image-link="/pmc/articles/PMC3030489/figure/fig5/" class="imagepopup">F igure  5.—.  From: The Putative Lipid Transporter, Arv1, Is Required for Activating Pheromone-Induced MAP Kinase <span class="highlight" style="background-color:">Signaling</span> in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div><i>arv1</i> cells harbor defects in pheromone response <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (A) Pheromone <span class="highlight" style="background-color:">signaling</span> is defective in <i>arv1</i> cells treated with α-factor. <i>MAT</i><b>a</b> <i>FUS1∷FUS1-lacZ∷LEU2</i> and <i>arv1 FUS1∷FUS1-lacZ∷LEU2</i> cells were transformed with plasmids containing full-length Arv1, AHD, and ΔAHD; the constructs are under the control of the <i>ARV1</i> promoter and are C-terminally tagged with 3HA in vector pRS416. <i>FUS1-lacZ</i> induction was measured after 2 hr of α-factor treatment (mean ± SEM, <i>n</i> = 3, *<i>P</i> &lt; 0.02, **<i>P</i> &lt; 0.006). (B) Ste12 overexpression rescues <span class="highlight" style="background-color:">signaling</span> defect in <i>arv1</i> cells. pGAL1-STE12 was expressed in the strains described in A. <i>FUS1-lacZ</i> induction was measured 4 hr after galactose treatment (without α-factor) (mean ± SEM, <i>n</i> = 3).</div></p></div><div class="aux"><div class="resc">Michelle L. Villasmil, et al. Genetics. 2011 February;187(2):455-465.</div><p class="links"><a class="dblinks" href="/pubmed/21098723" ref="ordinalpos=3672&amp;ncbi_uid=3616110&amp;link_uid=21098723">Citation</a><a class="dblinks" href="/pmc/articles/PMC3030489/" ref="ordinalpos=3672&amp;ncbi_uid=3616110&amp;link_uid=PMC3030489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3673.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4074810/figure/cancers-06-00926-f001/" ref="ordinalpos=3673&amp;ncbi_uid=6308249&amp;link_uid=PMC4074810" href="/pmc/articles/PMC4074810/"><img src="/pmc/articles/instance/4074810/bin/cancers-06-00926-g001.gif" src-large="/pmc/articles/instance/4074810/bin/cancers-06-00926-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4074810/" ref="ordinalpos=3673&amp;ncbi_uid=6308249&amp;link_uid=PMC4074810" image-link="/pmc/articles/PMC4074810/figure/cancers-06-00926-f001/" class="imagepopup">Figure 1.  From: Transcription Factor STAT3 as a Novel Molecular Target for Cancer Prevention. </a></p><div class="supp"><p class="details"><div>STAT family of transcription factors and STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>A</b>) Structure and function domains for STAT1-6. (<b>B</b>) STAT3 <span class="highlight" style="background-color:">signaling</span> and its activation signals from extracellular and intracellular routes.</div></p></div><div class="aux"><div class="resc">Ailian Xiong, et al. Cancers (Basel). 2014 June;6(2):926-957.</div><p class="links"><a class="dblinks" href="/pubmed/24743778" ref="ordinalpos=3673&amp;ncbi_uid=6308249&amp;link_uid=24743778">Citation</a><a class="dblinks" href="/pmc/articles/PMC4074810/" ref="ordinalpos=3673&amp;ncbi_uid=6308249&amp;link_uid=PMC4074810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3674.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418714/figure/F3/" ref="ordinalpos=3674&amp;ncbi_uid=5178919&amp;link_uid=PMC3418714" href="/pmc/articles/PMC3418714/"><img src="/pmc/articles/instance/3418714/bin/zh89991294640003.gif" src-large="/pmc/articles/instance/3418714/bin/zh89991294640003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418714/" ref="ordinalpos=3674&amp;ncbi_uid=5178919&amp;link_uid=PMC3418714" image-link="/pmc/articles/PMC3418714/figure/F3/" class="imagepopup">Figure 3.  From: The B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a therapeutic target in CLL. </a></p><div class="supp"><p class="details"><div><b>Kinase inhibitors in CLL.</b> (A) Chemical structures of signal kinase inhibitors. (B) Kinase inhibitors and the BCR <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jennifer A. Woyach, et al. Blood. 2012 August 9;120(6):1175-1184.</div><p class="links"><a class="dblinks" href="/pubmed/22715122" ref="ordinalpos=3674&amp;ncbi_uid=5178919&amp;link_uid=22715122">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418714/" ref="ordinalpos=3674&amp;ncbi_uid=5178919&amp;link_uid=PMC3418714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3675.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1502142/figure/pbio-0040258-g001/" ref="ordinalpos=3675&amp;ncbi_uid=765207&amp;link_uid=PMC1502142" href="/pmc/articles/PMC1502142/"><img src="/pmc/articles/instance/1543679/bin/pbio.0040258.g001.gif" src-large="/pmc/articles/instance/1543679/bin/pbio.0040258.g001.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1502142/" ref="ordinalpos=3675&amp;ncbi_uid=765207&amp;link_uid=PMC1502142" image-link="/pmc/articles/PMC1502142/figure/pbio-0040258-g001/" class="imagepopup">Figure. From: Inhibiting Hedgehog: New Insights into a Developmentally Important <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Structure of the vitamin D3 molecule overlaying a photograph of a zebrafish embryo stained for hedgehog <span class="highlight" style="background-color:">pathway</span> activity.</div></p></div><div class="aux"><div class="resc">Mary Hoff. PLoS Biol. 2006 August;4(8):e258.</div><p class="links"><a class="dblinks" href="/pubmed/20076617" ref="ordinalpos=3675&amp;ncbi_uid=765207&amp;link_uid=20076617">Citation</a><a class="dblinks" href="/pmc/articles/PMC1502142/" ref="ordinalpos=3675&amp;ncbi_uid=765207&amp;link_uid=PMC1502142">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3676.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3728089/figure/pone-0070338-g005/" ref="ordinalpos=3676&amp;ncbi_uid=5185677&amp;link_uid=PMC3728089" href="/pmc/articles/PMC3728089/"><img src="/pmc/articles/instance/3728089/bin/pone.0070338.g005.gif" src-large="/pmc/articles/instance/3728089/bin/pone.0070338.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3728089/" ref="ordinalpos=3676&amp;ncbi_uid=5185677&amp;link_uid=PMC3728089" image-link="/pmc/articles/PMC3728089/figure/pone-0070338-g005/" class="imagepopup">Figure 5. Proposed mechanism of NRGs neuroprotective action during inflammation..  From: Neuregulin Upregulates Microglial ?7 Nicotinic Acetylcholine Receptor Expression in Immortalized Cell Lines: Implications for Regulating Neuroinflammation. </a></p><div class="supp"><p class="details"><div>Here we propose a model where in NRG amplifies the cholinergic anti-inflammatory <span class="highlight" style="background-color:">pathway</span> mediated by ACh and α7 nAChR during an inflammatory response. In panel A, we describe the possible <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> responsible for the increased expression of α7nAChRs observed in microglia when treated with NRG. NRG <span class="highlight" style="background-color:">signaling</span> through ErbB4 activates Ras/Raf/ERK of the MAPK <span class="highlight" style="background-color:">pathway</span> which leads to cell survival, and in muscle has been shown to activate the N-box response elements associated with the transcription of AChRs. Also of importance to NRGs neuroprotective function is the activation of the p85 subunit of PI3K <span class="highlight" style="background-color:">pathway</span>, its downstream effector PKB/Akt, and Jak/STAT pathways [16,51,52] (not shown); <span class="highlight" style="background-color:">signaling</span> mechanisms also involved in cell survival. In panel B, we describe the well established cholinergic anti-inflammatory <span class="highlight" style="background-color:">pathway</span> [modified from 38]. In response to inflammatory mediators, activation of the α7nAChR by its ligand ACh inhibits the synthesis of TNF-α by preventing the nuclear translocation of the p65 subunit of NF-κB. In addition, the activation of Jak2/STAT3 supports the synthesis of anti-inflammatory cytokine IL-10 and further attenuates TNF-α production by STAT3 action on IκK, NF-κB and p65. In panel C, we propose that in LPS-challenged microglia NRG acting in concert with ACh leads to increased reduction of TNF-α by inducing increased expression of α7nAChR and as such amplifying the cholinergic anti-inflammatory <span class="highlight" style="background-color:">pathway</span> by providing more receptors for ligand binding.</div></p></div><div class="aux"><div class="resc">Malwina Mencel, et al. PLoS One. 2013;8(7):e70338.</div><p class="links"><a class="dblinks" href="/pubmed/23936190" ref="ordinalpos=3676&amp;ncbi_uid=5185677&amp;link_uid=23936190">Citation</a><a class="dblinks" href="/pmc/articles/PMC3728089/" ref="ordinalpos=3676&amp;ncbi_uid=5185677&amp;link_uid=PMC3728089">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3677.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3082499/figure/F2/" ref="ordinalpos=3677&amp;ncbi_uid=2906303&amp;link_uid=PMC3082499" href="/pmc/articles/PMC3082499/"><img src="/pmc/articles/instance/3082499/bin/nihms-290189-f0002.gif" src-large="/pmc/articles/instance/3082499/bin/nihms-290189-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3082499/" ref="ordinalpos=3677&amp;ncbi_uid=2906303&amp;link_uid=PMC3082499" image-link="/pmc/articles/PMC3082499/figure/F2/" class="imagepopup">Fig. 2.  From: Programmed cell death and new discoveries in the genetics of parkinsonism. </a></p><div class="supp"><p class="details"><div>Mediators of the extrinsic <span class="highlight" style="background-color:">pathway</span> of PCD by Fas and TNFR-1 <span class="highlight" style="background-color:">signaling</span>. (a) The protein domain structures of Fas and TNFR-1. Each receptor has cysteine-rich domains (CRD) in the ligand binding region, characteristic of members of the TNF superfamily of receptors. There is a death domain in the intracellular portion of each receptor. (b) Fas death <span class="highlight" style="background-color:">signaling</span> patheways. (c) TNFR-1 <span class="highlight" style="background-color:">signaling</span> pathways. See text for details.</div></p></div><div class="aux"><div class="resc">Robert E. Burke. J Neurochem. ;104(4):875-890.</div><p class="links"><a class="dblinks" href="/pubmed/17996022" ref="ordinalpos=3677&amp;ncbi_uid=2906303&amp;link_uid=17996022">Citation</a><a class="dblinks" href="/pmc/articles/PMC3082499/" ref="ordinalpos=3677&amp;ncbi_uid=2906303&amp;link_uid=PMC3082499">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3678.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4039177/figure/F3/" ref="ordinalpos=3678&amp;ncbi_uid=6181519&amp;link_uid=PMC4039177" href="/pmc/articles/PMC4039177/"><img src="/pmc/articles/instance/4039177/bin/nihms199097f3.gif" src-large="/pmc/articles/instance/4039177/bin/nihms199097f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4039177/" ref="ordinalpos=3678&amp;ncbi_uid=6181519&amp;link_uid=PMC4039177" image-link="/pmc/articles/PMC4039177/figure/F3/" class="imagepopup">Figure 3. Low density lipoprotein modulate the Hh <span class="highlight" style="background-color:">pathway</span> inhibitory effects of itraconazole.  From: Itraconazole, a commonly used anti-fungal that inhibits Hedgehog <span class="highlight" style="background-color:">pathway</span> activity and cancer growth. </a></p><div class="supp"><p class="details"><div>(A) High density lipoprotein (HDL) 100 µg/ml and (B) very low density lipoprotein (VLDL) 6.5 µg/ml cannot reverse Hh <span class="highlight" style="background-color:">pathway</span> inhibition by itraconazole. (C, D) Titration of low density lipoprotein (LDL) up to 120 µg/ml rescues either induced Hh <span class="highlight" style="background-color:">pathway</span> activity by ShhN ligand in Shh-Light2 cells (C) or constitutively active <span class="highlight" style="background-color:">pathway</span> in <i>Ptch−/−</i> cells (Goodrich et al., 1997; Taipale et al., 2000) (D) from the inhibitory effects of itraconazole. All <span class="highlight" style="background-color:">signaling</span> assays were performed in lipid depleted 10% calf serum media for Shh-Light2 cells (A–C) and 0.5% fetal bovine serum media for <i>Ptch−/−</i> cells. Data are mean of triplicates ± s.d.</div></p></div><div class="aux"><div class="resc">James Kim, et al. Cancer Cell. ;17(4):388-399.</div><p class="links"><a class="dblinks" href="/pubmed/20385363" ref="ordinalpos=3678&amp;ncbi_uid=6181519&amp;link_uid=20385363">Citation</a><a class="dblinks" href="/pmc/articles/PMC4039177/" ref="ordinalpos=3678&amp;ncbi_uid=6181519&amp;link_uid=PMC4039177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3679.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4142280/figure/fig4/" ref="ordinalpos=3679&amp;ncbi_uid=6549494&amp;link_uid=PMC4142280" href="/pmc/articles/PMC4142280/"><img src="/pmc/articles/instance/4142280/bin/JOPH2014-801787.004.gif" src-large="/pmc/articles/instance/4142280/bin/JOPH2014-801787.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4142280/" ref="ordinalpos=3679&amp;ncbi_uid=6549494&amp;link_uid=PMC4142280" image-link="/pmc/articles/PMC4142280/figure/fig4/" class="imagepopup">Figure 4.  From: Cell Models to Study Regulation of Cell Transformation in Pathologies of Retinal Pigment Epithelium. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> crosstalk between the TGF-<i>β</i>-activated <span class="highlight" style="background-color:">pathway</span> and other pathways during EMT (according to Xu et al. [94]).</div></p></div><div class="aux"><div class="resc">Alla V. Kuznetsova, et al. J Ophthalmol. 2014;2014:801787.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3679&amp;ncbi_uid=6549494&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4142280/" ref="ordinalpos=3679&amp;ncbi_uid=6549494&amp;link_uid=PMC4142280">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3680.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3963847/figure/pone-0091094-g008/" ref="ordinalpos=3680&amp;ncbi_uid=5951807&amp;link_uid=PMC3963847" href="/pmc/articles/PMC3963847/"><img src="/pmc/articles/instance/3963847/bin/pone.0091094.g008.gif" src-large="/pmc/articles/instance/3963847/bin/pone.0091094.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3963847/" ref="ordinalpos=3680&amp;ncbi_uid=5951807&amp;link_uid=PMC3963847" image-link="/pmc/articles/PMC3963847/figure/pone-0091094-g008/" class="imagepopup">Figure 8. Schematic summary of CNT mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> related with anti-angiogenic activity..  From: Convallatoxin, a Dual Inducer of Autophagy and Apoptosis, Inhibits Angiogenesis In Vitro and In Vivo. </a></p><div class="aux"><div class="resc">Seung Ya Yang, et al. PLoS One. 2014;9(3):e91094.</div><p class="links"><a class="dblinks" href="/pubmed/24663328" ref="ordinalpos=3680&amp;ncbi_uid=5951807&amp;link_uid=24663328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3963847/" ref="ordinalpos=3680&amp;ncbi_uid=5951807&amp;link_uid=PMC3963847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3681.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2840158/figure/fig06/" ref="ordinalpos=3681&amp;ncbi_uid=2450153&amp;link_uid=PMC2840158" href="/pmc/articles/PMC2840158/"><img src="/pmc/articles/instance/2840158/bin/pnas.0913860107fig06.gif" src-large="/pmc/articles/instance/2840158/bin/pnas.0913860107fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2840158/" ref="ordinalpos=3681&amp;ncbi_uid=2450153&amp;link_uid=PMC2840158" image-link="/pmc/articles/PMC2840158/figure/fig06/" class="imagepopup">Fig. 6. From: ATM signals to TSC2 in the cytoplasm to regulate mTORC1 in response to ROS. </a></p><div class="supp"><p class="details"><div>Schematic showing cytoplasmic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from ATM to TSC2 via LKB1 and AMPK to repress mTORC1 and induce autophagy.</div></p></div><div class="aux"><div class="resc">Angela Alexander, et al. Proc Natl Acad Sci U S A. 2010 March 2;107(9):4153-4158.</div><p class="links"><a class="dblinks" href="/pubmed/20160076" ref="ordinalpos=3681&amp;ncbi_uid=2450153&amp;link_uid=20160076">Citation</a><a class="dblinks" href="/pmc/articles/PMC2840158/" ref="ordinalpos=3681&amp;ncbi_uid=2450153&amp;link_uid=PMC2840158">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3682.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1440308/figure/F4/" ref="ordinalpos=3682&amp;ncbi_uid=579807&amp;link_uid=PMC1440308" href="/pmc/articles/PMC1440308/"><img src="/pmc/articles/instance/1440308/bin/1742-4682-3-13-4.gif" src-large="/pmc/articles/instance/1440308/bin/1742-4682-3-13-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1440308/" ref="ordinalpos=3682&amp;ncbi_uid=579807&amp;link_uid=PMC1440308" image-link="/pmc/articles/PMC1440308/figure/F4/" class="imagepopup">Figure 4.  From: A method for the generation of standardized qualitative dynamical systems of regulatory networks. </a></p><div class="supp"><p class="details"><div><b>Alternative Th network</b>. T helper <span class="highlight" style="background-color:">pathway</span> published in [69], reinterpreted as a <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Luis Mendoza, et al. Theor Biol Med Model. 2006;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/16542429" ref="ordinalpos=3682&amp;ncbi_uid=579807&amp;link_uid=16542429">Citation</a><a class="dblinks" href="/pmc/articles/PMC1440308/" ref="ordinalpos=3682&amp;ncbi_uid=579807&amp;link_uid=PMC1440308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3683.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1440308/figure/F6/" ref="ordinalpos=3683&amp;ncbi_uid=579801&amp;link_uid=PMC1440308" href="/pmc/articles/PMC1440308/"><img src="/pmc/articles/instance/1440308/bin/1742-4682-3-13-6.gif" src-large="/pmc/articles/instance/1440308/bin/1742-4682-3-13-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1440308/" ref="ordinalpos=3683&amp;ncbi_uid=579801&amp;link_uid=PMC1440308" image-link="/pmc/articles/PMC1440308/figure/F6/" class="imagepopup">Figure 6.  From: A method for the generation of standardized qualitative dynamical systems of regulatory networks. </a></p><div class="supp"><p class="details"><div><b>Alternative Th network</b>. T helper <span class="highlight" style="background-color:">pathway</span> published in [43], reinterpreted as a <span class="highlight" style="background-color:">signaling</span> network.</div></p></div><div class="aux"><div class="resc">Luis Mendoza, et al. Theor Biol Med Model. 2006;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/16542429" ref="ordinalpos=3683&amp;ncbi_uid=579801&amp;link_uid=16542429">Citation</a><a class="dblinks" href="/pmc/articles/PMC1440308/" ref="ordinalpos=3683&amp;ncbi_uid=579801&amp;link_uid=PMC1440308">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3684.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC84262/figure/F1/" ref="ordinalpos=3684&amp;ncbi_uid=172800&amp;link_uid=PMC84262" href="/pmc/articles/PMC84262/"><img src="/pmc/articles/instance/84262/bin/mb0791836001.gif" src-large="/pmc/articles/instance/84262/bin/mb0791836001.jpg" alt="FIG. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC84262/" ref="ordinalpos=3684&amp;ncbi_uid=172800&amp;link_uid=PMC84262" image-link="/pmc/articles/PMC84262/figure/F1/" class="imagepopup">FIG. 1.  From: Mutant Cells That Do Not Respond to Interleukin-1 (IL-1) Reveal a Novel Role for IL-1 Receptor-Associated Kinase. </a></p><div class="supp"><p class="details"><div>A model for the IL-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. See the text for details. PC, phosphatidylcholine.</div></p></div><div class="aux"><div class="resc">Xiaoxia Li, et al. Mol Cell Biol. 1999 July;19(7):4643-4652.</div><p class="links"><a class="dblinks" href="/pubmed/10373513" ref="ordinalpos=3684&amp;ncbi_uid=172800&amp;link_uid=10373513">Citation</a><a class="dblinks" href="/pmc/articles/PMC84262/" ref="ordinalpos=3684&amp;ncbi_uid=172800&amp;link_uid=PMC84262">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3685.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508196/figure/F1/" ref="ordinalpos=3685&amp;ncbi_uid=4394742&amp;link_uid=PMC3508196" href="/pmc/articles/PMC3508196/"><img src="/pmc/articles/instance/3508196/bin/nihms-398021-f0001.gif" src-large="/pmc/articles/instance/3508196/bin/nihms-398021-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508196/" ref="ordinalpos=3685&amp;ncbi_uid=4394742&amp;link_uid=PMC3508196" image-link="/pmc/articles/PMC3508196/figure/F1/" class="imagepopup">Figure 1. The MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: TARGETING ONCOGENIC BRAF IN HUMAN CANCER. </a></p><div class="supp"><p class="details"><div>The MAP kinase <span class="highlight" style="background-color:">pathway</span> is activated in human tumors by the binding of ligand to receptor tyrosine kinases (RTKs), by constitutive activation of an RTK, by loss of NF1 function, or by mutations in RAS, BRAF and MEK1. Phosphorylation and thus activation of ERK regulates transcription of target genes which promote cell cylce progression and tumor survival. The ERK <span class="highlight" style="background-color:">pathway</span> contains a classical feedback loop in which the expression of feedback elements such as SPRY and DUSP family proteins are regulated by ERK (dashed grey lines). Selected agents that target the MAP kinase and PI3 kinase/AKT <span class="highlight" style="background-color:">pathway</span> are shown (red).</div></p></div><div class="aux"><div class="resc">Christine Pratilas, et al. Curr Top Microbiol Immunol. ;355:83-98.</div><p class="links"><a class="dblinks" href="/pubmed/21818706" ref="ordinalpos=3685&amp;ncbi_uid=4394742&amp;link_uid=21818706">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508196/" ref="ordinalpos=3685&amp;ncbi_uid=4394742&amp;link_uid=PMC3508196">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3686.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3072701/figure/F3/" ref="ordinalpos=3686&amp;ncbi_uid=2927759&amp;link_uid=PMC3072701" href="/pmc/articles/PMC3072701/"><img src="/pmc/articles/instance/3072701/bin/nihms281664f3.gif" src-large="/pmc/articles/instance/3072701/bin/nihms281664f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3072701/" ref="ordinalpos=3686&amp;ncbi_uid=2927759&amp;link_uid=PMC3072701" image-link="/pmc/articles/PMC3072701/figure/F3/" class="imagepopup">Figure 3.  From: Roles for Netrin <span class="highlight" style="background-color:">signaling</span> outside of axon guidance: a view from the worm. </a></p><div class="supp"><p class="details"><div><b>(a)</b> Confocal micrographs of anchor cells expressing various fluorescent markers in <i>C. elegans</i>. UNC-6/Netrin <span class="highlight" style="background-color:">signaling</span> from the ventral nerve cord polarizes UNC-40/DCC ventrally (white arrowheads) towards the basement membrane and vulval tissue. In turn the Netrin <span class="highlight" style="background-color:">signaling</span> recruits cytoplasmic <span class="highlight" style="background-color:">signaling</span> effectors such as the MIG-2/Mtl Rac. Through this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the actin cytoskeleton is oriented towards the basement membrane for invasion. In <i>unc-6</i> mutants, polarity of UNC-40/DCC is lost and F-actin and MIG-2/Mtl Rac can accumulate on apical and lateral membranes (yellow arrowheads). Scale bar at bottom left is 5μm. <b>(b)</b> In one possible model for UNC-6/Netrin <span class="highlight" style="background-color:">signaling</span> through UNC-40/DCC receptors to polarize a cell, binding of the UNC-40/DCC to UNC-6/Netrin on neighboring cells or basement membranes concentrates the receptor. This leads to local recruitment and activation of the Rac-like proteins MIG-2/Mtl Rac and/or CED-10/Rac, perhaps by UNC-73/Trio, which can bind to the intracellular domains of UNC-40/DCC, Frazzled/DCC and Human DCC. Downstream of Rac activation the <span class="highlight" style="background-color:">pathway</span> may diverge, utilizing context specific regulators and effectors such as MIG-10/Lamellipodin, a known Rac effector and cytoskeletal regulator.</div></p></div><div class="aux"><div class="resc">Joshua W. Ziel, et al. Dev Dyn. ;239(5):1296-1305.</div><p class="links"><a class="dblinks" href="/pubmed/20108323" ref="ordinalpos=3686&amp;ncbi_uid=2927759&amp;link_uid=20108323">Citation</a><a class="dblinks" href="/pmc/articles/PMC3072701/" ref="ordinalpos=3686&amp;ncbi_uid=2927759&amp;link_uid=PMC3072701">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3687.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2905036/figure/F2/" ref="ordinalpos=3687&amp;ncbi_uid=2862031&amp;link_uid=PMC2905036" href="/pmc/articles/PMC2905036/"><img src="/pmc/articles/instance/2905036/bin/nihms-183350-f0002.gif" src-large="/pmc/articles/instance/2905036/bin/nihms-183350-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2905036/" ref="ordinalpos=3687&amp;ncbi_uid=2862031&amp;link_uid=PMC2905036" image-link="/pmc/articles/PMC2905036/figure/F2/" class="imagepopup">Figure 2.  From: Molecular Mechanisms of Hepcidin Regulation: Implications for the Anemia of CKD. </a></p><div class="supp"><p class="details"><div>Schematic diagram depicting the proposed role of bone morphogenetic protein (BMP) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, HJV, HFE, and TFR2 in iron sensing and hepcidin regulation in the liver. In response to iron, bone morphogenetic protein 6 (BMP6) binds to the BMP co-receptor hemojuvelin (HJV) as part of the BMP6/HJV/BMPR complex on the hepatocyte membrane to activate the SMAD1/5/8 <span class="highlight" style="background-color:">pathway</span>. Activated SMAD complexes bind directly to BMP responsive elements (BMP-REs) on the hepcidin promoter to induce hepcidin transcription. Holotransferrin (TF-Fe) competes for HFE binding to transferrin receptor 1 (TFR1), causing HFE displacement to form a complex with TFR2 and TF-Fe to induce hepcidin expression, possibly through an interaction with the BMP6-HJV-SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and/or alternative <span class="highlight" style="background-color:">signaling</span> pathways such as the ERK1/2 <span class="highlight" style="background-color:">pathway</span>. In the setting of iron deficiency, the serine protease TMPRSS6 inhibits hepcidin expression by cleaving membrane-bound HJV to form soluble HJV, thereby inhibiting downstream SMAD <span class="highlight" style="background-color:">signaling</span> by loss of membrane-bound HJV and sequestration of BMP6 ligand. Abbreviations: BMPR, BMP receptor, TF, transferrin.</div></p></div><div class="aux"><div class="resc">Jodie L. Babitt, et al. Am J Kidney Dis. ;55(4):726-741.</div><p class="links"><a class="dblinks" href="/pubmed/20189278" ref="ordinalpos=3687&amp;ncbi_uid=2862031&amp;link_uid=20189278">Citation</a><a class="dblinks" href="/pmc/articles/PMC2905036/" ref="ordinalpos=3687&amp;ncbi_uid=2862031&amp;link_uid=PMC2905036">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3688.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3013243/figure/F9/" ref="ordinalpos=3688&amp;ncbi_uid=3438800&amp;link_uid=PMC3013243" href="/pmc/articles/PMC3013243/"><img src="/pmc/articles/instance/3013243/bin/nihms-246118-f0009.gif" src-large="/pmc/articles/instance/3013243/bin/nihms-246118-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3013243/" ref="ordinalpos=3688&amp;ncbi_uid=3438800&amp;link_uid=PMC3013243" image-link="/pmc/articles/PMC3013243/figure/F9/" class="imagepopup">Figure 9. Proposed model of GRK2 regulation of TLR4-induced inflammatory response.  From: Myeloid-specific GPCR kinase-2 negatively regulates NF?B1p105-ERK <span class="highlight" style="background-color:">pathway</span> and limits endotoxemic shock in mice. </a></p><div class="supp"><p class="details"><div>TLR4 activates ERK <span class="highlight" style="background-color:">pathway</span> via IKKβ-mediated phosphorylation of p105. Based on the results presented here, we propose that GRK2 negatively regulates IKKβ-induced p105 phosphorylation and therefore the ERK <span class="highlight" style="background-color:">pathway</span>. Because p105 phosphorylation is also essential for p50 mediated NFκB activation, negative regulation of p105 by GRK2 also modulates a subset of cytokines that are regulated by this <span class="highlight" style="background-color:">pathway</span>. Note that there are two exceptions: 1. Because MCP-1 was inhibited by ERK inhibitor but not by the IKKβ inhibitor, it is possible that MCP-1 is being regulated by GRK2 in an ERK-dependent but p105-independent mechanism. 2. Also, because MIP1α was modestly inhibited by PD98059, it is possible that a small component of the ERK <span class="highlight" style="background-color:">pathway</span> may also regulate MIP1α. Previous studies in primary macrophages and neutrophils have demonstrated that TLR4 activation increases the expression of GRK2 significantly and thus GRK2 might act as a negative feedback regulator for TLR4 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sonika Patial, et al. J Cell Physiol. ;226(3):627-637.</div><p class="links"><a class="dblinks" href="/pubmed/20717897" ref="ordinalpos=3688&amp;ncbi_uid=3438800&amp;link_uid=20717897">Citation</a><a class="dblinks" href="/pmc/articles/PMC3013243/" ref="ordinalpos=3688&amp;ncbi_uid=3438800&amp;link_uid=PMC3013243">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3689.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1401500/figure/F6/" ref="ordinalpos=3689&amp;ncbi_uid=692666&amp;link_uid=PMC1401500" href="/pmc/articles/PMC1401500/"><img src="/pmc/articles/instance/1401500/bin/nihms-7653-0006.gif" src-large="/pmc/articles/instance/1401500/bin/nihms-7653-0006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1401500/" ref="ordinalpos=3689&amp;ncbi_uid=692666&amp;link_uid=PMC1401500" image-link="/pmc/articles/PMC1401500/figure/F6/" class="imagepopup">Figure 6. From: Advanced Glycation End Products Activate a Chymase-Dependent Angiotensin II Generating <span class="highlight" style="background-color:">Pathway</span> in Diabetic Complications. </a></p><div class="supp"><p class="details"><div><b>Western blot analysis demonstrates that AGEs signal through RAGE to activate the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and p38 <span class="highlight" style="background-color:">signaling</span> pathways.</b> (A) A representative Western blot shows that AGE-BSA (50 μg/ml) induces ERK1/2 and p38 phosphorylation in VSMC in a time-dependent manner, being significant by 30 minutes. BSA (50μg/ml) at 30 mins does not induce such phosphorylation. (B) AGE-induced phosphorylation of p38 and ERK1/2 is blocked by a neutralizing rabbit anti-RAGE Ab (10 μg/ml), but not by the isotype control rabbit IgG (10 μg/ml), confirming the involvement of RAGE in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Furthermore, addition of inhibitors to ERK1/2 (PD98059, 10μM) or p38 (SB203580, 10 μM) and infection with Adv-DN-ERK1/2 or Adv-DN-p38 (MOI of 50) also inhibit AGE-induced ERK1/2 and p38 phosphorylation respectively, while the control Adv-β-gal (MOI of 50) does not, confirming the efficacy of the dominant negative constructs. BSA (50 μg/ml) has no effect on either ERK1/2 or p38 phosphorylation.</div></p></div><div class="aux"><div class="resc">Vijay Koka, et al. Circulation. ;113(10):1353-1360.</div><p class="links"><a class="dblinks" href="/pubmed/16520412" ref="ordinalpos=3689&amp;ncbi_uid=692666&amp;link_uid=16520412">Citation</a><a class="dblinks" href="/pmc/articles/PMC1401500/" ref="ordinalpos=3689&amp;ncbi_uid=692666&amp;link_uid=PMC1401500">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3690.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3475297/figure/F1/" ref="ordinalpos=3690&amp;ncbi_uid=4306083&amp;link_uid=PMC3475297" href="/pmc/articles/PMC3475297/"><img src="/pmc/articles/instance/3475297/bin/fgene-03-00218-g0001.gif" src-large="/pmc/articles/instance/3475297/bin/fgene-03-00218-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3475297/" ref="ordinalpos=3690&amp;ncbi_uid=4306083&amp;link_uid=PMC3475297" image-link="/pmc/articles/PMC3475297/figure/F1/" class="imagepopup">Figure 1.  From: Regulation of lifespan by chemosensory and thermosensory systems: findings in invertebrates and their implications in mammalian aging. </a></p><div class="supp"><p class="details"><div><b><i>C. elegans</i> and <i>D. melanogaster</i> sensory systems influence lifespan. (A)</b> <i>C. elegans</i> chemosensory and thermosensory systems regulate lifespan through hormonal <span class="highlight" style="background-color:">signaling</span>. Specific chemosensory (olfactory or gustatory) neurons promote or limit longevity. Insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> can mediate this longevity response downstream of chemosensory neurons. Perturbation of thermosensory system accelerates aging at high temperatures by influencing the sterol hormonal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(B)</b> Chemosensory systems regulate <i>D. melanogaster</i> lifespan. Olfaction of nutrient-derived odorants promotes the aging of long-lived <i>D. melanogaster</i> upon dietary restriction (DR). Perturbation of the <i>D. melanogaster</i> olfactory system and inhibition of the CO<sub>2</sub>-sensing system both prolong lifespan. However, the <span class="highlight" style="background-color:">signaling</span> pathways regulated by <i>D. melanogaster</i> chemosensory systems to influence lifespan are unknown.</div></p></div><div class="aux"><div class="resc">Dae-Eun Jeong, et al. Front Genet. 2012;3:218.</div><p class="links"><a class="dblinks" href="/pubmed/23087711" ref="ordinalpos=3690&amp;ncbi_uid=4306083&amp;link_uid=23087711">Citation</a><a class="dblinks" href="/pmc/articles/PMC3475297/" ref="ordinalpos=3690&amp;ncbi_uid=4306083&amp;link_uid=PMC3475297">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3691.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3617635/figure/F6/" ref="ordinalpos=3691&amp;ncbi_uid=5978245&amp;link_uid=PMC3617635" href="/pmc/articles/PMC3617635/"><img src="/pmc/articles/instance/3617635/bin/2704f6.gif" src-large="/pmc/articles/instance/3617635/bin/2704f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3617635/" ref="ordinalpos=3691&amp;ncbi_uid=5978245&amp;link_uid=PMC3617635" image-link="/pmc/articles/PMC3617635/figure/F6/" class="imagepopup">Figure 6.  From: Chronic lymphocytic leukemia cells induce defective LFA-1-directed T-cell motility by altering Rho GTPase <span class="highlight" style="background-color:">signaling</span> that is reversible with lenalidomide. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of CLL-induced Rho GTPase <span class="highlight" style="background-color:">signaling</span> defects in T cells that suppress LFA-1 activation <span class="highlight" style="background-color:">signaling</span> and migration and how this is reversible with lenalidomide.</b> Direct contact of CLL cells with T cells (immunosuppressive <span class="highlight" style="background-color:">signaling</span>) downregulates RhoA and Rac1 (positive regulators of LFA-1 activation, green) and potentiates Cdc42 activity (negatively regulates LFA-1 activation, blue) in T cells inducing an important <span class="highlight" style="background-color:">signaling</span> block that negatively regulates LFA-1 activity. This attenuated LFA-1 activating <span class="highlight" style="background-color:">signaling</span> model is supported by reduced expression and trafficking of Rap1 at the LFA-1–expressing membrane in CLL T cells. PIP5KC (phosphatidylinositol-4-phosphate 5-kinase type I γ) and PtdIns(4,5)P2 (phosphatidylinositol-4,5-bisphosphate) are critical for LFA-1 affinity modulation downstream of RhoA and Rac. Diminished Rho GTPase activity and cytoskeletal <span class="highlight" style="background-color:">signaling</span> would suppress subsequent outside-in <span class="highlight" style="background-color:">signaling</span> and LFA-1–mediated adhesion and migration on the CD54 ligand. The ability of lenalidomide (red) to restore normal T-cell Rho GTPase activity and LFA-1 activation provides, for the first time, an important molecular definition of its mechanism of action. Cereblon expression is required for this immunomodulatory activity of lenalidomide. Dotted lines indicate that it is yet to be determined if lenalidomide activates Rho GTPases directly or indirectly via the cereblon <span class="highlight" style="background-color:">pathway</span>. These results indicate that lenalidomide represents a powerful immunotherapy to rescue global T-cell motility defects in cancer patients.</div></p></div><div class="aux"><div class="resc">Alan G. Ramsay, et al. Blood. 2013 April 4;121(14):2704-2714.</div><p class="links"><a class="dblinks" href="/pubmed/23325833" ref="ordinalpos=3691&amp;ncbi_uid=5978245&amp;link_uid=23325833">Citation</a><a class="dblinks" href="/pmc/articles/PMC3617635/" ref="ordinalpos=3691&amp;ncbi_uid=5978245&amp;link_uid=PMC3617635">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3692.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649616/figure/F3/" ref="ordinalpos=3692&amp;ncbi_uid=1730107&amp;link_uid=PMC2649616" href="/pmc/articles/PMC2649616/"><img src="/pmc/articles/instance/2649616/bin/org0301_0025_fig003.gif" src-large="/pmc/articles/instance/2649616/bin/org0301_0025_fig003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649616/" ref="ordinalpos=3692&amp;ncbi_uid=1730107&amp;link_uid=PMC2649616" image-link="/pmc/articles/PMC2649616/figure/F3/" class="imagepopup">Figure 3.  From: miRNAs: Miracle or Mirage?. </a></p><div class="supp"><p class="details"><div>Model for the role of microRNAs in the regulation of ‘tumor suppressor’ pathways in <i>Drosophila</i>. The miRNA <i>bantam</i> is a key target of the transcriptional regulator Yorkie. <i>Yorkie</i> is under tight control of the Hippo <span class="highlight" style="background-color:">pathway</span> that consists of <i>merlin</i> and <i>expanded</i> at the membrane integrating signals from an unknown transmembrane receptor. In certain cell types in <i>Drosophila</i>, expanded mRNA is regulated by <i>miR-278</i>. Warts and Mats transduce the signal from Merlin/Expanded and Hippo/Salvador. Warts and Mats are also elements of Fat/Dachsous <span class="highlight" style="background-color:">signaling</span> (another <i>Drosophila</i> tumor suppressor <span class="highlight" style="background-color:">pathway</span>) through Dachs. Fat also can interfere with canonical Wnt <span class="highlight" style="background-color:">signaling</span> and activate noncanonical Wnt <span class="highlight" style="background-color:">signaling</span>. This could theoretically occur via the inactivation of <i>Yorkie</i> and <i>Bantam</i>: one of <i>Bantam</i>'s target mRNAs is <i>Hid</i>, which has been shown to inhibit canonical Wnt <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Thomas Andl. Organogenesis. 2007 Jan-Mar;3(1):25-33.</div><p class="links"><a class="dblinks" href="/pubmed/19279697" ref="ordinalpos=3692&amp;ncbi_uid=1730107&amp;link_uid=19279697">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649616/" ref="ordinalpos=3692&amp;ncbi_uid=1730107&amp;link_uid=PMC2649616">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3693.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2990745/figure/F1/" ref="ordinalpos=3693&amp;ncbi_uid=2628116&amp;link_uid=PMC2990745" href="/pmc/articles/PMC2990745/"><img src="/pmc/articles/instance/2990745/bin/1752-0509-4-151-1.gif" src-large="/pmc/articles/instance/2990745/bin/1752-0509-4-151-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2990745/" ref="ordinalpos=3693&amp;ncbi_uid=2628116&amp;link_uid=PMC2990745" image-link="/pmc/articles/PMC2990745/figure/F1/" class="imagepopup">Figure 1.  From: Multi-level reproducibility of signature hubs in human interactome for breast cancer metastasis. </a></p><div class="supp"><p class="details"><div><b>Examples of pathways shared by a signature hub from the Wang dataset and a signature hub from the Desmedt dataset</b>. (a) JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; (b) Ribosome complex; (c) MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The yellow and red colors represent proteins (both hubs and their neighbors) identified from the Wang and Desmedt datasets, respectively. The orange colors represent the overlapped neighbors of these two hub proteins. Please see the main text for detailed explanation.</div></p></div><div class="aux"><div class="resc">Chen Yao, et al. BMC Syst Biol. 2010;4:151-151.</div><p class="links"><a class="dblinks" href="/pubmed/21059271" ref="ordinalpos=3693&amp;ncbi_uid=2628116&amp;link_uid=21059271">Citation</a><a class="dblinks" href="/pmc/articles/PMC2990745/" ref="ordinalpos=3693&amp;ncbi_uid=2628116&amp;link_uid=PMC2990745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3694.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3733450/figure/F2/" ref="ordinalpos=3694&amp;ncbi_uid=5165753&amp;link_uid=PMC3733450" href="/pmc/articles/PMC3733450/"><img src="/pmc/articles/instance/3733450/bin/nihms496673f2.gif" src-large="/pmc/articles/instance/3733450/bin/nihms496673f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3733450/" ref="ordinalpos=3694&amp;ncbi_uid=5165753&amp;link_uid=PMC3733450" image-link="/pmc/articles/PMC3733450/figure/F2/" class="imagepopup">Figure 2. mTOR in human disease.  From: The Kaposi's Sarcoma-Associated Herpesvirus G Protein Coupled Receptor: Lessons on Dysregulated Angiogenesis from a Viral Oncogene. </a></p><div class="supp"><p class="details"><div>Schematic showing the functional relationship between the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and several genetic diseases, some of them characterized by the development of vascular tumors. In a similar manner, vGPCR, the KSHV gene hypothesized to be responsible for KS development, appears to trigger Kaposi’s sarcomagenesis by dysregulating this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bruno C. Jham, et al. J Cell Biochem. ;110(1):1-9.</div><p class="links"><a class="dblinks" href="/pubmed/20213674" ref="ordinalpos=3694&amp;ncbi_uid=5165753&amp;link_uid=20213674">Citation</a><a class="dblinks" href="/pmc/articles/PMC3733450/" ref="ordinalpos=3694&amp;ncbi_uid=5165753&amp;link_uid=PMC3733450">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3695.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2635620/figure/f1-ijms-9-2217/" ref="ordinalpos=3695&amp;ncbi_uid=1724768&amp;link_uid=PMC2635620" href="/pmc/articles/PMC2635620/"><img src="/pmc/articles/instance/2635620/bin/ijms-9-2217f1.gif" src-large="/pmc/articles/instance/2635620/bin/ijms-9-2217f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2635620/" ref="ordinalpos=3695&amp;ncbi_uid=1724768&amp;link_uid=PMC2635620" image-link="/pmc/articles/PMC2635620/figure/f1-ijms-9-2217/" class="imagepopup">Figure 1. From: Contribution of Natural Inhibitors to the Understanding of the PI3K/PDK1/PKB <span class="highlight" style="background-color:">Pathway</span> in the Insulin-mediated Intracellular <span class="highlight" style="background-color:">Signaling</span> Cascade. </a></p><div class="supp"><p class="details"><div>Molecular structures of natural products that inhibit insulin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jae Youl Cho, et al. Int J Mol Sci. 2008 November;9(11):2217-2230.</div><p class="links"><a class="dblinks" href="/pubmed/19330070" ref="ordinalpos=3695&amp;ncbi_uid=1724768&amp;link_uid=19330070">Citation</a><a class="dblinks" href="/pmc/articles/PMC2635620/" ref="ordinalpos=3695&amp;ncbi_uid=1724768&amp;link_uid=PMC2635620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3696.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3066043/figure/F7/" ref="ordinalpos=3696&amp;ncbi_uid=2850321&amp;link_uid=PMC3066043" href="/pmc/articles/PMC3066043/"><img src="/pmc/articles/instance/3066043/bin/nihms276743f7.gif" src-large="/pmc/articles/instance/3066043/bin/nihms276743f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3066043/" ref="ordinalpos=3696&amp;ncbi_uid=2850321&amp;link_uid=PMC3066043" image-link="/pmc/articles/PMC3066043/figure/F7/" class="imagepopup">Figure 7. FOXO/4E-BP <span class="highlight" style="background-color:">Signaling</span> in Muscles Controls Proteostasis and Systemic Aging.  From: FOXO/4E-BP <span class="highlight" style="background-color:">Signaling</span> in Drosophila Muscles Regulates Organism-wide Proteostasis During Aging. </a></p><div class="supp"><p class="details"><div>Muscle aging is characterized by protein damage and accumulation of cytoplasmic aggregates. Loss of protein homeostasis (proteostasis) associates with the progressive decrease in muscle strength and can affect the lifespan of the organism. Pten/FOXO <span class="highlight" style="background-color:">signaling</span> induces multiple targets, including several folding chaperones and the regulator of protein translation 4E-BP. FOXO/4E-BP activity regulates muscle proteostasis at least in part via the autophagy/lysosome <span class="highlight" style="background-color:">pathway</span> of protein degradation, preserves muscle function, and extends lifespan. In addition, FOXO/4E-BP <span class="highlight" style="background-color:">signaling</span> in muscles decreases feeding behavior that, similar to fasting, results in reduced Insulin release from producing cells. This in turn promotes FOXO and 4E-BP activity in other tissues, preserving proteostasis organism-wide and mitigating systemic aging.</div></p></div><div class="aux"><div class="resc">Fabio Demontis, et al. Cell. ;143(5):813-825.</div><p class="links"><a class="dblinks" href="/pubmed/21111239" ref="ordinalpos=3696&amp;ncbi_uid=2850321&amp;link_uid=21111239">Citation</a><a class="dblinks" href="/pmc/articles/PMC3066043/" ref="ordinalpos=3696&amp;ncbi_uid=2850321&amp;link_uid=PMC3066043">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3697.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3653303/figure/F2/" ref="ordinalpos=3697&amp;ncbi_uid=4897954&amp;link_uid=PMC3653303" href="/pmc/articles/PMC3653303/"><img src="/pmc/articles/instance/3653303/bin/nihms227323f2.gif" src-large="/pmc/articles/instance/3653303/bin/nihms227323f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3653303/" ref="ordinalpos=3697&amp;ncbi_uid=4897954&amp;link_uid=PMC3653303" image-link="/pmc/articles/PMC3653303/figure/F2/" class="imagepopup">Figure 2.  From: IL28B genotype is associated with differential expression of intrahepatic interferon-stimulated genes in chronic hepatitis C patients. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis showed that genes showing differential expression by <i>IL28B</i>-type clustered in the canonical interferon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (FDR P value threshold=0.1). Several additional immune-related pathways reach significance in this analysis.</div></p></div><div class="aux"><div class="resc">Thomas J. Urban, et al. Hepatology. 2010 December;52(6):1888-1896.</div><p class="links"><a class="dblinks" href="/pubmed/20931559" ref="ordinalpos=3697&amp;ncbi_uid=4897954&amp;link_uid=20931559">Citation</a><a class="dblinks" href="/pmc/articles/PMC3653303/" ref="ordinalpos=3697&amp;ncbi_uid=4897954&amp;link_uid=PMC3653303">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3698.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923586/figure/F1/" ref="ordinalpos=3698&amp;ncbi_uid=2423110&amp;link_uid=PMC2923586" href="/pmc/articles/PMC2923586/"><img src="/pmc/articles/instance/2923586/bin/nihms223756f1.gif" src-large="/pmc/articles/instance/2923586/bin/nihms223756f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923586/" ref="ordinalpos=3698&amp;ncbi_uid=2423110&amp;link_uid=PMC2923586" image-link="/pmc/articles/PMC2923586/figure/F1/" class="imagepopup">Fig. 1.  From: Relative Expression Analysis for Identifying Perturbed Pathways. </a></p><div class="supp"><p class="details"><div>Example of tightly regulated <span class="highlight" style="background-color:">pathway</span> in GIST. A simplified diagram of the GS <span class="highlight" style="background-color:">pathway</span>, comprising six <span class="highlight" style="background-color:">signaling</span> proteins downstream of G-protein couple receptors, is shown above. The majority of GIST samples match the pairwise orderings in the GIST rank template exactly.</div></p></div><div class="aux"><div class="resc">James A. Eddy, et al. Conf Proc IEEE Eng Med Biol Soc. ;2009:5456-5459.</div><p class="links"><a class="dblinks" href="/pubmed/19964680" ref="ordinalpos=3698&amp;ncbi_uid=2423110&amp;link_uid=19964680">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923586/" ref="ordinalpos=3698&amp;ncbi_uid=2423110&amp;link_uid=PMC2923586">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3699.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4128997/figure/fig03/" ref="ordinalpos=3699&amp;ncbi_uid=6487838&amp;link_uid=PMC4128997" href="/pmc/articles/PMC4128997/"><img src="/pmc/articles/instance/4128997/bin/ijcep0007-3853-f3.gif" src-large="/pmc/articles/instance/4128997/bin/ijcep0007-3853-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4128997/" ref="ordinalpos=3699&amp;ncbi_uid=6487838&amp;link_uid=PMC4128997" image-link="/pmc/articles/PMC4128997/figure/fig03/" class="imagepopup">Figure 3.  From: Identification of collaboration patterns of dysfunctional pathways in breast cancer. </a></p><div class="supp"><p class="details"><div>The PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. *Genes with red are differentially expressed genes while genes with green are non-differentially expressed genes.</div></p></div><div class="aux"><div class="resc">Xinyong Zhu, et al. Int J Clin Exp Pathol. 2014;7(7):3853-3864.</div><p class="links"><a class="dblinks" href="/pubmed/25120762" ref="ordinalpos=3699&amp;ncbi_uid=6487838&amp;link_uid=25120762">Citation</a><a class="dblinks" href="/pmc/articles/PMC4128997/" ref="ordinalpos=3699&amp;ncbi_uid=6487838&amp;link_uid=PMC4128997">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3700.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4058194/figure/fig8/" ref="ordinalpos=3700&amp;ncbi_uid=6305736&amp;link_uid=PMC4058194" href="/pmc/articles/PMC4058194/"><img src="/pmc/articles/instance/4058194/bin/BMRI2014-375980.008.gif" src-large="/pmc/articles/instance/4058194/bin/BMRI2014-375980.008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4058194/" ref="ordinalpos=3700&amp;ncbi_uid=6305736&amp;link_uid=PMC4058194" image-link="/pmc/articles/PMC4058194/figure/fig8/" class="imagepopup">Figure 8.  From: Simulated Annealing Based Algorithm for Identifying Mutated Driver Pathways in Cancer. </a></p><div class="supp"><p class="details"><div>The genes sets (PTEN, EGFR, PIK3R1, and PIK3CA) are involved in the RTK/RAS/PI(3)K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which is reported in TCGA [15].</div></p></div><div class="aux"><div class="resc">Hai-Tao Li, et al. Biomed Res Int. 2014;2014:375980.</div><p class="links"><a class="dblinks" href="/pubmed/24982873" ref="ordinalpos=3700&amp;ncbi_uid=6305736&amp;link_uid=24982873">Citation</a><a class="dblinks" href="/pmc/articles/PMC4058194/" ref="ordinalpos=3700&amp;ncbi_uid=6305736&amp;link_uid=PMC4058194">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="36" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="37" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="38" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:43:17-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A218F5431210100000000007BB942;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A218F5431210100000000007BB942" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>