
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="10" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC51F353FEBE210000000000313357" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="9" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="10" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="11" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="10" /></div>
            <div><h2 class="result_count">Results: 901 to 1000 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>901.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067743/figure/f0010/" ref="ordinalpos=901&amp;ncbi_uid=6419431&amp;link_uid=PMC4067743" href="/pmc/articles/PMC4067743/"><img src="/pmc/articles/instance/4067743/bin/gr2.gif" src-large="/pmc/articles/instance/4067743/bin/gr2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067743/" ref="ordinalpos=901&amp;ncbi_uid=6419431&amp;link_uid=PMC4067743" image-link="/pmc/articles/PMC4067743/figure/f0010/" class="imagepopup">Fig. 2.  From: ?-Adrenoceptors and synaptic plasticity in the perirhinal cortex. </a></p><div class="supp"><p class="details"><div>β-ADR activation by isoprenaline induced LTP in the PRh–PRh <span class="highlight" style="background-color:">pathway</span>, that was dependent on NMDA receptors, VGCCs and PKA <span class="highlight" style="background-color:">signaling</span>. (A) Isoprenaline (10 μM) induced LTP in the PRh–PRh <span class="highlight" style="background-color:">pathway</span> but only a transient potentiation in the LA–PRh <span class="highlight" style="background-color:">pathway</span>. (A<sub>1</sub>) Pooled data (<i>N</i> = 6) demonstrating the isoprenaline-induced potentiation in the PRh–PRh <span class="highlight" style="background-color:">pathway</span> (●) and the LA–PRh <span class="highlight" style="background-color:">pathway</span> (○). (A<sub>2</sub>) Example of traces for the PRh–PRh <span class="highlight" style="background-color:">pathway</span>, during baseline (black) and 1 h after (red) bath application of isoprenaline. (A<sub>3</sub>) Example of traces for the LA–PRh <span class="highlight" style="background-color:">pathway</span> during baseline (black) and 10–20 min into (red) bath application of isoprenaline. (B) Isoprenaline-induced potentiation of the PRh–PRh input did not occur if stimulation to the slice was ceased during isoprenaline application, and 20 min following washout. (B<sub>1</sub>) Pooled data (<i>N</i> = 3) demonstrating that isoprenaline-induced potentiation of the PRh–PRh <span class="highlight" style="background-color:">pathway</span> (●) was prevented (PRh–PRh: <i>P</i> = 0.12). (B<sub>2</sub>) Example of traces of the PRh–PRh <span class="highlight" style="background-color:">pathway</span>, during baseline (black) and at the end of the experiment (red). (B<sub>3</sub>) Example of traces for the LA–PRh <span class="highlight" style="background-color:">pathway</span> during baseline (black) and at the end of the experiment (red). (C) Pre-application of the NMDAR antagonist L-689,560 (5 μM) blocked isoprenaline-induced potentiation. (C<sub>1</sub>) Pooled data (<i>N</i> = 6) demonstrating that isoprenaline-induced potentiation was blocked in both the PRh–PRh (●, <i>P</i> = 0.45) and the LA–PRh inputs (○, <i>P</i> = 0.24) when isoprenaline was co-applied with L-689,560. (C<sub>2</sub>) Example of traces of the PRh–PRh <span class="highlight" style="background-color:">pathway</span>, during baseline (black) and at (red) the end of the experiment. (C<sub>3</sub>) Example of traces of the LA–PRh <span class="highlight" style="background-color:">pathway</span>, during baseline (black) and during (red) the co-application of isoprenaline and L-689,560 (10–20 min into co-application). (D) PRh–PRh histogram highlighting that the potentiation observed with the bath application of isoprenaline was blocked by antagonism of β1- or β2-ADRs, VGCCs (verapamil 20 μM or nifedipine 10 μM) or PKA (KT 5720 200 nM or H89 15 μM) or. (E) LA–PRh histogram highlighting that the potentiation observed with the bath application of isoprenaline was blocked by antagonism of β1- or β2-ADRs, VGCCs (verapamil 20 μM or nifedipine 10 μM) or PKA (KT 5720 200 nM or H89 15 μM) Antagonism of either the β1-ADRs (metoprolol) or the β2-ADRs (ICI 118,551) was sufficient to prevent isoprenaline-induced potentiation (measurements taken 10–20 min into co-application). (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.)</div></p></div><div class="aux"><div class="resc">M. Laing, et al. Neuroscience. 2014 July 25;273(100):163-173.</div><p class="links"><a class="dblinks" href="/pubmed/24836853" ref="ordinalpos=901&amp;ncbi_uid=6419431&amp;link_uid=24836853">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067743/" ref="ordinalpos=901&amp;ncbi_uid=6419431&amp;link_uid=PMC4067743">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>902.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3279357/figure/pone-0031428-g002/" ref="ordinalpos=902&amp;ncbi_uid=3811255&amp;link_uid=PMC3279357" href="/pmc/articles/PMC3279357/"><img src="/pmc/articles/instance/3279357/bin/pone.0031428.g002.gif" src-large="/pmc/articles/instance/3279357/bin/pone.0031428.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3279357/" ref="ordinalpos=902&amp;ncbi_uid=3811255&amp;link_uid=PMC3279357" image-link="/pmc/articles/PMC3279357/figure/pone-0031428-g002/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Pathway</span> analysis of WT and p53−/− samples cultured in the presence of cytokine..  From: p53-Dependent Transcriptional Responses to Interleukin-3 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) List of activated and inactivated pathways identified by a <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Impact Analysis (SPIA) of array results from WT and <i>p53<sup>−/−</sup></i> samples. The ID is the KEGG ID, pSize indicates the number of genes in the KEGG <span class="highlight" style="background-color:">pathway</span>, NDE is the number of differentially expressed genes found within the <span class="highlight" style="background-color:">pathway</span> and pGFdr is the False Discovery pathways (FDR&lt;0.1). Significant pathways are shown. (B) The differentially expressed genes that account for the significant SPIA pathways (from Figure 2A) are depicted by the heatmap. To the right of the heatmap, the dots indicate the pathways to which each gene contributes and whether a gene is represented by several of the pathways.</div></p></div><div class="aux"><div class="resc">Anissa M. Jabbour, et al. PLoS One. 2012;7(2):e31428.</div><p class="links"><a class="dblinks" href="/pubmed/22348085" ref="ordinalpos=902&amp;ncbi_uid=3811255&amp;link_uid=22348085">Citation</a><a class="dblinks" href="/pmc/articles/PMC3279357/" ref="ordinalpos=902&amp;ncbi_uid=3811255&amp;link_uid=PMC3279357">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>903.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2795724/figure/pone-c9e68857-cd7e-4b1c-9765-04f66d8237d0-g001/" ref="ordinalpos=903&amp;ncbi_uid=6858764&amp;link_uid=PMC2795724" href="/pmc/articles/PMC2795724/"><img src="/pmc/articles/instance/2795724/bin/pone.c9e68857-cd7e-4b1c-9765-04f66d8237d0.g001.gif" src-large="/pmc/articles/instance/2795724/bin/pone.c9e68857-cd7e-4b1c-9765-04f66d8237d0.g001.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2795724/" ref="ordinalpos=903&amp;ncbi_uid=6858764&amp;link_uid=PMC2795724" image-link="/pmc/articles/PMC2795724/figure/pone-c9e68857-cd7e-4b1c-9765-04f66d8237d0-g001/" class="imagepopup">Figure 5. An AIs-Deletions matrix in the mating <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> in budding yeast..  From: Correction: Reverse Engineering a <span class="highlight" style="background-color:">Signaling</span> Network Using Alternative Inputs. </a></p><div class="aux"><div class="resc">Hiromasa Tanaka, et al. PLoS One. 2009;4(12):10.1371/annotation/c9e68857-cd7e-4b1c-9765-04f66d8237d0.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=903&amp;ncbi_uid=6858764&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC2795724/" ref="ordinalpos=903&amp;ncbi_uid=6858764&amp;link_uid=PMC2795724">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>904.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241422/figure/fig6/" ref="ordinalpos=904&amp;ncbi_uid=4404082&amp;link_uid=PMC3241422" href="/pmc/articles/PMC3241422/"><img src="/pmc/articles/instance/3241422/bin/1145fig6.gif" src-large="/pmc/articles/instance/3241422/bin/1145fig6.jpg" alt="Figure 6 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241422/" ref="ordinalpos=904&amp;ncbi_uid=4404082&amp;link_uid=PMC3241422" image-link="/pmc/articles/PMC3241422/figure/fig6/" class="imagepopup">Figure 6 .  From: Regulation of Cell Wall Biogenesis in Saccharomyces cerevisiae: The Cell Wall Integrity <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Control of the Skn7 transcription factor. (A) The Sln1 branch of the HOG <span class="highlight" style="background-color:">pathway</span>. The Sln1 osmosensor controls a phosphorelay <span class="highlight" style="background-color:">pathway</span>, which activates Skn7 under hypo-osmotic conditions to support cell wall biosynthesis and the Hog1 MAPK cascade under hyper-osmotic conditions. Active components are shaded. (B) Coordinate activation of CWI <span class="highlight" style="background-color:">signaling</span>, Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>, and Skn7. Rho1 may independently activate the Skn7 transcription factor (dashed line), which induces stabilization of the Crz1 transcripion factor, and may have additional effects on cell wall stress-induced transcription. The Mid1-Cch1 Ca<sup>2+</sup> channel is activated by many of the same stresses that activate CWI <span class="highlight" style="background-color:">signaling</span>. Additionally, activation of Mpk1 results in stimulation of the Mid1-Cch1 Ca<sup>2+</sup> channel, at least in response to ER stress, which activates the Ca<sup>2+</sup>-dependent protein phosphatase calcineurin. The Crz1 transcription factor is activated through dephosphorylation by calcineurin, which allows its entry to the nucleus. This interplay may coordinate control of gene expression by Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> and cell wall stress <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">David E. Levin. Genetics. 2011 December;189(4):1145-1175.</div><p class="links"><a class="dblinks" href="/pubmed/22174182" ref="ordinalpos=904&amp;ncbi_uid=4404082&amp;link_uid=22174182">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241422/" ref="ordinalpos=904&amp;ncbi_uid=4404082&amp;link_uid=PMC3241422">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>905.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2965086/figure/pone-0013543-g001/" ref="ordinalpos=905&amp;ncbi_uid=2560279&amp;link_uid=PMC2965086" href="/pmc/articles/PMC2965086/"><img src="/pmc/articles/instance/2965086/bin/pone.0013543.g001.gif" src-large="/pmc/articles/instance/2965086/bin/pone.0013543.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2965086/" ref="ordinalpos=905&amp;ncbi_uid=2560279&amp;link_uid=PMC2965086" image-link="/pmc/articles/PMC2965086/figure/pone-0013543-g001/" class="imagepopup">Figure 1. Role of each metric in assessing different aspects of <span class="highlight" style="background-color:">signaling</span> biology..  From: Functional Characterization of FLT3 Receptor <span class="highlight" style="background-color:">Signaling</span> Deregulation in Acute Myeloid Leukemia by Single Cell Network Profiling (SCNP). </a></p><div class="supp"><p class="details"><div>Summary schema of all metrics used in the two studies and the role each has in assessing different aspects of <span class="highlight" style="background-color:">signaling</span> and <span class="highlight" style="background-color:">pathway</span> biology. Median Fluorescence Intensities (MFI) were calculated for leukemic blast cells in each condition and used to compute metrics for Relative Expression of surface expression markers and Basal, Fold and Total measurements of <span class="highlight" style="background-color:">signaling</span> as described in Results. Definitions of metrics used to analyze surface marker expression, <span class="highlight" style="background-color:">signaling</span> data, and apoptosis readouts are shown (i.e. Basal  =  log<sub>2</sub>(Unmodulated<sub>MFI</sub>/Autofluorescence<sub>MFI</sub>).</div></p></div><div class="aux"><div class="resc">David B. Rosen, et al. PLoS One. 2010;5(10):e13543.</div><p class="links"><a class="dblinks" href="/pubmed/21048955" ref="ordinalpos=905&amp;ncbi_uid=2560279&amp;link_uid=21048955">Citation</a><a class="dblinks" href="/pmc/articles/PMC2965086/" ref="ordinalpos=905&amp;ncbi_uid=2560279&amp;link_uid=PMC2965086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>906.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634610/figure/F2/" ref="ordinalpos=906&amp;ncbi_uid=4827897&amp;link_uid=PMC3634610" href="/pmc/articles/PMC3634610/"><img src="/pmc/articles/instance/3634610/bin/IRI-32-134-g002.gif" src-large="/pmc/articles/instance/3634610/bin/IRI-32-134-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634610/" ref="ordinalpos=906&amp;ncbi_uid=4827897&amp;link_uid=PMC3634610" image-link="/pmc/articles/PMC3634610/figure/F2/" class="imagepopup">Figure 2. From: C-Type Lectin-Like Receptors of the Dectin-1 Cluster: Ligands and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div> <b><span class="highlight" style="background-color:">Signaling</span> network of Dectin-1.</b> Multiple pathways downstream of Dectin-1 rely on Syk. This can lead to PLCγ2/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> leading to PKC<i>δ</i> activation that controls calcineurin/NFAT and ROS/NLRP3 inflammasome, and the phosphorylation of CARD9-Bcl10-Malt1. The CARD9 complex is able to activate all NF-κB subunits including c-Rel by removing its inhibitor, IκB, through activation of the IKK complex. Syk can also activate IKK complex through the non-canonical NIK <span class="highlight" style="background-color:">pathway</span>, leading to the activation of NF-κB subunits RelB and p52. This <span class="highlight" style="background-color:">pathway</span> can be inhibited the Raf-1-mediated <span class="highlight" style="background-color:">pathway</span> involving p65. Phosphorylated and acetylated p65 can interact with RelB, sequestering it away from an active NF-κB complex. Resultant activation of NFAT and NF-κB results in modification of gene transcription including cytokines and chemokines. Dashed arrows represent a <span class="highlight" style="background-color:">pathway</span> that has yet to be fully defined.</div></p></div><div class="aux"><div class="resc">Anthony Plato, et al. Int Rev Immunol. 2013 April;32(2):134-156.</div><p class="links"><a class="dblinks" href="/pubmed/23570314" ref="ordinalpos=906&amp;ncbi_uid=4827897&amp;link_uid=23570314">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634610/" ref="ordinalpos=906&amp;ncbi_uid=4827897&amp;link_uid=PMC3634610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>907.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3461296/figure/fig2/" ref="ordinalpos=907&amp;ncbi_uid=4276569&amp;link_uid=PMC3461296" href="/pmc/articles/PMC3461296/"><img src="/pmc/articles/instance/3461296/bin/NRI2012-878030.002.gif" src-large="/pmc/articles/instance/3461296/bin/NRI2012-878030.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3461296/" ref="ordinalpos=907&amp;ncbi_uid=4276569&amp;link_uid=PMC3461296" image-link="/pmc/articles/PMC3461296/figure/fig2/" class="imagepopup">Figure 2.  From: Nrf2/ARE <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: Key Mediator in Oxidative Stress and Potential Therapeutic Target in ALS. </a></p><div class="supp"><p class="details"><div>Schematic representation of major pathways involved in motor neuron death. The Nrf2/ARE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a potential target for blocking multiple death pathways. Green arrows represent neuroprotective pathways, and black arrows point to known neuronal death pathways containing potential cellular targets for antioxidant and anti-inflammatory agents.</div></p></div><div class="aux"><div class="resc">Susanne Petri, et al. Neurol Res Int. 2012;2012:878030.</div><p class="links"><a class="dblinks" href="/pubmed/23050144" ref="ordinalpos=907&amp;ncbi_uid=4276569&amp;link_uid=23050144">Citation</a><a class="dblinks" href="/pmc/articles/PMC3461296/" ref="ordinalpos=907&amp;ncbi_uid=4276569&amp;link_uid=PMC3461296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>908.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3153373/figure/fig08/" ref="ordinalpos=908&amp;ncbi_uid=3164586&amp;link_uid=PMC3153373" href="/pmc/articles/PMC3153373/"><img src="/pmc/articles/instance/3153373/bin/jbmr0025-1167-f8.gif" src-large="/pmc/articles/instance/3153373/bin/jbmr0025-1167-f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3153373/" ref="ordinalpos=908&amp;ncbi_uid=3164586&amp;link_uid=PMC3153373" image-link="/pmc/articles/PMC3153373/figure/fig08/" class="imagepopup">Fig. 8.  From: Smad4-Shh-Nfic <span class="highlight" style="background-color:">Signaling</span> Cascade-Mediated Epithelial-Mesenchymal Interaction Is Crucial in Regulating Tooth Root Development. </a></p><div class="supp"><p class="details"><div>A Smad4-Shh-Gli1-Nfic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates molar root development. Schematic diagram of our model for the Smad4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during tooth root development. <i>Smad4</i> in the dental epithelium is required for <i>Shh</i> expression. Then Shh is released from the dental epithelium, acts through Gli1, and induces <i>Nfic</i> expression in the dental mesenchyme for normal root formation.</div></p></div><div class="aux"><div class="resc">Xiaofeng Huang, et al. J Bone Miner Res. 2010 May;25(5):1167-1178.</div><p class="links"><a class="dblinks" href="/pubmed/19888897" ref="ordinalpos=908&amp;ncbi_uid=3164586&amp;link_uid=19888897">Citation</a><a class="dblinks" href="/pmc/articles/PMC3153373/" ref="ordinalpos=908&amp;ncbi_uid=3164586&amp;link_uid=PMC3153373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>909.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997801/figure/F1/" ref="ordinalpos=909&amp;ncbi_uid=2621077&amp;link_uid=PMC2997801" href="/pmc/articles/PMC2997801/"><img src="/pmc/articles/instance/2997801/bin/halms482245f1.gif" src-large="/pmc/articles/instance/2997801/bin/halms482245f1.jpg" alt="Figure" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997801/" ref="ordinalpos=909&amp;ncbi_uid=2621077&amp;link_uid=PMC2997801" image-link="/pmc/articles/PMC2997801/figure/F1/" class="imagepopup">Figure. The main gene abnormalities targeting NF-κB <span class="highlight" style="background-color:">pathway</span> are displayed in red and italic font.  From: Positioning NF-?B in multiple myeloma. </a></p><div class="supp"><p class="details"><div>They result in aberrant activation of non-canonical and canonical NF-κB <span class="highlight" style="background-color:">pathway</span> that together with other <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation (JAK/STAT, MAPkinase, PI3K/AKT) trigger myeloma cell survival, proliferation and drug resistance.</div></p></div><div class="aux"><div class="resc">Bernard Klein. Blood. ;115(17):3422-3424.</div><p class="links"><a class="dblinks" href="/pubmed/20430962" ref="ordinalpos=909&amp;ncbi_uid=2621077&amp;link_uid=20430962">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997801/" ref="ordinalpos=909&amp;ncbi_uid=2621077&amp;link_uid=PMC2997801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>910.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3926821/figure/F2/" ref="ordinalpos=910&amp;ncbi_uid=5825497&amp;link_uid=PMC3926821" href="/pmc/articles/PMC3926821/"><img src="/pmc/articles/instance/3926821/bin/oncotarget-04-2212-g002.gif" src-large="/pmc/articles/instance/3926821/bin/oncotarget-04-2212-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3926821/" ref="ordinalpos=910&amp;ncbi_uid=5825497&amp;link_uid=PMC3926821" image-link="/pmc/articles/PMC3926821/figure/F2/" class="imagepopup">Figure 2. Activation of the STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by the secretome of senescent melanoma cells.  From: Secretome from senescent melanoma engages the STAT3 <span class="highlight" style="background-color:">pathway</span> to favor reprogramming of naive melanoma towards a tumor-initiating cell phenotype. </a></p><div class="aux"><div class="resc">Mickaël Ohanna, et al. Oncotarget. 2013 December;4(12):2212-2224.</div><p class="links"><a class="dblinks" href="/pubmed/24344100" ref="ordinalpos=910&amp;ncbi_uid=5825497&amp;link_uid=24344100">Citation</a><a class="dblinks" href="/pmc/articles/PMC3926821/" ref="ordinalpos=910&amp;ncbi_uid=5825497&amp;link_uid=PMC3926821">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>911.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3316974/figure/fig1/" ref="ordinalpos=911&amp;ncbi_uid=3963395&amp;link_uid=PMC3316974" href="/pmc/articles/PMC3316974/"><img src="/pmc/articles/instance/3316974/bin/NRI2012-454957.001.gif" src-large="/pmc/articles/instance/3316974/bin/NRI2012-454957.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3316974/" ref="ordinalpos=911&amp;ncbi_uid=3963395&amp;link_uid=PMC3316974" image-link="/pmc/articles/PMC3316974/figure/fig1/" class="imagepopup">Figure 1.  From: The Clinical and Prognostic Significance of Activated AKT-mTOR <span class="highlight" style="background-color:">Pathway</span> in Human Astrocytomas. </a></p><div class="supp"><p class="details"><div>Schematic representation of AKT-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing rapamycin inhibition sites in astrocytomas.</div></p></div><div class="aux"><div class="resc">Elias A. El Habr, et al. Neurol Res Int. 2012;2012:454957.</div><p class="links"><a class="dblinks" href="/pubmed/22530122" ref="ordinalpos=911&amp;ncbi_uid=3963395&amp;link_uid=22530122">Citation</a><a class="dblinks" href="/pmc/articles/PMC3316974/" ref="ordinalpos=911&amp;ncbi_uid=3963395&amp;link_uid=PMC3316974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>912.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3226976/figure/F5/" ref="ordinalpos=912&amp;ncbi_uid=4408997&amp;link_uid=PMC3226976" href="/pmc/articles/PMC3226976/"><img src="/pmc/articles/instance/3226976/bin/nihms-334218-f0005.gif" src-large="/pmc/articles/instance/3226976/bin/nihms-334218-f0005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3226976/" ref="ordinalpos=912&amp;ncbi_uid=4408997&amp;link_uid=PMC3226976" image-link="/pmc/articles/PMC3226976/figure/F5/" class="imagepopup">Fig. 5. Phenotypes of potential acy-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutants.  From: A Gain-of-function Mutation in Adenylate Cyclase Confers Isoflurane Resistance in C. elegans. </a></p><div class="supp"><p class="details"><div>(A) Isoflurane EC<sub>50</sub> values for various genes hypothesized to lie in the <i>acy-1</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (* <i>p</i> &lt; 0.01 <i>vs.</i> wild type). (B) <i>js127</i> increases sensitivity of <i>crh-1</i>(<i>n3315</i>) to 0.35mM aldicarb (<i>p</i> &lt; 0.01). (C) Aldicarb resistance (0.35mM) of <i>unc-10(md1117)</i> is not significantly suppressed by <i>js127</i>. (D) Aldicarb resistance (0.35mM) of <i>snb-1(md247)</i> is not significantly suppressed by <i>js127</i>. (E) Aldicarb resistance (0.35mM) of <i>unc-13(e376)</i> is not significantly suppressed by <i>js127</i>. (F) <i>js127</i> mutation enhances the locomotion of several lethargic mutants but not <i>unc-13</i>(<i>s69</i>) (* <i>p</i> &lt; 0.01 <i>vs.</i> lethargic single mutant).</div></p></div><div class="aux"><div class="resc">Owais Saifee, et al. Anesthesiology. ;115(6):1162-1171.</div><p class="links"><a class="dblinks" href="/pubmed/22024713" ref="ordinalpos=912&amp;ncbi_uid=4408997&amp;link_uid=22024713">Citation</a><a class="dblinks" href="/pmc/articles/PMC3226976/" ref="ordinalpos=912&amp;ncbi_uid=4408997&amp;link_uid=PMC3226976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>913.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3433406/figure/F3/" ref="ordinalpos=913&amp;ncbi_uid=5162054&amp;link_uid=PMC3433406" href="/pmc/articles/PMC3433406/"><img src="/pmc/articles/instance/3433406/bin/nihms386618f3.gif" src-large="/pmc/articles/instance/3433406/bin/nihms386618f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3433406/" ref="ordinalpos=913&amp;ncbi_uid=5162054&amp;link_uid=PMC3433406" image-link="/pmc/articles/PMC3433406/figure/F3/" class="imagepopup">Fig. 3. <span class="highlight" style="background-color:">Signaling</span> pathways involving identified phosphoproteins.  From: Large scale phosphoproteome analysis of LNCaP human prostate cancer cells. </a></p><div class="supp"><p class="details"><div>Different <span class="highlight" style="background-color:">signaling</span> pathways hits are represented by PANTHER analysis from all identified phosphoproteins in LNCaP cells. The major <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was Wnt, followed by cadherin, Rho GTPase, heterotrimeric G-protein and Gi alpha and Gs alpha mediated <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Jae-Kyung Myung, et al. Mol Biosyst. 2012 August;8(8):2174-2182.</div><p class="links"><a class="dblinks" href="/pubmed/22699815" ref="ordinalpos=913&amp;ncbi_uid=5162054&amp;link_uid=22699815">Citation</a><a class="dblinks" href="/pmc/articles/PMC3433406/" ref="ordinalpos=913&amp;ncbi_uid=5162054&amp;link_uid=PMC3433406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>914.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2804301/figure/F3/" ref="ordinalpos=914&amp;ncbi_uid=2710849&amp;link_uid=PMC2804301" href="/pmc/articles/PMC2804301/"><img src="/pmc/articles/instance/2804301/bin/btp640f3.gif" src-large="/pmc/articles/instance/2804301/bin/btp640f3.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2804301/" ref="ordinalpos=914&amp;ncbi_uid=2710849&amp;link_uid=PMC2804301" image-link="/pmc/articles/PMC2804301/figure/F3/" class="imagepopup">Fig. 3. From: <span class="highlight" style="background-color:">Pathway</span> analysis using random forests with bivariate node-split for survival outcomes. </a></p><div class="supp"><p class="details"><div>An outliers plot of 30 patients for TGF beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Herbert Pang, et al. Bioinformatics. 2010 January 15;26(2):250-258.</div><p class="links"><a class="dblinks" href="/pubmed/19933158" ref="ordinalpos=914&amp;ncbi_uid=2710849&amp;link_uid=19933158">Citation</a><a class="dblinks" href="/pmc/articles/PMC2804301/" ref="ordinalpos=914&amp;ncbi_uid=2710849&amp;link_uid=PMC2804301">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>915.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3922882/figure/pone-0088550-g002/" ref="ordinalpos=915&amp;ncbi_uid=5819302&amp;link_uid=PMC3922882" href="/pmc/articles/PMC3922882/"><img src="/pmc/articles/instance/3922882/bin/pone.0088550.g002.gif" src-large="/pmc/articles/instance/3922882/bin/pone.0088550.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3922882/" ref="ordinalpos=915&amp;ncbi_uid=5819302&amp;link_uid=PMC3922882" image-link="/pmc/articles/PMC3922882/figure/pone-0088550-g002/" class="imagepopup">Figure 2. Shh upregulates the Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and multiple cytokines in Shh transfected chondrocytes..  From: Sonic Hedgehog Improves Redifferentiation of Dedifferentiated Chondrocytes for Articular Cartilage Repair. </a></p><div class="supp"><p class="details"><div>To explore potential mechanisms responsible for the redifferentiation effect of Shh, we evaluated the Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Real time RT-PCR and western blot analysis verified the stronger expression of Ptc1, Gli1 and Sox9 in Shh transfected cells after 48 hours (A, B). The mRNA expression of BMP2 and IGF1 were upregulated by Shh after 48 hours (C). *p&lt;0.05.</div></p></div><div class="aux"><div class="resc">Lin Lin, et al. PLoS One. 2014;9(2):e88550.</div><p class="links"><a class="dblinks" href="/pubmed/24533105" ref="ordinalpos=915&amp;ncbi_uid=5819302&amp;link_uid=24533105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3922882/" ref="ordinalpos=915&amp;ncbi_uid=5819302&amp;link_uid=PMC3922882">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>916.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3644346/figure/F6/" ref="ordinalpos=916&amp;ncbi_uid=6078055&amp;link_uid=PMC3644346" href="/pmc/articles/PMC3644346/"><img src="/pmc/articles/instance/3644346/bin/nihms-450062-f0006.gif" src-large="/pmc/articles/instance/3644346/bin/nihms-450062-f0006.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3644346/" ref="ordinalpos=916&amp;ncbi_uid=6078055&amp;link_uid=PMC3644346" image-link="/pmc/articles/PMC3644346/figure/F6/" class="imagepopup">Fig. 6. An illustration showing the modulation of the BRAF/MEK/ERK/RSK2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by silybin in SK-MEL-5 cells.  From: Direct targeting of MEK1/2 and RSK2 by silybin induces cell cycle arrest and inhibits melanoma cell growth. </a></p><div class="supp"><p class="details"><div>In melanoma, the mutation of BRAF (V600E) constitutively activates the down–stream kinase MEK1/2, which phosphorylates ERK2. Phosphorylated ERK2 activates RSK2, which transmits a signal downstream to transcription factors, such as AP-1, NF-κB and STAT3, and induces their transcriptional activity to express cyclin D1. Silybin inhibits this oncogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by direct regulation of MEK1/2 and RSK2.</div></p></div><div class="aux"><div class="resc">Mee-Hyun Lee, et al. Cancer Prev Res (Phila). 2013 May;6(5):455-465.</div><p class="links"><a class="dblinks" href="/pubmed/23447564" ref="ordinalpos=916&amp;ncbi_uid=6078055&amp;link_uid=23447564">Citation</a><a class="dblinks" href="/pmc/articles/PMC3644346/" ref="ordinalpos=916&amp;ncbi_uid=6078055&amp;link_uid=PMC3644346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>917.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4102304/figure/F4/" ref="ordinalpos=917&amp;ncbi_uid=6383968&amp;link_uid=PMC4102304" href="/pmc/articles/PMC4102304/"><img src="/pmc/articles/instance/4102304/bin/nihms585242f4.gif" src-large="/pmc/articles/instance/4102304/bin/nihms585242f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4102304/" ref="ordinalpos=917&amp;ncbi_uid=6383968&amp;link_uid=PMC4102304" image-link="/pmc/articles/PMC4102304/figure/F4/" class="imagepopup">Figure 4. Simplified schematic of the proposed relationship between BMPR2 <span class="highlight" style="background-color:">signaling</span>, canonical TGFβ <span class="highlight" style="background-color:">signaling</span>, and caveolae.  From: HERITABLE FORMS OF PULMONARY ARTERIAL HYPERTENSION. </a></p><div class="supp"><p class="details"><div>Caveolae are receptor-rich regions of the plasma membrane, populated by multiple receptor types including BMPR2 and TGFβ receptors. A delicate balance between BMPR2 <span class="highlight" style="background-color:">pathway</span> and TGFβ <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> modulates pro-proliferative and apoptotic forces. A <i>BMPR2</i> mutation, or reduced wild-type <i>BMPR2</i> gene expression, may perturb this balance—perturbation may promote PAH pathogenesis. Disruption of caveolae structure or function could also disrupt cell <span class="highlight" style="background-color:">signaling</span> by reducing receptor density.</div></p></div><div class="aux"><div class="resc">Eric D. Austin, et al. Semin Respir Crit Care Med. ;34(5):568-580.</div><p class="links"><a class="dblinks" href="/pubmed/24037626" ref="ordinalpos=917&amp;ncbi_uid=6383968&amp;link_uid=24037626">Citation</a><a class="dblinks" href="/pmc/articles/PMC4102304/" ref="ordinalpos=917&amp;ncbi_uid=6383968&amp;link_uid=PMC4102304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>918.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F3/" ref="ordinalpos=918&amp;ncbi_uid=4064478&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f3.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=918&amp;ncbi_uid=4064478&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F3/" class="imagepopup">Figure 3. Deletion of plexin-D1 dramatically increases mEPSC frequency in direct <span class="highlight" style="background-color:">pathway</span> MSNs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Traces showing mEPSCs recorded from direct <span class="highlight" style="background-color:">pathway</span> MSNs from <i>Drd2-GFP;Drd1a</i>- Cre;<i>plexin-D1</i> conditional littermate control (<i>top</i>) and KO mice (<i>bottom</i>). (<b>b</b>) Cumulative probability plots of mEPSC inter-event intervals (<i>left</i>) and summary of mean mEPSC frequency (<i>right</i>) in direct <span class="highlight" style="background-color:">pathway</span> MSNs. (*<i>P</i>&lt;0.05; Mann-Whitney). (<b>c</b>) Cumulative probability plots of mEPSC amplitudes (<i>left</i>) and summary of mean mEPSC amplitude (<i>right</i>) in direct <span class="highlight" style="background-color:">pathway</span> MSNs. (<b>d</b>) Traces showing mEPSCs recorded from indirect <span class="highlight" style="background-color:">pathway</span> MSNs from <i>Drd2-GFP;Drd2</i>-Cre;<i>plexin-D1</i> littermate control (<i>top</i>) and conditional KO (<i>bottom</i>) mice. (<b>e</b>) Cumulative probability plots of mEPSC inter-event intervals (<i>left</i>) and summary of mean mEPSC frequency (<i>right</i>) in indirect <span class="highlight" style="background-color:">pathway</span> MSNs. (<b>f</b>) Cumulative probability plots of mEPSC amplitudes (<i>left</i>) summary of mean mEPSC amplitude (<i>right</i>) in indirect <span class="highlight" style="background-color:">pathway</span> MSNs. Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=918&amp;ncbi_uid=4064478&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=918&amp;ncbi_uid=4064478&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>919.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4051765/figure/pone-0099772-g001/" ref="ordinalpos=919&amp;ncbi_uid=6263291&amp;link_uid=PMC4051765" href="/pmc/articles/PMC4051765/"><img src="/pmc/articles/instance/4051765/bin/pone.0099772.g001.gif" src-large="/pmc/articles/instance/4051765/bin/pone.0099772.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4051765/" ref="ordinalpos=919&amp;ncbi_uid=6263291&amp;link_uid=PMC4051765" image-link="/pmc/articles/PMC4051765/figure/pone-0099772-g001/" class="imagepopup">Figure 1. Diagram of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modified by coding the change of transcripts in the BAT of rats responding to 4-hours of cold stimulation (4°C) as compared to the control group (22°C)..  From: Responses of the Insulin <span class="highlight" style="background-color:">Signaling</span> Pathways in the Brown Adipose Tissue of Rats following Cold Exposure. </a></p><div class="supp"><p class="details"><div>Up-regulated transcripts are red (the darker red, the more up-regulation). Down-regulated ones are green (the darker green, the more down-regulation). The unchanged transcripts have no color. The signs between two components represent activation (arrow) and inhibition (bar or bar crossing arrow). Our data show that 4-hours of cold stimulation regulated multiple transcripts of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in BAT of rats.</div></p></div><div class="aux"><div class="resc">Xiaofei Wang, et al. PLoS One. 2014;9(6):e99772.</div><p class="links"><a class="dblinks" href="/pubmed/24915042" ref="ordinalpos=919&amp;ncbi_uid=6263291&amp;link_uid=24915042">Citation</a><a class="dblinks" href="/pmc/articles/PMC4051765/" ref="ordinalpos=919&amp;ncbi_uid=6263291&amp;link_uid=PMC4051765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>920.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2464318/figure/f1-mol14_7p0538/" ref="ordinalpos=920&amp;ncbi_uid=6856008&amp;link_uid=PMC2464318" href="/pmc/articles/PMC2464318/"><img src="/pmc/articles/instance/2464318/bin/mol14_7p0538f1.gif" src-large="/pmc/articles/instance/2464318/bin/mol14_7p0538f1.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2464318/" ref="ordinalpos=920&amp;ncbi_uid=6856008&amp;link_uid=PMC2464318" image-link="/pmc/articles/PMC2464318/figure/f1-mol14_7p0538/" class="imagepopup">Figure 7. Proposed <span class="highlight" style="background-color:">pathway</span>.  From: ERRATUM. </a></p><div class="supp"><p class="details"><div>In this model, IGF-I peptide binds to IGF-IR to induce tyrosine autophosphorylation and phosphorylation of its adaptor protein Shc. Phosphorylated Shc then is associated with the Grb2-mSOS complex to activate p21/Ras, which leads to proliferation by activation of the Ras/Raf/MAPK <span class="highlight" style="background-color:">pathway</span>. The IGF-I and IGF-IR complex also autophosphorylates its docking protein IRS-I, which, in turn, activates the survival PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. However, on the basis of our studies, this <span class="highlight" style="background-color:">pathway</span> appears not to be playing a major role in leiomyoma growth. IRS-I also may recruit Grb2, but it appears that the Shc-Grb2 <span class="highlight" style="background-color:">pathway</span> is the predominant activator of p21/Ras in IGF-IR <span class="highlight" style="background-color:">signaling</span> in UtLM cells and uterine leiomyoma tissue.</div></p></div><div class="aux"><div class="resc">Mol Med. 2008 Jul-Aug;14(7-8):538-538.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=920&amp;ncbi_uid=6856008&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC2464318/" ref="ordinalpos=920&amp;ncbi_uid=6856008&amp;link_uid=PMC2464318">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>921.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3115238/figure/F1/" ref="ordinalpos=921&amp;ncbi_uid=3032825&amp;link_uid=PMC3115238" href="/pmc/articles/PMC3115238/"><img src="/pmc/articles/instance/3115238/bin/psb0511_1384_fig001.gif" src-large="/pmc/articles/instance/3115238/bin/psb0511_1384_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3115238/" ref="ordinalpos=921&amp;ncbi_uid=3032825&amp;link_uid=PMC3115238" image-link="/pmc/articles/PMC3115238/figure/F1/" class="imagepopup">Figure 1.  From: The more, the merrier. </a></p><div class="supp"><p class="details"><div>Phylogenetic relation of plant species used in cytokinin <span class="highlight" style="background-color:">signaling</span> research. Species written in grey were used in bioinformatic analysis.5 Black font marks species in which experimental data for cytokinin <span class="highlight" style="background-color:">signaling</span> components have been obtained on the RNA level. In species shaded in grey functional assays have been performed for members of the cytokinin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (tree based on Sitte et al. 2002).65</div></p></div><div class="aux"><div class="resc">Eva Hellmann, et al. Plant Signal Behav. 2010 November;5(11):1384-1390.</div><p class="links"><a class="dblinks" href="/pubmed/21045560" ref="ordinalpos=921&amp;ncbi_uid=3032825&amp;link_uid=21045560">Citation</a><a class="dblinks" href="/pmc/articles/PMC3115238/" ref="ordinalpos=921&amp;ncbi_uid=3032825&amp;link_uid=PMC3115238">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>922.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4180639/figure/fig7/" ref="ordinalpos=922&amp;ncbi_uid=6732466&amp;link_uid=PMC4180639" href="/pmc/articles/PMC4180639/"><img src="/pmc/articles/instance/4180639/bin/onc201469f7.gif" src-large="/pmc/articles/instance/4180639/bin/onc201469f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4180639/" ref="ordinalpos=922&amp;ncbi_uid=6732466&amp;link_uid=PMC4180639" image-link="/pmc/articles/PMC4180639/figure/fig7/" class="imagepopup">Figure 7.  From: Ahnak functions as a tumor suppressor via modulation of TGF?/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Putative model of Ahnak-dependent activation of TGFβ <span class="highlight" style="background-color:">signaling</span>. Ahnak protein interacts with R-Smad proteins for activation of R-Smad-dependent transcriptional regulation and with I-Smad for inhibiting I-Smad-mediated <span class="highlight" style="background-color:">pathway</span>, resulting in the downregulation of c-Myc and cyclin D protein. Downregulation of cyclin D initiates a series of events leading to the inactivation of CDK4 and dephosphorylation of Rb protein, resulting in cell cycle arrest at the G0/G1 stage. Thus, Ahnak serves as novel tumor suppressor through potentiation of TGFβ <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">I H Lee, et al. Oncogene. 2014 September 18;33(38):4675-4684.</div><p class="links"><a class="dblinks" href="/pubmed/24662814" ref="ordinalpos=922&amp;ncbi_uid=6732466&amp;link_uid=24662814">Citation</a><a class="dblinks" href="/pmc/articles/PMC4180639/" ref="ordinalpos=922&amp;ncbi_uid=6732466&amp;link_uid=PMC4180639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>923.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2947266/figure/fig10/" ref="ordinalpos=923&amp;ncbi_uid=3257938&amp;link_uid=PMC2947266" href="/pmc/articles/PMC2947266/"><img src="/pmc/articles/instance/2947266/bin/gr10.gif" src-large="/pmc/articles/instance/2947266/bin/gr10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2947266/" ref="ordinalpos=923&amp;ncbi_uid=3257938&amp;link_uid=PMC2947266" image-link="/pmc/articles/PMC2947266/figure/fig10/" class="imagepopup">Figure 10.  From: tPA Activates LDL Receptor-Related Protein 1-Mediated Mitogenic <span class="highlight" style="background-color:">Signaling</span> Involving the p90RSK and GSK3? <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> leading to tPA induced fibroblast proliferation. tPA binds to membrane receptor LRP-1, inducing LRP-1 phosphorylation at Tyr<sup>4507</sup> residue, which in turn triggers the phosphorylation of Erk 1/2, p90RSK, GSK3β, and induction of cyclin D1. Blockade of any step within this <span class="highlight" style="background-color:">signaling</span> cascade by different strategies (<b>A–E</b>) abolishes the mitogenic effect of tPA, whereas overexpression of Mek1 mimics tPA and promotes interstitial fibroblast/myofibroblast proliferation (<b>F</b>).</div></p></div><div class="aux"><div class="resc">Ling Lin, et al. Am J Pathol. 2010 October;177(4):1687-1696.</div><p class="links"><a class="dblinks" href="/pubmed/20724593" ref="ordinalpos=923&amp;ncbi_uid=3257938&amp;link_uid=20724593">Citation</a><a class="dblinks" href="/pmc/articles/PMC2947266/" ref="ordinalpos=923&amp;ncbi_uid=3257938&amp;link_uid=PMC2947266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>924.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3303995/figure/F1/" ref="ordinalpos=924&amp;ncbi_uid=4754900&amp;link_uid=PMC3303995" href="/pmc/articles/PMC3303995/"><img src="/pmc/articles/instance/3303995/bin/nihms-341094-f0001.gif" src-large="/pmc/articles/instance/3303995/bin/nihms-341094-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3303995/" ref="ordinalpos=924&amp;ncbi_uid=4754900&amp;link_uid=PMC3303995" image-link="/pmc/articles/PMC3303995/figure/F1/" class="imagepopup">Figure 1.  From: Integrating <span class="highlight" style="background-color:">Pathway</span> Analysis and Genetics of Gene Expression for Genome-wide Association Study of Basal Cell Carcinoma. </a></p><div class="supp"><p class="details"><div>Quantile Quantile Plots of the representative eSNP p values for two pathways: A) autoimmune thyroid disease <span class="highlight" style="background-color:">pathway</span> and B) Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mingfeng Zhang, et al. Hum Genet. ;131(4):615-623.</div><p class="links"><a class="dblinks" href="/pubmed/22006220" ref="ordinalpos=924&amp;ncbi_uid=4754900&amp;link_uid=22006220">Citation</a><a class="dblinks" href="/pmc/articles/PMC3303995/" ref="ordinalpos=924&amp;ncbi_uid=4754900&amp;link_uid=PMC3303995">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>925.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3039708/figure/F2/" ref="ordinalpos=925&amp;ncbi_uid=3654093&amp;link_uid=PMC3039708" href="/pmc/articles/PMC3039708/"><img src="/pmc/articles/instance/3039708/bin/nihms243353f2.gif" src-large="/pmc/articles/instance/3039708/bin/nihms243353f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3039708/" ref="ordinalpos=925&amp;ncbi_uid=3654093&amp;link_uid=PMC3039708" image-link="/pmc/articles/PMC3039708/figure/F2/" class="imagepopup">Figure 2. Regulatory mechanisms of PD-L1 and B7-H3 expression.  From: T cell coinhibition in prostate cancer: new immune evasion pathways and emerging therapeutics. </a></p><div class="supp"><p class="details"><div>The PD-L1 promoter has binding sites for transcription factors IRF-1 and NF-κB, so IFN-γ-mediated <span class="highlight" style="background-color:">signaling</span> through the JAK/STAT-1/IRF-1 <span class="highlight" style="background-color:">pathway</span> and TLR ligand-mediated <span class="highlight" style="background-color:">signaling</span> through the MEK/ERK/NF-κB <span class="highlight" style="background-color:">pathway</span> can upregulate PD-L1 gene transcription. <span class="highlight" style="background-color:">Signaling</span> through the PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>, usually inhibited by PTEN, can activate S6K1, which regulates the 5′ UTR of PD-L1. This results in recruitment of PD-L1 transcripts to polysomes and associated PD-L1 translation. By contrast, miR-513, which can be inhibited by IFN-γ <span class="highlight" style="background-color:">signaling</span>, prevents PD-L1 translation by binding to the PD-L1 3′ UTR. B7-H3 expression can also be enhanced by IFN-γ <span class="highlight" style="background-color:">signaling</span>, perhaps through blocking miR-29, which otherwise represses B7-H3 translation. It seems that different types of cells use overlapping yet divergent pathways to regulate the expression of PD-L1 and B7-H3.</div></p></div><div class="aux"><div class="resc">Yael S Barach, et al. Trends Mol Med. </div><p class="links"><a class="dblinks" href="/pubmed/20971039" ref="ordinalpos=925&amp;ncbi_uid=3654093&amp;link_uid=20971039">Citation</a><a class="dblinks" href="/pmc/articles/PMC3039708/" ref="ordinalpos=925&amp;ncbi_uid=3654093&amp;link_uid=PMC3039708">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>926.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3308032/figure/F2/" ref="ordinalpos=926&amp;ncbi_uid=3865871&amp;link_uid=PMC3308032" href="/pmc/articles/PMC3308032/"><img src="/pmc/articles/instance/3308032/bin/gm293-2.gif" src-large="/pmc/articles/instance/3308032/bin/gm293-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3308032/" ref="ordinalpos=926&amp;ncbi_uid=3865871&amp;link_uid=PMC3308032" image-link="/pmc/articles/PMC3308032/figure/F2/" class="imagepopup">Figure 2.  From: Gene expression and network-based analysis reveals a novel role for hsa-miR-9 and drug control over the p38 network in glioblastoma multiforme progression. </a></p><div class="supp"><p class="details"><div><b>hsa-miR-9 regulation of the p38 <span class="highlight" style="background-color:">pathway</span></b>. <b>(a) </b>Distribution of p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activity levels across groups. Group 1 (blue, higher survival rates) has low <span class="highlight" style="background-color:">pathway</span> activity, and group 2 (green, lower survival rates) has higher activity levels. This figure demonstrates the large range in the activity levels between groups, and the distinct difference between them. <b>(b) </b>The p38 <span class="highlight" style="background-color:">signaling</span> network. Genes highlighted in blue are in the p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and the genes in red boxes are those found by PITA to be possibly targeted by hsa-miR-9. <b>(c) </b>Correlation between hsa-miR-9 and p38 <span class="highlight" style="background-color:">pathway</span> levels. Group 1 (blue dots) has a significant, strong negative correlation between miRNA expression levels and <span class="highlight" style="background-color:">pathway</span> activity while group 2 (green dots) has a lower correlation value. The groups presented here are based on the p38 survival groups.</div></p></div><div class="aux"><div class="resc">Rotem Ben-Hamo, et al. Genome Med. 2011;3(11):77-77.</div><p class="links"><a class="dblinks" href="/pubmed/22122801" ref="ordinalpos=926&amp;ncbi_uid=3865871&amp;link_uid=22122801">Citation</a><a class="dblinks" href="/pmc/articles/PMC3308032/" ref="ordinalpos=926&amp;ncbi_uid=3865871&amp;link_uid=PMC3308032">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>927.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2776288/figure/fig02/" ref="ordinalpos=927&amp;ncbi_uid=2105710&amp;link_uid=PMC2776288" href="/pmc/articles/PMC2776288/"><img src="/pmc/articles/instance/2776288/bin/ajtr0001-0001-f2.gif" src-large="/pmc/articles/instance/2776288/bin/ajtr0001-0001-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2776288/" ref="ordinalpos=927&amp;ncbi_uid=2105710&amp;link_uid=PMC2776288" image-link="/pmc/articles/PMC2776288/figure/fig02/" class="imagepopup">Figure 2.  From: Overcoming cancer therapy resistance by targeting inhibitors of apoptosis proteins and nuclear factor-kappa B. </a></p><div class="supp"><p class="details"><div>Crosstalk between ionizing radiation-induced apoptosis and NF-κB <span class="highlight" style="background-color:">pathway</span>. IR activates both apoptosis and NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IR triggers instrinsic apoptosis <span class="highlight" style="background-color:">pathway</span> as described in Figure 1. Simultaneously, NF-κB <span class="highlight" style="background-color:">pathway</span> is activated by IR-induced DNA damage. IR induced double-strand breaks (DSB) directly activates initiator kinase ATM and PIDD, which further recruit NEMO, also known as IKKγ, by multiple steps of posttranslational modification. Modified NEMO joins formation of IKKs signalosome and further activates classical NF-κB <span class="highlight" style="background-color:">pathway</span>. Upon apoptotic stimuli (IR or TNFα), natural Smac can bind to both XIAP and cIAPs, and predominantly neutralize the suppression effect of XIAP on caspases. It has been shown that cIAP-1 physically binds to TRAF2 and regulates its function through ubiquitination in TNFα <span class="highlight" style="background-color:">signaling</span>. Smac-mimetics that function as IAP-antagonists, not only promote caspase activation by neutralizing IAPs, but also interferes the stability and interaction between cIAP-1/2 and TRAF2, thus modulate the balance between apoptosis and NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yao Dai, et al. Am J Transl Res. 2009;1(1):1-15.</div><p class="links"><a class="dblinks" href="/pubmed/19966933" ref="ordinalpos=927&amp;ncbi_uid=2105710&amp;link_uid=19966933">Citation</a><a class="dblinks" href="/pmc/articles/PMC2776288/" ref="ordinalpos=927&amp;ncbi_uid=2105710&amp;link_uid=PMC2776288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>928.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC64993/figure/F2/" ref="ordinalpos=928&amp;ncbi_uid=154429&amp;link_uid=PMC64993" href="/pmc/articles/PMC64993/"><img src="/pmc/articles/instance/64993/bin/pq2625739002.gif" src-large="/pmc/articles/instance/64993/bin/pq2625739002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC64993/" ref="ordinalpos=928&amp;ncbi_uid=154429&amp;link_uid=PMC64993" image-link="/pmc/articles/PMC64993/figure/F2/" class="imagepopup">Figure 2.  From: A genome-wide analysis of immune responses in Drosophila. </a></p><div class="supp"><p class="details"><div>Distal components of the dorso-ventral, Toll, and Imd <span class="highlight" style="background-color:">signaling</span>  pathways. (<i>A</i>) The distal part of the embryonic  dorso-ventral patterning <span class="highlight" style="background-color:">pathway</span>, showing the end of the proteolytic  cascade that leads to the cleavage of Spaetzle and intracellular  <span class="highlight" style="background-color:">signaling</span>. (<i>B</i>) Established elements of the distal part  of the Toll <span class="highlight" style="background-color:">pathway</span>, where a serpin, <i>nec</i>, regulated a  proteolytic cascade leading to the cleavage of Spaetzle. This <span class="highlight" style="background-color:">pathway</span>  reuses many of the components seen in embryonic dorso-ventral  patterning, and it is induced by infection with Gram-positive bacteria  and fungi. (<i>C</i>) Established elements of the Imd <span class="highlight" style="background-color:">pathway</span>,  which is induced by Gram-negative bacteria. At present, none of  components in the extracellular cascades are known for this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Phil Irving, et al. Proc Natl Acad Sci U S A. 2001 December 18;98(26):15119-15124.</div><p class="links"><a class="dblinks" href="/pubmed/11742098" ref="ordinalpos=928&amp;ncbi_uid=154429&amp;link_uid=11742098">Citation</a><a class="dblinks" href="/pmc/articles/PMC64993/" ref="ordinalpos=928&amp;ncbi_uid=154429&amp;link_uid=PMC64993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>929.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4107940/figure/F1/" ref="ordinalpos=929&amp;ncbi_uid=6467472&amp;link_uid=PMC4107940" href="/pmc/articles/PMC4107940/"><img src="/pmc/articles/instance/4107940/bin/fimmu-05-00343-g001.gif" src-large="/pmc/articles/instance/4107940/bin/fimmu-05-00343-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4107940/" ref="ordinalpos=929&amp;ncbi_uid=6467472&amp;link_uid=PMC4107940" image-link="/pmc/articles/PMC4107940/figure/F1/" class="imagepopup">Figure 1.  From: Pattern Recognition Receptors and DNA Repair: Starting to Put a Jigsaw Puzzle Together. </a></p><div class="supp"><p class="details"><div><b>The general interplay between the canonical TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the cytokine-mediated DNA repair feedback loop, and the growth factors-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. There are three main TLR-mediated pathways of DNA repair. The protein encoded by myeloid differentiation primary response gene 88 (MyD88) and its downstream <span class="highlight" style="background-color:">signaling</span> proteins (not shown) may inhibit mitogen-activated protein kinase phosphatase 3 (MKP3), which hinders phosphorylation of extracellular signal-regulated kinase (ERK), and therefore, prevents further <span class="highlight" style="background-color:">signaling</span> via Ras-MAPK <span class="highlight" style="background-color:">pathway</span>. In addition, MyD88 and its downstream <span class="highlight" style="background-color:">signaling</span> proteins (not shown) along with pERK activate AP-1 transcription factor, which promotes transcription of certain DNA repair genes. Finally, IL-12 and IL-23, which enhance DNA repair and whose transcription is also amplified by MyD88-regulated transcription factors, bind to their receptors, activate Janus kinase-signal transducer and activator of transcription (JAK-STAT) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and increase further transcription of their own encoding genes.</div></p></div><div class="aux"><div class="resc">Anton G. Kutikhin, et al. Front Immunol. 2014;5:343.</div><p class="links"><a class="dblinks" href="/pubmed/25101085" ref="ordinalpos=929&amp;ncbi_uid=6467472&amp;link_uid=25101085">Citation</a><a class="dblinks" href="/pmc/articles/PMC4107940/" ref="ordinalpos=929&amp;ncbi_uid=6467472&amp;link_uid=PMC4107940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>930.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC303340/figure/f8/" ref="ordinalpos=930&amp;ncbi_uid=340843&amp;link_uid=PMC303340" href="/pmc/articles/PMC303340/"><img src="/pmc/articles/instance/303340/bin/mb0140745008.gif" src-large="/pmc/articles/instance/303340/bin/mb0140745008.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC303340/" ref="ordinalpos=930&amp;ncbi_uid=340843&amp;link_uid=PMC303340" image-link="/pmc/articles/PMC303340/figure/f8/" class="imagepopup">FIG. 8. From: Activation of the RAS/Cyclic AMP <span class="highlight" style="background-color:">Pathway</span> Suppresses a TOR Deficiency in Yeast. </a></p><div class="supp"><p class="details"><div>Model of rapamycin-sensitive TOR <span class="highlight" style="background-color:">signaling</span> in the control of cell growth. Rapamycin-sensitive TOR <span class="highlight" style="background-color:">signaling</span> bifurcates into two separate effector pathways, TAP42/SIT4 and RAS/cAMP/PKA. Each effector <span class="highlight" style="background-color:">pathway</span> controls a specific subset of TOR readouts. For details, see Discussion.</div></p></div><div class="aux"><div class="resc">Tobias Schmelzle, et al. Mol Cell Biol. 2004 January;24(1):338-351.</div><p class="links"><a class="dblinks" href="/pubmed/14673167" ref="ordinalpos=930&amp;ncbi_uid=340843&amp;link_uid=14673167">Citation</a><a class="dblinks" href="/pmc/articles/PMC303340/" ref="ordinalpos=930&amp;ncbi_uid=340843&amp;link_uid=PMC303340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>931.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4152338/figure/pone-0095254-g009/" ref="ordinalpos=931&amp;ncbi_uid=6588233&amp;link_uid=PMC4152338" href="/pmc/articles/PMC4152338/"><img src="/pmc/articles/instance/4152338/bin/pone.0095254.g009.gif" src-large="/pmc/articles/instance/4152338/bin/pone.0095254.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4152338/" ref="ordinalpos=931&amp;ncbi_uid=6588233&amp;link_uid=PMC4152338" image-link="/pmc/articles/PMC4152338/figure/pone-0095254-g009/" class="imagepopup">Figure 9. Schematic <span class="highlight" style="background-color:">signaling</span> and possible function of aldosterone-induced galectin-3 expression in macrophages..  From: Aldosterone Induced Galectin-3 Secretion In Vitro and In Vivo: From Cells to Humans. </a></p><div class="supp"><p class="details"><div>Through a MR/PI3K/Akt/NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, aldosterone-induced galectin-3 expression enhanced fibrosis-related factor expression in fibroblasts.</div></p></div><div class="aux"><div class="resc">Yen-Hung Lin, et al. PLoS One. 2014;9(9):e95254.</div><p class="links"><a class="dblinks" href="/pubmed/25180794" ref="ordinalpos=931&amp;ncbi_uid=6588233&amp;link_uid=25180794">Citation</a><a class="dblinks" href="/pmc/articles/PMC4152338/" ref="ordinalpos=931&amp;ncbi_uid=6588233&amp;link_uid=PMC4152338">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>932.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3636761/figure/F1/" ref="ordinalpos=932&amp;ncbi_uid=4855826&amp;link_uid=PMC3636761" href="/pmc/articles/PMC3636761/"><img src="/pmc/articles/instance/3636761/bin/fonc-03-00093-g001.gif" src-large="/pmc/articles/instance/3636761/bin/fonc-03-00093-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3636761/" ref="ordinalpos=932&amp;ncbi_uid=4855826&amp;link_uid=PMC3636761" image-link="/pmc/articles/PMC3636761/figure/F1/" class="imagepopup">Figure 1.  From: Dissecting the PI3K <span class="highlight" style="background-color:">Signaling</span> Axis in Pediatric Solid Tumors: Novel Targets for Clinical Integration. </a></p><div class="supp"><p class="details"><div><b>The PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the downstream targets of Akt</b>.</div></p></div><div class="aux"><div class="resc">Amos H. P. Loh, et al. Front Oncol. 2013;3:93.</div><p class="links"><a class="dblinks" href="/pubmed/23638435" ref="ordinalpos=932&amp;ncbi_uid=4855826&amp;link_uid=23638435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3636761/" ref="ordinalpos=932&amp;ncbi_uid=4855826&amp;link_uid=PMC3636761">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>933.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3557825/figure/F1/" ref="ordinalpos=933&amp;ncbi_uid=4564630&amp;link_uid=PMC3557825" href="/pmc/articles/PMC3557825/"><img src="/pmc/articles/instance/3557825/bin/nihms-266932-f0001.gif" src-large="/pmc/articles/instance/3557825/bin/nihms-266932-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3557825/" ref="ordinalpos=933&amp;ncbi_uid=4564630&amp;link_uid=PMC3557825" image-link="/pmc/articles/PMC3557825/figure/F1/" class="imagepopup">Figure 1.  From: OXIDATIVE STRESS, INSULIN <span class="highlight" style="background-color:">SIGNALING</span> AND DIABETES. </a></p><div class="supp"><p class="details"><div>Schematic of the effect of chronic oxidative stress on the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Justin L. Rains, et al. Free Radic Biol Med. ;50(5):567-575.</div><p class="links"><a class="dblinks" href="/pubmed/21163346" ref="ordinalpos=933&amp;ncbi_uid=4564630&amp;link_uid=21163346">Citation</a><a class="dblinks" href="/pmc/articles/PMC3557825/" ref="ordinalpos=933&amp;ncbi_uid=4564630&amp;link_uid=PMC3557825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>934.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3183998/figure/F8/" ref="ordinalpos=934&amp;ncbi_uid=3456815&amp;link_uid=PMC3183998" href="/pmc/articles/PMC3183998/"><img src="/pmc/articles/instance/3183998/bin/nihms325221f8.gif" src-large="/pmc/articles/instance/3183998/bin/nihms325221f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3183998/" ref="ordinalpos=934&amp;ncbi_uid=3456815&amp;link_uid=PMC3183998" image-link="/pmc/articles/PMC3183998/figure/F8/" class="imagepopup">Figure 8. Model: The Wnt/β-catenin asymmetry <span class="highlight" style="background-color:">pathway</span> patterns LIN-32 expression during ray development to diversify ray cell fates.  From: The Wnt/?-catenin asymmetry <span class="highlight" style="background-color:">pathway</span> patterns the atonal ortholog lin-32 to diversify cell fate in a C. elegans sensory lineage. </a></p><div class="supp"><p class="details"><div>Shading of nuclei indicates LIN-32 expression; “SYS/POP” refers to the SYS-1:POP-1 ratio that determines the output of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In addition to regulation by <i>lin-17</i> and <i>lin-44</i>, our findings also indicate a possible role for cell-intrinsic polarity-generating mechanisms. See Discussion for details.</div></p></div><div class="aux"><div class="resc">Renee M. Miller, et al. J Neurosci. ;31(37):13281-13291.</div><p class="links"><a class="dblinks" href="/pubmed/21917811" ref="ordinalpos=934&amp;ncbi_uid=3456815&amp;link_uid=21917811">Citation</a><a class="dblinks" href="/pmc/articles/PMC3183998/" ref="ordinalpos=934&amp;ncbi_uid=3456815&amp;link_uid=PMC3183998">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>935.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2958340/figure/fig5/" ref="ordinalpos=935&amp;ncbi_uid=2534175&amp;link_uid=PMC2958340" href="/pmc/articles/PMC2958340/"><img src="/pmc/articles/instance/2958340/bin/JO2011-941876.005.gif" src-large="/pmc/articles/instance/2958340/bin/JO2011-941876.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2958340/" ref="ordinalpos=935&amp;ncbi_uid=2534175&amp;link_uid=PMC2958340" image-link="/pmc/articles/PMC2958340/figure/fig5/" class="imagepopup">Figure 5.  From: Targeting the Mechanisms of Resistance to Chemotherapy and  Radiotherapy with the Cancer Stem Cell Hypothesis. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the Sonic Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a potential modulator of cancer stem cell carcinogenesis with significant therapeutic implications. GDC-0449, XL-139, and IPI-926 are drugs under development targeting this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ryan Morrison, et al. J Oncol. 2011;2011:941876.</div><p class="links"><a class="dblinks" href="/pubmed/20981352" ref="ordinalpos=935&amp;ncbi_uid=2534175&amp;link_uid=20981352">Citation</a><a class="dblinks" href="/pmc/articles/PMC2958340/" ref="ordinalpos=935&amp;ncbi_uid=2534175&amp;link_uid=PMC2958340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>936.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1143566/figure/F5/" ref="ordinalpos=936&amp;ncbi_uid=494855&amp;link_uid=PMC1143566" href="/pmc/articles/PMC1143566/"><img src="/pmc/articles/instance/1143566/bin/bcr1021-5.gif" src-large="/pmc/articles/instance/1143566/bin/bcr1021-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1143566/" ref="ordinalpos=936&amp;ncbi_uid=494855&amp;link_uid=PMC1143566" image-link="/pmc/articles/PMC1143566/figure/F5/" class="imagepopup">Figure 5.  From: Mammary stem cells, self-renewal pathways, and carcinogenesis. </a></p><div class="supp"><p class="details"><div>A hypothetic interacting model in the regulation of stem cell self-renewal by the Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and B lymphoma Mo-MLV insertion region 1 (Bmi-1). Interactions between the Hedgehog, Notch, and Wnt pathways and Bmi-1 are shown by solid arrows; interactions between stem cell self-renewal regulation by the pathways and Bmi-1 are shown by dashed arrows; the question marks represent the postulated interactions.</div></p></div><div class="aux"><div class="resc">Suling Liu, et al. Breast Cancer Res. 2005;7(3):86-95.</div><p class="links"><a class="dblinks" href="/pubmed/15987436" ref="ordinalpos=936&amp;ncbi_uid=494855&amp;link_uid=15987436">Citation</a><a class="dblinks" href="/pmc/articles/PMC1143566/" ref="ordinalpos=936&amp;ncbi_uid=494855&amp;link_uid=PMC1143566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>937.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2746679/figure/F3/" ref="ordinalpos=937&amp;ncbi_uid=2053129&amp;link_uid=PMC2746679" href="/pmc/articles/PMC2746679/"><img src="/pmc/articles/instance/2746679/bin/nihms110736f3.gif" src-large="/pmc/articles/instance/2746679/bin/nihms110736f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2746679/" ref="ordinalpos=937&amp;ncbi_uid=2053129&amp;link_uid=PMC2746679" image-link="/pmc/articles/PMC2746679/figure/F3/" class="imagepopup">Figure 3. Negative- and positive-feedback loops involved in p53 <span class="highlight" style="background-color:">signaling</span>.  From: Guardian of corpulence: a hypothesis on p53 <span class="highlight" style="background-color:">signaling</span> in the fat cell. </a></p><div class="supp"><p class="details"><div>Negative- (closed arrow –•) and positive- (open arrow →) feedback loops involved in p53 <span class="highlight" style="background-color:">signaling</span> are highlighted and the manner in which they intertwine to synchronize regulation of the cell cycle with the regulation of protein synthesis. Indicated are classical components of the p53-mediated cell-cycle arrest <span class="highlight" style="background-color:">pathway</span>: Mdm2, Mdm4 and the p53-effector p21<sup>waf1</sup>; essential components of the metabolic fuel-sensing <span class="highlight" style="background-color:">pathway</span>: AMPK and mTORC; and the recently positioned sestrins, as discussed in the text.<br />mTORC: Mammalian target of rapamycin complex.</div></p></div><div class="aux"><div class="resc">Merlijn Bazuine, et al. Clin Lipidol. ;4(2):231.</div><p class="links"><a class="dblinks" href="/pubmed/20126301" ref="ordinalpos=937&amp;ncbi_uid=2053129&amp;link_uid=20126301">Citation</a><a class="dblinks" href="/pmc/articles/PMC2746679/" ref="ordinalpos=937&amp;ncbi_uid=2053129&amp;link_uid=PMC2746679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>938.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1257398/figure/fig9/" ref="ordinalpos=938&amp;ncbi_uid=678431&amp;link_uid=PMC1257398" href="/pmc/articles/PMC1257398/"><img src="/pmc/articles/instance/1257398/bin/2435fig9.gif" src-large="/pmc/articles/instance/1257398/bin/2435fig9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1257398/" ref="ordinalpos=938&amp;ncbi_uid=678431&amp;link_uid=PMC1257398" image-link="/pmc/articles/PMC1257398/figure/fig9/" class="imagepopup">Figure 9. From: Transcriptional feedback control of insulin receptor by dFOXO/FOXO1. </a></p><div class="supp"><p class="details"><div>Proposed mechanism by which FOXO1 regulates the sensitivity of the InR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through a feedback mechanism that involves up-regulation of InR and IRS-2 upon fasting. When nutrients are high, the InR <span class="highlight" style="background-color:">pathway</span> turns down FOXO1 activity and the levels of InR and IRS-2 drop, dampening the <span class="highlight" style="background-color:">signaling</span> through the InR <span class="highlight" style="background-color:">pathway</span>. When nutrient conditions change, FOXO1 is active and it sensitizes the <span class="highlight" style="background-color:">pathway</span> by activating the transcription of InR and IRS-2.</div></p></div><div class="aux"><div class="resc">Oscar Puig, et al. Genes Dev. 2005 October 15;19(20):2435-2446.</div><p class="links"><a class="dblinks" href="/pubmed/16230533" ref="ordinalpos=938&amp;ncbi_uid=678431&amp;link_uid=16230533">Citation</a><a class="dblinks" href="/pmc/articles/PMC1257398/" ref="ordinalpos=938&amp;ncbi_uid=678431&amp;link_uid=PMC1257398">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>939.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3518872/figure/F6/" ref="ordinalpos=939&amp;ncbi_uid=4810199&amp;link_uid=PMC3518872" href="/pmc/articles/PMC3518872/"><img src="/pmc/articles/instance/3518872/bin/nihms-416085-f0006.gif" src-large="/pmc/articles/instance/3518872/bin/nihms-416085-f0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3518872/" ref="ordinalpos=939&amp;ncbi_uid=4810199&amp;link_uid=PMC3518872" image-link="/pmc/articles/PMC3518872/figure/F6/" class="imagepopup">Figure 6.  From: Inflammatory Mediators Alter the Astrocyte Transcriptome and Calcium <span class="highlight" style="background-color:">Signaling</span> Elicited by Multiple G-Protein-Coupled Receptors. </a></p><div class="supp"><p class="details"><div>GPCR Canonical <span class="highlight" style="background-color:">Signaling</span> Pathways. <i><b>A–C</b></i>, Schematic diagrams of three significantly altered GPCR Canonical <span class="highlight" style="background-color:">Signaling</span> Pathways identified using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software (IPA; Ingenuity) as being affected by TGF-<i>β</i>1+LPS+IFN<i>γ</i>-treatment of astrocytes. As selected from Table 7, these pathways are<i>α</i>-Adrenergic <span class="highlight" style="background-color:">Signaling</span> (<i><b>A</b></i>), Endothelin-1 <span class="highlight" style="background-color:">Signaling</span> (<i><b>B</b></i>), and P2Y <span class="highlight" style="background-color:">Signaling</span> (<i><b>C</b></i>). Molecules are indicated by standard abbreviations. Relative changes in gene expression are depicted by gradated shades of color coding: red, up; green, down; white, no change. Direct and indirect interactions between molecules are depicted by solid and dotted lines, respectively.</div></p></div><div class="aux"><div class="resc">Mary E. Hamby, et al. J Neurosci. ;32(42):14489-14510.</div><p class="links"><a class="dblinks" href="/pubmed/23077035" ref="ordinalpos=939&amp;ncbi_uid=4810199&amp;link_uid=23077035">Citation</a><a class="dblinks" href="/pmc/articles/PMC3518872/" ref="ordinalpos=939&amp;ncbi_uid=4810199&amp;link_uid=PMC3518872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>940.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3932967/figure/f02/" ref="ordinalpos=940&amp;ncbi_uid=5893111&amp;link_uid=PMC3932967" href="/pmc/articles/PMC3932967/"><img src="/pmc/articles/instance/3932967/bin/1414-431X-bjmbr-47-01-00001-gf002.gif" src-large="/pmc/articles/instance/3932967/bin/1414-431X-bjmbr-47-01-00001-gf002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3932967/" ref="ordinalpos=940&amp;ncbi_uid=5893111&amp;link_uid=PMC3932967" image-link="/pmc/articles/PMC3932967/figure/f02/" class="imagepopup">Figure 2. Summary of cardiac developmental aspects of Notch <span class="highlight" style="background-color:">signaling</span>. A, Notch <span class="highlight" style="background-color:">signaling</span> affects AVC development via BMP 2/Tbx 2 <span class="highlight" style="background-color:">pathway</span>. B, Notch <span class="highlight" style="background-color:">signaling</span> initiates EMT via MMP-2, snail/VE-cadherin, VEGFR2/VEGF, and PI3K/Akt/eNOS pathways. EMT is a required step for valve development. C, Notch <span class="highlight" style="background-color:">signaling</span> promotes ventricular trabecular formation dependent on NRG1-ErbB2/4, Ephrin B2/Eph B4, and BMP-10 <span class="highlight" style="background-color:">signaling</span> pathways. D, Notch <span class="highlight" style="background-color:">signaling</span> stimulates smooth muscle differentiation during OFT development. E, Notch <span class="highlight" style="background-color:">signaling</span> modulates coronary vessel morphogenesis in which the embryonic epicardium actively participates. F, Notch <span class="highlight" style="background-color:">signaling</span> regulates cardiac conduction system function via effects on the atrioventricular node. AVC: atrioventricular canal; BMP-2: bone morphogenetic protein-2; Tbx2: T-box 2; EMT: epithelial-to-mesenchymal transition; MMP-2: matrix metalloproteinase-2; VEGFR 2: vascular endothelial growth factor receptor 2; VEGF: vascular endothelial growth factor; BMP-10: bone morphogenetic protein-10; OFT: outflow tract..  From: Role of Notch <span class="highlight" style="background-color:">signaling</span> in the mammalian heart. </a></p><div class="aux"><div class="resc">X.L. Zhou, et al. Braz J Med Biol Res. 2014 January;47(1):1-10.</div><p class="links"><a class="dblinks" href="/pubmed/24345875" ref="ordinalpos=940&amp;ncbi_uid=5893111&amp;link_uid=24345875">Citation</a><a class="dblinks" href="/pmc/articles/PMC3932967/" ref="ordinalpos=940&amp;ncbi_uid=5893111&amp;link_uid=PMC3932967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>941.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2779451/figure/pone-0008001-g003/" ref="ordinalpos=941&amp;ncbi_uid=2138098&amp;link_uid=PMC2779451" href="/pmc/articles/PMC2779451/"><img src="/pmc/articles/instance/2779451/bin/pone.0008001.g003.gif" src-large="/pmc/articles/instance/2779451/bin/pone.0008001.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2779451/" ref="ordinalpos=941&amp;ncbi_uid=2138098&amp;link_uid=PMC2779451" image-link="/pmc/articles/PMC2779451/figure/pone-0008001-g003/" class="imagepopup">Figure 3. Histograms of dynamic robustness in randomized <span class="highlight" style="background-color:">signaling</span> networks retrieved from databases..  From: Ground State Robustness as an Evolutionary Design Principle in <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>For the Adrenergic <span class="highlight" style="background-color:">pathway</span> <img src="/pmc/articles/instance/2779451/bin/pone.0008001.e144.jpg" class="inline-graphic" style="border-style: none;" /> random networks were evaluated, for all other <span class="highlight" style="background-color:">signaling</span> systems <img src="/pmc/articles/instance/2779451/bin/pone.0008001.e145.jpg" class="inline-graphic" style="border-style: none;" />. The distribution of the indices of the randomized networks are depicted in the left column. The right column shows the structural robustness values for the perturbed networks. The asterisks mark the values for the original networks. Perturbation mode a has been used to calculate <img src="/pmc/articles/instance/2779451/bin/pone.0008001.e146.jpg" class="inline-graphic" style="border-style: none;" />.</div></p></div><div class="aux"><div class="resc">Önder Kartal, et al. PLoS One. 2009;4(12):e8001.</div><p class="links"><a class="dblinks" href="/pubmed/19956601" ref="ordinalpos=941&amp;ncbi_uid=2138098&amp;link_uid=19956601">Citation</a><a class="dblinks" href="/pmc/articles/PMC2779451/" ref="ordinalpos=941&amp;ncbi_uid=2138098&amp;link_uid=PMC2779451">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>942.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F5/" ref="ordinalpos=942&amp;ncbi_uid=4064476&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f5.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=942&amp;ncbi_uid=4064476&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F5/" class="imagepopup">Figure 5. Deletion of sema3E increases mEPSC frequency and decreases spine density in direct <span class="highlight" style="background-color:">pathway</span> MSNs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Traces showing mEPSCs recorded from direct <span class="highlight" style="background-color:">pathway</span> MSNs from <i>Drd2</i>-GFP<i>:sema3E</i><sup>+/</sup><i><sup>+</sup></i> (<i>top</i>) and <i>Drd2</i>-GFP<i>:sema3E<sup>−/−</sup></i> (<i>bottom</i>) mice. (<b>b</b>) Summary of mean mEPSC frequency (<i>top</i>) and amplitude (<i>bottom</i>) in direct <span class="highlight" style="background-color:">pathway</span> MSNs. (*<i>P</i>&lt;0.05; Mann-Whitney). (<b>c</b>) Traces showing mEPSCs recorded from indirect <span class="highlight" style="background-color:">pathway</span> MSNs from <i>Drd2</i>-GFP<i>:sema3E</i><sup>+/</sup><i><sup>+</sup></i> (<i>top</i>) and <i>Drd2</i>-GFP<i>:sema3E<sup>−/−</sup></i> (<i>bottom</i>) mice. (<b>d</b>) Summary of mean mEPSC frequency (<i>top</i>) and amplitude (<i>bottom</i>) in direct <span class="highlight" style="background-color:">pathway</span> MSNs. (<b>e</b>) Images of spiny dendrites of direct (<i>top</i>) and indirect (<i>bottom</i>) <span class="highlight" style="background-color:">pathway</span> MSNs in wildtype (<i>left</i>) and <i>sema3E</i><sup>−/−</sup> (<i>right</i>) mice. (<b>f</b>) Summary of spine densities in direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs of the indicated genotypes. (*<i>P</i>&lt;0.05; Mann-Whitney). Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=942&amp;ncbi_uid=4064476&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=942&amp;ncbi_uid=4064476&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>943.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3102722/figure/pone-0020454-g002/" ref="ordinalpos=943&amp;ncbi_uid=3000912&amp;link_uid=PMC3102722" href="/pmc/articles/PMC3102722/"><img src="/pmc/articles/instance/3102722/bin/pone.0020454.g002.gif" src-large="/pmc/articles/instance/3102722/bin/pone.0020454.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3102722/" ref="ordinalpos=943&amp;ncbi_uid=3000912&amp;link_uid=PMC3102722" image-link="/pmc/articles/PMC3102722/figure/pone-0020454-g002/" class="imagepopup">Figure 2. An interaction network generated using IPA analysis shows genes regulated by NKX2-3 in HIMEC..  From: NKX2-3 Transcriptional Regulation of Endothelin-1 and VEGF <span class="highlight" style="background-color:">Signaling</span> in Human Intestinal Microvascular Endothelial Cells. </a></p><div class="supp"><p class="details"><div>(A) The top 25 <span class="highlight" style="background-color:">signaling</span> pathways affected by <i>NKX2-3</i> knockdown. (B) The EDN1 <span class="highlight" style="background-color:">pathway</span> is regulated by <i>NKX2-3</i> in 2 HIMEC. IPA analysis shows genes up-regulated (red) and down-regulated (green) by <i>NKX2-3</i> knockdown. (C) The VEGF-PI3K/AKT-eNOS <span class="highlight" style="background-color:">pathway</span> is regulated by <i>NKX2-3</i> in 2 HIMEC. IPA analysis shows genes up-regulated (red) and down-regulated (green) by <i>NKX2-3</i> knockdown in the VEGF-PI3K/AKT-eNOS <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wei Yu, et al. PLoS One. 2011;6(5):e20454.</div><p class="links"><a class="dblinks" href="/pubmed/21637825" ref="ordinalpos=943&amp;ncbi_uid=3000912&amp;link_uid=21637825">Citation</a><a class="dblinks" href="/pmc/articles/PMC3102722/" ref="ordinalpos=943&amp;ncbi_uid=3000912&amp;link_uid=PMC3102722">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>944.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4313785/figure/F1/" ref="ordinalpos=944&amp;ncbi_uid=7190375&amp;link_uid=PMC4313785" href="/pmc/articles/PMC4313785/"><img src="/pmc/articles/instance/4313785/bin/fendo-06-00012-g001.gif" src-large="/pmc/articles/instance/4313785/bin/fendo-06-00012-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4313785/" ref="ordinalpos=944&amp;ncbi_uid=7190375&amp;link_uid=PMC4313785" image-link="/pmc/articles/PMC4313785/figure/F1/" class="imagepopup">Figure 1.  From: Insulin/Insulin-Like Growth Factors in Cancer: New Roles for the Aryl Hydrocarbon Receptor, Tumor Resistance Mechanisms, and New Blocking Strategies. </a></p><div class="supp"><p class="details"><div><b>Figitumumab (CP) is a biased agonist of the IGF1R that selectively promotes β-arrestin1 (β-arr1) <span class="highlight" style="background-color:">signaling</span> in cancer cells</b>. <b>(A)</b> IGF1 binding to and activation of the IGF1R stimulates the β-arr1 <span class="highlight" style="background-color:">pathway</span> that leads to proteasomal degradation of the IGF1R through a ubiquitin (Ub)-mediated mechanism and ERK activation. IGF1 binding to the IGF1R also induces IGF1R autophosphorylation and this serves to increase the kinase <span class="highlight" style="background-color:">pathway</span> that promotes the phosphorylation and activation of AKT. <b>(B)</b> CP binding to the IGF1R preferentially increases β-arr1 <span class="highlight" style="background-color:">signaling</span>, and inhibits the IGF1R kinase <span class="highlight" style="background-color:">pathway</span>. By this mechanism, CP downregulates IGF1R and AKT <span class="highlight" style="background-color:">signaling</span>, and increases ERK-mediated mitogenic <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Travis B. Salisbury, et al. Front Endocrinol (Lausanne). 2015;6:12.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=944&amp;ncbi_uid=7190375&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4313785/" ref="ordinalpos=944&amp;ncbi_uid=7190375&amp;link_uid=PMC4313785">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>945.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3616369/figure/F1/" ref="ordinalpos=945&amp;ncbi_uid=4779724&amp;link_uid=PMC3616369" href="/pmc/articles/PMC3616369/"><img src="/pmc/articles/instance/3616369/bin/emss-52288-f0001.gif" src-large="/pmc/articles/instance/3616369/bin/emss-52288-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3616369/" ref="ordinalpos=945&amp;ncbi_uid=4779724&amp;link_uid=PMC3616369" image-link="/pmc/articles/PMC3616369/figure/F1/" class="imagepopup">Figure 1.  From: Diabetes Causes Bone Marrow Endothelial Barrier Dysfunction by Activation of the RhoA-Rho-Associated Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis of transcription-associated biofunctions and <span class="highlight" style="background-color:">signaling</span> pathways. <b>A</b>, Bar graph showing −log probability values of canonic biological functions associated with expressional changes induced by diabetes mellitus in bone marrow endothelial cells (BMECs). <b>B</b>, Bar graphs showing −log probability values (<b>i</b>), and enrichment of genes (<b>ii</b>) in <span class="highlight" style="background-color:">signaling</span> pathways modulated by diabetes mellitus in BMECs.</div></p></div><div class="aux"><div class="resc">Giuseppe Mangialardi, et al. Arterioscler Thromb Vasc Biol. 2013 March;33(3):555-564.</div><p class="links"><a class="dblinks" href="/pubmed/23307872" ref="ordinalpos=945&amp;ncbi_uid=4779724&amp;link_uid=23307872">Citation</a><a class="dblinks" href="/pmc/articles/PMC3616369/" ref="ordinalpos=945&amp;ncbi_uid=4779724&amp;link_uid=PMC3616369">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>946.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3800035/figure/pone-0076895-g007/" ref="ordinalpos=946&amp;ncbi_uid=5466459&amp;link_uid=PMC3800035" href="/pmc/articles/PMC3800035/"><img src="/pmc/articles/instance/3800035/bin/pone.0076895.g007.gif" src-large="/pmc/articles/instance/3800035/bin/pone.0076895.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3800035/" ref="ordinalpos=946&amp;ncbi_uid=5466459&amp;link_uid=PMC3800035" image-link="/pmc/articles/PMC3800035/figure/pone-0076895-g007/" class="imagepopup">Figure 7. Schematic representation of differential utilization of co-receptors and G-proteins for β-catenin-dependent and β-catenin-independent <span class="highlight" style="background-color:">signaling</span> pathways..  From: Heterotrimeric G-Protein, Gα16, Is a Critical Downstream Effector of Non-Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> and a Potent Inhibitor of Transformed Cell Growth in Non Small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div>Based on our current understanding of Wnt-sensitive pathways, we propose that Wnt binding to Fzd and co-receptor LRP5/6, and mediated by G<sub>αo</sub>, leads to activation of β-catenin-dependent gene transcription. While, Wnt binding to Fzd and co-receptor ROR1/2, and mediated by G<sub>α16</sub>, leads to the activation of β-catenin-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, mainly via activation of ERK5 and PPARγ.</div></p></div><div class="aux"><div class="resc">Sreedevi Avasarala, et al. PLoS One. 2013;8(10):e76895.</div><p class="links"><a class="dblinks" href="/pubmed/24204697" ref="ordinalpos=946&amp;ncbi_uid=5466459&amp;link_uid=24204697">Citation</a><a class="dblinks" href="/pmc/articles/PMC3800035/" ref="ordinalpos=946&amp;ncbi_uid=5466459&amp;link_uid=PMC3800035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>947.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679427/figure/fig3/" ref="ordinalpos=947&amp;ncbi_uid=1841946&amp;link_uid=PMC2679427" href="/pmc/articles/PMC2679427/"><img src="/pmc/articles/instance/2679427/bin/zbc0200971600003.gif" src-large="/pmc/articles/instance/2679427/bin/zbc0200971600003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679427/" ref="ordinalpos=947&amp;ncbi_uid=1841946&amp;link_uid=PMC2679427" image-link="/pmc/articles/PMC2679427/figure/fig3/" class="imagepopup">FIGURE 3. From: The Nrf2-Antioxidant Response Element <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Its  Activation by Oxidative  Stress. </a></p><div class="supp"><p class="details"><div><b>Proposed Nrf2-ARE <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Nrf2 is expressed  constitutively in the cell and translocates directly to the nucleus following  its synthesis. Following transactivation of its genes, Nrf2 is targeted for  degradation by Keap1 in the nucleus, a process that requires the transient  shuttling of Keap1 into this compartment. In cells under stress, stabilization  of Nrf2 is thought to be dependent on mechanisms that either prevent or reduce  access of Keap1 to Nrf2. This figure was reproduced from Ref.  12.</div></p></div><div class="aux"><div class="resc">Truyen Nguyen, et al. J Biol Chem. 2009 May 15;284(20):13291-13295.</div><p class="links"><a class="dblinks" href="/pubmed/19182219" ref="ordinalpos=947&amp;ncbi_uid=1841946&amp;link_uid=19182219">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679427/" ref="ordinalpos=947&amp;ncbi_uid=1841946&amp;link_uid=PMC2679427">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>948.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3170992/figure/F1/" ref="ordinalpos=948&amp;ncbi_uid=3219221&amp;link_uid=PMC3170992" href="/pmc/articles/PMC3170992/"><img src="/pmc/articles/instance/3170992/bin/nihms315651f1.gif" src-large="/pmc/articles/instance/3170992/bin/nihms315651f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3170992/" ref="ordinalpos=948&amp;ncbi_uid=3219221&amp;link_uid=PMC3170992" image-link="/pmc/articles/PMC3170992/figure/F1/" class="imagepopup">Figure 1.  From: Association Study of Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes in Bipolar Disorder. </a></p><div class="supp"><p class="details"><div>Association of 227 SNPs in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes with bipolar disorder. P-values for allelic single SNP analyses (black diamonds) and haplotype analyses (blue, green and red lines) on the log scale. Results with nominal p&lt;0.01 are labeled with the gene name.</div></p></div><div class="aux"><div class="resc">Peter P. Zandi, et al. Arch Gen Psychiatry. ;65(7):785-793.</div><p class="links"><a class="dblinks" href="/pubmed/18606951" ref="ordinalpos=948&amp;ncbi_uid=3219221&amp;link_uid=18606951">Citation</a><a class="dblinks" href="/pmc/articles/PMC3170992/" ref="ordinalpos=948&amp;ncbi_uid=3219221&amp;link_uid=PMC3170992">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>949.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3270444/figure/fig5/" ref="ordinalpos=949&amp;ncbi_uid=3792105&amp;link_uid=PMC3270444" href="/pmc/articles/PMC3270444/"><img src="/pmc/articles/instance/3270444/bin/MI2012-489810.005.gif" src-large="/pmc/articles/instance/3270444/bin/MI2012-489810.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3270444/" ref="ordinalpos=949&amp;ncbi_uid=3792105&amp;link_uid=PMC3270444" image-link="/pmc/articles/PMC3270444/figure/fig5/" class="imagepopup">Figure 5.  From: Nanostructured, Self-Assembling Peptide K5 Blocks TNF-α and PGE2 Production by Suppression of the AP-1/p38 <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Putative inhibitory <span class="highlight" style="background-color:">pathway</span> of LPS-activated inflammatory <span class="highlight" style="background-color:">signaling</span> responses by peptide K5.</div></p></div><div class="aux"><div class="resc">Woo Seok Yang, et al. Mediators Inflamm. 2012;2012:489810.</div><p class="links"><a class="dblinks" href="/pubmed/22315508" ref="ordinalpos=949&amp;ncbi_uid=3792105&amp;link_uid=22315508">Citation</a><a class="dblinks" href="/pmc/articles/PMC3270444/" ref="ordinalpos=949&amp;ncbi_uid=3792105&amp;link_uid=PMC3270444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>950.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3446243/figure/F1/" ref="ordinalpos=950&amp;ncbi_uid=4242479&amp;link_uid=PMC3446243" href="/pmc/articles/PMC3446243/"><img src="/pmc/articles/instance/3446243/bin/nihms388815f1.gif" src-large="/pmc/articles/instance/3446243/bin/nihms388815f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3446243/" ref="ordinalpos=950&amp;ncbi_uid=4242479&amp;link_uid=PMC3446243" image-link="/pmc/articles/PMC3446243/figure/F1/" class="imagepopup">Fig. 1.  From: Nuclear Factor ?B <span class="highlight" style="background-color:">Signaling</span> in Opioid Functions and Receptor Gene Expression. </a></p><div class="supp"><p class="details"><div>Possible cellular responses and transcriptional regulation initiated by activation of opioid receptor <span class="highlight" style="background-color:">signaling</span>. Arrows indicate activation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; the “⊣” signs indicate inhibition of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. AC, adenylyl cyclase; Akt, also called protein kinase B (PKB); cAMP, cyclic AMP; PI3K, phosphatidylinositol 3-kinase; PKC, protein kinase C; PLC, phospholipase C.</div></p></div><div class="aux"><div class="resc">Yulong L. Chen, et al. J Neuroimmune Pharmacol. 2006 September;1(3):270-279.</div><p class="links"><a class="dblinks" href="/pubmed/18040804" ref="ordinalpos=950&amp;ncbi_uid=4242479&amp;link_uid=18040804">Citation</a><a class="dblinks" href="/pmc/articles/PMC3446243/" ref="ordinalpos=950&amp;ncbi_uid=4242479&amp;link_uid=PMC3446243">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>951.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2743116/figure/F1/" ref="ordinalpos=951&amp;ncbi_uid=2007516&amp;link_uid=PMC2743116" href="/pmc/articles/PMC2743116/"><img src="/pmc/articles/instance/2743116/bin/nihms112312f1.gif" src-large="/pmc/articles/instance/2743116/bin/nihms112312f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2743116/" ref="ordinalpos=951&amp;ncbi_uid=2007516&amp;link_uid=PMC2743116" image-link="/pmc/articles/PMC2743116/figure/F1/" class="imagepopup">Figure 1.  From: The RASopathies: Developmental syndromes of Ras/MAPK <span class="highlight" style="background-color:">pathway</span> dysregulation. </a></p><div class="supp"><p class="details"><div>The Ras/mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and associated developmental syndromes (indicated by dashed lines). The MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of protein kinases is critically involved in cell proliferation, differentiation, motility, apoptosis and senescence. The Ras/MAPK <span class="highlight" style="background-color:">pathway</span> proteins with germline mutations in their respective genes are associated with Noonan, LEOPARD, gingival fibromatosis 1, neurofibromatosis 1, capillary malformation-arteriovenous malformation, Costello, autoimmune lymphoproliferative (ALPS), cardio-facio-cutaneous and Legius syndromes.</div></p></div><div class="aux"><div class="resc">William E. Tidyman, et al. Curr Opin Genet Dev. ;19(3):230-236.</div><p class="links"><a class="dblinks" href="/pubmed/19467855" ref="ordinalpos=951&amp;ncbi_uid=2007516&amp;link_uid=19467855">Citation</a><a class="dblinks" href="/pmc/articles/PMC2743116/" ref="ordinalpos=951&amp;ncbi_uid=2007516&amp;link_uid=PMC2743116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>952.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3126131/figure/F10/" ref="ordinalpos=952&amp;ncbi_uid=3247946&amp;link_uid=PMC3126131" href="/pmc/articles/PMC3126131/"><img src="/pmc/articles/instance/3126131/bin/zjv9990944190010.gif" src-large="/pmc/articles/instance/3126131/bin/zjv9990944190010.jpg" alt="Fig. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3126131/" ref="ordinalpos=952&amp;ncbi_uid=3247946&amp;link_uid=PMC3126131" image-link="/pmc/articles/PMC3126131/figure/F10/" class="imagepopup">Fig. 10. From: The IRF-3/Bax-Mediated Apoptotic <span class="highlight" style="background-color:">Pathway</span>, Activated by Viral Cytoplasmic RNA and DNA, Inhibits Virus Replication   . </a></p><div class="supp"><p class="details"><div>IRF-3- and Bax-mediated apoptosis acts as a host antiviral response. Infection of cells by DNA or RNA viruses triggers the cellular apoptotic <span class="highlight" style="background-color:">pathway</span> by RIG-I <span class="highlight" style="background-color:">signaling</span>-mediated activation of IRF-3 and Bax to cause their translocation to mitochondria, which in turn activates the apoptotic <span class="highlight" style="background-color:">pathway</span>. Cytosolic DNA, generated upon infection by DNA viruses or other microbes, activates RIG-I <span class="highlight" style="background-color:">signaling</span> via an intermediate dsRNA generated by RNA polymerase III. The IRF-3-mediated apoptotic <span class="highlight" style="background-color:">pathway</span>, in addition to the gene induction <span class="highlight" style="background-color:">pathway</span>, significantly contributes to the host antiviral response.</div></p></div><div class="aux"><div class="resc">Saurabh Chattopadhyay, et al. J Virol. 2011 April;85(8):3708-3716.</div><p class="links"><a class="dblinks" href="/pubmed/21307205" ref="ordinalpos=952&amp;ncbi_uid=3247946&amp;link_uid=21307205">Citation</a><a class="dblinks" href="/pmc/articles/PMC3126131/" ref="ordinalpos=952&amp;ncbi_uid=3247946&amp;link_uid=PMC3126131">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>953.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3445042/figure/F3/" ref="ordinalpos=953&amp;ncbi_uid=5280737&amp;link_uid=PMC3445042" href="/pmc/articles/PMC3445042/"><img src="/pmc/articles/instance/3445042/bin/nihms402300f3.gif" src-large="/pmc/articles/instance/3445042/bin/nihms402300f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3445042/" ref="ordinalpos=953&amp;ncbi_uid=5280737&amp;link_uid=PMC3445042" image-link="/pmc/articles/PMC3445042/figure/F3/" class="imagepopup">Figure 3.  From: Wnt <span class="highlight" style="background-color:">signaling</span> regulates post-embryonic hypothalamic progenitor differentiation. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> regulates hypothalamic progenitor proliferation and differentiation. (A) pH3<sup>+</sup> and BrdU<sup>+</sup>/Sox3<sup>+</sup> cell numbers in the 32hpf hypothalamus following 2 hour labeling and Wnt <span class="highlight" style="background-color:">pathway</span> inhibition or activation at 24hpf. (B) BrdU<sup>+</sup> cell numbers in the 4dpf posterior recess following 2 hour labeling and Wnt <span class="highlight" style="background-color:">pathway</span> inhibition or activation at 3dpf. (C) BrdU<sup>+</sup> cell fates in the 4dpf hypothalamus following labeling and Wnt <span class="highlight" style="background-color:">pathway</span> inhibition or activation at 3dpf. (D) Percentage of BrdU<sup>+</sup> cells expressing <i>dlx5/6:gfp</i> following Wnt <span class="highlight" style="background-color:">pathway</span> activation at 3dpf. (E) BrdU<sup>+</sup> cell fates in the adult hypothalamus following 2 day labeling and Wnt <span class="highlight" style="background-color:">pathway</span> inhibition or activation for 15 days. (F) Number of Gal4<sup>+</sup> and BLBP+ cells in the 4dpf posterior recess following Wnt <span class="highlight" style="background-color:">pathway</span> inhibition or activation at 3dpf. All cell counts were collected from ventral maximum intensity confocal Z-projections through 5 individual brains. The entire hypothalamus was counted at 32hpf, the entire posterior recess was counted at 4dpf, and a hemisphere of the posterior recess was counted in adults. *: p&lt;0.05, **: p&lt;0.005. Error=±SD. See also Figure S4.</div></p></div><div class="aux"><div class="resc">Xu Wang, et al. Dev Cell. 2012 September 11;23(3):624-636.</div><p class="links"><a class="dblinks" href="/pubmed/22975330" ref="ordinalpos=953&amp;ncbi_uid=5280737&amp;link_uid=22975330">Citation</a><a class="dblinks" href="/pmc/articles/PMC3445042/" ref="ordinalpos=953&amp;ncbi_uid=5280737&amp;link_uid=PMC3445042">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>954.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3376380/figure/F1/" ref="ordinalpos=954&amp;ncbi_uid=4950059&amp;link_uid=PMC3376380" href="/pmc/articles/PMC3376380/"><img src="/pmc/articles/instance/3376380/bin/nihms373397f1.gif" src-large="/pmc/articles/instance/3376380/bin/nihms373397f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3376380/" ref="ordinalpos=954&amp;ncbi_uid=4950059&amp;link_uid=PMC3376380" image-link="/pmc/articles/PMC3376380/figure/F1/" class="imagepopup">Fig. 1.  From: The Role and Regulation of mTOR in T Lymphocyte Function. </a></p><div class="supp"><p class="details"><div>mTOR <span class="highlight" style="background-color:">signaling</span> and its regulation by DGKs. mTOR <span class="highlight" style="background-color:">signaling</span> is mediated by two <span class="highlight" style="background-color:">signaling</span> complexes, raptor-containing mTORC1 and rictor containing mTORC2. DAG, generated by PLCγ1 following TCR engagement, activates the RasGRP1-Ras-Erk1/2 <span class="highlight" style="background-color:">pathway</span>. The <span class="highlight" style="background-color:">pathway</span> plays important roles for mTORC1 and mTORC2 activation in T cells. DGKs negatively control both mTORC1 and mTORC2 <span class="highlight" style="background-color:">signaling</span> by terminating DAG in T cells.</div></p></div><div class="aux"><div class="resc">Thomas F. O’Brien, et al. Arch Immunol Ther Exp (Warsz). ;60(3):173-181.</div><p class="links"><a class="dblinks" href="/pubmed/22484804" ref="ordinalpos=954&amp;ncbi_uid=4950059&amp;link_uid=22484804">Citation</a><a class="dblinks" href="/pmc/articles/PMC3376380/" ref="ordinalpos=954&amp;ncbi_uid=4950059&amp;link_uid=PMC3376380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>955.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3597535/figure/pcbi-1002967-g008/" ref="ordinalpos=955&amp;ncbi_uid=4724033&amp;link_uid=PMC3597535" href="/pmc/articles/PMC3597535/"><img src="/pmc/articles/instance/3597535/bin/pcbi.1002967.g008.gif" src-large="/pmc/articles/instance/3597535/bin/pcbi.1002967.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3597535/" ref="ordinalpos=955&amp;ncbi_uid=4724033&amp;link_uid=PMC3597535" image-link="/pmc/articles/PMC3597535/figure/pcbi-1002967-g008/" class="imagepopup">Figure 8. Simulated data on the TGFβ signalling <span class="highlight" style="background-color:">pathway</span>, power vs. <span class="highlight" style="background-color:">pathway</span> effect, variance, and sample size..  From: Differential Expression Analysis for Pathways. </a></p><div class="supp"><p class="details"><div>At the top, the KEGG TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is illustrated, with green, red, and grey nodes representing nodes whose simulated values were +μ, −μ, and 0, respectively [13]. The nodes are colored to indicate activity leading to G1 arrest in the cell cycle. At the bottom, power for detecting significant differential expression in this <span class="highlight" style="background-color:">pathway</span> is illustrated with respect to <span class="highlight" style="background-color:">pathway</span> effect, variance, and sample size. Figure adapted from http://www.genome.jp/kegg-bin/show_<span class="highlight" style="background-color:">pathway</span>?map04350 with permission from KEGG.</div></p></div><div class="aux"><div class="resc">Winston A. Haynes, et al. PLoS Comput Biol. 2013 March; 2013 April;9(3):e1002967.</div><p class="links"><a class="dblinks" href="/pubmed/23516350" ref="ordinalpos=955&amp;ncbi_uid=4724033&amp;link_uid=23516350">Citation</a><a class="dblinks" href="/pmc/articles/PMC3597535/" ref="ordinalpos=955&amp;ncbi_uid=4724033&amp;link_uid=PMC3597535">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>956.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3226194/figure/F7/" ref="ordinalpos=956&amp;ncbi_uid=3684536&amp;link_uid=PMC3226194" href="/pmc/articles/PMC3226194/"><img src="/pmc/articles/instance/3226194/bin/1471-2105-10-S11-S5-7.gif" src-large="/pmc/articles/instance/3226194/bin/1471-2105-10-S11-S5-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3226194/" ref="ordinalpos=956&amp;ncbi_uid=3684536&amp;link_uid=PMC3226194" image-link="/pmc/articles/PMC3226194/figure/F7/" class="imagepopup">Figure 7.  From: HPD: an online integrated human <span class="highlight" style="background-color:">pathway</span> database enabling systems biology studies. </a></p><div class="supp"><p class="details"><div><b>An integrated <span class="highlight" style="background-color:">pathway</span> model involving FOXA1 in breast cancer</b>. The figure shows how information from different <span class="highlight" style="background-color:">pathway</span> database sources are readily integrated, queried, and analyzed together in HPD for FOXA1-related breast cancer <span class="highlight" style="background-color:">signaling</span> studies.</div></p></div><div class="aux"><div class="resc">Sudhir R Chowbina, et al. BMC Bioinformatics. 2009;10(Suppl 11):S5-S5.</div><p class="links"><a class="dblinks" href="/pubmed/19811689" ref="ordinalpos=956&amp;ncbi_uid=3684536&amp;link_uid=19811689">Citation</a><a class="dblinks" href="/pmc/articles/PMC3226194/" ref="ordinalpos=956&amp;ncbi_uid=3684536&amp;link_uid=PMC3226194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>957.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566147/figure/pone-0052088-g013/" ref="ordinalpos=957&amp;ncbi_uid=4628969&amp;link_uid=PMC3566147" href="/pmc/articles/PMC3566147/"><img src="/pmc/articles/instance/3566147/bin/pone.0052088.g013.gif" src-large="/pmc/articles/instance/3566147/bin/pone.0052088.g013.jpg" alt="Figure 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566147/" ref="ordinalpos=957&amp;ncbi_uid=4628969&amp;link_uid=PMC3566147" image-link="/pmc/articles/PMC3566147/figure/pone-0052088-g013/" class="imagepopup">Figure 13. A proposed model for SARMs in the innate immune <span class="highlight" style="background-color:">signaling</span> pathways..  From: Litopenaeus vannamei Sterile-Alpha and Armadillo Motif Containing Protein (LvSARM) Is Involved in Regulation of Penaeidins and antilipopolysaccharide factors. </a></p><div class="supp"><p class="details"><div>SARM is a positive regulator of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i> for AMP regulation but functions as a negative regulator in the Toll <span class="highlight" style="background-color:">pathway</span> of <i>L. vannamei</i> and the TLR <span class="highlight" style="background-color:">pathway</span> of mammals.</div></p></div><div class="aux"><div class="resc">Pei-Hui Wang, et al. PLoS One. 2013;8(2):e52088.</div><p class="links"><a class="dblinks" href="/pubmed/23405063" ref="ordinalpos=957&amp;ncbi_uid=4628969&amp;link_uid=23405063">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566147/" ref="ordinalpos=957&amp;ncbi_uid=4628969&amp;link_uid=PMC3566147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>958.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4121214/figure/fig4/" ref="ordinalpos=958&amp;ncbi_uid=6482141&amp;link_uid=PMC4121214" href="/pmc/articles/PMC4121214/"><img src="/pmc/articles/instance/4121214/bin/BMRI2014-931825.004.gif" src-large="/pmc/articles/instance/4121214/bin/BMRI2014-931825.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4121214/" ref="ordinalpos=958&amp;ncbi_uid=6482141&amp;link_uid=PMC4121214" image-link="/pmc/articles/PMC4121214/figure/fig4/" class="imagepopup">Figure 4.  From: Integrated Analysis Identifies Interaction Patterns between Small Molecules and Pathways. </a></p><div class="supp"><p class="details"><div>Hierarchical clustering in the M-P link map for breast cancer from GSE5364. (a) Hierarchical clustering between 571 small molecules and 47 metabolic pathways. The corresponding cells are coloured red, where small molecules link to the pathways in the M-P link map. The labels for the corresponding pathways are shown on the right of the figure. (b) Zoomed-in plot of an sM-sP link between LY-294002 and the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The gene indicated by the arrow is the drug target of LY-294002. Differentially expressed genes in this <span class="highlight" style="background-color:">pathway</span> are coloured pink, while other genes in green are human disease-related genes. (c) Zoomed-in plot of mM-sP links between eight small molecules and the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. These eight small molecules target only the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The differentially expressed genes in this <span class="highlight" style="background-color:">pathway</span> are coloured pink, while other genes in green are human disease-related genes.</div></p></div><div class="aux"><div class="resc">Yan Li, et al. Biomed Res Int. 2014;2014:931825.</div><p class="links"><a class="dblinks" href="/pubmed/25114931" ref="ordinalpos=958&amp;ncbi_uid=6482141&amp;link_uid=25114931">Citation</a><a class="dblinks" href="/pmc/articles/PMC4121214/" ref="ordinalpos=958&amp;ncbi_uid=6482141&amp;link_uid=PMC4121214">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>959.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3623872/figure/pone-0060881-g005/" ref="ordinalpos=959&amp;ncbi_uid=4808859&amp;link_uid=PMC3623872" href="/pmc/articles/PMC3623872/"><img src="/pmc/articles/instance/3623872/bin/pone.0060881.g005.gif" src-large="/pmc/articles/instance/3623872/bin/pone.0060881.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3623872/" ref="ordinalpos=959&amp;ncbi_uid=4808859&amp;link_uid=PMC3623872" image-link="/pmc/articles/PMC3623872/figure/pone-0060881-g005/" class="imagepopup">Figure 5. Insulin and mTOR <span class="highlight" style="background-color:">signaling</span> pathways affected by artificial diet feeding..  From: Nutrigenomics in Arma chinensis: Transcriptome Analysis of Arma chinensis Fed on Artificial Diet and Chinese Oak Silk Moth Antheraea pernyi Pupae. </a></p><div class="supp"><p class="details"><div>(A) Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Red and green frames indicate genes and enriched functions that were up- and down-regulated in artificial diet-fed (AD_1) and prey-fed (CY_1) treatments.</div></p></div><div class="aux"><div class="resc">Deyu Zou, et al. PLoS One. 2013;8(4):e60881.</div><p class="links"><a class="dblinks" href="/pubmed/23593338" ref="ordinalpos=959&amp;ncbi_uid=4808859&amp;link_uid=23593338">Citation</a><a class="dblinks" href="/pmc/articles/PMC3623872/" ref="ordinalpos=959&amp;ncbi_uid=4808859&amp;link_uid=PMC3623872">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>960.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807212/figure/fig4/" ref="ordinalpos=960&amp;ncbi_uid=2213187&amp;link_uid=PMC2807212" href="/pmc/articles/PMC2807212/"><img src="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0002.gif" src-large="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0002.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807212/" ref="ordinalpos=960&amp;ncbi_uid=2213187&amp;link_uid=PMC2807212" image-link="/pmc/articles/PMC2807212/figure/fig4/" class="imagepopup">Figure 4.  From: Controlling Destiny through Chemistry: Small-Molecule Regulators of Cell Fate. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> proteins associated with Notch <span class="highlight" style="background-color:">signaling</span> are shown. Components that currently can be inhibited by synthetic molecules to achieve selective <span class="highlight" style="background-color:">pathway</span> control are labeled with red hexagons.</div></p></div><div class="aux"><div class="resc">Ari J. Firestone, et al. ACS Chem Biol. 2010 January 15;5(1):15-34.</div><p class="links"><a class="dblinks" href="/pubmed/20000447" ref="ordinalpos=960&amp;ncbi_uid=2213187&amp;link_uid=20000447">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807212/" ref="ordinalpos=960&amp;ncbi_uid=2213187&amp;link_uid=PMC2807212">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>961.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3465289/figure/pone-0045049-g002/" ref="ordinalpos=961&amp;ncbi_uid=4298292&amp;link_uid=PMC3465289" href="/pmc/articles/PMC3465289/"><img src="/pmc/articles/instance/3465289/bin/pone.0045049.g002.gif" src-large="/pmc/articles/instance/3465289/bin/pone.0045049.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3465289/" ref="ordinalpos=961&amp;ncbi_uid=4298292&amp;link_uid=PMC3465289" image-link="/pmc/articles/PMC3465289/figure/pone-0045049-g002/" class="imagepopup">Figure 2. Rho1 <span class="highlight" style="background-color:">signaling</span> in the context of the CWI <span class="highlight" style="background-color:">pathway</span>..  From: Rho <span class="highlight" style="background-color:">Signaling</span> Participates in Membrane Fluidity Homeostasis. </a></p><div class="supp"><p class="details"><div>All proteins shown have been implicated in the CWI <span class="highlight" style="background-color:">pathway</span> or in control of Rho1. All three known Rho1 GEFs (Rom1, Rom2, and Tus1) and all four known Rho1 GAPs (Lrg1, Bem2, Bag7, and Sac7) are shown. <span style="text-decoration: underline" class="underline">Underlining</span> indicates proteins found in the screen. Proteins drawn in red are those which when mutated cause sensitivity to C16:1 but not C18:1. Sac7 is drawn in blue because <i>sac7Δ</i> instead causes C16:1<sup>R</sup> C18:1<sup>S</sup>.</div></p></div><div class="aux"><div class="resc">Daniel Lockshon, et al. PLoS One. 2012;7(10):e45049.</div><p class="links"><a class="dblinks" href="/pubmed/23071506" ref="ordinalpos=961&amp;ncbi_uid=4298292&amp;link_uid=23071506">Citation</a><a class="dblinks" href="/pmc/articles/PMC3465289/" ref="ordinalpos=961&amp;ncbi_uid=4298292&amp;link_uid=PMC3465289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>962.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4172779/figure/Fig3/" ref="ordinalpos=962&amp;ncbi_uid=6648491&amp;link_uid=PMC4172779" href="/pmc/articles/PMC4172779/"><img src="/pmc/articles/instance/4172779/bin/13048_2014_87_Fig3_HTML.gif" src-large="/pmc/articles/instance/4172779/bin/13048_2014_87_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4172779/" ref="ordinalpos=962&amp;ncbi_uid=6648491&amp;link_uid=PMC4172779" image-link="/pmc/articles/PMC4172779/figure/Fig3/" class="imagepopup">Figure 3.  From: Reduced expression of SOX7 in ovarian cancer: a novel tumor suppressor through the Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Downstream genes chosen from KEGG annotated in Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span>.</b> Downstream genes in the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> are denoted in red. The figure is created based on KEGG <span class="highlight" style="background-color:">pathway</span>. Only a part of the <span class="highlight" style="background-color:">pathway</span> is shown for clarity.</div></p></div><div class="aux"><div class="resc">Huidi Liu, et al. J Ovarian Res. 2014;7(1):87.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=962&amp;ncbi_uid=6648491&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4172779/" ref="ordinalpos=962&amp;ncbi_uid=6648491&amp;link_uid=PMC4172779">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>963.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2077336/figure/F3/" ref="ordinalpos=963&amp;ncbi_uid=1069692&amp;link_uid=PMC2077336" href="/pmc/articles/PMC2077336/"><img src="/pmc/articles/instance/2077336/bin/1471-2407-7-182-3.gif" src-large="/pmc/articles/instance/2077336/bin/1471-2407-7-182-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2077336/" ref="ordinalpos=963&amp;ncbi_uid=1069692&amp;link_uid=PMC2077336" image-link="/pmc/articles/PMC2077336/figure/F3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Pathway</span> analysis of gene signatures predicting metastasis of node-negative primary breast cancer. </a></p><div class="supp"><p class="details"><div><b>Association of the expression of individual genes with DMFS time for selected over-represented pathways</b>. The Geneplot function in the Global Test program [12, 14] was applied and the contribution of the individual genes in each selected <span class="highlight" style="background-color:">pathway</span> is plotted. The numbers at the X-axis represent the number of genes in the respective <span class="highlight" style="background-color:">pathway</span> in ER-positive (Left) or ER-negative tumors (Right). The values at the Y-axis, represent the contribution (influence) of each individual gene in the selected <span class="highlight" style="background-color:">pathway</span> with DMFS. Negative values indicate there is no association between the gene expression and DMFS. Horizontal markers in a bar indicates one standard deviation away from the reference point, two or more horizontal markers in a bar indicates that the association of the corresponding gene with DMFS is statistically significant. The green bars reflect genes that are positively associated with DMFS, indicating a higher expression in tumors without metastatic capability. The red bars reflect genes that are negatively associated with DMFS, indicative of higher expression in tumors with metastatic capability. (<b>A</b>) ER-positive tumors: from top to bottom: "apoptosis" <span class="highlight" style="background-color:">pathway</span> consisting of 282 genes, "regulation of cell cycle" <span class="highlight" style="background-color:">pathway</span> consisting of 228 genes, "immune response" <span class="highlight" style="background-color:">pathway</span> consisting of 379 genes, and "mitosis"? <span class="highlight" style="background-color:">pathway</span> consisting of 100 genes. (<b>B</b>) ER-negative tumors: from top to bottom: "regulation of cell growth" <span class="highlight" style="background-color:">pathway</span> consisting of 58 genes, "cell adhesion" <span class="highlight" style="background-color:">pathway</span> consisting of 327 genes, "regulation of G-coupled receptor <span class="highlight" style="background-color:">signaling</span>" <span class="highlight" style="background-color:">pathway</span> consisting of 20 genes, and "skeletal development" <span class="highlight" style="background-color:">pathway</span> consisting of 105 genes.</div></p></div><div class="aux"><div class="resc">Jack X Yu, et al. BMC Cancer. 2007;7:182-182.</div><p class="links"><a class="dblinks" href="/pubmed/17894856" ref="ordinalpos=963&amp;ncbi_uid=1069692&amp;link_uid=17894856">Citation</a><a class="dblinks" href="/pmc/articles/PMC2077336/" ref="ordinalpos=963&amp;ncbi_uid=1069692&amp;link_uid=PMC2077336">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>964.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4133035/figure/f2-cgm-7-2014-033/" ref="ordinalpos=964&amp;ncbi_uid=6556608&amp;link_uid=PMC4133035" href="/pmc/articles/PMC4133035/"><img src="/pmc/articles/instance/4133035/bin/cgm-7-2014-033f2.gif" src-large="/pmc/articles/instance/4133035/bin/cgm-7-2014-033f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4133035/" ref="ordinalpos=964&amp;ncbi_uid=6556608&amp;link_uid=PMC4133035" image-link="/pmc/articles/PMC4133035/figure/f2-cgm-7-2014-033/" class="imagepopup">Figure 2.  From: Bone Disease in Multiple Myeloma: Pathophysiology and Management. </a></p><div class="supp"><p class="details"><div><b>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Wnt <span class="highlight" style="background-color:">signaling</span> stimulates OB differentiation from MSCs. Blockade of this <span class="highlight" style="background-color:">pathway</span> through DKK1 inhibits OB formation. Wnt <span class="highlight" style="background-color:">signaling</span> upregulates RANKL expression from OB precursors resulting in increased osteoclastic activity and bone resorption. Activation of Wnt <span class="highlight" style="background-color:">pathway</span> increases OPG production from OB which in turn downregulates RANKL-driven osteoclastogenesis.67</div></p></div><div class="aux"><div class="resc">Abdul Hameed, et al. Cancer Growth Metastasis. 2014;7:33-42.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=964&amp;ncbi_uid=6556608&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4133035/" ref="ordinalpos=964&amp;ncbi_uid=6556608&amp;link_uid=PMC4133035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>965.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2686490/figure/F2/" ref="ordinalpos=965&amp;ncbi_uid=1836163&amp;link_uid=PMC2686490" href="/pmc/articles/PMC2686490/"><img src="/pmc/articles/instance/2686490/bin/gkn892f2.gif" src-large="/pmc/articles/instance/2686490/bin/gkn892f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2686490/" ref="ordinalpos=965&amp;ncbi_uid=1836163&amp;link_uid=PMC2686490" image-link="/pmc/articles/PMC2686490/figure/F2/" class="imagepopup">Figure 2. From: Human Protein Reference Database--2009 update. </a></p><div class="supp"><p class="details"><div>Linking to human <span class="highlight" style="background-color:">signaling</span> pathways from HPRD. ‘Pathways’ button in the HPRD page of JAK2 is hyperlinked to its NetPath page. It shows the list of <span class="highlight" style="background-color:">signaling</span> pathways in which the protein is involved along with the description of its interactors in each <span class="highlight" style="background-color:">pathway</span>. Each interaction or catalysis event is linked to the PubMed abstract of the original article. The <span class="highlight" style="background-color:">pathway</span> name is linked to the specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> annotated in NetPath.</div></p></div><div class="aux"><div class="resc">T. S. Keshava Prasad, et al. Nucleic Acids Res. 2009 January;37(Database issue):D767-D772.</div><p class="links"><a class="dblinks" href="/pubmed/18988627" ref="ordinalpos=965&amp;ncbi_uid=1836163&amp;link_uid=18988627">Citation</a><a class="dblinks" href="/pmc/articles/PMC2686490/" ref="ordinalpos=965&amp;ncbi_uid=1836163&amp;link_uid=PMC2686490">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>966.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4152840/figure/F2/" ref="ordinalpos=966&amp;ncbi_uid=6556599&amp;link_uid=PMC4152840" href="/pmc/articles/PMC4152840/"><img src="/pmc/articles/instance/4152840/bin/nihms172116f2.gif" src-large="/pmc/articles/instance/4152840/bin/nihms172116f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4152840/" ref="ordinalpos=966&amp;ncbi_uid=6556599&amp;link_uid=PMC4152840" image-link="/pmc/articles/PMC4152840/figure/F2/" class="imagepopup">Fig. 2.  From: Notch <span class="highlight" style="background-color:">signaling</span> in pancreatic endocrine cell and diabetes. </a></p><div class="supp"><p class="details"><div>The bimodal <span class="highlight" style="background-color:">pathway</span> of Notch <span class="highlight" style="background-color:">signaling</span> during pancreas development. (a) Lateral inhibition. (b) Suppressive maintenance.</div></p></div><div class="aux"><div class="resc">Wook Kim, et al. Biochem Biophys Res Commun. ;392(3):247-251.</div><p class="links"><a class="dblinks" href="/pubmed/20035712" ref="ordinalpos=966&amp;ncbi_uid=6556599&amp;link_uid=20035712">Citation</a><a class="dblinks" href="/pmc/articles/PMC4152840/" ref="ordinalpos=966&amp;ncbi_uid=6556599&amp;link_uid=PMC4152840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>967.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3943477/figure/F3/" ref="ordinalpos=967&amp;ncbi_uid=5888500&amp;link_uid=PMC3943477" href="/pmc/articles/PMC3943477/"><img src="/pmc/articles/instance/3943477/bin/nihms452393f3.gif" src-large="/pmc/articles/instance/3943477/bin/nihms452393f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3943477/" ref="ordinalpos=967&amp;ncbi_uid=5888500&amp;link_uid=PMC3943477" image-link="/pmc/articles/PMC3943477/figure/F3/" class="imagepopup">Figure 3. Developing <span class="highlight" style="background-color:">Pathway</span> Assays.  From: Case Study Approaches for Implementing the 2007 NRC Report "Toxicity Testing in the 21st Century: A Vision and A Strategy". </a></p><div class="supp"><p class="details"><div>The steps shown here constitute the process of assuring that <span class="highlight" style="background-color:">pathway</span> assay are fit for purpose, i.e., for identifying activity in a specific <span class="highlight" style="background-color:">pathway</span>, assessing the structure and dynamics of the overall <span class="highlight" style="background-color:">signaling</span> network, and providing computational systems biology <span class="highlight" style="background-color:">pathway</span> models to assist dose-response modeling of assay results and low-dose extrapolations. The process of <span class="highlight" style="background-color:">pathway</span> development is fundamental to their use as part of a testing suite for risk assessment, as shown in Figure 2.</div></p></div><div class="aux"><div class="resc">Melvin E. Andersen, et al. ALTEX. 2011;28(3):175-182.</div><p class="links"><a class="dblinks" href="/pubmed/21993955" ref="ordinalpos=967&amp;ncbi_uid=5888500&amp;link_uid=21993955">Citation</a><a class="dblinks" href="/pmc/articles/PMC3943477/" ref="ordinalpos=967&amp;ncbi_uid=5888500&amp;link_uid=PMC3943477">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>968.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913631/figure/fig3/" ref="ordinalpos=968&amp;ncbi_uid=2409297&amp;link_uid=PMC2913631" href="/pmc/articles/PMC2913631/"><img src="/pmc/articles/instance/2913631/bin/PPAR2010-341671.003.gif" src-large="/pmc/articles/instance/2913631/bin/PPAR2010-341671.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913631/" ref="ordinalpos=968&amp;ncbi_uid=2409297&amp;link_uid=PMC2913631" image-link="/pmc/articles/PMC2913631/figure/fig3/" class="imagepopup">Figure 3.  From: PPAR-γ <span class="highlight" style="background-color:">Signaling</span> Crosstalk in Mesenchymal Stem Cells. </a></p><div class="supp"><p class="details"><div>Schematic model of crosstalk between PPAR-<i>γ</i> and Wnt-5a <span class="highlight" style="background-color:">signaling</span> in MSC. NLK activated by the Wnt5a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> phosphorylates SETDB1 and forms a complex with PPAR-<i>γ</i>/RXR and chromodomain containing protein 7 (CHD7).</div></p></div><div class="aux"><div class="resc">Ichiro Takada, et al. PPAR Res. 2010;2010:341671.</div><p class="links"><a class="dblinks" href="/pubmed/20706670" ref="ordinalpos=968&amp;ncbi_uid=2409297&amp;link_uid=20706670">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913631/" ref="ordinalpos=968&amp;ncbi_uid=2409297&amp;link_uid=PMC2913631">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>969.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1828071/figure/F3/" ref="ordinalpos=969&amp;ncbi_uid=996326&amp;link_uid=PMC1828071" href="/pmc/articles/PMC1828071/"><img src="/pmc/articles/instance/1828071/bin/nihms16392f12.gif" src-large="/pmc/articles/instance/1828071/bin/nihms16392f12.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1828071/" ref="ordinalpos=969&amp;ncbi_uid=996326&amp;link_uid=PMC1828071" image-link="/pmc/articles/PMC1828071/figure/F3/" class="imagepopup">Fig. 3.  From: The Role of Intracellular <span class="highlight" style="background-color:">Signaling</span> in Insulin-mediated Regulation of Drug Metabolizing Enzyme Gene and Protein Expression. </a></p><div class="supp"><p class="details"><div>A diagrammatic representation of insulin on growth factor <span class="highlight" style="background-color:">signaling</span> through the Ras-Raf, MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with downstream effects on gene transcription through ERK translocation.</div></p></div><div class="aux"><div class="resc">Sang K. Kim, et al. Pharmacol Ther. ;113(1):88-120.</div><p class="links"><a class="dblinks" href="/pubmed/17097148" ref="ordinalpos=969&amp;ncbi_uid=996326&amp;link_uid=17097148">Citation</a><a class="dblinks" href="/pmc/articles/PMC1828071/" ref="ordinalpos=969&amp;ncbi_uid=996326&amp;link_uid=PMC1828071">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>970.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3295293/figure/fig06/" ref="ordinalpos=970&amp;ncbi_uid=4197011&amp;link_uid=PMC3295293" href="/pmc/articles/PMC3295293/"><img src="/pmc/articles/instance/3295293/bin/pnas.1109407109fig06.gif" src-large="/pmc/articles/instance/3295293/bin/pnas.1109407109fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3295293/" ref="ordinalpos=970&amp;ncbi_uid=4197011&amp;link_uid=PMC3295293" image-link="/pmc/articles/PMC3295293/figure/fig06/" class="imagepopup">Fig. 6. From: Size control of the Drosophila hematopoietic niche by bone morphogenetic protein <span class="highlight" style="background-color:">signaling</span> reveals parallels with mammals. </a></p><div class="supp"><p class="details"><div>Role of BMP and Wg <span class="highlight" style="background-color:">signaling</span> in the <i>Drosophila</i> PSC: parallels with the mouse HSC niche. (<i>A</i>–<i>C</i>) Schematic representation of <i>Drosophila</i> LGs (<i>Left</i>) and the interactions among Col, Dpp, and Wg <span class="highlight" style="background-color:">signaling</span> in the PSC (niche, green) (<i>Right</i>). (<i>A</i>) In wt LGs, signals issued from the PSC maintain Jak-Stat <span class="highlight" style="background-color:">signaling</span> in hemocyte progenitors (red) and control the balance between prohemocyte maintenance and hemocyte differentiation (intermediate progenitors and differentiated hemocytes, blue). Col activity in the PSC both controls the expression of Hh, one prohemocyte maintenance signal (7) and the niche size by restricting BMP <span class="highlight" style="background-color:">signaling</span> to PSC cells, via activation of Dlp. A paracrine/autocrine Dpp &gt; dpp autoregulatory loop maintains a high level of BMP <span class="highlight" style="background-color:">signaling</span> and represses Dmyc in the PSC. Conversely, Wg <span class="highlight" style="background-color:">signaling</span> positively regulates Dmyc expression. Wg is epistatic to Dpp <span class="highlight" style="background-color:">signaling</span> in the PSC, but whether Dpp <span class="highlight" style="background-color:">signaling</span> only represses the Wg <span class="highlight" style="background-color:">pathway</span> or also acts in parallel to Wg in controlling <i>dmyc</i> expression remains an open question. *Represents the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The role of Dpp <span class="highlight" style="background-color:">signaling</span> in the PSC and roles of EBF2, BMPR1A, and Wnt in mouse osteoblasts (10, 44) highlight parallels between the <i>Drosophila</i> and mammalian hematopoietic niches. (<i>B</i>) Blocking Dpp activity in PSC cells leads to increased levels of Dmyc expression and increased number of cells. Because each PSC cell expresses normal levels of prohemocyte maintenance signals, the MZ receives more signal and more progenitors are maintained at the expense of differentiation. (<i>C</i>) Reducing Col expression in the larval PSC results in both down-regulation of Dlp expression and Dpp <span class="highlight" style="background-color:">signaling</span> and Hh expression, leading to an increase in the number of PSC cells, with each cell producing less Hh signal. The overall effect is restoring normal homeostasis in the LG.</div></p></div><div class="aux"><div class="resc">Delphine Pennetier, et al. Proc Natl Acad Sci U S A. 2012 February 28;109(9):3389-3394.</div><p class="links"><a class="dblinks" href="/pubmed/22331866" ref="ordinalpos=970&amp;ncbi_uid=4197011&amp;link_uid=22331866">Citation</a><a class="dblinks" href="/pmc/articles/PMC3295293/" ref="ordinalpos=970&amp;ncbi_uid=4197011&amp;link_uid=PMC3295293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>971.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3532475/figure/F6/" ref="ordinalpos=971&amp;ncbi_uid=5602022&amp;link_uid=PMC3532475" href="/pmc/articles/PMC3532475/"><img src="/pmc/articles/instance/3532475/bin/zh20241267810006.gif" src-large="/pmc/articles/instance/3532475/bin/zh20241267810006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3532475/" ref="ordinalpos=971&amp;ncbi_uid=5602022&amp;link_uid=PMC3532475" image-link="/pmc/articles/PMC3532475/figure/F6/" class="imagepopup">Fig. 6. From: Klotho protects against mouse renal fibrosis by inhibiting Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Microarray analysis of Wnt <span class="highlight" style="background-color:">signaling</span> component gene in kidney of WT and KLTG. Diagram of mammalian Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> superimposed on oligonucleotide-based chip data at 14 days after UUO. Downregulated genes are displayed in green and upregulated genes are displayed in red. <i>A</i>: illustration of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>B</i>: gene expression of KLTG sham compared with WT sham. <i>C</i>: gene expression of WT UUO compared with WT sham. <i>D</i>: gene expression of KLTG UUO compared with KLTG sham. Six samples in each group were mixed and used for microarray analysis.</div></p></div><div class="aux"><div class="resc">Minoru Satoh, et al. Am J Physiol Renal Physiol. 2012 December 15;303(12):F1641-F1651.</div><p class="links"><a class="dblinks" href="/pubmed/23034937" ref="ordinalpos=971&amp;ncbi_uid=5602022&amp;link_uid=23034937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3532475/" ref="ordinalpos=971&amp;ncbi_uid=5602022&amp;link_uid=PMC3532475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>972.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2494797/figure/f7/" ref="ordinalpos=972&amp;ncbi_uid=1442317&amp;link_uid=PMC2494797" href="/pmc/articles/PMC2494797/"><img src="/pmc/articles/instance/2494797/bin/zh50070852350007.gif" src-large="/pmc/articles/instance/2494797/bin/zh50070852350007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2494797/" ref="ordinalpos=972&amp;ncbi_uid=1442317&amp;link_uid=PMC2494797" image-link="/pmc/articles/PMC2494797/figure/f7/" class="imagepopup">Fig. 7. From: Involvement of RhoA/Rho kinase <span class="highlight" style="background-color:">signaling</span> in protection against monocrotaline-induced pulmonary hypertension in pneumonectomized rats by dehydroepiandrosterone. </a></p><div class="supp"><p class="details"><div>Possible mechanisms of protection by DHEA against pulmonary hypertension via inhibition of RhoA/ROCK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Noriyuki Homma, et al. Am J Physiol Lung Cell Mol Physiol. 2008 July;295(1):L71-L78.</div><p class="links"><a class="dblinks" href="/pubmed/18469113" ref="ordinalpos=972&amp;ncbi_uid=1442317&amp;link_uid=18469113">Citation</a><a class="dblinks" href="/pmc/articles/PMC2494797/" ref="ordinalpos=972&amp;ncbi_uid=1442317&amp;link_uid=PMC2494797">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>973.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045367/figure/pone-0016580-g001/" ref="ordinalpos=973&amp;ncbi_uid=2806739&amp;link_uid=PMC3045367" href="/pmc/articles/PMC3045367/"><img src="/pmc/articles/instance/3045367/bin/pone.0016580.g001.gif" src-large="/pmc/articles/instance/3045367/bin/pone.0016580.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045367/" ref="ordinalpos=973&amp;ncbi_uid=2806739&amp;link_uid=PMC3045367" image-link="/pmc/articles/PMC3045367/figure/pone-0016580-g001/" class="imagepopup">Figure 1. Critical node concept in the insulin <span class="highlight" style="background-color:">signaling</span> network..  From: Exploring the Evolutionary Relationship of Insulin Receptor Substrate Family Using Computational Biology. </a></p><div class="supp"><p class="details"><div>(A) Critical node is the nodal point which consists of the effect molecules for a further downstream action in a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) This <span class="highlight" style="background-color:">pathway</span> shows that IRS family members belong to a critical node. This node is one of the important node in a <span class="highlight" style="background-color:">signaling</span> path way.</div></p></div><div class="aux"><div class="resc">Chiranjib Chakraborty, et al. PLoS One. 2011;6(2):e16580.</div><p class="links"><a class="dblinks" href="/pubmed/21364910" ref="ordinalpos=973&amp;ncbi_uid=2806739&amp;link_uid=21364910">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045367/" ref="ordinalpos=973&amp;ncbi_uid=2806739&amp;link_uid=PMC3045367">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>974.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912738/figure/F1/" ref="ordinalpos=974&amp;ncbi_uid=5823319&amp;link_uid=PMC3912738" href="/pmc/articles/PMC3912738/"><img src="/pmc/articles/instance/3912738/bin/fendo-05-00010-g001.gif" src-large="/pmc/articles/instance/3912738/bin/fendo-05-00010-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912738/" ref="ordinalpos=974&amp;ncbi_uid=5823319&amp;link_uid=PMC3912738" image-link="/pmc/articles/PMC3912738/figure/F1/" class="imagepopup">Figure 1.  From: The Emerging Role of Insulin and Insulin-Like Growth Factor <span class="highlight" style="background-color:">Signaling</span> in Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of IGF <span class="highlight" style="background-color:">signaling</span> involvement in EMT</b>. The two main <span class="highlight" style="background-color:">signaling</span> pathways downstream the IGF-IR are both involved in the regulation of transcription factors of the Twist, Snail, and ZEB1 families, all involved in the EMT program. The IRS-1/PI3K/Akt <span class="highlight" style="background-color:">pathway</span> leads to the phosphorylation and inactivation of GSK3β, thus preventing the phosphorylation and proteasome targeting of Snail and Slug. The same <span class="highlight" style="background-color:">pathway</span> also induces the activation of NF-kB transcription factor, which then increases the transcription of Snail and ZEB1. On the other hand, the Ras/Raf/ERK <span class="highlight" style="background-color:">pathway</span> is involved in the transcription of Twist and ZEB1. In turn, Snail inhibits PTEN gene transcription, thus potentiating the IRS-1/PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. In addition, Slug increases the transcription of IGF-I, the main IGF-IR ligand.</div></p></div><div class="aux"><div class="resc">Roberta Malaguarnera, et al. Front Endocrinol (Lausanne). 2014;5:10.</div><p class="links"><a class="dblinks" href="/pubmed/24550888" ref="ordinalpos=974&amp;ncbi_uid=5823319&amp;link_uid=24550888">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912738/" ref="ordinalpos=974&amp;ncbi_uid=5823319&amp;link_uid=PMC3912738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>975.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3210519/figure/F1/" ref="ordinalpos=975&amp;ncbi_uid=3500917&amp;link_uid=PMC3210519" href="/pmc/articles/PMC3210519/"><img src="/pmc/articles/instance/3210519/bin/bioa0104_0200_fig001.gif" src-large="/pmc/articles/instance/3210519/bin/bioa0104_0200_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3210519/" ref="ordinalpos=975&amp;ncbi_uid=3500917&amp;link_uid=PMC3210519" image-link="/pmc/articles/PMC3210519/figure/F1/" class="imagepopup">Figure 1.  From: Ras and Rho GTPases on the move. </a></p><div class="supp"><p class="details"><div>Schematic representation of the balance between Cdc42 and RasGRF pathways. (A) The balance between both <span class="highlight" style="background-color:">signaling</span> pathways is in equilibrium and both are active to a similar extent. (B) RasGRF expression is higher and there is more <span class="highlight" style="background-color:">signaling</span> through the ERK <span class="highlight" style="background-color:">pathway</span> while Cdc42 <span class="highlight" style="background-color:">signaling</span> is low. (C) Tumor cell plasticity allows that both pathways signal on and off depending on environmental conditions. (D) RasGRF expression is lower and there is less <span class="highlight" style="background-color:">signaling</span> through the ER K <span class="highlight" style="background-color:">pathway</span> while Cdc42 <span class="highlight" style="background-color:">signaling</span> is high.</div></p></div><div class="aux"><div class="resc">Piero Crespo, et al. Bioarchitecture. 2011 Jul-Aug;1(4):200-204.</div><p class="links"><a class="dblinks" href="/pubmed/22069515" ref="ordinalpos=975&amp;ncbi_uid=3500917&amp;link_uid=22069515">Citation</a><a class="dblinks" href="/pmc/articles/PMC3210519/" ref="ordinalpos=975&amp;ncbi_uid=3500917&amp;link_uid=PMC3210519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>976.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3865727/figure/F3/" ref="ordinalpos=976&amp;ncbi_uid=5641693&amp;link_uid=PMC3865727" href="/pmc/articles/PMC3865727/"><img src="/pmc/articles/instance/3865727/bin/fphys-04-00370-g0003.gif" src-large="/pmc/articles/instance/3865727/bin/fphys-04-00370-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3865727/" ref="ordinalpos=976&amp;ncbi_uid=5641693&amp;link_uid=PMC3865727" image-link="/pmc/articles/PMC3865727/figure/F3/" class="imagepopup">Figure 3.  From: pH sensing and regulation in cancer. </a></p><div class="supp"><p class="details"><div><b>Cancer cells express proton-sensing GPCRs such as GPR4, TDAG8, and OGR1 to regulate their tumorigenesis and invasion</b>. TDAG8 and OGR1 sense extracellular protons, leading to activation of the cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. OGR1 is usually coupled with the PLC/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> through G<sub>q</sub>/11 proteins and GPR4 and TDAG8 are coupled with the adenylyl cyclase/cAMP <span class="highlight" style="background-color:">pathway</span> through G<sub>s</sub> proteins in the cells to PKA and ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that play pivotal roles in cancer progression.</div></p></div><div class="aux"><div class="resc">Mehdi Damaghi, et al. Front Physiol. 2013;4:370.</div><p class="links"><a class="dblinks" href="/pubmed/24381558" ref="ordinalpos=976&amp;ncbi_uid=5641693&amp;link_uid=24381558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3865727/" ref="ordinalpos=976&amp;ncbi_uid=5641693&amp;link_uid=PMC3865727">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>977.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3116122/figure/Fig1/" ref="ordinalpos=977&amp;ncbi_uid=3096403&amp;link_uid=PMC3116122" href="/pmc/articles/PMC3116122/"><img src="/pmc/articles/instance/3116122/bin/11060_2010_442_Fig1_HTML.gif" src-large="/pmc/articles/instance/3116122/bin/11060_2010_442_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3116122/" ref="ordinalpos=977&amp;ncbi_uid=3096403&amp;link_uid=PMC3116122" image-link="/pmc/articles/PMC3116122/figure/Fig1/" class="imagepopup">Fig. 1.  From: PI3Kinase <span class="highlight" style="background-color:">signaling</span> in glioblastoma. </a></p><div class="supp"><p class="details"><div>The PI3K <span class="highlight" style="background-color:">pathway</span> activates different processes that regulate proliferation, apoptosis and migration. Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activated by receptor tyrosine kinase (RTK) and the PI3K downstream activation. Designed by www.lebensart003.com</div></p></div><div class="aux"><div class="resc">M. M. Lino, et al. J Neurooncol. 2011 July;103(3):417-427.</div><p class="links"><a class="dblinks" href="/pubmed/21063898" ref="ordinalpos=977&amp;ncbi_uid=3096403&amp;link_uid=21063898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3116122/" ref="ordinalpos=977&amp;ncbi_uid=3096403&amp;link_uid=PMC3116122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>978.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2322867/figure/F2/" ref="ordinalpos=978&amp;ncbi_uid=1336696&amp;link_uid=PMC2322867" href="/pmc/articles/PMC2322867/"><img src="/pmc/articles/instance/2322867/bin/nihms45365f2.gif" src-large="/pmc/articles/instance/2322867/bin/nihms45365f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2322867/" ref="ordinalpos=978&amp;ncbi_uid=1336696&amp;link_uid=PMC2322867" image-link="/pmc/articles/PMC2322867/figure/F2/" class="imagepopup">Figure 2. Models for the mechanisms of cross-talk between receptor tyrosine kinase/growth factor and rapid estradiol-induced activation of ERK1/2.  From: Rapid <span class="highlight" style="background-color:">Signaling</span> Mechanisms of Estrogens in the Developing Cerebellum. </a></p><div class="supp"><p class="details"><div>This schematic diagram illustrates proposed mER-mediated mechanisms that can mediate rapid estradiol-induced ERK1/2 activation. <span class="highlight" style="background-color:">Pathway</span> 1 represents G-protein-like transactivation of epidermal growth factor receptor (EGFR) proposed to occur via GPR30 or ERα induction of MMP-mediated heprin-bound EGF (HB-EGF) shedding; <span class="highlight" style="background-color:">Pathway</span> 2 shows the mechanism proposed to involve direct interaction of ERα with Shc which acts to couple the activated ER to the ras/ERK <span class="highlight" style="background-color:">pathway</span>; <span class="highlight" style="background-color:">Pathway</span> 3 is a cascade involving the interaction of the ER with the adaptor protein PELP/MNAR 1, or directly interacting with c-Src to stimulate c-Src kinase, which in turn activates Ras to link the activated ER to ERK-MAPK <span class="highlight" style="background-color:">pathway</span>; <span class="highlight" style="background-color:">Pathway</span> 4 shows the G-protein and Src-kinase dependent, EGF-receptor independent <span class="highlight" style="background-color:">pathway</span> operative in cerebellar granule cells. The site of inhibition for specific antagonists used to elucidate the <span class="highlight" style="background-color:">pathway</span> of rapid cerebellar <span class="highlight" style="background-color:">signaling</span> are indicted. The dotted arrow indicates the identified rapid intracellular estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that stimulates protein phosphatase 2A (PP2A) activity in granule cells. This phosphatase <span class="highlight" style="background-color:">pathway</span> is independent of Gαi-<span class="highlight" style="background-color:">signaling</span> and is also PKA independent. PP2A activation is required, along with rapid ERK-activation, for estrogen to induce calpain-mediated cell death in granule cell precursors. Key protein kinase activation phosphorylation sites are indicated with a red P. The PI3K <span class="highlight" style="background-color:">pathway</span> is included to illustrate that EGF-receptor activation in granule cells activates both the ERK &amp; PI3K pathways, but estrogen only stimulates ERK-<span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Scott M. Belcher. Brain Res Rev. ;57(2):481-492.</div><p class="links"><a class="dblinks" href="/pubmed/17931703" ref="ordinalpos=978&amp;ncbi_uid=1336696&amp;link_uid=17931703">Citation</a><a class="dblinks" href="/pmc/articles/PMC2322867/" ref="ordinalpos=978&amp;ncbi_uid=1336696&amp;link_uid=PMC2322867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>979.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3720582/figure/pone-0069132-g002/" ref="ordinalpos=979&amp;ncbi_uid=5183320&amp;link_uid=PMC3720582" href="/pmc/articles/PMC3720582/"><img src="/pmc/articles/instance/3720582/bin/pone.0069132.g002.gif" src-large="/pmc/articles/instance/3720582/bin/pone.0069132.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3720582/" ref="ordinalpos=979&amp;ncbi_uid=5183320&amp;link_uid=PMC3720582" image-link="/pmc/articles/PMC3720582/figure/pone-0069132-g002/" class="imagepopup">Figure 2. Network picture of Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Structural and Logical Analysis of a Comprehensive Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> to Identify Alternative Drug Targets for Glioma, Colon and Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div>The colored circles represent the nodes or proteins of the <span class="highlight" style="background-color:">pathway</span> and the black arrows indicate an edge or connection between two nodes of the network. The nodes are colored according to their sub cellular locations in the cell (Figure 1) and are divided into four regions: Extracellular and Membrane (Blue), Cytoplasm (Red), Nuclear (Green) and Output proteins (Yellow) respectively. The size of the nodes is assigned according to their total number of connections or degree. Total degree of each node is followed by the name of the proteins. The “Bow-Tie” structure of the Hedgehog <span class="highlight" style="background-color:">signaling</span> network is easily visible, where the signals are converging towards GLI1 or GLI2 and diverging to its subsequent steps. The size the node GLI1 is biggest as it has highest number of connections or degree among all other proteins in the network.</div></p></div><div class="aux"><div class="resc">Saikat Chowdhury, et al. PLoS One. 2013;8(7):e69132.</div><p class="links"><a class="dblinks" href="/pubmed/23935937" ref="ordinalpos=979&amp;ncbi_uid=5183320&amp;link_uid=23935937">Citation</a><a class="dblinks" href="/pmc/articles/PMC3720582/" ref="ordinalpos=979&amp;ncbi_uid=5183320&amp;link_uid=PMC3720582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>980.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1087802/figure/f2/" ref="ordinalpos=980&amp;ncbi_uid=416573&amp;link_uid=PMC1087802" href="/pmc/articles/PMC1087802/"><img src="/pmc/articles/instance/1087802/bin/zek0030524280002.gif" src-large="/pmc/articles/instance/1087802/bin/zek0030524280002.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1087802/" ref="ordinalpos=980&amp;ncbi_uid=416573&amp;link_uid=PMC1087802" image-link="/pmc/articles/PMC1087802/figure/f2/" class="imagepopup">FIG. 2. From: Except in Every Detail: Comparing and Contrasting G-Protein <span class="highlight" style="background-color:">Signaling</span> in Saccharomyces cerevisiae and Schizosaccharomyces pombe. </a></p><div class="supp"><p class="details"><div>G-protein-mediated <span class="highlight" style="background-color:">signaling</span> pathways of <i>S. cerevisiae</i> and <i>S. pombe</i>. Schematics identify the key receptors, G-protein subunits, and effectors, as well as some additional relevant proteins for (A) the <i>S. cerevisiae</i> pheromone <span class="highlight" style="background-color:">pathway</span>, (B) the <i>S. pombe</i> pheromone <span class="highlight" style="background-color:">pathway</span>, (C) the <i>S. cerevisiae</i> glucose/cAMP <span class="highlight" style="background-color:">pathway</span> (note that the Gpb1/2-Gpg1 Gβγ mimic may not be associated with the membrane due to the absence of a CAAX box on Gpg1), and (D) the <i>S. pombe</i> glucose/cAMP <span class="highlight" style="background-color:">pathway</span> (note the lack of an N-terminal coiled-coil domain in the Git5 Gβ). Details for each <span class="highlight" style="background-color:">pathway</span> are provided in the text.</div></p></div><div class="aux"><div class="resc">Charles S. Hoffman. Eukaryot Cell. 2005 March;4(3):495-503.</div><p class="links"><a class="dblinks" href="/pubmed/15755912" ref="ordinalpos=980&amp;ncbi_uid=416573&amp;link_uid=15755912">Citation</a><a class="dblinks" href="/pmc/articles/PMC1087802/" ref="ordinalpos=980&amp;ncbi_uid=416573&amp;link_uid=PMC1087802">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>981.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3653035/figure/Fig2/" ref="ordinalpos=981&amp;ncbi_uid=4903992&amp;link_uid=PMC3653035" href="/pmc/articles/PMC3653035/"><img src="/pmc/articles/instance/3653035/bin/11064_2013_980_Fig2_HTML.gif" src-large="/pmc/articles/instance/3653035/bin/11064_2013_980_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3653035/" ref="ordinalpos=981&amp;ncbi_uid=4903992&amp;link_uid=PMC3653035" image-link="/pmc/articles/PMC3653035/figure/Fig2/" class="imagepopup">Fig. 2.  From: Physiological Role of ?-Catenin/TCF <span class="highlight" style="background-color:">Signaling</span> in Neurons of the Adult Brain. </a></p><div class="supp"><p class="details"><div>Wnt ligands induce diverse Wnt pathways. <b>a, b</b> Wnt ligands bind Frizzled receptors and activate canonical Wnt <span class="highlight" style="background-color:">signaling</span>. The inhibition of GSK3α/β in the canonical Wnt <span class="highlight" style="background-color:">pathway</span> can lead to the activation of β-catenin or influence other targets of GSK3α/β. These two possibilities can be called the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> and “divergent canonical” Wnt <span class="highlight" style="background-color:">pathway</span>, respectively. <b>c</b> Other WNT molecules and Frizzled receptors can activate planar cell polarity (PCP) <span class="highlight" style="background-color:">signaling</span>, in which downstream messenger are ROCK and JNK kinases. <b>d</b> WNT5a can regulate intracellular level of Ca<sup>2+</sup>, which is involved in many cellular signalings and can inhibit β-catenin/TCF <span class="highlight" style="background-color:">signaling</span>. <b>e, f</b> WNT molecules can also bind to the receptor tyrosine kinases (RTK) ROR and RYK. The Wnt/RYK <span class="highlight" style="background-color:">pathway</span> regulates neuronal differentiation</div></p></div><div class="aux"><div class="resc">Marta B. Wisniewska. Neurochem Res. 2013 June;38(6):1144-1155.</div><p class="links"><a class="dblinks" href="/pubmed/23377854" ref="ordinalpos=981&amp;ncbi_uid=4903992&amp;link_uid=23377854">Citation</a><a class="dblinks" href="/pmc/articles/PMC3653035/" ref="ordinalpos=981&amp;ncbi_uid=4903992&amp;link_uid=PMC3653035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>982.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3384423/figure/F1/" ref="ordinalpos=982&amp;ncbi_uid=4083485&amp;link_uid=PMC3384423" href="/pmc/articles/PMC3384423/"><img src="/pmc/articles/instance/3384423/bin/neo1405_0360_fig001.gif" src-large="/pmc/articles/instance/3384423/bin/neo1405_0360_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3384423/" ref="ordinalpos=982&amp;ncbi_uid=4083485&amp;link_uid=PMC3384423" image-link="/pmc/articles/PMC3384423/figure/F1/" class="imagepopup">Figure 1.  From: Targeting the mTOR-DEPTOR <span class="highlight" style="background-color:">Pathway</span> by CRL E3 Ubiquitin Ligases: Therapeutic Application. </a></p><div class="supp"><p class="details"><div>The mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Arrows stand for activation; bars represent inhibition.  represents phosphorylation. For details, see text.</div></p></div><div class="aux"><div class="resc">Yongchao Zhao, et al. Neoplasia. 2012 May;14(5):360-367.</div><p class="links"><a class="dblinks" href="/pubmed/22745582" ref="ordinalpos=982&amp;ncbi_uid=4083485&amp;link_uid=22745582">Citation</a><a class="dblinks" href="/pmc/articles/PMC3384423/" ref="ordinalpos=982&amp;ncbi_uid=4083485&amp;link_uid=PMC3384423">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>983.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678438/figure/fig9/" ref="ordinalpos=983&amp;ncbi_uid=5002282&amp;link_uid=PMC3678438" href="/pmc/articles/PMC3678438/"><img src="/pmc/articles/instance/3678438/bin/ECAM2013-459467.009.gif" src-large="/pmc/articles/instance/3678438/bin/ECAM2013-459467.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678438/" ref="ordinalpos=983&amp;ncbi_uid=5002282&amp;link_uid=PMC3678438" image-link="/pmc/articles/PMC3678438/figure/fig9/" class="imagepopup">Figure 9.  From: PI3K/Akt <span class="highlight" style="background-color:">Pathway</span> Contributes to Neurovascular Unit Protection of Xiao-Xu-Ming Decoction against Focal Cerebral Ischemia and Reperfusion Injury in Rats. </a></p><div class="supp"><p class="details"><div>The diagram of the cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> which was involved in NVU protection of XXMD against cerebral injury following focal cerebral ischemia and reperfusion. Reperfusion after stroke induced dysfunction of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and the PI3K/Akt inhibition led to dephosphorylation of PDK1, Akt, and GSK3<i>β</i> degradation, resulting in apoptosis. However, XXMD treatment inhibited apoptosis of different cells in NVU and increased expression levels of p-PTEN, p-PDK1, p-Akt, and p-GSK3<i>β</i>. Furthermore, PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is associated with Ras/MAPK <span class="highlight" style="background-color:">pathway</span>. In the present study, we also found that XXMD upregulated the level of p-c-raf. Above all, all the observations in the study indicated that XXMD may exert its NVU protection partly through the activation of PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Rui Lan, et al. Evid Based Complement Alternat Med. 2013;2013:459467.</div><p class="links"><a class="dblinks" href="/pubmed/23781261" ref="ordinalpos=983&amp;ncbi_uid=5002282&amp;link_uid=23781261">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678438/" ref="ordinalpos=983&amp;ncbi_uid=5002282&amp;link_uid=PMC3678438">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>984.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3413998/figure/fig5/" ref="ordinalpos=984&amp;ncbi_uid=4178343&amp;link_uid=PMC3413998" href="/pmc/articles/PMC3413998/"><img src="/pmc/articles/instance/3413998/bin/BMR2012-270425.005.gif" src-large="/pmc/articles/instance/3413998/bin/BMR2012-270425.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3413998/" ref="ordinalpos=984&amp;ncbi_uid=4178343&amp;link_uid=PMC3413998" image-link="/pmc/articles/PMC3413998/figure/fig5/" class="imagepopup">Figure 5.  From: Hematopoietic Stem Cell Development, Niches, and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>c-Mpl/TPO <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TPO signals to its receptor, c-Mpl, and induces the downstream <span class="highlight" style="background-color:">signaling</span> cascades: STATs, PI3K, MAPKs, and extracellular signal regulated kinases-1 and -2 (modified from [14, 15]).</div></p></div><div class="aux"><div class="resc">Kamonnaree Chotinantakul, et al. Bone Marrow Res. 2012;2012:270425.</div><p class="links"><a class="dblinks" href="/pubmed/22900188" ref="ordinalpos=984&amp;ncbi_uid=4178343&amp;link_uid=22900188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3413998/" ref="ordinalpos=984&amp;ncbi_uid=4178343&amp;link_uid=PMC3413998">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>985.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3364461/figure/F5/" ref="ordinalpos=985&amp;ncbi_uid=4043513&amp;link_uid=PMC3364461" href="/pmc/articles/PMC3364461/"><img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-g005.gif" src-large="/pmc/articles/instance/3364461/bin/fendo-02-00045-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3364461/" ref="ordinalpos=985&amp;ncbi_uid=4043513&amp;link_uid=PMC3364461" image-link="/pmc/articles/PMC3364461/figure/F5/" class="imagepopup">Figure 5.  From: Mapping the Follicle-Stimulating Hormone-Induced <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div><b>Proteins interacting at the FSHR and their role in <span class="highlight" style="background-color:">signaling</span></b>. A number of proteins have been reported to interact with the FSHR and to impact on FSH-induced <span class="highlight" style="background-color:">signaling</span> pathways. These include 14-3-3τ, FoxO1, and APPL1, the latter two being involved in the control of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> downstream of the FSHR.</div></p></div><div class="aux"><div class="resc">Pauline Gloaguen, et al. Front Endocrinol (Lausanne). 2011;2:45.</div><p class="links"><a class="dblinks" href="/pubmed/22666216" ref="ordinalpos=985&amp;ncbi_uid=4043513&amp;link_uid=22666216">Citation</a><a class="dblinks" href="/pmc/articles/PMC3364461/" ref="ordinalpos=985&amp;ncbi_uid=4043513&amp;link_uid=PMC3364461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>986.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC419826/figure/fig9/" ref="ordinalpos=986&amp;ncbi_uid=393914&amp;link_uid=PMC419826" href="/pmc/articles/PMC419826/"><img src="/pmc/articles/instance/419826/bin/pp1341500f09.gif" src-large="/pmc/articles/instance/419826/bin/pp1341500f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC419826/" ref="ordinalpos=986&amp;ncbi_uid=393914&amp;link_uid=PMC419826" image-link="/pmc/articles/PMC419826/figure/fig9/" class="imagepopup">Figure 9. From: A Rice WRKY Gene Encodes a Transcriptional Repressor of the Gibberellin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Aleurone Cells. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> components on the GA response <span class="highlight" style="background-color:">pathway</span> leading to expression of the <i>Amy32b</i> <i>α</i>-amylase gene in aleurone cells. SLN1 is a repressor, which is destabilized by GA treatment; GAMYB is a GA-inducible transcriptional activator. OsWRKY71 functions as a transcriptional repressor interacting with GAMYB to repress the GA response; GA down-regulates the expression of the <i>OsWRKY71</i> gene at transcriptional or posttranscriptional level and destabilizes OsWRKY71 protein.</div></p></div><div class="aux"><div class="resc">Zhong-Lin Zhang, et al. Plant Physiol. 2004 April;134(4):1500-1513.</div><p class="links"><a class="dblinks" href="/pubmed/15047897" ref="ordinalpos=986&amp;ncbi_uid=393914&amp;link_uid=15047897">Citation</a><a class="dblinks" href="/pmc/articles/PMC419826/" ref="ordinalpos=986&amp;ncbi_uid=393914&amp;link_uid=PMC419826">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>987.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC117624/figure/F6/" ref="ordinalpos=987&amp;ncbi_uid=323099&amp;link_uid=PMC117624" href="/pmc/articles/PMC117624/"><img src="/pmc/articles/instance/117624/bin/mk0621859006.gif" src-large="/pmc/articles/instance/117624/bin/mk0621859006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC117624/" ref="ordinalpos=987&amp;ncbi_uid=323099&amp;link_uid=PMC117624" image-link="/pmc/articles/PMC117624/figure/F6/" class="imagepopup">Figure 6.  From: Calmodulin Regulates Intracellular Trafficking of Epidermal  Growth Factor Receptor and the MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>W-13 effects on MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Dissection  of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Serum starved COS-1 cells grown in  100-mm dishes were incubated with W-13 (10 μg/ml) for 60 min, and the  last 15 min with the indicated concentrations of EGF, at 37°C.  Lysates with equal amount of protein were electrophoresed and the  stimulation of EGFR-phosphotyrosine (RC-20) (A), the Ras activation  (Ras-GTP) assessed by GST-RBD pull down (B), and the MEK activation (C)  were analyzed by Western blotting.</div></p></div><div class="aux"><div class="resc">Francesc Tebar, et al. Mol Biol Cell. 2002 June;13(6):2057-2068.</div><p class="links"><a class="dblinks" href="/pubmed/12058069" ref="ordinalpos=987&amp;ncbi_uid=323099&amp;link_uid=12058069">Citation</a><a class="dblinks" href="/pmc/articles/PMC117624/" ref="ordinalpos=987&amp;ncbi_uid=323099&amp;link_uid=PMC117624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>988.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2659402/figure/F3/" ref="ordinalpos=988&amp;ncbi_uid=1720663&amp;link_uid=PMC2659402" href="/pmc/articles/PMC2659402/"><img src="/pmc/articles/instance/2659402/bin/nihms84561f3.gif" src-large="/pmc/articles/instance/2659402/bin/nihms84561f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2659402/" ref="ordinalpos=988&amp;ncbi_uid=1720663&amp;link_uid=PMC2659402" image-link="/pmc/articles/PMC2659402/figure/F3/" class="imagepopup">Fig. 3.  From: Structures and biological functions of IL-31 and IL-31 receptors. </a></p><div class="supp"><p class="details"><div>Interleukin-31 acts through three <span class="highlight" style="background-color:">signaling</span> pathways: Jak/STAT <span class="highlight" style="background-color:">pathway</span>, PI3K/AKT <span class="highlight" style="background-color:">pathway</span> and MAPK <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Qing Zhang, et al. Cytokine Growth Factor Rev. ;19(5-6):347-356.</div><p class="links"><a class="dblinks" href="/pubmed/18926762" ref="ordinalpos=988&amp;ncbi_uid=1720663&amp;link_uid=18926762">Citation</a><a class="dblinks" href="/pmc/articles/PMC2659402/" ref="ordinalpos=988&amp;ncbi_uid=1720663&amp;link_uid=PMC2659402">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>989.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2671376/figure/F7-7952/" ref="ordinalpos=989&amp;ncbi_uid=1723718&amp;link_uid=PMC2671376" href="/pmc/articles/PMC2671376/"><img src="/pmc/articles/instance/2671376/bin/zjh0040979520007.gif" src-large="/pmc/articles/instance/2671376/bin/zjh0040979520007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2671376/" ref="ordinalpos=989&amp;ncbi_uid=1723718&amp;link_uid=PMC2671376" image-link="/pmc/articles/PMC2671376/figure/F7-7952/" class="imagepopup">Figure 7.  From: PTEN Contributes to Profound PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Deregulation in Dystrophin-Deficient Dog Muscle. </a></p><div class="supp"><p class="details"><div>Schematic diagram of <span class="highlight" style="background-color:">signaling</span> in the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and deregulations detected in GRMD skeletal muscle. Proteins shown in red, green, and yellow indicate decreased, increased, and unchanged, expression or activity of these enzymes in GRMD versus healthy muscle, respectively. As PP2A phosphatase activity was not statistically modulated in GRMD muscle, the protein is shown in yellow. The deregulations detected in GRMD skeletal muscle could lead to decreased protein synthesis and block compensatory hypertrophy.</div></p></div><div class="aux"><div class="resc">Marie Feron, et al. Am J Pathol. 2009 April;174(4):1459-1470.</div><p class="links"><a class="dblinks" href="/pubmed/19264909" ref="ordinalpos=989&amp;ncbi_uid=1723718&amp;link_uid=19264909">Citation</a><a class="dblinks" href="/pmc/articles/PMC2671376/" ref="ordinalpos=989&amp;ncbi_uid=1723718&amp;link_uid=PMC2671376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>990.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478298/figure/F1/" ref="ordinalpos=990&amp;ncbi_uid=4338324&amp;link_uid=PMC3478298" href="/pmc/articles/PMC3478298/"><img src="/pmc/articles/instance/3478298/bin/cc-11-3529-g1.gif" src-large="/pmc/articles/instance/3478298/bin/cc-11-3529-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478298/" ref="ordinalpos=990&amp;ncbi_uid=4338324&amp;link_uid=PMC3478298" image-link="/pmc/articles/PMC3478298/figure/F1/" class="imagepopup">Figure. From: <span class="highlight" style="background-color:">Signaling</span> dynamics and embryonic development. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Schematics of simple and adaptive responses. Green lines represent ligand dynamics, and blue lines represent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> output.</div></p></div><div class="aux"><div class="resc">Aryeh Warmflash, et al. Cell Cycle. 2012 October 1;11(19):3529-3530.</div><p class="links"><a class="dblinks" href="/pubmed/22935712" ref="ordinalpos=990&amp;ncbi_uid=4338324&amp;link_uid=22935712">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478298/" ref="ordinalpos=990&amp;ncbi_uid=4338324&amp;link_uid=PMC3478298">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>991.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694555/figure/F3/" ref="ordinalpos=991&amp;ncbi_uid=1936001&amp;link_uid=PMC2694555" href="/pmc/articles/PMC2694555/"><img src="/pmc/articles/instance/2694555/bin/CG-9-349_F3.gif" src-large="/pmc/articles/instance/2694555/bin/CG-9-349_F3.jpg" alt="Fig. (3)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694555/" ref="ordinalpos=991&amp;ncbi_uid=1936001&amp;link_uid=PMC2694555" image-link="/pmc/articles/PMC2694555/figure/F3/" class="imagepopup">Fig. (3).  From: Can Systems Biology Understand <span class="highlight" style="background-color:">Pathway</span> Activation? Gene Expression Signatures as Surrogate Markers for Understanding the Complexity of <span class="highlight" style="background-color:">Pathway</span> Activation. </a></p><div class="supp"><p class="details"><div>Response to <span class="highlight" style="background-color:">pathway</span> inhibitors of tumor cells depends on the mechanism of <span class="highlight" style="background-color:">pathway</span> activation. <b>(a)</b> Tumor cells with EGFR mutation are sensitive to EGFR inhibitor gefitinib or erlotinib, whereas those with RAS mutation are resistant to the same drug, because RAF-MEK-ERK <span class="highlight" style="background-color:">signaling</span> is not inhibited by the drugs. <b>(b)</b> When the PI3K <span class="highlight" style="background-color:">pathway</span> is hyperactivated by PI3K mutation or by PTEN loss, tumor cells are sensitive to rapamycin, which inhibits mTOR downstream of the <span class="highlight" style="background-color:">pathway</span>. However, the HER2 inhibitor Herceptin cannot stop the deregulated signal, because it does not inhibit hyperactivation downstream.</div></p></div><div class="aux"><div class="resc">Hiraku Itadani, et al. Curr Genomics. 2008 August;9(5):349-360.</div><p class="links"><a class="dblinks" href="/pubmed/19517027" ref="ordinalpos=991&amp;ncbi_uid=1936001&amp;link_uid=19517027">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694555/" ref="ordinalpos=991&amp;ncbi_uid=1936001&amp;link_uid=PMC2694555">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>992.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3082280/figure/F1/" ref="ordinalpos=992&amp;ncbi_uid=2905990&amp;link_uid=PMC3082280" href="/pmc/articles/PMC3082280/"><img src="/pmc/articles/instance/3082280/bin/nihms-288089-f0001.gif" src-large="/pmc/articles/instance/3082280/bin/nihms-288089-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3082280/" ref="ordinalpos=992&amp;ncbi_uid=2905990&amp;link_uid=PMC3082280" image-link="/pmc/articles/PMC3082280/figure/F1/" class="imagepopup">Figure 1.  From: Canonical and non-canonical JAK-STAT <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Canonical JAK–STAT <span class="highlight" style="background-color:">signaling</span>. A simplified schematic representation of the <i>Drosophila</i> JAK–STAT <span class="highlight" style="background-color:">pathway</span>, which is typical of the canonical JAK–STAT <span class="highlight" style="background-color:">pathway</span>, in which unphosphorylated STAT resides in the cytoplasm. JAK activates STAT by phosphorylation, leading to nuclear translocation of dimerized phospho-STAT, which functions as a transcription factor. Names of the <i>Drosophila</i> homologs are in parentheses.</div></p></div><div class="aux"><div class="resc">Willis X. Li. Trends Cell Biol. ;18(11):545-551.</div><p class="links"><a class="dblinks" href="/pubmed/18848449" ref="ordinalpos=992&amp;ncbi_uid=2905990&amp;link_uid=18848449">Citation</a><a class="dblinks" href="/pmc/articles/PMC3082280/" ref="ordinalpos=992&amp;ncbi_uid=2905990&amp;link_uid=PMC3082280">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>993.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4312599/figure/Fig5/" ref="ordinalpos=993&amp;ncbi_uid=7118284&amp;link_uid=PMC4312599" href="/pmc/articles/PMC4312599/"><img src="/pmc/articles/instance/4312599/bin/12985_2014_236_Fig5_HTML.gif" src-large="/pmc/articles/instance/4312599/bin/12985_2014_236_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4312599/" ref="ordinalpos=993&amp;ncbi_uid=7118284&amp;link_uid=PMC4312599" image-link="/pmc/articles/PMC4312599/figure/Fig5/" class="imagepopup">Figure 5.  From: Differential regulation of cytotoxicity <span class="highlight" style="background-color:">pathway</span> discriminating between HIV, HCV mono- and co-infection identified by transcriptome profiling of PBMCs. </a></p><div class="supp"><p class="details"><div> <b>Coordinately up-regulated genes of WNT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HCV versus HIV.</b> The <span class="highlight" style="background-color:">pathway</span> figure is adapted from Kyoto Encyclopedia of Genes and Genomes (KEGG; http://www.genome.jp/kegg/). The proteins encoded by the coordinately up-regulated genes in the HCV group are highlighted in red.</div></p></div><div class="aux"><div class="resc">Jing Qin Wu, et al. Virol J. 2015;12(1):4.</div><p class="links"><a class="dblinks" href="/pubmed/25623235" ref="ordinalpos=993&amp;ncbi_uid=7118284&amp;link_uid=25623235">Citation</a><a class="dblinks" href="/pmc/articles/PMC4312599/" ref="ordinalpos=993&amp;ncbi_uid=7118284&amp;link_uid=PMC4312599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>994.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4163447/figure/fig4/" ref="ordinalpos=994&amp;ncbi_uid=6636436&amp;link_uid=PMC4163447" href="/pmc/articles/PMC4163447/"><img src="/pmc/articles/instance/4163447/bin/BMRI2014-236385.004.gif" src-large="/pmc/articles/instance/4163447/bin/BMRI2014-236385.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4163447/" ref="ordinalpos=994&amp;ncbi_uid=6636436&amp;link_uid=PMC4163447" image-link="/pmc/articles/PMC4163447/figure/fig4/" class="imagepopup">Figure 4.  From: MicroRNA Dysregulation in Liver and Pancreas of CMP-Neu5Ac Hydroxylase Null Mice Disrupts Insulin/PI3K-AKT <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Identification of target genes of dysregulated miRNAs in<i> Cmah</i>-null mice. (a) RT-qPCR analysis was performed using randomly selected genes (<i>Mapk3</i>,<i> Raf1</i>, and<i> Ccnd2</i>), which involved the insulin/PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Expression of mRNAs in livers (left) and pancreas (right) of control and<i> Cmah</i>-null mice were downregulated (*<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01). (b) Dysregulated miRNAs and their target genes which involved in insulin/PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in liver and pancreas of<i> Cmah</i>-null mouse. Red and blue present the refined pathways and putative miRNAs, respectively. Black indicates putative target genes for their <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Deug-Nam Kwon, et al. Biomed Res Int. 2014;2014:236385.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=994&amp;ncbi_uid=6636436&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4163447/" ref="ordinalpos=994&amp;ncbi_uid=6636436&amp;link_uid=PMC4163447">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>995.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3776718/figure/f3-ijmm-32-02-0281/" ref="ordinalpos=995&amp;ncbi_uid=5322923&amp;link_uid=PMC3776718" href="/pmc/articles/PMC3776718/"><img src="/pmc/articles/instance/3776718/bin/IJMM-32-02-0281-g02.gif" src-large="/pmc/articles/instance/3776718/bin/IJMM-32-02-0281-g02.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3776718/" ref="ordinalpos=995&amp;ncbi_uid=5322923&amp;link_uid=PMC3776718" image-link="/pmc/articles/PMC3776718/figure/f3-ijmm-32-02-0281/" class="imagepopup">Figure 3.  From: Genome-wide analysis of murine bone marrow-derived very small embryonic-like stem cells reveals that mitogenic growth factor <span class="highlight" style="background-color:">signaling</span> pathways play a crucial role in the quiescence and ageing of these cells. </a></p><div class="supp"><p class="details"><div>Overlay of the insulin-like growth factor-1 (IGF-1) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with microarray data for comparison of very small embryonic-like stem cells (VSELs) with hematopoietic stem cells (HSCs). Relative expression is depicted as fold-change in gene expression. Fold-changes in gene expression between VSELs and HSCs are shown below the indicated genes. The canonical IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was built using ingenuity <span class="highlight" style="background-color:">pathway</span> analysis (IPA) software version 8.7.</div></p></div><div class="aux"><div class="resc">KATARZYNA MIERZEJEWSKA, et al. Int J Mol Med. 2013 August;32(2):281-290.</div><p class="links"><a class="dblinks" href="/pubmed/23708325" ref="ordinalpos=995&amp;ncbi_uid=5322923&amp;link_uid=23708325">Citation</a><a class="dblinks" href="/pmc/articles/PMC3776718/" ref="ordinalpos=995&amp;ncbi_uid=5322923&amp;link_uid=PMC3776718">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>996.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3493431/figure/F4/" ref="ordinalpos=996&amp;ncbi_uid=5434583&amp;link_uid=PMC3493431" href="/pmc/articles/PMC3493431/"><img src="/pmc/articles/instance/3493431/bin/cbt-13-1244-g4.gif" src-large="/pmc/articles/instance/3493431/bin/cbt-13-1244-g4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3493431/" ref="ordinalpos=996&amp;ncbi_uid=5434583&amp;link_uid=PMC3493431" image-link="/pmc/articles/PMC3493431/figure/F4/" class="imagepopup">Figure. From: Oridonin in combination with imatinib exerts synergetic anti-leukemia effect in Ph+ acute lymphoblastic leukemia cells in vitro by inhibiting activation of LYN/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 4.</b> Effects of imatinib alone or in combination with oridonin on growth-<span class="highlight" style="background-color:">signaling</span> pathways in SUP-B15 cells. The SUP-B15 cells were incubated with 1μM imtinib for 6, 12, 24 h, or with 1 μM imatinib plus 3 μM oridonin for 24 h. They were then harvested and total proteins were extracted. Equal amounts of protein from each sample were separated on SDS-PAGE and immunoblotted with indicated antibodies, GAPDH was used as a loading control. Imatinib upregulated Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (A), inhibited constitutively activation of MEK, ERK <span class="highlight" style="background-color:">signaling</span> proteins (C) and STAT5 <span class="highlight" style="background-color:">signaling</span> (E) and had no significant effect on RAF and LYN <span class="highlight" style="background-color:">signaling</span> proteins (C,E). Oridonin overcame the upregulation of Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by imatinib. (B), Oridonin plus imatinib had no synergetic inhibitory effects on activation of MEK, ERK (D) and STAT5 (F) <span class="highlight" style="background-color:">signaling</span> proteins. The combination of oridonin and imatinib exerted significant inhibitory effect on activation of LYN tyrosine kinase (F). The results shown were representative of two independent experiments.</div></p></div><div class="aux"><div class="resc">Yong Guo, et al. Cancer Biol Ther. 2012 November 1;13(13):1244-1254.</div><p class="links"><a class="dblinks" href="/pubmed/22895079" ref="ordinalpos=996&amp;ncbi_uid=5434583&amp;link_uid=22895079">Citation</a><a class="dblinks" href="/pmc/articles/PMC3493431/" ref="ordinalpos=996&amp;ncbi_uid=5434583&amp;link_uid=PMC3493431">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>997.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3444627/figure/F1/" ref="ordinalpos=997&amp;ncbi_uid=5441058&amp;link_uid=PMC3444627" href="/pmc/articles/PMC3444627/"><img src="/pmc/articles/instance/3444627/bin/nihms-402572-f0001.gif" src-large="/pmc/articles/instance/3444627/bin/nihms-402572-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3444627/" ref="ordinalpos=997&amp;ncbi_uid=5441058&amp;link_uid=PMC3444627" image-link="/pmc/articles/PMC3444627/figure/F1/" class="imagepopup">Fig. 1. Schematic illustration of intracellular and intercellular <span class="highlight" style="background-color:">signaling</span> and cellular dynamics in bone healing and bone remodeling.  From: Cytokine Combination Therapy Prediction for Bone Remodeling in Tissue Engineering Based on the Intracellular <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Bone regeneration or bone remodeling involves bone resorption by osteoclasts (OC) and the following bone formation by osteoblasts (OB) within basic multicellular units (BMU). Three cytokines were considered: TGFβ, Wnt and BMP2. Intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consists of Smad2/3, Smad 1/5, βCatenin, and Runx2 and Osx. Runx2 can promote the differentiation of mesenchymal stem cells (MSCs) into pre-osteoblasts (OBp) and can inhibit the differentiation of pre-osteoblasts into active osteoblasts (OBa). Osx also play a promoting role in the later stage of osteoblastic lineage which interacts with osteoclastic lineage through intercellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> RANK-RANKL OPG.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Sun, et al. Biomaterials. 2012 November;33(33):8265-8276.</div><p class="links"><a class="dblinks" href="/pubmed/22910219" ref="ordinalpos=997&amp;ncbi_uid=5441058&amp;link_uid=22910219">Citation</a><a class="dblinks" href="/pmc/articles/PMC3444627/" ref="ordinalpos=997&amp;ncbi_uid=5441058&amp;link_uid=PMC3444627">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>998.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2515871/figure/fig2/" ref="ordinalpos=998&amp;ncbi_uid=1761928&amp;link_uid=PMC2515871" href="/pmc/articles/PMC2515871/"><img src="/pmc/articles/instance/2638722/bin/molbiolevolmsn138f02_ht.gif" src-large="/pmc/articles/instance/2638722/bin/molbiolevolmsn138f02_ht.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;IG&lt;/span&gt; . 2.—" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2515871/" ref="ordinalpos=998&amp;ncbi_uid=1761928&amp;link_uid=PMC2515871" image-link="/pmc/articles/PMC2515871/figure/fig2/" class="imagepopup">F IG . 2.—.  From: Roles of cis- and trans-Changes in the Regulatory Evolution of Genes in the Gluconeogenic <span class="highlight" style="background-color:">Pathway</span> in Yeast. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in the regulation of gluconeogenesis by carbon source-responsive element-binding factors, Cat8p and Sip4p. Adapted from Schuller (2003).</div></p></div><div class="aux"><div class="resc">Ya-Wen Chang, et al. Mol Biol Evol. 2008 September;25(9):1863-1875.</div><p class="links"><a class="dblinks" href="/pubmed/18573843" ref="ordinalpos=998&amp;ncbi_uid=1761928&amp;link_uid=18573843">Citation</a><a class="dblinks" href="/pmc/articles/PMC2515871/" ref="ordinalpos=998&amp;ncbi_uid=1761928&amp;link_uid=PMC2515871">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>999.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3487967/figure/F8/" ref="ordinalpos=999&amp;ncbi_uid=4353695&amp;link_uid=PMC3487967" href="/pmc/articles/PMC3487967/"><img src="/pmc/articles/instance/3487967/bin/1471-2105-13-218-8.gif" src-large="/pmc/articles/instance/3487967/bin/1471-2105-13-218-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3487967/" ref="ordinalpos=999&amp;ncbi_uid=4353695&amp;link_uid=PMC3487967" image-link="/pmc/articles/PMC3487967/figure/F8/" class="imagepopup">Figure 8.  From: Multi-scale agent-based brain cancer modeling and prediction of TKI treatment response: Incorporating EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and angiogenesis. </a></p><div class="supp"><p class="details"><div><b>Sensitivity analysis of EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> The result shows that the migration potential of brain cancer cells is most sensitive to the initial concentration of EGFR (<i>X</i><sub>2</sub>) and TGFα (<i>X</i><sub>1</sub>). The result also shows us forward kinetic rates (<i>k</i><sub>1</sub>, <i>k</i><sub>2</sub>, <i>k</i><sub>3</sub>, <i>k</i><sub>5</sub>) are more important in the system and demonstrate that the developed intracellular <span class="highlight" style="background-color:">pathway</span> system is robust.</div></p></div><div class="aux"><div class="resc">Xiaoqiang Sun, et al. BMC Bioinformatics. 2012;13:218-218.</div><p class="links"><a class="dblinks" href="/pubmed/22935054" ref="ordinalpos=999&amp;ncbi_uid=4353695&amp;link_uid=22935054">Citation</a><a class="dblinks" href="/pmc/articles/PMC3487967/" ref="ordinalpos=999&amp;ncbi_uid=4353695&amp;link_uid=PMC3487967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1000.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2879143/figure/fig04/" ref="ordinalpos=1000&amp;ncbi_uid=2513022&amp;link_uid=PMC2879143" href="/pmc/articles/PMC2879143/"><img src="/pmc/articles/instance/2952290/bin/pmic0010-1316-f4.gif" src-large="/pmc/articles/instance/2952290/bin/pmic0010-1316-f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2879143/" ref="ordinalpos=1000&amp;ncbi_uid=2513022&amp;link_uid=PMC2879143" image-link="/pmc/articles/PMC2879143/figure/fig04/" class="imagepopup">Figure 4.  From: <span class="highlight" style="background-color:">Pathway</span> analysis of dilated cardiomyopathy using global proteomic profiling and enrichment maps. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> cluster and integrin <span class="highlight" style="background-color:">signaling</span>. Zoom-in of the Enrichment Map gene set cluster representing <span class="highlight" style="background-color:">signaling</span> pathways enriched at the early and mid-stages of heart failure. A summary description of the cluster was visualized as a “term cloud” using Wordle (http://www.wordle.net/) derived from the text descriptions of all gene sets. Term size indicates its frequency; thus, large terms best summarize the cluster (<i>i.e</i>. <span class="highlight" style="background-color:">signaling</span> pathways). Specific terms related to the integrin <span class="highlight" style="background-color:">pathway</span> are highlighted within this cluster and in the network.</div></p></div><div class="aux"><div class="resc">Ruth Isserlin, et al. Proteomics. 2010 March;10(6):1316-1327.</div><p class="links"><a class="dblinks" href="/pubmed/20127684" ref="ordinalpos=1000&amp;ncbi_uid=2513022&amp;link_uid=20127684">Citation</a><a class="dblinks" href="/pmc/articles/PMC2879143/" ref="ordinalpos=1000&amp;ncbi_uid=2513022&amp;link_uid=PMC2879143">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="9" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="10" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="11" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:22:15-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC51F353FEBE210000000000313357;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC51F353FEBE210000000000313357" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>