
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="27" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB0F6E543102210000000000A60676" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="26" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="27" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="28" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="27" /></div>
            <div><h2 class="result_count">Results: 2601 to 2700 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2601.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137639/figure/F5/" ref="ordinalpos=2601&amp;ncbi_uid=3087799&amp;link_uid=PMC3137639" href="/pmc/articles/PMC3137639/"><img src="/pmc/articles/instance/3137639/bin/nihms306822f5.gif" src-large="/pmc/articles/instance/3137639/bin/nihms306822f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137639/" ref="ordinalpos=2601&amp;ncbi_uid=3087799&amp;link_uid=PMC3137639" image-link="/pmc/articles/PMC3137639/figure/F5/" class="imagepopup">Figure 5. Connections iidentified using RTKIs and <span class="highlight" style="background-color:">pathway</span> specific inhibitors.  From: Akt-RSK-S6-kinase <span class="highlight" style="background-color:">Signaling</span> Networks Activated by Oncogenic Receptor Tyrosine Kinases. </a></p><div class="supp"><p class="details"><div>Proteins inhibited by two or more RTKIs and the indicated <span class="highlight" style="background-color:">pathway</span> inhibitor(s) are grouped by protein function. Color coding indicates sensitivity to <span class="highlight" style="background-color:">pathway</span> inhibitor.</div></p></div><div class="aux"><div class="resc">Albrecht Moritz, et al. Sci Signal. ;3(136):ra64-ra64.</div><p class="links"><a class="dblinks" href="/pubmed/20736484" ref="ordinalpos=2601&amp;ncbi_uid=3087799&amp;link_uid=20736484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137639/" ref="ordinalpos=2601&amp;ncbi_uid=3087799&amp;link_uid=PMC3137639">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2602.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2891200/figure/F3/" ref="ordinalpos=2602&amp;ncbi_uid=2553872&amp;link_uid=PMC2891200" href="/pmc/articles/PMC2891200/"><img src="/pmc/articles/instance/2891200/bin/nihms108036f3.gif" src-large="/pmc/articles/instance/2891200/bin/nihms108036f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2891200/" ref="ordinalpos=2602&amp;ncbi_uid=2553872&amp;link_uid=PMC2891200" image-link="/pmc/articles/PMC2891200/figure/F3/" class="imagepopup">Figure 3.  From: Targeting Eicosanoids <span class="highlight" style="background-color:">Pathway</span> in Non-Small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div>The increased understanding that selective COX-2 inhibitors are not free of unwanted side effects (particularly on renal and cardiovascular systems) have led to a reevaluation of other potential targets in the PG synthesis <span class="highlight" style="background-color:">pathway</span> downstream of COX for treatment and/or prevention of lung cancers, including PGE synthases and 15-PGDH. Another attractive target for inhibition of the activity of the PG synthesis <span class="highlight" style="background-color:">pathway</span> is inhibition of downstream receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Leora Horn, et al. Expert Opin Ther Targets. ;13(6):675-688.</div><p class="links"><a class="dblinks" href="/pubmed/19409031" ref="ordinalpos=2602&amp;ncbi_uid=2553872&amp;link_uid=19409031">Citation</a><a class="dblinks" href="/pmc/articles/PMC2891200/" ref="ordinalpos=2602&amp;ncbi_uid=2553872&amp;link_uid=PMC2891200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2603.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065864/figure/F4/" ref="ordinalpos=2603&amp;ncbi_uid=6283587&amp;link_uid=PMC4065864" href="/pmc/articles/PMC4065864/"><img src="/pmc/articles/instance/4065864/bin/1741-7015-12-103-4.gif" src-large="/pmc/articles/instance/4065864/bin/1741-7015-12-103-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065864/" ref="ordinalpos=2603&amp;ncbi_uid=6283587&amp;link_uid=PMC4065864" image-link="/pmc/articles/PMC4065864/figure/F4/" class="imagepopup">Figure 4.  From: Effects of palmitate on genome-wide mRNA expression and DNA methylation patterns in human pancreatic islets. </a></p><div class="supp"><p class="details"><div><b>Differential mRNA expression of genes in enriched KEGG pathways in palmitate-exposed human islets. a)</b> Differentially down-regulated genes in the glycolysis/gluconeogenesis <span class="highlight" style="background-color:">pathway</span>. <b>b)</b> Differentially expressed genes in the glutathione metabolism <span class="highlight" style="background-color:">pathway</span>. <b>c)</b> Differentially expressed genes in the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>d)</b> Differentially expressed genes in the biosynthesis of unsaturated fatty acids <span class="highlight" style="background-color:">pathway</span>. All data are presented as meansem. *<i>q</i> &lt;0.05.</div></p></div><div class="aux"><div class="resc">Elin Hall, et al. BMC Med. 2014;12:103-103.</div><p class="links"><a class="dblinks" href="/pubmed/24953961" ref="ordinalpos=2603&amp;ncbi_uid=6283587&amp;link_uid=24953961">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065864/" ref="ordinalpos=2603&amp;ncbi_uid=6283587&amp;link_uid=PMC4065864">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2604.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3448082/figure/nutrients-04-00967-f001/" ref="ordinalpos=2604&amp;ncbi_uid=4252921&amp;link_uid=PMC3448082" href="/pmc/articles/PMC3448082/"><img src="/pmc/articles/instance/3448082/bin/nutrients-04-00967-g001.gif" src-large="/pmc/articles/instance/3448082/bin/nutrients-04-00967-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3448082/" ref="ordinalpos=2604&amp;ncbi_uid=4252921&amp;link_uid=PMC3448082" image-link="/pmc/articles/PMC3448082/figure/nutrients-04-00967-f001/" class="imagepopup">Figure 1.  From: The Role of Ghrelin, Salivary Secretions, and Dental Care in Eating Disorders. </a></p><div class="supp"><p class="details"><div>A simplified model of the feeding regulation hypothesis involving the hypothalamic neuropeptide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Leptin, insulin, CCK, and PYY stimulate an anorexigenic <span class="highlight" style="background-color:">pathway</span> and inhibit an orexigenic <span class="highlight" style="background-color:">pathway</span>. The effect of ghrelin in the hypothalamus is opposite to that of leptin, insulin, CCK, and PYY. Ghrelin stimulates an orexigenic <span class="highlight" style="background-color:">pathway</span> and inhibits an anorexigenic <span class="highlight" style="background-color:">pathway</span>. Together, this system functions to regulate food intake to optimize metabolic energy balance. NPY: neuropeptide Y; POMC: proopiomelanocortin; AgRP: agouti-related protein; CART: cocaine- and amphetamine-regulated transcript; ARC: arcuate nucleus; PVN: paraventricular nucleus; MCH: melanin-concentrating hormone; LH: lateral hypothalamus; CRF: corticotrophin-releasing factor.</div></p></div><div class="aux"><div class="resc">Takakazu Yagi, et al. Nutrients. 2012 August;4(8):967-989.</div><p class="links"><a class="dblinks" href="/pubmed/23016127" ref="ordinalpos=2604&amp;ncbi_uid=4252921&amp;link_uid=23016127">Citation</a><a class="dblinks" href="/pmc/articles/PMC3448082/" ref="ordinalpos=2604&amp;ncbi_uid=4252921&amp;link_uid=PMC3448082">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2605.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3350500/figure/pone-0037262-g006/" ref="ordinalpos=2605&amp;ncbi_uid=4008447&amp;link_uid=PMC3350500" href="/pmc/articles/PMC3350500/"><img src="/pmc/articles/instance/3350500/bin/pone.0037262.g006.gif" src-large="/pmc/articles/instance/3350500/bin/pone.0037262.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3350500/" ref="ordinalpos=2605&amp;ncbi_uid=4008447&amp;link_uid=PMC3350500" image-link="/pmc/articles/PMC3350500/figure/pone-0037262-g006/" class="imagepopup">Figure 6. IGF <span class="highlight" style="background-color:">signaling</span> is activated in emerin-null myogenic progenitors..  From: Loss of Emerin Alters Myogenic <span class="highlight" style="background-color:">Signaling</span> and miRNA Expression in Mouse Myogenic Progenitors. </a></p><div class="supp"><p class="details"><div>Protein expression of p38 and phosphorylated p38 in emerin-null myogenic progenitors was monitored by western blotting (A) and quantitated by densitometry normalized to -tubulin (B) or total p38 (C) levels in wildtype progenitors. Antibodies specific for p38 (B) or phosphorylated p38 (C) were used to monitor activation of the IGF-1 <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Adam J. Koch, et al. PLoS One. 2012;7(5):e37262.</div><p class="links"><a class="dblinks" href="/pubmed/22606356" ref="ordinalpos=2605&amp;ncbi_uid=4008447&amp;link_uid=22606356">Citation</a><a class="dblinks" href="/pmc/articles/PMC3350500/" ref="ordinalpos=2605&amp;ncbi_uid=4008447&amp;link_uid=PMC3350500">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2606.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878841/figure/F2/" ref="ordinalpos=2606&amp;ncbi_uid=3082254&amp;link_uid=PMC2878841" href="/pmc/articles/PMC2878841/"><img src="/pmc/articles/instance/2878841/bin/nihms200227f2.gif" src-large="/pmc/articles/instance/2878841/bin/nihms200227f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878841/" ref="ordinalpos=2606&amp;ncbi_uid=3082254&amp;link_uid=PMC2878841" image-link="/pmc/articles/PMC2878841/figure/F2/" class="imagepopup">Fig. 2.  From: Disruption of IGF-1R <span class="highlight" style="background-color:">signaling</span> increases TRAIL-induced apoptosis: a new potential therapy for the treatment of melanoma. </a></p><div class="supp"><p class="details"><div>Effects of PPP on IGF-1-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in melanoma cells. AC. Western blot analysis was performed 424 h after treatment of WM35 and WM9 or LU1205 and WM793 melanoma cells with indicated doses of PPP. Analyzed proteins are indicated.</div></p></div><div class="aux"><div class="resc">Thomas B. Karasic, et al. Exp Cell Res. ;316(12):1994-2007.</div><p class="links"><a class="dblinks" href="/pubmed/20417200" ref="ordinalpos=2606&amp;ncbi_uid=3082254&amp;link_uid=20417200">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878841/" ref="ordinalpos=2606&amp;ncbi_uid=3082254&amp;link_uid=PMC2878841">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2607.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2660380/figure/F3/" ref="ordinalpos=2607&amp;ncbi_uid=1733995&amp;link_uid=PMC2660380" href="/pmc/articles/PMC2660380/"><img src="/pmc/articles/instance/2660380/bin/nihms96238f3.gif" src-large="/pmc/articles/instance/2660380/bin/nihms96238f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2660380/" ref="ordinalpos=2607&amp;ncbi_uid=1733995&amp;link_uid=PMC2660380" image-link="/pmc/articles/PMC2660380/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> pathways of LPA-induced transactivation of EGF-R.  From: Lysophosphatidic Acid <span class="highlight" style="background-color:">Signaling</span> in Airway Epithelium: Role in Airway Inflammation and Remodeling. </a></p><div class="supp"><p class="details"><div>Ligation of LPA to its G-protein-coupled receptors induces phosphorylation of EGF-R through PKC, Lyn kinase, MMP, and pro-HB-EGF <span class="highlight" style="background-color:">pathway</span>. PLD2 and PKC are involved in LPA-induced transactivation of EGF-R; however, the relationship between PLD2 and PKC or between PKC and Lyn kinase are unclear. Erk1/2 and C/EBP are downstream targets of EGF-R transactivation, which regulate cell proliferation and gene expression.</div></p></div><div class="aux"><div class="resc">Yutong Zhao, et al. Cell Signal. ;21(3):367-377.</div><p class="links"><a class="dblinks" href="/pubmed/18996473" ref="ordinalpos=2607&amp;ncbi_uid=1733995&amp;link_uid=18996473">Citation</a><a class="dblinks" href="/pmc/articles/PMC2660380/" ref="ordinalpos=2607&amp;ncbi_uid=1733995&amp;link_uid=PMC2660380">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2608.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3436582/figure/F10/" ref="ordinalpos=2608&amp;ncbi_uid=5202673&amp;link_uid=PMC3436582" href="/pmc/articles/PMC3436582/"><img src="/pmc/articles/instance/3436582/bin/zbc0371221070010.gif" src-large="/pmc/articles/instance/3436582/bin/zbc0371221070010.jpg" alt="FIGURE 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3436582/" ref="ordinalpos=2608&amp;ncbi_uid=5202673&amp;link_uid=PMC3436582" image-link="/pmc/articles/PMC3436582/figure/F10/" class="imagepopup">FIGURE 10. From: Angiotensin-II and MARCKS. </a></p><div class="supp"><p class="details"><div><b>Model of Ang-II <span class="highlight" style="background-color:">signaling</span> to the endothelial cytoskeleton via MARCKS and H<sub>2</sub>O<sub>2</sub>.</b> This scheme shows that the Ang-II-dependent activation of the AT1R receptor in vascular endothelila cells leads to Rac1 activation and to increased levels of H<sub>2</sub>O<sub>2</sub>, both of which are required for MARCKS phosphorylation and cytoskeleton responses. The AT1R blocker losartan blocks downstream Ang-II-stimulated <span class="highlight" style="background-color:">signaling</span> responses. siRNA-mediated knockdown of Rac1 attenuates Ang-II-stimulated MARCKS phosphorylation and translocation, as does the enzymatic degradation of H<sub>2</sub>O<sub>2</sub> after PEG-catalase treatment. MARCKS phosphorylation leads to increased levels of free PIP<sub>2</sub> and to the translocation of phospho-MARCKS from the membrane to internal sites, where phosphorylated MARCKS binds actin and promotes formation of adhesion complexes containing vinculin. siRNA-mediated knockdown of cAbl attenuates MARCKS phosphorylation, implicating this protein kinase in the <span class="highlight" style="background-color:">pathway</span> modulating MARCKS phosphorylation. Conversely, siRNA-mediated knockdown of MARCKS leads to an attenuation of c-Abl phosphorylation, suggesting reciprocal regulatory interactions between these two <span class="highlight" style="background-color:">signaling</span> proteins. Taken together, these observations help to establish a central role for MARCKS in Ang-II <span class="highlight" style="background-color:">signaling</span> in the vascular endothelium.</div></p></div><div class="aux"><div class="resc">Hermann Kalwa, et al. J Biol Chem. 2012 August 17;287(34):29147-29158.</div><p class="links"><a class="dblinks" href="/pubmed/22773836" ref="ordinalpos=2608&amp;ncbi_uid=5202673&amp;link_uid=22773836">Citation</a><a class="dblinks" href="/pmc/articles/PMC3436582/" ref="ordinalpos=2608&amp;ncbi_uid=5202673&amp;link_uid=PMC3436582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2609.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4266699/figure/fig01/" ref="ordinalpos=2609&amp;ncbi_uid=6969047&amp;link_uid=PMC4266699" href="/pmc/articles/PMC4266699/"><img src="/pmc/articles/instance/4266699/bin/ajcr0004-0608-f1.gif" src-large="/pmc/articles/instance/4266699/bin/ajcr0004-0608-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4266699/" ref="ordinalpos=2609&amp;ncbi_uid=6969047&amp;link_uid=PMC4266699" image-link="/pmc/articles/PMC4266699/figure/fig01/" class="imagepopup">Figure 1.  From: Redundant kinase activation and resistance of EGFR-tyrosine kinase inhibitors. </a></p><div class="supp"><p class="details"><div>HER family and EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ligand-bound receptors form functionally active homodimers or heterodimers, resulting in the activation of downstream <span class="highlight" style="background-color:">signaling</span> pathways such as PI3K/AKT, RAS/RAF/MAPK, PLC/PKC and JAK/STAT <span class="highlight" style="background-color:">pathway</span>, leading to cell proliferation, invasion, metastasis, survival and angiogenesis.</div></p></div><div class="aux"><div class="resc">Min Luo, et al. Am J Cancer Res. 2014;4(6):608-628.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2609&amp;ncbi_uid=6969047&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4266699/" ref="ordinalpos=2609&amp;ncbi_uid=6969047&amp;link_uid=PMC4266699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2610.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3928842/figure/F1/" ref="ordinalpos=2610&amp;ncbi_uid=5877475&amp;link_uid=PMC3928842" href="/pmc/articles/PMC3928842/"><img src="/pmc/articles/instance/3928842/bin/ABR-3-1-g002.gif" src-large="/pmc/articles/instance/3928842/bin/ABR-3-1-g002.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3928842/" ref="ordinalpos=2610&amp;ncbi_uid=5877475&amp;link_uid=PMC3928842" image-link="/pmc/articles/PMC3928842/figure/F1/" class="imagepopup">Figure 1.  From: Role of the innate immunity in female reproductive tract. </a></p><div class="supp"><p class="details"><div>TLRs <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Fatemehsadat Amjadi, et al. Adv Biomed Res. 2014;3:1.</div><p class="links"><a class="dblinks" href="/pubmed/24592358" ref="ordinalpos=2610&amp;ncbi_uid=5877475&amp;link_uid=24592358">Citation</a><a class="dblinks" href="/pmc/articles/PMC3928842/" ref="ordinalpos=2610&amp;ncbi_uid=5877475&amp;link_uid=PMC3928842">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2611.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3702373/figure/F1/" ref="ordinalpos=2611&amp;ncbi_uid=5861564&amp;link_uid=PMC3702373" href="/pmc/articles/PMC3702373/"><img src="/pmc/articles/instance/3702373/bin/nihms436358f1.gif" src-large="/pmc/articles/instance/3702373/bin/nihms436358f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3702373/" ref="ordinalpos=2611&amp;ncbi_uid=5861564&amp;link_uid=PMC3702373" image-link="/pmc/articles/PMC3702373/figure/F1/" class="imagepopup">Figure 1. PD-1 <span class="highlight" style="background-color:">signaling</span> in inflamed tissue.  From: Antagonist Antibodies to PD-1 and B7-H1 (PD-L1) in the Treatment of Advanced Human Cancer. </a></p><div class="supp"><p class="details"><div>Ligation of T-cell PD-1 by B7-H1 on APC leads to the formation of PD-1/TCR microclusters and the subsequent recruitment of SHP2 phosphatases, which dephosphorylate multiple members of the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This abrogates downstream effects of T-cell activation, including cytokine production, cell-cycle progression, and the expression of survival proteins. By preventing T-cell activation, PD-1 <span class="highlight" style="background-color:">signaling</span> contributes to the maintenance of tolerance to self-antigens and prevents immune-mediated damage of healthy tissue during the resolution of infection and other inflammatory responses.</div></p></div><div class="aux"><div class="resc">Mario Sznol, et al. Clin Cancer Res. ;19(5):1021-1034.</div><p class="links"><a class="dblinks" href="/pubmed/23460533" ref="ordinalpos=2611&amp;ncbi_uid=5861564&amp;link_uid=23460533">Citation</a><a class="dblinks" href="/pmc/articles/PMC3702373/" ref="ordinalpos=2611&amp;ncbi_uid=5861564&amp;link_uid=PMC3702373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2612.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3561807/figure/F9/" ref="ordinalpos=2612&amp;ncbi_uid=4592154&amp;link_uid=PMC3561807" href="/pmc/articles/PMC3561807/"><img src="/pmc/articles/instance/3561807/bin/JCI63324.f9.gif" src-large="/pmc/articles/instance/3561807/bin/JCI63324.f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3561807/" ref="ordinalpos=2612&amp;ncbi_uid=4592154&amp;link_uid=PMC3561807" image-link="/pmc/articles/PMC3561807/figure/F9/" class="imagepopup">Figure 9. A vicious circle of stress <span class="highlight" style="background-color:">signaling</span> in prostate cancer..  From: Behavioral stress accelerates prostate cancer development in mice. </a></p><div class="supp"><p class="details"><div>Psychoemotional stress activates the hypothalamic-pituitary-adrenal axis (HPA); as a result, blood levels of adrenaline increase and activate ADRB2/PKA/BAD antiapoptotic <span class="highlight" style="background-color:">signaling</span> in prostate cells. Constitutive activation of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in stressed patients (Supplemental Figure 12) may trigger a vicious circle: stress and anxiety from a diagnosis of prostate cancer increase adrenaline levels that in turn reduce efficacy of anticancer treatments.</div></p></div><div class="aux"><div class="resc">Sazzad Hassan, et al. J Clin Invest. 2013 February 1;123(2):874-886.</div><p class="links"><a class="dblinks" href="/pubmed/23348742" ref="ordinalpos=2612&amp;ncbi_uid=4592154&amp;link_uid=23348742">Citation</a><a class="dblinks" href="/pmc/articles/PMC3561807/" ref="ordinalpos=2612&amp;ncbi_uid=4592154&amp;link_uid=PMC3561807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2613.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3323931/figure/F1/" ref="ordinalpos=2613&amp;ncbi_uid=3900902&amp;link_uid=PMC3323931" href="/pmc/articles/PMC3323931/"><img src="/pmc/articles/instance/3323931/bin/tlo0502_0098_fig001.gif" src-large="/pmc/articles/instance/3323931/bin/tlo0502_0098_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3323931/" ref="ordinalpos=2613&amp;ncbi_uid=3900902&amp;link_uid=PMC3323931" image-link="/pmc/articles/PMC3323931/figure/F1/" class="imagepopup">Figure 1.  From: uPA and PAI-1-Related <span class="highlight" style="background-color:">Signaling</span> Pathways Differ between Primary Breast Cancers and Lymph Node Metastases. </a></p><div class="supp"><p class="details"><div>Parallel analysis of FAK, ILK, PI3K, and ROCK expression in correlation to PAI-1 expression. A selection of seven representative tumors is shown in the figure. Normalized signal intensity of the different proteins obtained by RPPA analysis is depicted. All four proteins of the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are expressed at similar levels compared with PAI-1 expression in six of seven patients. For one patient (no. 186), a single protein (PI3K) is expressed in a different manner than the rest of the <span class="highlight" style="background-color:">pathway</span>. In only 4 of the analyzed 52 patients, a single protein was regulated in a different manner than the rest of the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Katharina Malinowsky, et al. Transl Oncol. 2012 April;5(2):98-104.</div><p class="links"><a class="dblinks" href="/pubmed/22496926" ref="ordinalpos=2613&amp;ncbi_uid=3900902&amp;link_uid=22496926">Citation</a><a class="dblinks" href="/pmc/articles/PMC3323931/" ref="ordinalpos=2613&amp;ncbi_uid=3900902&amp;link_uid=PMC3323931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2614.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923852/figure/F1/" ref="ordinalpos=2614&amp;ncbi_uid=3058570&amp;link_uid=PMC2923852" href="/pmc/articles/PMC2923852/"><img src="/pmc/articles/instance/2923852/bin/nihms-208591-f0001.gif" src-large="/pmc/articles/instance/2923852/bin/nihms-208591-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923852/" ref="ordinalpos=2614&amp;ncbi_uid=3058570&amp;link_uid=PMC2923852" image-link="/pmc/articles/PMC2923852/figure/F1/" class="imagepopup">Figure 1.  From: GnRH <span class="highlight" style="background-color:">signaling</span>, the gonadotrope and endocrine control of fertility. </a></p><div class="supp"><p class="details"><div>Model for GnRH receptor (GnRHR) <span class="highlight" style="background-color:">signaling</span> within the pituitary gonadotrope that leads to LH biosynthesis. The GnRHR resides within discrete plasma membrane compartments (membrane rafts) characterized by increased presence of cholesterol (green within membrane) and sphingolipids (orange within membranes). Activation of the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> occurs within the membrane rafts facilitated by local influx of extracellular calcium through voltage dependent calcium channels (VGCC).</div></p></div><div class="aux"><div class="resc">Stuart P. Bliss, et al. Front Neuroendocrinol. ;31(3):322-340.</div><p class="links"><a class="dblinks" href="/pubmed/20451543" ref="ordinalpos=2614&amp;ncbi_uid=3058570&amp;link_uid=20451543">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923852/" ref="ordinalpos=2614&amp;ncbi_uid=3058570&amp;link_uid=PMC2923852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2615.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2946928/figure/pone-0013058-g004/" ref="ordinalpos=2615&amp;ncbi_uid=2504831&amp;link_uid=PMC2946928" href="/pmc/articles/PMC2946928/"><img src="/pmc/articles/instance/2946928/bin/pone.0013058.g004.gif" src-large="/pmc/articles/instance/2946928/bin/pone.0013058.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2946928/" ref="ordinalpos=2615&amp;ncbi_uid=2504831&amp;link_uid=PMC2946928" image-link="/pmc/articles/PMC2946928/figure/pone-0013058-g004/" class="imagepopup">Figure 4. LA activates the protein kinase A (PKA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Lipoic Acid Attenuates Inflammation via cAMP and Protein Kinase A <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) T cells were stimulated with 10 g/ml LA for 10 min. Total protein was harvested and used for western blot analysis for Lck phosphorylation, which is a downstream effector of PKA <span class="highlight" style="background-color:">signaling</span>. The top blot is phospho-Lck and the bottom panel is Lck. (B) Graphical representation of densitometric analysis of band intensity showing the ratio of pLck/Lck. N=9. *indicates statistical significance compared to untreated control (p&lt;0.05).</div></p></div><div class="aux"><div class="resc">Sonemany Salinthone, et al. PLoS One. 2010;5(9):e13058.</div><p class="links"><a class="dblinks" href="/pubmed/20927401" ref="ordinalpos=2615&amp;ncbi_uid=2504831&amp;link_uid=20927401">Citation</a><a class="dblinks" href="/pmc/articles/PMC2946928/" ref="ordinalpos=2615&amp;ncbi_uid=2504831&amp;link_uid=PMC2946928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2616.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4026697/figure/F3/" ref="ordinalpos=2616&amp;ncbi_uid=6166026&amp;link_uid=PMC4026697" href="/pmc/articles/PMC4026697/"><img src="/pmc/articles/instance/4026697/bin/fendo-05-00072-g003.gif" src-large="/pmc/articles/instance/4026697/bin/fendo-05-00072-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4026697/" ref="ordinalpos=2616&amp;ncbi_uid=6166026&amp;link_uid=PMC4026697" image-link="/pmc/articles/PMC4026697/figure/F3/" class="imagepopup">Figure 3.  From: Venus Kinase Receptors: Prospects in <span class="highlight" style="background-color:">Signaling</span> and Biological Functions of These Invertebrate Kinases. </a></p><div class="supp"><p class="details"><div><b>VKR <span class="highlight" style="background-color:">signaling</span> pathways</b>. Closure of VFT extracellular domains upon binding of arginine ligand promotes and reinforces receptor dimerization inducing kinase activation and autophosphorylation of VKR. Phosphotyrosines can bind different partners for the transduction of conserved RTK <span class="highlight" style="background-color:">signaling</span> pathways, such as the PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span> involved in protein synthesis and the Ras/MAPK ERK <span class="highlight" style="background-color:">pathway</span> important for cell growth and proliferation. Moreover, activated VKRs could activate through the specific binding of Shb (SH2-containing protein), the alternative JNK <span class="highlight" style="background-color:">pathway</span> to contribute in concert with Rho1, MEK7, and PP2C to cytoskeleton rearrangement and oocyte maturation. Phosphorylation of AKT, p70S6K, ERK, and JNK (circled in red) was confirmed in VKR-expressing <i>Xenopus</i> oocytes (22).</div></p></div><div class="aux"><div class="resc">Colette Dissous, et al. Front Endocrinol (Lausanne). 2014;5:72.</div><p class="links"><a class="dblinks" href="/pubmed/24860549" ref="ordinalpos=2616&amp;ncbi_uid=6166026&amp;link_uid=24860549">Citation</a><a class="dblinks" href="/pmc/articles/PMC4026697/" ref="ordinalpos=2616&amp;ncbi_uid=6166026&amp;link_uid=PMC4026697">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2617.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3742814/figure/f4-ol-06-01-0075/" ref="ordinalpos=2617&amp;ncbi_uid=5197434&amp;link_uid=PMC3742814" href="/pmc/articles/PMC3742814/"><img src="/pmc/articles/instance/3742814/bin/OL-06-01-0075-g03.gif" src-large="/pmc/articles/instance/3742814/bin/OL-06-01-0075-g03.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3742814/" ref="ordinalpos=2617&amp;ncbi_uid=5197434&amp;link_uid=PMC3742814" image-link="/pmc/articles/PMC3742814/figure/f4-ol-06-01-0075/" class="imagepopup">Figure 4. From: Telomerase reverse transcriptase promotes the proliferation of human laryngeal carcinoma cells through activation of the activator protein 1. </a></p><div class="supp"><p class="details"><div>Effect of TERT overexpression with modulation of the p38 and ERK <span class="highlight" style="background-color:">signaling</span> pathways on c-Fos and c-Jun expression and activation in the HEp-2 laryngeal carcinoma cells. (A) Western blot analysis of the effect of TERT overexpression (Ad-sh-TERT) and silencing (Ad-Sh-TERT) on the p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and c-Fos and c-Jun expression in the presence and absence of the p38 inhibitor, SB202190. (B) Western blot analysis of the effect of TERT overexpression (Ad-sh-TERT) and silencing (Ad-Sh-TERT) on the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and c-Fos and c-Jun expression in the presence and absence of the ERK inhibitor, U0126.</div></p></div><div class="aux"><div class="resc">YANG JIANG, et al. Oncol Lett. 2013 July;6(1):75-80.</div><p class="links"><a class="dblinks" href="/pubmed/23946780" ref="ordinalpos=2617&amp;ncbi_uid=5197434&amp;link_uid=23946780">Citation</a><a class="dblinks" href="/pmc/articles/PMC3742814/" ref="ordinalpos=2617&amp;ncbi_uid=5197434&amp;link_uid=PMC3742814">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2618.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1472009/figure/f6/" ref="ordinalpos=2618&amp;ncbi_uid=677855&amp;link_uid=PMC1472009" href="/pmc/articles/PMC1472009/"><img src="/pmc/articles/instance/1472009/bin/zjv0090676890006.gif" src-large="/pmc/articles/instance/1472009/bin/zjv0090676890006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1472009/" ref="ordinalpos=2618&amp;ncbi_uid=677855&amp;link_uid=PMC1472009" image-link="/pmc/articles/PMC1472009/figure/f6/" class="imagepopup">FIG. 6. From: Hepatitis C Virus Core Protein Inhibits Tumor Necrosis Factor Alpha-Mediated Apoptosis by a Protective Effect Involving Cellular FLICE Inhibitory Protein. </a></p><div class="supp"><p class="details"><div>A schematic diagram depicting the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for TNF--mediated apoptotic cell death and the inhibitory role of HCV core protein. TNFR1 associates with TRADD, which recruits FADD for activation of caspase-8, and initiates the apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. HCV core protein appears to inhibit TNF--mediated apoptosis by sustained c-FLIP expression, thus impairing activation of caspase-8 for downstream <span class="highlight" style="background-color:">signaling</span> of executioner caspases.</div></p></div><div class="aux"><div class="resc">Kousuke Saito, et al. J Virol. 2006 May;80(9):4372-4379.</div><p class="links"><a class="dblinks" href="/pubmed/16611896" ref="ordinalpos=2618&amp;ncbi_uid=677855&amp;link_uid=16611896">Citation</a><a class="dblinks" href="/pmc/articles/PMC1472009/" ref="ordinalpos=2618&amp;ncbi_uid=677855&amp;link_uid=PMC1472009">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2619.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3993994/figure/F1/" ref="ordinalpos=2619&amp;ncbi_uid=6041732&amp;link_uid=PMC3993994" href="/pmc/articles/PMC3993994/"><img src="/pmc/articles/instance/3993994/bin/nihms569654f1.gif" src-large="/pmc/articles/instance/3993994/bin/nihms569654f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3993994/" ref="ordinalpos=2619&amp;ncbi_uid=6041732&amp;link_uid=PMC3993994" image-link="/pmc/articles/PMC3993994/figure/F1/" class="imagepopup">Figure 1.  From: LIPID A-MEDIATED TOLERANCE AND CANCER THERAPY. </a></p><div class="supp"><p class="details"><div>Schematic diagram of hypothetical mechanisms of tolerance within the TLR4 <span class="highlight" style="background-color:">pathway</span>. While ST2, IRAK-M, and the splice variant of MyD88 (MyD88s) are thought to suppress association of IRAK-1 with MyD88 through competitive inhibition. TOLLIP has been shown to bind to IRAK-1 and inhibits its activity. SOCS-1 and SOCS-3 are inhibitors of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. While the mechanism by which SHIP induces tolerance is unknown, it is believed to inhibit the NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Cheryl E. Rockwell, et al. Adv Exp Med Biol. 2010;667:81-99.</div><p class="links"><a class="dblinks" href="/pubmed/20665202" ref="ordinalpos=2619&amp;ncbi_uid=6041732&amp;link_uid=20665202">Citation</a><a class="dblinks" href="/pmc/articles/PMC3993994/" ref="ordinalpos=2619&amp;ncbi_uid=6041732&amp;link_uid=PMC3993994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2620.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3965205/figure/F1/" ref="ordinalpos=2620&amp;ncbi_uid=5944332&amp;link_uid=PMC3965205" href="/pmc/articles/PMC3965205/"><img src="/pmc/articles/instance/3965205/bin/nihms570994f1.gif" src-large="/pmc/articles/instance/3965205/bin/nihms570994f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3965205/" ref="ordinalpos=2620&amp;ncbi_uid=5944332&amp;link_uid=PMC3965205" image-link="/pmc/articles/PMC3965205/figure/F1/" class="imagepopup">Fig. 1.  From: LIPID A-MEDIATED TOLERANCE AND CANCER THERAPY. </a></p><div class="supp"><p class="details"><div>Schematic diagram of hypothetical mechanisms of tolerance within the TLR4 <span class="highlight" style="background-color:">pathway</span>. While ST2, IRAK-M, and the splice variant of MyD88 (MyD88s) are thought to suppress association of IRAK-1 with MyD88 through competitive inhibition. TOLLIP has been shown to bind to IRAK-1 and inhibits its activity. SOCS-1 and SOCS-3 are inhibitors of the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. While the mechanism by which SHIP induces tolerance is unknown, it is believed to inhibit the NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Cheryl E. Rockwell, et al. Adv Exp Med Biol. 2009;667:81-99.</div><p class="links"><a class="dblinks" href="/pubmed/20429219" ref="ordinalpos=2620&amp;ncbi_uid=5944332&amp;link_uid=20429219">Citation</a><a class="dblinks" href="/pmc/articles/PMC3965205/" ref="ordinalpos=2620&amp;ncbi_uid=5944332&amp;link_uid=PMC3965205">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2621.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3179504/figure/pone-0025332-g008/" ref="ordinalpos=2621&amp;ncbi_uid=3251206&amp;link_uid=PMC3179504" href="/pmc/articles/PMC3179504/"><img src="/pmc/articles/instance/3179504/bin/pone.0025332.g008.gif" src-large="/pmc/articles/instance/3179504/bin/pone.0025332.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3179504/" ref="ordinalpos=2621&amp;ncbi_uid=3251206&amp;link_uid=PMC3179504" image-link="/pmc/articles/PMC3179504/figure/pone-0025332-g008/" class="imagepopup">Figure 8. Intracellular <span class="highlight" style="background-color:">signaling</span> mechanisms involved in NK1R-mediated membrane blebbing..  From: Substance P Induces Rapid and Transient Membrane Blebbing in U373MG Cells in a p21-Activated Kinase-Dependent Manner. </a></p><div class="supp"><p class="details"><div>Bleb formation mediated by the endogenous full-length NK1R in U373MG cells is dependent of PAK and is associated with tubulin depolymerization. The classic Gq-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by NK1R is not involved in morphological changes induced by SP.</div></p></div><div class="aux"><div class="resc">John Meshki, et al. PLoS One. 2011;6(9):e25332.</div><p class="links"><a class="dblinks" href="/pubmed/21966499" ref="ordinalpos=2621&amp;ncbi_uid=3251206&amp;link_uid=21966499">Citation</a><a class="dblinks" href="/pmc/articles/PMC3179504/" ref="ordinalpos=2621&amp;ncbi_uid=3251206&amp;link_uid=PMC3179504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2622.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3669032/figure/F6/" ref="ordinalpos=2622&amp;ncbi_uid=4960056&amp;link_uid=PMC3669032" href="/pmc/articles/PMC3669032/"><img src="/pmc/articles/instance/3669032/bin/1471-2164-14-282-6.gif" src-large="/pmc/articles/instance/3669032/bin/1471-2164-14-282-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3669032/" ref="ordinalpos=2622&amp;ncbi_uid=4960056&amp;link_uid=PMC3669032" image-link="/pmc/articles/PMC3669032/figure/F6/" class="imagepopup">Figure 6.  From: Gene expression patterns following unilateral traumatic brain injury reveals a local pro-inflammatory and remote anti-inflammatory response. </a></p><div class="supp"><p class="details"><div><b>Canonical <span class="highlight" style="background-color:">pathway</span> analysis. </b>The IL-6 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the relative expression values for all TBI-I IR genes (<b>A</b>) and all TBI-C IR genes (<b>B</b>) involved in this <span class="highlight" style="background-color:">pathway</span>. <i>red: relative increase in expression; green: relative decrease in expression; white: no change in expression</i>.</div></p></div><div class="aux"><div class="resc">Todd E White, et al. BMC Genomics. 2013;14:282-282.</div><p class="links"><a class="dblinks" href="/pubmed/23617241" ref="ordinalpos=2622&amp;ncbi_uid=4960056&amp;link_uid=23617241">Citation</a><a class="dblinks" href="/pmc/articles/PMC3669032/" ref="ordinalpos=2622&amp;ncbi_uid=4960056&amp;link_uid=PMC3669032">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2623.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3601063/figure/pone-0059285-g004/" ref="ordinalpos=2623&amp;ncbi_uid=4734088&amp;link_uid=PMC3601063" href="/pmc/articles/PMC3601063/"><img src="/pmc/articles/instance/3601063/bin/pone.0059285.g004.gif" src-large="/pmc/articles/instance/3601063/bin/pone.0059285.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3601063/" ref="ordinalpos=2623&amp;ncbi_uid=4734088&amp;link_uid=PMC3601063" image-link="/pmc/articles/PMC3601063/figure/pone-0059285-g004/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Signaling</span> pathways regulated by CSPGs..  From: Global Analysis of Neuronal Phosphoproteome Regulation by Chondroitin Sulfate Proteoglycans. </a></p><div class="supp"><p class="details"><div>PANTHER <span class="highlight" style="background-color:">Pathway</span> terms were extracted for each phosphopeptide that changed significantly with CSPGs and compared with <span class="highlight" style="background-color:">pathway</span> terms for all identified peptides which were used as background. The gene names for the corresponding phosphoproteins that changed with CSPGs are listed for each <span class="highlight" style="background-color:">pathway</span>. Only the pathways that contained more than one identified protein are included.</div></p></div><div class="aux"><div class="resc">Panpan Yu, et al. PLoS One. 2013;8(3):e59285.</div><p class="links"><a class="dblinks" href="/pubmed/23527152" ref="ordinalpos=2623&amp;ncbi_uid=4734088&amp;link_uid=23527152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3601063/" ref="ordinalpos=2623&amp;ncbi_uid=4734088&amp;link_uid=PMC3601063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2624.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3517183/figure/F1/" ref="ordinalpos=2624&amp;ncbi_uid=4435010&amp;link_uid=PMC3517183" href="/pmc/articles/PMC3517183/"><img src="/pmc/articles/instance/3517183/bin/nihms-421438-f0001.gif" src-large="/pmc/articles/instance/3517183/bin/nihms-421438-f0001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3517183/" ref="ordinalpos=2624&amp;ncbi_uid=4435010&amp;link_uid=PMC3517183" image-link="/pmc/articles/PMC3517183/figure/F1/" class="imagepopup">FIGURE 1.  From: Single-Cell Network Profiling of Peripheral Blood Mononuclear Cells from Healthy Donors Reveals Age- and Race-Associated Differences in Immune <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Activation. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> responses for the 28 modulated <span class="highlight" style="background-color:">signaling</span> nodes. <i>A</i>, Dendrogram showing the gating hierarchy for the total 12 cell populations including the 7 distinct immune cell subpopulations outlined in red. <i>B</i>, <span class="highlight" style="background-color:">Signaling</span> responses in all 12 of the cell populations. <span class="highlight" style="background-color:">Signaling</span> nodes with fold values &lt;0.25 are represented by gray squares (Response), and <span class="highlight" style="background-color:">signaling</span> nodes with values below this threshold are indicated with white squares (No response). C, Representative histograms from one donor with unmodulated p-Erk levels shown in red and R848-induced p-Erk levels shown in blue. <i>D</i>, Representative histograms from one donor with unmodulated p-Stat1 levels shown in red and IFN-induced p-Stat1 levels shown in blue.</div></p></div><div class="aux"><div class="resc">Diane M. Longo, et al. J Immunol. ;188(4):1717-1725.</div><p class="links"><a class="dblinks" href="/pubmed/22246624" ref="ordinalpos=2624&amp;ncbi_uid=4435010&amp;link_uid=22246624">Citation</a><a class="dblinks" href="/pmc/articles/PMC3517183/" ref="ordinalpos=2624&amp;ncbi_uid=4435010&amp;link_uid=PMC3517183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2625.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3845839/figure/F3/" ref="ordinalpos=2625&amp;ncbi_uid=5954062&amp;link_uid=PMC3845839" href="/pmc/articles/PMC3845839/"><img src="/pmc/articles/instance/3845839/bin/nihms-529254-f0003.gif" src-large="/pmc/articles/instance/3845839/bin/nihms-529254-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3845839/" ref="ordinalpos=2625&amp;ncbi_uid=5954062&amp;link_uid=PMC3845839" image-link="/pmc/articles/PMC3845839/figure/F3/" class="imagepopup">Fig. 3. Basal profiles and <span class="highlight" style="background-color:">signaling</span> profiles predict responses to targeted inhibitors.  From: Profiles of Basal and Stimulated Receptor <span class="highlight" style="background-color:">Signaling</span> Networks Predict Drug Response in Breast Cancer Lines. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Projection of the fit of the PLSR model based on the <span class="highlight" style="background-color:">signaling</span> profile data predicting <i>GI<sub>50</sub></i> values for erlotinib responses against the measured response for each cell line (red, resistant; blue, sensitive). Lower plot shows the trimodal distribution of the measured <i>GI<sub>50</sub></i> values across the cell lines. <b>(B)</b> Fraction of drugs for each class (see Table 1) predicted by the totality of the data (black), the <span class="highlight" style="background-color:">signaling</span> profile based on the maximal response to the 15 growth factors (white), and the basal profile (gray). <b>(C)</b> Distribution of the <i>q<sup>2</sup></i> for drugs significantly predicted by the basal or <span class="highlight" style="background-color:">signaling</span> profile grouped by primary drug targets (AKTp, AKT <span class="highlight" style="background-color:">pathway</span>; MAPKp, MAPK <span class="highlight" style="background-color:">pathway</span>; CC, cell cycle). Non-significant predictions (NS) are set to zero. Colors represent different classes of drugs.</div></p></div><div class="aux"><div class="resc">Mario Niepel, et al. Sci Signal. ;6(294):10.1126/scisignal.2004379.</div><p class="links"><a class="dblinks" href="/pubmed/24065145" ref="ordinalpos=2625&amp;ncbi_uid=5954062&amp;link_uid=24065145">Citation</a><a class="dblinks" href="/pmc/articles/PMC3845839/" ref="ordinalpos=2625&amp;ncbi_uid=5954062&amp;link_uid=PMC3845839">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2626.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3557298/figure/pone-0054762-g004/" ref="ordinalpos=2626&amp;ncbi_uid=4593550&amp;link_uid=PMC3557298" href="/pmc/articles/PMC3557298/"><img src="/pmc/articles/instance/3557298/bin/pone.0054762.g004.gif" src-large="/pmc/articles/instance/3557298/bin/pone.0054762.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3557298/" ref="ordinalpos=2626&amp;ncbi_uid=4593550&amp;link_uid=PMC3557298" image-link="/pmc/articles/PMC3557298/figure/pone-0054762-g004/" class="imagepopup">Figure 4. Summary of transcripts related to plant hormone signal transduction pathways and their response to PEG and NaCl treatments across cotton tissue samples..  From: mRNA-seq Analysis of the Gossypium arboreum transcriptome Reveals Tissue Selective <span class="highlight" style="background-color:">Signaling</span> in Response to Water Stress during Seedling Stage. </a></p><div class="supp"><p class="details"><div>The colored bars represent the percentage of the transcripts in each bin (re-annotated to MapMan classification) whether up-regulated (red) or down-regulated (blue) under PEG or NaCl treatment in different cotton tissues. <b>A</b>: represents the differential expression transcripts related to auxin (IAA) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>B</b>: represents the differential expression transcripts related to cytokinin (CK) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>C</b>: represents the differential expression transcripts related to gibberellin (GA) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>D</b>: represents the differential expression transcripts related to abscisic acid (ABA) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>E</b>: represents the differential expression transcripts related to ethylene signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>F</b>: represents the differential expression transcripts related to brassinosteroid (BR) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>G</b>: represents the differential expression transcripts related to jasmonate (JA) signalling transduction <span class="highlight" style="background-color:">pathway</span>. <b>H</b>: represents the differential expression transcripts related to salycylic acid (SA) signalling transduction <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Xueyan Zhang, et al. PLoS One. 2013;8(1):e54762.</div><p class="links"><a class="dblinks" href="/pubmed/23382961" ref="ordinalpos=2626&amp;ncbi_uid=4593550&amp;link_uid=23382961">Citation</a><a class="dblinks" href="/pmc/articles/PMC3557298/" ref="ordinalpos=2626&amp;ncbi_uid=4593550&amp;link_uid=PMC3557298">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2627.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3480408/figure/pone-0047797-g006/" ref="ordinalpos=2627&amp;ncbi_uid=4322633&amp;link_uid=PMC3480408" href="/pmc/articles/PMC3480408/"><img src="/pmc/articles/instance/3480408/bin/pone.0047797.g006.gif" src-large="/pmc/articles/instance/3480408/bin/pone.0047797.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3480408/" ref="ordinalpos=2627&amp;ncbi_uid=4322633&amp;link_uid=PMC3480408" image-link="/pmc/articles/PMC3480408/figure/pone-0047797-g006/" class="imagepopup">Figure 6. A schematic model for the roles of CKs in low K signal <span class="highlight" style="background-color:">pathway</span>..  From: Regulatory Roles of Cytokinins and Cytokinin <span class="highlight" style="background-color:">Signaling</span> in Response to Potassium Deficiency in Arabidopsis. </a></p><div class="supp"><p class="details"><div>Low K conditions lead to reduced levels of endogenous CKs, which are negative regulators under these conditions. Lower CK levels result in ROS production, <i>HAK5</i> expression, and then altered root growth. Previous studies showed that ethylene is a positive regulator in low K <span class="highlight" style="background-color:">signaling</span> that controls ROS production, <i>HAK5</i> expression as well as root hair growth [13]. ABA [30] and auxin [12] also are involved in controlling low K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Arrowhead line, positive regulation; bar-head lines, negative regulation; gray color, previously identified; black color, identified in this study.</div></p></div><div class="aux"><div class="resc">Youn-Jeong Nam, et al. PLoS One. 2012;7(10):e47797.</div><p class="links"><a class="dblinks" href="/pubmed/23112848" ref="ordinalpos=2627&amp;ncbi_uid=4322633&amp;link_uid=23112848">Citation</a><a class="dblinks" href="/pmc/articles/PMC3480408/" ref="ordinalpos=2627&amp;ncbi_uid=4322633&amp;link_uid=PMC3480408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2628.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4022607/figure/F3/" ref="ordinalpos=2628&amp;ncbi_uid=6146615&amp;link_uid=PMC4022607" href="/pmc/articles/PMC4022607/"><img src="/pmc/articles/instance/4022607/bin/tisb-2-e28127-g3.gif" src-large="/pmc/articles/instance/4022607/bin/tisb-2-e28127-g3.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4022607/" ref="ordinalpos=2628&amp;ncbi_uid=6146615&amp;link_uid=PMC4022607" image-link="/pmc/articles/PMC4022607/figure/F3/" class="imagepopup">Figure. From: The role of angiomotin phosphorylation in the Hippo <span class="highlight" style="background-color:">pathway</span> during preimplantation mouse development. </a></p><div class="supp"><p class="details"><div><b>Figure 3.</b> The molecular basis of differential Hippo <span class="highlight" style="background-color:">signaling</span> in preimplantation embryos. (<b>A</b>) <i>Amot</i> KO enhances Hippo <span class="highlight" style="background-color:">signaling</span> in the outer cells as shown by an increased level of phosphorylated Yap (p-Yap). The inner cells are encircled by dashed lines in the left panels. The graph shows the mean level of p-Yap in the outer cells of wildtype (wt) (n = 4) and <i>Amot</i> KO (n = 5) embryos. Error bars show the standard error of the mean. These data suggest that Hippo <span class="highlight" style="background-color:">signaling</span> is inhibited at the apical membrane. (<b>B</b>) A model of Hippo <span class="highlight" style="background-color:">pathway</span> regulation in preimplantation embryos. In the inner cells, phosphorylated Amot makes an active complex with MerlinLats at the AJs, activating Hippo <span class="highlight" style="background-color:">signaling</span>. In the outer cells, (1) cell polarity factors sequester Amot from the basolateral AJs, thereby attenuating E-cadherin-mediated Hippo <span class="highlight" style="background-color:">signaling</span>. (2) Nonphosphorylated apical Amot makes an inactive complex with MerlinLats, which binds cortical F-actin, suppressing Lats activity.</div></p></div><div class="aux"><div class="resc">Yoshikazu Hirate, et al. Tissue Barriers. 2014 January 1;2:e28127.</div><p class="links"><a class="dblinks" href="/pubmed/24843842" ref="ordinalpos=2628&amp;ncbi_uid=6146615&amp;link_uid=24843842">Citation</a><a class="dblinks" href="/pmc/articles/PMC4022607/" ref="ordinalpos=2628&amp;ncbi_uid=6146615&amp;link_uid=PMC4022607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2629.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3703460/figure/F4/" ref="ordinalpos=2629&amp;ncbi_uid=6434405&amp;link_uid=PMC3703460" href="/pmc/articles/PMC3703460/"><img src="/pmc/articles/instance/3703460/bin/nihms469526f4.gif" src-large="/pmc/articles/instance/3703460/bin/nihms469526f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3703460/" ref="ordinalpos=2629&amp;ncbi_uid=6434405&amp;link_uid=PMC3703460" image-link="/pmc/articles/PMC3703460/figure/F4/" class="imagepopup">Figure 4. Regulation of the myogenin mRNA stability by the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the ARE-BP KSRP.  From: <span class="highlight" style="background-color:">Signaling</span> Pathways That Control mRNA Turnover. </a></p><div class="supp"><p class="details"><div>PI3K/AKT activation in myoblasts C2C12 regulates myomiR maturation via the ARE-BP KSRP. Phosphorylation of KSRP by AKT inhibits its ability to promote the decay of myogenin mRNA by dissociating KSRP from myogenin mRNA in the cytoplasm. The released KSRP binds 14-3-3, translocates to the nucleus where it promotes the maturation of myogenic miRNAs.</div></p></div><div class="aux"><div class="resc">Roopa Thapar, et al. Cell Signal. ;25(8):1699-1710.</div><p class="links"><a class="dblinks" href="/pubmed/23602935" ref="ordinalpos=2629&amp;ncbi_uid=6434405&amp;link_uid=23602935">Citation</a><a class="dblinks" href="/pmc/articles/PMC3703460/" ref="ordinalpos=2629&amp;ncbi_uid=6434405&amp;link_uid=PMC3703460">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2630.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3475699/figure/pone-0047516-g004/" ref="ordinalpos=2630&amp;ncbi_uid=4311687&amp;link_uid=PMC3475699" href="/pmc/articles/PMC3475699/"><img src="/pmc/articles/instance/3475699/bin/pone.0047516.g004.gif" src-large="/pmc/articles/instance/3475699/bin/pone.0047516.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3475699/" ref="ordinalpos=2630&amp;ncbi_uid=4311687&amp;link_uid=PMC3475699" image-link="/pmc/articles/PMC3475699/figure/pone-0047516-g004/" class="imagepopup">Figure 4. Quercetin inhibits the activation of VEGFR2-mediated <span class="highlight" style="background-color:">signaling</span> pathways in HUVECs..  From: Quercetin Inhibits Angiogenesis Mediated Human Prostate Tumor Growth by Targeting VEGFR- 2 Regulated AKT/mTOR/P70S6K <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Quercetin suppressed the activation of VEGFR2 and their down stream AKT/mTOR/p70S6K <span class="highlight" style="background-color:">pathway</span> triggered by VEGF in HUVECs. Proteins from different treatments was tested by western blotting and probed with specific antibodies. Experiments were repeated for three times.</div></p></div><div class="aux"><div class="resc">Poyil Pratheeshkumar, et al. PLoS One. 2012;7(10):e47516.</div><p class="links"><a class="dblinks" href="/pubmed/23094058" ref="ordinalpos=2630&amp;ncbi_uid=4311687&amp;link_uid=23094058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3475699/" ref="ordinalpos=2630&amp;ncbi_uid=4311687&amp;link_uid=PMC3475699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2631.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2952323/figure/F3/" ref="ordinalpos=2631&amp;ncbi_uid=3228840&amp;link_uid=PMC2952323" href="/pmc/articles/PMC2952323/"><img src="/pmc/articles/instance/2952323/bin/nihms236675f3.gif" src-large="/pmc/articles/instance/2952323/bin/nihms236675f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2952323/" ref="ordinalpos=2631&amp;ncbi_uid=3228840&amp;link_uid=PMC2952323" image-link="/pmc/articles/PMC2952323/figure/F3/" class="imagepopup">Fig. 3.  From: Secreted Frizzled-Related Protein-2 (sFRP2) Augments Canonical Wnt3a-induced <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>sFRP2 augments Wnt3a-induced gene expression in HEK293A cells. Cells were treated with 3 nM Wnt3a alone or in combination with 6 nM sFRP2 for 6 h. RNA was isolated and analyzed using Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> RT<sup>2</sup> Profile PCR Arrays. Shown are mean  SEM of 3 independent experiments.</div></p></div><div class="aux"><div class="resc">Zofia von Marschall, et al. Biochem Biophys Res Commun. ;400(3):299-304.</div><p class="links"><a class="dblinks" href="/pubmed/20723538" ref="ordinalpos=2631&amp;ncbi_uid=3228840&amp;link_uid=20723538">Citation</a><a class="dblinks" href="/pmc/articles/PMC2952323/" ref="ordinalpos=2631&amp;ncbi_uid=3228840&amp;link_uid=PMC2952323">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2632.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2704161/figure/F1/" ref="ordinalpos=2632&amp;ncbi_uid=1929186&amp;link_uid=PMC2704161" href="/pmc/articles/PMC2704161/"><img src="/pmc/articles/instance/2704161/bin/zmk0130991040001.gif" src-large="/pmc/articles/instance/2704161/bin/zmk0130991040001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2704161/" ref="ordinalpos=2632&amp;ncbi_uid=1929186&amp;link_uid=PMC2704161" image-link="/pmc/articles/PMC2704161/figure/F1/" class="imagepopup">Figure 1. From: The <span class="highlight" style="background-color:">Signaling</span> Mucins Msb2 and Hkr1 Differentially Regulate the Filamentation Mitogen-activated Protein Kinase <span class="highlight" style="background-color:">Pathway</span> and Contribute to a Multimodal Response. </a></p><div class="supp"><p class="details"><div>The FG and HOG MAP kinase pathways. The two pathways are shown with factors that function in both pathways in black, <span class="highlight" style="background-color:">pathway</span>-specific factors for the FG <span class="highlight" style="background-color:">pathway</span> in green, and HOG <span class="highlight" style="background-color:">pathway</span>, red. The Sln1p branch of the HOG <span class="highlight" style="background-color:">pathway</span> is also shown. The Ste7p and Ste12p proteins are shown in black because they are required for activation of the FG and pheromone response pathways.</div></p></div><div class="aux"><div class="resc">Andrew Pitoniak, et al. Mol Biol Cell. 2009 July 1;20(13):3101-3114.</div><p class="links"><a class="dblinks" href="/pubmed/19439450" ref="ordinalpos=2632&amp;ncbi_uid=1929186&amp;link_uid=19439450">Citation</a><a class="dblinks" href="/pmc/articles/PMC2704161/" ref="ordinalpos=2632&amp;ncbi_uid=1929186&amp;link_uid=PMC2704161">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2633.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4349862/figure/pone.0118551.g006/" ref="ordinalpos=2633&amp;ncbi_uid=7289128&amp;link_uid=PMC4349862" href="/pmc/articles/PMC4349862/"><img src="/pmc/articles/instance/4349862/bin/pone.0118551.g006.gif" src-large="/pmc/articles/instance/4349862/bin/pone.0118551.g006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4349862/" ref="ordinalpos=2633&amp;ncbi_uid=7289128&amp;link_uid=PMC4349862" image-link="/pmc/articles/PMC4349862/figure/pone.0118551.g006/" class="imagepopup">Fig 6. A schematic overview of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by increased Sp1 level..  From: Overexpression of the Transcription Factor Sp1 Activates the OAS-RNAse L-RIG-I <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Upon Sp1 overexpression, <i>Oas2</i> and <i>Rnasel</i> genes are upregulated and small self-RNAs are produced. Sensing of small self-RNAs leads to the activation of the sensor RIG-I, the IRF3/7 transcription factors and downstream effector targets such as ISGs.</div></p></div><div class="aux"><div class="resc">Valryane Dupuis-Maurin, et al. PLoS One. 2015;10(3):e0118551.</div><p class="links"><a class="dblinks" href="/pubmed/25738304" ref="ordinalpos=2633&amp;ncbi_uid=7289128&amp;link_uid=25738304">Citation</a><a class="dblinks" href="/pmc/articles/PMC4349862/" ref="ordinalpos=2633&amp;ncbi_uid=7289128&amp;link_uid=PMC4349862">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2634.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC196255/figure/fig7/" ref="ordinalpos=2634&amp;ncbi_uid=239780&amp;link_uid=PMC196255" href="/pmc/articles/PMC196255/"><img src="/pmc/articles/instance/196255/bin/67858-09f7a_L1TT.gif" src-large="/pmc/articles/instance/196255/bin/67858-09f7a_L1TT.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC196255/figure/fig7/" ref="ordinalpos=2634&amp;ncbi_uid=239780&amp;link_uid=PMC196255" href="/pmc/articles/PMC196255/"><img src="/pmc/articles/instance/196255/bin/67858-09f7b_F1TT.gif" src-large="/pmc/articles/instance/196255/bin/67858-09f7b_F1TT.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC196255/" ref="ordinalpos=2634&amp;ncbi_uid=239780&amp;link_uid=PMC196255" image-link="/pmc/articles/PMC196255/figure/fig7/" class="imagepopup">Figure 7. From: Control of cell number by Drosophila FOXO: downstream and  feedback regulation of the insulin receptor <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>(<i>A</i>) dFOXO is a key element of insulin <span class="highlight" style="background-color:">signaling</span> in  <i>Drosophila</i>. The insulin receptor inactivates dFOXO through  dPI3K/dAkt. Activation of d4EBP may explain growth inhibition by dFOXO,  whereas activation of dInR may provide a novel transcriptionally induced  feedback control mechanism for the <span class="highlight" style="background-color:">pathway</span>. (<i>B</i>) Model to explain  regulation of growth by a feedback mechanism involving dInR and dFOXO.</div></p></div><div class="aux"><div class="resc">Oscar Puig, et al. Genes Dev. 2003 August 15;17(16):2006-2020.</div><p class="links"><a class="dblinks" href="/pubmed/12893776" ref="ordinalpos=2634&amp;ncbi_uid=239780&amp;link_uid=12893776">Citation</a><a class="dblinks" href="/pmc/articles/PMC196255/" ref="ordinalpos=2634&amp;ncbi_uid=239780&amp;link_uid=PMC196255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2635.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119514/figure/F1/" ref="ordinalpos=2635&amp;ncbi_uid=3462298&amp;link_uid=PMC3119514" href="/pmc/articles/PMC3119514/"><img src="/pmc/articles/instance/3119514/bin/nihms291316f1.gif" src-large="/pmc/articles/instance/3119514/bin/nihms291316f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119514/" ref="ordinalpos=2635&amp;ncbi_uid=3462298&amp;link_uid=PMC3119514" image-link="/pmc/articles/PMC3119514/figure/F1/" class="imagepopup">Fig. 1.  From: Genomics of the NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: hypothesized role in ovarian cancer. </a></p><div class="supp"><p class="details"><div>Key components of NF-B <span class="highlight" style="background-color:">signaling</span> pathways. Color-coding of molecules: activators = <i>yellow</i>, NF-B subunits = <i>blue</i>, inhibitors = <i>red</i>, degradators = <i>green</i>, nuclear function = <i>cream</i>. Canonical <span class="highlight" style="background-color:">pathway</span>: Binding of extracellular <span class="highlight" style="background-color:">signaling</span> molecules to receptors leads to recruitment of activation proteins, phosphorylation of IKK, activation of the IKK complex (IKK, IKK, and NEMO), and processing of p105 and p100 into functional p50 and p52 by the ubiquitin ligase SCF<sup>-TrCP</sup> complex. Activated IKK complex phosphorylates IB, marking it for ubiquitination by the SCF<sup>-TrCP</sup> complex and degradation. The main canonical NF-B dimer, p65/p50, is activated and translocates to the nucleus to initiate gene transcription with the RNA polymerase II preinitiation complex (PIC). Non-canonical <span class="highlight" style="background-color:">pathway</span>: Ligase and receptor coupling leads to NIK and IKK activation, which initiates processing of p100 into functional p52 by the SCF<sup>-TrCP</sup> complex. The p52/Rel-B dimer translocates to the nucleus to initiate gene expression through the RNA polymerase II PIC</div></p></div><div class="aux"><div class="resc">Kristin L. White, et al. Cancer Causes Control. ;22(5):785-801.</div><p class="links"><a class="dblinks" href="/pubmed/21359843" ref="ordinalpos=2635&amp;ncbi_uid=3462298&amp;link_uid=21359843">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119514/" ref="ordinalpos=2635&amp;ncbi_uid=3462298&amp;link_uid=PMC3119514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2636.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807212/figure/fig3/" ref="ordinalpos=2636&amp;ncbi_uid=2213191&amp;link_uid=PMC2807212" href="/pmc/articles/PMC2807212/"><img src="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0001.gif" src-large="/pmc/articles/instance/2807212/bin/cb-2009-00249y_0001.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807212/" ref="ordinalpos=2636&amp;ncbi_uid=2213191&amp;link_uid=PMC2807212" image-link="/pmc/articles/PMC2807212/figure/fig3/" class="imagepopup">Figure 3.  From: Controlling Destiny through Chemistry: Small-Molecule Regulators of Cell Fate. </a></p><div class="supp"><p class="details"><div>TGF superfamily and FGF <span class="highlight" style="background-color:">signaling</span> pathways. <span class="highlight" style="background-color:">Signaling</span> proteins associated with TGF (a) and FGF (b) <span class="highlight" style="background-color:">pathway</span> regulation are shown. Components or processes that currently can be targeted by small molecules to achieve selective <span class="highlight" style="background-color:">pathway</span> control are labeled with green (agonist) and red (antagonist) hexagons. Direct small moleculeprotein interactions are depicted when known.</div></p></div><div class="aux"><div class="resc">Ari J. Firestone, et al. ACS Chem Biol. 2010 January 15;5(1):15-34.</div><p class="links"><a class="dblinks" href="/pubmed/20000447" ref="ordinalpos=2636&amp;ncbi_uid=2213191&amp;link_uid=20000447">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807212/" ref="ordinalpos=2636&amp;ncbi_uid=2213191&amp;link_uid=PMC2807212">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2637.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2783976/figure/F10/" ref="ordinalpos=2637&amp;ncbi_uid=2171879&amp;link_uid=PMC2783976" href="/pmc/articles/PMC2783976/"><img src="/pmc/articles/instance/2783976/bin/nihms-130658-f0010.gif" src-large="/pmc/articles/instance/2783976/bin/nihms-130658-f0010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783976/" ref="ordinalpos=2637&amp;ncbi_uid=2171879&amp;link_uid=PMC2783976" image-link="/pmc/articles/PMC2783976/figure/F10/" class="imagepopup">Figure 10. Schematic illustration demonstrating putative <span class="highlight" style="background-color:">signaling</span> pathways involved in PDGF-mediated neuroprotection in rat primary neurons.  From: TRPC channel-mediated neuroprotection by PDGF involves Pyk2/ERK/CREB <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>PDGF-mediated engagement of the PDGF- receptor stimulates the PLC/IP3 R <span class="highlight" style="background-color:">pathway</span>, which in turn, activates TRPC channels resulting in elevation of [Ca<sup>2+</sup>]i transients. [Ca<sup>2+</sup>]i elevation in turn results in activation of Pyk2/ERK pathways leading to CREB activation and consequently neuronal survival. In addition to this, PDGF/PDGF-R axis can also activate another distinct <span class="highlight" style="background-color:">pathway</span> - PI3K/Akt, which can also lead to potentiation of neuronal survival.</div></p></div><div class="aux"><div class="resc">Honghong Yao, et al. Cell Death Differ. ;16(12):1681-1693.</div><p class="links"><a class="dblinks" href="/pubmed/19680266" ref="ordinalpos=2637&amp;ncbi_uid=2171879&amp;link_uid=19680266">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783976/" ref="ordinalpos=2637&amp;ncbi_uid=2171879&amp;link_uid=PMC2783976">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2638.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3367558/figure/A008045F3/" ref="ordinalpos=2638&amp;ncbi_uid=6185743&amp;link_uid=PMC3367558" href="/pmc/articles/PMC3367558/"><img src="/pmc/articles/instance/3367558/bin/cshperspect-WNT-008045_F3.gif" src-large="/pmc/articles/instance/3367558/bin/cshperspect-WNT-008045_F3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3367558/" ref="ordinalpos=2638&amp;ncbi_uid=6185743&amp;link_uid=PMC3367558" image-link="/pmc/articles/PMC3367558/figure/A008045F3/" class="imagepopup">Figure 3. From: Wnt <span class="highlight" style="background-color:">Signaling</span> in Neuromuscular Junction Development. </a></p><div class="supp"><p class="details"><div>Negative regulation of AChR clustering by Wnt3a. Muscle Wnt3a initiates a Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the muscle that leads to decreased Rapsyn levels. This is achieved through the -catenin-mediated negative regulation of <i>rapsyn</i> gene expression, and decreased Rapsyn levels result in the destabilization and dispersal of AChR clusters. In addition, Wnt3a, via -catenin, also triggers an unknown retrograde signal that affects presynaptic function and/or assembly.</div></p></div><div class="aux"><div class="resc">Kate Koles, et al. Cold Spring Harb Perspect Biol. 2012 June;4(6):a008045.</div><p class="links"><a class="dblinks" href="/pubmed/22510459" ref="ordinalpos=2638&amp;ncbi_uid=6185743&amp;link_uid=22510459">Citation</a><a class="dblinks" href="/pmc/articles/PMC3367558/" ref="ordinalpos=2638&amp;ncbi_uid=6185743&amp;link_uid=PMC3367558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2639.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3197365/figure/F8/" ref="ordinalpos=2639&amp;ncbi_uid=3529829&amp;link_uid=PMC3197365" href="/pmc/articles/PMC3197365/"><img src="/pmc/articles/instance/3197365/bin/zh40101100800008.gif" src-large="/pmc/articles/instance/3197365/bin/zh40101100800008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3197365/" ref="ordinalpos=2639&amp;ncbi_uid=3529829&amp;link_uid=PMC3197365" image-link="/pmc/articles/PMC3197365/figure/F8/" class="imagepopup">Fig. 8. From: SDF-1/CXCR4 mediates acute protection of cardiac function through myocardial STAT3 <span class="highlight" style="background-color:">signaling</span> following global ischemia/reperfusion injury. </a></p><div class="supp"><p class="details"><div>Akt <span class="highlight" style="background-color:">pathway</span> in SDF-1-improved myocardial function following I/R. Inhibition of the Akt <span class="highlight" style="background-color:">signaling</span> by LY294002 does not affect SDF-1-protected postischemic LVDP (<i>A</i>), +dP/d<i>t</i> (<i>B</i>), and dP/d<i>t</i> (<i>C</i>). Results are means  SE; <i>n</i> = 56/group. *<i>P</i> &lt; 0.05 vs. vehicle or LY294002 alone at corresponding time point.</div></p></div><div class="aux"><div class="resc">Chunyan Huang, et al. Am J Physiol Heart Circ Physiol. 2011 October;301(4):H1496-H1505.</div><p class="links"><a class="dblinks" href="/pubmed/21821779" ref="ordinalpos=2639&amp;ncbi_uid=3529829&amp;link_uid=21821779">Citation</a><a class="dblinks" href="/pmc/articles/PMC3197365/" ref="ordinalpos=2639&amp;ncbi_uid=3529829&amp;link_uid=PMC3197365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2640.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2922261/figure/fig02/" ref="ordinalpos=2640&amp;ncbi_uid=2746620&amp;link_uid=PMC2922261" href="/pmc/articles/PMC2922261/"><img src="/pmc/articles/instance/2922261/bin/pnas.1008397107fig02.gif" src-large="/pmc/articles/instance/2922261/bin/pnas.1008397107fig02.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2922261/" ref="ordinalpos=2640&amp;ncbi_uid=2746620&amp;link_uid=PMC2922261" image-link="/pmc/articles/PMC2922261/figure/fig02/" class="imagepopup">Fig. 2. From: Regulator of G protein <span class="highlight" style="background-color:">signaling</span> 5 protects against cardiac hypertrophy and fibrosis during biomechanical stress of pressure overload. </a></p><div class="supp"><p class="details"><div>Effect of Rgs5 on MEK-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Representative blots of ERK1/2, p38, and JNK, and AKT phosphorylation and their total protein expression at 4 wk after AB surgery in Rgs5 transgenic mice and WT mice or in Rgs5<sup>/</sup> and Rgs5<sup>+/+</sup> mice.</div></p></div><div class="aux"><div class="resc">Hongliang Li, et al. Proc Natl Acad Sci U S A. 2010 August 3;107(31):13818-13823.</div><p class="links"><a class="dblinks" href="/pubmed/20643937" ref="ordinalpos=2640&amp;ncbi_uid=2746620&amp;link_uid=20643937">Citation</a><a class="dblinks" href="/pmc/articles/PMC2922261/" ref="ordinalpos=2640&amp;ncbi_uid=2746620&amp;link_uid=PMC2922261">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2641.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2777340/figure/F4/" ref="ordinalpos=2641&amp;ncbi_uid=2117062&amp;link_uid=PMC2777340" href="/pmc/articles/PMC2777340/"><img src="/pmc/articles/instance/2777340/bin/jbt0050900230004.gif" src-large="/pmc/articles/instance/2777340/bin/jbt0050900230004.jpg" alt="FIGURE 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2777340/" ref="ordinalpos=2641&amp;ncbi_uid=2117062&amp;link_uid=PMC2777340" image-link="/pmc/articles/PMC2777340/figure/F4/" class="imagepopup">FIGURE 4.  From: Use of a Label-Free Quantitative Platform Based on MS/MS Average TIC to Calculate Dynamics of Protein Complexes in Insulin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The Drosophila insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PIP3, Phosphatidylinositol 3,4,5-triphosphate; PTEN, phosphatase and tensin homologue deleted on chromosome 10; dTor, target of rapamycin Drosophila; PDK1, phosphoinositide-dependent kinase-1; FKH, forkhead; RTK, receptor tyryosine kinase; PTP-ER, protein tyrosine phosphatase-ERK/enhancer.</div></p></div><div class="aux"><div class="resc">Xuemei Yang, et al. J Biomol Tech. 2009 December;20(5):272-277.</div><p class="links"><a class="dblinks" href="/pubmed/19949701" ref="ordinalpos=2641&amp;ncbi_uid=2117062&amp;link_uid=19949701">Citation</a><a class="dblinks" href="/pmc/articles/PMC2777340/" ref="ordinalpos=2641&amp;ncbi_uid=2117062&amp;link_uid=PMC2777340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2642.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3612991/figure/F2/" ref="ordinalpos=2642&amp;ncbi_uid=5563518&amp;link_uid=PMC3612991" href="/pmc/articles/PMC3612991/"><img src="/pmc/articles/instance/3612991/bin/nihms423722f2.gif" src-large="/pmc/articles/instance/3612991/bin/nihms423722f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3612991/" ref="ordinalpos=2642&amp;ncbi_uid=5563518&amp;link_uid=PMC3612991" image-link="/pmc/articles/PMC3612991/figure/F2/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Signaling</span> Crosstalk between TGF- and Other Pathways in Mouse and Human ESCs.  From: The Harmonies Played by TGF-? in Stem Cell Biology. </a></p><div class="supp"><p class="details"><div>(A) Self-renewal mechanisms in mouse ESCs. LIF and BMPs cooperate to maintain self-renewal. Tyrosine kinase receptor <span class="highlight" style="background-color:">signaling</span>, e.g. FGFs, typically activate downstream effectors ERK-MAPK, and induce differentiation. BMPs induce canonical Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> pathways and one of their key targets for ESCs was recently shown to be the gene encoding ERK-specific phosphatase Dusp9. Dusp9 negatively affects ERK activity and hence support ESC self-renewal. Through the simultaneous induction of another BMP target gene, <i>Id,</i> and through activation of the STAT3 <span class="highlight" style="background-color:">pathway</span> by LIF, these two factors further promote ESC self-renewal by inhibiting mesendoderm and neuroectoderm differentiation, respectively.<br />(B) Self-renewal mechanisms in human ESCs. Nodal/Activin and hyperactive PI3K/Akt <span class="highlight" style="background-color:">signaling</span> cooperate to maintain self-renewal. Nodal/Activin induce canonical Smad2/3 <span class="highlight" style="background-color:">signaling</span> pathways and a key target for ESCs is <i>Nanog</i>. Hyperactive PI3K/Akt <span class="highlight" style="background-color:">signaling</span> can be achieved by some growth factors, such as FGF and IGFs, and this leads to suppression of ERK activity. When PI3K/Akt <span class="highlight" style="background-color:">signaling</span> diminishes, ERK inhibition is relieved, which in turn, suppresses the GSK3 kinase. This results in stabilization of -catenin, which apparently associates with pSmad2/3-Smad4 to influence the target genes that become transcribed, thereby switching the ESC from a self-renewing to differentiating status.</div></p></div><div class="aux"><div class="resc">Naoki Oshimori, et al. Cell Stem Cell. 2012 December 7;11(6):751-764.</div><p class="links"><a class="dblinks" href="/pubmed/23217421" ref="ordinalpos=2642&amp;ncbi_uid=5563518&amp;link_uid=23217421">Citation</a><a class="dblinks" href="/pmc/articles/PMC3612991/" ref="ordinalpos=2642&amp;ncbi_uid=5563518&amp;link_uid=PMC3612991">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2643.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4367659/figure/f1/" ref="ordinalpos=2643&amp;ncbi_uid=7319032&amp;link_uid=PMC4367659" href="/pmc/articles/PMC4367659/"><img src="/pmc/articles/instance/4367659/bin/tlcr-03-02-075-f1.gif" src-large="/pmc/articles/instance/4367659/bin/tlcr-03-02-075-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4367659/" ref="ordinalpos=2643&amp;ncbi_uid=7319032&amp;link_uid=PMC4367659" image-link="/pmc/articles/PMC4367659/figure/f1/" class="imagepopup">Figure 1.  From: Hippo/YAP <span class="highlight" style="background-color:">pathway</span> for targeted therapy. </a></p><div class="supp"><p class="details"><div>In normal tissue repair NF2 activates the Hippo <span class="highlight" style="background-color:">pathway</span>, an essential regulator of cell proliferation. Key components of the Hippo <span class="highlight" style="background-color:">pathway</span> include two kinases: Mst and Lats. The sequential activation of these kinases leads to phosphorylation of the transcription factor YAP. When Hippo <span class="highlight" style="background-color:">signaling</span> is attenuated, e.g., in NF2-deficient cancer cells, YAP phosphorylation is reduced, resulting in its nuclear localization and regulation of target genes such as CTGF, Cyr61, AXL and survivin. Survivin is also controlled by nuclear NF2.</div></p></div><div class="aux"><div class="resc">Emanuela Felley-Bosco, et al. Transl Lung Cancer Res. 2014 April;3(2):75-83.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2643&amp;ncbi_uid=7319032&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4367659/" ref="ordinalpos=2643&amp;ncbi_uid=7319032&amp;link_uid=PMC4367659">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2644.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3870928/figure/F9/" ref="ordinalpos=2644&amp;ncbi_uid=6365789&amp;link_uid=PMC3870928" href="/pmc/articles/PMC3870928/"><img src="/pmc/articles/instance/3870928/bin/zns9991449750009.gif" src-large="/pmc/articles/instance/3870928/bin/zns9991449750009.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3870928/" ref="ordinalpos=2644&amp;ncbi_uid=6365789&amp;link_uid=PMC3870928" image-link="/pmc/articles/PMC3870928/figure/F9/" class="imagepopup">Figure 9. From: Rem2 Is an Activity-Dependent Negative Regulator of Dendritic Complexity In Vivo. </a></p><div class="supp"><p class="details"><div>Model of <span class="highlight" style="background-color:">signaling</span> pathways that mediate activity-dependent changes in dendritic complexity. Depolarization activates Rem2 <span class="highlight" style="background-color:">signaling</span> in at least two ways: (1) calcium entry through L-VGCCs triggers the transcription of Rem2, and (2) calcium activates CaMKII <span class="highlight" style="background-color:">signaling</span>. This Rem2/CaMKII <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> negatively regulates dendritic complexity (Ghiretti et al., 2013 and this manuscript). Calcium entry through L-VGCCs also activates signal transduction <span class="highlight" style="background-color:">pathway</span>(s) that positively regulate dendritic complexity (X) in parallel to Rem2. Thus, the net effect on dendritic morphology is due to the combination of these <span class="highlight" style="background-color:">signaling</span> pathways. Note that a role for constitutively expressed Rem2 in mediating dendritic complexity is not excluded by our results.</div></p></div><div class="aux"><div class="resc">Amy E. Ghiretti, et al. J Neurosci. 2014 January 8;34(2):392-407.</div><p class="links"><a class="dblinks" href="/pubmed/24403140" ref="ordinalpos=2644&amp;ncbi_uid=6365789&amp;link_uid=24403140">Citation</a><a class="dblinks" href="/pmc/articles/PMC3870928/" ref="ordinalpos=2644&amp;ncbi_uid=6365789&amp;link_uid=PMC3870928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2645.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3033794/figure/F11/" ref="ordinalpos=2645&amp;ncbi_uid=2755160&amp;link_uid=PMC3033794" href="/pmc/articles/PMC3033794/"><img src="/pmc/articles/instance/3033794/bin/1477-5956-9-3-11.gif" src-large="/pmc/articles/instance/3033794/bin/1477-5956-9-3-11.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3033794/" ref="ordinalpos=2645&amp;ncbi_uid=2755160&amp;link_uid=PMC3033794" image-link="/pmc/articles/PMC3033794/figure/F11/" class="imagepopup">Figure 11.  From: Cardiogenol C can induce Mouse Hair Bulge Progenitor Cells to Transdifferentiate into Cardiomyocyte-like Cells. </a></p><div class="supp"><p class="details"><div><b>Activation of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> following Cardiogenol C induction</b>. (A) Western blot analysis revealed that Kremen1 was down-regulated in Cardiogenol C-treated HBPCs. Tubulin served as an internal control. (B) Semi-quantitative RT-PCR analysis revealed Dkk1 and Kremen1, which are normally involved in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, were down-regulated in Cardiogenol C-treated HBPCs while Lef1 and Wnt11 expression were up-regulated.</div></p></div><div class="aux"><div class="resc">Winifred WY Yau, et al. Proteome Sci. 2011;9:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/21247432" ref="ordinalpos=2645&amp;ncbi_uid=2755160&amp;link_uid=21247432">Citation</a><a class="dblinks" href="/pmc/articles/PMC3033794/" ref="ordinalpos=2645&amp;ncbi_uid=2755160&amp;link_uid=PMC3033794">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2646.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4320882/figure/fig3/" ref="ordinalpos=2646&amp;ncbi_uid=7182910&amp;link_uid=PMC4320882" href="/pmc/articles/PMC4320882/"><img src="/pmc/articles/instance/4320882/bin/NP2015-975367.003.gif" src-large="/pmc/articles/instance/4320882/bin/NP2015-975367.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4320882/" ref="ordinalpos=2646&amp;ncbi_uid=7182910&amp;link_uid=PMC4320882" image-link="/pmc/articles/PMC4320882/figure/fig3/" class="imagepopup">Figure 3.  From: Odorant Receptors <span class="highlight" style="background-color:">Signaling</span> Instructs the Development and Plasticity of the Glomerular Map. </a></p><div class="supp"><p class="details"><div>OSNs in the OE and OR <span class="highlight" style="background-color:">signaling</span> pathways. (a) The OSN somas and cilia (dots), where the OR <span class="highlight" style="background-color:">signaling</span> pathways occur, transduce the information to the axons (lines) through action potentials that eventually reach the OB. (b) Diagram of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of a standard OR after activation. The interaction of odorants with their receptors on the OSN cilia and soma leads to Golf activation, which activates ACIII, thus causing increased cAMP levels. Next, cAMP activates CNG channels that allow the entry of sodium and calcium. Finally, this latter ion activates a chloride channel that contributes to depolarization enough to reach the threshold for action potentials. Modified from [38].</div></p></div><div class="aux"><div class="resc">Pablo Valle-Leija. Neural Plast. 2015;2015:975367.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2646&amp;ncbi_uid=7182910&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4320882/" ref="ordinalpos=2646&amp;ncbi_uid=7182910&amp;link_uid=PMC4320882">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2647.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2610355/figure/F9/" ref="ordinalpos=2647&amp;ncbi_uid=1627257&amp;link_uid=PMC2610355" href="/pmc/articles/PMC2610355/"><img src="/pmc/articles/instance/2610355/bin/nihms82917f9.gif" src-large="/pmc/articles/instance/2610355/bin/nihms82917f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2610355/" ref="ordinalpos=2647&amp;ncbi_uid=1627257&amp;link_uid=PMC2610355" image-link="/pmc/articles/PMC2610355/figure/F9/" class="imagepopup">Fig. 9. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of c-FLIP siRNA-triggered spontaneous apoptosis in MCF-7 cells.  From: c-FLIP Knockdown Induces Ligand-independent DR5-, FADD-, Caspase-8-, and Caspase-9-dependent Apoptosis in Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div>In this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, we propose that an apoptotic inhibitory complex (AIC) comprised of DR5, FADD, caspase-8, and c-FLIP<sub>L</sub> exists, and c-FLIP<sub>L</sub> inhibits caspase-8 activation. The knockdown of c-FLIP<sub>L</sub> using a c-FLIP-specific siRNA induces DISC activation by triggering DR5- and FADD-mediated activation of procaspase-8, forming active caspase-8. Active caspase-8 cleaves Bid to form tBid, leading to the release of cytochrome <i>c</i> from the mitochondria. Mitochondrial cytochrome <i>c</i> release activates procaspase-9 producing active caspase-9, which activates downstream caspases-6 and -7 thereby inducing apoptosis. The solid lines indicate protein binding partners and the brackets show protein complexes that comprise the AIC and DISC.</div></p></div><div class="aux"><div class="resc">Travis W. Day, et al. Biochem Pharmacol. ;76(12):1694-1704.</div><p class="links"><a class="dblinks" href="/pubmed/18840411" ref="ordinalpos=2647&amp;ncbi_uid=1627257&amp;link_uid=18840411">Citation</a><a class="dblinks" href="/pmc/articles/PMC2610355/" ref="ordinalpos=2647&amp;ncbi_uid=1627257&amp;link_uid=PMC2610355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2648.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3602516/figure/pone-0059045-g001/" ref="ordinalpos=2648&amp;ncbi_uid=4730872&amp;link_uid=PMC3602516" href="/pmc/articles/PMC3602516/"><img src="/pmc/articles/instance/3602516/bin/pone.0059045.g001.gif" src-large="/pmc/articles/instance/3602516/bin/pone.0059045.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3602516/" ref="ordinalpos=2648&amp;ncbi_uid=4730872&amp;link_uid=PMC3602516" image-link="/pmc/articles/PMC3602516/figure/pone-0059045-g001/" class="imagepopup">Figure 1. Canonical BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Identification of Small Molecule Activators of BMP <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>BMP canonical <span class="highlight" style="background-color:">signaling</span> is transduced by two type I (BMPR-IA and BMPR-IB) and two type II transmembrane serine/threonine kinase receptors, BMPR-II. BMPs bind to the heteromeric complex of type I and type II receptors. Subsequently, the type II receptor phosphorylates the type I receptor, which in turn facilitates phosphorylation of Smad1, 5 (R-Smads). R-Smads directly interact with the activated type I receptor and are released upon phosphorylation. Following release from the receptor complex, R-Smads complex with co-Smad 4 and translocate into the nucleus to modulate the transcription of target genes, including Id1 and Id2. Several natural extracellular proteins including Noggin and Chordin and the inhibitory Smads, Smad6 and Smad7, antagonize the BMP <span class="highlight" style="background-color:">pathway</span>. The small molecule dorsomorphin inhibits phosphorylation of Smads1, 5. Figure adapted from [15].</div></p></div><div class="aux"><div class="resc">Karen Vrijens, et al. PLoS One. 2013;8(3):e59045.</div><p class="links"><a class="dblinks" href="/pubmed/23527084" ref="ordinalpos=2648&amp;ncbi_uid=4730872&amp;link_uid=23527084">Citation</a><a class="dblinks" href="/pmc/articles/PMC3602516/" ref="ordinalpos=2648&amp;ncbi_uid=4730872&amp;link_uid=PMC3602516">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2649.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4118962/figure/pone-0103794-g007/" ref="ordinalpos=2649&amp;ncbi_uid=6462702&amp;link_uid=PMC4118962" href="/pmc/articles/PMC4118962/"><img src="/pmc/articles/instance/4118962/bin/pone.0103794.g007.gif" src-large="/pmc/articles/instance/4118962/bin/pone.0103794.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4118962/" ref="ordinalpos=2649&amp;ncbi_uid=6462702&amp;link_uid=PMC4118962" image-link="/pmc/articles/PMC4118962/figure/pone-0103794-g007/" class="imagepopup">Figure 7. Changes of proteins included in IL-1/TLR-induced NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by western blotting analysis invivo..  From: Interleukin-1/Toll-Like Receptor-Induced Nuclear Factor Kappa B <span class="highlight" style="background-color:">Signaling</span> Participates in Intima Hyperplasia after Carotid Artery Balloon Injury in Goto-Kakizaki Rats: A Potential Target Therapy <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Representative western blotting of TLR4 and IRAK4 (A), EMR1 and IB (B), p-p38 and HuR(C). Bar diagrams depicting the relative protein level of TLR4, IRAK4, EMR1, IB, p-p38 and HuR (D, E, F, G, H, I). Data are represented as mean  SEM. <sup></sup>indicates versus uninjured, p&lt;0.001; *indicates versus Day 3 post-injury, p&lt;0.05, **p&lt;0.01, ***p&lt;0.001;<sup> #</sup>indicates versus Day 7 post-injury, p&lt;0.05, <sup>##</sup>p&lt;0.01, <sup>###</sup>p&lt;0.001.</div></p></div><div class="aux"><div class="resc">Xiaotian Zhang, et al. PLoS One. 2014;9(8):e103794.</div><p class="links"><a class="dblinks" href="/pubmed/25083789" ref="ordinalpos=2649&amp;ncbi_uid=6462702&amp;link_uid=25083789">Citation</a><a class="dblinks" href="/pmc/articles/PMC4118962/" ref="ordinalpos=2649&amp;ncbi_uid=6462702&amp;link_uid=PMC4118962">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2650.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3340233/figure/F2/" ref="ordinalpos=2650&amp;ncbi_uid=4830365&amp;link_uid=PMC3340233" href="/pmc/articles/PMC3340233/"><img src="/pmc/articles/instance/3340233/bin/zbc0201206770002.gif" src-large="/pmc/articles/instance/3340233/bin/zbc0201206770002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3340233/" ref="ordinalpos=2650&amp;ncbi_uid=4830365&amp;link_uid=PMC3340233" image-link="/pmc/articles/PMC3340233/figure/F2/" class="imagepopup">FIGURE 2. From: Induction of Autophagy by Palmitic Acid via Protein Kinase C-mediated <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Independent of mTOR (Mammalian Target of Rapamycin). </a></p><div class="supp"><p class="details"><div><b>PA-induced autophagy is independent of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, MEFs were treated with BSA control or PA (0.25 m<span style="font-variant: small-caps" class="small-caps">m</span>) for various time points as indicated. Cell lysates were collected and subject to Western blot. <i>FM</i>, full medium. <i>B</i>, MEFs were treated with BSA control or PA (0.25 m<span style="font-variant: small-caps" class="small-caps">m</span>) for 4 h in full medium. Cells were also treated with EBSS and the mTOR inhibitor, rapamycin (20 n<span style="font-variant: small-caps" class="small-caps">m</span>), for 4 h as positive controls. CQ (10 <span style="font-variant: small-caps" class="small-caps">m</span>) was added to the treated cells for measuring autophagic flux.</div></p></div><div class="aux"><div class="resc">Shi Hao Tan, et al. J Biol Chem. 2012 April 27;287(18):14364-14376.</div><p class="links"><a class="dblinks" href="/pubmed/22408252" ref="ordinalpos=2650&amp;ncbi_uid=4830365&amp;link_uid=22408252">Citation</a><a class="dblinks" href="/pmc/articles/PMC3340233/" ref="ordinalpos=2650&amp;ncbi_uid=4830365&amp;link_uid=PMC3340233">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2651.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2844221/figure/F6/" ref="ordinalpos=2651&amp;ncbi_uid=2812708&amp;link_uid=PMC2844221" href="/pmc/articles/PMC2844221/"><img src="/pmc/articles/instance/2844221/bin/zbc0131007850006.gif" src-large="/pmc/articles/instance/2844221/bin/zbc0131007850006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2844221/" ref="ordinalpos=2651&amp;ncbi_uid=2812708&amp;link_uid=PMC2844221" image-link="/pmc/articles/PMC2844221/figure/F6/" class="imagepopup">FIGURE 6. From: Differentiation of Embryonic Stem Cells 1 (Dies1) Is a Component of Bone Morphogenetic Protein 4 (BMP4) <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Required for Proper Differentiation of Mouse Embryonic Stem Cells. </a></p><div class="supp"><p class="details"><div><b>Inhibition of Nodal/Activin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> masks the effects of Dies1 silencing in ESCs.</b> <i>A</i>, ESCs stably transfected with NS or Dies1 silencing shRNAs were grown for 6 days in the absence of LIF and then stained for AP. Cells were treated with SB-431542 or with DMSO as a control. % of colonies AP-positive was calculated in triplicate experiments. *, <i>p</i>  0.001. <i>B</i>, Western blot analysis of Oct3/4 and Nanog in the cells treated as described in <i>A</i>.</div></p></div><div class="aux"><div class="resc">Luigi Aloia, et al. J Biol Chem. 2010 March 5;285(10):7776-7783.</div><p class="links"><a class="dblinks" href="/pubmed/20042595" ref="ordinalpos=2651&amp;ncbi_uid=2812708&amp;link_uid=20042595">Citation</a><a class="dblinks" href="/pmc/articles/PMC2844221/" ref="ordinalpos=2651&amp;ncbi_uid=2812708&amp;link_uid=PMC2844221">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2652.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035979/figure/F3/" ref="ordinalpos=2652&amp;ncbi_uid=2759944&amp;link_uid=PMC3035979" href="/pmc/articles/PMC3035979/"><img src="/pmc/articles/instance/3035979/bin/nihms189517f3.gif" src-large="/pmc/articles/instance/3035979/bin/nihms189517f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035979/" ref="ordinalpos=2652&amp;ncbi_uid=2759944&amp;link_uid=PMC3035979" image-link="/pmc/articles/PMC3035979/figure/F3/" class="imagepopup">Figure 3.  From: Genome-wide gene expression profiling in GluR1 knockout mice: Key role of the calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in glutamatergically-mediated hippocampal transmission. </a></p><div class="supp"><p class="details"><div>Protein level evaluation of selected genes altered in GluR1 KO mice that are members of the calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Six GluR1 knockout mice and six WT mice were compared for hippocampal protein levels of a) NMDAR1; b) CACNA1G; c) CAMK2A; and d) MLCK. Protein levels were calibrated with the corresponding actin beta (ACTB) levels for loading amount. Bar graphs are in meanSEM format, *: p&lt; 0.05, <i>t</i>-test, GluR1 KO vs. control.</div></p></div><div class="aux"><div class="resc">Rulun Zhou, et al. Eur J Neurosci. ;30(12):2318-2326.</div><p class="links"><a class="dblinks" href="/pubmed/20092574" ref="ordinalpos=2652&amp;ncbi_uid=2759944&amp;link_uid=20092574">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035979/" ref="ordinalpos=2652&amp;ncbi_uid=2759944&amp;link_uid=PMC3035979">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2653.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1783462/figure/f8/" ref="ordinalpos=2653&amp;ncbi_uid=875326&amp;link_uid=PMC1783462" href="/pmc/articles/PMC1783462/"><img src="/pmc/articles/instance/1783462/bin/7601504f8.gif" src-large="/pmc/articles/instance/1783462/bin/7601504f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1783462/" ref="ordinalpos=2653&amp;ncbi_uid=875326&amp;link_uid=PMC1783462" image-link="/pmc/articles/PMC1783462/figure/f8/" class="imagepopup">Figure 8.  From: FLASH links the CD95 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to the cell nucleus and nuclear bodies. </a></p><div class="supp"><p class="details"><div>Model for the role of FLASH in the CD95 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In response to CD95 activation, DISC-activated caspase-8 triggers a feedforward loop triggering FLASH mobilization from nuclear bodies and its Crm1-dependent nucleo-cytoplasmic translocation. Cytoplasmic FLASH accumulates at the mitochondria, where it interacts with mitochondria-associated caspase-8, thereby regulating its proteolytic activation. CD95-induced FLASH translocation is blocked by FADD-DN expression, by caspase-8 inhibition through zIETD-fmk and by treatment with the nuclear export inhibitor leptomycin B (LMB). E1B19K inhibits CD95-induced caspase-8 activation through trapping FLASH and caspase-8 in a ternary complex at the mitochondria.</div></p></div><div class="aux"><div class="resc">Kristijana Milovic-Holm, et al. EMBO J. 2007 January 24;26(2):391-401.</div><p class="links"><a class="dblinks" href="/pubmed/17245429" ref="ordinalpos=2653&amp;ncbi_uid=875326&amp;link_uid=17245429">Citation</a><a class="dblinks" href="/pmc/articles/PMC1783462/" ref="ordinalpos=2653&amp;ncbi_uid=875326&amp;link_uid=PMC1783462">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2654.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1573867/figure/fig3/" ref="ordinalpos=2654&amp;ncbi_uid=799908&amp;link_uid=PMC1573867" href="/pmc/articles/PMC1573867/"><img src="/pmc/articles/instance/1573867/bin/139-0705266f3.gif" src-large="/pmc/articles/instance/1573867/bin/139-0705266f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1573867/" ref="ordinalpos=2654&amp;ncbi_uid=799908&amp;link_uid=PMC1573867" image-link="/pmc/articles/PMC1573867/figure/fig3/" class="imagepopup">Figure 3.  From: A3 adenosine and CB1 receptors activate a PKC-sensitive Cl current in human nonpigmented ciliary epithelial cells via a G?-coupled MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>A PTX-sensitive G<sub>i/o</sub>/G<i></i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates activation of <i>I</i><sub>Cl,Aden</sub>. Mean <i>I</i><sub>Cl,Aden</sub> (agonist-stimulated Cl<sup></sup> current minus basal Cl<sup></sup> current) recorded at 62 and +58 mV in: (a) NPCE cells before (IB-MECA, <i>n</i>=8) and after PTX treatment (IB-MECA+PTX, <i>n</i>=8), (b) NPCE cells transfected with control vector pIRES2-EGFP (IB-MECA+EGFP, <i>n</i>=6) or pIRES2-EGFP-<i></i>ARK (IB-MECA+EGFP-<i></i>ARK, <i>n</i>=6), (c) NPCE cells before (IB-MECA, <i>n</i>=3) and after 100 n<span style="font-variant: small-caps" class="small-caps">M</span> wortmannin treatment (IB-MECA+Wort, <i>n</i>=5) and (d) NPCE cells before (IB-MECA, <i>n</i>=5) and after 5 <i></i><span style="font-variant: small-caps" class="small-caps">M</span> PD98059 treatment (IB-MECA+PD, <i>n</i>=7). <sup>*</sup><i>P</i>&lt;0.05, <sup>**</sup><i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Chanjuan Shi, et al. Br J Pharmacol. 2003 June;139(3):475-486.</div><p class="links"><a class="dblinks" href="/pubmed/12788807" ref="ordinalpos=2654&amp;ncbi_uid=799908&amp;link_uid=12788807">Citation</a><a class="dblinks" href="/pmc/articles/PMC1573867/" ref="ordinalpos=2654&amp;ncbi_uid=799908&amp;link_uid=PMC1573867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2655.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC23386/figure/F5/" ref="ordinalpos=2655&amp;ncbi_uid=141905&amp;link_uid=PMC23386" href="/pmc/articles/PMC23386/"><img src="/pmc/articles/instance/23386/bin/pq1972355005.gif" src-large="/pmc/articles/instance/23386/bin/pq1972355005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC23386/" ref="ordinalpos=2655&amp;ncbi_uid=141905&amp;link_uid=PMC23386" image-link="/pmc/articles/PMC23386/figure/F5/" class="imagepopup">Figure 5.  From: A possible role for kinase-associated protein phosphatase in the Arabidopsis CLAVATA1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Model of CLV1 and KAPP Action. The model predicts that CLV1, upon binding a peptide ligand, dimerizes and allows intermolecular phosphorylation of the cytoplasmic kinase domains. The activated receptor then binds and phosphorylates KAPP. The KAPP protein may then modify the CLV1 signal by directly dephosphorylating CLV1. The CLV1 kinase domain probably interacts with other positively acting components (X), which may or may not be dephosphorylated by KAPP. Based on epistasis analysis, WUS may function downstream of CLV1 to promote cell divisions. SHOOTMERISTEMLESS may function independently but competitively with the CLV1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote cell division (24).</div></p></div><div class="aux"><div class="resc">Robert W. Williams, et al. Proc Natl Acad Sci U S A. 1997 September 16;94(19):10467-10472.</div><p class="links"><a class="dblinks" href="/pubmed/9294234" ref="ordinalpos=2655&amp;ncbi_uid=141905&amp;link_uid=9294234">Citation</a><a class="dblinks" href="/pmc/articles/PMC23386/" ref="ordinalpos=2655&amp;ncbi_uid=141905&amp;link_uid=PMC23386">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2656.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3953114/figure/pone-0089956-g006/" ref="ordinalpos=2656&amp;ncbi_uid=5922439&amp;link_uid=PMC3953114" href="/pmc/articles/PMC3953114/"><img src="/pmc/articles/instance/3953114/bin/pone.0089956.g006.gif" src-large="/pmc/articles/instance/3953114/bin/pone.0089956.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3953114/" ref="ordinalpos=2656&amp;ncbi_uid=5922439&amp;link_uid=PMC3953114" image-link="/pmc/articles/PMC3953114/figure/pone-0089956-g006/" class="imagepopup">Figure 6. IL-6-DNMT3b <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral cancer..  From: Significance of DNMT3b in Oral Cancer. </a></p><div class="aux"><div class="resc">Wen-Cheng Chen, et al. PLoS One. 2014;9(3):e89956.</div><p class="links"><a class="dblinks" href="/pubmed/24625449" ref="ordinalpos=2656&amp;ncbi_uid=5922439&amp;link_uid=24625449">Citation</a><a class="dblinks" href="/pmc/articles/PMC3953114/" ref="ordinalpos=2656&amp;ncbi_uid=5922439&amp;link_uid=PMC3953114">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2657.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2665046/figure/F6/" ref="ordinalpos=2657&amp;ncbi_uid=1804701&amp;link_uid=PMC2665046" href="/pmc/articles/PMC2665046/"><img src="/pmc/articles/instance/2665046/bin/nihms94225f6.gif" src-large="/pmc/articles/instance/2665046/bin/nihms94225f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2665046/" ref="ordinalpos=2657&amp;ncbi_uid=1804701&amp;link_uid=PMC2665046" image-link="/pmc/articles/PMC2665046/figure/F6/" class="imagepopup">Figure 6. The control of cardiac <span class="highlight" style="background-color:">signaling</span> and hypertrophy by RF-C11.  From: Multivalency-Assisted Control of Intracellular <span class="highlight" style="background-color:">Signaling</span> Pathways: Application for Ubiquitin-Dependent N-End Rule <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Shown is a model where RGS4, RGS5, and RGS16 are cotranslationally degraded through serial Cys-2 modifications (see the main text). In this model, the heterovalent interaction of RF-C11 to the N-recognin family inhibits the degradation of these RGS proteins in cardiomyocytes, leading to their metabolic stabilization and inactivation of G protein <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shashikanth M. Sriram, et al. Chem Biol. ;16(2):121-131.</div><p class="links"><a class="dblinks" href="/pubmed/19246002" ref="ordinalpos=2657&amp;ncbi_uid=1804701&amp;link_uid=19246002">Citation</a><a class="dblinks" href="/pmc/articles/PMC2665046/" ref="ordinalpos=2657&amp;ncbi_uid=1804701&amp;link_uid=PMC2665046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2658.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4017430/figure/F0001/" ref="ordinalpos=2658&amp;ncbi_uid=6135520&amp;link_uid=PMC4017430" href="/pmc/articles/PMC4017430/"><img src="/pmc/articles/instance/4017430/bin/GHFBB-4-192-g001.gif" src-large="/pmc/articles/instance/4017430/bin/GHFBB-4-192-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4017430/" ref="ordinalpos=2658&amp;ncbi_uid=6135520&amp;link_uid=PMC4017430" image-link="/pmc/articles/PMC4017430/figure/F0001/" class="imagepopup">Figure 1.  From: Association of leptin receptor gene Gln223Arg polymorphism with susceptibility to colorectal cancer. </a></p><div class="supp"><p class="details"><div>Leptin receptor <span class="highlight" style="background-color:">signaling</span>. Leptin binding to the long isoform of the leptin receptor results in autophosphorylation and activation of JAK2, which then leads to phosphorylation of tyrosine residues located in the intracellular domain of the leptin receptor (pY-985 and pY-1138). pY-1138 recruits the transcription factor STAT3, which is appears to be crucial in the action of leptin on energy expenditure. STAT3 activation also results in the transcription of the <span class="highlight" style="background-color:">signaling</span> inhibitor SOCS3. Leptin receptor <span class="highlight" style="background-color:">signaling</span> also results in the activation of the IRS-phosphorylationPI3-kinase <span class="highlight" style="background-color:">pathway</span>, probably through phosphorylation sites on JAK2. SOCS3 attenuates leptin receptor <span class="highlight" style="background-color:">signaling</span> by binding to pY-985 and to JAK2.</div></p></div><div class="aux"><div class="resc">Khatoon Karimi, et al. Gastroenterol Hepatol Bed Bench. 2011 Autumn;4(4):192-198.</div><p class="links"><a class="dblinks" href="/pubmed/24834182" ref="ordinalpos=2658&amp;ncbi_uid=6135520&amp;link_uid=24834182">Citation</a><a class="dblinks" href="/pmc/articles/PMC4017430/" ref="ordinalpos=2658&amp;ncbi_uid=6135520&amp;link_uid=PMC4017430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2659.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2669422/figure/F7/" ref="ordinalpos=2659&amp;ncbi_uid=1866519&amp;link_uid=PMC2669422" href="/pmc/articles/PMC2669422/"><img src="/pmc/articles/instance/2669422/bin/z380050972630007.gif" src-large="/pmc/articles/instance/2669422/bin/z380050972630007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2669422/" ref="ordinalpos=2659&amp;ncbi_uid=1866519&amp;link_uid=PMC2669422" image-link="/pmc/articles/PMC2669422/figure/F7/" class="imagepopup">Figure 7. From: GADD45a is a novel candidate gene in inflammatory lung injury via influences on Akt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Expression profiling reveals a signature of increased B-cell receptor <span class="highlight" style="background-color:">signaling</span> dysregulation in VILI-challenged <i>GADD45a</i><sup>/</sup> mice. IPA reveals canonical pathways enriched in dysregulated genes in VILI-challenged groups relative to spontaneously breathing WT control lungs. Dashed line represents threshold level for significance (1.3, negative log of <i>P</i> value 0.05 by Fishers exact test). The B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> exhibited the greatest dysregulation in the VILI-exposed <i>GADD45a</i><sup>/</sup> group, with components that include members of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nuala J. Meyer, et al. FASEB J. 2009 May;23(5):1325-1337.</div><p class="links"><a class="dblinks" href="/pubmed/19124556" ref="ordinalpos=2659&amp;ncbi_uid=1866519&amp;link_uid=19124556">Citation</a><a class="dblinks" href="/pmc/articles/PMC2669422/" ref="ordinalpos=2659&amp;ncbi_uid=1866519&amp;link_uid=PMC2669422">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2660.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2173691/figure/fig7/" ref="ordinalpos=2660&amp;ncbi_uid=1142635&amp;link_uid=PMC2173691" href="/pmc/articles/PMC2173691/"><img src="/pmc/articles/instance/2173691/bin/200308162f7.gif" src-large="/pmc/articles/instance/2173691/bin/200308162f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2173691/" ref="ordinalpos=2660&amp;ncbi_uid=1142635&amp;link_uid=PMC2173691" image-link="/pmc/articles/PMC2173691/figure/fig7/" class="imagepopup">Figure 7. From: Autoregulation of E-cadherin expression by cadherin-cadherin interactions. </a></p><div class="supp"><p class="details"><div><b>Mechanisms of E-cadherin regulation in sparse and dense cultures of colon cancer cells.</b> In sparse cultures, E-cadherin expression is suppressed by two different mechanisms. One involves activation of ERK (P-ERK) by receptor tyrosine kinases (RTK), such as ErbB-1 and ErbB-2. ERK activation induces <i>Slug</i> that inhibits E-cadherin expression by binding to the E-box of <i>E-cadherin</i>. The other <span class="highlight" style="background-color:">pathway</span> includes <i>Slug</i> induction by the -cateninTCF <span class="highlight" style="background-color:">pathway</span>. In dense cultures, the ERK <span class="highlight" style="background-color:">pathway</span> is inactive, the levels of ErbB-1, ErbB-2, and Slug are repressed and E-cadherin expression increases. This leads to recruitment of nuclear -catenin to adherens junctions together with E-cadherin and the inhibition of -catenin <span class="highlight" style="background-color:">signaling</span>. This further reduces Slug expression and relieves the inhibition on <i>E-cadherin</i> transcription resulting in elevated E-cadherin expression.</div></p></div><div class="aux"><div class="resc">Maralice Conacci-Sorrell, et al. J Cell Biol. 2003 November 24;163(4):847-857.</div><p class="links"><a class="dblinks" href="/pubmed/14623871" ref="ordinalpos=2660&amp;ncbi_uid=1142635&amp;link_uid=14623871">Citation</a><a class="dblinks" href="/pmc/articles/PMC2173691/" ref="ordinalpos=2660&amp;ncbi_uid=1142635&amp;link_uid=PMC2173691">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2661.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4389564/figure/F1/" ref="ordinalpos=2661&amp;ncbi_uid=7448020&amp;link_uid=PMC4389564" href="/pmc/articles/PMC4389564/"><img src="/pmc/articles/instance/4389564/bin/fimmu-06-00154-g001.gif" src-large="/pmc/articles/instance/4389564/bin/fimmu-06-00154-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4389564/" ref="ordinalpos=2661&amp;ncbi_uid=7448020&amp;link_uid=PMC4389564" image-link="/pmc/articles/PMC4389564/figure/F1/" class="imagepopup">Figure 1.  From: CD44 Acts as a <span class="highlight" style="background-color:">Signaling</span> Platform Controlling Tumor Progression and Metastasis. </a></p><div class="supp"><p class="details"><div><b>CD44 acts as a co-receptor for several cell surface receptors including RTKs, G-protein coupled receptors, and LRP6</b>. Hyaluronan binding to CD44 increases CXCL12-induced CXCR4-G Protein <span class="highlight" style="background-color:">signaling</span> (18). RTKs activation by their ligands [growth factors (GFs) such as HGF or VEGF] and downstream <span class="highlight" style="background-color:">signaling</span> are dependent on CD44v6 [reviewed in Ref. (11)]. In the Wnt--catenin <span class="highlight" style="background-color:">pathway</span>, LRP6 recruitment of CD44 leads to -catenin activation and translocation to the nucleus (3).</div></p></div><div class="aux"><div class="resc">Vronique Orian-Rousseau. Front Immunol. 2015;6:154.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2661&amp;ncbi_uid=7448020&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4389564/" ref="ordinalpos=2661&amp;ncbi_uid=7448020&amp;link_uid=PMC4389564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2662.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507493/figure/F14/" ref="ordinalpos=2662&amp;ncbi_uid=5431122&amp;link_uid=PMC3507493" href="/pmc/articles/PMC3507493/"><img src="/pmc/articles/instance/3507493/bin/cc-11-3972-g14.gif" src-large="/pmc/articles/instance/3507493/bin/cc-11-3972-g14.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507493/" ref="ordinalpos=2662&amp;ncbi_uid=5431122&amp;link_uid=PMC3507493" image-link="/pmc/articles/PMC3507493/figure/F14/" class="imagepopup">Figure. From: The milk protein ?-casein functions as a tumor suppressor via activation of STAT1 <span class="highlight" style="background-color:">signaling</span>, effectively preventing breast cancer tumor growth and metastasis. </a></p><div class="supp"><p class="details"><div><b>Figure 14.</b> -casein behaves as a <span class="highlight" style="background-color:">signaling</span> molecule. Mechanistically, -casein functionally induced interferon-associated STAT1-<span class="highlight" style="background-color:">signaling</span> and reduced the stemness of mammary tumor cells. Thus, -casein may mediate its tumor suppressor effects by conferring a more differentiated mammary cell phenotype. These findings define a previously unrecognized signal transduction <span class="highlight" style="background-color:">pathway</span> and identify -casein as a <span class="highlight" style="background-color:">signaling</span> molecule.</div></p></div><div class="aux"><div class="resc">Gloria Bonuccelli, et al. Cell Cycle. 2012 November 1;11(21):3972-3982.</div><p class="links"><a class="dblinks" href="/pubmed/23047602" ref="ordinalpos=2662&amp;ncbi_uid=5431122&amp;link_uid=23047602">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507493/" ref="ordinalpos=2662&amp;ncbi_uid=5431122&amp;link_uid=PMC3507493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2663.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000931/figure/fig2/" ref="ordinalpos=2663&amp;ncbi_uid=6111683&amp;link_uid=PMC4000931" href="/pmc/articles/PMC4000931/"><img src="/pmc/articles/instance/4000931/bin/BMRI2014-797898.002.gif" src-large="/pmc/articles/instance/4000931/bin/BMRI2014-797898.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000931/" ref="ordinalpos=2663&amp;ncbi_uid=6111683&amp;link_uid=PMC4000931" image-link="/pmc/articles/PMC4000931/figure/fig2/" class="imagepopup">Figure 2.  From: Therapeutic Potential of MicroRNA: A New Target to Treat Intrahepatic Portal Hypertension?. </a></p><div class="supp"><p class="details"><div>Summary of the miRNAs and the TGF-<i></i> <span class="highlight" style="background-color:">pathway</span> involved in HSCs. The particular TGF-<i></i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affected by miRNAs is described in detail in the review.</div></p></div><div class="aux"><div class="resc">Can-Jie Guo, et al. Biomed Res Int. 2014;2014:797898.</div><p class="links"><a class="dblinks" href="/pubmed/24812632" ref="ordinalpos=2663&amp;ncbi_uid=6111683&amp;link_uid=24812632">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000931/" ref="ordinalpos=2663&amp;ncbi_uid=6111683&amp;link_uid=PMC4000931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2664.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2442100/figure/F10/" ref="ordinalpos=2664&amp;ncbi_uid=1398086&amp;link_uid=PMC2442100" href="/pmc/articles/PMC2442100/"><img src="/pmc/articles/instance/2442100/bin/1471-2105-9-276-10.gif" src-large="/pmc/articles/instance/2442100/bin/1471-2105-9-276-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2442100/" ref="ordinalpos=2664&amp;ncbi_uid=1398086&amp;link_uid=PMC2442100" image-link="/pmc/articles/PMC2442100/figure/F10/" class="imagepopup">Figure 10.  From: Discovering functional interaction patterns in protein-protein interaction networks. </a></p><div class="supp"><p class="details"><div><b>A functional interaction pattern related to the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. A frequent functional interaction pattern which has a supporting embedding that completely overlaps with the MAPK signalling <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mehmet E Turanalp, et al. BMC Bioinformatics. 2008;9:276-276.</div><p class="links"><a class="dblinks" href="/pubmed/18547430" ref="ordinalpos=2664&amp;ncbi_uid=1398086&amp;link_uid=18547430">Citation</a><a class="dblinks" href="/pmc/articles/PMC2442100/" ref="ordinalpos=2664&amp;ncbi_uid=1398086&amp;link_uid=PMC2442100">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2665.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013405/figure/cjc-30-08-551-g003/" ref="ordinalpos=2665&amp;ncbi_uid=6134429&amp;link_uid=PMC4013405" href="/pmc/articles/PMC4013405/"><img src="/pmc/articles/instance/4013405/bin/cjc-30-08-551-g003.gif" src-large="/pmc/articles/instance/4013405/bin/cjc-30-08-551-g003.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013405/" ref="ordinalpos=2665&amp;ncbi_uid=6134429&amp;link_uid=PMC4013405" image-link="/pmc/articles/PMC4013405/figure/cjc-30-08-551-g003/" class="imagepopup">Figure 3.nAChR-mediated <span class="highlight" style="background-color:">signaling</span> in lung squamous cell carcinomas and adenocarcinomas..  From: Lung carcinoma <span class="highlight" style="background-color:">signaling</span> pathways activated by smoking. </a></p><div class="supp"><p class="details"><div>nAChR-mediated mitogen-activated protein kinase (MAPK) activation in squamous cell carcinoma and adenocarcinoma is mediated by the scaffolding protein -arrestin, which binds to and recruits <i>Src</i> to the receptors and stimulates the Raf/MAPK <span class="highlight" style="background-color:">pathway</span>. MAPK activation functions with the PI3K/Akt and the EGFR/Raf/MAPK <span class="highlight" style="background-color:">signaling</span> pathways, both activated by Ca<sup>2+</sup> influx through nAChRs, to induce phosphorylation and activation of transcription factors, further regulate downstream gene transcription and protein expression, provoke cell proliferation, anti-apoptosis, tumor invasion, and therapy resistance.</div></p></div><div class="aux"><div class="resc">Jing Wen, et al. Chin J Cancer. 2011 August;30(8):551-558.</div><p class="links"><a class="dblinks" href="/pubmed/21801603" ref="ordinalpos=2665&amp;ncbi_uid=6134429&amp;link_uid=21801603">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013405/" ref="ordinalpos=2665&amp;ncbi_uid=6134429&amp;link_uid=PMC4013405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2666.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3951857/figure/F1/" ref="ordinalpos=2666&amp;ncbi_uid=5913796&amp;link_uid=PMC3951857" href="/pmc/articles/PMC3951857/"><img src="/pmc/articles/instance/3951857/bin/nihms332125f1.gif" src-large="/pmc/articles/instance/3951857/bin/nihms332125f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3951857/" ref="ordinalpos=2666&amp;ncbi_uid=5913796&amp;link_uid=PMC3951857" image-link="/pmc/articles/PMC3951857/figure/F1/" class="imagepopup">Fig. 1.  From: Diseases of Wnt <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Wnt binds to the Lrp5 or 6 and Frizzled coreceptor complex which leads to the intracellular accumulation of -catenin. -catenin translocates to the nucleus where it binds to TCF/Lef transcription factors and regulates the expression of a number of target genes. Mutations in several components of the <span class="highlight" style="background-color:">pathway</span> have been shown to result in disease. Mutations in <i>LRP5</i> can result in diseases of increased or decreased bone mass and vision problems. The Lrp5 receptor and Wnt <span class="highlight" style="background-color:">pathway</span> may also play a key role in aortic valve calcification. Mutations in <i>APC</i>, <i>-catenin</i> and <i>Axin</i> have been shown to give rise to a number of different types of cancer. Downstream gene targets of -catenin such as osteoprotegerin (<i>OPG</i>) and the secreted frizzled related proteins (<i>sFRPs</i>) are known to regulate osteoclastogenesis and may play a role in metastatic bone diseases that result in osteolytic lesions. Mutations in modulators of Wnt <span class="highlight" style="background-color:">signaling</span> such as the <i>SOST</i> gene, which produces sclerostin, give rise to the increased bone mass disease, sclerosteosis. These are just a few examples of the variety of diseases that have been attributed to the Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mark L. Johnson, et al. Rev Endocr Metab Disord. 2006 June;7(0):41-49.</div><p class="links"><a class="dblinks" href="/pubmed/16944325" ref="ordinalpos=2666&amp;ncbi_uid=5913796&amp;link_uid=16944325">Citation</a><a class="dblinks" href="/pmc/articles/PMC3951857/" ref="ordinalpos=2666&amp;ncbi_uid=5913796&amp;link_uid=PMC3951857">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2667.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351446/figure/pone-0036975-g003/" ref="ordinalpos=2667&amp;ncbi_uid=4008403&amp;link_uid=PMC3351446" href="/pmc/articles/PMC3351446/"><img src="/pmc/articles/instance/3351446/bin/pone.0036975.g003.gif" src-large="/pmc/articles/instance/3351446/bin/pone.0036975.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351446/" ref="ordinalpos=2667&amp;ncbi_uid=4008403&amp;link_uid=PMC3351446" image-link="/pmc/articles/PMC3351446/figure/pone-0036975-g003/" class="imagepopup">Figure 3. KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analysis of aging-related hypothalamic proteins..  From: GIT2 Acts as a Potential Keystone Protein in Functional Hypothalamic Networks Associated with Age-Related Phenotypic Changes in Rats. </a></p><div class="supp"><p class="details"><div>Proteins significantly regulated in middle (M) or old (O) aged animals compared to young (Y) animals were used as input data for KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> population analysis. (A) Venn diagram analysis of middle-aged (grey line) and old-aged (black line) significantly-regulated KEGG pathways demonstrated 27 common KEGG terms between old and middle-aged tissues. The common (27) significantly populated pathways for middle-aged (grey bars) and old aged (black bars) animals were then rationally clustered into subgroups focused upon disease pathology (B), neurophysiological activity (C), and intermediary cellular <span class="highlight" style="background-color:">signaling</span> activity (D). For each significantly-populated KEGG <span class="highlight" style="background-color:">pathway</span> a hybrid score was generated which represents the log<sub>10</sub> of the enrichment probability multiplied with the relative enrichment factor compared to the background proteomic expression.</div></p></div><div class="aux"><div class="resc">Wayne Chadwick, et al. PLoS One. 2012;7(5):e36975.</div><p class="links"><a class="dblinks" href="/pubmed/22606319" ref="ordinalpos=2667&amp;ncbi_uid=4008403&amp;link_uid=22606319">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351446/" ref="ordinalpos=2667&amp;ncbi_uid=4008403&amp;link_uid=PMC3351446">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2668.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382892/figure/F2/" ref="ordinalpos=2668&amp;ncbi_uid=4096925&amp;link_uid=PMC3382892" href="/pmc/articles/PMC3382892/"><img src="/pmc/articles/instance/3382892/bin/onci-1-516-g2.gif" src-large="/pmc/articles/instance/3382892/bin/onci-1-516-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382892/" ref="ordinalpos=2668&amp;ncbi_uid=4096925&amp;link_uid=PMC3382892" image-link="/pmc/articles/PMC3382892/figure/F2/" class="imagepopup">Figure. From: Th17 cells have stem cell-like features and promote long-term immunity. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> Th17 stemness, HIF/Notch/Bcl2 <span class="highlight" style="background-color:">signaling</span> and T cell immunity. Human Th17 cells are resistant to apoptosis and long-lived memory T cells. Th17 cells express high levels of HIF-1, Notch <span class="highlight" style="background-color:">signaling</span> molecules and Bcl2, and multiple stem cell core genes. HIF/Notch/Bcl2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controls Th17 cell survival, and promote Th17 cell-mediated T cell immunity. Th17 cells are polyfunctional and highly plastic in the chronic inflammatory environment, and would be converted into regulatory T cell-type and Th-1 type effector cells. It remains unknown if HIF/Notch/Bcl2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates Th17 cell plasticity.</div></p></div><div class="aux"><div class="resc">Shuang Wei, et al. Oncoimmunology. 2012 July 1;1(4):516-519.</div><p class="links"><a class="dblinks" href="/pubmed/22754771" ref="ordinalpos=2668&amp;ncbi_uid=4096925&amp;link_uid=22754771">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382892/" ref="ordinalpos=2668&amp;ncbi_uid=4096925&amp;link_uid=PMC3382892">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2669.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1815248/figure/F5/" ref="ordinalpos=2669&amp;ncbi_uid=974855&amp;link_uid=PMC1815248" href="/pmc/articles/PMC1815248/"><img src="/pmc/articles/instance/1815248/bin/zpq0070752640005.gif" src-large="/pmc/articles/instance/1815248/bin/zpq0070752640005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1815248/" ref="ordinalpos=2669&amp;ncbi_uid=974855&amp;link_uid=PMC1815248" image-link="/pmc/articles/PMC1815248/figure/F5/" class="imagepopup">Fig. 5. From: Basement membrane remodeling is essential for Drosophila disc eversion and tumor invasion. </a></p><div class="supp"><p class="details"><div>Co-option of a developmental <span class="highlight" style="background-color:">pathway</span> by invasive tumors. A model summarizing the relationships between JNK <span class="highlight" style="background-color:">pathway</span>, MMPs, TIMP, and RECK and their role in BM degradation during the developmental invasive process of disc eversion and tumor invasion. During the disc eversion process, both MMP1 and MMP2 are under the control of the JNK <span class="highlight" style="background-color:">pathway</span> and result in BM degradation. In tumor invasion, part of the developmental <span class="highlight" style="background-color:">pathway</span> is hijacked to affect BM degradation and invasiveness. MMP1 and MMP2 function is inhibited by TIMP and/or RECK. It is possible that RECK could also block other proteases activated by <i>Ras<sup>V12</sup></i> and/or <i>scrib</i><sup>/</sup>, which are independent from JNK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Ajay Srivastava, et al. Proc Natl Acad Sci U S A. 2007 February 20;104(8):2721-2726.</div><p class="links"><a class="dblinks" href="/pubmed/17301221" ref="ordinalpos=2669&amp;ncbi_uid=974855&amp;link_uid=17301221">Citation</a><a class="dblinks" href="/pmc/articles/PMC1815248/" ref="ordinalpos=2669&amp;ncbi_uid=974855&amp;link_uid=PMC1815248">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2670.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763651/figure/pharmaceuticals-05-00417-f001/" ref="ordinalpos=2670&amp;ncbi_uid=5486562&amp;link_uid=PMC3763651" href="/pmc/articles/PMC3763651/"><img src="/pmc/articles/instance/3763651/bin/pharmaceuticals-05-00417-g001.gif" src-large="/pmc/articles/instance/3763651/bin/pharmaceuticals-05-00417-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763651/" ref="ordinalpos=2670&amp;ncbi_uid=5486562&amp;link_uid=PMC3763651" image-link="/pmc/articles/PMC3763651/figure/pharmaceuticals-05-00417-f001/" class="imagepopup">Figure 1.  From: Pathophysiology of GPCR Homo- and Heterodimerization: Special Emphasis on Somatostatin Receptors. </a></p><div class="supp"><p class="details"><div>Schematic presentation depicting the regulation of receptor <span class="highlight" style="background-color:">signaling</span> through G-proteins. GPCRs upon activation couples to G<sub>s</sub> (<b>A</b>), G<sub>i</sub> (<b>B</b>), G<sub>q</sub> (<b>C</b>) or G<sub>12</sub> (<b>D</b>) and regulates <span class="highlight" style="background-color:">signaling</span> in specific manner. Downstream <span class="highlight" style="background-color:">signaling</span> involving G<sub></sub> subunit is depicted in panel A [78,79,81]. Figure was made by using the online <span class="highlight" style="background-color:">pathway</span> builder from Protein Lounge [126].</div></p></div><div class="aux"><div class="resc">Rishi K. Somvanshi, et al. Pharmaceuticals (Basel). 2012 May;5(5):417-446.</div><p class="links"><a class="dblinks" href="/pubmed/24281555" ref="ordinalpos=2670&amp;ncbi_uid=5486562&amp;link_uid=24281555">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763651/" ref="ordinalpos=2670&amp;ncbi_uid=5486562&amp;link_uid=PMC3763651">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2671.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3657702/figure/F1/" ref="ordinalpos=2671&amp;ncbi_uid=4937506&amp;link_uid=PMC3657702" href="/pmc/articles/PMC3657702/"><img src="/pmc/articles/instance/3657702/bin/fimmu-04-00121-g001.gif" src-large="/pmc/articles/instance/3657702/bin/fimmu-04-00121-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3657702/" ref="ordinalpos=2671&amp;ncbi_uid=4937506&amp;link_uid=PMC3657702" image-link="/pmc/articles/PMC3657702/figure/F1/" class="imagepopup">Figure 1.  From: T Cell Aging: A Review of the Transcriptional Changes Determined from Genome-Wide Analysis. </a></p><div class="supp"><p class="details"><div><b>Age-related alteration of gene expression in the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The age-related genes were assembled from the literature based on the significant changes reported by the original articles which were applied for all figures (Chen et al., 2003; Mo et al., 2003; Yung and Mo, 2003; Fann et al., 2005; Czesnikiewicz-Guzik et al., 2008; Henson and Akbar, 2009; Lazuardi et al., 2009; Weng et al., 2009; Cao et al., 2010; Hernandez-Lopez et al., 2010; Alonso-Arias et al., 2011; Mirza et al., 2011; Cane et al., 2012). Three-hundred five up-regulated and 472 down-regulated genes in aged human T cells including CD4 and CD8 T cells were analyzed using KEGG analysis in the WEB-based GEne SeT AnaLysis Toolkit (WebGestalt). The entire TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is included and the molecules with lower gene expression compared with young are highlighted in red text and different border color indicating specific T cell subsets. The altered expressed genes in the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> include <i>CD3G</i>, <i>LCK</i>, <i>GRAP2</i> (encode GADS), <i>GRB2</i>, <i>ITK</i>, <i>NCK</i>, MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules <i>SOS2</i> and <i>MAPK1</i> (encode Erk), <i>CD28</i>, PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules <i>PI3K</i>, <i>PDK1</i>, and <i>AKT</i>, DLG1 (encode DLGH1) and <i>MAPK14</i> (encode p38).</div></p></div><div class="aux"><div class="resc">Guobing Chen, et al. Front Immunol. 2013;4:121.</div><p class="links"><a class="dblinks" href="/pubmed/23730304" ref="ordinalpos=2671&amp;ncbi_uid=4937506&amp;link_uid=23730304">Citation</a><a class="dblinks" href="/pmc/articles/PMC3657702/" ref="ordinalpos=2671&amp;ncbi_uid=4937506&amp;link_uid=PMC3657702">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2672.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2773643/figure/A002949F1/" ref="ordinalpos=2672&amp;ncbi_uid=2077599&amp;link_uid=PMC2773643" href="/pmc/articles/PMC2773643/"><img src="/pmc/articles/instance/2773643/bin/cshperspect-JCT-002949_F1.gif" src-large="/pmc/articles/instance/2773643/bin/cshperspect-JCT-002949_F1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2773643/" ref="ordinalpos=2672&amp;ncbi_uid=2077599&amp;link_uid=PMC2773643" image-link="/pmc/articles/PMC2773643/figure/A002949F1/" class="imagepopup">Figure 1. From: Adhesive and <span class="highlight" style="background-color:">Signaling</span> Functions of Cadherins and Catenins in Vertebrate Development. </a></p><div class="supp"><p class="details"><div>Cadherincatenin proteins and their connections to developmental cell <span class="highlight" style="background-color:">signaling</span> pathways. Diagram showing major proteins involved in formation of cadherin, -, -, -, and p120-catenin complexes at the plasma membranes of two juxtaposed cells. Although cadherins are implicated in regulation of receptor-type tyrosine kinases (RTKs) and protection from apoptosis, catenins have multiple overlapping and distinct functions. Both p120-catenin and -catenin regulate NFB and MAPK <span class="highlight" style="background-color:">signaling</span> pathways, cytokinesis, and cell proliferation. However, p120-catenin has a unique function in the control of RhoA <span class="highlight" style="background-color:">signaling</span> and microtubule cytoskeleton organization, whereas -catenin is implicated in regulation of RTKs, Hedgehog <span class="highlight" style="background-color:">signaling</span>, microtubule-mediated traffic, and functional coupling between the junctions and the actin cytoskeleton. -catenin is a principal player in the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Because RTKs, Hedgehog, and Wnt <span class="highlight" style="background-color:">signaling</span> mechanisms are heavily implicated in regulation of normal development, intriguing connections between cadherincatenin receptors and these pathways provides potential tools for integrating the efforts of individual cells into a smoothly orchestrated developmental process.</div></p></div><div class="aux"><div class="resc">Ewa Stepniak, et al. Cold Spring Harb Perspect Biol. 2009 November;1(5):a002949.</div><p class="links"><a class="dblinks" href="/pubmed/20066120" ref="ordinalpos=2672&amp;ncbi_uid=2077599&amp;link_uid=20066120">Citation</a><a class="dblinks" href="/pmc/articles/PMC2773643/" ref="ordinalpos=2672&amp;ncbi_uid=2077599&amp;link_uid=PMC2773643">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2673.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4017424/figure/F0001/" ref="ordinalpos=2673&amp;ncbi_uid=6134659&amp;link_uid=PMC4017424" href="/pmc/articles/PMC4017424/"><img src="/pmc/articles/instance/4017424/bin/GHFBB-4-127-g001.gif" src-large="/pmc/articles/instance/4017424/bin/GHFBB-4-127-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4017424/" ref="ordinalpos=2673&amp;ncbi_uid=6134659&amp;link_uid=PMC4017424" image-link="/pmc/articles/PMC4017424/figure/F0001/" class="imagepopup">Figure 1.  From: Colorectal cancer and NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>NF-B <span class="highlight" style="background-color:">signaling</span> pathways</div></p></div><div class="aux"><div class="resc">Parichehr Hassanzadeh. Gastroenterol Hepatol Bed Bench. 2011 Summer;4(3):127-132.</div><p class="links"><a class="dblinks" href="/pubmed/24834170" ref="ordinalpos=2673&amp;ncbi_uid=6134659&amp;link_uid=24834170">Citation</a><a class="dblinks" href="/pmc/articles/PMC4017424/" ref="ordinalpos=2673&amp;ncbi_uid=6134659&amp;link_uid=PMC4017424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2674.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4145234/figure/Fig1/" ref="ordinalpos=2674&amp;ncbi_uid=6537191&amp;link_uid=PMC4145234" href="/pmc/articles/PMC4145234/"><img src="/pmc/articles/instance/4145234/bin/12967_2014_231_Fig1_HTML.gif" src-large="/pmc/articles/instance/4145234/bin/12967_2014_231_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4145234/" ref="ordinalpos=2674&amp;ncbi_uid=6537191&amp;link_uid=PMC4145234" image-link="/pmc/articles/PMC4145234/figure/Fig1/" class="imagepopup">Figure 1.  From: The impact of microRNA-mediated PI3K/AKT <span class="highlight" style="background-color:">signaling</span> on epithelial-mesenchymal transition and cancer stemness in endometrial cancer. </a></p><div class="supp"><p class="details"><div> <b>Regulatory networks of miRNAs and PI3K/AKT <span class="highlight" style="background-color:">pathway</span> in controlling EMT and cancer stemness in human tumors including endometrial cancer.</b> Activation of PI3K/AKT <span class="highlight" style="background-color:">signaling</span> plays an essential role in promoting EMT and cancer stem cell phenotypes. Interactions between PI3K/AKT and other pathways regulate EMT/CSC. MiRNAs function as both upstream mediators and downstream effectors to affect PI3K/AKT <span class="highlight" style="background-color:">pathway</span> activities. References showing the regulatory interactions are indicated. Verified interactions in endometrial cancer were underlined.</div></p></div><div class="aux"><div class="resc">Peixin Dong, et al. J Transl Med. 2014;12(1):231.</div><p class="links"><a class="dblinks" href="/pubmed/25141911" ref="ordinalpos=2674&amp;ncbi_uid=6537191&amp;link_uid=25141911">Citation</a><a class="dblinks" href="/pmc/articles/PMC4145234/" ref="ordinalpos=2674&amp;ncbi_uid=6537191&amp;link_uid=PMC4145234">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2675.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3400851/figure/F1/" ref="ordinalpos=2675&amp;ncbi_uid=4133354&amp;link_uid=PMC3400851" href="/pmc/articles/PMC3400851/"><img src="/pmc/articles/instance/3400851/bin/WJG-18-3502-g001.gif" src-large="/pmc/articles/instance/3400851/bin/WJG-18-3502-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3400851/" ref="ordinalpos=2675&amp;ncbi_uid=4133354&amp;link_uid=PMC3400851" image-link="/pmc/articles/PMC3400851/figure/F1/" class="imagepopup">Figure 1.  From: Significance of regenerating islet-derived type IV gene expression in gastroenterological cancers. </a></p><div class="supp"><p class="details"><div>Schematic representation of the regenerating islet-derived type IV <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Regenerating islet-derived type IV (RegIV) contributes to cell proliferation, inhibition of apoptosis, and carcinogenesis <i>via</i> protein kinase B (Akt) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by transactivating the epidermal growth factor receptor (EGFR). The primary <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is now being investigated. MBS: Mannan-binding sites; PI3K: Phosphoinositide 3-kinase; PIP2: Phosphatidylinositol 4,5-bisphosphate; PIP3: Phosphatidylinositol 4,5-triphosphate; PDK1: Phosphoinositide-dependent kinase-1; Bcl-2: B-cell lymphoma 2.</div></p></div><div class="aux"><div class="resc">Masakatsu Numata, et al. World J Gastroenterol. 2012 July 21;18(27):3502-3510.</div><p class="links"><a class="dblinks" href="/pubmed/22826614" ref="ordinalpos=2675&amp;ncbi_uid=4133354&amp;link_uid=22826614">Citation</a><a class="dblinks" href="/pmc/articles/PMC3400851/" ref="ordinalpos=2675&amp;ncbi_uid=4133354&amp;link_uid=PMC3400851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2676.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2582243/figure/F4/" ref="ordinalpos=2676&amp;ncbi_uid=1581615&amp;link_uid=PMC2582243" href="/pmc/articles/PMC2582243/"><img src="/pmc/articles/instance/2582243/bin/1471-2407-8-302-4.gif" src-large="/pmc/articles/instance/2582243/bin/1471-2407-8-302-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2582243/" ref="ordinalpos=2676&amp;ncbi_uid=1581615&amp;link_uid=PMC2582243" image-link="/pmc/articles/PMC2582243/figure/F4/" class="imagepopup">Figure 4.  From: STAT3 can be activated through paracrine <span class="highlight" style="background-color:">signaling</span> in breast epithelial cells. </a></p><div class="supp"><p class="details"><div><b>Activation of STAT3 occurs through the JAK/STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> A. MCF-10A cells were treated with MDA-MB-468 conditioned media for 0.5 hours. Treatment with 10 M JSI-124 one hour prior to treatment with conditioned media reduced STAT3 activation. B. MCF-10A cells were treated with MDA-MB-231 conditioned media for 0.5 hours. Treatment with 10 M JSI-124 one hour prior to treatment with conditioned media reduced STAT3 activation. C. MDA-MB-453 cells were treated with MDA-MB-231 conditioned media for 0.5 hours. Treatment with 10 M JSI-124 one hour prior to treatment with conditioned media reduced STAT3 activation. 468: MDA-MB-468; 231: MDA-MB-231; 453: MDA-MB-453; No Tx: no treatment; CM: conditioned media.</div></p></div><div class="aux"><div class="resc">Jacqueline C Lieblein, et al. BMC Cancer. 2008;8:302-302.</div><p class="links"><a class="dblinks" href="/pubmed/18939993" ref="ordinalpos=2676&amp;ncbi_uid=1581615&amp;link_uid=18939993">Citation</a><a class="dblinks" href="/pmc/articles/PMC2582243/" ref="ordinalpos=2676&amp;ncbi_uid=1581615&amp;link_uid=PMC2582243">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2677.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3431500/figure/F1/" ref="ordinalpos=2677&amp;ncbi_uid=4229074&amp;link_uid=PMC3431500" href="/pmc/articles/PMC3431500/"><img src="/pmc/articles/instance/3431500/bin/fonc-02-00109-g001.gif" src-large="/pmc/articles/instance/3431500/bin/fonc-02-00109-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3431500/" ref="ordinalpos=2677&amp;ncbi_uid=4229074&amp;link_uid=PMC3431500" image-link="/pmc/articles/PMC3431500/figure/F1/" class="imagepopup">Figure 1.  From: Genomic Determinants of PI3K <span class="highlight" style="background-color:">Pathway</span> Inhibitor Response in Cancer. </a></p><div class="supp"><p class="details"><div><b>Class I PI3K signal transduction <span class="highlight" style="background-color:">pathway</span></b>. Components of the class I PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left) and of the mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">pathway</span> (right) recurrently targeted by genetic/epigenetic alterations in cancer are depicted with a red asterisk. Several PI3K <span class="highlight" style="background-color:">pathway</span> inhibitors downstream of RTKs are currently being tested in clinical trials (gray boxes). mTOR, mechanistic target of rapamycin; mTORC, mTOR complex; PI3K, phosphoinositide 3-kinase; PIP2, phosphatidylinositol 4,5-bisphosphate; PIP3, phosphatidylinositol (3,4,5)-triphosphate; PTEN, phosphatase and tensin homolog; RTK, receptor tyrosine kinase; TSC, tuberous sclerosis protein.</div></p></div><div class="aux"><div class="resc">Britta Weigelt, et al. Front Oncol. 2012;2:109.</div><p class="links"><a class="dblinks" href="/pubmed/22970424" ref="ordinalpos=2677&amp;ncbi_uid=4229074&amp;link_uid=22970424">Citation</a><a class="dblinks" href="/pmc/articles/PMC3431500/" ref="ordinalpos=2677&amp;ncbi_uid=4229074&amp;link_uid=PMC3431500">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2678.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3132733/figure/pone-0021835-g001/" ref="ordinalpos=2678&amp;ncbi_uid=3105073&amp;link_uid=PMC3132733" href="/pmc/articles/PMC3132733/"><img src="/pmc/articles/instance/3132733/bin/pone.0021835.g001.gif" src-large="/pmc/articles/instance/3132733/bin/pone.0021835.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3132733/" ref="ordinalpos=2678&amp;ncbi_uid=3105073&amp;link_uid=PMC3132733" image-link="/pmc/articles/PMC3132733/figure/pone-0021835-g001/" class="imagepopup">Figure 1. TRAIL <span class="highlight" style="background-color:">pathway</span> and the gene-gene interaction..  From: <span class="highlight" style="background-color:">Pathway</span>-Based Association Analyses Identified TRAIL <span class="highlight" style="background-color:">Pathway</span> for Osteoporotic Fractures. </a></p><div class="supp"><p class="details"><div>The binding of TRAIL to TRAILR2 may induce the apoptosis of osteoclast. Increased c-FLIP levels may decrease the apoptosis of Osteoclast. The osteoclast apoptosis signal transduced by TRAIL/TRAILR2 was transformed to activate NF-B. A key step in downstream <span class="highlight" style="background-color:">signaling</span> of RANKL/RANK is binding of TRAFs to RANK. Stimulation of RANK also results in strong NF-B activation. NF-B1(p50) is an important signal for osteoclast development and osteoclast function.</div></p></div><div class="aux"><div class="resc">Yin-Ping Zhang, et al. PLoS One. 2011;6(7):e21835.</div><p class="links"><a class="dblinks" href="/pubmed/21760914" ref="ordinalpos=2678&amp;ncbi_uid=3105073&amp;link_uid=21760914">Citation</a><a class="dblinks" href="/pmc/articles/PMC3132733/" ref="ordinalpos=2678&amp;ncbi_uid=3105073&amp;link_uid=PMC3132733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2679.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938833/figure/f2/" ref="ordinalpos=2679&amp;ncbi_uid=2987785&amp;link_uid=PMC2938833" href="/pmc/articles/PMC2938833/"><img src="/pmc/articles/instance/2938833/bin/fig-2.gif" src-large="/pmc/articles/instance/2938833/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938833/" ref="ordinalpos=2679&amp;ncbi_uid=2987785&amp;link_uid=PMC2938833" image-link="/pmc/articles/PMC2938833/figure/f2/" class="imagepopup">FIG. 2. From: The Actin-Cytoskeleton <span class="highlight" style="background-color:">Pathway</span> and Its Potential Role in Inflammatory Bowel Disease-Associated Human Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The actin-cytoskeleton <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the candidate genes involved in colorectal cancer progression in patients with ulcerative colitis. <i>FGF-2</i>, <i>Pfn-2</i>, <i>Rdx</i>, and <i>Abi-2</i> are up-regulated (black). <i>DIAPH1/DRF1</i> is down-regulated (light gray). <i>FGF-2</i>, fibroblast growth factor-2; <i>Pfn-2</i>, profilin-2; <i>Rdx</i>, radixin; <i>Abi-2</i>, Abelson interactor-2; <i>DIAPH1</i>, diaphanous homolog.</div></p></div><div class="aux"><div class="resc">Ziad Kanaan, et al. Genet Test Mol Biomarkers. 2010 June;14(3):347-353.</div><p class="links"><a class="dblinks" href="/pubmed/20406101" ref="ordinalpos=2679&amp;ncbi_uid=2987785&amp;link_uid=20406101">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938833/" ref="ordinalpos=2679&amp;ncbi_uid=2987785&amp;link_uid=PMC2938833">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2680.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2673908/figure/F2/" ref="ordinalpos=2680&amp;ncbi_uid=1762033&amp;link_uid=PMC2673908" href="/pmc/articles/PMC2673908/"><img src="/pmc/articles/instance/2673908/bin/nihms100740f2.gif" src-large="/pmc/articles/instance/2673908/bin/nihms100740f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2673908/" ref="ordinalpos=2680&amp;ncbi_uid=1762033&amp;link_uid=PMC2673908" image-link="/pmc/articles/PMC2673908/figure/F2/" class="imagepopup">Figure 2.  From: NONCANONICAL WNT11 <span class="highlight" style="background-color:">SIGNALING</span> AND CARDIOMYOGENIC DIFFERENTIATION. </a></p><div class="supp"><p class="details"><div>A unifying model of Wnt <span class="highlight" style="background-color:">signaling</span> with regard to cardiomyogenesis. Illustrated at the left is the canonical <span class="highlight" style="background-color:">pathway</span>, similar to that in Figure 1A, with the presence of Dkk-1. Dkk-1 can promote cardiac differentiation via canonical inhibition as well as by activating JNK via unclear mechanisms. The G-proteins have also been linked to the canonical <span class="highlight" style="background-color:">pathway</span>. The illustration to the right provides an alternative portrayal of the noncanonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a single entity. In this model, activation of the heterotrimeric G-protein-linked Fz receptor by Wnt ligand initiates a multifaceted <span class="highlight" style="background-color:">signaling</span> cascade that includes activation of phospholipase C, which generates inositol triphosphate (IP<sub>3</sub>) and diacyl glycerol (DAG). IP<sub>3</sub> mediates cytoplasmic Ca<sup>2+</sup> accumulation via its entry from the extracellular space and its release from the sarcoplasmic reticulum (SR). This Ca<sup>2+</sup> influx activates PKC (also activated via DAG) and CaMKII. Activation of Frz receptor also activates Dsh responsible for activation of Ca<sup>2+</sup>-dependent PKC and CaMKII <span class="highlight" style="background-color:">signaling</span> via direct interaction with the G-protein subunits, and/or by forming a Dsh/PKC protein complex that drives PKC-dependent JNK <span class="highlight" style="background-color:">signaling</span>. Rho/Rac may also activate JNK via PKC. Although PKC phosphorylates and activates JNK, cardiogenic <span class="highlight" style="background-color:">signaling</span> is incompletely inhibited with JNK inhibition, suggesting other (unknown) transducing elements downstream from PKC, perhaps ultimately involving (ATF/CREB)-mediated transcriptional regulation.</div></p></div><div class="aux"><div class="resc">Michael P. Flaherty, et al. Trends Cardiovasc Med. ;18(7):260-268.</div><p class="links"><a class="dblinks" href="/pubmed/19232955" ref="ordinalpos=2680&amp;ncbi_uid=1762033&amp;link_uid=19232955">Citation</a><a class="dblinks" href="/pmc/articles/PMC2673908/" ref="ordinalpos=2680&amp;ncbi_uid=1762033&amp;link_uid=PMC2673908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2681.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3105099/figure/pone-0020586-g007/" ref="ordinalpos=2681&amp;ncbi_uid=3005735&amp;link_uid=PMC3105099" href="/pmc/articles/PMC3105099/"><img src="/pmc/articles/instance/3105099/bin/pone.0020586.g007.gif" src-large="/pmc/articles/instance/3105099/bin/pone.0020586.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3105099/" ref="ordinalpos=2681&amp;ncbi_uid=3005735&amp;link_uid=PMC3105099" image-link="/pmc/articles/PMC3105099/figure/pone-0020586-g007/" class="imagepopup">Figure 7. A diagram of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for ATO-mediated BIM activation and apoptosis in ovarian caner cells..  From: BIM-Mediated AKT Phosphorylation Is a Key Modulator of Arsenic Trioxide-Induced Apoptosis in Cisplatin-Sensitive and -Resistant Ovarian Cancer Cells. </a></p><div class="supp"><p class="details"><div>ATO induced JNK <span class="highlight" style="background-color:">pathway</span> activation, and then activated JNK regulated BIM activation. BIM expression upregulated caspase-3 activation and subsequent apoptosis at the first stage. Activated caspase-3 triggered PP2A-linked AKT dephosphorylation, and then AKT dephosphorylation further enhanced caspase-3 cleavage and apoptosis at the second stage.</div></p></div><div class="aux"><div class="resc">Zhu Yuan, et al. PLoS One. 2011;6(5):e20586.</div><p class="links"><a class="dblinks" href="/pubmed/21655183" ref="ordinalpos=2681&amp;ncbi_uid=3005735&amp;link_uid=21655183">Citation</a><a class="dblinks" href="/pmc/articles/PMC3105099/" ref="ordinalpos=2681&amp;ncbi_uid=3005735&amp;link_uid=PMC3105099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2682.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4016011/figure/pone-0096797-g001/" ref="ordinalpos=2682&amp;ncbi_uid=6132116&amp;link_uid=PMC4016011" href="/pmc/articles/PMC4016011/"><img src="/pmc/articles/instance/4016011/bin/pone.0096797.g001.gif" src-large="/pmc/articles/instance/4016011/bin/pone.0096797.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4016011/" ref="ordinalpos=2682&amp;ncbi_uid=6132116&amp;link_uid=PMC4016011" image-link="/pmc/articles/PMC4016011/figure/pone-0096797-g001/" class="imagepopup">Figure 1. JNK MAP kinase <span class="highlight" style="background-color:">pathway</span> was restored by replacing JIP1-JNK docking interaction with PDZ interaction domains..  From: Reassembly of JIP1 Scaffold Complex in JNK MAP Kinase <span class="highlight" style="background-color:">Pathway</span> Using Heterologous Protein Interactions. </a></p><div class="supp"><p class="details"><div>A. The wiring strategy of the JNK MAP kinase <span class="highlight" style="background-color:">pathway</span> is illustrated. The diagram shows a conventional JIP1-dependent JNK MAP kinase <span class="highlight" style="background-color:">pathway</span> (left). A JIP1* scaffold containing mutations to disrupt docking interaction with JNK leads to release of JNK from the scaffold complex and, therefore, a decrease in the <span class="highlight" style="background-color:">signaling</span> response (middle). The functional scaffold complex is reassembled by re-recruiting missing JNK to the scaffold complex using fusions of syn-JIP1* and JNK1-nNOS, resulting in the restoration of JNK <span class="highlight" style="background-color:">signaling</span> (right). To recruit JNK to JIP1*, syn and nNOS PDZ domains were fused to JIP1* and JNK, respectively. B. JIP1, JIP1* or syn-JIP1* scaffold was co-expressed with Flag-JNK1-nNOS in 293T cells. Myc-tagged scaffold proteins were immunoprecipitated using an anti-myc antibody conjugated to agarose. The bound Flag-JNK1-nNOS was detected by immunoblotting. C. To verify that the alternatively assembled scaffold complex restored the <span class="highlight" style="background-color:">signaling</span> response, dual-phosphorylation of JNK1-nNOS in the whole cell lysates was detected by western blot analysis using an anti-phospho JNK antibody. Expression of HA-MLK3 was used to stimulate the JIP1-mediated JNK <span class="highlight" style="background-color:">pathway</span> in 293T cells [13]. Each experiment described here was performed in triplicate.</div></p></div><div class="aux"><div class="resc">Jiyoung Moon, et al. PLoS One. 2014;9(5):e96797.</div><p class="links"><a class="dblinks" href="/pubmed/24816971" ref="ordinalpos=2682&amp;ncbi_uid=6132116&amp;link_uid=24816971">Citation</a><a class="dblinks" href="/pmc/articles/PMC4016011/" ref="ordinalpos=2682&amp;ncbi_uid=6132116&amp;link_uid=PMC4016011">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2683.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4154660/figure/pgen-1004582-g006/" ref="ordinalpos=2683&amp;ncbi_uid=6593550&amp;link_uid=PMC4154660" href="/pmc/articles/PMC4154660/"><img src="/pmc/articles/instance/4154660/bin/pgen.1004582.g006.gif" src-large="/pmc/articles/instance/4154660/bin/pgen.1004582.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4154660/" ref="ordinalpos=2683&amp;ncbi_uid=6593550&amp;link_uid=PMC4154660" image-link="/pmc/articles/PMC4154660/figure/pgen-1004582-g006/" class="imagepopup">Figure 6. Interactions of PRO40 with components of the CWI <span class="highlight" style="background-color:">pathway</span>..  From: PRO40 Is a Scaffold Protein of the Cell Wall Integrity <span class="highlight" style="background-color:">Pathway</span>, Linking the MAP Kinase Module to the Upstream Activator Protein Kinase C. </a></p><div class="supp"><p class="details"><div>(A) Structures of proteins used for yeast two-hybrid analysis. Derivatives generated in addition to full-length constructs are shown below the protein structures. (B) Yeast two-hybrid analysis of CWI <span class="highlight" style="background-color:">pathway</span> components and PRO40. Yeast cells were drop-plated on SD medium lacking leucine, tryptophan, histidine, and adenine. Empty squares indicate that interactions were not tested. (C) Schematic overview of signal transduction and protein-protein interactions within the PRO40-CWI complex. <span class="highlight" style="background-color:">Signaling</span> through the <span class="highlight" style="background-color:">pathway</span> is depicted by gray arrows; interactions are depicted by black arrows. (D) Interaction sites between PRO40, PKC1, MIK1, MEK1, and MAK1. Black bars represent interaction sites tested in yeast two-hybrid analyses (A, B). For reasons of clarity, only PRO40 is depicted as homodimer.</div></p></div><div class="aux"><div class="resc">Ines Teichert, et al. PLoS Genet. 2014 September;10(9):e1004582.</div><p class="links"><a class="dblinks" href="/pubmed/25188365" ref="ordinalpos=2683&amp;ncbi_uid=6593550&amp;link_uid=25188365">Citation</a><a class="dblinks" href="/pmc/articles/PMC4154660/" ref="ordinalpos=2683&amp;ncbi_uid=6593550&amp;link_uid=PMC4154660">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2684.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3358973/figure/F9/" ref="ordinalpos=2684&amp;ncbi_uid=4018386&amp;link_uid=PMC3358973" href="/pmc/articles/PMC3358973/"><img src="/pmc/articles/instance/3358973/bin/nihms-250383-f0009.gif" src-large="/pmc/articles/instance/3358973/bin/nihms-250383-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3358973/" ref="ordinalpos=2684&amp;ncbi_uid=4018386&amp;link_uid=PMC3358973" image-link="/pmc/articles/PMC3358973/figure/F9/" class="imagepopup">Figure 9. E/sox2:EGFP+ cells exhibit differential expression of <span class="highlight" style="background-color:">pathway</span> specific transcripts.  From: Prospective identification, isolation, and profiling of a telomerase expressing subpopulation of human neural stem cells, using sox2 enhancer-directed FACS. </a></p><div class="supp"><p class="details"><div><b>A</b>, Parametric gene set enrichment analysis of KEGG <span class="highlight" style="background-color:">signaling</span> pathways was performed on the profiles of E/sox2:EGFP-sorted (E/sox2:EGFP<sup>+</sup>), depleted (E/sox2:EGFP<sup></sup>) and unsorted VZ/SVZ dissociates (n=4 each). Using a linear modeling approach, significant KEGG pathways were identified between E/sox2:EGFP<sup>+</sup> and E/sox2:EGFP<sup></sup> cells at 5% FDR. The heat map visualization shows relative enrichment within each <span class="highlight" style="background-color:">pathway</span>; red indicates over expression and blue lower expression relative to the mean of E/sox2:EGFP<sup></sup> profiles.<br /><b>B</b>, The MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was significantly depleted according to KEGG <span class="highlight" style="background-color:">pathway</span> analysis. The heat map shows expression of those MAPK <span class="highlight" style="background-color:">pathway</span> genes whose expression differed significantly between E/sox2:EGFP<sup>+</sup> and E/sox2:EGFP<sup></sup> cells (&gt;3 fold change, 5% FDR). Furthermore, activators of the MAPK pathways were identified as significantly enriched in E/sox2:EGFP<sup>+</sup> following GO-based GSEA.<br /><b>C</b>, The wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was significantly over expressed in E/sox2:EGFP<sup>+</sup> cells. As in <b>B</b>, the heat map shows only significant differentially expressed genes. GO-GSEA further confirmed the over-representation of wnt genes in E/sox2:EGFP<sup>+</sup> cells (q&lt;0.05). The color key indicates relative expression on a log<sub>2</sub> scale, with red indicating high expression, and green low expression values.</div></p></div><div class="aux"><div class="resc">Su Wang, et al. J Neurosci. ;30(44):14635-14648.</div><p class="links"><a class="dblinks" href="/pubmed/21048121" ref="ordinalpos=2684&amp;ncbi_uid=4018386&amp;link_uid=21048121">Citation</a><a class="dblinks" href="/pmc/articles/PMC3358973/" ref="ordinalpos=2684&amp;ncbi_uid=4018386&amp;link_uid=PMC3358973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2685.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3869759/figure/pone-0083312-g001/" ref="ordinalpos=2685&amp;ncbi_uid=5633687&amp;link_uid=PMC3869759" href="/pmc/articles/PMC3869759/"><img src="/pmc/articles/instance/3869759/bin/pone.0083312.g001.gif" src-large="/pmc/articles/instance/3869759/bin/pone.0083312.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3869759/" ref="ordinalpos=2685&amp;ncbi_uid=5633687&amp;link_uid=PMC3869759" image-link="/pmc/articles/PMC3869759/figure/pone-0083312-g001/" class="imagepopup">Figure 1. CHIP overexpression activated PI3K/AKT and inhibition of FoxO proteins..  From: CHIP Regulates AKT/FoxO/Bim <span class="highlight" style="background-color:">Signaling</span> in MCF7 and MCF10A Cells. </a></p><div class="supp"><p class="details"><div>(A) CHIP overexpression activated the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MCF7 and MCF10A cells were transfected with Myc-CHIP or Myc empty vectors as negative control. The cells were harvested and blotted after 48 h. (B) CHIP overexpression increased the phosphorylation of FoxO family proteins.</div></p></div><div class="aux"><div class="resc">Yanrong Lv, et al. PLoS One. 2013;8(12):e83312.</div><p class="links"><a class="dblinks" href="/pubmed/24376685" ref="ordinalpos=2685&amp;ncbi_uid=5633687&amp;link_uid=24376685">Citation</a><a class="dblinks" href="/pmc/articles/PMC3869759/" ref="ordinalpos=2685&amp;ncbi_uid=5633687&amp;link_uid=PMC3869759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2686.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404564/figure/F3/" ref="ordinalpos=2686&amp;ncbi_uid=5043218&amp;link_uid=PMC3404564" href="/pmc/articles/PMC3404564/"><img src="/pmc/articles/instance/3404564/bin/zh10091265220003.gif" src-large="/pmc/articles/instance/3404564/bin/zh10091265220003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404564/" ref="ordinalpos=2686&amp;ncbi_uid=5043218&amp;link_uid=PMC3404564" image-link="/pmc/articles/PMC3404564/figure/F3/" class="imagepopup">Fig. 3. From: cAMP <span class="highlight" style="background-color:">signaling</span> in skeletal muscle adaptation: hypertrophy, metabolism, and regeneration. </a></p><div class="supp"><p class="details"><div>GPCRs that induce myofiber hypertrophy. Four GPCRs (Fzd7, <sub>2</sub>-AR, CRFR2, and LPA receptor) have been shown to stimulate hypertrophy in myotubes and/or myofibers. Cognate ligands are italicized. A partial view of the known <span class="highlight" style="background-color:">signaling</span> mediators is shown. CRFR2 stimulates myofiber growth by an uncharacterized effector <span class="highlight" style="background-color:">pathway</span>. <sub>2</sub>-AR also induces fiber type transitions to fast-twitch fibers (not shown).</div></p></div><div class="aux"><div class="resc">Rebecca Berdeaux, et al. Am J Physiol Endocrinol Metab. 2012 July 1;303(1):E1-E17.</div><p class="links"><a class="dblinks" href="/pubmed/22354781" ref="ordinalpos=2686&amp;ncbi_uid=5043218&amp;link_uid=22354781">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404564/" ref="ordinalpos=2686&amp;ncbi_uid=5043218&amp;link_uid=PMC3404564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2687.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2637552/figure/F1/" ref="ordinalpos=2687&amp;ncbi_uid=1747763&amp;link_uid=PMC2637552" href="/pmc/articles/PMC2637552/"><img src="/pmc/articles/instance/2637552/bin/nihms-89014-f0001.gif" src-large="/pmc/articles/instance/2637552/bin/nihms-89014-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2637552/" ref="ordinalpos=2687&amp;ncbi_uid=1747763&amp;link_uid=PMC2637552" image-link="/pmc/articles/PMC2637552/figure/F1/" class="imagepopup">Figure 1.  From: TGF-? <span class="highlight" style="background-color:">signaling</span> alterations and susceptibility to colorectal cancer. </a></p><div class="supp"><p class="details"><div>TGF-/SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGF- (TGFB1, TGFB2 or TGFB3) binds to the type II TGF- receptor (TGFBR2: R2), which then binds to the type I TGF- receptor (TGFBR1: R1) and forms a heterodimeric complex in which TGFBR1 kinase becomes activated. SMAD2 and SMAD3 are phosphorylated by activated TGFBR1. Once phosphorylated, they associate with SMAD4 and move to the nucleus where they act as transcription factors.</div></p></div><div class="aux"><div class="resc">Yanfei Xu, et al. Hum Mol Genet. ;16(SPEC):R14-R20.</div><p class="links"><a class="dblinks" href="/pubmed/17613544" ref="ordinalpos=2687&amp;ncbi_uid=1747763&amp;link_uid=17613544">Citation</a><a class="dblinks" href="/pmc/articles/PMC2637552/" ref="ordinalpos=2687&amp;ncbi_uid=1747763&amp;link_uid=PMC2637552">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2688.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3189339/figure/F7/" ref="ordinalpos=2688&amp;ncbi_uid=3498565&amp;link_uid=PMC3189339" href="/pmc/articles/PMC3189339/"><img src="/pmc/articles/instance/3189339/bin/nihms319385f7.gif" src-large="/pmc/articles/instance/3189339/bin/nihms319385f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3189339/" ref="ordinalpos=2688&amp;ncbi_uid=3498565&amp;link_uid=PMC3189339" image-link="/pmc/articles/PMC3189339/figure/F7/" class="imagepopup">Figure 7. Raf/EMK/ERK-mediated AR regulation, growth inhibitory <span class="highlight" style="background-color:">signaling</span>, and differentiation.  From: The Raf/MEK/extracellular signal-regulated kinase 1/2 <span class="highlight" style="background-color:">pathway</span> can mediate growth inhibitory and differentiation <span class="highlight" style="background-color:">signaling</span> via androgen receptor downregulation in prostate cancer cells. </a></p><div class="supp"><p class="details"><div>Constitutively high MEK/ERK activity can mediate AR downregulation, growth inhibitory <span class="highlight" style="background-color:">signaling</span> and differentiation. In this <span class="highlight" style="background-color:">signaling</span>, AR downregulation is required to control NSE, NEP, p16<sup>INK4A</sup> and p21<sup>WAF</sup> expression (i). However, Rb phosphorylation and expression of E2F1 can be downregulated independently of AR downregulation (ii). Decreased AR transcriptional activity leads to PSA downregulation (i) but PSA is also downregulated <i>via</i> a non-transcriptional mechanism independently of AR downregulation (ii). Raf/EMK/ERK-mediated AR downregulation is also detected in a subset of PCa cells regardless of hormone sensitivity.</div></p></div><div class="aux"><div class="resc">Seung-Keun Hong, et al. Exp Cell Res. ;317(18):2671-2682.</div><p class="links"><a class="dblinks" href="/pubmed/21871886" ref="ordinalpos=2688&amp;ncbi_uid=3498565&amp;link_uid=21871886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3189339/" ref="ordinalpos=2688&amp;ncbi_uid=3498565&amp;link_uid=PMC3189339">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2689.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4287613/figure/ppat-1004597-g003/" ref="ordinalpos=2689&amp;ncbi_uid=7057458&amp;link_uid=PMC4287613" href="/pmc/articles/PMC4287613/"><img src="/pmc/articles/instance/4287613/bin/ppat.1004597.g003.gif" src-large="/pmc/articles/instance/4287613/bin/ppat.1004597.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4287613/" ref="ordinalpos=2689&amp;ncbi_uid=7057458&amp;link_uid=PMC4287613" image-link="/pmc/articles/PMC4287613/figure/ppat-1004597-g003/" class="imagepopup">Figure 3. KapB activation of MK2 can lead to RhoA through intermediates hsp27 and p115RhoGEF..  From: Viral Activation of MK2-hsp27-p115RhoGEF-RhoA <span class="highlight" style="background-color:">Signaling</span> Axis Causes Cytoskeletal Rearrangements, P-body Disruption and ARE-mRNA Stabilization. </a></p><div class="supp"><p class="details"><div>A recently described <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> links activated MK2 to RhoA through intermediates hsp27 and p115RhoGEF (63). Hsp27 phosphorylation by MK2 is an important switch that allows local recruitment of p115RhoGEF to a <span class="highlight" style="background-color:">signaling</span> complex, promoting RhoA activity. We hypothesize that KapB mediated activation of this <span class="highlight" style="background-color:">pathway</span> contributes to several downstream effects of active RhoA.To address the importance of individual proteins within this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the mechanism of KapB-mediated RhoA activation, PB dispersion and actin cytoskeletal rearrangements, we undertook several approaches to eliminate expression or activity of candidate proteins. These included the co-expression of dominant negative (DN) versions of RhoA and hsp27, co-expression of shRNAs designed to reduce expression of p115RhoGEF and MK2 using RNA interference, and chemical inhibitors that were specific for RhoA (C3) and its downstream effector kinase ROCK (Y-27632).</div></p></div><div class="aux"><div class="resc">Jennifer A. Corcoran, et al. PLoS Pathog. 2015 January;11(1):e1004597.</div><p class="links"><a class="dblinks" href="/pubmed/25569678" ref="ordinalpos=2689&amp;ncbi_uid=7057458&amp;link_uid=25569678">Citation</a><a class="dblinks" href="/pmc/articles/PMC4287613/" ref="ordinalpos=2689&amp;ncbi_uid=7057458&amp;link_uid=PMC4287613">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2690.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3687869/figure/F2/" ref="ordinalpos=2690&amp;ncbi_uid=5017618&amp;link_uid=PMC3687869" href="/pmc/articles/PMC3687869/"><img src="/pmc/articles/instance/3687869/bin/NMJ-54-79-g002.gif" src-large="/pmc/articles/instance/3687869/bin/NMJ-54-79-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3687869/" ref="ordinalpos=2690&amp;ncbi_uid=5017618&amp;link_uid=PMC3687869" image-link="/pmc/articles/PMC3687869/figure/F2/" class="imagepopup">Figure 2.  From: Phosphatase and tensin homologue deleted on chromosome 10. </a></p><div class="supp"><p class="details"><div>The phosphatidylinositol-3-kinase/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Imran Haruna Abdulkareem, et al. Niger Med J. 2013 Mar-Apr;54(2):79-86.</div><p class="links"><a class="dblinks" href="/pubmed/23798791" ref="ordinalpos=2690&amp;ncbi_uid=5017618&amp;link_uid=23798791">Citation</a><a class="dblinks" href="/pmc/articles/PMC3687869/" ref="ordinalpos=2690&amp;ncbi_uid=5017618&amp;link_uid=PMC3687869">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2691.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117317/figure/F0003/" ref="ordinalpos=2691&amp;ncbi_uid=3066995&amp;link_uid=PMC3117317" href="/pmc/articles/PMC3117317/"><img src="/pmc/articles/instance/3117317/bin/JAIM-1-257-g003.gif" src-large="/pmc/articles/instance/3117317/bin/JAIM-1-257-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117317/" ref="ordinalpos=2691&amp;ncbi_uid=3066995&amp;link_uid=PMC3117317" image-link="/pmc/articles/PMC3117317/figure/F0003/" class="imagepopup">Figure 3.  From: Beyond reverse pharmacology: Mechanism-based screening of Ayurvedic drugs. </a></p><div class="supp"><p class="details"><div>The NFKB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">R. D. Lele. J Ayurveda Integr Med. 2010 Oct-Dec;1(4):257-265.</div><p class="links"><a class="dblinks" href="/pubmed/21731372" ref="ordinalpos=2691&amp;ncbi_uid=3066995&amp;link_uid=21731372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117317/" ref="ordinalpos=2691&amp;ncbi_uid=3066995&amp;link_uid=PMC3117317">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2692.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2754637/figure/fig7/" ref="ordinalpos=2692&amp;ncbi_uid=2135263&amp;link_uid=PMC2754637" href="/pmc/articles/PMC2754637/"><img src="/pmc/articles/instance/2754637/bin/pp1510782f07.gif" src-large="/pmc/articles/instance/2754637/bin/pp1510782f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2754637/" ref="ordinalpos=2692&amp;ncbi_uid=2135263&amp;link_uid=PMC2754637" image-link="/pmc/articles/PMC2754637/figure/fig7/" class="imagepopup">Figure 7. From: Unraveling the Evolution of Cytokinin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Model of the evolution of the cytokinin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Birgit Pils, et al. Plant Physiol. 2009 October;151(2):782-791.</div><p class="links"><a class="dblinks" href="/pubmed/19675156" ref="ordinalpos=2692&amp;ncbi_uid=2135263&amp;link_uid=19675156">Citation</a><a class="dblinks" href="/pmc/articles/PMC2754637/" ref="ordinalpos=2692&amp;ncbi_uid=2135263&amp;link_uid=PMC2754637">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2693.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3247289/figure/pone-0029800-g007/" ref="ordinalpos=2693&amp;ncbi_uid=3738966&amp;link_uid=PMC3247289" href="/pmc/articles/PMC3247289/"><img src="/pmc/articles/instance/3247289/bin/pone.0029800.g007.gif" src-large="/pmc/articles/instance/3247289/bin/pone.0029800.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3247289/" ref="ordinalpos=2693&amp;ncbi_uid=3738966&amp;link_uid=PMC3247289" image-link="/pmc/articles/PMC3247289/figure/pone-0029800-g007/" class="imagepopup">Figure 7. cyp6a17 expression is regulated by PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Novel Cytochrome P450, cyp6a17, Is Required for Temperature Preference Behavior in Drosophila. </a></p><div class="supp"><p class="details"><div>(A) Temperature preference behavior of <i>MB247&gt;UAS-PKA<sup>CA</sup>, cyp6a17-RNAi</i>. Temperature preference profile and avoidance index (AI) are shown. Black bars, AI<sub>High</sub>; gray bars, AI<sub>Low</sub>. Two asterisks, <i>P</i>&lt;0.001. (BD) Determination of expression level of <i>cyp6a17</i> in the PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutant background. <i>CrebB</i>-<i>17A<sup>S162</sup></i> (B); <i>rut<sup>1</sup></i> (C); and <i>dnc<sup>M14</sup></i>/<i>dnc <sup>1</sup></i>(D). Relative transcript level was measured by real-time PCR (histogram). <i>gapdh1</i> was used as internal control. The number of tests: N=34. Asterisks, <i>P</i>&lt;0.001 by t test. Error bars indicates S.E.M.</div></p></div><div class="aux"><div class="resc">Jongkyun Kang, et al. PLoS One. 2011;6(12):e29800.</div><p class="links"><a class="dblinks" href="/pubmed/22216356" ref="ordinalpos=2693&amp;ncbi_uid=3738966&amp;link_uid=22216356">Citation</a><a class="dblinks" href="/pmc/articles/PMC3247289/" ref="ordinalpos=2693&amp;ncbi_uid=3738966&amp;link_uid=PMC3247289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2694.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3101770/figure/F2/" ref="ordinalpos=2694&amp;ncbi_uid=2968343&amp;link_uid=PMC3101770" href="/pmc/articles/PMC3101770/"><img src="/pmc/articles/instance/3101770/bin/nihms-260063-f0002.gif" src-large="/pmc/articles/instance/3101770/bin/nihms-260063-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3101770/" ref="ordinalpos=2694&amp;ncbi_uid=2968343&amp;link_uid=PMC3101770" image-link="/pmc/articles/PMC3101770/figure/F2/" class="imagepopup">Figure 2.  From: Specificity and <span class="highlight" style="background-color:">signaling</span> in the Drosophila immune response. </a></p><div class="supp"><p class="details"><div>IMD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is preferentially triggered by DAP-type peptidoglycan, common to Gram-negative bacteria and certain Gram-positives, especially <i>Bacillus spp.</i> DAP-type peptidoglycan can be recognized by different receptors, depending on its location and size. Large insoluble peptidoglycan is recognized on the cell surface by PGRP-LCx, while smaller peptidoglycan fragments, like TCT, can be recognized at the cell surface by a ligand-induced dimer of PGRP-LCx and PGRP-LCa. In addition, DAP-type peptidoglycan that reaches the cytosol can trigger another receptor, PGRP-LE. Both PGRP-LC and PGRP-LE trigger a similar intracellular signal transduction <span class="highlight" style="background-color:">pathway</span>, as outlined here, that culminates in the activation of the NF-B precursor Relish. In addition, recognition of intracellular DAP-type peptidoglycan by PGRP-LE also triggers an autophagic response, which is important in the protection against intracellular bacteria.</div></p></div><div class="aux"><div class="resc">N Silverman, et al. Invertebrate Surviv J. ;6(2):163-174.</div><p class="links"><a class="dblinks" href="/pubmed/21625362" ref="ordinalpos=2694&amp;ncbi_uid=2968343&amp;link_uid=21625362">Citation</a><a class="dblinks" href="/pmc/articles/PMC3101770/" ref="ordinalpos=2694&amp;ncbi_uid=2968343&amp;link_uid=PMC3101770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2695.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2740372/figure/F2/" ref="ordinalpos=2695&amp;ncbi_uid=1976837&amp;link_uid=PMC2740372" href="/pmc/articles/PMC2740372/"><img src="/pmc/articles/instance/2740372/bin/nihms124870f2.gif" src-large="/pmc/articles/instance/2740372/bin/nihms124870f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2740372/" ref="ordinalpos=2695&amp;ncbi_uid=1976837&amp;link_uid=PMC2740372" image-link="/pmc/articles/PMC2740372/figure/F2/" class="imagepopup">Fig. 2.  From: Old target new approach: an alternate NF-?B activation <span class="highlight" style="background-color:">pathway</span> via translation inhibition. </a></p><div class="supp"><p class="details"><div>The eIF2 kinase regulated <span class="highlight" style="background-color:">signaling</span> pathways. A number of stimuli achieve eIF2 phosphorylation through the four known eIF2 kinases. The graphical representation of the molecular network was generated through the use of <span class="highlight" style="background-color:">Pathway</span> Studio 6 (Ariadne Genomics). All lines are supported by at least one reference from previously published literature stored in the <span class="highlight" style="background-color:">Pathway</span> Studio database</div></p></div><div class="aux"><div class="resc">Csaba F. Lszl, et al. Mol Cell Biochem. ;328(1-2):9-16.</div><p class="links"><a class="dblinks" href="/pubmed/19224334" ref="ordinalpos=2695&amp;ncbi_uid=1976837&amp;link_uid=19224334">Citation</a><a class="dblinks" href="/pmc/articles/PMC2740372/" ref="ordinalpos=2695&amp;ncbi_uid=1976837&amp;link_uid=PMC2740372">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2696.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3632385/figure/F2/" ref="ordinalpos=2696&amp;ncbi_uid=4818971&amp;link_uid=PMC3632385" href="/pmc/articles/PMC3632385/"><img src="/pmc/articles/instance/3632385/bin/nihms455115f2.gif" src-large="/pmc/articles/instance/3632385/bin/nihms455115f2.jpg" alt="Figure 6.2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3632385/" ref="ordinalpos=2696&amp;ncbi_uid=4818971&amp;link_uid=PMC3632385" image-link="/pmc/articles/PMC3632385/figure/F2/" class="imagepopup">Figure 6.2.  From: G Protein-Coupled Estrogen Receptor in Energy Homeostasis and Obesity Pathogenesis. </a></p><div class="supp"><p class="details"><div>Schematic overview of estradiol-mediated genomic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via ER/ and rapid nongenomic <span class="highlight" style="background-color:">signaling</span> pathways via GPER, Gq-mER, and membrane subpopulation mER/.</div></p></div><div class="aux"><div class="resc">Haifei Shi, et al. Prog Mol Biol Transl Sci. 2013;114:193-250.</div><p class="links"><a class="dblinks" href="/pubmed/23317786" ref="ordinalpos=2696&amp;ncbi_uid=4818971&amp;link_uid=23317786">Citation</a><a class="dblinks" href="/pmc/articles/PMC3632385/" ref="ordinalpos=2696&amp;ncbi_uid=4818971&amp;link_uid=PMC3632385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2697.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3128904/figure/F1/" ref="ordinalpos=2697&amp;ncbi_uid=3101525&amp;link_uid=PMC3128904" href="/pmc/articles/PMC3128904/"><img src="/pmc/articles/instance/3128904/bin/kjh-46-69-g001.gif" src-large="/pmc/articles/instance/3128904/bin/kjh-46-69-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3128904/" ref="ordinalpos=2697&amp;ncbi_uid=3101525&amp;link_uid=PMC3128904" image-link="/pmc/articles/PMC3128904/figure/F1/" class="imagepopup">Fig. 1.  From: Innate immunity and transplantation tolerance: the potential role of TLRs/NLRs in GVHD. </a></p><div class="supp"><p class="details"><div>Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Toll-like receptors (TLRs), except for TLR3, use a MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to induce NF-B activation. TLR3, (and TLR4), on the other hand, can activate MyD88-independent, TRIF-dependent <span class="highlight" style="background-color:">pathway</span> to induce NF-B or type I interferon (IFN) activation. MyD88 recruits TRAF6 and members of the IRAK family, leading to the activation of the TAK1 complex. The activated TAK1 complex then activates the IKK complex, consisting of IKK, IKK and NEMO (IKK-), which catalyze the phosphorylation of IB proteins. IBs are degraded by the proteasome, allowing NF-B to translocate into the nucleus. Simultaneously, the TAK1 complex activates the MAPK <span class="highlight" style="background-color:">pathway</span>, which results in the phosphorylation and activation of AP-1. NF-B and AP-1 control inflammatory responses through the induction of inflammatory cytokines. The <span class="highlight" style="background-color:">signaling</span> cascade triggered by TLR3 uses MyD88-independent, TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. TRIF recruits TRAF3, which then interacts with TBK1 and IKKi. These kinases mediate phosphorylation of IRF3. Phosphorylated IRF3 dimerizes and translocates into the nucleus to regulate transcription. TRIF also interacts with TRAF6 and RIP1, which mediate NF-B activation. Activation of the IRF3, NF-B and MAPK pathways results in the induction of type I IFN, particularly IFN-.</div></p></div><div class="aux"><div class="resc">Ok S. Shin, et al. Korean J Hematol. 2011 June;46(2):69-79.</div><p class="links"><a class="dblinks" href="/pubmed/21747878" ref="ordinalpos=2697&amp;ncbi_uid=3101525&amp;link_uid=21747878">Citation</a><a class="dblinks" href="/pmc/articles/PMC3128904/" ref="ordinalpos=2697&amp;ncbi_uid=3101525&amp;link_uid=PMC3128904">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2698.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC133918/figure/f3/" ref="ordinalpos=2698&amp;ncbi_uid=269173&amp;link_uid=PMC133918" href="/pmc/articles/PMC133918/"><img src="/pmc/articles/instance/133918/bin/mb1321825003.gif" src-large="/pmc/articles/instance/133918/bin/mb1321825003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC133918/" ref="ordinalpos=2698&amp;ncbi_uid=269173&amp;link_uid=PMC133918" image-link="/pmc/articles/PMC133918/figure/f3/" class="imagepopup">FIG. 3. From: Dual Roles of Cripto as a Ligand and Coreceptor in the Nodal <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Cripto and Nodal act as secreted <span class="highlight" style="background-color:">signaling</span> factors. (A) Design of a coculture assay to assess the <span class="highlight" style="background-color:">signaling</span> activities of Cripto and Nodal. Two populations of 293T cells were transiently transfected and replated together to assay luciferase activity (panel B). Responding cells are distinguished from <span class="highlight" style="background-color:">signaling</span> cells by transfection with the <i>A3-lux</i> luciferase reporter plasmid. Alternatively (panel C), conditioned media from <span class="highlight" style="background-color:">signaling</span> cells were added without direct coculture. (B) Nodal is active when expressed by either <span class="highlight" style="background-color:">signaling</span> or responding cells. Cripto is active when expressed by responding cells, but also displays detectable activity when expressed by <span class="highlight" style="background-color:">signaling</span> cells. (C) Both Nodal and Cripto proteins expressed in conditioned media of transiently transfected 293T cells are active in this <span class="highlight" style="background-color:">signaling</span> assay; the left and right insets show the expression of Nodal and Cripto protein in conditioned media, respectively. (D) Conditioned media from two independent stable clones (#8 and #9) expressing mouse Nodal display activity; similarly, conditioned media from two independent stable clones (#37 and #44) expressing mouse Cripto are also active. The left and right insets show the expression of Nodal and Cripto protein, respectively, in conditioned media from these stable cell lines or from control stable lines containing the parental vector alone.</div></p></div><div class="aux"><div class="resc">Yu-Ting Yan, et al. Mol Cell Biol. 2002 July;22(13):4439-4449.</div><p class="links"><a class="dblinks" href="/pubmed/12052855" ref="ordinalpos=2698&amp;ncbi_uid=269173&amp;link_uid=12052855">Citation</a><a class="dblinks" href="/pmc/articles/PMC133918/" ref="ordinalpos=2698&amp;ncbi_uid=269173&amp;link_uid=PMC133918">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2699.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3215398/figure/F5/" ref="ordinalpos=2699&amp;ncbi_uid=3408180&amp;link_uid=PMC3215398" href="/pmc/articles/PMC3215398/"><img src="/pmc/articles/instance/3215398/bin/nihms334114f5.gif" src-large="/pmc/articles/instance/3215398/bin/nihms334114f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3215398/" ref="ordinalpos=2699&amp;ncbi_uid=3408180&amp;link_uid=PMC3215398" image-link="/pmc/articles/PMC3215398/figure/F5/" class="imagepopup">Fig. 5.  From: Phosphorylation of Mad Controls Competition Between Wingless and BMP <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model in which Mad and Medea bind to the Wg Pangolin-Armadillo transcriptional complex in the absence of BMP signals. Previously, Mad was believed to be transcriptionally active only when phosphorylated at its C terminus by BMP receptor. In this model, Mad promotes activation of Wg <span class="highlight" style="background-color:">pathway</span> target genes when not phosphorylated. In the presence of BMP, Wg serves to prolong the signal from Mad phosphorylated in the C-terminal domain by inhibiting GSK3-mediated phosphorylation of the linker region of Mad (8, 9). In addition, BMP <span class="highlight" style="background-color:">signaling</span> also diverts Mad or Medea from the Wg <span class="highlight" style="background-color:">pathway</span> to promote activation of BMP target genes, thereby causing competition between the BMP and the Wingless <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Edward Eivers, et al. Sci Signal. ;4(194):10.1126/scisignal.2002034 ra68.</div><p class="links"><a class="dblinks" href="/pubmed/21990430" ref="ordinalpos=2699&amp;ncbi_uid=3408180&amp;link_uid=21990430">Citation</a><a class="dblinks" href="/pmc/articles/PMC3215398/" ref="ordinalpos=2699&amp;ncbi_uid=3408180&amp;link_uid=PMC3215398">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2700.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053611/figure/Fig2/" ref="ordinalpos=2700&amp;ncbi_uid=6254423&amp;link_uid=PMC4053611" href="/pmc/articles/PMC4053611/"><img src="/pmc/articles/instance/4053611/bin/11010_2014_2021_Fig2_HTML.gif" src-large="/pmc/articles/instance/4053611/bin/11010_2014_2021_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053611/" ref="ordinalpos=2700&amp;ncbi_uid=6254423&amp;link_uid=PMC4053611" image-link="/pmc/articles/PMC4053611/figure/Fig2/" class="imagepopup">Fig. 2.  From: Estrogen improves the proliferation and differentiation of hBMSCs derived from postmenopausal osteoporosis through notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>17-estradiol promoted hBMSCs proliferation and differentiation partially by activating Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Growth curves (<b>a</b>) and plate colony formation assay (<b>b</b>) of hBMSCs cultured in different conditions.* <i>p</i> &lt; <i>0.05</i>, compared to the control. Osteogenic markers' expression with or without the addition of DAPT by real-time PCR. * <i>p</i> &lt; <i>0.05</i>, compared to the control (<b>c</b>). ALP Staining and Alizarin red staining with or without the addition of DAPT (<b>d</b>)</div></p></div><div class="aux"><div class="resc">Jin-Zhu Fan, et al. Mol Cell Biochem. 2014;392(1-2):85-93.</div><p class="links"><a class="dblinks" href="/pubmed/24390089" ref="ordinalpos=2700&amp;ncbi_uid=6254423&amp;link_uid=24390089">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053611/" ref="ordinalpos=2700&amp;ncbi_uid=6254423&amp;link_uid=PMC4053611">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="26" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="27" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="28" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:35:56-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB0F6E543102210000000000A60676;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB0F6E543102210000000000A60676" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>