
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="3" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="8A1A393946D25221_0091SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB7B7653EBEE61000000000054237B" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene">Gene</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=12" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="2" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="3" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="4" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="3" /></div>
            <div><h2 class="result_count">Results: 201 to 300 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>201.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2567351/figure/F1/" ref="ordinalpos=201&amp;ncbi_uid=1595506&amp;link_uid=PMC2567351" href="/pmc/articles/PMC2567351/"><img src="/pmc/articles/instance/2567351/bin/1471-2105-9-412-1.gif" src-large="/pmc/articles/instance/2567351/bin/1471-2105-9-412-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2567351/" ref="ordinalpos=201&amp;ncbi_uid=1595506&amp;link_uid=PMC2567351" image-link="/pmc/articles/PMC2567351/figure/F1/" class="imagepopup">Figure 1.  From: Gene and <span class="highlight" style="background-color:">pathway</span> identification with Lp penalized Bayesian logistic regression. </a></p><div class="supp"><p class="details"><div><b>T Cell Receiptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></b>. T-cell receipt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the associated genes identified with the integrated algorithm. The over-expressed and down-expressed genes on the <span class="highlight" style="background-color:">pathway</span> are shown in red and blue respectively.</div></p></div><div class="aux"><div class="resc">Zhenqiu Liu, et al. BMC Bioinformatics. 2008;9:412-412.</div><p class="links"><a class="dblinks" href="/pubmed/18834526" ref="ordinalpos=201&amp;ncbi_uid=1595506&amp;link_uid=18834526">Citation</a><a class="dblinks" href="/pmc/articles/PMC2567351/" ref="ordinalpos=201&amp;ncbi_uid=1595506&amp;link_uid=PMC2567351">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>202.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F6/" ref="ordinalpos=202&amp;ncbi_uid=5822246&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g006.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g006.jpg" alt="Figure 2d" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=202&amp;ncbi_uid=5822246&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F6/" class="imagepopup">Figure 2d.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancers -Type IIIb-Reverse Paracrine Ligand-dependent cancers</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=202&amp;ncbi_uid=5822246&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=202&amp;ncbi_uid=5822246&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>203.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3829470/figure/F1/" ref="ordinalpos=203&amp;ncbi_uid=5492641&amp;link_uid=PMC3829470" href="/pmc/articles/PMC3829470/"><img src="/pmc/articles/instance/3829470/bin/nihms-525698-f0001.gif" src-large="/pmc/articles/instance/3829470/bin/nihms-525698-f0001.jpg" alt="Fig. (1)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3829470/" ref="ordinalpos=203&amp;ncbi_uid=5492641&amp;link_uid=PMC3829470" image-link="/pmc/articles/PMC3829470/figure/F1/" class="imagepopup">Fig. (1).  From: <span class="highlight" style="background-color:">Signaling</span> Through Rho GTPase <span class="highlight" style="background-color:">Pathway</span> as Viable Drug Target. </a></p><div class="supp"><p class="details"><div>Schematic illustration of Rho GTPase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A. Receptor dependent and independent <span class="highlight" style="background-color:">signaling</span> through Rho GTPase <span class="highlight" style="background-color:">pathway</span> leading to actin reorganization. Numbered arrows indicate where the inhibitor targets are in the Rho GTPase <span class="highlight" style="background-color:">signaling</span> cascade: 1, Y27632, fasudil, and H-1152; 2, LM11A31; 3, statins; 4, NSC23766; 5, CCG-1423. B. Regulation of Rho GTPase activation and inactivation through Rho-GEFs, Rho-GAPs, and RhoGDIs.</div></p></div><div class="aux"><div class="resc">Qun Lu, et al. Curr Med Chem. 2009;16(11):1355-1365.</div><p class="links"><a class="dblinks" href="/pubmed/19355891" ref="ordinalpos=203&amp;ncbi_uid=5492641&amp;link_uid=19355891">Citation</a><a class="dblinks" href="/pmc/articles/PMC3829470/" ref="ordinalpos=203&amp;ncbi_uid=5492641&amp;link_uid=PMC3829470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>204.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281649/figure/F9/" ref="ordinalpos=204&amp;ncbi_uid=4610086&amp;link_uid=PMC3281649" href="/pmc/articles/PMC3281649/"><img src="/pmc/articles/instance/3281649/bin/zbc0091297680009.gif" src-large="/pmc/articles/instance/3281649/bin/zbc0091297680009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281649/" ref="ordinalpos=204&amp;ncbi_uid=4610086&amp;link_uid=PMC3281649" image-link="/pmc/articles/PMC3281649/figure/F9/" class="imagepopup">FIGURE 9. From: Dual Regulation of Notch1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Adaptor Protein Fe65. </a></p><div class="supp"><p class="details"><div><b>Proposed model for the inhibition of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Fe65.</b> Fe65 suppresses Notch1 <span class="highlight" style="background-color:">signaling</span> via an acceleration of the degradation of NEXT in the cytoplasm. Notch1 is capable of forming a trimeric complex with Itch and Fe65. Fe65 enhanced the protein degradation of NEXT via the Itch-dependent proteasomal <span class="highlight" style="background-color:">pathway</span>. Additionally, Fe65 suppresses Notch1 transcriptional activity via the dissociation of the Notch1-IC-RBP-Jk complex in the nucleus.</div></p></div><div class="aux"><div class="resc">Mi-Yeon Kim, et al. J Biol Chem. 2012 February 10;287(7):4690-4701.</div><p class="links"><a class="dblinks" href="/pubmed/22199353" ref="ordinalpos=204&amp;ncbi_uid=4610086&amp;link_uid=22199353">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281649/" ref="ordinalpos=204&amp;ncbi_uid=4610086&amp;link_uid=PMC3281649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>205.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4006025/figure/F2/" ref="ordinalpos=205&amp;ncbi_uid=6100668&amp;link_uid=PMC4006025" href="/pmc/articles/PMC4006025/"><img src="/pmc/articles/instance/4006025/bin/fcimb-04-00049-g0002.gif" src-large="/pmc/articles/instance/4006025/bin/fcimb-04-00049-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4006025/" ref="ordinalpos=205&amp;ncbi_uid=6100668&amp;link_uid=PMC4006025" image-link="/pmc/articles/PMC4006025/figure/F2/" class="imagepopup">Figure 2.  From: Tissue communication in regenerative inflammatory <span class="highlight" style="background-color:">signaling</span>: lessons from the fly gut. </a></p><div class="supp"><p class="details"><div><b>Signals derived from non-intestinal epithelial tissues control intestinal homeostasis in <i>Drosophila</i>. (A)</b> Physiological Homeostasis: Basal proliferation is controlled via the EGFR and the Wg pathways, whereas JAK/STAT controls differentiation. Vn secreted by the VM, as well as Krn and Spi coming from the EC and the EB, respectively, activate the EGFR <span class="highlight" style="background-color:">pathway</span>. Wg coming from the VM and the EB activates the Wg <span class="highlight" style="background-color:">pathway</span>. Dilp3 secreted from the VM and systemic Dilps activate InR <span class="highlight" style="background-color:">signaling</span> in response to nutrition. Dpp secreted by the VM and possibly by the ECs and the trachea activates the BMP <span class="highlight" style="background-color:">pathway</span>, which inhibits EGFR. Differentiation is regulated by JAK/STAT with the Upd cytokines coming mainly from the ECs. <b>(B)</b> Regenerative inflammatory <span class="highlight" style="background-color:">signaling</span>: Enteric infection and ingestion of chemicals induce intestinal damage that promotes regeneration via compensatory ISC proliferation. The EGFR, the Wg and the JAK/STAT pathways control ISC proliferation. JNK <span class="highlight" style="background-color:">signaling</span> is a stress sensor induced in the ECs. It activates EGFR <span class="highlight" style="background-color:">signaling</span> in ISCs and induces Wg in the EBs that activates Wg <span class="highlight" style="background-color:">signaling</span> in the ISCs. The EGFR ligands come from the VM (secreted Vn), the ECs (Krn) and the EBs (Spi). The JNK and JAK/STAT induce proliferation by activating EGFR <span class="highlight" style="background-color:">signaling</span>. Upd3 derived from damaged ECs induces the JAK/STAT <span class="highlight" style="background-color:">pathway</span> in the ISCs and the VM. In the VM, Vn is induced by JAK/STAT activity. BMP <span class="highlight" style="background-color:">signaling</span> is required for the shift from regeneration to basal maintenance; it inhibits EGFR <span class="highlight" style="background-color:">signaling</span>. The BMP ligand Dpp comes from the VM. Additionally, Dpp is expressed in the trachea but this source seems dispensable. Although the InR promotes proliferation, the source and identity of its ligands remain unclear. <span class="highlight" style="background-color:">Signaling</span> pathways are shown in bold and underlined. Abbreviations: EC, enterocyte; ISC, intestinal stem cell; EB, enteroblast; Vn, Vein; Spi, Spitz; Krn, Keren; Wg, Wingless.</div></p></div><div class="aux"><div class="resc">Kristina Kux, et al. Front Cell Infect Microbiol. 2014;4:49.</div><p class="links"><a class="dblinks" href="/pubmed/24795868" ref="ordinalpos=205&amp;ncbi_uid=6100668&amp;link_uid=24795868">Citation</a><a class="dblinks" href="/pmc/articles/PMC4006025/" ref="ordinalpos=205&amp;ncbi_uid=6100668&amp;link_uid=PMC4006025">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>206.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775871/figure/F1/" ref="ordinalpos=206&amp;ncbi_uid=6451364&amp;link_uid=PMC3775871" href="/pmc/articles/PMC3775871/"><img src="/pmc/articles/instance/3775871/bin/nihms468897f1.gif" src-large="/pmc/articles/instance/3775871/bin/nihms468897f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775871/" ref="ordinalpos=206&amp;ncbi_uid=6451364&amp;link_uid=PMC3775871" image-link="/pmc/articles/PMC3775871/figure/F1/" class="imagepopup">Figure 1. Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis demonstrating significant methylation changes in the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in invasive cells.  From: Epigenetic Regulation of SOX9 by the NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Pancreatic Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div>Genes highlighted in gray demonstrate demethylation in invasive cells.</div></p></div><div class="aux"><div class="resc">Lei Sun, et al. Stem Cells. 2013 August;31(8):1454-1466.</div><p class="links"><a class="dblinks" href="/pubmed/23592398" ref="ordinalpos=206&amp;ncbi_uid=6451364&amp;link_uid=23592398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775871/" ref="ordinalpos=206&amp;ncbi_uid=6451364&amp;link_uid=PMC3775871">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>207.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3307269/figure/F9/" ref="ordinalpos=207&amp;ncbi_uid=4637404&amp;link_uid=PMC3307269" href="/pmc/articles/PMC3307269/"><img src="/pmc/articles/instance/3307269/bin/zbc0111298810009.gif" src-large="/pmc/articles/instance/3307269/bin/zbc0111298810009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3307269/" ref="ordinalpos=207&amp;ncbi_uid=4637404&amp;link_uid=PMC3307269" image-link="/pmc/articles/PMC3307269/figure/F9/" class="imagepopup">FIGURE 9. From: Malin Regulates Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> through Degradation of Dishevelled2. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the regulation of Wnt <span class="highlight" style="background-color:">pathway</span> by malin and laforin.</b> Laforin is shown to function as a phosphatase for GSK-3β. It stimulates the degradation of β-catenin and down-regulates Wnt <span class="highlight" style="background-color:">signaling</span> by activating GSK-3β (20). Our finding shows that malin down-regulates Wnt <span class="highlight" style="background-color:">pathway</span> by promoting degradation of Dvl2. Therefore, the loss of function of either malin or laforin could result in aberrant Wnt <span class="highlight" style="background-color:">signaling</span>, which might be one of the possible cause of LD pathogenesis.</div></p></div><div class="aux"><div class="resc">Jaiprakash Sharma, et al. J Biol Chem. 2012 February 24;287(9):6830-6839.</div><p class="links"><a class="dblinks" href="/pubmed/22223637" ref="ordinalpos=207&amp;ncbi_uid=4637404&amp;link_uid=22223637">Citation</a><a class="dblinks" href="/pmc/articles/PMC3307269/" ref="ordinalpos=207&amp;ncbi_uid=4637404&amp;link_uid=PMC3307269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>208.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2923204/figure/F2/" ref="ordinalpos=208&amp;ncbi_uid=2862681&amp;link_uid=PMC2923204" href="/pmc/articles/PMC2923204/"><img src="/pmc/articles/instance/2923204/bin/nihms216902f2.gif" src-large="/pmc/articles/instance/2923204/bin/nihms216902f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2923204/" ref="ordinalpos=208&amp;ncbi_uid=2862681&amp;link_uid=PMC2923204" image-link="/pmc/articles/PMC2923204/figure/F2/" class="imagepopup">Figure 2. BDNF-TrkB <span class="highlight" style="background-color:">signaling</span> Regulates Multiple Events through PLCγ, PI3K and MAPK pathways.  From: Post-synaptic BDNF-TrkB <span class="highlight" style="background-color:">Signaling</span> in Synapse Maturation, Plasticity and Disease. </a></p><div class="supp"><p class="details"><div>(a) PI3K <span class="highlight" style="background-color:">pathway</span> regulates trafficking and PSD-95 to a synapse and cAMP regulates formation of synaptic PSD-95-TrkB complex.<br />(b) BDNF-TrkB <span class="highlight" style="background-color:">signaling</span> activates MAPK/Erk, increasese cAMP and activate CREB-regulated gene transcription.<br />(c) BDNF-TrkB <span class="highlight" style="background-color:">signaling</span> regulates protein translation through both MAPK/Erk and PI3K-Akt-mTOR pathways. PI3K-Akt also regulates protein trafficking in ER and Golgi.<br />(d) A diagram showing BDNF-TrkB <span class="highlight" style="background-color:">signaling</span> regulates multiple molecular events through three pathways in parallel.</div></p></div><div class="aux"><div class="resc">Akira Yoshii, et al. Dev Neurobiol. ;70(5):304-322.</div><p class="links"><a class="dblinks" href="/pubmed/20186705" ref="ordinalpos=208&amp;ncbi_uid=2862681&amp;link_uid=20186705">Citation</a><a class="dblinks" href="/pmc/articles/PMC2923204/" ref="ordinalpos=208&amp;ncbi_uid=2862681&amp;link_uid=PMC2923204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>209.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1431720/figure/F8/" ref="ordinalpos=209&amp;ncbi_uid=750768&amp;link_uid=PMC1431720" href="/pmc/articles/PMC1431720/"><img src="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-8.gif" src-large="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1431720/" ref="ordinalpos=209&amp;ncbi_uid=750768&amp;link_uid=PMC1431720" image-link="/pmc/articles/PMC1431720/figure/F8/" class="imagepopup">Figure 8.  From: Identification of <span class="highlight" style="background-color:">signaling</span> components required for the prediction of cytokine release in RAW 264.7 macrophages. </a></p><div class="supp"><p class="details"><div>Combined network of <span class="highlight" style="background-color:">signaling</span> components required for the production of cytokines. Upper row represents the different <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. Lower row represents the different cytokines. Bold face: <span class="highlight" style="background-color:">signaling</span> component identified from measured <span class="highlight" style="background-color:">signaling</span> pathways. Italic face: <span class="highlight" style="background-color:">signaling</span> component identified from residuals and representing ligand-specific unmeasured pathways.</div></p></div><div class="aux"><div class="resc">Sylvain Pradervand, et al. Genome Biol. 2006;7(2):R11-R11.</div><p class="links"><a class="dblinks" href="/pubmed/16507166" ref="ordinalpos=209&amp;ncbi_uid=750768&amp;link_uid=16507166">Citation</a><a class="dblinks" href="/pmc/articles/PMC1431720/" ref="ordinalpos=209&amp;ncbi_uid=750768&amp;link_uid=PMC1431720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>210.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508164/figure/F9/" ref="ordinalpos=210&amp;ncbi_uid=5657097&amp;link_uid=PMC3508164" href="/pmc/articles/PMC3508164/"><img src="/pmc/articles/instance/3508164/bin/nihms411023f9.gif" src-large="/pmc/articles/instance/3508164/bin/nihms411023f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508164/" ref="ordinalpos=210&amp;ncbi_uid=5657097&amp;link_uid=PMC3508164" image-link="/pmc/articles/PMC3508164/figure/F9/" class="imagepopup">Fig. 9.  From: Cripto-1 enhances the canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by binding to LRP5 and LRP6 co-receptors. </a></p><div class="supp"><p class="details"><div>Activation of the canonical Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is independent of a Smad2/3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in F9 cells. (A) Western blot analysis for phospho Smad2 (P-Smad2) and total Smad2 in F9 and F9 Cripto-1<sup>-/-</sup> cells stimulated with rhWnt3a (10 ng/ml) in the presence or absence of SB431542 inhibitor. Density ratio indicates densitometric values of phosphorylated Smad2/total Smad2. (B) SuperTOPFLASH luciferase reporter assay and (C) (n2)<sub>7</sub>-luc reporter assay in F9 and F9 Cripto-1<sup>-/-</sup> cells. Cells were incubated in the presence of the Alk4/5/7 inhibitor SB431542 to block activation of the Smad2/Smad3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tadahiro Nagaoka, et al. Cell Signal. ;25(1):178-189.</div><p class="links"><a class="dblinks" href="/pubmed/23022962" ref="ordinalpos=210&amp;ncbi_uid=5657097&amp;link_uid=23022962">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508164/" ref="ordinalpos=210&amp;ncbi_uid=5657097&amp;link_uid=PMC3508164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>211.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4026543/figure/cjc-33-05-231-g002/" ref="ordinalpos=211&amp;ncbi_uid=6148589&amp;link_uid=PMC4026543" href="/pmc/articles/PMC4026543/"><img src="/pmc/articles/instance/4026543/bin/cjc-33-05-231-g002.gif" src-large="/pmc/articles/instance/4026543/bin/cjc-33-05-231-g002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4026543/" ref="ordinalpos=211&amp;ncbi_uid=6148589&amp;link_uid=PMC4026543" image-link="/pmc/articles/PMC4026543/figure/cjc-33-05-231-g002/" class="imagepopup">Figure 2.EPH receptors crosstalk with EMT-related <span class="highlight" style="background-color:">signaling</span> pathways..  From: The role of EPH receptors in cancer-related epithelial-mesenchymal transition. </a></p><div class="supp"><p class="details"><div>EPH receptors initiate forward <span class="highlight" style="background-color:">signaling</span> to promote cancer progression via the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in both ligand-dependent and ligand-independent manners. EPHRIN-mediated reverse <span class="highlight" style="background-color:">signaling</span> also contributes to tumorigenicity by inhibiting apoptosis. The Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promotes tumor progression by up-regulating EPHB4 and down-regulating EPHB2, whereas EPHB3 inhibits the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. EPHA2 promotes EMT through the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. CBP, cyclic AMP-responsive related binding protein; ECM, extracellular matrix; ERK, extracellular regulated protein kinase; NF-κβ, nuclear factor kappa-light-chain-enhancer of activated B cell; MAPK, mitogen-activated protein kinase; mTORC1, mammalian target of rapamycin (mTOR) complex 1; PI3K, phosphatidylinositol 3-kinase; Red P, phosphorylation; RTKs, receptor tyrosine kinases; TCF, T-cell factor.</div></p></div><div class="aux"><div class="resc">Rui-Xin Li, et al. Chin J Cancer. 2014 May;33(5):231-240.</div><p class="links"><a class="dblinks" href="/pubmed/24103789" ref="ordinalpos=211&amp;ncbi_uid=6148589&amp;link_uid=24103789">Citation</a><a class="dblinks" href="/pmc/articles/PMC4026543/" ref="ordinalpos=211&amp;ncbi_uid=6148589&amp;link_uid=PMC4026543">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>212.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F5/" ref="ordinalpos=212&amp;ncbi_uid=5822247&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g005.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g005.jpg" alt="Figure 2c" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=212&amp;ncbi_uid=5822247&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F5/" class="imagepopup">Figure 2c.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancers -Type III- Paracrine Ligand-dependent cancers</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=212&amp;ncbi_uid=5822247&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=212&amp;ncbi_uid=5822247&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>213.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4320218/figure/Fig1/" ref="ordinalpos=213&amp;ncbi_uid=7166527&amp;link_uid=PMC4320218" href="/pmc/articles/PMC4320218/"><img src="/pmc/articles/instance/4320218/bin/40064_2014_1498_Fig1_HTML.gif" src-large="/pmc/articles/instance/4320218/bin/40064_2014_1498_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4320218/" ref="ordinalpos=213&amp;ncbi_uid=7166527&amp;link_uid=PMC4320218" image-link="/pmc/articles/PMC4320218/figure/Fig1/" class="imagepopup">Figure 1.  From: Diminished mTOR <span class="highlight" style="background-color:">signaling</span>: a common mode of action for endocrine longevity factors. </a></p><div class="supp"><p class="details"><div> <b>The PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Insulin, amino acids and glucose stimulate <span class="highlight" style="background-color:">signaling</span> through the PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to regulate ribosomal biogenesis, translation, autophagy, and metabolism and stress resistance.</div></p></div><div class="aux"><div class="resc">Dudley W Lamming. Springerplus. 2014;3:735.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=213&amp;ncbi_uid=7166527&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4320218/" ref="ordinalpos=213&amp;ncbi_uid=7166527&amp;link_uid=PMC4320218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>214.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3397502/figure/f7-ijms-13-06883/" ref="ordinalpos=214&amp;ncbi_uid=4139900&amp;link_uid=PMC3397502" href="/pmc/articles/PMC3397502/"><img src="/pmc/articles/instance/3397502/bin/ijms-13-06883f7.gif" src-large="/pmc/articles/instance/3397502/bin/ijms-13-06883f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3397502/" ref="ordinalpos=214&amp;ncbi_uid=4139900&amp;link_uid=PMC3397502" image-link="/pmc/articles/PMC3397502/figure/f7-ijms-13-06883/" class="imagepopup">Figure 7.  From: Fragment C of Tetanus Toxin: New Insights into Its Neuronal <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for fragment C. Based on the results obtained by Aguilera and co-workers, fragment C can induce cerebellar granular cell survival under stress conditions, activating <span class="highlight" style="background-color:">signaling</span> pathways associated with Trk receptors that include the activation of PLC, the Ras/MAPK <span class="highlight" style="background-color:">pathway</span> and the PI3 <span class="highlight" style="background-color:">pathway</span>, leading to the survival of the cell [88,89]. Another possible retrograde <span class="highlight" style="background-color:">pathway</span> of fragment C of tetanus toxin is shared by p75<sup>NTR</sup>, TrkB and BDNF, which is strongly dependent on the activities of the small GTPases Rab5 and Rab7 [56].</div></p></div><div class="aux"><div class="resc">Ana C. Calvo, et al. Int J Mol Sci. 2012;13(6):6883-6901.</div><p class="links"><a class="dblinks" href="/pubmed/22837670" ref="ordinalpos=214&amp;ncbi_uid=4139900&amp;link_uid=22837670">Citation</a><a class="dblinks" href="/pmc/articles/PMC3397502/" ref="ordinalpos=214&amp;ncbi_uid=4139900&amp;link_uid=PMC3397502">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>215.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2955995/figure/F2/" ref="ordinalpos=215&amp;ncbi_uid=3581516&amp;link_uid=PMC2955995" href="/pmc/articles/PMC2955995/"><img src="/pmc/articles/instance/2955995/bin/nihms-217025-f0002.gif" src-large="/pmc/articles/instance/2955995/bin/nihms-217025-f0002.jpg" alt="Figure-2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2955995/" ref="ordinalpos=215&amp;ncbi_uid=3581516&amp;link_uid=PMC2955995" image-link="/pmc/articles/PMC2955995/figure/F2/" class="imagepopup">Figure-2.  From: Targeting Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to overcome drug-resistance for cancer therapy. </a></p><div class="supp"><p class="details"><div>The role of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the progression of cancer, and during the acquisition of EMT phenotype, and the formation of cancer stem cells, leading to drug resistance. Natural agents and γ-secretase inhibitors could be useful for targeting Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins, which could enhance the sensitivity of chemotherapeutic drugs.</div></p></div><div class="aux"><div class="resc">Zhiwei Wang, et al. Biochim Biophys Acta. ;1806(2):258-267.</div><p class="links"><a class="dblinks" href="/pubmed/20600632" ref="ordinalpos=215&amp;ncbi_uid=3581516&amp;link_uid=20600632">Citation</a><a class="dblinks" href="/pmc/articles/PMC2955995/" ref="ordinalpos=215&amp;ncbi_uid=3581516&amp;link_uid=PMC2955995">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>216.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC59357/figure/F8/" ref="ordinalpos=216&amp;ncbi_uid=125193&amp;link_uid=PMC59357" href="/pmc/articles/PMC59357/"><img src="/pmc/articles/instance/59357/bin/pp0895650008.gif" src-large="/pmc/articles/instance/59357/bin/pp0895650008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC59357/" ref="ordinalpos=216&amp;ncbi_uid=125193&amp;link_uid=PMC59357" image-link="/pmc/articles/PMC59357/figure/F8/" class="imagepopup">Figure 8.  From: Leucine Aminopeptidase RNAs, Proteins, and Activities Increase in Response to Water Deficit, Salinity, and the Wound Signals Systemin, Methyl Jasmonate, and Abscisic Acid. </a></p><div class="supp"><p class="details"><div>Schematic diagram showing the four independent signal transduction pathways that activate wound and defense genes in tomato. The PR genes (<i>PR</i>-<i>1</i>, <i>PR</i>-<i>4</i>, and <i>GluB</i>) utilize two <span class="highlight" style="background-color:">signaling</span> pathways: a SA-dependent and an ethylene-dependent <span class="highlight" style="background-color:">pathway</span>. <i>PR</i>-<i>1</i> and <i>GluB</i> utilize a third <span class="highlight" style="background-color:">pathway</span>, a systemin-independent, JA-dependent <span class="highlight" style="background-color:">pathway</span>, which may be analogous to the SA-independent <span class="highlight" style="background-color:">pathway</span> used to induce defensins in Arabidopsis (Penninckx et al., 1995). The fourth <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is the octadecanoid <span class="highlight" style="background-color:">pathway</span>, which in tomato utilizes systemin, ABA, JA, and ethylene to activate the wound-response genes (<i>LapA1</i>, <i>LapA2</i>, and <i>pin2</i>) and SA to down-regulate the <span class="highlight" style="background-color:">pathway</span>. There is evidence for a fifth <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Arabidopsis. A JA-independent mechanism for wound-response gene activation was recently described, but tomato genes activated by this <span class="highlight" style="background-color:">pathway</span> have yet to be identified (Titarenko et al., 1997).</div></p></div><div class="aux"><div class="resc">Wun S. Chao, et al. Plant Physiol. 1999 August;120(4):979-992.</div><p class="links"><a class="dblinks" href="/pubmed/10444081" ref="ordinalpos=216&amp;ncbi_uid=125193&amp;link_uid=10444081">Citation</a><a class="dblinks" href="/pmc/articles/PMC59357/" ref="ordinalpos=216&amp;ncbi_uid=125193&amp;link_uid=PMC59357">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>217.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3690158/figure/F1/" ref="ordinalpos=217&amp;ncbi_uid=6079349&amp;link_uid=PMC3690158" href="/pmc/articles/PMC3690158/"><img src="/pmc/articles/instance/3690158/bin/nihms463323f1.gif" src-large="/pmc/articles/instance/3690158/bin/nihms463323f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3690158/" ref="ordinalpos=217&amp;ncbi_uid=6079349&amp;link_uid=PMC3690158" image-link="/pmc/articles/PMC3690158/figure/F1/" class="imagepopup">Figure 1.  From: Wnt <span class="highlight" style="background-color:">Signaling</span> during Cochlear Development. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> pathways. Canonical vs. non-canonical Wnt <span class="highlight" style="background-color:">signaling</span> is dependent on ligand, receptor and co-receptor interactions. Wnt ligands can activate both canonical pathways by binding to Fzd receptors and non-canonical pathways through co-receptors such as Ror and Ryk. Ror can inhibit β-catenin activity while activating Jnk. R-spondins directly bind to Lgr and Lrp co-receptors that interact with Fzd to stimulate/augment β-catenin <span class="highlight" style="background-color:">signaling</span>. Norrin can also activate the canonical <span class="highlight" style="background-color:">pathway</span>, independent of Wnts. Wnt Inhibitors such as sFRP/Wif also regulate Wnt <span class="highlight" style="background-color:">signaling</span> by sequestering Wnt ligands, while Dkk interacts with Lrp to inhibit Wnt <span class="highlight" style="background-color:">signaling</span>. The Notch and Fgfr <span class="highlight" style="background-color:">signaling</span> pathways have been found to act both, upstream and downstream of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>. Figure modified from [3].</div></p></div><div class="aux"><div class="resc">Vidhya Munnamalai, et al. Semin Cell Dev Biol. ;24(5):480-489.</div><p class="links"><a class="dblinks" href="/pubmed/23548730" ref="ordinalpos=217&amp;ncbi_uid=6079349&amp;link_uid=23548730">Citation</a><a class="dblinks" href="/pmc/articles/PMC3690158/" ref="ordinalpos=217&amp;ncbi_uid=6079349&amp;link_uid=PMC3690158">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>218.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4072777/figure/ppat-1004165-g003/" ref="ordinalpos=218&amp;ncbi_uid=6317527&amp;link_uid=PMC4072777" href="/pmc/articles/PMC4072777/"><img src="/pmc/articles/instance/4072777/bin/ppat.1004165.g003.gif" src-large="/pmc/articles/instance/4072777/bin/ppat.1004165.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4072777/" ref="ordinalpos=218&amp;ncbi_uid=6317527&amp;link_uid=PMC4072777" image-link="/pmc/articles/PMC4072777/figure/ppat-1004165-g003/" class="imagepopup">Figure 3. Activation of <span class="highlight" style="background-color:">signaling</span> proteins in the IGF-1R canonical <span class="highlight" style="background-color:">pathway</span> in splenic macrophages from hamsters with VL..  From: Growth Factor and Th2 Cytokine <span class="highlight" style="background-color:">Signaling</span> Pathways Converge at STAT6 to Promote Arginase Expression in Progressive Experimental Visceral Leishmaniasis. </a></p><div class="supp"><p class="details"><div>(<b>A–B, D–J</b>) Immunoblot analysis of expression of proteins in the IGF-1R canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in splenic macrophages was performed as described in Fig. 2. <b>C</b>) Detection of phospho-IGFR by immunoblot in BMDMs uninfected (Un) or infected <i>in vitro</i> with <i>L. donovani</i> for 20 min to 24 hrs. Shown is an immunoblot from a single experiment. <b>K</b>) Network analysis showing the activation of the FGF and IGF-1 canonical <span class="highlight" style="background-color:">signaling</span> pathways generated by comparing the fold change of 32 <span class="highlight" style="background-color:">signaling</span> proteins in splenic macrophages from uninfected and infected (7, 14, and 28 days) hamsters using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software (Ingenuity Systems). The –log of the <i>p</i> value (vertical axis) represents the probability that the association of the data set in that <span class="highlight" style="background-color:">pathway</span> is due to chance. <b>L</b>) Simplified schematic of the canonical IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for reference.</div></p></div><div class="aux"><div class="resc">E. Yaneth Osorio, et al. PLoS Pathog. 2014 June;10(6):e1004165.</div><p class="links"><a class="dblinks" href="/pubmed/24967908" ref="ordinalpos=218&amp;ncbi_uid=6317527&amp;link_uid=24967908">Citation</a><a class="dblinks" href="/pmc/articles/PMC4072777/" ref="ordinalpos=218&amp;ncbi_uid=6317527&amp;link_uid=PMC4072777">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>219.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3804493/figure/pone-0078880-g004/" ref="ordinalpos=219&amp;ncbi_uid=5470862&amp;link_uid=PMC3804493" href="/pmc/articles/PMC3804493/"><img src="/pmc/articles/instance/3804493/bin/pone.0078880.g004.gif" src-large="/pmc/articles/instance/3804493/bin/pone.0078880.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3804493/" ref="ordinalpos=219&amp;ncbi_uid=5470862&amp;link_uid=PMC3804493" image-link="/pmc/articles/PMC3804493/figure/pone-0078880-g004/" class="imagepopup">Figure 4. Model for alcohol interaction with the Hippo <span class="highlight" style="background-color:">Pathway</span>..  From: Alcohol Interacts with Genetic Alteration of the Hippo Tumor Suppressor <span class="highlight" style="background-color:">Pathway</span> to Modulate Tissue Growth in Drosophila. </a></p><div class="supp"><p class="details"><div>As established, when Hippo <span class="highlight" style="background-color:">signaling</span> is intact (A), Yki is properly regulated and tissues maintain proper growth control. We propose that alcohol promotes activation of the <span class="highlight" style="background-color:">pathway</span> at or upstream of Hippo to target alcohol-relevant growth-promoting activities (B) to maintain proper growth control upon alcohol exposure. When Hippo <span class="highlight" style="background-color:">signaling</span> is impaired (C), Yki becomes overactive and promotes tissue overgrowth. We propose that when Hippo <span class="highlight" style="background-color:">signaling</span> is impaired in the presence of alcohol (D), Yki acts to promote overgrowth as established, and additional alcohol-relevant targets of the <span class="highlight" style="background-color:">pathway</span> act to promote further overgrowth.</div></p></div><div class="aux"><div class="resc">Anoj Ilanges, et al. PLoS One. 2013;8(10):e78880.</div><p class="links"><a class="dblinks" href="/pubmed/24205337" ref="ordinalpos=219&amp;ncbi_uid=5470862&amp;link_uid=24205337">Citation</a><a class="dblinks" href="/pmc/articles/PMC3804493/" ref="ordinalpos=219&amp;ncbi_uid=5470862&amp;link_uid=PMC3804493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>220.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4119234/figure/f5/" ref="ordinalpos=220&amp;ncbi_uid=6726298&amp;link_uid=PMC4119234" href="/pmc/articles/PMC4119234/"><img src="/pmc/articles/instance/4119234/bin/mv-v20-1109-f5.gif" src-large="/pmc/articles/instance/4119234/bin/mv-v20-1109-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4119234/" ref="ordinalpos=220&amp;ncbi_uid=6726298&amp;link_uid=PMC4119234" image-link="/pmc/articles/PMC4119234/figure/f5/" class="imagepopup">Figure 5.  From: ERK1/2/COX-2/PGE2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediates GPR91-dependent VEGF release in streptozotocin-induced diabetes. </a></p><div class="supp"><p class="details"><div>Succinate and the GPR91 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the retinas of STZ-induced diabetic rats. Each <i>solid line and arrow</i> denotes a step in an activating <span class="highlight" style="background-color:">pathway</span>. Succinate accumulation triggered by hyperglycemia activates GPR91 <span class="highlight" style="background-color:">signaling</span>, sequentially activating ERK1/2 and then upregulating COX-2 and PGE<sub>2</sub> expression, and later increasing VEGF release in retinal ganglion cells. The increase of VEGF induces retinal vascular dysfunction and plays a key role in the pathology of diabetic retinopathy. The <i>long dashed line and vertical lines</i> denotes a step in an inhibiting <span class="highlight" style="background-color:">pathway</span>. U0126 blocks the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the downstream expression of COX-2, PGE<sub>2</sub>, and VEGF. NS-398 blocks COX-2 and PGE<sub>2</sub> and decreases PGE<sub>2</sub>-mediated VEGF synthesis. </div></p></div><div class="aux"><div class="resc">Tingting Li, et al. Mol Vis. 2014;20:1109-1121.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=220&amp;ncbi_uid=6726298&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4119234/" ref="ordinalpos=220&amp;ncbi_uid=6726298&amp;link_uid=PMC4119234">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>221.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g003/" ref="ordinalpos=221&amp;ncbi_uid=6143992&amp;link_uid=PMC4018296" href="/pmc/articles/PMC4018296/"><img src="/pmc/articles/instance/4018296/bin/pone.0096776.g003.gif" src-large="/pmc/articles/instance/4018296/bin/pone.0096776.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4018296/" ref="ordinalpos=221&amp;ncbi_uid=6143992&amp;link_uid=PMC4018296" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g003/" class="imagepopup">Figure 3. The concept for deduction of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Chemical Genomic-Based <span class="highlight" style="background-color:">Pathway</span> Analyses for Epidermal Growth Factor-Mediated <span class="highlight" style="background-color:">Signaling</span> in Migrating Cancer Cells. </a></p><div class="supp"><p class="details"><div>Detailed description is presented in the Results section and Experimental Procedures section.</div></p></div><div class="aux"><div class="resc">Shigeyuki Magi, et al. PLoS One. 2014;9(5):e96776.</div><p class="links"><a class="dblinks" href="/pubmed/24820097" ref="ordinalpos=221&amp;ncbi_uid=6143992&amp;link_uid=24820097">Citation</a><a class="dblinks" href="/pmc/articles/PMC4018296/" ref="ordinalpos=221&amp;ncbi_uid=6143992&amp;link_uid=PMC4018296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>222.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g006/" ref="ordinalpos=222&amp;ncbi_uid=1718058&amp;link_uid=PMC2634968" href="/pmc/articles/PMC2634968/"><img src="/pmc/articles/instance/2634968/bin/pcbi.1000292.g006.gif" src-large="/pmc/articles/instance/2634968/bin/pcbi.1000292.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634968/" ref="ordinalpos=222&amp;ncbi_uid=1718058&amp;link_uid=PMC2634968" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g006/" class="imagepopup">Figure 6. A simplified illustration of the IL-1 DIOS <span class="highlight" style="background-color:">pathway</span>..  From: Identification of Potential <span class="highlight" style="background-color:">Pathway</span> Mediation Targets in Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>There were three critical network reactions that controlled the IL-1 induced activation of NF-kappa-B. Uninhibited IL-1 <span class="highlight" style="background-color:">signaling</span> induced formation of a TRAF6/Ajuba/PKCz/SQST1 complex followed by autophosphorylation at the Thr-560 residue of PKCz. This activated complex then signaled downstream to NF-kappa-B via IKK phosphorylation. The three critical network reactions inhibited IL-1 induced NF-kappa-B activation by preventing the formation and subsequent autophosphorylation of the TRAF6/Ajuba/PKCz/SQST1 complex. Unlike inhibitors such as IL-1R2 and soluble IL-1R, which mediate IL-1 <span class="highlight" style="background-color:">signaling</span> by preventing the activation of the IL-1 receptors, the three critical network reactions worked by disrupting other components of the IL-1 DIOS <span class="highlight" style="background-color:">pathway</span> and did not affect the activation of IL-1 receptors.</div></p></div><div class="aux"><div class="resc">Fan Li, et al. PLoS Comput Biol. 2009 February;5(2):e1000292.</div><p class="links"><a class="dblinks" href="/pubmed/19229310" ref="ordinalpos=222&amp;ncbi_uid=1718058&amp;link_uid=19229310">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634968/" ref="ordinalpos=222&amp;ncbi_uid=1718058&amp;link_uid=PMC2634968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>223.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3432501/figure/F1/" ref="ordinalpos=223&amp;ncbi_uid=4211643&amp;link_uid=PMC3432501" href="/pmc/articles/PMC3432501/"><img src="/pmc/articles/instance/3432501/bin/nihms159254f1.gif" src-large="/pmc/articles/instance/3432501/bin/nihms159254f1.jpg" alt="Figure 24.1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3432501/" ref="ordinalpos=223&amp;ncbi_uid=4211643&amp;link_uid=PMC3432501" image-link="/pmc/articles/PMC3432501/figure/F1/" class="imagepopup">Figure 24.1.  From: In Vitro <span class="highlight" style="background-color:">Signaling</span> by MAPK and NFκB Pathways Inhibited by Yersinia YopJ. </a></p><div class="supp"><p class="details"><div>MAPK and NF<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> pathways. The MAPK <span class="highlight" style="background-color:">pathway</span> is activated <i>in vitro</i> by Ras or Raf and the phosphorylation of MAPKinase (P-MAPK) is observed to indicate activation of the <span class="highlight" style="background-color:">pathway</span>. The NF<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> is activated byTRAF6, NIK, or MEKK1, and the phosphorylation of I<i>κ</i>B (P-I<i>κ</i>B) is observed to indicate activation of the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Sohini Mukherjee, et al. Methods Enzymol. 2008;438:343-353.</div><p class="links"><a class="dblinks" href="/pubmed/18413260" ref="ordinalpos=223&amp;ncbi_uid=4211643&amp;link_uid=18413260">Citation</a><a class="dblinks" href="/pmc/articles/PMC3432501/" ref="ordinalpos=223&amp;ncbi_uid=4211643&amp;link_uid=PMC3432501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>224.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2387053/figure/F2/" ref="ordinalpos=224&amp;ncbi_uid=1379749&amp;link_uid=PMC2387053" href="/pmc/articles/PMC2387053/"><img src="/pmc/articles/instance/2387053/bin/nihms49842f2.gif" src-large="/pmc/articles/instance/2387053/bin/nihms49842f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2387053/" ref="ordinalpos=224&amp;ncbi_uid=1379749&amp;link_uid=PMC2387053" image-link="/pmc/articles/PMC2387053/figure/F2/" class="imagepopup">Figure 2. Schematic diagram of IGF1-R <span class="highlight" style="background-color:">signaling</span> in mammals and C. elegans.  From: Aging of the brain, neurotrophin <span class="highlight" style="background-color:">signaling</span>, and Alzheimer's disease: is IGF1-R the common culprit?. </a></p><div class="supp"><p class="details"><div>The description of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is in the text. The common elements in mammals and worms are shown in red.</div></p></div><div class="aux"><div class="resc">Luigi Puglielli. Neurobiol Aging. ;29(6):795-811.</div><p class="links"><a class="dblinks" href="/pubmed/17313996" ref="ordinalpos=224&amp;ncbi_uid=1379749&amp;link_uid=17313996">Citation</a><a class="dblinks" href="/pmc/articles/PMC2387053/" ref="ordinalpos=224&amp;ncbi_uid=1379749&amp;link_uid=PMC2387053">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>225.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g006/" ref="ordinalpos=225&amp;ncbi_uid=6143989&amp;link_uid=PMC4018296" href="/pmc/articles/PMC4018296/"><img src="/pmc/articles/instance/4018296/bin/pone.0096776.g006.gif" src-large="/pmc/articles/instance/4018296/bin/pone.0096776.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4018296/" ref="ordinalpos=225&amp;ncbi_uid=6143989&amp;link_uid=PMC4018296" image-link="/pmc/articles/PMC4018296/figure/pone-0096776-g006/" class="imagepopup">Figure 6. Comparison of <span class="highlight" style="background-color:">signaling</span> network between EC109 cells and TT cells..  From: Chemical Genomic-Based <span class="highlight" style="background-color:">Pathway</span> Analyses for Epidermal Growth Factor-Mediated <span class="highlight" style="background-color:">Signaling</span> in Migrating Cancer Cells. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Common <span class="highlight" style="background-color:">signaling</span> network between EC109 cells and TT cells. (<b>b</b>) Specific EGF-induced <span class="highlight" style="background-color:">signaling</span> network in EC109 cells. (<b>c</b>) Specific EGF-induced <span class="highlight" style="background-color:">signaling</span> network in TT cells. The edge color is decided based on the sign of edges; red indicates positive <span class="highlight" style="background-color:">signaling</span>, and blue indicates negative <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shigeyuki Magi, et al. PLoS One. 2014;9(5):e96776.</div><p class="links"><a class="dblinks" href="/pubmed/24820097" ref="ordinalpos=225&amp;ncbi_uid=6143989&amp;link_uid=24820097">Citation</a><a class="dblinks" href="/pmc/articles/PMC4018296/" ref="ordinalpos=225&amp;ncbi_uid=6143989&amp;link_uid=PMC4018296">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>226.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3045689/figure/F4/" ref="ordinalpos=226&amp;ncbi_uid=2811753&amp;link_uid=PMC3045689" href="/pmc/articles/PMC3045689/"><img src="/pmc/articles/instance/3045689/bin/jcbn10-69f04.gif" src-large="/pmc/articles/instance/3045689/bin/jcbn10-69f04.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3045689/" ref="ordinalpos=226&amp;ncbi_uid=2811753&amp;link_uid=PMC3045689" image-link="/pmc/articles/PMC3045689/figure/F4/" class="imagepopup">Fig. 4.  From: Rebamipide suppresses TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in regulating IRF3/7 and IFN-?/? reduction. </a></p><div class="supp"><p class="details"><div>Rebamipide suppressed TBK1-IRF3/7-IFN-α/β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in DSS mice. Effect of rebamipide on TLR-TBK1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in colon specimens from mice given oral DDS examined by real-time RT-PCR of <i>TBK1</i>, <i>IRF3</i>, <i>IRF7</i>, <i>IFN-α</i> and <i>IFN-β</i>. Messenger RNA expression of these genes was increased because of colonic inflammation in DSS mice, but this increase was suppressed in DSS mice given daily rectal rebamipide.</div></p></div><div class="aux"><div class="resc">Naotaka Ogasawara, et al. J Clin Biochem Nutr. 2011 March;48(2):154-160.</div><p class="links"><a class="dblinks" href="/pubmed/21373269" ref="ordinalpos=226&amp;ncbi_uid=2811753&amp;link_uid=21373269">Citation</a><a class="dblinks" href="/pmc/articles/PMC3045689/" ref="ordinalpos=226&amp;ncbi_uid=2811753&amp;link_uid=PMC3045689">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>227.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711381/figure/F3/" ref="ordinalpos=227&amp;ncbi_uid=5099965&amp;link_uid=PMC3711381" href="/pmc/articles/PMC3711381/"><img src="/pmc/articles/instance/3711381/bin/nihms-442491-f0003.gif" src-large="/pmc/articles/instance/3711381/bin/nihms-442491-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711381/" ref="ordinalpos=227&amp;ncbi_uid=5099965&amp;link_uid=PMC3711381" image-link="/pmc/articles/PMC3711381/figure/F3/" class="imagepopup">Figure 3.  From: A novel target for Huntington's disease: ERK at the crossroads of <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Htt affects ERK <span class="highlight" style="background-color:">signaling</span> in multiple ways<b>.</b> The ERK cascade potentiates pro-survival mechanisms, i.e. inhibition of apoptosis and transcriptional activation of pro-survival genes, in response to various stimuli. Mutant huntingtin activates the ERK <span class="highlight" style="background-color:">pathway</span> in a cell-autonomous manner but also interferes with both upstream and downstream <span class="highlight" style="background-color:">signaling</span> events. The points at which mutant Htt affects ERK <span class="highlight" style="background-color:">signaling</span> both positively and negatively are indicated.</div></p></div><div class="aux"><div class="resc">László Bodai, et al. Bioessays. ;34(2):142-148.</div><p class="links"><a class="dblinks" href="/pubmed/22334892" ref="ordinalpos=227&amp;ncbi_uid=5099965&amp;link_uid=22334892">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711381/" ref="ordinalpos=227&amp;ncbi_uid=5099965&amp;link_uid=PMC3711381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>228.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4023177/figure/F3/" ref="ordinalpos=228&amp;ncbi_uid=6142458&amp;link_uid=PMC4023177" href="/pmc/articles/PMC4023177/"><img src="/pmc/articles/instance/4023177/bin/1477-5956-12-22-3.gif" src-large="/pmc/articles/instance/4023177/bin/1477-5956-12-22-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4023177/" ref="ordinalpos=228&amp;ncbi_uid=6142458&amp;link_uid=PMC4023177" image-link="/pmc/articles/PMC4023177/figure/F3/" class="imagepopup">Figure 3.  From: Quantitative proteomics analysis of early recurrence/metastasis of huge hepatocellular carcinoma following radical resection. </a></p><div class="supp"><p class="details"><div><b>The key <span class="highlight" style="background-color:">signaling</span> pathways involved in the early recurrence/metastasis (R/M</b><sub><b>≤6months</b></sub><b>) and late recurrence/metastasis (R/M</b><sub><b>6-12months</b></sub><b>) of H-HCC. (A)</b> The alternations of the ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the R/M<sub>≤6months</sub> group, including 6 up-regulated proteins (red labeling) and 7 down-regulated (green labeling) proteins. <b>(B)</b> The alternations of the TNF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the R/M<sub>≤6months</sub> group, including 4 up-regulated proteins (red labeling) and 9 down-regulated proteins (green labeling). <b>(C)</b> The alternations of the TNF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the R/M<sub>6-12months</sub> group, including 7 up-regulated proteins (red labeling) and 5 down-regulated proteins (green labeling). <b>(D)</b> the alternations of the NFκB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the R/M<sub>6-12months</sub> group, including 9 up-regulated proteins (red labeling) and 2 down-regulated (green labeling) proteins.</div></p></div><div class="aux"><div class="resc">Xinhui Huang, et al. Proteome Sci. 2014;12:22-22.</div><p class="links"><a class="dblinks" href="/pubmed/24839399" ref="ordinalpos=228&amp;ncbi_uid=6142458&amp;link_uid=24839399">Citation</a><a class="dblinks" href="/pmc/articles/PMC4023177/" ref="ordinalpos=228&amp;ncbi_uid=6142458&amp;link_uid=PMC4023177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>229.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4152115/figure/fig05/" ref="ordinalpos=229&amp;ncbi_uid=6561222&amp;link_uid=PMC4152115" href="/pmc/articles/PMC4152115/"><img src="/pmc/articles/instance/4152115/bin/ijcep0007-5369-f5.gif" src-large="/pmc/articles/instance/4152115/bin/ijcep0007-5369-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4152115/" ref="ordinalpos=229&amp;ncbi_uid=6561222&amp;link_uid=PMC4152115" image-link="/pmc/articles/PMC4152115/figure/fig05/" class="imagepopup">Figure 5.  From: Cerium promotes bone marrow stromal cells migration and osteogenic differentiation via Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Plausible molecular mechanism of migration and osteogenic differentiation of BMSCs by Ce through Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ying Hu, et al. Int J Clin Exp Pathol. 2014;7(8):5369-5378.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=229&amp;ncbi_uid=6561222&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4152115/" ref="ordinalpos=229&amp;ncbi_uid=6561222&amp;link_uid=PMC4152115">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>230.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4069080/figure/pone-0100690-g004/" ref="ordinalpos=230&amp;ncbi_uid=6303384&amp;link_uid=PMC4069080" href="/pmc/articles/PMC4069080/"><img src="/pmc/articles/instance/4069080/bin/pone.0100690.g004.gif" src-large="/pmc/articles/instance/4069080/bin/pone.0100690.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4069080/" ref="ordinalpos=230&amp;ncbi_uid=6303384&amp;link_uid=PMC4069080" image-link="/pmc/articles/PMC4069080/figure/pone-0100690-g004/" class="imagepopup">Figure 4. Transcript Complexity Value analysis of RA risk genes..  From: Novel Insights into the Regulatory Architecture of CD4+ T Cells in Rheumatoid Arthritis. </a></p><div class="supp"><p class="details"><div>TCV from the RA CD4+ T cell and control LCLs analyzed in RA risk genes. <b>A</b>: JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>B</b>: T cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>C</b>: TNF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>D</b>: Cell adhesion <span class="highlight" style="background-color:">pathway</span>. <b>E</b>: Toll-like receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>F</b>: Antigen processing and presentation <span class="highlight" style="background-color:">pathway</span>. <b>G</b>: B cell development function <span class="highlight" style="background-color:">pathway</span>. The genes that are framed represent those genes showing significant genomic regulation profiles between RA CD4+ T cells and LCLs (P&lt;0.05).</div></p></div><div class="aux"><div class="resc">Adrià Aterido, et al. PLoS One. 2014;9(6):e100690.</div><p class="links"><a class="dblinks" href="/pubmed/24959711" ref="ordinalpos=230&amp;ncbi_uid=6303384&amp;link_uid=24959711">Citation</a><a class="dblinks" href="/pmc/articles/PMC4069080/" ref="ordinalpos=230&amp;ncbi_uid=6303384&amp;link_uid=PMC4069080">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>231.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3565315/figure/f1-ijms-14-01179/" ref="ordinalpos=231&amp;ncbi_uid=4694573&amp;link_uid=PMC3565315" href="/pmc/articles/PMC3565315/"><img src="/pmc/articles/instance/3565315/bin/ijms-14-01179f1.gif" src-large="/pmc/articles/instance/3565315/bin/ijms-14-01179f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3565315/" ref="ordinalpos=231&amp;ncbi_uid=4694573&amp;link_uid=PMC3565315" image-link="/pmc/articles/PMC3565315/figure/f1-ijms-14-01179/" class="imagepopup">Figure 1.  From: Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Ovarian Cancer. </a></p><div class="supp"><p class="details"><div>Overview of the Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: a simplified model for Hh <span class="highlight" style="background-color:">signaling</span> in mammalian cells. SMO is the key transducer of the Hh <span class="highlight" style="background-color:">pathway</span>. (<b>a</b>) In the absence of the Hh ligands, the putative Hh receptor PTC is localized in the cilium and inhibits SMO <span class="highlight" style="background-color:">signaling</span>. Gli molecules are processed with the help of Su(Fu)/KIF7 molecules into repressor forms, which deactivate the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>b</b>) In the presence of Hh, PTC is displaced out of the cilium and unable to inhibit SMO. Hh reception facilitates conformational changes in SMO, promoting Gli activation (GliA) and stimulation of Hh target gene expression. Su(Fu) and KIF7 can inhibit this process.</div></p></div><div class="aux"><div class="resc">Joanna Szkandera, et al. Int J Mol Sci. 2013 January;14(1):1179-1196.</div><p class="links"><a class="dblinks" href="/pubmed/23303278" ref="ordinalpos=231&amp;ncbi_uid=4694573&amp;link_uid=23303278">Citation</a><a class="dblinks" href="/pmc/articles/PMC3565315/" ref="ordinalpos=231&amp;ncbi_uid=4694573&amp;link_uid=PMC3565315">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>232.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2570709/figure/F1/" ref="ordinalpos=232&amp;ncbi_uid=1623100&amp;link_uid=PMC2570709" href="/pmc/articles/PMC2570709/"><img src="/pmc/articles/instance/2570709/bin/nihms73332f1.gif" src-large="/pmc/articles/instance/2570709/bin/nihms73332f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2570709/" ref="ordinalpos=232&amp;ncbi_uid=1623100&amp;link_uid=PMC2570709" image-link="/pmc/articles/PMC2570709/figure/F1/" class="imagepopup">Figure 1. Wnt <span class="highlight" style="background-color:">signaling</span> is mediated through the β-catenin (canonical) and the calcium (non-canonical) pathways.  From: Rogue proliferation versus restorative protection: Where do we draw the line for Wnt and Forkhead <span class="highlight" style="background-color:">signaling</span>?. </a></p><div class="supp"><p class="details"><div>A representative schematic for the pathways is shown. The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is referred to as the Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> since it can regulate β-catenin. All Wnt <span class="highlight" style="background-color:">signaling</span> pathways are initiated by interaction of Wnt proteins with Frizzled (FZD) receptors. Once Wnt protein binds to the FZD receptor and the co-receptor low-density lipoprotein receptor-related protein (LRP)-5/6, this is followed by recruitment of Dishevelled. Dishevelled can inhibit glycogen synthase kinase (GSK-3β) activity. The non-canonical or calcium Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has two intracellular <span class="highlight" style="background-color:">signaling</span> cascades that consist of the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> and the Wnt/planar cell polarity (PCP) <span class="highlight" style="background-color:">pathway</span>. In the Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span>, Wnt protein binds to FZD receptors on the cell surface resulting in several cellular processes that can involve stimulation of heterotrimeric G proteins and increased intracellular Ca<sup>2+</sup> release. Yet, the distinction between these pathways is sometimes not distinct and may involve non-canonical pathways, such as Wnt5a, that antagonize canonical pathways by leading to the degradation of β-catenin.<br />Lef: Lymphoid enhancer-binding factor; Tcf: T cell factor.</div></p></div><div class="aux"><div class="resc">Kenneth Maiese, et al. Expert Opin Ther Targets. ;12(7):905-916.</div><p class="links"><a class="dblinks" href="/pubmed/18554157" ref="ordinalpos=232&amp;ncbi_uid=1623100&amp;link_uid=18554157">Citation</a><a class="dblinks" href="/pmc/articles/PMC2570709/" ref="ordinalpos=232&amp;ncbi_uid=1623100&amp;link_uid=PMC2570709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>233.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F4/" ref="ordinalpos=233&amp;ncbi_uid=5822248&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g004.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g004.jpg" alt="Figure 2b" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=233&amp;ncbi_uid=5822248&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F4/" class="imagepopup">Figure 2b.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancers -Type II- Autocrine Ligand-dependent cancers</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=233&amp;ncbi_uid=5822248&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=233&amp;ncbi_uid=5822248&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>234.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2578836/figure/F7/" ref="ordinalpos=234&amp;ncbi_uid=1556901&amp;link_uid=PMC2578836" href="/pmc/articles/PMC2578836/"><img src="/pmc/articles/instance/2578836/bin/nihms63415f7.gif" src-large="/pmc/articles/instance/2578836/bin/nihms63415f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2578836/" ref="ordinalpos=234&amp;ncbi_uid=1556901&amp;link_uid=PMC2578836" image-link="/pmc/articles/PMC2578836/figure/F7/" class="imagepopup">Fig. 7.  From: Smad2 functions as a co-activator of canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> independent of Smad4 through histone acetyltransferase activity of p300. </a></p><div class="supp"><p class="details"><div>A proposed model for transcriptional regulatory mechanism of canonical Wnt/β-catenin/Tcf4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Smad2 and Smad4.</div></p></div><div class="aux"><div class="resc">Morihisa Hirota, et al. Cell Signal. ;20(9):1632-1641.</div><p class="links"><a class="dblinks" href="/pubmed/18595660" ref="ordinalpos=234&amp;ncbi_uid=1556901&amp;link_uid=18595660">Citation</a><a class="dblinks" href="/pmc/articles/PMC2578836/" ref="ordinalpos=234&amp;ncbi_uid=1556901&amp;link_uid=PMC2578836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>235.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3095674/figure/F4/" ref="ordinalpos=235&amp;ncbi_uid=3490074&amp;link_uid=PMC3095674" href="/pmc/articles/PMC3095674/"><img src="/pmc/articles/instance/3095674/bin/nihms-275584-f0004.gif" src-large="/pmc/articles/instance/3095674/bin/nihms-275584-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3095674/" ref="ordinalpos=235&amp;ncbi_uid=3490074&amp;link_uid=PMC3095674" image-link="/pmc/articles/PMC3095674/figure/F4/" class="imagepopup">Figure 4. CsA activates TGFβ activated kinase 1 (TAK1) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Procarcinogenic Effects of Cyclosporine A are Mediated through the Activation of TAK1/TAB1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Summary of the cascades of events triggered by CsA. .CsA enhances TAK1 by activating TGFβ-dependent <span class="highlight" style="background-color:">signaling</span> . CsA also augments the expression of upstream kinases ZMP and AMPK in addition to IRAK. The enhancement in the expression/activity of these <span class="highlight" style="background-color:">signaling</span> proteins contributes to MAP kinase and NFkB activation, which together triggers proliferation, reduced apoptosis and enhanced angiogenesis.</div></p></div><div class="aux"><div class="resc">Jianmin Xu, et al. Biochem Biophys Res Commun. ;408(3):363-368.</div><p class="links"><a class="dblinks" href="/pubmed/21333626" ref="ordinalpos=235&amp;ncbi_uid=3490074&amp;link_uid=21333626">Citation</a><a class="dblinks" href="/pmc/articles/PMC3095674/" ref="ordinalpos=235&amp;ncbi_uid=3490074&amp;link_uid=PMC3095674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>236.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2151969/figure/Fig5/" ref="ordinalpos=236&amp;ncbi_uid=1248631&amp;link_uid=PMC2151969" href="/pmc/articles/PMC2151969/"><img src="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig5_HTML.gif" src-large="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig5_HTML.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2151969/" ref="ordinalpos=236&amp;ncbi_uid=1248631&amp;link_uid=PMC2151969" image-link="/pmc/articles/PMC2151969/figure/Fig5/" class="imagepopup">Fig. 5.  From: FGF-2 increases osteogenic and chondrogenic differentiation potentials of human mesenchymal stem cells by inactivation of TGF-? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Genes up-regulated and down-regulated by FGF-2 in TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes up-regulated and down-regulated by FGF-2 (Fig. 2) were mapped with the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. The red color showed up-regulated genes and green color showed down-regulated genes</div></p></div><div class="aux"><div class="resc">Tomomi Ito, et al. Cytotechnology. 2008 January;56(1):1-7.</div><p class="links"><a class="dblinks" href="/pubmed/19002835" ref="ordinalpos=236&amp;ncbi_uid=1248631&amp;link_uid=19002835">Citation</a><a class="dblinks" href="/pmc/articles/PMC2151969/" ref="ordinalpos=236&amp;ncbi_uid=1248631&amp;link_uid=PMC2151969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>237.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3097097/figure/F2/" ref="ordinalpos=237&amp;ncbi_uid=3318172&amp;link_uid=PMC3097097" href="/pmc/articles/PMC3097097/"><img src="/pmc/articles/instance/3097097/bin/nihms231256f2.gif" src-large="/pmc/articles/instance/3097097/bin/nihms231256f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3097097/" ref="ordinalpos=237&amp;ncbi_uid=3318172&amp;link_uid=PMC3097097" image-link="/pmc/articles/PMC3097097/figure/F2/" class="imagepopup">Figure 2. The Interleukin-17 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (MetaCore).  From: Characterization of the myometrial transcriptome and biological pathways of spontaneous human labor at term. </a></p><div class="supp"><p class="details"><div>Display of differentially expressed genes in human myometrium during term labor mapped on the MetaCore Interleukin-17 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Red thermometers indicate gene upregulation in labor while blue thermometers indicate downregulation in labor. Of note, the majority of differentially regulated genes in this <span class="highlight" style="background-color:">pathway</span> are those involved in inflammation and chemotaxis (circled genes).</div></p></div><div class="aux"><div class="resc">Pooja Mittal, et al. J Perinat Med. ;38(6):617-643.</div><p class="links"><a class="dblinks" href="/pubmed/20629487" ref="ordinalpos=237&amp;ncbi_uid=3318172&amp;link_uid=20629487">Citation</a><a class="dblinks" href="/pmc/articles/PMC3097097/" ref="ordinalpos=237&amp;ncbi_uid=3318172&amp;link_uid=PMC3097097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>238.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4110525/figure/F4/" ref="ordinalpos=238&amp;ncbi_uid=6474943&amp;link_uid=PMC4110525" href="/pmc/articles/PMC4110525/"><img src="/pmc/articles/instance/4110525/bin/1476-4598-13-139-4.gif" src-large="/pmc/articles/instance/4110525/bin/1476-4598-13-139-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4110525/" ref="ordinalpos=238&amp;ncbi_uid=6474943&amp;link_uid=PMC4110525" image-link="/pmc/articles/PMC4110525/figure/F4/" class="imagepopup">Figure 4.  From: Targeting Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> positively affects the sensitivity of osteosarcoma to cisplatin by regulating the expression and/or activity of Caspase family. </a></p><div class="supp"><p class="details"><div><b>Inhibition of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> decreased the sensitivity of MG63 and Saos-2 to cisplatin.</b> Real-time PCR indicated that HES1 mRNA in MG63 and Saos-2 had significantly less expression after DAPT treatment <b>(A and B)</b>. The sensitivity of MG63 and Saos-2 to cisplatin has significantly decreased after the inhibition of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(C and D)</b>. (*P &lt; 0.05; **P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Lei Wang, et al. Mol Cancer. 2014;13:139-139.</div><p class="links"><a class="dblinks" href="/pubmed/24894297" ref="ordinalpos=238&amp;ncbi_uid=6474943&amp;link_uid=24894297">Citation</a><a class="dblinks" href="/pmc/articles/PMC4110525/" ref="ordinalpos=238&amp;ncbi_uid=6474943&amp;link_uid=PMC4110525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>239.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3654375/figure/F2/" ref="ordinalpos=239&amp;ncbi_uid=4898792&amp;link_uid=PMC3654375" href="/pmc/articles/PMC3654375/"><img src="/pmc/articles/instance/3654375/bin/nihms462306f2a.gif" src-large="/pmc/articles/instance/3654375/bin/nihms462306f2a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3654375/figure/F2/" ref="ordinalpos=239&amp;ncbi_uid=4898792&amp;link_uid=PMC3654375" href="/pmc/articles/PMC3654375/"><img src="/pmc/articles/instance/3654375/bin/nihms462306f2b.gif" src-large="/pmc/articles/instance/3654375/bin/nihms462306f2b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3654375/" ref="ordinalpos=239&amp;ncbi_uid=4898792&amp;link_uid=PMC3654375" image-link="/pmc/articles/PMC3654375/figure/F2/" class="imagepopup">Figure 2. Ephrin-Eph <span class="highlight" style="background-color:">signaling</span> in the nervous and vascular systems.  From: Guidance from above: common cues direct distinct <span class="highlight" style="background-color:">signaling</span> outcomes in vascular and neural patterning. </a></p><div class="supp"><p class="details"><div><b>A: ephrin-A-EphA <span class="highlight" style="background-color:">signaling</span> in the nervous system</b>. The Rap <span class="highlight" style="background-color:">pathway</span> is shown in yellow: EphA activation leads to tyrosine phosphorylation of SPAR, a RapGAP that binds to EphA4 through its PDZ domain. SPAR then inactivates Rap. The RhoA <span class="highlight" style="background-color:">pathway</span> is shown in light blue: Upon ephrin-A treatment, EphA recruits and phosphorylates Cdk5, which in turn phosphorylates and activates a RhoGEF, ephexin, activating RhoA. RhoA then activates ROCK, leading to dendritic spine retraction (as shown by the dashed pink box). The dashed blue box indicates the part of this <span class="highlight" style="background-color:">pathway</span> that has been shown to be important for growth cone collapse. The Rac <span class="highlight" style="background-color:">pathway</span> is shown in green: Activation of EphA receptors leads to binding and phosphorylation of the RacGAP α2-chimaerin, which then inactivaes Rac1 in cortical neurons, leading to growth cone collapse. In retinal axons, ephrin binding to EphA receptors leads to activation of the Rho family GEF Vav2, which then activates Rac1. Vav2-mediated activation of Rac1 leads to endocytosis of the membrane, eventually leading to growth cone collapse. In reverse <span class="highlight" style="background-color:">signaling</span>, ephrin-A associates with the p75-NTR receptor. This complex works to phosphorylate Fyn and cause axon repulsion from EphA-expressing cells when EphA binds ephrin-A. <b>B: ephrin–A-EphA <span class="highlight" style="background-color:">signaling</span> in the vascular system</b>. There is limited information on how this <span class="highlight" style="background-color:">signaling</span> regulates angiogenesis, but ephrin-A expression is known to be regulated by HIF-2α expression in surrounding tissue and <span class="highlight" style="background-color:">signaling</span> through EphA in the vascular system can lead to angiogenesis in tumor models. <b>C: ephrin–B-EphB <span class="highlight" style="background-color:">signaling</span> in the nervous system</b>. EphB is recruited to the membrane by GRIP1, a multi-PDZ domain scaffolding protein. The Rac1 <span class="highlight" style="background-color:">pathway</span> is shown in green: Rac1 is activated by the GEFs Kalirin and Tiam1. Both Tiam1 and Kalirin bind to and are phosphorylated by the activated receptor and then activate Rac1, leading to increased dendrite morphogenesis. The RhoA <span class="highlight" style="background-color:">pathway</span> is shown in blue: EphA activation leads to phosphorylation of FAK, which then activates RhoA, activating ROCK and leading to increased dendrite morphogenesis. For reverse <span class="highlight" style="background-color:">signaling</span> through the ephrin-B ligand, several adaptor proteins are required. GRIP1 binds to ephrin-B3 and is perhaps involved in clustering it at the synapse. The SH2-SH3 domain containing adaptor protein Grb4 binds ephrin-Bs and links them with multiple downstream regulators and enhancing dendrite morphogenesis in the ligand-expressing cell. <b>D: ephrin-B–EphB <span class="highlight" style="background-color:">signaling</span> in the vascular system</b>. EphB activation leads to the activation of RhoA and ROCK. In addition, EphB <span class="highlight" style="background-color:">signaling</span> enhances SFD/CXCR4 <span class="highlight" style="background-color:">signaling</span> as seen by increased phosphorylation of Akt. Both pathways lead to endothelial cell migration and sprouting. For reverse <span class="highlight" style="background-color:">signaling</span>, ephrin-B binds the SH2-SH3 domain containing adaptor protein Grb4, leading to vessel sprouting, presumably through multiple downstream regulators recruited by Grb4.</div></p></div><div class="aux"><div class="resc">Maria V. Gelfand, et al. Trends Cell Biol. 2009 March;19(3):99-110.</div><p class="links"><a class="dblinks" href="/pubmed/19200729" ref="ordinalpos=239&amp;ncbi_uid=4898792&amp;link_uid=19200729">Citation</a><a class="dblinks" href="/pmc/articles/PMC3654375/" ref="ordinalpos=239&amp;ncbi_uid=4898792&amp;link_uid=PMC3654375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>240.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2151969/figure/Fig4/" ref="ordinalpos=240&amp;ncbi_uid=1248635&amp;link_uid=PMC2151969" href="/pmc/articles/PMC2151969/"><img src="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig4_HTML.gif" src-large="/pmc/articles/instance/2151969/bin/10616_2007_9092_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2151969/" ref="ordinalpos=240&amp;ncbi_uid=1248635&amp;link_uid=PMC2151969" image-link="/pmc/articles/PMC2151969/figure/Fig4/" class="imagepopup">Fig. 4.  From: FGF-2 increases osteogenic and chondrogenic differentiation potentials of human mesenchymal stem cells by inactivation of TGF-? <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Genes up-regulated and down-regulated by FGF-2 in IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes up-regulated and down-regulated by FGF-2 (Fig. 2) were mapped with the IGF-I <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. The red color showed up-regulated genes and green color showed down-regulated genes</div></p></div><div class="aux"><div class="resc">Tomomi Ito, et al. Cytotechnology. 2008 January;56(1):1-7.</div><p class="links"><a class="dblinks" href="/pubmed/19002835" ref="ordinalpos=240&amp;ncbi_uid=1248635&amp;link_uid=19002835">Citation</a><a class="dblinks" href="/pmc/articles/PMC2151969/" ref="ordinalpos=240&amp;ncbi_uid=1248635&amp;link_uid=PMC2151969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>241.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3167819/figure/pone-0024225-g001/" ref="ordinalpos=241&amp;ncbi_uid=3217032&amp;link_uid=PMC3167819" href="/pmc/articles/PMC3167819/"><img src="/pmc/articles/instance/3167819/bin/pone.0024225.g001.gif" src-large="/pmc/articles/instance/3167819/bin/pone.0024225.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3167819/" ref="ordinalpos=241&amp;ncbi_uid=3217032&amp;link_uid=PMC3167819" image-link="/pmc/articles/PMC3167819/figure/pone-0024225-g001/" class="imagepopup">Figure 1. Schematic representation of the mathematical model of fate decision by breast cancer stem cells..  From: Dickkopf1 Regulates Fate Decision and Drives Breast Cancer Stem Cells to Differentiation: An Experimentally Supported Mathematical Model. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces LEF/TCF activation, resulting in up-regulation of Pf, Dkk1 and Notch <span class="highlight" style="background-color:">signaling</span> activators. Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also down-regulates E-cadherins. Dkk1 and E-cadherins inhibit Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by DSL up-regulates Hes expression, which inhibits Df. Pf, proliferation factor; Df, Differentiation factors; C<sub>P</sub>, Pf threshold; C<sub>M</sub>, Df threshold; DSL, Delta, Serrate, Lag-2; Dkk1, Dickkopf1 protein. Pale blue – Wnt protein; purple – Notch <span class="highlight" style="background-color:">pathway</span> protein; green arrow – activation; grey arrow – inhibition; dashed arrow – threshold-dependent effect.</div></p></div><div class="aux"><div class="resc">Zvia Agur, et al. PLoS One. 2011;6(9):e24225.</div><p class="links"><a class="dblinks" href="/pubmed/21915302" ref="ordinalpos=241&amp;ncbi_uid=3217032&amp;link_uid=21915302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3167819/" ref="ordinalpos=241&amp;ncbi_uid=3217032&amp;link_uid=PMC3167819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>242.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC201163/figure/f1/" ref="ordinalpos=242&amp;ncbi_uid=252384&amp;link_uid=PMC201163" href="/pmc/articles/PMC201163/"><img src="/pmc/articles/instance/201163/bin/ac1030622001.gif" src-large="/pmc/articles/instance/201163/bin/ac1030622001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC201163/" ref="ordinalpos=242&amp;ncbi_uid=252384&amp;link_uid=PMC201163" image-link="/pmc/articles/PMC201163/figure/f1/" class="imagepopup">FIG. 1. From: Cyclic AMP <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Modulates Susceptibility of Candida Species and Saccharomyces cerevisiae to Antifungal Azoles and Other Sterol Biosynthesis Inhibitors. </a></p><div class="supp"><p class="details"><div>Simplified model of the yeast cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (35).</div></p></div><div class="aux"><div class="resc">Pooja Jain, et al. Antimicrob Agents Chemother. 2003 October;47(10):3195-3201.</div><p class="links"><a class="dblinks" href="/pubmed/14506030" ref="ordinalpos=242&amp;ncbi_uid=252384&amp;link_uid=14506030">Citation</a><a class="dblinks" href="/pmc/articles/PMC201163/" ref="ordinalpos=242&amp;ncbi_uid=252384&amp;link_uid=PMC201163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>243.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2988058/figure/F6/" ref="ordinalpos=243&amp;ncbi_uid=2582265&amp;link_uid=PMC2988058" href="/pmc/articles/PMC2988058/"><img src="/pmc/articles/instance/2988058/bin/1471-2172-11-53-6.gif" src-large="/pmc/articles/instance/2988058/bin/1471-2172-11-53-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2988058/" ref="ordinalpos=243&amp;ncbi_uid=2582265&amp;link_uid=PMC2988058" image-link="/pmc/articles/PMC2988058/figure/F6/" class="imagepopup">Figure 6.  From: PAR1- and PAR2-induced innate immune markers are negatively regulated by PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral keratinocytes. </a></p><div class="supp"><p class="details"><div><b>PAR1 and PAR2 <span class="highlight" style="background-color:">signaling</span> involves MAPK and PI3K <span class="highlight" style="background-color:">signaling</span> pathways in the induction of innate immune responses</b>. Protease-activated receptors are G-protein-coupled receptors. Once activated, they signal via ERK1/2 and p38 MAPK to induce innate immune responses. This action is limited by activation of PI3K-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This inhibitory effect is predominantly via blocking of p38 activation.</div></p></div><div class="aux"><div class="resc">Maryam G Rohani, et al. BMC Immunol. 2010;11:53-53.</div><p class="links"><a class="dblinks" href="/pubmed/21029417" ref="ordinalpos=243&amp;ncbi_uid=2582265&amp;link_uid=21029417">Citation</a><a class="dblinks" href="/pmc/articles/PMC2988058/" ref="ordinalpos=243&amp;ncbi_uid=2582265&amp;link_uid=PMC2988058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>244.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2532600/figure/F2/" ref="ordinalpos=244&amp;ncbi_uid=1587649&amp;link_uid=PMC2532600" href="/pmc/articles/PMC2532600/"><img src="/pmc/articles/instance/2532600/bin/nihms56058f2.gif" src-large="/pmc/articles/instance/2532600/bin/nihms56058f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2532600/" ref="ordinalpos=244&amp;ncbi_uid=1587649&amp;link_uid=PMC2532600" image-link="/pmc/articles/PMC2532600/figure/F2/" class="imagepopup">Fig. 2. Wnt <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span>.  From: Wnt <span class="highlight" style="background-color:">signaling</span>: the good and the bad. </a></p><div class="supp"><p class="details"><div>(A) When Wnt <span class="highlight" style="background-color:">signaling</span> is “turned off”, cytoplasmic β-catenin is degraded by the Axin complex consisting of APC, GSK3 and CKIα. Within this complex, CKIα and GSK3 phosphorylate β-catenin, phosphorylated β-catenin is recognized by β-Trcp and degraded via the ubiquitination-proteasome <span class="highlight" style="background-color:">pathway</span>. Wnt protein can be inhibited by sFRPs, and LRP5/6 can be inhibited by DKK1. (B) When Wnt <span class="highlight" style="background-color:">signaling</span> is “turned on”, Wnt protein binds its receptor Frizzled and co-receptor LRP5/6 and stimulates LRP5/6 phosphorylation with the help of Dishevelled. Phosphorylated LRP5/6 recruits Axin to the membrane and disrupts the Axin complex. β-catenin accumulates in the cytoplasm and enters nucleus, where it binds TCF/LEF and co-activators, such as pygopus, legless and CBP/p300, and activates gene expression.</div></p></div><div class="aux"><div class="resc">Xi Chen, et al. Acta Biochim Biophys Sin (Shanghai). ;40(7):577-594.</div><p class="links"><a class="dblinks" href="/pubmed/18604449" ref="ordinalpos=244&amp;ncbi_uid=1587649&amp;link_uid=18604449">Citation</a><a class="dblinks" href="/pmc/articles/PMC2532600/" ref="ordinalpos=244&amp;ncbi_uid=1587649&amp;link_uid=PMC2532600">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>245.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2396493/figure/F1/" ref="ordinalpos=245&amp;ncbi_uid=1419144&amp;link_uid=PMC2396493" href="/pmc/articles/PMC2396493/"><img src="/pmc/articles/instance/2396493/bin/nihms47079f1.gif" src-large="/pmc/articles/instance/2396493/bin/nihms47079f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2396493/" ref="ordinalpos=245&amp;ncbi_uid=1419144&amp;link_uid=PMC2396493" image-link="/pmc/articles/PMC2396493/figure/F1/" class="imagepopup">Figure 1.  From: How does suppression of IGF-1 <span class="highlight" style="background-color:">signaling</span> by DNA damage affect aging and longevity?. </a></p><div class="supp"><p class="details"><div>Effects of p53 on the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A simplified version of the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is presented at the left. P53, shown to the right of the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, is activated by DNA damage and can induce cell cycle arrest, senescence or apoptosis. In addition, four entry points of p53 into the IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are indicated. From top to bottom: (1) p53 transcriptionally activates the IGF-BP3 gene, whose product binds to IGF-1 and inhibits its function; (2) p53 transcriptionally downregulates the IGF-1 receptor; (3) p53 transcriptionally inhibits the p110 subunit of the PI3K subunit; and (4) p53 transcriptionally activates the pTEN gene, which encodes a phosphatase that dephosphorylates PIP3 to downregulate IGF-1 <span class="highlight" style="background-color:">signaling</span>. The end result of suppression of IGF-1 <span class="highlight" style="background-color:">signaling</span> is decreased phoshorylation of Forkhead (FOXO) transcription factors by AKT and increased activity of Forkhead in transcriptionally activating anti-oxidant and DNA repair genes.</div></p></div><div class="aux"><div class="resc">George Hinkal, et al. Mech Ageing Dev. ;129(5):243-253.</div><p class="links"><a class="dblinks" href="/pubmed/18374391" ref="ordinalpos=245&amp;ncbi_uid=1419144&amp;link_uid=18374391">Citation</a><a class="dblinks" href="/pmc/articles/PMC2396493/" ref="ordinalpos=245&amp;ncbi_uid=1419144&amp;link_uid=PMC2396493">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>246.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3976829/figure/fig4/" ref="ordinalpos=246&amp;ncbi_uid=6046969&amp;link_uid=PMC3976829" href="/pmc/articles/PMC3976829/"><img src="/pmc/articles/instance/3976829/bin/BMRI2014-724274.004.gif" src-large="/pmc/articles/instance/3976829/bin/BMRI2014-724274.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3976829/" ref="ordinalpos=246&amp;ncbi_uid=6046969&amp;link_uid=PMC3976829" image-link="/pmc/articles/PMC3976829/figure/fig4/" class="imagepopup">Figure 4.  From: Analysis of Differentially Expressed Genes and <span class="highlight" style="background-color:">Signaling</span> Pathways Related to Intramuscular Fat Deposition in Skeletal Muscle of Sex-Linked Dwarf Chickens. </a></p><div class="supp"><p class="details"><div>Schematic illustrations for potential <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of IMF deposition in skeletal muscles regulated by GHR and SOCS3. GHR increases the expression of SOCS3. SOCS3 affected adipogenesis by inhibiting the expression of LERP and IRS1, participating in adipocytokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IGF2BP3 affected insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by regulating IGFs.</div></p></div><div class="aux"><div class="resc">Yaqiong Ye, et al. Biomed Res Int. 2014;2014:724274.</div><p class="links"><a class="dblinks" href="/pubmed/24757673" ref="ordinalpos=246&amp;ncbi_uid=6046969&amp;link_uid=24757673">Citation</a><a class="dblinks" href="/pmc/articles/PMC3976829/" ref="ordinalpos=246&amp;ncbi_uid=6046969&amp;link_uid=PMC3976829">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>247.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092286/figure/fig1-1947601911407325/" ref="ordinalpos=247&amp;ncbi_uid=3024891&amp;link_uid=PMC3092286" href="/pmc/articles/PMC3092286/"><img src="/pmc/articles/instance/3092286/bin/10.1177_1947601911407325-fig1.gif" src-large="/pmc/articles/instance/3092286/bin/10.1177_1947601911407325-fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092286/" ref="ordinalpos=247&amp;ncbi_uid=3024891&amp;link_uid=PMC3092286" image-link="/pmc/articles/PMC3092286/figure/fig1-1947601911407325/" class="imagepopup">Figure 1. From: PTEN Tumor Suppressor Network in PI3K-Akt <span class="highlight" style="background-color:">Pathway</span> Control. </a></p><div class="supp"><p class="details"><div>Nonlinear <span class="highlight" style="background-color:">signaling</span> through the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>. Growth factors activate receptor tyrosine kinases (RTKs) that further activate parallel growth pathways. The MAPK <span class="highlight" style="background-color:">pathway</span>, which signals via Ras and downstream effectors, is an example of a linear growth-promoting <span class="highlight" style="background-color:">pathway</span>. The PI3K-Akt <span class="highlight" style="background-color:">pathway</span> is an example of a nonlinear <span class="highlight" style="background-color:">pathway</span> that signals through protein and lipid effectors and through layers of oncoproteins (red) and growth suppressors (blue). Negative feedback loops of the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> are shown in green (repressing) or orange (activating). The effectors of the <span class="highlight" style="background-color:">pathway</span> whose alterations show combinatorial additive effects with PTEN inactivation in cancers are encircled.</div></p></div><div class="aux"><div class="resc">Maria-Magdalena Georgescu. Genes Cancer. 2010 December;1(12):1170-1177.</div><p class="links"><a class="dblinks" href="/pubmed/21779440" ref="ordinalpos=247&amp;ncbi_uid=3024891&amp;link_uid=21779440">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092286/" ref="ordinalpos=247&amp;ncbi_uid=3024891&amp;link_uid=PMC3092286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>248.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885591/figure/F2/" ref="ordinalpos=248&amp;ncbi_uid=3003975&amp;link_uid=PMC2885591" href="/pmc/articles/PMC2885591/"><img src="/pmc/articles/instance/2885591/bin/nihms208471f2.gif" src-large="/pmc/articles/instance/2885591/bin/nihms208471f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885591/" ref="ordinalpos=248&amp;ncbi_uid=3003975&amp;link_uid=PMC2885591" image-link="/pmc/articles/PMC2885591/figure/F2/" class="imagepopup">Figure 2. TOR <span class="highlight" style="background-color:">pathway</span>, a conserved mediator of lifespan extension in multiple species.  From: With TOR less is more: a key role for the conserved nutrient sensing TOR <span class="highlight" style="background-color:">pathway</span> in aging. </a></p><div class="supp"><p class="details"><div>The diagram represents the genetic interactions in the TOR <span class="highlight" style="background-color:">signaling</span> network that execute lifespan extension in yeast, worms, flies and mice. The diagram summarizes the lifespan data for genes in the TOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or those that demonstrate a genetic interaction with mutants in the TOR <span class="highlight" style="background-color:">pathway</span>. The shaded area refers to components of the TOR <span class="highlight" style="background-color:">pathway</span> that show conserved effects on lifespan extension in different species. See main text for details.</div></p></div><div class="aux"><div class="resc">Pankaj Kapahi, et al. Cell Metab. ;11(6):453-465.</div><p class="links"><a class="dblinks" href="/pubmed/20519118" ref="ordinalpos=248&amp;ncbi_uid=3003975&amp;link_uid=20519118">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885591/" ref="ordinalpos=248&amp;ncbi_uid=3003975&amp;link_uid=PMC2885591">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>249.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC324544/figure/F6/" ref="ordinalpos=249&amp;ncbi_uid=431989&amp;link_uid=PMC324544" href="/pmc/articles/PMC324544/"><img src="/pmc/articles/instance/324544/bin/JCI0419872.f6.gif" src-large="/pmc/articles/instance/324544/bin/JCI0419872.f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC324544/" ref="ordinalpos=249&amp;ncbi_uid=431989&amp;link_uid=PMC324544" image-link="/pmc/articles/PMC324544/figure/F6/" class="imagepopup">Figure 6.  From: Glycoprotein 130 regulates bone turnover and bone size by distinct downstream <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>Summary of the effects of gp130 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mutations on bone structure and bone cell function. (<b>a</b>) In WT mice, IL-6 stimulates osteoblast proliferation via the gp130-STAT1/3 <span class="highlight" style="background-color:">pathway</span>, and the osteoblasts in turn stimulate osteoclastogenesis by the RANKL <span class="highlight" style="background-color:">pathway</span>. Osteoclastogenesis is also inhibited by gp130 family cytokines, acting through the gp130-SHP2/Ras/MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> within the hematopoietic lineage. Chondrocyte proliferation is stimulated by gp130 family cytokines through the gp130-STAT1/3 <span class="highlight" style="background-color:">pathway</span>. (<b>b</b>) In the absence of the gp130-induced STAT1/3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, indicated by the red X (in Δ/Δ mice), chondrocyte proliferation is no longer stimulated through the gp130-STAT1/3 <span class="highlight" style="background-color:">pathway</span>, leading to reduced bone length. Inhibition of osteoclastogenesis through the gp130-SHP2/Ras/MAPK <span class="highlight" style="background-color:">pathway</span> remains intact, and while IL-6 stimulation of osteoblast differentiation can no longer occur through the gp130-STAT1/3 <span class="highlight" style="background-color:">pathway</span>, this is clearly a minor <span class="highlight" style="background-color:">pathway</span> not required for the normal level of bone formation, as bone turnover and bone structure remain at the same level as in WT mice. (<b>c</b>) In F/F mice, IL-6-stimulated osteoblast proliferation via gp130-STAT1/3 is enhanced, leading to a greater number of osteoblasts. Deletion of the gp130-SHP2/Ras/MAPK <span class="highlight" style="background-color:">pathway</span>, indicated by the red X, releases inhibition, resulting in increased osteoclastogenesis. The increase in resorption appears to be greater than the increase in bone formation, leading overall to a reduction in trabecular bone density.</div></p></div><div class="aux"><div class="resc">Natalie A. Sims, et al. J Clin Invest. 2004 February 1;113(3):379-389.</div><p class="links"><a class="dblinks" href="/pubmed/14755335" ref="ordinalpos=249&amp;ncbi_uid=431989&amp;link_uid=14755335">Citation</a><a class="dblinks" href="/pmc/articles/PMC324544/" ref="ordinalpos=249&amp;ncbi_uid=431989&amp;link_uid=PMC324544">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>250.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4253425/figure/F6/" ref="ordinalpos=250&amp;ncbi_uid=6900318&amp;link_uid=PMC4253425" href="/pmc/articles/PMC4253425/"><img src="/pmc/articles/instance/4253425/bin/oncotarget-05-9150-g006.gif" src-large="/pmc/articles/instance/4253425/bin/oncotarget-05-9150-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4253425/" ref="ordinalpos=250&amp;ncbi_uid=6900318&amp;link_uid=PMC4253425" image-link="/pmc/articles/PMC4253425/figure/F6/" class="imagepopup">Figure 6. Inhibition of ALK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by Crizotinib in PANC-1 cells.  From: Crizotinib Exhibits Antitumor Activity by Targeting ALK <span class="highlight" style="background-color:">Signaling</span> not c-MET in Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div><b>(A)</b> The ability of Crizotinib to target ALK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was analyzed by Phospho-Kinase array after PANC-1 cells were treated with Crizotinib (5 μM) for 2 hr (*p &lt; 0.05 and ** p &lt;0.01, compared to control). <b>(B)</b> The expression of p-STAT3, p-AKT, and p-ERK were measured by western blotting after PANC-1 cells were treated with Crizotinib at the indicated doses (0-20 μM) for 2 hr.</div></p></div><div class="aux"><div class="resc">Hong Hua Yan, et al. Oncotarget. 2014 October;5(19):9150-9168.</div><p class="links"><a class="dblinks" href="/pubmed/25193856" ref="ordinalpos=250&amp;ncbi_uid=6900318&amp;link_uid=25193856">Citation</a><a class="dblinks" href="/pmc/articles/PMC4253425/" ref="ordinalpos=250&amp;ncbi_uid=6900318&amp;link_uid=PMC4253425">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>251.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3216328/figure/fig8/" ref="ordinalpos=251&amp;ncbi_uid=3408496&amp;link_uid=PMC3216328" href="/pmc/articles/PMC3216328/"><img src="/pmc/articles/instance/3216328/bin/JCB_201009126R_RGB_Fig8.gif" src-large="/pmc/articles/instance/3216328/bin/JCB_201009126R_RGB_Fig8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3216328/" ref="ordinalpos=251&amp;ncbi_uid=3408496&amp;link_uid=PMC3216328" image-link="/pmc/articles/PMC3216328/figure/fig8/" class="imagepopup">Figure 8. From: Phosphoinositide 3-kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated by p110? regulates invadopodia formation. </a></p><div class="supp"><p class="details"><div><b>A model of the function of PI3K <span class="highlight" style="background-color:">signaling</span> in invadopodia formation and cell invasion.</b> p110α that is activated downstream of growth factor receptors produces the <span class="highlight" style="background-color:">signaling</span> lipid PI(3,4,5)P<sub>3</sub> to regulate invadopodia formation and cancer cell invasion. PI(3,4)P<sub>2</sub> that is generated via dephosphorylation of PI(3,4,5)P<sub>3</sub> by synaptojanin-2 (SJ2) may regulate invadopodia formation through the Tks5/N-WASP axis in parallel with PI(3,4,5)P<sub>3</sub>. PDK1 and Akt are activated by both PI(3,4,5)P<sub>3</sub> and PI(3,4)P<sub>2</sub> and act as mediators of the PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for invadopodia formation.</div></p></div><div class="aux"><div class="resc">Hideki Yamaguchi, et al. J Cell Biol. 2011 June 27;193(7):1275-1288.</div><p class="links"><a class="dblinks" href="/pubmed/21708979" ref="ordinalpos=251&amp;ncbi_uid=3408496&amp;link_uid=21708979">Citation</a><a class="dblinks" href="/pmc/articles/PMC3216328/" ref="ordinalpos=251&amp;ncbi_uid=3408496&amp;link_uid=PMC3216328">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>252.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634258/figure/F1/" ref="ordinalpos=252&amp;ncbi_uid=1709793&amp;link_uid=PMC2634258" href="/pmc/articles/PMC2634258/"><img src="/pmc/articles/instance/2634258/bin/org0402_0123_fig001.gif" src-large="/pmc/articles/instance/2634258/bin/org0402_0123_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634258/" ref="ordinalpos=252&amp;ncbi_uid=1709793&amp;link_uid=PMC2634258" image-link="/pmc/articles/PMC2634258/figure/F1/" class="imagepopup">Figure 1.  From: Wnt <span class="highlight" style="background-color:">signaling</span> in skin organogenesis. </a></p><div class="supp"><p class="details"><div>Schematic diagrams depicting the Wnt <span class="highlight" style="background-color:">signaling</span> pathways. The canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left 2) and planar cell polarity <span class="highlight" style="background-color:">pathway</span> (right) are shown. Wnts, represented by yellow circles represent a large family of <span class="highlight" style="background-color:">signaling</span> molecules. In general, Wnts 1, 3a, 8 and 8b induce the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> while Wnts 5a and 11 favor the non-canonical <span class="highlight" style="background-color:">pathway</span>; however, significant cross talk between these pathways has been observed. There are a number of other Wnt <span class="highlight" style="background-color:">signaling</span> pathways that have not been described here as their role in skin appendages is yet to be established. More detailed views are presented elsewhere.87,88</div></p></div><div class="aux"><div class="resc">Randall B Widelitz. Organogenesis. 2008 Apr-Jun;4(2):123-133.</div><p class="links"><a class="dblinks" href="/pubmed/19279724" ref="ordinalpos=252&amp;ncbi_uid=1709793&amp;link_uid=19279724">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634258/" ref="ordinalpos=252&amp;ncbi_uid=1709793&amp;link_uid=PMC2634258">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>253.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3776465/figure/F2/" ref="ordinalpos=253&amp;ncbi_uid=5418637&amp;link_uid=PMC3776465" href="/pmc/articles/PMC3776465/"><img src="/pmc/articles/instance/3776465/bin/msy-0004-0257-g02.gif" src-large="/pmc/articles/instance/3776465/bin/msy-0004-0257-g02.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3776465/" ref="ordinalpos=253&amp;ncbi_uid=5418637&amp;link_uid=PMC3776465" image-link="/pmc/articles/PMC3776465/figure/F2/" class="imagepopup">Fig. 2.  From: Mutations in the VEGFR3 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Explain 36% of Familial Lymphedema. </a></p><div class="supp"><p class="details"><div>The functions of the proteins encoded by the genes mutated in primary lymphedema cluster around a central molecular <span class="highlight" style="background-color:">pathway</span>: the VEGFR3 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">, et al. Mol Syndromol. 2013 September;4(6):257-266.</div><p class="links"><a class="dblinks" href="/pubmed/24167460" ref="ordinalpos=253&amp;ncbi_uid=5418637&amp;link_uid=24167460">Citation</a><a class="dblinks" href="/pmc/articles/PMC3776465/" ref="ordinalpos=253&amp;ncbi_uid=5418637&amp;link_uid=PMC3776465">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>254.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2815562/figure/f9/" ref="ordinalpos=254&amp;ncbi_uid=2185417&amp;link_uid=PMC2815562" href="/pmc/articles/PMC2815562/"><img src="/pmc/articles/instance/2815562/bin/zmb9991084670009.gif" src-large="/pmc/articles/instance/2815562/bin/zmb9991084670009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2815562/" ref="ordinalpos=254&amp;ncbi_uid=2185417&amp;link_uid=PMC2815562" image-link="/pmc/articles/PMC2815562/figure/f9/" class="imagepopup">FIG. 9. From: The Caenorhabditis elegans Ste20-Related Kinase and Rac-Type Small GTPase Regulate the c-Jun N-Terminal Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Mediating the Stress Response  . </a></p><div class="supp"><p class="details"><div>Proposed model for KGB-1 JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The circled P stands for phosphate.</div></p></div><div class="aux"><div class="resc">Kota Fujiki, et al. Mol Cell Biol. 2010 February;30(4):995-1003.</div><p class="links"><a class="dblinks" href="/pubmed/20008556" ref="ordinalpos=254&amp;ncbi_uid=2185417&amp;link_uid=20008556">Citation</a><a class="dblinks" href="/pmc/articles/PMC2815562/" ref="ordinalpos=254&amp;ncbi_uid=2185417&amp;link_uid=PMC2815562">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>255.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4110525/figure/F3/" ref="ordinalpos=255&amp;ncbi_uid=6474944&amp;link_uid=PMC4110525" href="/pmc/articles/PMC4110525/"><img src="/pmc/articles/instance/4110525/bin/1476-4598-13-139-3.gif" src-large="/pmc/articles/instance/4110525/bin/1476-4598-13-139-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4110525/" ref="ordinalpos=255&amp;ncbi_uid=6474944&amp;link_uid=PMC4110525" image-link="/pmc/articles/PMC4110525/figure/F3/" class="imagepopup">Figure 3.  From: Targeting Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> positively affects the sensitivity of osteosarcoma to cisplatin by regulating the expression and/or activity of Caspase family. </a></p><div class="supp"><p class="details"><div><b>Activation of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> increased the sensitivity of MG63 and Saos-2 to cisplatin.</b> Real-time PCR indicated that HES1 mRNA in MG63 and Saos-2 had significantly more expression after NICD-1 plasmid transfection <b>(A and B)</b>. The sensitivity of MG63 and Saos-2 to cisplatin has significantly increased after the activation of Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> <b>(C and D)</b>. (*P &lt; 0.05; **P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Lei Wang, et al. Mol Cancer. 2014;13:139-139.</div><p class="links"><a class="dblinks" href="/pubmed/24894297" ref="ordinalpos=255&amp;ncbi_uid=6474944&amp;link_uid=24894297">Citation</a><a class="dblinks" href="/pmc/articles/PMC4110525/" ref="ordinalpos=255&amp;ncbi_uid=6474944&amp;link_uid=PMC4110525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>256.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1601946/figure/F1/" ref="ordinalpos=256&amp;ncbi_uid=867277&amp;link_uid=PMC1601946" href="/pmc/articles/PMC1601946/"><img src="/pmc/articles/instance/1601946/bin/neo0808_0677_fig001.gif" src-large="/pmc/articles/instance/1601946/bin/neo0808_0677_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1601946/" ref="ordinalpos=256&amp;ncbi_uid=867277&amp;link_uid=PMC1601946" image-link="/pmc/articles/PMC1601946/figure/F1/" class="imagepopup">Figure 1.  From: Ubiquitin-Dependent Regulation of TGF? <span class="highlight" style="background-color:">Signaling</span> in Cancer. </a></p><div class="supp"><p class="details"><div>E3 ubiquitin ligases regulating the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. HECT domain, RING type, and U-box E3 ubiquitin ligases regulate both basal levels and activated components of the TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Black arrows illustrate TGFβ signal transduction. Positive (green arrows) and negative (red arrows) regulation of TGFβ <span class="highlight" style="background-color:">signaling</span> components by the ubiquitin/proteosome <span class="highlight" style="background-color:">pathway</span> are also indicated.</div></p></div><div class="aux"><div class="resc">Luisa Izzi, et al. Neoplasia. 2006 August;8(8):677-688.</div><p class="links"><a class="dblinks" href="/pubmed/16925950" ref="ordinalpos=256&amp;ncbi_uid=867277&amp;link_uid=16925950">Citation</a><a class="dblinks" href="/pmc/articles/PMC1601946/" ref="ordinalpos=256&amp;ncbi_uid=867277&amp;link_uid=PMC1601946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>257.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3404876/figure/F2/" ref="ordinalpos=257&amp;ncbi_uid=5035666&amp;link_uid=PMC3404876" href="/pmc/articles/PMC3404876/"><img src="/pmc/articles/instance/3404876/bin/cc-11-2452-g2.gif" src-large="/pmc/articles/instance/3404876/bin/cc-11-2452-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3404876/" ref="ordinalpos=257&amp;ncbi_uid=5035666&amp;link_uid=PMC3404876" image-link="/pmc/articles/PMC3404876/figure/F2/" class="imagepopup">Figure. From: Identification of novel metastasis suppressor <span class="highlight" style="background-color:">signaling</span> pathways for breast cancer. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> RKIP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates breast cancer metastasis to the bone. Highlighted with light blue rectangles are the components of the <span class="highlight" style="background-color:">pathway</span> that are also part of the RPMS. Highlighted with light blue ovals are the meta-genes that were added to the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to make the RPMS.</div></p></div><div class="aux"><div class="resc">Andy J. Minn, et al. Cell Cycle. 2012 July 1;11(13):2452-2457.</div><p class="links"><a class="dblinks" href="/pubmed/22659842" ref="ordinalpos=257&amp;ncbi_uid=5035666&amp;link_uid=22659842">Citation</a><a class="dblinks" href="/pmc/articles/PMC3404876/" ref="ordinalpos=257&amp;ncbi_uid=5035666&amp;link_uid=PMC3404876">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>258.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193469/figure/fig5/" ref="ordinalpos=258&amp;ncbi_uid=4203921&amp;link_uid=PMC3193469" href="/pmc/articles/PMC3193469/"><img src="/pmc/articles/instance/3193469/bin/cr2011118f5.gif" src-large="/pmc/articles/instance/3193469/bin/cr2011118f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193469/" ref="ordinalpos=258&amp;ncbi_uid=4203921&amp;link_uid=PMC3193469" image-link="/pmc/articles/PMC3193469/figure/fig5/" class="imagepopup">Figure 5.  From: Plant ubiquitin-proteasome <span class="highlight" style="background-color:">pathway</span> and its role in gibberellin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>GA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rice and <i>Arabidopsis</i>. There are three GA receptors and five DELLA homologues in <i>Arabidopsis</i>, in contrast with the single-gene encoded GID1 and SLR1 in rice.</div></p></div><div class="aux"><div class="resc">Feng Wang, et al. Cell Res. 2011 September;21(9):1286-1294.</div><p class="links"><a class="dblinks" href="/pubmed/21788985" ref="ordinalpos=258&amp;ncbi_uid=4203921&amp;link_uid=21788985">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193469/" ref="ordinalpos=258&amp;ncbi_uid=4203921&amp;link_uid=PMC3193469">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>259.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2963373/figure/F6/" ref="ordinalpos=259&amp;ncbi_uid=3292471&amp;link_uid=PMC2963373" href="/pmc/articles/PMC2963373/"><img src="/pmc/articles/instance/2963373/bin/zbc0451035750006.gif" src-large="/pmc/articles/instance/2963373/bin/zbc0451035750006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2963373/" ref="ordinalpos=259&amp;ncbi_uid=3292471&amp;link_uid=PMC2963373" image-link="/pmc/articles/PMC2963373/figure/F6/" class="imagepopup">FIGURE 6. From: ?-Tocotrienol but Not ?-Tocopherol Blocks STAT3 Cell <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> through Induction of Protein-tyrosine Phosphatase SHP-1 and Sensitizes Tumor Cells to Chemotherapeutic Agents. </a></p><div class="supp"><p class="details"><div><b>Schematic diagram showing the effect of γ-T3 on STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and apoptosis.</b></div></p></div><div class="aux"><div class="resc">Ramaswamy Kannappan, et al. J Biol Chem. 2010 October 22;285(43):33520-33528.</div><p class="links"><a class="dblinks" href="/pubmed/20720018" ref="ordinalpos=259&amp;ncbi_uid=3292471&amp;link_uid=20720018">Citation</a><a class="dblinks" href="/pmc/articles/PMC2963373/" ref="ordinalpos=259&amp;ncbi_uid=3292471&amp;link_uid=PMC2963373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>260.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163574/figure/pone-0023523-g002/" ref="ordinalpos=260&amp;ncbi_uid=3208467&amp;link_uid=PMC3163574" href="/pmc/articles/PMC3163574/"><img src="/pmc/articles/instance/3163574/bin/pone.0023523.g002.gif" src-large="/pmc/articles/instance/3163574/bin/pone.0023523.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163574/" ref="ordinalpos=260&amp;ncbi_uid=3208467&amp;link_uid=PMC3163574" image-link="/pmc/articles/PMC3163574/figure/pone-0023523-g002/" class="imagepopup">Figure 2. Schematic of the IFNγ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and the use of GAS-Cre as a reporter of IFNγ <span class="highlight" style="background-color:">signaling</span>..  From: Identification and Visualization of CD8+ T Cell Mediated IFN-? <span class="highlight" style="background-color:">Signaling</span> in Target Cells during an Antiviral Immune Response in the Brain. </a></p><div class="supp"><p class="details"><div>Schematic of IFNγ <span class="highlight" style="background-color:">signaling</span> through the JAK/STAT1 <span class="highlight" style="background-color:">pathway</span>. Binding of IFNγ to its receptor results in activation of JAKs, which phosphorylate cytoplasmic monomers of STAT1. Phosphorylated STAT1 dimerizes, translocates to the nucleus, and activates transcription from gene promoters containing Gamma Activated Sites (GAS). <i>Cre</i> recombinase translocates to the nucleus due to the presence of a nuclear localization signal.</div></p></div><div class="aux"><div class="resc">Mariana Puntel, et al. PLoS One. 2011;6(8):e23523.</div><p class="links"><a class="dblinks" href="/pubmed/21897844" ref="ordinalpos=260&amp;ncbi_uid=3208467&amp;link_uid=21897844">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163574/" ref="ordinalpos=260&amp;ncbi_uid=3208467&amp;link_uid=PMC3163574">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>261.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4306126/figure/f5/" ref="ordinalpos=261&amp;ncbi_uid=7129169&amp;link_uid=PMC4306126" href="/pmc/articles/PMC4306126/"><img src="/pmc/articles/instance/4306126/bin/srep08030-f5.gif" src-large="/pmc/articles/instance/4306126/bin/srep08030-f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4306126/" ref="ordinalpos=261&amp;ncbi_uid=7129169&amp;link_uid=PMC4306126" image-link="/pmc/articles/PMC4306126/figure/f5/" class="imagepopup">Figure 5. Hypothesized RSLVs-related <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in abiotic stress accompanied by oxidative stress..  From: Reactive short-chain leaf volatiles act as powerful inducers of abiotic stress-related gene expression. </a></p><div class="supp"><p class="details"><div>RSLV stimulates both HSFA1-dependent and –independent pathways (white arrows). RSLVs are possibly involved in the upstream of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by abiotic stresses accompanied by oxidative stress. Proteotoxic stress such as heat and ROS are accepted by HSFA1s to drive HSFA1-denpendent <span class="highlight" style="background-color:">pathway</span> (black arrows).</div></p></div><div class="aux"><div class="resc">Yasuo Yamauchi, et al. Sci Rep. 2015;5:8030.</div><p class="links"><a class="dblinks" href="/pubmed/25619826" ref="ordinalpos=261&amp;ncbi_uid=7129169&amp;link_uid=25619826">Citation</a><a class="dblinks" href="/pmc/articles/PMC4306126/" ref="ordinalpos=261&amp;ncbi_uid=7129169&amp;link_uid=PMC4306126">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>262.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711381/figure/F1/" ref="ordinalpos=262&amp;ncbi_uid=5099967&amp;link_uid=PMC3711381" href="/pmc/articles/PMC3711381/"><img src="/pmc/articles/instance/3711381/bin/nihms-442491-f0001.gif" src-large="/pmc/articles/instance/3711381/bin/nihms-442491-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711381/" ref="ordinalpos=262&amp;ncbi_uid=5099967&amp;link_uid=PMC3711381" image-link="/pmc/articles/PMC3711381/figure/F1/" class="imagepopup">Figure 1.  From: A novel target for Huntington's disease: ERK at the crossroads of <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The ERK <span class="highlight" style="background-color:">pathway</span>, its upstream activators and downstream targets. The <span class="highlight" style="background-color:">signaling</span> branches affected in Huntington’s disease are shown; not all components are indicated.</div></p></div><div class="aux"><div class="resc">László Bodai, et al. Bioessays. ;34(2):142-148.</div><p class="links"><a class="dblinks" href="/pubmed/22334892" ref="ordinalpos=262&amp;ncbi_uid=5099967&amp;link_uid=22334892">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711381/" ref="ordinalpos=262&amp;ncbi_uid=5099967&amp;link_uid=PMC3711381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>263.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4159867/figure/ijms-15-14546-f002/" ref="ordinalpos=263&amp;ncbi_uid=6619453&amp;link_uid=PMC4159867" href="/pmc/articles/PMC4159867/"><img src="/pmc/articles/instance/4159867/bin/ijms-15-14546-g002.gif" src-large="/pmc/articles/instance/4159867/bin/ijms-15-14546-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4159867/" ref="ordinalpos=263&amp;ncbi_uid=6619453&amp;link_uid=PMC4159867" image-link="/pmc/articles/PMC4159867/figure/ijms-15-14546-f002/" class="imagepopup">Figure 2.  From: Salmonella as an Innovative Therapeutic Antitumor Agent. </a></p><div class="supp"><p class="details"><div><i>Salmonella</i>-mediated cell death <span class="highlight" style="background-color:">pathway</span> in tumor cells. The arrow indicates that <i>Salmonella</i> induces <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The flat indicates that <i>Salmonella</i> inhibits <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Salmonella</i> induces tumor cell death by activating caspase or down-regulating the AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Wen-Wei Chang, et al. Int J Mol Sci. 2014 August;15(8):14546-14554.</div><p class="links"><a class="dblinks" href="/pubmed/25196596" ref="ordinalpos=263&amp;ncbi_uid=6619453&amp;link_uid=25196596">Citation</a><a class="dblinks" href="/pmc/articles/PMC4159867/" ref="ordinalpos=263&amp;ncbi_uid=6619453&amp;link_uid=PMC4159867">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>264.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2812679/figure/F8/" ref="ordinalpos=264&amp;ncbi_uid=2731986&amp;link_uid=PMC2812679" href="/pmc/articles/PMC2812679/"><img src="/pmc/articles/instance/2812679/bin/nihms-160238-f0011.gif" src-large="/pmc/articles/instance/2812679/bin/nihms-160238-f0011.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2812679/" ref="ordinalpos=264&amp;ncbi_uid=2731986&amp;link_uid=PMC2812679" image-link="/pmc/articles/PMC2812679/figure/F8/" class="imagepopup">Fig. 8.  From: A Novel <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> of Tissue Kallikrein in Promoting Keratinocyte Migration: Activation of Proteinase-Activated Receptor 1 and Epidermal Growth Factor Receptor. </a></p><div class="supp"><p class="details"><div>Schematic diagram of TK-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> promoting HaCaT keratinocyte migration.</div></p></div><div class="aux"><div class="resc">Lin Gao, et al. Exp Cell Res. ;316(3):376-389.</div><p class="links"><a class="dblinks" href="/pubmed/19879874" ref="ordinalpos=264&amp;ncbi_uid=2731986&amp;link_uid=19879874">Citation</a><a class="dblinks" href="/pmc/articles/PMC2812679/" ref="ordinalpos=264&amp;ncbi_uid=2731986&amp;link_uid=PMC2812679">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>265.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3890531/figure/F3/" ref="ordinalpos=265&amp;ncbi_uid=5704231&amp;link_uid=PMC3890531" href="/pmc/articles/PMC3890531/"><img src="/pmc/articles/instance/3890531/bin/1471-2164-14-904-3.gif" src-large="/pmc/articles/instance/3890531/bin/1471-2164-14-904-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3890531/" ref="ordinalpos=265&amp;ncbi_uid=5704231&amp;link_uid=PMC3890531" image-link="/pmc/articles/PMC3890531/figure/F3/" class="imagepopup">Figure 3.  From: An overview of gene expression dynamics during early ovarian folliculogenesis: specificity of follicular compartments and bi-directional dialog. </a></p><div class="supp"><p class="details"><div><b>Functional enrichment and canonical pathways within oocyte and GCs. A</b>: Functional enrichment. Genes differentially expressed between the 2 compartments were evaluated <i>in silico</i> using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA). The figure shows the most significant functional groups (p &lt; 0.05, FDR &lt; 5%). For each compartment, the gene list corresponds to up regulated genes compared to the other compartments. The bars represent the p-value at <i>logarithmic</i> scale. GC data are in red and oocyte data are in blue. Red and blue numbers correspond to the number of focus genes that contributed to the function concerned. <b>B</b>: Canonical pathways in oocytes and GCs. Significantly enriched canonical <span class="highlight" style="background-color:">pathway</span> categories (p-value &lt; 0.05) identified in the 2 compartments. The Y axis corresponds to percentage gene enrichment in the <span class="highlight" style="background-color:">pathway</span>. Red and blue numbers correspond to the number of focus genes that contributed to the <span class="highlight" style="background-color:">pathway</span> concerned. *: FDR &lt; 0.25 (likely to be valid 3 out of 4 times); **: FDR &lt;0.10; ***: FDR &lt; 0.05. <b>C</b>: Growth factor <span class="highlight" style="background-color:">signaling</span> pathways in oocytes and GCs. Significantly enriched growth factor <span class="highlight" style="background-color:">signaling</span> pathways (p-value &lt; 0.05) identified in the 2 compartments. The Y axis corresponds to percentage gene enrichment in the <span class="highlight" style="background-color:">pathway</span>. Red and blue numbers correspond to the number of focus genes that contributed to the <span class="highlight" style="background-color:">pathway</span> concerned. *: FDR &lt; 0.25; **: FDR &lt;0.10; ***: FDR &lt; 0.05. <b>D</b>: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> in oocytes and GCs. Significantly enriched <span class="highlight" style="background-color:">signaling</span> pathways (p-value &lt; 0.05) identified in the 2 compartments. The Y axis corresponds to percentage gene enrichment in the <span class="highlight" style="background-color:">pathway</span>. Red and blue numbers correspond to the number of focus genes that contributed to the <span class="highlight" style="background-color:">pathway</span> concerned. *: FDR &lt; 0.25; **: FDR &lt;0.10; ***: FDR &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Agnes Bonnet, et al. BMC Genomics. 2013;14:904-904.</div><p class="links"><a class="dblinks" href="/pubmed/24350644" ref="ordinalpos=265&amp;ncbi_uid=5704231&amp;link_uid=24350644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3890531/" ref="ordinalpos=265&amp;ncbi_uid=5704231&amp;link_uid=PMC3890531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>266.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3885983/figure/F8/" ref="ordinalpos=266&amp;ncbi_uid=5692459&amp;link_uid=PMC3885983" href="/pmc/articles/PMC3885983/"><img src="/pmc/articles/instance/3885983/bin/nihms542217f8.gif" src-large="/pmc/articles/instance/3885983/bin/nihms542217f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3885983/" ref="ordinalpos=266&amp;ncbi_uid=5692459&amp;link_uid=PMC3885983" image-link="/pmc/articles/PMC3885983/figure/F8/" class="imagepopup">Figure 8. Interactions between EGF-Ras-ERK and other <span class="highlight" style="background-color:">signaling</span> pathways.  From: Canonical RTK-Ras-ERK <span class="highlight" style="background-color:">signaling</span> and related alternative pathways. </a></p><div class="supp"><p class="details"><div>A) EGF-Ras-ERK and Wnt <span class="highlight" style="background-color:">signaling</span> cooperate to promote several cell fates. B) EGF-Ras-ERK and Notch <span class="highlight" style="background-color:">signaling</span> function antagonistically to promote distinct cell fates. C) The alternative EGF-Ras-Ral <span class="highlight" style="background-color:">pathway</span> and Notch cooperate to promote the secondary vulval fate. See text for details and references.</div></p></div><div class="aux"><div class="resc">Meera V. Sundaram. WormBook. :10.1895/wormbook.1.80.2.</div><p class="links"><a class="dblinks" href="/pubmed/23908058" ref="ordinalpos=266&amp;ncbi_uid=5692459&amp;link_uid=23908058">Citation</a><a class="dblinks" href="/pmc/articles/PMC3885983/" ref="ordinalpos=266&amp;ncbi_uid=5692459&amp;link_uid=PMC3885983">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>267.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3334516/figure/fig4/" ref="ordinalpos=267&amp;ncbi_uid=4851774&amp;link_uid=PMC3334516" href="/pmc/articles/PMC3334516/"><img src="/pmc/articles/instance/3334516/bin/carcinbgs114f04_4c.gif" src-large="/pmc/articles/instance/3334516/bin/carcinbgs114f04_4c.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3334516/" ref="ordinalpos=267&amp;ncbi_uid=4851774&amp;link_uid=PMC3334516" image-link="/pmc/articles/PMC3334516/figure/fig4/" class="imagepopup">Fig. 4. From: MiRNA-335 suppresses neuroblastoma cell invasiveness by direct targeting of multiple genes from the non-canonical TGF-? signalling <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic illustration of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members and potential <span class="highlight" style="background-color:">pathway</span> member LRG1. TGF-β <span class="highlight" style="background-color:">pathway</span> comprises the canonical and non-canonical <span class="highlight" style="background-color:">signaling</span> pathways. miR-335 regulates cell migration and invasion through targeting of the three branches of the non-canonical TGF-β <span class="highlight" style="background-color:">pathway</span>. In this study, miR-335 has been demonstrated to directly target <i>ROCK1</i>, <i>MAPK1</i> and putative TGF-β member <i>LRG1</i>. miR-335 inhibition of <i>ROCK1</i> prevents phosphorylation of MLC thereby reducing actomyosin assembly and contraction and ultimately the motile potential of the cells. Similarly, inhibition of <i>MAPK1</i> <span class="highlight" style="background-color:">signaling</span> also prevents phosphorylation of MLC.</div></p></div><div class="aux"><div class="resc">Jennifer Lynch, et al. Carcinogenesis. 2012 May;33(5):976-985.</div><p class="links"><a class="dblinks" href="/pubmed/22382496" ref="ordinalpos=267&amp;ncbi_uid=4851774&amp;link_uid=22382496">Citation</a><a class="dblinks" href="/pmc/articles/PMC3334516/" ref="ordinalpos=267&amp;ncbi_uid=4851774&amp;link_uid=PMC3334516">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>268.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3419804/figure/F1/" ref="ordinalpos=268&amp;ncbi_uid=5143477&amp;link_uid=PMC3419804" href="/pmc/articles/PMC3419804/"><img src="/pmc/articles/instance/3419804/bin/nihms372177f1.gif" src-large="/pmc/articles/instance/3419804/bin/nihms372177f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3419804/" ref="ordinalpos=268&amp;ncbi_uid=5143477&amp;link_uid=PMC3419804" image-link="/pmc/articles/PMC3419804/figure/F1/" class="imagepopup">Figure 1. Autophagy and <span class="highlight" style="background-color:">signaling</span> pathways.  From: Autophagy, Myocardial Protection and the Metabolic Syndrome. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> activates the mTOR <span class="highlight" style="background-color:">pathway</span> via Akt and thus suppresses autophagy. The mTOR inhibitor rapamycin and its derivatives enhance autophagy. The AMPK <span class="highlight" style="background-color:">pathway</span> modulates the mTOR <span class="highlight" style="background-color:">pathway</span> via multiple mechanisms including interaction with Akt, TSC1, mTOR and Ulk1. Biguanides such as metformin and phenformin, and thiazolidinediones (TZDs), rosiglitazone and pioglitazone, enhance AMPK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Zoltan Giricz, et al. J Cardiovasc Pharmacol. ;60(2):125-132.</div><p class="links"><a class="dblinks" href="/pubmed/22472909" ref="ordinalpos=268&amp;ncbi_uid=5143477&amp;link_uid=22472909">Citation</a><a class="dblinks" href="/pmc/articles/PMC3419804/" ref="ordinalpos=268&amp;ncbi_uid=5143477&amp;link_uid=PMC3419804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>269.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2700035/figure/F6/" ref="ordinalpos=269&amp;ncbi_uid=1872972&amp;link_uid=PMC2700035" href="/pmc/articles/PMC2700035/"><img src="/pmc/articles/instance/2700035/bin/nihms112886f6.gif" src-large="/pmc/articles/instance/2700035/bin/nihms112886f6.jpg" alt="FIG. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2700035/" ref="ordinalpos=269&amp;ncbi_uid=1872972&amp;link_uid=PMC2700035" image-link="/pmc/articles/PMC2700035/figure/F6/" class="imagepopup">FIG. 6.  From: Joint loading-driven bone formation and <span class="highlight" style="background-color:">signaling</span> pathways predicted from genome-wide expression profiles. </a></p><div class="supp"><p class="details"><div>Load-driven <span class="highlight" style="background-color:">signaling</span> pathways predicted with ankle loading at 1 h after the last loading. In the network diagrams the mRNA levels of the genes in the red boxes were altered by ankle loading. (A) PI3K <span class="highlight" style="background-color:">pathway</span>. (B) ECM-receptor interaction. (C) TGFβ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (D) Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ping Zhang, et al. Bone. ;44(5):989-998.</div><p class="links"><a class="dblinks" href="/pubmed/19442616" ref="ordinalpos=269&amp;ncbi_uid=1872972&amp;link_uid=19442616">Citation</a><a class="dblinks" href="/pmc/articles/PMC2700035/" ref="ordinalpos=269&amp;ncbi_uid=1872972&amp;link_uid=PMC2700035">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>270.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2519775/figure/f1/" ref="ordinalpos=270&amp;ncbi_uid=1525228&amp;link_uid=PMC2519775" href="/pmc/articles/PMC2519775/"><img src="/pmc/articles/instance/2519775/bin/zek0080831680001.gif" src-large="/pmc/articles/instance/2519775/bin/zek0080831680001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2519775/" ref="ordinalpos=270&amp;ncbi_uid=1525228&amp;link_uid=PMC2519775" image-link="/pmc/articles/PMC2519775/figure/f1/" class="imagepopup">FIG. 1. From: Characterization of KlGRR1 and SMS1 Genes, Two New Elements of the Glucose <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> of Kluyveromyces lactis . </a></p><div class="supp"><p class="details"><div>Comparison of glucose <span class="highlight" style="background-color:">signaling</span> pathways in <i>S. cerevisiae</i> and <i>K. lactis</i>. A simplified model of the Snf3/Rgt2/Rag4-dependent <span class="highlight" style="background-color:">pathway</span> in the two yeasts is presented. Phosphorylation events on Std1/Mth1, Rgt1, and Sck1 are indicated (P). The Gpr1/Gpa2-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, known to participate in ScRgt1 regulation (phosphorylation) in <i>S. cerevisiae</i>, is not represented.</div></p></div><div class="aux"><div class="resc">Martina Hnatova, et al. Eukaryot Cell. 2008 August;7(8):1299-1308.</div><p class="links"><a class="dblinks" href="/pubmed/18552281" ref="ordinalpos=270&amp;ncbi_uid=1525228&amp;link_uid=18552281">Citation</a><a class="dblinks" href="/pmc/articles/PMC2519775/" ref="ordinalpos=270&amp;ncbi_uid=1525228&amp;link_uid=PMC2519775">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>271.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2803041/figure/F1/" ref="ordinalpos=271&amp;ncbi_uid=2171350&amp;link_uid=PMC2803041" href="/pmc/articles/PMC2803041/"><img src="/pmc/articles/instance/2803041/bin/nihms-155225-f0001.gif" src-large="/pmc/articles/instance/2803041/bin/nihms-155225-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2803041/" ref="ordinalpos=271&amp;ncbi_uid=2171350&amp;link_uid=PMC2803041" image-link="/pmc/articles/PMC2803041/figure/F1/" class="imagepopup">Figure 1. Reconstitution of ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for stress responsive gene expression in Arabidopsis protoplasts.  From: In vitro Reconstitution of an ABA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>a</b>, SnRK2-mediated phosphorylation of ABF2 is sufficient for ABA-responsive gene expression. <b>b</b>, Reconstitution of ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by co-expression of PYR1, ABI1, SnRK2.6 and ABF2. <b>c</b>, econstitution of ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with different members of the PYR/PYL family. <b>d</b>, Reconstitution using different combinations of the core components. Protoplasts (2 × 10<sup>4</sup>) from the <i>snrk2.2/3/6</i> triple mutant were used except in (d), where protoplasts from the Col-0 wild-type plants were used. The <i>RD29B::LUC</i> and <i>ZmUBQ</i>::<i>GUS</i> were used as the ABA responsive reporter and internal control, respectively. After transfection, protoplasts were incubated for 5 h under light and in the presence of 0 (open bars) or 5 μM (solid bars) ABA. Error bars, mean ± s.e.m. (n=3).</div></p></div><div class="aux"><div class="resc">Hiroaki Fujii, et al. Nature. ;462(7273):660-664.</div><p class="links"><a class="dblinks" href="/pubmed/19924127" ref="ordinalpos=271&amp;ncbi_uid=2171350&amp;link_uid=19924127">Citation</a><a class="dblinks" href="/pmc/articles/PMC2803041/" ref="ordinalpos=271&amp;ncbi_uid=2171350&amp;link_uid=PMC2803041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>272.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2743320/figure/F7/" ref="ordinalpos=272&amp;ncbi_uid=2010881&amp;link_uid=PMC2743320" href="/pmc/articles/PMC2743320/"><img src="/pmc/articles/instance/2743320/bin/nihms135426f7.gif" src-large="/pmc/articles/instance/2743320/bin/nihms135426f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2743320/" ref="ordinalpos=272&amp;ncbi_uid=2010881&amp;link_uid=PMC2743320" image-link="/pmc/articles/PMC2743320/figure/F7/" class="imagepopup">Figure 7. Summary diagram for the proposed PACAP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: PACAP regulates immediate catecholamine release from adrenal chromaffin cells in an activity dependent manner through a protein kinase C-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Key components of the proposed PACAP-mediated <span class="highlight" style="background-color:">signaling</span> path as described in the text are shown.</div></p></div><div class="aux"><div class="resc">Barbara A. Kuri, et al. J Neurochem. ;110(4):1214-1225.</div><p class="links"><a class="dblinks" href="/pubmed/19508428" ref="ordinalpos=272&amp;ncbi_uid=2010881&amp;link_uid=19508428">Citation</a><a class="dblinks" href="/pmc/articles/PMC2743320/" ref="ordinalpos=272&amp;ncbi_uid=2010881&amp;link_uid=PMC2743320">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>273.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634987/figure/F1/" ref="ordinalpos=273&amp;ncbi_uid=1718384&amp;link_uid=PMC2634987" href="/pmc/articles/PMC2634987/"><img src="/pmc/articles/instance/2634987/bin/cam0202_0071_fig001.gif" src-large="/pmc/articles/instance/2634987/bin/cam0202_0071_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634987/" ref="ordinalpos=273&amp;ncbi_uid=1718384&amp;link_uid=PMC2634987" image-link="/pmc/articles/PMC2634987/figure/F1/" class="imagepopup">Figure 1.  From: The EGFR-GEP100-Arf6 <span class="highlight" style="background-color:">pathway</span> in breast cancer. </a></p><div class="supp"><p class="details"><div>The GEP100-Arf6 <span class="highlight" style="background-color:">pathway</span> to invasion. The Arf6 <span class="highlight" style="background-color:">pathway</span>, which involves its activation by GEP100 and employs the AMAP1-mediated signalsome for its downstream <span class="highlight" style="background-color:">signaling</span> constitutes a robust and central <span class="highlight" style="background-color:">pathway</span> for the induction of the invasion and metastasis of some breast cancer cells. GEP100 associates with tyrosine-phosphorylated EGFR for its activation. Therefore, activation of EGFR is required for the activation of this <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> also appears to perturb the formation and maintenance of E-cadherin-based cell-cell adhesion.</div></p></div><div class="aux"><div class="resc">Hisataka Sabe, et al. Cell Adh Migr. 2008 Apr-Jun;2(2):71-73.</div><p class="links"><a class="dblinks" href="/pubmed/19262097" ref="ordinalpos=273&amp;ncbi_uid=1718384&amp;link_uid=19262097">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634987/" ref="ordinalpos=273&amp;ncbi_uid=1718384&amp;link_uid=PMC2634987">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>274.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117959/figure/F1/" ref="ordinalpos=274&amp;ncbi_uid=3369810&amp;link_uid=PMC3117959" href="/pmc/articles/PMC3117959/"><img src="/pmc/articles/instance/3117959/bin/nihms292988f1.gif" src-large="/pmc/articles/instance/3117959/bin/nihms292988f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117959/" ref="ordinalpos=274&amp;ncbi_uid=3369810&amp;link_uid=PMC3117959" image-link="/pmc/articles/PMC3117959/figure/F1/" class="imagepopup">Figure 1.  From: Distinct effects of loss of classical estrogen receptor <span class="highlight" style="background-color:">signaling</span> versus complete deletion of estrogen receptor alpha on bone. </a></p><div class="supp"><p class="details"><div>ER <span class="highlight" style="background-color:">signaling</span> pathways. The classical <span class="highlight" style="background-color:">pathway</span> of ERα action (left) requires direct binding to EREs found in the control regions of estradiol (E<sub>2</sub>)-regulated genes. The non-classical <span class="highlight" style="background-color:">pathway</span> can involve either nuclear (middle) or membrane (right) <span class="highlight" style="background-color:">signaling</span>. In non-classical nuclear <span class="highlight" style="background-color:">signaling</span>, the ER interacts with other transcription factors, such as the jun/fos complex which binds to AP-1 sites, whereas in the membrane <span class="highlight" style="background-color:">pathway</span>, ER <span class="highlight" style="background-color:">signaling</span> is via alterations in kinase activity.</div></p></div><div class="aux"><div class="resc">Farhan A. Syed, et al. Bone. ;49(2):208-216.</div><p class="links"><a class="dblinks" href="/pubmed/21458604" ref="ordinalpos=274&amp;ncbi_uid=3369810&amp;link_uid=21458604">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117959/" ref="ordinalpos=274&amp;ncbi_uid=3369810&amp;link_uid=PMC3117959">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>275.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4026441/figure/Fig1/" ref="ordinalpos=275&amp;ncbi_uid=6162486&amp;link_uid=PMC4026441" href="/pmc/articles/PMC4026441/"><img src="/pmc/articles/instance/4026441/bin/12263_2014_399_Fig1_HTML.gif" src-large="/pmc/articles/instance/4026441/bin/12263_2014_399_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4026441/" ref="ordinalpos=275&amp;ncbi_uid=6162486&amp;link_uid=PMC4026441" image-link="/pmc/articles/PMC4026441/figure/Fig1/" class="imagepopup">Fig. 1.  From: High frequency of rare variants with a moderate-to-high predicted biological effect in protocadherin genes of extremely obese. </a></p><div class="supp"><p class="details"><div>Functional clustering analysis of genes with selected rare variants. <b>a</b> Pie chart of the top-ranked pathways with numbers referring to the names of the pathways as indicated in Table 1. <b>b</b> Pie chart of <span class="highlight" style="background-color:">pathway</span> 1 in <b>a</b>, the <i>Wnt <span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span>. <b>c</b> Pie chart of <span class="highlight" style="background-color:">pathway</span> 3 in <b>a</b>, the <i>integrin</i> <i><span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span>. <b>d</b> Pie chart of <span class="highlight" style="background-color:">pathway</span> 6 in <b>a</b>, the <i>nicotinic acetylcholine receptor <span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Edwin C. M. Mariman, et al. Genes Nutr. 2014 May;9(3):399.</div><p class="links"><a class="dblinks" href="/pubmed/24682882" ref="ordinalpos=275&amp;ncbi_uid=6162486&amp;link_uid=24682882">Citation</a><a class="dblinks" href="/pmc/articles/PMC4026441/" ref="ordinalpos=275&amp;ncbi_uid=6162486&amp;link_uid=PMC4026441">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>276.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4139835/figure/ijms-15-12149-f007/" ref="ordinalpos=276&amp;ncbi_uid=6520873&amp;link_uid=PMC4139835" href="/pmc/articles/PMC4139835/"><img src="/pmc/articles/instance/4139835/bin/ijms-15-12149-g007.gif" src-large="/pmc/articles/instance/4139835/bin/ijms-15-12149-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4139835/" ref="ordinalpos=276&amp;ncbi_uid=6520873&amp;link_uid=PMC4139835" image-link="/pmc/articles/PMC4139835/figure/ijms-15-12149-f007/" class="imagepopup">Figure 7.  From: Nrf2-Mediated HO-1 Induction Coupled with the ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Contributes to Indirect Antioxidant Capacity of Caffeic Acid Phenethyl Ester in HepG2 Cells. </a></p><div class="supp"><p class="details"><div>The ERK-Nrf2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of CAPE contributing to indirect cellular antioxidant capacity.</div></p></div><div class="aux"><div class="resc">Jin-Kyoung Kim, et al. Int J Mol Sci. 2014 July;15(7):12149-12165.</div><p class="links"><a class="dblinks" href="/pubmed/25007817" ref="ordinalpos=276&amp;ncbi_uid=6520873&amp;link_uid=25007817">Citation</a><a class="dblinks" href="/pmc/articles/PMC4139835/" ref="ordinalpos=276&amp;ncbi_uid=6520873&amp;link_uid=PMC4139835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>277.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3636549/figure/F2/" ref="ordinalpos=277&amp;ncbi_uid=4833021&amp;link_uid=PMC3636549" href="/pmc/articles/PMC3636549/"><img src="/pmc/articles/instance/3636549/bin/nihms460462f2.gif" src-large="/pmc/articles/instance/3636549/bin/nihms460462f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3636549/" ref="ordinalpos=277&amp;ncbi_uid=4833021&amp;link_uid=PMC3636549" image-link="/pmc/articles/PMC3636549/figure/F2/" class="imagepopup">Figure 2. Hedgehog <span class="highlight" style="background-color:">Pathway</span> intracellular <span class="highlight" style="background-color:">signaling</span>.  From: HEDGEHOG <span class="highlight" style="background-color:">SIGNALING</span> IN CHOLANGIOCYTES. </a></p><div class="supp"><p class="details"><div>(A) <b><span class="highlight" style="background-color:">PATHWAY</span> OFF.</b> In absence of Hedgehog ligands (Hh), Patched (Ptc), a membrane-spanning receptor on the surface of Hh-responsive cells, keeps the co-receptor Smoothened (Smo) in its inactive form. “Free”-Ptc silences the Smo-dependent down-stream intracellular <span class="highlight" style="background-color:">signaling</span>, and Hh-regulated transcription factors undergo multiple phosphorylations, leading to their inactivation and proteasome degradation. Thus, nuclear translocation of Hh-regulated transcription factors is prevented, and the <span class="highlight" style="background-color:">pathway</span> is OFF. (B) <b><span class="highlight" style="background-color:">PATHWAY</span> ON.</b> When the extracellular microenviroment becomes enriched with soluble HH ligands, they bind to the receptor PTC and this interaction SMO, which turns into its active form. Activation of Smo inhibits Hh transcription factor degradation, leading to an intracellular <span class="highlight" style="background-color:">signaling</span> cascade that ultimately drives the activation and nuclear translocation of Gli family zinc-finger transcription factors, which induce the expression of Hh target genes (e.g. Glis, Ptc, Smo and Hh interactive protein, Hhip).</div></p></div><div class="aux"><div class="resc">Alessia Omenetti, et al. Curr Opin Gastroenterol. 2011 May;27(3):268-275.</div><p class="links"><a class="dblinks" href="/pubmed/21423008" ref="ordinalpos=277&amp;ncbi_uid=4833021&amp;link_uid=21423008">Citation</a><a class="dblinks" href="/pmc/articles/PMC3636549/" ref="ordinalpos=277&amp;ncbi_uid=4833021&amp;link_uid=PMC3636549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>278.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3357548/figure/fig2/" ref="ordinalpos=278&amp;ncbi_uid=4028645&amp;link_uid=PMC3357548" href="/pmc/articles/PMC3357548/"><img src="/pmc/articles/instance/3357548/bin/JST2012-376470.002.gif" src-large="/pmc/articles/instance/3357548/bin/JST2012-376470.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3357548/" ref="ordinalpos=278&amp;ncbi_uid=4028645&amp;link_uid=PMC3357548" image-link="/pmc/articles/PMC3357548/figure/fig2/" class="imagepopup">Figure 2.  From: A Bioinformatics Resource for TWEAK-Fn14 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Illustration of the TWEAK page in NetPath. The image provides an outline of the TWEAK <span class="highlight" style="background-color:">pathway</span> as visualized in the NetPath webpage. The figure shows the statistical details of TWEAK <span class="highlight" style="background-color:">pathway</span>-based reactions (right upper corner). Under, “Molecules involved in TWEAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>”, each molecule has been linked to its respective NetPath page. Tabs have been provided which leads to the details of the “<span class="highlight" style="background-color:">pathway</span> authority,” “curators,” and “comments” tab—where the users can provide their feedback and the reaction tabs. “Access to data” indicates links to the events regulated by TWEAK. The <span class="highlight" style="background-color:">pathway</span> can be downloaded from the three standard formats provided at the bottom of the page.</div></p></div><div class="aux"><div class="resc">Mitali Bhattacharjee, et al. J Signal Transduct. 2012;2012:376470.</div><p class="links"><a class="dblinks" href="/pubmed/22649723" ref="ordinalpos=278&amp;ncbi_uid=4028645&amp;link_uid=22649723">Citation</a><a class="dblinks" href="/pmc/articles/PMC3357548/" ref="ordinalpos=278&amp;ncbi_uid=4028645&amp;link_uid=PMC3357548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>279.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3940060/figure/F1/" ref="ordinalpos=279&amp;ncbi_uid=5904857&amp;link_uid=PMC3940060" href="/pmc/articles/PMC3940060/"><img src="/pmc/articles/instance/3940060/bin/fgene-05-00049-g0001.gif" src-large="/pmc/articles/instance/3940060/bin/fgene-05-00049-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3940060/" ref="ordinalpos=279&amp;ncbi_uid=5904857&amp;link_uid=PMC3940060" image-link="/pmc/articles/PMC3940060/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> cloud regulation for in silico screening and ranking of the potential geroprotective drugs. </a></p><div class="supp"><p class="details"><div><b>Gene expression-based approach to <i>in silico</i> screening for drugs with geroprotective properties and estimating the predicted efficacy. (A)</b> Using <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> cloud regulation for theoretical <i>in silico</i> aging-suppressive drug identification and ranking. The proposed method for identifying and ranking of geroprotective drugs by evaluating the net effect on the many elements of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> cloud that brings the “old” metabolic state closer to the “young.” <b>(B)</b> An example of how multiple pathways are activated and down-regulated during aging. <b>(C)</b> <span class="highlight" style="background-color:">Pathway</span> Activation Strength (PAS) is the logarithmic additive value that characterizes the up-/downregulation of <span class="highlight" style="background-color:">signaling</span> pathways. <b>(D)</b> Function for the overall signal <span class="highlight" style="background-color:">pathway</span> cloud disturbance outcome (SPCD).</div></p></div><div class="aux"><div class="resc">Alex Zhavoronkov, et al. Front Genet. 2014;5:49.</div><p class="links"><a class="dblinks" href="/pubmed/24624136" ref="ordinalpos=279&amp;ncbi_uid=5904857&amp;link_uid=24624136">Citation</a><a class="dblinks" href="/pmc/articles/PMC3940060/" ref="ordinalpos=279&amp;ncbi_uid=5904857&amp;link_uid=PMC3940060">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>280.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig3/" ref="ordinalpos=280&amp;ncbi_uid=662999&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F03_4C.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F03_4C.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=280&amp;ncbi_uid=662999&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig3/" class="imagepopup">FIGURE 3.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div>Examples of extreme <span class="highlight" style="background-color:">signaling</span> pathways of the JAK-STAT <span class="highlight" style="background-color:">signaling</span> network. <span class="highlight" style="background-color:">Pathway</span> 119 (<i>A</i>) corresponds to the binding of the interferon <i>γ</i> ligand to its receptor and the subsequent formation of an activated receptor-ligand complex. <span class="highlight" style="background-color:">Pathway</span> 1 (<i>B</i>) corresponds to the activation of the STAT1-STAT2 heterodimer by the interferon <i>γ</i> and interferon <i>α</i>/<i>β</i> bound receptors. The names of the reactions are in italics above the reaction arrows (see supplementary material).</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=280&amp;ncbi_uid=662999&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=280&amp;ncbi_uid=662999&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>281.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2946555/figure/F2/" ref="ordinalpos=281&amp;ncbi_uid=2481509&amp;link_uid=PMC2946555" href="/pmc/articles/PMC2946555/"><img src="/pmc/articles/instance/2946555/bin/nihms-236013-f0002.gif" src-large="/pmc/articles/instance/2946555/bin/nihms-236013-f0002.jpg" alt="Fig. (2)" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2946555/" ref="ordinalpos=281&amp;ncbi_uid=2481509&amp;link_uid=PMC2946555" image-link="/pmc/articles/PMC2946555/figure/F2/" class="imagepopup">Fig. (2).  From: Merlin, a "Magic" Linker Between the Extracellular Cues and Intracellular <span class="highlight" style="background-color:">Signaling</span> Pathways that Regulate Cell Motility, Proliferation, and Survival. </a></p><div class="supp"><p class="details"><div>A working model of merlin-mediated <span class="highlight" style="background-color:">signaling</span> events and their potential cross-talks (the components of <i>Drosophila</i> Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are underlined): merlin functions upstream of the mammalian Hippo (merlin-MST1/2-LATS1/2-YAP) and JNK/p38 <span class="highlight" style="background-color:">signaling</span> pathways and plays an essential role in regulating the cell response to the stresses and stress-induced apoptosis as well as to proliferation/survival signals. Merlin antagonizes CD44 function and inhibits activities of RTKs and the RTK-derived growth and survival signals. CD44 function upstream of mammalian Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> attenuates the stress induced activation of the hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, however, enhances activities of RTKs.</div></p></div><div class="aux"><div class="resc">Ivan Stamenkovic, et al. Curr Protein Pept Sci. ;11(6):471-484.</div><p class="links"><a class="dblinks" href="/pubmed/20491622" ref="ordinalpos=281&amp;ncbi_uid=2481509&amp;link_uid=20491622">Citation</a><a class="dblinks" href="/pmc/articles/PMC2946555/" ref="ordinalpos=281&amp;ncbi_uid=2481509&amp;link_uid=PMC2946555">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>282.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124513/figure/pone-0021474-g006/" ref="ordinalpos=282&amp;ncbi_uid=3076736&amp;link_uid=PMC3124513" href="/pmc/articles/PMC3124513/"><img src="/pmc/articles/instance/3124513/bin/pone.0021474.g006.gif" src-large="/pmc/articles/instance/3124513/bin/pone.0021474.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124513/" ref="ordinalpos=282&amp;ncbi_uid=3076736&amp;link_uid=PMC3124513" image-link="/pmc/articles/PMC3124513/figure/pone-0021474-g006/" class="imagepopup">Figure 6. Proteins and pathways that are regulated by PMA directly and indirectly..  From: Integrated Bio-Entity Network: A System for Biological Knowledge Discovery. </a></p><div class="supp"><p class="details"><div>PMA affected pathways through IL2 are: Calcineurin-regulated NFAT-dependent transcription in lymphocytes, IL27-mediated <span class="highlight" style="background-color:">signaling</span> events, IL12-mediated <span class="highlight" style="background-color:">signaling</span> events, IL23-mediated <span class="highlight" style="background-color:">signaling</span> events, Glucocorticoid receptor regulatory network, IL2-mediated <span class="highlight" style="background-color:">signaling</span> events, IL2 <span class="highlight" style="background-color:">signaling</span> events mediated by PI3K, Calcium <span class="highlight" style="background-color:">signaling</span> in the CD4+ TCR <span class="highlight" style="background-color:">pathway</span>, Regulation of Telomerase, IL12 <span class="highlight" style="background-color:">signaling</span> mediated by STAT4, Downstream <span class="highlight" style="background-color:">signaling</span> in naïve CD8+ T cells and IL2 <span class="highlight" style="background-color:">signaling</span> events mediated by STAT5; PMA affected pathways through IGHE are: IL4-mediated <span class="highlight" style="background-color:">signaling</span> events and Fc-epsilon receptor I <span class="highlight" style="background-color:">signaling</span> in mast cells; PMA affected pathways through CEBPA are: E2F transcription factor network, Regulation of retinoblastoma protein, regulation of Androgen receptor activity, FOXA2 and FOXA3 transcription factor networks and C-MYB transcription factor network; PMA affected <span class="highlight" style="background-color:">pathway</span> through TPM3 is Muscle contraction.</div></p></div><div class="aux"><div class="resc">Lindsey Bell, et al. PLoS One. 2011;6(6):e21474.</div><p class="links"><a class="dblinks" href="/pubmed/21738677" ref="ordinalpos=282&amp;ncbi_uid=3076736&amp;link_uid=21738677">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124513/" ref="ordinalpos=282&amp;ncbi_uid=3076736&amp;link_uid=PMC3124513">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>283.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g001/" ref="ordinalpos=283&amp;ncbi_uid=3758048&amp;link_uid=PMC3256223" href="/pmc/articles/PMC3256223/"><img src="/pmc/articles/instance/3256223/bin/pone.0030140.g001.gif" src-large="/pmc/articles/instance/3256223/bin/pone.0030140.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3256223/" ref="ordinalpos=283&amp;ncbi_uid=3758048&amp;link_uid=PMC3256223" image-link="/pmc/articles/PMC3256223/figure/pone-0030140-g001/" class="imagepopup">Figure 1. Overview of the model..  From: Modeling of miRNA and Drug Action in the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>A</b>: The model of the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> contains 4 sub-pathways: MAPK <span class="highlight" style="background-color:">pathway</span>, PI3K-AKT <span class="highlight" style="background-color:">pathway</span>, Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, PAK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>B, C</b> are parts of the model network visualized by PyBioS. <b>B</b>: The mRNA BAD-1 is produced by the transcription reaction (1) of gene BAD and also takes part in decay reaction (2) and translation-reaction (3), which produces the protein BAD-1 (cytoplasm). The protein BAD-1 takes part in further three reactions that are the decay-reaction (4), phosphorylation reaction (5) and dephosphorylation reaction (6). The phosphorylated protein P-AKT (plasma membrane) catalyzes the phosphorylation reaction, in which protein BAD-1 is phosphorylated into P-BAD-1. Afterwards, the protein P-BAD-1 is then dephosphorylated; <b>C</b>: shows a simplified miRNA biogenesis, target recognition and competitive anti-miRNA effect. (1) miRNA-gene transcription; (2) miRNA translocation (from nucleus into cytoplasm); (3) miRNA-binding-target reaction; (4) miRNA binds to the miRNA inhibitor.</div></p></div><div class="aux"><div class="resc">Jian Li, et al. PLoS One. 2012;7(1):e30140.</div><p class="links"><a class="dblinks" href="/pubmed/22253908" ref="ordinalpos=283&amp;ncbi_uid=3758048&amp;link_uid=22253908">Citation</a><a class="dblinks" href="/pmc/articles/PMC3256223/" ref="ordinalpos=283&amp;ncbi_uid=3758048&amp;link_uid=PMC3256223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>284.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3072770/figure/F4/" ref="ordinalpos=284&amp;ncbi_uid=2882097&amp;link_uid=PMC3072770" href="/pmc/articles/PMC3072770/"><img src="/pmc/articles/instance/3072770/bin/bar010f4.gif" src-large="/pmc/articles/instance/3072770/bin/bar010f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3072770/" ref="ordinalpos=284&amp;ncbi_uid=2882097&amp;link_uid=PMC3072770" image-link="/pmc/articles/PMC3072770/figure/F4/" class="imagepopup">Figure 4. From: The Rat Genome Database <span class="highlight" style="background-color:">Pathway</span> Portal. </a></p><div class="supp"><p class="details"><div>Shows a partial view of the Ontology Report page for the ‘glucagon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>’ (<b>A)</b> and the path(s) to the ‘glucagon <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>’ term within the ontology (<b>B</b>).</div></p></div><div class="aux"><div class="resc">Victoria Petri, et al. Database (Oxford). 2011;2011:bar010.</div><p class="links"><a class="dblinks" href="/pubmed/21478484" ref="ordinalpos=284&amp;ncbi_uid=2882097&amp;link_uid=21478484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3072770/" ref="ordinalpos=284&amp;ncbi_uid=2882097&amp;link_uid=PMC3072770">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>285.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3696739/figure/F3/" ref="ordinalpos=285&amp;ncbi_uid=6214006&amp;link_uid=PMC3696739" href="/pmc/articles/PMC3696739/"><img src="/pmc/articles/instance/3696739/bin/nihms475406f3.gif" src-large="/pmc/articles/instance/3696739/bin/nihms475406f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3696739/" ref="ordinalpos=285&amp;ncbi_uid=6214006&amp;link_uid=PMC3696739" image-link="/pmc/articles/PMC3696739/figure/F3/" class="imagepopup">Figure 3. Downstream <span class="highlight" style="background-color:">signaling</span> engaged by PTHrP during embryonic mammary development.  From: Parathyroid Hormone-Related Protein Specifies the Mammary Mesenchyme and Regulates Embryonic Mammary Development. </a></p><div class="supp"><p class="details"><div>PTHrP is secreted from the mammary epithelium (pink) and acts on the mammary mesenchyme (blue) to activate <span class="highlight" style="background-color:">signaling</span> via β-catenin. This <span class="highlight" style="background-color:">pathway</span> directs sexual dimorphism, in part via regulation of AR expression, and possibly regulates the response to BMP <span class="highlight" style="background-color:">signaling</span>. PTHrP <span class="highlight" style="background-color:">signaling</span>, via PTHR1, also upregulates BMPR1a and sensitizes mammary mesenchymal cells to BMP4, possibly upregulating BMP <span class="highlight" style="background-color:">signaling</span> in the mammary mesenchyme. This <span class="highlight" style="background-color:">pathway</span> is required for bud outgrowth, via MMP2 and for inhibiting hair folliculogenesis in the nipple, via Msx2. PTHrP <span class="highlight" style="background-color:">signaling</span> also maintains mammary epithelial identity via <span class="highlight" style="background-color:">signaling</span> in the mammary mesenchyme, although the exact mechanism remains unknown.</div></p></div><div class="aux"><div class="resc">Minoti Hiremath, et al. J Mammary Gland Biol Neoplasia. ;18(2):171-177.</div><p class="links"><a class="dblinks" href="/pubmed/23640717" ref="ordinalpos=285&amp;ncbi_uid=6214006&amp;link_uid=23640717">Citation</a><a class="dblinks" href="/pmc/articles/PMC3696739/" ref="ordinalpos=285&amp;ncbi_uid=6214006&amp;link_uid=PMC3696739">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>286.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1959522/figure/F4/" ref="ordinalpos=286&amp;ncbi_uid=1097796&amp;link_uid=PMC1959522" href="/pmc/articles/PMC1959522/"><img src="/pmc/articles/instance/1959522/bin/1471-2164-8-230-4.gif" src-large="/pmc/articles/instance/1959522/bin/1471-2164-8-230-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1959522/" ref="ordinalpos=286&amp;ncbi_uid=1097796&amp;link_uid=PMC1959522" image-link="/pmc/articles/PMC1959522/figure/F4/" class="imagepopup">Figure 4.  From: Genome wide transcriptional analysis of resting and IL2 activated human natural killer cells: gene expression signatures indicative of novel molecular <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways. A) JAK/STAT <span class="highlight" style="background-color:">pathway</span>, B) TGFB1 <span class="highlight" style="background-color:">pathway</span>, C) PI3K-activation <span class="highlight" style="background-color:">pathway</span>, D) NFAT activation <span class="highlight" style="background-color:">pathway</span>, E) I: NF-κB <span class="highlight" style="background-color:">pathway</span> regulation and II: NF-κB target genes. (<i>See also figure 2 legend for details</i>)</div></p></div><div class="aux"><div class="resc">Karen Dybkaer, et al. BMC Genomics. 2007;8:230-230.</div><p class="links"><a class="dblinks" href="/pubmed/17623099" ref="ordinalpos=286&amp;ncbi_uid=1097796&amp;link_uid=17623099">Citation</a><a class="dblinks" href="/pmc/articles/PMC1959522/" ref="ordinalpos=286&amp;ncbi_uid=1097796&amp;link_uid=PMC1959522">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>287.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3708893/figure/pone-0068983-g005/" ref="ordinalpos=287&amp;ncbi_uid=5115840&amp;link_uid=PMC3708893" href="/pmc/articles/PMC3708893/"><img src="/pmc/articles/instance/3708893/bin/pone.0068983.g005.gif" src-large="/pmc/articles/instance/3708893/bin/pone.0068983.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3708893/" ref="ordinalpos=287&amp;ncbi_uid=5115840&amp;link_uid=PMC3708893" image-link="/pmc/articles/PMC3708893/figure/pone-0068983-g005/" class="imagepopup">Figure 5. Graphic representation of Death Receptor <span class="highlight" style="background-color:">Pathway</span> and Apoptosis <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in neutrophils demonstrates that they are altered in neutrophils in chronic inflammation..  From: Oral Neutrophil Transcriptome Changes Result in a Pro-Survival Phenotype in Periodontal Diseases. </a></p><div class="supp"><p class="details"><div>IPA canonical <span class="highlight" style="background-color:">pathway</span> analysis of Death Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in neutrophils from (A) healthy individuals and (B) chronic periodontitis patients. Red represents up-regulated genes, green are down-regulated genes and white symbols depict neighbouring genes in this analysis.</div></p></div><div class="aux"><div class="resc">Flavia S. Lakschevitz, et al. PLoS One. 2013;8(7):e68983.</div><p class="links"><a class="dblinks" href="/pubmed/23874838" ref="ordinalpos=287&amp;ncbi_uid=5115840&amp;link_uid=23874838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3708893/" ref="ordinalpos=287&amp;ncbi_uid=5115840&amp;link_uid=PMC3708893">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>288.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3185594/figure/f8-viruses-02-02618/" ref="ordinalpos=288&amp;ncbi_uid=3279786&amp;link_uid=PMC3185594" href="/pmc/articles/PMC3185594/"><img src="/pmc/articles/instance/3185594/bin/viruses-02-02618f8.gif" src-large="/pmc/articles/instance/3185594/bin/viruses-02-02618f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3185594/" ref="ordinalpos=288&amp;ncbi_uid=3279786&amp;link_uid=PMC3185594" image-link="/pmc/articles/PMC3185594/figure/f8-viruses-02-02618/" class="imagepopup">Figure 8.  From: Jaagsiekte Sheep Retrovirus Biology and Oncogenesis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways involved in JSRV Env-induced cell transformation. Depending on the type of cell line studied, three main pathways have been shown to be activated in JSRV transformed cells; the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, the Hyal2-RON <span class="highlight" style="background-color:">pathway</span>, and the Ras-MEK-ERK <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Signaling</span> through the TLR4-NFkB <span class="highlight" style="background-color:">pathway</span> is also depicted, since the JSRV Env interacts with TLR4, though activation of this <span class="highlight" style="background-color:">pathway</span> has yet to be formally shown. A fourth <span class="highlight" style="background-color:">pathway</span>; via <span class="highlight" style="background-color:">signaling</span> through Rac1, has recently been identified, but less is known about it [86]. Except for the Hyal2-RON <span class="highlight" style="background-color:">pathway</span>, how the JSRV Env engages the Ras-MEK-ERK or the PI3K/Akt pathways is not known. Possible interactions, or yet to be identified factors that may be involved in JSRV Env-mediated transformation, are indicated with question marks (?). Each <span class="highlight" style="background-color:">pathway</span> involved in JSRV Env-transformation leads to the activation of factors involved in cell survival, proliferation, apoptosis, and differentiation. See text for details.</div></p></div><div class="aux"><div class="resc">Andrew Hofacre, et al. Viruses. 2010 December;2(12):2618-2648.</div><p class="links"><a class="dblinks" href="/pubmed/21994634" ref="ordinalpos=288&amp;ncbi_uid=3279786&amp;link_uid=21994634">Citation</a><a class="dblinks" href="/pmc/articles/PMC3185594/" ref="ordinalpos=288&amp;ncbi_uid=3279786&amp;link_uid=PMC3185594">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>289.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2991117/figure/F5/" ref="ordinalpos=289&amp;ncbi_uid=2588315&amp;link_uid=PMC2991117" href="/pmc/articles/PMC2991117/"><img src="/pmc/articles/instance/2991117/bin/nihms242216f5.gif" src-large="/pmc/articles/instance/2991117/bin/nihms242216f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2991117/" ref="ordinalpos=289&amp;ncbi_uid=2588315&amp;link_uid=PMC2991117" image-link="/pmc/articles/PMC2991117/figure/F5/" class="imagepopup">Figure 5. Caspase-3 deficiency impairs B-cell maturation by affecting <span class="highlight" style="background-color:">signaling</span> via the Ras-Raf-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Hematopoietic stem cell responsiveness to exogenous signals limited by Caspase-3. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Analysis of B-cell development in presence or absence of Caspase-3 revealed a reduction of the pre-B cells fraction and a relative accumulation of the pro-B cells fraction within the B-cell compartment indicating an incomplete block at the pro-B to pre-B transition in Caspase-3<sup>−/−</sup>mice. Values are mean ±s.d., n=5, *, <i>P</i>&lt;0.05.<br /><b>(B)</b> Stimulation of WT and Caspase-3 KO pro-B cell line with IL-7 revealed defective <span class="highlight" style="background-color:">signaling</span> through the ERK-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while the JAK-STAT <span class="highlight" style="background-color:">pathway</span> remained unperturbed in the absence of Caspase-3. The experiment was performed twice with similar results.</div></p></div><div class="aux"><div class="resc">Viktor Janzen, et al. Cell Stem Cell. ;2(6):584-594.</div><p class="links"><a class="dblinks" href="/pubmed/18522851" ref="ordinalpos=289&amp;ncbi_uid=2588315&amp;link_uid=18522851">Citation</a><a class="dblinks" href="/pmc/articles/PMC2991117/" ref="ordinalpos=289&amp;ncbi_uid=2588315&amp;link_uid=PMC2991117">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>290.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193406/figure/fig1/" ref="ordinalpos=290&amp;ncbi_uid=3770402&amp;link_uid=PMC3193406" href="/pmc/articles/PMC3193406/"><img src="/pmc/articles/instance/3193406/bin/cr2010177f1.gif" src-large="/pmc/articles/instance/3193406/bin/cr2010177f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193406/" ref="ordinalpos=290&amp;ncbi_uid=3770402&amp;link_uid=PMC3193406" image-link="/pmc/articles/PMC3193406/figure/fig1/" class="imagepopup">Figure 1.  From: Non-canonical NF-?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Canonical and non-canonical NF-κB <span class="highlight" style="background-color:">signaling</span> pathways. Canonical <span class="highlight" style="background-color:">pathway</span> is triggered by numerous signals, including those mediated by innate and adaptive immune receptors. It involves activation of IKK complex by Tak1, IKK-mediated IκBα phosphorylation, and subsequent degradation, resulting in rapid and transient nuclear translocation of the prototypical NF-κB heterodimer RelA/p50. Non-canonical NF-κB <span class="highlight" style="background-color:">pathway</span> relies on phosphorylation-induced p100 processing, which is triggered by <span class="highlight" style="background-color:">signaling</span> from a subset of TNFR members. This <span class="highlight" style="background-color:">pathway</span> is dependent on NIK and IKKα, but not on the trimeric IKK complex, and mediates the persistent activation of RelB/p52 complex.</div></p></div><div class="aux"><div class="resc">Shao-Cong Sun. Cell Res. 2011 January;21(1):71-85.</div><p class="links"><a class="dblinks" href="/pubmed/21173796" ref="ordinalpos=290&amp;ncbi_uid=3770402&amp;link_uid=21173796">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193406/" ref="ordinalpos=290&amp;ncbi_uid=3770402&amp;link_uid=PMC3193406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>291.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2708714/figure/F2/" ref="ordinalpos=291&amp;ncbi_uid=1987578&amp;link_uid=PMC2708714" href="/pmc/articles/PMC2708714/"><img src="/pmc/articles/instance/2708714/bin/zpq9990985740002.gif" src-large="/pmc/articles/instance/2708714/bin/zpq9990985740002.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2708714/" ref="ordinalpos=291&amp;ncbi_uid=1987578&amp;link_uid=PMC2708714" image-link="/pmc/articles/PMC2708714/figure/F2/" class="imagepopup">Fig. 2. From: An integrated genome screen identifies the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a major target of WT1. </a></p><div class="supp"><p class="details"><div>Direct targets of WT1 identified in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes bound and regulated by WT1 are indicated in red bold. <i>PPP3CB</i>, a candidate WT1 direct target, is involved in noncanonical Wnt/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> (not shown here). Activation and repression by WT1 are shown with up and down arrows, respectively.</div></p></div><div class="aux"><div class="resc">Marianne K.-H. Kim, et al. Proc Natl Acad Sci U S A. 2009 July 7;106(27):11154-11159.</div><p class="links"><a class="dblinks" href="/pubmed/19549856" ref="ordinalpos=291&amp;ncbi_uid=1987578&amp;link_uid=19549856">Citation</a><a class="dblinks" href="/pmc/articles/PMC2708714/" ref="ordinalpos=291&amp;ncbi_uid=1987578&amp;link_uid=PMC2708714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>292.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3207424/figure/F6/" ref="ordinalpos=292&amp;ncbi_uid=3540051&amp;link_uid=PMC3207424" href="/pmc/articles/PMC3207424/"><img src="/pmc/articles/instance/3207424/bin/zbc0501185650006.gif" src-large="/pmc/articles/instance/3207424/bin/zbc0501185650006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3207424/" ref="ordinalpos=292&amp;ncbi_uid=3540051&amp;link_uid=PMC3207424" image-link="/pmc/articles/PMC3207424/figure/F6/" class="imagepopup">FIGURE 6. From: Enteric Commensal Bacteria Induce Extracellular Signal-regulated Kinase <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> via Formyl Peptide Receptor-dependent Redox Modulation of Dual Specific Phosphatase 3. </a></p><div class="supp"><p class="details"><div><b>Model for commensal bacteria-induced ERK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> via FPR-dependent redox modulation of DUSP3.</b></div></p></div><div class="aux"><div class="resc">Christy C. Wentworth, et al. J Biol Chem. 2011 November 4;286(44):38448-38455.</div><p class="links"><a class="dblinks" href="/pubmed/21921027" ref="ordinalpos=292&amp;ncbi_uid=3540051&amp;link_uid=21921027">Citation</a><a class="dblinks" href="/pmc/articles/PMC3207424/" ref="ordinalpos=292&amp;ncbi_uid=3540051&amp;link_uid=PMC3207424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>293.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382127/figure/pone-0039592-g007/" ref="ordinalpos=293&amp;ncbi_uid=4087216&amp;link_uid=PMC3382127" href="/pmc/articles/PMC3382127/"><img src="/pmc/articles/instance/3382127/bin/pone.0039592.g007.gif" src-large="/pmc/articles/instance/3382127/bin/pone.0039592.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382127/" ref="ordinalpos=293&amp;ncbi_uid=4087216&amp;link_uid=PMC3382127" image-link="/pmc/articles/PMC3382127/figure/pone-0039592-g007/" class="imagepopup">Figure 7. SOCS1 and SOCS3 inhibit TLR3 <span class="highlight" style="background-color:">signaling</span> in hMSC..  From: Toll-Like Receptor 3 and Suppressor of Cytokine <span class="highlight" style="background-color:">Signaling</span> Proteins Regulate CXCR4 and CXCR7 Expression in Bone Marrow-Derived Human Multipotent Stromal Cells. </a></p><div class="supp"><p class="details"><div>TLR3 <span class="highlight" style="background-color:">signaling</span> induces cytokine expression and mediates the internalization of both CXCR4 and CXCR7, as well as the degradation of CXCR4. A JAK2/STAT1 <span class="highlight" style="background-color:">pathway</span> is also activated, and subsequent upregulation of SOCS1 and SOCS3 inhibits TLR3-mediated <span class="highlight" style="background-color:">signaling</span> by targeting different branches of the <span class="highlight" style="background-color:">signaling</span> pathways. SOCS1 inhibits STAT1 phosphorylation and CXCR7 internalization, while SOCS3 inhibits IRF1 upregulation and the internalization of both CXCR4 and CXCR7.</div></p></div><div class="aux"><div class="resc">Suzanne L. Tomchuck, et al. PLoS One. 2012;7(6):e39592.</div><p class="links"><a class="dblinks" href="/pubmed/22745793" ref="ordinalpos=293&amp;ncbi_uid=4087216&amp;link_uid=22745793">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382127/" ref="ordinalpos=293&amp;ncbi_uid=4087216&amp;link_uid=PMC3382127">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>294.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3679355/figure/F2/" ref="ordinalpos=294&amp;ncbi_uid=6205738&amp;link_uid=PMC3679355" href="/pmc/articles/PMC3679355/"><img src="/pmc/articles/instance/3679355/bin/nihms461199f2.gif" src-large="/pmc/articles/instance/3679355/bin/nihms461199f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3679355/" ref="ordinalpos=294&amp;ncbi_uid=6205738&amp;link_uid=PMC3679355" image-link="/pmc/articles/PMC3679355/figure/F2/" class="imagepopup">Figure 2. Microarray expression of Wnt <span class="highlight" style="background-color:">signaling</span> regulators and <span class="highlight" style="background-color:">pathway</span> members.  From: Canonical Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> Drives Human Schwann Cell Transformation, Progression, and Tumor Maintenance. </a></p><div class="supp"><p class="details"><div>Heat-map depicting microarray expression analysis for Wnt <span class="highlight" style="background-color:">signaling</span> regulators and <span class="highlight" style="background-color:">pathway</span> members. NHSC = purified normal human Schwann cells, dNFSC= purified dermal neurofibroma Schwann cells, pNFSC = purified plexiform-neurofibroma Schwann cells, MPNST cell = human MPNST cell lines, NV = normal human nerve, dNF = dermal neurofibroma, pNF = plexiform-neurofibroma, MPNST = bulk MPNST tumor.</div></p></div><div class="aux"><div class="resc">Adrienne L. Watson, et al. Cancer Discov. ;3(6):674-689.</div><p class="links"><a class="dblinks" href="/pubmed/23535903" ref="ordinalpos=294&amp;ncbi_uid=6205738&amp;link_uid=23535903">Citation</a><a class="dblinks" href="/pmc/articles/PMC3679355/" ref="ordinalpos=294&amp;ncbi_uid=6205738&amp;link_uid=PMC3679355">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>295.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4263734/figure/pone-0114627-g005/" ref="ordinalpos=295&amp;ncbi_uid=6982367&amp;link_uid=PMC4263734" href="/pmc/articles/PMC4263734/"><img src="/pmc/articles/instance/4263734/bin/pone.0114627.g005.gif" src-large="/pmc/articles/instance/4263734/bin/pone.0114627.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4263734/" ref="ordinalpos=295&amp;ncbi_uid=6982367&amp;link_uid=PMC4263734" image-link="/pmc/articles/PMC4263734/figure/pone-0114627-g005/" class="imagepopup">Figure 5. Diagram of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Altered MicroRNA Expression Profile in Exosomes during Osteogenic Differentiation of Human Bone Marrow-Derived Mesenchymal Stem Cells. </a></p><div class="supp"><p class="details"><div>Four differentially expressed miRNAs (including miR-203, miR-299-5p, miR-885-5p, and miR-320c) were predicted to participate in Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through regulating their potential gene targets.</div></p></div><div class="aux"><div class="resc">Ji-Feng Xu, et al. PLoS One. 2014;9(12):e114627.</div><p class="links"><a class="dblinks" href="/pubmed/25503309" ref="ordinalpos=295&amp;ncbi_uid=6982367&amp;link_uid=25503309">Citation</a><a class="dblinks" href="/pmc/articles/PMC4263734/" ref="ordinalpos=295&amp;ncbi_uid=6982367&amp;link_uid=PMC4263734">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>296.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3883653/figure/pone-0083988-g005/" ref="ordinalpos=296&amp;ncbi_uid=5691423&amp;link_uid=PMC3883653" href="/pmc/articles/PMC3883653/"><img src="/pmc/articles/instance/3883653/bin/pone.0083988.g005.gif" src-large="/pmc/articles/instance/3883653/bin/pone.0083988.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3883653/" ref="ordinalpos=296&amp;ncbi_uid=5691423&amp;link_uid=PMC3883653" image-link="/pmc/articles/PMC3883653/figure/pone-0083988-g005/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Pathway</span> analysis indicates that male and female mice fed Rapa share 13 pathways..  From: Mice Fed Rapamycin Have an Increase in Lifespan Associated with Major Changes in the Liver Transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis was conducted using IPA for significantly changed transcripts in Rapa-2 males and Rapa females relative to respective sex controls with adjusted p&lt;0.05 and 15% change. IPA used Fisher's exact test to place genes into pathways, and we also used a FDR cutoff of B-H p&lt;0.05 to show the highly significant pathways. The 13 shared pathways ranked by B-H p-value are shown for Rapa-2 males (<b>A</b>) and Rapa females (<b>B</b>). For the graphical portion of A and B, the yellow line indicates the –log(B-H p-value). Red indicates up-regulated and green represents down-regulated genes significantly changed in Rapa compared to control, and white indicates the percentage of genes not significantly changed in the <span class="highlight" style="background-color:">pathway</span> relative to the total number of genes in that <span class="highlight" style="background-color:">pathway</span> (bolded number). Twenty-four other pathways were observed to change significantly in Rapa-2 males: acute phase response <span class="highlight" style="background-color:">signaling</span>, hypoxia <span class="highlight" style="background-color:">signaling</span>, PI3K/AKT <span class="highlight" style="background-color:">signaling</span>, remodeling of epithelial adherens junctions, sertoli cell-sertoli cell junction <span class="highlight" style="background-color:">signaling</span>, renal cell carcinoma <span class="highlight" style="background-color:">signaling</span>, JAK/Stat <span class="highlight" style="background-color:">signaling</span>, IL-3 <span class="highlight" style="background-color:">signaling</span>, myc mediated apoptosis <span class="highlight" style="background-color:">Signaling</span>, germ cell-sertoli cell junction <span class="highlight" style="background-color:">signaling</span>, pancreatic adenocarcinoma <span class="highlight" style="background-color:">signaling</span>, insulin receptor <span class="highlight" style="background-color:">signaling</span>, non-small cell lung cancer <span class="highlight" style="background-color:">signaling</span>, Rac <span class="highlight" style="background-color:">signaling</span>, small cell lung cancer <span class="highlight" style="background-color:">signaling</span>, PTEN <span class="highlight" style="background-color:">signaling</span>, prostate cancer <span class="highlight" style="background-color:">signaling</span>, neurotrophin/TRK <span class="highlight" style="background-color:">signaling</span>, PAK <span class="highlight" style="background-color:">signaling</span>, RAR activation, NGF <span class="highlight" style="background-color:">signaling</span>, clathrin-mediated endocytosis <span class="highlight" style="background-color:">signaling</span>, huntington's disease <span class="highlight" style="background-color:">signaling</span>, and TCA cycle II. Six other pathways were observed to change significantly in Rapa females: systemic lupus erythematosus <span class="highlight" style="background-color:">signaling</span>, mTOR <span class="highlight" style="background-color:">signaling</span>, actin nucleation by ARP-WASP complex, superpathway of methionine degradation, glutathione-mediated detoxification, and oncostatin M <span class="highlight" style="background-color:">signaling</span>. A list of the pathways that significantly changed with chronic Rapa feeding is shown in in File S2, tab2 for Rapa-2 male; tab3 for Rapa female and the transcripts that change for the 13 shared pathways are found in File S3. The asterisk indicates that the mitochondrial function <span class="highlight" style="background-color:">pathway</span> is indicated as mitochondrial dysfunction in IPA. The mitochondrial dysfunction <span class="highlight" style="background-color:">pathway</span> described in IPA is somewhat misleading because it contains genes involved in mitochondrial function in general, e.g., genes in the electron transport chain complexes and redox and antioxidant enzymes as well as other genes related to mitochondrial function.</div></p></div><div class="aux"><div class="resc">Wilson C. Fok, et al. PLoS One. 2014;9(1):e83988.</div><p class="links"><a class="dblinks" href="/pubmed/24409289" ref="ordinalpos=296&amp;ncbi_uid=5691423&amp;link_uid=24409289">Citation</a><a class="dblinks" href="/pmc/articles/PMC3883653/" ref="ordinalpos=296&amp;ncbi_uid=5691423&amp;link_uid=PMC3883653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>297.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2658630/figure/F7/" ref="ordinalpos=297&amp;ncbi_uid=1711384&amp;link_uid=PMC2658630" href="/pmc/articles/PMC2658630/"><img src="/pmc/articles/instance/2658630/bin/nihms85267f7.gif" src-large="/pmc/articles/instance/2658630/bin/nihms85267f7.jpg" alt="FIGURE 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2658630/" ref="ordinalpos=297&amp;ncbi_uid=1711384&amp;link_uid=PMC2658630" image-link="/pmc/articles/PMC2658630/figure/F7/" class="imagepopup">FIGURE 7. Lysophospholipid-mediated <span class="highlight" style="background-color:">signaling</span> pathways in MVECs.  From: ANASTELLIN, THE ANGIOSTATIC FIBRONECTIN PEPTIDE, IS A SELECTIVE INHIBITOR OF LYSOPHOSPHOLIPID <span class="highlight" style="background-color:">SIGNALING</span>. </a></p><div class="supp"><p class="details"><div>LPA and S1P activate specific G-protein-coupled receptors (LPA<sub>1–4</sub> and S1P<sub>1–5</sub>, respectively) and initiate Ras/ERK, PI3K/Akt, and Rho/ROCK <span class="highlight" style="background-color:">signaling</span> pathways in microvessel endothelial cells. Results presented here have demonstrated that both Ras/ERK and PI3K/Akt <span class="highlight" style="background-color:">signaling</span> is mediated by a G<sub>i</sub>-dependent <span class="highlight" style="background-color:">pathway</span> while Rho <span class="highlight" style="background-color:">signaling</span> occurs through an alternate G-protein, possibly G<sub>12/13</sub> or G<sub>q</sub> as has been shown in previous studies (5–9). In addition, G<sub>i</sub>-dependent activation of Ras/ERK and PI3K/Akt occur through independent pathways and are unaffected by inhibition of Rho <span class="highlight" style="background-color:">signaling</span>. These results are further supported by the finding that anastellin suppresses Ras activation by LPA and prevents ERK activation without affecting PI3K/Akt or Rho/ROCK <span class="highlight" style="background-color:">signaling</span>. The <i>dashed</i> line linking S1P receptors to G<sub>12/13</sub> indicates that a weak induction by S1P was observed in MVECs.</div></p></div><div class="aux"><div class="resc">Anthony Ambesi, et al. Mol Cancer Res. ;7(2):255-265.</div><p class="links"><a class="dblinks" href="/pubmed/19208746" ref="ordinalpos=297&amp;ncbi_uid=1711384&amp;link_uid=19208746">Citation</a><a class="dblinks" href="/pmc/articles/PMC2658630/" ref="ordinalpos=297&amp;ncbi_uid=1711384&amp;link_uid=PMC2658630">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>298.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3285398/figure/F2/" ref="ordinalpos=298&amp;ncbi_uid=5231230&amp;link_uid=PMC3285398" href="/pmc/articles/PMC3285398/"><img src="/pmc/articles/instance/3285398/bin/nihms354715f2.gif" src-large="/pmc/articles/instance/3285398/bin/nihms354715f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3285398/" ref="ordinalpos=298&amp;ncbi_uid=5231230&amp;link_uid=PMC3285398" image-link="/pmc/articles/PMC3285398/figure/F2/" class="imagepopup">Fig. 2.  From: MUCINS AND TOLL-LIKE RECEPTORS: KITH AND KIN IN INFECTION AND CANCER. </a></p><div class="supp"><p class="details"><div>The proposed mechanisms of interaction between TLRs and MUCs: I. Ligand-TLR interaction initiates <span class="highlight" style="background-color:">signaling</span> pathways leading to the induction of several pro-inflammatory cytokines MAPKs and NF-κB which are known to induce mucin gene expression. Alternatively these molecules can also induce TLR expression in turn regulating mucin expression thereby forming a self-perpetuating <span class="highlight" style="background-color:">signaling</span> loop. Constitutively active TLR pathways are commonly observed in several cancers.<br />II. Mucins can also activate <span class="highlight" style="background-color:">signaling</span> pathways such as their interaction with receptor tyrosine kinase ErbB2 and other molecules activates <span class="highlight" style="background-color:">signaling</span> pathways which have common downstream <span class="highlight" style="background-color:">signaling</span> components and lead to a cross-talk with TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. III. Mucins are large glycoprotein molecules which exhibit the <i>receptor masking</i> and can provide a steric hindrance for endogenous ligand binding to TLRs thus inhibiting their <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IV. The sequestration of TLR ligands in the dense mucin matrix can have two possible outcomes either activation or inhibition of TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. V. Altered glycosylation of mucin molecules can act as a ligand for TLRs and activate TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shikha Tarang, et al. Cancer Lett. ;321(2):110-119.</div><p class="links"><a class="dblinks" href="/pubmed/22306702" ref="ordinalpos=298&amp;ncbi_uid=5231230&amp;link_uid=22306702">Citation</a><a class="dblinks" href="/pmc/articles/PMC3285398/" ref="ordinalpos=298&amp;ncbi_uid=5231230&amp;link_uid=PMC3285398">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>299.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3733232/figure/F1/" ref="ordinalpos=299&amp;ncbi_uid=5165999&amp;link_uid=PMC3733232" href="/pmc/articles/PMC3733232/"><img src="/pmc/articles/instance/3733232/bin/nihms237672f1.gif" src-large="/pmc/articles/instance/3733232/bin/nihms237672f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3733232/" ref="ordinalpos=299&amp;ncbi_uid=5165999&amp;link_uid=PMC3733232" image-link="/pmc/articles/PMC3733232/figure/F1/" class="imagepopup">Figure 1. Schematic diagram of transforming growth factor-β (TGFβ) <span class="highlight" style="background-color:">signaling</span> and regulation.  From: TGF?-mediated <span class="highlight" style="background-color:">signaling</span> and transcriptional regulation in pancreatic development and cancer. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> includes two types of membrane Ser/Thr receptor kinases (TβR-I and TβR-II), the receptor Smads2/3, the co-Smad4, the inhibitory Smad7 and nuclear partner proteins (<i>eg</i>, DNA-binding factors). The Smad <span class="highlight" style="background-color:">signaling</span> is negatively regulated by Smad7, which itself can be induced by several crosstalks, including epidermal growth factor <span class="highlight" style="background-color:">signaling</span> and NFkB.</div></p></div><div class="aux"><div class="resc">Volker Ellenrieder, et al. Curr Opin Gastroenterol. ;17(5):434-440.</div><p class="links"><a class="dblinks" href="/pubmed/17031197" ref="ordinalpos=299&amp;ncbi_uid=5165999&amp;link_uid=17031197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3733232/" ref="ordinalpos=299&amp;ncbi_uid=5165999&amp;link_uid=PMC3733232">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>300.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2410087/figure/f1-cin-03-379/" ref="ordinalpos=300&amp;ncbi_uid=2106260&amp;link_uid=PMC2410087" href="/pmc/articles/PMC2410087/"><img src="/pmc/articles/instance/2776330/bin/CIN-03-379-g001.gif" src-large="/pmc/articles/instance/2776330/bin/CIN-03-379-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2410087/" ref="ordinalpos=300&amp;ncbi_uid=2106260&amp;link_uid=PMC2410087" image-link="/pmc/articles/PMC2410087/figure/f1-cin-03-379/" class="imagepopup">Figure 1. From: Public Databases and Software for the <span class="highlight" style="background-color:">Pathway</span> Analysis of Cancer Genomes. </a></p><div class="supp"><p class="details"><div>Example of EGFR-mediated <span class="highlight" style="background-color:">signaling</span> changes, a commonly disrupted <span class="highlight" style="background-color:">pathway</span> in lung cancer. The EGFR <span class="highlight" style="background-color:">pathway</span> could be disrupted by an increased expression of growth factor ligands. By targeting EGFR with tyrosine kinase inhibitors (TKIs) and MAb (monoclonal antibodies), EGFR activity can be eliminated. However, a downstream factor (e.g. MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) may also be activated to disrupt the <span class="highlight" style="background-color:">pathway</span>, thus making TKIs ineffective. <span class="highlight" style="background-color:">Pathway</span> data was obtained and selected from the Cancer Cell Map database and drawn using <i>Cytoscape</i>.</div></p></div><div class="aux"><div class="resc">Ivy F.L. Tsui, et al. Cancer Inform. 2007;3:379-397.</div><p class="links"><a class="dblinks" href="/pubmed/19455256" ref="ordinalpos=300&amp;ncbi_uid=2106260&amp;link_uid=19455256">Citation</a><a class="dblinks" href="/pmc/articles/PMC2410087/" ref="ordinalpos=300&amp;ncbi_uid=2106260&amp;link_uid=PMC2410087">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="2" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="3" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="4" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ebca2cbe05d491bc324d4">signaling pathway <span class="number">(471856)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebc89437716036a8a124a">disease pathway <span class="number">(392428)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553ebc72437716036a8a0f36">differentiation pathway <span class="number">(234089)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553ebc53cbe05d491bc316a6">metabolic pathway <span class="number">(214028)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553ebc19cbe05d491bc30c66">cancer pathways <span class="number">(277189)</span></a>
          <div class="tertiary">PMC</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-27T18:58:39-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB7B7653EBEE61000000000054237B;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB7B7653EBEE61000000000054237B" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="12" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="8A1A393946D25221_0091SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- 8A1A393946D25221_0091SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r451781 Wed, Nov 12 2014 02:48:02 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er451781+Wed%2C+Nov+12+2014+02%3A48%3A02'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>