
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="19" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A339953FF819100000000001F18F9" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="18" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="19" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="20" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="19" /></div>
            <div><h2 class="result_count">Results: 1801 to 1900 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>1801.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3942504/figure/F1/" ref="ordinalpos=1801&amp;ncbi_uid=5893267&amp;link_uid=PMC3942504" href="/pmc/articles/PMC3942504/"><img src="/pmc/articles/instance/3942504/bin/in-14-21-g001.gif" src-large="/pmc/articles/instance/3942504/bin/in-14-21-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3942504/" ref="ordinalpos=1801&amp;ncbi_uid=5893267&amp;link_uid=PMC3942504" image-link="/pmc/articles/PMC3942504/figure/F1/" class="imagepopup">Figure 1.  From: Insights into the Role of Follicular Helper T Cells in Autoimmunity. </a></p><div class="supp"><p class="details"><div>Molecular mechanisms of Bcl-6 expression in T cells. Bcl-6, the master regulator of T<sub>FH</sub> cell differentiation is controlled by a complex <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Co-stimulatory molecules such as CD28 and ICOS activate PI3K to induce Bcl-6 expression. PTEN, PHLPP2 inhibit Bcl-6 expression through interfering PI3K <span class="highlight" style="background-color:">signaling</span> and Foxo1 directly inhibits Bcl-6 expression. Various cytokines, such as IL-6, IL-21, IL-12, and IFN-γ induce Bcl-6 expression through JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> while high level of IL-2 in combination with IL-12 induces T-bet to inhibit Bcl-6. Blimp-1 and Bcl-6 is reciprocally regulating each other to make a decision of effector T cell fate between T<sub>FH</sub> and non-T<sub>FH</sub> effector cells. Some miRNA such as miR-17-92 induces Bcl-6 expression by interfering phosphatases, which inhibit PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> while miR-10a directly inhibits Bcl-6 expression.</div></p></div><div class="aux"><div class="resc">Hong-Jai Park, et al. Immune Netw. 2014 February;14(1):21-29.</div><p class="links"><a class="dblinks" href="/pubmed/24605077" ref="ordinalpos=1801&amp;ncbi_uid=5893267&amp;link_uid=24605077">Citation</a><a class="dblinks" href="/pmc/articles/PMC3942504/" ref="ordinalpos=1801&amp;ncbi_uid=5893267&amp;link_uid=PMC3942504">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1802.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1896004/figure/F4/" ref="ordinalpos=1802&amp;ncbi_uid=1023629&amp;link_uid=PMC1896004" href="/pmc/articles/PMC1896004/"><img src="/pmc/articles/instance/1896004/bin/gb-2007-8-4-r59-4.gif" src-large="/pmc/articles/instance/1896004/bin/gb-2007-8-4-r59-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1896004/" ref="ordinalpos=1802&amp;ncbi_uid=1023629&amp;link_uid=PMC1896004" image-link="/pmc/articles/PMC1896004/figure/F4/" class="imagepopup">Figure 4.  From: The innate immune repertoire in Cnidaria - ancestral complexity and stochastic gene loss. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways downstream of the Toll/TLRs. Pattern recognition, either indirectly or directly, by Toll/TLRs results in activation of NF-κB (vertebrates) or the Dif/Rel heterodimer (<i>Drosophila</i>) and thus transcription of appropriate immune response genes. At TRAF6, the classic Toll/TIR <span class="highlight" style="background-color:">pathway</span> (shown in the right branch) is linked to the JNK/p38 <span class="highlight" style="background-color:">pathway</span> (shown in the left branch) by the ECSIT protein, which acts as a regulator of MEKK-1 processing [35]. Components of both pathways downstream of Toll/TLRs are represented in the cnidarian datasets (Table 1). ECSIT may also act as a link between these and the TGF-b <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, since it forms complexes with BMP-<span class="highlight" style="background-color:">pathway</span> restricted Smads and is essential for regulation of the BMP-target gene <i>Tlx2 </i>[36]. All of the components of the TGF-b <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are also known from anthozoan cnidarians [13].</div></p></div><div class="aux"><div class="resc">David J Miller, et al. Genome Biol. 2007;8(4):R59-R59.</div><p class="links"><a class="dblinks" href="/pubmed/17437634" ref="ordinalpos=1802&amp;ncbi_uid=1023629&amp;link_uid=17437634">Citation</a><a class="dblinks" href="/pmc/articles/PMC1896004/" ref="ordinalpos=1802&amp;ncbi_uid=1023629&amp;link_uid=PMC1896004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1803.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3151525/figure/fig5/" ref="ordinalpos=1803&amp;ncbi_uid=3150054&amp;link_uid=PMC3151525" href="/pmc/articles/PMC3151525/"><img src="/pmc/articles/instance/3151525/bin/ECAM2011-140462.005.gif" src-large="/pmc/articles/instance/3151525/bin/ECAM2011-140462.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3151525/" ref="ordinalpos=1803&amp;ncbi_uid=3150054&amp;link_uid=PMC3151525" image-link="/pmc/articles/PMC3151525/figure/fig5/" class="imagepopup">Figure 5.  From: Dieckol from Ecklonia cava Regulates Invasion of Human Fibrosarcoma Cells and Modulates MMP-2 and MMP-9 Expression via NF-κB <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of dieckol on the NF-<i>κ</i>B and AP-1 cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and TIMP-1, -2 expressions in HT1080 cells. (a) Nucleus NF-<i>κ</i>B (p65 and p50) and I<i>κ</i>B (p-I<i>κ</i>B-<i>α</i>) protein expressions in the dieckol treated HT1080 cells. (b) (c-jun), MAPK <span class="highlight" style="background-color:">pathway</span> (ERK, JNK, p38). (c) Western blot analysis of TIMP-1 and TIMP-2.</div></p></div><div class="aux"><div class="resc">Chen Zhang, et al. Evid Based Complement Alternat Med. 2011;2011:140462.</div><p class="links"><a class="dblinks" href="/pubmed/21826183" ref="ordinalpos=1803&amp;ncbi_uid=3150054&amp;link_uid=21826183">Citation</a><a class="dblinks" href="/pmc/articles/PMC3151525/" ref="ordinalpos=1803&amp;ncbi_uid=3150054&amp;link_uid=PMC3151525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1804.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213687/figure/F2/" ref="ordinalpos=1804&amp;ncbi_uid=3401355&amp;link_uid=PMC3213687" href="/pmc/articles/PMC3213687/"><img src="/pmc/articles/instance/3213687/bin/1471-2105-12-81-2.gif" src-large="/pmc/articles/instance/3213687/bin/1471-2105-12-81-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213687/" ref="ordinalpos=1804&amp;ncbi_uid=3401355&amp;link_uid=PMC3213687" image-link="/pmc/articles/PMC3213687/figure/F2/" class="imagepopup">Figure 2.  From: Appearance frequency modulated gene set enrichment testing. </a></p><div class="supp"><p class="details"><div><b>Appearance frequencies of genes in the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The appearance frequency of each gene in the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is listed adjacent to the corresponding node in the map. Red color represents high appearance. Blue color represents low appearance. Notice that kinase genes (AKT) tend to have higher frequency compare with mTOR <span class="highlight" style="background-color:">pathway</span> specific genes (Raptor and Gβl).</div></p></div><div class="aux"><div class="resc">Jun Ma, et al. BMC Bioinformatics. 2011;12:81-81.</div><p class="links"><a class="dblinks" href="/pubmed/21418606" ref="ordinalpos=1804&amp;ncbi_uid=3401355&amp;link_uid=21418606">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213687/" ref="ordinalpos=1804&amp;ncbi_uid=3401355&amp;link_uid=PMC3213687">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1805.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3412102/figure/fig5/" ref="ordinalpos=1805&amp;ncbi_uid=4177733&amp;link_uid=PMC3412102" href="/pmc/articles/PMC3412102/"><img src="/pmc/articles/instance/3412102/bin/ECAM2012-871893.005.gif" src-large="/pmc/articles/instance/3412102/bin/ECAM2012-871893.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3412102/" ref="ordinalpos=1805&amp;ncbi_uid=4177733&amp;link_uid=PMC3412102" image-link="/pmc/articles/PMC3412102/figure/fig5/" class="imagepopup">Figure 5.  From: HRT, Herbal Formula, Induces G2/M Cell Cycle Arrest and Apoptosis via Suppressing Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>HRT inhibited cancer cell proliferation via suppressing Akt <span class="highlight" style="background-color:">signaling</span>. (a) The expression of phosphor-ERK or -Akt was examined in HCT116 cells treated with HRT in a time-dependent manner. HRT induced dephosphorylation of Akt not ERK at 24 h. (b) The effect of HRT on PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Phosphorylation levels of Akt, mTOR, and GSK3<i>β</i> were examined in HCT116 cells treated with LY294002 (10 <i>μ</i>M), HRT (300 <i>μ</i>g/mL) or cotreated with LY294002 and HRT for 24 h.</div></p></div><div class="aux"><div class="resc">Nam-Hui Yim, et al. Evid Based Complement Alternat Med. 2012;2012:871893.</div><p class="links"><a class="dblinks" href="/pubmed/22899960" ref="ordinalpos=1805&amp;ncbi_uid=4177733&amp;link_uid=22899960">Citation</a><a class="dblinks" href="/pmc/articles/PMC3412102/" ref="ordinalpos=1805&amp;ncbi_uid=4177733&amp;link_uid=PMC3412102">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1806.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979993/figure/F5/" ref="ordinalpos=1806&amp;ncbi_uid=6008779&amp;link_uid=PMC3979993" href="/pmc/articles/PMC3979993/"><img src="/pmc/articles/instance/3979993/bin/ijbsv10p0404g005.gif" src-large="/pmc/articles/instance/3979993/bin/ijbsv10p0404g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979993/" ref="ordinalpos=1806&amp;ncbi_uid=6008779&amp;link_uid=PMC3979993" image-link="/pmc/articles/PMC3979993/figure/F5/" class="imagepopup">Figure 5.  From: Andrographolide Suppress Tumor Growth by Inhibiting TLR4/NF-?B <span class="highlight" style="background-color:">Signaling</span> Activation in Insulinoma. </a></p><div class="supp"><p class="details"><div><b> Andro targets TLR4/NF-κB <span class="highlight" style="background-color:">signaling</span> in insulinoma. </b>Andro can significantly inhibited the protein expression of TLR4, MyD88, phosphorylated IκBα, p65, phosphorylated p65 (Ser536), and phosphorylated p65 (Ser276) of TLR4/NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in tumor tissues of RIP1-Tag2 mice compared with control groups. Immunohistochemical staining showed that Andro also inhibited the expression of TLR4 (B), MyD88 (C) and p65 (D) in tumor tissues of RIP1-Tag2 mice compared with control groups. Bar, 20 μm. ***<i>P</i> &lt; 0.001.</div></p></div><div class="aux"><div class="resc">Qian-Qian Zhang, et al. Int J Biol Sci. 2014;10(4):404-414.</div><p class="links"><a class="dblinks" href="/pubmed/24719558" ref="ordinalpos=1806&amp;ncbi_uid=6008779&amp;link_uid=24719558">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979993/" ref="ordinalpos=1806&amp;ncbi_uid=6008779&amp;link_uid=PMC3979993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1807.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3703268/figure/F6/" ref="ordinalpos=1807&amp;ncbi_uid=5074563&amp;link_uid=PMC3703268" href="/pmc/articles/PMC3703268/"><img src="/pmc/articles/instance/3703268/bin/1471-2164-14-440-6.gif" src-large="/pmc/articles/instance/3703268/bin/1471-2164-14-440-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3703268/" ref="ordinalpos=1807&amp;ncbi_uid=5074563&amp;link_uid=PMC3703268" image-link="/pmc/articles/PMC3703268/figure/F6/" class="imagepopup">Figure 6.  From: Large-scale integrative network-based analysis identifies common pathways disrupted by copy number alterations across cancers. </a></p><div class="supp"><p class="details"><div><b>Network view of TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> alterations in colorectal and ovarian cancer. </b><b>(A)</b> Network view of genes altered by copy number changes in colorectal cancer in the TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (diamond nodes) or genes directly interacting with TGF-beta <span class="highlight" style="background-color:">signaling</span> genes (circular nodes) based on the protein-protein interaction database. <b>(B)</b> The same network view for ovarian cancer. Size of node represents frequency of amplification or deletion in patient population. Color of node indicates whether gene is amplified (red), deleted (green), or unchanged (gray). Lines indicate interactions. Blue dotted line separates genes within the <span class="highlight" style="background-color:">pathway</span> from genes that interact with the <span class="highlight" style="background-color:">pathway</span> based on the protein-protein interaction database.</div></p></div><div class="aux"><div class="resc">Tae Hyun Hwang, et al. BMC Genomics. 2013;14:440-440.</div><p class="links"><a class="dblinks" href="/pubmed/23822816" ref="ordinalpos=1807&amp;ncbi_uid=5074563&amp;link_uid=23822816">Citation</a><a class="dblinks" href="/pmc/articles/PMC3703268/" ref="ordinalpos=1807&amp;ncbi_uid=5074563&amp;link_uid=PMC3703268">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1808.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2562016/figure/F5/" ref="ordinalpos=1808&amp;ncbi_uid=1769853&amp;link_uid=PMC2562016" href="/pmc/articles/PMC2562016/"><img src="/pmc/articles/instance/2639168/bin/btn451f5.gif" src-large="/pmc/articles/instance/2639168/bin/btn451f5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2562016/" ref="ordinalpos=1808&amp;ncbi_uid=1769853&amp;link_uid=PMC2562016" image-link="/pmc/articles/PMC2562016/figure/F5/" class="imagepopup">Fig. 5. From: Novel <span class="highlight" style="background-color:">pathway</span> compendium analysis elucidates mechanism of pro-angiogenic synthetic small molecule. </a></p><div class="supp"><p class="details"><div>Regulation of the TGF-β <span class="highlight" style="background-color:">pathway</span> by PNF1. (<b>A</b>) The core TGF-β <span class="highlight" style="background-color:">pathway</span> members, as rendered from KEGG (http://www.genome.jp/kegg/), that were significantly regulated by PNF1 over at least one timepoint (1–48 h). Shaded gene products represent PNF1-stimulated genes; gene products not shaded (white) were not significantly regulated by the drug but are required for <span class="highlight" style="background-color:">signaling</span> between PNF1-regulated genes. (<b>B</b>) Plots of the fold changes over time for the individual gene products in the TGF-β <span class="highlight" style="background-color:">pathway</span> that were significantly differentially expressed after PNF1 stimulation.</div></p></div><div class="aux"><div class="resc">Kristen A. Wieghaus, et al. Bioinformatics. 2008 October 15;24(20):2384-2390.</div><p class="links"><a class="dblinks" href="/pubmed/18718940" ref="ordinalpos=1808&amp;ncbi_uid=1769853&amp;link_uid=18718940">Citation</a><a class="dblinks" href="/pmc/articles/PMC2562016/" ref="ordinalpos=1808&amp;ncbi_uid=1769853&amp;link_uid=PMC2562016">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1809.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3529674/figure/F5/" ref="ordinalpos=1809&amp;ncbi_uid=4502398&amp;link_uid=PMC3529674" href="/pmc/articles/PMC3529674/"><img src="/pmc/articles/instance/3529674/bin/1756-8722-5-66-5.gif" src-large="/pmc/articles/instance/3529674/bin/1756-8722-5-66-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3529674/" ref="ordinalpos=1809&amp;ncbi_uid=4502398&amp;link_uid=PMC3529674" image-link="/pmc/articles/PMC3529674/figure/F5/" class="imagepopup">Figure 5.  From: MicroRNAs and Toll-like Receptor/Interleukin-1 Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>MyD88-independent TLR <span class="highlight" style="background-color:">signaling</span></b><b>.</b> Particular TLRs, like TLR3 and TLR4, can initiate MyD88-independent <span class="highlight" style="background-color:">signaling</span>. Receptor activation leads to the association of TRIF (TIR-domain-containing adaptor protein inducing IFN-β) protein. Specific to TLR4 <span class="highlight" style="background-color:">signaling</span>, the adaptor protein TRAM (TRIF-related adaptor molecule) also associates with the receptor to facilitate TRIF binding. As a result of TRIF binding, downstream NF-κB-directed transcription occurs along with IFN-α/β production. Although the exact <span class="highlight" style="background-color:">signaling</span> chronology is undetermined, it is known that two non-canonical IKKs (inhibitor of NF-κB kinase complex), IKKε and TBK1 (TANK-binding kinase 1) are activated. As a result, theses IKKs can initiate both NF-κB and IRF-3 (interferon-3) activity. In addition to IRF-3, IRF-7 is also known to be activated in the independent <span class="highlight" style="background-color:">pathway</span> leading to IFN-α/β production.</div></p></div><div class="aux"><div class="resc">Anthony Virtue, et al. J Hematol Oncol. 2012;5:66-66.</div><p class="links"><a class="dblinks" href="/pubmed/23078795" ref="ordinalpos=1809&amp;ncbi_uid=4502398&amp;link_uid=23078795">Citation</a><a class="dblinks" href="/pmc/articles/PMC3529674/" ref="ordinalpos=1809&amp;ncbi_uid=4502398&amp;link_uid=PMC3529674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1810.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2731830/figure/F1/" ref="ordinalpos=1810&amp;ncbi_uid=1999146&amp;link_uid=PMC2731830" href="/pmc/articles/PMC2731830/"><img src="/pmc/articles/instance/2731830/bin/nihms112315f1.gif" src-large="/pmc/articles/instance/2731830/bin/nihms112315f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2731830/" ref="ordinalpos=1810&amp;ncbi_uid=1999146&amp;link_uid=PMC2731830" image-link="/pmc/articles/PMC2731830/figure/F1/" class="imagepopup">Figure 1.  From: Endocytic regulation of Notch <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Overview of ligand and receptor endocytosis in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the signal-sending cell (top), endocytosis of Notch ligands (violet) is needed for their productive <span class="highlight" style="background-color:">signaling</span> activity. Ligand endocytosis generates the physical forces needed to dissociate and activate Notch receptors in nascent invaginating endocytic vesicles, and it might also concentrate ligand molecules through their trafficking and/or post-translational modification in the recycling compartment. In the signal-receiving cell (bottom), Notch receptors (blue) are internalized into endocytic vesicles and subsequently routed to early endosomes. Ligand-activated Notch is sorted into multivesicular bodies and thence to lysosomes for degradation; non-ligand-activated Notch can undergo trafficking to the cell surface via recycling endosomes. The identities of various endocytic factors involved in these trafficking steps are shown near their approximate sites of activity; those that promote <span class="highlight" style="background-color:">signaling</span> are indicated in green whereas those involved in <span class="highlight" style="background-color:">signaling</span> down-regulation are indicated in red. The γ-secretase-mediated cleavage of Notch that generates the active <span class="highlight" style="background-color:">signaling</span> fragment NICD can occur at different points in the endosomal trafficking <span class="highlight" style="background-color:">pathway</span> (see text for details).</div></p></div><div class="aux"><div class="resc">Mark E. Fortini, et al. Curr Opin Genet Dev. ;19(4):323-328.</div><p class="links"><a class="dblinks" href="/pubmed/19447603" ref="ordinalpos=1810&amp;ncbi_uid=1999146&amp;link_uid=19447603">Citation</a><a class="dblinks" href="/pmc/articles/PMC2731830/" ref="ordinalpos=1810&amp;ncbi_uid=1999146&amp;link_uid=PMC2731830">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1811.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3078531/figure/F5/" ref="ordinalpos=1811&amp;ncbi_uid=3390055&amp;link_uid=PMC3078531" href="/pmc/articles/PMC3078531/"><img src="/pmc/articles/instance/3078531/bin/nihms280576f5.gif" src-large="/pmc/articles/instance/3078531/bin/nihms280576f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3078531/" ref="ordinalpos=1811&amp;ncbi_uid=3390055&amp;link_uid=PMC3078531" image-link="/pmc/articles/PMC3078531/figure/F5/" class="imagepopup">Fig. 5. Proposed ROS-dependent <span class="highlight" style="background-color:">signaling</span> pathways for GATA4 activation.  From: Cell signalling pathways for the regulation of GATA4 transcription factor: Implications for cell growth and apoptosis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> stimuli trigger the generation of ROS, which in turn (i) activate MEK, phosphorylating ERK and (ii) activate RhoA/ROCK <span class="highlight" style="background-color:">pathway</span>, facilitating ERK nuclear translocation. In the nucleus, ERK then phosphorylates and activates GATA4.</div></p></div><div class="aux"><div class="resc">Yuichiro J. Suzuki. Cell Signal. ;23(7):1094-1099.</div><p class="links"><a class="dblinks" href="/pubmed/21376121" ref="ordinalpos=1811&amp;ncbi_uid=3390055&amp;link_uid=21376121">Citation</a><a class="dblinks" href="/pmc/articles/PMC3078531/" ref="ordinalpos=1811&amp;ncbi_uid=3390055&amp;link_uid=PMC3078531">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1812.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1435660/figure/f8/" ref="ordinalpos=1812&amp;ncbi_uid=575394&amp;link_uid=PMC1435660" href="/pmc/articles/PMC1435660/"><img src="/pmc/articles/instance/1435660/bin/75f8c.gif" src-large="/pmc/articles/instance/1435660/bin/75f8c.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1435660/" ref="ordinalpos=1812&amp;ncbi_uid=575394&amp;link_uid=PMC1435660" image-link="/pmc/articles/PMC1435660/figure/f8/" class="imagepopup">Figure 8. From: Hereditary Hemorrhagic Telangiectasia, a Vascular Dysplasia Affecting the TGF-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Hypothetical model for HHT. Model representing TGF-β receptor complex in healthy (non-HHT) endothelial cells as compared to HHT1 and HHT2 endothelial cells. In healthy endothelial cells (non-HHT), ALK1 and endoglin are cooperating in the TGF-β/Smad <span class="highlight" style="background-color:">pathway</span>, and endoglin levels are maintained to meet the physiological needs of the endothelial cell. However, in HHT1 and HHT2 cells, either endoglin or ALK1 fail in the cooperative <span class="highlight" style="background-color:">signaling</span>. As a result, endoglin expression decreases below a critical threshold leading to impaired TGF-β <span class="highlight" style="background-color:">signaling</span>, and abnormal cytoskeleton and tube formation in HHT endothelial cells. These altered endothelial cell functions may explain the HHT phenotype. Adapted from Fernández-L et al.75</div></p></div><div class="aux"><div class="resc">Africa Fernández-L, et al. Clin Med Res. 2006 March;4(1):66-78.</div><p class="links"><a class="dblinks" href="/pubmed/16595794" ref="ordinalpos=1812&amp;ncbi_uid=575394&amp;link_uid=16595794">Citation</a><a class="dblinks" href="/pmc/articles/PMC1435660/" ref="ordinalpos=1812&amp;ncbi_uid=575394&amp;link_uid=PMC1435660">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1813.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3609045/figure/F5/" ref="ordinalpos=1813&amp;ncbi_uid=5662678&amp;link_uid=PMC3609045" href="/pmc/articles/PMC3609045/"><img src="/pmc/articles/instance/3609045/bin/prio-7-2-g5.gif" src-large="/pmc/articles/instance/3609045/bin/prio-7-2-g5.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3609045/" ref="ordinalpos=1813&amp;ncbi_uid=5662678&amp;link_uid=PMC3609045" image-link="/pmc/articles/PMC3609045/figure/F5/" class="imagepopup">Figure. From: Dysregulation of neural calcium <span class="highlight" style="background-color:">signaling</span> in Alzheimer disease, bipolar disorder and schizophrenia. </a></p><div class="supp"><p class="details"><div><b>Figure 5.</b> Dysregulation of modulatory Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> pathways contribute to neural diseases. Hypofunction of the NMDA receptor results in a reduction in the gene transcription necessary to maintain the GABAergic phenotype resulting in schizophrenia. Hyperfunction of the Ins(1,4,5)<i>P</i><sub>3</sub>/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has been linked to both bipolar disorder and Alzheimer disease.</div></p></div><div class="aux"><div class="resc">Michael J. Berridge. Prion. 2013 January 1;7(1):2-13.</div><p class="links"><a class="dblinks" href="/pubmed/22895098" ref="ordinalpos=1813&amp;ncbi_uid=5662678&amp;link_uid=22895098">Citation</a><a class="dblinks" href="/pmc/articles/PMC3609045/" ref="ordinalpos=1813&amp;ncbi_uid=5662678&amp;link_uid=PMC3609045">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1814.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1142443/figure/fig5/" ref="ordinalpos=1814&amp;ncbi_uid=491508&amp;link_uid=PMC1142443" href="/pmc/articles/PMC1142443/"><img src="/pmc/articles/instance/1142443/bin/zmk0060571890005.gif" src-large="/pmc/articles/instance/1142443/bin/zmk0060571890005.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1142443/" ref="ordinalpos=1814&amp;ncbi_uid=491508&amp;link_uid=PMC1142443" image-link="/pmc/articles/PMC1142443/figure/fig5/" class="imagepopup">Figure 5. From: The Conserved Mec1/Rad53 Nuclear Checkpoint <span class="highlight" style="background-color:">Pathway</span> Regulates Mitochondrial DNA Copy Number in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Proposed model for how <span class="highlight" style="background-color:">signaling</span> via the Mec1/Rad53 <span class="highlight" style="background-color:">pathway</span> influences mtDNA copy number. It is established that the Mec1/Rad53 <span class="highlight" style="background-color:">pathway</span> is responsive to nuclear DNA damage, nuclear replication stress, and S-phase signals that result in increased dNTPs needed to allow replication and repair to proceed. Based on the present study, we propose that <span class="highlight" style="background-color:">signaling</span> through this <span class="highlight" style="background-color:">pathway</span> can also regulate mtDNA copy number, primarily (although not necessarily exclusively; see <i>Discussion</i>) through changes in RNR activity and up-regulation or alteration of cellular dNTP pools. Arrows denote points in the <span class="highlight" style="background-color:">pathway</span> that are activated by the indicated stimulus (for example, deletion of <i>SML1</i> results in increased activity of RNR). Several conditions tested in this work that resulted in increased mtDNA copy number and how they influence this <span class="highlight" style="background-color:">pathway</span> are shown, including deletion of <i>RRM3</i> (<i>rrm3</i>Δ) or <i>SML1</i> (<i>sml1</i>Δ) and overexpression of <i>RNR1</i>. Mitochondrial Pif1p is drawn at the end of this <span class="highlight" style="background-color:">pathway</span> because its petite-induction phenotype is rescued by overexpression of <i>RNR1</i> (O'Rourke et al., 2005), deletion of <i>RRM3</i> (O'Rourke et al., 2005), or deletion of <i>SML1</i> (Table 1), indicating that a process it is involved in is responsive to alterations in dNTP pools. Finally, Abf2p is shown as a separate <span class="highlight" style="background-color:">pathway</span> that can increase mtDNA copy number. That upstream signals which regulate the abundance (or activity) of Abf2p to regulate mtDNA copy number are currently unknown is indicated by a question mark.</div></p></div><div class="aux"><div class="resc">Sean D. Taylor, et al. Mol Biol Cell. 2005 June;16(6):3010-3018.</div><p class="links"><a class="dblinks" href="/pubmed/15829566" ref="ordinalpos=1814&amp;ncbi_uid=491508&amp;link_uid=15829566">Citation</a><a class="dblinks" href="/pmc/articles/PMC1142443/" ref="ordinalpos=1814&amp;ncbi_uid=491508&amp;link_uid=PMC1142443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1815.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4020487/figure/f7-ijmm-33-05-1097/" ref="ordinalpos=1815&amp;ncbi_uid=6132432&amp;link_uid=PMC4020487" href="/pmc/articles/PMC4020487/"><img src="/pmc/articles/instance/4020487/bin/IJMM-33-05-1097-g06.gif" src-large="/pmc/articles/instance/4020487/bin/IJMM-33-05-1097-g06.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4020487/" ref="ordinalpos=1815&amp;ncbi_uid=6132432&amp;link_uid=PMC4020487" image-link="/pmc/articles/PMC4020487/figure/f7-ijmm-33-05-1097/" class="imagepopup">Figure 7.  From: Activated Wnt <span class="highlight" style="background-color:">signaling</span> induces myofibroblast differentiation of mesenchymal stem cells, contributing to pulmonary fibrosis. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> regulates the differentiation of mesenchymal stem cells (MSCs). (A) The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was highly activated after lung injury. The expression of Wnt <span class="highlight" style="background-color:">signaling</span> components β-catenin and MMP-2 were increased in each group. The activated Wnt <span class="highlight" style="background-color:">signaling</span> may determine the differentiation of MSCs <i>in vivo</i>. (B and C) MSCs were cultured until they reached confluence and then treated with Wnt3α and DKK1 for 14 days. Activation of Wnt <span class="highlight" style="background-color:">signaling</span> induces MSCs to differentiate into myofibroblasts which was inhibited by treatment with DKK1. (B) Immunofluorescent analysis of β-catenin, vimentin and α-SMA expression in MSCs. Scale bar, 50 μm. (C) Expression of β-catenin, vimentin and α-SMA was evaluated in whole cell lysates by western blotting. β-actin was used as the control.</div></p></div><div class="aux"><div class="resc">ZHAORUI SUN, et al. Int J Mol Med. 2014 May;33(5):1097-1109.</div><p class="links"><a class="dblinks" href="/pubmed/24573542" ref="ordinalpos=1815&amp;ncbi_uid=6132432&amp;link_uid=24573542">Citation</a><a class="dblinks" href="/pmc/articles/PMC4020487/" ref="ordinalpos=1815&amp;ncbi_uid=6132432&amp;link_uid=PMC4020487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1816.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2423168/figure/f7/" ref="ordinalpos=1816&amp;ncbi_uid=1476366&amp;link_uid=PMC2423168" href="/pmc/articles/PMC2423168/"><img src="/pmc/articles/instance/2423168/bin/zmb0100874790007.gif" src-large="/pmc/articles/instance/2423168/bin/zmb0100874790007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2423168/" ref="ordinalpos=1816&amp;ncbi_uid=1476366&amp;link_uid=PMC2423168" image-link="/pmc/articles/PMC2423168/figure/f7/" class="imagepopup">FIG. 7. From: Cdc1p Is an Endoplasmic Reticulum-Localized Putative Lipid Phosphatase That Affects Golgi Inheritance and Actin Polarization by Activating Ca2+ <span class="highlight" style="background-color:">Signaling</span>   . </a></p><div class="supp"><p class="details"><div>A model for Cdc1p function and the link between Cdc1p and Golgi inheritance. (a) Cdc1p is presumed to be a lipid phosphatase, and we suggest that Per1p is needed to generate the substrate for Cdc1p. According to this model, inactivating Cdc1p causes its substrate to accumulate to toxic levels and also causes accumulation of a nontoxic precursor lipid. When Per1p is inactivated, only the nontoxic precursor accumulates, so a <i>per1</i>Δ mutation suppresses the lethality of <i>cdc1</i> mutations. This nontoxic precursor may correspond to the unidentified lipid that was visualized in the experiment shown in Fig. 6. (b) In <i>cdc1</i> cells, accumulation of the toxic phosphorylated lipid activates both Mpk1p- and Cch1p-dependent Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>. The Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> activates calcineurin. In a calcineurin-independent <span class="highlight" style="background-color:">pathway</span>, strong Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> depolarizes actin. We speculate that strong Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> also inhibits Golgi tethering. These two effects of strong Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> synergize to cause a pronounced defect in Golgi inheritance.</div></p></div><div class="aux"><div class="resc">Eugene Losev, et al. Mol Cell Biol. 2008 May;28(10):3336-3343.</div><p class="links"><a class="dblinks" href="/pubmed/18332110" ref="ordinalpos=1816&amp;ncbi_uid=1476366&amp;link_uid=18332110">Citation</a><a class="dblinks" href="/pmc/articles/PMC2423168/" ref="ordinalpos=1816&amp;ncbi_uid=1476366&amp;link_uid=PMC2423168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1817.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872207/figure/f3/" ref="ordinalpos=1817&amp;ncbi_uid=2370744&amp;link_uid=PMC2872207" href="/pmc/articles/PMC2872207/"><img src="/pmc/articles/instance/2872207/bin/fig-3.gif" src-large="/pmc/articles/instance/2872207/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872207/" ref="ordinalpos=1817&amp;ncbi_uid=2370744&amp;link_uid=PMC2872207" image-link="/pmc/articles/PMC2872207/figure/f3/" class="imagepopup">FIG. 3. From: Proinflammatory Stem Cell <span class="highlight" style="background-color:">Signaling</span> in Cardiac Ischemia. </a></p><div class="supp"><p class="details"><div><b>The Jak/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Signal activation occurs through binding of IL-6, CT-1, or LIF. Increased gene expression occurs after STAT is phosphorylated, dimerizes, and translocates to the nucleus. Inhibitory steps include the SOCS, SHP2, PIAS, and elongin proteins.</div></p></div><div class="aux"><div class="resc">Jeremy L. Herrmann, et al. Antioxid Redox Signal. 2009 August;11(8):1883-1896.</div><p class="links"><a class="dblinks" href="/pubmed/19187005" ref="ordinalpos=1817&amp;ncbi_uid=2370744&amp;link_uid=19187005">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872207/" ref="ordinalpos=1817&amp;ncbi_uid=2370744&amp;link_uid=PMC2872207">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1818.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2374695/figure/F1/" ref="ordinalpos=1818&amp;ncbi_uid=1331108&amp;link_uid=PMC2374695" href="/pmc/articles/PMC2374695/"><img src="/pmc/articles/instance/2374695/bin/nihms46978f1.gif" src-large="/pmc/articles/instance/2374695/bin/nihms46978f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2374695/" ref="ordinalpos=1818&amp;ncbi_uid=1331108&amp;link_uid=PMC2374695" image-link="/pmc/articles/PMC2374695/figure/F1/" class="imagepopup">Figure 1. The H2O2 stress <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that activates Spc1 MAPK in S. pombe.  From: Glycolytic enzyme GAPDH promotes peroxide stress <span class="highlight" style="background-color:">signaling</span> through multistep phosphorelay to a MAPK cascade. </a></p><div class="supp"><p class="details"><div>See text for details.</div></p></div><div class="aux"><div class="resc">Susumu Morigasaki, et al. Mol Cell. ;30(1):108-113.</div><p class="links"><a class="dblinks" href="/pubmed/18406331" ref="ordinalpos=1818&amp;ncbi_uid=1331108&amp;link_uid=18406331">Citation</a><a class="dblinks" href="/pmc/articles/PMC2374695/" ref="ordinalpos=1818&amp;ncbi_uid=1331108&amp;link_uid=PMC2374695">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1819.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3710532/figure/F1/" ref="ordinalpos=1819&amp;ncbi_uid=6324778&amp;link_uid=PMC3710532" href="/pmc/articles/PMC3710532/"><img src="/pmc/articles/instance/3710532/bin/nihms474423f1.gif" src-large="/pmc/articles/instance/3710532/bin/nihms474423f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3710532/" ref="ordinalpos=1819&amp;ncbi_uid=6324778&amp;link_uid=PMC3710532" image-link="/pmc/articles/PMC3710532/figure/F1/" class="imagepopup">Figure 1.  From: Frequent mutation of the PI3K <span class="highlight" style="background-color:">pathway</span> in head and neck cancer defines predictive biomarkers. </a></p><div class="supp"><p class="details"><div>Mutations in oncogenic <span class="highlight" style="background-color:">signaling</span> pathways in HNSCC. (A) Mutation rates of the major mitogenic pathways (the PI3K <span class="highlight" style="background-color:">pathway</span>, the MAPK <span class="highlight" style="background-color:">pathway</span> and the JAK/STAT <span class="highlight" style="background-color:">pathway</span>) in 151 HNSCC patient tumors determined by whole exome sequencing. Components of each <span class="highlight" style="background-color:">pathway</span> examined are displayed underneath each pie chart. (B) Bar graph detailing the number of mutations (dark bars) of each particular component of the PI3K <span class="highlight" style="background-color:">pathway</span> as well as the number of HNSCC tumors harboring these mutations (grey bars). (C) PI3K <span class="highlight" style="background-color:">pathway</span>-mutated HNSCC tumors have higher of non-synonymous mutations and (D) cancer gene mutations when compared to HNSCC tumors without any PI3K <span class="highlight" style="background-color:">pathway</span> mutations. Bar graph representing the average number of non-synonymous mutations per tumor (C) and the average number of cancer gene mutations (D) in 151 HNSCC tumors. Statistical significance was calculated by Fisher’s Exact test, P&lt;0.0001 (N=151). (E) Graphical representation of the number of HNSCC tumors with mutation of multiple components of the PI3K, MAPK and the JAK/STAT pathways, respectively.</div></p></div><div class="aux"><div class="resc">Vivian Wai Yan Lui, et al. Cancer Discov. ;3(7):761-769.</div><p class="links"><a class="dblinks" href="/pubmed/23619167" ref="ordinalpos=1819&amp;ncbi_uid=6324778&amp;link_uid=23619167">Citation</a><a class="dblinks" href="/pmc/articles/PMC3710532/" ref="ordinalpos=1819&amp;ncbi_uid=6324778&amp;link_uid=PMC3710532">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1820.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2853013/figure/F3/" ref="ordinalpos=1820&amp;ncbi_uid=2549615&amp;link_uid=PMC2853013" href="/pmc/articles/PMC2853013/"><img src="/pmc/articles/instance/2853013/bin/nihms184568f3.gif" src-large="/pmc/articles/instance/2853013/bin/nihms184568f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2853013/" ref="ordinalpos=1820&amp;ncbi_uid=2549615&amp;link_uid=PMC2853013" image-link="/pmc/articles/PMC2853013/figure/F3/" class="imagepopup">Figure 3. VEGF <span class="highlight" style="background-color:">signaling</span> in cultured primary MNs.  From: Vascular endothelial growth factor prevents G93A-SOD1-induced motor neuron degeneration. </a></p><div class="supp"><p class="details"><div>Western blotting of primary MNs treated with VEGF. (A) Time course of Akt and p44/42 MAPK phosphorylation in response to VEGF (250 ng/ml) treatment. (B) Pretreatment (1 h) with inhibitors to the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (LY294002; 20µM), p44/42 MAPK <span class="highlight" style="background-color:">pathway</span> (U0126; 20µM), and VEGFR2 (SU1948; 20µM) prevent activation of the respective <span class="highlight" style="background-color:">signaling</span> pathways in response to VEGF treatment.</div></p></div><div class="aux"><div class="resc">J. Simon Lunn, et al. Dev Neurobiol. ;69(13):871-884.</div><p class="links"><a class="dblinks" href="/pubmed/19672955" ref="ordinalpos=1820&amp;ncbi_uid=2549615&amp;link_uid=19672955">Citation</a><a class="dblinks" href="/pmc/articles/PMC2853013/" ref="ordinalpos=1820&amp;ncbi_uid=2549615&amp;link_uid=PMC2853013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1821.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634388/figure/F1/" ref="ordinalpos=1821&amp;ncbi_uid=1710941&amp;link_uid=PMC2634388" href="/pmc/articles/PMC2634388/"><img src="/pmc/articles/instance/2634388/bin/psb0310_0831_fig001.gif" src-large="/pmc/articles/instance/2634388/bin/psb0310_0831_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634388/" ref="ordinalpos=1821&amp;ncbi_uid=1710941&amp;link_uid=PMC2634388" image-link="/pmc/articles/PMC2634388/figure/F1/" class="imagepopup">Figure 1.  From: Chitin <span class="highlight" style="background-color:">signaling</span> and plant disease resistance. </a></p><div class="supp"><p class="details"><div>Chitin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> vs. flagellin-and EF-Tu-mediated pathways. Increasing evidence suggests these pathways share a common, downstream <span class="highlight" style="background-color:">pathway</span> to mediate plant innate immunity against different pathogens, although the initial stages are different, as partially reflected by different receptors. Blue circle: LysM motif, Yellow column: leucine-rich repeats; Red oval: kinase domain.</div></p></div><div class="aux"><div class="resc">Jinrong Wan, et al. Plant Signal Behav. 2008 October;3(10):831-833.</div><p class="links"><a class="dblinks" href="/pubmed/19704513" ref="ordinalpos=1821&amp;ncbi_uid=1710941&amp;link_uid=19704513">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634388/" ref="ordinalpos=1821&amp;ncbi_uid=1710941&amp;link_uid=PMC2634388">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1822.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3610942/figure/pgen-1003414-g005/" ref="ordinalpos=1822&amp;ncbi_uid=4777409&amp;link_uid=PMC3610942" href="/pmc/articles/PMC3610942/"><img src="/pmc/articles/instance/3610942/bin/pgen.1003414.g005.gif" src-large="/pmc/articles/instance/3610942/bin/pgen.1003414.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3610942/" ref="ordinalpos=1822&amp;ncbi_uid=4777409&amp;link_uid=PMC3610942" image-link="/pmc/articles/PMC3610942/figure/pgen-1003414-g005/" class="imagepopup">Figure 5. Glucose modulates ribosome-related modules through glucose response <span class="highlight" style="background-color:">pathway</span>..  From: Statistical Analysis Reveals Co-Expression Patterns of Many Pairs of Genes in Yeast Are Jointly Regulated by Interacting Loci. </a></p><div class="supp"><p class="details"><div>The regulatory loci in the modules contain genes involved in the glucose response <span class="highlight" style="background-color:">pathway</span> (red nodes). This indicates how the glucose condition modulates the co-expression pattern between ribosome genes (blue nodes) and glucose <span class="highlight" style="background-color:">signaling</span> or metabolic genes (green nodes).</div></p></div><div class="aux"><div class="resc">Lin Wang, et al. PLoS Genet. 2013 March;9(3):e1003414.</div><p class="links"><a class="dblinks" href="/pubmed/23555313" ref="ordinalpos=1822&amp;ncbi_uid=4777409&amp;link_uid=23555313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3610942/" ref="ordinalpos=1822&amp;ncbi_uid=4777409&amp;link_uid=PMC3610942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1823.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3572020/figure/pone-0056224-g004/" ref="ordinalpos=1823&amp;ncbi_uid=4631234&amp;link_uid=PMC3572020" href="/pmc/articles/PMC3572020/"><img src="/pmc/articles/instance/3572020/bin/pone.0056224.g004.gif" src-large="/pmc/articles/instance/3572020/bin/pone.0056224.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3572020/" ref="ordinalpos=1823&amp;ncbi_uid=4631234&amp;link_uid=PMC3572020" image-link="/pmc/articles/PMC3572020/figure/pone-0056224-g004/" class="imagepopup">Figure 4. LMWF inhibited MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in kidney after IR..  From: Low Molecular Weight Fucoidan against Renal Ischemia-Reperfusion Injury via Inhibition of the MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Expression levels of ERK, p38, JNK, CREB and c-fos of kidneys after reperfusion for 2 h were detected by Western blot analysis. Representative blotting (<i>left</i>) and quantification of protein levels (<i>right</i>) are shown. Mean ± SEM. n = 3; #<i>P</i>&lt;0.05, ##<i>P</i>&lt;0.01, ###<i>P</i>&lt;0.001 vs. sham, *<i>P</i>&lt;0.05, **<i>P</i>&lt;0.01 vs. saline treated IR group.</div></p></div><div class="aux"><div class="resc">Jihui Chen, et al. PLoS One. 2013;8(2):e56224.</div><p class="links"><a class="dblinks" href="/pubmed/23418539" ref="ordinalpos=1823&amp;ncbi_uid=4631234&amp;link_uid=23418539">Citation</a><a class="dblinks" href="/pmc/articles/PMC3572020/" ref="ordinalpos=1823&amp;ncbi_uid=4631234&amp;link_uid=PMC3572020">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1824.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2755866/figure/F9/" ref="ordinalpos=1824&amp;ncbi_uid=2009980&amp;link_uid=PMC2755866" href="/pmc/articles/PMC2755866/"><img src="/pmc/articles/instance/2755866/bin/zbc0360984750009.gif" src-large="/pmc/articles/instance/2755866/bin/zbc0360984750009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2755866/" ref="ordinalpos=1824&amp;ncbi_uid=2009980&amp;link_uid=PMC2755866" image-link="/pmc/articles/PMC2755866/figure/F9/" class="imagepopup">FIGURE 9. From: A Non-canonical MEK/ERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulates Autophagy via Regulating Beclin 1. </a></p><div class="supp"><p class="details"><div><b>A diagram for the non-canonical MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating autophagy via regulating Beclin 1.</b> Basal MEK/ERK activity maintains the activity of mTORC1 and mTORC2 by protecting them from being disassembled and thus maintains a basal Beclin 1 level incapable of triggering autophagic response. The non-canonical MEK/ERK module is activated by upstream regulator AMPK upon autophagy stimuli. Activation of MEK/ERK up-regulates the Beclin 1 level through disassembling mTORC1 and/or mTORC2 with a significantly sustained and elevated Beclin 1 level causing cytodestructive autophagy and moderately elevated Beclin 1 causing cytoprotective autophagy. Disassembling mTORC1 by MEK/ERK depends on TSC, whereas disassembling mTORC2 does not depend on TSC2. Although ERK is implicated in the regulation of autophagy, MEK can bypass ERK to trigger autophagic response.</div></p></div><div class="aux"><div class="resc">Jianrong Wang, et al. J Biol Chem. 2009 August 7;284(32):21412-21424.</div><p class="links"><a class="dblinks" href="/pubmed/19520853" ref="ordinalpos=1824&amp;ncbi_uid=2009980&amp;link_uid=19520853">Citation</a><a class="dblinks" href="/pmc/articles/PMC2755866/" ref="ordinalpos=1824&amp;ncbi_uid=2009980&amp;link_uid=PMC2755866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1825.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3042311/figure/f8/" ref="ordinalpos=1825&amp;ncbi_uid=3627977&amp;link_uid=PMC3042311" href="/pmc/articles/PMC3042311/"><img src="/pmc/articles/instance/3042311/bin/fig-8.gif" src-large="/pmc/articles/instance/3042311/bin/fig-8.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3042311/" ref="ordinalpos=1825&amp;ncbi_uid=3627977&amp;link_uid=PMC3042311" image-link="/pmc/articles/PMC3042311/figure/f8/" class="imagepopup">FIG. 8. From: Reactive Oxygen Species-Dependent Cell <span class="highlight" style="background-color:">Signaling</span> Regulates the Mosquito Immune Response to Plasmodium falciparum. </a></p><div class="supp"><p class="details"><div><b>Proposed model of ROS-mediated insulin <span class="highlight" style="background-color:">signaling</span> in mosquitoes.</b> <b>(1)</b> Human insulin signals in the mosquito midgut, <b>(2)</b> inducing the phosphorylation of MEK and Akt. <b>(3)</b> Activated MEK and Akt phosphorylate downstream effectors such as ERK and FOXO. <b>(4)</b> This <span class="highlight" style="background-color:">signaling</span> leads to increased ROS, which can positively feed back into the insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, increasing the phosphorylation of downstream effectors such as ERK and FOXO. <b>(5)</b> Ultimately, insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span>, acting in part through increased ROS levels, leads to a decrease in the mosquito immune response and <b>(6)</b> a subsequent increase in parasite development in the mosquito midgut epithelium.</div></p></div><div class="aux"><div class="resc">Win Surachetpong, et al. Antioxid Redox Signal. 2011 March 15;14(6):943-955.</div><p class="links"><a class="dblinks" href="/pubmed/21126166" ref="ordinalpos=1825&amp;ncbi_uid=3627977&amp;link_uid=21126166">Citation</a><a class="dblinks" href="/pmc/articles/PMC3042311/" ref="ordinalpos=1825&amp;ncbi_uid=3627977&amp;link_uid=PMC3042311">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1826.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3784243/figure/F1/" ref="ordinalpos=1826&amp;ncbi_uid=5334991&amp;link_uid=PMC3784243" href="/pmc/articles/PMC3784243/"><img src="/pmc/articles/instance/3784243/bin/nihms-516261-f0001.gif" src-large="/pmc/articles/instance/3784243/bin/nihms-516261-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3784243/" ref="ordinalpos=1826&amp;ncbi_uid=5334991&amp;link_uid=PMC3784243" image-link="/pmc/articles/PMC3784243/figure/F1/" class="imagepopup">Figure 1. Effect of PD-1 <span class="highlight" style="background-color:">signaling</span> in T cells.  From: Role of the PD-1 <span class="highlight" style="background-color:">Pathway</span> in the Immune Response. </a></p><div class="supp"><p class="details"><div>PD-1 <span class="highlight" style="background-color:">signaling</span> dephosphorylates proximal <span class="highlight" style="background-color:">signaling</span> molecules and augments PTEN expression, inhibiting PI3K and AKT activation. The consequences include decreased T cell proliferation, cytokine production and cell survival. PD-1 inhibition can be overcome by strong TCR <span class="highlight" style="background-color:">signaling</span>, CD28 <span class="highlight" style="background-color:">signaling</span> or IL-2 <span class="highlight" style="background-color:">signaling</span> (not shown). SHP2, protein tyrosine phosphatase; PI3K, phosphatidylinositol 3-kinase; PTEN, phosphatase and tensin homolog; AKT, serine/threonine protein kinase; PIP3, Phosphatidylinositol (3,4,5)-triphosphate.</div></p></div><div class="aux"><div class="resc">Leonardo V. Riella, et al. Am J Transplant. 2012 October;12(10):2575-2587.</div><p class="links"><a class="dblinks" href="/pubmed/22900886" ref="ordinalpos=1826&amp;ncbi_uid=5334991&amp;link_uid=22900886">Citation</a><a class="dblinks" href="/pmc/articles/PMC3784243/" ref="ordinalpos=1826&amp;ncbi_uid=5334991&amp;link_uid=PMC3784243">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1827.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3874981/figure/fig3/" ref="ordinalpos=1827&amp;ncbi_uid=5699710&amp;link_uid=PMC3874981" href="/pmc/articles/PMC3874981/"><img src="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.003.gif" src-large="/pmc/articles/instance/3874981/bin/SCIENTIFICA2013-684736.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3874981/" ref="ordinalpos=1827&amp;ncbi_uid=5699710&amp;link_uid=PMC3874981" image-link="/pmc/articles/PMC3874981/figure/fig3/" class="imagepopup">Figure 3.  From: Review of <span class="highlight" style="background-color:">Signaling</span> Pathways Governing MSC Osteogenic and Adipogenic Differentiation. </a></p><div class="supp"><p class="details"><div>Schematic of <i>β</i>-catenin dependent and independent Wnt <span class="highlight" style="background-color:">signaling</span> pathways. Wnt <span class="highlight" style="background-color:">signaling</span> transduction occurs via <i>β</i>-catenin dependent or <i>β</i>-catenin independent <span class="highlight" style="background-color:">signaling</span> pathways. In <i>β</i>-catenin dependent <span class="highlight" style="background-color:">signaling</span>, extracellular Wnt ligands bind to the LRP5-Frizzled (Frz) complex to activate intracellular disheveled (DSH). This subsequently inhibits the intracellular complex comprised of axin, glycogen synthase kinase 3 (GSK3), and adenomatosis polyposis coli (APC) protein. This inhibits the cytosolic degradation of <i>β</i>-catenin, which accumulates and is free to enter the nucleus to heterodimerize with lymphoid enhancer-binding factor/T cell factor (LEF/TCF1) and mediate effects of gene transcription. Under the <i>β</i>-catenin independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, a similar transmembrane complex forms between Wnt, Frz, DSH, and Ror2 and activates secondary messengers.</div></p></div><div class="aux"><div class="resc">Aaron W. James. Scientifica (Cairo). 2013;2013:684736.</div><p class="links"><a class="dblinks" href="/pubmed/24416618" ref="ordinalpos=1827&amp;ncbi_uid=5699710&amp;link_uid=24416618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3874981/" ref="ordinalpos=1827&amp;ncbi_uid=5699710&amp;link_uid=PMC3874981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1828.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3748137/figure/F5/" ref="ordinalpos=1828&amp;ncbi_uid=6447875&amp;link_uid=PMC3748137" href="/pmc/articles/PMC3748137/"><img src="/pmc/articles/instance/3748137/bin/nihms-504168-f0005.gif" src-large="/pmc/articles/instance/3748137/bin/nihms-504168-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3748137/" ref="ordinalpos=1828&amp;ncbi_uid=6447875&amp;link_uid=PMC3748137" image-link="/pmc/articles/PMC3748137/figure/F5/" class="imagepopup">Figure 5. Insulin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Alcohol Consumption Improves Insulin <span class="highlight" style="background-color:">Signaling</span> in the Myocardium. </a></p><div class="supp"><p class="details"><div>The effects of insulin on glucose uptake and insulin sensitivity presented here are discussed in detail in the text. PIP3 Phosphatidylinostitol Triphospate; PDK1/2 3-Phosphoinostitide-Dependent Kinase 1; GS Glycogen Synthase.</div></p></div><div class="aux"><div class="resc">Nassrene Y. Elmadhun, et al. Surgery. ;154(2):320-327.</div><p class="links"><a class="dblinks" href="/pubmed/23889958" ref="ordinalpos=1828&amp;ncbi_uid=6447875&amp;link_uid=23889958">Citation</a><a class="dblinks" href="/pmc/articles/PMC3748137/" ref="ordinalpos=1828&amp;ncbi_uid=6447875&amp;link_uid=PMC3748137">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1829.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2996105/figure/F7/" ref="ordinalpos=1829&amp;ncbi_uid=2949523&amp;link_uid=PMC2996105" href="/pmc/articles/PMC2996105/"><img src="/pmc/articles/instance/2996105/bin/nihms253117f7.gif" src-large="/pmc/articles/instance/2996105/bin/nihms253117f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2996105/" ref="ordinalpos=1829&amp;ncbi_uid=2949523&amp;link_uid=PMC2996105" image-link="/pmc/articles/PMC2996105/figure/F7/" class="imagepopup">Figure 7.  From: ProNGF induces PTEN via p75NTR to suppress Trk -mediated survival <span class="highlight" style="background-color:">signaling</span> in brain neurons. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the interaction of p75<sup>NTR</sup> and TrkB <span class="highlight" style="background-color:">signaling</span>. Apoptosis induced by proNGF via p75<sup>NTR</sup> requires suppression of the TrkB-induced PI<sub>3</sub> kinase <span class="highlight" style="background-color:">pathway</span> by PTEN. When PTEN is inhibited, BDNF-induced activation of Akt can inhibit apoptosis.</div></p></div><div class="aux"><div class="resc">Wenyu Song, et al. J Neurosci. ;30(46):15608-15615.</div><p class="links"><a class="dblinks" href="/pubmed/21084616" ref="ordinalpos=1829&amp;ncbi_uid=2949523&amp;link_uid=21084616">Citation</a><a class="dblinks" href="/pmc/articles/PMC2996105/" ref="ordinalpos=1829&amp;ncbi_uid=2949523&amp;link_uid=PMC2996105">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1830.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2844640/figure/F3/" ref="ordinalpos=1830&amp;ncbi_uid=2282232&amp;link_uid=PMC2844640" href="/pmc/articles/PMC2844640/"><img src="/pmc/articles/instance/2844640/bin/nihms-183419-f0003.gif" src-large="/pmc/articles/instance/2844640/bin/nihms-183419-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2844640/" ref="ordinalpos=1830&amp;ncbi_uid=2282232&amp;link_uid=PMC2844640" image-link="/pmc/articles/PMC2844640/figure/F3/" class="imagepopup">Figure 3. Radiation induced SMAD7, a bottleneck in radiotherapy.  From: Role of Radiation-induced TGF-beta <span class="highlight" style="background-color:">Signaling</span> in Cancer Therapy. </a></p><div class="supp"><p class="details"><div>As a negative feedback mechanism, radiation-induced TGF-β, elevates the expression of SMAD7, which blocks the activity of this <span class="highlight" style="background-color:">pathway</span>. TIEG2 (also known as KLF11) is a tumor suppressor gene that causes repression of SMAD7 expression and can be a useful target for adjuvant radiotherapy to harness the negative growth effects of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Thus, over expression of TIEG2 will drive the TGF-β <span class="highlight" style="background-color:">pathway</span> towards the growth suppressive functions and enhance the radiation response.</div></p></div><div class="aux"><div class="resc">Horatiu C. Dancea, et al. Mol Cell Pharmacol. ;1(1):44-56.</div><p class="links"><a class="dblinks" href="/pubmed/20336170" ref="ordinalpos=1830&amp;ncbi_uid=2282232&amp;link_uid=20336170">Citation</a><a class="dblinks" href="/pmc/articles/PMC2844640/" ref="ordinalpos=1830&amp;ncbi_uid=2282232&amp;link_uid=PMC2844640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1831.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3056154/figure/F1/" ref="ordinalpos=1831&amp;ncbi_uid=2827338&amp;link_uid=PMC3056154" href="/pmc/articles/PMC3056154/"><img src="/pmc/articles/instance/3056154/bin/nihms276493f1.gif" src-large="/pmc/articles/instance/3056154/bin/nihms276493f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3056154/" ref="ordinalpos=1831&amp;ncbi_uid=2827338&amp;link_uid=PMC3056154" image-link="/pmc/articles/PMC3056154/figure/F1/" class="imagepopup">Figure 1. Factors that influence the <span class="highlight" style="background-color:">signaling</span> behavior of the human β2AR.  From: Energy landscapes as a tool to integrate GPCR structure, dynamics and function. </a></p><div class="supp"><p class="details"><div>Ligands with different efficacies (inset) can modulate a given <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Moreover, the efficacy of a drug may differ for different down stream <span class="highlight" style="background-color:">signaling</span> profiles. For instance, certain drugs induce routes involving <span class="highlight" style="background-color:">signaling</span> and regulatory proteins (e.g. Gi, arrestin) that complement the predominant <span class="highlight" style="background-color:">pathway</span> through Gs. In addition to ligand binding, other factors as lipid bilayer composition, pH and salt gradients, membrane potential, protein-protein interactions, either within the membrane (homo- or hetero-oligomerization and interaction with other membrane proteins) or with extracellular and intracellular partners, or post-translational modifications, as glycosilation, phosphorylation or acylation, can also influence the <span class="highlight" style="background-color:">signaling</span> behavior of the receptor.</div></p></div><div class="aux"><div class="resc">Xavier Deupi, et al. Physiology (Bethesda). ;25(5):293-303.</div><p class="links"><a class="dblinks" href="/pubmed/20940434" ref="ordinalpos=1831&amp;ncbi_uid=2827338&amp;link_uid=20940434">Citation</a><a class="dblinks" href="/pmc/articles/PMC3056154/" ref="ordinalpos=1831&amp;ncbi_uid=2827338&amp;link_uid=PMC3056154">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1832.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3241928/figure/F8/" ref="ordinalpos=1832&amp;ncbi_uid=4444517&amp;link_uid=PMC3241928" href="/pmc/articles/PMC3241928/"><img src="/pmc/articles/instance/3241928/bin/nihms336504f8.gif" src-large="/pmc/articles/instance/3241928/bin/nihms336504f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3241928/" ref="ordinalpos=1832&amp;ncbi_uid=4444517&amp;link_uid=PMC3241928" image-link="/pmc/articles/PMC3241928/figure/F8/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Signaling</span> and phenotypic overview of murine melanoma models.  From: ?-catenin <span class="highlight" style="background-color:">signaling</span> controls metastasis in Braf-activated Pten-deficient melanomas. </a></p><div class="supp"><p class="details"><div>(A) <i>Pten/Braf/Bcat-STA</i> melanomas are characterized by rapid growth, high degree of melanocytic differentiation, and extensive metastasis. These phenotypes are associated with increased MAPK/Erk and PI3K/Akt <span class="highlight" style="background-color:">signaling</span> as well as elevated Mitf and Pin1 levels. Bold text: relatively increased levels and/or activity compared to other models, blue shading: relatively decreased compared to other models, red shading: genetically inactivated, *: stabilized.<br />(B) <i>Pten/Braf</i> melanomas exhibit rapid tumor growth, but only intermediate melanocytic differentiation and metastasis. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> activation, Mitf, and Pin1 are intermediate in this model.<br />(C) <i>Pten/Braf/Bcat-KO</i> melanomas exhibit tumor growth, but significantly reduced melanocytic differentiation and almost no metastasis. Inactivation of β-catenin in this model is associated with reduced PI3K/Akt <span class="highlight" style="background-color:">signaling</span> and Mitf levels.<br />(D) <i>Braf/Bcat-STA</i> melanomas are characterized by long latency and slow tumor growth, as well as lack of metastases. These tumors are differentiated and heavily pigmented due to high Mitf levels, but show reduced PI3K/Akt <span class="highlight" style="background-color:">pathway</span> activation.<br />(E) A three-<span class="highlight" style="background-color:">pathway</span> synergy is observed when MAPK/Erk, PI3K/Akt, and β-catenin/Mitf <span class="highlight" style="background-color:">signaling</span> pathways are simultaneously activated in melanoma. If any of these three changes is not present, the metastatic melanoma phenotype is abrogated.</div></p></div><div class="aux"><div class="resc">William E. Damsky, et al. Cancer Cell. ;20(6):741-754.</div><p class="links"><a class="dblinks" href="/pubmed/22172720" ref="ordinalpos=1832&amp;ncbi_uid=4444517&amp;link_uid=22172720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3241928/" ref="ordinalpos=1832&amp;ncbi_uid=4444517&amp;link_uid=PMC3241928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1833.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC98946/figure/F8/" ref="ordinalpos=1833&amp;ncbi_uid=49733&amp;link_uid=PMC98946" href="/pmc/articles/PMC98946/"><img src="/pmc/articles/instance/98946/bin/mr0480048008.gif" src-large="/pmc/articles/instance/98946/bin/mr0480048008.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC98946/" ref="ordinalpos=1833&amp;ncbi_uid=49733&amp;link_uid=PMC98946" image-link="/pmc/articles/PMC98946/figure/F8/" class="imagepopup">FIG. 8.  From: MAP Kinase Pathways in the Yeast Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Spore wall assembly <span class="highlight" style="background-color:">pathway</span> of <i>S. cerevisiae</i>. Whether <span class="highlight" style="background-color:">signaling</span> in the <span class="highlight" style="background-color:">pathway</span> is modulated solely by expression of <span class="highlight" style="background-color:">pathway</span> proteins or whether an upstream activating physiological stimulus exists has not yet been determined. A MEKK and MEK for this <span class="highlight" style="background-color:">pathway</span> have not been identified, and so it remains an open question whether this is a typical three-kinase MAPK cascade.</div></p></div><div class="aux"><div class="resc">Michael C. Gustin, et al. Microbiol Mol Biol Rev. 1998 December;62(4):1264-1300.</div><p class="links"><a class="dblinks" href="/pubmed/9841672" ref="ordinalpos=1833&amp;ncbi_uid=49733&amp;link_uid=9841672">Citation</a><a class="dblinks" href="/pmc/articles/PMC98946/" ref="ordinalpos=1833&amp;ncbi_uid=49733&amp;link_uid=PMC98946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1834.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411054/figure/F6/" ref="ordinalpos=1834&amp;ncbi_uid=5161752&amp;link_uid=PMC3411054" href="/pmc/articles/PMC3411054/"><img src="/pmc/articles/instance/3411054/bin/zbc0331217470006.gif" src-large="/pmc/articles/instance/3411054/bin/zbc0331217470006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411054/" ref="ordinalpos=1834&amp;ncbi_uid=5161752&amp;link_uid=PMC3411054" image-link="/pmc/articles/PMC3411054/figure/F6/" class="imagepopup">FIGURE 6. From: Targeting the Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> in Keratocystic Odontogenic Tumor. </a></p><div class="supp"><p class="details"><div><b>The protein expression of SHH <span class="highlight" style="background-color:">pathway</span> components was down-regulated by cyclopamine treatment.</b> KCOT-1 cells were treated with cyclopamine (<i>C</i>) (10, 20, and 25 μ<span style="font-variant: small-caps" class="small-caps">m</span>) or tomatidine (<i>T</i>) (5 and 10 μ<span style="font-variant: small-caps" class="small-caps">m</span>) or no treatment (<i>NT</i>) for 48 h. <i>A</i>, protein expression for the SHH <span class="highlight" style="background-color:">pathway</span>. <i>B</i>, densitometric analysis of the calculated reduction of the protein levels of the SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>OD Ratio</i>, optical density ratio.</div></p></div><div class="aux"><div class="resc">Changchun Ren, et al. J Biol Chem. 2012 August 3;287(32):27117-27125.</div><p class="links"><a class="dblinks" href="/pubmed/22679015" ref="ordinalpos=1834&amp;ncbi_uid=5161752&amp;link_uid=22679015">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411054/" ref="ordinalpos=1834&amp;ncbi_uid=5161752&amp;link_uid=PMC3411054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1835.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479345/figure/F3/" ref="ordinalpos=1835&amp;ncbi_uid=713501&amp;link_uid=PMC1479345" href="/pmc/articles/PMC1479345/"><img src="/pmc/articles/instance/1479345/bin/1476-511X-5-10-3.gif" src-large="/pmc/articles/instance/1479345/bin/1476-511X-5-10-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479345/" ref="ordinalpos=1835&amp;ncbi_uid=713501&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F3/" class="imagepopup">Figure 3.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. Figures 3–7 represent <span class="highlight" style="background-color:">signaling</span> pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following <span class="highlight" style="background-color:">signaling</span> cascades are shown: JUN, TNF, and CDKN2A <span class="highlight" style="background-color:">signaling</span> cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell <span class="highlight" style="background-color:">signaling</span>; gene expression; and cell cycle (Fig. 5); INS1/hRAS <span class="highlight" style="background-color:">signaling</span> cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC <span class="highlight" style="background-color:">signaling</span> cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the <span class="highlight" style="background-color:">signaling</span> diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/<span class="highlight" style="background-color:">signaling</span> categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></p></div><div class="aux"><div class="resc">Alvin Berger, et al. Lipids Health Dis. 2006;5:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/16623957" ref="ordinalpos=1835&amp;ncbi_uid=713501&amp;link_uid=16623957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479345/" ref="ordinalpos=1835&amp;ncbi_uid=713501&amp;link_uid=PMC1479345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1836.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479345/figure/F5/" ref="ordinalpos=1836&amp;ncbi_uid=713497&amp;link_uid=PMC1479345" href="/pmc/articles/PMC1479345/"><img src="/pmc/articles/instance/1479345/bin/1476-511X-5-10-5.gif" src-large="/pmc/articles/instance/1479345/bin/1476-511X-5-10-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479345/" ref="ordinalpos=1836&amp;ncbi_uid=713497&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F5/" class="imagepopup">Figure 5.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. Figures 3-7 represent <span class="highlight" style="background-color:">signaling</span> pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following <span class="highlight" style="background-color:">signaling</span> cascades are shown: JUN, TNF, and CDKN2A <span class="highlight" style="background-color:">signaling</span> cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell <span class="highlight" style="background-color:">signaling</span>; gene expression; and cell cycle (Fig. 5); INS1/hRAS <span class="highlight" style="background-color:">signaling</span> cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC <span class="highlight" style="background-color:">signaling</span> cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the <span class="highlight" style="background-color:">signaling</span> diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/<span class="highlight" style="background-color:">signaling</span> categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></p></div><div class="aux"><div class="resc">Alvin Berger, et al. Lipids Health Dis. 2006;5:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/16623957" ref="ordinalpos=1836&amp;ncbi_uid=713497&amp;link_uid=16623957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479345/" ref="ordinalpos=1836&amp;ncbi_uid=713497&amp;link_uid=PMC1479345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1837.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479345/figure/F7/" ref="ordinalpos=1837&amp;ncbi_uid=713490&amp;link_uid=PMC1479345" href="/pmc/articles/PMC1479345/"><img src="/pmc/articles/instance/1479345/bin/1476-511X-5-10-7.gif" src-large="/pmc/articles/instance/1479345/bin/1476-511X-5-10-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479345/" ref="ordinalpos=1837&amp;ncbi_uid=713490&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F7/" class="imagepopup">Figure 7.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. Figures 3-7 represent <span class="highlight" style="background-color:">signaling</span> pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following <span class="highlight" style="background-color:">signaling</span> cascades are shown: JUN, TNF, and CDKN2A <span class="highlight" style="background-color:">signaling</span> cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell <span class="highlight" style="background-color:">signaling</span>; gene expression; and cell cycle (Fig. 5); INS1/hRAS <span class="highlight" style="background-color:">signaling</span> cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC <span class="highlight" style="background-color:">signaling</span> cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the <span class="highlight" style="background-color:">signaling</span> diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/<span class="highlight" style="background-color:">signaling</span> categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></p></div><div class="aux"><div class="resc">Alvin Berger, et al. Lipids Health Dis. 2006;5:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/16623957" ref="ordinalpos=1837&amp;ncbi_uid=713490&amp;link_uid=16623957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479345/" ref="ordinalpos=1837&amp;ncbi_uid=713490&amp;link_uid=PMC1479345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1838.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3743506/figure/F9/" ref="ordinalpos=1838&amp;ncbi_uid=6477884&amp;link_uid=PMC3743506" href="/pmc/articles/PMC3743506/"><img src="/pmc/articles/instance/3743506/bin/zbc0361358130009.gif" src-large="/pmc/articles/instance/3743506/bin/zbc0361358130009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3743506/" ref="ordinalpos=1838&amp;ncbi_uid=6477884&amp;link_uid=PMC3743506" image-link="/pmc/articles/PMC3743506/figure/F9/" class="imagepopup">FIGURE 9. From: Calmodulin-dependent Protein Kinase II/cAMP Response Element-binding Protein/Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Cascade Regulates Angiotensin II-induced Podocyte Injury and Albuminuria. </a></p><div class="supp"><p class="details"><div><b>Blockade of Wnt <span class="highlight" style="background-color:">signaling</span> with ectopic expression of Dkk1 ameliorates Ang II-induced podocyte injury and albuminuria in mice.</b> <i>A,</i> urinary albumin excretion in mice, the <i>asterisk</i> indicates <i>p</i> &lt; 0.05 compared with controls (<i>n</i> = 5), # indicates <i>p</i> &lt; 0.05 compared with Ang II plus pcDNA3-treated mice (<i>n</i> = 5). <i>B</i> and <i>C,</i> real-time PCR analysis for nephrin and podocin mRNA, the <i>asterisk</i> indicates <i>p</i> &lt; 0.05 compared with controls (<i>n</i> = 5), # indicates <i>p</i> &lt; 0.05 compared with Ang II plus pcDNA3-treated mice (<i>n</i> = 5). <i>D,</i> schematic model illustrates that Ang II can stimulate a cascade of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in podocytes, including activation of CaMK II, CREB, and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, which leads to podocyte injury and albuminuria. Inhibition of this <span class="highlight" style="background-color:">signaling</span> cascade with Dkk1, β-catenin siRNA, CREB siRNA, and CK59 ameliorates Ang II-induced podocyte injury.</div></p></div><div class="aux"><div class="resc">Lei Jiang, et al. J Biol Chem. 2013 August 9;288(32):23368-23379.</div><p class="links"><a class="dblinks" href="/pubmed/23803607" ref="ordinalpos=1838&amp;ncbi_uid=6477884&amp;link_uid=23803607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3743506/" ref="ordinalpos=1838&amp;ncbi_uid=6477884&amp;link_uid=PMC3743506">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1839.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3582304/figure/fig2-1759720X12466608/" ref="ordinalpos=1839&amp;ncbi_uid=4721195&amp;link_uid=PMC3582304" href="/pmc/articles/PMC3582304/"><img src="/pmc/articles/instance/3582304/bin/10.1177_1759720X12466608-fig2.gif" src-large="/pmc/articles/instance/3582304/bin/10.1177_1759720X12466608-fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3582304/" ref="ordinalpos=1839&amp;ncbi_uid=4721195&amp;link_uid=PMC3582304" image-link="/pmc/articles/PMC3582304/figure/fig2-1759720X12466608/" class="imagepopup">Figure 2. From: Wnt <span class="highlight" style="background-color:">signaling</span> in bone formation and its therapeutic potential for bone diseases. </a></p><div class="supp"><p class="details"><div>Potential therapeutic targets of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the presence of sclerostin or dickkopf (Dkk) proteins, the ligands cannot bind their receptors to activate the Wnt <span class="highlight" style="background-color:">signaling</span>. The inhibitory effects of sclerostin and Dkk proteins can be neutralized by their respective antibodies, thus activating the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream. Secreted Frizzled-related proteins (SFRPs) and Wif inhibit the Wnt <span class="highlight" style="background-color:">signaling</span> by binding the Wnt ligands, but SFRP inhibitors such as diphenylsulfonyl sulfonamide can inhibit SFRPs from binding the Wnt ligands. Glycogen synthase kinase 3β (GSK3β) normally targets β catenin for degradation by phosphorylation. However, lithium or other pharmacological GSK3β-inhibiting compounds can inhibit the GSK3β activity, thus allowing β catenin to accumulate in the cytoplasm and the nucleus expressing the target genes. Chibby (Cby) is an endogenous antagonist, which interrupts the binding of β catenin to transcriptional factors T-cell factor (Tcf)/lymphoid enhancer-binding factor (Lef-1).</div></p></div><div class="aux"><div class="resc">Jeong Hwan Kim, et al. Ther Adv Musculoskelet Dis. 2013 February;5(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/23514963" ref="ordinalpos=1839&amp;ncbi_uid=4721195&amp;link_uid=23514963">Citation</a><a class="dblinks" href="/pmc/articles/PMC3582304/" ref="ordinalpos=1839&amp;ncbi_uid=4721195&amp;link_uid=PMC3582304">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1840.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3060910/figure/pone-0017957-g001/" ref="ordinalpos=1840&amp;ncbi_uid=2843614&amp;link_uid=PMC3060910" href="/pmc/articles/PMC3060910/"><img src="/pmc/articles/instance/3060910/bin/pone.0017957.g001.gif" src-large="/pmc/articles/instance/3060910/bin/pone.0017957.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3060910/" ref="ordinalpos=1840&amp;ncbi_uid=2843614&amp;link_uid=PMC3060910" image-link="/pmc/articles/PMC3060910/figure/pone-0017957-g001/" class="imagepopup">Figure 1. In silico analysis of Notch <span class="highlight" style="background-color:">pathway</span> gene                             promoters..  From: The Notch-2 Gene Is Regulated by Wnt <span class="highlight" style="background-color:">Signaling</span> in                     Cultured Colorectal Cancer Cells. </a></p><div class="supp"><p class="details"><div><i>In silico</i> analysis of the genetic networks directly                             involved in Notch and Wnt <span class="highlight" style="background-color:">signaling</span> suggest overlap and direct crosstalk                             via Notch target gene activation through canonical Wnt <span class="highlight" style="background-color:">signaling</span>. By                             means of MatInspector, [33] gene promoters in the Notch <span class="highlight" style="background-color:">pathway</span> were                             found to contain at least one putative LEF-1/TCF-site (number of sites                             in per gene is described in square brackets). Genes in gray boxes were                             subjected for further semi quantitative RT-PCR analysis.</div></p></div><div class="aux"><div class="resc">Jonas Ungerbäck, et al. PLoS One. 2011;6(3):e17957.</div><p class="links"><a class="dblinks" href="/pubmed/21437251" ref="ordinalpos=1840&amp;ncbi_uid=2843614&amp;link_uid=21437251">Citation</a><a class="dblinks" href="/pmc/articles/PMC3060910/" ref="ordinalpos=1840&amp;ncbi_uid=2843614&amp;link_uid=PMC3060910">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1841.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3532050/figure/fig1/" ref="ordinalpos=1841&amp;ncbi_uid=6523875&amp;link_uid=PMC3532050" href="/pmc/articles/PMC3532050/"><img src="/pmc/articles/instance/4002810/bin/cmi201159f1.gif" src-large="/pmc/articles/instance/4002810/bin/cmi201159f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3532050/" ref="ordinalpos=1841&amp;ncbi_uid=6523875&amp;link_uid=PMC3532050" image-link="/pmc/articles/PMC3532050/figure/fig1/" class="imagepopup">Figure 1.  From: Regulation of NF-?B <span class="highlight" style="background-color:">signaling</span> by the A20 deubiquitinase. </a></p><div class="supp"><p class="details"><div>Mechanisms of NF-κB inhibition by A20 in the TNFR1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A20 is induced by NF-κB in the TNFR1 <span class="highlight" style="background-color:">pathway</span> and functions in a negative feedback loop. The A20 adaptor molecule TAX1BP1 is phosphorylated on Ser593 and Ser624 by IKKα which nucleates the A20 ubiquitin-editing complex and is essential for interactions between TAX1BP1, Itch, RNF11 and A20. The A20 ubiquitin-editing complex inhibits RIP1 K63-linked polyubiquitination to terminate NF-κB <span class="highlight" style="background-color:">signaling</span> downstream of TNFR1. IKK, IκB kinase; RNF11, ring finger protein 11; TAX1BP1, Tax1-binding protein 1; TNFR1, TNF receptor 1.</div></p></div><div class="aux"><div class="resc">Noula Shembade, et al. Cell Mol Immunol. ;9(2):123-130.</div><p class="links"><a class="dblinks" href="/pubmed/22343828" ref="ordinalpos=1841&amp;ncbi_uid=6523875&amp;link_uid=22343828">Citation</a><a class="dblinks" href="/pmc/articles/PMC3532050/" ref="ordinalpos=1841&amp;ncbi_uid=6523875&amp;link_uid=PMC3532050">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1842.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2759143/figure/f2-grsb-2007-217/" ref="ordinalpos=1842&amp;ncbi_uid=2044644&amp;link_uid=PMC2759143" href="/pmc/articles/PMC2759143/"><img src="/pmc/articles/instance/2759143/bin/grsb-2007-217f2.gif" src-large="/pmc/articles/instance/2759143/bin/grsb-2007-217f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2759143/" ref="ordinalpos=1842&amp;ncbi_uid=2044644&amp;link_uid=PMC2759143" image-link="/pmc/articles/PMC2759143/figure/f2-grsb-2007-217/" class="imagepopup">Figure 2.  From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms of Endogenous Angiogenesis Inhibitors Derived from Type IV Collagen. </a></p><div class="supp"><p class="details"><div>Schematic illustration of distinct <span class="highlight" style="background-color:">signaling</span> pathways induced by α2(IV)NC1. α2(IV)NC1 binds to αVβ3 and αVβ5 integrins on endothelial and tumor cells, and initiates two distinct <span class="highlight" style="background-color:">signaling</span> pathways. (1) Inhibition of phosphorylation of FAK/PI-3K/eIF4E/4E-BP1. (2) Activation of caspase-8 and -9 leading to activation of caspase-3. α2(IV)NC1 activates pro-caspase-8 and -9 directly through inhibition of FAK/PI3K/Akt/mTOR <span class="highlight" style="background-color:">pathway</span>. α2(IV)NC1 also indirectly enhances the mitochondrial <span class="highlight" style="background-color:">pathway</span> through Fas dependent caspase-8 activation, which results in inhibition of protein synthesis, DNA damage and cell death.</div></p></div><div class="aux"><div class="resc">Akulapalli Sudhakar, et al. Gene Regul Syst Bio. 2007;1:217-226.</div><p class="links"><a class="dblinks" href="/pubmed/19936090" ref="ordinalpos=1842&amp;ncbi_uid=2044644&amp;link_uid=19936090">Citation</a><a class="dblinks" href="/pmc/articles/PMC2759143/" ref="ordinalpos=1842&amp;ncbi_uid=2044644&amp;link_uid=PMC2759143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1843.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2833197/figure/pone-0009575-g007/" ref="ordinalpos=1843&amp;ncbi_uid=2199389&amp;link_uid=PMC2833197" href="/pmc/articles/PMC2833197/"><img src="/pmc/articles/instance/2833197/bin/pone.0009575.g007.gif" src-large="/pmc/articles/instance/2833197/bin/pone.0009575.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2833197/" ref="ordinalpos=1843&amp;ncbi_uid=2199389&amp;link_uid=PMC2833197" image-link="/pmc/articles/PMC2833197/figure/pone-0009575-g007/" class="imagepopup">Figure 7. VEGFA expression is regulated by ER stress..  From: Transcriptional Regulation of VEGF-A by the Unfolded Protein Response <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>ER stress <span class="highlight" style="background-color:">signaling</span> regulates VEGFA expression via its principal regulators Ire1α, Perk and ATF6α. Proposed <span class="highlight" style="background-color:">pathway</span> shows how ER stress <span class="highlight" style="background-color:">signaling</span> might affect growth and survival of both developing placenta as well as tumor through angiogenesis. This is accomplished by activating VEGFA transcription via different regulatory regions on the VEGFA gene.</div></p></div><div class="aux"><div class="resc">Rajarshi Ghosh, et al. PLoS One. 2010;5(3):e9575.</div><p class="links"><a class="dblinks" href="/pubmed/20221394" ref="ordinalpos=1843&amp;ncbi_uid=2199389&amp;link_uid=20221394">Citation</a><a class="dblinks" href="/pmc/articles/PMC2833197/" ref="ordinalpos=1843&amp;ncbi_uid=2199389&amp;link_uid=PMC2833197">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1844.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758746/figure/F6/" ref="ordinalpos=1844&amp;ncbi_uid=2041532&amp;link_uid=PMC2758746" href="/pmc/articles/PMC2758746/"><img src="/pmc/articles/instance/2758746/bin/2320fig6.gif" src-large="/pmc/articles/instance/2758746/bin/2320fig6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758746/" ref="ordinalpos=1844&amp;ncbi_uid=2041532&amp;link_uid=PMC2758746" image-link="/pmc/articles/PMC2758746/figure/F6/" class="imagepopup">Figure 6. From: A heritable switch in carbon source utilization driven by an unusual yeast prion. </a></p><div class="supp"><p class="details"><div>Pma1 and the Rgt2/Snf3 glucose <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> We propose that Pma1 acts as a part of the Rgt2/Snf3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) In [<i>gar</i><sup>−</sup>] glucose-grown cells, Pma1 associates with Mth1. The glucose signal is propagated through Snf3 and Rgt2 to Yck1 and Yck2, which phosphorylate Mth1 and Std1. This phosphorylation marks Mth1 and Std1 for degredation, leaving their interacting partner, Rgt1, free in the cytosol, where it does not repress transcription at the <i>HXT3</i> locus. (<i>B</i>) Under [<i>GAR</i><sup>+</sup>] conditions, <i>HXT3</i> transcription is repressed, which resembles that of cells grown in a carbon source other than glucose. Pma1 associates with Std1, which somehow facilitates the repression of <i>HXT3</i>, possibly by altering the affinity of Std1 for Rgt1. Association with Std1 has been shown previously to facilitate the binding of Rgt1 to DNA (Lakshmanan et al. 2003). The association between Pma1 can either be transient or stable, but either way it aids in the establishment of an altered <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. This altered <span class="highlight" style="background-color:">pathway</span> is then maintained either by the contained association between Std1 and Pma1 or by a feedback loop within the <span class="highlight" style="background-color:">signaling</span> cascade itself.</div></p></div><div class="aux"><div class="resc">Jessica C.S. Brown, et al. Genes Dev. 2009 October 1;23(19):2320-2332.</div><p class="links"><a class="dblinks" href="/pubmed/19797769" ref="ordinalpos=1844&amp;ncbi_uid=2041532&amp;link_uid=19797769">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758746/" ref="ordinalpos=1844&amp;ncbi_uid=2041532&amp;link_uid=PMC2758746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1845.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3587995/figure/f3-ijms-14-02431/" ref="ordinalpos=1845&amp;ncbi_uid=4705037&amp;link_uid=PMC3587995" href="/pmc/articles/PMC3587995/"><img src="/pmc/articles/instance/3587995/bin/ijms-14-02431f3.gif" src-large="/pmc/articles/instance/3587995/bin/ijms-14-02431f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3587995/" ref="ordinalpos=1845&amp;ncbi_uid=4705037&amp;link_uid=PMC3587995" image-link="/pmc/articles/PMC3587995/figure/f3-ijms-14-02431/" class="imagepopup">Figure 3.  From: The Yin-Yang of DNA Damage Response: Roles in Tumorigenesis and Cellular Senescence. </a></p><div class="supp"><p class="details"><div>Yin and Yang of the ATM-CHK2-p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> upon BRCA1 mutation-associated premature aging and tumorigenesis. ATM-CHK2-p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> senses DNA damage/genomic instability and acts as a gatekeeper to eliminate mutations, but, as a side effect, it may also lead to premature aging.</div></p></div><div class="aux"><div class="resc">Xiaoman Li, et al. Int J Mol Sci. 2013 February;14(2):2431-2448.</div><p class="links"><a class="dblinks" href="/pubmed/23354477" ref="ordinalpos=1845&amp;ncbi_uid=4705037&amp;link_uid=23354477">Citation</a><a class="dblinks" href="/pmc/articles/PMC3587995/" ref="ordinalpos=1845&amp;ncbi_uid=4705037&amp;link_uid=PMC3587995">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1846.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3089884/figure/F5/" ref="ordinalpos=1846&amp;ncbi_uid=2951692&amp;link_uid=PMC3089884" href="/pmc/articles/PMC3089884/"><img src="/pmc/articles/instance/3089884/bin/97320630006111F5.gif" src-large="/pmc/articles/instance/3089884/bin/97320630006111F5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3089884/" ref="ordinalpos=1846&amp;ncbi_uid=2951692&amp;link_uid=PMC3089884" image-link="/pmc/articles/PMC3089884/figure/F5/" class="imagepopup">Figure 5.  From: Deciphering the key molecular and cellular events in neutrophil transmigration during acute inflammation. </a></p><div class="supp"><p class="details"><div>Chemokine <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. KEGG <span class="highlight" style="background-color:">Pathway</span> Analysis of Genes Implicated in the Chemokine <span class="highlight" style="background-color:">Signaling</span> (Red Stars) in the colonic epithelial tissue of 3% DSS-induced Colitis in C57BL/6j mice.</div></p></div><div class="aux"><div class="resc">Srinivasan Dinesh Kumar, et al. Bioinformation. 2011;6(3):111-114.</div><p class="links"><a class="dblinks" href="/pubmed/21584186" ref="ordinalpos=1846&amp;ncbi_uid=2951692&amp;link_uid=21584186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3089884/" ref="ordinalpos=1846&amp;ncbi_uid=2951692&amp;link_uid=PMC3089884">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1847.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2780819/figure/F3/" ref="ordinalpos=1847&amp;ncbi_uid=2153238&amp;link_uid=PMC2780819" href="/pmc/articles/PMC2780819/"><img src="/pmc/articles/instance/2780819/bin/CCR-4-179_F3.gif" src-large="/pmc/articles/instance/2780819/bin/CCR-4-179_F3.jpg" alt="Fig. (3). " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2780819/" ref="ordinalpos=1847&amp;ncbi_uid=2153238&amp;link_uid=PMC2780819" image-link="/pmc/articles/PMC2780819/figure/F3/" class="imagepopup">Fig. (3).  From: Roles of IL-6-gp130 <span class="highlight" style="background-color:">Signaling</span> in Vascular Inflammation. </a></p><div class="supp"><p class="details"><div><b>Negative regulation of the JAK-STAT <span class="highlight" style="background-color:">pathway</span>.</b> Shown are the major negative autoregulatory pathways of IL-6 induced STAT3 <span class="highlight" style="background-color:">signaling</span>. IL-6 activated STAT3 both engages the suppressor of cytokine <span class="highlight" style="background-color:">signaling</span> (SOCS3) gene, inducing its expression and recruits SOC3 to gp130 where it subsequently terminates STAT3 activation <i>via</i> JAK1 inactivation. SHP2 phosphatase activity also inactivates gp130 and JAKs.</div></p></div><div class="aux"><div class="resc">Tieying Hou, et al. Curr Cardiol Rev. 2008 August;4(3):179-192.</div><p class="links"><a class="dblinks" href="/pubmed/19936194" ref="ordinalpos=1847&amp;ncbi_uid=2153238&amp;link_uid=19936194">Citation</a><a class="dblinks" href="/pmc/articles/PMC2780819/" ref="ordinalpos=1847&amp;ncbi_uid=2153238&amp;link_uid=PMC2780819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1848.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821554/figure/F1/" ref="ordinalpos=1848&amp;ncbi_uid=5477229&amp;link_uid=PMC3821554" href="/pmc/articles/PMC3821554/"><img src="/pmc/articles/instance/3821554/bin/nihms518906f1.gif" src-large="/pmc/articles/instance/3821554/bin/nihms518906f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821554/" ref="ordinalpos=1848&amp;ncbi_uid=5477229&amp;link_uid=PMC3821554" image-link="/pmc/articles/PMC3821554/figure/F1/" class="imagepopup">Figure 1.  From: Transforming growth factor-? <span class="highlight" style="background-color:">signaling</span> and ubiquitinators in cancer. </a></p><div class="supp"><p class="details"><div>The TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is regulated at multiple points by ubiquitin-mediated targeting and proteasomal degradation. Receptors, adaptor proteins, Smads, co-activator/co-repressor and DNA binding co-factors are targeted by E3 ubiquitin ligases for ubiquitination and degredation by the proteasome. These protein degredation pathways impart precise and specific control over the cellular response to TGF-β <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Eric Glasgow, et al. Endocr Relat Cancer. 2008 March;15(1):10.1677/ERC-07-0168.</div><p class="links"><a class="dblinks" href="/pubmed/18310276" ref="ordinalpos=1848&amp;ncbi_uid=5477229&amp;link_uid=18310276">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821554/" ref="ordinalpos=1848&amp;ncbi_uid=5477229&amp;link_uid=PMC3821554">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1849.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC139432/figure/F1/" ref="ordinalpos=1849&amp;ncbi_uid=352469&amp;link_uid=PMC139432" href="/pmc/articles/PMC139432/"><img src="/pmc/articles/instance/139432/bin/bcr42-1.gif" src-large="/pmc/articles/instance/139432/bin/bcr42-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC139432/" ref="ordinalpos=1849&amp;ncbi_uid=352469&amp;link_uid=PMC139432" image-link="/pmc/articles/PMC139432/figure/F1/" class="imagepopup">Figure 1.  From: Transforming growth factor-? and breast cancer: Transforming growth factor-?/SMAD <span class="highlight" style="background-color:">signaling</span> defects and cancer. </a></p><div class="supp"><p class="details"><div>This model schematically illustrates the various molecular mechanisms that can contribute to negative regulation of SMAD <span class="highlight" style="background-color:">signaling</span> and thereby to a loss of transforming growth factor (TGF)β-induced growth inhibition. Inactivating mutations or loss of expression of components of the TGFβ/SMAD <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are not depicted here but may also be important. For details, see text.</div></p></div><div class="aux"><div class="resc">Marcus Kretzschmar. Breast Cancer Res. 2000;2(2):107-115.</div><p class="links"><a class="dblinks" href="/pubmed/11250700" ref="ordinalpos=1849&amp;ncbi_uid=352469&amp;link_uid=11250700">Citation</a><a class="dblinks" href="/pmc/articles/PMC139432/" ref="ordinalpos=1849&amp;ncbi_uid=352469&amp;link_uid=PMC139432">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1850.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2668855/figure/F7/" ref="ordinalpos=1850&amp;ncbi_uid=1858814&amp;link_uid=PMC2668855" href="/pmc/articles/PMC2668855/"><img src="/pmc/articles/instance/2668855/bin/zh80170934070007.gif" src-large="/pmc/articles/instance/2668855/bin/zh80170934070007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2668855/" ref="ordinalpos=1850&amp;ncbi_uid=1858814&amp;link_uid=PMC2668855" image-link="/pmc/articles/PMC2668855/figure/F7/" class="imagepopup">Figure 7.  From: Loss of pleckstrin defines a novel <span class="highlight" style="background-color:">pathway</span> for PKC-mediated exocytosis. </a></p><div class="supp"><p class="details"><div><b>Model of <span class="highlight" style="background-color:">signaling</span> by pleckstrin leading to platelet activation</b>. Stimulation of an agonist receptor leads to the activation of PI3K and PLC. These 2 enzymes participate in alternative <span class="highlight" style="background-color:">signaling</span> pathways, leading to platelet aggregation. Activation of PLC leads to the production of DAG, PKC activation, and pleckstrin phosphorylation. Once phosphorylated, pleckstrin binds to the cell membrane and contributes to integrin activation, actin assembly, and exocytosis. An alternative <span class="highlight" style="background-color:">pathway</span> to platelet activation involving PI3K is pleckstrin-independent.</div></p></div><div class="aux"><div class="resc">Lurong Lian, et al. Blood. 2009 April 9;113(15):3577-3584.</div><p class="links"><a class="dblinks" href="/pubmed/19190246" ref="ordinalpos=1850&amp;ncbi_uid=1858814&amp;link_uid=19190246">Citation</a><a class="dblinks" href="/pmc/articles/PMC2668855/" ref="ordinalpos=1850&amp;ncbi_uid=1858814&amp;link_uid=PMC2668855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1851.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478307/figure/F2/" ref="ordinalpos=1851&amp;ncbi_uid=5359244&amp;link_uid=PMC3478307" href="/pmc/articles/PMC3478307/"><img src="/pmc/articles/instance/3478307/bin/cc-11-3559-g2.gif" src-large="/pmc/articles/instance/3478307/bin/cc-11-3559-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478307/" ref="ordinalpos=1851&amp;ncbi_uid=5359244&amp;link_uid=PMC3478307" image-link="/pmc/articles/PMC3478307/figure/F2/" class="imagepopup">Figure. From: Toll-like receptors, <span class="highlight" style="background-color:">signaling</span> adapters and regulation of the pro-inflammatory response by PI3K. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> BCAP links TLRs to PI3K. TLR ligation initiates a <span class="highlight" style="background-color:">signaling</span> cascade leading to activation of NFκB. Activation of NFκB by TLRs is mediated through recruitment of <span class="highlight" style="background-color:">signaling</span> adaptors, including MyD88. The MyD88-dependent <span class="highlight" style="background-color:">pathway</span>, which is utilized by all TLRs except TLR3, mediates assembly of a ternary complex of IRAK proteins. This complex then engages and activates TRAF6, which ultimately leads to activation of NFκB. In addition to activation of NFκB, MyD88-dependent TLR <span class="highlight" style="background-color:">signaling</span> also leads to activation of PI3K. Activation of PI3K by TLRs critically requires the TIR-domain containing TLR adaptor BCAP. Upon stimulation, tyrosine phosphorylated BCAP recruits the PI3K complex. Activated PI3K then phosphorylates its lipid substrate PtdIns(4,5)P<sub>2</sub>. Increased concentrations of PtdIns(3,4,5)P<sub>3</sub> leads to activation of the serine/threonine kinase Akt by PDPK1. Activation of the PI3K <span class="highlight" style="background-color:">signaling</span> axis represents a major branch of TLR mediated <span class="highlight" style="background-color:">signaling</span> and contributes to a variety of cellular outcomes, including regulation of the inflammatory response, cell proliferation and cell survival.</div></p></div><div class="aux"><div class="resc">Ty Dale Troutman, et al. Cell Cycle. 2012 October 1;11(19):3559-3567.</div><p class="links"><a class="dblinks" href="/pubmed/22895011" ref="ordinalpos=1851&amp;ncbi_uid=5359244&amp;link_uid=22895011">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478307/" ref="ordinalpos=1851&amp;ncbi_uid=5359244&amp;link_uid=PMC3478307">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1852.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2861953/figure/f2/" ref="ordinalpos=1852&amp;ncbi_uid=2603244&amp;link_uid=PMC2861953" href="/pmc/articles/PMC2861953/"><img src="/pmc/articles/instance/2861953/bin/fig-2.gif" src-large="/pmc/articles/instance/2861953/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2861953/" ref="ordinalpos=1852&amp;ncbi_uid=2603244&amp;link_uid=PMC2861953" image-link="/pmc/articles/PMC2861953/figure/f2/" class="imagepopup">FIG. 2. From: Lessons from Mouse Models of Thyroid Cancer. </a></p><div class="supp"><p class="details"><div>Altered <span class="highlight" style="background-color:">signaling</span> pathways and mouse models of thyroid cancer. Mouse models are indicated by an (*). Aberrant <span class="highlight" style="background-color:">signaling</span> through receptor tyrosine kinases (RTK) such as the <i>RET</i> and <i>NTRK1</i> receptors induce papillary thyroid cancer (PTC). Overactivation of the RAS/RAF/MEK/ERK <span class="highlight" style="background-color:">pathway</span> (or MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) leads to PTC. The <i>BRAF<sup>V600E</sup></i> mutation results in constitutive activation of BRAF and is commonly found in PTC. <i>RAS</i> overactivation is common to both papillary and follicular thyroid cancer (FTC) models. TSH receptor (TSHR) <span class="highlight" style="background-color:">signaling</span> via adenyl cyclases/cAMP/protein kinase A (PKA) is important for normal thyroid growth, but only a few models of TSHR overactivation develop thyroid cancer (<i>Prkar1a<sup>Δ2</sup>/<sup>+</sup>, Rap1b<sup>G12V</sup></i>). The adrenergic receptor may also contribute to thyroid cancer development also through increased cAMP and <span class="highlight" style="background-color:">signaling</span> via pathways such as MAPK and protein kinase C (PKC). Overactivation in the PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via cell-surface receptors such TSHR or inactivation of the phosphatase, <i>PTEN</i>, are important for FTC tumorigenesis. <i>TRß<sup>PV/PV</sup></i> mice also have PI3K-AKT overactivation and develop FTC. Please see text for additional details regarding these mouse models.</div></p></div><div class="aux"><div class="resc">Caroline S. Kim, et al. Thyroid. 2009 December;19(12):1317-1331.</div><p class="links"><a class="dblinks" href="/pubmed/20001715" ref="ordinalpos=1852&amp;ncbi_uid=2603244&amp;link_uid=20001715">Citation</a><a class="dblinks" href="/pmc/articles/PMC2861953/" ref="ordinalpos=1852&amp;ncbi_uid=2603244&amp;link_uid=PMC2861953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1853.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052099/figure/F1/" ref="ordinalpos=1853&amp;ncbi_uid=6291370&amp;link_uid=PMC4052099" href="/pmc/articles/PMC4052099/"><img src="/pmc/articles/instance/4052099/bin/fgene-05-00176-g0001.gif" src-large="/pmc/articles/instance/4052099/bin/fgene-05-00176-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052099/" ref="ordinalpos=1853&amp;ncbi_uid=6291370&amp;link_uid=PMC4052099" image-link="/pmc/articles/PMC4052099/figure/F1/" class="imagepopup">Figure 1.  From: Revisiting glucose uptake and metabolism in schistosomes: new molecular insights for improved schistosomiasis therapies. </a></p><div class="supp"><p class="details"><div><b>Predicted insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in schistosomes based on the characterization of schistosome insulin receptors, the published <i>S. japonicum</i> KEGG insulin <span class="highlight" style="background-color:">pathway</span> (www.chgc.sh.cn/japonicum/sjpathway) and the human insulin <span class="highlight" style="background-color:">pathway</span> (http://www.google.com/imgres?imgurl=&amp;imgrefurl=http%3A%2F%2Fwww.abcam.com%2Findex.html%3Fpageconfig%3Dresource%26rid%3D10602&amp;h=0&amp;w=0&amp;tbnid=6ZKuhlAsT2VZXM&amp;zoom=1&amp;tbnh=184&amp;tbnw=274&amp;docid=ETqKf7RmabxHnM&amp;tbm=isch&amp;ei=IU2JU6fSDKmysQSx34BQ&amp;ved=0CAIQsCUoAA)</b>.</div></p></div><div class="aux"><div class="resc">Hong You, et al. Front Genet. 2014;5:176.</div><p class="links"><a class="dblinks" href="/pubmed/24966871" ref="ordinalpos=1853&amp;ncbi_uid=6291370&amp;link_uid=24966871">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052099/" ref="ordinalpos=1853&amp;ncbi_uid=6291370&amp;link_uid=PMC4052099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1854.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3616967/figure/f4/" ref="ordinalpos=1854&amp;ncbi_uid=4798675&amp;link_uid=PMC3616967" href="/pmc/articles/PMC3616967/"><img src="/pmc/articles/instance/3616967/bin/ehp.1205659.g004.gif" src-large="/pmc/articles/instance/3616967/bin/ehp.1205659.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3616967/" ref="ordinalpos=1854&amp;ncbi_uid=4798675&amp;link_uid=PMC3616967" image-link="/pmc/articles/PMC3616967/figure/f4/" class="imagepopup">Figure 4.  From: Systems Biology and Birth Defects Prevention: Blockade of the Glucocorticoid Receptor Prevents Arsenic-Induced Birth Defects. </a></p><div class="supp"><p class="details"><div>Proposed model for iAs-induced birth defects via the GR <span class="highlight" style="background-color:">pathway</span>. iAs appears to induce birth defects in the chick embryo as a result of <span class="highlight" style="background-color:">signaling</span> through the GR [also known as NR3C1 (nuclear receptor subfamily 3, group C, member 1)] <span class="highlight" style="background-color:">pathway</span>. Gene members of the GR <span class="highlight" style="background-color:">pathway</span> are shown in red (associated with embryonic development) or green (associated with developmental disorders). In embryos with inhibited <span class="highlight" style="background-color:">signaling</span> of the GR <span class="highlight" style="background-color:">pathway</span> via cortexolone, the iAs-induced birth defects are prevented.</div></p></div><div class="aux"><div class="resc">Bhavesh K. Ahir, et al. Environ Health Perspect. 2013 March;121(3):332-338.</div><p class="links"><a class="dblinks" href="/pubmed/23458687" ref="ordinalpos=1854&amp;ncbi_uid=4798675&amp;link_uid=23458687">Citation</a><a class="dblinks" href="/pmc/articles/PMC3616967/" ref="ordinalpos=1854&amp;ncbi_uid=4798675&amp;link_uid=PMC3616967">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1855.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479345/figure/F4/" ref="ordinalpos=1855&amp;ncbi_uid=713499&amp;link_uid=PMC1479345" href="/pmc/articles/PMC1479345/"><img src="/pmc/articles/instance/1479345/bin/1476-511X-5-10-4.gif" src-large="/pmc/articles/instance/1479345/bin/1476-511X-5-10-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479345/" ref="ordinalpos=1855&amp;ncbi_uid=713499&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F4/" class="imagepopup">Figure 4.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. Figures 3-7 represent <span class="highlight" style="background-color:">signaling</span> pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following <span class="highlight" style="background-color:">signaling</span> cascades are shown: JUN, TNF, and CDKN2A <span class="highlight" style="background-color:">signaling</span> cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell <span class="highlight" style="background-color:">signaling</span>; gene expression; and cell cycle (Fig. 5); INS1/hRAS <span class="highlight" style="background-color:">signaling</span> cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC <span class="highlight" style="background-color:">signaling</span> cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the <span class="highlight" style="background-color:">signaling</span> diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/<span class="highlight" style="background-color:">signaling</span> categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></p></div><div class="aux"><div class="resc">Alvin Berger, et al. Lipids Health Dis. 2006;5:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/16623957" ref="ordinalpos=1855&amp;ncbi_uid=713499&amp;link_uid=16623957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479345/" ref="ordinalpos=1855&amp;ncbi_uid=713499&amp;link_uid=PMC1479345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1856.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1479345/figure/F6/" ref="ordinalpos=1856&amp;ncbi_uid=713494&amp;link_uid=PMC1479345" href="/pmc/articles/PMC1479345/"><img src="/pmc/articles/instance/1479345/bin/1476-511X-5-10-6.gif" src-large="/pmc/articles/instance/1479345/bin/1476-511X-5-10-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1479345/" ref="ordinalpos=1856&amp;ncbi_uid=713494&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F6/" class="imagepopup">Figure 6.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis. Figures 3-7 represent <span class="highlight" style="background-color:">signaling</span> pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following <span class="highlight" style="background-color:">signaling</span> cascades are shown: JUN, TNF, and CDKN2A <span class="highlight" style="background-color:">signaling</span> cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 <span class="highlight" style="background-color:">signaling</span> cascade affecting: cell <span class="highlight" style="background-color:">signaling</span>; gene expression; and cell cycle (Fig. 5); INS1/hRAS <span class="highlight" style="background-color:">signaling</span> cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC <span class="highlight" style="background-color:">signaling</span> cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the <span class="highlight" style="background-color:">signaling</span> diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/<span class="highlight" style="background-color:">signaling</span> categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></p></div><div class="aux"><div class="resc">Alvin Berger, et al. Lipids Health Dis. 2006;5:10-10.</div><p class="links"><a class="dblinks" href="/pubmed/16623957" ref="ordinalpos=1856&amp;ncbi_uid=713494&amp;link_uid=16623957">Citation</a><a class="dblinks" href="/pmc/articles/PMC1479345/" ref="ordinalpos=1856&amp;ncbi_uid=713494&amp;link_uid=PMC1479345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1857.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4100156/figure/ijms-15-10365-f003/" ref="ordinalpos=1857&amp;ncbi_uid=6382271&amp;link_uid=PMC4100156" href="/pmc/articles/PMC4100156/"><img src="/pmc/articles/instance/4100156/bin/ijms-15-10365-g003.gif" src-large="/pmc/articles/instance/4100156/bin/ijms-15-10365-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4100156/" ref="ordinalpos=1857&amp;ncbi_uid=6382271&amp;link_uid=PMC4100156" image-link="/pmc/articles/PMC4100156/figure/ijms-15-10365-f003/" class="imagepopup">Figure 3.  From: Up-Regulation of Rhoa/Rho Kinase <span class="highlight" style="background-color:">Pathway</span> by Translationally Controlled Tumor Protein in Vascular Smooth Muscle Cells. </a></p><div class="supp"><p class="details"><div>Role of TCTP in the RhoA-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in TCTP<sup>+/−</sup> mice. Representative photomicrographs indicating immunoreactivity of TCTP, RhoA, and p-MLC in the sections of aortic tissues from WT and TCTP<sup>+/−</sup> mice. IHC was performed using anti-TCTP, -RhoA, and p-MLC antibodies (<i>N</i> = 4–5).</div></p></div><div class="aux"><div class="resc">Jeehye Maeng, et al. Int J Mol Sci. 2014 June;15(6):10365-10376.</div><p class="links"><a class="dblinks" href="/pubmed/24918292" ref="ordinalpos=1857&amp;ncbi_uid=6382271&amp;link_uid=24918292">Citation</a><a class="dblinks" href="/pmc/articles/PMC4100156/" ref="ordinalpos=1857&amp;ncbi_uid=6382271&amp;link_uid=PMC4100156">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1858.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2654730/figure/F4/" ref="ordinalpos=1858&amp;ncbi_uid=1795082&amp;link_uid=PMC2654730" href="/pmc/articles/PMC2654730/"><img src="/pmc/articles/instance/2654730/bin/zee0030945770004.gif" src-large="/pmc/articles/instance/2654730/bin/zee0030945770004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2654730/" ref="ordinalpos=1858&amp;ncbi_uid=1795082&amp;link_uid=PMC2654730" image-link="/pmc/articles/PMC2654730/figure/F4/" class="imagepopup">Figure 4.  From: Regulation of Myogenic Differentiation by Androgens: Cross Talk between Androgen Receptor/ ?-Catenin and Follistatin/Transforming Growth Factor-? <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>Regulation of TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes by testosterone treatment. A, C3H 10T1/2 cells were treated with (+T) or without 100 n<span style="font-variant: small-caps" class="small-caps">m</span> testosterone (Con) for 3 h. Total cellular RNA was isolated, and mouse TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> genes were analyzed by SuperArray. <i>Open circular spots</i> represent some of the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target genes, which were regulated by testosterone treatment. The experiment was repeated two times, and a representative array is shown. B, Relative changes in some TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> target genes after normalization with GAPDH intensity. C, Verification of SuperArray data by RT and quantitative real-time PCR analysis. Total RNA isolated from C3H 10T1/2 cells treated with medium alone (con, <i>black bars</i>) or 100 n<span style="font-variant: small-caps" class="small-caps">m</span> testosterone (+T, <i>white bars</i>) were analyzed by RT and real-time PCR, using gene-specific primers for FKBP, BMP10, SMAD7, and FST. Three independent experiments were performed, and data shown are mean ± <span style="font-variant: small-caps" class="small-caps">sem</span>. Statistical analysis was performed by Student’s <i>t</i> test. **, <i>P</i> ≤ 0.01 compared with control group.</div></p></div><div class="aux"><div class="resc">Rajan Singh, et al. Endocrinology. 2009 March;150(3):1259-1268.</div><p class="links"><a class="dblinks" href="/pubmed/18948405" ref="ordinalpos=1858&amp;ncbi_uid=1795082&amp;link_uid=18948405">Citation</a><a class="dblinks" href="/pmc/articles/PMC2654730/" ref="ordinalpos=1858&amp;ncbi_uid=1795082&amp;link_uid=PMC2654730">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1859.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3931971/figure/Fig1/" ref="ordinalpos=1859&amp;ncbi_uid=5875334&amp;link_uid=PMC3931971" href="/pmc/articles/PMC3931971/"><img src="/pmc/articles/instance/3931971/bin/13671_2014_69_Fig1_HTML.gif" src-large="/pmc/articles/instance/3931971/bin/13671_2014_69_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3931971/" ref="ordinalpos=1859&amp;ncbi_uid=5875334&amp;link_uid=PMC3931971" image-link="/pmc/articles/PMC3931971/figure/Fig1/" class="imagepopup">Fig. 1.  From: Advanced Basal Cell Carcinoma: Epidemiology and Therapeutic Innovations. </a></p><div class="supp"><p class="details"><div>Simplified schematic of common mutations in the Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to basal cell carcinoma from <span class="highlight" style="background-color:">pathway</span> activation (<b>a</b>), and therapeutic targets of this <span class="highlight" style="background-color:">pathway</span> (<b>b</b>)</div></p></div><div class="aux"><div class="resc">Shalini V. Mohan, et al. Curr Dermatol Rep. 2014;3(1):40-45.</div><p class="links"><a class="dblinks" href="/pubmed/24587976" ref="ordinalpos=1859&amp;ncbi_uid=5875334&amp;link_uid=24587976">Citation</a><a class="dblinks" href="/pmc/articles/PMC3931971/" ref="ordinalpos=1859&amp;ncbi_uid=5875334&amp;link_uid=PMC3931971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1860.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4103951/figure/F3/" ref="ordinalpos=1860&amp;ncbi_uid=6393296&amp;link_uid=PMC4103951" href="/pmc/articles/PMC4103951/"><img src="/pmc/articles/instance/4103951/bin/nihms-601821-f0003.gif" src-large="/pmc/articles/instance/4103951/bin/nihms-601821-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4103951/" ref="ordinalpos=1860&amp;ncbi_uid=6393296&amp;link_uid=PMC4103951" image-link="/pmc/articles/PMC4103951/figure/F3/" class="imagepopup">Fig. 3.  From: Disruption of the Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Contributes to the Hair Follicle Cycling Deficiency in Vdr Knockout Mice. </a></p><div class="supp"><p class="details"><div>Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> agonist stimulates hair growth in wild-type and VdrKO but not in Rhino mice. Five to 11 weeks old mice were shaved and treated daily for 8 consecutive days with either Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> agonist or vehicle solution. Both wild-type and VdrKO mice display hair growth following treatment with the agonist where as vehicle wild-type and agonist treated Rhinomice do not.Pictures were taken at the beginning of the treatment (day 1), after the end of the treatment (day 9) and at subsequent days 15 and 22 to monitor hair growth.</div></p></div><div class="aux"><div class="resc">ARNAUD TEICHERT, et al. J Cell Physiol. ;225(2):482-489.</div><p class="links"><a class="dblinks" href="/pubmed/20458748" ref="ordinalpos=1860&amp;ncbi_uid=6393296&amp;link_uid=20458748">Citation</a><a class="dblinks" href="/pmc/articles/PMC4103951/" ref="ordinalpos=1860&amp;ncbi_uid=6393296&amp;link_uid=PMC4103951">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1861.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3349800/figure/F3/" ref="ordinalpos=1861&amp;ncbi_uid=4850274&amp;link_uid=PMC3349800" href="/pmc/articles/PMC3349800/"><img src="/pmc/articles/instance/3349800/bin/nihms-371655-f0003.gif" src-large="/pmc/articles/instance/3349800/bin/nihms-371655-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3349800/" ref="ordinalpos=1861&amp;ncbi_uid=4850274&amp;link_uid=PMC3349800" image-link="/pmc/articles/PMC3349800/figure/F3/" class="imagepopup">Fig. 3. Effects of resveratrol on the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The anticancer effects of resveratrol - Modulation of transcription factors. </a></p><div class="supp"><p class="details"><div>Activation by cytokine results in IKK phosphorylation of IκBα, resulting in IκBα degradation, and facilitation of NF-κB nuclear translocation and subsequent activation of transcription (49). Resveratrol is demonstrated it inhibit NF-κB <span class="highlight" style="background-color:">signaling</span> at all steps (see text for references).</div></p></div><div class="aux"><div class="resc">Nichelle C. Whitlock, et al. Nutr Cancer. ;64(4):493-502.</div><p class="links"><a class="dblinks" href="/pubmed/22482424" ref="ordinalpos=1861&amp;ncbi_uid=4850274&amp;link_uid=22482424">Citation</a><a class="dblinks" href="/pmc/articles/PMC3349800/" ref="ordinalpos=1861&amp;ncbi_uid=4850274&amp;link_uid=PMC3349800">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1862.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4377021/figure/Fig5/" ref="ordinalpos=1862&amp;ncbi_uid=7340266&amp;link_uid=PMC4377021" href="/pmc/articles/PMC4377021/"><img src="/pmc/articles/instance/4377021/bin/12864_2015_1372_Fig5_HTML.gif" src-large="/pmc/articles/instance/4377021/bin/12864_2015_1372_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4377021/" ref="ordinalpos=1862&amp;ncbi_uid=7340266&amp;link_uid=PMC4377021" image-link="/pmc/articles/PMC4377021/figure/Fig5/" class="imagepopup">Figure 5.  From: Genome-wide expression analysis suggests a crucial role of dysregulation of matrix metalloproteinases <span class="highlight" style="background-color:">pathway</span> in undifferentiated thyroid carcinoma. </a></p><div class="supp"><p class="details"><div> <b><span class="highlight" style="background-color:">Pathway</span> for a network related to cell to cell <span class="highlight" style="background-color:">signaling</span> process in the anaplastic thyroid carcinoma.</b> Please notice the overexpression of PLAU, PLAUR and SPP1.</div></p></div><div class="aux"><div class="resc">Jesús Espinal-Enríquez, et al. BMC Genomics. 2015;16(1):207.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1862&amp;ncbi_uid=7340266&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4377021/" ref="ordinalpos=1862&amp;ncbi_uid=7340266&amp;link_uid=PMC4377021">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1863.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3493907/figure/F7/" ref="ordinalpos=1863&amp;ncbi_uid=5462807&amp;link_uid=PMC3493907" href="/pmc/articles/PMC3493907/"><img src="/pmc/articles/instance/3493907/bin/zbc0491230640007.gif" src-large="/pmc/articles/instance/3493907/bin/zbc0491230640007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3493907/" ref="ordinalpos=1863&amp;ncbi_uid=5462807&amp;link_uid=PMC3493907" image-link="/pmc/articles/PMC3493907/figure/F7/" class="imagepopup">FIGURE 7. From: Withdrawal of Essential Amino Acids Increases Autophagy by a <span class="highlight" style="background-color:">Pathway</span> Involving Ca2+/Calmodulin-dependent Kinase Kinase-? (CaMKK-?). </a></p><div class="supp"><p class="details"><div><b>Suggested Ca<sup>2+</sup>-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from amino acid starvation to autophagy.</b> Detection methods/experimental approaches and assays/tools used in this study to monitor the indicated events are shown, respectively, on the <i>left</i> (marked in <i>blue</i>) and on the right (marked in <i>red</i>).</div></p></div><div class="aux"><div class="resc">Ghita Ghislat, et al. J Biol Chem. 2012 November 9;287(46):38625-38636.</div><p class="links"><a class="dblinks" href="/pubmed/23027865" ref="ordinalpos=1863&amp;ncbi_uid=5462807&amp;link_uid=23027865">Citation</a><a class="dblinks" href="/pmc/articles/PMC3493907/" ref="ordinalpos=1863&amp;ncbi_uid=5462807&amp;link_uid=PMC3493907">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1864.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949398/figure/pone-0013180-g002/" ref="ordinalpos=1864&amp;ncbi_uid=2519735&amp;link_uid=PMC2949398" href="/pmc/articles/PMC2949398/"><img src="/pmc/articles/instance/2949398/bin/pone.0013180.g002.gif" src-large="/pmc/articles/instance/2949398/bin/pone.0013180.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949398/" ref="ordinalpos=1864&amp;ncbi_uid=2519735&amp;link_uid=PMC2949398" image-link="/pmc/articles/PMC2949398/figure/pone-0013180-g002/" class="imagepopup">Figure 2. (A) Significance of cancer genes with co-occurring mutations existing in the same <span class="highlight" style="background-color:">signaling</span> pathways..  From: A Network of Cancer Genes with Co-Occurring and Anti-Co-Occurring Mutations. </a></p><div class="supp"><p class="details"><div>The red arrow indicates the real number of cancer gene pairs that exist in the same <span class="highlight" style="background-color:">pathway</span>. The curve is the distribution of the random number of cancer gene pairs that exist in the same <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Distribution of number of activation and repression signals among cancer gene pairs with co-occurring mutations. The red circle (pointed by a red arrow) indicates the real number of activation signals and repression signals that exist among the cancer gene pairs in the human <span class="highlight" style="background-color:">signaling</span> network. The heatmap indicates the union distribution of the random number of activation signals and repression signals that exist among the cancer gene pairs in the human <span class="highlight" style="background-color:">signaling</span> network. (<b>C</b>) Correlation between the distance of cancer genes in the CCA network and their expression similarity.</div></p></div><div class="aux"><div class="resc">Qinghua Cui. PLoS One. 2010;5(10):e13180.</div><p class="links"><a class="dblinks" href="/pubmed/20957180" ref="ordinalpos=1864&amp;ncbi_uid=2519735&amp;link_uid=20957180">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949398/" ref="ordinalpos=1864&amp;ncbi_uid=2519735&amp;link_uid=PMC2949398">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1865.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3092699/figure/F5/" ref="ordinalpos=1865&amp;ncbi_uid=2945963&amp;link_uid=PMC3092699" href="/pmc/articles/PMC3092699/"><img src="/pmc/articles/instance/3092699/bin/nihms289536f5.gif" src-large="/pmc/articles/instance/3092699/bin/nihms289536f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3092699/" ref="ordinalpos=1865&amp;ncbi_uid=2945963&amp;link_uid=PMC3092699" image-link="/pmc/articles/PMC3092699/figure/F5/" class="imagepopup">Figure 5. A differentiation-stage dependency of GM-CSF cell <span class="highlight" style="background-color:">signaling</span> model.  From: MOLECULAR CROSSTALK BETWEEN p70S6K AND MAPK CELL <span class="highlight" style="background-color:">SIGNALING</span> PATHWAYS. </a></p><div class="supp"><p class="details"><div>GM-CSF activates the Ras/MAPK and the PI3-K/p70S6K pathways in hematopoietic cells, but at different rates. In rapidly dividing cells (e.g., myeloid progenitors, leukemic cells) the former <span class="highlight" style="background-color:">pathway</span> is favored, while in mature cells (e.g., neutrophils) the latter <span class="highlight" style="background-color:">pathway</span> is more relevant. According to the data presented here, the MAPK <span class="highlight" style="background-color:">pathway</span> exerts a positive feedback on the p70S6K <span class="highlight" style="background-color:">pathway</span>, leading to an increased chemotaxis response in terminally differentiated cells. This molecular crosstalk becomes important in cells incapable of mitotic division (such as the neutrophil).</div></p></div><div class="aux"><div class="resc">Jason A. Lehman, et al. Biochem Biophys Res Commun. ;293(1):463-469.</div><p class="links"><a class="dblinks" href="/pubmed/12054624" ref="ordinalpos=1865&amp;ncbi_uid=2945963&amp;link_uid=12054624">Citation</a><a class="dblinks" href="/pmc/articles/PMC3092699/" ref="ordinalpos=1865&amp;ncbi_uid=2945963&amp;link_uid=PMC3092699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1866.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2717887/figure/F2/" ref="ordinalpos=1866&amp;ncbi_uid=2003372&amp;link_uid=PMC2717887" href="/pmc/articles/PMC2717887/"><img src="/pmc/articles/instance/2717887/bin/zee0080949230002.gif" src-large="/pmc/articles/instance/2717887/bin/zee0080949230002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2717887/" ref="ordinalpos=1866&amp;ncbi_uid=2003372&amp;link_uid=PMC2717887" image-link="/pmc/articles/PMC2717887/figure/F2/" class="imagepopup">Figure 2.  From: A <span class="highlight" style="background-color:">Signaling</span> Network in Phenylephrine-Induced Benign Prostatic Hyperplasia. </a></p><div class="supp"><p class="details"><div>Clusterin as a critical <span class="highlight" style="background-color:">signaling</span> node. A, Candidate genes identified by DNA microarray were analyzed with the Ingenuity software tool to construct an objective <span class="highlight" style="background-color:">signaling</span> network. Up-regulated genes in the PE-treated rats are shown in <i>red</i>; down-regulated genes are depicted in <i>green</i>. B and C, Analysis of the network demonstrates a link between clusterin and the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Components of the network are listed in B with fold change compared with control. Regulatory network showing potential interactions between molecules is shown in C.</div></p></div><div class="aux"><div class="resc">Jayoung Kim, et al. Endocrinology. 2009 August;150(8):3576-3583.</div><p class="links"><a class="dblinks" href="/pubmed/19443575" ref="ordinalpos=1866&amp;ncbi_uid=2003372&amp;link_uid=19443575">Citation</a><a class="dblinks" href="/pmc/articles/PMC2717887/" ref="ordinalpos=1866&amp;ncbi_uid=2003372&amp;link_uid=PMC2717887">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1867.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2966904/figure/f7-0951832/" ref="ordinalpos=1867&amp;ncbi_uid=2560570&amp;link_uid=PMC2966904" href="/pmc/articles/PMC2966904/"><img src="/pmc/articles/instance/2966904/bin/951832.fig7.gif" src-large="/pmc/articles/instance/2966904/bin/951832.fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2966904/" ref="ordinalpos=1867&amp;ncbi_uid=2560570&amp;link_uid=PMC2966904" image-link="/pmc/articles/PMC2966904/figure/f7-0951832/" class="imagepopup">Figure 7. From: Transferrin receptor 2 and HFE regulate furin expression via mitogen-activated protein kinase/extracellular signal-regulated kinase (MAPK/Erk) <span class="highlight" style="background-color:">signaling</span>. Implications for transferrin-dependent hepcidin regulation. </a></p><div class="supp"><p class="details"><div>Proposed scheme of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by TfR2 and HFE. Holotransferrin by binding to TfR2 in a complex with HFE, induces Erk1/2 phosphorylation. This, in turn, induces furin expression possibly acting also on the SMAD1/5/8 <span class="highlight" style="background-color:">pathway</span>. Furin participates in the maturation of hepcidin, and of the BMP, which induce hepcidin expression. It also produces the soluble form of HJV, which has an inhibitory effect on hepcidin expression.</div></p></div><div class="aux"><div class="resc">Maura Poli, et al. Haematologica. 2010 November;95(11):1832-1840.</div><p class="links"><a class="dblinks" href="/pubmed/20634490" ref="ordinalpos=1867&amp;ncbi_uid=2560570&amp;link_uid=20634490">Citation</a><a class="dblinks" href="/pmc/articles/PMC2966904/" ref="ordinalpos=1867&amp;ncbi_uid=2560570&amp;link_uid=PMC2966904">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1868.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC24169/figure/F5/" ref="ordinalpos=1868&amp;ncbi_uid=155969&amp;link_uid=PMC24169" href="/pmc/articles/PMC24169/"><img src="/pmc/articles/instance/24169/bin/pq2392867005.gif" src-large="/pmc/articles/instance/24169/bin/pq2392867005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC24169/" ref="ordinalpos=1868&amp;ncbi_uid=155969&amp;link_uid=PMC24169" image-link="/pmc/articles/PMC24169/figure/F5/" class="imagepopup">Figure 5.  From: Ligation of intestinal epithelial CD1d induces bioactive IL-10: Critical role of the cytoplasmic tail in autocrine <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Proposed model of CD1d <span class="highlight" style="background-color:">signaling</span> in polarized intestinal epithelia. Results from these studies define a signal transduction <span class="highlight" style="background-color:">pathway</span> through surface activation of epithelial CD1d and illustrate that this <span class="highlight" style="background-color:">pathway</span> may serve to dampen proinflammatory signals at the level of mucosal epithelia through the autocrine actions of epithelial-cell-derived IL-10.</div></p></div><div class="aux"><div class="resc">Sean P. Colgan, et al. Proc Natl Acad Sci U S A. 1999 November 23;96(24):13938-13943.</div><p class="links"><a class="dblinks" href="/pubmed/10570177" ref="ordinalpos=1868&amp;ncbi_uid=155969&amp;link_uid=10570177">Citation</a><a class="dblinks" href="/pmc/articles/PMC24169/" ref="ordinalpos=1868&amp;ncbi_uid=155969&amp;link_uid=PMC24169">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1869.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4356078/figure/F5/" ref="ordinalpos=1869&amp;ncbi_uid=7326499&amp;link_uid=PMC4356078" href="/pmc/articles/PMC4356078/"><img src="/pmc/articles/instance/4356078/bin/fncel-09-00090-g0005.gif" src-large="/pmc/articles/instance/4356078/bin/fncel-09-00090-g0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4356078/" ref="ordinalpos=1869&amp;ncbi_uid=7326499&amp;link_uid=PMC4356078" image-link="/pmc/articles/PMC4356078/figure/F5/" class="imagepopup">Figure 5.  From: Caloric restriction protects against electrical kindling of the amygdala by inhibiting the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Phosphorylation of proteins participating in the mechanistic target of rapamycin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the neocortex (Cx) and hippocampus (Hp) of animals allowed to feed <i>ad libitum</i> (AL) or subjected to a 15% caloric restriction (CR). (A)</b> Expression of phospho-AMPK against total AMPK. <b>(B)</b> Proportion of phospho-PKB vs. total PKB. <b>(C)</b> Phospho-S6 to total S6 ratio. <sup>*</sup><i>p</i> ≤ 0.05, <sup>**</sup><i>p</i> ≤ 0.01, AL vs. CR.</div></p></div><div class="aux"><div class="resc">Bryan V. Phillips-Farfán, et al. Front Cell Neurosci. 2015;9:90.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1869&amp;ncbi_uid=7326499&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4356078/" ref="ordinalpos=1869&amp;ncbi_uid=7326499&amp;link_uid=PMC4356078">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1870.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1759898/figure/F3/" ref="ordinalpos=1870&amp;ncbi_uid=774078&amp;link_uid=PMC1759898" href="/pmc/articles/PMC1759898/"><img src="/pmc/articles/instance/1759898/bin/43fig3.gif" src-large="/pmc/articles/instance/1759898/bin/43fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1759898/" ref="ordinalpos=1870&amp;ncbi_uid=774078&amp;link_uid=PMC1759898" image-link="/pmc/articles/PMC1759898/figure/F3/" class="imagepopup">Figure 3. From: The DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is a critical mediator of oncogene-induced senescence. </a></p><div class="supp"><p class="details"><div>Characterization of the DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in STAT5A-induced senescence. (<i>A</i>) Colocalization 53BP1 foci with phospho-ATM<sup>S1981</sup> in STAT5A1*6-expressing cells by confocal immunofluorescence. (<i>B</i>) Colocalization γH2AX foci with phospho-S/TQ epitopes in STAT5A1*6-expressing cells by confocal immunofluorescence. (<i>C</i>) Quantitation of DNA damage foci; asterisk represents a <i>p</i> &lt; 0.001 using the Student’s <i>t</i>-test. (<i>D</i>) Immunoblots for Chk1 and Chk2 phosphorylation. (V) Vector; (R) RasV12; (5A) STAT5A. (<i>E</i>) ATM is required for phosphorylation of Chk2 at Thr 68 in ca-STAT5A-induced senescence.</div></p></div><div class="aux"><div class="resc">Frédérick A. Mallette, et al. Genes Dev. 2007 January 1;21(1):43-48.</div><p class="links"><a class="dblinks" href="/pubmed/17210786" ref="ordinalpos=1870&amp;ncbi_uid=774078&amp;link_uid=17210786">Citation</a><a class="dblinks" href="/pmc/articles/PMC1759898/" ref="ordinalpos=1870&amp;ncbi_uid=774078&amp;link_uid=PMC1759898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1871.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3734838/figure/fig6/" ref="ordinalpos=1871&amp;ncbi_uid=5171605&amp;link_uid=PMC3734838" href="/pmc/articles/PMC3734838/"><img src="/pmc/articles/instance/3734838/bin/cddis20133f6.gif" src-large="/pmc/articles/instance/3734838/bin/cddis20133f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3734838/" ref="ordinalpos=1871&amp;ncbi_uid=5171605&amp;link_uid=PMC3734838" image-link="/pmc/articles/PMC3734838/figure/fig6/" class="imagepopup">Figure 6.  From: PTEN-regulated AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> contributes to reactive oxygen species-mediated apoptosis in selenite-treated colorectal cancer cells. </a></p><div class="supp"><p class="details"><div>A schematic illustration delineating the role of the AKT/Foxo3a/Bim <span class="highlight" style="background-color:">pathway</span> in selenite-induced apoptosis of CRC cells. Selenite treatment inhibited Src/PI3K/PDK1/AKT <span class="highlight" style="background-color:">pathway</span> and accumulation of FoxO3a in the nucleus possibly through induced ROS in CRC cells. On one hand, upregulated FoxO3a enhanced the transcription and expression of bim, which led to apoptosis. Additionally, PTEN was also upregulated by FoxO3a, indicating further regulation of AKT/FoxO3a/Bim <span class="highlight" style="background-color:">signaling</span> and apoptosis in response to selenite</div></p></div><div class="aux"><div class="resc">H Luo, et al. Cell Death Dis. 2013 February;4(2):e481.</div><p class="links"><a class="dblinks" href="/pubmed/23392169" ref="ordinalpos=1871&amp;ncbi_uid=5171605&amp;link_uid=23392169">Citation</a><a class="dblinks" href="/pmc/articles/PMC3734838/" ref="ordinalpos=1871&amp;ncbi_uid=5171605&amp;link_uid=PMC3734838">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1872.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3997674/figure/f1-ol-07-05-1327/" ref="ordinalpos=1872&amp;ncbi_uid=6053801&amp;link_uid=PMC3997674" href="/pmc/articles/PMC3997674/"><img src="/pmc/articles/instance/3997674/bin/OL-07-05-1327-g00.gif" src-large="/pmc/articles/instance/3997674/bin/OL-07-05-1327-g00.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3997674/" ref="ordinalpos=1872&amp;ncbi_uid=6053801&amp;link_uid=PMC3997674" image-link="/pmc/articles/PMC3997674/figure/f1-ol-07-05-1327/" class="imagepopup">Figure 1.  From: TRAIL combinations: The new 'trail' for cancer therapy (Review). </a></p><div class="supp"><p class="details"><div>Dual opposing <span class="highlight" style="background-color:">signaling</span> pathways of TRAIL. (A) The apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (B) The resistance <span class="highlight" style="background-color:">pathway</span> developed against TRAIL-induced apoptosis. TRAIL, tumor necrosis factor-related apoptosis-inducing ligand.</div></p></div><div class="aux"><div class="resc">ALAA REFAAT, et al. Oncol Lett. 2014 May;7(5):1327-1332.</div><p class="links"><a class="dblinks" href="/pubmed/24765133" ref="ordinalpos=1872&amp;ncbi_uid=6053801&amp;link_uid=24765133">Citation</a><a class="dblinks" href="/pmc/articles/PMC3997674/" ref="ordinalpos=1872&amp;ncbi_uid=6053801&amp;link_uid=PMC3997674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1873.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3845556/figure/cjc-32-05-253-g001/" ref="ordinalpos=1873&amp;ncbi_uid=5588254&amp;link_uid=PMC3845556" href="/pmc/articles/PMC3845556/"><img src="/pmc/articles/instance/3845556/bin/cjc-32-05-253-g001.gif" src-large="/pmc/articles/instance/3845556/bin/cjc-32-05-253-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3845556/" ref="ordinalpos=1873&amp;ncbi_uid=5588254&amp;link_uid=PMC3845556" image-link="/pmc/articles/PMC3845556/figure/cjc-32-05-253-g001/" class="imagepopup">Figure 1. Schematic diagram of <span class="highlight" style="background-color:">signaling</span> through the phosphoinositide 3-kinase-AKT (PI3K-AKT) <span class="highlight" style="background-color:">pathway</span> with current and future treatment strategies..  From: Targeting the PI3K-AKT-mTOR <span class="highlight" style="background-color:">signaling</span> network in cancer. </a></p><div class="supp"><p class="details"><div>PI3K <span class="highlight" style="background-color:">signaling</span> impacts on cell growth, survival, differentiation, and proliferation. The pointed arrows represent activation of substrates, while blunt arrows represent inhibition. Activation of membrane kinases, such as insulin-like growth factor-1 (IGF1), by external growth factors can initiate activation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways. AKT is phosphorylated downstream of PI3K with various effects, including the activation of mammalian target of rapamycin (mTOR). mTOR phosphorylates p70S6 and 4E-binding protein 1 (4EBP-1), which then leads to an increased translation of mRNA that encodes several cell cycle regulators. These effects are controlled by the tuberous sclerosis 1 (TSC1)-TSC2 complex. The Ras-Raf-MEK-MAPK <span class="highlight" style="background-color:">pathway</span> has its own distinct downstream effects, but also converges with the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. Future combination regimens involving targeted agents against this <span class="highlight" style="background-color:">signaling</span> network include the concomitant or sequential blockade of these pathways. PTEN, phosphatase and tensin homolog; HIF-1α, hypoxia-inducible factor-1α; MAPK, mitogen-activated protein kinase.</div></p></div><div class="aux"><div class="resc">Khurum H. Khan, et al. Chin J Cancer. 2013 May;32(5):253-265.</div><p class="links"><a class="dblinks" href="/pubmed/23642907" ref="ordinalpos=1873&amp;ncbi_uid=5588254&amp;link_uid=23642907">Citation</a><a class="dblinks" href="/pmc/articles/PMC3845556/" ref="ordinalpos=1873&amp;ncbi_uid=5588254&amp;link_uid=PMC3845556">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1874.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g005/" ref="ordinalpos=1874&amp;ncbi_uid=5370220&amp;link_uid=PMC3789883" href="/pmc/articles/PMC3789883/"><img src="/pmc/articles/instance/3789883/bin/pgen.1003770.g005.gif" src-large="/pmc/articles/instance/3789883/bin/pgen.1003770.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3789883/" ref="ordinalpos=1874&amp;ncbi_uid=5370220&amp;link_uid=PMC3789883" image-link="/pmc/articles/PMC3789883/figure/pgen-1003770-g005/" class="imagepopup">Figure 5. Enrichment hypotheses with multiple enriched pathways show increased support from data..  From: Integrated Enrichment Analysis of Variants and Pathways in Genome-Wide Association Studies Indicates Central Role for IL-2 <span class="highlight" style="background-color:">Signaling</span> Genes in Type 1 Diabetes, and Cytokine <span class="highlight" style="background-color:">Signaling</span> Genes in Crohn's Disease. </a></p><div class="supp"><p class="details"><div>Each row gives <span class="highlight" style="background-color:">pathway</span>, or combination of 2 or 3 pathways, with largest BF for enrichment of disease associations. See Figure 1 for legend and abbreviations used. All enrichment hypotheses for RA and T1D shown here also include enrichment of the MHC, allowing for a different level of enrichment within the MHC. Unlike the BFs in Figures 1 and 2, BFs here are all defined relative to null hypothesis of no enrichment, so that they can be easily compared. Counts of genes and SNPs only include those that are not already assigned to other enriched pathways; for example, 37 genes belong to the IL-23 <span class="highlight" style="background-color:">pathway</span>, and of those 15 are already cytokine <span class="highlight" style="background-color:">signaling</span> genes, so inclusion of IL-23 <span class="highlight" style="background-color:">signaling</span> adds 22 more genes. Databases and database identifiers for pathways in this figure: “IL2-mediated <span class="highlight" style="background-color:">signaling</span> events” (PID il2_1pathway, BS 137976); “ErbB receptor <span class="highlight" style="background-color:">signaling</span> network” (PID erbb_network_<span class="highlight" style="background-color:">pathway</span>, BS 138016); “Inositol pyrophosphates biosynthesis” (HumanCyc 6369, PC); “Measles” (KEGG hsa05162, BS 213306); “Wnt” (Cancer Cell Map, PC); “Cytokine <span class="highlight" style="background-color:">signaling</span> in immune system” (Reactome 75790, BS 366171); “IL23-mediated <span class="highlight" style="background-color:">signaling</span> events” (PID il23pathway, BS 138000); “Methionine salvage <span class="highlight" style="background-color:">pathway</span>” (Reactome 75881, BS 366245).</div></p></div><div class="aux"><div class="resc">Peter Carbonetto, et al. PLoS Genet. 2013 October;9(10):e1003770.</div><p class="links"><a class="dblinks" href="/pubmed/24098138" ref="ordinalpos=1874&amp;ncbi_uid=5370220&amp;link_uid=24098138">Citation</a><a class="dblinks" href="/pmc/articles/PMC3789883/" ref="ordinalpos=1874&amp;ncbi_uid=5370220&amp;link_uid=PMC3789883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1875.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657316/figure/fig3/" ref="ordinalpos=1875&amp;ncbi_uid=2257572&amp;link_uid=PMC2657316" href="/pmc/articles/PMC2657316/"><img src="/pmc/articles/instance/2824233/bin/sgas44-462-f3.gif" src-large="/pmc/articles/instance/2824233/bin/sgas44-462-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657316/" ref="ordinalpos=1875&amp;ncbi_uid=2257572&amp;link_uid=PMC2657316" image-link="/pmc/articles/PMC2657316/figure/fig3/" class="imagepopup">Figure 3.  From: Possible regulation of genes associated with intracellular <span class="highlight" style="background-color:">signaling</span> cascade in rat liver regeneration. </a></p><div class="supp"><p class="details"><div>Expression dynamics and interaction of 54 intracellular <span class="highlight" style="background-color:">signaling</span> cascade-associated genes during rat liver regeneration (LR). The interactions of intracellular <span class="highlight" style="background-color:">signaling</span> cascade-associated genes were assayed by <span class="highlight" style="background-color:">pathway</span> studio 5.0 software. Red, green, gray and white shapes denote the up-regulated, the down-regulated, the up/down-regulated and the meaninglessly expressed genes, respectively. A. The interactions of 54 genes with closer relationships. B. Expression changes at each phase during LR.</div></p></div><div class="aux"><div class="resc">Cun-Shuan Xu, et al. Scand J Gastroenterol. 2009 April;44(1):462-470.</div><p class="links"><a class="dblinks" href="/pubmed/18991167" ref="ordinalpos=1875&amp;ncbi_uid=2257572&amp;link_uid=18991167">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657316/" ref="ordinalpos=1875&amp;ncbi_uid=2257572&amp;link_uid=PMC2657316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1876.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3704530/figure/pone-0068451-g007/" ref="ordinalpos=1876&amp;ncbi_uid=5095637&amp;link_uid=PMC3704530" href="/pmc/articles/PMC3704530/"><img src="/pmc/articles/instance/3704530/bin/pone.0068451.g007.gif" src-large="/pmc/articles/instance/3704530/bin/pone.0068451.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3704530/" ref="ordinalpos=1876&amp;ncbi_uid=5095637&amp;link_uid=PMC3704530" image-link="/pmc/articles/PMC3704530/figure/pone-0068451-g007/" class="imagepopup">Figure 7. Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of pMSC derived-exosomes proteins..  From: Exosomal <span class="highlight" style="background-color:">Signaling</span> during Hypoxia Mediates Microvascular Endothelial Cell Migration and Vasculogenesis. </a></p><div class="supp"><p class="details"><div>(A) The Venn diagram depicts the distribution of common and unique proteins identified by nanospray LC-MS/MS (ABSciex 5600) in exosomes released from pMSC exposed to 1%, 3% and 8% oxygen. Comparison of canonical pathways: (B) actin cytoskeleton <span class="highlight" style="background-color:">signaling</span>, (C) growth hormone <span class="highlight" style="background-color:">signaling</span>, (D) VEGF <span class="highlight" style="background-color:">signaling</span>, and (E) clathrin-mediated endocytosis <span class="highlight" style="background-color:">signaling</span> identified by IPA core analysis. Values are mean ± SEM. In B, C, D and E, <i>*p</i>&lt;0.005 versus all condition.</div></p></div><div class="aux"><div class="resc">Carlos Salomon, et al. PLoS One. 2013;8(7):e68451.</div><p class="links"><a class="dblinks" href="/pubmed/23861904" ref="ordinalpos=1876&amp;ncbi_uid=5095637&amp;link_uid=23861904">Citation</a><a class="dblinks" href="/pmc/articles/PMC3704530/" ref="ordinalpos=1876&amp;ncbi_uid=5095637&amp;link_uid=PMC3704530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1877.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3426219/figure/F2/" ref="ordinalpos=1877&amp;ncbi_uid=4412902&amp;link_uid=PMC3426219" href="/pmc/articles/PMC3426219/"><img src="/pmc/articles/instance/3426219/bin/nihms394813f2.gif" src-large="/pmc/articles/instance/3426219/bin/nihms394813f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3426219/" ref="ordinalpos=1877&amp;ncbi_uid=4412902&amp;link_uid=PMC3426219" image-link="/pmc/articles/PMC3426219/figure/F2/" class="imagepopup">Figure 2.  From: CSPG4, a potential therapeutic target, facilitates malignant progression of melanoma. </a></p><div class="supp"><p class="details"><div>Chondroitin sulfate proteoglycan 4 (CSPG4) <span class="highlight" style="background-color:">Signaling</span> Pathways. CSPG4 functions to activate two major overlapping but distinct <span class="highlight" style="background-color:">signaling</span> cascades: integrin/focal adhesion kinase (FAK) <span class="highlight" style="background-color:">signaling</span> (A) and MAPK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> (B). Through these two branches, CSPG4 ultimately promotes tumor progression through a variety of cellular functions. (A) CSPG4 influences integrin function and <span class="highlight" style="background-color:">signaling</span> pathways. CSPG4 promotes Src-FAK complexing through its interaction with the scaffold protein Syntenin. This leads to FAK activation by Src, prompting a number of <span class="highlight" style="background-color:">signaling</span> cascades including FAK–integrin–extracellular matrix (ECM) complex assembly, activation of Rac through p130cas, and activation of the PI3K/AKT/NFκB <span class="highlight" style="background-color:">signaling</span> cascade; activation of these pathways mediate the effects of CSPG4 on cytoskeletal reorganization, survival, chemoresistance, and migration. Activation of MMPs by CSPG4 via direct binding to the CS chain leads to local invasion of cancer cells. CSPG4/NG2 can also enhance survival as a result of its constitutive activation of integrin-related signals. (B) CSPG4 promotes MAPK <span class="highlight" style="background-color:">signaling</span> through receptor tyrosine (RTK)-dependent and independent mechanisms. In human melanomas, CSPG4 impacts activation of the ERK 1,2 <span class="highlight" style="background-color:">pathway</span>, likely by impacting on the growth factor-induced activation of RTKs. Note the expression of BRAF<sup>V600E</sup> in this <span class="highlight" style="background-color:">pathway</span>, which results in the constitutive activation of this kinase (see text). In human melanoma cells expressing this BRAF-activating mutation, constitutive activation of the ERK 1,2 <span class="highlight" style="background-color:">pathway</span> requires the presence of CSPG4. There are several possible downstream oncogenic targets of the ERK 1,2 <span class="highlight" style="background-color:">pathway</span>, including MITF and activated c-Met, which are implicated in epithelial to mesenchymal transition. Inhibition of IKK by ERK 1,2 downstream of CSPG4 could also lead to cell survival and chemoresistance.</div></p></div><div class="aux"><div class="resc">Matthew A. Price, et al. Pigment Cell Melanoma Res. ;24(6):1148-1157.</div><p class="links"><a class="dblinks" href="/pubmed/22004131" ref="ordinalpos=1877&amp;ncbi_uid=4412902&amp;link_uid=22004131">Citation</a><a class="dblinks" href="/pmc/articles/PMC3426219/" ref="ordinalpos=1877&amp;ncbi_uid=4412902&amp;link_uid=PMC3426219">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1878.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC27070/figure/F1/" ref="ordinalpos=1878&amp;ncbi_uid=15208&amp;link_uid=PMC27070" href="/pmc/articles/PMC27070/"><img src="/pmc/articles/instance/27070/bin/pq1901337001.gif" src-large="/pmc/articles/instance/27070/bin/pq1901337001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC27070/" ref="ordinalpos=1878&amp;ncbi_uid=15208&amp;link_uid=PMC27070" image-link="/pmc/articles/PMC27070/figure/F1/" class="imagepopup">Figure 1.  From: A cGMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a subset of olfactory sensory neurons. </a></p><div class="supp"><p class="details"><div>Two alternative hypotheses of cGMP <span class="highlight" style="background-color:">signaling</span> in a subset of OSNs. (<i>Upper</i>) An odorant receptor (OR) activates a cAMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving a G protein (G<sub>olf</sub>), an adenylyl cyclase (ACIII), a cyclic nucleotide-gated (CNG) channel (α3α4β1b), and a chloride channel (ClC). The cAMP is degraded by a CaM-dependent phosphodiesterase (PDE1C2). (<i>Lower</i>) Components of a cGMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and putative targets of cGMP: receptor guanylyl cyclase GC-D; cGMP-regulated PDE2; an unknown cGMP-regulated ion channel; and the known CNG channel of the cAMP-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Mike R. Meyer, et al. Proc Natl Acad Sci U S A. 2000 September 12;97(19):10595-10600.</div><p class="links"><a class="dblinks" href="/pubmed/10984544" ref="ordinalpos=1878&amp;ncbi_uid=15208&amp;link_uid=10984544">Citation</a><a class="dblinks" href="/pmc/articles/PMC27070/" ref="ordinalpos=1878&amp;ncbi_uid=15208&amp;link_uid=PMC27070">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1879.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2515624/figure/F1/" ref="ordinalpos=1879&amp;ncbi_uid=1479192&amp;link_uid=PMC2515624" href="/pmc/articles/PMC2515624/"><img src="/pmc/articles/instance/2515624/bin/nihms56053f1.gif" src-large="/pmc/articles/instance/2515624/bin/nihms56053f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2515624/" ref="ordinalpos=1879&amp;ncbi_uid=1479192&amp;link_uid=PMC2515624" image-link="/pmc/articles/PMC2515624/figure/F1/" class="imagepopup">Fig. 1. Simplified diagram of Hh <span class="highlight" style="background-color:">signaling</span>.  From: Implications of hedgehog <span class="highlight" style="background-color:">signaling</span> antagonists for cancer therapy. </a></p><div class="supp"><p class="details"><div>SMO is the key signal transducer in the Hh <span class="highlight" style="background-color:">pathway</span>. In the absence of Hh, the Hh receptor PTC can suppress SMO activity and repressor forms of Gli (GliR) are generated, leading to downregulation of Hh target genes. In the presence of Hh ligands (Shh, Ihh or Dhh), PTC is unable to affect SMO <span class="highlight" style="background-color:">signaling</span>. SMO somehow can promote formation of activated forms of Gli (GliA), resulting in upregulation of Hh target genes. Gene mutations (PTC, SMO) or abnormal over-expression of Hh ligands all can lead to elevated expression of Hh target genes.</div></p></div><div class="aux"><div class="resc">Jingwu Xie. Acta Biochim Biophys Sin (Shanghai). ;40(7):670-680.</div><p class="links"><a class="dblinks" href="/pubmed/18604459" ref="ordinalpos=1879&amp;ncbi_uid=1479192&amp;link_uid=18604459">Citation</a><a class="dblinks" href="/pmc/articles/PMC2515624/" ref="ordinalpos=1879&amp;ncbi_uid=1479192&amp;link_uid=PMC2515624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1880.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2662467/figure/f6-ijms-10-00184/" ref="ordinalpos=1880&amp;ncbi_uid=1762269&amp;link_uid=PMC2662467" href="/pmc/articles/PMC2662467/"><img src="/pmc/articles/instance/2662467/bin/ijms-10-00184f6.gif" src-large="/pmc/articles/instance/2662467/bin/ijms-10-00184f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2662467/" ref="ordinalpos=1880&amp;ncbi_uid=1762269&amp;link_uid=PMC2662467" image-link="/pmc/articles/PMC2662467/figure/f6-ijms-10-00184/" class="imagepopup">Figure 6. From: Molecular Neuropathology of Gliomas. </a></p><div class="supp"><p class="details"><div>Core pathways involved in the pathogenesis of gliomas.<br />Note the interrelationships between p53, RB, growth factor receptor, PTEN/PI3K/AKT and RAS <span class="highlight" style="background-color:">signaling</span> on the regulation of cell proliferation and apoptosis. While <i>TP53</i> mutation, amplification of <i>MDM2/MDM4</i> or <i>p14</i><i><sup>ARF</sup></i> deletion/methylation inhibits apoptosis, alterations in p16<sup>INK4a</sup>, p15<sup>INK4b</sup>, p18<sup>INK4c</sup> and p21<sup>waf1</sup> disinhibit cell cycle progression at the G<sub>1</sub>/S-phase checkpoint via cyclin-dependent kinases by phosphorylation of RB1 and release of E2F transcription factors. Amplification, overexpression or mutation of growth factor receptors stimulates cell proliferation and inhibits apoptosis through both the RAS as well as the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The RAS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be alternatively activated by mutations in the <i>NF1</i>, the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by mutations in the <i>PTEN</i> gene and less commonly, the <i>PIK3CA</i> or the <i>PIK3R1</i> gene.</div></p></div><div class="aux"><div class="resc">Markus J. Riemenschneider, et al. Int J Mol Sci. 2009 January;10(1):184-212.</div><p class="links"><a class="dblinks" href="/pubmed/19333441" ref="ordinalpos=1880&amp;ncbi_uid=1762269&amp;link_uid=19333441">Citation</a><a class="dblinks" href="/pmc/articles/PMC2662467/" ref="ordinalpos=1880&amp;ncbi_uid=1762269&amp;link_uid=PMC2662467">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1881.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814921/figure/F4/" ref="ordinalpos=1881&amp;ncbi_uid=2732741&amp;link_uid=PMC2814921" href="/pmc/articles/PMC2814921/"><img src="/pmc/articles/instance/2814921/bin/nihms-160240-f0004.gif" src-large="/pmc/articles/instance/2814921/bin/nihms-160240-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814921/" ref="ordinalpos=1881&amp;ncbi_uid=2732741&amp;link_uid=PMC2814921" image-link="/pmc/articles/PMC2814921/figure/F4/" class="imagepopup">Figure 4.  From: THE EFFECT OF PROPRANOLOL ON GENE EXPRESSION DURING THE BLOOD ALCOHOL CYCLE OF RATS FED ETHANOL INTRAGASTRICALLY. </a></p><div class="supp"><p class="details"><div>KEGG functional pathways that were changed by feeding ethanol+propranolol compared with control rats are shown. The most pathways affected were the cytokine-cytokine receptor interaction, Jak-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the Mapk <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jun Li, et al. Exp Mol Pathol. ;88(1):32.</div><p class="links"><a class="dblinks" href="/pubmed/19925788" ref="ordinalpos=1881&amp;ncbi_uid=2732741&amp;link_uid=19925788">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814921/" ref="ordinalpos=1881&amp;ncbi_uid=2732741&amp;link_uid=PMC2814921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1882.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2537433/figure/F8/" ref="ordinalpos=1882&amp;ncbi_uid=1461140&amp;link_uid=PMC2537433" href="/pmc/articles/PMC2537433/"><img src="/pmc/articles/instance/2537433/bin/z380100870580008.gif" src-large="/pmc/articles/instance/2537433/bin/z380100870580008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2537433/" ref="ordinalpos=1882&amp;ncbi_uid=1461140&amp;link_uid=PMC2537433" image-link="/pmc/articles/PMC2537433/figure/F8/" class="imagepopup">Figure 8. From: Deletion of Go2? abolishes cocaine-induced behavioral sensitization by disturbing the striatal dopamine system. </a></p><div class="supp"><p class="details"><div>Effects of D1- and D2-receptor activation on motor activity. Dopamine enhances motor activity both by promoting the direct <span class="highlight" style="background-color:">pathway</span> <i>via</i> the stimulatory D1 receptor <span class="highlight" style="background-color:">signaling</span> and by slowing down the indirect <span class="highlight" style="background-color:">pathway</span> <i>via</i> the inhibitory D2-receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Irene Brunk, et al. FASEB J. 2008 October;22(10):3736-3746.</div><p class="links"><a class="dblinks" href="/pubmed/18606864" ref="ordinalpos=1882&amp;ncbi_uid=1461140&amp;link_uid=18606864">Citation</a><a class="dblinks" href="/pmc/articles/PMC2537433/" ref="ordinalpos=1882&amp;ncbi_uid=1461140&amp;link_uid=PMC2537433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1883.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4164261/figure/fig1/" ref="ordinalpos=1883&amp;ncbi_uid=6643694&amp;link_uid=PMC4164261" href="/pmc/articles/PMC4164261/"><img src="/pmc/articles/instance/4164261/bin/BMRI2014-759473.001.gif" src-large="/pmc/articles/instance/4164261/bin/BMRI2014-759473.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4164261/" ref="ordinalpos=1883&amp;ncbi_uid=6643694&amp;link_uid=PMC4164261" image-link="/pmc/articles/PMC4164261/figure/fig1/" class="imagepopup">Figure 1.  From: The Nerve Growth Factor <span class="highlight" style="background-color:">Signaling</span> and Its Potential as Therapeutic Target for Glaucoma. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> of NGF receptors. NGF is formed by cleavage from Pro-NGF, which is the precursor protein form of NGF. TrkA receptor is the high affinity receptor for NGF; NGF binding to TrkA causes the phosphorylation of TrkA and activation of multiple <span class="highlight" style="background-color:">signaling</span> pathways such as the PI3K/Akt, Ras/Raf/MEK/ERK1/2, or PLC<i>γ</i>/PKC <span class="highlight" style="background-color:">signaling</span> pathways. Activation of these pathways eventually leads to different biological functions including the prevention of apoptosis. The other NGF receptor, p75NTR, is a low affinity receptor. The precise role of p75NTR depends upon the cellular context; it can enhance cell survival through NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> or promote cell death through JNK/c-Jun signal <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Haitao Wang, et al. Biomed Res Int. 2014;2014:759473.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=1883&amp;ncbi_uid=6643694&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4164261/" ref="ordinalpos=1883&amp;ncbi_uid=6643694&amp;link_uid=PMC4164261">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1884.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3295657/figure/F6/" ref="ordinalpos=1884&amp;ncbi_uid=3844581&amp;link_uid=PMC3295657" href="/pmc/articles/PMC3295657/"><img src="/pmc/articles/instance/3295657/bin/2044-5040-2-3-6.gif" src-large="/pmc/articles/instance/3295657/bin/2044-5040-2-3-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3295657/" ref="ordinalpos=1884&amp;ncbi_uid=3844581&amp;link_uid=PMC3295657" image-link="/pmc/articles/PMC3295657/figure/F6/" class="imagepopup">Figure 6.  From: TAK-1/p38/nNF?B <span class="highlight" style="background-color:">signaling</span> inhibits myoblast differentiation by increasing levels of Activin A. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span> model for interleukin (IL)-1<b>α </b>and tumor necrosis factor (TNF)-<b>α</b>-induced inhibition of human skeletal muscle cell (HuSKMC) differentiation</b>. IL-1α and TNF-α induced secretion of Activin A in differentiating HuSKMCs cells and this secreted Activin A mediated their effects. Release of Activin A was dependent on <b>t</b>ransforming growth factor-β-activated kinase-1/p38/nuclear factor κB <span class="highlight" style="background-color:">pathway</span> activation, but independent of SMAD2/3 <span class="highlight" style="background-color:">signaling</span>. The released Activin A subsequently inhibited differentiation via ALK/SMAD2/3 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Anne Ulrike Trendelenburg, et al. Skelet Muscle. 2012;2:3-3.</div><p class="links"><a class="dblinks" href="/pubmed/22313861" ref="ordinalpos=1884&amp;ncbi_uid=3844581&amp;link_uid=22313861">Citation</a><a class="dblinks" href="/pmc/articles/PMC3295657/" ref="ordinalpos=1884&amp;ncbi_uid=3844581&amp;link_uid=PMC3295657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1885.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3887612/figure/F005/" ref="ordinalpos=1885&amp;ncbi_uid=5787842&amp;link_uid=PMC3887612" href="/pmc/articles/PMC3887612/"><img src="/pmc/articles/instance/3887612/bin/ksmcb-31-483-g005.gif" src-large="/pmc/articles/instance/3887612/bin/ksmcb-31-483-g005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3887612/" ref="ordinalpos=1885&amp;ncbi_uid=5787842&amp;link_uid=PMC3887612" image-link="/pmc/articles/PMC3887612/figure/F005/" class="imagepopup">Fig. 5.A proposed model for the tau <span class="highlight" style="background-color:">signaling</span> cascade in LPA induced neurite retraction in differentiated Neuro2a cells. LPA increased tau phosphorylation, which was followed by activation of the GSK-3β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The activated GSK-3β and PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediated the hyperphosphorylation of tau, which subsequently resulted in differentiated Neuro2a cell neurite retraction..  From: Lysophosphatidic Acid Induces Neurite Retraction in Differentiated Neuroblastoma Cells via GSK-3? Activation. </a></p><div class="aux"><div class="resc">Yuanjie Sun, et al. Mol Cells. 2011 May 31;31(5):483-489.</div><p class="links"><a class="dblinks" href="/pubmed/21499833" ref="ordinalpos=1885&amp;ncbi_uid=5787842&amp;link_uid=21499833">Citation</a><a class="dblinks" href="/pmc/articles/PMC3887612/" ref="ordinalpos=1885&amp;ncbi_uid=5787842&amp;link_uid=PMC3887612">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1886.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2798675/figure/F8/" ref="ordinalpos=1886&amp;ncbi_uid=2132257&amp;link_uid=PMC2798675" href="/pmc/articles/PMC2798675/"><img src="/pmc/articles/instance/2798675/bin/JCI37964.f8.gif" src-large="/pmc/articles/instance/2798675/bin/JCI37964.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2798675/" ref="ordinalpos=1886&amp;ncbi_uid=2132257&amp;link_uid=PMC2798675" image-link="/pmc/articles/PMC2798675/figure/F8/" class="imagepopup">Figure 8. mTOR regulates STAT3/p63/Notch <span class="highlight" style="background-color:">signaling</span> in human cancers..  From: Mammalian target of rapamycin regulates murine and human cell differentiation through STAT3/p63/Jagged/Notch cascade. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Human breast (MCF7 and MDA-MB-468), prostate (PC3), lung (A549), pancreatic (PANC-1), and liver (HepG2) cancer cell lines were treated with 10 nM rapamycin for 24 hours and then subjected to immunoblotting for components of the mTOR/STAT/p63/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) PC3 cells were transduced with either pLXIN-hyg retroviruses (V) or pLXIN-hyg-PTEN retroviruses (PTEN) and then subjected to immunoblotting for PTEN and mTOR/STAT/p63/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components. (<b>C</b>) Human breast cancer tissues were immunoblotted for components of the mTOR/STAT/p63/Notch <span class="highlight" style="background-color:">signaling</span> cascade. A representative blot is shown. The differentiation states (Diff.) of cancer cells were as indicated.    </div></p></div><div class="aux"><div class="resc">Jianhui Ma, et al. J Clin Invest. 2010 January 4;120(1):103-114.</div><p class="links"><a class="dblinks" href="/pubmed/20038814" ref="ordinalpos=1886&amp;ncbi_uid=2132257&amp;link_uid=20038814">Citation</a><a class="dblinks" href="/pmc/articles/PMC2798675/" ref="ordinalpos=1886&amp;ncbi_uid=2132257&amp;link_uid=PMC2798675">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1887.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3288347/figure/fig7/" ref="ordinalpos=1887&amp;ncbi_uid=3829453&amp;link_uid=PMC3288347" href="/pmc/articles/PMC3288347/"><img src="/pmc/articles/instance/3288347/bin/cddis20125f7.gif" src-large="/pmc/articles/instance/3288347/bin/cddis20125f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3288347/" ref="ordinalpos=1887&amp;ncbi_uid=3829453&amp;link_uid=PMC3288347" image-link="/pmc/articles/PMC3288347/figure/fig7/" class="imagepopup">Figure 7.  From: JNK modulates FOXO3a for the expression of the mitochondrial death and mitophagy marker BNIP3 in pathological hypertrophy and in heart failure. </a></p><div class="supp"><p class="details"><div>Chronic JNK <span class="highlight" style="background-color:">signaling</span> dominates over AKT <span class="highlight" style="background-color:">signaling</span> and overrides the inhibitory effect of AKT on FOXO3a for the transcription of its effectors, BNIP3 and MurF-1 in POH and in heart failure. JNK and AKT <span class="highlight" style="background-color:">signaling</span> are activated in response to pressure overload. However, despite the increase in AKT activity, BNIP3 expression increased significantly in POH and peaked in HF. On the other hand, the decrease in JNK activity by Ad-DN-JNK and 3 MA <i>in vivo</i> decreased AKT activity and decreased the expression of FOXO3a effectors, BNIP3 and MurF-1. In other words, the expression of FOXO3a effectors, BNIP3 and MurF-1, are directly related to JNK activity and that the chronic activation of JNK dominates over AKT <span class="highlight" style="background-color:">signaling</span> for the activation of FOXO3a and for the transcription of its effectors, BNIP3 and MurF-1. Moreover, the increase in JNK activity phosphorylates Bcl-2 and increases Bax to Bcl-2 ratio, further promoting mitochondria-induced apoptosis. Green arrows: promote <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, Red arrows: inhibit <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">A H Chaanine, et al. Cell Death Dis. 2012 February;3(2):265.</div><p class="links"><a class="dblinks" href="/pubmed/22297293" ref="ordinalpos=1887&amp;ncbi_uid=3829453&amp;link_uid=22297293">Citation</a><a class="dblinks" href="/pmc/articles/PMC3288347/" ref="ordinalpos=1887&amp;ncbi_uid=3829453&amp;link_uid=PMC3288347">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1888.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2976807/figure/F6/" ref="ordinalpos=1888&amp;ncbi_uid=3584651&amp;link_uid=PMC2976807" href="/pmc/articles/PMC2976807/"><img src="/pmc/articles/instance/2976807/bin/nihms238183f6.gif" src-large="/pmc/articles/instance/2976807/bin/nihms238183f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2976807/" ref="ordinalpos=1888&amp;ncbi_uid=3584651&amp;link_uid=PMC2976807" image-link="/pmc/articles/PMC2976807/figure/F6/" class="imagepopup">Figure 6. Models for asymmetric activation of the Wnt <span class="highlight" style="background-color:">pathway</span> in posterior seam cell daughters.  From: Wnt <span class="highlight" style="background-color:">signaling</span> controls the stem cell-like asymmetric division of the epithelial seam cells during C. elegans larval development. </a></p><div class="supp"><p class="details"><div>A generalized mother seam cell (SC) divides in a stem cell-like manner to generate an anterior daughter that adopts a non-seam cell fate, and a posterior daughter that retains the seam cell fate. In the posterior daughter, activation of the Wnt <span class="highlight" style="background-color:">pathway</span> leads to formation of a POP-1/SYS-1 complex that promotes expression of target genes required for the seam cell fate. In the anterior daughter, POP-1 interacts with transcriptional repressors to keep these Wnt <span class="highlight" style="background-color:">pathway</span> targets off. The curvy line at the membrane indicates expression of Wnt receptor. A) An anterior-posterior gradient of Wnt ligands (higher in the posterior) leads to asymmetric activation of the <span class="highlight" style="background-color:">pathway</span> in the posterior daughter. B) The asymmetric localization of Wnt <span class="highlight" style="background-color:">pathway</span> components (such as the receptor, shown here) to the posterior cell allows only that daughter to respond to a permissive Wnt signal in the environment. Alternatively, the asymmetric distribution of a negative <span class="highlight" style="background-color:">pathway</span> regulator (shown by a triangle) to the anterior daughter could keep the <span class="highlight" style="background-color:">pathway</span> off in that cell, even in the presence of permissive Wnt signal and appropriate receptor. C) Asymmetric localization of some Wnt <span class="highlight" style="background-color:">pathway</span> components to the posterior daughter (such as the receptor) and other factors to the anterior daughter, leads to activation of the <span class="highlight" style="background-color:">pathway</span> in the posterior daughter in a Wnt-independent manner. For example, a non-Wnt ligand localized to the anterior daughter (shown by a star) could activate the Wnt receptor in the posterior daughter.</div></p></div><div class="aux"><div class="resc">Julie E. Gleason, et al. Dev Biol. ;348(1):58-66.</div><p class="links"><a class="dblinks" href="/pubmed/20849842" ref="ordinalpos=1888&amp;ncbi_uid=3584651&amp;link_uid=20849842">Citation</a><a class="dblinks" href="/pmc/articles/PMC2976807/" ref="ordinalpos=1888&amp;ncbi_uid=3584651&amp;link_uid=PMC2976807">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1889.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F4/" ref="ordinalpos=1889&amp;ncbi_uid=4064477&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f4.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=1889&amp;ncbi_uid=4064477&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F4/" class="imagepopup">Figure 4. Deletion of plexin-D1 slightly decreases spine density in direct <span class="highlight" style="background-color:">pathway</span> MSNs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) 2PLSM image of a brain slice from a <i>Drd2-</i>GFP<i>;Drd1a-</i>Cre<i>;plexin-D1</i><sup>f/+</sup> mouse showing GFP-expressing indirect <span class="highlight" style="background-color:">pathway</span> MSNs (green) and one MSN that was filled with Alexa Fluor-594 (red) through the whole-cell recording pipette. (<b>b</b>) High magnification images of spiny dendritic segments of direct (<i>left</i>) and indirect (<i>right</i>) <span class="highlight" style="background-color:">pathway</span> MSNs in mice that have selectively lost one (<i>top</i>) or both (<i>bottom</i>) copies of <i>plexin-D1</i> in MSNs of each <span class="highlight" style="background-color:">pathway</span>. Direct <span class="highlight" style="background-color:">pathway</span> MSNs in <i>Drd2-GFP;Drd1a</i>-Cre;<i>plexin-D1</i><sup>f/f</sup> mice show decreased spine density compared to their littermate controls. In contrast, indirect <span class="highlight" style="background-color:">pathway</span> MSNs in <i>Drd2</i>-specific <i>plexin-D1</i> KO mice (<i>Drd2</i>-GFP;<i>Drd2</i>-Cre;<i>plexin-D1</i><sup>f/f</sup>) show no difference in spine density compared to their littermate controls (<i>Drd2</i>-GFP;<i>Drd2</i>-Cre;<i>plexin-D1</i><sup>f/+</sup>) (<b>c</b>) Summary of spine densities in direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs of the indicated genotypes. There is a small but significant loss of spines in direct <span class="highlight" style="background-color:">pathway</span> MSNs in <i>Drd1a</i>-Cre <i>plexin-D1</i> conditional KO mice compared to Cre− (fl/fl Ctrl) and Cre+ (Het Ctrl) control mice (*<i>P</i>&lt;0.05; Mann-Whitney). Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=1889&amp;ncbi_uid=4064477&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=1889&amp;ncbi_uid=4064477&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1890.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713013/figure/F7/" ref="ordinalpos=1890&amp;ncbi_uid=1932307&amp;link_uid=PMC2713013" href="/pmc/articles/PMC2713013/"><img src="/pmc/articles/instance/2713013/bin/nihms-100233-f0007.gif" src-large="/pmc/articles/instance/2713013/bin/nihms-100233-f0007.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713013/" ref="ordinalpos=1890&amp;ncbi_uid=1932307&amp;link_uid=PMC2713013" image-link="/pmc/articles/PMC2713013/figure/F7/" class="imagepopup">Fig. 7. Unchecked IL-1 <span class="highlight" style="background-color:">signaling</span> causes inflammation and autoimmunity in Ptpn6spin mice.  From: MICROBE SENSING, POSITIVE FEEDBACK LOOPS, AND THE PATHOGENESIS OF INFLAMMATORY DISEASES. </a></p><div class="supp"><p class="details"><div>(A) The IL-1 <span class="highlight" style="background-color:">signaling</span> loop is initiated by microbial stimulation of TLRs, which through canonical TLR <span class="highlight" style="background-color:">signaling</span> via IRAK1, IRAK4, TRAF6, IKK complex, and NF-κB activation, leads to IL-1β production. IL-1β activates its own receptor, which also signals via the same TLR <span class="highlight" style="background-color:">pathway</span> components to produce IL-1β. Through an unknown mechanism, impaired SHP1 function in <i>Ptpn6<sup>spin</sup></i> cells results in a failure to downregulate IL-1 <span class="highlight" style="background-color:">signaling</span>, which continues to stimulate its own production. This uninhibited IL-1 <span class="highlight" style="background-color:">signaling</span> loop (B) ultimately causes chronic inflammation and autoimmunity in <i>Ptpn6<sup>spin</sup></i> mice. Derivation into a germfree environment and inactivating mutations in MyD88, IRAK4, and IL-1R suppress disease in <i>Ptpn6<sup>spin</sup></i> mutants.</div></p></div><div class="aux"><div class="resc">Bruce Beutler. Immunol Rev. ;227(1):248-263.</div><p class="links"><a class="dblinks" href="/pubmed/19120489" ref="ordinalpos=1890&amp;ncbi_uid=1932307&amp;link_uid=19120489">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713013/" ref="ordinalpos=1890&amp;ncbi_uid=1932307&amp;link_uid=PMC2713013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1891.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2753670/figure/F3/" ref="ordinalpos=1891&amp;ncbi_uid=2125670&amp;link_uid=PMC2753670" href="/pmc/articles/PMC2753670/"><img src="/pmc/articles/instance/2753670/bin/nihms122747f3.gif" src-large="/pmc/articles/instance/2753670/bin/nihms122747f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2753670/" ref="ordinalpos=1891&amp;ncbi_uid=2125670&amp;link_uid=PMC2753670" image-link="/pmc/articles/PMC2753670/figure/F3/" class="imagepopup">Figure 3.  From: Cross-regulation of <span class="highlight" style="background-color:">signaling</span> by ITAM-associated receptors. </a></p><div class="supp"><p class="details"><div>Integration of ITAM and cytokine Jak–STAT <span class="highlight" style="background-color:">signaling</span>. Low-avidity ligation of ITAM-associated receptors leads to tonic calcium <span class="highlight" style="background-color:">signaling</span> that maintains basal activity of a CaMK–Pyk2 <span class="highlight" style="background-color:">pathway</span> that amplifies Jak activity and enhances downstream activation of STAT1. Cytokine receptors and Jaks also activate Pyk2, and Pyk2 can thus relay signals between ITAM-associated receptors and cytokine receptors bidirectionally. ITAM-containing DAP12 also modulates cell responses to IL-4. Cytokines such as IFN-γ, IFN-α, IL-4, IL-10 and IL-27 regulate expression of ITAM-associated receptors and STAT1, and thus cytokine and ITAM-associated receptor <span class="highlight" style="background-color:">signaling</span> pathways are tightly integrated. Adapted from <i>Nat. Immunol.</i> <b>9</b>, 186–193 (2008).</div></p></div><div class="aux"><div class="resc">Lionel B Ivashkiv. Nat Immunol. ;10(4):340-347.</div><p class="links"><a class="dblinks" href="/pubmed/19295630" ref="ordinalpos=1891&amp;ncbi_uid=2125670&amp;link_uid=19295630">Citation</a><a class="dblinks" href="/pmc/articles/PMC2753670/" ref="ordinalpos=1891&amp;ncbi_uid=2125670&amp;link_uid=PMC2753670">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1892.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035573/figure/F9/" ref="ordinalpos=1892&amp;ncbi_uid=2758818&amp;link_uid=PMC3035573" href="/pmc/articles/PMC3035573/"><img src="/pmc/articles/instance/3035573/bin/nihms186746f9.gif" src-large="/pmc/articles/instance/3035573/bin/nihms186746f9.jpg" alt="FIGURE 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035573/" ref="ordinalpos=1892&amp;ncbi_uid=2758818&amp;link_uid=PMC3035573" image-link="/pmc/articles/PMC3035573/figure/F9/" class="imagepopup">FIGURE 9. Model of thrombin-induced platelet activation through PAR1 and PAR4.  From: PAR4, but Not PAR1, Signals Human Platelet Aggregation via Ca2+ Mobilization and Synergistic P2Y12 Receptor Activation. </a></p><div class="supp"><p class="details"><div>Thrombin activates both PAR1 and PAR4 following vascular injury. PAR1 activates G<i>α</i><sub>i/o</sub>, G<i>α</i><sub>q</sub>, and G<i>α</i><sub>12/13</sub>, whereas PAR4 activates G<i>α</i><sub>q</sub> and G<i>α</i><sub>12/13</sub>. All three of these G proteins play a role in platelet aggregation. Additionally, through secretion of the dense granules, ADP is released and acts as an autocrine signal on the P2Y<sub>12</sub> receptor. PAR4 <span class="highlight" style="background-color:">signaling</span> calcium mobilization downstream of phospholipase C<i>β</i> through the G<i>α</i><sub>q</sub> <span class="highlight" style="background-color:">pathway</span> converges with <span class="highlight" style="background-color:">signaling</span> through the P2Y<sub>12</sub> <span class="highlight" style="background-color:">pathway</span> to induce platelet aggregation. Finally, Rap1 activation is partially dependent on P2Y<sub>12</sub> <span class="highlight" style="background-color:">signaling</span> but may also be activated directly through PAR1 or PAR4 most likely through <span class="highlight" style="background-color:">signaling</span> in the G<i>α</i><sub>q</sub> or G<i>α</i><sub>12/13</sub> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Michael Holinstat, et al. J Biol Chem. ;281(36):26665-26674.</div><p class="links"><a class="dblinks" href="/pubmed/16837456" ref="ordinalpos=1892&amp;ncbi_uid=2758818&amp;link_uid=16837456">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035573/" ref="ordinalpos=1892&amp;ncbi_uid=2758818&amp;link_uid=PMC3035573">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1893.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1899305/figure/fig5/" ref="ordinalpos=1893&amp;ncbi_uid=1030807&amp;link_uid=PMC1899305" href="/pmc/articles/PMC1899305/"><img src="/pmc/articles/instance/1899305/bin/AJRCMB361114f5.gif" src-large="/pmc/articles/instance/1899305/bin/AJRCMB361114f5.jpg" alt="&lt;b&gt;Figure 5.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1899305/" ref="ordinalpos=1893&amp;ncbi_uid=1030807&amp;link_uid=PMC1899305" image-link="/pmc/articles/PMC1899305/figure/fig5/" class="imagepopup">Figure 5..  From: GM-CSF Regulates a PU.1-Dependent Transcriptional Program Determining the Pulmonary Response to LPS. </a></p><div class="supp"><p class="details"><div>GM-CSF, via PU.1, regulates mRNA levels for multiple components of the TLR-4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in AMs. mRNA transcripts encoding both positive and negative <span class="highlight" style="background-color:">signaling</span> components of the TLR-4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cultured alveolar macrophages were detected by RT-PCR. Note the absence of transcripts for CD14, RP105, and IRAK-M in mAM cells.</div></p></div><div class="aux"><div class="resc">Pierre-Yves Berclaz, et al. Am J Respir Cell Mol Biol. 2007 January;36(1):114-121.</div><p class="links"><a class="dblinks" href="/pubmed/16917076" ref="ordinalpos=1893&amp;ncbi_uid=1030807&amp;link_uid=16917076">Citation</a><a class="dblinks" href="/pmc/articles/PMC1899305/" ref="ordinalpos=1893&amp;ncbi_uid=1030807&amp;link_uid=PMC1899305">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1894.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4200133/figure/Fig1/" ref="ordinalpos=1894&amp;ncbi_uid=6731988&amp;link_uid=PMC4200133" href="/pmc/articles/PMC4200133/"><img src="/pmc/articles/instance/4200133/bin/12865_2014_46_Fig1_HTML.gif" src-large="/pmc/articles/instance/4200133/bin/12865_2014_46_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4200133/" ref="ordinalpos=1894&amp;ncbi_uid=6731988&amp;link_uid=PMC4200133" image-link="/pmc/articles/PMC4200133/figure/Fig1/" class="imagepopup">Figure 1.  From: C-Jun N-terminal kinase (JNK) isoforms play differing roles in otitis media. </a></p><div class="supp"><p class="details"><div> <b>Regulation of JNK <span class="highlight" style="background-color:">signaling</span> genes during OM.</b> ME expression of genes related to the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in WT mice during OM, assessed by DNA microarray. Genes involved in TLR JNK <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">William Yao, et al. BMC Immunol. 2014;15(1):46.</div><p class="links"><a class="dblinks" href="/pubmed/25311344" ref="ordinalpos=1894&amp;ncbi_uid=6731988&amp;link_uid=25311344">Citation</a><a class="dblinks" href="/pmc/articles/PMC4200133/" ref="ordinalpos=1894&amp;ncbi_uid=6731988&amp;link_uid=PMC4200133">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1895.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1951132/figure/f1/" ref="ordinalpos=1895&amp;ncbi_uid=1062610&amp;link_uid=PMC1951132" href="/pmc/articles/PMC1951132/"><img src="/pmc/articles/instance/1951132/bin/zek0080729420001.gif" src-large="/pmc/articles/instance/1951132/bin/zek0080729420001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1951132/" ref="ordinalpos=1895&amp;ncbi_uid=1062610&amp;link_uid=PMC1951132" image-link="/pmc/articles/PMC1951132/figure/f1/" class="imagepopup">FIG. 1. From: MpkA-Dependent and -Independent Cell Wall Integrity <span class="highlight" style="background-color:">Signaling</span> in Aspergillus nidulans  . </a></p><div class="supp"><p class="details"><div>In silico reconstruction of the <i>Aspergillus</i> CWIS <span class="highlight" style="background-color:">pathway</span> based on <i>Aspergillus</i> and <i>S. cerevisiae</i> genome information. The putative orthologous proteins involved in CWIS in <i>Aspergillus</i> and <i>S. cerevisiae</i> are represented; An, Ao, and Af indicate the number of BLAST hits for <i>A. nidulans</i>, <i>A. oryzae</i>, and <i>A. fumigatus</i>, respectively. Most genes for CWIS are nonredundant and are well conserved among the <i>Aspergillus</i> fungi and <i>S. cerevisiae</i>. Shaded circles represent transcription factors. Rectangles with rounded edges represent components of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ellipses represent signals that are not part of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MAPKK, MAPK kinase; MAPKKK, MAPK kinase kinase.</div></p></div><div class="aux"><div class="resc">Tomonori Fujioka, et al. Eukaryot Cell. 2007 August;6(8):1497-1510.</div><p class="links"><a class="dblinks" href="/pubmed/17601879" ref="ordinalpos=1895&amp;ncbi_uid=1062610&amp;link_uid=17601879">Citation</a><a class="dblinks" href="/pmc/articles/PMC1951132/" ref="ordinalpos=1895&amp;ncbi_uid=1062610&amp;link_uid=PMC1951132">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1896.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847488/figure/F6/" ref="ordinalpos=1896&amp;ncbi_uid=5567190&amp;link_uid=PMC3847488" href="/pmc/articles/PMC3847488/"><img src="/pmc/articles/instance/3847488/bin/1476-4598-12-96-6.gif" src-large="/pmc/articles/instance/3847488/bin/1476-4598-12-96-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847488/" ref="ordinalpos=1896&amp;ncbi_uid=5567190&amp;link_uid=PMC3847488" image-link="/pmc/articles/PMC3847488/figure/F6/" class="imagepopup">Figure 6.  From: Histone acetyltransferase PCAF Up-regulated cell apoptosis in hepatocellular carcinoma via acetylating histone H4 and inactivating AKT <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Working model for the anti-HCC effect of PCAF and downstream <span class="highlight" style="background-color:">pathway</span>.</b> PCAF promotes apoptosis and inhibits growth of HCC through acetylating histone H4 and inactivating AKT <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Xin Zheng, et al. Mol Cancer. 2013;12:96-96.</div><p class="links"><a class="dblinks" href="/pubmed/23981651" ref="ordinalpos=1896&amp;ncbi_uid=5567190&amp;link_uid=23981651">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847488/" ref="ordinalpos=1896&amp;ncbi_uid=5567190&amp;link_uid=PMC3847488">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1897.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3319314/figure/F2/" ref="ordinalpos=1897&amp;ncbi_uid=3888313&amp;link_uid=PMC3319314" href="/pmc/articles/PMC3319314/"><img src="/pmc/articles/instance/3319314/bin/nihms347427f2.gif" src-large="/pmc/articles/instance/3319314/bin/nihms347427f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3319314/" ref="ordinalpos=1897&amp;ncbi_uid=3888313&amp;link_uid=PMC3319314" image-link="/pmc/articles/PMC3319314/figure/F2/" class="imagepopup">Fig. 2.  From: APE1/Ref-1 Role in Redox <span class="highlight" style="background-color:">Signaling</span>: Translational Applications of Targeting the Redox Function of the DNA Repair/Redox Protein APE1/Ref-1. </a></p><div class="supp"><p class="details"><div>Inhibition of APE1 redox mechanism affects multiple cancer <span class="highlight" style="background-color:">pathway</span> targets. Blocking APE1 redox <span class="highlight" style="background-color:">signaling</span> affects various downstream targets that can impact on tumor growth factors, angiogenesis, growth and proliferation and metastasis and migration of the tumor.</div></p></div><div class="aux"><div class="resc">Mark R. Kelley, et al. Curr Mol Pharmacol. ;5(1):36-53.</div><p class="links"><a class="dblinks" href="/pubmed/22122463" ref="ordinalpos=1897&amp;ncbi_uid=3888313&amp;link_uid=22122463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3319314/" ref="ordinalpos=1897&amp;ncbi_uid=3888313&amp;link_uid=PMC3319314">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1898.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2630181/figure/F8/" ref="ordinalpos=1898&amp;ncbi_uid=1657717&amp;link_uid=PMC2630181" href="/pmc/articles/PMC2630181/"><img src="/pmc/articles/instance/2630181/bin/nihms-82845-f0008.gif" src-large="/pmc/articles/instance/2630181/bin/nihms-82845-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2630181/" ref="ordinalpos=1898&amp;ncbi_uid=1657717&amp;link_uid=PMC2630181" image-link="/pmc/articles/PMC2630181/figure/F8/" class="imagepopup">Figure 8.  From: Acute doxorubicin cardiotoxicity is associated with p53-induced inhibition of the mTOR <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Western blot analysis of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hearts from NON-TXG and MHC-mTORca mice treated with saline or DOX (see Supplemental Data for densitometric signal quantitation).</div></p></div><div class="aux"><div class="resc">Wuqiang Zhu, et al. Circulation. ;119(1):99-106.</div><p class="links"><a class="dblinks" href="/pubmed/19103993" ref="ordinalpos=1898&amp;ncbi_uid=1657717&amp;link_uid=19103993">Citation</a><a class="dblinks" href="/pmc/articles/PMC2630181/" ref="ordinalpos=1898&amp;ncbi_uid=1657717&amp;link_uid=PMC2630181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1899.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3152494/figure/F7/" ref="ordinalpos=1899&amp;ncbi_uid=3414439&amp;link_uid=PMC3152494" href="/pmc/articles/PMC3152494/"><img src="/pmc/articles/instance/3152494/bin/zh89991175410007.gif" src-large="/pmc/articles/instance/3152494/bin/zh89991175410007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3152494/" ref="ordinalpos=1899&amp;ncbi_uid=3414439&amp;link_uid=PMC3152494" image-link="/pmc/articles/PMC3152494/figure/F7/" class="imagepopup">Figure 7.  From: Crosstalk between NOTCH and AKT <span class="highlight" style="background-color:">signaling</span> during murine megakaryocyte lineage specification. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the crosstalk between the NOTCH and AKT pathways at the molecular and cellular levels.</b> (A) Scheme of the molecular pathways. NOTCH <span class="highlight" style="background-color:">pathway</span> activation inhibits PTEN expression leading to AKT activation, FOXO phosphorylation, and enhanced RBPJ-mediated transcription. We hypothesize that this regulatory loop links cytokine receptor <span class="highlight" style="background-color:">signaling</span> and RBPJ-mediated transcription. (B) Scheme of the megakaryocyte lineage development pathways. Our results support the hypothesis that megakaryocyte can develop from immature hematopoietic cells through a NOTCH-induced AKT-independent <span class="highlight" style="background-color:">pathway</span>. Megakaryocyte can also develop from committed myeloid progenitor through an AKT-dependent <span class="highlight" style="background-color:">pathway</span> that is activated by cytokine receptor activation (eg, MPL) and may also be stimulated by the NOTCH <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Melanie G. Cornejo, et al. Blood. 2011 August 4;118(5):1264-1273.</div><p class="links"><a class="dblinks" href="/pubmed/21653327" ref="ordinalpos=1899&amp;ncbi_uid=3414439&amp;link_uid=21653327">Citation</a><a class="dblinks" href="/pmc/articles/PMC3152494/" ref="ordinalpos=1899&amp;ncbi_uid=3414439&amp;link_uid=PMC3152494">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>1900.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2587394/figure/F9/" ref="ordinalpos=1900&amp;ncbi_uid=1649594&amp;link_uid=PMC2587394" href="/pmc/articles/PMC2587394/"><img src="/pmc/articles/instance/2587394/bin/nihms55911f9.gif" src-large="/pmc/articles/instance/2587394/bin/nihms55911f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2587394/" ref="ordinalpos=1900&amp;ncbi_uid=1649594&amp;link_uid=PMC2587394" image-link="/pmc/articles/PMC2587394/figure/F9/" class="imagepopup">Figure 9. PKD1 interacts with TLR9 <span class="highlight" style="background-color:">signaling</span> complex in response to CpG-B DNA.  From: Protein Kinase D1: A New Component in Toll-Like Receptor 9 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>RAW264.7 cells stably expressing empty vector, FLAG-tagged PKD1, FLAG-tagged PKD2, or FLAG-tagged PKD3 were stimulated with medium (med) or CpG-B DNA for 30 min. <b>A,</b> FLAG-tagged PKD family proteins in the cell lysates were immunoprecipitated. The presence of proteins in the TLR9-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and phosphorylated forms of PKD in the resulting immunoprecipitates were detected by Western blot. <b>B,</b> The indicated endogenous proteins in the TLR9-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the cell lysates were immunoprecipitated. The presence of FLAG-tagged PKD family proteins and proteins in the TLR9-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the resulting immunoprecipitates were detected by Western blot.</div></p></div><div class="aux"><div class="resc">Jeoung-Eun Park, et al. J Immunol. ;181(3):2044-2055.</div><p class="links"><a class="dblinks" href="/pubmed/18641342" ref="ordinalpos=1900&amp;ncbi_uid=1649594&amp;link_uid=18641342">Citation</a><a class="dblinks" href="/pmc/articles/PMC2587394/" ref="ordinalpos=1900&amp;ncbi_uid=1649594&amp;link_uid=PMC2587394">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="18" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="19" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="20" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553ff746437716036ad09555">Saving and Managing Searches - My NCBI Help</a>
          <div class="ralinkpop offscreen_noflow">Saving and Managing Searches - My NCBI Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553fef47cbe05d491bf71396">signaling pathway AND (free full text[sb]) <span class="number">(225335)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553fef249892033523880147">signaling pathway <span class="number">(491236)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553fedbe437716036ace4747">PMC Help</a>
          <div class="ralinkpop offscreen_noflow">PMC Help<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553fedb5cbe05d491bf6bb4a">NCBI Help Manual</a>
          <div class="ralinkpop offscreen_noflow">NCBI Help Manual<div class="brieflinkpopdesc"></div></div>
          <div class="tertiary"></div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T17:14:19-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A339953FF819100000000001F18F9;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A339953FF819100000000001F18F9" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>