
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="47" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A6B615431392100000000009436B8" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="46" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="47" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="48" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="47" /></div>
            <div><h2 class="result_count">Results: 4601 to 4700 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4601.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2290851/figure/fig9/" ref="ordinalpos=4601&amp;ncbi_uid=1295384&amp;link_uid=PMC2290851" href="/pmc/articles/PMC2290851/"><img src="/pmc/articles/instance/2290851/bin/jcb1801277f09.gif" src-large="/pmc/articles/instance/2290851/bin/jcb1801277f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2290851/" ref="ordinalpos=4601&amp;ncbi_uid=1295384&amp;link_uid=PMC2290851" image-link="/pmc/articles/PMC2290851/figure/fig9/" class="imagepopup">Figure 9. From: Collagen I-mediated up-regulation of N-cadherin requires cooperative signals from integrins and discoidin domain receptor 1. </a></p><div class="supp"><p class="details"><div>Model of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> active in BxPC3 cells from collagen I outside the cell to induction of cell scattering and up-regulation of N-cadherin.</div></p></div><div class="aux"><div class="resc">Yasushi Shintani, et al. J Cell Biol. 2008 March 24;180(6):1277-1289.</div><p class="links"><a class="dblinks" href="/pubmed/18362184" ref="ordinalpos=4601&amp;ncbi_uid=1295384&amp;link_uid=18362184">Citation</a><a class="dblinks" href="/pmc/articles/PMC2290851/" ref="ordinalpos=4601&amp;ncbi_uid=1295384&amp;link_uid=PMC2290851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4602.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4322980/figure/F4/" ref="ordinalpos=4602&amp;ncbi_uid=7163122&amp;link_uid=PMC4322980" href="/pmc/articles/PMC4322980/"><img src="/pmc/articles/instance/4322980/bin/oncotarget-05-12151-g004.gif" src-large="/pmc/articles/instance/4322980/bin/oncotarget-05-12151-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4322980/" ref="ordinalpos=4602&amp;ncbi_uid=7163122&amp;link_uid=PMC4322980" image-link="/pmc/articles/PMC4322980/figure/F4/" class="imagepopup">Figure 4. GANT-61 treatment dampens Shh and m-TOR <span class="highlight" style="background-color:">signaling</span> pathways.  From: GLI inhibitor GANT-61 diminishes embryonal and alveolar rhabdomyosarcoma growth by inhibiting Shh/AKT-mTOR axis. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Semi-quantitative RT-PCR analysis of Gli1/2 in RD and RH30 cells at various concentrations of GANT-61. <b>(B)</b> Immunofluorescence staining of GLI1 in vehicle-and GANT-61-treated RH30 cells-derived xenograft tumors. <b>(C)</b> Real time PCR analysis of Gli1 mRNA expression relative to GAPDH as an internal control in GANT-61-treated xenograft tumors as compared to vehicle-treated control tumors. AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in GANT-61-treated RD and RH30 cells-derived tumors vs. vehicle-treated control tumors: <b>(D)</b> Real time PCR analysis and <b>(E-I &amp; EII)</b> western blot analysis. <b>(F)</b> Western blot analysis of GANT-61 on the phosphorylation of AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins (p-AKT, p-mTOR, p-P70S6K) in RD and RH30 RMS cells. T1 to T4 represents tumors excised from four different mice. <i>P</i> value represents level of significant difference in GANT-61-treated and vehicle-treated controls.</div></p></div><div class="aux"><div class="resc">Ritesh K. Srivastava, et al. Oncotarget. 2014 December;5(23):12151-12165.</div><p class="links"><a class="dblinks" href="/pubmed/25432075" ref="ordinalpos=4602&amp;ncbi_uid=7163122&amp;link_uid=25432075">Citation</a><a class="dblinks" href="/pmc/articles/PMC4322980/" ref="ordinalpos=4602&amp;ncbi_uid=7163122&amp;link_uid=PMC4322980">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4603.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472971/figure/F1/" ref="ordinalpos=4603&amp;ncbi_uid=5391362&amp;link_uid=PMC3472971" href="/pmc/articles/PMC3472971/"><img src="/pmc/articles/instance/3472971/bin/jnci.j_djs396_f0001.gif" src-large="/pmc/articles/instance/3472971/bin/jnci.j_djs396_f0001.jpg" alt="Figure 1. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472971/" ref="ordinalpos=4603&amp;ncbi_uid=5391362&amp;link_uid=PMC3472971" image-link="/pmc/articles/PMC3472971/figure/F1/" class="imagepopup">Figure 1.  From: Selective Inhibition of HER2-Positive Breast Cancer Cells by the HIV Protease Inhibitor Nelfinavir. </a></p><div class="supp"><p class="details"><div>Effect of nelfinavir (NFV) on human epidermal growth factor receptor 2 (HER2) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>A</b>) Effect of increasing concentrations of NFV on HER2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and caspase cascades in HER2-positive and -negative breast cancer cells. Cells were treated with various concentrations of NFV for 24 hours, and each protein level was analyzed by immunoblots. <b>B</b>) A time-course experiment of the NFV effect on HER2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and caspase cascades in HER2-positive breast cancer, HCC1954 cells. <b>C</b>) Effect of proteasome inhibitors and caspase inhibitors on NFV effects on HER2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and caspase cascades in HCC1954 cells. For the cotreatment experiments, cells were pretreated with either caspase inhibitors or proteasome inhibitors for 30 minutes and then incubated with NFV for 24 hours. z-DEVD (30 M z-DEVD-fmk, caspase-3 inhibitor), z-IETD (30 M z-IETD-fmk, caspase-8 inhibitor), z-VAD (30 M z-VAD-fmk, pan caspase inhibitor), MG132 (0.4 M, proteasome inhibitor), and BTZ (20nM bortezomib, proteasome inhibitor) were used for this study. Act-cas-3 = active form of caspase 3; Act-cas-8 = active form of caspase 8; Act-cas-9 = active form of caspase 9; NS = nonspecific band; pERK1 and -2; phospho ERK1 and -2; pHER2 = phospho-HER2; Pro-cas-8 = proform of caspase 8; Pro-cas-9 = proform of caspase 9.</div></p></div><div class="aux"><div class="resc">Joong Sup Shim, et al. J Natl Cancer Inst. 2012 October 17;104(20):1576-1590.</div><p class="links"><a class="dblinks" href="/pubmed/23042933" ref="ordinalpos=4603&amp;ncbi_uid=5391362&amp;link_uid=23042933">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472971/" ref="ordinalpos=4603&amp;ncbi_uid=5391362&amp;link_uid=PMC3472971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4604.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3808934/figure/fig02/" ref="ordinalpos=4604&amp;ncbi_uid=5445325&amp;link_uid=PMC3808934" href="/pmc/articles/PMC3808934/"><img src="/pmc/articles/instance/3808934/bin/ajcei0002-0195-f2.gif" src-large="/pmc/articles/instance/3808934/bin/ajcei0002-0195-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3808934/" ref="ordinalpos=4604&amp;ncbi_uid=5445325&amp;link_uid=PMC3808934" image-link="/pmc/articles/PMC3808934/figure/fig02/" class="imagepopup">Figure 2.  From: Collaboration of Toll-like and RIG-I-like receptors in human dendritic cells: tRIGgering antiviral innate immune responses. </a></p><div class="supp"><p class="details"><div>The interplay of TLR and RLR <span class="highlight" style="background-color:">signaling</span>. TLRs expressed on the cell surface or localized on intracellular membranes recognize various sets of pathogenic structures among them viral genomes or replication intermediates and transduce signals through the NF-B/type I IFN pathways. The TLR3 and TLR4 mediated <span class="highlight" style="background-color:">signaling</span> pathways are independent of MyD88 and IRAKs (IRAK-1/2/4), whereas the other TLRs use the MyD88 <span class="highlight" style="background-color:">pathway</span>. TLR4 is capable of using both MyD88-dependent and independent <span class="highlight" style="background-color:">signaling</span>. TRAF3 and TRAF6 have a cardinal role in both the collaboration and the essential <span class="highlight" style="background-color:">signaling</span> processes of TLRs and RLRs.</div></p></div><div class="aux"><div class="resc">Attila Szabo, et al. Am J Clin Exp Immunol. 2013;2(3):195-207.</div><p class="links"><a class="dblinks" href="/pubmed/24179728" ref="ordinalpos=4604&amp;ncbi_uid=5445325&amp;link_uid=24179728">Citation</a><a class="dblinks" href="/pmc/articles/PMC3808934/" ref="ordinalpos=4604&amp;ncbi_uid=5445325&amp;link_uid=PMC3808934">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4605.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3545208/figure/F9/" ref="ordinalpos=4605&amp;ncbi_uid=5663209&amp;link_uid=PMC3545208" href="/pmc/articles/PMC3545208/"><img src="/pmc/articles/instance/3545208/bin/zmg0011351210009.gif" src-large="/pmc/articles/instance/3545208/bin/zmg0011351210009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3545208/" ref="ordinalpos=4605&amp;ncbi_uid=5663209&amp;link_uid=PMC3545208" image-link="/pmc/articles/PMC3545208/figure/F9/" class="imagepopup">Fig. 9. From: Cardiac PI3K-Akt Impairs Insulin-Stimulated Glucose Uptake Independent of mTORC1 and GLUT4 Translocation. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> and glucose transport in normal hearts and in insulin resistance and type 2 diabetes. In a healthy heart, insulin activates PI3K-Akt <span class="highlight" style="background-color:">signaling</span> and promotes GLUT4 translocation to the cell surface to facilitate glucose transport. Insulin resistance and type 2 diabetes is characterized by hyperinsulinemia. A persistent increase in circulating insulin activates PI3K-Akt leading to mTOR-S6K negative feedback <span class="highlight" style="background-color:">signaling</span> to IRS1/2. Activation of PI3K-Akt <span class="highlight" style="background-color:">signaling</span> in the heart rapidly impairs insulin-stimulated glucose uptake before inactivation of IRS by this negative feedback <span class="highlight" style="background-color:">pathway</span>. Although GLUT4 translocation may be maintained or increased, glucose uptake is decreased, suggesting the existence of a GLUT4 postinsertion activation mechanism, which is impaired in hearts with chronic activation of PI3K and Akt.</div></p></div><div class="aux"><div class="resc">Yi Zhu, et al. Mol Endocrinol. 2013 January;27(1):172-184.</div><p class="links"><a class="dblinks" href="/pubmed/23204326" ref="ordinalpos=4605&amp;ncbi_uid=5663209&amp;link_uid=23204326">Citation</a><a class="dblinks" href="/pmc/articles/PMC3545208/" ref="ordinalpos=4605&amp;ncbi_uid=5663209&amp;link_uid=PMC3545208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4606.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2577427/figure/f9/" ref="ordinalpos=4606&amp;ncbi_uid=1548825&amp;link_uid=PMC2577427" href="/pmc/articles/PMC2577427/"><img src="/pmc/articles/instance/2577427/bin/zmb0200877470009.gif" src-large="/pmc/articles/instance/2577427/bin/zmb0200877470009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2577427/" ref="ordinalpos=4606&amp;ncbi_uid=1548825&amp;link_uid=PMC2577427" image-link="/pmc/articles/PMC2577427/figure/f9/" class="imagepopup">FIG. 9. From: A Novel Nuclear <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> for Thromboxane A2 Receptors in Oligodendrocytes: Evidence for <span class="highlight" style="background-color:">Signaling</span> Compartmentalization during Differentiation . </a></p><div class="supp"><p class="details"><div>Model depicting <span class="highlight" style="background-color:">signaling</span> pathways for TPR-mediated modulation of OLG development and function. See Discussion for details. Cox1/TXS, cyclooxygenase 1/thromboxane synthase; PLA<sub>2</sub>, phospholipase A<sub>2</sub>; PIP<sub>2</sub>, phosphatidylinositol 4,5-biphosphate.</div></p></div><div class="aux"><div class="resc">Fozia Mir, et al. Mol Cell Biol. 2008 October;28(20):6329-6341.</div><p class="links"><a class="dblinks" href="/pubmed/18710937" ref="ordinalpos=4606&amp;ncbi_uid=1548825&amp;link_uid=18710937">Citation</a><a class="dblinks" href="/pmc/articles/PMC2577427/" ref="ordinalpos=4606&amp;ncbi_uid=1548825&amp;link_uid=PMC2577427">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4607.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938097/figure/F7/" ref="ordinalpos=4607&amp;ncbi_uid=3059919&amp;link_uid=PMC2938097" href="/pmc/articles/PMC2938097/"><img src="/pmc/articles/instance/2938097/bin/zjw0071036280007.gif" src-large="/pmc/articles/instance/2938097/bin/zjw0071036280007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938097/" ref="ordinalpos=4607&amp;ncbi_uid=3059919&amp;link_uid=PMC2938097" image-link="/pmc/articles/PMC2938097/figure/F7/" class="imagepopup">Fig. 7. From: Studies of Phosphoproteomic Changes Induced by Nucleophosmin-Anaplastic Lymphoma Kinase (ALK) Highlight Deregulation of Tumor Necrosis Factor (TNF)/Fas/TNF-related Apoptosis-induced Ligand <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in ALK-positive Anaplastic Large Cell Lymphoma. </a></p><div class="supp"><p class="details"><div><b>Representative MS/MS spectra of phosphopeptides from three proteins in TNF/Fas/TRAIL <span class="highlight" style="background-color:">pathway</span>: TRAP1 (<i>a</i>), RIP (<i>b</i>), and FAF1 (<i>c</i>).</b></div></p></div><div class="aux"><div class="resc">Fang Wu, et al. Mol Cell Proteomics. 2010 July;9(7):1616-1632.</div><p class="links"><a class="dblinks" href="/pubmed/20393185" ref="ordinalpos=4607&amp;ncbi_uid=3059919&amp;link_uid=20393185">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938097/" ref="ordinalpos=4607&amp;ncbi_uid=3059919&amp;link_uid=PMC2938097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4608.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047870/figure/fig1/" ref="ordinalpos=4608&amp;ncbi_uid=6244327&amp;link_uid=PMC4047870" href="/pmc/articles/PMC4047870/"><img src="/pmc/articles/instance/4047870/bin/cddis2014178f1.gif" src-large="/pmc/articles/instance/4047870/bin/cddis2014178f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047870/" ref="ordinalpos=4608&amp;ncbi_uid=6244327&amp;link_uid=PMC4047870" image-link="/pmc/articles/PMC4047870/figure/fig1/" class="imagepopup">Figure 1.  From: Rapamycin reverses insulin resistance (IR) in high-glucose medium without causing IR in normoglycemic medium. </a></p><div class="supp"><p class="details"><div>The insulin/mTOR/HIF-1 <span class="highlight" style="background-color:">pathway</span>: a simplified schema. Glucose activates mTOR, which in turn blocks insulin <span class="highlight" style="background-color:">signaling</span> (a feedback loop). The mTOR <span class="highlight" style="background-color:">pathway</span> increases translation of the HIF-1<i></i> RNA. However, accumulation of the HIF-1<i></i> protein is tightly limited by its degradation via a PHD-feedback loop under normoxia.<sup>28, 66</sup> Red lines, inhibitory feedback loops</div></p></div><div class="aux"><div class="resc">O V Leontieva, et al. Cell Death Dis. 2014 May;5(5):e1214.</div><p class="links"><a class="dblinks" href="/pubmed/24810050" ref="ordinalpos=4608&amp;ncbi_uid=6244327&amp;link_uid=24810050">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047870/" ref="ordinalpos=4608&amp;ncbi_uid=6244327&amp;link_uid=PMC4047870">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4609.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3800607/figure/fig6/" ref="ordinalpos=4609&amp;ncbi_uid=5471760&amp;link_uid=PMC3800607" href="/pmc/articles/PMC3800607/"><img src="/pmc/articles/instance/3800607/bin/ECAM2013-548929.006.gif" src-large="/pmc/articles/instance/3800607/bin/ECAM2013-548929.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3800607/" ref="ordinalpos=4609&amp;ncbi_uid=5471760&amp;link_uid=PMC3800607" image-link="/pmc/articles/PMC3800607/figure/fig6/" class="imagepopup">Figure 6.  From: Destruxin B Isolated from Entomopathogenic Fungus Metarhizium anisopliae Induces Apoptosis via a Bcl-2 Family-Dependent Mitochondrial <span class="highlight" style="background-color:">Pathway</span> in Human Nonsmall Cell Lung Cancer Cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of destruxin B preparation and mechanistic illustration of destruxin-B-induced apoptosis. Destruxin B isolated from <i>Metarhizium anisopliae </i>induces apoptosis in human nonsmall cell lung cancer cells by activating a mitochondrial-dependent caspase activation <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Chun-Chi Wu, et al. Evid Based Complement Alternat Med. 2013;2013:548929.</div><p class="links"><a class="dblinks" href="/pubmed/24204395" ref="ordinalpos=4609&amp;ncbi_uid=5471760&amp;link_uid=24204395">Citation</a><a class="dblinks" href="/pmc/articles/PMC3800607/" ref="ordinalpos=4609&amp;ncbi_uid=5471760&amp;link_uid=PMC3800607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4610.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2957443/figure/pone-0013500-g003/" ref="ordinalpos=4610&amp;ncbi_uid=2530293&amp;link_uid=PMC2957443" href="/pmc/articles/PMC2957443/"><img src="/pmc/articles/instance/2957443/bin/pone.0013500.g003.gif" src-large="/pmc/articles/instance/2957443/bin/pone.0013500.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2957443/" ref="ordinalpos=4610&amp;ncbi_uid=2530293&amp;link_uid=PMC2957443" image-link="/pmc/articles/PMC2957443/figure/pone-0013500-g003/" class="imagepopup">Figure 3. Biological processes deregulated in the hormone-refractory sublines..  From: Bypass Mechanisms of the Androgen Receptor <span class="highlight" style="background-color:">Pathway</span> in Therapy-Resistant Prostate Cancer Cell Models. </a></p><div class="supp"><p class="details"><div>Top 5 biological functions enriched in the therapy-resistant sublines: (A) diseases and disorders, (B) molecular and cellular functions. (C) Example of Network analysis for PC346DCC showing deregulation of hormone and growth-factor receptor <span class="highlight" style="background-color:">signaling</span>: up-regulated genes are represented in red and repressed genes in green. Analysis was performed using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software (www.ingenuity.com).</div></p></div><div class="aux"><div class="resc">Rute B. Marques, et al. PLoS One. 2010;5(10):e13500.</div><p class="links"><a class="dblinks" href="/pubmed/20976069" ref="ordinalpos=4610&amp;ncbi_uid=2530293&amp;link_uid=20976069">Citation</a><a class="dblinks" href="/pmc/articles/PMC2957443/" ref="ordinalpos=4610&amp;ncbi_uid=2530293&amp;link_uid=PMC2957443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4611.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3572966/figure/pone-0056891-g005/" ref="ordinalpos=4611&amp;ncbi_uid=4671421&amp;link_uid=PMC3572966" href="/pmc/articles/PMC3572966/"><img src="/pmc/articles/instance/3572966/bin/pone.0056891.g005.gif" src-large="/pmc/articles/instance/3572966/bin/pone.0056891.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3572966/" ref="ordinalpos=4611&amp;ncbi_uid=4671421&amp;link_uid=PMC3572966" image-link="/pmc/articles/PMC3572966/figure/pone-0056891-g005/" class="imagepopup">Figure 5. ERK1/2 is the predominant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> facilitating IL-1-mediated astrocyte TIMP-1 expression..  From: Extracellular Regulated Kinase 1/2 <span class="highlight" style="background-color:">Signaling</span> Is a Critical Regulator of Interleukin-1?-Mediated Astrocyte Tissue Inhibitor of Metalloproteinase-1 Expression. </a></p><div class="supp"><p class="details"><div>Neuroinflammation activates bystander microglia and astrocytes to secrete IL-1. Astrocytes respond to IL-1 by secreting TIMP-1. In this study we use a panel of small molecule <span class="highlight" style="background-color:">pathway</span>-selective inhibitors to delineate the signal transduction mechanism facilitating IL-1-induced astrocyte TIMP-1 expression. p38K- and ERK1/2-selective inhibitors reduced and completely blocked IL-1-induced astrocyte TIMP-1 mRNA and protein expression, respectively. U0126, the ERK1/2-selective inhibitor, also blocked TIMP-1 promoter activity.</div></p></div><div class="aux"><div class="resc">Jerel Fields, et al. PLoS One. 2013;8(2):e56891.</div><p class="links"><a class="dblinks" href="/pubmed/23457635" ref="ordinalpos=4611&amp;ncbi_uid=4671421&amp;link_uid=23457635">Citation</a><a class="dblinks" href="/pmc/articles/PMC3572966/" ref="ordinalpos=4611&amp;ncbi_uid=4671421&amp;link_uid=PMC3572966">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4612.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3329486/figure/pone-0034488-g001/" ref="ordinalpos=4612&amp;ncbi_uid=3961951&amp;link_uid=PMC3329486" href="/pmc/articles/PMC3329486/"><img src="/pmc/articles/instance/3329486/bin/pone.0034488.g001.gif" src-large="/pmc/articles/instance/3329486/bin/pone.0034488.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3329486/" ref="ordinalpos=4612&amp;ncbi_uid=3961951&amp;link_uid=PMC3329486" image-link="/pmc/articles/PMC3329486/figure/pone-0034488-g001/" class="imagepopup">Figure 1. Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> network models..  From: Cross-Talk and Information Transfer in Mammalian and Bacterial <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(<b>Model A</b>) A single channel of one RSmad with a single output. (<b>Model B</b>) A single channel with two outputs, the phosphorylated RSmad and the RSmad:Co-Smad heterodimer. (<b>Model C</b>) The dual channel with two distinct RSmads and two outputs. The insets diagram the information transmission topology of signal (ligand), channel, and output (complexes). Note that this diagram represents a phosphorylation reaction by the receptors, not a phosphotransfer.</div></p></div><div class="aux"><div class="resc">Samanthe M. Lyons, et al. PLoS One. 2012;7(4):e34488.</div><p class="links"><a class="dblinks" href="/pubmed/22529918" ref="ordinalpos=4612&amp;ncbi_uid=3961951&amp;link_uid=22529918">Citation</a><a class="dblinks" href="/pmc/articles/PMC3329486/" ref="ordinalpos=4612&amp;ncbi_uid=3961951&amp;link_uid=PMC3329486">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4613.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3130521/figure/F1/" ref="ordinalpos=4613&amp;ncbi_uid=3510431&amp;link_uid=PMC3130521" href="/pmc/articles/PMC3130521/"><img src="/pmc/articles/instance/3130521/bin/nihms302786f1.gif" src-large="/pmc/articles/instance/3130521/bin/nihms302786f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3130521/" ref="ordinalpos=4613&amp;ncbi_uid=3510431&amp;link_uid=PMC3130521" image-link="/pmc/articles/PMC3130521/figure/F1/" class="imagepopup">Figure 1. Common <span class="highlight" style="background-color:">signaling</span> pathways identified during preconditioning.  From: Multiple preconditioning paradigms converge on IRF-dependent <span class="highlight" style="background-color:">signaling</span> to promote tolerance to ischemic brain injury. </a></p><div class="supp"><p class="details"><div><b>A)</b> Venn diagrams comparing the regulated* genes in each of the preconditioning paradigms. <b>B)</b> Z-score analysis of Kegg <span class="highlight" style="background-color:">pathway</span> terms associated with genes regulated* in the brain following preconditioning (3, 24 and 72 hr). Z-scores greater than 2 were considered significant (black line). *p&lt;0.01; 2 fold regulation compared to non-treated mice.</div></p></div><div class="aux"><div class="resc">Susan L. Stevens, et al. J Neurosci. ;31(23):8456-8463.</div><p class="links"><a class="dblinks" href="/pubmed/21653850" ref="ordinalpos=4613&amp;ncbi_uid=3510431&amp;link_uid=21653850">Citation</a><a class="dblinks" href="/pmc/articles/PMC3130521/" ref="ordinalpos=4613&amp;ncbi_uid=3510431&amp;link_uid=PMC3130521">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4614.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3172806/figure/F7/" ref="ordinalpos=4614&amp;ncbi_uid=3269599&amp;link_uid=PMC3172806" href="/pmc/articles/PMC3172806/"><img src="/pmc/articles/instance/3172806/bin/zh89991177840007.gif" src-large="/pmc/articles/instance/3172806/bin/zh89991177840007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3172806/" ref="ordinalpos=4614&amp;ncbi_uid=3269599&amp;link_uid=PMC3172806" image-link="/pmc/articles/PMC3172806/figure/F7/" class="imagepopup">Figure 7.  From: Pericytes promote endothelial cell survival through induction of autocrine VEGF-A <span class="highlight" style="background-color:">signaling</span> and Bcl-w expression. </a></p><div class="supp"><p class="details"><div><b>A model of pericyte-induced EC survival <span class="highlight" style="background-color:">signaling</span>.</b> Endothelial cells and pericytes communicate through the reciprocal exchange of growth factors. PDGF-BB produced by ECs acts to recruit pericytes along growing vascular sprouts. In turn, pericytes confer survival advantage to ECs through PDGF-dependent secretion of vitronectin, (1) which through an integrin <sub>v</sub> (2) and NF-B (3) mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> results in up-regulation of intracrine VEGF-A <span class="highlight" style="background-color:">signaling</span> (4) and Bcl-w expression. (5) Previously described survival factors induced by the paracrine action of pericyte- or tumor cell-derived VEGF-A, such as survivin, Bcl-2, and XIAP, are presumably sensitive to the action of both intra- and extra-cellularly acting VEGFR inhibitors (AG-028262 and sFlt1, respectively), whereas the pericyte-induced autocrine <span class="highlight" style="background-color:">signaling</span> by VEGF-A is only sensitive to intra-cellularly acting inhibitors.</div></p></div><div class="aux"><div class="resc">Marcela Franco, et al. Blood. 2011 September 8;118(10):2906-2917.</div><p class="links"><a class="dblinks" href="/pubmed/21778339" ref="ordinalpos=4614&amp;ncbi_uid=3269599&amp;link_uid=21778339">Citation</a><a class="dblinks" href="/pmc/articles/PMC3172806/" ref="ordinalpos=4614&amp;ncbi_uid=3269599&amp;link_uid=PMC3172806">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4615.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3523093/figure/F1/" ref="ordinalpos=4615&amp;ncbi_uid=5147969&amp;link_uid=PMC3523093" href="/pmc/articles/PMC3523093/"><img src="/pmc/articles/instance/3523093/bin/nihms-393916-f0001.gif" src-large="/pmc/articles/instance/3523093/bin/nihms-393916-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3523093/" ref="ordinalpos=4615&amp;ncbi_uid=5147969&amp;link_uid=PMC3523093" image-link="/pmc/articles/PMC3523093/figure/F1/" class="imagepopup">Figure 1. The JNK Activation <span class="highlight" style="background-color:">Pathway</span> and Domain Structure and.  From: A Liver Full of JNK: <span class="highlight" style="background-color:">Signaling</span> in Regulation of Cell Function and Disease Pathogenesis, and Clinical Approaches. </a></p><div class="supp"><p class="details"><div><b>A.</b> number of stimuli, including growth factors, cytokines, PAMPs, ROS, and environmental stresses activate the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through a 3-tier kinase cascade that includes MAP3Ks (ASK1, TAK1, MEKKs and MLKs), MAP2Ks (MKK4 and MKK7), and JNKs. Activated JNKs phosphorylate their substrates, which include transcription factors (c-Jun, JunB, JunD, p53, and c-Myc) and mitochondrial proteins (Bcl-2, Bid, Bim, Bax, and Mcl-1), to induce various biological responses.<br /><b>B.</b> The JNK protein contains a conserved dual phosphorylation Thr-Pro-Tyr motif in its activation loop and has a common substrate docking site in C-terminus and a glutamate/aspartate domain in its N-terminus to mediate interactions with MAP2Ks, phosphatases, and substrates.</div></p></div><div class="aux"><div class="resc">Ekihiro Seki, et al. Gastroenterology. ;143(2):307-320.</div><p class="links"><a class="dblinks" href="/pubmed/22705006" ref="ordinalpos=4615&amp;ncbi_uid=5147969&amp;link_uid=22705006">Citation</a><a class="dblinks" href="/pmc/articles/PMC3523093/" ref="ordinalpos=4615&amp;ncbi_uid=5147969&amp;link_uid=PMC3523093">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4616.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3699820/figure/fig5/" ref="ordinalpos=4616&amp;ncbi_uid=6299362&amp;link_uid=PMC3699820" href="/pmc/articles/PMC3699820/"><img src="/pmc/articles/instance/3699820/bin/ASN.2012050476f5.gif" src-large="/pmc/articles/instance/3699820/bin/ASN.2012050476f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3699820/" ref="ordinalpos=4616&amp;ncbi_uid=6299362&amp;link_uid=PMC3699820" image-link="/pmc/articles/PMC3699820/figure/fig5/" class="imagepopup">Figure 5. From: Rheb/mTORC1 <span class="highlight" style="background-color:">Signaling</span> Promotes Kidney Fibroblast Activation and Fibrosis. </a></p><div class="supp"><p class="details"><div>Ectopic expression of Rheb promotes fibroblast activation. Primary cultured kidney interstitial fibroblasts are treated with rapamycin. (A) Activation of mTORC1 <span class="highlight" style="background-color:">signaling</span> in Rheb overexpression cells. (B) Semi-quantitative analysis of mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> protein abundance in primary cultured fibroblasts. *<i>P</i>&lt;0.05 compared with controls; <sup>#</sup><i>P</i>&lt;0.05 compared with TGF1-treated cells (<i>n</i>=3). (C) Elevation of FN and -SMA protein expression level in Rheb overexpression fibroblasts. (D) Semi-quantitative analysis of FN and -SMA protein expression in primary cultured fibroblasts. *<i>P</i>&lt;0.05 compared with controls; <sup>#</sup><i>P</i>&lt;0.05 compared with TGF1-treated cells (<i>n</i>=3). (E) Representative immunofluorescence images showing FN and -SMA expression in the primary cultured fibroblasts. Slides are counterstained with DAPI for nuclear visualization. DAPI, 4',6-diamidino-2-phenylindole; WT, wild type; Rapa, rapamycin.</div></p></div><div class="aux"><div class="resc">Lei Jiang, et al. J Am Soc Nephrol. 2013 June 28;24(7):1114-1126.</div><p class="links"><a class="dblinks" href="/pubmed/23661807" ref="ordinalpos=4616&amp;ncbi_uid=6299362&amp;link_uid=23661807">Citation</a><a class="dblinks" href="/pmc/articles/PMC3699820/" ref="ordinalpos=4616&amp;ncbi_uid=6299362&amp;link_uid=PMC3699820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4617.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3974699/figure/pone-0092926-g006/" ref="ordinalpos=4617&amp;ncbi_uid=6008449&amp;link_uid=PMC3974699" href="/pmc/articles/PMC3974699/"><img src="/pmc/articles/instance/3974699/bin/pone.0092926.g006.gif" src-large="/pmc/articles/instance/3974699/bin/pone.0092926.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3974699/" ref="ordinalpos=4617&amp;ncbi_uid=6008449&amp;link_uid=PMC3974699" image-link="/pmc/articles/PMC3974699/figure/pone-0092926-g006/" class="imagepopup">Figure 6. Schema for the mechanism of cardiac hypertrophy via AT1-R-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induced by AngII or mediated by mechanical stretch, respectively..  From: Src Is Required for Mechanical Stretch-Induced Cardiomyocyte Hypertrophy through Angiotensin II Type 1 Receptor-Dependent ?-Arrestin2 Pathways. </a></p><div class="aux"><div class="resc">Shijun Wang, et al. PLoS One. 2014;9(4):e92926.</div><p class="links"><a class="dblinks" href="/pubmed/24699426" ref="ordinalpos=4617&amp;ncbi_uid=6008449&amp;link_uid=24699426">Citation</a><a class="dblinks" href="/pmc/articles/PMC3974699/" ref="ordinalpos=4617&amp;ncbi_uid=6008449&amp;link_uid=PMC3974699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4618.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4104550/figure/F1/" ref="ordinalpos=4618&amp;ncbi_uid=6469048&amp;link_uid=PMC4104550" href="/pmc/articles/PMC4104550/"><img src="/pmc/articles/instance/4104550/bin/fonc-04-00174-g001.gif" src-large="/pmc/articles/instance/4104550/bin/fonc-04-00174-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4104550/" ref="ordinalpos=4618&amp;ncbi_uid=6469048&amp;link_uid=PMC4104550" image-link="/pmc/articles/PMC4104550/figure/F1/" class="imagepopup">Figure 1.  From: A Systemic Review of Resistance Mechanisms and Ongoing Clinical Trials in ALK-Rearranged Non-Small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div><b>The ALK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with its cross-talk with other pathways involved in the resistance to ALK-inhibitors</b>. Adapted from Tabbo et al. (47).</div></p></div><div class="aux"><div class="resc">Khashayar Esfahani, et al. Front Oncol. 2014;4:174.</div><p class="links"><a class="dblinks" href="/pubmed/25101240" ref="ordinalpos=4618&amp;ncbi_uid=6469048&amp;link_uid=25101240">Citation</a><a class="dblinks" href="/pmc/articles/PMC4104550/" ref="ordinalpos=4618&amp;ncbi_uid=6469048&amp;link_uid=PMC4104550">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4619.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3205057/figure/F5/" ref="ordinalpos=4619&amp;ncbi_uid=3322331&amp;link_uid=PMC3205057" href="/pmc/articles/PMC3205057/"><img src="/pmc/articles/instance/3205057/bin/1757-2215-4-17-5.gif" src-large="/pmc/articles/instance/3205057/bin/1757-2215-4-17-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3205057/" ref="ordinalpos=4619&amp;ncbi_uid=3322331&amp;link_uid=PMC3205057" image-link="/pmc/articles/PMC3205057/figure/F5/" class="imagepopup">Figure 5.  From: MicroRNA signature of cis-platin resistant vs. cis-platin sensitive ovarian cancer cell lines. </a></p><div class="supp"><p class="details"><div><b>Effect of miR-300 on TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis was used to identify the putative targets (yellow) for miR-300.</div></p></div><div class="aux"><div class="resc">Smriti Kumar, et al. J Ovarian Res. 2011;4:17-17.</div><p class="links"><a class="dblinks" href="/pubmed/21939554" ref="ordinalpos=4619&amp;ncbi_uid=3322331&amp;link_uid=21939554">Citation</a><a class="dblinks" href="/pmc/articles/PMC3205057/" ref="ordinalpos=4619&amp;ncbi_uid=3322331&amp;link_uid=PMC3205057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4620.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1851054/figure/F6/" ref="ordinalpos=4620&amp;ncbi_uid=924407&amp;link_uid=PMC1851054" href="/pmc/articles/PMC1851054/"><img src="/pmc/articles/instance/1851054/bin/zpq0090755020006.gif" src-large="/pmc/articles/instance/1851054/bin/zpq0090755020006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1851054/" ref="ordinalpos=4620&amp;ncbi_uid=924407&amp;link_uid=PMC1851054" image-link="/pmc/articles/PMC1851054/figure/F6/" class="imagepopup">Fig. 6. From: Many NK cell receptors activate ERK2 and JNK1 to trigger microtubule organizing center and granule polarization and cytotoxicity. </a></p><div class="supp"><p class="details"><div>Schematic representation showing that many NK activating receptors employ the same SFK-dependent PI3K  ERK2 <span class="highlight" style="background-color:">pathway</span> and phospholipase C  JNK1 <span class="highlight" style="background-color:">pathway</span> to polarize the MTOC and cytolytic granules during NK cell cytotoxicity, independent of the proximal <span class="highlight" style="background-color:">signaling</span> modules.</div></p></div><div class="aux"><div class="resc">Xi Chen, et al. Proc Natl Acad Sci U S A. 2007 April 10;104(15):6329-6334.</div><p class="links"><a class="dblinks" href="/pubmed/17395718" ref="ordinalpos=4620&amp;ncbi_uid=924407&amp;link_uid=17395718">Citation</a><a class="dblinks" href="/pmc/articles/PMC1851054/" ref="ordinalpos=4620&amp;ncbi_uid=924407&amp;link_uid=PMC1851054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4621.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4322988/figure/F6/" ref="ordinalpos=4621&amp;ncbi_uid=7161732&amp;link_uid=PMC4322988" href="/pmc/articles/PMC4322988/"><img src="/pmc/articles/instance/4322988/bin/oncotarget-05-12097-g006.gif" src-large="/pmc/articles/instance/4322988/bin/oncotarget-05-12097-g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4322988/" ref="ordinalpos=4621&amp;ncbi_uid=7161732&amp;link_uid=PMC4322988" image-link="/pmc/articles/PMC4322988/figure/F6/" class="imagepopup">Figure 6. ESL-1 activates RAS-MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: E-selectin ligand-1 controls circulating prostate cancer cell rolling/adhesion and metastasis. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Western blot analysis of key molecules of RAS-MAP kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from cell lysate of shESL-1 and scESL-1 of PC-3 cells (left panel) and prostate tissues from the orthotopic mice receiving rolling and floating PC-3 cells (right panel). They are ESL-1, nRAS, total MEK, phosphoMEK, total ERK1/2, phosphoERK1/2, and its downstream transcription factor, cFOS antibody. GAPDH was used as a housekeeping gene. <b>(B)</b> Integrin-mediated cell adhesion and migration PCR array. Cell adhesion and migration PCR array was performed to compare the cytoskeleton gene expression in scESL-1 and shESL-1 PC-3 cell line. Bar graph represents the relative fold expression of 3 clusters of genes representing Actin, Myosin, and Rho family.</div></p></div><div class="aux"><div class="resc">Sayeda Yasmin-Karim, et al. Oncotarget. 2014 December;5(23):12097-12110.</div><p class="links"><a class="dblinks" href="/pubmed/25301730" ref="ordinalpos=4621&amp;ncbi_uid=7161732&amp;link_uid=25301730">Citation</a><a class="dblinks" href="/pmc/articles/PMC4322988/" ref="ordinalpos=4621&amp;ncbi_uid=7161732&amp;link_uid=PMC4322988">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4622.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3448731/figure/pone-0045976-g006/" ref="ordinalpos=4622&amp;ncbi_uid=4266114&amp;link_uid=PMC3448731" href="/pmc/articles/PMC3448731/"><img src="/pmc/articles/instance/3448731/bin/pone.0045976.g006.gif" src-large="/pmc/articles/instance/3448731/bin/pone.0045976.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3448731/" ref="ordinalpos=4622&amp;ncbi_uid=4266114&amp;link_uid=PMC3448731" image-link="/pmc/articles/PMC3448731/figure/pone-0045976-g006/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> participated in the AQP1translocation and degradation induced by AZA. .  From: Aquaporin-1 Translocation and Degradation Mediates the Water Transportation Mechanism of Acetazolamide. </a></p><div class="supp"><p class="details"><div><b>A</b>, The effects of AZA on the activation of MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in HK-2 cells. For the MAPK immunoblotting, cells were incubated with 310<sup>6</sup> mol/L AZA for a series of times. Cells were harvested. Immunoblot analysis was performed by using antibodies to p-ERK, ERK, p-p38 kinase, p38 kinase, p-JNK, JNK, respectively. Representative blots and summary data are shown. Data is expressed as meansS.E.M. **<i>p</i>&lt;0.01 compared to Control. <b>B</b>, The effect of AZA on AQP1 expression in the presence of ERK inhibitor U0126. After pretreated with U0126 (2 mol/L) or vehicle for 30 min, cells were incubated with AZA for 24 hours. Values are as meansS.E.M. <b>*</b><i>p</i>&lt;0.05 compared to Control. <b>C</b>, The effect of AZA on the activation of MLC in HK-2 cells. For p-MLC immunoblotting, cells were incubated with 310<sup>6</sup> mol/L AZA for a series of times. Cells were harvested. Immunoblot analysis was performed by using antibodies to p-MLC and summary data is shown here. Values are presented as meansS.E.M. <b>*</b><i>p</i>&lt;0.05, **<i>p</i>&lt;0.01 compared to Control. <b>D</b>, The effect of AZA on MLCK activity. Cells were incubated with 310<sup>6</sup> mol/L AZA for a series of times. Cells were harvested and MLCK activity was analyzed by ELISA assay. Summary data is shown. Values are expressed as meansS.E.M. <b>*</b><i>p</i>&lt;0.05, **<i>p</i>&lt;0.01 compared to Control.</div></p></div><div class="aux"><div class="resc">Jianzhao Zhang, et al. PLoS One. 2012;7(9):e45976.</div><p class="links"><a class="dblinks" href="/pubmed/23029347" ref="ordinalpos=4622&amp;ncbi_uid=4266114&amp;link_uid=23029347">Citation</a><a class="dblinks" href="/pmc/articles/PMC3448731/" ref="ordinalpos=4622&amp;ncbi_uid=4266114&amp;link_uid=PMC3448731">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4623.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2665189/figure/F2/" ref="ordinalpos=4623&amp;ncbi_uid=1807603&amp;link_uid=PMC2665189" href="/pmc/articles/PMC2665189/"><img src="/pmc/articles/instance/2665189/bin/nihms93577f2.gif" src-large="/pmc/articles/instance/2665189/bin/nihms93577f2.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2665189/" ref="ordinalpos=4623&amp;ncbi_uid=1807603&amp;link_uid=PMC2665189" image-link="/pmc/articles/PMC2665189/figure/F2/" class="imagepopup">Fig 2. Oral glucose tolerance and insulin <span class="highlight" style="background-color:">signaling</span>.  From: STEAROYL-CoA DESATURASE-1 DEFICIENCY ATTENUATES OBESITY AND INSULIN RESISTANCE IN LEPTIN-RESISTANT OBESE MICE. </a></p><div class="supp"><p class="details"><div>A. Oral glucose tolerance was assessed in fasted animals by measuring plasma glucose at 0 and 90 minutes post-gavage of 10% glucose at a dose of 2 g/kg body weight.<br />B. Fasting plasma insulin levels were measured by RIA.<br />C. Components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were measured in liver and red gastrocnemius.<br />IR, insulin receptor. <sup>**</sup>compared to respective <i>Scd1<sup>+/+</sup></i> counterparts.</div></p></div><div class="aux"><div class="resc">Makoto Miyazaki, et al. Biochem Biophys Res Commun. ;380(4):818-822.</div><p class="links"><a class="dblinks" href="/pubmed/19338759" ref="ordinalpos=4623&amp;ncbi_uid=1807603&amp;link_uid=19338759">Citation</a><a class="dblinks" href="/pmc/articles/PMC2665189/" ref="ordinalpos=4623&amp;ncbi_uid=1807603&amp;link_uid=PMC2665189">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4624.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4033076/figure/F3/" ref="ordinalpos=4624&amp;ncbi_uid=6199574&amp;link_uid=PMC4033076" href="/pmc/articles/PMC4033076/"><img src="/pmc/articles/instance/4033076/bin/fphar-05-00113-g003.gif" src-large="/pmc/articles/instance/4033076/bin/fphar-05-00113-g003.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4033076/" ref="ordinalpos=4624&amp;ncbi_uid=6199574&amp;link_uid=PMC4033076" image-link="/pmc/articles/PMC4033076/figure/F3/" class="imagepopup">FIGURE 3.  From: Mechanisms of iron metabolism in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div><b>Pathways regulating cellular iron metabolism in <i>C. elegans</i>.</b> <i>Iron deficiency-HIF-1 <span class="highlight" style="background-color:">pathway</span>:</i> this <span class="highlight" style="background-color:">pathway</span> is activated in response to iron deficiency or hypoxia and results in HIF-1 dependent repression of <i>ftn-1</i> and <i>ftn-2</i> and activation of <i>smf-3</i>. This leads to reduced iron storage and increased iron uptake, ultimately increasing the cellular iron pool and promoting survival during iron limitation. <i>Insulin/insulin growth factor-1 (IGF-1) <span class="highlight" style="background-color:">signaling</span> (IIS) <span class="highlight" style="background-color:">pathway</span>:</i> the IIS <span class="highlight" style="background-color:">pathway</span> is a conserved <span class="highlight" style="background-color:">pathway</span> in worms, flies and in vertebrates that regulates the transcription factor DAF-16/FOXO. DAF-16/FOXO regulates genes in essential processes, such as metabolism, stress resistance, pathogen defense and lifespan extension. Activated IIS initiates a phosphorylation cascade that leads to the cytoplasmic retention of DAF-16/FOXO and inhibits its function. Nutrient deprivation reduces IIS leading to the nuclear localization of DAF-16 to activate <i>ftn-1.</i> Glucose upregulates <i>smf-3</i> expression, but whether DAF-16 directly regulates <i>smf-3</i> remains to be determined. This <span class="highlight" style="background-color:">pathway</span> provides a mechanism to increase cellular iron by reducing <i>ftn-1</i> when nutrients are abundant to promote growth and to reduce cellular iron during stress by increasing <i>ftn-1</i> to limit iron-catalyzed oxidative stress.</div></p></div><div class="aux"><div class="resc">Cole P. Anderson, et al. Front Pharmacol. 2014;5:113.</div><p class="links"><a class="dblinks" href="/pubmed/24904417" ref="ordinalpos=4624&amp;ncbi_uid=6199574&amp;link_uid=24904417">Citation</a><a class="dblinks" href="/pmc/articles/PMC4033076/" ref="ordinalpos=4624&amp;ncbi_uid=6199574&amp;link_uid=PMC4033076">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4625.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3865371/figure/F3/" ref="ordinalpos=4625&amp;ncbi_uid=5640934&amp;link_uid=PMC3865371" href="/pmc/articles/PMC3865371/"><img src="/pmc/articles/instance/3865371/bin/fphys-04-00375-g0003.gif" src-large="/pmc/articles/instance/3865371/bin/fphys-04-00375-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3865371/" ref="ordinalpos=4625&amp;ncbi_uid=5640934&amp;link_uid=PMC3865371" image-link="/pmc/articles/PMC3865371/figure/F3/" class="imagepopup">Figure 3.  From: Host-microbe interactions in the gut of Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div><b>The Imd <span class="highlight" style="background-color:">pathway</span> and its negative regulators in the gut.</b> The Imd <span class="highlight" style="background-color:">pathway</span> is activated by the recognition of diaminopimelic acid-type peptidoglycan (DAP-PGN) by the membrane receptor PGRP-LC, or tracheal cytotoxin (TCT) by the intracellular receptor PGRP-LE. Activated PGRP-LC or PGRP-LE then recruits the adaptor protein Imd, which triggers the <span class="highlight" style="background-color:">signaling</span> cascade of the <span class="highlight" style="background-color:">pathway</span> and induces the nuclear translocation of Relish, a nuclear factor-B (NF-B) transcription factor of <i>Drosophila</i>. The NF-B protein Relish activates the expression of antimicrobial peptides (AMPs) and negative regulators of this <span class="highlight" style="background-color:">pathway</span>. PGRP-LB and PGRP-SC catalyze the degradation of peptidoglycan and TCT, resulting in a decrease in the amount of ligands for the recognition receptors PGRP-LC and PGRP-LE. PGRP-LF and Pirk inhibit the activity of those receptors, transglutaminase (TG) blocks the nuclear translocation of Relish, and Caudal modulates the transcriptional response of Relish.</div></p></div><div class="aux"><div class="resc">Takayuki Kuraishi, et al. Front Physiol. 2013;4:375.</div><p class="links"><a class="dblinks" href="/pubmed/24381562" ref="ordinalpos=4625&amp;ncbi_uid=5640934&amp;link_uid=24381562">Citation</a><a class="dblinks" href="/pmc/articles/PMC3865371/" ref="ordinalpos=4625&amp;ncbi_uid=5640934&amp;link_uid=PMC3865371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4626.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3185776/figure/f2-viruses-03-00886/" ref="ordinalpos=4626&amp;ncbi_uid=3280180&amp;link_uid=PMC3185776" href="/pmc/articles/PMC3185776/"><img src="/pmc/articles/instance/3185776/bin/viruses-03-00886f2.gif" src-large="/pmc/articles/instance/3185776/bin/viruses-03-00886f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3185776/" ref="ordinalpos=4626&amp;ncbi_uid=3280180&amp;link_uid=PMC3185776" image-link="/pmc/articles/PMC3185776/figure/f2-viruses-03-00886/" class="imagepopup">Figure 2. From: Targeting HTLV-1 Activation of NF?B in Mouse Models and ATLL Patients. </a></p><div class="supp"><p class="details"><div>Tax targets the NFB <span class="highlight" style="background-color:">pathway</span>. A simplified schematic of NFB <span class="highlight" style="background-color:">signaling</span> pathways highlighting regulatory proteins commonly targeted by viruses [1,2,13]. HTLV-1 Tax targets the <span class="highlight" style="background-color:">pathway</span> at several points. (<b>a</b>) Tax leads to the activation of receptor-associated kinases that signal through the NFB <span class="highlight" style="background-color:">pathway</span> [16,17]. (<b>b</b>) Tax directly binds to IKK (NEMO) which leads to constitutive phosphorylation and degradation of the NFB repressor IB [15]. (<b>c</b>) Tax activates and recruits IKK to p100 stimulating phosphorylation, ubiquitination, and processing to p52 leading to nuclear translocation [19]. (<b>d</b>) Tax alters binding or recognition of a variety of transcription factors and DNA binding proteins increasing the number of genes regulated by the NFB <span class="highlight" style="background-color:">pathway</span> [18].</div></p></div><div class="aux"><div class="resc">Daniel A. Rauch, et al. Viruses. 2011 June;3(6):886-900.</div><p class="links"><a class="dblinks" href="/pubmed/21994759" ref="ordinalpos=4626&amp;ncbi_uid=3280180&amp;link_uid=21994759">Citation</a><a class="dblinks" href="/pmc/articles/PMC3185776/" ref="ordinalpos=4626&amp;ncbi_uid=3280180&amp;link_uid=PMC3185776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4627.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2065877/figure/ppat-0030166-g006/" ref="ordinalpos=4627&amp;ncbi_uid=1114741&amp;link_uid=PMC2065877" href="/pmc/articles/PMC2065877/"><img src="/pmc/articles/instance/2098818/bin/ppat.0030166.g006.gif" src-large="/pmc/articles/instance/2098818/bin/ppat.0030166.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2065877/" ref="ordinalpos=4627&amp;ncbi_uid=1114741&amp;link_uid=PMC2065877" image-link="/pmc/articles/PMC2065877/figure/ppat-0030166-g006/" class="imagepopup">Figure 6. LMP1 Activates Akt <span class="highlight" style="background-color:">Signaling</span> and Deregulates the Rb Cell Cycle <span class="highlight" style="background-color:">Pathway</span>.  From: EBV Latent Membrane Protein 1 Activates Akt, NF?B, and Stat3 in B Cell Lymphomas. </a></p><div class="supp"><p class="details"><div>(A and B) Immunoblot analysis of purified B cells (CD19+) from the spleens of WT and LMP1 transgenic mice for Akt <span class="highlight" style="background-color:">signaling</span>, probing for (A) activated pAkt and downstream targets, including inactivated pGSK3/, and (B) activated p-mTOR, and total levels of FoxO1. Arrows indicate the positions of  and  isoforms of GSK3. The white line indicates that intervening lanes have been spliced out.<br />(C) Immunoblot analysis for cell cycle proteins regulating the Rb <span class="highlight" style="background-color:">pathway</span>, probing for activated pRb, and total levels of Cdk2 and the Cdk inhibitor p27. Actin was used as a loading control.</div></p></div><div class="aux"><div class="resc">Kathy H. Y Shair, et al. PLoS Pathog. 2007 November;3(11):e166.</div><p class="links"><a class="dblinks" href="/pubmed/17997602" ref="ordinalpos=4627&amp;ncbi_uid=1114741&amp;link_uid=17997602">Citation</a><a class="dblinks" href="/pmc/articles/PMC2065877/" ref="ordinalpos=4627&amp;ncbi_uid=1114741&amp;link_uid=PMC2065877">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4628.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2912204/figure/F1/" ref="ordinalpos=4628&amp;ncbi_uid=2654865&amp;link_uid=PMC2912204" href="/pmc/articles/PMC2912204/"><img src="/pmc/articles/instance/2912204/bin/JCI44026.f1.gif" src-large="/pmc/articles/instance/2912204/bin/JCI44026.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2912204/" ref="ordinalpos=4628&amp;ncbi_uid=2654865&amp;link_uid=PMC2912204" image-link="/pmc/articles/PMC2912204/figure/F1/" class="imagepopup">Figure 1. The PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span>..  From: PIK3CA and KRAS mutations predict for response to everolimus therapy: now that's RAD001             . </a></p><div class="supp"><p class="details"><div>Receptor tyrosine kinases bind with ligand and initiate the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via intermediate molecules (IRS). PI3K becomes activated, which results in phosphorylation of phosphatidylinositol 4,5-bisphosphate (PIP<sub>2</sub>) to phosphatidylinositol 3,4,5-trisphosphate (PIP<sub>3</sub>), a process that is reversed by PTEN. At the cell membrane, proteins with pleckstrin homology domains then become phosphorylated via PIP<sub>3</sub> (phosphoinositide-dependent protein kinase1 [PDK1] and AKT). PDK1 can also phosphorylate critical residues on AKT. The tumor suppressor complex of TSC1/TSC2 normally inhibits mTOR activation via Ras homolog enriched in brain (Rheb). Activated AKT prevents this inhibition, leading to activation of the mTOR/Raptor complex known as mTOR complex 1 (mTORC1). This complex can be inhibited by rapamycin and its analogs, including everolimus. Ultimately, mTORC1 leads to the activation of downstream proteins involved in the initiation of protein synthesis, resulting in cellular growth. Receptor tyrosine kinase activation also initiates MAPK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>, which leads to cell cycle progression and proliferation. MAPK <span class="highlight" style="background-color:">pathway</span> activation can also augment PI3K <span class="highlight" style="background-color:">signaling</span>. MEK, MAPK/ERK kinase.    </div></p></div><div class="aux"><div class="resc">Morassa Mohseni, et al. J Clin Invest. 2010 August 2;120(8):2655-2658.</div><p class="links"><a class="dblinks" href="/pubmed/20664174" ref="ordinalpos=4628&amp;ncbi_uid=2654865&amp;link_uid=20664174">Citation</a><a class="dblinks" href="/pmc/articles/PMC2912204/" ref="ordinalpos=4628&amp;ncbi_uid=2654865&amp;link_uid=PMC2912204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4629.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1853162/figure/fig2/" ref="ordinalpos=4629&amp;ncbi_uid=950597&amp;link_uid=PMC1853162" href="/pmc/articles/PMC1853162/"><img src="/pmc/articles/instance/1853162/bin/biophysj00090084F02_RGB.gif" src-large="/pmc/articles/instance/1853162/bin/biophysj00090084F02_RGB.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1853162/" ref="ordinalpos=4629&amp;ncbi_uid=950597&amp;link_uid=PMC1853162" image-link="/pmc/articles/PMC1853162/figure/fig2/" class="imagepopup">FIGURE 2.  From: Mathematical Models of Specificity in Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A simple network consisting of two parallel cascades with cross talk. The network consists of two pathways, <i>X</i> and <i>Y</i>. <span class="highlight" style="background-color:">Pathway</span> <i>X</i> has three components (<i>x</i><sub>0</sub>, <i>x</i><sub>1</sub>, and <i>x</i><sub>2</sub>); component <i>x</i><sub>0</sub> is input signal itself, or is activated in manner that is strictly proportional to the input signal. (Because the input signal can have any shape, <i>x</i><sub>0</sub> can abstractly represent any upstream component, e.g., a receptor, a G-protein, a kinase, etc.) The parameters <i>a</i><sub>1</sub> and <i>a</i><sub>2</sub> are activation rate constants, and <img src="/pmc/articles/instance/1853162/equ/M139" class="equation" style="vertical-align: middle; border-style: none;" /> and <img src="/pmc/articles/instance/1853162/equ/M140" class="equation" style="vertical-align: middle; border-style: none;" /> are deactivation, or decay, rate constants. For example, <i>a</i><sub>1</sub> is the rate constant for the activation of <i>x</i><sub>1</sub> by <i>x</i><sub>0</sub>, and <img src="/pmc/articles/instance/1853162/equ/M141" class="equation" style="vertical-align: middle; border-style: none;" /> is the rate constant for the deactivation of <i>x</i><sub>1</sub>. (<i>B</i>) Outputs of this network in response to <i>X</i> <span class="highlight" style="background-color:">signaling</span> (that is, <i>X</i> input) and <i>Y</i> <span class="highlight" style="background-color:">signaling</span>. Signal <i>x</i><sub>0</sub> does not lead to the production of <i>y</i><sub>2</sub>, because none of the components of <i>X</i> can activate or inhibit components of <i>Y</i>. Thus, the specificity of <i>X</i> with respect to <i>Y</i> and fidelity of <i>Y</i> with respect to <i>X</i> are complete (see text for definitions). In contrast, <i>y</i><sub>1</sub> (which might be a kinase of <span class="highlight" style="background-color:">pathway</span> <i>Y</i>) weakly activates <i>x</i><sub>2</sub> (which might be a transcription factor for <span class="highlight" style="background-color:">pathway</span> <i>X</i>), at a rate characterized by the leak constant, <i>h</i><sub>leak.</sub> Thus, the specificity of <i>Y</i> and the fidelity of <i>X</i> are finite functions of <i>h</i><sub>leak</sub> and other key parameters of the network. In particular, in the text it is shown that network specificity is proportional to 1/<i>h</i><sub>leak</sub>. (<i>C</i>) Depiction of the ratios equal to the specificity of <span class="highlight" style="background-color:">pathway</span> <i>Y</i> and the fidelity of <span class="highlight" style="background-color:">pathway</span> <i>X</i>.</div></p></div><div class="aux"><div class="resc">Lee Bardwell, et al. Biophys J. 2007 May 15;92(10):3425-3441.</div><p class="links"><a class="dblinks" href="/pubmed/17325015" ref="ordinalpos=4629&amp;ncbi_uid=950597&amp;link_uid=17325015">Citation</a><a class="dblinks" href="/pmc/articles/PMC1853162/" ref="ordinalpos=4629&amp;ncbi_uid=950597&amp;link_uid=PMC1853162">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4630.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2917029/figure/F8/" ref="ordinalpos=4630&amp;ncbi_uid=2403257&amp;link_uid=PMC2917029" href="/pmc/articles/PMC2917029/"><img src="/pmc/articles/instance/2917029/bin/bcr2588-8.gif" src-large="/pmc/articles/instance/2917029/bin/bcr2588-8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2917029/" ref="ordinalpos=4630&amp;ncbi_uid=2403257&amp;link_uid=PMC2917029" image-link="/pmc/articles/PMC2917029/figure/F8/" class="imagepopup">Figure 8.  From: Progesterone reverses the mesenchymal phenotypes of basal phenotype breast cancer cells via a membrane progesterone receptor mediated <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic illustration of molecular <span class="highlight" style="background-color:">pathway</span> initiated by P4 <span class="highlight" style="background-color:">signaling</span> in BPBC cells</b>. MB468 cells usually maintain the activities of PI3K/Akt at high levels because these cells contain a mutant <i>pten </i>gene and over expressed epidermal growth factor receptor (EGFR). Activation of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> is a vital signal for cell survival and epithelial to mesenchymal transition (EMT). It is assumed that the binding of progesterone (P4) to membrane progesterone receptor (mPR)  in the caveolar membrane of the cells inhibits EMT-relevant events either directly through Caveolar (Cav) 1 activation or through a cross interaction with EGFR and trans-activation of Cav-1 which subsequently inactivate PI3K/Akt <span class="highlight" style="background-color:">pathway</span>. Inactivation of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> subsequently inhibits the nuclear translocation of snail and then modulates expression of other EMT-relevant proteins.</div></p></div><div class="aux"><div class="resc">Lian Zuo, et al. Breast Cancer Res. 2010;12(3):R34-R34.</div><p class="links"><a class="dblinks" href="/pubmed/20540763" ref="ordinalpos=4630&amp;ncbi_uid=2403257&amp;link_uid=20540763">Citation</a><a class="dblinks" href="/pmc/articles/PMC2917029/" ref="ordinalpos=4630&amp;ncbi_uid=2403257&amp;link_uid=PMC2917029">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4631.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3083852/figure/F1/" ref="ordinalpos=4631&amp;ncbi_uid=3381196&amp;link_uid=PMC3083852" href="/pmc/articles/PMC3083852/"><img src="/pmc/articles/instance/3083852/bin/nihms-265332-f0001.gif" src-large="/pmc/articles/instance/3083852/bin/nihms-265332-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3083852/" ref="ordinalpos=4631&amp;ncbi_uid=3381196&amp;link_uid=PMC3083852" image-link="/pmc/articles/PMC3083852/figure/F1/" class="imagepopup">Fig. 1.  From: NF-B/Rel Proteins and the Humoral Immune Responses of Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div><i>Drosophila</i> Toll <span class="highlight" style="background-color:">pathway</span>. <i>Drosophila</i> Toll <span class="highlight" style="background-color:">pathway</span> is a cytokine receptor <span class="highlight" style="background-color:">pathway</span> that responds indirectly to microbial infection. Distinct circulating receptor proteins recognize different microbial structures or activities and activate a protease cascade, which culminates in the processing of Sptzle. Lys-type PGN, from bacterial cell walls, is recognized by PGRP-SA/PGRP-SD and GNBP1, while GNBP3 detects <i></i>-(1, 3)-glucan from fungal cell walls. These upstream recognition events trigger a proteolytic cascade which proceeds through the serine proteases ModSP and Grass to activate Sptzle Processing Enzyme (SPE) which in turn cleaves pro-Sptzle to release the mature Toll ligand, Sptzle. In addition, virulence-associated proteases, released by pathogens, cleave and activate another <span class="highlight" style="background-color:">pathway</span> that requires the protease Persephone and converges on the activation of SPE. The intracellular Toll <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is very homologous to the mammalian MyD88-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. It signals through an upstream complex containing the adaptor proteins dMyD88, Tube and the kinase Pelle (an IRAK homolog). Pelle triggers the phosphorylation, K48-ubiquitination and proteasomal degradation of the I<i></i>B protein, Cactus, thereby releasing the <i>Drosophila</i> NF-<i></i>B/Rel factors DIF/Dorsal to translocate into the nucleus and transcribe target AMP genes, such as <i>Drosomycin</i></div></p></div><div class="aux"><div class="resc">Sandhya Ganesan, et al. Curr Top Microbiol Immunol. ;349:25-60.</div><p class="links"><a class="dblinks" href="/pubmed/20852987" ref="ordinalpos=4631&amp;ncbi_uid=3381196&amp;link_uid=20852987">Citation</a><a class="dblinks" href="/pmc/articles/PMC3083852/" ref="ordinalpos=4631&amp;ncbi_uid=3381196&amp;link_uid=PMC3083852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4632.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2904940/figure/F7/" ref="ordinalpos=4632&amp;ncbi_uid=2710391&amp;link_uid=PMC2904940" href="/pmc/articles/PMC2904940/"><img src="/pmc/articles/instance/2904940/bin/1496fig7.gif" src-large="/pmc/articles/instance/2904940/bin/1496fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2904940/" ref="ordinalpos=4632&amp;ncbi_uid=2710391&amp;link_uid=PMC2904940" image-link="/pmc/articles/PMC2904940/figure/F7/" class="imagepopup">Figure 7. From: The CNK1 scaffold binds cytohesins and promotes insulin <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>CNK is required for IR to IRS/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> but not for PDGF to PI3K/AKT <span class="highlight" style="background-color:">signaling</span>. (<i>A</i>) HeLa cells stably expressing vector control, WT-IRS1, or Myr-IRS1 were transfected with control or CNK1-siRNAs prior to serum starvation and insulin treatment. Lysates were prepared and examined by immunoblot analysis. (<i>B</i>) HeLa cells stably expressing PDGFR were transfected with control or CNK1-siRNAs prior to serum starvation and treatment with either insulin or PDGF. Lysates or membrane fractions were prepared and examined by immunoblot analysis. (<i>C</i>) Model for CNK1 function in insulin <span class="highlight" style="background-color:">signaling</span> (see the text for details).</div></p></div><div class="aux"><div class="resc">Junghwa Lim, et al. Genes Dev. 2010 July 15;24(14):1496-1506.</div><p class="links"><a class="dblinks" href="/pubmed/20634316" ref="ordinalpos=4632&amp;ncbi_uid=2710391&amp;link_uid=20634316">Citation</a><a class="dblinks" href="/pmc/articles/PMC2904940/" ref="ordinalpos=4632&amp;ncbi_uid=2710391&amp;link_uid=PMC2904940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4633.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2860937/figure/F1/" ref="ordinalpos=4633&amp;ncbi_uid=2654742&amp;link_uid=PMC2860937" href="/pmc/articles/PMC2860937/"><img src="/pmc/articles/instance/2860937/bin/JCI43030.f1.gif" src-large="/pmc/articles/instance/2860937/bin/JCI43030.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2860937/" ref="ordinalpos=4633&amp;ncbi_uid=2654742&amp;link_uid=PMC2860937" image-link="/pmc/articles/PMC2860937/figure/F1/" class="imagepopup">Figure 1. mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in genetic syndromes..  From: Caveat mTOR: aberrant <span class="highlight" style="background-color:">signaling</span> disrupts corticogenesis. </a></p><div class="supp"><p class="details"><div>Shown is the relationship among different proteins encoded by genes known to be responsible for genetic syndromes associated with activation of mTORC1. Activation of mTORC1 ultimately leads to increased protein synthesis, and thereby increased cell growth. Germline mutations that result in mTORC1 hyperactivation are therefore associated with syndromes characterized by cellular overgrowth, usually in the form of hamartomas. LKB1 also activates an mTORC1-independent <span class="highlight" style="background-color:">pathway</span> that controls neuronal polarity specification; it is therefore possible that the neuronal migration defects seen in individuals with PMSE may be due to effects on this <span class="highlight" style="background-color:">pathway</span>, and not to hyperactivation of mTOR <span class="highlight" style="background-color:">signaling</span>. This would explain the lack of neuronal migration defects in disorders associated with germline mutations that do not affect LKB1 localization or function. 4E-BP1, eIF4E-binding protein 1; eIF4E, eukaryotic translation initiation factor 4E; MAP, microtubule-associate protein; PDK1, phosphoinositide-dependent protein kinase1; Rheb, Ras homolog enriched in brain; S6K1, S6 kinase, 70 kDa, polypeptide 1.</div></p></div><div class="aux"><div class="resc">Lucy R. Osborne. J Clin Invest. 2010 May 3;120(5):1392-1395.</div><p class="links"><a class="dblinks" href="/pubmed/20424320" ref="ordinalpos=4633&amp;ncbi_uid=2654742&amp;link_uid=20424320">Citation</a><a class="dblinks" href="/pmc/articles/PMC2860937/" ref="ordinalpos=4633&amp;ncbi_uid=2654742&amp;link_uid=PMC2860937">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4634.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1461420/figure/fig4/" ref="ordinalpos=4634&amp;ncbi_uid=654802&amp;link_uid=PMC1461420" href="/pmc/articles/PMC1461420/"><img src="/pmc/articles/instance/1461420/bin/GEN1731177f4.gif" src-large="/pmc/articles/instance/1461420/bin/GEN1731177f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1461420/" ref="ordinalpos=4634&amp;ncbi_uid=654802&amp;link_uid=PMC1461420" image-link="/pmc/articles/PMC1461420/figure/fig4/" class="imagepopup">Figure 4. From: A Novel Gain-of-Function Mutant of the Cyclic GMP-Dependent Protein Kinase egl-4 Affects Multiple Physiological Processes in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Distinct <span class="highlight" style="background-color:">signaling</span> pathways mediate <i>egl-4</i> phenotypes. <i>egl-4</i> promotes a reduced longevity by negatively regulating the activity of the insulin <span class="highlight" style="background-color:">signaling</span> transcription factor <i>daf-16</i> (Hirose et al. 2003), promotes a reduced body size by negatively regulating the activity of the <i>dbl-1</i> TGF- (Fujiwara et al. 2002; Hirose et al. 2003), promotes a pale intestine and nondauer development by positively regulating the <i>daf-7</i> TGF-, as well as by another unidentified <span class="highlight" style="background-color:">pathway</span> (Trent et al. 1983; Daniels et al. 2000) (this work), and promotes a reduced locomotion and feeding rate in the presence of food by an as yet unidentified <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">David M. Raizen, et al. Genetics. 2006 May;173(1):177-187.</div><p class="links"><a class="dblinks" href="/pubmed/16547093" ref="ordinalpos=4634&amp;ncbi_uid=654802&amp;link_uid=16547093">Citation</a><a class="dblinks" href="/pmc/articles/PMC1461420/" ref="ordinalpos=4634&amp;ncbi_uid=654802&amp;link_uid=PMC1461420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4635.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3415084/figure/fig2/" ref="ordinalpos=4635&amp;ncbi_uid=4178372&amp;link_uid=PMC3415084" href="/pmc/articles/PMC3415084/"><img src="/pmc/articles/instance/3415084/bin/JP2012-631038.002.gif" src-large="/pmc/articles/instance/3415084/bin/JP2012-631038.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3415084/" ref="ordinalpos=4635&amp;ncbi_uid=4178372&amp;link_uid=PMC3415084" image-link="/pmc/articles/PMC3415084/figure/fig2/" class="imagepopup">Figure 2.  From: Developmental Programming in Response to Intrauterine Growth Restriction Impairs Myoblast Function and Skeletal Muscle Metabolism. </a></p><div class="supp"><p class="details"><div>Impact of placental insufficiency on endocrine responsiveness in fetal myoblasts and myofibers. Adrenergic activity increases due to greater circulating catecholamines. Adrenergic receptor <i></i> subtype-specific desensitization results in a greater proportion of <span class="highlight" style="background-color:">signaling</span> through Adr<i></i>1 and Adr<i></i>3 because Adr<i></i>2 expression is reduced. Insulin <span class="highlight" style="background-color:">signaling</span> is reduced due to adrenergic suppression of insulin secretion in pancreatic <i></i>-cells and by muscle adrenergic <span class="highlight" style="background-color:">signaling</span> that negatively influences the insulin-Akt2 intercellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. These developmental adaptations reduce rates of myoblast proliferation and differentiation as well as glucose metabolism in skeletal muscle.</div></p></div><div class="aux"><div class="resc">D. T. Yates, et al. J Pregnancy. 2012;2012:631038.</div><p class="links"><a class="dblinks" href="/pubmed/22900186" ref="ordinalpos=4635&amp;ncbi_uid=4178372&amp;link_uid=22900186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3415084/" ref="ordinalpos=4635&amp;ncbi_uid=4178372&amp;link_uid=PMC3415084">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4636.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3428657/figure/F1/" ref="ordinalpos=4636&amp;ncbi_uid=4197584&amp;link_uid=PMC3428657" href="/pmc/articles/PMC3428657/"><img src="/pmc/articles/instance/3428657/bin/1471-2164-13-306-1.gif" src-large="/pmc/articles/instance/3428657/bin/1471-2164-13-306-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3428657/" ref="ordinalpos=4636&amp;ncbi_uid=4197584&amp;link_uid=PMC3428657" image-link="/pmc/articles/PMC3428657/figure/F1/" class="imagepopup">Figure 1.  From: Let-7b regulates the expression of the growth hormone receptor gene in deletion-type dwarf chickens. </a></p><div class="supp"><p class="details"><div><b>Schematic illustration of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of</b><b><i>GHR</i></b><b>regulated by let-7b.</b> miRNA let-7b inhibits the expression of <i>GHR</i> and regulates <i>SOCS3</i><b>.</b><i>SOCS3</i> regulates the growth and development of chickens through three <span class="highlight" style="background-color:">signaling</span> pathways: inhibiting the phosphorylation of tyrosine in <i>IRS1</i>; inhibiting <i>LEPR</i>; and inhibiting <i>JAK</i>. <i>IRS1</i>, <i>LEPR</i> and <i>JAK</i> are involved in growth, fat synthesis, and cell proliferation.</div></p></div><div class="aux"><div class="resc">Shumao Lin, et al. BMC Genomics. 2012;13:306-306.</div><p class="links"><a class="dblinks" href="/pubmed/22781587" ref="ordinalpos=4636&amp;ncbi_uid=4197584&amp;link_uid=22781587">Citation</a><a class="dblinks" href="/pmc/articles/PMC3428657/" ref="ordinalpos=4636&amp;ncbi_uid=4197584&amp;link_uid=PMC3428657">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4637.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2573245/figure/f6/" ref="ordinalpos=4637&amp;ncbi_uid=1499033&amp;link_uid=PMC2573245" href="/pmc/articles/PMC2573245/"><img src="/pmc/articles/instance/2573245/bin/zjv0220812040006.gif" src-large="/pmc/articles/instance/2573245/bin/zjv0220812040006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2573245/" ref="ordinalpos=4637&amp;ncbi_uid=1499033&amp;link_uid=PMC2573245" image-link="/pmc/articles/PMC2573245/figure/f6/" class="imagepopup">FIG. 6. From: Hepatitis C Virus Core Protein Augments Androgen Receptor-Mediated <span class="highlight" style="background-color:">Signaling</span> . </a></p><div class="supp"><p class="details"><div>Proposed model for HCV-induced AR activation. Inflammatory cytokine IL-6 activates STAT3 <span class="highlight" style="background-color:">signaling</span>. HCV also enhances the phosphorylation of Tyr-705 and Ser-727 of STAT3 through the core protein and enhances AR expression. AG490, a JAK-1 protein inhibitor, inhibits the JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, suggesting an association of STAT3 as a downstream modulator.</div></p></div><div class="aux"><div class="resc">Tatsuo Kanda, et al. J Virol. 2008 November;82(22):11066-11072.</div><p class="links"><a class="dblinks" href="/pubmed/18768969" ref="ordinalpos=4637&amp;ncbi_uid=1499033&amp;link_uid=18768969">Citation</a><a class="dblinks" href="/pmc/articles/PMC2573245/" ref="ordinalpos=4637&amp;ncbi_uid=1499033&amp;link_uid=PMC2573245">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4638.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2596750/figure/F8/" ref="ordinalpos=4638&amp;ncbi_uid=1532159&amp;link_uid=PMC2596750" href="/pmc/articles/PMC2596750/"><img src="/pmc/articles/instance/2596750/bin/nihms70476f8.gif" src-large="/pmc/articles/instance/2596750/bin/nihms70476f8.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2596750/" ref="ordinalpos=4638&amp;ncbi_uid=1532159&amp;link_uid=PMC2596750" image-link="/pmc/articles/PMC2596750/figure/F8/" class="imagepopup">FIGURE 8.  From: Lymphotoxin-12 and LIGHT Induce Classical and Noncanonical NF-B-Dependent Proinflammatory Gene Expression in Vascular Endothelial Cells. </a></p><div class="supp"><p class="details"><div>Model of TNFR1 and LT<i></i>R ligation-induced <span class="highlight" style="background-color:">signaling</span> in vascular EC. This model depicts the relative strengths of activation of classical and noncanonical NF-<i></i>B <span class="highlight" style="background-color:">signaling</span> and the JNK <span class="highlight" style="background-color:">pathway</span> following TNF and LT<i></i>R ligation in vascular EC. Strong activation is indicated by the thick arrow, weaker induction by the thin arrow, and minor activation by the dotted arrow, respectively.</div></p></div><div class="aux"><div class="resc">Lisa A. Madge, et al. J Immunol. ;180(5):3467-3477.</div><p class="links"><a class="dblinks" href="/pubmed/18292573" ref="ordinalpos=4638&amp;ncbi_uid=1532159&amp;link_uid=18292573">Citation</a><a class="dblinks" href="/pmc/articles/PMC2596750/" ref="ordinalpos=4638&amp;ncbi_uid=1532159&amp;link_uid=PMC2596750">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4639.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC540032/figure/gki078f1/" ref="ordinalpos=4639&amp;ncbi_uid=438923&amp;link_uid=PMC540032" href="/pmc/articles/PMC540032/"><img src="/pmc/articles/instance/540032/bin/gki078f1.gif" src-large="/pmc/articles/instance/540032/bin/gki078f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC540032/" ref="ordinalpos=4639&amp;ncbi_uid=438923&amp;link_uid=PMC540032" image-link="/pmc/articles/PMC540032/figure/gki078f1/" class="imagepopup">Figure 1.  From: The PANTHER database of protein families, subfamilies, functions and pathways. </a></p><div class="supp"><p class="details"><div> CellDesigner (6) diagram of the insulin/IGF receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Proteins (blue and brown boxes) are mapped onto PANTHER HMMs. Active forms (dashed-line boxes) and phosphorylated forms (small circles around the letter P) of proteins are clearly indicated in the diagram. Over 60 pathways (mostly <span class="highlight" style="background-color:">signaling</span> pathways) are currently available.</div></p></div><div class="aux"><div class="resc">Huaiyu Mi, et al. Nucleic Acids Res. 2005 January 1;33(Database issue):D284-D288.</div><p class="links"><a class="dblinks" href="/pubmed/15608197" ref="ordinalpos=4639&amp;ncbi_uid=438923&amp;link_uid=15608197">Citation</a><a class="dblinks" href="/pmc/articles/PMC540032/" ref="ordinalpos=4639&amp;ncbi_uid=438923&amp;link_uid=PMC540032">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4640.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2270852/figure/F5/" ref="ordinalpos=4640&amp;ncbi_uid=1350517&amp;link_uid=PMC2270852" href="/pmc/articles/PMC2270852/"><img src="/pmc/articles/instance/2270852/bin/1471-2407-8-43-5.gif" src-large="/pmc/articles/instance/2270852/bin/1471-2407-8-43-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2270852/" ref="ordinalpos=4640&amp;ncbi_uid=1350517&amp;link_uid=PMC2270852" image-link="/pmc/articles/PMC2270852/figure/F5/" class="imagepopup">Figure 5.  From: Immunohistochemical analysis of changes in <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation downstream of growth factor receptors in pancreatic duct cell carcinogenesis. </a></p><div class="supp"><p class="details"><div><b>A schematic of <span class="highlight" style="background-color:">signaling</span> pathways in PDAC</b>. Higher (red) and lower (green) <span class="highlight" style="background-color:">signaling</span> protein levels characterized tumors compared with non-neoplastic ducts. Relationships between proteins are positive (line), stimulatory (arrows) or inhibitory (T-bar).</div></p></div><div class="aux"><div class="resc">Nhu-An Pham, et al. BMC Cancer. 2008;8:43-43.</div><p class="links"><a class="dblinks" href="/pubmed/18254976" ref="ordinalpos=4640&amp;ncbi_uid=1350517&amp;link_uid=18254976">Citation</a><a class="dblinks" href="/pmc/articles/PMC2270852/" ref="ordinalpos=4640&amp;ncbi_uid=1350517&amp;link_uid=PMC2270852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4641.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2878507/figure/F4/" ref="ordinalpos=4641&amp;ncbi_uid=2998993&amp;link_uid=PMC2878507" href="/pmc/articles/PMC2878507/"><img src="/pmc/articles/instance/2878507/bin/zbc0271020120004.gif" src-large="/pmc/articles/instance/2878507/bin/zbc0271020120004.jpg" alt="FIGURE 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2878507/" ref="ordinalpos=4641&amp;ncbi_uid=2998993&amp;link_uid=PMC2878507" image-link="/pmc/articles/PMC2878507/figure/F4/" class="imagepopup">FIGURE 4. From: Full-length, Membrane-anchored TWEAK Can Function as a Juxtacrine <span class="highlight" style="background-color:">Signaling</span> Molecule and Activate the NF-?B <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Membrane TWEAK expression in HEK293/NFB-luc/Fn14 cells can activate the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> <i>A</i>, stably transfected HEK293/NFB-luc/Fn14 cells were either left untransfected (<i>No DNA</i>) or transfected with either the pcDNA3 vector or the pcDNA3/TWK-dF1/F2 plasmid encoding Myc-tagged non-cleavable TWEAK. The cells were harvested 1 day later and lysed. Equal amounts of protein were used for Western blot analysis using either an anti-Myc epitope (<i>top</i>) or anti-tubulin antibody. <i>B</i>, NF-B reporter activation in the control and transiently transfected cells was measured using a luminometer. The values shown are mean  S.D. (<i>error bars</i>) of triplicate wells from one representative experiment of three independent experiments.</div></p></div><div class="aux"><div class="resc">Sharron A. N. Brown, et al. J Biol Chem. 2010 June 4;285(23):17432-17441.</div><p class="links"><a class="dblinks" href="/pubmed/20385556" ref="ordinalpos=4641&amp;ncbi_uid=2998993&amp;link_uid=20385556">Citation</a><a class="dblinks" href="/pmc/articles/PMC2878507/" ref="ordinalpos=4641&amp;ncbi_uid=2998993&amp;link_uid=PMC2878507">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4642.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3511450/figure/pone-0050085-g002/" ref="ordinalpos=4642&amp;ncbi_uid=4429597&amp;link_uid=PMC3511450" href="/pmc/articles/PMC3511450/"><img src="/pmc/articles/instance/3511450/bin/pone.0050085.g002.gif" src-large="/pmc/articles/instance/3511450/bin/pone.0050085.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3511450/" ref="ordinalpos=4642&amp;ncbi_uid=4429597&amp;link_uid=PMC3511450" image-link="/pmc/articles/PMC3511450/figure/pone-0050085-g002/" class="imagepopup">Figure 2. Optimization of a toy model to <span class="highlight" style="background-color:">signaling</span> data..  From: Non Linear Programming (NLP) Formulation for Quantitative Modeling of Protein Signal Transduction Pathways. </a></p><div class="supp"><p class="details"><div>(A) Generic <span class="highlight" style="background-color:">pathway</span> is represented as a signed directed graph, also refers as PKN. Green nodes refer to different cytokines (ligands) where the <span class="highlight" style="background-color:">signaling</span> process initiates; Red nodes refer to inhibitors present in the in-silico dataset; Grey nodes refer to measured proteins; White nodes refer to latent species, i.e. proteins whose activation state is not measured. (B) In-silico <span class="highlight" style="background-color:">signaling</span> data under combinatorial treatment with stimuli (TGF, TNF, no-treatment) and inhibitors (mek12i, pi3ki, no-inhibitor). Each subplot shows the average activation level within 30 minutes upon stimulation [37]. Red background refers to model-prediction mismatches (C) Optimized <span class="highlight" style="background-color:">pathway</span>, grey arrows refer to reactions with limited activity (<i>z<sub>i</sub><sup>k</sup></i>) (caused by <i>a</i> parameters being close to 0). The opacity of each edge corresponds to the activity (<i>z<sub>i</sub><sup>k</sup></i>) of the corresponding reaction. (D) In silico <span class="highlight" style="background-color:">signaling</span> dataset and fitness error after the optimization procedure. Decrease in the red background color shows the optimized model is in accordance to the <span class="highlight" style="background-color:">signaling</span> dataset. (E) Optimized transfer functions presented in C.</div></p></div><div class="aux"><div class="resc">Alexander Mitsos, et al. PLoS One. 2012;7(11):e50085.</div><p class="links"><a class="dblinks" href="/pubmed/23226239" ref="ordinalpos=4642&amp;ncbi_uid=4429597&amp;link_uid=23226239">Citation</a><a class="dblinks" href="/pmc/articles/PMC3511450/" ref="ordinalpos=4642&amp;ncbi_uid=4429597&amp;link_uid=PMC3511450">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4643.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4358430/figure/fig09/" ref="ordinalpos=4643&amp;ncbi_uid=7291505&amp;link_uid=PMC4358430" href="/pmc/articles/PMC4358430/"><img src="/pmc/articles/instance/4358430/bin/ijcem0008-0058-f9.gif" src-large="/pmc/articles/instance/4358430/bin/ijcem0008-0058-f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4358430/" ref="ordinalpos=4643&amp;ncbi_uid=7291505&amp;link_uid=PMC4358430" image-link="/pmc/articles/PMC4358430/figure/fig09/" class="imagepopup">Figure 9.  From: AMPK is required for PM2.5-induced autophagy in human lung epithelial A549 cells. </a></p><div class="supp"><p class="details"><div>Schematic diagram describing PM<sub>2.5</sub> induced autophagy mediated by the AMPK <span class="highlight" style="background-color:">pathway</span>. PM<sub>2.5</sub> exposure in A549 cells causes the activation of AMPK, potentially through the ROS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which subsequently leads to autophagy. The AKT, MAPK and p53 <span class="highlight" style="background-color:">signaling</span> pathways are also activated, with an inhibitory effect by the AKT, ERK and p38 pathways and no apparent effect on autophagy by JNK and p53 <span class="highlight" style="background-color:">signaling</span> pathways. These results suggest there is a balance of activating and inhibitory pathways, and that activating pathways (ROS and AMPK) predominately regulate the response to PM<sub>2.5</sub> to promote an overall induction of autophagy. The DNA damage induced by PM<sub>2.5</sub> also activates apoptosis, which is counteracted in part by the induction of autophagy.</div></p></div><div class="aux"><div class="resc">Yahong Wang, et al. Int J Clin Exp Med. 2015;8(1):58-72.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4643&amp;ncbi_uid=7291505&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4358430/" ref="ordinalpos=4643&amp;ncbi_uid=7291505&amp;link_uid=PMC4358430">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4644.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399319/figure/F1/" ref="ordinalpos=4644&amp;ncbi_uid=4126042&amp;link_uid=PMC3399319" href="/pmc/articles/PMC3399319/"><img src="/pmc/articles/instance/3399319/bin/ijbsv08p0964g01.gif" src-large="/pmc/articles/instance/3399319/bin/ijbsv08p0964g01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399319/" ref="ordinalpos=4644&amp;ncbi_uid=4126042&amp;link_uid=PMC3399319" image-link="/pmc/articles/PMC3399319/figure/F1/" class="imagepopup">Figure 1.  From: Complexities of TGF-? Targeted Cancer Therapy. </a></p><div class="supp"><p class="details"><div><b> TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. TGF- ligand is secreted as a latent precursor protein, bound to LAP. Activation of TGF- involves cleavage of LAP from the ligand, which then binds to the type II receptor, and drives hetero-tetramerization with the type I receptor. The canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involves phosphorylation of R-Smads (mainly Smad2 and Smad3) by activated TRI. Phosphorylated R-Smads form a complex with the Co-Smad (Smad4), which translocates into the nucleus to bind gene promoters and activate expression of target genes. There are several non-canonical (non-Smad) <span class="highlight" style="background-color:">signaling</span> pathways, whereby TGF- signals through the TGF- receptors to activate TGF- activated kinase 1 (Tak1), Ras and PI3K as well as other pathways.</div></p></div><div class="aux"><div class="resc">Erin C. Connolly, et al. Int J Biol Sci. 2012;8(7):964-978.</div><p class="links"><a class="dblinks" href="/pubmed/22811618" ref="ordinalpos=4644&amp;ncbi_uid=4126042&amp;link_uid=22811618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399319/" ref="ordinalpos=4644&amp;ncbi_uid=4126042&amp;link_uid=PMC3399319">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4645.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539252/figure/f3/" ref="ordinalpos=4645&amp;ncbi_uid=5661006&amp;link_uid=PMC3539252" href="/pmc/articles/PMC3539252/"><img src="/pmc/articles/instance/3539252/bin/fig-3.gif" src-large="/pmc/articles/instance/3539252/bin/fig-3.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539252/" ref="ordinalpos=4645&amp;ncbi_uid=5661006&amp;link_uid=PMC3539252" image-link="/pmc/articles/PMC3539252/figure/f3/" class="imagepopup">FIG. 3. From: The Blood Transcriptional Signature of Chronic Hepatitis C Virus Is Consistent with an Ongoing Interferon-Mediated Antiviral Response. </a></p><div class="supp"><p class="details"><div>The upregulated HCV gene signature is enriched for interferon (IFN) <span class="highlight" style="background-color:">pathway</span> genes. Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis was used to find significantly enriched pathways and networks among the genes that were upregulated in chronic HCV infection. <b>(a)</b> The IFN <span class="highlight" style="background-color:">pathway</span> was found to be significantly enriched (<i>P</i>&lt;0.0001), with several signature genes (shaded) occurring downstream of Jak-Stat <span class="highlight" style="background-color:">signaling</span>. <b>(b)</b> Many direct (<i>solid arrow</i>) and indirect (<i>dashed arrow</i>) interactions were identified among the signature genes (shaded), and others related to the IFN response.</div></p></div><div class="aux"><div class="resc">Christopher R. Bolen, et al. J Interferon Cytokine Res. 2013 January;33(1):15-23.</div><p class="links"><a class="dblinks" href="/pubmed/23067362" ref="ordinalpos=4645&amp;ncbi_uid=5661006&amp;link_uid=23067362">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539252/" ref="ordinalpos=4645&amp;ncbi_uid=5661006&amp;link_uid=PMC3539252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4646.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4141883/figure/F1/" ref="ordinalpos=4646&amp;ncbi_uid=6528117&amp;link_uid=PMC4141883" href="/pmc/articles/PMC4141883/"><img src="/pmc/articles/instance/4141883/bin/nihms613168f1.gif" src-large="/pmc/articles/instance/4141883/bin/nihms613168f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4141883/" ref="ordinalpos=4646&amp;ncbi_uid=6528117&amp;link_uid=PMC4141883" image-link="/pmc/articles/PMC4141883/figure/F1/" class="imagepopup">Fig. 1.  From: Glioma Cell Growth Inhibition Following Photochemical Internalization Enhanced Non-Viral PTEN Gene Transfection. </a></p><div class="supp"><p class="details"><div>Graphic representation of the Pten/PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PTEN has major implication in PI3 kinase (PI3K) signal transduction <span class="highlight" style="background-color:">pathway</span> and negatively controls PI3K phosphorylation. PTEN is implicated in cell cycle progression and cell death control through inhibition of PI3K-Akt signal transduction <span class="highlight" style="background-color:">pathway</span> and in the control of cell migration and spreading through its interaction with focal adhesion kinase (FAK).</div></p></div><div class="aux"><div class="resc">Marlon S. Mathews, et al. Lasers Surg Med. ;44(9):746-754.</div><p class="links"><a class="dblinks" href="/pubmed/23018764" ref="ordinalpos=4646&amp;ncbi_uid=6528117&amp;link_uid=23018764">Citation</a><a class="dblinks" href="/pmc/articles/PMC4141883/" ref="ordinalpos=4646&amp;ncbi_uid=6528117&amp;link_uid=PMC4141883">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4647.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3445712/figure/F4/" ref="ordinalpos=4647&amp;ncbi_uid=5305636&amp;link_uid=PMC3445712" href="/pmc/articles/PMC3445712/"><img src="/pmc/articles/instance/3445712/bin/nihms-396903-f0004.gif" src-large="/pmc/articles/instance/3445712/bin/nihms-396903-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3445712/" ref="ordinalpos=4647&amp;ncbi_uid=5305636&amp;link_uid=PMC3445712" image-link="/pmc/articles/PMC3445712/figure/F4/" class="imagepopup">Figure 4.  From: B-Raf activation cooperates with PTEN loss to drive c-Myc expression in advanced prostate cancer. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analyses of mouse prostate tumors. Western blot analyses were done to assess activation of the PI3-kinaseAktmTOR <span class="highlight" style="background-color:">pathway</span> (A), MAP kinase <span class="highlight" style="background-color:">signaling</span> (B), RAS <span class="highlight" style="background-color:">pathway</span> activation (C), and expression of Myc and Cyclin D1 (D). Western blots were performed using total protein extracts prepared from dorsolateral prostate or prostate tumors from the indicated genotypes. Western Blots were quantified using ImageJ.</div></p></div><div class="aux"><div class="resc">Jingqiang Wang, et al. Cancer Res. 2012 September 15;72(18):4765-4776.</div><p class="links"><a class="dblinks" href="/pubmed/22836754" ref="ordinalpos=4647&amp;ncbi_uid=5305636&amp;link_uid=22836754">Citation</a><a class="dblinks" href="/pmc/articles/PMC3445712/" ref="ordinalpos=4647&amp;ncbi_uid=5305636&amp;link_uid=PMC3445712">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4648.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3149098/figure/F6/" ref="ordinalpos=4648&amp;ncbi_uid=3427567&amp;link_uid=PMC3149098" href="/pmc/articles/PMC3149098/"><img src="/pmc/articles/instance/3149098/bin/zjw0071139090006.gif" src-large="/pmc/articles/instance/3149098/bin/zjw0071139090006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3149098/" ref="ordinalpos=4648&amp;ncbi_uid=3427567&amp;link_uid=PMC3149098" image-link="/pmc/articles/PMC3149098/figure/F6/" class="imagepopup">Fig. 6. From: GProX, a User-Friendly Platform for Bioinformatics Analysis and Visualization of Quantitative Proteomics Data. </a></p><div class="supp"><p class="details"><div><b>GProX <span class="highlight" style="background-color:">Pathway</span> Analysis of the Cellular Response to EGF Stimulation.</b> The regulation after 30 min stimulation with EGF was mapped to the ErbB KEGG <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Nodes corresponding to proteins identified in the example data are color-coded according to regulation so up-regulated proteins are colored in shades of red, down-regulated proteins are colored in shades of blue and not regulated proteins are gray. This analysis illustrate that after 30 min of EGF stimulation the proteins upstream in the <span class="highlight" style="background-color:">pathway</span> have returned to the control state or are down-regulated while downstream effector proteins remain up-regulated.</div></p></div><div class="aux"><div class="resc">Kristoffer T. G. Rigbolt, et al. Mol Cell Proteomics. 2011 August;10(8):O110.007450.</div><p class="links"><a class="dblinks" href="/pubmed/21602510" ref="ordinalpos=4648&amp;ncbi_uid=3427567&amp;link_uid=21602510">Citation</a><a class="dblinks" href="/pmc/articles/PMC3149098/" ref="ordinalpos=4648&amp;ncbi_uid=3427567&amp;link_uid=PMC3149098">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4649.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3038716/figure/fig02/" ref="ordinalpos=4649&amp;ncbi_uid=3146910&amp;link_uid=PMC3038716" href="/pmc/articles/PMC3038716/"><img src="/pmc/articles/instance/3038716/bin/pnas.1010643108fig02.gif" src-large="/pmc/articles/instance/3038716/bin/pnas.1010643108fig02.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3038716/" ref="ordinalpos=4649&amp;ncbi_uid=3146910&amp;link_uid=PMC3038716" image-link="/pmc/articles/PMC3038716/figure/fig02/" class="imagepopup">Fig. 2. From: Experimental selection of hypoxia-tolerant Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div>Enrichment of fixed SNPs and Indels in an extended Notch <span class="highlight" style="background-color:">pathway</span>. The Notch <span class="highlight" style="background-color:">pathway</span> was adapted from KEGG (42) by adding Notch interactors from the literature to create an expanded Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Genes differentially expressed in larva (expression levels from Zhou et al.) (1) are cyan (up-regulated) or yellow (down-regulated), genes showing no change in expression are gray, and untested genes are white. Genes for which one more SNP and/or indels became fixed are indicated with stars: red for genes located within a hypoxia-selected region and blue for all others.</div></p></div><div class="aux"><div class="resc">Dan Zhou, et al. Proc Natl Acad Sci U S A. 2011 February 8;108(6):2349-2354.</div><p class="links"><a class="dblinks" href="/pubmed/21262834" ref="ordinalpos=4649&amp;ncbi_uid=3146910&amp;link_uid=21262834">Citation</a><a class="dblinks" href="/pmc/articles/PMC3038716/" ref="ordinalpos=4649&amp;ncbi_uid=3146910&amp;link_uid=PMC3038716">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4650.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3816224/figure/fig8/" ref="ordinalpos=4650&amp;ncbi_uid=5454897&amp;link_uid=PMC3816224" href="/pmc/articles/PMC3816224/"><img src="/pmc/articles/instance/3816224/bin/oncsis201334f8.gif" src-large="/pmc/articles/instance/3816224/bin/oncsis201334f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3816224/" ref="ordinalpos=4650&amp;ncbi_uid=5454897&amp;link_uid=PMC3816224" image-link="/pmc/articles/PMC3816224/figure/fig8/" class="imagepopup">Figure 8.  From: Centrosomal kinase Nek2 cooperates with oncogenic pathways to promote metastasis. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) Analysis of Oncomine data showed that median <i>hNek2</i> mRNA levels were consistently upregulated in patient tumors in the following cancersbreast,<sup>41</sup> pancreas,<sup>42</sup> colorectal,<sup>43</sup> lymphoma,<sup>44</sup> esophagus,<sup>45</sup> liver,<sup>46</sup> bladder,<sup>47</sup> lung,<sup>48</sup> and cervical.<sup>49</sup> (<b>b</b>) The chemical structure of Nek2 pharmacophore identified for possible SAR studies. (<b>c</b>) Model showing how Nek2 <span class="highlight" style="background-color:">signaling</span> can promote metastasis by cooperating with mitogen-activated protein kinase (MAPK) and RTK <span class="highlight" style="background-color:">signaling</span>; chromosomal instability (CIN). The PI3K-Akt <span class="highlight" style="background-color:">pathway</span> is an axis through which Nek2 cooperates with other pathways, indicated by heavy black arrow. Akt <span class="highlight" style="background-color:">pathway</span> upregulation would inhibit GSK3 leading to stabilization of -catenin and increase in Wnt <span class="highlight" style="background-color:">signaling</span>. Wingless/Wnt itself is also upregulated by Nek2 expression, creating the possibility of a complex feedback loop through interaction with other <span class="highlight" style="background-color:">signaling</span> pathways such as the Hippo cascade.<sup>50</sup> Red inhibitory signs indicate points along the <span class="highlight" style="background-color:">signaling</span> cascade where we can pharmacologically inhibit Nek2-driven tumorigenesis more effectively.</div></p></div><div class="aux"><div class="resc">T K Das, et al. Oncogenesis. 2013 September;2(9):e69.</div><p class="links"><a class="dblinks" href="/pubmed/24018644" ref="ordinalpos=4650&amp;ncbi_uid=5454897&amp;link_uid=24018644">Citation</a><a class="dblinks" href="/pmc/articles/PMC3816224/" ref="ordinalpos=4650&amp;ncbi_uid=5454897&amp;link_uid=PMC3816224">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4651.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4385402/figure/F1/" ref="ordinalpos=4651&amp;ncbi_uid=7384765&amp;link_uid=PMC4385402" href="/pmc/articles/PMC4385402/"><img src="/pmc/articles/instance/4385402/bin/nihms667486f1.gif" src-large="/pmc/articles/instance/4385402/bin/nihms667486f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4385402/" ref="ordinalpos=4651&amp;ncbi_uid=7384765&amp;link_uid=PMC4385402" image-link="/pmc/articles/PMC4385402/figure/F1/" class="imagepopup">Figure 1. Schematic of PGE2 synthesis, transport, <span class="highlight" style="background-color:">signaling</span>, and metabolism.  From: Multiple Drug Resistance-Associated Protein 4 (MRP4), Prostaglandin Transporter (PGT), and 15-Hydroxyprostaglandin Dehydrogenase (15-PGDH) as Determinants of PGE2 Levels in Cancer. </a></p><div class="supp"><p class="details"><div>The cyclooxygenase-2 (COX-2) enzyme is the rate-limiting step in the production of prostaglandin E<sub>2</sub> (PGE<sub>2</sub>) from arachidonic acid. PGE<sub>2</sub> is exported from the cell via multiple drug resistance-associated protein 4 (MRP4). In an autocrine or paracrine manner, extracellular PGE<sub>2</sub> binds any of its four cognate EP receptors, EP1-4, and initiates diverse <span class="highlight" style="background-color:">signaling</span> pathways via activation of G-proteins. Alternatively, PGE<sub>2</sub> is imported via the prostaglandin transporter (PGT) and inactivated by 15-hydroxyprostaglandin dehydrogenase (15-PGDH). The bold arrows indicate members of this <span class="highlight" style="background-color:">pathway</span> up-regulated in cancer while the dotted arrows indicate members of this <span class="highlight" style="background-color:">pathway</span> down-regulated in cancer.</div></p></div><div class="aux"><div class="resc">Tyler J. Kochel, et al. Prostaglandins Other Lipid Mediat. ;0:99-103.</div><p class="links"><a class="dblinks" href="/pubmed/25433169" ref="ordinalpos=4651&amp;ncbi_uid=7384765&amp;link_uid=25433169">Citation</a><a class="dblinks" href="/pmc/articles/PMC4385402/" ref="ordinalpos=4651&amp;ncbi_uid=7384765&amp;link_uid=PMC4385402">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4652.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4294653/figure/f9-dddt-9-349/" ref="ordinalpos=4652&amp;ncbi_uid=7087775&amp;link_uid=PMC4294653" href="/pmc/articles/PMC4294653/"><img src="/pmc/articles/instance/4294653/bin/dddt-9-349Fig9.gif" src-large="/pmc/articles/instance/4294653/bin/dddt-9-349Fig9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4294653/" ref="ordinalpos=4652&amp;ncbi_uid=7087775&amp;link_uid=PMC4294653" image-link="/pmc/articles/PMC4294653/figure/f9-dddt-9-349/" class="imagepopup">Figure 9.  From: Plumbagin elicits differential proteomic responses mainly involving cell cycle, apoptosis, autophagy, and epithelial-to-mesenchymal transition pathways in human prostate cancer PC-3 and DU145 cells. </a></p><div class="supp"><p class="details"><div>Proteomic analysis revealed networks of <span class="highlight" style="background-color:">signaling</span> pathways regulated by PLB in DU145 cells.<br /><b>Notes:</b> Networks of <span class="highlight" style="background-color:">signaling</span> pathways were analyzed by IPA according to 267 molecules and 107 related pathways which were regulated by PLB in DU145 cells.<br /><b>Abbreviations:</b> cAMP, cyclic adenosine monophosphate; IPA, Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis; PLB, plumbagin.</div></p></div><div class="aux"><div class="resc">Jia-Xuan Qiu, et al. Drug Des Devel Ther. 2015;9:349-417.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4652&amp;ncbi_uid=7087775&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4294653/" ref="ordinalpos=4652&amp;ncbi_uid=7087775&amp;link_uid=PMC4294653">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4653.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2899674/figure/F1/" ref="ordinalpos=4653&amp;ncbi_uid=2322268&amp;link_uid=PMC2899674" href="/pmc/articles/PMC2899674/"><img src="/pmc/articles/instance/2899674/bin/nihms-209954-f0001.gif" src-large="/pmc/articles/instance/2899674/bin/nihms-209954-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2899674/" ref="ordinalpos=4653&amp;ncbi_uid=2322268&amp;link_uid=PMC2899674" image-link="/pmc/articles/PMC2899674/figure/F1/" class="imagepopup">Fig. 1.  From: Specification of arterial, venous, and lymphatic endothelial cells during embryonic development. </a></p><div class="supp"><p class="details"><div>A model for molecular <span class="highlight" style="background-color:">pathway</span> for arterial, venous and lymphatic cell fate determination in vertebrate embryo. The VEGF-VEGFR2/Nrp1 <span class="highlight" style="background-color:">pathway</span> activates Notch <span class="highlight" style="background-color:">signaling</span> in angioblasts, leading to the specification of arterial cell fate, while crlr controls VEGF expression. Dep1 acts upstream of the VEGF-activated PI3K <span class="highlight" style="background-color:">pathway</span>. Foxc1 and Foxc2 directly regulate Dll4 and Hey2 expression, presumably downstream of VEGF <span class="highlight" style="background-color:">signaling</span>. Sox7/18 and snrk-1 may act upstream of grl (the Hey2 homolog in zebrafish). EphrinB2 is a downstream target of Notch. In venous endothelial progenitors, COUP-TFII suppresses Nrp1 and Notch activation and promotes venous cell fate. A subpopulation of venous endothelial cells subsequently expresses Sox18 and Prox1, thereby inducing lymphatic cell fate. COUP-TFII also contributes to lymphatic gene expression by interacting with Prox1.</div></p></div><div class="aux"><div class="resc">Tsutomu Kume. Histol Histopathol. ;25(5):637-646.</div><p class="links"><a class="dblinks" href="/pubmed/20238301" ref="ordinalpos=4653&amp;ncbi_uid=2322268&amp;link_uid=20238301">Citation</a><a class="dblinks" href="/pmc/articles/PMC2899674/" ref="ordinalpos=4653&amp;ncbi_uid=2322268&amp;link_uid=PMC2899674">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4654.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3785513/figure/pone-0075551-g002/" ref="ordinalpos=4654&amp;ncbi_uid=5359740&amp;link_uid=PMC3785513" href="/pmc/articles/PMC3785513/"><img src="/pmc/articles/instance/3785513/bin/pone.0075551.g002.gif" src-large="/pmc/articles/instance/3785513/bin/pone.0075551.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3785513/" ref="ordinalpos=4654&amp;ncbi_uid=5359740&amp;link_uid=PMC3785513" image-link="/pmc/articles/PMC3785513/figure/pone-0075551-g002/" class="imagepopup">Figure 2. The IRS-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in leptin-mediated increase of IL-6 expression..  From: Leptin Induces IL-6 Expression through OBRl Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Human Synovial Fibroblasts. </a></p><div class="supp"><p class="details"><div>(A and B) OASFs were transfected with IRS-1 siRNA for 24 h. Media and total RNA were collected, and the expression of IL-6 was examined by qPCR and ELISA assay (n=4). (C) OASFs were incubated with leptin for the indicated time intervals. Total protein was collected and the phosphorylation of IRS-1 was examined by western blotting assay (n=4). (D) OASFs were transfected with OBRl AS-ODN and MM-ODN for 24 h and then stimulated with leptin (3 M) for 30 min. Total protein was collected and the phosphorylation of IRS-1 was examined by western blotting (n=4). Results are expressed as mean  S.E.M. of four independent experiments. *: <i>p</i> &lt; 0.05 as compared with basal level. #: <i>p</i> &lt; 0.05 as compared with the leptin-treated group.</div></p></div><div class="aux"><div class="resc">Wei-Hung Yang, et al. PLoS One. 2013;8(9):e75551.</div><p class="links"><a class="dblinks" href="/pubmed/24086566" ref="ordinalpos=4654&amp;ncbi_uid=5359740&amp;link_uid=24086566">Citation</a><a class="dblinks" href="/pmc/articles/PMC3785513/" ref="ordinalpos=4654&amp;ncbi_uid=5359740&amp;link_uid=PMC3785513">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4655.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC60086/figure/F5/" ref="ordinalpos=4655&amp;ncbi_uid=134299&amp;link_uid=PMC60086" href="/pmc/articles/PMC60086/"><img src="/pmc/articles/instance/60086/bin/pq2211086005.gif" src-large="/pmc/articles/instance/60086/bin/pq2211086005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC60086/" ref="ordinalpos=4655&amp;ncbi_uid=134299&amp;link_uid=PMC60086" image-link="/pmc/articles/PMC60086/figure/F5/" class="imagepopup">Figure 5.  From: Sonic hedgehog restricts adhesion and migration of neural crest cells independently of the Patched- Smoothened-Gli <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The effect of Shh on NCC adhesion and migration is not mediated by the Smoothened-Gli <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) Effects of actinomycin-D at 1 g/ml, cycloheximide at 1 g/ml, forskolin at 25 M, and cyclopamine at 240 nM added to the culture medium on NCC spreading on FN in the presence of Shh at 20 g/ml. Spreading was measured by using NCCs that have been collected from explants previously cultured on FN in the absence of Shh. (<i>B</i><i>E</i>) Effects of forskolin at 25 M (<i>B</i> and <i>C</i>) and cyclopamine at 240 nM (<i>D</i> and <i>E</i>) on NCC migration from neural tube explants cultured on FN without (<i>B</i> and <i>D</i>) or with (<i>C</i> and <i>E</i>) Shh at 20 g/ml. No reversion of the inhibition of NCC adhesion and migration can be observed when transcription or protein synthesis are blocked or when Smoothened and Gli activities are inhibited. (Bar = 50 m.)</div></p></div><div class="aux"><div class="resc">Sandrine Testaz, et al. Proc Natl Acad Sci U S A. 2001 October 23;98(22):12521-12526.</div><p class="links"><a class="dblinks" href="/pubmed/11592978" ref="ordinalpos=4655&amp;ncbi_uid=134299&amp;link_uid=11592978">Citation</a><a class="dblinks" href="/pmc/articles/PMC60086/" ref="ordinalpos=4655&amp;ncbi_uid=134299&amp;link_uid=PMC60086">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4656.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4275034/figure/F1/" ref="ordinalpos=4656&amp;ncbi_uid=7042201&amp;link_uid=PMC4275034" href="/pmc/articles/PMC4275034/"><img src="/pmc/articles/instance/4275034/bin/fendo-05-00224-g001.gif" src-large="/pmc/articles/instance/4275034/bin/fendo-05-00224-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4275034/" ref="ordinalpos=4656&amp;ncbi_uid=7042201&amp;link_uid=PMC4275034" image-link="/pmc/articles/PMC4275034/figure/F1/" class="imagepopup">Figure 1.  From: IGF-IR Targeted Therapy: Past, Present and Future. </a></p><div class="supp"><p class="details"><div><b>When considering all possible combinations of homodimer and hybrid receptors of the insulin/IGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, there are at least six potentially tyrosine kinase receptors involved in signal transduction</b>.</div></p></div><div class="aux"><div class="resc">Joseph A. M. J. L. Janssen, et al. Front Endocrinol (Lausanne). 2014;5:224.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4656&amp;ncbi_uid=7042201&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4275034/" ref="ordinalpos=4656&amp;ncbi_uid=7042201&amp;link_uid=PMC4275034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4657.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3909570/figure/Fig4/" ref="ordinalpos=4657&amp;ncbi_uid=5792740&amp;link_uid=PMC3909570" href="/pmc/articles/PMC3909570/"><img src="/pmc/articles/instance/3909570/bin/13205_2013_121_Fig4_HTML.gif" src-large="/pmc/articles/instance/3909570/bin/13205_2013_121_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3909570/" ref="ordinalpos=4657&amp;ncbi_uid=5792740&amp;link_uid=PMC3909570" image-link="/pmc/articles/PMC3909570/figure/Fig4/" class="imagepopup">Fig. 4.  From: Pharmacological and therapeutic potential of Cordyceps with special reference to Cordycepin. </a></p><div class="supp"><p class="details"><div>Cordycepin presumably activates the AMPK by some unknown mechanism which further negatively regulates the translation of mTOR <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> by the formation of a translational repressor, 4-E-binding protein-1 (4EBP1)</div></p></div><div class="aux"><div class="resc">Hardeep S. Tuli, et al. 3 Biotech. 2014 February;4(1):1-12.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4657&amp;ncbi_uid=5792740&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC3909570/" ref="ordinalpos=4657&amp;ncbi_uid=5792740&amp;link_uid=PMC3909570">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4658.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3598860/figure/pone-0058911-g008/" ref="ordinalpos=4658&amp;ncbi_uid=4769375&amp;link_uid=PMC3598860" href="/pmc/articles/PMC3598860/"><img src="/pmc/articles/instance/3598860/bin/pone.0058911.g008.gif" src-large="/pmc/articles/instance/3598860/bin/pone.0058911.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3598860/" ref="ordinalpos=4658&amp;ncbi_uid=4769375&amp;link_uid=PMC3598860" image-link="/pmc/articles/PMC3598860/figure/pone-0058911-g008/" class="imagepopup">Figure 8. Model of Brap2 function..  From: Brap2 Regulates Temporal Control of NF-?B Localization Mediated by Inflammatory Response. </a></p><div class="supp"><p class="details"><div>(A) Model of Brap2 interaction with Cul1 and its effect on NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<b>B</b>) Time course of NF-B translocation and target gene expression by each Brap2 constructs.</div></p></div><div class="aux"><div class="resc">Osamu Takashima, et al. PLoS One. 2013;8(3):e58911.</div><p class="links"><a class="dblinks" href="/pubmed/23554956" ref="ordinalpos=4658&amp;ncbi_uid=4769375&amp;link_uid=23554956">Citation</a><a class="dblinks" href="/pmc/articles/PMC3598860/" ref="ordinalpos=4658&amp;ncbi_uid=4769375&amp;link_uid=PMC3598860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4659.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3281420/figure/f1-cancers-03-01639/" ref="ordinalpos=4659&amp;ncbi_uid=5273359&amp;link_uid=PMC3281420" href="/pmc/articles/PMC3281420/"><img src="/pmc/articles/instance/3757383/bin/cancers-03-01639f1.gif" src-large="/pmc/articles/instance/3757383/bin/cancers-03-01639f1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3281420/" ref="ordinalpos=4659&amp;ncbi_uid=5273359&amp;link_uid=PMC3281420" image-link="/pmc/articles/PMC3281420/figure/f1-cancers-03-01639/" class="imagepopup">Figure 1. From: Targeting the Anti-Apoptotic Protein c-FLIP for Cancer Therapy. </a></p><div class="supp"><p class="details"><div>Schematic overview of the multifunctional roles of c-FLIP in the TRAIL-triggered apoptosis <span class="highlight" style="background-color:">pathway</span> as well as activating various anti-apoptotic and cell survival <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Ahmad R. Safa, et al. Cancers (Basel). 2011 June;3(2):1639-1671.</div><p class="links"><a class="dblinks" href="/pubmed/22348197" ref="ordinalpos=4659&amp;ncbi_uid=5273359&amp;link_uid=22348197">Citation</a><a class="dblinks" href="/pmc/articles/PMC3281420/" ref="ordinalpos=4659&amp;ncbi_uid=5273359&amp;link_uid=PMC3281420">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4660.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1702561/figure/pbio-0050010-g001/" ref="ordinalpos=4660&amp;ncbi_uid=820657&amp;link_uid=PMC1702561" href="/pmc/articles/PMC1702561/"><img src="/pmc/articles/instance/1769435/bin/pbio.0050010.g001.gif" src-large="/pmc/articles/instance/1769435/bin/pbio.0050010.g001.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1702561/" ref="ordinalpos=4660&amp;ncbi_uid=820657&amp;link_uid=PMC1702561" image-link="/pmc/articles/PMC1702561/figure/pbio-0050010-g001/" class="imagepopup">Figure. From: Manipulating Cellular pH Suggests Novel Anticancer Therapy. </a></p><div class="supp"><p class="details"><div>A novel <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by DNA damage leads to the upregulation of the NHE-1 antiport, increased intracellular pH, Bcl-x<sub>L</sub> deamidation, and finally apoptosis.</div></p></div><div class="aux"><div class="resc">Liza Gross. PLoS Biol. 2007 January;5(1):e10.</div><p class="links"><a class="dblinks" href="/pubmed/20076643" ref="ordinalpos=4660&amp;ncbi_uid=820657&amp;link_uid=20076643">Citation</a><a class="dblinks" href="/pmc/articles/PMC1702561/" ref="ordinalpos=4660&amp;ncbi_uid=820657&amp;link_uid=PMC1702561">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4661.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1143751/figure/f6/" ref="ordinalpos=4661&amp;ncbi_uid=498106&amp;link_uid=PMC1143751" href="/pmc/articles/PMC1143751/"><img src="/pmc/articles/instance/1143751/bin/zjv0130564460006.gif" src-large="/pmc/articles/instance/1143751/bin/zjv0130564460006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1143751/" ref="ordinalpos=4661&amp;ncbi_uid=498106&amp;link_uid=PMC1143751" image-link="/pmc/articles/PMC1143751/figure/f6/" class="imagepopup">FIG. 6. From: Hepatitis C Virus E2-CD81 Interaction Induces Hypermutation of the Immunoglobulin Gene in B Cells . </a></p><div class="supp"><p class="details"><div>A postulated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for induction of hypermutation of Ig gene in B cells or hepatocytes by E2 binding. BCR, B-cell receptor complex.</div></p></div><div class="aux"><div class="resc">Keigo Machida, et al. J Virol. 2005 July;79(13):8079-8089.</div><p class="links"><a class="dblinks" href="/pubmed/15956553" ref="ordinalpos=4661&amp;ncbi_uid=498106&amp;link_uid=15956553">Citation</a><a class="dblinks" href="/pmc/articles/PMC1143751/" ref="ordinalpos=4661&amp;ncbi_uid=498106&amp;link_uid=PMC1143751">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4662.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3903585/figure/F4/" ref="ordinalpos=4662&amp;ncbi_uid=5742954&amp;link_uid=PMC3903585" href="/pmc/articles/PMC3903585/"><img src="/pmc/articles/instance/3903585/bin/nihms-546630-f0004.gif" src-large="/pmc/articles/instance/3903585/bin/nihms-546630-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3903585/" ref="ordinalpos=4662&amp;ncbi_uid=5742954&amp;link_uid=PMC3903585" image-link="/pmc/articles/PMC3903585/figure/F4/" class="imagepopup">Figure 4. High-throughput screen for Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> modulators.  From: A High-Throughput Screen for Wnt/?-catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Modulators in Human iPSC-derived Neural Progenitors. </a></p><div class="supp"><p class="details"><div>The screen was conducted in three conditions: media, EC<sub>25</sub> Wnt3a-CM (shown here), or 10 mM lithium.</div></p></div><div class="aux"><div class="resc">Wen-Ning Zhao, et al. J Biomol Screen. ;17(9):1252-1263.</div><p class="links"><a class="dblinks" href="/pubmed/22923789" ref="ordinalpos=4662&amp;ncbi_uid=5742954&amp;link_uid=22923789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3903585/" ref="ordinalpos=4662&amp;ncbi_uid=5742954&amp;link_uid=PMC3903585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4663.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3454471/figure/F8/" ref="ordinalpos=4663&amp;ncbi_uid=4847743&amp;link_uid=PMC3454471" href="/pmc/articles/PMC3454471/"><img src="/pmc/articles/instance/3454471/bin/nihms351530f8.gif" src-large="/pmc/articles/instance/3454471/bin/nihms351530f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3454471/" ref="ordinalpos=4663&amp;ncbi_uid=4847743&amp;link_uid=PMC3454471" image-link="/pmc/articles/PMC3454471/figure/F8/" class="imagepopup">Fig. 8.  From: Reactive oxygen species (ROS) homeostasis and redox regulation in cellular <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic of ATM <span class="highlight" style="background-color:">signaling</span> upon oxidative stress and double-strand DNA breaks. Double-strand DNA breaks mediate the phospho-ATMChk2 <span class="highlight" style="background-color:">pathway</span>; however, oxidative stress elicits both the phospho-ATMLKB1 and the ATMhomodimer pathways (ataxiatelangiectasia mutated (ATM); liver kinase B1 (LKB1); AMP activated protein kinase (AMPK); tuberous sclerosis complex 2 (TSC2); mammalian target of rapamycin complex1 (mTORC1)).</div></p></div><div class="aux"><div class="resc">Paul D. Ray, et al. Cell Signal. 2012 May;24(5):981-990.</div><p class="links"><a class="dblinks" href="/pubmed/22286106" ref="ordinalpos=4663&amp;ncbi_uid=4847743&amp;link_uid=22286106">Citation</a><a class="dblinks" href="/pmc/articles/PMC3454471/" ref="ordinalpos=4663&amp;ncbi_uid=4847743&amp;link_uid=PMC3454471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4664.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3018149/figure/F5/" ref="ordinalpos=4664&amp;ncbi_uid=3403250&amp;link_uid=PMC3018149" href="/pmc/articles/PMC3018149/"><img src="/pmc/articles/instance/3018149/bin/nihms-241911-f0005.gif" src-large="/pmc/articles/instance/3018149/bin/nihms-241911-f0005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3018149/" ref="ordinalpos=4664&amp;ncbi_uid=3403250&amp;link_uid=PMC3018149" image-link="/pmc/articles/PMC3018149/figure/F5/" class="imagepopup">Figure 5.  From: Up Regulation of DLX5 Promotes Ovarian Cancer Cell Proliferation by Enhancing IRS-2-AKT <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A wobbled <i>DLX5</i> cDNA prevents <i>DLX5</i> knockdown-induced cell cycle arrest. (A) WST-1 assays were performed to determine cell viability/proliferation after transduction of <i>DLX5</i> shRNA2 and a Flag-<i>DLX5</i> cDNA containing three wobbled codons at the shRNA2 targeted sites. (B) The wobbled <i>DLX5</i> cDNA restores cyclin levels as well as AKT <span class="highlight" style="background-color:">signaling</span>. Immunoblotting was performed to analyze expression levels of DLX5, cyclins A, B1 and D2, various AKT <span class="highlight" style="background-color:">pathway</span> members, as well as IRS-2 and MET.</div></p></div><div class="aux"><div class="resc">Yinfei Tan, et al. Cancer Res. ;70(22):9197-9206.</div><p class="links"><a class="dblinks" href="/pubmed/21045156" ref="ordinalpos=4664&amp;ncbi_uid=3403250&amp;link_uid=21045156">Citation</a><a class="dblinks" href="/pmc/articles/PMC3018149/" ref="ordinalpos=4664&amp;ncbi_uid=3403250&amp;link_uid=PMC3018149">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4665.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2818861/figure/F1/" ref="ordinalpos=4665&amp;ncbi_uid=2601881&amp;link_uid=PMC2818861" href="/pmc/articles/PMC2818861/"><img src="/pmc/articles/instance/2818861/bin/nihms-167495-f0001.gif" src-large="/pmc/articles/instance/2818861/bin/nihms-167495-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2818861/" ref="ordinalpos=4665&amp;ncbi_uid=2601881&amp;link_uid=PMC2818861" image-link="/pmc/articles/PMC2818861/figure/F1/" class="imagepopup">Figure 1.  From: Estrogen Receptor <span class="highlight" style="background-color:">Signaling</span> in Lung Cancer. </a></p><div class="supp"><p class="details"><div>Available strategies to target the estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for lung cancer include 1) inhibition of the final step in the synthesis of estrogen, by inhibiting the enzyme aromatase with the inhibitor anastrozole; 2) down-regulation of classical ERs using the pure antiestrogen fulvestrant; 3) inhibition of GPR30-dependent <span class="highlight" style="background-color:">signaling</span> through the newly identified GPR30 antagonist, G15; 4) targeting the growth factor pathways that are activated by estrogens, mainly the EGF and VEGF pathways using the drugs gefitinib, erlotinib or vandetanib. These strategies can be used alone or in combination.</div></p></div><div class="aux"><div class="resc">Jill M. Siegfried, et al. Semin Oncol. ;36(6):524.</div><p class="links"><a class="dblinks" href="/pubmed/19995644" ref="ordinalpos=4665&amp;ncbi_uid=2601881&amp;link_uid=19995644">Citation</a><a class="dblinks" href="/pmc/articles/PMC2818861/" ref="ordinalpos=4665&amp;ncbi_uid=2601881&amp;link_uid=PMC2818861">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4666.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3071501/figure/F10/" ref="ordinalpos=4666&amp;ncbi_uid=2896414&amp;link_uid=PMC3071501" href="/pmc/articles/PMC3071501/"><img src="/pmc/articles/instance/3071501/bin/fnins-05-00161-g010.gif" src-large="/pmc/articles/instance/3071501/bin/fnins-05-00161-g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3071501/" ref="ordinalpos=4666&amp;ncbi_uid=2896414&amp;link_uid=PMC3071501" image-link="/pmc/articles/PMC3071501/figure/F10/" class="imagepopup">Figure 10.  From: Transcriptomic Analysis in a Drosophila Model Identifies Previously Implicated and Novel Pathways in the Therapeutic Mechanism in Neuropsychiatric Disorders. </a></p><div class="supp"><p class="details"><div><b>Genes mapping to KEGG <span class="highlight" style="background-color:">pathway</span> for hedgehog <span class="highlight" style="background-color:">signaling</span></b>. Pink boxes represent upregulated genes in AED untreated flies. These genes are <i>Rab23</i> (CG2108 gene product from transcript CG2108-RA), <i>dpp</i> (decapentaplegic), and <i>fu</i> (fused). Green boxes represent other members in the <span class="highlight" style="background-color:">pathway</span> database for <i>Drosophila melanogaster</i>.</div></p></div><div class="aux"><div class="resc">Priyanka Singh, et al. Front Neurosci. 2011;5:161.</div><p class="links"><a class="dblinks" href="/pubmed/21503142" ref="ordinalpos=4666&amp;ncbi_uid=2896414&amp;link_uid=21503142">Citation</a><a class="dblinks" href="/pmc/articles/PMC3071501/" ref="ordinalpos=4666&amp;ncbi_uid=2896414&amp;link_uid=PMC3071501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4667.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4202139/figure/F5/" ref="ordinalpos=4667&amp;ncbi_uid=6741321&amp;link_uid=PMC4202139" href="/pmc/articles/PMC4202139/"><img src="/pmc/articles/instance/4202139/bin/oncotarget-05-7498-g005.gif" src-large="/pmc/articles/instance/4202139/bin/oncotarget-05-7498-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4202139/" ref="ordinalpos=4667&amp;ncbi_uid=6741321&amp;link_uid=PMC4202139" image-link="/pmc/articles/PMC4202139/figure/F5/" class="imagepopup">Figure 5. Aur-A activated mTOR <span class="highlight" style="background-color:">signaling</span> by inhibiting GSK3 under metabolic stress.  From: Aurora kinase A suppresses metabolic stress-induced autophagic cell death by activating mTOR <span class="highlight" style="background-color:">signaling</span> in breast cancer cells. </a></p><div class="supp"><p class="details"><div>A-B, mTOR activity (indicated by p70S6K phospho-Thr389 level) and GSK3 activity (indicated by phospho-Ser9 level) were assessed by Western blot after HBSS starvation in control and Aur-A overexpressed SKBR-3 cells. C, GSK3 activity was reconstituated by transfection of V5-GSK3-CA plasmid in control and Aur-A overexpressed SKBR-3 cells. After HBSS starvation, mTOR activity (indicated by p70S6K phospho-Thr389 level) and LC3 conversion (LC3 II/total) were assessed by Western blot. D, Cells were pretreated with rapamycin 12 hours before subjected to HBSS treatment. After starvation, cell viabilities were measured by MTT assay. Data were mean  S.D. of three independent experiments done in parallel (<sup>**</sup><i>p</i>&lt;0.01). E, A working model to elucidate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that mediated action of Aur-A under metabolic stress.</div></p></div><div class="aux"><div class="resc">Ling-Zhi Xu, et al. Oncotarget. 2014 September;5(17):7498-7511.</div><p class="links"><a class="dblinks" href="/pubmed/25115395" ref="ordinalpos=4667&amp;ncbi_uid=6741321&amp;link_uid=25115395">Citation</a><a class="dblinks" href="/pmc/articles/PMC4202139/" ref="ordinalpos=4667&amp;ncbi_uid=6741321&amp;link_uid=PMC4202139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4668.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3292340/figure/F1/" ref="ordinalpos=4668&amp;ncbi_uid=3981917&amp;link_uid=PMC3292340" href="/pmc/articles/PMC3292340/"><img src="/pmc/articles/instance/3292340/bin/1632fig1.gif" src-large="/pmc/articles/instance/3292340/bin/1632fig1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3292340/" ref="ordinalpos=4668&amp;ncbi_uid=3981917&amp;link_uid=PMC3292340" image-link="/pmc/articles/PMC3292340/figure/F1/" class="imagepopup">FIG. 1. From: Drosophila Insulin <span class="highlight" style="background-color:">Pathway</span> Mutants Affect Visual Physiology and Brain Function Besides Growth, Lipid, and Carbohydrate Metabolism. </a></p><div class="supp"><p class="details"><div>Mutant flies for insulin <span class="highlight" style="background-color:">signaling</span> show defective growth. <i>A</i>: Histogram shows the percentage of reduction in body weight of heteroallelic and homozygous mutants of the insulin <span class="highlight" style="background-color:">pathway</span> compared with their respective paired heterozygous sibling controls (<i>n</i> = 16). *<i>P</i> &lt; 0.01. The error bars represent the SEM. <i>B</i>: Heteroallelic <i>PKB</i><sup>1/3</sup> mutant flies are smaller than <i>PKB</i><sup>1/+</sup> or <i>PKB</i><sup>3/+</sup> heterozygous flies (controls). <i>C</i>: The wing area of insulin <span class="highlight" style="background-color:">pathway</span> mutant flies is reduced compared with paired sibling controls (<i>n</i> = 30). *<i>P</i> &lt; 0.01. The error bars represent SEM. <i>D</i>: Heteroallelic mutant <i>PKB</i><sup>1/3</sup> wings are smaller than heterozygous <i>PKB</i><sup>1/+</sup> or <i>PKB</i><sup>3/+</sup> wings (controls). (A high-quality color representation of this figure is available in the online issue.)</div></p></div><div class="aux"><div class="resc">Juan M. Murillo-Maldonado, et al. Diabetes. 2011 May;60(5):1632-1636.</div><p class="links"><a class="dblinks" href="/pubmed/21464442" ref="ordinalpos=4668&amp;ncbi_uid=3981917&amp;link_uid=21464442">Citation</a><a class="dblinks" href="/pmc/articles/PMC3292340/" ref="ordinalpos=4668&amp;ncbi_uid=3981917&amp;link_uid=PMC3292340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4669.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3808542/figure/fig2s1/" ref="ordinalpos=4669&amp;ncbi_uid=5422233&amp;link_uid=PMC3808542" href="/pmc/articles/PMC3808542/"><img src="/pmc/articles/instance/3808542/bin/elife01236fs004.gif" src-large="/pmc/articles/instance/3808542/bin/elife01236fs004.jpg" alt="Figure 2figure supplement 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3808542/" ref="ordinalpos=4669&amp;ncbi_uid=5422233&amp;link_uid=PMC3808542" image-link="/pmc/articles/PMC3808542/figure/fig2s1/" class="imagepopup">Figure 2figure supplement 1.Subgroup analysis from biological pathways showing changes in translation efficiency from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis..  From: Global cellular response to chemotherapy-induced apoptosis. </a></p><div class="supp"><p class="details"><div>Genes included in IPA as XBP1 downstream targets (left) and Genes included as part of the eIF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in IPA (right). Green boxed area indicates numerous ribosome structural proteins with decreased footprint reads relative to mRNA. <i>SEC61B</i> and <i>RPL7A</i> (shown in Figure 2) are highlighted with red arrows, as is PERK (<i>EIF2AK3</i>). All data are shown as Log<sub>2</sub> expression ratio vs 0 hr.<br /><b>DOI:</b> http://dx.doi.org/10.7554/eLife.01236.009</div></p></div><div class="aux"><div class="resc">Arun P Wiita, et al. eLife. 2013; 2013;2:e01236.</div><p class="links"><a class="dblinks" href="/pubmed/24171104" ref="ordinalpos=4669&amp;ncbi_uid=5422233&amp;link_uid=24171104">Citation</a><a class="dblinks" href="/pmc/articles/PMC3808542/" ref="ordinalpos=4669&amp;ncbi_uid=5422233&amp;link_uid=PMC3808542">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4670.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2262155/figure/pone-0001782-g003/" ref="ordinalpos=4670&amp;ncbi_uid=1258573&amp;link_uid=PMC2262155" href="/pmc/articles/PMC2262155/"><img src="/pmc/articles/instance/2262155/bin/pone.0001782.g003.gif" src-large="/pmc/articles/instance/2262155/bin/pone.0001782.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2262155/" ref="ordinalpos=4670&amp;ncbi_uid=1258573&amp;link_uid=PMC2262155" image-link="/pmc/articles/PMC2262155/figure/pone-0001782-g003/" class="imagepopup">Figure 3. The 29 <span class="highlight" style="background-color:">pathway</span> structures that possess positive feedback by ERK to Raf..  From: Topological Analysis of MAPK Cascade for Kinetic ErbB <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The 29 <span class="highlight" style="background-color:">pathway</span> structures that possess positive feedback by ERK to Raf isoforms can be classified into three types of regulation depending on which isoform is positively regulated: B-Raf (nos. 110), Raf-1 (nos. 1120), or both B-Raf and Raf-1 (nos. 2129). The symbol for MEK is omitted here for clarity.</div></p></div><div class="aux"><div class="resc">Takashi Nakakuki, et al. PLoS ONE. 2008;3(3):e1782.</div><p class="links"><a class="dblinks" href="/pubmed/18335053" ref="ordinalpos=4670&amp;ncbi_uid=1258573&amp;link_uid=18335053">Citation</a><a class="dblinks" href="/pmc/articles/PMC2262155/" ref="ordinalpos=4670&amp;ncbi_uid=1258573&amp;link_uid=PMC2262155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4671.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3540819/figure/fig1/" ref="ordinalpos=4671&amp;ncbi_uid=4531047&amp;link_uid=PMC3540819" href="/pmc/articles/PMC3540819/"><img src="/pmc/articles/instance/3540819/bin/JTR2012-818204.001.gif" src-large="/pmc/articles/instance/3540819/bin/JTR2012-818204.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3540819/" ref="ordinalpos=4671&amp;ncbi_uid=4531047&amp;link_uid=PMC3540819" image-link="/pmc/articles/PMC3540819/figure/fig1/" class="imagepopup">Figure 1.  From: New Molecular Targeted Therapy and Redifferentiation Therapy for Radioiodine-Refractory Advanced Papillary Thyroid Carcinoma: Literature Review. </a></p><div class="supp"><p class="details"><div>The main <span class="highlight" style="background-color:">signaling</span> pathways involved in thyroid carcinogenesis are the MAPK and PI3K-AKT <span class="highlight" style="background-color:">pathway</span> (reproduced with permission) [15].</div></p></div><div class="aux"><div class="resc">Kai-Pun Wong, et al. J Thyroid Res. 2012;2012:818204.</div><p class="links"><a class="dblinks" href="/pubmed/23320248" ref="ordinalpos=4671&amp;ncbi_uid=4531047&amp;link_uid=23320248">Citation</a><a class="dblinks" href="/pmc/articles/PMC3540819/" ref="ordinalpos=4671&amp;ncbi_uid=4531047&amp;link_uid=PMC3540819">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4672.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3379801/figure/NOS114F8/" ref="ordinalpos=4672&amp;ncbi_uid=5044020&amp;link_uid=PMC3379801" href="/pmc/articles/PMC3379801/"><img src="/pmc/articles/instance/3379801/bin/nos11408.gif" src-large="/pmc/articles/instance/3379801/bin/nos11408.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3379801/" ref="ordinalpos=4672&amp;ncbi_uid=5044020&amp;link_uid=PMC3379801" image-link="/pmc/articles/PMC3379801/figure/NOS114F8/" class="imagepopup">Fig. 8. From: p16-Cdk4-Rb axis controls sensitivity to a cyclin-dependent kinase inhibitor PD0332991 in glioblastoma xenograft cells. </a></p><div class="supp"><p class="details"><div>p16-Cdk4-Rb <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> diagram. The components that were evaluated in this study and associated with response to PD0332991 therapy are highlighted in grey color-filled boxes.</div></p></div><div class="aux"><div class="resc">Ling Cen, et al. Neuro Oncol. 2012 July;14(7):870-881.</div><p class="links"><a class="dblinks" href="/pubmed/22711607" ref="ordinalpos=4672&amp;ncbi_uid=5044020&amp;link_uid=22711607">Citation</a><a class="dblinks" href="/pmc/articles/PMC3379801/" ref="ordinalpos=4672&amp;ncbi_uid=5044020&amp;link_uid=PMC3379801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4673.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2063843/figure/fig8/" ref="ordinalpos=4673&amp;ncbi_uid=1184593&amp;link_uid=PMC2063843" href="/pmc/articles/PMC2063843/"><img src="/pmc/articles/instance/2063843/bin/jcb1730431f08.gif" src-large="/pmc/articles/instance/2063843/bin/jcb1730431f08.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2063843/" ref="ordinalpos=4673&amp;ncbi_uid=1184593&amp;link_uid=PMC2063843" image-link="/pmc/articles/PMC2063843/figure/fig8/" class="imagepopup">Figure 8. From: E-cadherin engagement stimulates proliferation via Rac1. </a></p><div class="supp"><p class="details"><div><b>Schematic of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading cellcell contact to inhibition or stimulation of proliferation.</b> Cellcell contact stimulates proliferation through activation of Rac1 that depends on E-cadherin engagement and p120. Inhibition of proliferation by cellcell contact results from a crowding-dependent, cadherin-independent decrease in cellECM spreading.</div></p></div><div class="aux"><div class="resc">Wendy F. Liu, et al. J Cell Biol. 2006 May 8;173(3):431-441.</div><p class="links"><a class="dblinks" href="/pubmed/16682529" ref="ordinalpos=4673&amp;ncbi_uid=1184593&amp;link_uid=16682529">Citation</a><a class="dblinks" href="/pmc/articles/PMC2063843/" ref="ordinalpos=4673&amp;ncbi_uid=1184593&amp;link_uid=PMC2063843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4674.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC25483/figure/F11/" ref="ordinalpos=4674&amp;ncbi_uid=184027&amp;link_uid=PMC25483" href="/pmc/articles/PMC25483/"><img src="/pmc/articles/instance/25483/bin/mk0880660011.gif" src-large="/pmc/articles/instance/25483/bin/mk0880660011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC25483/" ref="ordinalpos=4674&amp;ncbi_uid=184027&amp;link_uid=PMC25483" image-link="/pmc/articles/PMC25483/figure/F11/" class="imagepopup">Figure 11.  From: Na-H Exchange Acts Downstream of RhoA to Regulate Integrin-induced Cell Adhesion and Spreading. </a></p><div class="supp"><p class="details"><div>A proposed <span class="highlight" style="background-color:">pathway</span> of integrinRhoANHE1 <span class="highlight" style="background-color:">signaling</span> for induction of focal adhesions and stress fibers. MCL-P, phosphorylated myosin light chain.</div></p></div><div class="aux"><div class="resc">Tomoko Tominaga, et al. Mol Biol Cell. 1998 August;9(8):2287-2303.</div><p class="links"><a class="dblinks" href="/pubmed/9693382" ref="ordinalpos=4674&amp;ncbi_uid=184027&amp;link_uid=9693382">Citation</a><a class="dblinks" href="/pmc/articles/PMC25483/" ref="ordinalpos=4674&amp;ncbi_uid=184027&amp;link_uid=PMC25483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4675.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3006139/figure/pntd-0000924-g006/" ref="ordinalpos=4675&amp;ncbi_uid=2694518&amp;link_uid=PMC3006139" href="/pmc/articles/PMC3006139/"><img src="/pmc/articles/instance/3006139/bin/pntd.0000924.g006.gif" src-large="/pmc/articles/instance/3006139/bin/pntd.0000924.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3006139/" ref="ordinalpos=4675&amp;ncbi_uid=2694518&amp;link_uid=PMC3006139" image-link="/pmc/articles/PMC3006139/figure/pntd-0000924-g006/" class="imagepopup">Figure 6. Model illustrating TLR and TLR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> collapse during DENV-Ab complex infection..  From: Subversion of Innate Defenses by the Interplay between DENV and Pre-Existing Enhancing Antibodies: TLRs <span class="highlight" style="background-color:">Signaling</span> Collapse. </a></p><div class="supp"><p class="details"><div>DENV-ADE infection down-regulates TLR-4 and endosomal TLRs synthesis while strongly stimulates <i>SARM</i> and <i>TANK</i>, negative regulators of <i>TRIF</i> and <i>TRAF-6</i>, respectively. These events result in suppression of innate responses mediated through the TLR-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Naphak Modhiran, et al. PLoS Negl Trop Dis. 2010 December;4(12):e924.</div><p class="links"><a class="dblinks" href="/pubmed/21200427" ref="ordinalpos=4675&amp;ncbi_uid=2694518&amp;link_uid=21200427">Citation</a><a class="dblinks" href="/pmc/articles/PMC3006139/" ref="ordinalpos=4675&amp;ncbi_uid=2694518&amp;link_uid=PMC3006139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4676.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1087413/figure/f3/" ref="ordinalpos=4676&amp;ncbi_uid=410896&amp;link_uid=PMC1087413" href="/pmc/articles/PMC1087413/"><img src="/pmc/articles/instance/1087413/bin/zii0040547300003.gif" src-large="/pmc/articles/instance/1087413/bin/zii0040547300003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1087413/" ref="ordinalpos=4676&amp;ncbi_uid=410896&amp;link_uid=PMC1087413" image-link="/pmc/articles/PMC1087413/figure/f3/" class="imagepopup">FIG. 3. From: Apoptosis, Pyroptosis, and Necrosis: Mechanistic Description of Dead and Dying Eukaryotic Cells . </a></p><div class="supp"><p class="details"><div>Biological input differentially influences utilization of cell death pathways. Factors influencing cell death include the stimulus, cell type, and the surrounding milieu of the cell and, therefore, its physiological state during receipt of the stimulus, which together dictate the <span class="highlight" style="background-color:">pathway</span> of cell death. In column A1, a stimulus (1) delivered to a cell type of interest in its baseline physiological state () dictates cell death via the primary <span class="highlight" style="background-color:">pathway</span> (P). Altering the magnitude of the stimulus (1), e.g., multiplicity of infection, concentration, or concentration per unit time, can alter the path to cell death (A2, leading to <span class="highlight" style="background-color:">pathway</span> Q), as does the physiological state () of the cell during stimulation (A3, leading to <span class="highlight" style="background-color:">pathway</span> R) and the cell type being studied (A4, leading to <span class="highlight" style="background-color:">pathway</span> S). A single stimulus can simultaneously trigger multiple pathways (column B), and blockade of the primary <span class="highlight" style="background-color:">pathway</span> for a given cell type (column C versus column A1) can result in the use of alternate <span class="highlight" style="background-color:">pathway</span>(s) (column C). Finally, multiple stimuli (2,3, or 4), which may utilize a variety of upstream <span class="highlight" style="background-color:">signaling</span> cascades, may converge in the use of biologically conserved effector pathways and result in cell death (column D).</div></p></div><div class="aux"><div class="resc">Susan L. Fink, et al. Infect Immun. 2005 April;73(4):1907-1916.</div><p class="links"><a class="dblinks" href="/pubmed/15784530" ref="ordinalpos=4676&amp;ncbi_uid=410896&amp;link_uid=15784530">Citation</a><a class="dblinks" href="/pmc/articles/PMC1087413/" ref="ordinalpos=4676&amp;ncbi_uid=410896&amp;link_uid=PMC1087413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4677.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2808051/figure/F1/" ref="ordinalpos=4677&amp;ncbi_uid=2220662&amp;link_uid=PMC2808051" href="/pmc/articles/PMC2808051/"><img src="/pmc/articles/instance/2808051/bin/ijbsv06p0009g01.gif" src-large="/pmc/articles/instance/2808051/bin/ijbsv06p0009g01.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2808051/" ref="ordinalpos=4677&amp;ncbi_uid=2220662&amp;link_uid=PMC2808051" image-link="/pmc/articles/PMC2808051/figure/F1/" class="imagepopup">Figure 1.  From: An Overview of Stress Response and Hypometabolic Strategies in Caenorhabditis elegans: Conserved and Contrasting Signals with the Mammalian System. </a></p><div class="supp"><p class="details"><div>Summary of the insulin/IGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, using the terminology for mammalian systems. Binding of the ligand to the insulin like growth factor-I Receptor initiates the <span class="highlight" style="background-color:">signaling</span> cascade, which is propagated by PI3K. Akt acts as the <span class="highlight" style="background-color:">signaling</span> hub for the majority of the downstream effects such as cell cycle control, glycogen synthesis and apoptotic suppression. The phosphorylation state of Akt is critically regulated by protein phosphatases (PTEN/PP2) and protein kinases (PDK-1).</div></p></div><div class="aux"><div class="resc">Benjamin Lant, et al. Int J Biol Sci. 2010;6(1):9-50.</div><p class="links"><a class="dblinks" href="/pubmed/20087441" ref="ordinalpos=4677&amp;ncbi_uid=2220662&amp;link_uid=20087441">Citation</a><a class="dblinks" href="/pmc/articles/PMC2808051/" ref="ordinalpos=4677&amp;ncbi_uid=2220662&amp;link_uid=PMC2808051">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4678.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4283135/figure/F2/" ref="ordinalpos=4678&amp;ncbi_uid=7079106&amp;link_uid=PMC4283135" href="/pmc/articles/PMC4283135/"><img src="/pmc/articles/instance/4283135/bin/fonc-04-00341-g002.gif" src-large="/pmc/articles/instance/4283135/bin/fonc-04-00341-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4283135/" ref="ordinalpos=4678&amp;ncbi_uid=7079106&amp;link_uid=PMC4283135" image-link="/pmc/articles/PMC4283135/figure/F2/" class="imagepopup">Figure 2.  From: Notching on Cancers Door: Notch <span class="highlight" style="background-color:">Signaling</span> in Brain Tumors. </a></p><div class="supp"><p class="details"><div><b>Notch <span class="highlight" style="background-color:">signaling</span> modules relevant for brain tumors</b>. Notch <span class="highlight" style="background-color:">signaling</span> has been shown to be modulated on multiple levels in glioma cells and is linked upstream and downstream to other tumorigenic pathways. Its expression is induced by Ras and Akt, while Notch itself induces expression of epidermal growth factor (EGFR) <i>via</i> p53 (indirectly; hence dashed arrow) and pro-migratory glycoprotein tenascin C (TNC). Hypoxia-inducible factor 1 (HIF-1) and HIF-2 compete for NICD binding. The Notch inhibitor HIF-2 is displaced by HIF-1 under hypoxic conditions. Further, several proteins modulate the Notch <span class="highlight" style="background-color:">pathway</span> at the level of NICD. Examples are Numb, which promotes Notch degradation <i>via</i> ubiquitin ligases such as FBW7 or Itch, or Numb4d7 and ligand of Notch protein X (LNX), which stimulate Notch <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Marcin Teodorczyk, et al. Front Oncol. 2014;4:341.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4678&amp;ncbi_uid=7079106&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4283135/" ref="ordinalpos=4678&amp;ncbi_uid=7079106&amp;link_uid=PMC4283135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4679.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3634610/figure/F4/" ref="ordinalpos=4679&amp;ncbi_uid=4827895&amp;link_uid=PMC3634610" href="/pmc/articles/PMC3634610/"><img src="/pmc/articles/instance/3634610/bin/IRI-32-134-g004.gif" src-large="/pmc/articles/instance/3634610/bin/IRI-32-134-g004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3634610/" ref="ordinalpos=4679&amp;ncbi_uid=4827895&amp;link_uid=PMC3634610" image-link="/pmc/articles/PMC3634610/figure/F4/" class="imagepopup">Figure 4. From: C-Type Lectin-Like Receptors of the Dectin-1 Cluster: Ligands and <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div> <b>ITIM-bearing CTLRs within the Dectin-1 cluster.</b> The ITIM motif within MICL and MAH is known to interact with SHP-1 and SHP-2; however, pathways downstream of these phosphatases are only partially known. MICL is known to inhibit cytokine transcription instigated by other receptors as a method of control. On the other hand, there is evidence to suggest that MICL induces activatory pathways through MAPK, which is somewhat contradictory to the known functions of its motif. Downstream <span class="highlight" style="background-color:">signaling</span> of MAH is largely unknown. Dashed arrows represent a <span class="highlight" style="background-color:">pathway</span> that has yet to be fully defined.</div></p></div><div class="aux"><div class="resc">Anthony Plato, et al. Int Rev Immunol. 2013 April;32(2):134-156.</div><p class="links"><a class="dblinks" href="/pubmed/23570314" ref="ordinalpos=4679&amp;ncbi_uid=4827895&amp;link_uid=23570314">Citation</a><a class="dblinks" href="/pmc/articles/PMC3634610/" ref="ordinalpos=4679&amp;ncbi_uid=4827895&amp;link_uid=PMC3634610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4680.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3564525/figure/F9/" ref="ordinalpos=4680&amp;ncbi_uid=4805448&amp;link_uid=PMC3564525" href="/pmc/articles/PMC3564525/"><img src="/pmc/articles/instance/3564525/bin/385fig9.gif" src-large="/pmc/articles/instance/3564525/bin/385fig9.jpg" alt="FIGURE 9:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3564525/" ref="ordinalpos=4680&amp;ncbi_uid=4805448&amp;link_uid=PMC3564525" image-link="/pmc/articles/PMC3564525/figure/F9/" class="imagepopup">FIGURE 9:.  From: Reduced TOR <span class="highlight" style="background-color:">signaling</span> sustains hyphal development in Candida albicans by lowering Hog1 basal activity. </a></p><div class="supp"><p class="details"><div>A schematic diagram depicting the mechanism of how the Tor1 <span class="highlight" style="background-color:">signaling</span> and HOG MAPK <span class="highlight" style="background-color:">pathway</span> integrate to sustain hyphal elongation. Rapamycin reduces Tor1 activity, which in turn represses the basal activity of Hog1 via two tyrosine phosphatases, Ptp2 and Ptp3. Inactive Hog1 fails to keep the repressor Sko1 on the promoter of <i>BRG1</i>, allowing the expression of Brg1 and hyphal elongation. Black lines represent active regulatory relationships; gray lines represent relationships that are inactive.</div></p></div><div class="aux"><div class="resc">Chang Su, et al. Mol Biol Cell. 2013 February 1;24(3):385-397.</div><p class="links"><a class="dblinks" href="/pubmed/23171549" ref="ordinalpos=4680&amp;ncbi_uid=4805448&amp;link_uid=23171549">Citation</a><a class="dblinks" href="/pmc/articles/PMC3564525/" ref="ordinalpos=4680&amp;ncbi_uid=4805448&amp;link_uid=PMC3564525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4681.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3135929/figure/fig9/" ref="ordinalpos=4681&amp;ncbi_uid=3137412&amp;link_uid=PMC3135929" href="/pmc/articles/PMC3135929/"><img src="/pmc/articles/instance/3135929/bin/PP_175257_gs_f9.gif" src-large="/pmc/articles/instance/3135929/bin/PP_175257_gs_f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3135929/" ref="ordinalpos=4681&amp;ncbi_uid=3137412&amp;link_uid=PMC3135929" image-link="/pmc/articles/PMC3135929/figure/fig9/" class="imagepopup">Figure 9. From: Vacuolar Ca2+/H+ Transport Activity Is Required for Systemic Phosphate Homeostasis Involving Shoot-to-Root <span class="highlight" style="background-color:">Signaling</span> in Arabidopsis1[W][OA]. </a></p><div class="supp"><p class="details"><div>Working hypothesis of the CAX1/CAX3-mediated P<sub>i</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in Arabidopsis. When P<sub>i</sub> is limited, repression of shoot CAX1 leads to abrogated cellular Ca<sup>2+</sup> homeostasis involving altered [Ca<sup>2+</sup>]<sub>cyt</sub> and/or misregulated compartmentation of Ca<sup>2+</sup>, which initiates the activation of a subset of shoot PSR genes. A shoot-born signal (X) is thereby generated and moves toward the root to activate the P<sub>i</sub> transporter system, including PHT1;1, at the posttranslational level.</div></p></div><div class="aux"><div class="resc">Tzu-Yin Liu, et al. Plant Physiol. 2011 July;156(3):1176-1189.</div><p class="links"><a class="dblinks" href="/pubmed/21546457" ref="ordinalpos=4681&amp;ncbi_uid=3137412&amp;link_uid=21546457">Citation</a><a class="dblinks" href="/pmc/articles/PMC3135929/" ref="ordinalpos=4681&amp;ncbi_uid=3137412&amp;link_uid=PMC3135929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4682.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2940575/figure/NOP022F1/" ref="ordinalpos=4682&amp;ncbi_uid=2744081&amp;link_uid=PMC2940575" href="/pmc/articles/PMC2940575/"><img src="/pmc/articles/instance/2940575/bin/nop02201.gif" src-large="/pmc/articles/instance/2940575/bin/nop02201.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2940575/" ref="ordinalpos=4682&amp;ncbi_uid=2744081&amp;link_uid=PMC2940575" image-link="/pmc/articles/PMC2940575/figure/NOP022F1/" class="imagepopup">Fig. 1. From: The functional role of Notch <span class="highlight" style="background-color:">signaling</span> in human gliomas. </a></p><div class="supp"><p class="details"><div>Schematic overview of the canonical Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Notch receptors are produced as large proteins that are cleaved (S1) and inserted in the membrane as heterodimers. Upon ligand binding, two consecutive cleavages occur (S2 and S3), which release the ICN. In the nucleus, ICN forms a multimeric protein complex together with CSL and co-activators, and initiates transcription of target genes. See text for further details.</div></p></div><div class="aux"><div class="resc">Marie-Thrse Stockhausen, et al. Neuro Oncol. 2010 February;12(2):199-211.</div><p class="links"><a class="dblinks" href="/pubmed/20150387" ref="ordinalpos=4682&amp;ncbi_uid=2744081&amp;link_uid=20150387">Citation</a><a class="dblinks" href="/pmc/articles/PMC2940575/" ref="ordinalpos=4682&amp;ncbi_uid=2744081&amp;link_uid=PMC2940575">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4683.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2787930/figure/Fig8/" ref="ordinalpos=4683&amp;ncbi_uid=2208433&amp;link_uid=PMC2787930" href="/pmc/articles/PMC2787930/"><img src="/pmc/articles/instance/2787930/bin/12307_2009_30_Fig8_HTML.gif" src-large="/pmc/articles/instance/2787930/bin/12307_2009_30_Fig8_HTML.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2787930/" ref="ordinalpos=4683&amp;ncbi_uid=2208433&amp;link_uid=PMC2787930" image-link="/pmc/articles/PMC2787930/figure/Fig8/" class="imagepopup">Fig. 8.  From: The NF-?B/AKT-dependent Induction of Wnt <span class="highlight" style="background-color:">Signaling</span> in Colon Cancer Cells by Macrophages and IL-1?. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> whereby tumor associated macrophages promote Wnt <span class="highlight" style="background-color:">signaling</span> in tumor cells. Peripheral blood monocytes (Mo) were cultured with control medium or with conditioned medium from HCT116 or Hke-3 cells for 48 h. As shown here, soluble factor(s) from HCT116 and Hke-3 cells induced maturation of normal peripheral blood monocytes (Mo), demonstrated by phalloidin/DAPI staining, coupled to the release of IL-1. IL-1, through activation of NF-B, induced phosphorylation of PDK1 and AKT, which inactivates GSK3, leading to enhanced -catenin/TCF4 transcriptional activity, and increased expression of Wnt target genes in tumor cells, including c-myc and c-jun</div></p></div><div class="aux"><div class="resc">Pawan Kaler, et al. Cancer Microenviron. 2009 December;2(1):69-80.</div><p class="links"><a class="dblinks" href="/pubmed/19779850" ref="ordinalpos=4683&amp;ncbi_uid=2208433&amp;link_uid=19779850">Citation</a><a class="dblinks" href="/pmc/articles/PMC2787930/" ref="ordinalpos=4683&amp;ncbi_uid=2208433&amp;link_uid=PMC2787930">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4684.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2656602/figure/F7/" ref="ordinalpos=4684&amp;ncbi_uid=1688906&amp;link_uid=PMC2656602" href="/pmc/articles/PMC2656602/"><img src="/pmc/articles/instance/2656602/bin/nihms-85459-f0007.gif" src-large="/pmc/articles/instance/2656602/bin/nihms-85459-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2656602/" ref="ordinalpos=4684&amp;ncbi_uid=1688906&amp;link_uid=PMC2656602" image-link="/pmc/articles/PMC2656602/figure/F7/" class="imagepopup">Figure 7.  From: D2 dopamine receptor activation facilitates endocannabinoid-mediated LTD of GABAergic synaptic transmission in midbrain dopamine neurons via cAMP-PKA <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> mechanisms underlying the cooperation between D2 receptors and group I mGluRs to induce I-LTD in dopamine neurons. (<i>A</i>) In this model, D2 receptor activation induces the release of AEA and group I mGluR activation induces the release of 2-AG from dopamine neurons. These two eCBs travel across the synapses and activate presynaptic CB1 receptors, resulting in the inhibition of cAMP/PKA activity and I-LTD induction. (<i>B</i>) In this model, group I mGluR activation induces the release of 2-AG, which activates presynaptic CB1 receptors to inhibit cAMP/PKA activity, while D2 receptor activation directly inhibits cAMP/PKA activity. D2 receptors and CB1 receptors target the same downstream effector cAMP/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to induce I-LTD.</div></p></div><div class="aux"><div class="resc">Bin Pan, et al. J Neurosci. ;28(52):14018-14030.</div><p class="links"><a class="dblinks" href="/pubmed/19109485" ref="ordinalpos=4684&amp;ncbi_uid=1688906&amp;link_uid=19109485">Citation</a><a class="dblinks" href="/pmc/articles/PMC2656602/" ref="ordinalpos=4684&amp;ncbi_uid=1688906&amp;link_uid=PMC2656602">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4685.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC275405/figure/cdg548f4/" ref="ordinalpos=4685&amp;ncbi_uid=310730&amp;link_uid=PMC275405" href="/pmc/articles/PMC275405/"><img src="/pmc/articles/instance/275405/bin/cdg548f4.gif" src-large="/pmc/articles/instance/275405/bin/cdg548f4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC275405/" ref="ordinalpos=4685&amp;ncbi_uid=310730&amp;link_uid=PMC275405" image-link="/pmc/articles/PMC275405/figure/cdg548f4/" class="imagepopup">Figure. From: scribble mutants cooperate with oncogenic Ras or Notch to cause neoplastic overgrowth in Drosophila. </a></p><div class="supp"><p class="details"><div><b>Fig. 4.</b> Ectopic activation of Ras and Notch <span class="highlight" style="background-color:">signaling</span> pathways, but not Wg, Dpp, Hh or PI3 kinase pathways, in <i>scrib</i> mutant eye clones induces massive cooperative overgrowth. (AL) The MARCM system was used ectopically to activate the EGFR/Ras <span class="highlight" style="background-color:">pathway</span> (Ras<sup>ACT</sup>; <b>A</b> and <b>B</b>); Notch <span class="highlight" style="background-color:">pathway</span> (N<sup>ACT</sup>; <b>C</b> and <b>D</b>); Wg <span class="highlight" style="background-color:">pathway</span> (Arm<sup>ACT</sup>; <b>E</b> and <b>F</b>); Dpp <span class="highlight" style="background-color:">pathway</span> (Tkv<sup>ACT</sup>; <b>G</b> and <b>H</b>); Hh <span class="highlight" style="background-color:">pathway</span> (Ci-155; <b>I</b> and <b>J</b>); or PI3 kinase <span class="highlight" style="background-color:">pathway</span> (PI3K; <b>K</b> and <b>L</b>) in either wild-type control eye clones (A, C, E, G, I and K) or <i>scrib</i><sup>1</sup> clones (B, D, F, H, J and L). All clones are marked by the expression of GFP. Developing photoreceptors are shown by Elav staining (A1L1; merged with GFP in A2L2). The resulting adult eyes (A3L3), or pharate adults dissected from the pupal case (C3), are shown where applicable. Expression of Ras<sup>ACT</sup> in control clones (A) induces precocious photoreceptor differentiation anterior to the MF (arrowhead). In <i>scrib</i><sup>1</sup> clones (B), Ras<sup>ACT</sup> fails to initiate differentiation, and instead, massive three-dimensional tissue overgrowth is induced, resulting in fusion of the eye/antennal discs to each other as well as the brain lobes (arrowheads). The insert (B1) shows wild-type brain lobes (arrowheads) and eye antennal discs (arrows) at the same magnification (magnification is half that of the other panels in this figure) for comparison. Expression of N<sup>ACT</sup> in control clones (C) interferes with photoreceptor differentiation (arrowheads), and adult flies die before eclosure with overgrown eyes. In <i>scrib</i><sup></sup> clones (D), N<sup>ACT</sup> induces a similar degree of three-dimensional overgrowth, and lack of differentiation, as Ras<sup>ACT</sup> (a planar section is shown, but overgrowth is in three dimensions). Ectopic activation of Wg <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> effectively blocks photoreceptor differentiation in control clones (E; arrowhead), and this remains the case in <i>scrib</i><sup>1</sup> clones (F). Ectopic activation of Dpp or Hh <span class="highlight" style="background-color:">signaling</span> induces patterning defects in both control clones (G and I), as well as <i>scrib</i><sup>1</sup> clones (H and J). Ectopic PI3 kinase <span class="highlight" style="background-color:">signaling</span> in <i>scrib</i><sup>1</sup> clones (L) has little effect.</div></p></div><div class="aux"><div class="resc">Anthony M. Brumby, et al. EMBO J. 2003 November 3;22(21):5769-5779.</div><p class="links"><a class="dblinks" href="/pubmed/14592975" ref="ordinalpos=4685&amp;ncbi_uid=310730&amp;link_uid=14592975">Citation</a><a class="dblinks" href="/pmc/articles/PMC275405/" ref="ordinalpos=4685&amp;ncbi_uid=310730&amp;link_uid=PMC275405">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4686.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3148410/figure/F1/" ref="ordinalpos=4686&amp;ncbi_uid=3487830&amp;link_uid=PMC3148410" href="/pmc/articles/PMC3148410/"><img src="/pmc/articles/instance/3148410/bin/nihms-301477-f0001.gif" src-large="/pmc/articles/instance/3148410/bin/nihms-301477-f0001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3148410/" ref="ordinalpos=4686&amp;ncbi_uid=3487830&amp;link_uid=PMC3148410" image-link="/pmc/articles/PMC3148410/figure/F1/" class="imagepopup">FIGURE 1. Schematic representation of CLF1 secretion and <span class="highlight" style="background-color:">signaling</span>.  From: Cytokine-Like Factor 1 (CLF1): Life after Development?. </a></p><div class="supp"><p class="details"><div>Composite cytokines of CLF1 and CLC or CLF1 and the p28 subunit of IL-27 are assembled in the cell and secreted. It is unknown if CLF1 is secreted in the absence of its partners. CLF1/CLC has been found to signal via the tripartite receptor complex of CNTFRa, LIFR, and gp130. CLF1/p28 was also found to bind IL-6Ra complexed with gp130 and WSX-1. Engagement of these receptors leads to downstream <span class="highlight" style="background-color:">signaling</span> events via the JAK/STAT <span class="highlight" style="background-color:">pathway</span> (particularly STAT3), MEK/ERK, and PI3K/AKT. The full range of cells that are potentially responsive to CLF1 stimulation is unknown. The effect of CLF1 on these cells also remains unknown.</div></p></div><div class="aux"><div class="resc">Daniel J Kass. Cytokine. ;55(3):325-329.</div><p class="links"><a class="dblinks" href="/pubmed/21715184" ref="ordinalpos=4686&amp;ncbi_uid=3487830&amp;link_uid=21715184">Citation</a><a class="dblinks" href="/pmc/articles/PMC3148410/" ref="ordinalpos=4686&amp;ncbi_uid=3487830&amp;link_uid=PMC3148410">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4687.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3146975/figure/F1/" ref="ordinalpos=4687&amp;ncbi_uid=3624606&amp;link_uid=PMC3146975" href="/pmc/articles/PMC3146975/"><img src="/pmc/articles/instance/3146975/bin/nihms285225f1.gif" src-large="/pmc/articles/instance/3146975/bin/nihms285225f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3146975/" ref="ordinalpos=4687&amp;ncbi_uid=3624606&amp;link_uid=PMC3146975" image-link="/pmc/articles/PMC3146975/figure/F1/" class="imagepopup">Figure 1. T-cell receptor <span class="highlight" style="background-color:">signaling</span> pathways regulated by GREs.  From: Coordinate Regulation of mRNA Decay Networks by GU-rich Elements and CELF1. </a></p><div class="supp"><p class="details"><div>The network diagram depicts the coordinate regulation of GRE-containing transcripts involved in T cell receptor <span class="highlight" style="background-color:">signaling</span>. Transcripts in bold are GRE-containing. Transcripts in grey were identified as CELF1 targets in HeLa cells by RNA-IP [43].<br />Transcripts in green represent short-lived GRE-containing transcripts expressed in primary human T cells [33], [37]. Transcripts labeled with an asterisk exhibited changes in steady state levels following T-cell receptor stimulation [33]. This network diagram was built using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Assistant Software.</div></p></div><div class="aux"><div class="resc">Irina Vlasova-St. Louis, et al. Curr Opin Genet Dev. ;21(4):444-451.</div><p class="links"><a class="dblinks" href="/pubmed/21497082" ref="ordinalpos=4687&amp;ncbi_uid=3624606&amp;link_uid=21497082">Citation</a><a class="dblinks" href="/pmc/articles/PMC3146975/" ref="ordinalpos=4687&amp;ncbi_uid=3624606&amp;link_uid=PMC3146975">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4688.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3076051/figure/F4/" ref="ordinalpos=4688&amp;ncbi_uid=3548392&amp;link_uid=PMC3076051" href="/pmc/articles/PMC3076051/"><img src="/pmc/articles/instance/3076051/bin/nihms253030f4.gif" src-large="/pmc/articles/instance/3076051/bin/nihms253030f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3076051/" ref="ordinalpos=4688&amp;ncbi_uid=3548392&amp;link_uid=PMC3076051" image-link="/pmc/articles/PMC3076051/figure/F4/" class="imagepopup">Figure 4. Modulation of vascular BK-1 expression by Akt/FOXO-3a/FBXO <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Muscle-Specific F-Box Only Proteins Facilitate BK Channel ?1 Subunit Down-Regulation in Vascular Smooth Muscle Cells of Diabetes Mellitus. </a></p><div class="supp"><p class="details"><div>Reduced FOXO-3a(T-32) phosphorylation with unchanged total FOXO-3a expression in STZ-induced diabetic rats (<b>A</b>) and in human CASMCs cultured with HG (<b>B</b>). 24-h treatment with 7 mol/L LY294002 (LY) attenuated the phosphorylation of Akt and FOXO-3a, enhanced the expression of total FOXO-3a and atrogin-1, and reduced BK-<sub>1</sub> protein level in human CASMCs cultured with NG (<b>C</b>). Illustration showing the <span class="highlight" style="background-color:">signaling</span> mechanisms that underlie vascular BK-<sub>1</sub> down-regulation in diabetes (<b>D</b>).</div></p></div><div class="aux"><div class="resc">Daiming Zhang, et al. Circ Res. ;107(12):1454-1459.</div><p class="links"><a class="dblinks" href="/pubmed/20966391" ref="ordinalpos=4688&amp;ncbi_uid=3548392&amp;link_uid=20966391">Citation</a><a class="dblinks" href="/pmc/articles/PMC3076051/" ref="ordinalpos=4688&amp;ncbi_uid=3548392&amp;link_uid=PMC3076051">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4689.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2699825/figure/F3/" ref="ordinalpos=4689&amp;ncbi_uid=1870370&amp;link_uid=PMC2699825" href="/pmc/articles/PMC2699825/"><img src="/pmc/articles/instance/2699825/bin/CG-10-93_F3.gif" src-large="/pmc/articles/instance/2699825/bin/CG-10-93_F3.jpg" alt="Fig. (3). Schematic representation of a canonical signaling pathway from the plasma membrane to the nucleus." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2699825/" ref="ordinalpos=4689&amp;ncbi_uid=1870370&amp;link_uid=PMC2699825" image-link="/pmc/articles/PMC2699825/figure/F3/" class="imagepopup">Fig. (3). Schematic representation of a canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from the plasma membrane to the nucleus. From: Current Screens Based on the AlphaScreen(TM) Technology for Deciphering Cell Signalling Pathways. </a></p><div class="supp"><p class="details"><div>The receptor-ligand interaction occurring in the extracellular space leads to the activation of the membrane receptor which transduces a signal into the cytoplasm. At this stage, <span class="highlight" style="background-color:">signaling</span> can involve post-translational modifications of proteins (PTM), formation of complexes (scaffolds), modification of lipids and enzyme cascade activation (GTPases, kinases). Activated complexes can then translocate in the nucleus where transcriptional activation processes can take place, followed by mRNA maturation, expression and translation into active proteins.</div></p></div><div class="aux"><div class="resc">Sad Taouji, et al. Curr Genomics. 2009 April;10(2):93-101.</div><p class="links"><a class="dblinks" href="/pubmed/19794881" ref="ordinalpos=4689&amp;ncbi_uid=1870370&amp;link_uid=19794881">Citation</a><a class="dblinks" href="/pmc/articles/PMC2699825/" ref="ordinalpos=4689&amp;ncbi_uid=1870370&amp;link_uid=PMC2699825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4690.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2688285/figure/F4/" ref="ordinalpos=4690&amp;ncbi_uid=1854861&amp;link_uid=PMC2688285" href="/pmc/articles/PMC2688285/"><img src="/pmc/articles/instance/2688285/bin/gb-2009-10-2-r20-4.gif" src-large="/pmc/articles/instance/2688285/bin/gb-2009-10-2-r20-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2688285/" ref="ordinalpos=4690&amp;ncbi_uid=1854861&amp;link_uid=PMC2688285" image-link="/pmc/articles/PMC2688285/figure/F4/" class="imagepopup">Figure 4.  From: PDGF/VEGF <span class="highlight" style="background-color:">signaling</span> controls cell size in Drosophila. </a></p><div class="supp"><p class="details"><div>Pvf/Pvr <span class="highlight" style="background-color:">signaling</span> activates a network of <span class="highlight" style="background-color:">signaling</span> modules upstream of the Tor <span class="highlight" style="background-color:">pathway</span>. <b>(a) </b><i>Gap1 </i>RNAi epistasis. <i>Ras1</i>, <i>Rheb</i>, <i>Tor </i>and <i>S6k </i>dominate in their effect on cell size, <i>ERK </i>suppresses the <i>Gap1 </i>RNAi phenotype, whereas <i>Raf</i>, <i>p110 </i>and <i>Akt </i>only partially ameliorate the large increase in cell size seen following <i>Gap1 </i>RNAi. The insulin receptor has little effect in this assay. <b>(b) </b><i>Tsc2 </i>RNAi epistasis. <i>Rheb </i>and <i>S6k </i>dominate, placing them genetically downstream of <i>Tsc2</i>. <i>Pvr</i>, <i>Ras1 </i>and members of the MAPK and PI3K pathways fail to have a dramatic impact on the <i>Tsc2 </i>RNAi phenotype. <b>(c) </b><i>Raf </i>(<i>pole hole</i>) RNAi epistasis. Silencing of <i>Raf </i>leads to a minor reduction in cell size. However, silencing of <i>Raf </i>in conjuction with <i>p110 </i>or <i>Akt1 </i>causes a large reduction in cell size, like that seen in <i>Pvr</i>, <i>Ras</i>, <i>Tor</i>, <i>Rheb</i>, and <i>S6K </i>RNAi experiments. <b>(d) </b>MAPK and PI3K <span class="highlight" style="background-color:">pathway</span> genetic interactions. <i>Raf</i>, <i>ERK </i>and <i>Ksr </i>all show additive or synergistic genetic interactions with <i>p110 </i>and <i>Akt1</i>, but not with <i>InR</i>. Furthermore, these genetic interactions are stronger than those seen when combining <i>p110 </i>or <i>Akt1 </i>dsRNA with dsRNA targeting upstream or downstream <span class="highlight" style="background-color:">pathway</span> components of this putative growth <span class="highlight" style="background-color:">signaling</span> network (<i>Pvr</i>/<i>Ras1 </i>and <i>Tor</i>/<i>S6k</i>, respectively). Error bars indicate the standard error of the mean.</div></p></div><div class="aux"><div class="resc">David Sims, et al. Genome Biol. 2009;10(2):R20-R20.</div><p class="links"><a class="dblinks" href="/pubmed/19216764" ref="ordinalpos=4690&amp;ncbi_uid=1854861&amp;link_uid=19216764">Citation</a><a class="dblinks" href="/pmc/articles/PMC2688285/" ref="ordinalpos=4690&amp;ncbi_uid=1854861&amp;link_uid=PMC2688285">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4691.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3703147/figure/F1/" ref="ordinalpos=4691&amp;ncbi_uid=5068771&amp;link_uid=PMC3703147" href="/pmc/articles/PMC3703147/"><img src="/pmc/articles/instance/3703147/bin/nihms484447f1.gif" src-large="/pmc/articles/instance/3703147/bin/nihms484447f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3703147/" ref="ordinalpos=4691&amp;ncbi_uid=5068771&amp;link_uid=PMC3703147" image-link="/pmc/articles/PMC3703147/figure/F1/" class="imagepopup">Figure 1. IFN <span class="highlight" style="background-color:">signaling</span> and the regulatory role of ubiquitination.  From: Ubiquitination-mediated regulation of interferon responses. </a></p><div class="supp"><p class="details"><div>Upon engaging a cognate receptor complex (IFNAR1-IFNAR2 for Type 1 IFN, IFNGR1-IFNGR2 for Type 2 IFN or IL28RA-IL10R2 for Type 3 IFN) IFN activate receptor-associated JAKs. A resulting tyrosine phosphorylation of STAT proteins leads to the formation of transcriptionally active complexes (STAT1 homodimers for all types and additional STAT1-STAT2-IRF9 complexes for Types 1/3 IFN) and ensuing induction of ISGs. Cellular (-Trcp, SOCS, SLIM) and diverse viral E3 ubiquitin ligases facilitate the ubiquitination of various mediators of IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to limit the extent and duration of this <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Serge Y. Fuchs. Growth Factors. ;30(3):141-148.</div><p class="links"><a class="dblinks" href="/pubmed/22394219" ref="ordinalpos=4691&amp;ncbi_uid=5068771&amp;link_uid=22394219">Citation</a><a class="dblinks" href="/pmc/articles/PMC3703147/" ref="ordinalpos=4691&amp;ncbi_uid=5068771&amp;link_uid=PMC3703147">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4692.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2901811/figure/F6/" ref="ordinalpos=4692&amp;ncbi_uid=2495769&amp;link_uid=PMC2901811" href="/pmc/articles/PMC2901811/"><img src="/pmc/articles/instance/2901811/bin/org0602_0076_fig006.gif" src-large="/pmc/articles/instance/2901811/bin/org0602_0076_fig006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2901811/" ref="ordinalpos=4692&amp;ncbi_uid=2495769&amp;link_uid=PMC2901811" image-link="/pmc/articles/PMC2901811/figure/F6/" class="imagepopup">Figure 6.  From: The regulation of organ size in Drosophila. </a></p><div class="supp"><p class="details"><div>A model of the Fat/Hippo regulation of growth. Dpp <span class="highlight" style="background-color:">signaling</span> may affect <span class="highlight" style="background-color:">signaling</span> through the <span class="highlight" style="background-color:">pathway</span> via its effects on Dachsous and Fat. Wingless <span class="highlight" style="background-color:">signaling</span> may affect the activity of Yorkie through Vestigials interaction with Scalloped, a Yorkie co-factor. How Vestigial affects the interaction between Yorkie and Scalloped is, however, unclear. Mechanical forces may affect the activity of Hippo through their effect on Merlin and Expanded. Solid lines indicate protein-protein binding, solid arrows indicate biochemical interactions, dotted arrows indicate genetic interactions, gray arrow indicates hypothetical interaction. Data from references 96, 99 and 103.</div></p></div><div class="aux"><div class="resc">Alexander W Shingleton. Organogenesis. 2010 Apr-Jun;6(2):76-87.</div><p class="links"><a class="dblinks" href="/pubmed/20885854" ref="ordinalpos=4692&amp;ncbi_uid=2495769&amp;link_uid=20885854">Citation</a><a class="dblinks" href="/pmc/articles/PMC2901811/" ref="ordinalpos=4692&amp;ncbi_uid=2495769&amp;link_uid=PMC2901811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4693.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1904494/figure/F6/" ref="ordinalpos=4693&amp;ncbi_uid=1053619&amp;link_uid=PMC1904494" href="/pmc/articles/PMC1904494/"><img src="/pmc/articles/instance/1904494/bin/nihms4410f6.gif" src-large="/pmc/articles/instance/1904494/bin/nihms4410f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1904494/" ref="ordinalpos=4693&amp;ncbi_uid=1053619&amp;link_uid=PMC1904494" image-link="/pmc/articles/PMC1904494/figure/F6/" class="imagepopup">Figure 6.  From: Integrin ?v?3 is a Coreceptor for Human Cytomegalovirus Infection. </a></p><div class="supp"><p class="details"><div>Coordination between EGFR- and v3-mediated <span class="highlight" style="background-color:">signaling</span> is required for HCMV-induced downregulation of RhoA activity, disruption of stress fibers, and virus nuclear translocation.<br />(<b>a</b>) HCMV induced downregulation of the RhoA/cofilin <span class="highlight" style="background-color:">pathway</span>, (<b>b</b>) RhoA/cofilin downregulation was inhibited by blocking EGFR- or v3-mediated <span class="highlight" style="background-color:">signaling</span>, (<b>c</b>) Ly29 or PP2 inhibited HCMV nuclear translocation (determined by real-time PCR), which was relieved by Y-27632. (<b>d</b>) HCMV induced stress fiber disassembly, which correlated with UL94 nuclear translocation. (<b>e</b>) Stress fiber disassembly and UL94 nuclear translocation were inhibited by blocking PI3-K or Src <span class="highlight" style="background-color:">signaling</span>. The inhibition was relieved by Y-27632. Cytochalasin B (Cyto B) facilitated HCMV nuclear translocation.</div></p></div><div class="aux"><div class="resc">Xin Wang, et al. Nat Med. ;11(5):515-521.</div><p class="links"><a class="dblinks" href="/pubmed/15834425" ref="ordinalpos=4693&amp;ncbi_uid=1053619&amp;link_uid=15834425">Citation</a><a class="dblinks" href="/pmc/articles/PMC1904494/" ref="ordinalpos=4693&amp;ncbi_uid=1053619&amp;link_uid=PMC1904494">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4694.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99013/figure/F21/" ref="ordinalpos=4694&amp;ncbi_uid=52523&amp;link_uid=PMC99013" href="/pmc/articles/PMC99013/"><img src="/pmc/articles/instance/99013/bin/mr0400030021.gif" src-large="/pmc/articles/instance/99013/bin/mr0400030021.jpg" alt="FIG. 21" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99013/" ref="ordinalpos=4694&amp;ncbi_uid=52523&amp;link_uid=PMC99013" image-link="/pmc/articles/PMC99013/figure/F21/" class="imagepopup">FIG. 21.  From: Signal Transduction Cascades Regulating Fungal  Development and Virulence. </a></p><div class="supp"><p class="details"><div>Model for signal transduction pathways in <i>N. crassa</i>. A cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> consisting of G protein  subunits (GNA1 and GNA2), adenylyl cyclase (CR1), and a PKA regulatory subunit (MCB) is involved in conidiation, morphogenesis, mating, and stress tolerance. Ras-, MAP kinase (MAPK)-, and PKA-related kinase-mediated <span class="highlight" style="background-color:">signaling</span> pathways also regulate conidiation. How <span class="highlight" style="background-color:">signaling</span> is coordinated between these pathways remains to be elucidated.</div></p></div><div class="aux"><div class="resc">Klaus B. Lengeler, et al. Microbiol Mol Biol Rev. 2000 December;64(4):746-785.</div><p class="links"><a class="dblinks" href="/pubmed/11104818" ref="ordinalpos=4694&amp;ncbi_uid=52523&amp;link_uid=11104818">Citation</a><a class="dblinks" href="/pmc/articles/PMC99013/" ref="ordinalpos=4694&amp;ncbi_uid=52523&amp;link_uid=PMC99013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4695.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3326002/figure/pone-0034016-g002/" ref="ordinalpos=4695&amp;ncbi_uid=3954940&amp;link_uid=PMC3326002" href="/pmc/articles/PMC3326002/"><img src="/pmc/articles/instance/3326002/bin/pone.0034016.g002.gif" src-large="/pmc/articles/instance/3326002/bin/pone.0034016.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3326002/" ref="ordinalpos=4695&amp;ncbi_uid=3954940&amp;link_uid=PMC3326002" image-link="/pmc/articles/PMC3326002/figure/pone-0034016-g002/" class="imagepopup">Figure 2. Pum overrides EGFR <span class="highlight" style="background-color:">signaling</span>..  From: Negative Regulation of EGFR/MAPK <span class="highlight" style="background-color:">Pathway</span> by Pumilio in Drosophila melanogaster. </a></p><div class="supp"><p class="details"><div><i>ser-Gal4</i> derived expression of Pum and EGFR (A-C). Pum expression (<i>UAS-Pum</i>/+; <i>ser-GAL4</i>/<i>+</i>) resulted in notching in the wing boundary (A). EGFR expression (<i>ser-GAL4</i>/<i>UAS-EGFR)</i> caused extra-wing veins (arrowhead, B). Concomitant <i>pum</i> expression (<i>UAS-Pum</i>/+; <i>ser-GAL4</i>/<i>UAS-EGFR</i>) diminished extra wing veins induced by enhanced EGFR <span class="highlight" style="background-color:">signaling</span> and generated notch around wing boundary (C). <i>dpp-Gal4</i> derived expression of Pum and <i>rl<sup>[Sem]</sup></i> (D-F). Pum expression (<i>UAS-Pum</i>/+; <i>dpp-GAL4</i>/+) eliminated wing veins (arrow, D), whereas <i>rl<sup>[Sem]</sup></i> expression (<i>dpp-GAL4&gt;&gt;UAS-rl<sup>[Sem]</sup></i>) produced extra wing veins (arrowheads in E) at the A/P boundary. Concomitant Pum expression diminished extra wing veins induced by enhanced Rl <span class="highlight" style="background-color:">signaling</span> (<i>UAS-Pum</i>/<i>UAS- rl<sup>[Sem]</sup></i>; <i>dpp-GAL4</i>/+) (F).</div></p></div><div class="aux"><div class="resc">Sung Yun Kim, et al. PLoS One. 2012;7(4):e34016.</div><p class="links"><a class="dblinks" href="/pubmed/22514614" ref="ordinalpos=4695&amp;ncbi_uid=3954940&amp;link_uid=22514614">Citation</a><a class="dblinks" href="/pmc/articles/PMC3326002/" ref="ordinalpos=4695&amp;ncbi_uid=3954940&amp;link_uid=PMC3326002">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4696.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3140648/figure/F1/" ref="ordinalpos=4696&amp;ncbi_uid=3141544&amp;link_uid=PMC3140648" href="/pmc/articles/PMC3140648/"><img src="/pmc/articles/instance/3140648/bin/fnana-05-00043-g001.gif" src-large="/pmc/articles/instance/3140648/bin/fnana-05-00043-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3140648/" ref="ordinalpos=4696&amp;ncbi_uid=3141544&amp;link_uid=PMC3140648" image-link="/pmc/articles/PMC3140648/figure/F1/" class="imagepopup">Figure 1.  From: Mechanisms for the Modulation of Dopamine D1 Receptor <span class="highlight" style="background-color:">Signaling</span> in Striatal Neurons. </a></p><div class="supp"><p class="details"><div><b>The D<sub>1</sub> receptor <span class="highlight" style="background-color:">signaling</span> cascades in striatonigral/direct <span class="highlight" style="background-color:">pathway</span> neurons</b>. D<sub>1</sub> receptors couple to at least three distinct <span class="highlight" style="background-color:">signaling</span> cascades: (1) G<sub>s/olf</sub>/adenylyl cyclase (AC)/cAMP/PKA/DARPP-32/protein phosphatase-1 (PP-1) <span class="highlight" style="background-color:">signaling</span> (blue; Svenningsson et al., 2004; Stipanovich et al., 2008), (2) G<sub>q</sub>/phospholipase C (PLC)/inositol 1,4,5-trisphosphate (IP<sub>3</sub>)/IP<sub>3</sub> receptor/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> (orange; Rashid et al., 2007a; Kuroiwa et al., 2008; Hasbi et al., 2009), (3) G<sub></sub>/Src family kinase (SFK)/NMDA receptor NR2B subunit/Ca<sup>2+</sup>/Ras-guanine nucleotide-releasing factor 1 (Ras-GRF1)/ mitogen-activated protein kinase/ERK kinase (MEK)/ERK <span class="highlight" style="background-color:">signaling</span> (green; Girault et al., 2007; Pascoli et al., 2011). The phosphorylation levels of DARPP-32 are low at Thr34 and high at Thr75, Ser97, and Ser130 under basal conditions. Activation of PKA induces the phosphorylation of DARPP-32 at Thr34 and the dephosphorylation of DARPP-32 at Thr75 and Ser97 by PP-2A/B56 complex, and phospho-Thr34/dephospho-Ser97 DARPP-32 accumulates in nucleus and inhibits PP-1, leading to the increase in histone H3 phosphorylation (Stipanovich et al., 2008). ERK, activated by two D<sub>1</sub> receptor pathways, induces mitogen- and stress-activated kinase 1 (MSK1) activation and histone H3 and cAMP-response element binding protein (CREB) phosphorylation in the nucleus (Girault et al., 2007; Pascoli et al., 2011). Thus, D<sub>1</sub> receptor-mediated activation of PKA, intracellular Ca<sup>2+</sup>, and ERK <span class="highlight" style="background-color:">signaling</span> induces the changes in downstream <span class="highlight" style="background-color:">signaling</span> cascades and the transcriptional activation of many genes. CaMK, Ca<sup>2+</sup>/calmodulin-dependent protein kinase; DAG, diacylglycerol; PDE, phosphodiesterase; STEP, striatal-enriched tyrosine phosphatase.</div></p></div><div class="aux"><div class="resc">Akinori Nishi, et al. Front Neuroanat. 2011;5:43.</div><p class="links"><a class="dblinks" href="/pubmed/21811441" ref="ordinalpos=4696&amp;ncbi_uid=3141544&amp;link_uid=21811441">Citation</a><a class="dblinks" href="/pmc/articles/PMC3140648/" ref="ordinalpos=4696&amp;ncbi_uid=3141544&amp;link_uid=PMC3140648">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4697.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3275361/figure/F3/" ref="ordinalpos=4697&amp;ncbi_uid=4571147&amp;link_uid=PMC3275361" href="/pmc/articles/PMC3275361/"><img src="/pmc/articles/instance/3275361/bin/zeg0021285840003.gif" src-large="/pmc/articles/instance/3275361/bin/zeg0021285840003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3275361/" ref="ordinalpos=4697&amp;ncbi_uid=4571147&amp;link_uid=PMC3275361" image-link="/pmc/articles/PMC3275361/figure/F3/" class="imagepopup">Fig. 3. From: Update on Bone Anabolics in Osteoporosis Treatment: Rationale, Current Status, and Perspectives. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> and cross talk of the PTH and Wnt <span class="highlight" style="background-color:">signaling</span> pathways in osteocytes. In the osteocyte (and late osteoblasts), activation of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> occurs upon simultaneous binding of the secreted glycoprotein Wnt3a (or other Wnts like Wnt 10b for instance) to the seven-helix-receptor frizzled (Fz) family and the coreceptors Lrp 5/6. Binding of Wnt3a to Lrp5/6 changes the conformation of the cytoplasmic receptor domain, causing the recruitment of Axin2 and preventing the phosphorylation of -catenin by GSK3 and its proteasomal degradation. -Catenin accumulates in the cytosol and translocates into the nucleus, thereby stimulating the expression of the Lrp5/6 antagonists Dkk1 and sclerostin, and the RANKL inhibitor OPG, via the T-cell factor/lymphoid enhancer factor (Tcf/Lef). PTH binds to its seven-transmembrane-spanning receptor and activates phosphatidyl inositol-specific phospholipase C (PLC), cAMP-dependent protein kinase A (PKA), and the protein kinase C (PKC) downstream <span class="highlight" style="background-color:">signaling</span> cascades, all contributing to the bone anabolic effect of PTH. In addition, PTH <span class="highlight" style="background-color:">signaling</span> cross talks with the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by associating with Lrp6, inhibiting GSK3, stabilizing -catenin, and inhibiting the expression of both sclerostin and Dkk1.</div></p></div><div class="aux"><div class="resc">Roland Baron, et al. J Clin Endocrinol Metab. 2012 February;97(2):311-325.</div><p class="links"><a class="dblinks" href="/pubmed/22238383" ref="ordinalpos=4697&amp;ncbi_uid=4571147&amp;link_uid=22238383">Citation</a><a class="dblinks" href="/pmc/articles/PMC3275361/" ref="ordinalpos=4697&amp;ncbi_uid=4571147&amp;link_uid=PMC3275361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4698.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1797620/figure/ppat-0030021-g003/" ref="ordinalpos=4698&amp;ncbi_uid=934378&amp;link_uid=PMC1797620" href="/pmc/articles/PMC1797620/"><img src="/pmc/articles/instance/1803010/bin/ppat.0030021.g003.gif" src-large="/pmc/articles/instance/1803010/bin/ppat.0030021.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1797620/" ref="ordinalpos=4698&amp;ncbi_uid=934378&amp;link_uid=PMC1797620" image-link="/pmc/articles/PMC1797620/figure/ppat-0030021-g003/" class="imagepopup">Figure 3. OspF Inhibits the Yeast CWI <span class="highlight" style="background-color:">Pathway</span> by Inhibition of MAPK Phosphorylation.  From: Yeast Functional Genomic Screens Lead to Identification of a Role for a Bacterial Effector in Innate Immunity Regulation. </a></p><div class="supp"><p class="details"><div>(A) Summary of the activity of an RLM1-regulated -galactosidase reporter in response to heat shock in the presence or absence of OspF.<br />(B) Outline of the CWI <span class="highlight" style="background-color:">pathway</span>.<br />(C) Yeast containing empty vector or a plasmid that conditionally expresses OspF were subjected to the designated stress 2 h after the induction of expression of OspF (see Materials and Methods). Representative immunoblots used to assay for activation of each of four MAPK <span class="highlight" style="background-color:">signaling</span> pathways: SLT2 (CWI <span class="highlight" style="background-color:">pathway</span>), FUS3 (mating <span class="highlight" style="background-color:">pathway</span>), KSS1 (invasive growth <span class="highlight" style="background-color:">pathway</span>), and HOG1 (high-osmolarity glycerol <span class="highlight" style="background-color:">pathway</span>) are shown. The circled P denotes phosphorylated versions of the proteins. The blots were probed with the anti-PSTAIRE antibody that recognizes CDC28 and PHO85 as a loading control. Each experiment was conducted at least in triplicate with similar results.</div></p></div><div class="aux"><div class="resc">Roger W Kramer, et al. PLoS Pathog. 2007 February;3(2):e21.</div><p class="links"><a class="dblinks" href="/pubmed/17305427" ref="ordinalpos=4698&amp;ncbi_uid=934378&amp;link_uid=17305427">Citation</a><a class="dblinks" href="/pmc/articles/PMC1797620/" ref="ordinalpos=4698&amp;ncbi_uid=934378&amp;link_uid=PMC1797620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4699.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3304673/figure/fig1/" ref="ordinalpos=4699&amp;ncbi_uid=3897018&amp;link_uid=PMC3304673" href="/pmc/articles/PMC3304673/"><img src="/pmc/articles/instance/3304673/bin/JBB2012-594056.001.gif" src-large="/pmc/articles/instance/3304673/bin/JBB2012-594056.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3304673/" ref="ordinalpos=4699&amp;ncbi_uid=3897018&amp;link_uid=PMC3304673" image-link="/pmc/articles/PMC3304673/figure/fig1/" class="imagepopup">Figure 1.  From: Effects of IRF5 Lupus Risk Haplotype on Pathways Predicted to Influence B Cell Functions. </a></p><div class="supp"><p class="details"><div>Association of the canonical interferon <span class="highlight" style="background-color:">pathway</span> with the <i>IRF5</i> haplotype. Nave B cells were either exposed to EBV for 16 hours or left unstimulated. Whole-genome expression levels were compared between cells with the risk and protective haplotypes using the Illumina platform. Significant association of the data set with the canonical IFN <span class="highlight" style="background-color:">pathway</span> was discovered using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (<i>P</i> &lt; 0.01). Interactions between genes in the interferon <span class="highlight" style="background-color:">pathway</span> based on Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis are shown for both the unstimulated (a) and the EBV-infected conditions (b). Blue gene symbols represent genes with relatively lower expression in the cells with the risk haplotype, while genes with red shading are upregulated in the risk cells. IFIT3: IFN-induced protein with tetratricopeptide repeats 3, SOCS: suppressor of cytokine <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Joel M. Guthridge, et al. J Biomed Biotechnol. 2012;2012:594056.</div><p class="links"><a class="dblinks" href="/pubmed/22500098" ref="ordinalpos=4699&amp;ncbi_uid=3897018&amp;link_uid=22500098">Citation</a><a class="dblinks" href="/pmc/articles/PMC3304673/" ref="ordinalpos=4699&amp;ncbi_uid=3897018&amp;link_uid=PMC3304673">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4700.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3801191/figure/F1/" ref="ordinalpos=4700&amp;ncbi_uid=5596005&amp;link_uid=PMC3801191" href="/pmc/articles/PMC3801191/"><img src="/pmc/articles/instance/3801191/bin/nihms519277f1.gif" src-large="/pmc/articles/instance/3801191/bin/nihms519277f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3801191/" ref="ordinalpos=4700&amp;ncbi_uid=5596005&amp;link_uid=PMC3801191" image-link="/pmc/articles/PMC3801191/figure/F1/" class="imagepopup">Figure 1. Summary of the Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: Estrogen Regulation of Dkk1 and Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> in Neurodegenerative Disease. </a></p><div class="supp"><p class="details"><div>This figure briefly summarizes canonical Wnt <span class="highlight" style="background-color:">signaling</span>. <b><i>A,</i></b> In the absence of Wnt ligand, cytosolic -Catenin is phosphorylated, ubiquitinated, and degraded by the proteasome to prevent the expression of Wnt target genes. <b><i>B,</i></b> However, in the presence of Wnt ligand, cytosolic -Catenin is stabilized and translocates to the nucleus, where it acts as a co-activator and facilitates the transcription of Wnt target genes. Dkk1 antagonizes canonical Wnt <span class="highlight" style="background-color:">signaling</span> by binding to the Wnt co-receptor LRP5/6 and preventing formation of the Wnt-Frizzled-LRP5/6 complex. See text for further details. APC, <i>adenomatous polyposis coli</i>, -Cat, -Catenin; -TrCP, Beta-Transducing repeat-Containing Protein; Dvl, Disheveled; CKI, Casein Kinase I; Dkk1, Dickkopf-1; GSK3, Glycogen Synthase Kinase 3; LRP5/6, Low Density Lipoprotein-Related Protein 5/6; TCF/LEF, T Cell Factor/Lymphoid Enhancing Factor; Ub, Ubiquitin; Wnt, Wingless.</div></p></div><div class="aux"><div class="resc">Erin L. Scott, et al. Brain Res. 2013 June 13;1514:63-74.</div><p class="links"><a class="dblinks" href="/pubmed/23261660" ref="ordinalpos=4700&amp;ncbi_uid=5596005&amp;link_uid=23261660">Citation</a><a class="dblinks" href="/pmc/articles/PMC3801191/" ref="ordinalpos=4700&amp;ncbi_uid=5596005&amp;link_uid=PMC3801191">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="46" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="47" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="48" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:50:09-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A6B615431392100000000009436B8;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A6B615431392100000000009436B8" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>