
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="41" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A3A6B5431296100000000007C4954" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="40" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="41" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="42" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="41" /></div>
            <div><h2 class="result_count">Results: 4001 to 4100 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4001.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2786213/figure/F12/" ref="ordinalpos=4001&amp;ncbi_uid=2190812&amp;link_uid=PMC2786213" href="/pmc/articles/PMC2786213/"><img src="/pmc/articles/instance/2786213/bin/nihms133142f12.gif" src-large="/pmc/articles/instance/2786213/bin/nihms133142f12.jpg" alt="Scheme 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2786213/" ref="ordinalpos=4001&amp;ncbi_uid=2190812&amp;link_uid=PMC2786213" image-link="/pmc/articles/PMC2786213/figure/F12/" class="imagepopup">Scheme 1.  From: Mapping of DDR1 Distribution and Oligomerization on the Cell Surface by FRET Microscopy. </a></p><div class="supp"><p class="details"><div>Proposed model of DDR1 activation mechanism. DDR1 exists on the cell membrane as a mixture of monomers and dimers. High-affinity interaction between DDR1 and collagen monomers induces internalization of DDR1 dimers and activation of the endosomal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Endosomal <span class="highlight" style="background-color:">signaling</span> induces DDR1 activation, followed by recycling of DDR1 dimers to the cell surface.</div></p></div><div class="aux"><div class="resc">Cosmin Mihai, et al. J Mol Biol. ;385(2):432-445.</div><p class="links"><a class="dblinks" href="/pubmed/19007791" ref="ordinalpos=4001&amp;ncbi_uid=2190812&amp;link_uid=19007791">Citation</a><a class="dblinks" href="/pmc/articles/PMC2786213/" ref="ordinalpos=4001&amp;ncbi_uid=2190812&amp;link_uid=PMC2786213">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4002.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3668831/figure/F1/" ref="ordinalpos=4002&amp;ncbi_uid=5665716&amp;link_uid=PMC3668831" href="/pmc/articles/PMC3668831/"><img src="/pmc/articles/instance/3668831/bin/JCI67225.f1.gif" src-large="/pmc/articles/instance/3668831/bin/JCI67225.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3668831/" ref="ordinalpos=4002&amp;ncbi_uid=5665716&amp;link_uid=PMC3668831" image-link="/pmc/articles/PMC3668831/figure/F1/" class="imagepopup">Figure 1. Models for control of hepcidin expression by iron..  From: Iron regulation by hepcidin. </a></p><div class="supp"><p class="details"><div>Efficient hepcidin expression in hepatocytes requires an intact BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, HJV, neogenin, TfR2, HFE, and BMP6. (<b>A</b>) Under high iron conditions, increased loading of Tf with iron stabilizes TfR2, disrupts the HFE-TfR1 interaction, and induces BMP6 secretion from the nonparenchymal cells of the liver, which facilitates the formation of a complex consisting of the BMP receptor/BMP6/HJV/neogenin/TfR2/HFE to induce hepcidin expression. (<b>B</b>) Low iron conditions increase MT2, which induces the cleavage of hepatic HJV. Decreased Tf saturation in the circulation destabilizes TfR2 protein and facilitates the HFE-TfR1 interaction. Low iron levels in the liver reduce BMP6 secretion from the nonparenchymal cells, consequently blunting BMP <span class="highlight" style="background-color:">signaling</span> and lowering hepcidin expression. (<b>C</b>) Inflammation induces the expression of IL-6 and activin B in the liver, which activates the transcription of hepcidin via the STAT3/JAK2 <span class="highlight" style="background-color:">pathway</span> and the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, respectively. </div></p></div><div class="aux"><div class="resc">Ningning Zhao, et al. J Clin Invest. 2013 June 3;123(6):2337-2343.</div><p class="links"><a class="dblinks" href="/pubmed/23722909" ref="ordinalpos=4002&amp;ncbi_uid=5665716&amp;link_uid=23722909">Citation</a><a class="dblinks" href="/pmc/articles/PMC3668831/" ref="ordinalpos=4002&amp;ncbi_uid=5665716&amp;link_uid=PMC3668831">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4003.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3246786/figure/fig3/" ref="ordinalpos=4003&amp;ncbi_uid=3746429&amp;link_uid=PMC3246786" href="/pmc/articles/PMC3246786/"><img src="/pmc/articles/instance/3246786/bin/JSC2011-614097.003.gif" src-large="/pmc/articles/instance/3246786/bin/JSC2011-614097.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3246786/" ref="ordinalpos=4003&amp;ncbi_uid=3746429&amp;link_uid=PMC3246786" image-link="/pmc/articles/PMC3246786/figure/fig3/" class="imagepopup">Figure 3.  From: Tumor Suppressor Function of CYLD in Nonmelanoma Skin Cancer. </a></p><div class="supp"><p class="details"><div>Downregulation of CYLD in BCC mediated by hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of ligand, the hedgehog (Hh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is inactive (left). Patched (PTCH) inhibits the activity of Smoothened (SMO), which in turn is unable to activate GLI transcription factors through interactions with FUSED and Suppressor of FUSED (SUFU). The binding of SUFU also prevents the transcription of Hh target genes. Binding of the Hh ligand inhibits PTCH and activates hedgehog <span class="highlight" style="background-color:">pathway</span> (right) through derepression of SMO and translocation of GlLI to the nucleus. Nuclear GLI activates target gene expression, including PTCH, GLI, and Snail. Expression of Snail leads to transcriptional inactivation of CYLD by recruitment of Snail to the CYLD promoter.</div></p></div><div class="aux"><div class="resc">K. C. Masoumi, et al. J Skin Cancer. 2011;2011:614097.</div><p class="links"><a class="dblinks" href="/pubmed/22235375" ref="ordinalpos=4003&amp;ncbi_uid=3746429&amp;link_uid=22235375">Citation</a><a class="dblinks" href="/pmc/articles/PMC3246786/" ref="ordinalpos=4003&amp;ncbi_uid=3746429&amp;link_uid=PMC3246786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4004.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3479556/figure/fig01/" ref="ordinalpos=4004&amp;ncbi_uid=4768560&amp;link_uid=PMC3479556" href="/pmc/articles/PMC3479556/"><img src="/pmc/articles/instance/3479556/bin/pnas.1214113109fig01.gif" src-large="/pmc/articles/instance/3479556/bin/pnas.1214113109fig01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3479556/" ref="ordinalpos=4004&amp;ncbi_uid=4768560&amp;link_uid=PMC3479556" image-link="/pmc/articles/PMC3479556/figure/fig01/" class="imagepopup">Fig. 1. From: Distinct mTORC1 pathways for transcription and cleavage of SREBP-1c. </a></p><div class="supp"><p class="details"><div>Control of SREBP-1c transcription, cleavage, and stability via the AKT-mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (<i>A</i>) During fasting or in the absence of growth factors, SREBP-1c transcription and cleavage are depressed and nuclear SREBP-1c (nSREBP-1c) is destabilized and degraded. (<i>B</i>) SREBP-1c abundance, stability, and processing are increased on stimulation of the AKT-mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, leading to an increase in the transcription of lipogenic genes. 4EBP, 4E binding protein 1; P, phosphorylated; SCF, Skp, cullin, F-box containing complex; Ub, ubiquitin.</div></p></div><div class="aux"><div class="resc">William J. Quinn, et al. Proc Natl Acad Sci U S A. 2012 October 2;109(40):15974-15975.</div><p class="links"><a class="dblinks" href="/pubmed/23012450" ref="ordinalpos=4004&amp;ncbi_uid=4768560&amp;link_uid=23012450">Citation</a><a class="dblinks" href="/pmc/articles/PMC3479556/" ref="ordinalpos=4004&amp;ncbi_uid=4768560&amp;link_uid=PMC3479556">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4005.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4323058/figure/Fig3/" ref="ordinalpos=4005&amp;ncbi_uid=7162116&amp;link_uid=PMC4323058" href="/pmc/articles/PMC4323058/"><img src="/pmc/articles/instance/4323058/bin/40170_2014_138_Fig3_HTML.gif" src-large="/pmc/articles/instance/4323058/bin/40170_2014_138_Fig3_HTML.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4323058/" ref="ordinalpos=4005&amp;ncbi_uid=7162116&amp;link_uid=PMC4323058" image-link="/pmc/articles/PMC4323058/figure/Fig3/" class="imagepopup">Figure 3.  From: Mitochondrial reactive oxygen species and cancer. </a></p><div class="supp"><p class="details"><div> <b>Reactive oxygen species modify cellular <span class="highlight" style="background-color:">signaling</span>.</b> Hydrogen peroxide derived from either NOXs or the mitochondria can activate the PI3K <span class="highlight" style="background-color:">pathway</span>, the hypoxia-inducible factor (HIF) <span class="highlight" style="background-color:">pathway</span>, and metabolic adaptations. These modifications are essential to allowing the survival, growth, and proliferation fundamental to tumorigenesis.</div></p></div><div class="aux"><div class="resc">Lucas B Sullivan, et al. Cancer Metab. 2014;2:17.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4005&amp;ncbi_uid=7162116&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4323058/" ref="ordinalpos=4005&amp;ncbi_uid=7162116&amp;link_uid=PMC4323058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4006.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3726361/figure/F4/" ref="ordinalpos=4006&amp;ncbi_uid=5134015&amp;link_uid=PMC3726361" href="/pmc/articles/PMC3726361/"><img src="/pmc/articles/instance/3726361/bin/1471-2164-14-496-4.gif" src-large="/pmc/articles/instance/3726361/bin/1471-2164-14-496-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3726361/" ref="ordinalpos=4006&amp;ncbi_uid=5134015&amp;link_uid=PMC3726361" image-link="/pmc/articles/PMC3726361/figure/F4/" class="imagepopup">Figure 4.  From: Gene bionetworks that regulate ovarian primordial follicle assembly. </a></p><div class="supp"><p class="details"><div><b>Focal adhesion <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> highlighting those genes that are differentially expressed compared to untreated control ovaries and identifying by color the treatment group(s) from which the differentially expressed genes came.</b> Adapted from KEGG <span class="highlight" style="background-color:">pathway</span> rno04510, Kanehisa Laboratories, Kyoto University, Japan.</div></p></div><div class="aux"><div class="resc">Eric Nilsson, et al. BMC Genomics. 2013;14:496-496.</div><p class="links"><a class="dblinks" href="/pubmed/23875758" ref="ordinalpos=4006&amp;ncbi_uid=5134015&amp;link_uid=23875758">Citation</a><a class="dblinks" href="/pmc/articles/PMC3726361/" ref="ordinalpos=4006&amp;ncbi_uid=5134015&amp;link_uid=PMC3726361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4007.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2573882/figure/F1/" ref="ordinalpos=4007&amp;ncbi_uid=1503110&amp;link_uid=PMC2573882" href="/pmc/articles/PMC2573882/"><img src="/pmc/articles/instance/2573882/bin/1756-0500-1-81-1.gif" src-large="/pmc/articles/instance/2573882/bin/1756-0500-1-81-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2573882/" ref="ordinalpos=4007&amp;ncbi_uid=1503110&amp;link_uid=PMC2573882" image-link="/pmc/articles/PMC2573882/figure/F1/" class="imagepopup">Figure 1.  From: Cellular phosphatases facilitate combinatorial processing of receptor-activated signals. </a></p><div class="supp"><p class="details"><div><b>Influence of Phosphatase knockdowns on BCR-dependent activation of <span class="highlight" style="background-color:">signaling</span> intermediates</b>. Panel 'A' depicts the kinetics of phosphorylation of select intermediates in the BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Normalized phosphorylation profiles are represented here as a heat map with rows showing various siRNA knockdowns and columns showing different <span class="highlight" style="background-color:">signaling</span> intermediates whose phosphorylation were measured. The blown out region shows such profiles of PKD under all the knockdown conditions. The color bar shows relative magnitude of phosphorylation at individual time points. Panel 'B' shows cumulative area under curve calculated for activation profile of all the <span class="highlight" style="background-color:">signaling</span> molecules under individual knockdown condition (see text). Total area under activation curve as well as individual contribution of <span class="highlight" style="background-color:">signaling</span> intermediates varied across the siRNA knockdown conditions. The height of each bar gives the net strength of BCR signal generated (see text) following transfection of cells either with Mock, or individual phosphatase-specific siRNA. The colored regions within each bar identify the <span class="highlight" style="background-color:">signaling</span> intermediates examined, and the area of spread represents their proportional contribution to the net signal strength.</div></p></div><div class="aux"><div class="resc">Dhiraj Kumar, et al. BMC Res Notes. 2008;1:81-81.</div><p class="links"><a class="dblinks" href="/pubmed/18798986" ref="ordinalpos=4007&amp;ncbi_uid=1503110&amp;link_uid=18798986">Citation</a><a class="dblinks" href="/pmc/articles/PMC2573882/" ref="ordinalpos=4007&amp;ncbi_uid=1503110&amp;link_uid=PMC2573882">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4008.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3513982/figure/f1/" ref="ordinalpos=4008&amp;ncbi_uid=5535616&amp;link_uid=PMC3513982" href="/pmc/articles/PMC3513982/"><img src="/pmc/articles/instance/3513982/bin/fig-1.gif" src-large="/pmc/articles/instance/3513982/bin/fig-1.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3513982/" ref="ordinalpos=4008&amp;ncbi_uid=5535616&amp;link_uid=PMC3513982" image-link="/pmc/articles/PMC3513982/figure/f1/" class="imagepopup">FIG. 1. From: Reconstruction of Biological Networks by Incorporating Prior Knowledge into Bayesian Network Models. </a></p><div class="supp"><p class="details"><div>True network structures: <b>(a)</b> Subgraph of the ALARM network. <b>(b)</b> Schematic of a RAF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Baikang Pei, et al. J Comput Biol. 2012 December;19(12):1324-1334.</div><p class="links"><a class="dblinks" href="/pubmed/23210479" ref="ordinalpos=4008&amp;ncbi_uid=5535616&amp;link_uid=23210479">Citation</a><a class="dblinks" href="/pmc/articles/PMC3513982/" ref="ordinalpos=4008&amp;ncbi_uid=5535616&amp;link_uid=PMC3513982">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4009.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399428/figure/fig1/" ref="ordinalpos=4009&amp;ncbi_uid=4136446&amp;link_uid=PMC3399428" href="/pmc/articles/PMC3399428/"><img src="/pmc/articles/instance/3399428/bin/MBI2012-307628.001.gif" src-large="/pmc/articles/instance/3399428/bin/MBI2012-307628.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399428/" ref="ordinalpos=4009&amp;ncbi_uid=4136446&amp;link_uid=PMC3399428" image-link="/pmc/articles/PMC3399428/figure/fig1/" class="imagepopup">Figure 1.  From: Hippo and rassf1a Pathways: A Growing Affair. </a></p><div class="supp"><p class="details"><div>Hpo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila.</i> Schematic representation of Hippo kinases cascade and of its modulation by apical transmenbrame protein complexes.</div></p></div><div class="aux"><div class="resc">Francesca Fausti, et al. Mol Biol Int. 2012;2012:307628.</div><p class="links"><a class="dblinks" href="/pubmed/22830020" ref="ordinalpos=4009&amp;ncbi_uid=4136446&amp;link_uid=22830020">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399428/" ref="ordinalpos=4009&amp;ncbi_uid=4136446&amp;link_uid=PMC3399428">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4010.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3219440/figure/F4/" ref="ordinalpos=4010&amp;ncbi_uid=3423637&amp;link_uid=PMC3219440" href="/pmc/articles/PMC3219440/"><img src="/pmc/articles/instance/3219440/bin/nihms71642f4.gif" src-large="/pmc/articles/instance/3219440/bin/nihms71642f4.jpg" alt="Scheme 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3219440/" ref="ordinalpos=4010&amp;ncbi_uid=3423637&amp;link_uid=PMC3219440" image-link="/pmc/articles/PMC3219440/figure/F4/" class="imagepopup">Scheme 1.  From: High-throughput screening and mechanism-based evaluation of estrogenic botanical extracts. </a></p><div class="supp"><p class="details"><div>Simplified depiction of the classical estrogen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from ligand binding to functional protein.</div></p></div><div class="aux"><div class="resc">Cassia R. Overk, et al. Comb Chem High Throughput Screen. ;11(4):283-293.</div><p class="links"><a class="dblinks" href="/pubmed/18473738" ref="ordinalpos=4010&amp;ncbi_uid=3423637&amp;link_uid=18473738">Citation</a><a class="dblinks" href="/pmc/articles/PMC3219440/" ref="ordinalpos=4010&amp;ncbi_uid=3423637&amp;link_uid=PMC3219440">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4011.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC371122/figure/f7/" ref="ordinalpos=4011&amp;ncbi_uid=479256&amp;link_uid=PMC371122" href="/pmc/articles/PMC371122/"><img src="/pmc/articles/instance/371122/bin/zmb0070416580007.gif" src-large="/pmc/articles/instance/371122/bin/zmb0070416580007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC371122/" ref="ordinalpos=4011&amp;ncbi_uid=479256&amp;link_uid=PMC371122" image-link="/pmc/articles/PMC371122/figure/f7/" class="imagepopup">FIG. 7. From: p38 Mitogen-Activated Protein Kinase Is the Central Regulator of Cyclic AMP-Dependent Transcription of the Brown Fat Uncoupling Protein 1 Gene. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of UCP1 transcription in brown fat.</div></p></div><div class="aux"><div class="resc">Wenhong Cao, et al. Mol Cell Biol. 2004 April;24(7):3057-3067.</div><p class="links"><a class="dblinks" href="/pubmed/15024092" ref="ordinalpos=4011&amp;ncbi_uid=479256&amp;link_uid=15024092">Citation</a><a class="dblinks" href="/pmc/articles/PMC371122/" ref="ordinalpos=4011&amp;ncbi_uid=479256&amp;link_uid=PMC371122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4012.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3984840/figure/F3/" ref="ordinalpos=4012&amp;ncbi_uid=6020526&amp;link_uid=PMC3984840" href="/pmc/articles/PMC3984840/"><img src="/pmc/articles/instance/3984840/bin/nihms-376300-f0003.gif" src-large="/pmc/articles/instance/3984840/bin/nihms-376300-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3984840/" ref="ordinalpos=4012&amp;ncbi_uid=6020526&amp;link_uid=PMC3984840" image-link="/pmc/articles/PMC3984840/figure/F3/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Pathway</span> and Gene Ontology analysis of Subgroup-Specific Splicing Events Identifies Recurrent Targeting of Cerebellar Development Pathways in non-WNT medulloblastomas.  From: Subgroup Specific Alternative Splicing in Medulloblastoma. </a></p><div class="supp"><p class="details"><div>(a) Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) of the top ten pathways affected by alternative splicing across each molecular subgroup of medulloblastoma. Known <span class="highlight" style="background-color:">signaling</span> pathways: such as Tight junction <span class="highlight" style="background-color:">signaling</span> (WNT, P&lt;1.49E-2) and CREB <span class="highlight" style="background-color:">signaling</span> (SHH, P&lt;1.70E-4) were identified in our analysis as well as an abundance of neuronal pathways in non-WNT medulloblastomas.<br />(b) Cytoscape BINGO analysis of the significant Gene Ontologies (GO) targeted by alternative splicing in Group 3 tumors, after subtracting events present in the normal cerebella, identifies neuronal pathways targeting axonogenesis and glutamatergic synaptic transmission.</div></p></div><div class="aux"><div class="resc">Adrian M Dubuc, et al. Acta Neuropathol. 2012 April;123(4):485-499.</div><p class="links"><a class="dblinks" href="/pubmed/22358458" ref="ordinalpos=4012&amp;ncbi_uid=6020526&amp;link_uid=22358458">Citation</a><a class="dblinks" href="/pmc/articles/PMC3984840/" ref="ordinalpos=4012&amp;ncbi_uid=6020526&amp;link_uid=PMC3984840">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4013.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC446107/figure/f1/" ref="ordinalpos=4013&amp;ncbi_uid=433592&amp;link_uid=PMC446107" href="/pmc/articles/PMC446107/"><img src="/pmc/articles/instance/446107/bin/zjv0150449010001.gif" src-large="/pmc/articles/instance/446107/bin/zjv0150449010001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC446107/" ref="ordinalpos=4013&amp;ncbi_uid=433592&amp;link_uid=PMC446107" image-link="/pmc/articles/PMC446107/figure/f1/" class="imagepopup">FIG. 1. From: Innate Sensing of Viruses by Toll-Like Receptors. </a></p><div class="supp"><p class="details"><div>Schematic of TLR <span class="highlight" style="background-color:">signaling</span> pathways. TLR2 (in combination with TLR1 or TLR6) and TLR4 utilize MyD88 and TIRAP/MAL as primary adaptors to activate NF-B and inflammatory cytokine secretion. Also, TLR4 uses TRIF and the TRIF-related adaptor molecule (TRAM) to activate IRF3 and the IFN <span class="highlight" style="background-color:">pathway</span>. TLRs 3, 7, 8, and 9 typically localize to endocytic compartments, where they detect a variety of nucleic acids and activate the IFN <span class="highlight" style="background-color:">pathway</span>. TLR3 utilizes TRIF, but not TRAM, to activate IRF3. TLRs 7, 8, and 9 trigger inflammatory cytokine secretion and the IFN <span class="highlight" style="background-color:">pathway</span> through MyD88. Other than MyD88, the <span class="highlight" style="background-color:">signaling</span> intermediates utilized by TLRs 7, 8, and 9 to activate IFN responses remain undefined.</div></p></div><div class="aux"><div class="resc">Karl W. Boehme, et al. J Virol. 2004 August;78(15):7867-7873.</div><p class="links"><a class="dblinks" href="/pubmed/15254159" ref="ordinalpos=4013&amp;ncbi_uid=433592&amp;link_uid=15254159">Citation</a><a class="dblinks" href="/pmc/articles/PMC446107/" ref="ordinalpos=4013&amp;ncbi_uid=433592&amp;link_uid=PMC446107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4014.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067518/figure/F7/" ref="ordinalpos=4014&amp;ncbi_uid=6284652&amp;link_uid=PMC4067518" href="/pmc/articles/PMC4067518/"><img src="/pmc/articles/instance/4067518/bin/1477-7827-12-50-7.gif" src-large="/pmc/articles/instance/4067518/bin/1477-7827-12-50-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067518/" ref="ordinalpos=4014&amp;ncbi_uid=6284652&amp;link_uid=PMC4067518" image-link="/pmc/articles/PMC4067518/figure/F7/" class="imagepopup">Figure 7.  From: Molecular aspects of development and regulation of endometriosis. </a></p><div class="supp"><p class="details"><div><b>JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in endometriosis.</b> The effects of PDGF, FGF, IL-6, HGF, and VEGF are all primarily achieved through STAT activation. The JAK/STAT <span class="highlight" style="background-color:">pathway</span> is a tempting target for novel therapeutics because it is relatively simple mechanistically, providing a direct translation of extracellular signals into a transcriptional response. Of the pathways identified, perhaps the most viable as a biomarker are the STATs. Among the small molecule inhibitors of this <span class="highlight" style="background-color:">pathway</span> are Leflunomide (a JAK inhibitor) and Atiprimod (a STAT3 inhibitor). </div></p></div><div class="aux"><div class="resc">Yana B Aznaurova, et al. Reprod Biol Endocrinol. 2014;12:50-50.</div><p class="links"><a class="dblinks" href="/pubmed/24927773" ref="ordinalpos=4014&amp;ncbi_uid=6284652&amp;link_uid=24927773">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067518/" ref="ordinalpos=4014&amp;ncbi_uid=6284652&amp;link_uid=PMC4067518">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4015.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3552145/figure/F10/" ref="ordinalpos=4015&amp;ncbi_uid=5753780&amp;link_uid=PMC3552145" href="/pmc/articles/PMC3552145/"><img src="/pmc/articles/instance/3552145/bin/nihms-425513-f0010.gif" src-large="/pmc/articles/instance/3552145/bin/nihms-425513-f0010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3552145/" ref="ordinalpos=4015&amp;ncbi_uid=5753780&amp;link_uid=PMC3552145" image-link="/pmc/articles/PMC3552145/figure/F10/" class="imagepopup">Figure 10.  From: Regulation of TLR2-mediated tolerance and cross-tolerance through IRAK4 modulation by miR-132/-212. </a></p><div class="supp"><p class="details"><div>PGN-induced miR-132 and miR-212 in TLR <span class="highlight" style="background-color:">signaling</span>. (AB) A schematic summary of PGN-mediated induction of miR-132 and miR-212 targeting IRAK4 in tolerized and untolerized THP-1 monocytes. (C) LPS, PGN, and flagellin from bacterial infection bind to their respective TLRs and activate the MyD88-dependent <span class="highlight" style="background-color:">pathway</span>. This eventually leads to gene transcription with the production of TNF- and other cytokines. The MyD88 <span class="highlight" style="background-color:">pathway</span> leads to activation of NF-kB and production of miR-146a whereas activation of CREB leads to miR-132/-212 production. miR-132/-212 targets IRAK4 whereas miR-146a targets IRAK2/1 and TRAF6. IRAK4 and IRAK2/1 are critical components of the myddosome which is in turn critical for activation of the MyD88-dependent <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Md A Nahid, et al. J Immunol. ;190(3):1250-1263.</div><p class="links"><a class="dblinks" href="/pubmed/23264652" ref="ordinalpos=4015&amp;ncbi_uid=5753780&amp;link_uid=23264652">Citation</a><a class="dblinks" href="/pmc/articles/PMC3552145/" ref="ordinalpos=4015&amp;ncbi_uid=5753780&amp;link_uid=PMC3552145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4016.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1360160/figure/fig7/" ref="ordinalpos=4016&amp;ncbi_uid=611279&amp;link_uid=PMC1360160" href="/pmc/articles/PMC1360160/"><img src="/pmc/articles/instance/1449610/bin/8695f7.gif" src-large="/pmc/articles/instance/1449610/bin/8695f7.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;igure&lt;/span&gt;  7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1360160/" ref="ordinalpos=4016&amp;ncbi_uid=611279&amp;link_uid=PMC1360160" image-link="/pmc/articles/PMC1360160/figure/fig7/" class="imagepopup">F igure  7..  From: A Systematic High-Throughput Screen of a Yeast Deletion Collection for Mutants Defective in PHO5 Regulation. </a></p><div class="supp"><p class="details"><div>The <i>PHO</i> <span class="highlight" style="background-color:">pathway</span> might be networked with other cellular <span class="highlight" style="background-color:">signaling</span> pathways that are important for cell survival. Genes required for efficient <i>PHO5</i> induction under no-phosphate conditions are boxed on the left next to the known <i>PHO</i> <span class="highlight" style="background-color:">pathway</span>, whereas genes required for <i>PHO5</i> repression under high-phosphate conditions are boxed on the right. Regulation of adenosine nucleotides might have an effect on the <i>PHO</i> <span class="highlight" style="background-color:">pathway</span> by acting upstream of <i>PHO81</i>, while the other processes seem to contribute to <i>PHO5</i> regulation at levels of transcription and other processes downstream of <i>PHO4</i>.</div></p></div><div class="aux"><div class="resc">Sidong Huang, et al. Genetics. 2005 April;169(4):1859-1871.</div><p class="links"><a class="dblinks" href="/pubmed/15695358" ref="ordinalpos=4016&amp;ncbi_uid=611279&amp;link_uid=15695358">Citation</a><a class="dblinks" href="/pmc/articles/PMC1360160/" ref="ordinalpos=4016&amp;ncbi_uid=611279&amp;link_uid=PMC1360160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4017.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508164/figure/F2/" ref="ordinalpos=4017&amp;ncbi_uid=5657104&amp;link_uid=PMC3508164" href="/pmc/articles/PMC3508164/"><img src="/pmc/articles/instance/3508164/bin/nihms411023f2.gif" src-large="/pmc/articles/instance/3508164/bin/nihms411023f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508164/" ref="ordinalpos=4017&amp;ncbi_uid=5657104&amp;link_uid=PMC3508164" image-link="/pmc/articles/PMC3508164/figure/F2/" class="imagepopup">Fig. 2.  From: Cripto-1 enhances the canonical Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by binding to LRP5 and LRP6 co-receptors. </a></p><div class="supp"><p class="details"><div>Cripto-1 lacking GPI-anchor does not enhance the canonical Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> stimulated by Wnt3a in 293T cells. (A) Full-length (CR-1 WT) and GPI-anchor signal sequence deletion mutant (CR-1 C) of human Cripto-1 in a 3FLAG expression vector. (B) SuperTOPFLASH luciferase assay in 293T cells transiently transfected with different concentrations of 3FLAG-CR-1 WT or 3FLAG-CR-1 C (10 and 100 ng) together with 50 ng of SuperTOPFLASH luciferase vector. Cells were subsequently treated with 50 ng/ml of rhWnt3a protein for 16-20 hr. These results are the mean SD of triplicates from one of three separate experiments.</div></p></div><div class="aux"><div class="resc">Tadahiro Nagaoka, et al. Cell Signal. ;25(1):178-189.</div><p class="links"><a class="dblinks" href="/pubmed/23022962" ref="ordinalpos=4017&amp;ncbi_uid=5657104&amp;link_uid=23022962">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508164/" ref="ordinalpos=4017&amp;ncbi_uid=5657104&amp;link_uid=PMC3508164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4018.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3351464/figure/pone-0036739-g004/" ref="ordinalpos=4018&amp;ncbi_uid=4008132&amp;link_uid=PMC3351464" href="/pmc/articles/PMC3351464/"><img src="/pmc/articles/instance/3351464/bin/pone.0036739.g004.gif" src-large="/pmc/articles/instance/3351464/bin/pone.0036739.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3351464/" ref="ordinalpos=4018&amp;ncbi_uid=4008132&amp;link_uid=PMC3351464" image-link="/pmc/articles/PMC3351464/figure/pone-0036739-g004/" class="imagepopup">Figure 4. MEK-ERK1/2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is required for radiation-induced c-Jun phosphorylation..  From: Radiation-Induced c-Jun Activation Depends on MEK1-ERK1/2 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Microglial Cells. </a></p><div class="supp"><p class="details"><div>(A) Cell lysates collected from 10 Gy-irradiated BV2 cells were immunoblotted with anti-phosphorylated MEK1/2 (sc-7995R) antibody. (B) BV2 cells were transfected with HA-MEK1, HA-MEK1<sup>K97M</sup> (kinase-dead mutant), and vector control. Protein levels of p-c-Jun and p-ERK1/2 (upper and lower panels) are shown. (C) BV2 cells that were treated or untreated with 10 M U0126 were irradiated with 10 Gy. p-c-Jun and p-ERK1/2 levels were analyzed by Western blot. (D) Two shRNAs (sh-MEK1-212 and sh-MEK1-495) were lentivirally delivered into BV2 cells to knockdown MEK1 protein. Knockdown efficiency and protein levels of p-c-Jun and p-ERK1/2 were detected with the indicated antibodies. The quantification results for (C) and (D) are shown in Figure S2.</div></p></div><div class="aux"><div class="resc">Zhiyong Deng, et al. PLoS One. 2012;7(5):e36739.</div><p class="links"><a class="dblinks" href="/pubmed/22606284" ref="ordinalpos=4018&amp;ncbi_uid=4008132&amp;link_uid=22606284">Citation</a><a class="dblinks" href="/pmc/articles/PMC3351464/" ref="ordinalpos=4018&amp;ncbi_uid=4008132&amp;link_uid=PMC3351464">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4019.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3243079/figure/F2/" ref="ordinalpos=4019&amp;ncbi_uid=3721958&amp;link_uid=PMC3243079" href="/pmc/articles/PMC3243079/"><img src="/pmc/articles/instance/3243079/bin/JC-10-25-g002.gif" src-large="/pmc/articles/instance/3243079/bin/JC-10-25-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3243079/" ref="ordinalpos=4019&amp;ncbi_uid=3721958&amp;link_uid=PMC3243079" image-link="/pmc/articles/PMC3243079/figure/F2/" class="imagepopup">Figure 2.  From: Breaking the cycle: An insight into the role of ER? in eukaryotic cell cycles. </a></p><div class="supp"><p class="details"><div>Non-genomic ER <span class="highlight" style="background-color:">pathway</span>. Upon stimulation by growth factors, ER is able to initiate cytoplasmic <span class="highlight" style="background-color:">signaling</span> cascades, such as the MAPK and PI3K pathways. ER is phosphorylated in the non-genomic <span class="highlight" style="background-color:">pathway</span> by a tyrosine kinase receptor (TKR) in a ligand-independent manner and can promote unregulated gene transcription</div></p></div><div class="aux"><div class="resc">Sonia Javan Moghadam, et al. J Carcinog. 2011;10:25.</div><p class="links"><a class="dblinks" href="/pubmed/22190867" ref="ordinalpos=4019&amp;ncbi_uid=3721958&amp;link_uid=22190867">Citation</a><a class="dblinks" href="/pmc/articles/PMC3243079/" ref="ordinalpos=4019&amp;ncbi_uid=3721958&amp;link_uid=PMC3243079">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4020.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2519632/figure/f6/" ref="ordinalpos=4020&amp;ncbi_uid=1522042&amp;link_uid=PMC2519632" href="/pmc/articles/PMC2519632/"><img src="/pmc/articles/instance/2519632/bin/zjv0170809790006.gif" src-large="/pmc/articles/instance/2519632/bin/zjv0170809790006.jpg" alt="FIG. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2519632/" ref="ordinalpos=4020&amp;ncbi_uid=1522042&amp;link_uid=PMC2519632" image-link="/pmc/articles/PMC2519632/figure/f6/" class="imagepopup">FIG. 6. From: Activation of Akt through gp130 Receptor <span class="highlight" style="background-color:">Signaling</span> Is Required for Kaposi's Sarcoma-Associated Herpesvirus-Induced Lymphatic Reprogramming of Endothelial Cells . </a></p><div class="supp"><p class="details"><div>Proposed model of KSHV-induced lymphatic reprogramming of endothelial cells. KSHV infection of endothelial cells induces the secretion of a soluble factor that activates the gp130 receptor. Activation of the gp130 receptor activates JAK2, which directly phosphorylates and activates STAT3. Downstream of gp130, JAK2 also promotes the activation of PI3K to phosphorylate Akt. Prox1 is induced downstream of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> and regulates VEGFR-3 induction and podoplanin expression. STAT3 regulates gp130 induction, and its persistent activation controls a positive autoregulatory loop of the entire <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Arrows with solid lines, direct regulation; arrows with dashed lines, indirect regulation of the target.</div></p></div><div class="aux"><div class="resc">Valerie A. Morris, et al. J Virol. 2008 September;82(17):8771-8779.</div><p class="links"><a class="dblinks" href="/pubmed/18579585" ref="ordinalpos=4020&amp;ncbi_uid=1522042&amp;link_uid=18579585">Citation</a><a class="dblinks" href="/pmc/articles/PMC2519632/" ref="ordinalpos=4020&amp;ncbi_uid=1522042&amp;link_uid=PMC2519632">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4021.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4303926/figure/f1/" ref="ordinalpos=4021&amp;ncbi_uid=7121531&amp;link_uid=PMC4303926" href="/pmc/articles/PMC4303926/"><img src="/pmc/articles/instance/4303926/bin/srep07983-f1.gif" src-large="/pmc/articles/instance/4303926/bin/srep07983-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4303926/" ref="ordinalpos=4021&amp;ncbi_uid=7121531&amp;link_uid=PMC4303926" image-link="/pmc/articles/PMC4303926/figure/f1/" class="imagepopup">Figure 1. Activation of SHH <span class="highlight" style="background-color:">signaling</span> shortens QT interval in healthy animals..  From: Activation of Sonic hedgehog <span class="highlight" style="background-color:">signaling</span> in ventricular cardiomyocytes exerts cardioprotection against ischemia reperfusion injuries. </a></p><div class="supp"><p class="details"><div>(A) ECGs were acquired over 6hours after different treatments (<i>i.p.-</i>injected), in control animals. Representative ECGs recorded in control, N-SHH- and MPs<sup>SHH+</sup>-treated animals indicate a shortening of QT interval after SHH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. (B) Mean QTc values recorded before (t = 0, left column) and 6hours after injection (t = 6, right column) of the vehicle (control, n = 5), N-SHH (n = 8), N-SHH+Hexamethonium (N-SHH+Hex, n = 6), MPs<sup>SHH+</sup> (n = 7), N-SHH and cyclopamine (N-SHH+Ccl, n = 6) and cyclopamine alone (Control + Ccl, n = 6). (* p &lt; 0.05, QTc at 6h <i>vs</i> t = 0).</div></p></div><div class="aux"><div class="resc">Ludovit Paulis, et al. Sci Rep. 2015;5:7983.</div><p class="links"><a class="dblinks" href="/pubmed/25613906" ref="ordinalpos=4021&amp;ncbi_uid=7121531&amp;link_uid=25613906">Citation</a><a class="dblinks" href="/pmc/articles/PMC4303926/" ref="ordinalpos=4021&amp;ncbi_uid=7121531&amp;link_uid=PMC4303926">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4022.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4342183/figure/f8-dddt-9-1103/" ref="ordinalpos=4022&amp;ncbi_uid=7224299&amp;link_uid=PMC4342183" href="/pmc/articles/PMC4342183/"><img src="/pmc/articles/instance/4342183/bin/dddt-9-1103Fig8.gif" src-large="/pmc/articles/instance/4342183/bin/dddt-9-1103Fig8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4342183/" ref="ordinalpos=4022&amp;ncbi_uid=7224299&amp;link_uid=PMC4342183" image-link="/pmc/articles/PMC4342183/figure/f8-dddt-9-1103/" class="imagepopup">Figure 8.  From: Hsa-microRNA-181a is a regulator of a number of cancer genes and a biomarker for endometrial carcinoma in patients: a bioinformatic and clinical study and the therapeutic implication. </a></p><div class="supp"><p class="details"><div>VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the target list of hsa-miR-181a-5p based on miRTarBase 4.0.<br /><b>Notes:</b> hsa-miR-181a-5p can regulate the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. VEGF is an important <span class="highlight" style="background-color:">signaling</span> protein involved in both vasculogenesis and angiogenesis. All members of the VEGF family stimulate cellular responses by binding to tyrosine kinase receptors (the VEGFRs) on the cell surface, causing them to dimerize and become activated through transphosphorylation. This triggers a <span class="highlight" style="background-color:">signaling</span> cascade that activates several <span class="highlight" style="background-color:">signaling</span> pathways, such as PI3K/Akt, Erk1/2, Smad, and Notch, and results in endothelial cell proliferation and migration. A number of studies have shown that VEGF and its associated proteins are aberrant in EC.117122 These proteins represent useful targets in the treatment of EC.<br /><b>Abbreviations:</b> EC, endometrial cancer; VEGF, vascular endothelial growth factor.</div></p></div><div class="aux"><div class="resc">Shuming He, et al. Drug Des Devel Ther. 2015;9:1103-1175.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4022&amp;ncbi_uid=7224299&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4342183/" ref="ordinalpos=4022&amp;ncbi_uid=7224299&amp;link_uid=PMC4342183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4023.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1986796/figure/F8/" ref="ordinalpos=4023&amp;ncbi_uid=1037691&amp;link_uid=PMC1986796" href="/pmc/articles/PMC1986796/"><img src="/pmc/articles/instance/1986796/bin/nihms27267f8.gif" src-large="/pmc/articles/instance/1986796/bin/nihms27267f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1986796/" ref="ordinalpos=4023&amp;ncbi_uid=1037691&amp;link_uid=PMC1986796" image-link="/pmc/articles/PMC1986796/figure/F8/" class="imagepopup">Figure 8. From: NO Mobilizes Intracellular Zn2+ via cGMP/PKG <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Prevents Mitochondrial Oxidant Damage in Cardiomyocytes. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> mechanisms whereby NO mobilizes intracellular zinc and prevents the mitochondrial oxidant damage.</div></p></div><div class="aux"><div class="resc">Youngho Jang, et al. Cardiovasc Res. ;75(2):426-433.</div><p class="links"><a class="dblinks" href="/pubmed/17570352" ref="ordinalpos=4023&amp;ncbi_uid=1037691&amp;link_uid=17570352">Citation</a><a class="dblinks" href="/pmc/articles/PMC1986796/" ref="ordinalpos=4023&amp;ncbi_uid=1037691&amp;link_uid=PMC1986796">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4024.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3563200/figure/fig03/" ref="ordinalpos=4024&amp;ncbi_uid=4616857&amp;link_uid=PMC3563200" href="/pmc/articles/PMC3563200/"><img src="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f3.gif" src-large="/pmc/articles/instance/3563200/bin/ijcep0006-0431-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3563200/" ref="ordinalpos=4024&amp;ncbi_uid=4616857&amp;link_uid=PMC3563200" image-link="/pmc/articles/PMC3563200/figure/fig03/" class="imagepopup">Figure 3.  From: Indoleamine 2,3-dioxygenase-1 (IDO1) enhances survival and invasiveness of endometrial stromal cells via the activation of JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> responsible for enhancement of the ESCs viability, proliferation and invasion by IDO1. IDO1-overexpressing and IDO1-deficiency ESCs were treated with JNK inhibitor (SP600125, 20 M) or vehicle control (1 DMSO) for 24 h. Thereafter, MTT assay (A), apoptosis assay (B), BrdU assay (C) and Matrigel invasion assay (D) were applied to analyze the viability, apoptosis, proliferation and invasion of transfected ESCs, respectively. The columns represent the mean  SD calculus of 8 different experiments. *P&lt;.05 vs. vector control transfected ESCs without SP600125; **P&lt;.01 vs. vector control transfected ESCs without SP600125; #P&lt;.05, IDO1-overexpressing ESCs with SP600125 vs. IDO1-overexpressing ESCs without SP600125; ##P&lt;.01, IDO1-overexpressing ESCs with SP600125 vs. IDO1-overexpressing ESCs without SP600125.</div></p></div><div class="aux"><div class="resc">Jie Mei, et al. Int J Clin Exp Pathol. 2013;6(3):431-444.</div><p class="links"><a class="dblinks" href="/pubmed/23411497" ref="ordinalpos=4024&amp;ncbi_uid=4616857&amp;link_uid=23411497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3563200/" ref="ordinalpos=4024&amp;ncbi_uid=4616857&amp;link_uid=PMC3563200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4025.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3740866/figure/fig06/" ref="ordinalpos=4025&amp;ncbi_uid=5795879&amp;link_uid=PMC3740866" href="/pmc/articles/PMC3740866/"><img src="/pmc/articles/instance/3740866/bin/pnas.1304250110fig06.gif" src-large="/pmc/articles/instance/3740866/bin/pnas.1304250110fig06.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3740866/" ref="ordinalpos=4025&amp;ncbi_uid=5795879&amp;link_uid=PMC3740866" image-link="/pmc/articles/PMC3740866/figure/fig06/" class="imagepopup">Fig. 6. From: Convergence of auxin and gibberellin <span class="highlight" style="background-color:">signaling</span> on the regulation of the GATA transcription factors GNC and GNL in Arabidopsis thaliana. </a></p><div class="supp"><p class="details"><div>Model of the control mechanisms acting on the transcriptional control of <i>GNC and GNL</i>. <i>GNC</i> and <i>GNL</i> expression is repressed by ARF2 (red) and ARF7 (blue). In addition, the abundance of their transcripts is under the direct repression control of the GA <span class="highlight" style="background-color:">signaling</span> cascade and the PIF repressors as well as indirect GA control by the stabilization of ARF2 (green). Other genes that are under ARF and PIF control may also be regulated in a manner similar to <i>GNC</i> and <i>GNL</i>. In addition, gene expression events that are under control of the auxin <span class="highlight" style="background-color:">pathway</span> may also be regulated by GA <span class="highlight" style="background-color:">signaling</span> because of the control of ARF2 abundance by GA. The individual pathways are shown in distinct colors.</div></p></div><div class="aux"><div class="resc">Ren Richter, et al. Proc Natl Acad Sci U S A. 2013 August 6;110(32):13192-13197.</div><p class="links"><a class="dblinks" href="/pubmed/23878229" ref="ordinalpos=4025&amp;ncbi_uid=5795879&amp;link_uid=23878229">Citation</a><a class="dblinks" href="/pmc/articles/PMC3740866/" ref="ordinalpos=4025&amp;ncbi_uid=5795879&amp;link_uid=PMC3740866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4026.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1578699/figure/F1/" ref="ordinalpos=4026&amp;ncbi_uid=829167&amp;link_uid=PMC1578699" href="/pmc/articles/PMC1578699/"><img src="/pmc/articles/instance/1578699/bin/2739fig1.gif" src-large="/pmc/articles/instance/1578699/bin/2739fig1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1578699/" ref="ordinalpos=4026&amp;ncbi_uid=829167&amp;link_uid=PMC1578699" image-link="/pmc/articles/PMC1578699/figure/F1/" class="imagepopup">FIGURE 1.  From: Sustained Notch <span class="highlight" style="background-color:">signaling</span> in progenitors is required for sequential emergence of distinct cell lineages during organogenesis. </a></p><div class="supp"><p class="details"><div>In situ analysis of expression pattern of core components of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during pituitary development. Two ligands (<i>Dll1</i>, <i>Jag1</i>), two receptors (<i>Notch2</i>, <i>Notch3</i>), and the downstream target of Notch/Rbp-J <span class="highlight" style="background-color:">signaling</span> (<i>Hes1</i>) are expressed in the oral ectoderm and the Rathke's pouch (RP) by E12.5. At E13.5, <i>Dll1</i>, <i>Jag1</i>, <i>Notch2</i>, <i>Notch3</i>, and <i>Hes1</i> expression are down-regulated in the anterior pituitary (AP) and are further confined to the perilumenal cells. At E14.5, <i>Jag1</i> is expressed in the mesenchymal cells surrounding and within the pituitary gland. (IL) Intermediate lobe.</div></p></div><div class="aux"><div class="resc">Xiaoyan Zhu, et al. Genes Dev. 2006 October 1;20(19):2739-2753.</div><p class="links"><a class="dblinks" href="/pubmed/17015435" ref="ordinalpos=4026&amp;ncbi_uid=829167&amp;link_uid=17015435">Citation</a><a class="dblinks" href="/pmc/articles/PMC1578699/" ref="ordinalpos=4026&amp;ncbi_uid=829167&amp;link_uid=PMC1578699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4027.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4053348/figure/F1/" ref="ordinalpos=4027&amp;ncbi_uid=6236755&amp;link_uid=PMC4053348" href="/pmc/articles/PMC4053348/"><img src="/pmc/articles/instance/4053348/bin/nihms-426635-f0001.gif" src-large="/pmc/articles/instance/4053348/bin/nihms-426635-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4053348/" ref="ordinalpos=4027&amp;ncbi_uid=6236755&amp;link_uid=PMC4053348" image-link="/pmc/articles/PMC4053348/figure/F1/" class="imagepopup">Figure 1.  From: A New "Brew" of MALT1 Inhibitors. </a></p><div class="supp"><p class="details"><div>Role of MALT1 in <span class="highlight" style="background-color:">signaling</span> to NF-B in ABC DLBCL. Shown are two prominent pathways leading to NF-B activation in ABC DLBCL: chronic active BCR <span class="highlight" style="background-color:">signaling</span> and constitutive MYD88 <span class="highlight" style="background-color:">signaling</span>. MALT1 plays a key role in the BCR <span class="highlight" style="background-color:">pathway</span> in two ways. First, as a component of the CBM complex with CARD11 and BCL10, MALT1 helps recruit and activate IB kinase (IKK). Second, MALT1 protease activity potentiates NF-B <span class="highlight" style="background-color:">signaling</span> by cleaving and inactivating two negative regulators of IKK, A20 (TNFAIP3) and CYLD. Shown are recurrent mutations in ABC DLBCL tumors that cause or intensify NF-B activity.</div></p></div><div class="aux"><div class="resc">Ryan M. Young, et al. Cancer Cell. ;22(6):706-707.</div><p class="links"><a class="dblinks" href="/pubmed/23238011" ref="ordinalpos=4027&amp;ncbi_uid=6236755&amp;link_uid=23238011">Citation</a><a class="dblinks" href="/pmc/articles/PMC4053348/" ref="ordinalpos=4027&amp;ncbi_uid=6236755&amp;link_uid=PMC4053348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4028.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3294445/figure/F8/" ref="ordinalpos=4028&amp;ncbi_uid=4207704&amp;link_uid=PMC3294445" href="/pmc/articles/PMC3294445/"><img src="/pmc/articles/instance/3294445/bin/zek9990938620008.gif" src-large="/pmc/articles/instance/3294445/bin/zek9990938620008.jpg" alt="Fig 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3294445/" ref="ordinalpos=4028&amp;ncbi_uid=4207704&amp;link_uid=PMC3294445" image-link="/pmc/articles/PMC3294445/figure/F8/" class="imagepopup">Fig 8.  From: Vacuolar H+-ATPase Works in Parallel with the HOG <span class="highlight" style="background-color:">Pathway</span> To Adapt Saccharomyces cerevisiae Cells to Osmotic Stress. </a></p><div class="supp"><p class="details"><div>V-ATPase is important in salt detoxification during hyperosmotic stress. The HOG MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the V-ATPase act in parallel pathways to mitigate osmotic shock during salt stress. Activation of the HOG <span class="highlight" style="background-color:">pathway</span> leads to changes in gene expression that promote production of glycerol, an osmoprotectant. The HOG <span class="highlight" style="background-color:">pathway</span> also activates transporters at the plasma membrane, resulting in the extrusion of NaCl into the extracellular space. On the other hand, the V-ATPase becomes activated in response to salt stress, and higher activation of the enzyme is required for the activity of vacuolar cation transporters that recruit excess sodium ions into the vacuolar lumen.</div></p></div><div class="aux"><div class="resc">Sheena Claire Li, et al. Eukaryot Cell. 2012 March;11(3):282-291.</div><p class="links"><a class="dblinks" href="/pubmed/22210831" ref="ordinalpos=4028&amp;ncbi_uid=4207704&amp;link_uid=22210831">Citation</a><a class="dblinks" href="/pmc/articles/PMC3294445/" ref="ordinalpos=4028&amp;ncbi_uid=4207704&amp;link_uid=PMC3294445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4029.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2783859/figure/F3/" ref="ordinalpos=4029&amp;ncbi_uid=2546287&amp;link_uid=PMC2783859" href="/pmc/articles/PMC2783859/"><img src="/pmc/articles/instance/2783859/bin/nihms140228f3a.gif" src-large="/pmc/articles/instance/2783859/bin/nihms140228f3a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2783859/figure/F3/" ref="ordinalpos=4029&amp;ncbi_uid=2546287&amp;link_uid=PMC2783859" href="/pmc/articles/PMC2783859/"><img src="/pmc/articles/instance/2783859/bin/nihms140228f3b.gif" src-large="/pmc/articles/instance/2783859/bin/nihms140228f3b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2783859/" ref="ordinalpos=4029&amp;ncbi_uid=2546287&amp;link_uid=PMC2783859" image-link="/pmc/articles/PMC2783859/figure/F3/" class="imagepopup">Figure 3. Regulated genes in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> following carcinogen treatment at 10 weeks.  From: Identification of Actively Translated mRNA Transcripts in a Rat Model of Early Stage Colon Carcinogenesis. </a></p><div class="supp"><p class="details"><div>See legend to Figure 2 for details. Gene expression in AOM and saline treated rats was compared using GenMAPP <span class="highlight" style="background-color:">pathway</span> analysis. Colors indicate significantly (p &lt; 0.05) up-regulated genes. Numbers show the fold changes. <b>(A)</b> Polysome RNA fraction, <b>(B)</b> Total RNA fraction.</div></p></div><div class="aux"><div class="resc">Laurie A. Davidson, et al. Cancer Prev Res (Phila). ;2(11):984-994.</div><p class="links"><a class="dblinks" href="/pubmed/19843688" ref="ordinalpos=4029&amp;ncbi_uid=2546287&amp;link_uid=19843688">Citation</a><a class="dblinks" href="/pmc/articles/PMC2783859/" ref="ordinalpos=4029&amp;ncbi_uid=2546287&amp;link_uid=PMC2783859">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4030.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382590/figure/pone-0039691-g004/" ref="ordinalpos=4030&amp;ncbi_uid=4103026&amp;link_uid=PMC3382590" href="/pmc/articles/PMC3382590/"><img src="/pmc/articles/instance/3382590/bin/pone.0039691.g004.gif" src-large="/pmc/articles/instance/3382590/bin/pone.0039691.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382590/" ref="ordinalpos=4030&amp;ncbi_uid=4103026&amp;link_uid=PMC3382590" image-link="/pmc/articles/PMC3382590/figure/pone-0039691-g004/" class="imagepopup">Figure 4. Hairless mutant epidermis shows constitutive activation of the NFB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Loss of Hairless Confers Susceptibility to UVB-Induced Tumorigenesis via Disruption of NF-kappaB <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A) Western Blot and ImageJ densitometry analysis using anti-p65 antibody on isolated <i>Hr<sup>/</sup></i> epidermal extracts after 1, 8 and 24 hrs post UVB (180 mJ/cm<sup>2</sup>) irradiation. B) Immunohistochemistry using p-IB antibody to determine the extent of NFB <span class="highlight" style="background-color:">signaling</span> activity in the epidermis of WT (<i>Hr<sup>+/+</sup></i>) and <i>Hr<sup>/</sup></i> animals before and after a single, acute UVB (180 mJ/cm<sup>2</sup>) irradiation. IB is increased constitutively and after irradiation throughout the epidermal layer. C) Immunohistochemistry using p-IB antibody to determine the extent of NFB <span class="highlight" style="background-color:">signaling</span> activity in the epidermis of WT (<i>Hr<sup>+/+</sup></i>) and <i>Hr<sup>/</sup></i> animals at the indicated time points during chronic UVB irradiation (180 mJ/cm<sup>2</sup> UVB three times a week for a total of 50 weeks). IB is increased constitutively and after irradiation throughout the epidermal layer.</div></p></div><div class="aux"><div class="resc">Hyunmi Kim, et al. PLoS One. 2012;7(6):e39691.</div><p class="links"><a class="dblinks" href="/pubmed/22761871" ref="ordinalpos=4030&amp;ncbi_uid=4103026&amp;link_uid=22761871">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382590/" ref="ordinalpos=4030&amp;ncbi_uid=4103026&amp;link_uid=PMC3382590">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4031.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2660608/figure/F6/" ref="ordinalpos=4031&amp;ncbi_uid=1736780&amp;link_uid=PMC2660608" href="/pmc/articles/PMC2660608/"><img src="/pmc/articles/instance/2660608/bin/nihms88012f6.gif" src-large="/pmc/articles/instance/2660608/bin/nihms88012f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2660608/" ref="ordinalpos=4031&amp;ncbi_uid=1736780&amp;link_uid=PMC2660608" image-link="/pmc/articles/PMC2660608/figure/F6/" class="imagepopup">Figure 6. <span class="highlight" style="background-color:">Signaling</span> pathways mediating GDNF hyperalgesia.  From: GDNF hyperalgesia is mediated by PLC?, MAPK/ERK, PI3K, CDK5 and Src family kinase <span class="highlight" style="background-color:">signaling</span> and dependent on the IB4-binding protein versican. </a></p><div class="supp"><p class="details"><div>GDNF-induced mechanical hyperalgesia is mediated by five different <span class="highlight" style="background-color:">signaling</span> pathways. Rats received a single intradermal injection of 10 ng GDNF and 1 g inhibitor for each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Readings of the mechanical nociceptive threshold were taken 30 min post-injection. As shown inhibitors of PLC (U73122, <i>n</i> = 6), PI3K (Wortmannin and LY294002, both n = 4), MEK 1/2 (U0126, n = 6), SFK (PP2 and SU6656, both n = 4) and CDK5 (Roscovitine, n = 6) all significantly attenuated GDNF hyperalgesia (all p&lt;0.001) whereas inhibitors to PKA (PKA inhibitor 622 amide, n=6) and PKC (Bisindolylmaleimide 1 hydrochloride, n=6) had no significant effect on GDNF hyperalgesia (p=1.000, in both cases). Note that 10% DMSO alone did not change paw-pressure threshold.</div></p></div><div class="aux"><div class="resc">Oliver Bogen, et al. Eur J Neurosci. ;28(1):12-19.</div><p class="links"><a class="dblinks" href="/pubmed/18616564" ref="ordinalpos=4031&amp;ncbi_uid=1736780&amp;link_uid=18616564">Citation</a><a class="dblinks" href="/pmc/articles/PMC2660608/" ref="ordinalpos=4031&amp;ncbi_uid=1736780&amp;link_uid=PMC2660608">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4032.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC506992/figure/f7/" ref="ordinalpos=4032&amp;ncbi_uid=466699&amp;link_uid=PMC506992" href="/pmc/articles/PMC506992/"><img src="/pmc/articles/instance/506992/bin/zmb0170443420007.gif" src-large="/pmc/articles/instance/506992/bin/zmb0170443420007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC506992/" ref="ordinalpos=4032&amp;ncbi_uid=466699&amp;link_uid=PMC506992" image-link="/pmc/articles/PMC506992/figure/f7/" class="imagepopup">FIG. 7. From: MUC20 Suppresses the Hepatocyte Growth Factor-Induced Grb2-Ras <span class="highlight" style="background-color:">Pathway</span> by Binding to a Multifunctional Docking Site of Met. </a></p><div class="supp"><p class="details"><div>Model of MUC20 suppression. (A) A MUC20 monomer has less ability to associate with Met. In this condition, Grb2 is recruited to HGF-activated Met, followed by activation of <span class="highlight" style="background-color:">signaling</span> cascades leading to proliferation. (B) MUC20 oligomerization caused by either an overproduction of MUC20 or an unknown endogenous factor(s) leads to the association with Met, blocking Grb2 recruitment to Met and suppressing the Grb2-Ras <span class="highlight" style="background-color:">pathway</span>, but does not affect Gab1 recruitment. The oligomerizing domain in MUC20 is referred to as the self-oligomerizing domain (SOD). PLC, phospholipase C-.</div></p></div><div class="aux"><div class="resc">Toshio Higuchi, et al. Mol Cell Biol. 2004 September;24(17):7456-7468.</div><p class="links"><a class="dblinks" href="/pubmed/15314156" ref="ordinalpos=4032&amp;ncbi_uid=466699&amp;link_uid=15314156">Citation</a><a class="dblinks" href="/pmc/articles/PMC506992/" ref="ordinalpos=4032&amp;ncbi_uid=466699&amp;link_uid=PMC506992">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4033.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3116598/figure/F1/" ref="ordinalpos=4033&amp;ncbi_uid=3023251&amp;link_uid=PMC3116598" href="/pmc/articles/PMC3116598/"><img src="/pmc/articles/instance/3116598/bin/sgtp0102_0124_fig001.gif" src-large="/pmc/articles/instance/3116598/bin/sgtp0102_0124_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3116598/" ref="ordinalpos=4033&amp;ncbi_uid=3023251&amp;link_uid=PMC3116598" image-link="/pmc/articles/PMC3116598/figure/F1/" class="imagepopup">Figure 1.  From: p21-activated kinases in Erbb2-positive breast cancer. </a></p><div class="supp"><p class="details"><div>Dimerization of ErbB2 receptors leads to phosphorylation and activation of several intracellular catalytic substrates, including the Ras/Raf/MEK/Erk, PI3K/Akt and other important <span class="highlight" style="background-color:">signaling</span> pathways that regulate apoptosis, protein synthesis and cellular proliferation. Our experimental results summarized in this Extra View, demonstrate that ErbB2 <span class="highlight" style="background-color:">signaling</span> activates a Rac-Pak <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that contributes to ErbB2 mediated transformation through the Erk and Akt pathways.</div></p></div><div class="aux"><div class="resc">Luis E Arias-Romero, et al. Small GTPases. 2010 Sep-Oct;1(2):124-128.</div><p class="links"><a class="dblinks" href="/pubmed/21686266" ref="ordinalpos=4033&amp;ncbi_uid=3023251&amp;link_uid=21686266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3116598/" ref="ordinalpos=4033&amp;ncbi_uid=3023251&amp;link_uid=PMC3116598">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4034.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3549968/figure/fig9/" ref="ordinalpos=4034&amp;ncbi_uid=5114089&amp;link_uid=PMC3549968" href="/pmc/articles/PMC3549968/"><img src="/pmc/articles/instance/3549968/bin/JCB_201207036_Fig9.gif" src-large="/pmc/articles/instance/3549968/bin/JCB_201207036_Fig9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3549968/" ref="ordinalpos=4034&amp;ncbi_uid=5114089&amp;link_uid=PMC3549968" image-link="/pmc/articles/PMC3549968/figure/fig9/" class="imagepopup">Figure 9. From: Perlecan regulates bidirectional Wnt <span class="highlight" style="background-color:">signaling</span> at the Drosophila neuromuscular junction. </a></p><div class="supp"><p class="details"><div><b>Proposed model for Trol function in Wg <span class="highlight" style="background-color:">pathway</span> at pre- and postsynaptic cells.</b> In the wild type (left), presynaptically released Wg sends its signal to both presynaptic and postsynaptic cells, which is required for proliferation of boutons and formation of postsynaptic SSR, respectively. Trol binds and sequesters Wg on the surface of SSR. In <i>trol</i> mutants (right), stability of Wg at the postsynaptic site is decreased, and therefore, Wg might preferably bind its receptor DFz2 on the presynaptic membrane, inducing overproliferation of boutons. Thus, Trol regulates bidirectional activity of Wg <span class="highlight" style="background-color:">signaling</span> at the NMJ.</div></p></div><div class="aux"><div class="resc">Keisuke Kamimura, et al. J Cell Biol. 2013 January 21;200(2):219-233.</div><p class="links"><a class="dblinks" href="/pubmed/23319599" ref="ordinalpos=4034&amp;ncbi_uid=5114089&amp;link_uid=23319599">Citation</a><a class="dblinks" href="/pmc/articles/PMC3549968/" ref="ordinalpos=4034&amp;ncbi_uid=5114089&amp;link_uid=PMC3549968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4035.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2671893/figure/F11/" ref="ordinalpos=4035&amp;ncbi_uid=1730617&amp;link_uid=PMC2671893" href="/pmc/articles/PMC2671893/"><img src="/pmc/articles/instance/2671893/bin/zee0050947400011.gif" src-large="/pmc/articles/instance/2671893/bin/zee0050947400011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2671893/" ref="ordinalpos=4035&amp;ncbi_uid=1730617&amp;link_uid=PMC2671893" image-link="/pmc/articles/PMC2671893/figure/F11/" class="imagepopup">Figure 11.  From: Human Uterine Smooth Muscle and Leiomyoma Cells Differ in Their Rapid 17?-Estradiol <span class="highlight" style="background-color:">Signaling</span>: Implications for Proliferation. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">pathway</span>. In this model, E2 binds to the ER, resulting in PKC activation, which in turn promotes ERK1/2 activation and proliferation in leiomyoma cells. Likewise, E2 binds to the ER in normal UtSM-ht cells to activate PKC in normal UtSM-ht cells. However, activated PKC promotes cAMP-PKA <span class="highlight" style="background-color:">signaling</span>, which in turn inhibits ERK1/2 activation, E2-dependent genomic response, and proliferation in normal uterine smooth muscle. RTK, Receptor: tyrosine kinase; MEK, mitogen-activated/extracellular signal-regulated kinase.</div></p></div><div class="aux"><div class="resc">Erica N. Nierth-Simpson, et al. Endocrinology. 2009 May;150(5):2436-2445.</div><p class="links"><a class="dblinks" href="/pubmed/19179429" ref="ordinalpos=4035&amp;ncbi_uid=1730617&amp;link_uid=19179429">Citation</a><a class="dblinks" href="/pmc/articles/PMC2671893/" ref="ordinalpos=4035&amp;ncbi_uid=1730617&amp;link_uid=PMC2671893">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4036.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2636929/figure/f9/" ref="ordinalpos=4036&amp;ncbi_uid=1738471&amp;link_uid=PMC2636929" href="/pmc/articles/PMC2636929/"><img src="/pmc/articles/instance/2636929/bin/zh30010952680009.gif" src-large="/pmc/articles/instance/2636929/bin/zh30010952680009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2636929/" ref="ordinalpos=4036&amp;ncbi_uid=1738471&amp;link_uid=PMC2636929" image-link="/pmc/articles/PMC2636929/figure/f9/" class="imagepopup">Fig. 9. From: GABA <span class="highlight" style="background-color:">signaling</span> in the nucleus tractus solitarius sets the level of activity in dorsal motor nucleus of the vagus cholinergic neurons in the vagovagal circuit. </a></p><div class="supp"><p class="details"><div>Simplified schematic representation of proposed circuitry for local intra-nucleus tractus solitarius (NTS) GABAergic <span class="highlight" style="background-color:">signaling</span> and the DMV efferent <span class="highlight" style="background-color:">pathway</span> controlling gastric motility. Black circles are GABA interneurons, gray circles are GABA projection neurons, and white circles are DMV preganglionic neurons. Oval is the nodose sensory ganglion. Plus signs in the NTS represent excitatory (ionotropic glutamate receptor-mediated) <span class="highlight" style="background-color:">signaling</span>. Minus signs represent inhibitory (GABA<sub>A</sub> receptor-mediated) <span class="highlight" style="background-color:">signaling</span>. The question mark placed in the NTS area reflects the lack of knowledge on definite organization of interneurons between the vagal afferent terminals and the NTS projection neurons.</div></p></div><div class="aux"><div class="resc">Melissa A. Herman, et al. Am J Physiol Gastrointest Liver Physiol. 2009 January;296(1):G101-G111.</div><p class="links"><a class="dblinks" href="/pubmed/19008339" ref="ordinalpos=4036&amp;ncbi_uid=1738471&amp;link_uid=19008339">Citation</a><a class="dblinks" href="/pmc/articles/PMC2636929/" ref="ordinalpos=4036&amp;ncbi_uid=1738471&amp;link_uid=PMC2636929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4037.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2875095/figure/F1/" ref="ordinalpos=4037&amp;ncbi_uid=2958973&amp;link_uid=PMC2875095" href="/pmc/articles/PMC2875095/"><img src="/pmc/articles/instance/2875095/bin/zh89991052540001.gif" src-large="/pmc/articles/instance/2875095/bin/zh89991052540001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2875095/" ref="ordinalpos=4037&amp;ncbi_uid=2958973&amp;link_uid=PMC2875095" image-link="/pmc/articles/PMC2875095/figure/F1/" class="imagepopup">Figure 1.  From: Kindlins in FERM adhesion. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from an agonist stimulation to integrin activation</b>. CalDAG GEFI, a GDP/GTP exchange factor specific for Rap small G proteins, and protein kinase C (PKC) activated by a receptor-dependent stimulation of phospholipase C. As a result, Rap1*GTP interaction with RIAM protein (not shown) stimulates talin-1 function in the inside-out <span class="highlight" style="background-color:">pathway</span>. Protein kinase C has been implicated in the <span class="highlight" style="background-color:">signaling</span> as well, although the substrates important for the <span class="highlight" style="background-color:">signaling</span> are currently unknown. Talin-1 and Kindlin FERM domain are shown as spheres with 3 subdomains; the intersecting PH domain in Kindlin is shown as the dark-colored segment.</div></p></div><div class="aux"><div class="resc">Nikolay L. Malinin, et al. Blood. 2010 May 20;115(20):4011-4017.</div><p class="links"><a class="dblinks" href="/pubmed/20228270" ref="ordinalpos=4037&amp;ncbi_uid=2958973&amp;link_uid=20228270">Citation</a><a class="dblinks" href="/pmc/articles/PMC2875095/" ref="ordinalpos=4037&amp;ncbi_uid=2958973&amp;link_uid=PMC2875095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4038.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2899688/figure/F9/" ref="ordinalpos=4038&amp;ncbi_uid=2322363&amp;link_uid=PMC2899688" href="/pmc/articles/PMC2899688/"><img src="/pmc/articles/instance/2899688/bin/nihms214620f9.gif" src-large="/pmc/articles/instance/2899688/bin/nihms214620f9.jpg" alt="FIG. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2899688/" ref="ordinalpos=4038&amp;ncbi_uid=2322363&amp;link_uid=PMC2899688" image-link="/pmc/articles/PMC2899688/figure/F9/" class="imagepopup">FIG. 9.  From: Structural Basis of Protein Kinase C Isoform Function. </a></p><div class="supp"><p class="details"><div>Evolving concepts of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. <i>A:</i> the original concepts of linear signal transduction pathways held that stimuli evoke responses via the actions of transducers (such as G proteins or PKC isoforms). <i>B:</i> models of signal transduction were first revised to allow for the presence of multiple stimuli that can converge on a single <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and alter the amplitude of a <span class="highlight" style="background-color:">signaling</span> response. This could reflect the activation of different pools of a single enzyme. Alternatively, amplitude control might be due to the differential stimulatory properties of full or partial agonists. <i>C:</i> more current models of <span class="highlight" style="background-color:">signaling</span> transduction allow for a high level of signal integration by proteins (such as PKC<i></i>) that are regulated through conformational changes, translocation events, and various posttranslational modifications (phosphorylation, oxidation, tyrosine nitration, etc.) that can underlie stimulus-specific <span class="highlight" style="background-color:">signaling</span> responses.</div></p></div><div class="aux"><div class="resc">SUSAN F. STEINBERG. Physiol Rev. ;88(4):1341-1378.</div><p class="links"><a class="dblinks" href="/pubmed/18923184" ref="ordinalpos=4038&amp;ncbi_uid=2322363&amp;link_uid=18923184">Citation</a><a class="dblinks" href="/pmc/articles/PMC2899688/" ref="ordinalpos=4038&amp;ncbi_uid=2322363&amp;link_uid=PMC2899688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4039.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872423/figure/fig07/" ref="ordinalpos=4039&amp;ncbi_uid=2512596&amp;link_uid=PMC2872423" href="/pmc/articles/PMC2872423/"><img src="/pmc/articles/instance/2872423/bin/pnas.0910060107fig07.gif" src-large="/pmc/articles/instance/2872423/bin/pnas.0910060107fig07.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872423/" ref="ordinalpos=4039&amp;ncbi_uid=2512596&amp;link_uid=PMC2872423" image-link="/pmc/articles/PMC2872423/figure/fig07/" class="imagepopup">Fig. 7. From: Notch-activated <span class="highlight" style="background-color:">signaling</span> cascade interacts with mitochondrial remodeling proteins to regulate cell survival. </a></p><div class="supp"><p class="details"><div>Schematic summarizing key features of the NIC-activated antiapoptotic cascade. Cytoplasmic NIC activity, inhibited apoptosis coordinated by mitochondria in cell lines (triggered using Bax), and primary T cells. The mitochondrial remodeling proteinsMitofusinsare identified in this work as key intermediates in the antiapoptotic cascade, regulating mitochondrial function and connectivity downstream of the NIC-Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Despite evidence of mitochondrial pools of NIC and Akt, we do not detect stable, direct associations between NIC-MFN-AKT suggesting the possibility of dynamic interactions among these molecules regulating <span class="highlight" style="background-color:">signaling</span> outputs.</div></p></div><div class="aux"><div class="resc">Lakshmi R. Perumalsamy, et al. Proc Natl Acad Sci U S A. 2010 April 13;107(15):6882-6887.</div><p class="links"><a class="dblinks" href="/pubmed/20339081" ref="ordinalpos=4039&amp;ncbi_uid=2512596&amp;link_uid=20339081">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872423/" ref="ordinalpos=4039&amp;ncbi_uid=2512596&amp;link_uid=PMC2872423">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4040.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3443857/figure/F1/" ref="ordinalpos=4040&amp;ncbi_uid=5016614&amp;link_uid=PMC3443857" href="/pmc/articles/PMC3443857/"><img src="/pmc/articles/instance/3443857/bin/nihms339414f1.gif" src-large="/pmc/articles/instance/3443857/bin/nihms339414f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3443857/" ref="ordinalpos=4040&amp;ncbi_uid=5016614&amp;link_uid=PMC3443857" image-link="/pmc/articles/PMC3443857/figure/F1/" class="imagepopup">Fig. 1.  From: The progesterone receptor regulates implantation, decidualization, and glandular development via a complex paracrine <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div>The progesterone regulated paracrine <span class="highlight" style="background-color:">signaling</span> model depicted within the uterine compartments at the time of embryo implantation. <i>Ihh</i> is a progesterone target activated within the epithelium which signals downstream to COUP-TFII in the stroma establishing the <i>Ihh</i>COUP-TFII axis within the dual uterine compartments. COUP-TFII has demonstrated to be a critical effector of decidualization and the maintenance of pregnancy during post-implantation through the activation of <i>Bmp2</i> and <i>Wnt4</i>. COUP-TFII may also play a role in the activation of <i>Hand2</i> in the stroma leading to the inhibition of the FGF <span class="highlight" style="background-color:">pathway</span>, a <span class="highlight" style="background-color:">pathway</span> known to be involved in the promotion of epithelial proliferation by estrogen <span class="highlight" style="background-color:">signaling</span>. The arrows represent activation. The bars denote inhibition. The question marks next to arrows represent an activation that is lacking substantial evidence.</div></p></div><div class="aux"><div class="resc">Margeaux Wetendorf, et al. Mol Cell Endocrinol. 2012 June 24;357(1-2):108-118.</div><p class="links"><a class="dblinks" href="/pubmed/22115959" ref="ordinalpos=4040&amp;ncbi_uid=5016614&amp;link_uid=22115959">Citation</a><a class="dblinks" href="/pmc/articles/PMC3443857/" ref="ordinalpos=4040&amp;ncbi_uid=5016614&amp;link_uid=PMC3443857">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4041.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2889938/figure/F5/" ref="ordinalpos=4041&amp;ncbi_uid=2342702&amp;link_uid=PMC2889938" href="/pmc/articles/PMC2889938/"><img src="/pmc/articles/instance/2889938/bin/1471-2105-11-308-5.gif" src-large="/pmc/articles/instance/2889938/bin/1471-2105-11-308-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2889938/" ref="ordinalpos=4041&amp;ncbi_uid=2342702&amp;link_uid=PMC2889938" image-link="/pmc/articles/PMC2889938/figure/F5/" class="imagepopup">Figure 5.  From: Computational modeling with forward and reverse engineering links <span class="highlight" style="background-color:">signaling</span> network and genomic regulatory responses: NF-?B <span class="highlight" style="background-color:">signaling</span>-induced gene expression responses in inflammation. </a></p><div class="supp"><p class="details"><div><b>Linking the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and gene responses by mapping the pattern of NF-B profiles</b>. By simulating the NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, one of the NF-B profiles (pink data points) that is induced by a specified IKK profile (green line) matches the pattern of the NF-B profile (purple line) reconstructed from the gene expression data produced by NCA among the three different doses of LPS stimulus. In each conditions of LPS stimulus strength, the pattern of the two NF-B profiles can be mapped at the time points of 0, 0.5, 2, 4, 6, and 12 hours with a Pearson correlation of ~0.99.</div></p></div><div class="aux"><div class="resc">Shih Chi Peng, et al. BMC Bioinformatics. 2010;11:308-308.</div><p class="links"><a class="dblinks" href="/pubmed/20529327" ref="ordinalpos=4041&amp;ncbi_uid=2342702&amp;link_uid=20529327">Citation</a><a class="dblinks" href="/pmc/articles/PMC2889938/" ref="ordinalpos=4041&amp;ncbi_uid=2342702&amp;link_uid=PMC2889938">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4042.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091047/figure/fig9/" ref="ordinalpos=4042&amp;ncbi_uid=2963274&amp;link_uid=PMC3091047" href="/pmc/articles/PMC3091047/"><img src="/pmc/articles/instance/3091047/bin/PP_172502_wc_f9.gif" src-large="/pmc/articles/instance/3091047/bin/PP_172502_wc_f9.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091047/" ref="ordinalpos=4042&amp;ncbi_uid=2963274&amp;link_uid=PMC3091047" image-link="/pmc/articles/PMC3091047/figure/fig9/" class="imagepopup">Figure 9. From: Auxin and Ethylene Induce Flavonol Accumulation through Distinct Transcriptional Networks1[C][W][OA]. </a></p><div class="supp"><p class="details"><div>Order of <span class="highlight" style="background-color:">signaling</span> events in hormone-induced flavonol metabolism. A combination of auxin- and ethylene-specific and MYB-dependent transcriptional control may explain the observed data. TIR1- and ARF-dependent auxin <span class="highlight" style="background-color:">signaling</span> activates the auxin-responsive elements of the <span class="highlight" style="background-color:">pathway</span> genes shown, which include <i>F3H</i>. In parallel, MYB12 is activated, leading to further up-regulation of the same <span class="highlight" style="background-color:">pathway</span> genes and PAP1 and TTG1 through their MYB-responsive elements. The line between MYB12 and PAP1 and TTG1 is dotted based on the complexity of interpreting these mutant data, as described in the text. EIN2- and ETR1-dependent ethylene <span class="highlight" style="background-color:">signaling</span> also increases the expression of a subset of <span class="highlight" style="background-color:">pathway</span> genes, excluding <i>F3H</i>, through the activation of ethylene-responsive elements. This auxin-specific regulation of <i>F3H</i> may explain the observed differences in the Q/K ratio after treatment with IAA and ACC. [See online article for color version of this figure.]</div></p></div><div class="aux"><div class="resc">Daniel R. Lewis, et al. Plant Physiol. 2011 May;156(1):144-164.</div><p class="links"><a class="dblinks" href="/pubmed/21427279" ref="ordinalpos=4042&amp;ncbi_uid=2963274&amp;link_uid=21427279">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091047/" ref="ordinalpos=4042&amp;ncbi_uid=2963274&amp;link_uid=PMC3091047">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4043.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4243489/figure/F2/" ref="ordinalpos=4043&amp;ncbi_uid=6948078&amp;link_uid=PMC4243489" href="/pmc/articles/PMC4243489/"><img src="/pmc/articles/instance/4243489/bin/fpls-05-00650-g002.gif" src-large="/pmc/articles/instance/4243489/bin/fpls-05-00650-g002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4243489/" ref="ordinalpos=4043&amp;ncbi_uid=6948078&amp;link_uid=PMC4243489" image-link="/pmc/articles/PMC4243489/figure/F2/" class="imagepopup">FIGURE 2.  From: The roles of ethylene and transcription factors in the regulation of onset of leaf senescence. </a></p><div class="supp"><p class="details"><div><b>Scheme of the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to the onset of leaf senescence. (A)</b> In mature leaves, the detection of ethylene activates the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to SAG induction and leaf senescence. <b>(B)</b> In young and mature leaves, the receptors constitutively repress the downstream <span class="highlight" style="background-color:">signaling</span> in the absence of ethylene. <b>(C)</b> In young leaves, the detection of ethylene activates the downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, but does not itself induce leaf senescence. Note that EIN2 and EIN3 are active and induce some ethylene responses, but not leaf senescence by an uncharacterized mechanism, in which some regulators of leaf development are likely involved. Arrows and bars at the end of each line show positive and negative regulations, respectively. Solid lines and black gene names designate the active form, while dotted lines and gray gene names indicate the inactive form. Several transcription factors (TFs) and signals such as jasmonic acid are not drawn in this scheme owing to space limitations. A detailed description on the scheme is presented in the main text.</div></p></div><div class="aux"><div class="resc">Tomotsugu Koyama. Front Plant Sci. 2014;5:650.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4043&amp;ncbi_uid=6948078&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4243489/" ref="ordinalpos=4043&amp;ncbi_uid=6948078&amp;link_uid=PMC4243489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4044.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3652064/figure/F7/" ref="ordinalpos=4044&amp;ncbi_uid=6139073&amp;link_uid=PMC3652064" href="/pmc/articles/PMC3652064/"><img src="/pmc/articles/instance/3652064/bin/zh40101307420007.gif" src-large="/pmc/articles/instance/3652064/bin/zh40101307420007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3652064/" ref="ordinalpos=4044&amp;ncbi_uid=6139073&amp;link_uid=PMC3652064" image-link="/pmc/articles/PMC3652064/figure/F7/" class="imagepopup">Fig. 7. From: Physiologic and molecular characterization of a murine model of right ventricular volume overload. </a></p><div class="supp"><p class="details"><div>Heatmap and <span class="highlight" style="background-color:">pathway</span> analysis. <i>A</i>: heatmap at 1 and 3 mo comparing PI with sham-operated controls. 372 genes were downregulated by &gt;2-fold at 1 mo. Corrected <i>P</i> value &lt;0.05. At 3 mo 434 genes were upregulated and 307 were downregulated by &gt;2-fold. Corrected <i>P</i> value &lt;0.05. <i>B</i>: <span class="highlight" style="background-color:">pathway</span> analysis at 3 mo after PI demonstrated key nodal points highlighted here in red including upregulation in transforming growth factor-<sub>1</sub> (TGF-<sub>1</sub>), thrombospondin, and TNF <span class="highlight" style="background-color:">signaling</span> and downregulation in NF-B1 <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Sushma Reddy, et al. Am J Physiol Heart Circ Physiol. 2013 May 15;304(10):H1314-H1327.</div><p class="links"><a class="dblinks" href="/pubmed/23504182" ref="ordinalpos=4044&amp;ncbi_uid=6139073&amp;link_uid=23504182">Citation</a><a class="dblinks" href="/pmc/articles/PMC3652064/" ref="ordinalpos=4044&amp;ncbi_uid=6139073&amp;link_uid=PMC3652064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4045.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3469060/figure/fig6/" ref="ordinalpos=4045&amp;ncbi_uid=5428119&amp;link_uid=PMC3469060" href="/pmc/articles/PMC3469060/"><img src="/pmc/articles/instance/3469060/bin/cdd201260f6.gif" src-large="/pmc/articles/instance/3469060/bin/cdd201260f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3469060/" ref="ordinalpos=4045&amp;ncbi_uid=5428119&amp;link_uid=PMC3469060" image-link="/pmc/articles/PMC3469060/figure/fig6/" class="imagepopup">Figure 6.  From: CD147 induces UPR to inhibit apoptosis and chemosensitivity by increasing the transcription of Bip in hepatocellular carcinoma. </a></p><div class="supp"><p class="details"><div>Putative mechanisms responsible for UPR activation by CD147 in human HCC cells. CD147 is induced during ERS in human HCC cells. By interacting with integrin <i></i>3<i></i>1, CD147 activates the FAKSrc <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which promotes p-TFII-I nuclear translocation and the subsequent transcription of Bip via binding to the ERSE sites. By inhibiting ERS-induced cell apoptosis and decreasing chemosensitivity, this CD147-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may contribute to cell survival after ERS in human HCC cells</div></p></div><div class="aux"><div class="resc">J Tang, et al. Cell Death Differ. 2012 November;19(11):1779-1790.</div><p class="links"><a class="dblinks" href="/pubmed/22595757" ref="ordinalpos=4045&amp;ncbi_uid=5428119&amp;link_uid=22595757">Citation</a><a class="dblinks" href="/pmc/articles/PMC3469060/" ref="ordinalpos=4045&amp;ncbi_uid=5428119&amp;link_uid=PMC3469060">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4046.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2855397/figure/F2/" ref="ordinalpos=4046&amp;ncbi_uid=2885504&amp;link_uid=PMC2855397" href="/pmc/articles/PMC2855397/"><img src="/pmc/articles/instance/2855397/bin/nihms-182523-f0002.gif" src-large="/pmc/articles/instance/2855397/bin/nihms-182523-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2855397/" ref="ordinalpos=4046&amp;ncbi_uid=2885504&amp;link_uid=PMC2855397" image-link="/pmc/articles/PMC2855397/figure/F2/" class="imagepopup">Figure 2. djub affects expression of DIAP1 and Cyclin E and genetically interacts with the Hippo <span class="highlight" style="background-color:">pathway</span>.  From: Ajuba LIM Proteins are Negative Regulators of the Hippo <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A-C) Female third-instar larval eye imaginal discs containing <i>djub</i><sup>I</sup> clones (GFP ve, yellow arrows) generated by <i>Eyeless-flp</i> and stained for DIAP1 (A), DIAP1-lacZ (B), or Cyclin E (C) expression. Anterior is to the left for all larval eye imaginal discs. (D-K) SEMs of adult female <i>Drosophila</i> eyes of wt (E) and Hippo <span class="highlight" style="background-color:">pathway</span> mutants (F,H,J) and <i>djub</i><sup>I</sup> (E) or Hippo <span class="highlight" style="background-color:">pathway</span> mutants containing a deletion of a single copy of <i>djub</i> (G,I,K), as indicated. Mid-pupal eye dissection of Hippo <span class="highlight" style="background-color:">pathway</span> mutants (L,N,P) or Hippo <span class="highlight" style="background-color:">pathway</span> mutants containing a deletion of a single copy of <i>djub</i> (M,O,Q), as indicated, and stained for DE-cadherin to identify interommatidial cells. Scale bars in (D-K) equal 100m and (L-Q) equal 10m. (R) Quantification of interommatidial cell numbers in 10 random fields containing 10 ommatidia each of genotypes (D,F-K). Data are shown as mean percentages +/ standard deviation and (***) represents p-value  0.001.</div></p></div><div class="aux"><div class="resc">Meghna Das Thakur, et al. Curr Biol. ;20(7):657-662.</div><p class="links"><a class="dblinks" href="/pubmed/20303269" ref="ordinalpos=4046&amp;ncbi_uid=2885504&amp;link_uid=20303269">Citation</a><a class="dblinks" href="/pmc/articles/PMC2855397/" ref="ordinalpos=4046&amp;ncbi_uid=2885504&amp;link_uid=PMC2855397">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4047.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4239019/figure/pone-0112909-g004/" ref="ordinalpos=4047&amp;ncbi_uid=6882073&amp;link_uid=PMC4239019" href="/pmc/articles/PMC4239019/"><img src="/pmc/articles/instance/4239019/bin/pone.0112909.g004.gif" src-large="/pmc/articles/instance/4239019/bin/pone.0112909.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4239019/" ref="ordinalpos=4047&amp;ncbi_uid=6882073&amp;link_uid=PMC4239019" image-link="/pmc/articles/PMC4239019/figure/pone-0112909-g004/" class="imagepopup">Figure 4. Granule neuron signatures of Group 4 medulloblastoma..  From: Gene Expression Analyses of the Spatio-Temporal Relationships of Human Medulloblastoma Subgroups during Early Human Neurogenesis. </a></p><div class="supp"><p class="details"><div>a. Differential expression of reelin <span class="highlight" style="background-color:">signaling</span> members b. Semaphorin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members and downstream mediators and c. Dopamine/-synuclein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> members are shown for all sample groups using Box-Whisker plots. <i>* p&lt;0.05, ** p&lt;0.01, *** p&lt;0.001,  p&lt;0.05 compared to all control groups but not exclusive for medulloblastoma subgroup, NFB - normal fetal brain, NFGM - normal fetal germinal matrix, NPC - neural progenitor cells, NSC - neural stem cells.</i></div></p></div><div class="aux"><div class="resc">Cornelia M. Hooper, et al. PLoS One. 2014;9(11):e112909.</div><p class="links"><a class="dblinks" href="/pubmed/25412507" ref="ordinalpos=4047&amp;ncbi_uid=6882073&amp;link_uid=25412507">Citation</a><a class="dblinks" href="/pmc/articles/PMC4239019/" ref="ordinalpos=4047&amp;ncbi_uid=6882073&amp;link_uid=PMC4239019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4048.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4333806/figure/F1/" ref="ordinalpos=4048&amp;ncbi_uid=7235848&amp;link_uid=PMC4333806" href="/pmc/articles/PMC4333806/"><img src="/pmc/articles/instance/4333806/bin/fpls-06-00088-g0001.gif" src-large="/pmc/articles/instance/4333806/bin/fpls-06-00088-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4333806/" ref="ordinalpos=4048&amp;ncbi_uid=7235848&amp;link_uid=PMC4333806" image-link="/pmc/articles/PMC4333806/figure/F1/" class="imagepopup">Figure 1.  From: Structural basis and functions of abscisic acid receptors PYLs. </a></p><div class="supp"><p class="details"><div><b>Summary of the ABA perception and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. ABA receptors include nucleocytoplasmic PYR/PYL/RCARs (PYLs), most likely the plastid-localized Mg-chelatase H subunit (CHLH/GUN5/ABAR) and plasma membranelocalized GPCR type G-proteins (GPCRs). A core <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (black dotted rectangle enlarged in left) consists of PYLs, PP2Cs, SnRK2s, and downstream substrates, which control gene expression (long ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>) and stomatal closure (short ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>). Dimeric PYLs cannot bind to PP2Cs in the absence of ABA; thus, SnRK2s activity is inhibited by PP2Cs. Nevertheless, a subfamily of monomeric PYLs exhibits constitutive inhibitory activity on PP2Cs. In the presence of ABA, PYLs contact and inhibit PP2Cs, leading to the activation of SnRK2s through autophosphorylation. Then, the activated SnRK2 targets ABA-responsive element binding factors (such as bZIP) to regulate gene expression in the nucleus and the cation channel SLOW ANION CHANNEL-ASSOCIATED 1 (SLAC1) and POTASSIUM CHANNEL IN <i>ARABIDOPSIS THALIANA</i> 1 (KAT1) to cause stomatal closure in cytoplasm. The solid line with double-headed arrows indicates the equilibrium between dimeric PYLs and monomeric PYLs. The solid line with an arrow indicates direct positive interactions. The solid line with a bar indicates repression.</div></p></div><div class="aux"><div class="resc">Xing L. Zhang, et al. Front Plant Sci. 2015;6:88.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4048&amp;ncbi_uid=7235848&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4333806/" ref="ordinalpos=4048&amp;ncbi_uid=7235848&amp;link_uid=PMC4333806">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4049.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2862916/figure/Fig8/" ref="ordinalpos=4049&amp;ncbi_uid=2975534&amp;link_uid=PMC2862916" href="/pmc/articles/PMC2862916/"><img src="/pmc/articles/instance/2862916/bin/10162_2009_205_Fig8_HTML.gif" src-large="/pmc/articles/instance/2862916/bin/10162_2009_205_Fig8_HTML.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2862916/" ref="ordinalpos=4049&amp;ncbi_uid=2975534&amp;link_uid=PMC2862916" image-link="/pmc/articles/PMC2862916/figure/Fig8/" class="imagepopup">FIG. 8.  From: cAMP-induced Auditory Supporting Cell Proliferation is Mediated by ERK MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A model for cAMP/PKA and ERK MAPK <span class="highlight" style="background-color:">signaling</span> during FSK-induced supporting cells proliferation.</div></p></div><div class="aux"><div class="resc">Thomas J. Bell, et al. J Assoc Res Otolaryngol. 2010 June;11(2):173-185.</div><p class="links"><a class="dblinks" href="/pubmed/20107853" ref="ordinalpos=4049&amp;ncbi_uid=2975534&amp;link_uid=20107853">Citation</a><a class="dblinks" href="/pmc/articles/PMC2862916/" ref="ordinalpos=4049&amp;ncbi_uid=2975534&amp;link_uid=PMC2862916">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4050.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3387977/figure/F3/" ref="ordinalpos=4050&amp;ncbi_uid=4096215&amp;link_uid=PMC3387977" href="/pmc/articles/PMC3387977/"><img src="/pmc/articles/instance/3387977/bin/nihms381454f3.gif" src-large="/pmc/articles/instance/3387977/bin/nihms381454f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3387977/" ref="ordinalpos=4050&amp;ncbi_uid=4096215&amp;link_uid=PMC3387977" image-link="/pmc/articles/PMC3387977/figure/F3/" class="imagepopup">Fig. 3.  From: Epidermal growth factor receptor inhibition in lung cancer: the evolving role of individualized therapy. </a></p><div class="supp"><p class="details"><div>Potential targets for inhibition within the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Multiple points along the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, indicated with <i>arrows</i>, are currently being evaluated as therapeutic targets in the treatment of NSCLC. Inhibition of EGFR activation can occur by targeting EGFR itself and/or its downstream mediators. Small molecule TKIs and other agents are shown associated with their respective targets. Figure adapted with permission from [12]</div></p></div><div class="aux"><div class="resc">Adi F. Gazdar. Cancer Metastasis Rev. ;29(1):37-48.</div><p class="links"><a class="dblinks" href="/pubmed/20127143" ref="ordinalpos=4050&amp;ncbi_uid=4096215&amp;link_uid=20127143">Citation</a><a class="dblinks" href="/pmc/articles/PMC3387977/" ref="ordinalpos=4050&amp;ncbi_uid=4096215&amp;link_uid=PMC3387977">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4051.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g003/" ref="ordinalpos=4051&amp;ncbi_uid=6137742&amp;link_uid=PMC3137255" href="/pmc/articles/PMC3137255/"><img src="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g003.gif" src-large="/pmc/articles/instance/4012260/bin/cjc-30-01-013-g003.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137255/" ref="ordinalpos=4051&amp;ncbi_uid=6137742&amp;link_uid=PMC3137255" image-link="/pmc/articles/PMC3137255/figure/cjc-30-01-013-g003/" class="imagepopup">Figure 3.Activation of Hh <span class="highlight" style="background-color:">signaling</span> in human cancer. Following the discovery of Hh <span class="highlight" style="background-color:">signaling</span> activation in Gorlin syndrome, increasing evidence suggests that Hh <span class="highlight" style="background-color:">signaling</span> is frequently activated in human cancer. Based on current findings, we group these cancers into three groups. Group one is associated with Gorlin syndrome, including basal cell carcinomas, medulloblastomas, and rhabdomyosarcomas (in muscle) (in red). Group two includes cancer types with reproducible data of Hh <span class="highlight" style="background-color:">signaling</span> activation from several groups, such as oral cancer and many gastrointestinal cancers (in blue). Group three includes cancer types with limited or variable results from different groups (in black). Several common cancer types are included in group three; further investigation will provide insight as to the significance of Hh <span class="highlight" style="background-color:">signaling</span> in these different types of cancer..  From: Clinical implications of hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors. </a></p><div class="aux"><div class="resc">Hailan Liu, et al. Chin J Cancer. 2011 January;30(1):13-26.</div><p class="links"><a class="dblinks" href="/pubmed/21192841" ref="ordinalpos=4051&amp;ncbi_uid=6137742&amp;link_uid=21192841">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137255/" ref="ordinalpos=4051&amp;ncbi_uid=6137742&amp;link_uid=PMC3137255">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4052.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763466/figure/fig5/" ref="ordinalpos=4052&amp;ncbi_uid=5291510&amp;link_uid=PMC3763466" href="/pmc/articles/PMC3763466/"><img src="/pmc/articles/instance/3763466/bin/cddis2013314f5.gif" src-large="/pmc/articles/instance/3763466/bin/cddis2013314f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763466/" ref="ordinalpos=4052&amp;ncbi_uid=5291510&amp;link_uid=PMC3763466" image-link="/pmc/articles/PMC3763466/figure/fig5/" class="imagepopup">Figure 5.  From: Cuprous oxide nanoparticles inhibit the growth and metastasis of melanoma by targeting mitochondria. </a></p><div class="supp"><p class="details"><div>CONPs can significantly reduce the growth of melanoma <i>in vivo by</i> targeting the mitochondria and initiating the mitochondrion-mediated apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Y Wang, et al. Cell Death Dis. 2013 August;4(8):e783.</div><p class="links"><a class="dblinks" href="/pubmed/23990023" ref="ordinalpos=4052&amp;ncbi_uid=5291510&amp;link_uid=23990023">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763466/" ref="ordinalpos=4052&amp;ncbi_uid=5291510&amp;link_uid=PMC3763466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4053.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4076348/figure/i0006-3363-89-1-13-f04/" ref="ordinalpos=4053&amp;ncbi_uid=6351116&amp;link_uid=PMC4076348" href="/pmc/articles/PMC4076348/"><img src="/pmc/articles/instance/4076348/bin/i0006-3363-89-1-13-f04.gif" src-large="/pmc/articles/instance/4076348/bin/i0006-3363-89-1-13-f04.jpg" alt="FIG. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4076348/" ref="ordinalpos=4053&amp;ncbi_uid=6351116&amp;link_uid=PMC4076348" image-link="/pmc/articles/PMC4076348/figure/i0006-3363-89-1-13-f04/" class="imagepopup">FIG. 4.  From: Changes in Mouse Uterine Transcriptome in Estrus and Proestrus. </a></p><div class="supp"><p class="details"><div>Activation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse uterus in estrus versus proestrus. Upregulated genes are marked by red stars.</div></p></div><div class="aux"><div class="resc">Kerri Stanley Yip, et al. Biol Reprod. 2013 July;89(1):13-13.</div><p class="links"><a class="dblinks" href="/pubmed/23740946" ref="ordinalpos=4053&amp;ncbi_uid=6351116&amp;link_uid=23740946">Citation</a><a class="dblinks" href="/pmc/articles/PMC4076348/" ref="ordinalpos=4053&amp;ncbi_uid=6351116&amp;link_uid=PMC4076348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4054.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3472558/figure/fig1/" ref="ordinalpos=4054&amp;ncbi_uid=4309351&amp;link_uid=PMC3472558" href="/pmc/articles/PMC3472558/"><img src="/pmc/articles/instance/3472558/bin/CHRP2012-198197.001.gif" src-large="/pmc/articles/instance/3472558/bin/CHRP2012-198197.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472558/" ref="ordinalpos=4054&amp;ncbi_uid=4309351&amp;link_uid=PMC3472558" image-link="/pmc/articles/PMC3472558/figure/fig1/" class="imagepopup">Figure 1.  From: Antiepidermal Growth Factor Receptor Monoclonal Antibodies: Applications in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (reprinted with permission from BioCarta Pathways. All rights reserved).</div></p></div><div class="aux"><div class="resc">Efat Azizi, et al. Chemother Res Pract. 2012;2012:198197.</div><p class="links"><a class="dblinks" href="/pubmed/23091721" ref="ordinalpos=4054&amp;ncbi_uid=4309351&amp;link_uid=23091721">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472558/" ref="ordinalpos=4054&amp;ncbi_uid=4309351&amp;link_uid=PMC3472558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4055.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3322146/figure/pone-0035331-g008/" ref="ordinalpos=4055&amp;ncbi_uid=3900206&amp;link_uid=PMC3322146" href="/pmc/articles/PMC3322146/"><img src="/pmc/articles/instance/3322146/bin/pone.0035331.g008.gif" src-large="/pmc/articles/instance/3322146/bin/pone.0035331.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3322146/" ref="ordinalpos=4055&amp;ncbi_uid=3900206&amp;link_uid=PMC3322146" image-link="/pmc/articles/PMC3322146/figure/pone-0035331-g008/" class="imagepopup">Figure 8. Model of miR148a associated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Role of miR-148a in Hepatitis B Associated Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>miR148a increases -catenin expression by inhibiting PTEN. See the text for additional details.</div></p></div><div class="aux"><div class="resc">Ke Yuan, et al. PLoS One. 2012;7(4):e35331.</div><p class="links"><a class="dblinks" href="/pubmed/22496917" ref="ordinalpos=4055&amp;ncbi_uid=3900206&amp;link_uid=22496917">Citation</a><a class="dblinks" href="/pmc/articles/PMC3322146/" ref="ordinalpos=4055&amp;ncbi_uid=3900206&amp;link_uid=PMC3322146">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4056.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3091413/figure/f4-cin-2011-093/" ref="ordinalpos=4056&amp;ncbi_uid=2952275&amp;link_uid=PMC3091413" href="/pmc/articles/PMC3091413/"><img src="/pmc/articles/instance/3091413/bin/cin-2011-093f4.gif" src-large="/pmc/articles/instance/3091413/bin/cin-2011-093f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3091413/" ref="ordinalpos=4056&amp;ncbi_uid=2952275&amp;link_uid=PMC3091413" image-link="/pmc/articles/PMC3091413/figure/f4-cin-2011-093/" class="imagepopup">Figure 4.  From: Associating GWAS Information with the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Using Transcription Profiling. </a></p><div class="supp"><p class="details"><div>Results of <span class="highlight" style="background-color:">pathway</span> prediction and network modeling showing interactions between genes containing SNPs associated with risk for breast cancer and genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> and other biological pathways based on the Caucasian population. Nodes represent genes and vertices represent interactions. The color code denotes the biological process in which the genes are involved as defined in Figure 3. The color codes in the vertices indicate the experimental techniques or a combination thereof used to confirm the relationship between the genes as determined by the experiments reported in the literature. Candidate genes containing SNPs associated with risk for breast cancer are shown in red, genes involved in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in blue and new genes not reported in GWAS studies are shown in black. For the full names of genes and number of SNPs per gene including GWAS references, please refer Table A in supplementary data.</div></p></div><div class="aux"><div class="resc">Chindo Hicks, et al. Cancer Inform. 2011;10:93-108.</div><p class="links"><a class="dblinks" href="/pubmed/21584266" ref="ordinalpos=4056&amp;ncbi_uid=2952275&amp;link_uid=21584266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3091413/" ref="ordinalpos=4056&amp;ncbi_uid=2952275&amp;link_uid=PMC3091413">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4057.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3731360/figure/pone-0070352-g006/" ref="ordinalpos=4057&amp;ncbi_uid=5184580&amp;link_uid=PMC3731360" href="/pmc/articles/PMC3731360/"><img src="/pmc/articles/instance/3731360/bin/pone.0070352.g006.gif" src-large="/pmc/articles/instance/3731360/bin/pone.0070352.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3731360/" ref="ordinalpos=4057&amp;ncbi_uid=5184580&amp;link_uid=PMC3731360" image-link="/pmc/articles/PMC3731360/figure/pone-0070352-g006/" class="imagepopup">Figure 6. Hypothetical model for the role of CTLA4 in CLL cell proliferation/survival..  From: Role of CTLA4 in the Proliferation and Survival of Chronic Lymphocytic Leukemia. </a></p><div class="supp"><p class="details"><div><b>Panel A:</b> When CLL cells express low CD38, but high CTLA4, CTLA4 inhibits the CD38/BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at multiple levels. CTLA4 downregulates NFATC2 and proliferation-associated molecules such as c-Fos and c-Myc. Downregulation of NFATC2 may also be associated with an autoregulatory loop for CTLA4, which would downregulate CTLA4 transcription. CTLA4 also downregulates the expression of Bcl-2, thus decreasing the survival of CLL cells. CTLA4 inhibits the expression of STAT1, thus deregulating the JAK/STAT <span class="highlight" style="background-color:">pathway</span> and inhibiting CLL cell growth. <b>Panel B:</b> When CLL cells express high CD38, but low CTLA4, activated CD38/BCR <span class="highlight" style="background-color:">signaling</span> upregulates downstream molecules in the <span class="highlight" style="background-color:">pathway</span>, such as NFATC2, c-Fos, and Bcl-2. These molecules will increase proliferation and survival of CLL cells. Low expression of CTLA4 does not interfere with the expression of STAT1, which favors CLL cell growth.</div></p></div><div class="aux"><div class="resc">Amit K. Mittal, et al. PLoS One. 2013;8(8):e70352.</div><p class="links"><a class="dblinks" href="/pubmed/23936412" ref="ordinalpos=4057&amp;ncbi_uid=5184580&amp;link_uid=23936412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3731360/" ref="ordinalpos=4057&amp;ncbi_uid=5184580&amp;link_uid=PMC3731360">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4058.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2151853/figure/F10/" ref="ordinalpos=4058&amp;ncbi_uid=1247386&amp;link_uid=PMC2151853" href="/pmc/articles/PMC2151853/"><img src="/pmc/articles/instance/2151853/bin/cc5704-10.gif" src-large="/pmc/articles/instance/2151853/bin/cc5704-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2151853/" ref="ordinalpos=4058&amp;ncbi_uid=1247386&amp;link_uid=PMC2151853" image-link="/pmc/articles/PMC2151853/figure/F10/" class="imagepopup">Figure 10.  From: Hyperoxia increases ventilator-induced lung injury via mitogen-activated protein kinases: a prospective, controlled animal experiment. </a></p><div class="supp"><p class="details"><div>Differences in <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation with lung stretch with mechanical ventilation with and without hyperoxia. Activation of the JNK <span class="highlight" style="background-color:">pathway</span> was involved in ventilator-induced neutrophil infiltration, cytokine production, and microvascular permeability with and without hyperoxia. Both ERK and JNK were involved in airway epithelial cell apoptosis with and without hyperoxia. The major difference in the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation was that ERK1/2 activation was involved in increased MIP-2 production and neutrophil infiltration only in mice exposed to high-tidal volume ventilation with hyperoxia, and not in mice exposed to high-tidal volume ventilation without hyperoxia. JNK activation was involved with and without hyperoxia. ERK, extracellular signal-regulated kinase; JNK, c-Jun NH<sub>2</sub>-terminal kinase; MIP-2, macrophage inflammatory protein-2.</div></p></div><div class="aux"><div class="resc">Li-Fu Li, et al. Crit Care. 2007;11(1):R25-R25.</div><p class="links"><a class="dblinks" href="/pubmed/17316425" ref="ordinalpos=4058&amp;ncbi_uid=1247386&amp;link_uid=17316425">Citation</a><a class="dblinks" href="/pmc/articles/PMC2151853/" ref="ordinalpos=4058&amp;ncbi_uid=1247386&amp;link_uid=PMC2151853">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4059.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3108474/figure/F2/" ref="ordinalpos=4059&amp;ncbi_uid=3020644&amp;link_uid=PMC3108474" href="/pmc/articles/PMC3108474/"><img src="/pmc/articles/instance/3108474/bin/fphar-02-00111-g002.gif" src-large="/pmc/articles/instance/3108474/bin/fphar-02-00111-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3108474/" ref="ordinalpos=4059&amp;ncbi_uid=3020644&amp;link_uid=PMC3108474" image-link="/pmc/articles/PMC3108474/figure/F2/" class="imagepopup">Figure 2.  From: MT-MMPS as Regulators of Vessel Stability Associated with Angiogenesis. </a></p><div class="supp"><p class="details"><div><b>Membrane type-MMPs regulate vascular cell <span class="highlight" style="background-color:">signaling</span> in cancer</b>. MT1-MMP regulates cell migration through both ECM proteolysis and non-proteolytic-dependent TIMP-2 activation of ERK1/2 <span class="highlight" style="background-color:">pathway</span>. MT1-MMP regulates VEGF gene expression through Src, Akt, and mTOR activation and stimulates tumor angiogenesis. In addition, it induces RANKL shedding and <span class="highlight" style="background-color:">signaling</span> through Src leading to cell migration. MT1-MMP cooperates with platelet derived S1P to induce <span class="highlight" style="background-color:">signaling</span> through G protein-coupled-receptors (GPCRs) and regulates endothelial cell migration and tubulogenesis. MT1-MMP regulates TGF bioavailability and its <span class="highlight" style="background-color:">signaling</span> through ALK5 leading to vessel maturation. Shedding of Tie-2 or degradation of decorin by MT1-MMP stimulates angiogenesis, but shedding of endoglin from endothelial cell surface inhibits angiogenesis. ProTGF can also be activated by MMP2 and MMP9. MT1-MMP activates PDGFR <span class="highlight" style="background-color:">signaling</span> and regulates pericyte migration and vessel maturation. MT4-MMP a metastatic MMP, is highly expressed by cancer cells and induces tumor vessel destabilization through pericyte detachment.</div></p></div><div class="aux"><div class="resc">Nor Eddine Sounni, et al. Front Pharmacol. 2011;2:111.</div><p class="links"><a class="dblinks" href="/pubmed/21687519" ref="ordinalpos=4059&amp;ncbi_uid=3020644&amp;link_uid=21687519">Citation</a><a class="dblinks" href="/pmc/articles/PMC3108474/" ref="ordinalpos=4059&amp;ncbi_uid=3020644&amp;link_uid=PMC3108474">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4060.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4256424/figure/pone-0114470-g006/" ref="ordinalpos=4060&amp;ncbi_uid=6954190&amp;link_uid=PMC4256424" href="/pmc/articles/PMC4256424/"><img src="/pmc/articles/instance/4256424/bin/pone.0114470.g006.gif" src-large="/pmc/articles/instance/4256424/bin/pone.0114470.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4256424/" ref="ordinalpos=4060&amp;ncbi_uid=6954190&amp;link_uid=PMC4256424" image-link="/pmc/articles/PMC4256424/figure/pone-0114470-g006/" class="imagepopup">Figure 6. Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in dexamethasone pretreatment-induced neuroprotection against neonatal hypoxic-ischemic (HI) brain injury..  From: Dexamethasone Protects Neonatal Hypoxic-Ischemic Brain Injury via L-PGDS-Dependent PGD2-DP1-pERK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Dexamethasone interacts with the glucocorticoids receptor (GR) in the cytoplasm, promotes the translocation of GR into the nucleus, and increases L-PGDS expression, which stimulates conversion of PGH<sub>2</sub> to PGD<sub>2</sub>. PGD<sub>2</sub> then binds to DP<sub>1</sub> receptor (DP<sub>1</sub>R) and initiates a series of intracellular <span class="highlight" style="background-color:">signaling</span> pathways, of which MEK1/pERK1 function as the major downstream effectors. Upregulation of pERK1 confers dexamethasone pretreatment-induced neuroprotective effects in neonatal HI brain insult. Inhibition of L-PGDS, PGD<sub>2</sub> or MEK1 <i>via</i> correspondent inhibitors (SeCl<sub>4,</sub> MK-0524 or PD98059) thus inhibits dexamethasone-induced neuroprotection, suggesting the interaction of glucocorticoids/GR and L-PGDS-PGD<sub>2</sub>-DP<sub>1</sub>-pERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> underpinning the neuroprotective effect of dexamethasone on neonatal HI brain injury. GR: glucocorticoid receptor; L-PGDS: lipocalin-type prostaglandin D synthase; PGH<sub>2</sub>: prostaglandin H<sub>2</sub>; PGD<sub>2</sub>: prostaglandin D<sub>2</sub>; DP<sub>1</sub>: D prostanoid; MEK1: mitogen-activated kinase/ERK kinase 1; pERK-42/44: phosphorylated extracellular signal regulated kinase 42/44; SeCl<sub>4</sub>: selenium chloride, a selective inhibitor of lipocalin-type prostaglandin D synthase (L-PGDS); PD98059: a selective inhibitor of MEK1; MK-0524: a selective antagonist for prostaglandin D<sub>2</sub> receptor (DP<sub>1</sub>).</div></p></div><div class="aux"><div class="resc">Pablo J. Gonzalez-Rodriguez, et al. PLoS One. 2014;9(12):e114470.</div><p class="links"><a class="dblinks" href="/pubmed/25474649" ref="ordinalpos=4060&amp;ncbi_uid=6954190&amp;link_uid=25474649">Citation</a><a class="dblinks" href="/pmc/articles/PMC4256424/" ref="ordinalpos=4060&amp;ncbi_uid=6954190&amp;link_uid=PMC4256424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4061.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4042834/figure/F4/" ref="ordinalpos=4061&amp;ncbi_uid=6257623&amp;link_uid=PMC4042834" href="/pmc/articles/PMC4042834/"><img src="/pmc/articles/instance/4042834/bin/1753-6561-7-S7-S2-4.gif" src-large="/pmc/articles/instance/4042834/bin/1753-6561-7-S7-S2-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4042834/" ref="ordinalpos=4061&amp;ncbi_uid=6257623&amp;link_uid=PMC4042834" image-link="/pmc/articles/PMC4042834/figure/F4/" class="imagepopup">Figure 4.  From: Conceptualization of molecular findings by mining gene annotations. </a></p><div class="supp"><p class="details"><div><b>An example of a KEGG <span class="highlight" style="background-color:">pathway</span> containing genes involved in multiple processes</b>. The "MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>" (hsa04010) is shown. Two functionally coherent subsets are highlighted. The genes summarized by GO:0023014 (<i>Signal transduction by phosphorylation</i>) are in green; the genes summarized by GO:0006915 (<i>Apoptotic process</i>) are in blue. Genes involved in both biological processes are in yellow.</div></p></div><div class="aux"><div class="resc">Vicky Chen, et al. BMC Proc. 2013;7(Suppl 7):S2-S2.</div><p class="links"><a class="dblinks" href="/pubmed/24564884" ref="ordinalpos=4061&amp;ncbi_uid=6257623&amp;link_uid=24564884">Citation</a><a class="dblinks" href="/pmc/articles/PMC4042834/" ref="ordinalpos=4061&amp;ncbi_uid=6257623&amp;link_uid=PMC4042834">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4062.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2934865/figure/F2/" ref="ordinalpos=4062&amp;ncbi_uid=3286910&amp;link_uid=PMC2934865" href="/pmc/articles/PMC2934865/"><img src="/pmc/articles/instance/2934865/bin/nihms222776f2.gif" src-large="/pmc/articles/instance/2934865/bin/nihms222776f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2934865/" ref="ordinalpos=4062&amp;ncbi_uid=3286910&amp;link_uid=PMC2934865" image-link="/pmc/articles/PMC2934865/figure/F2/" class="imagepopup">Figure 2.  From: Emerging Roles of SUMO Modification in Arthritis. </a></p><div class="supp"><p class="details"><div>Hypothetical model for SUMO regulation in NFB <span class="highlight" style="background-color:">pathway</span>. Sumoylation of IB mediated by Ubc9 and RSUME competes off the ubiquitination <span class="highlight" style="background-color:">pathway</span>, which stabilizes IB and inhibits NFB <span class="highlight" style="background-color:">signaling</span>. IL-1R is also sumoylated upon stimulation, which conduces to signal transduction. The question marks indicate processes which have not been fully substantiated experimentally.</div></p></div><div class="aux"><div class="resc">Dongyao Yan, et al. Gene. ;466(1-2):1-15.</div><p class="links"><a class="dblinks" href="/pubmed/20627123" ref="ordinalpos=4062&amp;ncbi_uid=3286910&amp;link_uid=20627123">Citation</a><a class="dblinks" href="/pmc/articles/PMC2934865/" ref="ordinalpos=4062&amp;ncbi_uid=3286910&amp;link_uid=PMC2934865">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4063.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3081620/figure/F1/" ref="ordinalpos=4063&amp;ncbi_uid=3386427&amp;link_uid=PMC3081620" href="/pmc/articles/PMC3081620/"><img src="/pmc/articles/instance/3081620/bin/nihms-275732-f0001.gif" src-large="/pmc/articles/instance/3081620/bin/nihms-275732-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3081620/" ref="ordinalpos=4063&amp;ncbi_uid=3386427&amp;link_uid=PMC3081620" image-link="/pmc/articles/PMC3081620/figure/F1/" class="imagepopup">Fig. 1.  From: Activation of The Transforming Growth Factor-Beta <span class="highlight" style="background-color:">Pathway</span> in Bacterial Otitis Media. </a></p><div class="supp"><p class="details"><div>A representative cascade of the TGF- <span class="highlight" style="background-color:">pathway</span>. TGF-1~3, TGF- receptors (I &amp; II), Smads, and transcriptional factor (TF) constitute the cascade. TGF- ligands (including TGF-1~3, activins, BMPs, etc.) with the assistance of accessory receptor (betaglycan) activate the formation of heterotetrameric complexes of type I and II serine/threonine kinase receptors which in turn activate the intracellular effectors of TGF- <span class="highlight" style="background-color:">signaling</span>, Smad2/3 by phosphorylation. The action recruits Smad4 to the complex and translocate into the cellular nucleus where Smad/2/3/4, working together with a cellular context-specific transcriptional factor (TF), bind to DNA and trigger the expression of a battery of genes including collagens, fibronectins, laminins, etc. Eventually, the TGF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to fibrotic responses.</div></p></div><div class="aux"><div class="resc">Yun-Woo Lee, et al. Ann Otol Rhinol Laryngol. ;120(3):204-213.</div><p class="links"><a class="dblinks" href="/pubmed/21510147" ref="ordinalpos=4063&amp;ncbi_uid=3386427&amp;link_uid=21510147">Citation</a><a class="dblinks" href="/pmc/articles/PMC3081620/" ref="ordinalpos=4063&amp;ncbi_uid=3386427&amp;link_uid=PMC3081620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4064.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g003/" ref="ordinalpos=4064&amp;ncbi_uid=4834034&amp;link_uid=PMC3633842" href="/pmc/articles/PMC3633842/"><img src="/pmc/articles/instance/3633842/bin/pone.0062845.g003.gif" src-large="/pmc/articles/instance/3633842/bin/pone.0062845.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633842/" ref="ordinalpos=4064&amp;ncbi_uid=4834034&amp;link_uid=PMC3633842" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g003/" class="imagepopup">Figure 3. Role of pattern recognition receptors in recognition of bacteria and viruses <span class="highlight" style="background-color:">pathway</span>..  From: Decreased Pattern Recognition Receptor <span class="highlight" style="background-color:">Signaling</span>, Interferon-Signature, and Bactericidal/Permeability-Increasing Protein Gene Expression in Cord Blood of Term Low Birth Weight Human Newborns. </a></p><div class="supp"><p class="details"><div>The canonical <span class="highlight" style="background-color:">pathway</span> indicates the down-regulated genes in LBW newborns as green nodes. The uncolored nodes are the genes inferred by IPA from its knowledge base. The interferon stimulated genes in this <span class="highlight" style="background-color:">pathway</span> are marked as red diamond ().</div></p></div><div class="aux"><div class="resc">Vikas Vikram Singh, et al. PLoS One. 2013;8(4):e62845.</div><p class="links"><a class="dblinks" href="/pubmed/23626859" ref="ordinalpos=4064&amp;ncbi_uid=4834034&amp;link_uid=23626859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633842/" ref="ordinalpos=4064&amp;ncbi_uid=4834034&amp;link_uid=PMC3633842">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4065.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3039836/figure/fig7/" ref="ordinalpos=4065&amp;ncbi_uid=2782212&amp;link_uid=PMC3039836" href="/pmc/articles/PMC3039836/"><img src="/pmc/articles/instance/3039836/bin/cddis201037f7.gif" src-large="/pmc/articles/instance/3039836/bin/cddis201037f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3039836/" ref="ordinalpos=4065&amp;ncbi_uid=2782212&amp;link_uid=PMC3039836" image-link="/pmc/articles/PMC3039836/figure/fig7/" class="imagepopup">Figure 7.  From: MEK-ERK-dependent multiple caspase activation by mitochondrial proapoptotic Bcl-2 family proteins is essential for heavy ion irradiation-induced glioma cell death. </a></p><div class="supp"><p class="details"><div>Schematic summary of the present findings. On the basis of the results of the current study, we propose that CB irradiation triggers reactivation of the MEKERK <span class="highlight" style="background-color:">pathway</span> subsequent to initial downregulation, which then initiates the mitochondrial <span class="highlight" style="background-color:">pathway</span> of caspase cascade activation, culminating in apoptotic death of glioma cells. Molecules involved in this cell death <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> could be new targets for augmentation of CB therapy against gliomas. For instance, EGF may be used for MEKERK <span class="highlight" style="background-color:">pathway</span> activation, ABT-737 for the activation of mitochondrial apoptosis, and SMAC mimetics for caspase cascade activation</div></p></div><div class="aux"><div class="resc">A Tomiyama, et al. Cell Death Dis. 2010 July;1(7):e60.</div><p class="links"><a class="dblinks" href="/pubmed/21364665" ref="ordinalpos=4065&amp;ncbi_uid=2782212&amp;link_uid=21364665">Citation</a><a class="dblinks" href="/pmc/articles/PMC3039836/" ref="ordinalpos=4065&amp;ncbi_uid=2782212&amp;link_uid=PMC3039836">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4066.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3408479/figure/pone-0039120-g008/" ref="ordinalpos=4066&amp;ncbi_uid=4154067&amp;link_uid=PMC3408479" href="/pmc/articles/PMC3408479/"><img src="/pmc/articles/instance/3408479/bin/pone.0039120.g008.gif" src-large="/pmc/articles/instance/3408479/bin/pone.0039120.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3408479/" ref="ordinalpos=4066&amp;ncbi_uid=4154067&amp;link_uid=PMC3408479" image-link="/pmc/articles/PMC3408479/figure/pone-0039120-g008/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Signaling</span> of p53 expression by NF-B during endoplasmic reticulum stress..  From: Endoplasmic Reticulum Stress Stimulates p53 Expression through NF-?B Activation. </a></p><div class="supp"><p class="details"><div>Accumulation of unfolded or misfolded proteins in the endoplasmic reticulum induces activation of multiple <span class="highlight" style="background-color:">signaling</span> pathways. Many studies have indicated that activation of IRE1, ATF6, and PERK represents the standard UPR pathways and turns on the expression of many downstream genes such as <i>GRP78</i>. ER stress induced caspase-12/caspase-9/caspase-3, PERK/ATF-4/CHOP, IRE1/Ask1/JNK, and PERK/eIF2/NF-B <span class="highlight" style="background-color:">signaling</span> pathways and then those pathways may alter cellular homeostasis by regulating multiple genes, resulting in regulation of cell death or survival. This <span class="highlight" style="background-color:">pathway</span> (indicated by <i>heavy arrows</i>) is demonstrated in this report. We demonstrated that induction of p53 expression is mediated by NF-B during ER stress.</div></p></div><div class="aux"><div class="resc">Wan-Chi Lin, et al. PLoS One. 2012;7(7):e39120.</div><p class="links"><a class="dblinks" href="/pubmed/22859938" ref="ordinalpos=4066&amp;ncbi_uid=4154067&amp;link_uid=22859938">Citation</a><a class="dblinks" href="/pmc/articles/PMC3408479/" ref="ordinalpos=4066&amp;ncbi_uid=4154067&amp;link_uid=PMC3408479">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4067.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3932994/figure/F1/" ref="ordinalpos=4067&amp;ncbi_uid=5840106&amp;link_uid=PMC3932994" href="/pmc/articles/PMC3932994/"><img src="/pmc/articles/instance/3932994/bin/nihms554782f1.gif" src-large="/pmc/articles/instance/3932994/bin/nihms554782f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3932994/" ref="ordinalpos=4067&amp;ncbi_uid=5840106&amp;link_uid=PMC3932994" image-link="/pmc/articles/PMC3932994/figure/F1/" class="imagepopup">Fig. 1. Crosstalk between the <span class="highlight" style="background-color:">signaling</span> pathways that mediate nutrient sensing and the mating response.  From: <span class="highlight" style="background-color:">Signaling</span> Crosstalk: Integrating Nutrient Availability and Sex. </a></p><div class="supp"><p class="details"><div>The mating response <span class="highlight" style="background-color:">pathway</span> is initiated when the pheromone  factor binds to the GPCR Ste2 at the plasma membrane. Ligand binding stimulates the exchange of GDP for GTP by Gpa1 and the dissociation of Gpa1 from the Ste4-Ste18 dimer of the G protein. Ste4-Ste18 interacts with Ste20 to stimulate the MAPK <span class="highlight" style="background-color:">signaling</span> cascade, culminating in the activation of Fus3 and the mating response. During conditions of glucose abundance, the Ras and Gpa2 pathways activate PKA, which promotes the Glc7-Reg1dependent dephosphorylation of Gpa1 to maintain an efficient mating response. However, when glucose concentrations are limiting, PKA and the Glc7-Reg1 phosphatase complex are inactivated. Thus, the kinase Sak1 phosphorylates both Snf1 (the AMPK homolog), to initiate metabolic adaptation, and the Gpa1 protein, to reduce mating efficiency. The exact consequences of Gpa1 phosphorylation on G protein <span class="highlight" style="background-color:">signaling</span> remain to be determined.</div></p></div><div class="aux"><div class="resc">Martin C. Schmidt. Sci Signal. ;6(291):pe28-pe28.</div><p class="links"><a class="dblinks" href="/pubmed/24003253" ref="ordinalpos=4067&amp;ncbi_uid=5840106&amp;link_uid=24003253">Citation</a><a class="dblinks" href="/pmc/articles/PMC3932994/" ref="ordinalpos=4067&amp;ncbi_uid=5840106&amp;link_uid=PMC3932994">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4068.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3355621/figure/F3/" ref="ordinalpos=4068&amp;ncbi_uid=4019402&amp;link_uid=PMC3355621" href="/pmc/articles/PMC3355621/"><img src="/pmc/articles/instance/3355621/bin/fpls-03-00051-g003.gif" src-large="/pmc/articles/instance/3355621/bin/fpls-03-00051-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3355621/" ref="ordinalpos=4068&amp;ncbi_uid=4019402&amp;link_uid=PMC3355621" image-link="/pmc/articles/PMC3355621/figure/F3/" class="imagepopup">Figure 3.  From: Crosstalk between Phospholipase D and Sphingosine Kinase in Plant Stress <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Proposed model for crosstalk between PLD1/PA and SPHK/phyto-S1P in ABA-mediated stomatal closure <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. ABA may be perceived by the receptor (PYR/PYL/RCAR) in the cytosol, leading to activation of SPHK to produce phyto-S1P which initiates a cascade to activate PLD1. PLD1 hydrolyzes phospholipids to increase PA level in membrane (plasma membrane and tonoplast). PLD1-deprived PA promotes the ABA effect through three targets: (i) PA binds to ABI1 and tethers ABI1 to the membrane to inhibit its negative effect; (ii) PA stimulates plasma membrane-localized NADPH oxidase to form secondary messenger: ROS; (iii) Increased PA in tonoplast interacts with SPHK and promotes its activity to form a positive loop. PLD1/PA- and SPHK/phyto-S1P-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates ion channel activity, leading to ion flux in guard cell and finally stomatal closure. Note that this model summarizes the crosstalk between PLD1/PA and SPHK/phyto-S1P and their roles in ABA-mediated stomatal closure, not all ABA <span class="highlight" style="background-color:">signaling</span> components are included in this model. Arrow indicates positive regulation, bar indicates repression. Red arrow represents reactions which produce secondary <span class="highlight" style="background-color:">signaling</span> molecules.</div></p></div><div class="aux"><div class="resc">Liang Guo, et al. Front Plant Sci. 2012;3:51.</div><p class="links"><a class="dblinks" href="/pubmed/22639650" ref="ordinalpos=4068&amp;ncbi_uid=4019402&amp;link_uid=22639650">Citation</a><a class="dblinks" href="/pmc/articles/PMC3355621/" ref="ordinalpos=4068&amp;ncbi_uid=4019402&amp;link_uid=PMC3355621">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4069.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4308786/figure/F5/" ref="ordinalpos=4069&amp;ncbi_uid=7127273&amp;link_uid=PMC4308786" href="/pmc/articles/PMC4308786/"><img src="/pmc/articles/instance/4308786/bin/NRR-7-1370-g006.gif" src-large="/pmc/articles/instance/4308786/bin/NRR-7-1370-g006.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4308786/" ref="ordinalpos=4069&amp;ncbi_uid=7127273&amp;link_uid=PMC4308786" image-link="/pmc/articles/PMC4308786/figure/F5/" class="imagepopup">Figure 5.  From: Mitogen activated protein kinase <span class="highlight" style="background-color:">signaling</span> pathways participate in the active principle region of Buyang Huanwu decoction-induced differentiation of bone marrow mesenchymal stem cells. </a></p><div class="supp"><p class="details"><div>Reverse transcription-PCR electropherogram of neuron specific enolase mRNA expression after the mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> was blocked.<br />1: apr-BYHWD group; 2: apr-BYHWD + SB203580 group; 3: apr-BYHWD + PD98059 group; M: Marker; BYHWD: <i>Buyang Huanwu</i> decoction; apr-BYHWD: active principle region of BYHWD; PD98059 and SB203580: cellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocking agents.</div></p></div><div class="aux"><div class="resc">Jinghui Zheng, et al. Neural Regen Res. 2012 June 25;7(18):1370-1377.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4069&amp;ncbi_uid=7127273&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4308786/" ref="ordinalpos=4069&amp;ncbi_uid=7127273&amp;link_uid=PMC4308786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4070.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3376044/figure/F1/" ref="ordinalpos=4070&amp;ncbi_uid=4123842&amp;link_uid=PMC3376044" href="/pmc/articles/PMC3376044/"><img src="/pmc/articles/instance/3376044/bin/cib-5-122-g1.gif" src-large="/pmc/articles/instance/3376044/bin/cib-5-122-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3376044/" ref="ordinalpos=4070&amp;ncbi_uid=4123842&amp;link_uid=PMC3376044" image-link="/pmc/articles/PMC3376044/figure/F1/" class="imagepopup">Figure. From: Comparative analysis of calcineurin <span class="highlight" style="background-color:">signaling</span> between Candida dubliniensis and Candida albicans. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Comparison of the calcineurin <span class="highlight" style="background-color:">pathway</span> in <i>C. albicans</i> and <i>C. dubliniensis</i>. The roles of calcineurin and Crz1 in hyphal growth are unclear in <i>C. albicans</i> (dotted arrows) while their roles in hyphal growth of <i>C. dubliniensis</i> (right panel) are demonstrable. Calcineurin is required for virulence in both Candida species. ER stress responses are controlled by Crz1-dependent and Crz1-independent calcineurin <span class="highlight" style="background-color:">signaling</span> in <i>C. albicans</i> and <i>C. dubliniensis</i>, respectively. Drug tolerance and alkaline pH homeostasis are governed by the Crz1-dependent calcineurin <span class="highlight" style="background-color:">pathway</span> in both Candida species. <i>C. dubliniensis</i> calcineurin plays a larger role (bold arrow) in controlling acidic pH homeostasis compared with <i>C. albicans</i>. In <i>C. albicans</i> calcineurin plays a greater role (bold arrow) in serum survival compared with <i>C. dubliniensis</i>.</div></p></div><div class="aux"><div class="resc">Jing Zhang, et al. Commun Integr Biol. 2012 March 1;5(2):122-126.</div><p class="links"><a class="dblinks" href="/pubmed/22808313" ref="ordinalpos=4070&amp;ncbi_uid=4123842&amp;link_uid=22808313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3376044/" ref="ordinalpos=4070&amp;ncbi_uid=4123842&amp;link_uid=PMC3376044">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4071.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1456095/figure/fig6/" ref="ordinalpos=4071&amp;ncbi_uid=625275&amp;link_uid=PMC1456095" href="/pmc/articles/PMC1456095/"><img src="/pmc/articles/instance/1456095/bin/GEN17141683f6.gif" src-large="/pmc/articles/instance/1456095/bin/GEN17141683f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1456095/" ref="ordinalpos=4071&amp;ncbi_uid=625275&amp;link_uid=PMC1456095" image-link="/pmc/articles/PMC1456095/figure/fig6/" class="imagepopup">Figure 6. From: A Genetic Screen Targeting the Tumor Necrosis Factor/Eiger <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: Identification of Drosophila TAB2 as a Functionally Conserved Component. </a></p><div class="supp"><p class="details"><div>Proposed model for Eiger <span class="highlight" style="background-color:">signaling</span> in Drosophila. Following the binding of Eiger to Wengen a <span class="highlight" style="background-color:">signaling</span> complex consisting of Msn-dTRAF1-dTAB2-dTAK1 is stabilized, which allows the phosphorylation and activation of dTAK1 by Msn. Subsequently dTAK1 activates the core JNKK-JNK module, consisting of the Drosophila homologs Hep and Bsk.</div></p></div><div class="aux"><div class="resc">Peter Geuking, et al. Genetics. 2005 December;171(4):1683-1694.</div><p class="links"><a class="dblinks" href="/pubmed/16079232" ref="ordinalpos=4071&amp;ncbi_uid=625275&amp;link_uid=16079232">Citation</a><a class="dblinks" href="/pmc/articles/PMC1456095/" ref="ordinalpos=4071&amp;ncbi_uid=625275&amp;link_uid=PMC1456095">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4072.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3979765/figure/pone-0094114-g009/" ref="ordinalpos=4072&amp;ncbi_uid=6018290&amp;link_uid=PMC3979765" href="/pmc/articles/PMC3979765/"><img src="/pmc/articles/instance/3979765/bin/pone.0094114.g009.gif" src-large="/pmc/articles/instance/3979765/bin/pone.0094114.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3979765/" ref="ordinalpos=4072&amp;ncbi_uid=6018290&amp;link_uid=PMC3979765" image-link="/pmc/articles/PMC3979765/figure/pone-0094114-g009/" class="imagepopup">Figure 9. Memo has an important role in cell-autonomous S1PR <span class="highlight" style="background-color:">signaling</span>..  From: Memo Has a Novel Role in S1P <span class="highlight" style="background-color:">Signaling</span> and Crucial for Vascular Development. </a></p><div class="supp"><p class="details"><div>S1PR1 can be activated by extracellular (red arrow) and intracellular (blue arrow) S1P. Our <i>in vitro</i> data demonstrate that Memo has an important role in the cell-autonomous <span class="highlight" style="background-color:">pathway</span> activating S1PR1 by intracellular S1P. In MEFs, absence of Memo results in a defect in cell-autonomous activation of S1PR1 during cell migration induced by PDGF. In HUVECs, knockdown of Memo results in defects in survival and junction stabilization, both of which were rescued by addition of exogenous S1P.</div></p></div><div class="aux"><div class="resc">Shunya Kondo, et al. PLoS One. 2014;9(4):e94114.</div><p class="links"><a class="dblinks" href="/pubmed/24714781" ref="ordinalpos=4072&amp;ncbi_uid=6018290&amp;link_uid=24714781">Citation</a><a class="dblinks" href="/pmc/articles/PMC3979765/" ref="ordinalpos=4072&amp;ncbi_uid=6018290&amp;link_uid=PMC3979765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4073.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035094/figure/F6/" ref="ordinalpos=4073&amp;ncbi_uid=3621374&amp;link_uid=PMC3035094" href="/pmc/articles/PMC3035094/"><img src="/pmc/articles/instance/3035094/bin/DEV057729F6.gif" src-large="/pmc/articles/instance/3035094/bin/DEV057729F6.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035094/" ref="ordinalpos=4073&amp;ncbi_uid=3621374&amp;link_uid=PMC3035094" image-link="/pmc/articles/PMC3035094/figure/F6/" class="imagepopup">Fig. 6. From: Capicua DNA-binding sites are general response elements for RTK <span class="highlight" style="background-color:">signaling</span> in Drosophila. </a></p><div class="supp"><p class="details"><div><b>Cic regulatory elements mediate Torso and EGFR responses.</b> (<b>A</b>) Sequential activation of the Torso (gray) and EGFR (blue) RTK pathways downregulates Cic along the AP and DV embryonic axes. Both pathways relieve Cic repression mediated by common cis-regulatory elements. Developmental stages (St.) are indicated. (<b>B</b>) EGFR <span class="highlight" style="background-color:">signaling</span> (blue) induces <i>argos</i> expression via Cic sites; activation of the <span class="highlight" style="background-color:">pathway</span> in vein cells leads to downregulation of Cic repressor activity, thereby derepressing <i>argos</i> transcription.</div></p></div><div class="aux"><div class="resc">Leiore Ajuria, et al. Development. 2011 March 1;138(5):915-924.</div><p class="links"><a class="dblinks" href="/pubmed/21270056" ref="ordinalpos=4073&amp;ncbi_uid=3621374&amp;link_uid=21270056">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035094/" ref="ordinalpos=4073&amp;ncbi_uid=3621374&amp;link_uid=PMC3035094">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4074.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2666844/figure/f5-gcr1_4ap0013/" ref="ordinalpos=4074&amp;ncbi_uid=1827685&amp;link_uid=PMC2666844" href="/pmc/articles/PMC2666844/"><img src="/pmc/articles/instance/2666844/bin/gcr1_4ap0013f5.gif" src-large="/pmc/articles/instance/2666844/bin/gcr1_4ap0013f5.jpg" alt="Figure 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2666844/" ref="ordinalpos=4074&amp;ncbi_uid=1827685&amp;link_uid=PMC2666844" image-link="/pmc/articles/PMC2666844/figure/f5-gcr1_4ap0013/" class="imagepopup">Figure 5. From: Liver Stem Cells and Molecular <span class="highlight" style="background-color:">Signaling</span> Pathways in Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>Simplified schematic of hepatocyte growth factor (HGF)/MET <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activation of phosphorylation of the kinase domain in cMET, a receptor tyrosine kinase (RTK), by HGF results in activation of adaptor proteins: growth factor receptor-bound protein2 (Grb2) and Grb2-associated binding protein (Grb2 and Gab1). Activation of these adaptor proteins leads to activation of various downstream effectors resulting in transcription of various genes important in mitogenesis, angiogenesis, and morphogenesis.</div></p></div><div class="aux"><div class="resc">Krit Kitisin, et al. Gastrointest Cancer Res. 2007 Jul-Aug;1(4 Suppl 2):S13-S21.</div><p class="links"><a class="dblinks" href="/pubmed/19360142" ref="ordinalpos=4074&amp;ncbi_uid=1827685&amp;link_uid=19360142">Citation</a><a class="dblinks" href="/pmc/articles/PMC2666844/" ref="ordinalpos=4074&amp;ncbi_uid=1827685&amp;link_uid=PMC2666844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4075.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3219658/figure/pone-0026960-g005/" ref="ordinalpos=4075&amp;ncbi_uid=3672160&amp;link_uid=PMC3219658" href="/pmc/articles/PMC3219658/"><img src="/pmc/articles/instance/3219658/bin/pone.0026960.g005.gif" src-large="/pmc/articles/instance/3219658/bin/pone.0026960.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3219658/" ref="ordinalpos=4075&amp;ncbi_uid=3672160&amp;link_uid=PMC3219658" image-link="/pmc/articles/PMC3219658/figure/pone-0026960-g005/" class="imagepopup">Figure 5. Involvement of host-parasite interactions in <span class="highlight" style="background-color:">signaling</span> pathways..  From: Computational Prediction of Host-Parasite Protein Interactions between P. falciparum and H. sapiens. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) In a bipartite matrix a parasite protein was linked to a <span class="highlight" style="background-color:">pathway</span> if the corresponding targeted proteins were enriched in the given <span class="highlight" style="background-color:">pathway</span> (FDR&lt;0.05, Fisher's exact test). Utilizing the combined set of host-parasite interactions the matrix was composed of 83 parasite proteins that interacted with proteins in 117 pathways. In addition, a large fraction of pathways had &gt;95% of their proteins being expressed in liver. Specifically, a cluster of prominent <span class="highlight" style="background-color:">signaling</span> pathways emerged that were largely under the control of parasite specific chaperone proteins. In (<b>B</b>) all interactions between parasite chaperones and host proteins that appeared in the cluster of pathways were mapped. Significantly, such a network revolves around parasite proteins PF08_0054 and PFI0875w that garnered most of the host-parasite interactions and largely interfered with proteins of the TNF <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Stefan Wuchty. PLoS One. 2011;6(11):e26960.</div><p class="links"><a class="dblinks" href="/pubmed/22114664" ref="ordinalpos=4075&amp;ncbi_uid=3672160&amp;link_uid=22114664">Citation</a><a class="dblinks" href="/pmc/articles/PMC3219658/" ref="ordinalpos=4075&amp;ncbi_uid=3672160&amp;link_uid=PMC3219658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4076.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC309600/figure/f9/" ref="ordinalpos=4076&amp;ncbi_uid=361882&amp;link_uid=PMC309600" href="/pmc/articles/PMC309600/"><img src="/pmc/articles/instance/309600/bin/mb2430106009.gif" src-large="/pmc/articles/instance/309600/bin/mb2430106009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC309600/" ref="ordinalpos=4076&amp;ncbi_uid=361882&amp;link_uid=PMC309600" image-link="/pmc/articles/PMC309600/figure/f9/" class="imagepopup">FIG. 9. From: Smad6 Recruits Transcription Corepressor CtBP To Repress Bone Morphogenetic Protein-Induced Transcription. </a></p><div class="supp"><p class="details"><div>Working model for Smad6 functions in BMP <span class="highlight" style="background-color:">signaling</span>. The BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is illustrated with points of Smad6 action. Smad6, an immediate-early gene product of BMP <span class="highlight" style="background-color:">signaling</span>, inhibits R-Smad activation and R-Smad-Smad4 heteromerization and directly represses transcription in the nucleus. RI, BMP type I receptor; RII, BMP type II receptor; GTF, general transcription factors such as TATA-binding proteins and associated factors.</div></p></div><div class="aux"><div class="resc">Xia Lin, et al. Mol Cell Biol. 2003 December;23(24):9081-9093.</div><p class="links"><a class="dblinks" href="/pubmed/14645520" ref="ordinalpos=4076&amp;ncbi_uid=361882&amp;link_uid=14645520">Citation</a><a class="dblinks" href="/pmc/articles/PMC309600/" ref="ordinalpos=4076&amp;ncbi_uid=361882&amp;link_uid=PMC309600">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4077.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3243074/figure/F3/" ref="ordinalpos=4077&amp;ncbi_uid=4483141&amp;link_uid=PMC3243074" href="/pmc/articles/PMC3243074/"><img src="/pmc/articles/instance/3243074/bin/nihms335978f3.gif" src-large="/pmc/articles/instance/3243074/bin/nihms335978f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3243074/" ref="ordinalpos=4077&amp;ncbi_uid=4483141&amp;link_uid=PMC3243074" image-link="/pmc/articles/PMC3243074/figure/F3/" class="imagepopup">Figure 3. Therapies modulating the autophagy lysosomal <span class="highlight" style="background-color:">pathway</span>.  From: AUTOPHAGY IN BRAIN TUMORS: A NEW TARGET FOR THERAPEUTIC INTERVENTION. </a></p><div class="supp"><p class="details"><div>Tyrosine kinase inhibitors (monoclonal antibodies and small molecules) block RTK/Ras <span class="highlight" style="background-color:">signaling</span> upstream of the PI3K-Akt-mTOR axis. Inhibitors for PI3K and/or mTOR act on their specific targets resulting in the inhibition of mTOR which induces autophagy. BH3 mimetics induce autophagy by disrupting the Beclin-Bcl-2 interaction. These agents when combined with lysosomotropic drugs result in the disruption of the autophagy-lysosomal <span class="highlight" style="background-color:">pathway</span>, leading to accumulation of autophagic vacuoles ultimately resulting in tumor cell death.</div></p></div><div class="aux"><div class="resc">Niroop Kaza, et al. Brain Pathol. ;22(1):89-98.</div><p class="links"><a class="dblinks" href="/pubmed/22150924" ref="ordinalpos=4077&amp;ncbi_uid=4483141&amp;link_uid=22150924">Citation</a><a class="dblinks" href="/pmc/articles/PMC3243074/" ref="ordinalpos=4077&amp;ncbi_uid=4483141&amp;link_uid=PMC3243074">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4078.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590361/figure/Fig1/" ref="ordinalpos=4078&amp;ncbi_uid=4709289&amp;link_uid=PMC3590361" href="/pmc/articles/PMC3590361/"><img src="/pmc/articles/instance/3590361/bin/12079_2012_184_Fig1_HTML.gif" src-large="/pmc/articles/instance/3590361/bin/12079_2012_184_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590361/" ref="ordinalpos=4078&amp;ncbi_uid=4709289&amp;link_uid=PMC3590361" image-link="/pmc/articles/PMC3590361/figure/Fig1/" class="imagepopup">Fig. 1.  From: LPS and PAN-induced podocyte injury in an in vitro model of minimal change disease: changes in TLR profile. </a></p><div class="supp"><p class="details"><div>Schematic representation of major TLR <span class="highlight" style="background-color:">signaling</span> pathways. The toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> pathways converge through two major <span class="highlight" style="background-color:">signaling</span> molecules, MyD88 or TRIF. The MyD88-dependent <span class="highlight" style="background-color:">pathway</span> activates NF-B and transcription of inflammatory cytokines. The TRIF <span class="highlight" style="background-color:">pathway</span> leads to transcription of type I Interferons. [MyD88: myeloid differentiation primary-response protein 88, IFN: Interferon, IRF-3: IFN-regulatory factor-3, NF-B: Nuclear factor-B, LRR: leucine rich repeats, TIR: Toll/interleukin-1 receptor, TRIF: TIR-domain-containing adaptor protein inducing IFN-]</div></p></div><div class="aux"><div class="resc">Tarak Srivastava, et al. J Cell Commun Signal. 2013 March;7(1):49-60.</div><p class="links"><a class="dblinks" href="/pubmed/23161414" ref="ordinalpos=4078&amp;ncbi_uid=4709289&amp;link_uid=23161414">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590361/" ref="ordinalpos=4078&amp;ncbi_uid=4709289&amp;link_uid=PMC3590361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4079.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2118122/figure/fig1/" ref="ordinalpos=4079&amp;ncbi_uid=1142546&amp;link_uid=PMC2118122" href="/pmc/articles/PMC2118122/"><img src="/pmc/articles/instance/2118122/bin/jem2032409f01.gif" src-large="/pmc/articles/instance/2118122/bin/jem2032409f01.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2118122/" ref="ordinalpos=4079&amp;ncbi_uid=1142546&amp;link_uid=PMC2118122" image-link="/pmc/articles/PMC2118122/figure/fig1/" class="imagepopup">Figure 1. From: Giving blood: a new role for CD40 in tumorigenesis. </a></p><div class="supp"><p class="details"><div><b>CD40 <span class="highlight" style="background-color:">signaling</span> regulates angiogenesis.</b> (A) CD40-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in endothelial cells. Interaction of CD40L with CD40 on the surface of endothelial cells results in activation of the Ras <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Ras associates with PI3 to induce the expression of VEGF, which in turn stimulates angiogenesis. Platelets are likely to be a major, but not unique, source of CD40L (see The two faces of CD40). (B) Platelets induce neoangiogenesis. Activation of platelets causes them to release a soluble form of CD40L, which can bind CD40 on the surface of endothelial cells. Stimulation of CD40 on endothelial cells activates them and, in a tumor, leads to neovascularization of the tumor. The antiplatelet drug Clopidogrel blocks CD40L secretion by platelets and thereby inhibits neovascularization.</div></p></div><div class="aux"><div class="resc">Stephan Bergmann, et al. J Exp Med. 2006 October 30;203(11):2409-2412.</div><p class="links"><a class="dblinks" href="/pubmed/17043147" ref="ordinalpos=4079&amp;ncbi_uid=1142546&amp;link_uid=17043147">Citation</a><a class="dblinks" href="/pmc/articles/PMC2118122/" ref="ordinalpos=4079&amp;ncbi_uid=1142546&amp;link_uid=PMC2118122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4080.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3153641/figure/fig10/" ref="ordinalpos=4080&amp;ncbi_uid=3417092&amp;link_uid=PMC3153641" href="/pmc/articles/PMC3153641/"><img src="/pmc/articles/instance/3153641/bin/JCB_201011077_RGB_Fig10.gif" src-large="/pmc/articles/instance/3153641/bin/JCB_201011077_RGB_Fig10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3153641/" ref="ordinalpos=4080&amp;ncbi_uid=3417092&amp;link_uid=PMC3153641" image-link="/pmc/articles/PMC3153641/figure/fig10/" class="imagepopup">Figure 10. From: FGF and retinoic acid activity gradients control the timing of neural crest cell emigration in the trunk. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> pathways involved in the control of NCC specification and emigration during trunk elongation.</b> The FGF <span class="highlight" style="background-color:">signaling</span> (pink), acting through the MAPK <span class="highlight" style="background-color:">pathway</span>, prevents the onset of NCC (green) specification in the caudal neural tube and, through the regulation of <i>Wnt1</i> onset of expression (pale red) and <i>Noggin</i> (orange) expression, controls the initiation of NCC emigration. The RA <span class="highlight" style="background-color:">signaling</span> (blue) controls the initiation of NCC emigration by modulating NCC genes (dashed arrow) or by controlling the onset of <i>Wnt1</i> expression. The BMP <span class="highlight" style="background-color:">signaling</span> gradient (pale green) promotes the onset and maintenance of NCC emigration through the control of <i>Wnt1</i> expression (gray lines; data from Burstyn-Cohen et al., 2004).</div></p></div><div class="aux"><div class="resc">Patricia L. Martnez-Morales, et al. J Cell Biol. 2011 August 8;194(3):489-503.</div><p class="links"><a class="dblinks" href="/pubmed/21807879" ref="ordinalpos=4080&amp;ncbi_uid=3417092&amp;link_uid=21807879">Citation</a><a class="dblinks" href="/pmc/articles/PMC3153641/" ref="ordinalpos=4080&amp;ncbi_uid=3417092&amp;link_uid=PMC3153641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4081.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4241914/figure/f4-jenb-17-4-151/" ref="ordinalpos=4081&amp;ncbi_uid=7043127&amp;link_uid=PMC4241914" href="/pmc/articles/PMC4241914/"><img src="/pmc/articles/instance/4241914/bin/jenb-17-4-151f4.gif" src-large="/pmc/articles/instance/4241914/bin/jenb-17-4-151f4.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4241914/" ref="ordinalpos=4081&amp;ncbi_uid=7043127&amp;link_uid=PMC4241914" image-link="/pmc/articles/PMC4241914/figure/f4-jenb-17-4-151/" class="imagepopup">Fig. 4. From: Neuroprotective effects of treadmill exercise on BDNF and PI3-K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the cortex of transgenic mice model of Alzheimers disease. </a></p><div class="supp"><p class="details"><div>Treadmill exercise increase BDNF, PI3-K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and HSP70 in the cortex of Tg-NSE/hPS2m mice. (A) Expression of BDNF, p-PI3K/t-PI3K, p-Akt/t-Akt and in the cortex were analyzed by Western blot. BDNF, PI3K/Akt and HSP70 differed significantly between non-Tg and Tg mice in the cortex (B-E). Treadmill exercised mice showed a significant increase in levels of cortex BDNF (B), p-PI3K/t-PI3K ratio (C), p-Akt/t-Akt ratio (E) and HSP70 proteins (D) compared to control mice. <sup>a</sup>p &lt; .05, <sup>d</sup>p &lt; .001 vs. non-Tg-control; <sup>b</sup>p &lt; .05, <sup>e</sup>p &lt; .001 vs. non-Tg-exercise; <sup>c</sup>p &lt; .05, <sup>f</sup>p &lt; .001 vs. Tg-control mice. GAPDH was probed as an internal control. Values are the mean  SEM.</div></p></div><div class="aux"><div class="resc">Jung Hoon Koo, et al. J Exerc Nutrition Biochem. 2013 December;17(4):151-160.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4081&amp;ncbi_uid=7043127&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4241914/" ref="ordinalpos=4081&amp;ncbi_uid=7043127&amp;link_uid=PMC4241914">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4082.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3382585/figure/pone-0039738-g009/" ref="ordinalpos=4082&amp;ncbi_uid=4103169&amp;link_uid=PMC3382585" href="/pmc/articles/PMC3382585/"><img src="/pmc/articles/instance/3382585/bin/pone.0039738.g009.gif" src-large="/pmc/articles/instance/3382585/bin/pone.0039738.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3382585/" ref="ordinalpos=4082&amp;ncbi_uid=4103169&amp;link_uid=PMC3382585" image-link="/pmc/articles/PMC3382585/figure/pone-0039738-g009/" class="imagepopup">Figure 9. Uric acid activates NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in kidney of hyperuricemia mice..  From: Uric Acid Induces Renal Inflammation via Activating Tubular NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(a) Western blot demonstrated that uric acid induced p65 NF-B phosphorylation and activation in kidney of hyperuricemia mice. (b) Western blot showed that uric acid induced IB phosphorylation <i>in vivo</i>. (c) Graphic presentation of relative phospho-p65 NF-B and phospho-IB protein abundance normalized to p65 NF-B and IB, respectively. *<i>P</i>&lt;0.05 versus control (<i>n</i>=3). (d) Immunofluorescent staining demonstrated that p65 NF-B underwent nuclear translocation in kidney of hyperuricemia mice. Kidney sections were immunostained for total p65 (red) and the nuclei (blue). Arrowheads indicate nuclear staining of p65 NF-B. (e) Enlarged image shows nuclear translocation of p65 NF-B in tubules of hyperuricemia mice (day 7) demonstrated by immunofluorescent staining.</div></p></div><div class="aux"><div class="resc">Yang Zhou, et al. PLoS One. 2012;7(6):e39738.</div><p class="links"><a class="dblinks" href="/pubmed/22761883" ref="ordinalpos=4082&amp;ncbi_uid=4103169&amp;link_uid=22761883">Citation</a><a class="dblinks" href="/pmc/articles/PMC3382585/" ref="ordinalpos=4082&amp;ncbi_uid=4103169&amp;link_uid=PMC3382585">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4083.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3327736/figure/F5/" ref="ordinalpos=4083&amp;ncbi_uid=4192834&amp;link_uid=PMC3327736" href="/pmc/articles/PMC3327736/"><img src="/pmc/articles/instance/3327736/bin/nihms361687f5.gif" src-large="/pmc/articles/instance/3327736/bin/nihms361687f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3327736/" ref="ordinalpos=4083&amp;ncbi_uid=4192834&amp;link_uid=PMC3327736" image-link="/pmc/articles/PMC3327736/figure/F5/" class="imagepopup">Figure 5. Genetic perturbations tie the Msn2/4 noise regulon to PKA <span class="highlight" style="background-color:">signaling</span> and demonstrate that mean expression of a <span class="highlight" style="background-color:">pathway</span> and covariance within a <span class="highlight" style="background-color:">pathway</span> have a complex relationship.  From: Cellular Noise Regulons Underlie Fluctuations in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>(a) Noise-induced covariance between Hsp12-RFP and Pgm2-YFP is altered in heterozygous deletes of PKA <span class="highlight" style="background-color:">pathway</span> members, showing significant decreases in <i>PDE2/pde2, IRA2/ira2,</i> and <i>GBP1/gbp1</i> strains and an increase in the <i>RAS2/ras2</i> strain. Error bars represent standard error of means (n=6)<br />(b) Covariance of Pgm2 and Arg4 as a function of mean Pgm2 expression as the levels of Msn2 are varied. Overexpression of constitutively active Msn2 to various levels results in a log-linear increase in covariance between Arg4 and Pgm2 as a function of Pgm2, and non-loglinear increase in covariance between Pgm2 and Hsp12, two proteins whose genes promoters are targets of Msn2. Error bars represent standard error of means (n=3).</div></p></div><div class="aux"><div class="resc">Jacob Stewart-Ornstein, et al. Mol Cell. ;45(4):483-493.</div><p class="links"><a class="dblinks" href="/pubmed/22365828" ref="ordinalpos=4083&amp;ncbi_uid=4192834&amp;link_uid=22365828">Citation</a><a class="dblinks" href="/pmc/articles/PMC3327736/" ref="ordinalpos=4083&amp;ncbi_uid=4192834&amp;link_uid=PMC3327736">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4084.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193404/figure/fig1/" ref="ordinalpos=4084&amp;ncbi_uid=3770446&amp;link_uid=PMC3193404" href="/pmc/articles/PMC3193404/"><img src="/pmc/articles/instance/3193404/bin/cr2010168f1.gif" src-large="/pmc/articles/instance/3193404/bin/cr2010168f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193404/" ref="ordinalpos=4084&amp;ncbi_uid=3770446&amp;link_uid=PMC3193404" image-link="/pmc/articles/PMC3193404/figure/fig1/" class="imagepopup">Figure 1.  From: Regulation of NF-?B <span class="highlight" style="background-color:">signaling</span> by caspases and MALT1 paracaspase. </a></p><div class="supp"><p class="details"><div>NF-B <span class="highlight" style="background-color:">signaling</span> pathways initiated by TLR4, TCR, TNF receptor 1, CD40 and genotoxic stress. Left: canonical NF-B <span class="highlight" style="background-color:">signaling</span> is initiated by the inducible formation of multiprotein complexes that can undergo non-degradative polyubiquitination (K63-linked or linear) and which are recognized by TAK1 and IKK kinase complexes. This leads to the activation of IKK, which phosphorylates the NF-B inhibitor protein IB, resulting in its proteasomal degradation and nuclear translocation of NF-B (depicted as p50/p65). TNF receptor 1 can also initiate the formation of a caspase-8-containing complex that signals apoptosis. The places where caspases have been proposed to regulate NF-B <span class="highlight" style="background-color:">signaling</span> are indicated. Middle: some receptors initiate a non-canonical NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that results in the activation of IKK, which phosphorylates the NF-B inhibitor p100, resulting in its processing to p52 and the formation of p52/RelB NF-B complexes. Receptors initiating this <span class="highlight" style="background-color:">pathway</span> also initiate the formation of multiprotein complexes and ubiquitination (not shown in the figure), which results in the activation of the kinase NIK, which activates IKK. Right: NF-B activation by genotoxic stress results in the activation of a multiprotein complex (PIDDosome) in which the IKK adaptor NEMO is recruited. This is the beginning of a sequential series of modifications on NEMO, which alter its subcellular localization, providing a means to link genotoxic stress to the cytosolic IKK complex. An alternative PIDDosome complex recruits caspase-2 and initiates apoptosis.</div></p></div><div class="aux"><div class="resc">Jens Staal, et al. Cell Res. 2011 January;21(1):40-54.</div><p class="links"><a class="dblinks" href="/pubmed/21119681" ref="ordinalpos=4084&amp;ncbi_uid=3770446&amp;link_uid=21119681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193404/" ref="ordinalpos=4084&amp;ncbi_uid=3770446&amp;link_uid=PMC3193404">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4085.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4381337/figure/f2-0080385/" ref="ordinalpos=4085&amp;ncbi_uid=7419566&amp;link_uid=PMC4381337" href="/pmc/articles/PMC4381337/"><img src="/pmc/articles/instance/4381337/bin/DMM018887F2.gif" src-large="/pmc/articles/instance/4381337/bin/DMM018887F2.jpg" alt="Fig. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4381337/" ref="ordinalpos=4085&amp;ncbi_uid=7419566&amp;link_uid=PMC4381337" image-link="/pmc/articles/PMC4381337/figure/f2-0080385/" class="imagepopup">Fig. 2. From: Downregulation of Wnt <span class="highlight" style="background-color:">signaling</span> by sonic hedgehog activation promotes repopulation of human tumor cell lines. </a></p><div class="supp"><p class="details"><div><b>Evidence for Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibition in irradiated Panc1 and HT29 cells.</b> (A) Wnt reporter luciferase activity in irradiated and non-irradiated Panc1 cells. **<i>P</i>&lt;0.01. (B) Wnt reporter luciferase activity in irradiated and non-irradiated HT29 cells. **<i>P</i>&lt;0.01. Results are means.d. (C) Nuclear -catenin protein levels in Panc1 cells that were irradiated at 6 Gy or were not irradiated. (D) Nuclear -catenin protein levels in HT29 cells that were irradiated at 6 Gy or were not irradiated. Lamin B1 serves as a control.</div></p></div><div class="aux"><div class="resc">Jingjing Ma, et al. Dis Model Mech. 2015 April;8(4):385-391.</div><p class="links"><a class="dblinks" href="/pubmed/25713298" ref="ordinalpos=4085&amp;ncbi_uid=7419566&amp;link_uid=25713298">Citation</a><a class="dblinks" href="/pmc/articles/PMC4381337/" ref="ordinalpos=4085&amp;ncbi_uid=7419566&amp;link_uid=PMC4381337">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4086.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3668741/figure/F1/" ref="ordinalpos=4086&amp;ncbi_uid=6181722&amp;link_uid=PMC3668741" href="/pmc/articles/PMC3668741/"><img src="/pmc/articles/instance/3668741/bin/zbc0271351460001.gif" src-large="/pmc/articles/instance/3668741/bin/zbc0271351460001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3668741/" ref="ordinalpos=4086&amp;ncbi_uid=6181722&amp;link_uid=PMC3668741" image-link="/pmc/articles/PMC3668741/figure/F1/" class="imagepopup">FIGURE 1. From: The Intermediate Conductance Calcium-activated Potassium Channel KCa3.1 Regulates Vascular Smooth Muscle Cell Proliferation via Controlling Calcium-dependent <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>KCa3.1 regulates PDGF-induced rise in [Ca<sup>2+</sup>]<i><sub>i</sub></i> and <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation.</b> <i>A</i> and <i>B</i>, lowering [Ca<sup>2+</sup>]<i><sub>i</sub></i> with BAPTA suppressed (<i>A</i>) and a forced rise in [Ca<sup>2+</sup>]<i><sub>i</sub></i> with A23187 enhanced (<i>B</i>) PDGF-induced proliferation of HCSMCs. <i>C</i> and <i>D</i>, both blockade with TRAM-34 and siRNA knockdown of KCa3.1 reduced PDGF-induced rise in [Ca<sup>2+</sup>]<i><sub>i</sub></i> (<i>C</i>) and inhibited PDGF-induced expression of c-Fos and NOR-1 (<i>D</i>). *, p &lt; 0.05 <i>versus</i> control, and , p &lt; 0.05 <i>versus</i> PDGF alone. neg. siRNA, negative control siRNA.</div></p></div><div class="aux"><div class="resc">Dan Bi, et al. J Biol Chem. 2013 May 31;288(22):15843-15853.</div><p class="links"><a class="dblinks" href="/pubmed/23609438" ref="ordinalpos=4086&amp;ncbi_uid=6181722&amp;link_uid=23609438">Citation</a><a class="dblinks" href="/pmc/articles/PMC3668741/" ref="ordinalpos=4086&amp;ncbi_uid=6181722&amp;link_uid=PMC3668741">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4087.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3688259/figure/F2/" ref="ordinalpos=4087&amp;ncbi_uid=5916635&amp;link_uid=PMC3688259" href="/pmc/articles/PMC3688259/"><img src="/pmc/articles/instance/3688259/bin/nihms435303f2.gif" src-large="/pmc/articles/instance/3688259/bin/nihms435303f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3688259/" ref="ordinalpos=4087&amp;ncbi_uid=5916635&amp;link_uid=PMC3688259" image-link="/pmc/articles/PMC3688259/figure/F2/" class="imagepopup">Figure 2.  From: Caveolin-1-LRP6 <span class="highlight" style="background-color:">Signaling</span> Module Stimulates Aerobic Glycolysis in Prostate Cancer. </a></p><div class="supp"><p class="details"><div>Cav-1 stimulation of the Wnt-cat <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> A, Cav-1 was overexpressed in LNCaP cells after infection with AdCa . B, PC-3 cells were treated with Cavsi or control NCsi. Both LNCaP and PC-3 cells were2 incubated for 48 hours and treated with rhWnt3a (25 ng/mL) for the indicated times. C, Cav-1 knockdown (Cavsh11 and Cavsh11G) and NC stable clones of DU145 cells were treated with rhWnt3a (25 ng/mL) for 45 minutes. D, PC-3 and LNCaP cells treated with rhWnt3a (50 ng/mL) and rhDKK1 (100 ng/mL) for 1 hour.</div></p></div><div class="aux"><div class="resc">Salahaldin A. Tahir, et al. Cancer Res. ;73(6):1900-1911.</div><p class="links"><a class="dblinks" href="/pubmed/23302227" ref="ordinalpos=4087&amp;ncbi_uid=5916635&amp;link_uid=23302227">Citation</a><a class="dblinks" href="/pmc/articles/PMC3688259/" ref="ordinalpos=4087&amp;ncbi_uid=5916635&amp;link_uid=PMC3688259">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4088.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3613228/figure/F1/" ref="ordinalpos=4088&amp;ncbi_uid=4767049&amp;link_uid=PMC3613228" href="/pmc/articles/PMC3613228/"><img src="/pmc/articles/instance/3613228/bin/nihms-443725-f0001.gif" src-large="/pmc/articles/instance/3613228/bin/nihms-443725-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3613228/" ref="ordinalpos=4088&amp;ncbi_uid=4767049&amp;link_uid=PMC3613228" image-link="/pmc/articles/PMC3613228/figure/F1/" class="imagepopup">Fig. 1.  From: Rapamycin Reverses Elevated mTORC1 <span class="highlight" style="background-color:">Signaling</span> in Lamin A/C-Deficient Mice, Rescues Cardiac and Skeletal Muscle Function, and Extends Survival. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> through the mTORC1 <span class="highlight" style="background-color:">pathway</span> is increased in <i>Lmna</i><sup>/</sup> mice. (<b>A</b>) Western blot analysis of <i>Lmna</i><sup>+/+</sup> (<i>n</i> = 7) and <i>Lmna</i><sup>/</sup> (<i>n</i> = 8) mice heart tissue lysates. Phosphorylated mTOR<sup>(S2448)</sup>, S6 kinase<sup>(T389)</sup>, rpS6<sup>(S235/S236)</sup>, and 4E-BP1<sup>(S65)</sup> are increased in <i>Lmna</i><sup>/</sup> mice compared to <i>Lmna</i><sup>+/+</sup> mice. (<b>B</b>) Western blot analysis of <i>Lmna</i><sup>+/+</sup> (<i>n</i> = 7) and <i>Lmna</i><sup>/</sup> (<i>n</i> = 8) mice skeletal muscle (quadriceps) tissue lysates. Phosphorylated rpS6<sup>(S235/S236)</sup> and 4E-BP1<sup>(S65)</sup> are increased in <i>Lmna</i><sup>/</sup> mice compared to <i>Lmna</i><sup>+/+</sup> mice. However, phosphorylated S6 kinase<sup>(T389)</sup> is not significantly increased. *<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01.</div></p></div><div class="aux"><div class="resc">Fresnida J. Ramos, et al. Sci Transl Med. 2012 July 25;4(144):144ra103-144ra103.</div><p class="links"><a class="dblinks" href="/pubmed/22837538" ref="ordinalpos=4088&amp;ncbi_uid=4767049&amp;link_uid=22837538">Citation</a><a class="dblinks" href="/pmc/articles/PMC3613228/" ref="ordinalpos=4088&amp;ncbi_uid=4767049&amp;link_uid=PMC3613228">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4089.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2768835/figure/fig7/" ref="ordinalpos=4089&amp;ncbi_uid=2142815&amp;link_uid=PMC2768835" href="/pmc/articles/PMC2768835/"><img src="/pmc/articles/instance/2768835/bin/JCB_200905118_RGB_Fig7.gif" src-large="/pmc/articles/instance/2768835/bin/JCB_200905118_RGB_Fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2768835/" ref="ordinalpos=4089&amp;ncbi_uid=2142815&amp;link_uid=PMC2768835" image-link="/pmc/articles/PMC2768835/figure/fig7/" class="imagepopup">Figure 7. From: Spred2 interaction with the late endosomal protein NBR1 down-regulates fibroblast growth factor receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Schematic representation of the proposed mechanism for Spred.</b> Via interacting with NBR1, Spred redirects activated receptors to the lysosomal degradation <span class="highlight" style="background-color:">pathway</span>. Down-regulation of downstream <span class="highlight" style="background-color:">signaling</span> can be selective, as those pathways like ERK1/2 that need specific endosomal compartments for efficient activation are affected the most from a shift in the balance between different trafficking routes (e.g., sorting/recycling versus lysosomal).</div></p></div><div class="aux"><div class="resc">Faraz K. Mardakheh, et al. J Cell Biol. 2009 October 19;187(2):265-277.</div><p class="links"><a class="dblinks" href="/pubmed/19822672" ref="ordinalpos=4089&amp;ncbi_uid=2142815&amp;link_uid=19822672">Citation</a><a class="dblinks" href="/pmc/articles/PMC2768835/" ref="ordinalpos=4089&amp;ncbi_uid=2142815&amp;link_uid=PMC2768835">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4090.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC262421/figure/f4/" ref="ordinalpos=4090&amp;ncbi_uid=300070&amp;link_uid=PMC262421" href="/pmc/articles/PMC262421/"><img src="/pmc/articles/instance/262421/bin/mb2231072004.gif" src-large="/pmc/articles/instance/262421/bin/mb2231072004.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC262421/" ref="ordinalpos=4090&amp;ncbi_uid=300070&amp;link_uid=PMC262421" image-link="/pmc/articles/PMC262421/figure/f4/" class="imagepopup">FIG. 4. From: Discrete Functions of TRAF1 and TRAF2 in Drosophila melanogaster Mediated by c-Jun N-Terminal Kinase and NF-?B-Dependent <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div>DTRAF1 induces activation of the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and apoptosis. (A and B) The eye imaginal discs were immunostained with an anti-phospho-specific JNK antibody as described in Materials and Methods. (A) <i>gmr</i>-GAL4/+. (B) <i>gmr</i>-GAL4, EP(2)0578/+. (C and D) <i>puckered</i>-LacZ reporter assays were also conducted in the eye discs. (C) <i>gmr-</i>GAL4<i>/+</i>; <i>puckered</i>-LacZ<i>/+</i>. (D) <i>gmr</i>-GAL4, EP(2)0578/+; <i>puckered</i>-LacZ/+. (E to H) DTRAF1-induced apoptosis in the eye discs was examined by TUNEL assays. (E) <i>gmr</i>-GAL4/+. (F) <i>gmr</i>-GAL4, EP(2)0578/+. (G) <i>gmr</i>-GAL4, EP(2)0578/EP(2)0578. (H) <i>hep<sup>1</sup></i>/Y; <i>gmr-</i>GAL4, EP(2)0578/+.</div></p></div><div class="aux"><div class="resc">Guang-Ho Cha, et al. Mol Cell Biol. 2003 November;23(22):7982-7991.</div><p class="links"><a class="dblinks" href="/pubmed/14585960" ref="ordinalpos=4090&amp;ncbi_uid=300070&amp;link_uid=14585960">Citation</a><a class="dblinks" href="/pmc/articles/PMC262421/" ref="ordinalpos=4090&amp;ncbi_uid=300070&amp;link_uid=PMC262421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4091.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC137942/figure/F1/" ref="ordinalpos=4091&amp;ncbi_uid=341581&amp;link_uid=PMC137942" href="/pmc/articles/PMC137942/"><img src="/pmc/articles/instance/137942/bin/bcr543-1.gif" src-large="/pmc/articles/instance/137942/bin/bcr543-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC137942/" ref="ordinalpos=4091&amp;ncbi_uid=341581&amp;link_uid=PMC137942" image-link="/pmc/articles/PMC137942/figure/F1/" class="imagepopup">Figure 1.  From: Prolactin <span class="highlight" style="background-color:">signaling</span> and Stat5: going their own separate ways?. </a></p><div class="supp"><p class="details"><div>Interdependence of prolactin receptor (PrlR) and signal transducer and activator of transcription 5 (Stat5) <span class="highlight" style="background-color:">signaling</span>. <b>(a)</b> The 'classical' PrlR-Jak2-Stat5 <span class="highlight" style="background-color:">pathway</span> contributes to alveolar development. <b>(b)</b> Alternative routes to activate Stat5 are required for alveologenesis. <b>(c)</b> Potential contribution of multiple PrlR-dependent pathways to the proliferative response to estrogen and progesterone. PI-3K, PI 3-kinase; MAPK, mitogen-activated protein kinase.</div></p></div><div class="aux"><div class="resc">Cathrin Brisken, et al. Breast Cancer Res. 2002;4(6):209-212.</div><p class="links"><a class="dblinks" href="/pubmed/12473164" ref="ordinalpos=4091&amp;ncbi_uid=341581&amp;link_uid=12473164">Citation</a><a class="dblinks" href="/pmc/articles/PMC137942/" ref="ordinalpos=4091&amp;ncbi_uid=341581&amp;link_uid=PMC137942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4092.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2811393/figure/F2/" ref="ordinalpos=4092&amp;ncbi_uid=2148080&amp;link_uid=PMC2811393" href="/pmc/articles/PMC2811393/"><img src="/pmc/articles/instance/2811393/bin/nihms169267f2.gif" src-large="/pmc/articles/instance/2811393/bin/nihms169267f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2811393/" ref="ordinalpos=4092&amp;ncbi_uid=2148080&amp;link_uid=PMC2811393" image-link="/pmc/articles/PMC2811393/figure/F2/" class="imagepopup">Figure 2. p110 is the predominant mediator of PI3K <span class="highlight" style="background-color:">signaling</span> in breast cancer cells.  From: PIK3CA and PIK3CB inhibition produce synthetic lethality when combined with estrogen deprivation in estrogen receptor positive breast cancer. </a></p><div class="supp"><p class="details"><div><i>A</i>, effect of p110 and p110 knockdown on PI3K <span class="highlight" style="background-color:">signaling</span>. Cells were transfected with control siRNAs (<i>Control</i>) or siRNAs against <i>PIK3CB</i> (<i>PIK3CB</i>) or <i>PIK3CA</i> (<i>PIK3CA</i>). Three days after transfection, serum-deprived cells were stimulated with 20% FBS (final concentration) and lysates were analyzed for effects on PI3K <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> through phospho-Akt (<i>p-Akt</i>) and phospho-S6 (<i>p-S6</i>) immunoblotting. Shown are representative immunoblots obtained from at least three experiments per cell line. <i>B,</i> effect of p110/p110 dual knockdown on PI3K <span class="highlight" style="background-color:">signaling</span>. Cells were transfected with control siRNAs or a mixture of <i>PIK3CA</i> and <i>PIK3CB</i> siRNAs (<i>CA/CB</i>), treated as above and subjected to immunoblot analysis. Representative results obtained in at least two experiments per cell line are shown.</div></p></div><div class="aux"><div class="resc">Robert J. Crowder, et al. Cancer Res. ;69(9):3955-3962.</div><p class="links"><a class="dblinks" href="/pubmed/19366795" ref="ordinalpos=4092&amp;ncbi_uid=2148080&amp;link_uid=19366795">Citation</a><a class="dblinks" href="/pmc/articles/PMC2811393/" ref="ordinalpos=4092&amp;ncbi_uid=2148080&amp;link_uid=PMC2811393">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4093.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950298/figure/pone-0091730-g009/" ref="ordinalpos=4093&amp;ncbi_uid=5906987&amp;link_uid=PMC3950298" href="/pmc/articles/PMC3950298/"><img src="/pmc/articles/instance/3950298/bin/pone.0091730.g009.gif" src-large="/pmc/articles/instance/3950298/bin/pone.0091730.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950298/" ref="ordinalpos=4093&amp;ncbi_uid=5906987&amp;link_uid=PMC3950298" image-link="/pmc/articles/PMC3950298/figure/pone-0091730-g009/" class="imagepopup">Figure 9. Model of PC1 mediating Akt-GSK-3--catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in osteoblast mechanotransduction..  From: Polycystin-1 Mediates Mechanical Strain-Induced Osteoblastic Mechanoresponses via Potentiation of Intracellular Calcium and Akt/?-Catenin <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>PC1 functions as a mechanosensory molecule that senses external mechanical stimuli, thereby converting these mechanical signals into potentiation of intracellular calcium. Then calcium-dependent Akt phosphorylates and then leads to the phosphorylation of GSK-3. The phospho-GSK-3 inhibits -catenin for proteolysis, which leads to active -catenin nuclear translocation and transcriptional regulation of genes involved in osteoblastic proliferation and differentiation such as Runx2.</div></p></div><div class="aux"><div class="resc">Hua Wang, et al. PLoS One. 2014;9(3):e91730.</div><p class="links"><a class="dblinks" href="/pubmed/24618832" ref="ordinalpos=4093&amp;ncbi_uid=5906987&amp;link_uid=24618832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950298/" ref="ordinalpos=4093&amp;ncbi_uid=5906987&amp;link_uid=PMC3950298">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4094.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3868569/figure/pgen-1004020-g001/" ref="ordinalpos=4094&amp;ncbi_uid=5645561&amp;link_uid=PMC3868569" href="/pmc/articles/PMC3868569/"><img src="/pmc/articles/instance/3868569/bin/pgen.1004020.g001.gif" src-large="/pmc/articles/instance/3868569/bin/pgen.1004020.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3868569/" ref="ordinalpos=4094&amp;ncbi_uid=5645561&amp;link_uid=PMC3868569" image-link="/pmc/articles/PMC3868569/figure/pgen-1004020-g001/" class="imagepopup">Figure 1. Diagram of daf-2/IGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and tyrosine metabolic <span class="highlight" style="background-color:">pathway</span>..  From: TATN-1 Mutations Reveal a Novel Role for Tyrosine as a Metabolic Signal That Influences Developmental Decisions and Longevity in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>(A) For dauer formation in <i>C. elegans</i> the <i>daf-2</i>/IGFR receptor lies upstream of a PI3 kinase <span class="highlight" style="background-color:">signaling</span> cascade consisting of <i>age-1</i>/PI3 kinase, the phosphoinoside-dependent kinase <i>pdk-1</i>, and the protein kinase B family kinases <i>akt-1</i> and <i>akt-2</i>. Both AKT-1 and AKT-2 act to phosphorylate DAF-16/FOXO and prevent entry of this protein into the nucleus. Inhibition of AKT-1 leads to entry of DAF-16/FOXO into the nucleus without the activation of DAF-16 target genes. This suggested that other pathways acted to control the transcriptional activity of nuclear DAF-16/FOXO. The <i>eak</i> genes are a group of structurally unrelated genes which act in a cell non-autonomous manner to restrain the transcriptional activity of nuclear DAF-16/FOXO through an undefined molecular mechanism. (B) Tyrosine is degraded to fumarate and acetoacetate via a five step degradation <span class="highlight" style="background-color:">pathway</span>. Shown are the names and structures of the intermediates as well as the names of the enzymes which catalyze each step. Also shown are mutant alleles affecting enzymes in the <span class="highlight" style="background-color:">pathway</span> that are studied in this work.</div></p></div><div class="aux"><div class="resc">Annabel A. Ferguson, et al. PLoS Genet. 2013 December;9(12):e1004020.</div><p class="links"><a class="dblinks" href="/pubmed/24385923" ref="ordinalpos=4094&amp;ncbi_uid=5645561&amp;link_uid=24385923">Citation</a><a class="dblinks" href="/pmc/articles/PMC3868569/" ref="ordinalpos=4094&amp;ncbi_uid=5645561&amp;link_uid=PMC3868569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4095.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3176120/figure/fig9/" ref="ordinalpos=4095&amp;ncbi_uid=3562398&amp;link_uid=PMC3176120" href="/pmc/articles/PMC3176120/"><img src="/pmc/articles/instance/3176120/bin/177fig9.gif" src-large="/pmc/articles/instance/3176120/bin/177fig9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3176120/" ref="ordinalpos=4095&amp;ncbi_uid=3562398&amp;link_uid=PMC3176120" image-link="/pmc/articles/PMC3176120/figure/fig9/" class="imagepopup">Figure 9.  From: CLAVATA <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Receptors of Arabidopsis Regulate Cell Proliferation in Fruit Organ Formation as well as in Meristems. </a></p><div class="supp"><p class="details"><div>Model for development of valves in the wild type, weak, intermediate, and strong CLV <span class="highlight" style="background-color:">pathway</span> mutants.</div></p></div><div class="aux"><div class="resc">Amanda R. Durbak, et al. Genetics. 2011 September;189(1):177-194.</div><p class="links"><a class="dblinks" href="/pubmed/21705761" ref="ordinalpos=4095&amp;ncbi_uid=3562398&amp;link_uid=21705761">Citation</a><a class="dblinks" href="/pmc/articles/PMC3176120/" ref="ordinalpos=4095&amp;ncbi_uid=3562398&amp;link_uid=PMC3176120">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4096.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2665046/figure/F2/" ref="ordinalpos=4096&amp;ncbi_uid=1804716&amp;link_uid=PMC2665046" href="/pmc/articles/PMC2665046/"><img src="/pmc/articles/instance/2665046/bin/nihms94225f2.gif" src-large="/pmc/articles/instance/2665046/bin/nihms94225f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2665046/" ref="ordinalpos=4096&amp;ncbi_uid=1804716&amp;link_uid=PMC2665046" image-link="/pmc/articles/PMC2665046/figure/F2/" class="imagepopup">Figure 2. The ubiquitin-proteasome system.  From: Multivalency-Assisted Control of Intracellular <span class="highlight" style="background-color:">Signaling</span> Pathways: Application for Ubiquitin-Dependent N-End Rule <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The substrates are ubiquitylated through multiple rounds of a linear reaction catalyzed by E1, E2 and E3. Shown as an example is the N-end rule <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Shashikanth M. Sriram, et al. Chem Biol. ;16(2):121-131.</div><p class="links"><a class="dblinks" href="/pubmed/19246002" ref="ordinalpos=4096&amp;ncbi_uid=1804716&amp;link_uid=19246002">Citation</a><a class="dblinks" href="/pmc/articles/PMC2665046/" ref="ordinalpos=4096&amp;ncbi_uid=1804716&amp;link_uid=PMC2665046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4097.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4330251/figure/F3/" ref="ordinalpos=4097&amp;ncbi_uid=7202746&amp;link_uid=PMC4330251" href="/pmc/articles/PMC4330251/"><img src="/pmc/articles/instance/4330251/bin/gni-12-171-g003.gif" src-large="/pmc/articles/instance/4330251/bin/gni-12-171-g003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4330251/" ref="ordinalpos=4097&amp;ncbi_uid=7202746&amp;link_uid=PMC4330251" image-link="/pmc/articles/PMC4330251/figure/F3/" class="imagepopup">Fig. 3.  From: In silico Identification of SFRP1 as a Hypermethylated Gene in Colorectal Cancers. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Kyoto Encyclopedia of Genes and Genomes (KEGG) (SFRP genes are in red box).</div></p></div><div class="aux"><div class="resc">Jongbum Kim, et al. Genomics Inform. 2014 December;12(4):171-180.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4097&amp;ncbi_uid=7202746&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4330251/" ref="ordinalpos=4097&amp;ncbi_uid=7202746&amp;link_uid=PMC4330251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4098.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4032276/figure/pone-0098185-g007/" ref="ordinalpos=4098&amp;ncbi_uid=6176687&amp;link_uid=PMC4032276" href="/pmc/articles/PMC4032276/"><img src="/pmc/articles/instance/4032276/bin/pone.0098185.g007.gif" src-large="/pmc/articles/instance/4032276/bin/pone.0098185.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4032276/" ref="ordinalpos=4098&amp;ncbi_uid=6176687&amp;link_uid=PMC4032276" image-link="/pmc/articles/PMC4032276/figure/pone-0098185-g007/" class="imagepopup">Figure 7. Schematic diagram illustrating the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involved in bupivacaine-induced WDR35 expression in Neuro2a cells..  From: Enhanced Expression of WD Repeat-Containing Protein 35 via CaMKK/AMPK Activation in Bupivacaine-Treated Neuro2a Cells. </a></p><div class="aux"><div class="resc">Lei Huang, et al. PLoS One. 2014;9(5):e98185.</div><p class="links"><a class="dblinks" href="/pubmed/24859235" ref="ordinalpos=4098&amp;ncbi_uid=6176687&amp;link_uid=24859235">Citation</a><a class="dblinks" href="/pmc/articles/PMC4032276/" ref="ordinalpos=4098&amp;ncbi_uid=6176687&amp;link_uid=PMC4032276">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4099.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4013644/figure/f3-ijms-15-06517/" ref="ordinalpos=4099&amp;ncbi_uid=6112046&amp;link_uid=PMC4013644" href="/pmc/articles/PMC4013644/"><img src="/pmc/articles/instance/4013644/bin/ijms-15-06517f3.gif" src-large="/pmc/articles/instance/4013644/bin/ijms-15-06517f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4013644/" ref="ordinalpos=4099&amp;ncbi_uid=6112046&amp;link_uid=PMC4013644" image-link="/pmc/articles/PMC4013644/figure/f3-ijms-15-06517/" class="imagepopup">Figure 3. From: The Biomolecular Basis of Adipogenic Differentiation of Adipose-Derived Stem Cells. </a></p><div class="supp"><p class="details"><div>Adipogenic differentiation of ASCs. Schematic representation of adipogenic differentiation of ASCs: FGFR-1 activity and ErbB2 <span class="highlight" style="background-color:">signaling</span> downregulation are involved in the upregulation of Akt <span class="highlight" style="background-color:">pathway</span> and downregulation of Erk-1 in adipogenic differentiation of ASCs.</div></p></div><div class="aux"><div class="resc">Maria Giovanna Scioli, et al. Int J Mol Sci. 2014 April;15(4):6517-6526.</div><p class="links"><a class="dblinks" href="/pubmed/24743893" ref="ordinalpos=4099&amp;ncbi_uid=6112046&amp;link_uid=24743893">Citation</a><a class="dblinks" href="/pmc/articles/PMC4013644/" ref="ordinalpos=4099&amp;ncbi_uid=6112046&amp;link_uid=PMC4013644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4100.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3366009/figure/F8/" ref="ordinalpos=4100&amp;ncbi_uid=4953844&amp;link_uid=PMC3366009" href="/pmc/articles/PMC3366009/"><img src="/pmc/articles/instance/3366009/bin/zbc0261211280008.gif" src-large="/pmc/articles/instance/3366009/bin/zbc0261211280008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3366009/" ref="ordinalpos=4100&amp;ncbi_uid=4953844&amp;link_uid=PMC3366009" image-link="/pmc/articles/PMC3366009/figure/F8/" class="imagepopup">FIGURE 8. From: Molecular Basis for T Cell Response Induced by Altered Peptide Ligand of Type II Collagen. </a></p><div class="supp"><p class="details"><div><b>Hypothetical model of T cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> used by A2 and A9.</b></div></p></div><div class="aux"><div class="resc">Jeoung-Eun Park, et al. J Biol Chem. 2012 June 1;287(23):19765-19774.</div><p class="links"><a class="dblinks" href="/pubmed/22511761" ref="ordinalpos=4100&amp;ncbi_uid=4953844&amp;link_uid=22511761">Citation</a><a class="dblinks" href="/pmc/articles/PMC3366009/" ref="ordinalpos=4100&amp;ncbi_uid=4953844&amp;link_uid=PMC3366009">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="40" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="41" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="42" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:45:26-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A3A6B5431296100000000007C4954;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A3A6B5431296100000000007C4954" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>