
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="40" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB72EE543121D10000000000E37DAB" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="39" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="40" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="41" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="40" /></div>
            <div><h2 class="result_count">Results: 3901 to 4000 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3901.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC139760/figure/fig8/" ref="ordinalpos=3901&amp;ncbi_uid=357921&amp;link_uid=PMC139760" href="/pmc/articles/PMC139760/"><img src="/pmc/articles/instance/139760/bin/99-0353f8.gif" src-large="/pmc/articles/instance/139760/bin/99-0353f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC139760/" ref="ordinalpos=3901&amp;ncbi_uid=357921&amp;link_uid=PMC139760" image-link="/pmc/articles/PMC139760/figure/fig8/" class="imagepopup">Figure 8. From: RPT2. </a></p><div class="supp"><p class="details"><div>Models of the <span class="highlight" style="background-color:">Signaling</span> Pathways of the Phototropic Response in Hypocotyls of Arabidopsis Seedlings.<br />The <span class="highlight" style="background-color:">pathway</span> marked by an asterisk indicates the inhibitory effect of NPH1 to the RPT2-independent <span class="highlight" style="background-color:">pathway</span> by adaptation or unknown mechanisms at high fluence rates. See the text for more details.</div></p></div><div class="aux"><div class="resc">Tatsuya Sakai, et al. Plant Cell. 2000 February;12(2):225-236.</div><p class="links"><a class="dblinks" href="/pubmed/10662859" ref="ordinalpos=3901&amp;ncbi_uid=357921&amp;link_uid=10662859">Citation</a><a class="dblinks" href="/pmc/articles/PMC139760/" ref="ordinalpos=3901&amp;ncbi_uid=357921&amp;link_uid=PMC139760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3902.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3047755/figure/F7/" ref="ordinalpos=3902&amp;ncbi_uid=3241433&amp;link_uid=PMC3047755" href="/pmc/articles/PMC3047755/"><img src="/pmc/articles/instance/3047755/bin/cc0919_4013_fig007.gif" src-large="/pmc/articles/instance/3047755/bin/cc0919_4013_fig007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3047755/" ref="ordinalpos=3902&amp;ncbi_uid=3241433&amp;link_uid=PMC3047755" image-link="/pmc/articles/PMC3047755/figure/F7/" class="imagepopup">Figure 7.  From: ?-Arrestin-1 links mitogenic sonic hedgehog <span class="highlight" style="background-color:">signaling</span> to the cell cycle exit machinery in neural precursors. </a></p><div class="supp"><p class="details"><div>Proposed model for how Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> negatively regulates its mitogenic potential through β-arrestin 1. Activation of Shh <span class="highlight" style="background-color:">signaling</span> leads to cell cycle progression via the upregulation/activation of Gli, N-Myc and IRS1. Shh also activates a negative feedback loop through its induction and phosphorylation of βArr1, mediating its nuclear translocation, where it enhances <i>p27</i> transcription in complex with CREB and p300. This sets the stage for accumulation of p27 to ultimately drive CGNP cell cycle exit.</div></p></div><div class="aux"><div class="resc">Susana R Parathath, et al. Cell Cycle. 2010 October 1;9(19):4013-4024.</div><p class="links"><a class="dblinks" href="/pubmed/20935513" ref="ordinalpos=3902&amp;ncbi_uid=3241433&amp;link_uid=20935513">Citation</a><a class="dblinks" href="/pmc/articles/PMC3047755/" ref="ordinalpos=3902&amp;ncbi_uid=3241433&amp;link_uid=PMC3047755">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3903.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3373245/figure/DDS129F4/" ref="ordinalpos=3903&amp;ncbi_uid=5041289&amp;link_uid=PMC3373245" href="/pmc/articles/PMC3373245/"><img src="/pmc/articles/instance/3373245/bin/dds12904.gif" src-large="/pmc/articles/instance/3373245/bin/dds12904.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3373245/" ref="ordinalpos=3903&amp;ncbi_uid=5041289&amp;link_uid=PMC3373245" image-link="/pmc/articles/PMC3373245/figure/DDS129F4/" class="imagepopup">Figure 4. From: miRNA-34c regulates Notch <span class="highlight" style="background-color:">signaling</span> during bone development. </a></p><div class="supp"><p class="details"><div>miR-34c targets the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during osteoblast development <i>in vivo</i>. Total RNA was extracted from P2 calvarial tissues of WT and miR-34c Tg mice. The expression level of Notch receptors (<b>A</b>), Notch ligands (<i>Jag1</i> and <i>Dll4</i>), <i>Rbpjk</i> and <i>Hey1</i> (<b>B</b>), osteoblast markers (<b>C</b>) and <i>Opg</i> and <i>Rankl</i> (<b>D</b>) were quantified by qRT–PCR and normalized to B2m (<i>n</i> = 6). *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01. Error bar is SEM. (<b>E</b>) Diagram of miR-34c-mediated regulation of bone development. MiR-34c regulates both osteoclasts and osteoblasts via targeting multiple factors in the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and <i>Runx2</i> and <i>Satb2</i>, respectively, during skeletal development. The direct targets of miR-34c are in bold.</div></p></div><div class="aux"><div class="resc">Yangjin Bae, et al. Hum Mol Genet. 2012 July 1;21(13):2991-3000.</div><p class="links"><a class="dblinks" href="/pubmed/22498974" ref="ordinalpos=3903&amp;ncbi_uid=5041289&amp;link_uid=22498974">Citation</a><a class="dblinks" href="/pmc/articles/PMC3373245/" ref="ordinalpos=3903&amp;ncbi_uid=5041289&amp;link_uid=PMC3373245">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3904.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC195990/figure/F5/" ref="ordinalpos=3904&amp;ncbi_uid=236651&amp;link_uid=PMC195990" href="/pmc/articles/PMC195990/"><img src="/pmc/articles/instance/195990/bin/59046-10f5_C4TT.gif" src-large="/pmc/articles/instance/195990/bin/59046-10f5_C4TT.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC195990/" ref="ordinalpos=3904&amp;ncbi_uid=236651&amp;link_uid=PMC195990" image-link="/pmc/articles/PMC195990/figure/F5/" class="imagepopup">Figure 5.  From: Feedback control of mammalian Hedgehog <span class="highlight" style="background-color:">signaling</span> by the Hedgehog-binding protein, Hip1, modulates Fgf <span class="highlight" style="background-color:">signaling</span> during branching morphogenesis of the lung. </a></p><div class="supp"><p class="details"><div>A model of Hip1's role in lung branching morphogenesis. During lung branching morphogenesis, the two primary buds initially elongate and later bulge to generate the secondary buds. Localized <i>Fgf10</i> (green) expression in the distal mesenchyme induces primary bud formation. The mechanisms by which <i>Fgf10</i> expression is initiated and localized are not clear. <i>Shh</i> (blue) is expressed in the epithelium and is up-regulated at the distal tips of the primary buds. In this model, in the wild-type lung, secondary bud formation is also induced by localized <i>Fgf10</i> expression in the mesenchyme corresponding to the future sites of secondary bud formation. It is likely that similar mechanisms (indicated by a question mark in the diagram) are used to localize <i>Fgf10</i> to these sites. Epithelial expression of Hh may help restrict <i>Fgf10</i> expression to the sites of secondary bud formation by inhibiting <i>Fgf10</i> expression. The Hh <span class="highlight" style="background-color:">pathway</span> transcriptionally activates <i>Hip1</i> (gray) and <i>Ptch1</i> (not shown), antagonists of Hh <span class="highlight" style="background-color:">signaling</span>, to down-regulate the Hh <span class="highlight" style="background-color:">pathway</span> at the sites of bud formation. As a result, the signals that activate <i>Fgf10</i> expression overcome the mild inhibition (broken line), if any, by Hh <span class="highlight" style="background-color:">signaling</span>. In contrast, in the interbud regions, the Hh <span class="highlight" style="background-color:">pathway</span> effectively inhibits <i>Fgf10</i> expression. In the <i>Hip1</i> mutant lung, there is a uniform up-regulation of the Hh <span class="highlight" style="background-color:">pathway</span> along the entire epithelium due to the absence of negative regulation by <i>Hip1</i>. Consequently, high levels of Hh inhibit <i>Fgf10</i> expression and disrupt secondary bud formation. Thus, <i>Hip1</i>'s negative regulation of the Hh <span class="highlight" style="background-color:">pathway</span> is necessary to ensure proper secondary bud formation.</div></p></div><div class="aux"><div class="resc">Pao-Tien Chuang, et al. Genes Dev. 2003 February 1;17(3):342-347.</div><p class="links"><a class="dblinks" href="/pubmed/12569124" ref="ordinalpos=3904&amp;ncbi_uid=236651&amp;link_uid=12569124">Citation</a><a class="dblinks" href="/pmc/articles/PMC195990/" ref="ordinalpos=3904&amp;ncbi_uid=236651&amp;link_uid=PMC195990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3905.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3477470/figure/F7/" ref="ordinalpos=3905&amp;ncbi_uid=5407709&amp;link_uid=PMC3477470" href="/pmc/articles/PMC3477470/"><img src="/pmc/articles/instance/3477470/bin/nihms409809f7.gif" src-large="/pmc/articles/instance/3477470/bin/nihms409809f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3477470/" ref="ordinalpos=3905&amp;ncbi_uid=5407709&amp;link_uid=PMC3477470" image-link="/pmc/articles/PMC3477470/figure/F7/" class="imagepopup">Figure 7. Regulation of Mct1 by cAMP-dependent internalization into caveolae and early endosomes, and an unregulated clathrin-dependent recycling <span class="highlight" style="background-color:">pathway</span>.  From: Regulation of Mct1 by cAMP-dependent internalization in rat brain endothelial cells. </a></p><div class="supp"><p class="details"><div>Right side: In the regulated <span class="highlight" style="background-color:">pathway</span>, activation of β-adrenergic receptors leads to cAMP production and activation of a phosphatase which dephosphorylates plasma membrane Mct1, and caveolin-1 (hairpin structures), facilitating an interaction between Mct1 and caveolin-1 to promote internalization of Mct1 into caveolae (CAV) and early endosomes (EE). From here Mct1 could be trafficked to various compartments such as late endosomes and lysosomes (LE/LYS), similar to the protein-kinase dependent trafficking of kainite receptors and aquaporins (Martin and Henley 2004; Sidhaye et al., 2005). Left side: In the constitutive <span class="highlight" style="background-color:">pathway</span>, Mct1 is internalized into clathrin coated vesicles (CCV) where it is trafficked to recycling endosomes (RE) for redistribution on the plasma membrane, however, adrenergic <span class="highlight" style="background-color:">signaling</span> does not affect this trafficking <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jeffrey P. Smith, et al. Brain Res. 2012 October 22;1480:1-11.</div><p class="links"><a class="dblinks" href="/pubmed/22925948" ref="ordinalpos=3905&amp;ncbi_uid=5407709&amp;link_uid=22925948">Citation</a><a class="dblinks" href="/pmc/articles/PMC3477470/" ref="ordinalpos=3905&amp;ncbi_uid=5407709&amp;link_uid=PMC3477470">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3906.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3669920/figure/fig2/" ref="ordinalpos=3906&amp;ncbi_uid=4958138&amp;link_uid=PMC3669920" href="/pmc/articles/PMC3669920/"><img src="/pmc/articles/instance/3669920/bin/tp201333f2.gif" src-large="/pmc/articles/instance/3669920/bin/tp201333f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3669920/" ref="ordinalpos=3906&amp;ncbi_uid=4958138&amp;link_uid=PMC3669920" image-link="/pmc/articles/PMC3669920/figure/fig2/" class="imagepopup">Figure 2.  From: Multiple variants aggregate in the neuregulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a subset of schizophrenia patients. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> clustering model. Schematic representation of the proposed <span class="highlight" style="background-color:">pathway</span> clustering model for schizophrenia demonstrating how clustering of multiple protein damaging variants within different well-defined molecular pathways could represent an unrecognized factor of SZ risk and phenotypic heterogeneity.</div></p></div><div class="aux"><div class="resc">A Hatzimanolis, et al. Transl Psychiatry. 2013 May;3(5):e264.</div><p class="links"><a class="dblinks" href="/pubmed/23715299" ref="ordinalpos=3906&amp;ncbi_uid=4958138&amp;link_uid=23715299">Citation</a><a class="dblinks" href="/pmc/articles/PMC3669920/" ref="ordinalpos=3906&amp;ncbi_uid=4958138&amp;link_uid=PMC3669920">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3907.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3325914/figure/pone-0035012-g007/" ref="ordinalpos=3907&amp;ncbi_uid=3954691&amp;link_uid=PMC3325914" href="/pmc/articles/PMC3325914/"><img src="/pmc/articles/instance/3325914/bin/pone.0035012.g007.gif" src-large="/pmc/articles/instance/3325914/bin/pone.0035012.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3325914/" ref="ordinalpos=3907&amp;ncbi_uid=3954691&amp;link_uid=PMC3325914" image-link="/pmc/articles/PMC3325914/figure/pone-0035012-g007/" class="imagepopup">Figure 7. Insulin <span class="highlight" style="background-color:">signaling</span> and biologic response-associated RNA changes and inferred <span class="highlight" style="background-color:">signaling</span> pathways..  From: Comparative Transcriptional Network Modeling of Three PPAR-?/? Co-Agonists Reveals Distinct Metabolic Gene Signatures in Primary Human Hepatocytes. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Observed RNA state changes related to insulin <span class="highlight" style="background-color:">signaling</span> and biologic processes inferred to increase or decrease upon treatment with aleglitazar, tesaglitazar or Pio/Feno. See <i>Figure 2A</i> for treatment dosages. (<b>B</b>) Molecular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that could lead to changes in insulin <span class="highlight" style="background-color:">signaling</span> to decrease cellular proliferation. Depiction is based on data derived from treatment with the medium concentration of aleglitazar. Numbers in the blue or yellow boxes indicate the number of gene expression state changes that support the inference: negative numbers indicate an inferred decrease and positive numbers indicate an inferred increase. Numbers in the green or red boxes indicate the log fold change. AMP, adenosine monophosphate; CDK, cyclin-dependent kinase; Med, medium.</div></p></div><div class="aux"><div class="resc">Renée Deehan, et al. PLoS One. 2012;7(4):e35012.</div><p class="links"><a class="dblinks" href="/pubmed/22514701" ref="ordinalpos=3907&amp;ncbi_uid=3954691&amp;link_uid=22514701">Citation</a><a class="dblinks" href="/pmc/articles/PMC3325914/" ref="ordinalpos=3907&amp;ncbi_uid=3954691&amp;link_uid=PMC3325914">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3908.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3032584/figure/F1/" ref="ordinalpos=3908&amp;ncbi_uid=3663141&amp;link_uid=PMC3032584" href="/pmc/articles/PMC3032584/"><img src="/pmc/articles/instance/3032584/bin/nihms265455f1.gif" src-large="/pmc/articles/instance/3032584/bin/nihms265455f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3032584/" ref="ordinalpos=3908&amp;ncbi_uid=3663141&amp;link_uid=PMC3032584" image-link="/pmc/articles/PMC3032584/figure/F1/" class="imagepopup">Fig. 1. Pathways for NF-κB activation.  From: NF-kappaB, a mediator for lung carcinogenesis and a target for lung cancer prevention and therapy. </a></p><div class="supp"><p class="details"><div>The canonical <span class="highlight" style="background-color:">pathway</span> is activated by cytokines such as TNF-α. When TNF-α binds to the its receptor 1 (TNFR1), a <span class="highlight" style="background-color:">signaling</span> complex is formed to recruit and activate IKK, which leads to phosphorylation on IκB. IκB is subsequently ubiquitinated and degradated in the proteasome, resulting in NF-κB complex (p65/p50) translocation to the nucleus and activates gene transcription. The noncanonical <span class="highlight" style="background-color:">pathway</span> is activated by cytokines such as CD40L and lymphotoxin β. This <span class="highlight" style="background-color:">pathway</span> involves NIK-mediated IKKα activation and cleavage of p100 to create p52. The NF-κB complex (p52/RelB) moves to the nucleus to activate gene transcription. The atypical <span class="highlight" style="background-color:">pathway</span> is activated by a variety of stimuli such as UV. In this parhway, IKK-independent mechanisms are involved in IκB phosphorylation. Representative antiapoptotic factors induced by NF-κB are shown.</div></p></div><div class="aux"><div class="resc">Wenshu Chen, et al. Front Biosci. ;16:1172-1185.</div><p class="links"><a class="dblinks" href="/pubmed/21196225" ref="ordinalpos=3908&amp;ncbi_uid=3663141&amp;link_uid=21196225">Citation</a><a class="dblinks" href="/pmc/articles/PMC3032584/" ref="ordinalpos=3908&amp;ncbi_uid=3663141&amp;link_uid=PMC3032584">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3909.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3012004/figure/F2/" ref="ordinalpos=3909&amp;ncbi_uid=2684669&amp;link_uid=PMC3012004" href="/pmc/articles/PMC3012004/"><img src="/pmc/articles/instance/3012004/bin/nihms259273f2.gif" src-large="/pmc/articles/instance/3012004/bin/nihms259273f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3012004/" ref="ordinalpos=3909&amp;ncbi_uid=2684669&amp;link_uid=PMC3012004" image-link="/pmc/articles/PMC3012004/figure/F2/" class="imagepopup">Fig. 2.  From: Targeting the RTK-PI3K-mTOR Axis in Malignant Glioma: Overcoming Resistance. </a></p><div class="supp"><p class="details"><div>Sites of action for inhibitors and shRNAs in the EGFR-PI3K-Akt-mTOR <span class="highlight" style="background-color:">pathway</span>. Agents that inhibit the EGFR-PI3K-Akt-mTOR <span class="highlight" style="background-color:">pathway</span> at multiple sites may contribute to anticancer effects in malignant glioma. Agents that inhibit only one target within the EGFR-PI3K-Akt-mTOR axis generally show only modest efficacy and fail to induce appreciable apoptosis. This disappointing efficacy stems in-part from multiple nodes of activation in the EGFR-PI3K-Akt-mTOR axis. For example, loss of PTEN activates PI3K <span class="highlight" style="background-color:">signaling</span>. A feedback loop between S6K and IRS-1 also lead to PI3K activation. Inhibition of the PI3K-Akt-mTOR <span class="highlight" style="background-color:">pathway</span> may also induce autophagy, enabling cancer cells to survive some small molecule inhibitors of this <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Qi-Wen Fan, et al. Curr Top Microbiol Immunol. ;347:279-296.</div><p class="links"><a class="dblinks" href="/pubmed/20535652" ref="ordinalpos=3909&amp;ncbi_uid=2684669&amp;link_uid=20535652">Citation</a><a class="dblinks" href="/pmc/articles/PMC3012004/" ref="ordinalpos=3909&amp;ncbi_uid=2684669&amp;link_uid=PMC3012004">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3910.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2259449/figure/F2/" ref="ordinalpos=3910&amp;ncbi_uid=1255907&amp;link_uid=PMC2259449" href="/pmc/articles/PMC2259449/"><img src="/pmc/articles/instance/2259449/bin/neo1003_0198_fig002.gif" src-large="/pmc/articles/instance/2259449/bin/neo1003_0198_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2259449/" ref="ordinalpos=3910&amp;ncbi_uid=1255907&amp;link_uid=PMC2259449" image-link="/pmc/articles/PMC2259449/figure/F2/" class="imagepopup">Figure 2.  From: The Sodium Pump ?1 Subunit as a Potential Target to Combat Apoptosis-Resistant Glioblastomas. </a></p><div class="supp"><p class="details"><div>The molecular regulation of autophagy and the link with apoptosis. In the presence of growth factors, growth factor receptor <span class="highlight" style="background-color:">signaling</span> activates cascades to the mTOR, resulting in the inhibition of autophagy or type II programmed cell death (PCD; right-hand side of the figure). In contrast, inactivation of mTOR can induce both apoptosis and autophagy. The left-hand side of the figure represents a simplified version of the two main <span class="highlight" style="background-color:">signaling</span> pathways of apoptosis. The intrinsic <span class="highlight" style="background-color:">pathway</span> includes the mitochondria. The extrinsic cell death <span class="highlight" style="background-color:">pathway</span> is mediated by death receptors (DR). After formation of the DR/death-inducing <span class="highlight" style="background-color:">signaling</span> complex (DISC) and recruitment of caspases, downstream <span class="highlight" style="background-color:">signaling</span> pathways lead to apoptosis.</div></p></div><div class="aux"><div class="resc">Florence Lefranc, et al. Neoplasia. 2008 March;10(3):198-206.</div><p class="links"><a class="dblinks" href="/pubmed/18323016" ref="ordinalpos=3910&amp;ncbi_uid=1255907&amp;link_uid=18323016">Citation</a><a class="dblinks" href="/pmc/articles/PMC2259449/" ref="ordinalpos=3910&amp;ncbi_uid=1255907&amp;link_uid=PMC2259449">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3911.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3817236/figure/pone-0078398-g004/" ref="ordinalpos=3911&amp;ncbi_uid=5474725&amp;link_uid=PMC3817236" href="/pmc/articles/PMC3817236/"><img src="/pmc/articles/instance/3817236/bin/pone.0078398.g004.gif" src-large="/pmc/articles/instance/3817236/bin/pone.0078398.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3817236/" ref="ordinalpos=3911&amp;ncbi_uid=5474725&amp;link_uid=PMC3817236" image-link="/pmc/articles/PMC3817236/figure/pone-0078398-g004/" class="imagepopup">Figure 4. Differential expression of ERK/Wnt <span class="highlight" style="background-color:">pathway</span> proteins in parental and SU11274/Erlotinib resistant H2170 cells by western blotting..  From: Alternative <span class="highlight" style="background-color:">Signaling</span> Pathways as Potential Therapeutic Targets for Overcoming EGFR and c-Met Inhibitor Resistance in Non-Small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div>A. In SR H2170 cells, HGF induced pronounced p-ERK <span class="highlight" style="background-color:">signaling</span> compared to parental cells. Cells were starved for 48 hours and then stimulated with 40 ng/mL of HGF. Western blotting in SR H2170 indicated that, HGF activated p-ERK (T202/Y204) remained high for 120 minutes compared to parental lines. Basal levels of active β-catenin were also 2-fold higher and remained high (3.6-fold) for 120 minutes after HGF treatment in SR H2170 cells compared to those in parental cells over 60 minutes incubation. These experiments were done in triplicate. Relative densitometry of p-ERK/β-actin in SR H2170 cells was depicted which is an average of three independent experiments (n = 3, p&lt;0.01). B. Regulation of proteins in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after treatment of H2170 with SU11274. Upregulation of pLRP6 (2 to 3.0-fold) and β-catenin (3 to 8.0-fold) were seen in resistant H2170 cells in the presence or absence of SU11274. C. Regulation of proteins in the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after treatment of ER H2170 cells with erlotinib. Upregulation of LRP6 (2 to 5-fold), and Axin1 (2 to 3.5-fold) were seen in resistant H2170 cells in the presence or absence of erlotinib.</div></p></div><div class="aux"><div class="resc">Jason T. Fong, et al. PLoS One. 2013;8(11):e78398.</div><p class="links"><a class="dblinks" href="/pubmed/24223799" ref="ordinalpos=3911&amp;ncbi_uid=5474725&amp;link_uid=24223799">Citation</a><a class="dblinks" href="/pmc/articles/PMC3817236/" ref="ordinalpos=3911&amp;ncbi_uid=5474725&amp;link_uid=PMC3817236">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3912.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2695499/figure/F10/" ref="ordinalpos=3912&amp;ncbi_uid=1946996&amp;link_uid=PMC2695499" href="/pmc/articles/PMC2695499/"><img src="/pmc/articles/instance/2695499/bin/nihms106796f10.gif" src-large="/pmc/articles/instance/2695499/bin/nihms106796f10.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2695499/" ref="ordinalpos=3912&amp;ncbi_uid=1946996&amp;link_uid=PMC2695499" image-link="/pmc/articles/PMC2695499/figure/F10/" class="imagepopup">Fig. 10.  From: Phenotype reversion in fetal human liver epithelial cells identifies the role of an intermediate mesoendodermal stage before hepatic maturation. </a></p><div class="supp"><p class="details"><div>Changes in OSM <span class="highlight" style="background-color:">signaling</span>. (a-c) Microarray data analysis of OSM <span class="highlight" style="background-color:">signaling</span> in PP cells versus cultured cells. Genes with higher expression in PP cells are in pink, those expressed less well in PP cells are in green, those not expressed are in grey (IL6ST, interleukin-6 signal transducer). The OSM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was intact, although STAT3 was upregulated in cultured cells, probably explaining the responsiveness of P3 cells to added OSM.</div></p></div><div class="aux"><div class="resc">Mari Inada, et al. J Cell Sci. ;121(Pt 7):1002-1013.</div><p class="links"><a class="dblinks" href="/pubmed/18319302" ref="ordinalpos=3912&amp;ncbi_uid=1946996&amp;link_uid=18319302">Citation</a><a class="dblinks" href="/pmc/articles/PMC2695499/" ref="ordinalpos=3912&amp;ncbi_uid=1946996&amp;link_uid=PMC2695499">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3913.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2136409/figure/F7/" ref="ordinalpos=3913&amp;ncbi_uid=1190373&amp;link_uid=PMC2136409" href="/pmc/articles/PMC2136409/"><img src="/pmc/articles/instance/2136409/bin/nihms34093f7.gif" src-large="/pmc/articles/instance/2136409/bin/nihms34093f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2136409/" ref="ordinalpos=3913&amp;ncbi_uid=1190373&amp;link_uid=PMC2136409" image-link="/pmc/articles/PMC2136409/figure/F7/" class="imagepopup">Figure 7. Schematic diagram of FGF <span class="highlight" style="background-color:">signaling</span> in lens differentiation.  From: Genetic and epigenetic mechanisms of gene regulation during lens development. </a></p><div class="supp"><p class="details"><div>FGF binding to the outside of FGFR induces FGFR dimerization, receptor autophosphorylation followed by phosphorylation of the lipid-anchored docking protein Frs2α. Tyrosine-phosphorylated Frs2α serves as a focal protein for a multiprotein complex assembly that signals via two branches of FGF <span class="highlight" style="background-color:">signaling</span>, PI3K/Akt and FGF/MAPK. Additional components of this <span class="highlight" style="background-color:">pathway</span> such as heparin, N-cadherin, MKP3 and Sprouty (not shown) are discussed in text.</div></p></div><div class="aux"><div class="resc">Ales Cvekl, et al. Prog Retin Eye Res. ;26(6):555-597.</div><p class="links"><a class="dblinks" href="/pubmed/17905638" ref="ordinalpos=3913&amp;ncbi_uid=1190373&amp;link_uid=17905638">Citation</a><a class="dblinks" href="/pmc/articles/PMC2136409/" ref="ordinalpos=3913&amp;ncbi_uid=1190373&amp;link_uid=PMC2136409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3914.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885424/figure/pone-0011100-g006/" ref="ordinalpos=3914&amp;ncbi_uid=2309564&amp;link_uid=PMC2885424" href="/pmc/articles/PMC2885424/"><img src="/pmc/articles/instance/2885424/bin/pone.0011100.g006.gif" src-large="/pmc/articles/instance/2885424/bin/pone.0011100.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885424/" ref="ordinalpos=3914&amp;ncbi_uid=2309564&amp;link_uid=PMC2885424" image-link="/pmc/articles/PMC2885424/figure/pone-0011100-g006/" class="imagepopup">Figure 6. Heat map summary and hierarchical clustering of gene expression..  From: Transcriptional Regulation of the IGF <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> by Amino Acids and Insulin-Like Growth Factors during Myogenesis in Atlantic Salmon. </a></p><div class="supp"><p class="details"><div>Heat map summary and hierarchical clustering for MRFs, <i>MEF2A</i>, <i>pax7</i>, <i>FGF2</i> and components of the IGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during Atlantic salmon myogenic cell culture at 2 d, 5 d, 8 d, 11 d, 14 d, 17 d and 20 d. Rows are standardized to have mean 0 and standard deviation 1 so that red indicates high and green indicates low values.</div></p></div><div class="aux"><div class="resc">Neil I. Bower, et al. PLoS One. 2010;5(6):e11100.</div><p class="links"><a class="dblinks" href="/pubmed/20559434" ref="ordinalpos=3914&amp;ncbi_uid=2309564&amp;link_uid=20559434">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885424/" ref="ordinalpos=3914&amp;ncbi_uid=2309564&amp;link_uid=PMC2885424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3915.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3283740/figure/pone-0032204-g004/" ref="ordinalpos=3915&amp;ncbi_uid=3822086&amp;link_uid=PMC3283740" href="/pmc/articles/PMC3283740/"><img src="/pmc/articles/instance/3283740/bin/pone.0032204.g004.gif" src-large="/pmc/articles/instance/3283740/bin/pone.0032204.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3283740/" ref="ordinalpos=3915&amp;ncbi_uid=3822086&amp;link_uid=PMC3283740" image-link="/pmc/articles/PMC3283740/figure/pone-0032204-g004/" class="imagepopup">Figure 4. Involvement of the Plasminogen Activator system in the <span class="highlight" style="background-color:">signaling</span> to regulate the transition between mesenchymal and amoeboid migration..  From: Matrix-Bound PAI-1 Supports Cell Blebbing via RhoA/ROCK1 <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>uPAR can interact with different types of Integrins and participates in the activation of the MEK/ERK and Rac pathways, both leading to mesenchymal migration. Active, matrix-bound PAI-1 blocks the association of Integrin and uPAR with the extracellular matrix and subsequently supports cell blebbing by activating the RhoA <span class="highlight" style="background-color:">pathway</span>. If only matrix-bound PAI-1 is available, as in our experimental model, the RhoA <span class="highlight" style="background-color:">pathway</span> is activated either through biomechanical transmission (implicating an unknown co-actor) or transduction (implicating a transmembrane receptor; altogether these events might lead to amoeboid migration.</div></p></div><div class="aux"><div class="resc">Amandine Cartier-Michaud, et al. PLoS One. 2012;7(2):e32204.</div><p class="links"><a class="dblinks" href="/pubmed/22363817" ref="ordinalpos=3915&amp;ncbi_uid=3822086&amp;link_uid=22363817">Citation</a><a class="dblinks" href="/pmc/articles/PMC3283740/" ref="ordinalpos=3915&amp;ncbi_uid=3822086&amp;link_uid=PMC3283740">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3916.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2048736/figure/fig9/" ref="ordinalpos=3916&amp;ncbi_uid=1170720&amp;link_uid=PMC2048736" href="/pmc/articles/PMC2048736/"><img src="/pmc/articles/instance/2048736/bin/pp1450450f09.gif" src-large="/pmc/articles/instance/2048736/bin/pp1450450f09.jpg" alt="Figure 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2048736/" ref="ordinalpos=3916&amp;ncbi_uid=1170720&amp;link_uid=PMC2048736" image-link="/pmc/articles/PMC2048736/figure/fig9/" class="imagepopup">Figure 9. From: Dual Regulation Role of GH3.5 in Salicylic Acid and Auxin <span class="highlight" style="background-color:">Signaling</span> during Arabidopsis-Pseudomonas syringae Interaction. </a></p><div class="supp"><p class="details"><div>Function model of GH3.5 in Arabidopsis-<i>P. syringae</i> interactions. A, In the compatible interaction, <i>GH3.5</i> is activated to modulate the auxin <span class="highlight" style="background-color:">pathway</span> resulting in enhanced disease susceptibility through increasing IAA biosynthesis and derepressing auxin <span class="highlight" style="background-color:">signaling</span>. IAA also induces <i>GH3.5</i> to enlarge those processes. In addition, GH3.5 might also function as an IAA-amido synthetase to regulate IAA homeostasis. B, In the compatible/incompatible interactions, <i>GH3.5</i> positively modulates the SA <span class="highlight" style="background-color:">pathway</span> to enhance plant defense response through elevating SA biosynthesis, activating SA-induced genes, <i>WRKY</i>s, and basal defense-related genes. Feedback regulation of <i>GH3.5</i> by SA amplifies those effects. In this case, GH3.5 might also synthesize SA-Asp with unknown function during the interactions.</div></p></div><div class="aux"><div class="resc">Zhongqin Zhang, et al. Plant Physiol. 2007 October;145(2):450-464.</div><p class="links"><a class="dblinks" href="/pubmed/17704230" ref="ordinalpos=3916&amp;ncbi_uid=1170720&amp;link_uid=17704230">Citation</a><a class="dblinks" href="/pmc/articles/PMC2048736/" ref="ordinalpos=3916&amp;ncbi_uid=1170720&amp;link_uid=PMC2048736">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3917.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3528090/figure/f4/" ref="ordinalpos=3917&amp;ncbi_uid=5655651&amp;link_uid=PMC3528090" href="/pmc/articles/PMC3528090/"><img src="/pmc/articles/instance/3528090/bin/fig-4.gif" src-large="/pmc/articles/instance/3528090/bin/fig-4.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3528090/" ref="ordinalpos=3917&amp;ncbi_uid=5655651&amp;link_uid=PMC3528090" image-link="/pmc/articles/PMC3528090/figure/f4/" class="imagepopup">FIG. 4. From: Human Mesenchymal Stem Cells Inhibit Endothelial Proliferation and Angiogenesis via Cell-Cell Contact Through Modulation of the VE-Cadherin/?-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Activation of the Wnt <span class="highlight" style="background-color:">pathway</span> abrogates EC adherens junctions and inhibits vascular network formation in matrigel. <b>(A)</b> ECs cocultured with Wnt3a-transfected MSCs display diminished localization of β-catenin at the membrane, which results in a restoration of vascular network formation in matrigel in vitro shown in <b>(B)</b>. <b>(B)</b> also shows MSC-EC cultures treated with the recombinant Wnt3a protein display the same effect as MSC transfected with a Wnt3a expressing construct. Effect is quantitatively represented by changes in branch length <b>(C)</b> and branch point number <b>(D)</b>. <b>(E)</b> Schematic depicting effect of Wnt/β-catenin/VE-Cadherin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on ECs. * indicates p&lt;0.05. Color images available online at www.liebertpub.com/scd</div></p></div><div class="aux"><div class="resc">Tyler Menge, et al. Stem Cells Dev. 2013 January 1;22(1):148-157.</div><p class="links"><a class="dblinks" href="/pubmed/22734943" ref="ordinalpos=3917&amp;ncbi_uid=5655651&amp;link_uid=22734943">Citation</a><a class="dblinks" href="/pmc/articles/PMC3528090/" ref="ordinalpos=3917&amp;ncbi_uid=5655651&amp;link_uid=PMC3528090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3918.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3730284/figure/F3/" ref="ordinalpos=3918&amp;ncbi_uid=5160883&amp;link_uid=PMC3730284" href="/pmc/articles/PMC3730284/"><img src="/pmc/articles/instance/3730284/bin/nihms491633f3.gif" src-large="/pmc/articles/instance/3730284/bin/nihms491633f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3730284/" ref="ordinalpos=3918&amp;ncbi_uid=5160883&amp;link_uid=PMC3730284" image-link="/pmc/articles/PMC3730284/figure/F3/" class="imagepopup">Figure 3. Common Mechanisms Promoting PH Due to Hemodynamic Stress May Rely upon the Intersecting BMPR-II and Serotoninergic Pathways.  From: Pulmonary Vascular Disease Related to Hemodynamic Stress in the Pulmonary Circulation. </a></p><div class="supp"><p class="details"><div>In the serotoninergic <span class="highlight" style="background-color:">pathway</span>, both genetic predisposition and specific exogenous stimuli lead to vessel remodeling via increased serotonin levels and <span class="highlight" style="background-color:">signaling</span> activity. In the BMPR-II <span class="highlight" style="background-color:">pathway</span>, genetic predisposition (such as heterozygous loss-of-function +/− <i>BMPR2</i> mutations) and, perhaps, exogenous stimuli lead to decreased receptor expression, alteration of downstream <span class="highlight" style="background-color:">signaling</span>, and resulting vessel remodeling. Additional pathogenic mechanisms may upregulate the angiopoietin-1/Tie2 receptor <span class="highlight" style="background-color:">pathway</span> with resulting repression of BMPR-1A function and alteration of BMPR-II <span class="highlight" style="background-color:">signaling</span>. Interestingly, serotonin can directly upregulate angiopoietin-1 and can modulate downstream Smad function, thereby influencing the BMP <span class="highlight" style="background-color:">signaling</span> cascade. This functional intersection has yet to be fully described, but may represent a common, unifying mechanism of pathogenesis. The precise details of how hemodynamic stressors regulate these pathways are under investigation. This figure is reproduced with modifications (73).</div></p></div><div class="aux"><div class="resc">Stephen Y. Chan, et al. Compr Physiol. ;1(1):123-139.</div><p class="links"><a class="dblinks" href="/pubmed/23737167" ref="ordinalpos=3918&amp;ncbi_uid=5160883&amp;link_uid=23737167">Citation</a><a class="dblinks" href="/pmc/articles/PMC3730284/" ref="ordinalpos=3918&amp;ncbi_uid=5160883&amp;link_uid=PMC3730284">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3919.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4358957/figure/pone.0122509.g006/" ref="ordinalpos=3919&amp;ncbi_uid=7313188&amp;link_uid=PMC4358957" href="/pmc/articles/PMC4358957/"><img src="/pmc/articles/instance/4358957/bin/pone.0122509.g006.gif" src-large="/pmc/articles/instance/4358957/bin/pone.0122509.g006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4358957/" ref="ordinalpos=3919&amp;ncbi_uid=7313188&amp;link_uid=PMC4358957" image-link="/pmc/articles/PMC4358957/figure/pone.0122509.g006/" class="imagepopup">Fig 6. Working model illustrating the multiple targets of miR-185 in the calcium-dependent cardiac hypertrophy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: miR-185 Plays an Anti-Hypertrophic Role in the Heart via Multiple Targets in the Calcium-<span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="aux"><div class="resc">Jin Ock Kim, et al. PLoS One. 2015;10(3):e0122509.</div><p class="links"><a class="dblinks" href="/pubmed/25767890" ref="ordinalpos=3919&amp;ncbi_uid=7313188&amp;link_uid=25767890">Citation</a><a class="dblinks" href="/pmc/articles/PMC4358957/" ref="ordinalpos=3919&amp;ncbi_uid=7313188&amp;link_uid=PMC4358957">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3920.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3319201/figure/F3/" ref="ordinalpos=3920&amp;ncbi_uid=4532205&amp;link_uid=PMC3319201" href="/pmc/articles/PMC3319201/"><img src="/pmc/articles/instance/3319201/bin/nihms328806f3.gif" src-large="/pmc/articles/instance/3319201/bin/nihms328806f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3319201/" ref="ordinalpos=3920&amp;ncbi_uid=4532205&amp;link_uid=PMC3319201" image-link="/pmc/articles/PMC3319201/figure/F3/" class="imagepopup">Figure 3.  From: Consensus Recommendations for Current Treatments and Accelerating Clinical Trials for Patients with Neurofibromatosis Type 2. </a></p><div class="supp"><p class="details"><div>The primary therapeutic targets in the Ras downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jaishri O Blakeley, et al. Am J Med Genet A. ;158A(1):24-41.</div><p class="links"><a class="dblinks" href="/pubmed/22140088" ref="ordinalpos=3920&amp;ncbi_uid=4532205&amp;link_uid=22140088">Citation</a><a class="dblinks" href="/pmc/articles/PMC3319201/" ref="ordinalpos=3920&amp;ncbi_uid=4532205&amp;link_uid=PMC3319201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3921.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2938460/figure/fig1/" ref="ordinalpos=3921&amp;ncbi_uid=2475598&amp;link_uid=PMC2938460" href="/pmc/articles/PMC2938460/"><img src="/pmc/articles/instance/2938460/bin/PPAR2010-417646.001.gif" src-large="/pmc/articles/instance/2938460/bin/PPAR2010-417646.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2938460/" ref="ordinalpos=3921&amp;ncbi_uid=2475598&amp;link_uid=PMC2938460" image-link="/pmc/articles/PMC2938460/figure/fig1/" class="imagepopup">Figure 1.  From: Peroxisome Proliferator-Activated Receptors Protect against Apoptosis via 14-3-3. </a></p><div class="supp"><p class="details"><div>Schematic illustration of proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which all three PPAR isoforms exert anti-apoptotic actions via 14-3-3<i>ε</i> upregulation.</div></p></div><div class="aux"><div class="resc">Kenneth K. Wu. PPAR Res. 2010;2010:417646.</div><p class="links"><a class="dblinks" href="/pubmed/20862376" ref="ordinalpos=3921&amp;ncbi_uid=2475598&amp;link_uid=20862376">Citation</a><a class="dblinks" href="/pmc/articles/PMC2938460/" ref="ordinalpos=3921&amp;ncbi_uid=2475598&amp;link_uid=PMC2938460">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3922.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2799181/figure/F2/" ref="ordinalpos=3922&amp;ncbi_uid=2548262&amp;link_uid=PMC2799181" href="/pmc/articles/PMC2799181/"><img src="/pmc/articles/instance/2799181/bin/asn0110942130002.gif" src-large="/pmc/articles/instance/2799181/bin/asn0110942130002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2799181/" ref="ordinalpos=3922&amp;ncbi_uid=2548262&amp;link_uid=PMC2799181" image-link="/pmc/articles/PMC2799181/figure/F2/" class="imagepopup">Figure 2. From: The PPAR? Agonist Pioglitazone Ameliorates Aging-Related Progressive Renal Injury. </a></p><div class="supp"><p class="details"><div>Pioglitazone inhibits TGFβ activation. Smad2 phosphorylation, a major downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of TGFβ, was significantly reduced by pioglitazone.</div></p></div><div class="aux"><div class="resc">Hai-Chun Yang, et al. J Am Soc Nephrol. 2009 November;20(11):2380-2388.</div><p class="links"><a class="dblinks" href="/pubmed/19797472" ref="ordinalpos=3922&amp;ncbi_uid=2548262&amp;link_uid=19797472">Citation</a><a class="dblinks" href="/pmc/articles/PMC2799181/" ref="ordinalpos=3922&amp;ncbi_uid=2548262&amp;link_uid=PMC2799181">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3923.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634143/figure/F2/" ref="ordinalpos=3923&amp;ncbi_uid=1708872&amp;link_uid=PMC2634143" href="/pmc/articles/PMC2634143/"><img src="/pmc/articles/instance/2634143/bin/psb0204_0263_fig002.gif" src-large="/pmc/articles/instance/2634143/bin/psb0204_0263_fig002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634143/" ref="ordinalpos=3923&amp;ncbi_uid=1708872&amp;link_uid=PMC2634143" image-link="/pmc/articles/PMC2634143/figure/F2/" class="imagepopup">Figure 2.  From: Does Aluminum Generate a Bonafide Phospholipd Signal Cascade?. </a></p><div class="supp"><p class="details"><div>Model of Al<sup>3+</sup> effect on phospholipid signalling, and its possible consequence on other cell <span class="highlight" style="background-color:">signaling</span> components (X, inhibition of the <span class="highlight" style="background-color:">pathway</span> by Al<sup>3+</sup>).</div></p></div><div class="aux"><div class="resc">Ana Ramos-Díaz, et al. Plant Signal Behav. 2007 Jul-Aug;2(4):263-264.</div><p class="links"><a class="dblinks" href="/pubmed/19704674" ref="ordinalpos=3923&amp;ncbi_uid=1708872&amp;link_uid=19704674">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634143/" ref="ordinalpos=3923&amp;ncbi_uid=1708872&amp;link_uid=PMC2634143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3924.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3287101/figure/F4/" ref="ordinalpos=3924&amp;ncbi_uid=3828156&amp;link_uid=PMC3287101" href="/pmc/articles/PMC3287101/"><img src="/pmc/articles/instance/3287101/bin/1756-6606-5-1-4.gif" src-large="/pmc/articles/instance/3287101/bin/1756-6606-5-1-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3287101/" ref="ordinalpos=3924&amp;ncbi_uid=3828156&amp;link_uid=PMC3287101" image-link="/pmc/articles/PMC3287101/figure/F4/" class="imagepopup">Figure 4.  From: NMDA receptor activation stimulates transcription-independent rapid wnt5a protein synthesis via the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>NMDAR activation stimulates Wnt5a protein synthesis via the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. A. Primary cortical neurons (10 DIV) were pre-treated with vehicle (Control) or 20 μM PD98059 (PD98) for 30 min, and then stimulated with 50 μΜ NMDA for 15 min. Wnt5a protein was measured by Western blotting and quantified data were presented in graphs (mean ± SEM; n = 3, *p &lt; 0.05; One-way ANOVA). B. Primary cortical neurons (10 DIV) were pre-treated with vehicle (Control) or 20 μM U0126 for 30 min, followed by 50 μΜ NMDA for 15 min. Graphs (mean ± SEM) are from three independent experiments (*, p &lt; 0.05; **, p &lt; 0.01; One-way ANOVA).</div></p></div><div class="aux"><div class="resc">Yichen Li, et al. Mol Brain. 2012;5:1-1.</div><p class="links"><a class="dblinks" href="/pubmed/22217152" ref="ordinalpos=3924&amp;ncbi_uid=3828156&amp;link_uid=22217152">Citation</a><a class="dblinks" href="/pmc/articles/PMC3287101/" ref="ordinalpos=3924&amp;ncbi_uid=3828156&amp;link_uid=PMC3287101">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3925.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4123209/figure/F2/" ref="ordinalpos=3925&amp;ncbi_uid=6471155&amp;link_uid=PMC4123209" href="/pmc/articles/PMC4123209/"><img src="/pmc/articles/instance/4123209/bin/nihms-617278-f0002.gif" src-large="/pmc/articles/instance/4123209/bin/nihms-617278-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4123209/" ref="ordinalpos=3925&amp;ncbi_uid=6471155&amp;link_uid=PMC4123209" image-link="/pmc/articles/PMC4123209/figure/F2/" class="imagepopup">Figure 2. IL-17 and IL-22 <span class="highlight" style="background-color:">signaling</span> pathways.  From: A tale of two cytokines: IL-17 and IL-22 in asthma and infection. </a></p><div class="supp"><p class="details"><div>IL-17 and IL-22 bind to transmembrane heterodimeric cell surface receptors to induce cellular <span class="highlight" style="background-color:">signaling</span> pathways. Specifically, IL-17A or IL-17F binds to the IL-17RA and IL-17RC, respectively as homodimers or heterodimers. The binding of IL-17 to its receptor leads to the recruitment of adaptor protein and E3 ligase, Act1. Scaffold proteins TRAF6 and TAK1 interact with Act1 to activate NF-κB expression, which leads to subsequent induction of proinflammatory genes. IL-17 <span class="highlight" style="background-color:">signaling</span> through its receptor also induces ERK, p38, and JNK activation via Act1 binding with TRAF2 and TRAF5. Further, this MAP kinase activation can induce downstream <span class="highlight" style="background-color:">signaling</span> to activate AP-1 and CCAAT/enhancer binding protein (C/EBP). Receptor <span class="highlight" style="background-color:">signaling</span> of IL-22 through its heterodimer receptor of IL-10R2 and IL-22R1 induces phosphorylation of tyrosine kinases Jak1 and Tyk2, which activate the transcription factor STATs. In some cases, MAP kinases (ERK, p38, and JNK) are also activated through a distinct <span class="highlight" style="background-color:">pathway</span> following IL-22R activation. IL-22 binding protein (IL-22BP) acts as a soluble antagonist.</div></p></div><div class="aux"><div class="resc">Michelle L Manni, et al. Expert Rev Respir Med. ;8(1):25-42.</div><p class="links"><a class="dblinks" href="/pubmed/24325586" ref="ordinalpos=3925&amp;ncbi_uid=6471155&amp;link_uid=24325586">Citation</a><a class="dblinks" href="/pmc/articles/PMC4123209/" ref="ordinalpos=3925&amp;ncbi_uid=6471155&amp;link_uid=PMC4123209">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3926.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4028780/figure/F3/" ref="ordinalpos=3926&amp;ncbi_uid=6154580&amp;link_uid=PMC4028780" href="/pmc/articles/PMC4028780/"><img src="/pmc/articles/instance/4028780/bin/1297-9716-44-120-3.gif" src-large="/pmc/articles/instance/4028780/bin/1297-9716-44-120-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4028780/" ref="ordinalpos=3926&amp;ncbi_uid=6154580&amp;link_uid=PMC4028780" image-link="/pmc/articles/PMC4028780/figure/F3/" class="imagepopup">Figure 3.  From: Pyroptosis and adaptive immunity mechanisms are promptly engendered in mesenteric lymph-nodes during pig infections with Salmonella enterica serovar Typhimurium. </a></p><div class="supp"><p class="details"><div><b>Interaction between CTLA-4 <span class="highlight" style="background-color:">signaling</span> in cytotoxic T lymphocytes and Clathrin-mediated endocytosis <span class="highlight" style="background-color:">signaling</span> pathways. A</b>: visual representation of CTLA-4 <span class="highlight" style="background-color:">signaling</span> in cytotoxic T lymphocyte <span class="highlight" style="background-color:">pathway</span> stressing the regulation of molecules taking part in Clathrin-mediated endocytosis <span class="highlight" style="background-color:">signaling</span>. Red and grey nodes are input and IPA-inserted molecules respectively. <b>B</b>: Western blot analysis of CTLA4 and CLTA in MLN at 1, 2 and 6 dpi. <b>C</b>: mRNA quantification of <i>CTLA4</i> by qPCR. <b>D</b>: expression pattern of <i>CLTA</i> mRNA by microarray analysis.</div></p></div><div class="aux"><div class="resc">Rodrigo Prado Martins, et al. Vet Res. 2013;44(1):120-120.</div><p class="links"><a class="dblinks" href="/pubmed/24308825" ref="ordinalpos=3926&amp;ncbi_uid=6154580&amp;link_uid=24308825">Citation</a><a class="dblinks" href="/pmc/articles/PMC4028780/" ref="ordinalpos=3926&amp;ncbi_uid=6154580&amp;link_uid=PMC4028780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3927.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634038/figure/F1/" ref="ordinalpos=3927&amp;ncbi_uid=1708182&amp;link_uid=PMC2634038" href="/pmc/articles/PMC2634038/"><img src="/pmc/articles/instance/2634038/bin/psb0203_0135_fig001.gif" src-large="/pmc/articles/instance/2634038/bin/psb0203_0135_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634038/" ref="ordinalpos=3927&amp;ncbi_uid=1708182&amp;link_uid=PMC2634038" image-link="/pmc/articles/PMC2634038/figure/F1/" class="imagepopup">Figure 1.  From: Abscisic Acid and Abiotic Stress <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>ABA biosynthesis <span class="highlight" style="background-color:">pathway</span> and its regulation. ABA is synthesized from β- carotene via the oxidative cleavage of neoxanthin and conversion of xanthoxin to ABA via ABA-aldehyde. Abiotic stresses (Dehydration, cold, salinity) stimulate ABA biosynthesis and accumulation by activating genes involved in the ABA biosynthetic <span class="highlight" style="background-color:">pathway</span>, which itself could be mediated by calcium-dependent phosphorylation cascade. ABA could also upregulate expression of ABA biosynthetic genes via calcium <span class="highlight" style="background-color:">signaling</span> pathways (see Xiong et al., 2002; Zhu, 2002). AAO, ABA-aldehyde oxidase; MCSU, molybdenium cofactor sulfurase, NCED, 9-cis-epoxycarotenoid dioxygenase, ZEP, zeaxanthin epoxidase.</div></p></div><div class="aux"><div class="resc">Narendra Tuteja. Plant Signal Behav. 2007 May-Jun;2(3):135-138.</div><p class="links"><a class="dblinks" href="/pubmed/19516981" ref="ordinalpos=3927&amp;ncbi_uid=1708182&amp;link_uid=19516981">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634038/" ref="ordinalpos=3927&amp;ncbi_uid=1708182&amp;link_uid=PMC2634038">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3928.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3068163/figure/pone-0018257-g001/" ref="ordinalpos=3928&amp;ncbi_uid=2881685&amp;link_uid=PMC3068163" href="/pmc/articles/PMC3068163/"><img src="/pmc/articles/instance/3068163/bin/pone.0018257.g001.gif" src-large="/pmc/articles/instance/3068163/bin/pone.0018257.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3068163/" ref="ordinalpos=3928&amp;ncbi_uid=2881685&amp;link_uid=PMC3068163" image-link="/pmc/articles/PMC3068163/figure/pone-0018257-g001/" class="imagepopup">Figure 1. The two major <span class="highlight" style="background-color:">signaling</span> pathways implicated in melanoma are the MAPK <span class="highlight" style="background-color:">pathway</span> (red) and the AKT/PI3K (green) <span class="highlight" style="background-color:">pathway</span> which regulate cell growth, proliferation and cell death..  From: A Melanoma Molecular Disease Model. </a></p><div class="supp"><p class="details"><div>There is a lot of cross-talk between these pathways and their downstream effectors, which we have classified into 8 pathways for simplicity to account for differences in treatment modalities (e.g. <span class="highlight" style="background-color:">signaling</span> through NRAS could affect both MAPK and AKT/PI3K pathways). The additional 6 pathways are: c-KIT (pink), CDK (blue), GNAQ/GNA11 (brown), MITF (orange), NRAS (yellow), and P53/BCL (purple). The complex relationship among BRAF, ARF/INK4A (via dashed line), p16, and p14ARF connotes an alternative splicing relationship.</div></p></div><div class="aux"><div class="resc">Smruti J. Vidwans, et al. PLoS One. 2011;6(3):e18257.</div><p class="links"><a class="dblinks" href="/pubmed/21479172" ref="ordinalpos=3928&amp;ncbi_uid=2881685&amp;link_uid=21479172">Citation</a><a class="dblinks" href="/pmc/articles/PMC3068163/" ref="ordinalpos=3928&amp;ncbi_uid=2881685&amp;link_uid=PMC3068163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3929.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741361/figure/pone-0071038-g002/" ref="ordinalpos=3929&amp;ncbi_uid=5201727&amp;link_uid=PMC3741361" href="/pmc/articles/PMC3741361/"><img src="/pmc/articles/instance/3741361/bin/pone.0071038.g002.gif" src-large="/pmc/articles/instance/3741361/bin/pone.0071038.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741361/" ref="ordinalpos=3929&amp;ncbi_uid=5201727&amp;link_uid=PMC3741361" image-link="/pmc/articles/PMC3741361/figure/pone-0071038-g002/" class="imagepopup">Figure 2. Expression of genes involved in the plant hormone <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in rice seedling shoots and roots regulated by H2..  From: Molecular Hydrogen Is Involved in Phytohormone <span class="highlight" style="background-color:">Signaling</span> and Stress Responses in Plants. </a></p><div class="supp"><p class="details"><div>Rice seeds were germinated and maintained in hydrogen-rich water (HW) or de-hydrogen water (Control) for 2 weeks. RT-qPCR was performed to estimate the transcription rate of hormone receptor genes and of a few key factors involved in related <span class="highlight" style="background-color:">signaling</span> pathways (<b>a–e</b>, ETH receptor genes, <i>OsERS1, OsERS2</i>, <i>OsETR2</i>, <i>OsETR3</i>, and <i>OsETR4</i>. <b>f</b>, ACC synthetase gene, <i>OsACS1.</i> <b>g</b>, ABA receptor gene, <i>OsPYL</i>. <b>h</b>, CTK receptor gene, <i>OsHk6</i>. <b>I</b>, GA receptor gene, <i>OsGID1</i>. <b>j</b>, a gene for a key transcript factor (DELLA protein OsSLR1) in the GA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>k–m</b>, IAA receptor genes, <i>OsAFB</i> and <i>OsTIR</i>. <b>n</b>: SA receptor gene, <i>OsNPR4</i>. Data are presented as mean ± SE, n = 3, *<i>P</i>&lt;0.05, **<i>P</i>&lt;0.01, ***<i>P</i>&lt;0.001 compared with the control group (<i>t</i>-test).</div></p></div><div class="aux"><div class="resc">Jiqing Zeng, et al. PLoS One. 2013;8(8):e71038.</div><p class="links"><a class="dblinks" href="/pubmed/23951075" ref="ordinalpos=3929&amp;ncbi_uid=5201727&amp;link_uid=23951075">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741361/" ref="ordinalpos=3929&amp;ncbi_uid=5201727&amp;link_uid=PMC3741361">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3930.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3137877/figure/fig9/" ref="ordinalpos=3930&amp;ncbi_uid=3095631&amp;link_uid=PMC3137877" href="/pmc/articles/PMC3137877/"><img src="/pmc/articles/instance/3137877/bin/ECAM2011-984154.009.gif" src-large="/pmc/articles/instance/3137877/bin/ECAM2011-984154.009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3137877/" ref="ordinalpos=3930&amp;ncbi_uid=3095631&amp;link_uid=PMC3137877" image-link="/pmc/articles/PMC3137877/figure/fig9/" class="imagepopup">Figure 9.  From: Targeting PML-RARα and Oncogenic <span class="highlight" style="background-color:">Signaling</span> Pathways by Chinese Herbal Mixture Tien-Hsien Liquid in Acute Promyelocytic Leukemia NB4 Cells. </a></p><div class="supp"><p class="details"><div>Schematic illustration of the inhibitory effects of THL and its active fraction EAS5 on the oncogenic <span class="highlight" style="background-color:">signaling</span> pathways in NB4 cells. The arrow indicates promotion, and the T-shaped bar indicates repression. The dramatic inhibition on ERK <span class="highlight" style="background-color:">pathway</span> is indicated by a thicker T-shaped bar. The data of THL are presented in Figure 8, and the data of EAS5 are presented in Figure 13.</div></p></div><div class="aux"><div class="resc">Chih-Jung Yao, et al. Evid Based Complement Alternat Med. 2011;2011:984154.</div><p class="links"><a class="dblinks" href="/pubmed/19897545" ref="ordinalpos=3930&amp;ncbi_uid=3095631&amp;link_uid=19897545">Citation</a><a class="dblinks" href="/pmc/articles/PMC3137877/" ref="ordinalpos=3930&amp;ncbi_uid=3095631&amp;link_uid=PMC3137877">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3931.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3443809/figure/fig3/" ref="ordinalpos=3931&amp;ncbi_uid=5194406&amp;link_uid=PMC3443809" href="/pmc/articles/PMC3443809/"><img src="/pmc/articles/instance/3443809/bin/AJRCCM1864359fig3.gif" src-large="/pmc/articles/instance/3443809/bin/AJRCCM1864359fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3443809/" ref="ordinalpos=3931&amp;ncbi_uid=5194406&amp;link_uid=PMC3443809" image-link="/pmc/articles/PMC3443809/figure/fig3/" class="imagepopup">Figure 3. From: A Novel Molecular Signature for Elevated Tricuspid Regurgitation Velocity in Sickle Cell Disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> analysis of genes associated with an elevated tricuspid regurgitation jet velocity (TRV). <span class="highlight" style="background-color:">Pathway</span> analysis of the 631 elevated TRV-associated genes reveals Wnt <span class="highlight" style="background-color:">signaling</span>, axon guidance, calcium <span class="highlight" style="background-color:">signaling</span>, neuroactive ligand–receptor interaction, vascular smooth muscle contraction, and cancer pathways as the most significantly represented in patients with SCD.</div></p></div><div class="aux"><div class="resc">Ankit A. Desai, et al. Am J Respir Crit Care Med. 2012 August 15;186(4):359-368.</div><p class="links"><a class="dblinks" href="/pubmed/22679008" ref="ordinalpos=3931&amp;ncbi_uid=5194406&amp;link_uid=22679008">Citation</a><a class="dblinks" href="/pmc/articles/PMC3443809/" ref="ordinalpos=3931&amp;ncbi_uid=5194406&amp;link_uid=PMC3443809">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3932.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847587/figure/F5/" ref="ordinalpos=3932&amp;ncbi_uid=5567490&amp;link_uid=PMC3847587" href="/pmc/articles/PMC3847587/"><img src="/pmc/articles/instance/3847587/bin/1743-422X-10-202-5.gif" src-large="/pmc/articles/instance/3847587/bin/1743-422X-10-202-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847587/" ref="ordinalpos=3932&amp;ncbi_uid=5567490&amp;link_uid=PMC3847587" image-link="/pmc/articles/PMC3847587/figure/F5/" class="imagepopup">Figure 5.  From: HeLa cell response proteome alterations induced by mammalian reovirus T3D infection. </a></p><div class="supp"><p class="details"><div><b>Significantly affected canonical <span class="highlight" style="background-color:">pathway</span> “Interferon <span class="highlight" style="background-color:">signaling</span>”, as determined by IPA</b><sup><b>™ </b></sup><b>analysis.</b> Red indicates highly up-regulated proteins, light green indicates down-regulated, grey represents not significantly affected, and white represents molecules known to be part of the <span class="highlight" style="background-color:">pathway</span> but not identified by SILAC.</div></p></div><div class="aux"><div class="resc">Kevin M Coombs. Virol J. 2013;10:202-202.</div><p class="links"><a class="dblinks" href="/pubmed/23799967" ref="ordinalpos=3932&amp;ncbi_uid=5567490&amp;link_uid=23799967">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847587/" ref="ordinalpos=3932&amp;ncbi_uid=5567490&amp;link_uid=PMC3847587">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3933.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3255702/figure/F6/" ref="ordinalpos=3933&amp;ncbi_uid=4095031&amp;link_uid=PMC3255702" href="/pmc/articles/PMC3255702/"><img src="/pmc/articles/instance/3255702/bin/zmb9991093250006.gif" src-large="/pmc/articles/instance/3255702/bin/zmb9991093250006.jpg" alt="Fig 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3255702/" ref="ordinalpos=3933&amp;ncbi_uid=4095031&amp;link_uid=PMC3255702" image-link="/pmc/articles/PMC3255702/figure/F6/" class="imagepopup">Fig 6.  From: TIEG1 Inhibits Breast Cancer Invasion and Metastasis by Inhibition of Epidermal Growth Factor Receptor (EGFR) Transcription and the EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>TIEG1 inhibits breast cancer cell invasion by inhibition of EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Overexpression of TIEG1 inhibited the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TIEG1 expression vector or control vector was transfected into MDA-MB-231HM or MDA-MB-468 cells and generated stable transfectants, MDA-MB-231HM/TIEG1 and MDA-MB-231HM/Vector or MDA-MB-468/TIEG1 and MDA-MB-468/Vector, respectively. Western blot analysis was performed as described in Materials and Methods. (B) Knockdown of TIEG1 induced the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. MDA-MB-231 or MCF-7 cells were treated with 100 nM TIEG1 siRNA or nontargeting siRNA for 48 h. Western blot analysis was performed as described in Materials and Methods. (C and D) Expression of TIEG1 inhibited breast cancer cell invasion; overexpression of EGFR restored levels of invaded cells. MDA-MB-231HM/TIEG1 (231HM/TIEG1), MDA-MB-231HM/Vector (231HM/Vector), MDA-MB-468/TIEG1 (468/TIEG1), and MDA-MB-468/Vector (468/Vector) cells, as indicated, were transfected with 2 μg of EGFR expression vector or control vector and analyzed for Matrigel invasion. *, <i>P</i> &lt; 0.01 versus controls. (E and F) Knockdown of TIEG1 promoted cancer cell invasion. MDA-MB-231 cells or MCF-7 cells transfected with nontargeting siRNA (control siRNA) or TIEG1-directed siRNA were analyzed for Matrigel invasion and Western blotting. *, <i>P</i> &lt; 0.01 versus controls. Values in panels C to F indicate invaded cells/mean numbers in five microscope fields.</div></p></div><div class="aux"><div class="resc">Wei Jin, et al. Mol Cell Biol. 2012 January;32(1):50-63.</div><p class="links"><a class="dblinks" href="/pubmed/22025675" ref="ordinalpos=3933&amp;ncbi_uid=4095031&amp;link_uid=22025675">Citation</a><a class="dblinks" href="/pmc/articles/PMC3255702/" ref="ordinalpos=3933&amp;ncbi_uid=4095031&amp;link_uid=PMC3255702">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3934.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3391878/figure/F1/" ref="ordinalpos=3934&amp;ncbi_uid=4174500&amp;link_uid=PMC3391878" href="/pmc/articles/PMC3391878/"><img src="/pmc/articles/instance/3391878/bin/JMedLife-05-168-g001.gif" src-large="/pmc/articles/instance/3391878/bin/JMedLife-05-168-g001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3391878/" ref="ordinalpos=3934&amp;ncbi_uid=4174500&amp;link_uid=PMC3391878" image-link="/pmc/articles/PMC3391878/figure/F1/" class="imagepopup">Fig. 1.  From: Evaluating the treatment of metastatic colorectal cancer with monoclonal antibodies. </a></p><div class="supp"><p class="details"><div>The Role of RAS Protein in EGFR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (Source: Medscape General Surgery) </div></p></div><div class="aux"><div class="resc">C Popa, et al. J Med Life. 2012 June 12;5(2):168-172.</div><p class="links"><a class="dblinks" href="/pubmed/22802884" ref="ordinalpos=3934&amp;ncbi_uid=4174500&amp;link_uid=22802884">Citation</a><a class="dblinks" href="/pmc/articles/PMC3391878/" ref="ordinalpos=3934&amp;ncbi_uid=4174500&amp;link_uid=PMC3391878">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3935.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2727408/figure/F1/" ref="ordinalpos=3935&amp;ncbi_uid=2418483&amp;link_uid=PMC2727408" href="/pmc/articles/PMC2727408/"><img src="/pmc/articles/instance/2727408/bin/zh89990940210001.gif" src-large="/pmc/articles/instance/2727408/bin/zh89990940210001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2727408/" ref="ordinalpos=3935&amp;ncbi_uid=2418483&amp;link_uid=PMC2727408" image-link="/pmc/articles/PMC2727408/figure/F1/" class="imagepopup">Figure 1.  From: Activation of Wnt5A <span class="highlight" style="background-color:">signaling</span> is required for CXC chemokine ligand 12-mediated T-cell migration. </a></p><div class="supp"><p class="details"><div><b>Microarray analysis reveals up-regulation of noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> in CXCL12-mediated T-cell migration.</b> (A) Primary T cells from 3 different donors were treated with human CXCL12 at 100 ng/mL for 12 hours. Cells from the same 3 donors were incubated in media only as a control. RNA was extracted from each sample and subjected to microarray analysis (6 arrays total). Genes with a 2.5-fold cutoff (either up or down) were considered significantly changed. Genes involved in noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> (arrows) were found to be significantly up-regulated (&gt; 2.5-fold increase, <i>P</i> &lt; .05) in the CXCL12-treated T-cell groups. (B) Primary T cells from 4 additional donors were treated with human CXCL12 at 100 ng/mL for 12 hours. Cells from these 4 donors were incubated in media only as a control. RNA was extracted and cDNA was used to perform real-time PCR analysis using the Open Biosystems Superarray <span class="highlight" style="background-color:">pathway</span> reverse transcription–PCR kits. Superarray reverse transcription–PCR analysis confirms that gene members of the noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are up-regulated (B), whereas members of the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are down-regulated (C). All genes are normalized to actin. Interestingly, the only gene up-regulated in the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is the Wnt inhibitory factor-1 (<i>WIF1</i>), an inhibitor of canonical Wnt <span class="highlight" style="background-color:">signaling</span>. To validate the RNA results, Western blot analysis was performed. (D) Primary human T cells (pool of 3 donors) were treated with 100 ng/mL CXCL12 for the times indicated, then examined by Western blot analysis for β-catenin, Wnt5A, and Fzd2 expression. These data demonstrate that β-catenin is decreased by CXCL12 treatment, but Wnt5A and Fzd2 expression are increased (E). (F) Wnt5A up-regulation is specific to CXCL12, as Western blot analysis demonstrates that CCL19 (100 ng/mL, 12 hours) does not increase Wnt5A expression. Western blots were repeated 3 times, and a representative blot is shown.</div></p></div><div class="aux"><div class="resc">Manik C. Ghosh, et al. Blood. 2009 August 13;114(7):1366-1373.</div><p class="links"><a class="dblinks" href="/pubmed/19520808" ref="ordinalpos=3935&amp;ncbi_uid=2418483&amp;link_uid=19520808">Citation</a><a class="dblinks" href="/pmc/articles/PMC2727408/" ref="ordinalpos=3935&amp;ncbi_uid=2418483&amp;link_uid=PMC2727408">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3936.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855406/figure/F2/" ref="ordinalpos=3936&amp;ncbi_uid=5579469&amp;link_uid=PMC3855406" href="/pmc/articles/PMC3855406/"><img src="/pmc/articles/instance/3855406/bin/nihms505287f2.gif" src-large="/pmc/articles/instance/3855406/bin/nihms505287f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855406/" ref="ordinalpos=3936&amp;ncbi_uid=5579469&amp;link_uid=PMC3855406" image-link="/pmc/articles/PMC3855406/figure/F2/" class="imagepopup">Figure 2.  From: NF?B <span class="highlight" style="background-color:">signaling</span> regulates embryonic and adult neurogenesis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways of NFκB activation. In the classical <span class="highlight" style="background-color:">pathway</span>, the classical stimulators like TNFα bind to receptor, leading to recruitment of the adaptor molecule TRADD and the <span class="highlight" style="background-color:">signaling</span> proteins TRAF2 and RIP1. TRAF2 recruits the IKK complex that phosphorylates IκBα at serine residue 32 and 36, leading to its ubiquitination and degradation by the proteasome, allowing the release of the classical heterodimer of p65/p50, which then translocate to the nucleus to regulate the transcription of target genes. In the non-classical <span class="highlight" style="background-color:">pathway</span>, NIK-induced phosphorylation of IKKα leads to the phosphorylation of p100 and proteasome-mediated processing of p100 into p52, allowing the nuclear translocation of RelB/p52 to regulate target gene expression. In the atypical <span class="highlight" style="background-color:">pathway</span>, the p38-activated casein kinase 2 (CK2) phosphorylates IκBα at a cluster of serine residues within C-terminal domain, while oxidative stress induces spleen tyrosine kinase (Syk)-mediated phosphorylation of IκBα at tyrosine residue (Y-42) within N-terminal domain.</div></p></div><div class="aux"><div class="resc">Yonggang ZHANG, et al. Front Biol (Beijing). ;7(4):10.1007/s11515-012-1233-z.</div><p class="links"><a class="dblinks" href="/pubmed/24324484" ref="ordinalpos=3936&amp;ncbi_uid=5579469&amp;link_uid=24324484">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855406/" ref="ordinalpos=3936&amp;ncbi_uid=5579469&amp;link_uid=PMC3855406">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3937.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2814417/figure/F7/" ref="ordinalpos=3937&amp;ncbi_uid=2178825&amp;link_uid=PMC2814417" href="/pmc/articles/PMC2814417/"><img src="/pmc/articles/instance/2814417/bin/nihms169815f7.gif" src-large="/pmc/articles/instance/2814417/bin/nihms169815f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2814417/" ref="ordinalpos=3937&amp;ncbi_uid=2178825&amp;link_uid=PMC2814417" image-link="/pmc/articles/PMC2814417/figure/F7/" class="imagepopup">Fig. 7.  From: Raf-1 <span class="highlight" style="background-color:">Signaling</span> Is Required for the Later Stages of 1,25-Dihydroxyvitamin D3-Induced Differentiation of HL60 Cells But Is Not Mediated by the MEK/ERK Module. </a></p><div class="supp"><p class="details"><div>Schematic representation of the postulated <span class="highlight" style="background-color:">signaling</span> of MAP kinase <span class="highlight" style="background-color:">pathway</span> in proliferating or differentiating HL60 cells. <b>A</b>: In the early stage of HL60 cell differentiation, when proliferation still continues (hematopoietic lineage amplification), activation of ERK MAP kinase <span class="highlight" style="background-color:">pathway</span> is important in regulating proliferation-related genes, such as c-myc, Ets, and other targets. <b>B</b>: In the later stages of HL60 cell differentiation activation of both MEK-1 and ERK kinases is low, while the activation of RAf-1 <span class="highlight" style="background-color:">signaling</span> is continuously increasing, and appears to activate downstream targets such as p90RSK, while bypassing the MEK/ERK module. KSR-1 may amplify the Raf-RSK <span class="highlight" style="background-color:">pathway</span>, and result in phosphorylation of targets such as C/EBPβ. The C/EBPβ is one of the effectors of monocytic differentiation, perhaps through a C/EBP element present in the promoter of the monocyte marker CD14.</div></p></div><div class="aux"><div class="resc">Xuening Wang, et al. J Cell Physiol. ;209(2):253.</div><p class="links"><a class="dblinks" href="/pubmed/16883571" ref="ordinalpos=3937&amp;ncbi_uid=2178825&amp;link_uid=16883571">Citation</a><a class="dblinks" href="/pmc/articles/PMC2814417/" ref="ordinalpos=3937&amp;ncbi_uid=2178825&amp;link_uid=PMC2814417">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3938.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC326656/figure/f4/" ref="ordinalpos=3938&amp;ncbi_uid=432819&amp;link_uid=PMC326656" href="/pmc/articles/PMC326656/"><img src="/pmc/articles/instance/326656/bin/ek0630157004.gif" src-large="/pmc/articles/instance/326656/bin/ek0630157004.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC326656/" ref="ordinalpos=3938&amp;ncbi_uid=432819&amp;link_uid=PMC326656" image-link="/pmc/articles/PMC326656/figure/f4/" class="imagepopup">FIG. 4. From: The Yeast Protein Kinase C Cell Integrity <span class="highlight" style="background-color:">Pathway</span> Mediates Tolerance to the Antifungal Drug Caspofungin through Activation of Slt2p Mitogen-Activated Protein Kinase <span class="highlight" style="background-color:">Signaling</span> . </a></p><div class="supp"><p class="details"><div>Diagram depicting the yeast PKC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The basic components of the cell integrity signal transduction <span class="highlight" style="background-color:">pathway</span> are depicted. Cell surface sensors transduce extracellular signals through a MAP kinase cascade to induce transcription via two main downstream regulators, Rlm1p and Swi4p, so as to respond to a variety of external cell wall stresses.</div></p></div><div class="aux"><div class="resc">Cristina Reinoso-Martín, et al. Eukaryot Cell. 2003 December;2(6):1200-1210.</div><p class="links"><a class="dblinks" href="/pubmed/14665455" ref="ordinalpos=3938&amp;ncbi_uid=432819&amp;link_uid=14665455">Citation</a><a class="dblinks" href="/pmc/articles/PMC326656/" ref="ordinalpos=3938&amp;ncbi_uid=432819&amp;link_uid=PMC326656">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3939.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2118421/figure/fig7/" ref="ordinalpos=3939&amp;ncbi_uid=1147771&amp;link_uid=PMC2118421" href="/pmc/articles/PMC2118421/"><img src="/pmc/articles/instance/2118421/bin/jem2040093f07.gif" src-large="/pmc/articles/instance/2118421/bin/jem2040093f07.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2118421/" ref="ordinalpos=3939&amp;ncbi_uid=1147771&amp;link_uid=PMC2118421" image-link="/pmc/articles/PMC2118421/figure/fig7/" class="imagepopup">Figure 7. From: An essential role for RasGRP1 in mast cell function and IgE-mediated allergic response. </a></p><div class="supp"><p class="details"><div><b>A proposed model of RasGRP1 in Fc<i>ɛ</i>RI-mediated <span class="highlight" style="background-color:">signaling</span>.</b> Upon Fc<i>ɛ</i>RI cross-linking and PIP<sub>2</sub> hydrolysis, RasGRP1 is recruited to the membrane by DAG to activate N-Ras. N-Ras interacts with the catalytic p110 subunit of PI3K, which binds phosphorylated Gab2, and activates the PI3K <span class="highlight" style="background-color:">pathway</span>. RasGRP1 also controls RhoA activation indirectly through the PI3K <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yan Liu, et al. J Exp Med. 2007 January 22;204(1):93-103.</div><p class="links"><a class="dblinks" href="/pubmed/17190838" ref="ordinalpos=3939&amp;ncbi_uid=1147771&amp;link_uid=17190838">Citation</a><a class="dblinks" href="/pmc/articles/PMC2118421/" ref="ordinalpos=3939&amp;ncbi_uid=1147771&amp;link_uid=PMC2118421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3940.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1183194/figure/F5/" ref="ordinalpos=3940&amp;ncbi_uid=566755&amp;link_uid=PMC1183194" href="/pmc/articles/PMC1183194/"><img src="/pmc/articles/instance/1183194/bin/1471-2407-5-74-5.gif" src-large="/pmc/articles/instance/1183194/bin/1471-2407-5-74-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1183194/" ref="ordinalpos=3940&amp;ncbi_uid=566755&amp;link_uid=PMC1183194" image-link="/pmc/articles/PMC1183194/figure/F5/" class="imagepopup">Figure 5.  From: Response rate of fibrosarcoma cells to cytotoxic drugs on the expression level correlates to the therapeutic response rate of fibrosarcomas and is mediated by regulation of apoptotic pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span>-analyses based on decreased (green) and increased (red) genes detected in fibrosarcoma cells after 24 h treatment with doxorubicin. Results shown for caspases. </b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> maps were identified using the software package <span class="highlight" style="background-color:">Pathway</span> Assist. A minimum of 3 interacting genes is the required criteria for creation of a map (Software: <span class="highlight" style="background-color:">Pathway</span> Assist™ Stratagene).</div></p></div><div class="aux"><div class="resc">Marcus Lehnhardt, et al. BMC Cancer. 2005;5:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/16001973" ref="ordinalpos=3940&amp;ncbi_uid=566755&amp;link_uid=16001973">Citation</a><a class="dblinks" href="/pmc/articles/PMC1183194/" ref="ordinalpos=3940&amp;ncbi_uid=566755&amp;link_uid=PMC1183194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3941.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3226213/figure/fig8/" ref="ordinalpos=3941&amp;ncbi_uid=3692687&amp;link_uid=PMC3226213" href="/pmc/articles/PMC3226213/"><img src="/pmc/articles/instance/3226213/bin/PC_084608_gs_f8.gif" src-large="/pmc/articles/instance/3226213/bin/PC_084608_gs_f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3226213/" ref="ordinalpos=3941&amp;ncbi_uid=3692687&amp;link_uid=PMC3226213" image-link="/pmc/articles/PMC3226213/figure/fig8/" class="imagepopup">Figure 8. From: Arabidopsis Class I KNOTTED-Like Homeobox Proteins Act Downstream in the IDA-HAE/HSL2 Floral Abscission <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>[C][W][OA]. </a></p><div class="supp"><p class="details"><div>Models of IDA <span class="highlight" style="background-color:">Signaling</span>.<br /><b>(A)</b> to <b>(C)</b> IDA can be imagined to bind the extracellular LRRs of a HAE homodimer <b>(A)</b>, a HSL2 homodimer <b>(B)</b>, or a HAE-HSL2 heterodimer <b>(C)</b> and to cause autophosphorylation (P) of the kinase receptor domains. The protein phosphatase KAPP dephosphorylates HAE and returns the receptor to an off state (Stone et al., 1994).<br /><b>(D)</b> The signal from IDA-HAE/HSL2 is suggested to be transduced via a MAPK cascade that includes MKK4, MKK5, MPK3, and MPK6. Genetic evidence suggests that BP/KNAT1 acts as a negative downstream component of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by restricting cell wall degradation and consequently cell separation and the expression of <i>KNAT2</i> and <i>KNAT6</i>. Upon activation of the IDA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the BP/KNAT1 restriction of <i>KNAT2</i> and <i>KNAT6</i> is elevated, and these transcription factors can act as positive regulators of floral organ separation. In addition, BP/KNAT1 is a positive regulator of <i>EVR</i>, which influences cell elongation by a separate parallel <span class="highlight" style="background-color:">pathway</span> (Leslie et al., 2010).</div></p></div><div class="aux"><div class="resc">Chun-Lin Shi, et al. Plant Cell. 2011 July;23(7):2553-2567.</div><p class="links"><a class="dblinks" href="/pubmed/21742991" ref="ordinalpos=3941&amp;ncbi_uid=3692687&amp;link_uid=21742991">Citation</a><a class="dblinks" href="/pmc/articles/PMC3226213/" ref="ordinalpos=3941&amp;ncbi_uid=3692687&amp;link_uid=PMC3226213">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3942.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2758746/figure/F2/" ref="ordinalpos=3942&amp;ncbi_uid=2041542&amp;link_uid=PMC2758746" href="/pmc/articles/PMC2758746/"><img src="/pmc/articles/instance/2758746/bin/2320fig2.gif" src-large="/pmc/articles/instance/2758746/bin/2320fig2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2758746/" ref="ordinalpos=3942&amp;ncbi_uid=2041542&amp;link_uid=PMC2758746" image-link="/pmc/articles/PMC2758746/figure/F2/" class="imagepopup">Figure 2. From: A heritable switch in carbon source utilization driven by an unusual yeast prion. </a></p><div class="supp"><p class="details"><div>The Snf3/Rgt2 glucose <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> affects [<i>GAR</i><sup>+</sup>]. (<i>A</i>) Hxt3-GFP signal in [<i>gar</i><sup>−</sup>] and [<i>GAR</i><sup>+</sup>] cells (S288c background) by fluorescence microscopy. (<i>B</i>) Frequency of [<i>GAR</i><sup>+</sup>] in knockouts of members of the Snf3/Rgt2 glucose <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Δ<i>snf3</i> is completely resistant to glucosamine, and therefore [<i>GAR</i><sup>+</sup>] frequency could not be measured. Furthermore, the frequency of spontaneous glucosamine-resistant colonies in the Δ<i>rgt1</i>, Δ<i>std1</i>, and Δ<i>mths1</i> strains was close to the rate of genetic mutation, and therefore these colonies might not carry the actual [<i>GAR</i><sup>+</sup>] element. Overall, this <span class="highlight" style="background-color:">pathway</span> is enriched for genes that alter [<i>GAR</i><sup>+</sup>] frequency when knocked out relative to the library of nonessential genes (<i>P</i> = 8 × 10<sup>−6</sup>, Fisher's exact test). (<i>C</i>) The Snf3/Rgt2 glucose <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (Adapted with permission from Moriya and Johnston 2004; ©2004 National Academy of Sciences, USA.) (<i>D</i>) Measurement of [<i>GAR</i><sup>+</sup>] frequency following overexpression of Snf3/Rgt2 <span class="highlight" style="background-color:">pathway</span> members. Data are shown as mean ± standard deviation (<i>n</i> = 6). <i>STD1</i> strongly induces conversion to [<i>GAR</i><sup>+</sup>] and <i>MTH1</i> blocks it. (<i>E</i>, <i>top</i>) Tetrad spores from a [<i>GAR</i><sup>+</sup>] diploid with the genotype <i>std1∷kanMX/STD1</i>. (<i>Bottom</i>) Spores from <i>top</i> crossed to a [<i>gar</i><sup>−</sup>] strain with a wild-type <i>STD1</i> allele.</div></p></div><div class="aux"><div class="resc">Jessica C.S. Brown, et al. Genes Dev. 2009 October 1;23(19):2320-2332.</div><p class="links"><a class="dblinks" href="/pubmed/19797769" ref="ordinalpos=3942&amp;ncbi_uid=2041542&amp;link_uid=19797769">Citation</a><a class="dblinks" href="/pmc/articles/PMC2758746/" ref="ordinalpos=3942&amp;ncbi_uid=2041542&amp;link_uid=PMC2758746">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3943.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193242/figure/fig07/" ref="ordinalpos=3943&amp;ncbi_uid=3296032&amp;link_uid=PMC3193242" href="/pmc/articles/PMC3193242/"><img src="/pmc/articles/instance/3193242/bin/pnas.1114194108fig07.gif" src-large="/pmc/articles/instance/3193242/bin/pnas.1114194108fig07.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193242/" ref="ordinalpos=3943&amp;ncbi_uid=3296032&amp;link_uid=PMC3193242" image-link="/pmc/articles/PMC3193242/figure/fig07/" class="imagepopup">Fig. 7. From: The TRAF-associated protein TANK facilitates cross-talk within the I?B kinase family during Toll-like receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Model for the cross-talk between IKK family members in the MyD88 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. K63-pUb chains formed by TRAF6 in response to IL-1 or TLR stimulation bind to TAB2/3–TAK1 and IKKα/β–NEMO–TANK–TBK1/IKKε complexes enabling the sequential activation of the canonical IKKs by TAK1 and then the activation of the IKK-related kinases by the canonical IKKs and by autoactivation, as discussed in the text. The IKK-related kinases then negatively regulate the canonical IKKs by phosphorylating sites that inhibit activity.</div></p></div><div class="aux"><div class="resc">Kristopher Clark, et al. Proc Natl Acad Sci U S A. 2011 October 11;108(41):17093-17098.</div><p class="links"><a class="dblinks" href="/pubmed/21949249" ref="ordinalpos=3943&amp;ncbi_uid=3296032&amp;link_uid=21949249">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193242/" ref="ordinalpos=3943&amp;ncbi_uid=3296032&amp;link_uid=PMC3193242">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3944.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4109565/figure/F3/" ref="ordinalpos=3944&amp;ncbi_uid=6473086&amp;link_uid=PMC4109565" href="/pmc/articles/PMC4109565/"><img src="/pmc/articles/instance/4109565/bin/fphar-05-00164-g0003.gif" src-large="/pmc/articles/instance/4109565/bin/fphar-05-00164-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4109565/" ref="ordinalpos=3944&amp;ncbi_uid=6473086&amp;link_uid=PMC4109565" image-link="/pmc/articles/PMC4109565/figure/F3/" class="imagepopup">Figure 3.  From: Composition and applications of focus libraries to phenotypic assays. </a></p><div class="supp"><p class="details"><div><b>Analysis of the entire bioactivity dataset.</b> Examining biological annotations for the entire dataset including activity outside the extremes can rescue weak activity. <b>(A)</b> Depicted are three cases where compounds modulate cell <span class="highlight" style="background-color:">signaling</span>. In case 1, a compound shows high activity through, for example, activation of apoptosis. In case 2, three compounds are annotated to targets known to function in the same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> but each compound shows weak activity in the assay. In case 3, a compound shows strong activity in the assay acting through inhibition of general mechanisms such as transcription/translation. <b>(B)</b> While compounds in case 1 and 3 are easily captured using standard statistical thresholds, compounds in case 2 may only be rescued by examining the entire dataset and considering the target annotations in the context of biological <span class="highlight" style="background-color:">pathway</span> analysis. <b>(C)</b> Plot representing the aggregate effect of compounds perturbing a <span class="highlight" style="background-color:">pathway</span> to identify weakly active compounds (case 2). Each circle on the plot represents a <span class="highlight" style="background-color:">pathway</span>. The x-axis represents the aggregate effect of compounds hitting targets from the same <span class="highlight" style="background-color:">pathway</span>. For example, case 2 is depicted as having three compounds that individually show weak activity but are annotated as operating in the same <span class="highlight" style="background-color:">pathway</span>. The likelihood of getting such a score by chance, given the number of genes/proteins, is given on the y-axis (given as −log<sub>10</sub>, e.g., <i>p</i> = 0.01, 2). Using such an approach, relevant biological processes with subtle effects can be identified.</div></p></div><div class="aux"><div class="resc">Anne Mai Wassermann, et al. Front Pharmacol. 2014;5:164.</div><p class="links"><a class="dblinks" href="/pubmed/25104937" ref="ordinalpos=3944&amp;ncbi_uid=6473086&amp;link_uid=25104937">Citation</a><a class="dblinks" href="/pmc/articles/PMC4109565/" ref="ordinalpos=3944&amp;ncbi_uid=6473086&amp;link_uid=PMC4109565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3945.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2785564/figure/F1/" ref="ordinalpos=3945&amp;ncbi_uid=2527911&amp;link_uid=PMC2785564" href="/pmc/articles/PMC2785564/"><img src="/pmc/articles/instance/2785564/bin/zbc0460992430001.gif" src-large="/pmc/articles/instance/2785564/bin/zbc0460992430001.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2785564/" ref="ordinalpos=3945&amp;ncbi_uid=2527911&amp;link_uid=PMC2785564" image-link="/pmc/articles/PMC2785564/figure/F1/" class="imagepopup">FIGURE 1. From: CD38/cADPR/Ca2+ <span class="highlight" style="background-color:">Pathway</span> Promotes Cell Proliferation and Delays Nerve Growth Factor-induced Differentiation in PC12 Cells. </a></p><div class="supp"><p class="details"><div><b>Components of the cADPR/CD38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in PC12 cells.</b> <i>A</i>, expression of CD38 and RyR2 mRNA in PC12 cells were determined by RT-PCR. <i>B</i>, protein expression of CD38 and RyR2 in GFP-shRNA or CD38-shRNA-infected PC12 cells were determined by immunoblot analysis using either anti-CD38 or anti-Ryr antibody. <i>C</i>, Ca<sup>2+</sup> increases in intact PC12 cells induced by a cell-permeant cADPR agonist, cIDPRE (200 μ<span style="font-variant: small-caps" class="small-caps">m</span>), in the absence or presence of an antagonist of cADPR, 8-Br-cADPR (200 μ<span style="font-variant: small-caps" class="small-caps">m</span>).</div></p></div><div class="aux"><div class="resc">Jianbo Yue, et al. J Biol Chem. 2009 October 23;284(43):29335-29342.</div><p class="links"><a class="dblinks" href="/pubmed/19696022" ref="ordinalpos=3945&amp;ncbi_uid=2527911&amp;link_uid=19696022">Citation</a><a class="dblinks" href="/pmc/articles/PMC2785564/" ref="ordinalpos=3945&amp;ncbi_uid=2527911&amp;link_uid=PMC2785564">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3946.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1356384/figure/f8-19/" ref="ordinalpos=3946&amp;ncbi_uid=586819&amp;link_uid=PMC1356384" href="/pmc/articles/PMC1356384/"><img src="/pmc/articles/instance/1356384/bin/19FF8.gif" src-large="/pmc/articles/instance/1356384/bin/19FF8.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1356384/" ref="ordinalpos=3946&amp;ncbi_uid=586819&amp;link_uid=PMC1356384" image-link="/pmc/articles/PMC1356384/figure/f8-19/" class="imagepopup">Figure. From: Trauma-Hemorrhagic Shock Mesenteric Lymph Induces Endothelial Apoptosis That Involves Both Caspase-Dependent and Caspase-Independent Mechanisms. </a></p><div class="supp"><p class="details"><div><b>FIGURE 8.</b> Schematic diagram illustrating the major pro-apoptotic components of the caspase-dependent and caspase-independent apoptotic pathways as well as the sites where they are inhibited by Z-VAD-fmk. Caspase-dependent apoptosis occurs either through the death receptor or the mitochondrial-mediated <span class="highlight" style="background-color:">pathway</span>. In the death receptor <span class="highlight" style="background-color:">pathway</span>, the initiator caspase, caspase-8 is activated while, in the mitochondrial <span class="highlight" style="background-color:">pathway</span>, the initiator caspase, caspase-9 is activated. Both caspase-8 and caspase-9 ultimately turn on the effector caspase arm of the <span class="highlight" style="background-color:">pathway</span> as reflected in activated caspase-3. In caspase-independent apoptosis, mitochondrial membrane permeabilization triggers the release of death-promoting factors such as AIF or the serine protease Omni/htra2, which cause caspase-independent death <span class="highlight" style="background-color:">signaling</span> downstream from the mitochondria. As illustrated, z-VAD-fmk, the pan-caspase inhibitor, blocks both caspase-dependent apoptotic pathways at multiple sites. However, it has no effect on the caspase-independent apoptotic <span class="highlight" style="background-color:">pathway</span>. The results of the current study indicate that T/HS lymph-induced HUVEC apoptosis is occurring through all 3 apoptotic pathways and thus involves caspase-dependent and caspase-independent processes.</div></p></div><div class="aux"><div class="resc">Marson T. Davidson, et al. Ann Surg. 2004 July;240(1):123-131.</div><p class="links"><a class="dblinks" href="/pubmed/15213628" ref="ordinalpos=3946&amp;ncbi_uid=586819&amp;link_uid=15213628">Citation</a><a class="dblinks" href="/pmc/articles/PMC1356384/" ref="ordinalpos=3946&amp;ncbi_uid=586819&amp;link_uid=PMC1356384">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3947.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949723/figure/F3/" ref="ordinalpos=3947&amp;ncbi_uid=2506482&amp;link_uid=PMC2949723" href="/pmc/articles/PMC2949723/"><img src="/pmc/articles/instance/2949723/bin/1744-8069-6-56-3.gif" src-large="/pmc/articles/instance/2949723/bin/1744-8069-6-56-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949723/" ref="ordinalpos=3947&amp;ncbi_uid=2506482&amp;link_uid=PMC2949723" image-link="/pmc/articles/PMC2949723/figure/F3/" class="imagepopup">Figure 3.  From: Evaluation of phenoxybenzamine in the CFA model of pain following gene expression studies and connectivity mapping. </a></p><div class="supp"><p class="details"><div><b>The eicosanoid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software</b>. Only part of the <span class="highlight" style="background-color:">pathway</span> is shown. The highlighted genes (in yellow) were down-regulated in the 4 day CFA model of inflammatory pain. PLA2: phospholipase A2 group IVA; PTGS: prostaglandin-endoperoxide synthase 1; ALOX12, ALOX15: rat arachidonate 12/15-lipoxygenase.</div></p></div><div class="aux"><div class="resc">Meiping Chang, et al. Mol Pain. 2010;6:56-56.</div><p class="links"><a class="dblinks" href="/pubmed/20846436" ref="ordinalpos=3947&amp;ncbi_uid=2506482&amp;link_uid=20846436">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949723/" ref="ordinalpos=3947&amp;ncbi_uid=2506482&amp;link_uid=PMC2949723">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3948.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1183194/figure/F4/" ref="ordinalpos=3948&amp;ncbi_uid=566758&amp;link_uid=PMC1183194" href="/pmc/articles/PMC1183194/"><img src="/pmc/articles/instance/1183194/bin/1471-2407-5-74-4.gif" src-large="/pmc/articles/instance/1183194/bin/1471-2407-5-74-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1183194/" ref="ordinalpos=3948&amp;ncbi_uid=566758&amp;link_uid=PMC1183194" image-link="/pmc/articles/PMC1183194/figure/F4/" class="imagepopup">Figure 4.  From: Response rate of fibrosarcoma cells to cytotoxic drugs on the expression level correlates to the therapeutic response rate of fibrosarcomas and is mediated by regulation of apoptotic pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span>-analyses based on decreased (green) and increased (red) genes detected in fibrosarcoma cells after 24 h treatment with doxorubicin. Results shown for apoptosis. </b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> maps were identified using the software package <span class="highlight" style="background-color:">Pathway</span> Assist. A minimum of 3 interacting genes is the required criteria for creation of a map (Software: <span class="highlight" style="background-color:">Pathway</span> Assist™ Stratagene).</div></p></div><div class="aux"><div class="resc">Marcus Lehnhardt, et al. BMC Cancer. 2005;5:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/16001973" ref="ordinalpos=3948&amp;ncbi_uid=566758&amp;link_uid=16001973">Citation</a><a class="dblinks" href="/pmc/articles/PMC1183194/" ref="ordinalpos=3948&amp;ncbi_uid=566758&amp;link_uid=PMC1183194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3949.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1183194/figure/F6/" ref="ordinalpos=3949&amp;ncbi_uid=566752&amp;link_uid=PMC1183194" href="/pmc/articles/PMC1183194/"><img src="/pmc/articles/instance/1183194/bin/1471-2407-5-74-6.gif" src-large="/pmc/articles/instance/1183194/bin/1471-2407-5-74-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1183194/" ref="ordinalpos=3949&amp;ncbi_uid=566752&amp;link_uid=PMC1183194" image-link="/pmc/articles/PMC1183194/figure/F6/" class="imagepopup">Figure 6.  From: Response rate of fibrosarcoma cells to cytotoxic drugs on the expression level correlates to the therapeutic response rate of fibrosarcomas and is mediated by regulation of apoptotic pathways. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Pathway</span>-analyses based on decreased (green) and increased (red) genes detected in fibrosarcoma cells after 24 h treatment with doxorubicin. Results shown for FAS-signalling. </b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> maps were identified using the software package <span class="highlight" style="background-color:">Pathway</span> Assist. A minimum of 3 interacting genes is the required criteria for creation of a map (Software: <span class="highlight" style="background-color:">Pathway</span> Assist™ Stratagene).</div></p></div><div class="aux"><div class="resc">Marcus Lehnhardt, et al. BMC Cancer. 2005;5:74-74.</div><p class="links"><a class="dblinks" href="/pubmed/16001973" ref="ordinalpos=3949&amp;ncbi_uid=566752&amp;link_uid=16001973">Citation</a><a class="dblinks" href="/pmc/articles/PMC1183194/" ref="ordinalpos=3949&amp;ncbi_uid=566752&amp;link_uid=PMC1183194">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3950.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3107990/figure/F3/" ref="ordinalpos=3950&amp;ncbi_uid=3003197&amp;link_uid=PMC3107990" href="/pmc/articles/PMC3107990/"><img src="/pmc/articles/instance/3107990/bin/nihms291095f3.gif" src-large="/pmc/articles/instance/3107990/bin/nihms291095f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3107990/" ref="ordinalpos=3950&amp;ncbi_uid=3003197&amp;link_uid=PMC3107990" image-link="/pmc/articles/PMC3107990/figure/F3/" class="imagepopup">Figure 3. Srcasm may negatively regulate tyrosine kinase <span class="highlight" style="background-color:">signaling</span> in human esophageal squamous cell carcinoma.  From: Decreased Srcasm expression in esophageal squamous cell arcinoma in a Chinese population. </a></p><div class="supp"><p class="details"><div>Srcasm is a component of the EGFR and SFK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and act as a negative regulator of both molecules <sup>9, 12</sup>. Our data show that Srcasm protein levels are decreased in esophageal SCC compared to esophageal mucosa. We hypothesize that decreased Srcasm levels promote increased EGFR and SFK activity.</div></p></div><div class="aux"><div class="resc">Yu Qi, et al. Anticancer Res. ;30(9):3535-3539.</div><p class="links"><a class="dblinks" href="/pubmed/20944134" ref="ordinalpos=3950&amp;ncbi_uid=3003197&amp;link_uid=20944134">Citation</a><a class="dblinks" href="/pmc/articles/PMC3107990/" ref="ordinalpos=3950&amp;ncbi_uid=3003197&amp;link_uid=PMC3107990">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3951.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3341346/figure/pone-0036548-g007/" ref="ordinalpos=3951&amp;ncbi_uid=3985936&amp;link_uid=PMC3341346" href="/pmc/articles/PMC3341346/"><img src="/pmc/articles/instance/3341346/bin/pone.0036548.g007.gif" src-large="/pmc/articles/instance/3341346/bin/pone.0036548.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3341346/" ref="ordinalpos=3951&amp;ncbi_uid=3985936&amp;link_uid=PMC3341346" image-link="/pmc/articles/PMC3341346/figure/pone-0036548-g007/" class="imagepopup">Figure 7. Models of <span class="highlight" style="background-color:">signaling</span> outputs generated by dual kinase activity of the Baboon receptor..  From: R-Smad Competition Controls Activin Receptor Output in Drosophila. </a></p><div class="supp"><p class="details"><div>Given the potential of Baboon to phosphorylate and activate dSmad2 and Mad, we present several configurations of Baboon, dSmad2 and Mad interaction that would generate different <span class="highlight" style="background-color:">signaling</span> outputs. In each schematic, only the most relevant <span class="highlight" style="background-color:">pathway</span> components are drawn. The first model represents a logic gate, where the ability of Baboon to phosphorylate Mad is strictly controlled by the expression of dSmad2 in that cell (A). Another model incorporates the observed degradation of dSmad2 in response to Baboon <span class="highlight" style="background-color:">signaling</span> (B). The final model (C) incorporates feed-back or feed-forward of Mad activity through <i>dSmad2</i> regulation. If P-Mad positively regulated <i>dSmad2</i> this would prevent the complete loss of dSmad2. Conversely, if P-Mad negatively regulated <i>dSmad2</i> expression, this would enforce the switch from Baboon <span class="highlight" style="background-color:">signaling</span> through dSmad2 towards Mad.</div></p></div><div class="aux"><div class="resc">Aidan J. Peterson, et al. PLoS One. 2012;7(5):e36548.</div><p class="links"><a class="dblinks" href="/pubmed/22563507" ref="ordinalpos=3951&amp;ncbi_uid=3985936&amp;link_uid=22563507">Citation</a><a class="dblinks" href="/pmc/articles/PMC3341346/" ref="ordinalpos=3951&amp;ncbi_uid=3985936&amp;link_uid=PMC3341346">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3952.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4362472/figure/F1/" ref="ordinalpos=3952&amp;ncbi_uid=7392183&amp;link_uid=PMC4362472" href="/pmc/articles/PMC4362472/"><img src="/pmc/articles/instance/4362472/bin/fphar-06-00046-g001.gif" src-large="/pmc/articles/instance/4362472/bin/fphar-06-00046-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4362472/" ref="ordinalpos=3952&amp;ncbi_uid=7392183&amp;link_uid=PMC4362472" image-link="/pmc/articles/PMC4362472/figure/F1/" class="imagepopup">FIGURE 1.  From: Central serotonin-2A (5-HT2A) receptor dysfunction in depression and epilepsy: the missing link?. </a></p><div class="supp"><p class="details"><div><b>Anatomical and pharmacological properties of the 5-HT2A receptors in the brain. (A)</b> 5-HT<sub>2A</sub> receptors (5-HT<sub>2A</sub>-Rs) are located in brain regions involved in emotionality and epilepsy. <b>(B)</b> Interactions between 5-HT<sub>2A</sub>-Rs and beta-arrestin 2. According to the nature of the 5-HT<sub>2A</sub>-Rs agonist (endogenous/exogenous), 5-HT<sub>2A</sub>-Rs-mediated <span class="highlight" style="background-color:">signaling</span> may recruit beta-arrestin2-dependant or -independent pathways (<span class="highlight" style="background-color:">signaling</span> phases). Such a beta-arrestin2 is also involved in the down regulation/internalization of the 5-HT<sub>2A</sub>-Rs (arresting phase). <b>(C)</b> Dimerization of the 5-HT<sub>2A</sub>-Rs with GPCRs is necessary to activate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Bruno P. Guiard, et al. Front Pharmacol. 2015;6:46.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3952&amp;ncbi_uid=7392183&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4362472/" ref="ordinalpos=3952&amp;ncbi_uid=7392183&amp;link_uid=PMC4362472">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3953.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4293489/figure/Fig2/" ref="ordinalpos=3953&amp;ncbi_uid=7088213&amp;link_uid=PMC4293489" href="/pmc/articles/PMC4293489/"><img src="/pmc/articles/instance/4293489/bin/18_2014_1762_Fig2_HTML.gif" src-large="/pmc/articles/instance/4293489/bin/18_2014_1762_Fig2_HTML.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4293489/" ref="ordinalpos=3953&amp;ncbi_uid=7088213&amp;link_uid=PMC4293489" image-link="/pmc/articles/PMC4293489/figure/Fig2/" class="imagepopup">Fig. 2.  From: Co-operation of TLR4 and raft proteins in LPS-induced pro-inflammatory <span class="highlight" style="background-color:">signaling</span> . </a></p><div class="supp"><p class="details"><div>Participation of CD14 in TLR4 <span class="highlight" style="background-color:">signaling</span> pathways triggered by sLPS and rLPS. <b>a</b> In the presence of CD14, sLPS and rLPS activate TLR4 and trigger MyD88- and TRIF-dependent pathways with similar intensity. <b>b</b>, <b>c</b> Studies performed on macrophages bearing the Heedless mutation of <i>Cd14</i> have indicated that sLPS requires CD14 to activate TLR4 (<b>b</b>), while rLPS can induce TRIF-dependent <span class="highlight" style="background-color:">signaling</span> of TLR4 in CD14-deficient cells (<b>c</b>). <b>d</b>–<b>f</b> Other studies suggest that the requirement of CD14 for activation of TLR4 varies depending on the concentration of the endotoxin. At low concentrations, sLPS or rLPS are unable to activate TLR4 without the involvement of CD14 (<b>d</b>). At higher doses of sLPS, CD14 is dispensable for initiation of MyD88-dependent <span class="highlight" style="background-color:">pathway</span> of TLR4, although the production of TNF-α is submaximal in these conditions (<b>e</b>). When present in relatively high concentrations, rLPS can induce submaximal activation of both <span class="highlight" style="background-color:">signaling</span> pathways of TLR4 without binding to CD14 (<b>f</b>)</div></p></div><div class="aux"><div class="resc">Agnieszka Płóciennikowska, et al. Cell Mol Life Sci. 2015;72:557-581.</div><p class="links"><a class="dblinks" href="/pubmed/25332099" ref="ordinalpos=3953&amp;ncbi_uid=7088213&amp;link_uid=25332099">Citation</a><a class="dblinks" href="/pmc/articles/PMC4293489/" ref="ordinalpos=3953&amp;ncbi_uid=7088213&amp;link_uid=PMC4293489">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3954.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067946/figure/F3/" ref="ordinalpos=3954&amp;ncbi_uid=6306674&amp;link_uid=PMC4067946" href="/pmc/articles/PMC4067946/"><img src="/pmc/articles/instance/4067946/bin/1471-2105-14-S19-S2-3.gif" src-large="/pmc/articles/instance/4067946/bin/1471-2105-14-S19-S2-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067946/" ref="ordinalpos=3954&amp;ncbi_uid=6306674&amp;link_uid=PMC4067946" image-link="/pmc/articles/PMC4067946/figure/F3/" class="imagepopup">Figure 3.  From: iVUN: interactive Visualization of Uncertain biochemical reaction Networks. </a></p><div class="supp"><p class="details"><div><b>Analysis of parameter correlations in the JAK2/STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. By linking the graph view (A) with the Pearson correlation matrix (B), scatter plots and histograms (C), our visual analytics systems <i>iVUN </i>allowed for a step-by-step exploration of the MCMC sample obtained for the JAK2/STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [34]. The graph view (A) shows the reactions (links), states (nodes), and outputs (subsets of nodes surrounded by a semitransparent area). The sample mean of fluxes and states is mapped to the color of links and nodes respectively. The Pearson correlation matrix (B) facilitates the efficient search for pairs of strongly correlated parameters. The selection of individual parameter pairs in the Pearson correlation matrix followed by the automatic highlighting of associated edges in the graph view (A) allows for the identification of these parameters in the BRN. In the JAK2/STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, pairs of strongly correlated parameters in general influence similar states and outputs of the network. In addition, there are some parameters that alter several reaction fluxes, e.g., 'SOC3Inh', and which therefore correlate with many parameters. This yields clusters of strong parameter-wise correlations, which are recognizable in the matrix view. Beyond the analysis of linear correlations using the Pearson correlation matrix, scatter plots (C) reveal nonlinear correlation structures as shown for 'SOCS3RNADelay' and 'SOCS3RNATurn'. The parameter sample for the JAK2/STAT5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shows only few strongly nonlinear correlations, although histograms (C) reveal that, e.g., the distribution for the parameter 'SOCS3RNATurn' is bimodal.</div></p></div><div class="aux"><div class="resc">Corinna Vehlow, et al. BMC Bioinformatics. 2013;14(Suppl 19):S2-S2.</div><p class="links"><a class="dblinks" href="/pubmed/24564335" ref="ordinalpos=3954&amp;ncbi_uid=6306674&amp;link_uid=24564335">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067946/" ref="ordinalpos=3954&amp;ncbi_uid=6306674&amp;link_uid=PMC4067946">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3955.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2855202/figure/F3/" ref="ordinalpos=3955&amp;ncbi_uid=2908172&amp;link_uid=PMC2855202" href="/pmc/articles/PMC2855202/"><img src="/pmc/articles/instance/2855202/bin/nihms-155216-f0003.gif" src-large="/pmc/articles/instance/2855202/bin/nihms-155216-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2855202/" ref="ordinalpos=3955&amp;ncbi_uid=2908172&amp;link_uid=PMC2855202" image-link="/pmc/articles/PMC2855202/figure/F3/" class="imagepopup">Fig. 3.  From: Hexosamine Flux, the O-GlcNAc Modification, and the Development of Insulin Resistance in Adipocytes. </a></p><div class="supp"><p class="details"><div>Cross-talk between the hexosamine biosynthesis <span class="highlight" style="background-color:">pathway</span>, O-GlcNAc modification of proteins, <span class="highlight" style="background-color:">signaling</span> events downstream of insulin action, and glucose-induced adipocytokine secretion. Many of the proteins involved in this <span class="highlight" style="background-color:">signaling</span> network are known to be O-GlcNAc modified, but the functional roles of this modification in each case remain to be established.</div></p></div><div class="aux"><div class="resc">Chin Fen Teo, et al. Mol Cell Endocrinol. ;318(1-2):44-53.</div><p class="links"><a class="dblinks" href="/pubmed/19799964" ref="ordinalpos=3955&amp;ncbi_uid=2908172&amp;link_uid=19799964">Citation</a><a class="dblinks" href="/pmc/articles/PMC2855202/" ref="ordinalpos=3955&amp;ncbi_uid=2908172&amp;link_uid=PMC2855202">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3956.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2650750/figure/F1/" ref="ordinalpos=3956&amp;ncbi_uid=1745820&amp;link_uid=PMC2650750" href="/pmc/articles/PMC2650750/"><img src="/pmc/articles/instance/2650750/bin/nihms75884f1.gif" src-large="/pmc/articles/instance/2650750/bin/nihms75884f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2650750/" ref="ordinalpos=3956&amp;ncbi_uid=1745820&amp;link_uid=PMC2650750" image-link="/pmc/articles/PMC2650750/figure/F1/" class="imagepopup">Fig. 1. Mechanisms of inflammation-mediated alteration of insulin <span class="highlight" style="background-color:">signaling</span>.  From: Emerging Role of Adipose Tissue Hypoxia in Obesity and Insulin Resistance. </a></p><div class="supp"><p class="details"><div>Hypoxia and FFAs in obesity lead to activation of NF-kB <span class="highlight" style="background-color:">pathway</span>, which induces expression of inflammatory cytokines (TNF-a, IL-1, IL-6, et al) in adipose tissue. Through receptor-mediated signal transduction, these cytokines activates serine kinases including IKK, JNK, PKC and p70S6K to inhibit IRS-1 function. NF-kB inhibits PPARγ function through nuclear corepressor and results in suppression of the gene transcription of CAP and IRS-2, which are <span class="highlight" style="background-color:">signaling</span> molecules in PI(3)K-independent and -dependent <span class="highlight" style="background-color:">signaling</span> pathways for insulin-induced GLUT4 translocation. Hypoxia also induces FFA level through lipolysis.</div></p></div><div class="aux"><div class="resc">Jianping Ye. Int J Obes (Lond). ;33(1):54-66.</div><p class="links"><a class="dblinks" href="/pubmed/19050672" ref="ordinalpos=3956&amp;ncbi_uid=1745820&amp;link_uid=19050672">Citation</a><a class="dblinks" href="/pmc/articles/PMC2650750/" ref="ordinalpos=3956&amp;ncbi_uid=1745820&amp;link_uid=PMC2650750">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3957.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2213034/figure/fig8/" ref="ordinalpos=3957&amp;ncbi_uid=1136893&amp;link_uid=PMC2213034" href="/pmc/articles/PMC2213034/"><img src="/pmc/articles/instance/2213034/bin/20042085f8.gif" src-large="/pmc/articles/instance/2213034/bin/20042085f8.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2213034/" ref="ordinalpos=3957&amp;ncbi_uid=1136893&amp;link_uid=PMC2213034" image-link="/pmc/articles/PMC2213034/figure/fig8/" class="imagepopup">Figure 8. From: Anti-CD63 antibodies suppress IgE-dependent allergic reactions in vitro and in vivo. </a></p><div class="supp"><p class="details"><div>Schematic representation of our model for the mechanism of anti-CD63 inhibition. The Lyn–Ca<sup>2+</sup>-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for degranulation is shown in gray. The Ca<sup>2+</sup>-independent PI3K–Gab2–PKCδ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shared with the integrin–tetraspanin complex is shown in black and yellow. Mechanism 1 (steric hindrance of integrin binding to their ECM substrates) is represented in green. Mechanism 2 (general disruption of the membrane complexes containing CD63) is represented in blue. Mechanism 3 (sequestration of PI4K) is represented in red.</div></p></div><div class="aux"><div class="resc">Stefan Kraft, et al. J Exp Med. 2005 February 7;201(3):385-396.</div><p class="links"><a class="dblinks" href="/pubmed/15684326" ref="ordinalpos=3957&amp;ncbi_uid=1136893&amp;link_uid=15684326">Citation</a><a class="dblinks" href="/pmc/articles/PMC2213034/" ref="ordinalpos=3957&amp;ncbi_uid=1136893&amp;link_uid=PMC2213034">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3958.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4076348/figure/i0006-3363-89-1-13-f05/" ref="ordinalpos=3958&amp;ncbi_uid=6351115&amp;link_uid=PMC4076348" href="/pmc/articles/PMC4076348/"><img src="/pmc/articles/instance/4076348/bin/i0006-3363-89-1-13-f05.gif" src-large="/pmc/articles/instance/4076348/bin/i0006-3363-89-1-13-f05.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4076348/" ref="ordinalpos=3958&amp;ncbi_uid=6351115&amp;link_uid=PMC4076348" image-link="/pmc/articles/PMC4076348/figure/i0006-3363-89-1-13-f05/" class="imagepopup">FIG. 5.  From: Changes in Mouse Uterine Transcriptome in Estrus and Proestrus. </a></p><div class="supp"><p class="details"><div>Activation of Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse uterus in estrus versus proestrus. Upregulated genes are marked by red stars.</div></p></div><div class="aux"><div class="resc">Kerri Stanley Yip, et al. Biol Reprod. 2013 July;89(1):13-13.</div><p class="links"><a class="dblinks" href="/pubmed/23740946" ref="ordinalpos=3958&amp;ncbi_uid=6351115&amp;link_uid=23740946">Citation</a><a class="dblinks" href="/pmc/articles/PMC4076348/" ref="ordinalpos=3958&amp;ncbi_uid=6351115&amp;link_uid=PMC4076348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3959.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3203697/figure/fig7/" ref="ordinalpos=3959&amp;ncbi_uid=4040533&amp;link_uid=PMC3203697" href="/pmc/articles/PMC3203697/"><img src="/pmc/articles/instance/3203697/bin/cr2010181f7.gif" src-large="/pmc/articles/instance/3203697/bin/cr2010181f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3203697/" ref="ordinalpos=3959&amp;ncbi_uid=4040533&amp;link_uid=PMC3203697" image-link="/pmc/articles/PMC3203697/figure/fig7/" class="imagepopup">Figure 7.  From: The endoplasmic reticulum-associated degradation is necessary for plant salt tolerance. </a></p><div class="supp"><p class="details"><div>Proposed model for the role of ERAD in the salt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in plants. UP: unfolded protein.</div></p></div><div class="aux"><div class="resc">Lijing Liu, et al. Cell Res. 2011 June;21(6):957-969.</div><p class="links"><a class="dblinks" href="/pubmed/21187857" ref="ordinalpos=3959&amp;ncbi_uid=4040533&amp;link_uid=21187857">Citation</a><a class="dblinks" href="/pmc/articles/PMC3203697/" ref="ordinalpos=3959&amp;ncbi_uid=4040533&amp;link_uid=PMC3203697">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3960.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3600401/figure/F10/" ref="ordinalpos=3960&amp;ncbi_uid=5443070&amp;link_uid=PMC3600401" href="/pmc/articles/PMC3600401/"><img src="/pmc/articles/instance/3600401/bin/nihms-421160-f0013.gif" src-large="/pmc/articles/instance/3600401/bin/nihms-421160-f0013.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3600401/" ref="ordinalpos=3960&amp;ncbi_uid=5443070&amp;link_uid=PMC3600401" image-link="/pmc/articles/PMC3600401/figure/F10/" class="imagepopup">Fig. 10. The mechanistic <span class="highlight" style="background-color:">pathway</span> for inhibition of Ang II-induced MC contraction by synthetic triterpenoids.  From: The synthetic triterpenoid, RTA405, increases glomerular filtration rate and reduces angiotensin II-induced contraction of glomerular mesangial cells. </a></p><div class="supp"><p class="details"><div>In MCs, Ang II generates ROS by activating NADPH oxdiases (NOX) through AT1 receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. ROS subsequently raises [Ca<sup>2+</sup>]<sub>i</sub> either by stimulating Ca<sup>2+</sup> release from endoplastic reticulum (ER)/sarcoplastic reticulum (SR) or by acutely activating Ca<sup>2+</sup> permeable channels in the plasma membrane or both. An elevation of intracellular Ca<sup>2+</sup> level triggers MC contraction and eventually results in a decrease in GFR. Synthetic triterpenoids bind to Keap1 in the cytoplasm of MC, which activates the Nrf2 transcription factor. In the nucleus of MCs, activated Nrf2 then promotes transcription of a variety of genes encoding antioxidant enzymes. The increased abundance of antioxidant enzymes reduces intracellular ROS produced by the Ang II <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, and consequently maintains an intracellular redox balance during Ang II stimulation (for simplicity, a classical Ang II-AT1 receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was omitted).</div></p></div><div class="aux"><div class="resc">Yanfeng Ding, et al. Kidney Int. 2013 May;83(5):845-854.</div><p class="links"><a class="dblinks" href="/pubmed/23235569" ref="ordinalpos=3960&amp;ncbi_uid=5443070&amp;link_uid=23235569">Citation</a><a class="dblinks" href="/pmc/articles/PMC3600401/" ref="ordinalpos=3960&amp;ncbi_uid=5443070&amp;link_uid=PMC3600401">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3961.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2785474/figure/f2-09_97_xiang/" ref="ordinalpos=3961&amp;ncbi_uid=2134718&amp;link_uid=PMC2785474" href="/pmc/articles/PMC2785474/"><img src="/pmc/articles/instance/2798896/bin/09_97_xiangf2.gif" src-large="/pmc/articles/instance/2798896/bin/09_97_xiangf2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2785474/" ref="ordinalpos=3961&amp;ncbi_uid=2134718&amp;link_uid=PMC2785474" image-link="/pmc/articles/PMC2785474/figure/f2-09_97_xiang/" class="imagepopup">Figure 2.  From: Pattern Recognition Receptor-Dependent Mechanisms of Acute Lung Injury. </a></p><div class="supp"><p class="details"><div>TLR <span class="highlight" style="background-color:">signaling</span> (an example in macrophages and dendritic cells). TLR2 (TLR2 in association with TLR1 or TLR6), TLR4, TLR5 and TLR11 are localized on the cell surface for ligand recognition. TLR3, TLR7 and TLR9 are localized in the endosome for ligand recognition in the lumen of endosome. All TLRs, except TLR3, recruit MyD88, and TLR1, TLR2, TLR4 and TLR6 recruit the additional adaptor TIRAP, which links the TIR domain with MyD88. TLR3 and TLR4 recruit TRIF. TLR4 requires the additional linker adaptor TRAM, which links the TIR domain of TLR4 with TRIF. Stimulation of the cells with TLR1, TLR2, TLR5, TLR6 and TLR11 ligands initiates the MyD88-dependent <span class="highlight" style="background-color:">pathway</span> whereas TLR3 ligands initiate the TRIF-dependent <span class="highlight" style="background-color:">pathway</span>. TLR4 activates both MyD88-dependent and TRIF-dependent pathways. In the MyD88-dependent <span class="highlight" style="background-color:">pathway</span>, MyD88 recruits the IRAK family of proteins and TRAF6. In turn, TRAF6 activates TAK1. The activated TAK1 activates the IKK complex, which activates NF-κB subunits. The activated TAK1 also activates the MAPK <span class="highlight" style="background-color:">pathway</span>. In the TRIF-dependent <span class="highlight" style="background-color:">pathway</span>, TRIF interacts with RIP1 and TRAF6. Activated TRAF6 and RIP1 activate NF-κB and MAPKs. TRIF also interacts with TRAF3 and activates TBK1/IKKi, which activate IRF3 and IRF7. Cells stimulated with TLR7 and TLR9 ligands activate NF-κB and MAPKs via the MyD88-dependent <span class="highlight" style="background-color:">pathway</span>. To induce type I IFNs, MyD88 associates with the IRAK family of proteins. IRAK1 and IKKα activate IRF7. IRAK1 also interacts with TRAF3 and activates IRF7. The activated NF-κB subunits and IRFs are translocated to the nucleus. NF-κB and MAPKs initiate the transcription of inflammatory cytokine genes whereas IRFs initiate the transcription of type I interferons. <br />(The figure is adapted from [13] and used with permission from Elsevier.)</div></p></div><div class="aux"><div class="resc">Meng Xiang, et al. Mol Med. 2010 Jan-Feb;16(1-2):69-82.</div><p class="links"><a class="dblinks" href="/pubmed/19949486" ref="ordinalpos=3961&amp;ncbi_uid=2134718&amp;link_uid=19949486">Citation</a><a class="dblinks" href="/pmc/articles/PMC2785474/" ref="ordinalpos=3961&amp;ncbi_uid=2134718&amp;link_uid=PMC2785474">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3962.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3912804/figure/F1/" ref="ordinalpos=3962&amp;ncbi_uid=5822251&amp;link_uid=PMC3912804" href="/pmc/articles/PMC3912804/"><img src="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g001.gif" src-large="/pmc/articles/instance/3912804/bin/IJPharm-46-3-g001.jpg" alt="Figure 1a" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3912804/" ref="ordinalpos=3962&amp;ncbi_uid=5822251&amp;link_uid=PMC3912804" image-link="/pmc/articles/PMC3912804/figure/F1/" class="imagepopup">Figure 1a.  From: Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A novel target for cancer therapy: Vismodegib, a promising therapeutic option in treatment of basal cell carcinomas. </a></p><div class="supp"><p class="details"><div>Hedgehog <span class="highlight" style="background-color:">signaling</span> pathways in vertebrates-without hedgehog ligand</div></p></div><div class="aux"><div class="resc">Afroz Abidi. Indian J Pharmacol. 2014 Jan-Feb;46(1):3-12.</div><p class="links"><a class="dblinks" href="/pubmed/24550577" ref="ordinalpos=3962&amp;ncbi_uid=5822251&amp;link_uid=24550577">Citation</a><a class="dblinks" href="/pmc/articles/PMC3912804/" ref="ordinalpos=3962&amp;ncbi_uid=5822251&amp;link_uid=PMC3912804">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3963.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3417165/figure/F2/" ref="ordinalpos=3963&amp;ncbi_uid=4180958&amp;link_uid=PMC3417165" href="/pmc/articles/PMC3417165/"><img src="/pmc/articles/instance/3417165/bin/fimmu-03-00245-g002.gif" src-large="/pmc/articles/instance/3417165/bin/fimmu-03-00245-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3417165/" ref="ordinalpos=3963&amp;ncbi_uid=4180958&amp;link_uid=PMC3417165" image-link="/pmc/articles/PMC3417165/figure/F2/" class="imagepopup">Figure 2.  From: The Role of the PI3K <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in CD4+ T Cell Differentiation and Function. </a></p><div class="supp"><p class="details"><div><b>Intracellular “tug-of-war” determines T cell fate</b>. Green arrow indicates activation of PI3K <span class="highlight" style="background-color:">signaling</span>; red arrow indicates inhibition of PI3K <span class="highlight" style="background-color:">signaling</span>. The strength of PI3K <span class="highlight" style="background-color:">signaling</span> is influenced by opposing activation and inhibitory signals that are integrated from extracellular stimuli such as co-stimulation, co-inhibition, and cytokines/adipokines. Ultimately, the outcome of this “tug-of-war” determines whether a T cell becomes a T<sub>reg</sub> to mediate tolerance, or inflammatory Th subsets to mediate immunity.</div></p></div><div class="aux"><div class="resc">Jonathan M. Han, et al. Front Immunol. 2012;3:245.</div><p class="links"><a class="dblinks" href="/pubmed/22905034" ref="ordinalpos=3963&amp;ncbi_uid=4180958&amp;link_uid=22905034">Citation</a><a class="dblinks" href="/pmc/articles/PMC3417165/" ref="ordinalpos=3963&amp;ncbi_uid=4180958&amp;link_uid=PMC3417165">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3964.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3164466/figure/F6/" ref="ordinalpos=3964&amp;ncbi_uid=3405883&amp;link_uid=PMC3164466" href="/pmc/articles/PMC3164466/"><img src="/pmc/articles/instance/3164466/bin/3206fig6.gif" src-large="/pmc/articles/instance/3164466/bin/3206fig6.jpg" alt="FIGURE 6:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3164466/" ref="ordinalpos=3964&amp;ncbi_uid=3405883&amp;link_uid=PMC3164466" image-link="/pmc/articles/PMC3164466/figure/F6/" class="imagepopup">FIGURE 6:.  From: Phosphorylation of Serine 114 on Atg32 mediates mitophagy. </a></p><div class="supp"><p class="details"><div>Model of mitophagy in yeast. The initial trigger-inducing mitophagy comes from mitochondria (such as mitochondrial damage) and/or the cellular environment (such as dramatic nutrient change and oxidative stresses). The initial trigger is through an unidentified mitophagy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and it reaches kinase X, which is believed to be a Ser/Thr–specific protein kinase. This kinase phosphorylates Ser-114 and Ser-119 on Atg32. Phosphorylation of Atg32, especially phosphorylation of Ser-114, mediates the Atg11–Atg32 interaction. Atg11 recruits mitochondria to the PAS, where the autophagosome is generated to enclose the mitochondria. Hog1 and Pbs2 affect the mitophagy <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or affect kinase X.</div></p></div><div class="aux"><div class="resc">Yoshimasa Aoki, et al. Mol Biol Cell. 2011 September 1;22(17):3206-3217.</div><p class="links"><a class="dblinks" href="/pubmed/21757540" ref="ordinalpos=3964&amp;ncbi_uid=3405883&amp;link_uid=21757540">Citation</a><a class="dblinks" href="/pmc/articles/PMC3164466/" ref="ordinalpos=3964&amp;ncbi_uid=3405883&amp;link_uid=PMC3164466">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3965.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3815519/figure/F3/" ref="ordinalpos=3965&amp;ncbi_uid=5677576&amp;link_uid=PMC3815519" href="/pmc/articles/PMC3815519/"><img src="/pmc/articles/instance/3815519/bin/nihms504402f3.gif" src-large="/pmc/articles/instance/3815519/bin/nihms504402f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3815519/" ref="ordinalpos=3965&amp;ncbi_uid=5677576&amp;link_uid=PMC3815519" image-link="/pmc/articles/PMC3815519/figure/F3/" class="imagepopup">Figure 3.  From: Genetic interactions within inositol-related pathways are associated with longitudinal changes in ventricle size. </a></p><div class="supp"><p class="details"><div>Phosphatidylinositol and Akt/PKB Survival Signalling <span class="highlight" style="background-color:">Pathway</span>, adapted from the KEGG Phosphatidylinositol <span class="highlight" style="background-color:">signaling</span> system (hsa04070, found at http://www.genome.jp/dbget-bin/www_bget?hsa04070)</div></p></div><div class="aux"><div class="resc">Mary Ellen I. Koran, et al. J Alzheimers Dis. 2014 January 1;38(1):10.3233/JAD-130989.</div><p class="links"><a class="dblinks" href="/pubmed/24077433" ref="ordinalpos=3965&amp;ncbi_uid=5677576&amp;link_uid=24077433">Citation</a><a class="dblinks" href="/pmc/articles/PMC3815519/" ref="ordinalpos=3965&amp;ncbi_uid=5677576&amp;link_uid=PMC3815519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3966.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3644182/figure/F1/" ref="ordinalpos=3966&amp;ncbi_uid=5760724&amp;link_uid=PMC3644182" href="/pmc/articles/PMC3644182/"><img src="/pmc/articles/instance/3644182/bin/nihms402865f1.gif" src-large="/pmc/articles/instance/3644182/bin/nihms402865f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3644182/" ref="ordinalpos=3966&amp;ncbi_uid=5760724&amp;link_uid=PMC3644182" image-link="/pmc/articles/PMC3644182/figure/F1/" class="imagepopup">Fig. 1.  From: Improving outcomes after cardiac arrest using NO inhalation. </a></p><div class="supp"><p class="details"><div>Nitric oxide <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. cGMP, cyclic guanosine monophosphate; GTP, guanosine triphosphate; GMP, guanosine monophosphate.</div></p></div><div class="aux"><div class="resc">Fumito Ichinose. Trends Cardiovasc Med. 2013 February;23(2):52-58.</div><p class="links"><a class="dblinks" href="/pubmed/23291033" ref="ordinalpos=3966&amp;ncbi_uid=5760724&amp;link_uid=23291033">Citation</a><a class="dblinks" href="/pmc/articles/PMC3644182/" ref="ordinalpos=3966&amp;ncbi_uid=5760724&amp;link_uid=PMC3644182">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3967.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440590/figure/fig01/" ref="ordinalpos=3967&amp;ncbi_uid=4234870&amp;link_uid=PMC3440590" href="/pmc/articles/PMC3440590/"><img src="/pmc/articles/instance/3440590/bin/bies0034-0542-f1.gif" src-large="/pmc/articles/instance/3440590/bin/bies0034-0542-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440590/" ref="ordinalpos=3967&amp;ncbi_uid=4234870&amp;link_uid=PMC3440590" image-link="/pmc/articles/PMC3440590/figure/fig01/" class="imagepopup">Figure 1.  From: Short-term information processing, long-term responses: Insights by mathematical modeling of signal transduction. </a></p><div class="supp"><p class="details"><div>The integrated activation curve of a key mediator can predict cell fate. <b>A:</b> Scheme of an exemplary <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, in which stimulation with two different ligands leads to activation of distinct <span class="highlight" style="background-color:">signaling</span> molecules. This information is summarized by a key mediator protein and processed by a signal integrator, resulting in a specific cell fate. Lines with open circles: reaction catalysis; crossed circle: unspecified source or sink. Layout according to Systems Biology Graphical Notation 39. <b>B:</b> Activation dynamics of the <span class="highlight" style="background-color:">signaling</span> molecules shown in (A). The two different ligands stimulate different <span class="highlight" style="background-color:">signaling</span> molecules both of which can activate the same downstream molecule. Since this molecule summarizes the information from both upstream <span class="highlight" style="background-color:">signaling</span> molecules and transfers it to another molecule, it functions as a key mediator. The integral of the activated key mediator, defined as the area under the <span class="highlight" style="background-color:">signaling</span> curve, is illustrated by the filled area. This quantity directly correlates with the dynamic behavior of a long-living signal integrator and is thus a decoder for the information from the input signals. Potential signal integrators are stable mRNA molecules and the proteins they encode, which then execute the specific cell fate.</div></p></div><div class="aux"><div class="resc">Annette Schneider, et al. Bioessays. 2012 July;34(7):542-550.</div><p class="links"><a class="dblinks" href="/pubmed/22528856" ref="ordinalpos=3967&amp;ncbi_uid=4234870&amp;link_uid=22528856">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440590/" ref="ordinalpos=3967&amp;ncbi_uid=4234870&amp;link_uid=PMC3440590">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3968.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2430908/figure/pbio-0060123-g008/" ref="ordinalpos=3968&amp;ncbi_uid=1349131&amp;link_uid=PMC2430908" href="/pmc/articles/PMC2430908/"><img src="/pmc/articles/instance/2430908/bin/pbio.0060123.g008.gif" src-large="/pmc/articles/instance/2430908/bin/pbio.0060123.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2430908/" ref="ordinalpos=3968&amp;ncbi_uid=1349131&amp;link_uid=PMC2430908" image-link="/pmc/articles/PMC2430908/figure/pbio-0060123-g008/" class="imagepopup">Figure 8. Model of the Role of the Notch <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Setting up Competent Skin Barrier and Preventing B-LPD in Newborn Animals.  From: Notch-Deficient Skin Induces a Lethal Systemic B-Lymphoproliferative Disorder by Secreting TSLP, a Sentinel for Epidermal Integrity. </a></p><div class="supp"><p class="details"><div>RBP-j-dependent and independent Notch <span class="highlight" style="background-color:">signaling</span> is required for proper epidermal differentiation and lipid biogenesis in the skin to form a competent barrier. The RBP-j-independent signals may be non-cell autonomous. Embryonic loss of Notch <span class="highlight" style="background-color:">signaling</span> leads to aberrant epidermal differentiation/lipid biogenesis and, therefore, the defective barrier formation. In response, TSLP is released by keratinocytes that fail to differentiate, triggering a systemic B-LPD in newborns that results in neonatal lethality at the highest level.</div></p></div><div class="aux"><div class="resc">Shadmehr Demehri, et al. PLoS Biol. 2008 May;6(5):e123.</div><p class="links"><a class="dblinks" href="/pubmed/18507503" ref="ordinalpos=3968&amp;ncbi_uid=1349131&amp;link_uid=18507503">Citation</a><a class="dblinks" href="/pmc/articles/PMC2430908/" ref="ordinalpos=3968&amp;ncbi_uid=1349131&amp;link_uid=PMC2430908">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3969.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4163447/figure/fig3/" ref="ordinalpos=3969&amp;ncbi_uid=6636437&amp;link_uid=PMC4163447" href="/pmc/articles/PMC4163447/"><img src="/pmc/articles/instance/4163447/bin/BMRI2014-236385.003.gif" src-large="/pmc/articles/instance/4163447/bin/BMRI2014-236385.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4163447/" ref="ordinalpos=3969&amp;ncbi_uid=6636437&amp;link_uid=PMC4163447" image-link="/pmc/articles/PMC4163447/figure/fig3/" class="imagepopup">Figure 3.  From: MicroRNA Dysregulation in Liver and Pancreas of CMP-Neu5Ac Hydroxylase Null Mice Disrupts Insulin/PI3K-AKT <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Expression changes in the insulin/PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (a) Expression levels of<i> IRS1</i>,<i> IRS2</i>,<i> AKT1</i>, and<i> FOXO1</i> mRNAs in livers (left) and pancreas (right) of control and<i> Cmah</i>-null mice were determined using real-time reverse transcription PCR (RT-qPCR) (*<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01; ***<i>P</i> &lt; 0.001). (b) Western blot analysis of proteins involved in the target <span class="highlight" style="background-color:">pathway</span> in liver (left) and pancreas (right) of control and<i> Cmah</i>-null mice. Actin was used as a loading control. Band intensities were quantified by image processing and analysis was made using ImageJ v1.32.</div></p></div><div class="aux"><div class="resc">Deug-Nam Kwon, et al. Biomed Res Int. 2014;2014:236385.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3969&amp;ncbi_uid=6636437&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4163447/" ref="ordinalpos=3969&amp;ncbi_uid=6636437&amp;link_uid=PMC4163447">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3970.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3364461/figure/F1/" ref="ordinalpos=3970&amp;ncbi_uid=4043517&amp;link_uid=PMC3364461" href="/pmc/articles/PMC3364461/"><img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-g001.gif" src-large="/pmc/articles/instance/3364461/bin/fendo-02-00045-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3364461/" ref="ordinalpos=3970&amp;ncbi_uid=4043517&amp;link_uid=PMC3364461" image-link="/pmc/articles/PMC3364461/figure/F1/" class="imagepopup">Figure 1.  From: Mapping the Follicle-Stimulating Hormone-Induced <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div><b>Multiple transduction mechanisms induced at the FSHR</b>. The Gαs/cAMP/protein kinase A (PKA) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has classically been considered as the key <span class="highlight" style="background-color:">signaling</span> mechanism triggered at the FSHR. Over the last decade, mounting evidence have shown that FSHR also engages other transduction mechanisms mainly coupling to other Gα subunits, β-arrestin-dependent <span class="highlight" style="background-color:">signaling</span>, EGFR transactivation, and APPL1-mediated signals. Bibliographic references are indicated on the map by the first author’s name with two digits for the year of publication. The CellDesigner program has been used to represent <span class="highlight" style="background-color:">signaling</span> pathways as shown in Figures 1–7. Complexes are surrounded by a box. Dashed lines indicate indirect reactions. The following semantic has been used: proteins (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i001.jpg" class="inline-graphic" style="border-style: none;" />); active protein (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i002.jpg" class="inline-graphic" style="border-style: none;" />); receptor (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i003.jpg" class="inline-graphic" style="border-style: none;" />); transcript (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i004.jpg" class="inline-graphic" style="border-style: none;" />); gene (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i005.jpg" class="inline-graphic" style="border-style: none;" />); catalysis (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i006.jpg" class="inline-graphic" style="border-style: none;" />); association (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i007.jpg" class="inline-graphic" style="border-style: none;" />); dissociation (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i008.jpg" class="inline-graphic" style="border-style: none;" />); inhibition (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i009.jpg" class="inline-graphic" style="border-style: none;" />); phosphorylation (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i010.jpg" class="inline-graphic" style="border-style: none;" />); degradation (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i011.jpg" class="inline-graphic" style="border-style: none;" />); known transition omitted (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i012.jpg" class="inline-graphic" style="border-style: none;" />); unknown transition (<img src="/pmc/articles/instance/3364461/bin/fendo-02-00045-i013.jpg" class="inline-graphic" style="border-style: none;" />).</div></p></div><div class="aux"><div class="resc">Pauline Gloaguen, et al. Front Endocrinol (Lausanne). 2011;2:45.</div><p class="links"><a class="dblinks" href="/pubmed/22666216" ref="ordinalpos=3970&amp;ncbi_uid=4043517&amp;link_uid=22666216">Citation</a><a class="dblinks" href="/pmc/articles/PMC3364461/" ref="ordinalpos=3970&amp;ncbi_uid=4043517&amp;link_uid=PMC3364461">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3971.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3372387/figure/F1/" ref="ordinalpos=3971&amp;ncbi_uid=4855695&amp;link_uid=PMC3372387" href="/pmc/articles/PMC3372387/"><img src="/pmc/articles/instance/3372387/bin/cc-11-1746-g1.gif" src-large="/pmc/articles/instance/3372387/bin/cc-11-1746-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3372387/" ref="ordinalpos=3971&amp;ncbi_uid=4855695&amp;link_uid=PMC3372387" image-link="/pmc/articles/PMC3372387/figure/F1/" class="imagepopup">Figure. From: Pro-apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by CDK5 and MEKK1. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> <span class="highlight" style="background-color:">Signaling</span> pathways from the stressed ER. The green arrow indicates the three known Unfolded Protein Response pathways mediated by transcription factors ATF4, ATF6 and XBP1. The predominant effect of these pathways is to induce quality control genes that help the cell survive ER stress. Although ATF4 induces CHOP that can aggravate stress, this aspect is a byproduct of a quality control enzyme, Ero-1L, involved in oxidative protein folding. The red arrow shows the newly discovered CDK5/MEKK1 <span class="highlight" style="background-color:">pathway</span> that mediates cell death in a Drosophila model for Autosomal Dominant Retinitis Pigmentosa.</div></p></div><div class="aux"><div class="resc">Hyung Don Ryoo. Cell Cycle. 2012 May 1;11(9):1746-1747.</div><p class="links"><a class="dblinks" href="/pubmed/22510567" ref="ordinalpos=3971&amp;ncbi_uid=4855695&amp;link_uid=22510567">Citation</a><a class="dblinks" href="/pmc/articles/PMC3372387/" ref="ordinalpos=3971&amp;ncbi_uid=4855695&amp;link_uid=PMC3372387">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3972.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3442418/figure/F8/" ref="ordinalpos=3972&amp;ncbi_uid=4397766&amp;link_uid=PMC3442418" href="/pmc/articles/PMC3442418/"><img src="/pmc/articles/instance/3442418/bin/3722fig8.gif" src-large="/pmc/articles/instance/3442418/bin/3722fig8.jpg" alt="FIGURE 8:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3442418/" ref="ordinalpos=3972&amp;ncbi_uid=4397766&amp;link_uid=PMC3442418" image-link="/pmc/articles/PMC3442418/figure/F8/" class="imagepopup">FIGURE 8:.  From: Interplay between MEK-ERK <span class="highlight" style="background-color:">signaling</span>, cyclin D1, and cyclin-dependent kinase 5 regulates cell cycle reentry and apoptosis of neurons. </a></p><div class="supp"><p class="details"><div>A model for cell cycle reentry–mediated apoptosis. Neurotoxic signals might trigger aberrant activation of the MEK-ERK <span class="highlight" style="background-color:">pathway</span>, resulting in an increase in cyclin D1 levels, which can be by transcription factor STAT3. Cyclin D1 prevents p35-mediated cdk5 activation, which might facilitate the increase in MEK-ERK <span class="highlight" style="background-color:">signaling</span> activity. These events can contribute to cell cycle reentry of neurons, which results in their death.</div></p></div><div class="aux"><div class="resc">Prashant Kumar Modi, et al. Mol Biol Cell. 2012 September 15;23(18):3722-3730.</div><p class="links"><a class="dblinks" href="/pubmed/22833568" ref="ordinalpos=3972&amp;ncbi_uid=4397766&amp;link_uid=22833568">Citation</a><a class="dblinks" href="/pmc/articles/PMC3442418/" ref="ordinalpos=3972&amp;ncbi_uid=4397766&amp;link_uid=PMC3442418">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3973.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC209287/figure/F1/" ref="ordinalpos=3973&amp;ncbi_uid=264313&amp;link_uid=PMC209287" href="/pmc/articles/PMC209287/"><img src="/pmc/articles/instance/209287/bin/JCI0112853.f1.gif" src-large="/pmc/articles/instance/209287/bin/JCI0112853.f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC209287/" ref="ordinalpos=3973&amp;ncbi_uid=264313&amp;link_uid=PMC209287" image-link="/pmc/articles/PMC209287/figure/F1/" class="imagepopup">Figure 1.  From: A TWIST in the fate of human osteoblasts identifies <span class="highlight" style="background-color:">signaling</span> molecules involved in skull development. </a></p><div class="supp"><p class="details"><div>Elements involved in the molecular <span class="highlight" style="background-color:">pathway</span> of sutural osteoblast differentiation (modified from ref. 15). Top: normal cellular differentiation and relationship of TWIST and FGFR <span class="highlight" style="background-color:">signaling</span>. Bottom: altered gene expression in TWIST Y103X mutant osteoblasts from a Saethre-Chotzen syndrome patient and in FGFR2 S252W mutant osteoblasts from an Apert syndrome patient. Bone sialoprotein is a mature osteoblast marker.</div></p></div><div class="aux"><div class="resc">Ethylin Wang Jabs. J Clin Invest. 2001 May 1;107(9):1075-1077.</div><p class="links"><a class="dblinks" href="/pubmed/11342569" ref="ordinalpos=3973&amp;ncbi_uid=264313&amp;link_uid=11342569">Citation</a><a class="dblinks" href="/pmc/articles/PMC209287/" ref="ordinalpos=3973&amp;ncbi_uid=264313&amp;link_uid=PMC209287">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3974.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3139433/figure/F4/" ref="ordinalpos=3974&amp;ncbi_uid=3620209&amp;link_uid=PMC3139433" href="/pmc/articles/PMC3139433/"><img src="/pmc/articles/instance/3139433/bin/nihms297490f4.gif" src-large="/pmc/articles/instance/3139433/bin/nihms297490f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3139433/" ref="ordinalpos=3974&amp;ncbi_uid=3620209&amp;link_uid=PMC3139433" image-link="/pmc/articles/PMC3139433/figure/F4/" class="imagepopup">Figure 4. Interactome map generated by the Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis tool.  From: Identification of predictive markers of cytarabine response in AML by integrative analysis of gene-expression profiles with multiple phenotypes. </a></p><div class="supp"><p class="details"><div>A total of 60 validated genes were analyzed for any reported association with genes belonging to apoptosis <span class="highlight" style="background-color:">signaling</span> and/or cell cycle and proliferation networks in Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. Only those genes that were directly associated with our gene(s) of interest were retained in the network shown. For the sake of simplicity the resulting network was additionally trimmed to represent relationships with validated genes.</div></p></div><div class="aux"><div class="resc">Jatinder K Lamba, et al. Pharmacogenomics. ;12(3):327-339.</div><p class="links"><a class="dblinks" href="/pubmed/21449673" ref="ordinalpos=3974&amp;ncbi_uid=3620209&amp;link_uid=21449673">Citation</a><a class="dblinks" href="/pmc/articles/PMC3139433/" ref="ordinalpos=3974&amp;ncbi_uid=3620209&amp;link_uid=PMC3139433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3975.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3198763/figure/F2/" ref="ordinalpos=3975&amp;ncbi_uid=3298341&amp;link_uid=PMC3198763" href="/pmc/articles/PMC3198763/"><img src="/pmc/articles/instance/3198763/bin/1741-7007-9-69-2.gif" src-large="/pmc/articles/instance/3198763/bin/1741-7007-9-69-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3198763/" ref="ordinalpos=3975&amp;ncbi_uid=3298341&amp;link_uid=PMC3198763" image-link="/pmc/articles/PMC3198763/figure/F2/" class="imagepopup">Figure 2.  From: Appetite for destruction: the inhibition of glycolysis as a therapy for tuberous sclerosis complex-related tumors. </a></p><div class="supp"><p class="details"><div><b>The mTORC1 <span class="highlight" style="background-color:">pathway</span> controls cellular metabolism</b>. The mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controls metabolic pathways active in proliferating cells. This schematic shows our current understanding of how glycolysis, oxidative phosphorylation, the pentose phosphate <span class="highlight" style="background-color:">pathway</span> (PPP), and glutamine metabolism are interconnected in proliferating cells. The mTORC1 <span class="highlight" style="background-color:">pathway</span> participates in this metabolic rewiring by controlling the expression of genes (depicted in blue) encoding enzymes involved in glycolysis, the PPP, and lipid synthesis. This metabolic control requires the up-regulation of c-Myc and HIF1α (glycolysis), and SREBP1 (lipid biosynthesis and the PPP). mTORC1-dependent metabolic regulation allows for production of both NADPH and intermediates for macromolecular synthesis (depicted in red). c-Myc drives glutamine metabolism, which also supports NADH production.</div></p></div><div class="aux"><div class="resc">Alfredo Csibi, et al. BMC Biol. 2011;9:69-69.</div><p class="links"><a class="dblinks" href="/pubmed/22018140" ref="ordinalpos=3975&amp;ncbi_uid=3298341&amp;link_uid=22018140">Citation</a><a class="dblinks" href="/pmc/articles/PMC3198763/" ref="ordinalpos=3975&amp;ncbi_uid=3298341&amp;link_uid=PMC3198763">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3976.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4074168/figure/pone-0101277-g007/" ref="ordinalpos=3976&amp;ncbi_uid=6315602&amp;link_uid=PMC4074168" href="/pmc/articles/PMC4074168/"><img src="/pmc/articles/instance/4074168/bin/pone.0101277.g007.gif" src-large="/pmc/articles/instance/4074168/bin/pone.0101277.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4074168/" ref="ordinalpos=3976&amp;ncbi_uid=6315602&amp;link_uid=PMC4074168" image-link="/pmc/articles/PMC4074168/figure/pone-0101277-g007/" class="imagepopup">Figure 7. The proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> triggered by NB and Cur in combination in A375 cells..  From: Synergistic Apoptosis-Inducing Effects on A375 Human Melanoma Cells of Natural Borneol and Curcumin. </a></p><div class="supp"><p class="details"><div>NB increases the cellular uptake of Cur, which subsequently improves intracellular ROS generation. Overproduction of ROS leads to accumulation of DNA damage. DNA damage activates p53 <span class="highlight" style="background-color:">pathway</span>, which causes mitochondrial dysfunction. Mitochondrial dysfunction activates several caspase cascades, and finally leads to cell apoptosis. In addition, ROS also enhances the phosphorylation of JNK and inactivation of Akt and ERK, which enhances the p53 activation and promotes the apoptotic cascade.</div></p></div><div class="aux"><div class="resc">Jianping Chen, et al. PLoS One. 2014;9(6):e101277.</div><p class="links"><a class="dblinks" href="/pubmed/24971451" ref="ordinalpos=3976&amp;ncbi_uid=6315602&amp;link_uid=24971451">Citation</a><a class="dblinks" href="/pmc/articles/PMC4074168/" ref="ordinalpos=3976&amp;ncbi_uid=6315602&amp;link_uid=PMC4074168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3977.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3991588/figure/pone-0095259-g006/" ref="ordinalpos=3977&amp;ncbi_uid=6042500&amp;link_uid=PMC3991588" href="/pmc/articles/PMC3991588/"><img src="/pmc/articles/instance/3991588/bin/pone.0095259.g006.gif" src-large="/pmc/articles/instance/3991588/bin/pone.0095259.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3991588/" ref="ordinalpos=3977&amp;ncbi_uid=6042500&amp;link_uid=PMC3991588" image-link="/pmc/articles/PMC3991588/figure/pone-0095259-g006/" class="imagepopup">Figure 6. Orexin-A-induced astrocytes migration is dependent on Ca2+ <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Orexin-A Promotes Cell Migration in Cultured Rat Astrocytes via Ca2+-Dependent PKC? and ERK1/2 Signals. </a></p><div class="supp"><p class="details"><div>A: Representative wound healing images showing the effects of various Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> inhibitors on orexin-A-induced astrocytes migration. B: statistical analysis of wound healing assay. Data are expressed as means ± SEM, n = 5, *p&lt;0.05 <i>vs.</i> control. Scale bar  = 20 µm. The images were selected from five individual experiments.</div></p></div><div class="aux"><div class="resc">Qing Shu, et al. PLoS One. 2014;9(4):e95259.</div><p class="links"><a class="dblinks" href="/pubmed/24748172" ref="ordinalpos=3977&amp;ncbi_uid=6042500&amp;link_uid=24748172">Citation</a><a class="dblinks" href="/pmc/articles/PMC3991588/" ref="ordinalpos=3977&amp;ncbi_uid=6042500&amp;link_uid=PMC3991588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3978.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3995118/figure/f5/" ref="ordinalpos=3978&amp;ncbi_uid=6104980&amp;link_uid=PMC3995118" href="/pmc/articles/PMC3995118/"><img src="/pmc/articles/instance/3995118/bin/fig-5.gif" src-large="/pmc/articles/instance/3995118/bin/fig-5.jpg" alt="&lt;b&gt;FIG. 5.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3995118/" ref="ordinalpos=3978&amp;ncbi_uid=6104980&amp;link_uid=PMC3995118" image-link="/pmc/articles/PMC3995118/figure/f5/" class="imagepopup">FIG. 5..  From: Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> Contributes to Gastric Cancer Cell Growth and Proliferation. </a></p><div class="supp"><p class="details"><div>Blockage of the Shh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> attenuated tumor growth in NOD/SCID mice xenografts model. <b>(A)</b> Tumor growth curve at 7 days after cyclopamine treatment (0.4 mg intraperitoneally). <b>(B)</b> Photograph of tumors resected from NOD/SCID mice. <b>(C)</b> Quantitative analysis of tumor weight. Data are presented as mean±SD. *<i>p</i>&lt;0.05; **<i>p</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Jianhua Wan, et al. Biores Open Access. 2014 April 1;3(2):53-59.</div><p class="links"><a class="dblinks" href="/pubmed/24804165" ref="ordinalpos=3978&amp;ncbi_uid=6104980&amp;link_uid=24804165">Citation</a><a class="dblinks" href="/pmc/articles/PMC3995118/" ref="ordinalpos=3978&amp;ncbi_uid=6104980&amp;link_uid=PMC3995118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3979.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3922208/figure/F1/" ref="ordinalpos=3979&amp;ncbi_uid=5875266&amp;link_uid=PMC3922208" href="/pmc/articles/PMC3922208/"><img src="/pmc/articles/instance/3922208/bin/nihms-434133-f0001.gif" src-large="/pmc/articles/instance/3922208/bin/nihms-434133-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3922208/" ref="ordinalpos=3979&amp;ncbi_uid=5875266&amp;link_uid=PMC3922208" image-link="/pmc/articles/PMC3922208/figure/F1/" class="imagepopup">Fig. 1. Upd activated long-range JAK/STAT <span class="highlight" style="background-color:">signaling</span>.  From: Drosophila glypicans Dally and Dally-like are essential regulators for JAK/STAT <span class="highlight" style="background-color:">signaling</span> and Unpaired distribution in eye development. </a></p><div class="supp"><p class="details"><div><b>A-B:</b> The JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was activated in eye discs in 2<sup>nd</sup> instar larval stage (A-A”) and early 3<sup>rd</sup> instar larval stage (B-B”), which was shown by 10XSTAT92E-GFP (green) and pSTAT staining (red). <b>C-D:</b> JAK/STAT signal can be activated in the cells distant to cells expressing <i>upd</i> in both 2<sup>nd</sup> instar larval stage (C-C”) and early 3<sup>rd</sup> instar larval stage (D-D”). <i>Upd-lacZ</i> expression at the posterior center (red in C’ and D’) marks the cells secreting Upd. <b>E-F:</b> Upd was distributed far from secreting cells in both 2<sup>nd</sup> instar larval stage (E-E”) and 3<sup>rd</sup> instar larval stage (F-F”). Extracellular Upd staining (green in E’ and F’) can be detected in the entire eye disc. In all images, the anterior compartment faces left, and the dorsal compartment faces up.</div></p></div><div class="aux"><div class="resc">Yan Zhang, et al. Dev Biol. ;375(1):23-32.</div><p class="links"><a class="dblinks" href="/pubmed/23313126" ref="ordinalpos=3979&amp;ncbi_uid=5875266&amp;link_uid=23313126">Citation</a><a class="dblinks" href="/pmc/articles/PMC3922208/" ref="ordinalpos=3979&amp;ncbi_uid=5875266&amp;link_uid=PMC3922208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3980.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3141090/figure/F6/" ref="ordinalpos=3980&amp;ncbi_uid=3091946&amp;link_uid=PMC3141090" href="/pmc/articles/PMC3141090/"><img src="/pmc/articles/instance/3141090/bin/nihms300851f6.gif" src-large="/pmc/articles/instance/3141090/bin/nihms300851f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3141090/" ref="ordinalpos=3980&amp;ncbi_uid=3091946&amp;link_uid=PMC3141090" image-link="/pmc/articles/PMC3141090/figure/F6/" class="imagepopup">Figure 6. Loss of either sds22 or PP1 leads to increased JNK <span class="highlight" style="background-color:">signaling</span> and MMP1 expression.  From: Sds22/PP1 Links Epithelial Integrity and Tumor Suppression via Regulation of Myosin II and JNK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A–C) <i>ptc-GAL4</i>-expressing cells are visualized by GFP (green) and <i>puc-lacZ</i> reporter expression is revealed by anti-βGal (red). Transcription of a <i>puc-lacZ</i> reporter gene is normally low or absent in the wing disc (A). Increased <i>puc-lacZ</i> expression in <i>sds22</i> or <i>PP1</i><b><i>β</i></b> mutant migrating cells (B″,C″, green arrowheads) suggests that loss of either <i>sds22</i> or <i>PP1</i><b><i>β</i></b> activates <i>puc</i> transcription in response to JNK <span class="highlight" style="background-color:">pathway</span> activation. (D–F) Third instar eye-antennal discs are labeled with MMP1, another downstream target of JNK <span class="highlight" style="background-color:">signaling</span>. Compared to the control disc (D), <i>sds22</i> (E) or <i>PP1</i><b><i>α</i></b> (F) mutant discs show strong activation of MMP1 expression.</div></p></div><div class="aux"><div class="resc">Yuwei Jiang, et al. Oncogene. ;30(29):3248-3260.</div><p class="links"><a class="dblinks" href="/pubmed/21399659" ref="ordinalpos=3980&amp;ncbi_uid=3091946&amp;link_uid=21399659">Citation</a><a class="dblinks" href="/pmc/articles/PMC3141090/" ref="ordinalpos=3980&amp;ncbi_uid=3091946&amp;link_uid=PMC3141090">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3981.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4069097/figure/pone-0100787-g006/" ref="ordinalpos=3981&amp;ncbi_uid=6300723&amp;link_uid=PMC4069097" href="/pmc/articles/PMC4069097/"><img src="/pmc/articles/instance/4069097/bin/pone.0100787.g006.gif" src-large="/pmc/articles/instance/4069097/bin/pone.0100787.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4069097/" ref="ordinalpos=3981&amp;ncbi_uid=6300723&amp;link_uid=PMC4069097" image-link="/pmc/articles/PMC4069097/figure/pone-0100787-g006/" class="imagepopup">Figure 6. Putative mechanism underlying inhibition of NF-κB by pORF3..  From: Open Reading Frame 3 of Genotype 1 Hepatitis E Virus Inhibits Nuclear Factor-?appa B <span class="highlight" style="background-color:">Signaling</span> Induced by Tumor Necrosis Factor-? in Human A549 Lung Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The pORF3 activates transiently NF-κB and up-regulates A20 via the ATF6 <span class="highlight" style="background-color:">pathway</span> in the early phase, and causes degradation or inactivation of RIP1 in the late phase, leading to the blockade of the TNF-α-induced NF-κB <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jian Xu, et al. PLoS One. 2014;9(6):e100787.</div><p class="links"><a class="dblinks" href="/pubmed/24959724" ref="ordinalpos=3981&amp;ncbi_uid=6300723&amp;link_uid=24959724">Citation</a><a class="dblinks" href="/pmc/articles/PMC4069097/" ref="ordinalpos=3981&amp;ncbi_uid=6300723&amp;link_uid=PMC4069097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3982.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3014155/figure/f1/" ref="ordinalpos=3982&amp;ncbi_uid=3062103&amp;link_uid=PMC3014155" href="/pmc/articles/PMC3014155/"><img src="/pmc/articles/instance/3014155/bin/zjv9990940750001.gif" src-large="/pmc/articles/instance/3014155/bin/zjv9990940750001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3014155/" ref="ordinalpos=3982&amp;ncbi_uid=3062103&amp;link_uid=PMC3014155" image-link="/pmc/articles/PMC3014155/figure/f1/" class="imagepopup">FIG. 1. From: Dominant Inhibition of Akt/Protein Kinase B <span class="highlight" style="background-color:">Signaling</span> by the Matrix Protein of a Negative-Strand RNA Virus  . </a></p><div class="supp"><p class="details"><div>Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> changes following infection with VSV. BHK cells were either mock infected or infected with VSV at an MOI of 10. Cell lysates were collected at various time points and assayed by immunoblotting with antibodies specific to Akt, p-Akt (Thr308), p-Akt (Ser473), mTOR, p-mTOR (Ser2481), GSK3β, p-GSK3β (Ser9), VSV matrix protein (M), and β-actin.</div></p></div><div class="aux"><div class="resc">Ewan F. Dunn, et al. J Virol. 2011 January;85(1):422-431.</div><p class="links"><a class="dblinks" href="/pubmed/20980511" ref="ordinalpos=3982&amp;ncbi_uid=3062103&amp;link_uid=20980511">Citation</a><a class="dblinks" href="/pmc/articles/PMC3014155/" ref="ordinalpos=3982&amp;ncbi_uid=3062103&amp;link_uid=PMC3014155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3983.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3685590/figure/F9/" ref="ordinalpos=3983&amp;ncbi_uid=5004192&amp;link_uid=PMC3685590" href="/pmc/articles/PMC3685590/"><img src="/pmc/articles/instance/3685590/bin/nihms458790f9.gif" src-large="/pmc/articles/instance/3685590/bin/nihms458790f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3685590/" ref="ordinalpos=3983&amp;ncbi_uid=5004192&amp;link_uid=PMC3685590" image-link="/pmc/articles/PMC3685590/figure/F9/" class="imagepopup">Figure 9. Proposed C. neoformans Ras1 <span class="highlight" style="background-color:">pathway</span> for growth and thermotolerance.  From: Two CDC42 paralogs modulate C. neoformans thermotolerance and morphogenesis under host physiological conditions. </a></p><div class="supp"><p class="details"><div>Ras1 regulates mating, morphogenesis, and thermotolerance via a branched <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In this work, we have investigated the contribution of Cdc42 paralogs to morphogenesis and thermotolerance. Based on data presented here, as well as previously published work, we propose a <span class="highlight" style="background-color:">pathway</span> in which Ras1 signals through multiple Rho-GTPases: Cdc42, Cdc420, Rac1, and Rac2; with Cdc42 and Rac1 playing the major roles, and Cdc420, and possibly Rac2, playing minor roles. Rac1 primarily is required for hyphal morphogenesis, and may act through Ste20 and Pak1 to organize the actin cytoskeleton during cytokinesis. Cdc42, and to a lesser extent Cdc420, organize the septins during cytokinesis, and play a role in actin organization during growth under stress conditions. We additionally propose the existence of multiple as yet uncharacterized GEFs, which may provide specificity to Ras1 Rho-GTPase-mediated <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Elizabeth R. Ballou, et al. Mol Microbiol. ;75(3):763-780.</div><p class="links"><a class="dblinks" href="/pubmed/20025659" ref="ordinalpos=3983&amp;ncbi_uid=5004192&amp;link_uid=20025659">Citation</a><a class="dblinks" href="/pmc/articles/PMC3685590/" ref="ordinalpos=3983&amp;ncbi_uid=5004192&amp;link_uid=PMC3685590">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3984.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124252/figure/F6/" ref="ordinalpos=3984&amp;ncbi_uid=3042140&amp;link_uid=PMC3124252" href="/pmc/articles/PMC3124252/"><img src="/pmc/articles/instance/3124252/bin/nihms302160f6.gif" src-large="/pmc/articles/instance/3124252/bin/nihms302160f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124252/" ref="ordinalpos=3984&amp;ncbi_uid=3042140&amp;link_uid=PMC3124252" image-link="/pmc/articles/PMC3124252/figure/F6/" class="imagepopup">Fig. 6. The TGF-β Dauer <span class="highlight" style="background-color:">pathway</span> regulates DAF-16 localization and sod-3 transcription.  From: The C. elegans TGF-? Dauer <span class="highlight" style="background-color:">Pathway</span> Regulates Longevity via Insulin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>(A) 100× and (B) 400× images of DAF-16::GFP animals. DAF-16::GFP remains diffuse in a wild-type background, but is excluded from nuclei in many <i>daf-3</i> animals, and is partially nuclearly localized in many <i>daf-7</i> mutants. (C) Ratios of animals with diffuse (black), cytoplasmic (red), and nuclearly-localized (blue) DAF-16::GFP in wild type and <i>daf-7(e1372)</i> and <i>daf-3(mgDf90)</i> mutant backgrounds. (D) <i>Psod-3::gfp</i>, <i>Psod-3::gfp</i>;<i>daf-7(e1372), Psod-3::gfp;daf-3(mgDf90)</i>, and <i>daf-2(e1370);Psod-3::gfp</i> animal treated with <i>daf-3</i> RNAi. (E) Model of TGF-β Dauer and IIS <span class="highlight" style="background-color:">pathway</span> interactions regulating longevity.</div></p></div><div class="aux"><div class="resc">Wendy M. Shaw, et al. Curr Biol. ;17(19):1635-1645.</div><p class="links"><a class="dblinks" href="/pubmed/17900898" ref="ordinalpos=3984&amp;ncbi_uid=3042140&amp;link_uid=17900898">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124252/" ref="ordinalpos=3984&amp;ncbi_uid=3042140&amp;link_uid=PMC3124252">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3985.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3190986/figure/F2/" ref="ordinalpos=3985&amp;ncbi_uid=3500100&amp;link_uid=PMC3190986" href="/pmc/articles/PMC3190986/"><img src="/pmc/articles/instance/3190986/bin/zbc0371172470002.gif" src-large="/pmc/articles/instance/3190986/bin/zbc0371172470002.jpg" alt="FIGURE 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3190986/" ref="ordinalpos=3985&amp;ncbi_uid=3500100&amp;link_uid=PMC3190986" image-link="/pmc/articles/PMC3190986/figure/F2/" class="imagepopup">FIGURE 2. From: Dentin Matrix Protein 1 (DMP1) Signals via Cell Surface Integrin. </a></p><div class="supp"><p class="details"><div><b>DMP1 activates <span class="highlight" style="background-color:">signaling</span> through the MAPK <span class="highlight" style="background-color:">pathway</span> in hMSC cells.</b> hMSC cells were first treated with DMP1, then analyzed by Western blot for the activation of known components of the MAPK <span class="highlight" style="background-color:">signaling</span> cascade. <i>a</i>, sharp increases in phosphorylated JNK, ERK, and MEK were detected in response to rDMP1F treatment. <i>b</i>, the activation of ERK by rDMP1 <span class="highlight" style="background-color:">signaling</span> can be effectively blocked with the ERK-specific inhibitor, U0126.</div></p></div><div class="aux"><div class="resc">Hong Wu, et al. J Biol Chem. 2011 August 26;286(34):29462-29469.</div><p class="links"><a class="dblinks" href="/pubmed/21642437" ref="ordinalpos=3985&amp;ncbi_uid=3500100&amp;link_uid=21642437">Citation</a><a class="dblinks" href="/pmc/articles/PMC3190986/" ref="ordinalpos=3985&amp;ncbi_uid=3500100&amp;link_uid=PMC3190986">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3986.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3302843/figure/pone-0033250-g010/" ref="ordinalpos=3986&amp;ncbi_uid=3860528&amp;link_uid=PMC3302843" href="/pmc/articles/PMC3302843/"><img src="/pmc/articles/instance/3302843/bin/pone.0033250.g010.gif" src-large="/pmc/articles/instance/3302843/bin/pone.0033250.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3302843/" ref="ordinalpos=3986&amp;ncbi_uid=3860528&amp;link_uid=PMC3302843" image-link="/pmc/articles/PMC3302843/figure/pone-0033250-g010/" class="imagepopup">Figure 10. Schematic illustration of oxygen→ARSB→C4S→galectin-3→AP-1→HIF-1α <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Hypoxia Reduces Arylsulfatase B Activity and Silencing Arylsulfatase B Replicates and Mediates the Effects of Hypoxia. </a></p><div class="supp"><p class="details"><div>Normoxic and hypoxic conditions lead to differences in <span class="highlight" style="background-color:">signaling</span>, attributable to reduced ARSB activity leading to increase in chondroitin-4-sulfation, reduced galectin-3 binding to the more highly sulfated C4S, increased nuclear translocalization of galectin-3, and increased transcriptional events in association with activation of AP-1, including increased expression of HIF-1α. [ARSB = arylsulfatase B; FGE = formylglycine (oxoalanine) generating enzyme; C4S = chondroitin-4-sulfate].</div></p></div><div class="aux"><div class="resc">Sumit Bhattacharyya, et al. PLoS One. 2012;7(3):e33250.</div><p class="links"><a class="dblinks" href="/pubmed/22428001" ref="ordinalpos=3986&amp;ncbi_uid=3860528&amp;link_uid=22428001">Citation</a><a class="dblinks" href="/pmc/articles/PMC3302843/" ref="ordinalpos=3986&amp;ncbi_uid=3860528&amp;link_uid=PMC3302843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3987.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888582/figure/pone-0011240-g006/" ref="ordinalpos=3987&amp;ncbi_uid=2334495&amp;link_uid=PMC2888582" href="/pmc/articles/PMC2888582/"><img src="/pmc/articles/instance/2888582/bin/pone.0011240.g006.gif" src-large="/pmc/articles/instance/2888582/bin/pone.0011240.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888582/" ref="ordinalpos=3987&amp;ncbi_uid=2334495&amp;link_uid=PMC2888582" image-link="/pmc/articles/PMC2888582/figure/pone-0011240-g006/" class="imagepopup">Figure 6. Cell-to-cell <span class="highlight" style="background-color:">signaling</span> and interaction network..  From: Proteomics-Based Systems Biology Modeling of Bovine Germinal Vesicle Stage Oocyte and Cumulus Cell Interaction. </a></p><div class="supp"><p class="details"><div>Network generated with proteins involved in cell-to-cell <span class="highlight" style="background-color:">signaling</span> biological process using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) as described in materials and methods. Each node represents a protein; proteins in shaded nodes were found in either GV oocyte or cumulus cell or both (see Table 3). Proteins in red and green nodes were higher and lower, respectively, in cumulus cells compared with GV oocytes.</div></p></div><div class="aux"><div class="resc">Divyaswetha Peddinti, et al. PLoS One. 2010;5(6):e11240.</div><p class="links"><a class="dblinks" href="/pubmed/20574525" ref="ordinalpos=3987&amp;ncbi_uid=2334495&amp;link_uid=20574525">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888582/" ref="ordinalpos=3987&amp;ncbi_uid=2334495&amp;link_uid=PMC2888582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3988.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4283519/figure/F1/" ref="ordinalpos=3988&amp;ncbi_uid=7078999&amp;link_uid=PMC4283519" href="/pmc/articles/PMC4283519/"><img src="/pmc/articles/instance/4283519/bin/fphys-05-00501-g0001.gif" src-large="/pmc/articles/instance/4283519/bin/fphys-05-00501-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4283519/" ref="ordinalpos=3988&amp;ncbi_uid=7078999&amp;link_uid=PMC4283519" image-link="/pmc/articles/PMC4283519/figure/F1/" class="imagepopup">Figure 1.  From: Platelet hemostasis in patients with metabolic syndrome and type 2 diabetes mellitus: cGMP- and NO-dependent mechanisms in the insulin-mediated platelet aggregation. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> interaction and effects of modulators:</b> cGMP and cAMP-dependent intracellular <span class="highlight" style="background-color:">signaling</span> systems in the insulin-mediated regulation of platelet aggregation in patients with metabolic disorders.</div></p></div><div class="aux"><div class="resc">Tatiana E. Suslova, et al. Front Physiol. 2014;5:501.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3988&amp;ncbi_uid=7078999&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4283519/" ref="ordinalpos=3988&amp;ncbi_uid=7078999&amp;link_uid=PMC4283519">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3989.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC120796/figure/f4/" ref="ordinalpos=3989&amp;ncbi_uid=333331&amp;link_uid=PMC120796" href="/pmc/articles/PMC120796/"><img src="/pmc/articles/instance/120796/bin/mr0320019004.gif" src-large="/pmc/articles/instance/120796/bin/mr0320019004.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC120796/" ref="ordinalpos=3989&amp;ncbi_uid=333331&amp;link_uid=PMC120796" image-link="/pmc/articles/PMC120796/figure/f4/" class="imagepopup">FIG. 4. From: Regulation of Gene Expression by Ambient pH in Filamentous Fungi and Yeasts. </a></p><div class="supp"><p class="details"><div>Current two-step model of PacC proteolytic activation. Details are described in the text. PacC<sup>72</sup>, PacC<sup>53</sup>, and PacC<sup>27</sup> refer to the 72-kDa full-length, the 53-kDa intermediate, and the 27-kDa processed PacC forms, respectively. The <span class="highlight" style="background-color:">signaling</span> cleavage step requires alkaline pH <span class="highlight" style="background-color:">signaling</span> through the <i>pal</i> <span class="highlight" style="background-color:">pathway</span> and leads to the intermediate, which is committed to the ambient pH-independent processing cleavage step. The processing protease has not yet been identified, but PalB would appear to be the <span class="highlight" style="background-color:">signaling</span> protease recognizing the <span class="highlight" style="background-color:">signaling</span> protease box (shown as an open box). Commitment to processing results from removal of interacting region C, which is also removed by <i>pacC<sup>c</sup></i>-truncating mutations. That the <span class="highlight" style="background-color:">signaling</span> cleavage occurs in the cytosol is supported by the nuclear exclusion of its PacC<sup>72</sup> substrate and the preferential nuclear localization of its PacC<sup>53</sup> product (82). The processing cleavage might be in the cytosol and/or in the nucleus, as indicated by the dotted lines connecting PacC<sup>53</sup> and PacC<sup>27</sup>. <i>pacC</i> is itself an alkali-expressed gene (122).</div></p></div><div class="aux"><div class="resc">Miguel A. Peñalva, et al. Microbiol Mol Biol Rev. 2002 September;66(3):426-446.</div><p class="links"><a class="dblinks" href="/pubmed/12208998" ref="ordinalpos=3989&amp;ncbi_uid=333331&amp;link_uid=12208998">Citation</a><a class="dblinks" href="/pmc/articles/PMC120796/" ref="ordinalpos=3989&amp;ncbi_uid=333331&amp;link_uid=PMC120796">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3990.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2528845/figure/F2/" ref="ordinalpos=3990&amp;ncbi_uid=1571039&amp;link_uid=PMC2528845" href="/pmc/articles/PMC2528845/"><img src="/pmc/articles/instance/2528845/bin/zef0020826790002.gif" src-large="/pmc/articles/instance/2528845/bin/zef0020826790002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2528845/" ref="ordinalpos=3990&amp;ncbi_uid=1571039&amp;link_uid=PMC2528845" image-link="/pmc/articles/PMC2528845/figure/F2/" class="imagepopup">Figure 2.  From: Cellular <span class="highlight" style="background-color:">Signaling</span> by Fibroblast Growth Factors (FGFs) and Their Receptors (FGFRs) in Male Reproduction. </a></p><div class="supp"><p class="details"><div>Intracellular <span class="highlight" style="background-color:">signaling</span> activated through FGFRs. Formation of the FGF-Syndecan/heparin-FGFR complex leads to receptor autophosphorylation and activation of intracellular <span class="highlight" style="background-color:">signaling</span> cascades of the Ras/MAPK <span class="highlight" style="background-color:">pathway</span>, the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (<i>left</i>), and the PLCγ/Ca<sup>2+</sup> /PKC <span class="highlight" style="background-color:">pathway</span> (<i>right</i>) (4,57,90,91). Syndecans facilitate the binding of FGFs to FGFRs, forming a ternary structure. EphA receptors induce tyrosine phosphorylation of FGFRs and FRS2 by forming a complex with the FGFR. The Ras/MAPK <span class="highlight" style="background-color:">signaling</span> cascade is activated by the binding of Grb2 to phosphorylated FRS2. The subsequent formation of the Grb2/SOS complex results in the activation of Ras. There are three routes by which FGFRs are able to activate the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>: Gab1 can bind to FRS2 indirectly via Grb2, resulting in the tyrosine phosphorylation and activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> (2,96); PI3K can bind directly to a phosphorylated tyrosine residue of the FGFR (88); and lastly, activated Ras can induce membrane localization and activation of the PI3K catalytic subunit (57). The negative feedback signals that are mediated by or imposed on FRS2 are marked by the <i>dotted line</i> (94). The PLCγ/Ca<sup>2+</sup> <span class="highlight" style="background-color:">pathway</span> is activated when autophosphorylation on a tyrosine residue in the carboxy terminal tail of the FGFR creates a specific binding site for the SH2 domain of PLCγ (91). The last activated components of the above-mentioned signal transduction cascades translocate into the nucleus and phosphorylate specific transcription factors of the Ets family, which in turn activate expression of FGF target genes. DAG, Diacylglycerol; IP3, inositol-1,4,5-triphosphate; SOS, son of sevenless.</div></p></div><div class="aux"><div class="resc">Leanne M. Cotton, et al. Endocr Rev. 2008 April;29(2):193-216.</div><p class="links"><a class="dblinks" href="/pubmed/18216218" ref="ordinalpos=3990&amp;ncbi_uid=1571039&amp;link_uid=18216218">Citation</a><a class="dblinks" href="/pmc/articles/PMC2528845/" ref="ordinalpos=3990&amp;ncbi_uid=1571039&amp;link_uid=PMC2528845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3991.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3003688/figure/pone-0015600-g006/" ref="ordinalpos=3991&amp;ncbi_uid=2642877&amp;link_uid=PMC3003688" href="/pmc/articles/PMC3003688/"><img src="/pmc/articles/instance/3003688/bin/pone.0015600.g006.gif" src-large="/pmc/articles/instance/3003688/bin/pone.0015600.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3003688/" ref="ordinalpos=3991&amp;ncbi_uid=2642877&amp;link_uid=PMC3003688" image-link="/pmc/articles/PMC3003688/figure/pone-0015600-g006/" class="imagepopup">Figure 6. NGF-induced phosphorylation of Akt is mediated through the PI3-K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: ?-Amyloid 1-42 Oligomers Impair Function of Human Embryonic Stem Cell-Derived Forebrain Cholinergic Neurons. </a></p><div class="supp"><p class="details"><div>To measure the levels of phosphorylated Akt (p-Akt) mediated through stimulation of the PI3-K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, hES cells were treated with the PI3-K inhibitor LY294002 (10 µM) 5 h prior to stimulation with NGF (50 ng/ml) or AβO<sub>1–40</sub> (5 µM). Cell lysates were analyzed by Western blotting, using phospho-specific antibodies, and β-actin served as a loading control. (A) Following stimulation with NGF or AβO<sub>1–40</sub>, p-Akt levels were 1.74±0.74 and 1.66±1.54 relative to controls, respectively. Induction of p-Akt by NGF was inhibited by LY294002. However, the AβO<sub>1–40</sub>-dependent phosphorylation of Akt was not inhibited by LY294002. Values are expressed as mean ± SD (relative to controls), *<i>p</i>&lt;0.05 (unpaired Student's <i>t</i>-test). (B) Representative immunoblots of p-Akt, total Akt and β-actin.</div></p></div><div class="aux"><div class="resc">Linn Wicklund, et al. PLoS One. 2010;5(12):e15600.</div><p class="links"><a class="dblinks" href="/pubmed/21179413" ref="ordinalpos=3991&amp;ncbi_uid=2642877&amp;link_uid=21179413">Citation</a><a class="dblinks" href="/pmc/articles/PMC3003688/" ref="ordinalpos=3991&amp;ncbi_uid=2642877&amp;link_uid=PMC3003688">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3992.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2807288/figure/F7/" ref="ordinalpos=3992&amp;ncbi_uid=2721040&amp;link_uid=PMC2807288" href="/pmc/articles/PMC2807288/"><img src="/pmc/articles/instance/2807288/bin/zbc0071003560007.gif" src-large="/pmc/articles/instance/2807288/bin/zbc0071003560007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2807288/" ref="ordinalpos=3992&amp;ncbi_uid=2721040&amp;link_uid=PMC2807288" image-link="/pmc/articles/PMC2807288/figure/F7/" class="imagepopup">FIGURE 7. From: BCAR3 Regulates Src/p130Cas Association, Src Kinase Activity, and Breast Cancer Adhesion <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Proposed mechanism of BCAR3-mediated enhancement of c-Src activation, cell spreading, and adhesion.</b> <i>Step 1</i>, cytosolically localized Cas is recruited to membrane-proximal regions by BCAR3 through an indirect or direct interaction with BCAR3. Once at the membrane (<i>step 2</i>), Cas interacts with and activates c-Src. c-Src activation results in the phosphorylation of Cas, which leads to the recruitment of substrate domain binding partners such as Crk (<i>step 3</i>). <i>Step 4</i>, Cas/Crk interactions activate downstream <span class="highlight" style="background-color:">signaling</span> pathways important for cell spreading, migration, and invasion. BCAR3-dependent and BCAR3-independent c-Src <span class="highlight" style="background-color:">signaling</span> can also activate other <span class="highlight" style="background-color:">signaling</span> pathways important for cell proliferation and survival in the presence of antiestrogens. <i>Step 5</i>, BCAR3 itself activates an unidentified c-Src-independent <span class="highlight" style="background-color:">pathway</span> important for cell spreading.</div></p></div><div class="aux"><div class="resc">Natasha R. Schuh, et al. J Biol Chem. 2010 January 22;285(4):2309-2317.</div><p class="links"><a class="dblinks" href="/pubmed/19940159" ref="ordinalpos=3992&amp;ncbi_uid=2721040&amp;link_uid=19940159">Citation</a><a class="dblinks" href="/pmc/articles/PMC2807288/" ref="ordinalpos=3992&amp;ncbi_uid=2721040&amp;link_uid=PMC2807288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3993.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478260/figure/F4/" ref="ordinalpos=3993&amp;ncbi_uid=5051300&amp;link_uid=PMC3478260" href="/pmc/articles/PMC3478260/"><img src="/pmc/articles/instance/3478260/bin/cam-6-374-g4.gif" src-large="/pmc/articles/instance/3478260/bin/cam-6-374-g4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478260/" ref="ordinalpos=3993&amp;ncbi_uid=5051300&amp;link_uid=PMC3478260" image-link="/pmc/articles/PMC3478260/figure/F4/" class="imagepopup">Figure. From: L1CAM. </a></p><div class="supp"><p class="details"><div><b>Figure 4.</b> L1CAM can trigger different <span class="highlight" style="background-color:">signaling</span> pathways. Schematic illustration of distinct <span class="highlight" style="background-color:">signaling</span> pathways triggered by L1CAM due to the interacting with different binding partners. (A) L1CAM homophilic interactions promote static cell-cell binding and trigger predominantly the MAPK <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">signaling</span> can be modulated by interactions with growth factor receptors (GFR). (B) The binding of L1CAM to integrins triggers NFκB activation and renders cells more motile and invasive. Cleavage of L1CAM from the membrane generates soluble L1CAM. The intracellular fragment L1-ICD can translocate into the nucleus and activate gene transcription. (C) Upregulation of L1CAM by TGF-β allows L1CAM-integrin downstream <span class="highlight" style="background-color:">signaling</span> via FAK-Src. This <span class="highlight" style="background-color:">pathway</span> induces production and release of IL-1β that in turn activates NFκB via binding to the IL-1 receptor (IL-1RI).</div></p></div><div class="aux"><div class="resc">Helena Kiefel, et al. Cell Adh Migr. 2012 July 1;6(4):374-384.</div><p class="links"><a class="dblinks" href="/pubmed/22796939" ref="ordinalpos=3993&amp;ncbi_uid=5051300&amp;link_uid=22796939">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478260/" ref="ordinalpos=3993&amp;ncbi_uid=5051300&amp;link_uid=PMC3478260">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3994.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713330/figure/fig1/" ref="ordinalpos=3994&amp;ncbi_uid=1937110&amp;link_uid=PMC2713330" href="/pmc/articles/PMC2713330/"><img src="/pmc/articles/instance/2713330/bin/PROCATS25403f1.gif" src-large="/pmc/articles/instance/2713330/bin/PROCATS25403f1.jpg" alt="&lt;b&gt;Figure 1.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713330/" ref="ordinalpos=3994&amp;ncbi_uid=1937110&amp;link_uid=PMC2713330" image-link="/pmc/articles/PMC2713330/figure/fig1/" class="imagepopup">Figure 1..  From: Innate Immunity in the Lungs. </a></p><div class="supp"><p class="details"><div>Simplified diagram of some Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> pathways. Flagellin binds to TLR5; lipoproteins bind to cooperative clusters of TLR2, -1, and -6; LPS binds to the complex formed by CD14, MD-2, and TLR4; and bacterial DNA binds to TLR9, which is located in the membrane of phagolysosomes. MyD88 has a key role in the proximal <span class="highlight" style="background-color:">signaling</span> by these TLRs. Two LPS <span class="highlight" style="background-color:">signaling</span> pathways diverge from TLR4, LPS-1 and LPS-2. The LPS-1 <span class="highlight" style="background-color:">pathway</span> is mediated by MyD88 and through IRAK-4 and IRAK-1 leads to nuclear factor (NF)-kB activation and cytokine production. The LPS-2 <span class="highlight" style="background-color:">pathway</span> is mediated by TRAM and TRIF and leads to the production of Type I interferons and nitric oxide, and has a role in the adjuvant effect of LPS in adaptive immune responses. A more complete <span class="highlight" style="background-color:">signaling</span> map can be found in Reference 1.</div></p></div><div class="aux"><div class="resc">Thomas R. Martin, et al. Proc Am Thorac Soc. 2005 December;2(5):403-411.</div><p class="links"><a class="dblinks" href="/pubmed/16322590" ref="ordinalpos=3994&amp;ncbi_uid=1937110&amp;link_uid=16322590">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713330/" ref="ordinalpos=3994&amp;ncbi_uid=1937110&amp;link_uid=PMC2713330">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3995.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g003/" ref="ordinalpos=3995&amp;ncbi_uid=6232468&amp;link_uid=PMC4041745" href="/pmc/articles/PMC4041745/"><img src="/pmc/articles/instance/4041745/bin/pone.0098229.g003.gif" src-large="/pmc/articles/instance/4041745/bin/pone.0098229.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4041745/" ref="ordinalpos=3995&amp;ncbi_uid=6232468&amp;link_uid=PMC4041745" image-link="/pmc/articles/PMC4041745/figure/pone-0098229-g003/" class="imagepopup">Figure 3. Top 25 GeneGO pathways enriched simultaneously by the 0.01 gene list and general metastasis signature; red numbers = significant at FDR of 0.05; green box = <span class="highlight" style="background-color:">pathway</span> significantly enriched by both gene lists; <span class="highlight" style="background-color:">Pathway</span> “Airway smooth muscle contraction in asthma” was placed at rank 7, <span class="highlight" style="background-color:">pathway</span> “Cytoskeleton remodeling_Cytoskeleton remodeling” was placed at rank 9..  From: Systematic <span class="highlight" style="background-color:">Pathway</span> Enrichment Analysis of a Genome-Wide Association Study on Breast Cancer Survival Reveals an Influence of Genes Involved in Cell Adhesion and Calcium <span class="highlight" style="background-color:">Signaling</span> on the Patients' Clinical Outcome. </a></p><div class="aux"><div class="resc">Andrea Woltmann, et al. PLoS One. 2014;9(6):e98229.</div><p class="links"><a class="dblinks" href="/pubmed/24886783" ref="ordinalpos=3995&amp;ncbi_uid=6232468&amp;link_uid=24886783">Citation</a><a class="dblinks" href="/pmc/articles/PMC4041745/" ref="ordinalpos=3995&amp;ncbi_uid=6232468&amp;link_uid=PMC4041745">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3996.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4366815/figure/f7/" ref="ordinalpos=3996&amp;ncbi_uid=7344370&amp;link_uid=PMC4366815" href="/pmc/articles/PMC4366815/"><img src="/pmc/articles/instance/4366815/bin/srep09369-f7.gif" src-large="/pmc/articles/instance/4366815/bin/srep09369-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4366815/" ref="ordinalpos=3996&amp;ncbi_uid=7344370&amp;link_uid=PMC4366815" image-link="/pmc/articles/PMC4366815/figure/f7/" class="imagepopup">Figure 7. Mechanical illustration of the positively-charged surface with tertiary amines upregulating osteogenic differentiation of BMSCs via the iNOS <span class="highlight" style="background-color:">pathway</span> signaled by the surface charge..  From: Upregulation of BMSCs Osteogenesis by Positively-Charged Tertiary Amines on Polymeric Implants via Charge/iNOS <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="aux"><div class="resc">Wei Zhang, et al. Sci Rep. 2015;5:9369.</div><p class="links"><a class="dblinks" href="/pubmed/25791957" ref="ordinalpos=3996&amp;ncbi_uid=7344370&amp;link_uid=25791957">Citation</a><a class="dblinks" href="/pmc/articles/PMC4366815/" ref="ordinalpos=3996&amp;ncbi_uid=7344370&amp;link_uid=PMC4366815">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3997.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3747267/figure/pone-0071349-g008/" ref="ordinalpos=3997&amp;ncbi_uid=5219877&amp;link_uid=PMC3747267" href="/pmc/articles/PMC3747267/"><img src="/pmc/articles/instance/3747267/bin/pone.0071349.g008.gif" src-large="/pmc/articles/instance/3747267/bin/pone.0071349.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3747267/" ref="ordinalpos=3997&amp;ncbi_uid=5219877&amp;link_uid=PMC3747267" image-link="/pmc/articles/PMC3747267/figure/pone-0071349-g008/" class="imagepopup">Figure 8. TBCK has no significant effect on MAPK and PDK1/AKT <span class="highlight" style="background-color:">pathway</span>..  From: TBCK Influences Cell Proliferation, Cell Size and mTOR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>–<b>B</b>) HEK293 cells infected with the indicated lentivirus were subjected to Western blotting with the antibodies as shown.</div></p></div><div class="aux"><div class="resc">Yueli Liu, et al. PLoS One. 2013;8(8):e71349.</div><p class="links"><a class="dblinks" href="/pubmed/23977024" ref="ordinalpos=3997&amp;ncbi_uid=5219877&amp;link_uid=23977024">Citation</a><a class="dblinks" href="/pmc/articles/PMC3747267/" ref="ordinalpos=3997&amp;ncbi_uid=5219877&amp;link_uid=PMC3747267">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3998.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3416717/figure/F1/" ref="ordinalpos=3998&amp;ncbi_uid=4171063&amp;link_uid=PMC3416717" href="/pmc/articles/PMC3416717/"><img src="/pmc/articles/instance/3416717/bin/1742-2094-9-119-1.gif" src-large="/pmc/articles/instance/3416717/bin/1742-2094-9-119-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3416717/" ref="ordinalpos=3998&amp;ncbi_uid=4171063&amp;link_uid=PMC3416717" image-link="/pmc/articles/PMC3416717/figure/F1/" class="imagepopup">Figure 1.  From: Immunodominant fragments of myelin basic protein initiate T cell-dependent pain. </a></p><div class="supp"><p class="details"><div><b>T-helper cell <span class="highlight" style="background-color:">signaling</span> is induced after CCI.</b> Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis of the canonical pathways (<b>A</b>) and <span class="highlight" style="background-color:">signaling</span> cascades (<b>B</b>) using the proteins unique to sciatic nerve proteome at day 7 after CCI or sham operation. The bars represent -log (<i>P</i> value) for a system or a <span class="highlight" style="background-color:">pathway</span> to be represented with a threshold set at 1.4 (<i>P</i> &lt; 0.05) in <i>n</i> = 3/group. CCI presents as an infectious disease, with CD28 Th cell <span class="highlight" style="background-color:">signaling</span> and T cell apoptosis among unique pathways activated relative to sham-operated nerve (asterisks).</div></p></div><div class="aux"><div class="resc">Huaqing Liu, et al. J Neuroinflammation. 2012;9:119-119.</div><p class="links"><a class="dblinks" href="/pubmed/22676642" ref="ordinalpos=3998&amp;ncbi_uid=4171063&amp;link_uid=22676642">Citation</a><a class="dblinks" href="/pmc/articles/PMC3416717/" ref="ordinalpos=3998&amp;ncbi_uid=4171063&amp;link_uid=PMC3416717">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3999.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3141289/figure/F1/" ref="ordinalpos=3999&amp;ncbi_uid=3096928&amp;link_uid=PMC3141289" href="/pmc/articles/PMC3141289/"><img src="/pmc/articles/instance/3141289/bin/nihms307485f1.gif" src-large="/pmc/articles/instance/3141289/bin/nihms307485f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3141289/" ref="ordinalpos=3999&amp;ncbi_uid=3096928&amp;link_uid=PMC3141289" image-link="/pmc/articles/PMC3141289/figure/F1/" class="imagepopup">Figure 1.  From: Signal Transducer and Activator of Transcription-3, Inflammation, and Cancer. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to signal transducer and activator of transcription (STAT)-3 activation (see text for definitions of abbreviations).</div></p></div><div class="aux"><div class="resc">Bharat B. Aggarwal, et al. Ann N Y Acad Sci. ;1171:59-76.</div><p class="links"><a class="dblinks" href="/pubmed/19723038" ref="ordinalpos=3999&amp;ncbi_uid=3096928&amp;link_uid=19723038">Citation</a><a class="dblinks" href="/pmc/articles/PMC3141289/" ref="ordinalpos=3999&amp;ncbi_uid=3096928&amp;link_uid=PMC3141289">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4000.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119544/figure/F4/" ref="ordinalpos=4000&amp;ncbi_uid=3466153&amp;link_uid=PMC3119544" href="/pmc/articles/PMC3119544/"><img src="/pmc/articles/instance/3119544/bin/nihms292230f4.gif" src-large="/pmc/articles/instance/3119544/bin/nihms292230f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119544/" ref="ordinalpos=4000&amp;ncbi_uid=3466153&amp;link_uid=PMC3119544" image-link="/pmc/articles/PMC3119544/figure/F4/" class="imagepopup">Figure 4. BMP <span class="highlight" style="background-color:">signaling</span> is disrupted in Arl13bhnn embryos.  From: Disrupted dorsal neural tube BMP <span class="highlight" style="background-color:">signaling</span> in the cilia mutant Arl13bhnn stems from abnormal Shh <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>(A-D) Expression of the BMP ligand <i>Gdf7</i> is discontinuous/absent in the caudal neural tube of E10.5 <i>Arl13b<sup>hnn</sup></i> embryos. C and D are higher-magnification views of the region of interest.<br />(E-L) Expression of the BMP <span class="highlight" style="background-color:">pathway</span> target genes <i>Msx1</i> (E-H) and <i>Lhx2</i> (I-L) is discontinuous/absent in the caudal neural tube of E10.5 and E12.5 <i>Arl13b<sup>hnn</sup></i> embryos, respectively. G, H, K, and L are higher-magnification views of the region of interest. Scale bars represent approximately 300 microns.</div></p></div><div class="aux"><div class="resc">Vanessa L. Horner, et al. Dev Biol. ;355(1):43-54.</div><p class="links"><a class="dblinks" href="/pubmed/21539826" ref="ordinalpos=4000&amp;ncbi_uid=3466153&amp;link_uid=21539826">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119544/" ref="ordinalpos=4000&amp;ncbi_uid=3466153&amp;link_uid=PMC3119544">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="39" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="40" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="41" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:44:52-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB72EE543121D10000000000E37DAB;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB72EE543121D10000000000E37DAB" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>