
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="22" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FC26905430F2F10000000000D8448B" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="21" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="22" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="23" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="22" /></div>
            <div><h2 class="result_count">Results: 2101 to 2200 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>2101.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2695974/figure/F10/" ref="ordinalpos=2101&amp;ncbi_uid=1816430&amp;link_uid=PMC2695974" href="/pmc/articles/PMC2695974/"><img src="/pmc/articles/instance/2695974/bin/nihms106785f10.gif" src-large="/pmc/articles/instance/2695974/bin/nihms106785f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2695974/" ref="ordinalpos=2101&amp;ncbi_uid=1816430&amp;link_uid=PMC2695974" image-link="/pmc/articles/PMC2695974/figure/F10/" class="imagepopup">Figure 10. <span class="highlight" style="background-color:">Signaling</span> pathways involved in ET-1-induced c-jun expression.  From: Endothelin-1 regulates astrocyte proliferation and reactive gliosis via a JNK/c-Jun <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Representative scheme of the main <span class="highlight" style="background-color:">signaling</span> pathways involved in ET-1-induced/ETB-R-mediated phosphorylation of downstream transcription factors in the <i>c-jun</i> promoter. Increased expression of the immediate early genes <i>c-fos</i> and <i>c-jun,</i> which bind the AP-1 site in promoters of target genes, modulate astrocyte proliferation. Dashed lines and question marks indicate unknown links between elements of <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Ana Gadea, et al. J Neurosci. ;28(10):2394-2408.</div><p class="links"><a class="dblinks" href="/pubmed/18322086" ref="ordinalpos=2101&amp;ncbi_uid=1816430&amp;link_uid=18322086">Citation</a><a class="dblinks" href="/pmc/articles/PMC2695974/" ref="ordinalpos=2101&amp;ncbi_uid=1816430&amp;link_uid=PMC2695974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2102.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3476735/figure/F5/" ref="ordinalpos=2102&amp;ncbi_uid=4306812&amp;link_uid=PMC3476735" href="/pmc/articles/PMC3476735/"><img src="/pmc/articles/instance/3476735/bin/nihms399340f5.gif" src-large="/pmc/articles/instance/3476735/bin/nihms399340f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3476735/" ref="ordinalpos=2102&amp;ncbi_uid=4306812&amp;link_uid=PMC3476735" image-link="/pmc/articles/PMC3476735/figure/F5/" class="imagepopup">Figure 5. Flow diagram showing the molecular mechanism of metformin-mediated reduction in SCC growth.  From: Metformin, an anti-diabetic agent reduces growth of cutaneous squamous cell carcinoma by targeting mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Metformin targets multiple <span class="highlight" style="background-color:">signaling</span> pathways to reduce the growth of cutaneous SCCs in a xenograft murine model system. The major pathways blocked by metformin treatment include mTOR, NFkB, Akt and MAP kinases. Metformin activates AMPK which blocks mTOR <span class="highlight" style="background-color:">signaling</span> while inhibition in NFkB <span class="highlight" style="background-color:">pathway</span> reduces proliferation via blocking activation of genes involved in cell cycle progression and enhancing genes that augment apoptosis.</div></p></div><div class="aux"><div class="resc">Sandeep C. Chaudhary, et al. Photochem Photobiol. 2012 Sep-Oct;88(5):1149-1156.</div><p class="links"><a class="dblinks" href="/pubmed/22540890" ref="ordinalpos=2102&amp;ncbi_uid=4306812&amp;link_uid=22540890">Citation</a><a class="dblinks" href="/pmc/articles/PMC3476735/" ref="ordinalpos=2102&amp;ncbi_uid=4306812&amp;link_uid=PMC3476735">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2103.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975375/figure/F2/" ref="ordinalpos=2103&amp;ncbi_uid=2567457&amp;link_uid=PMC2975375" href="/pmc/articles/PMC2975375/"><img src="/pmc/articles/instance/2975375/bin/nihms246516f2.gif" src-large="/pmc/articles/instance/2975375/bin/nihms246516f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975375/" ref="ordinalpos=2103&amp;ncbi_uid=2567457&amp;link_uid=PMC2975375" image-link="/pmc/articles/PMC2975375/figure/F2/" class="imagepopup">Fig. 2. Recombination of protein domains results in diversification in <span class="highlight" style="background-color:">signaling</span> behaviors.  From: Rapid Diversification of Cell <span class="highlight" style="background-color:">Signaling</span> Phenotypes by Modular Domain Recombination. </a></p><div class="supp"><p class="details"><div><b>A</b>, mating <span class="highlight" style="background-color:">pathway</span> activation was measured by flow cytometry, using a GFP reporter under the control of a promoter (from the Fus1 gene) that responds to <span class="highlight" style="background-color:">pathway</span> activation, in an a-type Far1 strain. Time course measurements of GFP fluorescence were done to calculate the baseline and slope of <span class="highlight" style="background-color:">pathway</span> activation, under conditions of linear <span class="highlight" style="background-color:">pathway</span> response. Baseline and slopes were then normalized relative to wild type values and the resulting values were plotted in <span class="highlight" style="background-color:">pathway</span> morphospace. <b>B</b>, gene duplications had minor effects on mating <span class="highlight" style="background-color:">pathway</span> response dynamics, with most values clustered around wild type. <b>C</b>, domain duplications had also minor effects on mating <span class="highlight" style="background-color:">pathway</span> response dynamics; duplication of Ste50 [N] (Ste50s SAM domain), Ste5 [N] (which includes Ste5s RING domain) and Ste11 [N] (Ste11s SAM domain) are exceptions with low slopes and may act as dominant negative. <b>D</b>, domain recombination led to a diverse set of novel <span class="highlight" style="background-color:">signaling</span> behaviors. Recombination variants with dynamic behaviors most different from wild type and from the corresponding co-expressed N- and C-domain pair (Fig. S3) are shown in red. <b>E</b>, these behaviors could not be recapitulated by co-expression of the unlinked corresponding pairs of domains. Fluorescent values were measured in at least two independent experiments, each time in triplicates. Mean values +/ standard deviation (error bars) are depicted.</div></p></div><div class="aux"><div class="resc">Sergio G. Peisajovich, et al. Science. ;328(5976):368-372.</div><p class="links"><a class="dblinks" href="/pubmed/20395511" ref="ordinalpos=2103&amp;ncbi_uid=2567457&amp;link_uid=20395511">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975375/" ref="ordinalpos=2103&amp;ncbi_uid=2567457&amp;link_uid=PMC2975375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2104.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418714/figure/F2/" ref="ordinalpos=2104&amp;ncbi_uid=5178920&amp;link_uid=PMC3418714" href="/pmc/articles/PMC3418714/"><img src="/pmc/articles/instance/3418714/bin/zh89991294640002.gif" src-large="/pmc/articles/instance/3418714/bin/zh89991294640002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418714/" ref="ordinalpos=2104&amp;ncbi_uid=5178920&amp;link_uid=PMC3418714" image-link="/pmc/articles/PMC3418714/figure/F2/" class="imagepopup">Figure 2.  From: The B-cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a therapeutic target in CLL. </a></p><div class="supp"><p class="details"><div><b>Integrin <span class="highlight" style="background-color:">signaling</span> in CLL.</b> (A) Inside-out integrin <span class="highlight" style="background-color:">signaling</span>. <span class="highlight" style="background-color:">Signaling</span> through the BCR, dependent on Btk and PLC-2, activates the cytoplasmic domain of the integrin, causing a conformational change to the open, active form. This form has high affinity for ligand binding and induces cell migration and adhesion. (B) Outside-in integrin <span class="highlight" style="background-color:">signaling</span>. Ligand binding to an integrin induces signals that can lead to growth, differentiation, survival, or apoptosis.</div></p></div><div class="aux"><div class="resc">Jennifer A. Woyach, et al. Blood. 2012 August 9;120(6):1175-1184.</div><p class="links"><a class="dblinks" href="/pubmed/22715122" ref="ordinalpos=2104&amp;ncbi_uid=5178920&amp;link_uid=22715122">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418714/" ref="ordinalpos=2104&amp;ncbi_uid=5178920&amp;link_uid=PMC3418714">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2105.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418396/figure/F5/" ref="ordinalpos=2105&amp;ncbi_uid=5186948&amp;link_uid=PMC3418396" href="/pmc/articles/PMC3418396/"><img src="/pmc/articles/instance/3418396/bin/nihms378923f5.gif" src-large="/pmc/articles/instance/3418396/bin/nihms378923f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418396/" ref="ordinalpos=2105&amp;ncbi_uid=5186948&amp;link_uid=PMC3418396" image-link="/pmc/articles/PMC3418396/figure/F5/" class="imagepopup">Figure 5.  From: The Akt-SRPK-SR Axis Constitutes a Major <span class="highlight" style="background-color:">Pathway</span> in Transducing EGF <span class="highlight" style="background-color:">Signaling</span> to Regulate Alternative Splicing in the Nucleus. </a></p><div class="supp"><p class="details"><div>EGF <span class="highlight" style="background-color:">signaling</span> modulates the interaction of SRPKs with heat shock and 14-3-3 proteins. (<b>A</b>) Co-immunoprecipitation of SRPK1 and SRPK2 with Hsp70 and Hsp90 along with their co-chaperones in the presence of serum. (<b>B</b>) EGF-induced release of Hsp70 and its co-chaperone Hsp40 from SRPKs while enhanced the association of the splicing kinases with Hsp90 and its co-chaperone Aha1. (<b>C</b>) The differential interaction of SRPKs with heat shock proteins required the activation of the PI3K <span class="highlight" style="background-color:">pathway</span>. (<b>D</b>-<b>F</b>) EGF induced the interaction of SRPKs with 14-3-3. Both SRPKs showed similar association in proliferating HEK293T cells (D). Interestingly, SRPK2 seems to be more inducible than SRPK1 in association with 14-3-3 in response to EGF <span class="highlight" style="background-color:">signaling</span> (E). Increased 14-3-3 was able to progressively suppress the interaction of SRPKs with both Hsp70 and Hsp90 in EGF-treated cells (F).</div></p></div><div class="aux"><div class="resc">Zhihong Zhou, et al. Mol Cell. ;47(3):422-433.</div><p class="links"><a class="dblinks" href="/pubmed/22727668" ref="ordinalpos=2105&amp;ncbi_uid=5186948&amp;link_uid=22727668">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418396/" ref="ordinalpos=2105&amp;ncbi_uid=5186948&amp;link_uid=PMC3418396">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2106.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553363/figure/F1/" ref="ordinalpos=2106&amp;ncbi_uid=1512755&amp;link_uid=PMC2553363" href="/pmc/articles/PMC2553363/"><img src="/pmc/articles/instance/2553363/bin/nihms60357f1.gif" src-large="/pmc/articles/instance/2553363/bin/nihms60357f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553363/" ref="ordinalpos=2106&amp;ncbi_uid=1512755&amp;link_uid=PMC2553363" image-link="/pmc/articles/PMC2553363/figure/F1/" class="imagepopup">Figure 1.  From: Cellular and Molecular Mechanisms of Liver Injury. </a></p><div class="supp"><p class="details"><div>Extrinsic and intrinsic pathways of hepatocyte apoptosis. The extrinsic <span class="highlight" style="background-color:">pathway</span> is activated by death receptors. Fas or TRAIL (depicted here) bind to their cognate receptors, leading to the formation of the death-inducing <span class="highlight" style="background-color:">signaling</span> complex (DISC), with caspase 8 activation, Bid cleavage, and subsequent mitochondrial permeabilization. Bim activation can also occur downstream of death receptor <span class="highlight" style="background-color:">signaling</span>, leading to Bax activation and mitochondrial permeabilization. The TNF- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> also leads to Bid cleavage with lysosomal permeabilization, leading to release of lysosomal contents and mitochondrial permeabilization. The intrinsic <span class="highlight" style="background-color:">pathway</span> of cell death can be initiated by myriad intracellular stressors that can activate the ER stress <span class="highlight" style="background-color:">pathway</span>, lysosomal permeabilization, or JNK activation. These cascades lead to inhibition of the antiapoptotic proteins (Bcl-xL, Bcl-2) and activation of the proapoptotic proteins (Bax, Bim, Bad, Bid). Mitochondrial permeabilization occurs eventually and is required for hepatocyte apoptosis.</div></p></div><div class="aux"><div class="resc">Harmeet Malhi, et al. Gastroenterology. ;134(6):1641-1654.</div><p class="links"><a class="dblinks" href="/pubmed/18471544" ref="ordinalpos=2106&amp;ncbi_uid=1512755&amp;link_uid=18471544">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553363/" ref="ordinalpos=2106&amp;ncbi_uid=1512755&amp;link_uid=PMC2553363">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2107.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1456092/figure/fig6/" ref="ordinalpos=2107&amp;ncbi_uid=625164&amp;link_uid=PMC1456092" href="/pmc/articles/PMC1456092/"><img src="/pmc/articles/instance/1456092/bin/GEN17141643f6.gif" src-large="/pmc/articles/instance/1456092/bin/GEN17141643f6.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1456092/" ref="ordinalpos=2107&amp;ncbi_uid=625164&amp;link_uid=PMC1456092" image-link="/pmc/articles/PMC1456092/figure/fig6/" class="imagepopup">Figure 6. From: Genetic Evidence That Drosophila frizzled Controls Planar Cell Polarity and Armadillo <span class="highlight" style="background-color:">Signaling</span> by a Common Mechanism. </a></p><div class="supp"><p class="details"><div>Model of Fz activation in PCP and Arm <span class="highlight" style="background-color:">signaling</span>. In the absence of a Wnt ligand, Fz has an intrinsic ability to signal to the PCP <span class="highlight" style="background-color:">pathway</span>. This may occur via a direct interaction with Dsh. When Wg is present, the coreceptor Arr is brought into proximity to Fz, which results in the close apposition of Dsh and Axn. This combination of intracellular components switches the output of Fz from PCP to Arm <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Michael Povelones, et al. Genetics. 2005 December;171(4):1643-1654.</div><p class="links"><a class="dblinks" href="/pubmed/16085697" ref="ordinalpos=2107&amp;ncbi_uid=625164&amp;link_uid=16085697">Citation</a><a class="dblinks" href="/pmc/articles/PMC1456092/" ref="ordinalpos=2107&amp;ncbi_uid=625164&amp;link_uid=PMC1456092">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2108.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g003/" ref="ordinalpos=2108&amp;ncbi_uid=4649218&amp;link_uid=PMC3578810" href="/pmc/articles/PMC3578810/"><img src="/pmc/articles/instance/3578810/bin/pone.0057231.g003.gif" src-large="/pmc/articles/instance/3578810/bin/pone.0057231.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3578810/" ref="ordinalpos=2108&amp;ncbi_uid=4649218&amp;link_uid=PMC3578810" image-link="/pmc/articles/PMC3578810/figure/pone-0057231-g003/" class="imagepopup">Figure 3. Injection of apelin increases glycogen level and improves insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in liver tissues of TNF--treated C57BL/6J mice..  From: Apelin Ameliorates TNF-?-Induced Reduction of Glycogen Synthesis in the Hepatocytes through G Protein-Coupled Receptor APJ. </a></p><div class="supp"><p class="details"><div>An intraperitoneal injection of 20 nmol/kg apelin-13 led to increased glycogen level in liver tissues of C57BL/6J mice treated by TNF- (A). Moreover, insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was impaired in liver tissues treated with TNF-. These changes of JNK, IRS-1, AKT and GSK induced by TNF- were reversed via apelin injection (B). Data represent the means  S.E.M., n=5 mice per group. * p&lt;0.05; ** p&lt;0.01 and *** p&lt;0.001 by ANOVA test (v.s. control or TNF-).</div></p></div><div class="aux"><div class="resc">Jiaojiao Chu, et al. PLoS One. 2013;8(2):e57231.</div><p class="links"><a class="dblinks" href="/pubmed/23437347" ref="ordinalpos=2108&amp;ncbi_uid=4649218&amp;link_uid=23437347">Citation</a><a class="dblinks" href="/pmc/articles/PMC3578810/" ref="ordinalpos=2108&amp;ncbi_uid=4649218&amp;link_uid=PMC3578810">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2109.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3847898/figure/F1/" ref="ordinalpos=2109&amp;ncbi_uid=5610539&amp;link_uid=PMC3847898" href="/pmc/articles/PMC3847898/"><img src="/pmc/articles/instance/3847898/bin/fncel-07-00224-g001.gif" src-large="/pmc/articles/instance/3847898/bin/fncel-07-00224-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3847898/" ref="ordinalpos=2109&amp;ncbi_uid=5610539&amp;link_uid=PMC3847898" image-link="/pmc/articles/PMC3847898/figure/F1/" class="imagepopup">FIGURE 1.  From: Wnts in adult brain: from synaptic plasticity to cognitive deficiencies. </a></p><div class="supp"><p class="details"><div><b>Wnt <span class="highlight" style="background-color:">signaling</span> pathways and the effects in mature synapses. (A)</b> activation of canonical Wnt/-catenin dependent <span class="highlight" style="background-color:">pathway</span> starts with the binding of Wnt ligand to Fz receptor and to the co-receptor LRP5/6, which induces the recruitment of Dvl and causes the inhibition of the -catenin destruction complex formed by Axin, APC, GSK3, and CK1. The inhibition of the -catenin destruction complex causes the accumulation of -catenin, which is no longer phosphorylated and it translocates to the nucleus where it activates transcription of Wnt target genes mediated by TCF/LEF. Canonical Wnt <span class="highlight" style="background-color:">pathway</span> participates in synaptic plasticity and cell survival. A divergent <span class="highlight" style="background-color:">pathway</span> involving proteins of the -catenin destruction complex participates in the clustering of pre-synaptic proteins and the neurotransmitter release. The Wnt/-catenin <span class="highlight" style="background-color:">pathway</span> can be blocked by Dkk-1 which binds to LRP, or through the sFRP which binds directly to Wnt ligand. <b>(B)</b> In the non-canonical Wnt/JNK <span class="highlight" style="background-color:">pathway</span>, activation of Dvl by the binding of Wnt to Fz, induces the activation of Rho and Rac small GTPases. Activation of Rho signals through ROCK and the activation of Rac signals through the JNK. This <span class="highlight" style="background-color:">pathway</span> is involved in the clustering of post-synaptic proteins and can also lead gene transcription mediated by ATF2. <b>(C)</b> In the non-canonical Wnt/Ca<sup>2</sup><sup>+</sup> <span class="highlight" style="background-color:">pathway</span>, the binding of Wnt to Fz, activates the heterotrimeric G-proteins which in turn signal through PLC and IP<sub>3</sub> to induce the release of the intracellular Ca<sup>2</sup><sup>+</sup> and the activation of PKC and CaMKII. In this <span class="highlight" style="background-color:">pathway</span>, the transcription factor CREB is activated. Wnt/Ca<sup>2</sup><sup>+</sup> <span class="highlight" style="background-color:">pathway</span> is involved in the clustering of post-synaptic proteins and the dendritic spine morphogenesis. This figure is a courtesy of Felipe Serrano, Ph.D. student.</div></p></div><div class="aux"><div class="resc">Carolina A. Oliva, et al. Front Cell Neurosci. 2013;7:224.</div><p class="links"><a class="dblinks" href="/pubmed/24348327" ref="ordinalpos=2109&amp;ncbi_uid=5610539&amp;link_uid=24348327">Citation</a><a class="dblinks" href="/pmc/articles/PMC3847898/" ref="ordinalpos=2109&amp;ncbi_uid=5610539&amp;link_uid=PMC3847898">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2110.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2635620/figure/f2-ijms-9-2217/" ref="ordinalpos=2110&amp;ncbi_uid=1724764&amp;link_uid=PMC2635620" href="/pmc/articles/PMC2635620/"><img src="/pmc/articles/instance/2635620/bin/ijms-9-2217f2.gif" src-large="/pmc/articles/instance/2635620/bin/ijms-9-2217f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2635620/" ref="ordinalpos=2110&amp;ncbi_uid=1724764&amp;link_uid=PMC2635620" image-link="/pmc/articles/PMC2635620/figure/f2-ijms-9-2217/" class="imagepopup">Figure 2. From: Contribution of Natural Inhibitors to the Understanding of the PI3K/PDK1/PKB <span class="highlight" style="background-color:">Pathway</span> in the Insulin-mediated Intracellular <span class="highlight" style="background-color:">Signaling</span> Cascade. </a></p><div class="supp"><p class="details"><div>Inhibition of each <span class="highlight" style="background-color:">signaling</span> molecules by natural products in insulin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Jae Youl Cho, et al. Int J Mol Sci. 2008 November;9(11):2217-2230.</div><p class="links"><a class="dblinks" href="/pubmed/19330070" ref="ordinalpos=2110&amp;ncbi_uid=1724764&amp;link_uid=19330070">Citation</a><a class="dblinks" href="/pmc/articles/PMC2635620/" ref="ordinalpos=2110&amp;ncbi_uid=1724764&amp;link_uid=PMC2635620">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2111.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3570895/figure/F3/" ref="ordinalpos=2111&amp;ncbi_uid=4624168&amp;link_uid=PMC3570895" href="/pmc/articles/PMC3570895/"><img src="/pmc/articles/instance/3570895/bin/fncel-07-00004-g0003.gif" src-large="/pmc/articles/instance/3570895/bin/fncel-07-00004-g0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3570895/" ref="ordinalpos=2111&amp;ncbi_uid=4624168&amp;link_uid=PMC3570895" image-link="/pmc/articles/PMC3570895/figure/F3/" class="imagepopup">Figure 3.  From: ErbB1-4-dependent EGF/neuregulin signals and their cross talk in the central nervous system: pathological implications in schizophrenia and Parkinson's disease. </a></p><div class="supp"><p class="details"><div><b>Typical signal transduction from the ErbB4:ErbB1 and ErbB4:ErbB4 complex.</b> Once ErbB1 is phosphorylated by the partner ErbB, the following signal cascades are activated; <b>(1)</b> In the PLC-PKC <span class="highlight" style="background-color:">pathway</span>, phosphorylated ErbB1 recruits and associates with PLC. As a result, PLC itself is phosphorylated to activate DAG/IP3 <span class="highlight" style="background-color:">signaling</span> (Chen et al., 1996). <b>(2)</b> In the Ras-MAPK <span class="highlight" style="background-color:">pathway</span>, phosphorylated ErbB1 associates with Shc and interacts with Grb2/Sos1. Activated Sos1 triggers GDP/GTP exchange in Ras and activates Ras, driving the sequential kinase reactions of Raf(MAPKKK), MEK(MAPKK), and Erk(MAPK). <b>(3)</b> In the PI3K-Akt <span class="highlight" style="background-color:">pathway</span>, the activated ErbB dimer interacts with Grb2/Gab1 and forms complexes with activated PI3Kinase, leading to the conversion of PIP2 to PIP3 and Akt activation. <b>(4)</b> In the JAK-STAT <span class="highlight" style="background-color:">pathway</span>, ErbB kinase phosphorylates and induces JAK to bind to ErbB1. Activated JAK phosphorylates STAT and allows STAT to homodimerize and translocate into the nucleus. Once ErbB4 is phosphorylated by ErbB4, the <span class="highlight" style="background-color:">signaling</span> cascades linked to differentiation become activated, notably the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> and the Ras-MAPK <span class="highlight" style="background-color:">pathway</span> with longer durations.</div></p></div><div class="aux"><div class="resc">Yuriko Iwakura, et al. Front Cell Neurosci. 2013;7:4.</div><p class="links"><a class="dblinks" href="/pubmed/23408472" ref="ordinalpos=2111&amp;ncbi_uid=4624168&amp;link_uid=23408472">Citation</a><a class="dblinks" href="/pmc/articles/PMC3570895/" ref="ordinalpos=2111&amp;ncbi_uid=4624168&amp;link_uid=PMC3570895">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2112.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2654975/figure/fig1/" ref="ordinalpos=2112&amp;ncbi_uid=1798183&amp;link_uid=PMC2654975" href="/pmc/articles/PMC2654975/"><img src="/pmc/articles/instance/2654975/bin/AJRCCM1796457f1.gif" src-large="/pmc/articles/instance/2654975/bin/AJRCCM1796457f1.jpg" alt="&lt;b&gt;Figure 1.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2654975/" ref="ordinalpos=2112&amp;ncbi_uid=1798183&amp;link_uid=PMC2654975" image-link="/pmc/articles/PMC2654975/figure/fig1/" class="imagepopup">Figure 1..  From: Down-regulation of the Notch <span class="highlight" style="background-color:">Pathway</span> in Human Airway Epithelium in Association with Smoking and Chronic Obstructive Pulmonary Disease. </a></p><div class="supp"><p class="details"><div>The Notch <span class="highlight" style="background-color:">pathway</span> in mammals. The <span class="highlight" style="background-color:">pathway</span> is triggered by the Delta-like or Jagged ligands interacting with one or more of the Notch receptors. The <span class="highlight" style="background-color:">pathway</span> is activated by intracellular cleavage of a Notch receptor, releasing the Notch intracellular domain, which translocates to the nucleus where it activates CBF1 (gene symbol, RBPSUH), the primary transcription factor to activate downstream Notch effector pathways (Hes1, 2, 5; Hey1, 2 and L). A variety of transcriptional coactivators and cytoplasmic and nuclear modulators tune the <span class="highlight" style="background-color:">pathway</span>, providing multiple points of control. When Notch <span class="highlight" style="background-color:">signaling</span> is active, differentiation is suppressed. When Notch <span class="highlight" style="background-color:">signaling</span> is turned off, differentiation to specific cell fates is allowed to proceed.</div></p></div><div class="aux"><div class="resc">Ann E. Tilley, et al. Am J Respir Crit Care Med. 2009 March 15;179(6):457-466.</div><p class="links"><a class="dblinks" href="/pubmed/19106307" ref="ordinalpos=2112&amp;ncbi_uid=1798183&amp;link_uid=19106307">Citation</a><a class="dblinks" href="/pmc/articles/PMC2654975/" ref="ordinalpos=2112&amp;ncbi_uid=1798183&amp;link_uid=PMC2654975">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2113.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3738179/figure/F2/" ref="ordinalpos=2113&amp;ncbi_uid=5178414&amp;link_uid=PMC3738179" href="/pmc/articles/PMC3738179/"><img src="/pmc/articles/instance/3738179/bin/nihms361943f2.gif" src-large="/pmc/articles/instance/3738179/bin/nihms361943f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3738179/" ref="ordinalpos=2113&amp;ncbi_uid=5178414&amp;link_uid=PMC3738179" image-link="/pmc/articles/PMC3738179/figure/F2/" class="imagepopup">Figure 2. The Role of Cilia in Hedgehog <span class="highlight" style="background-color:">Pathway</span> Activation in Cancer Cells.  From: Molecular Pathways: The Role of Primary Cilia in Cancer Progression and Therapeutics with a Focus on Hedgehog <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>A.</b> Cancer-Associated overexpression of Hh ligands or mutations in genes such as Ptch1 or Smo, which lie upstream of cilia, will only result in activation of the Hh <span class="highlight" style="background-color:">pathway</span> by increasing Gli<sup>A</sup> levels if cilia are present. If cilia are present, then inhibitors targeting Hh ligand, Smo, and Gli trafficking (grey boxes) will be effective. Inhibitors that target Gli activity downstream of cilia (white box) will also be effective in reducing the Hh <span class="highlight" style="background-color:">pathway</span> in this context. <b>B.</b> Cancer-associated overexpression of Gli1 (Gli<sup>A</sup>) in the presence of cilia will result in low levels of Hh <span class="highlight" style="background-color:">pathway</span> activation. In this context, cilia make the repressor form of Gli (Gli<sup>R</sup>) counterbalancing Gli<sup>A</sup> to reduce over activation of the Hh <span class="highlight" style="background-color:">pathway</span>. Mutations in REN(KCTD11) can also result in increased Gli<sup>A</sup> activity. As this activation is downstream of cilia only the downstream Gli targeting inhibitors (white box) are predicted to be effective. <b>C.</b> Cancer-associated overexpression of Hh ligands or mutations in genes such as Ptch or Smo, which lie upstream of cilia, will not activate the Hh <span class="highlight" style="background-color:">pathway</span> in the absence of cilia. <b>D.</b> Cancer-associated mutations downstream of cilia such as overexpression of Gli1 (Gli<sup>A</sup>) or mutations in Ren(KCTD11) have been documented in cancers and will turn on the Hh <span class="highlight" style="background-color:">pathway</span> in the absence of cilia due to high Gli<sup>A</sup> and low Gli<sup>R</sup>. Therefore, only downstream Gli targeting inhibitors (white box) are predicted to be effective in this scenario.</div></p></div><div class="aux"><div class="resc">Nadia B. Hassounah, et al. Clin Cancer Res. ;18(9):2429-2435.</div><p class="links"><a class="dblinks" href="/pubmed/22415315" ref="ordinalpos=2113&amp;ncbi_uid=5178414&amp;link_uid=22415315">Citation</a><a class="dblinks" href="/pmc/articles/PMC3738179/" ref="ordinalpos=2113&amp;ncbi_uid=5178414&amp;link_uid=PMC3738179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2114.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1356091/figure/pcbi-0020003-g003/" ref="ordinalpos=2114&amp;ncbi_uid=581494&amp;link_uid=PMC1356091" href="/pmc/articles/PMC1356091/"><img src="/pmc/articles/instance/1356091/bin/pcbi.0020003.g003.gif" src-large="/pmc/articles/instance/1356091/bin/pcbi.0020003.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1356091/" ref="ordinalpos=2114&amp;ncbi_uid=581494&amp;link_uid=PMC1356091" image-link="/pmc/articles/PMC1356091/figure/pcbi-0020003-g003/" class="imagepopup">Figure 3. <span class="highlight" style="background-color:">Signaling</span> and Trafficking in the TGF- <span class="highlight" style="background-color:">Pathway</span>.  From: Signal Processing in the TGF-? Superfamily Ligand-Receptor Network. </a></p><div class="supp"><p class="details"><div>Receptors in the plasma membrane interact with the <span class="highlight" style="background-color:">signaling</span> peptides of the TGF- superfamily to form active complexes. Receptors and activated ligand-receptor complexes can internalize via clathrin-coated pits into endosomes, from where the active ligand-receptor complexes phosphorylate the cytoplasmic R-Smads (receptor Smads, either the Smad1/5/8 or the Smad2/3 group) [33].<br />The phosphorylated R-Smads form complexes with the Co-Smad (Smad4) and then translocate into the nucleus where they act as transcriptional regulators of about 300 target genes.<br />The internalized receptors recycle back to the plasma membrane (with a characteristic time of ~30 min) via a rab11-dependent, rab4-independent <span class="highlight" style="background-color:">pathway</span> [14]. After returning to the plasma membrane, the receptors that were actively <span class="highlight" style="background-color:">signaling</span> can be targeted for degradation or be used for further ligand-binding or internalization [14]. Receptors that did not bind ligands are simply returned to the plasma membrane. As a consequence of the trafficking processes, only about 5%10% of receptors are present in the plasma membrane [15].<br />In addition to the traditional clathrin <span class="highlight" style="background-color:">pathway</span>, active ligand-receptor complexes can recruit Smad7-Smurf2 [28], which then targets them to lipid raftcaveolar compartments (right) for degradation [15].<br />The ligands do not return back to the plasma membrane, but disassociate from the receptors before recycling and undergo direct degradation via the lysosomes [14].<br />Note that, in addition to the ligand-induced receptor degradation, we also consider a receptor degradation <span class="highlight" style="background-color:">pathway</span> that functions independently of ligand-binding; this represents a constitutive or ligand-independent degradation <span class="highlight" style="background-color:">pathway</span> (left).</div></p></div><div class="aux"><div class="resc">Jose M. G Vilar, et al. PLoS Comput Biol. 2006 January;2(1):e3.</div><p class="links"><a class="dblinks" href="/pubmed/16446785" ref="ordinalpos=2114&amp;ncbi_uid=581494&amp;link_uid=16446785">Citation</a><a class="dblinks" href="/pmc/articles/PMC1356091/" ref="ordinalpos=2114&amp;ncbi_uid=581494&amp;link_uid=PMC1356091">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2115.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3510811/figure/F8/" ref="ordinalpos=2115&amp;ncbi_uid=5532257&amp;link_uid=PMC3510811" href="/pmc/articles/PMC3510811/"><img src="/pmc/articles/instance/3510811/bin/zbc0521232590008.gif" src-large="/pmc/articles/instance/3510811/bin/zbc0521232590008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3510811/" ref="ordinalpos=2115&amp;ncbi_uid=5532257&amp;link_uid=PMC3510811" image-link="/pmc/articles/PMC3510811/figure/F8/" class="imagepopup">FIGURE 8. From: A Novel Regulatory Factor Recruits the Nucleosome Remodeling Complex to Wingless Integrated (Wnt) <span class="highlight" style="background-color:">Signaling</span> Gene Promoters in Mouse Embryonic Stem Cells. </a></p><div class="supp"><p class="details"><div><b>NuRD, differentiation, and Wnt in relation to Cdk2ap1 in a global context.</b> A complete metanetwork map of mESCs to explain how nucleosome remodeling, differentiation, and Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are interconnected and linked to CDK2AP1.</div></p></div><div class="aux"><div class="resc">Jeffrey J. Kim, et al. J Biol Chem. 2012 November 30;287(49):41103-41117.</div><p class="links"><a class="dblinks" href="/pubmed/23074223" ref="ordinalpos=2115&amp;ncbi_uid=5532257&amp;link_uid=23074223">Citation</a><a class="dblinks" href="/pmc/articles/PMC3510811/" ref="ordinalpos=2115&amp;ncbi_uid=5532257&amp;link_uid=PMC3510811">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2116.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3016855/figure/F8/" ref="ordinalpos=2116&amp;ncbi_uid=2949224&amp;link_uid=PMC3016855" href="/pmc/articles/PMC3016855/"><img src="/pmc/articles/instance/3016855/bin/nihms261074f8.gif" src-large="/pmc/articles/instance/3016855/bin/nihms261074f8.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3016855/" ref="ordinalpos=2116&amp;ncbi_uid=2949224&amp;link_uid=PMC3016855" image-link="/pmc/articles/PMC3016855/figure/F8/" class="imagepopup">FIGURE 8.  From: Calcium-Activated Pathways and Oxidative Burst Mediate Zymosan-Induced <span class="highlight" style="background-color:">Signaling</span> and IL-10 Production in Human Macrophages. </a></p><div class="supp"><p class="details"><div>In human primary macrophages, zymosan activates a calcium-dependent CaMKIIPyk2ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream of Dectin-1 and ITAM-associated kinases Src and Syk. Calcium pathways also induce CREB phosphorylation and generation of ROS. ERK, CREB, and ROS contribute to calcium-mediated IL-10 production.</div></p></div><div class="aux"><div class="resc">Erin K. Kelly, et al. J Immunol. ;184(10):5545-5552.</div><p class="links"><a class="dblinks" href="/pubmed/20400701" ref="ordinalpos=2116&amp;ncbi_uid=2949224&amp;link_uid=20400701">Citation</a><a class="dblinks" href="/pmc/articles/PMC3016855/" ref="ordinalpos=2116&amp;ncbi_uid=2949224&amp;link_uid=PMC3016855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2117.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3317801/figure/fig1/" ref="ordinalpos=2117&amp;ncbi_uid=4260585&amp;link_uid=PMC3317801" href="/pmc/articles/PMC3317801/"><img src="/pmc/articles/instance/3317801/bin/jcb.1971iffig1.gif" src-large="/pmc/articles/instance/3317801/bin/jcb.1971iffig1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3317801/" ref="ordinalpos=2117&amp;ncbi_uid=4260585&amp;link_uid=PMC3317801" image-link="/pmc/articles/PMC3317801/figure/fig1/" class="imagepopup">Figure. From: Big enough for two. </a></p><div class="supp"><p class="details"><div><b>FOCAL POINT</b>(Left to right) Tracy MacDonough, Steph Anastasia, Doug Kellogg, Duy Nguyen, and colleagues (not pictured) uncover a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that links membrane transport to mitotic entry in budding yeast. Cells arrest in early mitosis when <span class="highlight" style="background-color:">signaling</span> through the <span class="highlight" style="background-color:">pathway</span> is disrupted, either by blocking membrane traffic or by mutating <span class="highlight" style="background-color:">pathway</span> components like protein kinase C (right, compared with wild-type yeast, center). The researchers think that the <span class="highlight" style="background-color:">pathway</span> may allow cells to control their size by coordinating cell growth and division.<br />PHOTO COURTESY OF VU THAI</div></p></div><div class="aux"><div class="resc">Ben Short. J Cell Biol. 2012 April 2;197(1):3-3.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2117&amp;ncbi_uid=4260585&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC3317801/" ref="ordinalpos=2117&amp;ncbi_uid=4260585&amp;link_uid=PMC3317801">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2118.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2789116/figure/F3/" ref="ordinalpos=2118&amp;ncbi_uid=2219379&amp;link_uid=PMC2789116" href="/pmc/articles/PMC2789116/"><img src="/pmc/articles/instance/2789116/bin/nihms113201f3.gif" src-large="/pmc/articles/instance/2789116/bin/nihms113201f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2789116/" ref="ordinalpos=2118&amp;ncbi_uid=2219379&amp;link_uid=PMC2789116" image-link="/pmc/articles/PMC2789116/figure/F3/" class="imagepopup">Figure 3.  From: Regulation of Neural Specification from Human Embryonic Stem Cells by BMP and FGF. </a></p><div class="supp"><p class="details"><div>Summary of gene expression changes in the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during hESC neural induction. (A) A schematic model of the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [4]. (B) A summary of expression changes of key components in the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Reported changes are ratios of expression values of samples from day 6 or day 10 compared to day 0 (ES cell stage). Yellow boxes indicate no change in expression from ESC stage to day 6 or day 10. Red boxes indicate an increase in expression level.</div></p></div><div class="aux"><div class="resc">Timothy M. LaVaute, et al. Stem Cells. ;27(8):1741-1749.</div><p class="links"><a class="dblinks" href="/pubmed/19544434" ref="ordinalpos=2118&amp;ncbi_uid=2219379&amp;link_uid=19544434">Citation</a><a class="dblinks" href="/pmc/articles/PMC2789116/" ref="ordinalpos=2118&amp;ncbi_uid=2219379&amp;link_uid=PMC2789116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2119.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3763548/figure/gkt563-F8/" ref="ordinalpos=2119&amp;ncbi_uid=5289626&amp;link_uid=PMC3763548" href="/pmc/articles/PMC3763548/"><img src="/pmc/articles/instance/3763548/bin/gkt563f8p.gif" src-large="/pmc/articles/instance/3763548/bin/gkt563f8p.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3763548/" ref="ordinalpos=2119&amp;ncbi_uid=5289626&amp;link_uid=PMC3763548" image-link="/pmc/articles/PMC3763548/figure/gkt563-F8/" class="imagepopup">Figure 8. From: The eIF2?/ATF4 <span class="highlight" style="background-color:">pathway</span> is essential for stress-induced autophagy gene expression. </a></p><div class="supp"><p class="details"><div>A model for the role of the eIF2/ATF4 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the transcriptional activation of autophagy genes. The mode by which each gene is regulated is indicated in parenthesis. Three classes of autophagy genes have been identified according to their dependence on ATF4 and CHOP and the binding of these factors to the promoter.</div></p></div><div class="aux"><div class="resc">Wafa Bchir, et al. Nucleic Acids Res. 2013 September;41(16):7683-7699.</div><p class="links"><a class="dblinks" href="/pubmed/23804767" ref="ordinalpos=2119&amp;ncbi_uid=5289626&amp;link_uid=23804767">Citation</a><a class="dblinks" href="/pmc/articles/PMC3763548/" ref="ordinalpos=2119&amp;ncbi_uid=5289626&amp;link_uid=PMC3763548">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2120.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4338248/figure/pone.0118458.g002/" ref="ordinalpos=2120&amp;ncbi_uid=7228567&amp;link_uid=PMC4338248" href="/pmc/articles/PMC4338248/"><img src="/pmc/articles/instance/4338248/bin/pone.0118458.g002.gif" src-large="/pmc/articles/instance/4338248/bin/pone.0118458.g002.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4338248/" ref="ordinalpos=2120&amp;ncbi_uid=7228567&amp;link_uid=PMC4338248" image-link="/pmc/articles/PMC4338248/figure/pone.0118458.g002/" class="imagepopup">Fig 2. Analysis of 79 differently expressed genes of TLR4/ NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in normal and mastitis mammary tissues (P&lt;0.01)..  From: Altered Molecular Expression of the TLR4/NF-B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Mammary Tissue of Chinese Holstein Cattle with Mastitis. </a></p><div class="supp"><p class="details"><div>The microarray heat map demonstrates the log2 ratio of gene signal difference between normal and mastitis tissues. Up-regulated genes are shown in red, while down-regulated genes are shown in green. The left column on the figure represents the level of expression in normal mammary tissue, while the right column represents the level of corresponding gene expression in mastitis mammary tissue.</div></p></div><div class="aux"><div class="resc">Jie Wu, et al. PLoS One. 2015;10(2):e0118458.</div><p class="links"><a class="dblinks" href="/pubmed/25706977" ref="ordinalpos=2120&amp;ncbi_uid=7228567&amp;link_uid=25706977">Citation</a><a class="dblinks" href="/pmc/articles/PMC4338248/" ref="ordinalpos=2120&amp;ncbi_uid=7228567&amp;link_uid=PMC4338248">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2121.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g003/" ref="ordinalpos=2121&amp;ncbi_uid=2941430&amp;link_uid=PMC3086880" href="/pmc/articles/PMC3086880/"><img src="/pmc/articles/instance/3086880/bin/pone.0019240.g003.gif" src-large="/pmc/articles/instance/3086880/bin/pone.0019240.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3086880/" ref="ordinalpos=2121&amp;ncbi_uid=2941430&amp;link_uid=PMC3086880" image-link="/pmc/articles/PMC3086880/figure/pone-0019240-g003/" class="imagepopup">Figure 3. Verification of the novelty of signalogs (protein  <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> annotations)..  From: Signalogs: Orthology-Based Identification of Novel <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Components in Three Metazoans. </a></p><div class="supp"><p class="details"><div>a) Classification of signalog proteins based on their known properties in the literature found by manual curation and searches in PubMed, UniProt, GO, Wormbase, Flybase, iHOP, and Chilibot [32][35]. bc) Comparison of signalog and interolog data between Signalink and one of the WI8, DroID or HomoMINT databases. The signalog proteins and possible interologs that are only present in SignaLink have not been predicted by previous orthology based (interolog) studies. This underscores the novelty of the signalogs. For details on interologs predicted from SignaLink data, see the main text.</div></p></div><div class="aux"><div class="resc">Tams Korcsmros, et al. PLoS One. 2011;6(5):e19240.</div><p class="links"><a class="dblinks" href="/pubmed/21559328" ref="ordinalpos=2121&amp;ncbi_uid=2941430&amp;link_uid=21559328">Citation</a><a class="dblinks" href="/pmc/articles/PMC3086880/" ref="ordinalpos=2121&amp;ncbi_uid=2941430&amp;link_uid=PMC3086880">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2122.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3899443/figure/F3/" ref="ordinalpos=2122&amp;ncbi_uid=5730467&amp;link_uid=PMC3899443" href="/pmc/articles/PMC3899443/"><img src="/pmc/articles/instance/3899443/bin/1479-5876-12-16-3.gif" src-large="/pmc/articles/instance/3899443/bin/1479-5876-12-16-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3899443/" ref="ordinalpos=2122&amp;ncbi_uid=5730467&amp;link_uid=PMC3899443" image-link="/pmc/articles/PMC3899443/figure/F3/" class="imagepopup">Figure 3.  From: The therapeutic target of estrogen receptor-alpha36 in estrogen-dependent tumors. </a></p><div class="supp"><p class="details"><div><b>The genomic activity <span class="highlight" style="background-color:">signaling</span> of nuclear estrogen receptors (ERs) can be inhibited by ER-36.</b> The classical <span class="highlight" style="background-color:">pathway</span> of estrogen <span class="highlight" style="background-color:">signaling</span> is ligand-activated ERs bind specifically to estrogen response elements (EREs) in the promoter of target genes. The ligand-dependent indirect genomic regulation of gene transcription includes interactions with other transcription factors (TF). In ligand-independent <span class="highlight" style="background-color:">pathway</span>, ERs can be stimulated by other <span class="highlight" style="background-color:">signaling</span> pathways, such as growth factor <span class="highlight" style="background-color:">signaling</span> which eliciting genomic effects in the absence of ligands. In this case, activated kinases phosphorylate ERs, it thereby activate them to dimerize, bind DNA and regulate genes. ER-36 can suppress both estrogen-dependent and estrogen-independent transactivation functions which <span class="highlight" style="background-color:">signaling</span> through nuclear ERs like ER-66 and ER-.</div></p></div><div class="aux"><div class="resc">Yu Gu, et al. J Transl Med. 2014;12:16-16.</div><p class="links"><a class="dblinks" href="/pubmed/24447535" ref="ordinalpos=2122&amp;ncbi_uid=5730467&amp;link_uid=24447535">Citation</a><a class="dblinks" href="/pmc/articles/PMC3899443/" ref="ordinalpos=2122&amp;ncbi_uid=5730467&amp;link_uid=PMC3899443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2123.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4301981/figure/F4/" ref="ordinalpos=2123&amp;ncbi_uid=7084991&amp;link_uid=PMC4301981" href="/pmc/articles/PMC4301981/"><img src="/pmc/articles/instance/4301981/bin/nihms-491846-f0004.gif" src-large="/pmc/articles/instance/4301981/bin/nihms-491846-f0004.jpg" alt="FIG. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4301981/" ref="ordinalpos=2123&amp;ncbi_uid=7084991&amp;link_uid=PMC4301981" image-link="/pmc/articles/PMC4301981/figure/F4/" class="imagepopup">FIG. 4.  From: Functional Proteomics Analysis to Study ATM Dependent <span class="highlight" style="background-color:">Signaling</span> in Response to Ionizing Radiation. </a></p><div class="supp"><p class="details"><div>Functional <span class="highlight" style="background-color:">pathway</span> analysis reveals differential stress response <span class="highlight" style="background-color:">signaling</span> in an ATM dependent manner 30 min after irradiation. Differential protein expression was determined by comparing samples that were harvested after radiation treatment to unirradiated samples for each cell type. Canonical <span class="highlight" style="background-color:">pathway</span> enrichment analysis shows a robust oxidative stress response <span class="highlight" style="background-color:">signaling</span> by ATCL8 with an enrichment of proteins in integrin, VEGF and Ephrin receptor <span class="highlight" style="background-color:">signaling</span> (Panel A) in contrast to AT5BIVA (Panel B).</div></p></div><div class="aux"><div class="resc">Amrita K. Cheema, et al. Radiat Res. ;179(6):674-683.</div><p class="links"><a class="dblinks" href="/pubmed/23642045" ref="ordinalpos=2123&amp;ncbi_uid=7084991&amp;link_uid=23642045">Citation</a><a class="dblinks" href="/pmc/articles/PMC4301981/" ref="ordinalpos=2123&amp;ncbi_uid=7084991&amp;link_uid=PMC4301981">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2124.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3390208/figure/F8/" ref="ordinalpos=2124&amp;ncbi_uid=4444497&amp;link_uid=PMC3390208" href="/pmc/articles/PMC3390208/"><img src="/pmc/articles/instance/3390208/bin/nihms-385665-f0008.gif" src-large="/pmc/articles/instance/3390208/bin/nihms-385665-f0008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3390208/" ref="ordinalpos=2124&amp;ncbi_uid=4444497&amp;link_uid=PMC3390208" image-link="/pmc/articles/PMC3390208/figure/F8/" class="imagepopup">Figure 8.  From: BDNF activation of CaM-kinase kinase via TRPC channels induces the translation and synaptic incorporation of GluA1 containing calcium-permeable AMPARs. </a></p><div class="supp"><p class="details"><div>Schematic of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> involving CaMKK regulation of GluA1 synthesis and synaptic incorporation as CP-AMPARs. <span class="highlight" style="background-color:">Signaling</span> shown in bold font is documented in the current report whereas <span class="highlight" style="background-color:">signaling</span> in normal font is taken from the literature as cited in the text.</div></p></div><div class="aux"><div class="resc">Dale A. Fortin, et al. J Neurosci. ;32(24):8127-8137.</div><p class="links"><a class="dblinks" href="/pubmed/22699894" ref="ordinalpos=2124&amp;ncbi_uid=4444497&amp;link_uid=22699894">Citation</a><a class="dblinks" href="/pmc/articles/PMC3390208/" ref="ordinalpos=2124&amp;ncbi_uid=4444497&amp;link_uid=PMC3390208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2125.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3247139/figure/F1/" ref="ordinalpos=2125&amp;ncbi_uid=3735628&amp;link_uid=PMC3247139" href="/pmc/articles/PMC3247139/"><img src="/pmc/articles/instance/3247139/bin/1752-0509-5-178-1.gif" src-large="/pmc/articles/instance/3247139/bin/1752-0509-5-178-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3247139/" ref="ordinalpos=2125&amp;ncbi_uid=3735628&amp;link_uid=PMC3247139" image-link="/pmc/articles/PMC3247139/figure/F1/" class="imagepopup">Figure 1.  From: Time-resolved in silico modeling of fine-tuned cAMP <span class="highlight" style="background-color:">signaling</span> in platelets: feedback loops, titrated phosphorylations and pharmacological modulation. </a></p><div class="supp"><p class="details"><div><b>Topological scheme of modeled cyclic nucleotide <span class="highlight" style="background-color:">signaling</span> pathways</b>. Main components of cyclic nucleotide <span class="highlight" style="background-color:">signaling</span> pathways and their cross-talk and downstream effects are depicted: Major <span class="highlight" style="background-color:">signaling</span> components of the simplified cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are shown in yellow, pivotal components of cGMP <span class="highlight" style="background-color:">signaling</span> in green and the three central cyclic nucleotide degrading phosphodiesterases (PDEs) are depicted in blue. Both, cAMP and cGMP <span class="highlight" style="background-color:">signaling</span> share in downstream effects on VASP (brown), which is phosphorylated at both, Ser157 and Ser239 differentially upon cyclic nucleotide dependent activation of their corresponding protein kinases (PKA, PKG). Levels of cAMP and cGMP react sensitively to <span class="highlight" style="background-color:">pathway</span> modulating compounds such as inhibitors of PDEs (Cilostamide, Milrinone) as well as adenylyl cyclase (AC) stimulating drugs (Iloprost, Forskolin) shown in red rectangles. This direct activation (green dashed arrow) and/or inhibition (red arrow) leads to an elevated cAMP level which subsequently feeds back on PDE3 (positive feedback) as well as on AC (negative feedback), as indicated by bold green and red colored arrows.</div></p></div><div class="aux"><div class="resc">Gaby Wangorsch, et al. BMC Syst Biol. 2011;5:178-178.</div><p class="links"><a class="dblinks" href="/pubmed/22034949" ref="ordinalpos=2125&amp;ncbi_uid=3735628&amp;link_uid=22034949">Citation</a><a class="dblinks" href="/pmc/articles/PMC3247139/" ref="ordinalpos=2125&amp;ncbi_uid=3735628&amp;link_uid=PMC3247139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2126.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3574187/figure/F8/" ref="ordinalpos=2126&amp;ncbi_uid=5818483&amp;link_uid=PMC3574187" href="/pmc/articles/PMC3574187/"><img src="/pmc/articles/instance/3574187/bin/nihms435041f8.gif" src-large="/pmc/articles/instance/3574187/bin/nihms435041f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3574187/" ref="ordinalpos=2126&amp;ncbi_uid=5818483&amp;link_uid=PMC3574187" image-link="/pmc/articles/PMC3574187/figure/F8/" class="imagepopup">Figure 8. Proposed Model for EC-mediated Paracrine Activation of Notch <span class="highlight" style="background-color:">Signaling</span> in Colorectal Cancer Cells.  From: Endothelial Cells Promote the Colorectal Cancer Stem Cell Phenotype Through a Soluble Form of Jagged-1. </a></p><div class="supp"><p class="details"><div>(A) The conical model for Notch <span class="highlight" style="background-color:">pathway</span> activation, where membrane bound ligands such as Jagged-1 activates Notch <span class="highlight" style="background-color:">signaling</span> in contacting cells.<br />(B) Schematic summarizing our proposed model for paracrine activation of the Notch <span class="highlight" style="background-color:">pathway</span> in CRC cells. ADAM17 cleaves membrane bound Jagged-1 on ECs, releasing an N-terminal soluble fragment that binds, and activates, Notch on CRC cells.</div></p></div><div class="aux"><div class="resc">Jia Lu, et al. Cancer Cell. ;23(2):171-185.</div><p class="links"><a class="dblinks" href="/pubmed/23375636" ref="ordinalpos=2126&amp;ncbi_uid=5818483&amp;link_uid=23375636">Citation</a><a class="dblinks" href="/pmc/articles/PMC3574187/" ref="ordinalpos=2126&amp;ncbi_uid=5818483&amp;link_uid=PMC3574187">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2127.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3381956/figure/F1/" ref="ordinalpos=2127&amp;ncbi_uid=4077053&amp;link_uid=PMC3381956" href="/pmc/articles/PMC3381956/"><img src="/pmc/articles/instance/3381956/bin/nihms-381274-f0001.gif" src-large="/pmc/articles/instance/3381956/bin/nihms-381274-f0001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3381956/" ref="ordinalpos=2127&amp;ncbi_uid=4077053&amp;link_uid=PMC3381956" image-link="/pmc/articles/PMC3381956/figure/F1/" class="imagepopup">FIGURE 1.  From: Targeted Therapies for Lung Cancer. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways and their targeted therapies in NSCLC. Aberrant <span class="highlight" style="background-color:">signaling</span> resulting in activation of growth stimulatory pathways or interference of growth inhibitory pathways has been implicated in lung cancer pathogenesis. Activation of RTKs through ligand binding leads to up-regulation of multiple <span class="highlight" style="background-color:">signaling</span> pathways including the RAS/RAF/MEK <span class="highlight" style="background-color:">pathway</span>, the PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span>, and the STAT. GF, growth factor.</div></p></div><div class="aux"><div class="resc">Jill E. Larsen, et al. Cancer J. ;17(6):512-527.</div><p class="links"><a class="dblinks" href="/pubmed/22157296" ref="ordinalpos=2127&amp;ncbi_uid=4077053&amp;link_uid=22157296">Citation</a><a class="dblinks" href="/pmc/articles/PMC3381956/" ref="ordinalpos=2127&amp;ncbi_uid=4077053&amp;link_uid=PMC3381956">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2128.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3421445/figure/f1/" ref="ordinalpos=2128&amp;ncbi_uid=4179785&amp;link_uid=PMC3421445" href="/pmc/articles/PMC3421445/"><img src="/pmc/articles/instance/3421445/bin/msb201230-f1.gif" src-large="/pmc/articles/instance/3421445/bin/msb201230-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3421445/" ref="ordinalpos=2128&amp;ncbi_uid=4179785&amp;link_uid=PMC3421445" image-link="/pmc/articles/PMC3421445/figure/f1/" class="imagepopup">Figure 1.  From: Metabolic rewiring drives resistance to targeted cancer therapy. </a></p><div class="supp"><p class="details"><div>Metabolic adaptations drive acquired drug resistance in breast cancer cells. (Left) In drug-sensitive cells, cell growth and proliferation are dependent on receptor tyrosine kinase (RTK) <span class="highlight" style="background-color:">signaling</span>. RTK <span class="highlight" style="background-color:">signaling</span> induces Ras and PI3K <span class="highlight" style="background-color:">pathway</span> activity that in turn modulates (arrows) multiple aspects of cellular metabolism that are required for cell growth and proliferation. Upon treatment of Lapatinib, RTK <span class="highlight" style="background-color:">signaling</span> is disrupted and activity of its downstream effectors is abrogated. Consequently, Ras and PI3K <span class="highlight" style="background-color:">signaling</span> is inhibited and oncogene-dependent metabolic rewiring is prevented. As a result, cancer cell death is observed. (Right) In drug-resistant cells, the Ras and PI3K pathways remain inhibited by Lapatinib treatment. However, disruption of these <span class="highlight" style="background-color:">signaling</span> pathways is insufficient to alter cellular metabolism and stress responses. As a result, cells remain viable and continue to proliferate in the absence of RTK <span class="highlight" style="background-color:">signaling</span> and Ras and PI3K <span class="highlight" style="background-color:">pathway</span> activity.</div></p></div><div class="aux"><div class="resc">Jason W Locasale. Mol Syst Biol. 2012;8:597-597.</div><p class="links"><a class="dblinks" href="/pubmed/22806144" ref="ordinalpos=2128&amp;ncbi_uid=4179785&amp;link_uid=22806144">Citation</a><a class="dblinks" href="/pmc/articles/PMC3421445/" ref="ordinalpos=2128&amp;ncbi_uid=4179785&amp;link_uid=PMC3421445">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2129.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g002/" ref="ordinalpos=2129&amp;ncbi_uid=4696889&amp;link_uid=PMC3590201" href="/pmc/articles/PMC3590201/"><img src="/pmc/articles/instance/3590201/bin/pone.0057022.g002.gif" src-large="/pmc/articles/instance/3590201/bin/pone.0057022.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3590201/" ref="ordinalpos=2129&amp;ncbi_uid=4696889&amp;link_uid=PMC3590201" image-link="/pmc/articles/PMC3590201/figure/pone-0057022-g002/" class="imagepopup">Figure 2. KEGG <span class="highlight" style="background-color:">pathway</span> map for TGF-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: The Identification of <span class="highlight" style="background-color:">Pathway</span> Markers in Intracranial Aneurysm Using Genome-Wide Association Data from Two Different Populations. </a></p><div class="supp"><p class="details"><div>The shade of red color in genes indicates the number of targeted SNPs in JP population per base pair of the gene. Red refers to the highest targeted gene, whereas white refers to a gene product, not targeted by the SNPs. Blue border indicates that the gene is found to be differentially expressed.</div></p></div><div class="aux"><div class="resc">Burcu Bakir-Gungor, et al. PLoS One. 2013;8(3):e57022.</div><p class="links"><a class="dblinks" href="/pubmed/23483893" ref="ordinalpos=2129&amp;ncbi_uid=4696889&amp;link_uid=23483893">Citation</a><a class="dblinks" href="/pmc/articles/PMC3590201/" ref="ordinalpos=2129&amp;ncbi_uid=4696889&amp;link_uid=PMC3590201">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2130.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3288889/figure/F5/" ref="ordinalpos=2130&amp;ncbi_uid=4577785&amp;link_uid=PMC3288889" href="/pmc/articles/PMC3288889/"><img src="/pmc/articles/instance/3288889/bin/nihms337472f5.gif" src-large="/pmc/articles/instance/3288889/bin/nihms337472f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3288889/" ref="ordinalpos=2130&amp;ncbi_uid=4577785&amp;link_uid=PMC3288889" image-link="/pmc/articles/PMC3288889/figure/F5/" class="imagepopup">Figure 5.  From: "non-SH2/PDZ reverse <span class="highlight" style="background-color:">signaling</span> by ephrins". </a></p><div class="supp"><p class="details"><div>Ephrin-B cleavage and <span class="highlight" style="background-color:">signaling</span>. A) Upon bind the EphB receptor, ephrin-B may be proteolytically processed by MMPs and presenilin 1/-gamma secretase to release a C-terminal fragment. The free fragment may translocate to the nucleus to perform a yet unknown function, or possibly the fragment binds to the Src kinase and inhibits its association with inhibitory kinase Csk, and allows Src autophosphorylation. Src, in turn, phosphorylates ephrin-B2, which can recruit certain <span class="highlight" style="background-color:">signaling</span> partners, such as Grb4, leading to functional effects. B) Unliganded ephrin-Bs may be cleaved by ADAM13 leading to an increase in canonical Wnt <span class="highlight" style="background-color:">signaling</span> through the cognate EphB receptor. Upon binding to the EphB receptor, ephrin-Bs may be resistant to ADAM13 cleavage, thus activating non-canonical Wnt/PCP <span class="highlight" style="background-color:">pathway</span> (not pictured) at the expense of Wnt/beta-catenin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Ira O. Daar. Semin Cell Dev Biol. ;23(1):65-74.</div><p class="links"><a class="dblinks" href="/pubmed/22040914" ref="ordinalpos=2130&amp;ncbi_uid=4577785&amp;link_uid=22040914">Citation</a><a class="dblinks" href="/pmc/articles/PMC3288889/" ref="ordinalpos=2130&amp;ncbi_uid=4577785&amp;link_uid=PMC3288889">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2131.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2650409/figure/pone-0004686-g006/" ref="ordinalpos=2131&amp;ncbi_uid=1741465&amp;link_uid=PMC2650409" href="/pmc/articles/PMC2650409/"><img src="/pmc/articles/instance/2650409/bin/pone.0004686.g006.gif" src-large="/pmc/articles/instance/2650409/bin/pone.0004686.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2650409/" ref="ordinalpos=2131&amp;ncbi_uid=1741465&amp;link_uid=PMC2650409" image-link="/pmc/articles/PMC2650409/figure/pone-0004686-g006/" class="imagepopup">Figure 6. Model of upstream activation of Akt by ghrelin..  From: c-Src Regulates Akt <span class="highlight" style="background-color:">Signaling</span> in Response to Ghrelin via ?-Arrestin <span class="highlight" style="background-color:">Signaling</span>-Independent and -Dependent Mechanisms. </a></p><div class="supp"><p class="details"><div>Binding of ghrelin to GHS-R1a activates the generation of the second messenger PtdIns(3,4,5)P<sub>3</sub> (PIP3) by PI3K activation through G<sub>i/o</sub>-protein dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Akt translocates to the plasma membrane by binding to PIP3 where it is phosphorylated at Y by the membrane bound c-Src. This phosphorylation (Y) is followed by phosphorylation of Akt A-loop (T308) and HM (S473) by PDK1 and mTORC2 respectively. Once the receptor is activated, a second <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is mediated by -arrestins 1 and 2, involving the recruitment of GHS-R1a, c-Src and Akt into an -arrestin-scaffolded complex.</div></p></div><div class="aux"><div class="resc">Maria Lodeiro, et al. PLoS ONE. 2009;4(3):e4686.</div><p class="links"><a class="dblinks" href="/pubmed/19262695" ref="ordinalpos=2131&amp;ncbi_uid=1741465&amp;link_uid=19262695">Citation</a><a class="dblinks" href="/pmc/articles/PMC2650409/" ref="ordinalpos=2131&amp;ncbi_uid=1741465&amp;link_uid=PMC2650409">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2132.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f004/" ref="ordinalpos=2132&amp;ncbi_uid=5948545&amp;link_uid=PMC3967167" href="/pmc/articles/PMC3967167/"><img src="/pmc/articles/instance/3967167/bin/viruses-05-03192-g004.gif" src-large="/pmc/articles/instance/3967167/bin/viruses-05-03192-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3967167/" ref="ordinalpos=2132&amp;ncbi_uid=5948545&amp;link_uid=PMC3967167" image-link="/pmc/articles/PMC3967167/figure/viruses-05-03192-f004/" class="imagepopup">Figure 4.  From: Make Yourself at Home: Viral Hijacking of the PI3K/Akt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>As a key regulator of cellular survival the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is often triggered by viruses. Upon incoming survival signals, the Akt-mediated inhibitory phosphorylation of pro-apoptotic molecules such as BAD prevents the induction of the caspase cascade. In addition, the inhibitory phosphorylation of the transcription factor FOXO1 by Akt blocks its translocation into the nucleus and thus prevents the expression of pro-apoptotic genes. A variety of viral proteins steps into this regulatory network to assure cell survival, and thus maintaining the cellular environment for viral replication. Known viral proteins interacting with the cellular member of the <span class="highlight" style="background-color:">signaling</span> pathways are highlighted in red.</div></p></div><div class="aux"><div class="resc">Nora Diehl, et al. Viruses. 2013 December;5(12):3192-3212.</div><p class="links"><a class="dblinks" href="/pubmed/24351799" ref="ordinalpos=2132&amp;ncbi_uid=5948545&amp;link_uid=24351799">Citation</a><a class="dblinks" href="/pmc/articles/PMC3967167/" ref="ordinalpos=2132&amp;ncbi_uid=5948545&amp;link_uid=PMC3967167">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2133.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2771431/figure/F4/" ref="ordinalpos=2133&amp;ncbi_uid=2169023&amp;link_uid=PMC2771431" href="/pmc/articles/PMC2771431/"><img src="/pmc/articles/instance/2771431/bin/nihms119374f4.gif" src-large="/pmc/articles/instance/2771431/bin/nihms119374f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2771431/" ref="ordinalpos=2133&amp;ncbi_uid=2169023&amp;link_uid=PMC2771431" image-link="/pmc/articles/PMC2771431/figure/F4/" class="imagepopup">Fig. 4.  From: Dynamic <span class="highlight" style="background-color:">Signaling</span> Network for the Specification of Embryonic Pancreas and Liver Progenitors. </a></p><div class="supp"><p class="details"><div>TGF <span class="highlight" style="background-color:">signaling</span> restrains the specification of ventral pancreatic progenitors. A,C,E. Whole mount X-gal staining (blue) for pancreatic progenitors. B, D, F. Sagittal sections, counterstained for Ki67 (brown). G. Dynamic network inducing the earliest genes for liver and pancreas specification. Arrows, positive <span class="highlight" style="background-color:">signaling</span>; barred lines, negative. The thicker lines for BMP and thinner lines for TGF at 5-6S indicate BMP as the dominant <i>Smad4</i>-dependent <span class="highlight" style="background-color:">pathway</span> for <i>alb1</i>, <i>Hnf6</i>, and <i>Pdx1</i>.</div></p></div><div class="aux"><div class="resc">Ewa Wandzioch, et al. Science. ;324(5935):1707-1710.</div><p class="links"><a class="dblinks" href="/pubmed/19556507" ref="ordinalpos=2133&amp;ncbi_uid=2169023&amp;link_uid=19556507">Citation</a><a class="dblinks" href="/pmc/articles/PMC2771431/" ref="ordinalpos=2133&amp;ncbi_uid=2169023&amp;link_uid=PMC2771431">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2134.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3654375/figure/F3/" ref="ordinalpos=2134&amp;ncbi_uid=4898791&amp;link_uid=PMC3654375" href="/pmc/articles/PMC3654375/"><img src="/pmc/articles/instance/3654375/bin/nihms462306f3.gif" src-large="/pmc/articles/instance/3654375/bin/nihms462306f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3654375/" ref="ordinalpos=2134&amp;ncbi_uid=4898791&amp;link_uid=PMC3654375" image-link="/pmc/articles/PMC3654375/figure/F3/" class="imagepopup">Figure 3. VEGF-VEGFR2 <span class="highlight" style="background-color:">signaling</span> in the nervous and vascular systems.  From: Guidance from above: common cues direct distinct <span class="highlight" style="background-color:">signaling</span> outcomes in vascular and neural patterning. </a></p><div class="supp"><p class="details"><div><b>A: VEGF <span class="highlight" style="background-color:">signaling</span> in the vascular system:</b> The PI3K-Akt <span class="highlight" style="background-color:">pathway</span> is shown in red: VEGF binding to VEGFR2 causes autophosphorylation of the receptor. Phosphorylation of the receptor at tyrosine 1175 causes the recruitment of the adaptor protein Shb, which then activates PI3K. PI3K phosphorylates Akt, which acts through substrates like the actin binding protein Girdin to regulate endothelial cell migration. The Rac1 <span class="highlight" style="background-color:">pathway</span> is shown in green: PI3K also activates Rac1, which regulates migration through the effectors WAVE2 and the actin binding protein Profilin. Phosphorylation of Tyr1175 also causes activation of PLC, which activates the small GTPases Rac1 and RhoA to regulate endothelial cell migration. <b>B: VEGF-VEGFR2 <span class="highlight" style="background-color:">signaling</span> in the nervous system</b>: Many of the same proteins used by VEGFR2 <span class="highlight" style="background-color:">signaling</span> in the vascular system are also implicated to act in the nervous system, but the evidence for this is less clear (indicated by question marks). PI3K inhibitors abrogate VEGFs neuroprotective effect, while PLC and PKC inhibitors prevented the neurotrophic effect exerted by VEGF on cortical neurons.</div></p></div><div class="aux"><div class="resc">Maria V. Gelfand, et al. Trends Cell Biol. 2009 March;19(3):99-110.</div><p class="links"><a class="dblinks" href="/pubmed/19200729" ref="ordinalpos=2134&amp;ncbi_uid=4898791&amp;link_uid=19200729">Citation</a><a class="dblinks" href="/pmc/articles/PMC3654375/" ref="ordinalpos=2134&amp;ncbi_uid=4898791&amp;link_uid=PMC3654375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2135.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676390/figure/pone-0065027-g005/" ref="ordinalpos=2135&amp;ncbi_uid=4995156&amp;link_uid=PMC3676390" href="/pmc/articles/PMC3676390/"><img src="/pmc/articles/instance/3676390/bin/pone.0065027.g005.gif" src-large="/pmc/articles/instance/3676390/bin/pone.0065027.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676390/" ref="ordinalpos=2135&amp;ncbi_uid=4995156&amp;link_uid=PMC3676390" image-link="/pmc/articles/PMC3676390/figure/pone-0065027-g005/" class="imagepopup">Figure 5. Schematic <span class="highlight" style="background-color:">pathway</span> predicting the role of MeJA induced genes in G. lucidum..  From: Profiling and Quantifying Differential Gene Transcription Provide Insights into Ganoderic Acid Biosynthesis in Ganoderma lucidum in Response to Methyl Jasmonate. </a></p><div class="supp"><p class="details"><div>Integrated <span class="highlight" style="background-color:">pathway</span> map shows the role of MeJA-induced genes involved in GA biosynthesis, primary metabolism, <span class="highlight" style="background-color:">signaling</span> regulation and transcriptional regulation. Dashed lines indicate the probable <span class="highlight" style="background-color:">pathway</span> involved in GA biosynthesis. Solid lines indicate the result supported by cDNA-AFLP and real time PCR. The ESTs from cDNA-AFLP results are indicated in stars.</div></p></div><div class="aux"><div class="resc">Ang Ren, et al. PLoS One. 2013;8(6):e65027.</div><p class="links"><a class="dblinks" href="/pubmed/23762280" ref="ordinalpos=2135&amp;ncbi_uid=4995156&amp;link_uid=23762280">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676390/" ref="ordinalpos=2135&amp;ncbi_uid=4995156&amp;link_uid=PMC3676390">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2136.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3318965/figure/F1/" ref="ordinalpos=2136&amp;ncbi_uid=5016473&amp;link_uid=PMC3318965" href="/pmc/articles/PMC3318965/"><img src="/pmc/articles/instance/3318965/bin/nihms344969f1.gif" src-large="/pmc/articles/instance/3318965/bin/nihms344969f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3318965/" ref="ordinalpos=2136&amp;ncbi_uid=5016473&amp;link_uid=PMC3318965" image-link="/pmc/articles/PMC3318965/figure/F1/" class="imagepopup">Figure 1. Model for progesterone regulation of cell proliferation in the mammary epithelium mediated by the RANK-RANKL paracrine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The Biology of Progesterone Receptor in the Normal Mammary gland and in Breast Cancer. </a></p><div class="supp"><p class="details"><div>The mechanism depicted is based on work with the mouse mammary gland as an experimental model. RANKL is a direct target of PR induced by progesterone (P4) and is then released to interact with the RANK receptor on either PR<sup>+</sup> or nearby PR<sup></sup> cells. In PR<sup></sup> cells, RANKL activates the downstream IKK/IkB/NFkB/cyclin D1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to stimulate proliferation. RANKL does not generate a sustained activation of the NFkB/cyclin D1 proliferative <span class="highlight" style="background-color:">pathway</span> in PR<sup>+</sup> cells due to the suppressive actions of TGF and/or other unknown mechanisms.</div></p></div><div class="aux"><div class="resc">Alison Obr, et al. Mol Cell Endocrinol. ;357(1-2):4-17.</div><p class="links"><a class="dblinks" href="/pubmed/22193050" ref="ordinalpos=2136&amp;ncbi_uid=5016473&amp;link_uid=22193050">Citation</a><a class="dblinks" href="/pmc/articles/PMC3318965/" ref="ordinalpos=2136&amp;ncbi_uid=5016473&amp;link_uid=PMC3318965">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2137.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4113511/figure/F1/" ref="ordinalpos=2137&amp;ncbi_uid=6423038&amp;link_uid=PMC4113511" href="/pmc/articles/PMC4113511/"><img src="/pmc/articles/instance/4113511/bin/nihms601822f1.gif" src-large="/pmc/articles/instance/4113511/bin/nihms601822f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4113511/" ref="ordinalpos=2137&amp;ncbi_uid=6423038&amp;link_uid=PMC4113511" image-link="/pmc/articles/PMC4113511/figure/F1/" class="imagepopup">Figure 1. The Sonic hedgehog (Shh) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: The Vitamin D Receptor: A Tumor Suppressor in Skin. </a></p><div class="supp"><p class="details"><div>In the absence of Shh, Ptch 1 suppresses <span class="highlight" style="background-color:">signaling</span> by smoothened (Smoh). Binding of Shh to Ptch 1 relieves this inhibition. Activation of Smoh leads to the activation and translocation of transcription factors of the Gli family into the nucleus, with subsequent changes in gene expression.</div></p></div><div class="aux"><div class="resc">Daniel David Bikle, et al. Discov Med. ;11(56):7-17.</div><p class="links"><a class="dblinks" href="/pubmed/21276406" ref="ordinalpos=2137&amp;ncbi_uid=6423038&amp;link_uid=21276406">Citation</a><a class="dblinks" href="/pmc/articles/PMC4113511/" ref="ordinalpos=2137&amp;ncbi_uid=6423038&amp;link_uid=PMC4113511">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2138.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2717566/figure/fig7/" ref="ordinalpos=2138&amp;ncbi_uid=1998485&amp;link_uid=PMC2717566" href="/pmc/articles/PMC2717566/"><img src="/pmc/articles/instance/2717566/bin/JBB2009-181282.007.gif" src-large="/pmc/articles/instance/2717566/bin/JBB2009-181282.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2717566/" ref="ordinalpos=2138&amp;ncbi_uid=1998485&amp;link_uid=PMC2717566" image-link="/pmc/articles/PMC2717566/figure/fig7/" class="imagepopup">Figure 7.  From: p,p?-DDE Induces Apoptosis of Rat Sertoli Cells via a FasL-Dependent <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Proposed model of  p,p-DDE-induced <span class="highlight" style="background-color:">signaling</span> pathways leading to apoptosis. ROS generation might play a critical role in the initiation of p,p-DDE-induced apoptosis of Sertoli cells through two mechanisms, one was the mitochondria-mediated <span class="highlight" style="background-color:">pathway</span> including elevation of ROS, decrease in <sub><i>m</i></sub> along with the cytochrome c release from mitochondria into the cytosol, and activation of the caspase-9 and -3 in our previous study; and the other was the elevation of ROS, which resulted in activation of NF-<i></i>B, expression of FasL and triggered FasL-dependent <span class="highlight" style="background-color:">pathway</span> (FasL/caspase-8/-3 <span class="highlight" style="background-color:">signaling</span> module) in our present study.</div></p></div><div class="aux"><div class="resc">Yuqin Shi, et al. J Biomed Biotechnol. 2009;2009:181282.</div><p class="links"><a class="dblinks" href="/pubmed/19644561" ref="ordinalpos=2138&amp;ncbi_uid=1998485&amp;link_uid=19644561">Citation</a><a class="dblinks" href="/pmc/articles/PMC2717566/" ref="ordinalpos=2138&amp;ncbi_uid=1998485&amp;link_uid=PMC2717566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2139.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3154848/figure/ppat-1002177-g010/" ref="ordinalpos=2139&amp;ncbi_uid=3161614&amp;link_uid=PMC3154848" href="/pmc/articles/PMC3154848/"><img src="/pmc/articles/instance/3154848/bin/ppat.1002177.g010.gif" src-large="/pmc/articles/instance/3154848/bin/ppat.1002177.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3154848/" ref="ordinalpos=2139&amp;ncbi_uid=3161614&amp;link_uid=PMC3154848" image-link="/pmc/articles/PMC3154848/figure/ppat-1002177-g010/" class="imagepopup">Figure 10. The proposed role of the UPR <span class="highlight" style="background-color:">pathway</span> as core stress response to various stresses in C. neoformans..  From: Unique Evolution of the UPR <span class="highlight" style="background-color:">Pathway</span> with a Novel bZIP Transcription Factor, Hxl1, for Controlling Pathogenicity of Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>The Ire1-Hxl1-mediated UPR system plays important roles in ER quality control and is essential during growth and cell survival for cryptococcal disease. Notably, Ire1 has a broad range of physiological roles in <i>C. neoformans</i> not only in Hxl1-dependent but also Hxl1-independent manners. Solid arrow indicates direct involvement of the UPR <span class="highlight" style="background-color:">pathway</span> (e.g., ER stress) and dotted arrow indicates the direct or indirect activation of the UPR <span class="highlight" style="background-color:">pathway</span> via potential cross-talk with other <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Seon Ah Cheon, et al. PLoS Pathog. 2011 August;7(8):e1002177.</div><p class="links"><a class="dblinks" href="/pubmed/21852949" ref="ordinalpos=2139&amp;ncbi_uid=3161614&amp;link_uid=21852949">Citation</a><a class="dblinks" href="/pmc/articles/PMC3154848/" ref="ordinalpos=2139&amp;ncbi_uid=3161614&amp;link_uid=PMC3154848">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2140.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3510826/figure/F8/" ref="ordinalpos=2140&amp;ncbi_uid=5531872&amp;link_uid=PMC3510826" href="/pmc/articles/PMC3510826/"><img src="/pmc/articles/instance/3510826/bin/zbc0511232250008.gif" src-large="/pmc/articles/instance/3510826/bin/zbc0511232250008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3510826/" ref="ordinalpos=2140&amp;ncbi_uid=5531872&amp;link_uid=PMC3510826" image-link="/pmc/articles/PMC3510826/figure/F8/" class="imagepopup">FIGURE 8. From: The Src Family Kinases and Protein Kinase C Synergize to Mediate Gq-dependent Platelet Activation. </a></p><div class="supp"><p class="details"><div><b>Important roles of SFKs in G protein-coupled receptor-induced platelet activation.</b> Although the G<sub>i</sub> <span class="highlight" style="background-color:">pathway</span> induces SFK activation by a direct interaction, G<sub>q</sub> coupling receptors such as PAR1 and PAR4 induce SFK activation downstream from the PLC/Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>. SFKs play important roles in G<sub>i</sub>- and G<sub>q</sub>-dependent activation of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, TXA<sub>2</sub> synthesis, platelet secretion, and activation. The Ca<sup>2+</sup>/SFKs/PI3K <span class="highlight" style="background-color:">pathway</span>, together with the DAG/PKC <span class="highlight" style="background-color:">pathway</span>, mediates G<sub>q</sub>-dependent platelet activation.</div></p></div><div class="aux"><div class="resc">Binggang Xiang, et al. J Biol Chem. 2012 November 30;287(49):41277-41287.</div><p class="links"><a class="dblinks" href="/pubmed/23066026" ref="ordinalpos=2140&amp;ncbi_uid=5531872&amp;link_uid=23066026">Citation</a><a class="dblinks" href="/pmc/articles/PMC3510826/" ref="ordinalpos=2140&amp;ncbi_uid=5531872&amp;link_uid=PMC3510826">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2141.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4012993/figure/pone-0091376-g006/" ref="ordinalpos=2141&amp;ncbi_uid=6121817&amp;link_uid=PMC4012993" href="/pmc/articles/PMC4012993/"><img src="/pmc/articles/instance/4012993/bin/pone.0091376.g006.gif" src-large="/pmc/articles/instance/4012993/bin/pone.0091376.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4012993/" ref="ordinalpos=2141&amp;ncbi_uid=6121817&amp;link_uid=PMC4012993" image-link="/pmc/articles/PMC4012993/figure/pone-0091376-g006/" class="imagepopup">Figure 6. Epithelial Scap deletion in the perinatal lung suppresses Wnt/-catenin <span class="highlight" style="background-color:">signaling</span>..  From: Epithelial SCAP/INSIG/SREBP <span class="highlight" style="background-color:">Signaling</span> Regulates Multiple Biological Processes during Perinatal Lung Maturation. </a></p><div class="supp"><p class="details"><div>(A) A schematic representation of mRNAs changed in the canonical Wnt/-catenin <span class="highlight" style="background-color:">pathway</span> in <i>Scap<sup>/</sup></i> perinatal lung (E17.5) is shown. Green nodes indicate mRNAs that were decreased in <i>Scap<sup>/</sup></i> lungs with respect to control lungs. Wnt <span class="highlight" style="background-color:">pathway</span> genes that were not changed in response to <i>Scap</i> deletion are depicted as empty nodes. (B) Heatmap demonstrates decreased expression of genes encoding a number of key components in the Wnt/-catenin <span class="highlight" style="background-color:">pathway</span> in <i>Scap<sup>/</sup></i> vs. control mice, n=3/group.</div></p></div><div class="aux"><div class="resc">James P. Bridges, et al. PLoS One. 2014;9(5):e91376.</div><p class="links"><a class="dblinks" href="/pubmed/24806461" ref="ordinalpos=2141&amp;ncbi_uid=6121817&amp;link_uid=24806461">Citation</a><a class="dblinks" href="/pmc/articles/PMC4012993/" ref="ordinalpos=2141&amp;ncbi_uid=6121817&amp;link_uid=PMC4012993">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2142.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4050444/figure/F7/" ref="ordinalpos=2142&amp;ncbi_uid=6235473&amp;link_uid=PMC4050444" href="/pmc/articles/PMC4050444/"><img src="/pmc/articles/instance/4050444/bin/2045-3701-4-24-7.gif" src-large="/pmc/articles/instance/4050444/bin/2045-3701-4-24-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4050444/" ref="ordinalpos=2142&amp;ncbi_uid=6235473&amp;link_uid=PMC4050444" image-link="/pmc/articles/PMC4050444/figure/F7/" class="imagepopup">Figure 7.  From: Adipose-derived stem cells cooperate with fractional carbon dioxide laser in antagonizing photoaging: a potential role of Wnt and ?-catenin <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>Effect of ADSC-CM on the expression of wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins and photoaging relevant proteins.</b> ADSC-CM significantly increased the protein expression of wnt3a and -catenin from 24 h, expression of TGF-2 and COLI were upregulated as well.</div></p></div><div class="aux"><div class="resc">Xiao Xu, et al. Cell Biosci. 2014;4:24-24.</div><p class="links"><a class="dblinks" href="/pubmed/24917925" ref="ordinalpos=2142&amp;ncbi_uid=6235473&amp;link_uid=24917925">Citation</a><a class="dblinks" href="/pmc/articles/PMC4050444/" ref="ordinalpos=2142&amp;ncbi_uid=6235473&amp;link_uid=PMC4050444">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2143.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3191566/figure/F6/" ref="ordinalpos=2143&amp;ncbi_uid=3487584&amp;link_uid=PMC3191566" href="/pmc/articles/PMC3191566/"><img src="/pmc/articles/instance/3191566/bin/zh00101167240006.gif" src-large="/pmc/articles/instance/3191566/bin/zh00101167240006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3191566/" ref="ordinalpos=2143&amp;ncbi_uid=3487584&amp;link_uid=PMC3191566" image-link="/pmc/articles/PMC3191566/figure/F6/" class="imagepopup">Fig. 6. From: Implication of microRNAs in atrial natriuretic peptide and nitric oxide <span class="highlight" style="background-color:">signaling</span> in vascular smooth muscle cells. </a></p><div class="supp"><p class="details"><div>Molecular <span class="highlight" style="background-color:">pathway</span> by which miR-21 regulates HVSMC contraction. ANP and SNAP via cGMP/cGK <span class="highlight" style="background-color:">signaling</span> reduce the expression of miR-21, which has a feedback loop inhibitory effect on cGK-1. miR-21 targets (CFL-2, M-RIP, and cGK-1) and their effects on HVSMC contraction/relaxation are depicted. NO, nitric oxide.</div></p></div><div class="aux"><div class="resc">Kumar U. Kotlo, et al. Am J Physiol Cell Physiol. 2011 October;301(4):C929-C937.</div><p class="links"><a class="dblinks" href="/pubmed/21734186" ref="ordinalpos=2143&amp;ncbi_uid=3487584&amp;link_uid=21734186">Citation</a><a class="dblinks" href="/pmc/articles/PMC3191566/" ref="ordinalpos=2143&amp;ncbi_uid=3487584&amp;link_uid=PMC3191566">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2144.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2994985/figure/F1/" ref="ordinalpos=2144&amp;ncbi_uid=3458753&amp;link_uid=PMC2994985" href="/pmc/articles/PMC2994985/"><img src="/pmc/articles/instance/2994985/bin/nihms246185f1.gif" src-large="/pmc/articles/instance/2994985/bin/nihms246185f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2994985/" ref="ordinalpos=2144&amp;ncbi_uid=3458753&amp;link_uid=PMC2994985" image-link="/pmc/articles/PMC2994985/figure/F1/" class="imagepopup">Figure 1.  From: Thirsty plants and beyond: Structural mechanisms of abscisic acid perception and <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Schematic presentation of the core ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Active components are shown in green, inactive ones in red, and ABA as yellow ball. Cylinders depict ion channels that regulate stomatal closure. TFs: transcription factors, P:phosphate.</div></p></div><div class="aux"><div class="resc">Karsten Melcher, et al. Curr Opin Struct Biol. ;20(6):722-729.</div><p class="links"><a class="dblinks" href="/pubmed/20951573" ref="ordinalpos=2144&amp;ncbi_uid=3458753&amp;link_uid=20951573">Citation</a><a class="dblinks" href="/pmc/articles/PMC2994985/" ref="ordinalpos=2144&amp;ncbi_uid=3458753&amp;link_uid=PMC2994985">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2145.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2857176/figure/A001032F3/" ref="ordinalpos=2145&amp;ncbi_uid=2265910&amp;link_uid=PMC2857176" href="/pmc/articles/PMC2857176/"><img src="/pmc/articles/instance/2857176/bin/cshperspect-PFT-001032_F3.gif" src-large="/pmc/articles/instance/2857176/bin/cshperspect-PFT-001032_F3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2857176/" ref="ordinalpos=2145&amp;ncbi_uid=2265910&amp;link_uid=PMC2857176" image-link="/pmc/articles/PMC2857176/figure/A001032F3/" class="imagepopup">Figure 3. From: Single-nucleotide Polymorphisms in the p53 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Map of selected p53 <span class="highlight" style="background-color:">pathway</span>-associated genes that harbor known or potential functional SNPs (octagon-shaped boxes) and their closely interacting genes (rectangular boxes). The map depicts genes involved in the core regulation of p53 (light red boxes), the DNA-damage response (yellow), the extrinsic apoptotic response (green), the cell-cycle (turquoise) and oncogenic <span class="highlight" style="background-color:">signaling</span> (dark blue). The arrows () show downstream activation and the inhibitory interactions are delineated by T-shaped () ends. A network map representing the p53 <span class="highlight" style="background-color:">pathway</span> is illustrated in the distant background.</div></p></div><div class="aux"><div class="resc">Lukasz F. Grochola, et al. Cold Spring Harb Perspect Biol. 2010 May;2(5):a001032.</div><p class="links"><a class="dblinks" href="/pubmed/20452958" ref="ordinalpos=2145&amp;ncbi_uid=2265910&amp;link_uid=20452958">Citation</a><a class="dblinks" href="/pmc/articles/PMC2857176/" ref="ordinalpos=2145&amp;ncbi_uid=2265910&amp;link_uid=PMC2857176">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2146.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4350097/figure/f6/" ref="ordinalpos=2146&amp;ncbi_uid=7273637&amp;link_uid=PMC4350097" href="/pmc/articles/PMC4350097/"><img src="/pmc/articles/instance/4350097/bin/srep08792-f6.gif" src-large="/pmc/articles/instance/4350097/bin/srep08792-f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4350097/" ref="ordinalpos=2146&amp;ncbi_uid=7273637&amp;link_uid=PMC4350097" image-link="/pmc/articles/PMC4350097/figure/f6/" class="imagepopup">Figure 6. Proposed working model for the CHS3-mediated defense <span class="highlight" style="background-color:">pathway</span>..  From: Autoimmunity conferred by chs3-2D relies on CSA1, its adjacent TNL-encoding neighbour. </a></p><div class="supp"><p class="details"><div>The LIM domain was proposed to repress CHS3 activities in the absence of pathogens. SGT1b, which can form a chaperone complex with HSP90 and RAR1, is probably required for the proper assembly of CHS3 activation complex. CHS3 activation relies on its TNL neighbour CSA1, and this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> appears to primarily rely on SAG101 and EDS1-90.</div></p></div><div class="aux"><div class="resc">Fang Xu, et al. Sci Rep. 2015;5:8792.</div><p class="links"><a class="dblinks" href="/pubmed/25740259" ref="ordinalpos=2146&amp;ncbi_uid=7273637&amp;link_uid=25740259">Citation</a><a class="dblinks" href="/pmc/articles/PMC4350097/" ref="ordinalpos=2146&amp;ncbi_uid=7273637&amp;link_uid=PMC4350097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2147.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC99013/figure/F8/" ref="ordinalpos=2147&amp;ncbi_uid=52569&amp;link_uid=PMC99013" href="/pmc/articles/PMC99013/"><img src="/pmc/articles/instance/99013/bin/mr0400030008.gif" src-large="/pmc/articles/instance/99013/bin/mr0400030008.jpg" alt="FIG. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC99013/" ref="ordinalpos=2147&amp;ncbi_uid=52569&amp;link_uid=PMC99013" image-link="/pmc/articles/PMC99013/figure/F8/" class="imagepopup">FIG. 8.  From: Signal Transduction Cascades Regulating Fungal  Development and Virulence. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> cascades regulating virulence of <i>C. albicans</i>. Two parallel <span class="highlight" style="background-color:">signaling</span> cascades involving a MAP kinase and a cAMP-PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulate filamentation and virulence of this human fungal pathogen. AC, adenylycyclase.</div></p></div><div class="aux"><div class="resc">Klaus B. Lengeler, et al. Microbiol Mol Biol Rev. 2000 December;64(4):746-785.</div><p class="links"><a class="dblinks" href="/pubmed/11104818" ref="ordinalpos=2147&amp;ncbi_uid=52569&amp;link_uid=11104818">Citation</a><a class="dblinks" href="/pmc/articles/PMC99013/" ref="ordinalpos=2147&amp;ncbi_uid=52569&amp;link_uid=PMC99013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2148.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3813610/figure/pone-0078344-g004/" ref="ordinalpos=2148&amp;ncbi_uid=5465479&amp;link_uid=PMC3813610" href="/pmc/articles/PMC3813610/"><img src="/pmc/articles/instance/3813610/bin/pone.0078344.g004.gif" src-large="/pmc/articles/instance/3813610/bin/pone.0078344.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3813610/" ref="ordinalpos=2148&amp;ncbi_uid=5465479&amp;link_uid=PMC3813610" image-link="/pmc/articles/PMC3813610/figure/pone-0078344-g004/" class="imagepopup">Figure 4. Western blot analysis of VEGFR2 downstream <span class="highlight" style="background-color:">signaling</span> transduction..  From: MiR-200c Increases the Radiosensitivity of Non-Small-Cell Lung Cancer Cell Line A549 by Targeting VEGF-VEGFR2 <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Ectopic expression of miR-200c inhibited phosphorylation of Akt and ERK1/2, inhibition of miR-200c increased activation of Akt and ERK1/2 <span class="highlight" style="background-color:">pathway</span>. Each experiment was repeated three times.</div></p></div><div class="aux"><div class="resc">Liangliang Shi, et al. PLoS One. 2013;8(10):e78344.</div><p class="links"><a class="dblinks" href="/pubmed/24205206" ref="ordinalpos=2148&amp;ncbi_uid=5465479&amp;link_uid=24205206">Citation</a><a class="dblinks" href="/pmc/articles/PMC3813610/" ref="ordinalpos=2148&amp;ncbi_uid=5465479&amp;link_uid=PMC3813610">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2149.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4034118/figure/F6/" ref="ordinalpos=2149&amp;ncbi_uid=6168730&amp;link_uid=PMC4034118" href="/pmc/articles/PMC4034118/"><img src="/pmc/articles/instance/4034118/bin/nihms581332f6.gif" src-large="/pmc/articles/instance/4034118/bin/nihms581332f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4034118/" ref="ordinalpos=2149&amp;ncbi_uid=6168730&amp;link_uid=PMC4034118" image-link="/pmc/articles/PMC4034118/figure/F6/" class="imagepopup">Figure 6.  From: Functional screen analysis reveals miR-26b and miR-128 as central regulators of pituitary somatomammotrophic tumor growth through activation the PTEN-AKT <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>MicroRNA-gene networks regulating pituitary oncogenesis. (<b>a</b>) Correlation (r represents the correlation coefficient) in the expression levels of miR-26b, miR-128, PTEN and BMI1 in human pituitary tumors. (<b>b</b>) Schematic of miR-26b and miR-128 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pituitary oncogenesis.</div></p></div><div class="aux"><div class="resc">Tiziana Palumbo, et al. Oncogene. ;32(13):1651-1659.</div><p class="links"><a class="dblinks" href="/pubmed/22614013" ref="ordinalpos=2149&amp;ncbi_uid=6168730&amp;link_uid=22614013">Citation</a><a class="dblinks" href="/pmc/articles/PMC4034118/" ref="ordinalpos=2149&amp;ncbi_uid=6168730&amp;link_uid=PMC4034118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2150.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2905627/figure/F7/" ref="ordinalpos=2150&amp;ncbi_uid=2361160&amp;link_uid=PMC2905627" href="/pmc/articles/PMC2905627/"><img src="/pmc/articles/instance/2905627/bin/nihms217549f7.gif" src-large="/pmc/articles/instance/2905627/bin/nihms217549f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2905627/" ref="ordinalpos=2150&amp;ncbi_uid=2361160&amp;link_uid=PMC2905627" image-link="/pmc/articles/PMC2905627/figure/F7/" class="imagepopup">Figure 7.  From: GDNF Induces CREB-1, ATF-1, and CREM-1 Phosphorylation and Up-regulates c-fos Transcription via the Ras/ERK1/2 <span class="highlight" style="background-color:">Pathway</span> to Promote Mouse Spermatogonial Stem Cell Proliferation. </a></p><div class="supp"><p class="details"><div>The schematic diagram demonstrates intracellular <span class="highlight" style="background-color:">signaling</span> events in the Ras/ERK1/2 <span class="highlight" style="background-color:">pathway</span> as well as the upstream and downstream cascades activated by GDNF in the C18-4 cells. P indicates phosphorylate, and A denotes activate.</div></p></div><div class="aux"><div class="resc">Zuping He, et al. Stem Cells. ;26(1):266-278.</div><p class="links"><a class="dblinks" href="/pubmed/17962702" ref="ordinalpos=2150&amp;ncbi_uid=2361160&amp;link_uid=17962702">Citation</a><a class="dblinks" href="/pmc/articles/PMC2905627/" ref="ordinalpos=2150&amp;ncbi_uid=2361160&amp;link_uid=PMC2905627">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2151.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC317251/figure/F6/" ref="ordinalpos=2151&amp;ncbi_uid=417735&amp;link_uid=PMC317251" href="/pmc/articles/PMC317251/"><img src="/pmc/articles/instance/317251/bin/gad.11f6.gif" src-large="/pmc/articles/instance/317251/bin/gad.11f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC317251/" ref="ordinalpos=2151&amp;ncbi_uid=417735&amp;link_uid=PMC317251" image-link="/pmc/articles/PMC317251/figure/F6/" class="imagepopup">Figure 6.  From: Nuclear events in ethylene <span class="highlight" style="background-color:">signaling</span>: a transcriptional cascade mediated by ETHYLENE-INSENSITIVE3 and ETHYLENE-RESPONSE-FACTOR1. </a></p><div class="supp"><p class="details"><div>ERF1 acts downstream of EIN2 and EIN3 in the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Transgenic plants overexpressing <i>ERF1</i> in wild-type (Col-0), <i>ein2, ein3-1</i>, and <i>ein3-3</i> mutant backgrounds grown in continuous flowthrough cambers with hydrocarbon-free air for 5 weeks. Untransformed wild-type, <i>ctr1-1</i>, and <i>EIN3</i>-overexpressing plants are shown for comparison.</div></p></div><div class="aux"><div class="resc">Roberto Solano, et al. Genes Dev. 1998 December 1;12(23):3703-3714.</div><p class="links"><a class="dblinks" href="/pubmed/9851977" ref="ordinalpos=2151&amp;ncbi_uid=417735&amp;link_uid=9851977">Citation</a><a class="dblinks" href="/pmc/articles/PMC317251/" ref="ordinalpos=2151&amp;ncbi_uid=417735&amp;link_uid=PMC317251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2152.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1356578/figure/fig10/" ref="ordinalpos=2152&amp;ncbi_uid=589373&amp;link_uid=PMC1356578" href="/pmc/articles/PMC1356578/"><img src="/pmc/articles/instance/1356578/bin/zmk0020674940010.gif" src-large="/pmc/articles/instance/1356578/bin/zmk0020674940010.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1356578/" ref="ordinalpos=2152&amp;ncbi_uid=589373&amp;link_uid=PMC1356578" image-link="/pmc/articles/PMC1356578/figure/fig10/" class="imagepopup">Figure 10. From: G Protein-coupled Receptor Gpr4 Senses Amino Acids and Activates the cAMP-PKA <span class="highlight" style="background-color:">Pathway</span> in Cryptococcus neoformans. </a></p><div class="supp"><p class="details"><div>Proposed model for the Gpr4, Gpa1-activated cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. neoformans</i>. Two signal inputs to the cAMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are proposed in this model and involve nutrient transport and sensing. Fermentable carbon sources, such as glucose, are transported into the cell via hexose transporters and phosphorylated by hexose kinases to produce glucose-6-phosphate (Glu6P), which in turn regulates Cac1 or Gpa1 activation to engage cAMP <span class="highlight" style="background-color:">signaling</span>. Glucose may also be sensed by an unknown receptor to activate the cAMP <span class="highlight" style="background-color:">pathway</span>. Gpr4 may also play a role in low glucose concentration sensing based on the cAMP assay results. In this model, Gpr4 is an amino acid sensor and interacts with Gpa1 to regulate downstream elements of the cAMP <span class="highlight" style="background-color:">pathway</span>. This model does not exclude that other proteins such as methionine permease may participate in ligand sensing via direct or indirect interactions with Gpr4. Hxt, hexose transporters; Hxk, hexose kinases; Glu6P, glucose-6-phosphate.</div></p></div><div class="aux"><div class="resc">Chaoyang Xue, et al. Mol Biol Cell. 2006 February;17(2):667-679.</div><p class="links"><a class="dblinks" href="/pubmed/16291861" ref="ordinalpos=2152&amp;ncbi_uid=589373&amp;link_uid=16291861">Citation</a><a class="dblinks" href="/pmc/articles/PMC1356578/" ref="ordinalpos=2152&amp;ncbi_uid=589373&amp;link_uid=PMC1356578">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2153.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3751848/figure/pone-0074420-g002/" ref="ordinalpos=2153&amp;ncbi_uid=5268802&amp;link_uid=PMC3751848" href="/pmc/articles/PMC3751848/"><img src="/pmc/articles/instance/3751848/bin/pone.0074420.g002.gif" src-large="/pmc/articles/instance/3751848/bin/pone.0074420.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3751848/" ref="ordinalpos=2153&amp;ncbi_uid=5268802&amp;link_uid=PMC3751848" image-link="/pmc/articles/PMC3751848/figure/pone-0074420-g002/" class="imagepopup">Figure 2. Metformin activates AMPK, Akt, and UPR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> proteins in ALL primary and cell line models..  From: Metformin Induces Apoptosis through AMPK-Dependent Inhibition of UPR <span class="highlight" style="background-color:">Signaling</span> in ALL Lymphoblasts. </a></p><div class="supp"><p class="details"><div><b>A</b>) Western blot analysis of proteins associated with AMPK, and Akt/mTOR <span class="highlight" style="background-color:">signaling</span> pathways in CCRF-CEM and NALM6 cells treated with metformin (MET, 2.5 and 5.0 mM) for 48 h. The density value of each band was normalized to -actin level and expressed relative to control (shown as fold induction). <b>B</b>) Immunoblotting of UPR <span class="highlight" style="background-color:">signaling</span> factors in CCRF-CEM and NALM6 cells treated with metformin (MET, 2.5 and 5.0 mM) for 48 h. <b>C</b>) Western blots of AMPK, Akt/mTOR and UPR <span class="highlight" style="background-color:">signaling</span> proteins in representative sample of primary T- and Bp-ALL cells treated with metformin (MET, 10 mM) for 24 h. <b>D</b>) Western blot analysis of GRP78 expression in CCRF-CEM and NALM6 cells treated with metformin (MET, 10mM) and tunicamycin (TUN, 2.5 g/ml for NALM6; 5.0 g/ml for CCRF-CEM), either alone or in combination for 48 h.</div></p></div><div class="aux"><div class="resc">Gilles M. Leclerc, et al. PLoS One. 2013;8(8):e74420.</div><p class="links"><a class="dblinks" href="/pubmed/24009772" ref="ordinalpos=2153&amp;ncbi_uid=5268802&amp;link_uid=24009772">Citation</a><a class="dblinks" href="/pmc/articles/PMC3751848/" ref="ordinalpos=2153&amp;ncbi_uid=5268802&amp;link_uid=PMC3751848">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2154.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872207/figure/f4/" ref="ordinalpos=2154&amp;ncbi_uid=2370743&amp;link_uid=PMC2872207" href="/pmc/articles/PMC2872207/"><img src="/pmc/articles/instance/2872207/bin/fig-4.gif" src-large="/pmc/articles/instance/2872207/bin/fig-4.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872207/" ref="ordinalpos=2154&amp;ncbi_uid=2370743&amp;link_uid=PMC2872207" image-link="/pmc/articles/PMC2872207/figure/f4/" class="imagepopup">FIG. 4. From: Proinflammatory Stem Cell <span class="highlight" style="background-color:">Signaling</span> in Cardiac Ischemia. </a></p><div class="supp"><p class="details"><div><b>The Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Growth factor <span class="highlight" style="background-color:">signaling</span> results in activation of PI3K and conversion of PIP2 to PIP3. Proteins containing a PH domain are recruited to the membrane and participate in phosphorylation reactions. PTEN, PP2A, and Rheb-GDP negatively regulate the <span class="highlight" style="background-color:">pathway</span>. PIP2, phosphatidylinositol (3,4)-bisphosphate; PIP3, phosphatidylinositol (3,4,5)-trisphosphate; PH, pleckstrin homology domain; HM, C-terminal hydrophobic motif; 4E-BP, inhibitory eIF4E-binding proteins; S6K, ribosomal S6 kinases; PP2A, protein phosphatase 2A.</div></p></div><div class="aux"><div class="resc">Jeremy L. Herrmann, et al. Antioxid Redox Signal. 2009 August;11(8):1883-1896.</div><p class="links"><a class="dblinks" href="/pubmed/19187005" ref="ordinalpos=2154&amp;ncbi_uid=2370743&amp;link_uid=19187005">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872207/" ref="ordinalpos=2154&amp;ncbi_uid=2370743&amp;link_uid=PMC2872207">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2155.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4120774/figure/F6/" ref="ordinalpos=2155&amp;ncbi_uid=6460826&amp;link_uid=PMC4120774" href="/pmc/articles/PMC4120774/"><img src="/pmc/articles/instance/4120774/bin/nihms-314392-f0006.gif" src-large="/pmc/articles/instance/4120774/bin/nihms-314392-f0006.jpg" alt="Figure - 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4120774/" ref="ordinalpos=2155&amp;ncbi_uid=6460826&amp;link_uid=PMC4120774" image-link="/pmc/articles/PMC4120774/figure/F6/" class="imagepopup">Figure - 6. Molecular targets and cell <span class="highlight" style="background-color:">signaling</span> pathways altered by DIM.  From: ATTENUATION OF MULTI-TARGETED PROLIFERATION-LINKED <span class="highlight" style="background-color:">SIGNALING</span> BY 3,3?-DIINDOLYLMETHANE (DIM): FROM BENCH TO CLINIC. </a></p><div class="supp"><p class="details"><div>Multiple growth factor receptors are activated at the cell surface during cancer. Activation of these receptors activates several downstream <span class="highlight" style="background-color:">signaling</span> pathways. Among these pathways the Ras-MAPK, the P13K-Akt, mTOR and NF-B pathways are of significance and targets of DIM. By dephosphorylating these molecules, DIM modulates downstream <span class="highlight" style="background-color:">signaling</span> pathways impinging on proliferation, angiogenesis and apoptosis. NF-kB <span class="highlight" style="background-color:">pathway</span> is inactive as a result of the binding of p50 and p65 to IB. When IB is phosphorylated by IKKs and degraded, p50 and p65 are set free and are translocated into the nucleus to activate a specific set of genes. This <span class="highlight" style="background-color:">pathway</span> has been shown to be inhibited both in vitro and in vivo by DIM. Activation of AP-1 is primarily mediated by MAPK, especially through the phosphorylation by c-Jun N-terminal kinase (JNK), which leads to translocation of AP-1 and subsequent transcriptional activation of target genes</div></p></div><div class="aux"><div class="resc">Sanjeev Banerjee, et al. Mutat Res. ;728(0):47-66.</div><p class="links"><a class="dblinks" href="/pubmed/21703360" ref="ordinalpos=2155&amp;ncbi_uid=6460826&amp;link_uid=21703360">Citation</a><a class="dblinks" href="/pmc/articles/PMC4120774/" ref="ordinalpos=2155&amp;ncbi_uid=6460826&amp;link_uid=PMC4120774">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2156.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2754063/figure/F8/" ref="ordinalpos=2156&amp;ncbi_uid=2128657&amp;link_uid=PMC2754063" href="/pmc/articles/PMC2754063/"><img src="/pmc/articles/instance/2754063/bin/nihms125511f8.gif" src-large="/pmc/articles/instance/2754063/bin/nihms125511f8.jpg" alt="FIGURE 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2754063/" ref="ordinalpos=2156&amp;ncbi_uid=2128657&amp;link_uid=PMC2754063" image-link="/pmc/articles/PMC2754063/figure/F8/" class="imagepopup">FIGURE 8. This model depicts the putative <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by which 4 integrin regulates the organizational state of laminin-332 in the extracellular matrix in motile versus stationary keratinocytes.  From: The Slingshot Family of Phosphatases Mediates Rac1 Regulation of Cofilin Phosphorylation, Laminin-332 Organization, and Motility Behavior of Keratinocytes. </a></p><div class="supp"><p class="details"><div>Our data indicate that it does so through <span class="highlight" style="background-color:">signaling</span> via Rac1, SSH proteins, 14-3-3 proteins, cofilin, and the actin cytoskeleton.</div></p></div><div class="aux"><div class="resc">Kristina Kligys, et al. J Biol Chem. ;282(44):32520-32528.</div><p class="links"><a class="dblinks" href="/pubmed/17848544" ref="ordinalpos=2156&amp;ncbi_uid=2128657&amp;link_uid=17848544">Citation</a><a class="dblinks" href="/pmc/articles/PMC2754063/" ref="ordinalpos=2156&amp;ncbi_uid=2128657&amp;link_uid=PMC2754063">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2157.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4372037/figure/Fig5/" ref="ordinalpos=2157&amp;ncbi_uid=7320829&amp;link_uid=PMC4372037" href="/pmc/articles/PMC4372037/"><img src="/pmc/articles/instance/4372037/bin/12864_2015_1392_Fig5_HTML.gif" src-large="/pmc/articles/instance/4372037/bin/12864_2015_1392_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4372037/" ref="ordinalpos=2157&amp;ncbi_uid=7320829&amp;link_uid=PMC4372037" image-link="/pmc/articles/PMC4372037/figure/Fig5/" class="imagepopup">Figure 5.  From: NMDA receptor activation upstream of methyl farnesoate <span class="highlight" style="background-color:">signaling</span> for short day-induced male offspring production in the water flea, Daphnia pulex. </a></p><div class="supp"><p class="details"><div> <b>A model for the function of NMDA receptors and protein kinase C <span class="highlight" style="background-color:">signaling</span> in the JH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for male production of</b> <b><i>D. pulex</i></b> <b>in response to the short-day, and MF-treated conditions.</b> iGluRs: ionotropic glutamate receptors, PKC: protein kinase C.</div></p></div><div class="aux"><div class="resc">Kenji Toyota, et al. BMC Genomics. 2015;16(1):186.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2157&amp;ncbi_uid=7320829&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4372037/" ref="ordinalpos=2157&amp;ncbi_uid=7320829&amp;link_uid=PMC4372037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2158.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193496/figure/fig1/" ref="ordinalpos=2158&amp;ncbi_uid=4090101&amp;link_uid=PMC3193496" href="/pmc/articles/PMC3193496/"><img src="/pmc/articles/instance/3193496/bin/cr201186f1.gif" src-large="/pmc/articles/instance/3193496/bin/cr201186f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193496/" ref="ordinalpos=2158&amp;ncbi_uid=4090101&amp;link_uid=PMC3193496" image-link="/pmc/articles/PMC3193496/figure/fig1/" class="imagepopup">Figure 1.  From: Wnt <span class="highlight" style="background-color:">signaling</span> through T-cell factor phosphorylation. </a></p><div class="supp"><p class="details"><div>Two conserved branches of the canonical Wnt <span class="highlight" style="background-color:">pathway</span>. In vertebrates, Wnt <span class="highlight" style="background-color:">signaling</span> acts to prevent -catenin degradation and promote its ability to activate transcription. SYS-1 is a functional substitute of -catenin in the early <i>C. elegans</i> embryo. The other conserved <span class="highlight" style="background-color:">signaling</span> branch is to inhibit TCF3 (vertebrate embryos) or POP-1 (<i>C. elegans</i>) repressive activity by phosphorylation.</div></p></div><div class="aux"><div class="resc">Sergei Y Sokol. Cell Res. 2011 July;21(7):1002-1012.</div><p class="links"><a class="dblinks" href="/pubmed/21606952" ref="ordinalpos=2158&amp;ncbi_uid=4090101&amp;link_uid=21606952">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193496/" ref="ordinalpos=2158&amp;ncbi_uid=4090101&amp;link_uid=PMC3193496">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2159.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2840073/figure/F4/" ref="ordinalpos=2159&amp;ncbi_uid=2825263&amp;link_uid=PMC2840073" href="/pmc/articles/PMC2840073/"><img src="/pmc/articles/instance/2840073/bin/nihms172763f4.gif" src-large="/pmc/articles/instance/2840073/bin/nihms172763f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2840073/" ref="ordinalpos=2159&amp;ncbi_uid=2825263&amp;link_uid=PMC2840073" image-link="/pmc/articles/PMC2840073/figure/F4/" class="imagepopup">Figure 4. CD44 functions upstream of the Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and plays an essential role in attenuating the apoptotic response of glioma cells induced by oxidative stress.  From: CD44 attenuates activation of the Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and is a prime therapeutic target for glioblastoma. </a></p><div class="supp"><p class="details"><div>Western blots were performed using the cell lysates derived from U87MG-NT cells infected with a mixture lentiviruses carrying non-targeting shRNAs (A and C), and U87MGshRNA-CD44 cells infected with a mixture of lentiviruses carrying CD44-specific shRNAs (B and D). These cells were treated with or without 60m H<sub>2</sub>O<sub>2</sub> for 30min, 2h, 24h, 48h, and 72h as indicated in the panels. 100g of total protein were loaded in each lane. Actin was included as an internal control for protein loading. The antibodies used against the different <span class="highlight" style="background-color:">signaling</span> mediators are indicated in the panels.</div></p></div><div class="aux"><div class="resc">Yin Xu, et al. Cancer Res. ;70(6):2455-2464.</div><p class="links"><a class="dblinks" href="/pubmed/20197461" ref="ordinalpos=2159&amp;ncbi_uid=2825263&amp;link_uid=20197461">Citation</a><a class="dblinks" href="/pmc/articles/PMC2840073/" ref="ordinalpos=2159&amp;ncbi_uid=2825263&amp;link_uid=PMC2840073">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2160.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3485417/figure/F1/" ref="ordinalpos=2160&amp;ncbi_uid=5548600&amp;link_uid=PMC3485417" href="/pmc/articles/PMC3485417/"><img src="/pmc/articles/instance/3485417/bin/nihms-393074-f0001.gif" src-large="/pmc/articles/instance/3485417/bin/nihms-393074-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3485417/" ref="ordinalpos=2160&amp;ncbi_uid=5548600&amp;link_uid=PMC3485417" image-link="/pmc/articles/PMC3485417/figure/F1/" class="imagepopup">Figure 1.  From: HLA Class I: an Unexpected Role in Integrin ?4 <span class="highlight" style="background-color:">Signaling</span> in Endothelial Cells. </a></p><div class="supp"><p class="details"><div>HLA-I antibodies stimulate intracellular signals via integrin 4 in endothelial cells and smooth muscle cells in the setting of transplant atherosclerosis. Crosslinking of HLA-I with antibodies increases its association with integrin 4, which in turn activates the Src/FAK <span class="highlight" style="background-color:">signaling</span> pathways. Phosphorylation of Src/FAK causes the activation of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> which stimulates the formation of mTORC1. mTORC1 initiates protein synthesis and cell proliferation via 4EBP1 and S6K. Phosphorylation of Src/FAK also stimulates mTORC2 <span class="highlight" style="background-color:">signaling</span>. Activation of mTORC2 promotes cell survival or proliferation though the AKT and ERK <span class="highlight" style="background-color:">signaling</span> pathways, respectively.</div></p></div><div class="aux"><div class="resc">Xiaohai Zhang, et al. Hum Immunol. 2012 December;73(12):1239-1244.</div><p class="links"><a class="dblinks" href="/pubmed/22789625" ref="ordinalpos=2160&amp;ncbi_uid=5548600&amp;link_uid=22789625">Citation</a><a class="dblinks" href="/pmc/articles/PMC3485417/" ref="ordinalpos=2160&amp;ncbi_uid=5548600&amp;link_uid=PMC3485417">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2161.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3372268/figure/F11/" ref="ordinalpos=2161&amp;ncbi_uid=4416225&amp;link_uid=PMC3372268" href="/pmc/articles/PMC3372268/"><img src="/pmc/articles/instance/3372268/bin/zmb9991095120011.gif" src-large="/pmc/articles/instance/3372268/bin/zmb9991095120011.jpg" alt="Fig 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3372268/" ref="ordinalpos=2161&amp;ncbi_uid=4416225&amp;link_uid=PMC3372268" image-link="/pmc/articles/PMC3372268/figure/F11/" class="imagepopup">Fig 11.  From: MicroRNA-155 Is Required for Mycobacterium bovis BCG-Mediated Apoptosis of Macrophages. </a></p><div class="supp"><p class="details"><div>Model depicting miR-155-mediated modulation of macrophage apoptosis. M. bovis BCG induces the upregulation of miR-155 by triggering the TLR2/PI3K <span class="highlight" style="background-color:">pathway</span>. PKC and RAS-RAF1-MAPKs play vital roles in M. bovis BCG-induced miR-155 expression through transcriptional factors NF-B and c-ETS. Induced miR-155 targets PKI-, a negative regulator of PKA <span class="highlight" style="background-color:">signaling</span>, resulting in the robust activation of the PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Overall, miR-155 induces the upregulation of proapoptotic genes by modulating PKA <span class="highlight" style="background-color:">signaling</span>, resulting in macrophage apoptosis.</div></p></div><div class="aux"><div class="resc">Devram Sampat Ghorpade, et al. Mol Cell Biol. 2012 June;32(12):2239-2253.</div><p class="links"><a class="dblinks" href="/pubmed/22473996" ref="ordinalpos=2161&amp;ncbi_uid=4416225&amp;link_uid=22473996">Citation</a><a class="dblinks" href="/pmc/articles/PMC3372268/" ref="ordinalpos=2161&amp;ncbi_uid=4416225&amp;link_uid=PMC3372268">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2162.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2323333/figure/f1-mol14_7p0517/" ref="ordinalpos=2162&amp;ncbi_uid=1451658&amp;link_uid=PMC2323333" href="/pmc/articles/PMC2323333/"><img src="/pmc/articles/instance/2464322/bin/mol14_7p0517f1.gif" src-large="/pmc/articles/instance/2464322/bin/mol14_7p0517f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2323333/" ref="ordinalpos=2162&amp;ncbi_uid=1451658&amp;link_uid=PMC2323333" image-link="/pmc/articles/PMC2323333/figure/f1-mol14_7p0517/" class="imagepopup">Figure 1.  From: Estrogen, NF?B, and the Heat Shock Response. </a></p><div class="supp"><p class="details"><div>Genomic and nongenomic E2 <span class="highlight" style="background-color:">signaling</span>. In the classical genomic <span class="highlight" style="background-color:">pathway</span>, E2 binds to cytosolic ER, and translocates to the nucleus where it binds directly to the ERE or binds to other transcription factors (TF) tethered to response elements and induces gene transcription. In addition, E2 is able to exert rapid cellular effects through several nongenomic mechanisms. E2 can bind to plasma membrane bound ER and directly activate <span class="highlight" style="background-color:">signaling</span> cascades, such as the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. Alternatively, upon binding E2, ER can recruit adaptor proteins, which activate <span class="highlight" style="background-color:">signaling</span> cascades. Lastly, E2 can bind non-membrane bound receptors and directly activate <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">James P Stice, et al. Mol Med. 2008 Jul-Aug;14(7-8):517-527.</div><p class="links"><a class="dblinks" href="/pubmed/18431462" ref="ordinalpos=2162&amp;ncbi_uid=1451658&amp;link_uid=18431462">Citation</a><a class="dblinks" href="/pmc/articles/PMC2323333/" ref="ordinalpos=2162&amp;ncbi_uid=1451658&amp;link_uid=PMC2323333">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2163.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3163686/figure/pone-0023989-g002/" ref="ordinalpos=2163&amp;ncbi_uid=3208672&amp;link_uid=PMC3163686" href="/pmc/articles/PMC3163686/"><img src="/pmc/articles/instance/3163686/bin/pone.0023989.g002.gif" src-large="/pmc/articles/instance/3163686/bin/pone.0023989.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3163686/" ref="ordinalpos=2163&amp;ncbi_uid=3208672&amp;link_uid=PMC3163686" image-link="/pmc/articles/PMC3163686/figure/pone-0023989-g002/" class="imagepopup">Figure 2. Models of ST2L inhibiting the TLR2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: In Silico Approach to Inhibition of <span class="highlight" style="background-color:">Signaling</span> Pathways of Toll-Like Receptors 2 and 4 by ST2L. </a></p><div class="supp"><p class="details"><div>Pictorial representation of ST2L inhibiting MyD88-dependent TLR2 (TLR2/1 and TLR2/6) <span class="highlight" style="background-color:">signaling</span> pathways. ST2L heterodimerizes with MyD88, thereby preventing the engagement of adapter protein MyD88 into the post-receptor <span class="highlight" style="background-color:">signaling</span> complexes (TLR2/1-MyD88 tetramer and TLR2/6-MyD88 tetramer) and thus exhibiting its inhibitory effect. Full-length structures of TLR2, 1 and 6 are colored in sky blue, tan and light green, respectively. The TIR domains of ST2L and MyD88 are colored in hot pink and gold, respectively. The available TLR2/1 and TLR2/6 ECD structure coordinates along with their respective ligands were taken from the PDB (2Z80 and 3A79). The red color dotted line along with the bar represents inhibition symbol.</div></p></div><div class="aux"><div class="resc">Shaherin Basith, et al. PLoS One. 2011;6(8):e23989.</div><p class="links"><a class="dblinks" href="/pubmed/21897866" ref="ordinalpos=2163&amp;ncbi_uid=3208672&amp;link_uid=21897866">Citation</a><a class="dblinks" href="/pmc/articles/PMC3163686/" ref="ordinalpos=2163&amp;ncbi_uid=3208672&amp;link_uid=PMC3163686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2164.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2491588/figure/pone-0002942-g007/" ref="ordinalpos=2164&amp;ncbi_uid=1510686&amp;link_uid=PMC2491588" href="/pmc/articles/PMC2491588/"><img src="/pmc/articles/instance/2491588/bin/pone.0002942.g007.gif" src-large="/pmc/articles/instance/2491588/bin/pone.0002942.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2491588/" ref="ordinalpos=2164&amp;ncbi_uid=1510686&amp;link_uid=PMC2491588" image-link="/pmc/articles/PMC2491588/figure/pone-0002942-g007/" class="imagepopup">Figure 7. PTHR1 <span class="highlight" style="background-color:">Signaling</span> in Osteocytes Leads to Increased Bone Mass and Remodeling via Distinct Mechanisms..  From: Control of Bone Mass and Remodeling by PTH Receptor <span class="highlight" style="background-color:">Signaling</span> in Osteocytes. </a></p><div class="supp"><p class="details"><div>In the proposed model, PTHR1 <span class="highlight" style="background-color:">signaling</span> in osteocytes activates at least two distinct pathways: one leading to increased bone mass and the other leading to increased bone remodeling. Suppression of Sost/sclerostin and activation of LRP5 <span class="highlight" style="background-color:">signaling</span> increase osteoblast numbers and are required for the increase in bone mass. The elevation of bone resorption is independent of the Sclerostin/LRP5 <span class="highlight" style="background-color:">pathway</span>. The question mark indicates uncertainty of the cellular source of RANKL and M-CSF (osteocytes versus stromal/osteoblastic cells).</div></p></div><div class="aux"><div class="resc">Charles A. O'Brien, et al. PLoS ONE. 2008;3(8):e2942.</div><p class="links"><a class="dblinks" href="/pubmed/18698360" ref="ordinalpos=2164&amp;ncbi_uid=1510686&amp;link_uid=18698360">Citation</a><a class="dblinks" href="/pmc/articles/PMC2491588/" ref="ordinalpos=2164&amp;ncbi_uid=1510686&amp;link_uid=PMC2491588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2165.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4342183/figure/f2-dddt-9-1103/" ref="ordinalpos=2165&amp;ncbi_uid=7224305&amp;link_uid=PMC4342183" href="/pmc/articles/PMC4342183/"><img src="/pmc/articles/instance/4342183/bin/dddt-9-1103Fig2.gif" src-large="/pmc/articles/instance/4342183/bin/dddt-9-1103Fig2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4342183/" ref="ordinalpos=2165&amp;ncbi_uid=7224305&amp;link_uid=PMC4342183" image-link="/pmc/articles/PMC4342183/figure/f2-dddt-9-1103/" class="imagepopup">Figure 2.  From: Hsa-microRNA-181a is a regulator of a number of cancer genes and a biomarker for endometrial carcinoma in patients: a bioinformatic and clinical study and the therapeutic implication. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the target list of hsa-miR-181a-5p, based on TarBase 6.0.<br /><b>Notes:</b> hsa-181a-5p is a regulator of Wnt. Three Wnt <span class="highlight" style="background-color:">signaling</span> pathways have been characterized: the canonical Wnt <span class="highlight" style="background-color:">pathway</span>, the noncanonical planar cell polarity <span class="highlight" style="background-color:">pathway</span>, and the noncanonical Wnt/calcium <span class="highlight" style="background-color:">pathway</span>.103 All three Wnt <span class="highlight" style="background-color:">signaling</span> pathways are activated by the binding of a Wnt-protein ligand to a Frizzled family receptor, which passes the biological signal to the protein Dishevelled inside the cell. The canonical Wnt <span class="highlight" style="background-color:">pathway</span> leads to regulation of gene transcription, the noncanonical planar cell polarity <span class="highlight" style="background-color:">pathway</span> regulates the cytoskeleton that is responsible for the shape of the cell, and the noncanonical Wnt/calcium <span class="highlight" style="background-color:">pathway</span> regulates calcium level inside the cell. Wnt <span class="highlight" style="background-color:">signaling</span> pathways are highly evolutionarily conserved. Wnt <span class="highlight" style="background-color:">signaling</span> has been implicated in the development of breast cancer, EC, and other types of cancer.103,104 Changes in CTNNB1 expression, which is the gene that encodes -catenin, can be measured in, not just breast cancer but also, colorectal cancer, melanoma, prostate cancer, lung cancer, EC, and several other cancer types. Increased expression of Wnt ligand-proteins, such as Wnt 1, Wnt2, and Wnt7A, has been observed in the development of glioblastoma, esophageal cancer, EC, and ovarian cancer. There is clinical and experimental evidence that Wnt/-catenin pathways are deregulated and play an important role in the initiation, development, growth, and metastasis of EC.105109 Targets of hsa-miR-181a-5p are marked with a red star.<br /><b>Abbreviation:</b> EC, endometrial cancer.</div></p></div><div class="aux"><div class="resc">Shuming He, et al. Drug Des Devel Ther. 2015;9:1103-1175.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=2165&amp;ncbi_uid=7224305&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4342183/" ref="ordinalpos=2165&amp;ncbi_uid=7224305&amp;link_uid=PMC4342183">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2166.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2771222/figure/F1/" ref="ordinalpos=2166&amp;ncbi_uid=2166503&amp;link_uid=PMC2771222" href="/pmc/articles/PMC2771222/"><img src="/pmc/articles/instance/2771222/bin/nihms112832f1.gif" src-large="/pmc/articles/instance/2771222/bin/nihms112832f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2771222/" ref="ordinalpos=2166&amp;ncbi_uid=2166503&amp;link_uid=PMC2771222" image-link="/pmc/articles/PMC2771222/figure/F1/" class="imagepopup">Figure 1.  From: Lack of evidence for activation of the Hedgehog <span class="highlight" style="background-color:">pathway</span> in psoriasis. </a></p><div class="supp"><p class="details"><div>Recent reports have suggested that the Hedgehog (Hh) <span class="highlight" style="background-color:">pathway</span> is activated in lesional psoriatic skin (Kuenzli et al, 2004; Tas and Avci, 2004; Endo et al, 2006; Meth and Weinberg, 2006), and that pharmacological inhibition of this <span class="highlight" style="background-color:">pathway</span> using cyclopamine may lead to rapid resolution of the disease (Kuenzli et al, 2004; Tas and Avci, 2004; Meth and Weinberg, 2006). The Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is one of the major <span class="highlight" style="background-color:">signaling</span> pathways involved in embryonic development (Ingham and Placzek, 2006). During physiologic Hh <span class="highlight" style="background-color:">signaling</span>, Hh proteins bind to the cell surface receptor Patched (PTCH1), thereby releasing Smoothened (SMO) from PTCH-mediated inhibition. SMO activation then triggers a series of intracellular events, culminating in alterations in gene expression mediated by the Gli family of transcription factors (GLI1, GLI2 and GLI3) (Ruiz i Altaba et al, 2007) (Figure 1). The transcripts for PTCH1 and GLI1 are reliable markers for both physiologic and pathologic Hh <span class="highlight" style="background-color:">signaling</span> activity as they are consistently induced when the Hh <span class="highlight" style="background-color:">pathway</span> is activated (McMahon et al, 2003; Hutchin et al, 2005). This <span class="highlight" style="background-color:">pathway</span> has been shown to be a crucial regulator of hair follicle growth and sebaceous gland biology (Allen et al, 2003; Niemann et al, 2003). On the other hand, sustained activation of the Hh <span class="highlight" style="background-color:">pathway</span> appears to be the driving force for BCC development (Daya-Grosjean and Couve-Privat, 2005).</div></p></div><div class="aux"><div class="resc">JE Gudjonsson, et al. J Invest Dermatol. ;129(3):635-640.</div><p class="links"><a class="dblinks" href="/pubmed/18754037" ref="ordinalpos=2166&amp;ncbi_uid=2166503&amp;link_uid=18754037">Citation</a><a class="dblinks" href="/pmc/articles/PMC2771222/" ref="ordinalpos=2166&amp;ncbi_uid=2166503&amp;link_uid=PMC2771222">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2167.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4095906/figure/F3/" ref="ordinalpos=2167&amp;ncbi_uid=6369284&amp;link_uid=PMC4095906" href="/pmc/articles/PMC4095906/"><img src="/pmc/articles/instance/4095906/bin/nihms496289f3.gif" src-large="/pmc/articles/instance/4095906/bin/nihms496289f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4095906/" ref="ordinalpos=2167&amp;ncbi_uid=6369284&amp;link_uid=PMC4095906" image-link="/pmc/articles/PMC4095906/figure/F3/" class="imagepopup">Figure 3.  From: B cell Toll-like receptors and immunoglobulin class-switch DNA recombination. </a></p><div class="supp"><p class="details"><div>TLR and BCR <span class="highlight" style="background-color:">signaling</span> and integration in CSR and antibody responses. TLR ligands located on the surface of microorganisms are recognized by the extracellular TLRs, whereas nucleic acid TLR ligands located inside microorganisms are recognized by endosomal TLRs. TLR <span class="highlight" style="background-color:">signaling</span> is initially relayed through homotypic TIR-TIR interactions with TIR-domain containing adapters such as MyD88 or TRIF, and eventually results in activation of several pathways, including the canonical NF-kB <span class="highlight" style="background-color:">pathway</span>. This induces, among other genes, expression of the non-canonical NF-kB precursor p100. Proximal BCR <span class="highlight" style="background-color:">signaling</span> activates the PI3K and PLC2 pathways, converging to nodes such as TAK1/TAB2/3/TRAF6 for NF-B and MAPK <span class="highlight" style="background-color:">pathway</span> activation. Dual BCR/TLR <span class="highlight" style="background-color:">signaling</span> integrates to induce p100 processing to p52, thereby activating the non-canonical NF-B <span class="highlight" style="background-color:">pathway</span>, which is required for induction of AID. TNFR member CD40 induces both canonical and non-canonical NF-B activation, thereby inducing CSR.</div></p></div><div class="aux"><div class="resc">Egest J. Pone, et al. Front Biosci (Landmark Ed). ;17:2594-2615.</div><p class="links"><a class="dblinks" href="/pubmed/22652800" ref="ordinalpos=2167&amp;ncbi_uid=6369284&amp;link_uid=22652800">Citation</a><a class="dblinks" href="/pmc/articles/PMC4095906/" ref="ordinalpos=2167&amp;ncbi_uid=6369284&amp;link_uid=PMC4095906">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2168.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2675913/figure/F1/" ref="ordinalpos=2168&amp;ncbi_uid=1790372&amp;link_uid=PMC2675913" href="/pmc/articles/PMC2675913/"><img src="/pmc/articles/instance/2675913/bin/nihms96630f1.gif" src-large="/pmc/articles/instance/2675913/bin/nihms96630f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2675913/" ref="ordinalpos=2168&amp;ncbi_uid=1790372&amp;link_uid=PMC2675913" image-link="/pmc/articles/PMC2675913/figure/F1/" class="imagepopup">Figure 1.  From: ON SENSITIVITY AMPLIFICATION IN INTRACELLULAR <span class="highlight" style="background-color:">SIGNALING</span> CASCADES. </a></p><div class="supp"><p class="details"><div>A typical input-output curve of a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (a) and its response coefficients (b). The input values yielding 10% and 90% outputs, <i>x</i><sub>0.1</sub> and <i>x</i><sub>0.9</sub>, are used to define Hill sensitivities, <i>S</i><sub>H</sub>; the logarithmic sensitivity, <i>S</i><sub>L</sub>, is defined as a maximal response coefficient in (b).</div></p></div><div class="aux"><div class="resc">Eva Racz, et al. Phys Biol. ;5(3):36004-36004.</div><p class="links"><a class="dblinks" href="/pubmed/18663279" ref="ordinalpos=2168&amp;ncbi_uid=1790372&amp;link_uid=18663279">Citation</a><a class="dblinks" href="/pmc/articles/PMC2675913/" ref="ordinalpos=2168&amp;ncbi_uid=1790372&amp;link_uid=PMC2675913">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2169.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131208/figure/f4-viruses-03-00714/" ref="ordinalpos=2169&amp;ncbi_uid=3280182&amp;link_uid=PMC3131208" href="/pmc/articles/PMC3131208/"><img src="/pmc/articles/instance/3185777/bin/viruses-03-00714f4.gif" src-large="/pmc/articles/instance/3185777/bin/viruses-03-00714f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131208/" ref="ordinalpos=2169&amp;ncbi_uid=3280182&amp;link_uid=PMC3131208" image-link="/pmc/articles/PMC3131208/figure/f4-viruses-03-00714/" class="imagepopup">Figure 4. From: Human T-Cell Lymphotropic Virus: A Model of NF-?B-Associated Tumorigenesis. </a></p><div class="supp"><p class="details"><div>NF-B <span class="highlight" style="background-color:">signaling</span> pathways. Although the canonical and non-canonical <span class="highlight" style="background-color:">signaling</span> pathways primarily activate the RelA/p50 and RelB/p52 dimers, respectively, all NF-B members may be activated by either <span class="highlight" style="background-color:">pathway</span> or both. In fact, the RelA/p50 dimers may be sequestered in the cytoplasm by p100 and can be activated through p100 processing. On the other hand, NF-B dimers containing p52 may be sequestered in the cytoplasm by IB and can be activated through IB degradation. Furthermore, activation of the canonical NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be induced through inducible degradation of IB, IB and p105, a process similar to the inducible IB degradation, although their degradation dynamics can be different.</div></p></div><div class="aux"><div class="resc">Zhaoxia Qu, et al. Viruses. 2011 June;3(6):714-749.</div><p class="links"><a class="dblinks" href="/pubmed/21743832" ref="ordinalpos=2169&amp;ncbi_uid=3280182&amp;link_uid=21743832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131208/" ref="ordinalpos=2169&amp;ncbi_uid=3280182&amp;link_uid=PMC3131208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2170.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2802668/figure/fig3/" ref="ordinalpos=2170&amp;ncbi_uid=2677710&amp;link_uid=PMC2802668" href="/pmc/articles/PMC2802668/"><img src="/pmc/articles/instance/2802668/bin/carcinbgp231f03_4c.gif" src-large="/pmc/articles/instance/2802668/bin/carcinbgp231f03_4c.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2802668/" ref="ordinalpos=2170&amp;ncbi_uid=2677710&amp;link_uid=PMC2802668" image-link="/pmc/articles/PMC2802668/figure/fig3/" class="imagepopup">Fig. 3. From: Nrf2: friend or foe for chemoprevention?. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) U-Shaped modulation of cancer risk through the KEAP1NRF2 <span class="highlight" style="background-color:">pathway</span>. Optimal activation of the <span class="highlight" style="background-color:">pathway</span> lies in a pharmacological range between the biologically effective dose (BED) that minimally activates the <span class="highlight" style="background-color:">pathway</span> and a maximal-tolerated dose (MTD) that not only activates the <span class="highlight" style="background-color:">pathway</span> but also may produce dose-limiting off target toxicities as well. Single-nucleotide polymorphisms (SNPs) in the <i>Nrf2</i> promoter may diminish constitutive or inducible capacity of the <span class="highlight" style="background-color:">pathway</span>, whereas mutations or epigenetic silencing of <i>Keap1</i> leads to sustained hyperactivation. (<b>B</b>) Comparison of the kinetics of induction of NRF2-regulated genes by pharmacological intervention versus genetic disruption of KEAP1 function. Chemopreventive agents that activate NRF2 <span class="highlight" style="background-color:">signaling</span> are typically administered (in both preclinical and clinical settings) on daily to weekly schedules, leading to pronounced but transient induction of downstream genes (dotted lines). In contrast, genetic disruption of the <span class="highlight" style="background-color:">pathway</span>, such as by conditional, tissue-specific targeted disruption of <i>Keap1</i> (cKeap1) in the mouse or through somatic mutations acquired by cancer cells in <i>KEAP1</i> or <i>NRF2</i>, leads to markedly elevated and sustained activation of the <span class="highlight" style="background-color:">pathway</span> (solid line). Chemopreventive agents cannot replicate the magnitude of response seen with genetic perturbation of the <span class="highlight" style="background-color:">pathway</span> (95).</div></p></div><div class="aux"><div class="resc">Thomas W. Kensler, et al. Carcinogenesis. 2010 January;31(1):90-99.</div><p class="links"><a class="dblinks" href="/pubmed/19793802" ref="ordinalpos=2170&amp;ncbi_uid=2677710&amp;link_uid=19793802">Citation</a><a class="dblinks" href="/pmc/articles/PMC2802668/" ref="ordinalpos=2170&amp;ncbi_uid=2677710&amp;link_uid=PMC2802668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2171.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3595309/figure/F5/" ref="ordinalpos=2171&amp;ncbi_uid=5868710&amp;link_uid=PMC3595309" href="/pmc/articles/PMC3595309/"><img src="/pmc/articles/instance/3595309/bin/cbt-14-262-g5.gif" src-large="/pmc/articles/instance/3595309/bin/cbt-14-262-g5.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3595309/" ref="ordinalpos=2171&amp;ncbi_uid=5868710&amp;link_uid=PMC3595309" image-link="/pmc/articles/PMC3595309/figure/F5/" class="imagepopup">Figure. From: Cadherin-17 induces tumorigenesis and lymphatic metastasis in gastric cancer through activation of NF?B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Figure 5.</b> Schematic exhibition of the proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulated by CDH17 in gastric cancer cells. Coupling CDH17 activates the IKK complex, which subsequently phosphorylates the IB-. Then the phosphorylated IB- undergoes proteasome-dependent decomposition, which releases the heterodimers of p65/p50 into cytoplasm and transferred into the nucleus. Finally, the p65 binds to its responsive gene and promotes the transcription of downstream proteins including VEGF-C and MMP-9. On the other hand, RNAi mediated inhibition of CDH17 attenuates the activation of NFB in gastric cancer cells, leading to a concomitant reduction in downstream proteins.</div></p></div><div class="aux"><div class="resc">Jin Wang, et al. Cancer Biol Ther. 2013 March 1;14(3):262-270.</div><p class="links"><a class="dblinks" href="/pubmed/23298905" ref="ordinalpos=2171&amp;ncbi_uid=5868710&amp;link_uid=23298905">Citation</a><a class="dblinks" href="/pmc/articles/PMC3595309/" ref="ordinalpos=2171&amp;ncbi_uid=5868710&amp;link_uid=PMC3595309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2172.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3972686/figure/cells-02-00361-f008/" ref="ordinalpos=2172&amp;ncbi_uid=6002001&amp;link_uid=PMC3972686" href="/pmc/articles/PMC3972686/"><img src="/pmc/articles/instance/3972686/bin/cells-02-00361-g008.gif" src-large="/pmc/articles/instance/3972686/bin/cells-02-00361-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3972686/" ref="ordinalpos=2172&amp;ncbi_uid=6002001&amp;link_uid=PMC3972686" image-link="/pmc/articles/PMC3972686/figure/cells-02-00361-f008/" class="imagepopup">Figure 8.  From: Monte Carlo Study Elucidates the Type 1/Type 2 Choice in Apoptotic Death <span class="highlight" style="background-color:">Signaling</span> in Healthy and Cancer Cells. </a></p><div class="supp"><p class="details"><div>A previously developed model of minimal network [8] is utilized to elucidate some of the robust systems level regulatory mechanisms in apoptosis <span class="highlight" style="background-color:">signaling</span>. This <span class="highlight" style="background-color:">signaling</span> network can be thought of as a highly coarse-grained representation of the full apoptotic <span class="highlight" style="background-color:">signaling</span> network. The following rate constants were used: 2  10<sup>4</sup> (for the type 1 <span class="highlight" style="background-color:">pathway</span>); 1.0, 10<sup>7</sup> and 10<sup>1</sup> (for the type 2 <span class="highlight" style="background-color:">pathway</span>). X<sub>0</sub> measures the strength of an apoptotic stimulus (varied in Figure 9). Initial numbers for all other the molecules (X<sub>1</sub>, X<sub>2</sub>, X<sub>3</sub>) were assumed to be 100. </div></p></div><div class="aux"><div class="resc">Subhadip Raychaudhuri, et al. Cells. 2013 June;2(2):361-392.</div><p class="links"><a class="dblinks" href="/pubmed/24709706" ref="ordinalpos=2172&amp;ncbi_uid=6002001&amp;link_uid=24709706">Citation</a><a class="dblinks" href="/pmc/articles/PMC3972686/" ref="ordinalpos=2172&amp;ncbi_uid=6002001&amp;link_uid=PMC3972686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2173.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2864782/figure/F1/" ref="ordinalpos=2173&amp;ncbi_uid=2328021&amp;link_uid=PMC2864782" href="/pmc/articles/PMC2864782/"><img src="/pmc/articles/instance/2864782/bin/nihms197972f1.gif" src-large="/pmc/articles/instance/2864782/bin/nihms197972f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2864782/" ref="ordinalpos=2173&amp;ncbi_uid=2328021&amp;link_uid=PMC2864782" image-link="/pmc/articles/PMC2864782/figure/F1/" class="imagepopup">Fig. 1.  From: Ameliorating prefrontal cortical dysfunction in mental illness: inhibition of phosphotidyl inositol-protein kinase C <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>High levels of phosphotidyl inositol-protein kinase C <span class="highlight" style="background-color:">signaling</span> in the PFC markedly impair PFC function at the behavioral and cellular levels. Molecules that normally serve as brakes on this <span class="highlight" style="background-color:">pathway</span> and are highly associated with genetic/biochemical alteration in mental illness are indicated in <i>gray</i>. Patients with these genetic changes would have weaker regulation of phosphotidyl inositol-protein kinase C <span class="highlight" style="background-color:">signaling</span>. Many treatments for mental illness inhibit this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> through either extracellular or intracellular actions; these treatments are indicated in <i>italics</i>. Tamoxifen is a proof of concept (<i>p.o.c.</i>) treatment; prazosin is used to treat PTSD but not schizophrenia or bipolar disorder; chelerythrine is undergoing pre-clinical safety testing</div></p></div><div class="aux"><div class="resc">A. F. T. Arnsten. Psychopharmacology (Berl). ;202(1-3):445-455.</div><p class="links"><a class="dblinks" href="/pubmed/18719894" ref="ordinalpos=2173&amp;ncbi_uid=2328021&amp;link_uid=18719894">Citation</a><a class="dblinks" href="/pmc/articles/PMC2864782/" ref="ordinalpos=2173&amp;ncbi_uid=2328021&amp;link_uid=PMC2864782">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2174.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3425953/figure/F1/" ref="ordinalpos=2174&amp;ncbi_uid=4190613&amp;link_uid=PMC3425953" href="/pmc/articles/PMC3425953/"><img src="/pmc/articles/instance/3425953/bin/nihms398364f1.gif" src-large="/pmc/articles/instance/3425953/bin/nihms398364f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3425953/" ref="ordinalpos=2174&amp;ncbi_uid=4190613&amp;link_uid=PMC3425953" image-link="/pmc/articles/PMC3425953/figure/F1/" class="imagepopup">Figure 1.  From: High-Throughput Screening Strategies for Targeted Identification Small-Molecule Compounds Towards Activated Pathways in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>The activity of Wnt canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the normal and tumor cells. Left panel presents the activity of the Wnt <span class="highlight" style="background-color:">pathway</span> in normal cells (steady and active states, respectively). Right panel shows hyperactivation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the intestinal tumor cells upon APC and -catenin mutations, respectively. Red star  activation; red crossed lines  inactivation.</div></p></div><div class="aux"><div class="resc">Agnieszka B. Bialkowska, et al. Future Oncol. ;8(3):259-272.</div><p class="links"><a class="dblinks" href="/pubmed/22409463" ref="ordinalpos=2174&amp;ncbi_uid=4190613&amp;link_uid=22409463">Citation</a><a class="dblinks" href="/pmc/articles/PMC3425953/" ref="ordinalpos=2174&amp;ncbi_uid=4190613&amp;link_uid=PMC3425953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2175.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3566061/figure/pone-0056025-g004/" ref="ordinalpos=2175&amp;ncbi_uid=4621609&amp;link_uid=PMC3566061" href="/pmc/articles/PMC3566061/"><img src="/pmc/articles/instance/3566061/bin/pone.0056025.g004.gif" src-large="/pmc/articles/instance/3566061/bin/pone.0056025.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3566061/" ref="ordinalpos=2175&amp;ncbi_uid=4621609&amp;link_uid=PMC3566061" image-link="/pmc/articles/PMC3566061/figure/pone-0056025-g004/" class="imagepopup">Figure 4. Gene expression of Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulators:.  From: Impact of Laminitis on the Canonical Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Basal Epithelial Cells of the Equine Digital Laminae. </a></p><div class="supp"><p class="details"><div>Expression of genes encoding <b>A</b> - Wnt4, <b>B</b> - FZD4, <b>C</b> - LRP6, <b>D</b> - Dsh, <b>E</b> - PP1, <b>F</b> - Axin1, <b>G</b> - APC, <b>H</b> - CKI, <b>I</b> - GSK3 relative to the gene expression of GapDH. Horizontal lines indicate mean  standard error of mean of n=8 CON, n=6 OG1, and n=5 OG3, animals. *=p&lt;0.05, **=p&lt;0.01, as calculated by one-way ANOVA.</div></p></div><div class="aux"><div class="resc">Le Wang, et al. PLoS One. 2013;8(2):e56025.</div><p class="links"><a class="dblinks" href="/pubmed/23405249" ref="ordinalpos=2175&amp;ncbi_uid=4621609&amp;link_uid=23405249">Citation</a><a class="dblinks" href="/pmc/articles/PMC3566061/" ref="ordinalpos=2175&amp;ncbi_uid=4621609&amp;link_uid=PMC3566061">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2176.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2719350/figure/F9/" ref="ordinalpos=2176&amp;ncbi_uid=2023656&amp;link_uid=PMC2719350" href="/pmc/articles/PMC2719350/"><img src="/pmc/articles/instance/2719350/bin/zbc0300980580009.gif" src-large="/pmc/articles/instance/2719350/bin/zbc0300980580009.jpg" alt="FIGURE 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2719350/" ref="ordinalpos=2176&amp;ncbi_uid=2023656&amp;link_uid=PMC2719350" image-link="/pmc/articles/PMC2719350/figure/F9/" class="imagepopup">FIGURE 9. From: Thromboxane A2 Receptor Activates a Rho-associated Kinase/LKB1/PTEN <span class="highlight" style="background-color:">Pathway</span> to Attenuate Endothelium Insulin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>TP mediates HFD-induced inhibition of both Akt and eNOS in murine aortas.</b> Male C57BL/6J mice were fed a normal diet (<i>ND</i>) or a HFD for 13 weeks and administrated vehicle or SQ29548 (5 g/kg/day) for the following 5 weeks. <i>AC</i>, aortic homogenates were analyzed for levels of total and phosphorylated eNOS (phosphorylation at Ser<sup>1177</sup>) (<i>A</i>), Akt (phosphorylation at Ser<sup>473</sup>) (<i>B</i>), and PTEN (phosphorylation at Ser<sup>380</sup>/Thr<sup>382/383</sup>) (<i>C</i>). <i>D</i>, proposed <span class="highlight" style="background-color:">pathway</span> underlying TP-induced attenuation of insulin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Ping Song, et al. J Biol Chem. 2009 June 19;284(25):17120-17128.</div><p class="links"><a class="dblinks" href="/pubmed/19403525" ref="ordinalpos=2176&amp;ncbi_uid=2023656&amp;link_uid=19403525">Citation</a><a class="dblinks" href="/pmc/articles/PMC2719350/" ref="ordinalpos=2176&amp;ncbi_uid=2023656&amp;link_uid=PMC2719350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2177.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2951022/figure/F1/" ref="ordinalpos=2177&amp;ncbi_uid=2803681&amp;link_uid=PMC2951022" href="/pmc/articles/PMC2951022/"><img src="/pmc/articles/instance/2951022/bin/nihms235327f1.gif" src-large="/pmc/articles/instance/2951022/bin/nihms235327f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2951022/" ref="ordinalpos=2177&amp;ncbi_uid=2803681&amp;link_uid=PMC2951022" image-link="/pmc/articles/PMC2951022/figure/F1/" class="imagepopup">Figure 1.  From: Membrane TLR <span class="highlight" style="background-color:">Signaling</span> Mechanisms in the Gastrointestinal Tract during Sepsis. </a></p><div class="supp"><p class="details"><div>Toll-like receptor (TLR) <span class="highlight" style="background-color:">signaling</span> pathways. All TLRs, except TLR<i>3</i> and the alternate TLR<i>4</i>-TRIF <span class="highlight" style="background-color:">pathway</span>, utilize intracellular adaptor protein MyD88 to activate NF-B and the MAPKs. Classical immune cell generated inflammatory cytokines/mediators indirectly modulate motility, but MyD88-dependent <span class="highlight" style="background-color:">signaling</span> on non-bone derived cells also plays a substantial role in <i>TLR4</i> induced ileus.</div></p></div><div class="aux"><div class="resc">Bettina M. Buchholz, et al. Neurogastroenterol Motil. ;22(3):232-245.</div><p class="links"><a class="dblinks" href="/pubmed/20377787" ref="ordinalpos=2177&amp;ncbi_uid=2803681&amp;link_uid=20377787">Citation</a><a class="dblinks" href="/pmc/articles/PMC2951022/" ref="ordinalpos=2177&amp;ncbi_uid=2803681&amp;link_uid=PMC2951022">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2178.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3444139/figure/F3/" ref="ordinalpos=2178&amp;ncbi_uid=4287057&amp;link_uid=PMC3444139" href="/pmc/articles/PMC3444139/"><img src="/pmc/articles/instance/3444139/bin/fmicb-03-00334-g003.gif" src-large="/pmc/articles/instance/3444139/bin/fmicb-03-00334-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3444139/" ref="ordinalpos=2178&amp;ncbi_uid=4287057&amp;link_uid=PMC3444139" image-link="/pmc/articles/PMC3444139/figure/F3/" class="imagepopup">Figure 3.  From: Molecular Hallmarks of Adult T Cell Leukemia. </a></p><div class="supp"><p class="details"><div><b>NF-B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ATL</b>. Compared to normal T cell, HTLV-1-infected cell shows intense activity of NF-B <span class="highlight" style="background-color:">pathway</span> through the Tax interactome. In contrast, leukemic cell sustains strong NF-B activity without Tax by acquisition of miR-31-dependent NIK expression. Activation of the non-canonical <span class="highlight" style="background-color:">pathway</span> ultimately joins canonical <span class="highlight" style="background-color:">pathway</span>, resulting in transactivation of numerous target genes important for cell survival, cytokine production, and invasiveness.</div></p></div><div class="aux"><div class="resc">Makoto Yamagishi, et al. Front Microbiol. 2012;3:334.</div><p class="links"><a class="dblinks" href="/pubmed/23060864" ref="ordinalpos=2178&amp;ncbi_uid=4287057&amp;link_uid=23060864">Citation</a><a class="dblinks" href="/pmc/articles/PMC3444139/" ref="ordinalpos=2178&amp;ncbi_uid=4287057&amp;link_uid=PMC3444139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2179.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2885580/figure/F9/" ref="ordinalpos=2179&amp;ncbi_uid=3118959&amp;link_uid=PMC2885580" href="/pmc/articles/PMC2885580/"><img src="/pmc/articles/instance/2885580/bin/nihms-193346-f0009.gif" src-large="/pmc/articles/instance/2885580/bin/nihms-193346-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2885580/" ref="ordinalpos=2179&amp;ncbi_uid=3118959&amp;link_uid=PMC2885580" image-link="/pmc/articles/PMC2885580/figure/F9/" class="imagepopup">Figure 9. Decreased Cn <span class="highlight" style="background-color:">signaling</span> downregulates PGC-1 during chronic diabetes mellitus.  From: Calcineurin <span class="highlight" style="background-color:">signaling</span> and PGC-1? expression are suppressed during muscle atrophy due to diabetes. </a></p><div class="supp"><p class="details"><div>Functional capacity and muscle mass are regulated by contractile and activity and the associated calcium <span class="highlight" style="background-color:">signaling</span>. Two parallel calcium-dependent pathways involving calcium-calmodulin kinase (CaMK) and calcineurin (Cn) can regulate MEF2 whereas Cn is the primary <span class="highlight" style="background-color:">pathway</span> that regulates NFATc. Since both NFATc and MEF2 activities are attenuated during chronic diabetes mellitus as well as in both CnA and CnA knockout mice, we propose that reduced Cn <span class="highlight" style="background-color:">signaling</span> is predominantly responsible for the decreased level of PGC-1 expression. Consequently, mitochondrial biogenesis, and the expression of proteins associated with oxidative metabolism in skeletal muscle are likely to be affected. Suppression of Cn/NFATc/MEF2 <span class="highlight" style="background-color:">signaling</span>, and subsequent downregulation of PGC-1 may perpetuate the decrease in muscle functional capacity and mass that accompanies diabetes mellitus.</div></p></div><div class="aux"><div class="resc">Tiffany K. Roberts-Wilson, et al. Biochim Biophys Acta. ;1803(8):960-967.</div><p class="links"><a class="dblinks" href="/pubmed/20359506" ref="ordinalpos=2179&amp;ncbi_uid=3118959&amp;link_uid=20359506">Citation</a><a class="dblinks" href="/pmc/articles/PMC2885580/" ref="ordinalpos=2179&amp;ncbi_uid=3118959&amp;link_uid=PMC2885580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2180.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3920780/figure/F2/" ref="ordinalpos=2180&amp;ncbi_uid=6438567&amp;link_uid=PMC3920780" href="/pmc/articles/PMC3920780/"><img src="/pmc/articles/instance/3920780/bin/S162fig2.gif" src-large="/pmc/articles/instance/3920780/bin/S162fig2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3920780/" ref="ordinalpos=2180&amp;ncbi_uid=6438567&amp;link_uid=PMC3920780" image-link="/pmc/articles/PMC3920780/figure/F2/" class="imagepopup">Figure 2.  From: In Vivo Actions of Peroxisome Proliferator-Activated Receptors. </a></p><div class="supp"><p class="details"><div>Insulin signal transduction in healthy nondiabetic (<i>left panel</i>) and T2DM (<i>right panel</i>) subjects. Thiazolidinediones improve insulin <span class="highlight" style="background-color:">signaling</span> through the PI-3 kinase <span class="highlight" style="background-color:">pathway</span>, while inhibiting insulin <span class="highlight" style="background-color:">signaling</span> through the MAP kinase <span class="highlight" style="background-color:">pathway</span>. Reprinted with permission from DeFronzo (61).</div></p></div><div class="aux"><div class="resc">Roy Eldor, et al. Diabetes Care. 2013 August;36(Suppl 2):S162-S174.</div><p class="links"><a class="dblinks" href="/pubmed/23882042" ref="ordinalpos=2180&amp;ncbi_uid=6438567&amp;link_uid=23882042">Citation</a><a class="dblinks" href="/pmc/articles/PMC3920780/" ref="ordinalpos=2180&amp;ncbi_uid=6438567&amp;link_uid=PMC3920780">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2181.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877642/figure/fig1/" ref="ordinalpos=2181&amp;ncbi_uid=5713247&amp;link_uid=PMC3877642" href="/pmc/articles/PMC3877642/"><img src="/pmc/articles/instance/3877642/bin/JDR2013-946432.001.gif" src-large="/pmc/articles/instance/3877642/bin/JDR2013-946432.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877642/" ref="ordinalpos=2181&amp;ncbi_uid=5713247&amp;link_uid=PMC3877642" image-link="/pmc/articles/PMC3877642/figure/fig1/" class="imagepopup">Figure 1.  From: Effects of Acute Exercise and Chronic Exercise on the Liver Leptin-AMPK-ACC <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Rats with Type 2 Diabetes. </a></p><div class="supp"><p class="details"><div>Histograph showing the effects of exercise on the leptin-AMPK-ACC <span class="highlight" style="background-color:">pathway</span> in type 2 diabetes rats. The relative levels of protein phosphorylation and protein expression related to the leptin-AMPK-ACC <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on different groups were shown. (a) Leptin and its receptor. (b) AMPK1/2 and p-AMPK1/2 (Thr<sup>172</sup>). (c) AMPK<i></i>1 and p-AMPK<i></i>1 (Thr<sup>172</sup>). (d) AMPK<i></i>2 and p-AMPK<i></i>2 (Thr<sup>172</sup>). (e) ACC and p-ACC (Ser<sup>79</sup>). Note: in comparison with the control group: **<i>P</i> &lt; 0.01, *<i>P</i> &lt; 0.05; in comparison with the diabetes group: <sup>##</sup><i>P</i> &lt; 0.01, <sup>#</sup><i>P</i> &lt; 0.05 (<i>n</i> = 8).</div></p></div><div class="aux"><div class="resc">Xuejie Yi, et al. J Diabetes Res. 2013;2013:946432.</div><p class="links"><a class="dblinks" href="/pubmed/24455748" ref="ordinalpos=2181&amp;ncbi_uid=5713247&amp;link_uid=24455748">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877642/" ref="ordinalpos=2181&amp;ncbi_uid=5713247&amp;link_uid=PMC3877642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2182.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3652647/figure/F7/" ref="ordinalpos=2182&amp;ncbi_uid=4924986&amp;link_uid=PMC3652647" href="/pmc/articles/PMC3652647/"><img src="/pmc/articles/instance/3652647/bin/WJBC-4-18-g007.gif" src-large="/pmc/articles/instance/3652647/bin/WJBC-4-18-g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3652647/" ref="ordinalpos=2182&amp;ncbi_uid=4924986&amp;link_uid=PMC3652647" image-link="/pmc/articles/PMC3652647/figure/F7/" class="imagepopup">Figure 7.  From: Refractoriness of interferon-beta <span class="highlight" style="background-color:">signaling</span> through NOD1 <span class="highlight" style="background-color:">pathway</span> in mouse respiratory epithelial cells using the anticancer xanthone compound. </a></p><div class="supp"><p class="details"><div>Schematic model of probable <span class="highlight" style="background-color:">pathway</span> of refractoriness of interferon- signal. Previous works have shown that 5,6-dimethylxanthenone-4-acetic acid (DMXAA) can activate TANK-binding kinase1 (TBK1) to initiate interferon (IFN) regulatory factor 3 (IRF3) <span class="highlight" style="background-color:">signaling</span> and nucleotide oligomerization domain 1 (NOD1) that interacts with rip-like interacting caspase-like apoptosis-regulatory protein kinase to activate nuclear factor-B (NF-B) signal <span class="highlight" style="background-color:">pathway</span> that regulates production of inflammatory cytokines/chemokines[11-13]. Our results indicate that DMXAA can induce refractoriness of interferon- signal through NOD1 tolerance in mouse respiratory epithelial cells.</div></p></div><div class="aux"><div class="resc">Zaifang Yu, et al. World J Biol Chem. 2013 May 26;4(2):18-29.</div><p class="links"><a class="dblinks" href="/pubmed/23710296" ref="ordinalpos=2182&amp;ncbi_uid=4924986&amp;link_uid=23710296">Citation</a><a class="dblinks" href="/pmc/articles/PMC3652647/" ref="ordinalpos=2182&amp;ncbi_uid=4924986&amp;link_uid=PMC3652647">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2183.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2913371/figure/F8-8685/" ref="ordinalpos=2183&amp;ncbi_uid=3124798&amp;link_uid=PMC2913371" href="/pmc/articles/PMC2913371/"><img src="/pmc/articles/instance/2913371/bin/zjh0081086850008.gif" src-large="/pmc/articles/instance/2913371/bin/zjh0081086850008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2913371/" ref="ordinalpos=2183&amp;ncbi_uid=3124798&amp;link_uid=PMC2913371" image-link="/pmc/articles/PMC2913371/figure/F8-8685/" class="imagepopup">Figure 8.  From: Morphine Suppresses Tumor Angiogenesis through a HIF-1?/p38MAPK <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>A <span class="highlight" style="background-color:">signaling</span> model for the effect of morphine on hypoxia-induced HIF-1. Based on our current findings we propose that morphine alters the hypoxic activation of the p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Hypoxia increased p38 MAPK activity, resulting in decreased HIF-1 hydroxylation and increased phosphorylation leading to an increased HIF-1 nuclear accumulation. Morphine treatment reduced hypoxia induced MAPK p38 and thus HIF-1 activation.</div></p></div><div class="aux"><div class="resc">Lisa Koodie, et al. Am J Pathol. 2010 August;177(2):984-997.</div><p class="links"><a class="dblinks" href="/pubmed/20616349" ref="ordinalpos=2183&amp;ncbi_uid=3124798&amp;link_uid=20616349">Citation</a><a class="dblinks" href="/pmc/articles/PMC2913371/" ref="ordinalpos=2183&amp;ncbi_uid=3124798&amp;link_uid=PMC2913371">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2184.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2980340/figure/fig1/" ref="ordinalpos=2184&amp;ncbi_uid=2586039&amp;link_uid=PMC2980340" href="/pmc/articles/PMC2980340/"><img src="/pmc/articles/instance/2980340/bin/adt.2009.0198_fig_01.gif" src-large="/pmc/articles/instance/2980340/bin/adt.2009.0198_fig_01.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2980340/" ref="ordinalpos=2184&amp;ncbi_uid=2586039&amp;link_uid=PMC2980340" image-link="/pmc/articles/PMC2980340/figure/fig1/" class="imagepopup">Fig. 1. From: Discovery of Chemical Modulators of a Conserved Translational Control <span class="highlight" style="background-color:">Pathway</span> by Parallel Screening in Yeast. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> to <i>GCN4</i> via eIF2B. (<b>Left</b>) In wild-type cells amino acids (aa) repress kinase Gcn2p, maintaining low eIF2 phosphorylation (eIF2P) and active eIF2B. This represses <i>GCN4</i> translation. (<b>Middle</b>) Starvation for one or more aa activates Gcn2p, inhibiting eIF2B and activating translation of <i>GCN4</i> and transcription of its target genes. (<b>Right</b>) Mutations in eIF2B (eIF2B<sup>M</sup>) override this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and activate <i>GCN4</i> constitutively.</div></p></div><div class="aux"><div class="resc">Nuzhat Motlekar, et al. Assay Drug Dev Technol. 2009 October;7(5):479-494.</div><p class="links"><a class="dblinks" href="/pubmed/19715453" ref="ordinalpos=2184&amp;ncbi_uid=2586039&amp;link_uid=19715453">Citation</a><a class="dblinks" href="/pmc/articles/PMC2980340/" ref="ordinalpos=2184&amp;ncbi_uid=2586039&amp;link_uid=PMC2980340">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2185.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2575623/figure/F7/" ref="ordinalpos=2185&amp;ncbi_uid=1525461&amp;link_uid=PMC2575623" href="/pmc/articles/PMC2575623/"><img src="/pmc/articles/instance/2575623/bin/ar2451-7.gif" src-large="/pmc/articles/instance/2575623/bin/ar2451-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2575623/" ref="ordinalpos=2185&amp;ncbi_uid=1525461&amp;link_uid=PMC2575623" image-link="/pmc/articles/PMC2575623/figure/F7/" class="imagepopup">Figure 7.  From: Low-intensity pulsed ultrasound activates the phosphatidylinositol 3 kinase/Akt <span class="highlight" style="background-color:">pathway</span> and stimulates the growth of chondrocytes in three-dimensional cultures: a basic science study. </a></p><div class="supp"><p class="details"><div><b>Signal transduction pathways activated by low-intensity pulsed ultrasound</b>. Area enclosed with a black broken line is the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> specified in the present experiment. One of the receptors of low-intensity pulsed ultrasound (LIPUS) is through integrin, and the integrin/mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">pathway</span> is activated to the same extent in both the LIPUS group and the control group. The integrin/phosphatidylinositol 3 kinase (PI3K)/Akt <span class="highlight" style="background-color:">pathway</span>, however, was further activated by LIPUS. The expression of -catenin, which is downstream of the Akt <span class="highlight" style="background-color:">signaling</span> pathways, is also increased by LIPUS. FAK, focal adhesion kinase; GSK-3, glycogen synthase kinase-3; Pax, Paxillin.</div></p></div><div class="aux"><div class="resc">Ryohei Takeuchi, et al. Arthritis Res Ther. 2008;10(4):R77-R77.</div><p class="links"><a class="dblinks" href="/pubmed/18616830" ref="ordinalpos=2185&amp;ncbi_uid=1525461&amp;link_uid=18616830">Citation</a><a class="dblinks" href="/pmc/articles/PMC2575623/" ref="ordinalpos=2185&amp;ncbi_uid=1525461&amp;link_uid=PMC2575623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2186.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4232733/figure/Fig1/" ref="ordinalpos=2186&amp;ncbi_uid=6844175&amp;link_uid=PMC4232733" href="/pmc/articles/PMC4232733/"><img src="/pmc/articles/instance/4232733/bin/12885_2014_4985_Fig1_HTML.gif" src-large="/pmc/articles/instance/4232733/bin/12885_2014_4985_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4232733/" ref="ordinalpos=2186&amp;ncbi_uid=6844175&amp;link_uid=PMC4232733" image-link="/pmc/articles/PMC4232733/figure/Fig1/" class="imagepopup">Figure 1.  From: Association between insulin-like growth factor-1 receptor (IGF1R) negativity and poor prognosis in a cohort of women with primary breast cancer. </a></p><div class="supp"><p class="details"><div> <b>Schematic illustration of the IGF1R/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> resulting in growth and survival of the cell.</b> Examples of cross-talk between the IGF1R <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and estrogen and the estrogen receptor (ER) are shown.</div></p></div><div class="aux"><div class="resc">Kristina E Aaltonen, et al. BMC Cancer. 2014;14(1):794.</div><p class="links"><a class="dblinks" href="/pubmed/25362932" ref="ordinalpos=2186&amp;ncbi_uid=6844175&amp;link_uid=25362932">Citation</a><a class="dblinks" href="/pmc/articles/PMC4232733/" ref="ordinalpos=2186&amp;ncbi_uid=6844175&amp;link_uid=PMC4232733">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2187.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3863673/figure/F5/" ref="ordinalpos=2187&amp;ncbi_uid=5612065&amp;link_uid=PMC3863673" href="/pmc/articles/PMC3863673/"><img src="/pmc/articles/instance/3863673/bin/ceo-6-237-g005.gif" src-large="/pmc/articles/instance/3863673/bin/ceo-6-237-g005.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3863673/" ref="ordinalpos=2187&amp;ncbi_uid=5612065&amp;link_uid=PMC3863673" image-link="/pmc/articles/PMC3863673/figure/F5/" class="imagepopup">Fig. 5.  From: Effect of Epigallocatechin-3-Gallate on PMA-Induced MUC5B Expression in Human Airway Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The schematic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of EGCG on PMA-induced MUC5B expression. EGCG can down-regulate PMA-induced MUC5B expression, which is associated with p38 MAPK dependent MMP-9 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. PMA, phorbol 12-myristate 13-acetate; EGCG, epigallocatechin-3-gallate; MAPK, mitogen-activated protein kinase; MMP-9, matrix metalloproteinase-9.</div></p></div><div class="aux"><div class="resc">Eun Jung Song, et al. Clin Exp Otorhinolaryngol. 2013 December;6(4):237-242.</div><p class="links"><a class="dblinks" href="/pubmed/24353864" ref="ordinalpos=2187&amp;ncbi_uid=5612065&amp;link_uid=24353864">Citation</a><a class="dblinks" href="/pmc/articles/PMC3863673/" ref="ordinalpos=2187&amp;ncbi_uid=5612065&amp;link_uid=PMC3863673">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2188.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4109802/figure/F2/" ref="ordinalpos=2188&amp;ncbi_uid=6413001&amp;link_uid=PMC4109802" href="/pmc/articles/PMC4109802/"><img src="/pmc/articles/instance/4109802/bin/nihms589452f2.gif" src-large="/pmc/articles/instance/4109802/bin/nihms589452f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4109802/" ref="ordinalpos=2188&amp;ncbi_uid=6413001&amp;link_uid=PMC4109802" image-link="/pmc/articles/PMC4109802/figure/F2/" class="imagepopup">Figure 2. Scheme of neuroimmunomodulation of CD200 <span class="highlight" style="background-color:">signaling</span> by FGF2 <span class="highlight" style="background-color:">signaling</span>.  From: Fibroblast growth factor-2 <span class="highlight" style="background-color:">signaling</span> in neurogenesis and neurodegeneration. </a></p><div class="supp"><p class="details"><div>Activation of FGFR leads to enhanced CD200 expression, which leads to subsequent activation of anti-inflammatory pathways via microglial CD200 receptor (CD200R) activation. This <span class="highlight" style="background-color:">pathway</span> may be disturbed in AD patients due to CD200 decrease by amyloid- (A), pathogen-associated molecular patterns (PAMPs), or damage-associated molecular patterns (DAMPs) recognized by microglia, leading to neuroinflammation-induced suppression of LTP, synaptic transmission and neuronal viability in AD brain.</div></p></div><div class="aux"><div class="resc">Maya E. Woodbury, et al. J Neuroimmune Pharmacol. ;9(2):92-101.</div><p class="links"><a class="dblinks" href="/pubmed/24057103" ref="ordinalpos=2188&amp;ncbi_uid=6413001&amp;link_uid=24057103">Citation</a><a class="dblinks" href="/pmc/articles/PMC4109802/" ref="ordinalpos=2188&amp;ncbi_uid=6413001&amp;link_uid=PMC4109802">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2189.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig6/" ref="ordinalpos=2189&amp;ncbi_uid=662988&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F06_LW.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F06_LW.jpg" alt="FIGURE 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=2189&amp;ncbi_uid=662988&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig6/" class="imagepopup">FIGURE 6.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span>, metabolic, and transcriptional regulatory networks are interconnected. For example, ATP and GTP, primary currencies of <span class="highlight" style="background-color:">signaling</span> networks, are principal products of metabolic networks.</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=2189&amp;ncbi_uid=662988&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=2189&amp;ncbi_uid=662988&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2190.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2824644/figure/F7/" ref="ordinalpos=2190&amp;ncbi_uid=2262549&amp;link_uid=PMC2824644" href="/pmc/articles/PMC2824644/"><img src="/pmc/articles/instance/2824644/bin/nihms70483f7.gif" src-large="/pmc/articles/instance/2824644/bin/nihms70483f7.jpg" alt="FIGURE 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2824644/" ref="ordinalpos=2190&amp;ncbi_uid=2262549&amp;link_uid=PMC2824644" image-link="/pmc/articles/PMC2824644/figure/F7/" class="imagepopup">FIGURE 7. Distinct modes of IKK complex activity transduce TNF and IL-1 <span class="highlight" style="background-color:">signaling</span> to NF-B.  From: Interleukin-1-induced NF-B Activation Is NEMO-dependent but Does Not Require IKK. </a></p><div class="supp"><p class="details"><div>Our findings verify that TNF-induced I<i></i>B<i></i> degradation and classical NF-<i></i>B activation is critically dependent upon NEMO and IKK<i></i> (<i>left</i>). Similarly, IL-1 <span class="highlight" style="background-color:">signaling</span> absolutely requires intact NEMO (<i>right</i>). However, IL-1-induced I<i></i>B<i></i> degradation, NF-<i></i>B nuclear translocation, and NF-<i></i>B transcriptional activity occurs in the absence of IKK<i></i>, demonstrating that NEMO and IKK<i></i> can form a <span class="highlight" style="background-color:">signaling</span> complex capable of activating the classical NF-<i></i>B <span class="highlight" style="background-color:">pathway</span> in response to certain pro-inflammatory stimuli.</div></p></div><div class="aux"><div class="resc">Laura A. Solt, et al. J Biol Chem. ;282(12):8724-8733.</div><p class="links"><a class="dblinks" href="/pubmed/17244613" ref="ordinalpos=2190&amp;ncbi_uid=2262549&amp;link_uid=17244613">Citation</a><a class="dblinks" href="/pmc/articles/PMC2824644/" ref="ordinalpos=2190&amp;ncbi_uid=2262549&amp;link_uid=PMC2824644">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2191.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3262803/figure/pone-0030215-g008/" ref="ordinalpos=2191&amp;ncbi_uid=3772710&amp;link_uid=PMC3262803" href="/pmc/articles/PMC3262803/"><img src="/pmc/articles/instance/3262803/bin/pone.0030215.g008.gif" src-large="/pmc/articles/instance/3262803/bin/pone.0030215.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3262803/" ref="ordinalpos=2191&amp;ncbi_uid=3772710&amp;link_uid=PMC3262803" image-link="/pmc/articles/PMC3262803/figure/pone-0030215-g008/" class="imagepopup">Figure 8. Schematic model for the mechanism of JNK mediated activation of the p53 <span class="highlight" style="background-color:">pathway</span> in RITA-induced MM cells..  From: Targeting p53 via JNK <span class="highlight" style="background-color:">Pathway</span>: A Novel Role of RITA for Apoptotic <span class="highlight" style="background-color:">Signaling</span> in Multiple Myeloma. </a></p><div class="supp"><p class="details"><div>RITA induces the activation of p53 through c-Jun/JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by inducing phosphorylation of ASK1-MKK-4 and c-Jun which in turn induces apoptosis by up-regulation of p53 and Noxa; downregulation of Mcl-1, and cleavage of caspase-8, caspase-3 and PARP. Pharmacological (SP600125) or genetic inhibition (JNK siRNA) of activation of c-Jun inhibits RITA-induced activation of p53 and attenuate apoptosis induction by RITA. In addition, treatment of MM cells with p53-transcriptional inhibitor, PFT- or p53 siRNA abrogates RITA-induced activation of c-Jun and induction of apoptosis. These results suggest the presence of a positive feedback loop between p53 and JNK. Depending on the cellular context, p53 can be directly activated by JNK, although the mechanism is not clear.</div></p></div><div class="aux"><div class="resc">Manujendra N. Saha, et al. PLoS One. 2012;7(1):e30215.</div><p class="links"><a class="dblinks" href="/pubmed/22276160" ref="ordinalpos=2191&amp;ncbi_uid=3772710&amp;link_uid=22276160">Citation</a><a class="dblinks" href="/pmc/articles/PMC3262803/" ref="ordinalpos=2191&amp;ncbi_uid=3772710&amp;link_uid=PMC3262803">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2192.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3022534/figure/pbio-1000582-g005/" ref="ordinalpos=2192&amp;ncbi_uid=2725698&amp;link_uid=PMC3022534" href="/pmc/articles/PMC3022534/"><img src="/pmc/articles/instance/3022534/bin/pbio.1000582.g005.gif" src-large="/pmc/articles/instance/3022534/bin/pbio.1000582.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3022534/" ref="ordinalpos=2192&amp;ncbi_uid=2725698&amp;link_uid=PMC3022534" image-link="/pmc/articles/PMC3022534/figure/pbio-1000582-g005/" class="imagepopup">Figure 5. Expression sites of Wnt <span class="highlight" style="background-color:">signaling</span> components in the E14.5 mouse kidney..  From: A High-Resolution Anatomical Atlas of the Transcriptome in the Mouse Embryo. </a></p><div class="supp"><p class="details"><div>(A) The matrix shows the level of expression of all 58 regionally expressed genes in ten different renal structures that are defined in (B). Colors represent expression strength: strong (red), moderate (light red), weak (pink), and not detected (white). The Wnt <span class="highlight" style="background-color:">signaling</span> components are grouped into seven blocks (ligands, receptors, extracellular inhibitors, canonical <span class="highlight" style="background-color:">signaling</span>, Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span>, PCP <span class="highlight" style="background-color:">signaling</span>, and GO Wnt receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>). (B) The scheme in the center illustrates the ten main anatomical structures characterizing the developing kidney. The image gallery composed of low- and high-power (inset) images reveals that each of the ten structures characteristically expresses a particular Wnt component. 1: <i>Wnt7b</i>; 2: <i>Wnt11</i>; 3: <i>Dkk1</i>; 4: <i>Sfrp2</i>; 5: <i>Lrp6</i>; 6: <i>Slc9a3r1</i>; 7: <i>Tle4</i>; 8: <i>Tcf4</i>; 9: <i>Wnt5a</i>; 10: <i>Rspo3</i>. (C) Wnt <span class="highlight" style="background-color:">signaling</span> components involved in the mesenchyme-to-epithelium transition. <i>Wnt9b</i> is expressed in the ureteric bud (white arrowhead) and acts upstream of WNT4, which is expressed in condensing mesenchyme (black arrowhead). The Wnt receptors FZD3 (black arrowhead) and FZD4 (black arrowhead) are expressed in a way that allows them to function as candidate transducers for WNT9B/WNT4 <span class="highlight" style="background-color:">signaling</span> and could possibly underlie a shift from canonical to noncanonical <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Graciana Diez-Roux, et al. PLoS Biol. 2011 January;9(1):e1000582.</div><p class="links"><a class="dblinks" href="/pubmed/21267068" ref="ordinalpos=2192&amp;ncbi_uid=2725698&amp;link_uid=21267068">Citation</a><a class="dblinks" href="/pmc/articles/PMC3022534/" ref="ordinalpos=2192&amp;ncbi_uid=2725698&amp;link_uid=PMC3022534">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2193.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3683424/figure/fig1/" ref="ordinalpos=2193&amp;ncbi_uid=5050832&amp;link_uid=PMC3683424" href="/pmc/articles/PMC3683424/"><img src="/pmc/articles/instance/3683424/bin/ECAM2013-849054.001.gif" src-large="/pmc/articles/instance/3683424/bin/ECAM2013-849054.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3683424/" ref="ordinalpos=2193&amp;ncbi_uid=5050832&amp;link_uid=PMC3683424" image-link="/pmc/articles/PMC3683424/figure/fig1/" class="imagepopup">Figure 1.  From: The Mechanisms of Chansu in Inducing Efficient Apoptosis in Colon Cancer Cells. </a></p><div class="supp"><p class="details"><div>Cytotoxic effects of CBF in human cancer cell lines. (a) Cytotoxicity assays of CBF. A variety of human cancer cell lines were exposed to CBF for 24 hours and exhibited a reduction in cell numbers. The EC<sub>50</sub> values were measured by using MTT assays. Results are means with standard errors from 9 replicates. (b) Annexin V/PI analysis by flow cytometry. HCT116 and HT29 cells were treated with 0 or 1<i></i>M CBF for 24 hours and stained with Annexin V/PI dyes. The lower right percentage indicated the apoptotic cells stained with Annexin V and represented a significant increment after exposure to CBF, as well as the proportion of dying cells stained with both dyes (upper right). No elevation in PI stained cells (upper left) was observed and a reduction was in living cell number (lower left) after CBF treatment. (c) The occurrence of mitochondrial permeabilisation in CBF-induced cell death. Untreated and CBF-treated HCT116 and HT29 cells were stained with JC-1 dye, which shows red in living cells and green in dying cells. The fraction (green/red) between treated and untreated cells indicated the change of mitochondrial potential transition. Results are means with standard errors from 3 replicates. (d) Deregulation in cytotoxicity <span class="highlight" style="background-color:">signaling</span> multipathway arrays after CBF treatment. The most significant change is in HIF-1<i></i>-regulated <span class="highlight" style="background-color:">pathway</span>, in which HIF-1<i></i> level was 2.8 times as low as the untreated cells. On the other hand, the most upregulation occurred in the MAPK/ERK <span class="highlight" style="background-color:">pathway</span>, where the expression of SRF/Elk-1 increased by 2.4 times. The logarithm of untreated cell luciferase intensity was set to baseline. p53, <span class="highlight" style="background-color:">pathway</span> of p53/DNA damage; HIF-1, <span class="highlight" style="background-color:">pathway</span> of hypoxia; NF<i></i>B, <span class="highlight" style="background-color:">pathway</span> of NF<i></i>B; GR, <span class="highlight" style="background-color:">pathway</span> of glucocorticoid receptor; E2F, <span class="highlight" style="background-color:">pathway</span> of cell cycle/pRb-E2F; SRF/Elk-1, <span class="highlight" style="background-color:">pathway</span> of MAPK/ERK; AP-1, <span class="highlight" style="background-color:">pathway</span> of MAPK/JNK; NFAT, <span class="highlight" style="background-color:">pathway</span> of PKC/Ca<sup>2+</sup>; SMAD2/SMAD3/SMAD4, <span class="highlight" style="background-color:">pathway</span> of TGF; and Myc/Max, <span class="highlight" style="background-color:">pathway</span> of Myc/Max. Results are means with standard errors from 4 replicates.</div></p></div><div class="aux"><div class="resc">Chun Li, et al. Evid Based Complement Alternat Med. 2013;2013:849054.</div><p class="links"><a class="dblinks" href="/pubmed/23818933" ref="ordinalpos=2193&amp;ncbi_uid=5050832&amp;link_uid=23818933">Citation</a><a class="dblinks" href="/pmc/articles/PMC3683424/" ref="ordinalpos=2193&amp;ncbi_uid=5050832&amp;link_uid=PMC3683424">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2194.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2958907/figure/F1/" ref="ordinalpos=2194&amp;ncbi_uid=2527109&amp;link_uid=PMC2958907" href="/pmc/articles/PMC2958907/"><img src="/pmc/articles/instance/2958907/bin/1752-0509-4-137-1.gif" src-large="/pmc/articles/instance/2958907/bin/1752-0509-4-137-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2958907/" ref="ordinalpos=2194&amp;ncbi_uid=2527109&amp;link_uid=PMC2958907" image-link="/pmc/articles/PMC2958907/figure/F1/" class="imagepopup">Figure 1.  From: <span class="highlight" style="background-color:">Signaling</span> network of dendritic cells in response to pathogens: a community-input supported knowledgebase. </a></p><div class="supp"><p class="details"><div><b>Web-accessible DC <span class="highlight" style="background-color:">pathway</span> navigator</b>. Snapshot of the <span class="highlight" style="background-color:">pathway</span> navigator that displays a selected view of the DC <span class="highlight" style="background-color:">signaling</span> map. Entities are clickable, such that corresponding annotations, namely the interactions in which those entities are involved, and the related PMIDs are displayed in the bottom left-hand frame. Additionally, other frames include an Interaction list, a Protein list, a Gene list, and an RNA list. Hyperlinks to the relevant NCBI Entrez Gene page(s) and DC wiki pages are also provided. A zoom rectangle located in the upper left corner of the image facilitates navigation through the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Sonali Patil, et al. BMC Syst Biol. 2010;4:137-137.</div><p class="links"><a class="dblinks" href="/pubmed/20929569" ref="ordinalpos=2194&amp;ncbi_uid=2527109&amp;link_uid=20929569">Citation</a><a class="dblinks" href="/pmc/articles/PMC2958907/" ref="ordinalpos=2194&amp;ncbi_uid=2527109&amp;link_uid=PMC2958907">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2195.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3146006/figure/F3/" ref="ordinalpos=2195&amp;ncbi_uid=3147639&amp;link_uid=PMC3146006" href="/pmc/articles/PMC3146006/"><img src="/pmc/articles/instance/3146006/bin/lar-27-117-g003.gif" src-large="/pmc/articles/instance/3146006/bin/lar-27-117-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3146006/" ref="ordinalpos=2195&amp;ncbi_uid=3147639&amp;link_uid=PMC3146006" image-link="/pmc/articles/PMC3146006/figure/F3/" class="imagepopup">Figure 3.  From: Effects of Steaming Time and Frequency for Manufactured Red Liriope platyphylla on the Insulin Secretion Ability and Insulin Receptor <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of the different steaming times of <i>Liriope platyphylla</i> on the down-stream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of the insulin receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via Western blotting. Total cell lysates were prepared from INS cells treated with vehicle, KWG, KRG and five different LPs prepared under different steaming conditions, as described in the Materials and Methods section. Fifty micrograms of protein per sample were immunoblotted with antibodies for each protein. Three samples were assayed in triplicate via Western blotting. The values are expressed as meansSD. <sup>*</sup><i>P</i>&lt;0.05 is the significance level compared to the vehicle-treated group.</div></p></div><div class="aux"><div class="resc">Sun Il Choi, et al. Lab Anim Res. 2011 June;27(2):117-126.</div><p class="links"><a class="dblinks" href="/pubmed/21826171" ref="ordinalpos=2195&amp;ncbi_uid=3147639&amp;link_uid=21826171">Citation</a><a class="dblinks" href="/pmc/articles/PMC3146006/" ref="ordinalpos=2195&amp;ncbi_uid=3147639&amp;link_uid=PMC3146006">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2196.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2896193/figure/F4/" ref="ordinalpos=2196&amp;ncbi_uid=2389666&amp;link_uid=PMC2896193" href="/pmc/articles/PMC2896193/"><img src="/pmc/articles/instance/2896193/bin/gkq424f4.gif" src-large="/pmc/articles/instance/2896193/bin/gkq424f4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2896193/" ref="ordinalpos=2196&amp;ncbi_uid=2389666&amp;link_uid=PMC2896193" image-link="/pmc/articles/PMC2896193/figure/F4/" class="imagepopup">Figure 4. From: Discovering causal <span class="highlight" style="background-color:">signaling</span> pathways through gene-expression patterns. </a></p><div class="supp"><p class="details"><div>SPEED analysis of downregulated genes in high-risk AML patients with CEBPa mutations. Screenshot of SPEED results shows TNF- as the top-ranking upstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for target genes downregulated in high-risk AML patients with CEBPa mutations versus CEBPa wild-type. The pathways are sorted by FDR and graphical outputs aid in interpretation of the results.</div></p></div><div class="aux"><div class="resc">Jignesh R. Parikh, et al. Nucleic Acids Res. 2010 July 1;38(Web Server issue):W109-W117.</div><p class="links"><a class="dblinks" href="/pubmed/20494976" ref="ordinalpos=2196&amp;ncbi_uid=2389666&amp;link_uid=20494976">Citation</a><a class="dblinks" href="/pmc/articles/PMC2896193/" ref="ordinalpos=2196&amp;ncbi_uid=2389666&amp;link_uid=PMC2896193">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2197.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2129233/figure/F2/" ref="ordinalpos=2197&amp;ncbi_uid=1106668&amp;link_uid=PMC2129233" href="/pmc/articles/PMC2129233/"><img src="/pmc/articles/instance/2171136/bin/JCI0832086.f2.gif" src-large="/pmc/articles/instance/2171136/bin/JCI0832086.f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2129233/" ref="ordinalpos=2197&amp;ncbi_uid=1106668&amp;link_uid=PMC2129233" image-link="/pmc/articles/PMC2129233/figure/F2/" class="imagepopup">Figure 2. The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is active in CCRCC cells..  From: Suppression of renal cell carcinoma growth by inhibition of Notch <span class="highlight" style="background-color:">signaling</span> in vitro and in vivo. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Inhibition of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 786-O cells with a single concentration (10 M) of the -secretase inhibitor DAPT for indicated time periods as monitored by Hes-1 levels. Cells were harvested at indicated time points, and cell lysates were analyzed by immunoblotting. (<b>B</b>) Inhibition of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in 786-O cells with increasing concentrations of the -secretase inhibitor DAPT for 24 hours. Immunoblotting experiments to measure expression levels of the Notch target Hes-1. (<b>C</b>) The effects of L-685458 and DAPT on Hes-1 protein levels in 786-O cells treated for 24 hours compared with DMSO-treated () cells. (<b>D</b>) DAPT (+) treatment compared with vehicle control () treatment of PRC3 and WT7 cells. Cells were harvested after 24 hours of treatment, and cell lysates were analyzed for Hes-1 protein expression. (<b>E</b>) DAPT (+) treatment compared with vehicle control () treatment in a panel of CCRCC cells. Cells were harvested after 24 hours of treatment, and cell lysates were analyzed for Hes-1 protein expression. (<b>F</b>) <i>Hes-1</i> mRNA levels assessed by Q-PCR in DAPT-treated CCRCC cells. Cells were harvested after 24 hours of treatment. DMSO-treated samples were designated as 100%, and data shown are mean + SD of representative experiment performed in triplicate.                </div></p></div><div class="aux"><div class="resc">Jonas Sjlund, et al. J Clin Invest. 2008 January 2;118(1):217-228.</div><p class="links"><a class="dblinks" href="/pubmed/18079963" ref="ordinalpos=2197&amp;ncbi_uid=1106668&amp;link_uid=18079963">Citation</a><a class="dblinks" href="/pmc/articles/PMC2129233/" ref="ordinalpos=2197&amp;ncbi_uid=1106668&amp;link_uid=PMC2129233">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2198.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3398046/figure/pone-0040676-g005/" ref="ordinalpos=2198&amp;ncbi_uid=4129239&amp;link_uid=PMC3398046" href="/pmc/articles/PMC3398046/"><img src="/pmc/articles/instance/3398046/bin/pone.0040676.g005.gif" src-large="/pmc/articles/instance/3398046/bin/pone.0040676.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3398046/" ref="ordinalpos=2198&amp;ncbi_uid=4129239&amp;link_uid=PMC3398046" image-link="/pmc/articles/PMC3398046/figure/pone-0040676-g005/" class="imagepopup">Figure 5. Functional <span class="highlight" style="background-color:">pathway</span> network analysis of 346 differentially regulated mRNAs..  From: MicroRNA Profiling in Mucosal Biopsies of Eosinophilic Esophagitis Patients Pre and Post Treatment with Steroids and Relationship with mRNA Targets. </a></p><div class="supp"><p class="details"><div>One of the primary <span class="highlight" style="background-color:">pathway</span> networks is associated with network functions of Inflammatory Response, Cell-To-Cell <span class="highlight" style="background-color:">Signaling</span> and Interaction, Hematological System Development and Function.</div></p></div><div class="aux"><div class="resc">Shaolei Lu, et al. PLoS One. 2012;7(7):e40676.</div><p class="links"><a class="dblinks" href="/pubmed/22815788" ref="ordinalpos=2198&amp;ncbi_uid=4129239&amp;link_uid=22815788">Citation</a><a class="dblinks" href="/pmc/articles/PMC3398046/" ref="ordinalpos=2198&amp;ncbi_uid=4129239&amp;link_uid=PMC3398046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2199.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3827948/figure/F6/" ref="ordinalpos=2199&amp;ncbi_uid=5486664&amp;link_uid=PMC3827948" href="/pmc/articles/PMC3827948/"><img src="/pmc/articles/instance/3827948/bin/nihms494905f6.gif" src-large="/pmc/articles/instance/3827948/bin/nihms494905f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3827948/" ref="ordinalpos=2199&amp;ncbi_uid=5486664&amp;link_uid=PMC3827948" image-link="/pmc/articles/PMC3827948/figure/F6/" class="imagepopup">Fig. 6.  From: Syndecan 4 Regulates FGFR1 <span class="highlight" style="background-color:">Signaling</span> in Endothelial Cells by Directing Macropinocytosis. </a></p><div class="supp"><p class="details"><div>S4 controls FGF2-mediated <span class="highlight" style="background-color:">signaling</span> through RhoG-dependent macropinocytosis of the FGFR1-S4-FGF2 <span class="highlight" style="background-color:">signaling</span> complex. (<b>A</b>) Before FGFR1 and S4 are activated by FGF2, RhoG is kept inactive by S4 (in complex with synectin and RhoGDI1), and Rab5 activity and phosphorylation of ERK1/2 are low at baseline. (<b>B</b>) Upon ligand binding, FGFR1 and S4 form a <span class="highlight" style="background-color:">signaling</span> complex, and independent <span class="highlight" style="background-color:">signaling</span> pathways are initiated by each co-receptor: Tyrosine kinase phosphorylation of FGFR1 activates several parallel <span class="highlight" style="background-color:">signaling</span> networks, and S4 releases inactive RhoG to become activated by GEFs such as SGEF. This induces membrane ruffling and subsequent macropinocytosis of the entire receptor complex. In the absence of functional Rab5, nascent vesicles cannot mature and become fully functional <span class="highlight" style="background-color:">signaling</span> endosomes, which leads to inadequate activation of the MAPK <span class="highlight" style="background-color:">pathway</span> (as indicated by diminished phosphorylation of ERK1/2). (<b>C</b>) Physiological <span class="highlight" style="background-color:">signaling</span> (designated by a double-border) is restored as functional Rab5 localizes to nascent macropinosomes containing FGFR1-S4-FGF2 and becomes activated, facilitating vesicle maturation. This permits effective activation of the MAPK <span class="highlight" style="background-color:">pathway</span> (as indicated by ERK1/2 phosphorylation). (<b>D</b>) In the absence of S4 or S4s cytoplasmic PDZ domain, RhoG activity is high at baseline, causing increased membrane ruffling and macropinocytosis. Enhanced macropinocytosis leads to increased co-receptor internalization. Rab5 promotes vesicular maturation, leading to potentiated ERK1/2 activity with slower deactivation kinetics upon FGF2 stimulation. Green and black denote activation, and purple denotes the process of vesicular maturation into early/<span class="highlight" style="background-color:">signaling</span> endosomes.</div></p></div><div class="aux"><div class="resc">Arye Elfenbein, et al. Sci Signal. 2012 May 8;5(223):10.1126/scisignal.2002495.</div><p class="links"><a class="dblinks" href="/pubmed/22569333" ref="ordinalpos=2199&amp;ncbi_uid=5486664&amp;link_uid=22569333">Citation</a><a class="dblinks" href="/pmc/articles/PMC3827948/" ref="ordinalpos=2199&amp;ncbi_uid=5486664&amp;link_uid=PMC3827948">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>2200.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2809802/figure/F1/" ref="ordinalpos=2200&amp;ncbi_uid=2731163&amp;link_uid=PMC2809802" href="/pmc/articles/PMC2809802/"><img src="/pmc/articles/instance/2809802/bin/nihms-162035-f0001.gif" src-large="/pmc/articles/instance/2809802/bin/nihms-162035-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2809802/" ref="ordinalpos=2200&amp;ncbi_uid=2731163&amp;link_uid=PMC2809802" image-link="/pmc/articles/PMC2809802/figure/F1/" class="imagepopup">Figure 1. Schematic representation of agonist-selective <span class="highlight" style="background-color:">signaling</span>.  From: Agonist-Selective <span class="highlight" style="background-color:">Signaling</span> of G Protein-Coupled Receptor: Mechanisms and Implications. </a></p><div class="supp"><p class="details"><div>Agonist I and II exhibit differential efficacies in <span class="highlight" style="background-color:">pathway</span> A and B. The Full Agonist and Neutral Antagonist were used to indicate the different intrinsic efficacies of these agonists on the two pathways. Also, they can be Partial Agonist or Inverse Agonist.</div></p></div><div class="aux"><div class="resc">Hui Zheng, et al. IUBMB Life. ;62(2):112-119.</div><p class="links"><a class="dblinks" href="/pubmed/20058265" ref="ordinalpos=2200&amp;ncbi_uid=2731163&amp;link_uid=20058265">Citation</a><a class="dblinks" href="/pmc/articles/PMC2809802/" ref="ordinalpos=2200&amp;ncbi_uid=2731163&amp;link_uid=PMC2809802">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="21" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="22" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="23" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:32:54-04:00&amp;Host=ptpmc202&amp;Data=+NCBI_Phid:+F4FC26905430F2F10000000000D8448B;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FC26905430F2F10000000000D8448B" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc202 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>