
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="35" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A202D543120810000000000471BF0" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="34" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="35" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="36" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="35" /></div>
            <div><h2 class="result_count">Results: 3401 to 3500 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>3401.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1531692/figure/F1/" ref="ordinalpos=3401&amp;ncbi_uid=743416&amp;link_uid=PMC1531692" href="/pmc/articles/PMC1531692/"><img src="/pmc/articles/instance/1531692/bin/neo0403_0191_fig001.gif" src-large="/pmc/articles/instance/1531692/bin/neo0403_0191_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1531692/" ref="ordinalpos=3401&amp;ncbi_uid=743416&amp;link_uid=PMC1531692" image-link="/pmc/articles/PMC1531692/figure/F1/" class="imagepopup">Figure 1.  From: The Role of αvβ3 in Prostate Cancer Progression. </a></p><div class="supp"><p class="details"><div>α<sub>v</sub>β<sub>3</sub> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells. Ligation of α<sub>v</sub>β<sub>3</sub> with multiple ligands (i.e., osteopontin, BSP, or vitronectin) activates FAK, which interacts and activates PI-3 kinase [23]. The products of PI-3 kinase activity recruit PKB/Akt to the cell membrane where it is activated by phosphorylation and it phosphorylates several substrates to elicit a variety of biological responses, including cell survival, adhesion, and migration. Although NF-κB is potentially involved in the α<sub>v</sub>β<sub>3</sub> signal transduction <span class="highlight" style="background-color:">pathway</span> [8], its role in this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is not known for prostate cancer cells.</div></p></div><div class="aux"><div class="resc">Carlton R Cooper, et al. Neoplasia. 2002 May;4(3):191-194.</div><p class="links"><a class="dblinks" href="/pubmed/11988838" ref="ordinalpos=3401&amp;ncbi_uid=743416&amp;link_uid=11988838">Citation</a><a class="dblinks" href="/pmc/articles/PMC1531692/" ref="ordinalpos=3401&amp;ncbi_uid=743416&amp;link_uid=PMC1531692">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3402.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2649808/figure/f5/" ref="ordinalpos=3402&amp;ncbi_uid=1733300&amp;link_uid=PMC2649808" href="/pmc/articles/PMC2649808/"><img src="/pmc/articles/instance/2649808/bin/zjw0120833050005.gif" src-large="/pmc/articles/instance/2649808/bin/zjw0120833050005.jpg" alt="F &lt;span style=&quot;font-variant: small-caps&quot; class=&quot;small-caps&quot;&gt;ig&lt;/span&gt; . 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2649808/" ref="ordinalpos=3402&amp;ncbi_uid=1733300&amp;link_uid=PMC2649808" image-link="/pmc/articles/PMC2649808/figure/f5/" class="imagepopup">F ig . 5. From: Comparative Proteomic Phenotyping of Cell Lines and Primary Cells to Assess Preservation of Cell Type-specific Functions . </a></p><div class="supp"><p class="details"><div><b>Phenotypic proteome comparison at the <span class="highlight" style="background-color:">pathway</span> level.</b> <i>A</i>, mapping of protein ratios on the fatty acid metabolism <span class="highlight" style="background-color:">pathway</span> reveals that almost the entire module is down-regulated several-fold in Hepa1–6. Proteins are color-coded according to their relative expression in the two cell types according to the scheme in Fig. 2. <i>B</i>, KEGG <span class="highlight" style="background-color:">pathway</span> mapping of ErbB and PI3K <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shows that they are equally present in primary cells and the cell line. <i>PI3K</i>, phosphatidylinositol 3-kinase.</div></p></div><div class="aux"><div class="resc">Cuiping Pan, et al. Mol Cell Proteomics. 2009 March;8(3):443-450.</div><p class="links"><a class="dblinks" href="/pubmed/18952599" ref="ordinalpos=3402&amp;ncbi_uid=1733300&amp;link_uid=18952599">Citation</a><a class="dblinks" href="/pmc/articles/PMC2649808/" ref="ordinalpos=3402&amp;ncbi_uid=1733300&amp;link_uid=PMC2649808">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3403.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3409633/figure/F1/" ref="ordinalpos=3403&amp;ncbi_uid=4663407&amp;link_uid=PMC3409633" href="/pmc/articles/PMC3409633/"><img src="/pmc/articles/instance/3409633/bin/zgb0031255630001.gif" src-large="/pmc/articles/instance/3409633/bin/zgb0031255630001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3409633/" ref="ordinalpos=3403&amp;ncbi_uid=4663407&amp;link_uid=PMC3409633" image-link="/pmc/articles/PMC3409633/figure/F1/" class="imagepopup">Figure 1.TLR <span class="highlight" style="background-color:">signaling</span> controls transcription and mRNA half-life..  From: Cell type- and stimulus-specific mechanisms for post-transcriptional control of neutrophil chemokine gene expression. </a></p><div class="supp"><p class="details"><div>TLR–ligand interaction intiates <span class="highlight" style="background-color:">signaling</span> by recruiting the TLR adaptor Myd88, leading to the assembly of a complex containing, additionally, IRAK1, IRAK4, TRAF6, and TGF-β-activated kinase 1 (TAK1). The signal bifurcates to activation of NF-κB through the IKK complex or to mRNA stabilization via the p38 MAPK <span class="highlight" style="background-color:">pathway</span>. MAPKAP2 phosphorylates TTP and inactivates its mRNA decay function.</div></p></div><div class="aux"><div class="resc">Thomas Hamilton, et al. J Leukoc Biol. 2012 March;91(3):377-383.</div><p class="links"><a class="dblinks" href="/pubmed/22167720" ref="ordinalpos=3403&amp;ncbi_uid=4663407&amp;link_uid=22167720">Citation</a><a class="dblinks" href="/pmc/articles/PMC3409633/" ref="ordinalpos=3403&amp;ncbi_uid=4663407&amp;link_uid=PMC3409633">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3404.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3341116/figure/F9/" ref="ordinalpos=3404&amp;ncbi_uid=4476524&amp;link_uid=PMC3341116" href="/pmc/articles/PMC3341116/"><img src="/pmc/articles/instance/3341116/bin/zh30021260890009.gif" src-large="/pmc/articles/instance/3341116/bin/zh30021260890009.jpg" alt="Fig. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3341116/" ref="ordinalpos=3404&amp;ncbi_uid=4476524&amp;link_uid=PMC3341116" image-link="/pmc/articles/PMC3341116/figure/F9/" class="imagepopup">Fig. 9. From: Tumor necrosis factor receptor 1 functions as a tumor suppressor. </a></p><div class="supp"><p class="details"><div>Putative <span class="highlight" style="background-color:">pathway</span> by which TNFR1 functions as a tumor suppressor. <i>A</i>: intact TNFR1 <span class="highlight" style="background-color:">signaling</span> modulates Wnt <span class="highlight" style="background-color:">signaling</span> or otherwise maintains β-catenin inactivated in the cytosol. Normal levels of TNF bind to normal numbers of both receptors and maintain gut homeostasis. <i>B</i>: absent TNFR1 <span class="highlight" style="background-color:">signaling</span>, increased levels of TNF bind to increased numbers of TNFR2 and result in increased epithelial cell proliferation. As well, possibly because of unmitigated Wnt <span class="highlight" style="background-color:">signaling</span>, activated β-catenin translocates to the nucleus, driving proliferation and neoplastic transformation.</div></p></div><div class="aux"><div class="resc">Fengqi Chang, et al. Am J Physiol Gastrointest Liver Physiol. 2012 January;302(2):G195-G206.</div><p class="links"><a class="dblinks" href="/pubmed/22052015" ref="ordinalpos=3404&amp;ncbi_uid=4476524&amp;link_uid=22052015">Citation</a><a class="dblinks" href="/pmc/articles/PMC3341116/" ref="ordinalpos=3404&amp;ncbi_uid=4476524&amp;link_uid=PMC3341116">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3405.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2724029/figure/F4/" ref="ordinalpos=3405&amp;ncbi_uid=1908168&amp;link_uid=PMC2724029" href="/pmc/articles/PMC2724029/"><img src="/pmc/articles/instance/2724029/bin/fnmol-02-007-g004.gif" src-large="/pmc/articles/instance/2724029/bin/fnmol-02-007-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2724029/" ref="ordinalpos=3405&amp;ncbi_uid=1908168&amp;link_uid=PMC2724029" image-link="/pmc/articles/PMC2724029/figure/F4/" class="imagepopup">Figure 4.  From: Major <span class="highlight" style="background-color:">Signaling</span> Pathways in Migrating Neuroblasts. </a></p><div class="supp"><p class="details"><div><b><span class="highlight" style="background-color:">Signaling</span> networks upregulated in migrating neuroblasts</b>. <b>(A)</b> Calmodulin-<span class="highlight" style="background-color:">signaling</span> network. <b>(B)</b> Akt1-DNA transcription network. Networks were identified by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis and subsequently modified using Bibliosphere, PathwayArchitect, GO and <span class="highlight" style="background-color:">pathway</span> databases. The intensity of the red color indicates the extent of gene upregulation. Connecting lines with arrowheads indicate activation of proteins, without arrowheads protein–protein interaction. Continuous and dashed lines are direct and indirect activation/interaction, respectively. Node description is indicated in the right upper corner.</div></p></div><div class="aux"><div class="resc">Konstantin Khodosevich, et al. Front Mol Neurosci. 2009;2:7.</div><p class="links"><a class="dblinks" href="/pubmed/19668709" ref="ordinalpos=3405&amp;ncbi_uid=1908168&amp;link_uid=19668709">Citation</a><a class="dblinks" href="/pmc/articles/PMC2724029/" ref="ordinalpos=3405&amp;ncbi_uid=1908168&amp;link_uid=PMC2724029">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3406.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1440875/figure/pgen-0020057-g005/" ref="ordinalpos=3406&amp;ncbi_uid=615434&amp;link_uid=PMC1440875" href="/pmc/articles/PMC1440875/"><img src="/pmc/articles/instance/1449898/bin/pgen.0020057.g005.gif" src-large="/pmc/articles/instance/1449898/bin/pgen.0020057.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1440875/" ref="ordinalpos=3406&amp;ncbi_uid=615434&amp;link_uid=PMC1440875" image-link="/pmc/articles/PMC1440875/figure/pgen-0020057-g005/" class="imagepopup">Figure 5. Analysis of Molecular <span class="highlight" style="background-color:">Signaling</span> Events Controlling Muscular Contraction in C. elegans.  From: Chemical Genetics Reveals an RGS/G-Protein Role in the Action of a Compound. </a></p><div class="supp"><p class="details"><div>The diagram shows the molecular components of the <span class="highlight" style="background-color:">signaling</span> pathways downstream of the muscarinic GPCRs in <i>C. elegans</i>. Alleles of genes encoding proteins in the <span class="highlight" style="background-color:">pathway</span> were tested for their effect on the Egl-d phenotype caused by treatment with BMS-192364. For the <span class="highlight" style="background-color:">pathway</span> members indicated by stars, certain alleles altered the response to treatment with BMS-192364. Specifically, resistance to the Egl-d phenotype was conferred by two gain-of-function alleles of <i>egl-19</i> and one of <i>egl-30,</i> and by two loss-of-function alleles of <i>eat-16</i> and one of <i>goa-1</i>.</div></p></div><div class="aux"><div class="resc">Kevin Fitzgerald, et al. PLoS Genet. 2006 April;2(4):e57.</div><p class="links"><a class="dblinks" href="/pubmed/16683034" ref="ordinalpos=3406&amp;ncbi_uid=615434&amp;link_uid=16683034">Citation</a><a class="dblinks" href="/pmc/articles/PMC1440875/" ref="ordinalpos=3406&amp;ncbi_uid=615434&amp;link_uid=PMC1440875">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3407.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3621011/figure/F1/" ref="ordinalpos=3407&amp;ncbi_uid=6087575&amp;link_uid=PMC3621011" href="/pmc/articles/PMC3621011/"><img src="/pmc/articles/instance/3621011/bin/nihms408962f1.gif" src-large="/pmc/articles/instance/3621011/bin/nihms408962f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3621011/" ref="ordinalpos=3407&amp;ncbi_uid=6087575&amp;link_uid=PMC3621011" image-link="/pmc/articles/PMC3621011/figure/F1/" class="imagepopup">Figure 1. Mechanisms of VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> inhibitors.  From: Cardiovascular Complications Associated with Novel Angiogenesis Inhibitors: Emerging Evidence and Evolving Perspectives. </a></p><div class="supp"><p class="details"><div>Multiple strategies have been employed to pharmacologically target the VEGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. One anti-VEGF monoclonal antibody has been developed (bevacizumab). Multiple tyrosine kinase inhibitors (TKIs) target the intracellular kinase domain on the VEGF receptors, as well as on other important tyrosine kinases. Five currently approved VSP inhibitors fall into this category (sunitinib, sorafenib, axitinib, pazopanib, and vandetanib). Numerous other TKIs are in various stages of development or trials. Other approaches to VSP inhibition that are currently in development or clinical trials include a soluble VEGF receptor, or VEGF-Trap, and a monoclonal antibody generated against VEGFR2.</div></p></div><div class="aux"><div class="resc">Steven M. Bair, et al. Trends Cardiovasc Med. 2013 May;23(4):104-113.</div><p class="links"><a class="dblinks" href="/pubmed/23290365" ref="ordinalpos=3407&amp;ncbi_uid=6087575&amp;link_uid=23290365">Citation</a><a class="dblinks" href="/pmc/articles/PMC3621011/" ref="ordinalpos=3407&amp;ncbi_uid=6087575&amp;link_uid=PMC3621011">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3408.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3186766/figure/pone-0024371-g002/" ref="ordinalpos=3408&amp;ncbi_uid=3275315&amp;link_uid=PMC3186766" href="/pmc/articles/PMC3186766/"><img src="/pmc/articles/instance/3186766/bin/pone.0024371.g002.gif" src-large="/pmc/articles/instance/3186766/bin/pone.0024371.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3186766/" ref="ordinalpos=3408&amp;ncbi_uid=3275315&amp;link_uid=PMC3186766" image-link="/pmc/articles/PMC3186766/figure/pone-0024371-g002/" class="imagepopup">Figure 2. Glutamate and Dopamine Receptor <span class="highlight" style="background-color:">Signaling</span> Pathways..  From: Gene Expression Analysis in the Thalamus and Cerebrum of Horses Experimentally Infected with West Nile Virus. </a></p><div class="supp"><p class="details"><div>(A) The diagrams represents <span class="highlight" style="background-color:">signaling</span> in the synaptic cleft (square = pre-synaptic neuron, oval = post-synaptic neuron, circle = glial cell). Green represents downregulation of pertinent receptors, red represents upregulation of pertinent receptors. WNV induces downregulation of glutamate receptors on the post-synaptic cleft as well as glutamate uptake receptors on glial cells. (B) Dopamine receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The graphic shown is for the exposure analysis, but was similar in all analyses. Green represents down-regulation of transcripts, red represents upregulation. Dopamine receptors (DRD5) and downstream <span class="highlight" style="background-color:">signaling</span> pathways as well as enzymes that create dopamine (TH) were downregulated, while MAO (enzyme that degrades dopamine) was upregulated. This network was generated through the use of IPA (Ingenuity Systems, www.ingenuity.com).</div></p></div><div class="aux"><div class="resc">Melissa A. Bourgeois, et al. PLoS One. 2011;6(10):e24371.</div><p class="links"><a class="dblinks" href="/pubmed/21991302" ref="ordinalpos=3408&amp;ncbi_uid=3275315&amp;link_uid=21991302">Citation</a><a class="dblinks" href="/pmc/articles/PMC3186766/" ref="ordinalpos=3408&amp;ncbi_uid=3275315&amp;link_uid=PMC3186766">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3409.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2775826/figure/fig4/" ref="ordinalpos=3409&amp;ncbi_uid=2100896&amp;link_uid=PMC2775826" href="/pmc/articles/PMC2775826/"><img src="/pmc/articles/instance/2775826/bin/gr4.gif" src-large="/pmc/articles/instance/2775826/bin/gr4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2775826/" ref="ordinalpos=3409&amp;ncbi_uid=2100896&amp;link_uid=PMC2775826" image-link="/pmc/articles/PMC2775826/figure/fig4/" class="imagepopup">Figure 4.  From: Mutations in TRPM1 Are a Common Cause of Complete Congenital Stationary Night Blindness. </a></p><div class="supp"><p class="details"><div>Rod <span class="highlight" style="background-color:">Signaling</span> in the Mammalian Retina Travels via Three Routes<br />(Ai) The primary rod <span class="highlight" style="background-color:">pathway</span> (rod → ON bipolar cell→ AII amacrine cell). Rods release glutamate onto rod ON bipolar cells, which provide an excitatory input to AII amacrine cells, which are electrically coupled to cone ON bipolar cells and inhibit cone OFF bipolar cells. <span class="highlight" style="background-color:">Signaling</span> in this <span class="highlight" style="background-color:">pathway</span> completely depends on the mGluR6 <span class="highlight" style="background-color:">signaling</span> cascade (B). This <span class="highlight" style="background-color:">pathway</span> is affected in cCNSB.<br />(Aii) The secondary rod <span class="highlight" style="background-color:">pathway</span> (rod → cone → bipolar cell). The electrical coupling between rods and cones allows the flow of rod signals into the cones and glutamate release into the cone ON and OFF bipolar cells.<br />(Aiii) The tertiary rod <span class="highlight" style="background-color:">pathway</span> (rod →OFF bipolar cell). A direct glutamatergic input from rods to cone OFF bipolar cell. Note that the latter two pathways are in principle independent of the rod ON bipolar cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (Ai). Red arrows indicate sign-inverting synapses, and green arrows indicate sign-preserving synapses. Abbreviations are as follows: BC, bipolar cell; AC, amacrine cell; GC, ganglion cell.<br />(B) Schematic representation of <span class="highlight" style="background-color:">signaling</span> between photoreceptors and bipolar cells (marked by dashed red circle in panel Ai) and the mGluR6 <span class="highlight" style="background-color:">signaling</span> cascade. Glutamate release from rods depends on the intracellular Ca<sup>2+</sup> concentration, which is regulated by an L-type voltage-gated Ca<sup>2+</sup> channel (<i>CACNA1F</i>) in the rod synaptic terminal. Glutamate binds to metabotropic glutamate receptors mGluR6 (<i>GRM6</i>) and activates an intracellular cascade involving GαO and Gβ5, leading to the closure of the TRPM1 channel. NYX also plays a role, but its function is currently unknown. GRM6, NYX, and TRPM1 have now been implicated in human cCSNB. 2,6  It is less likely that human patients with mutations in either GαO or Gβ5 will be found, given that the GαO 22  mutation is lethal in mice and Gβ5 has a primary photoreceptor-degeneration phenotype. 23 </div></p></div><div class="aux"><div class="resc">Maria M. van Genderen, et al. Am J Hum Genet. 2009 November 13;85(5):730-736.</div><p class="links"><a class="dblinks" href="/pubmed/19896109" ref="ordinalpos=3409&amp;ncbi_uid=2100896&amp;link_uid=19896109">Citation</a><a class="dblinks" href="/pmc/articles/PMC2775826/" ref="ordinalpos=3409&amp;ncbi_uid=2100896&amp;link_uid=PMC2775826">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3410.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2430211/figure/F5/" ref="ordinalpos=3410&amp;ncbi_uid=1341203&amp;link_uid=PMC2430211" href="/pmc/articles/PMC2430211/"><img src="/pmc/articles/instance/2430211/bin/1471-2407-8-163-5.gif" src-large="/pmc/articles/instance/2430211/bin/1471-2407-8-163-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2430211/" ref="ordinalpos=3410&amp;ncbi_uid=1341203&amp;link_uid=PMC2430211" image-link="/pmc/articles/PMC2430211/figure/F5/" class="imagepopup">Figure 5.  From: Involvement of TSC genes and differential expression of other members of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral squamous cell carcinoma. </a></p><div class="supp"><p class="details"><div><b>Expression of other members of the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in oral tumors.</b> a) mRNA expression of 10 genes in 16 matched normal and tumor samples. Horizontal lines represent mean values of mRNA expression across normal or tumor samples. b) Western blot analysis of matched normal (N) and tumor (T) samples from eight patients. Numbers represent patient numbers. Blots were probed with antibodies for anti-AKT, anti-p-AKT (Thr308), anti-p70S6K1 and anti-p-70S6K1 (Thr389).</div></p></div><div class="aux"><div class="resc">Sanjukta Chakraborty, et al. BMC Cancer. 2008;8:163-163.</div><p class="links"><a class="dblinks" href="/pubmed/18538015" ref="ordinalpos=3410&amp;ncbi_uid=1341203&amp;link_uid=18538015">Citation</a><a class="dblinks" href="/pmc/articles/PMC2430211/" ref="ordinalpos=3410&amp;ncbi_uid=1341203&amp;link_uid=PMC2430211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3411.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3950623/figure/Fig8/" ref="ordinalpos=3411&amp;ncbi_uid=5930161&amp;link_uid=PMC3950623" href="/pmc/articles/PMC3950623/"><img src="/pmc/articles/instance/3950623/bin/12035_2013_8568_Fig8_HTML.gif" src-large="/pmc/articles/instance/3950623/bin/12035_2013_8568_Fig8_HTML.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3950623/" ref="ordinalpos=3411&amp;ncbi_uid=5930161&amp;link_uid=PMC3950623" image-link="/pmc/articles/PMC3950623/figure/Fig8/" class="imagepopup">Fig. 8.  From: Opposing <span class="highlight" style="background-color:">Signaling</span> of ROCK1 and ROCK2 Determines the Switching of Substrate Specificity and the Mode of Migration of Glioblastoma Cells. </a></p><div class="supp"><p class="details"><div>Second part of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analyses of cells with altered ROCK1 and ROCK2 expressions. Here, we analyzed the protein expression of RhoA (<b>d</b>), β1-integrin (<b>e</b>), β-catenin (f), pERK1 (<b>g</b>), and pERK2 (<b>h</b>) using densitometric measurement in cells with ROCK1 knockdown (<b>a</b>), ROCk2 knockdown (<b>b</b>), and cells treated with Rho kinase inhibitor Y27632 (<b>c</b>)</div></p></div><div class="aux"><div class="resc">Sonja Mertsch, et al. Mol Neurobiol. 2014;49(2):900-915.</div><p class="links"><a class="dblinks" href="/pubmed/24170433" ref="ordinalpos=3411&amp;ncbi_uid=5930161&amp;link_uid=24170433">Citation</a><a class="dblinks" href="/pmc/articles/PMC3950623/" ref="ordinalpos=3411&amp;ncbi_uid=5930161&amp;link_uid=PMC3950623">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3412.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3173947/figure/F1/" ref="ordinalpos=3412&amp;ncbi_uid=3221627&amp;link_uid=PMC3173947" href="/pmc/articles/PMC3173947/"><img src="/pmc/articles/instance/3173947/bin/nihms261526f1.gif" src-large="/pmc/articles/instance/3173947/bin/nihms261526f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3173947/" ref="ordinalpos=3412&amp;ncbi_uid=3221627&amp;link_uid=PMC3173947" image-link="/pmc/articles/PMC3173947/figure/F1/" class="imagepopup">Fig. 1.  From: Regulation of Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> by Protein Kinases. </a></p><div class="supp"><p class="details"><div>Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>A:</b> In the absence of Wnt ligand, cytoplasmic b-catenin is targeted for destruction by the Axin–scaffolded protein complex containing Axin, APC, GSK3, and CKI. Association of DNA-bound Tcf with Gro and other transcriptional corepressors inhibits Wnt target genes. <b>B:</b> Upon ligand binding, the destruction complex is inhibited and cytoplasmic β-catenin accumulates and enters the nucleus where it can activate gene expression through Tcf and CBP binding.</div></p></div><div class="aux"><div class="resc">Esther M. Verheyen, et al. Dev Dyn. ;239(1):34-44.</div><p class="links"><a class="dblinks" href="/pubmed/19623618" ref="ordinalpos=3412&amp;ncbi_uid=3221627&amp;link_uid=19623618">Citation</a><a class="dblinks" href="/pmc/articles/PMC3173947/" ref="ordinalpos=3412&amp;ncbi_uid=3221627&amp;link_uid=PMC3173947">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3413.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2708738/figure/F3/" ref="ordinalpos=3413&amp;ncbi_uid=1987885&amp;link_uid=PMC2708738" href="/pmc/articles/PMC2708738/"><img src="/pmc/articles/instance/2708738/bin/zpq9990985010003.gif" src-large="/pmc/articles/instance/2708738/bin/zpq9990985010003.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2708738/" ref="ordinalpos=3413&amp;ncbi_uid=1987885&amp;link_uid=PMC2708738" image-link="/pmc/articles/PMC2708738/figure/F3/" class="imagepopup">Fig. 3. From: <span class="highlight" style="background-color:">Signaling</span> dynamics of the KSR1 scaffold complex. </a></p><div class="supp"><p class="details"><div>Binding of KSR1 to MEK and ERK regulates the KSR1/B-Raf interaction. (<i>A</i>) KSR1<sup>−/−</sup> MEFs stably expressing the indicated KSR1 proteins were serum starved and treated with EGF. Pyo-KSR1 proteins were immunoprecipitated and examined for binding of endogenous B-Raf and MEK proteins. Tyrosine phosphorylation of EGFr indicates activation of EGFr <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. (<i>B</i>) Serum-starved WT- or AxAP-KSR1 MEFs were treated as indicated using EGF and U0126. Pyo-KSR1 proteins were immunoprecipitated and examined for binding of endogenous B-Raf, MEK, and pERK. Lysates were probed for pERK levels to confirm that U0126 blocked ERK activation.</div></p></div><div class="aux"><div class="resc">Melissa M. McKay, et al. Proc Natl Acad Sci U S A. 2009 July 7;106(27):11022-11027.</div><p class="links"><a class="dblinks" href="/pubmed/19541618" ref="ordinalpos=3413&amp;ncbi_uid=1987885&amp;link_uid=19541618">Citation</a><a class="dblinks" href="/pmc/articles/PMC2708738/" ref="ordinalpos=3413&amp;ncbi_uid=1987885&amp;link_uid=PMC2708738">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3414.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2639747/figure/F2/" ref="ordinalpos=3414&amp;ncbi_uid=1775865&amp;link_uid=PMC2639747" href="/pmc/articles/PMC2639747/"><img src="/pmc/articles/instance/2639747/bin/nihms-90748-f0002.gif" src-large="/pmc/articles/instance/2639747/bin/nihms-90748-f0002.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2639747/" ref="ordinalpos=3414&amp;ncbi_uid=1775865&amp;link_uid=PMC2639747" image-link="/pmc/articles/PMC2639747/figure/F2/" class="imagepopup">FIGURE 2.  From: Roles of Smad3 in TGF-? <span class="highlight" style="background-color:">Signaling</span> During Carcinogenesis. </a></p><div class="supp"><p class="details"><div>Mechanisms of Smad3 in TGF-β–mediated proapoptotic transcriptional responses. An activated Smad3/4 complex is required in the nucleus to activate the transcription of <i>GADD45</i> (along with <i>FOXO</i>) and <i>Bim</i> or DAPK and to repress <i>Bcl-2</i>. p38 MAPK is likely involved in the repression of <i>Bcl-2</i>, but the mechanism is not clear. Three mechanisms have been reported for the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> to dampen the proapoptotic <span class="highlight" style="background-color:">signaling</span> of Smad3: Akt can sequester Smad3 in the cytoplasm by the binding of Smad3, activity of Smad3 can be inhibited by mTOR, or Akt may inhibit FOXO by sequestering it from participating in Smad3-mediated transcription of <i>GADD45</i>.</div></p></div><div class="aux"><div class="resc">Caroline Millet, et al. Crit Rev Eukaryot Gene Expr. ;17(4):281-293.</div><p class="links"><a class="dblinks" href="/pubmed/17725494" ref="ordinalpos=3414&amp;ncbi_uid=1775865&amp;link_uid=17725494">Citation</a><a class="dblinks" href="/pmc/articles/PMC2639747/" ref="ordinalpos=3414&amp;ncbi_uid=1775865&amp;link_uid=PMC2639747">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3415.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3711381/figure/F2/" ref="ordinalpos=3415&amp;ncbi_uid=5099966&amp;link_uid=PMC3711381" href="/pmc/articles/PMC3711381/"><img src="/pmc/articles/instance/3711381/bin/nihms-442491-f0002.gif" src-large="/pmc/articles/instance/3711381/bin/nihms-442491-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3711381/" ref="ordinalpos=3415&amp;ncbi_uid=5099966&amp;link_uid=PMC3711381" image-link="/pmc/articles/PMC3711381/figure/F2/" class="imagepopup">Figure 2.  From: A novel target for Huntington's disease: ERK at the crossroads of <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The ERK <span class="highlight" style="background-color:">pathway</span> is implicated in Htt pathogenesis in a complex manner. Mutant Htt may upregulate the ERK <span class="highlight" style="background-color:">pathway</span> either cell autonomously (a) or by enhancing the activating effect of glutamate <span class="highlight" style="background-color:">signaling</span> (b). Activated ERK contributes to cell survival by inhibiting caspase activation (c) and directing a transcriptional response (d). In glial cells, mutant Htt downregulates EGF and ERK dependent expression of EAAT glutamate transporters (e), which might lead to increased glutamate <span class="highlight" style="background-color:">signaling</span> in neuronal cells and eventual potential excitotoxicity (f). The protective transcriptional effects of corticostriatal BDNF <span class="highlight" style="background-color:">signaling</span> are mediated by the ERK <span class="highlight" style="background-color:">pathway</span>. Mutant Htt compromises the BDNF response both upstream and downstream of ERK, by inhibiting the activation of the Ras GTPase (g) or the MSK-1 kinase (h) that phosphorylates H3S10 in the chromatin and contributes to <i>c-fos</i> expression, respectively. Thus, the response to Htt challenge is mixed and involves both activation and dampening of different up- or downstream branches of the ERK cascade. Pharmacologic intervention with compounds that can selectively activate or enhance desirable responses while avoiding the non-desirable ones would provide attractive agents for treatment.</div></p></div><div class="aux"><div class="resc">László Bodai, et al. Bioessays. ;34(2):142-148.</div><p class="links"><a class="dblinks" href="/pubmed/22334892" ref="ordinalpos=3415&amp;ncbi_uid=5099966&amp;link_uid=22334892">Citation</a><a class="dblinks" href="/pmc/articles/PMC3711381/" ref="ordinalpos=3415&amp;ncbi_uid=5099966&amp;link_uid=PMC3711381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3416.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2933925/figure/fig4-1947601910373795/" ref="ordinalpos=3416&amp;ncbi_uid=3024506&amp;link_uid=PMC2933925" href="/pmc/articles/PMC2933925/"><img src="/pmc/articles/instance/3092205/bin/10.1177_1947601910373795-fig4.gif" src-large="/pmc/articles/instance/3092205/bin/10.1177_1947601910373795-fig4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2933925/" ref="ordinalpos=3416&amp;ncbi_uid=3024506&amp;link_uid=PMC2933925" image-link="/pmc/articles/PMC2933925/figure/fig4-1947601910373795/" class="imagepopup">Figure 4. From: The Role of B-RAF Mutations in Melanoma and the Induction of EMT via Dysregulation of the NF-?B/Snail/RKIP/PTEN Circuit. </a></p><div class="supp"><p class="details"><div>Activation of NF-κB by the PI3K/AKT and MAPK pathways. (<b>A</b>) Activation of NF-κB via the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. Activation of the PI3K <span class="highlight" style="background-color:">pathway</span> by RAS results in the production of AKT. AKT activates NF-κB through the activation of IKK. Activation of NF-κB results in Snail-induced EMT. PI3K <span class="highlight" style="background-color:">signaling</span> is antagonized by PTEN; however, up-regulation in Snail inhibits PTEN, resulting in a self-amplification cycle. (<b>B</b>) Activation of NF-κB via the MAPK <span class="highlight" style="background-color:">pathway</span> NF-κB is regulated through MEK-induced activation of IKK and ERK-induced up-regulation of Snail. (<b>C</b>) Regulation of Snail via RKIP and PTEN up-regulation of Snail through the PI3K/AKT and MAPK pathways inhibits both PTEN and RKIP, which then, in turn, amplifies the expression of Snail.</div></p></div><div class="aux"><div class="resc">Kimberly Lin, et al. Genes Cancer. 2010 May;1(5):409-420.</div><p class="links"><a class="dblinks" href="/pubmed/20827424" ref="ordinalpos=3416&amp;ncbi_uid=3024506&amp;link_uid=20827424">Citation</a><a class="dblinks" href="/pmc/articles/PMC2933925/" ref="ordinalpos=3416&amp;ncbi_uid=3024506&amp;link_uid=PMC2933925">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3417.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3794382/figure/F1/" ref="ordinalpos=3417&amp;ncbi_uid=5390264&amp;link_uid=PMC3794382" href="/pmc/articles/PMC3794382/"><img src="/pmc/articles/instance/3794382/bin/fphys-04-00278-g0001.gif" src-large="/pmc/articles/instance/3794382/bin/fphys-04-00278-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3794382/" ref="ordinalpos=3417&amp;ncbi_uid=5390264&amp;link_uid=PMC3794382" image-link="/pmc/articles/PMC3794382/figure/F1/" class="imagepopup">Figure 1.  From: Methods and approaches in the topology-based analysis of biological pathways. </a></p><div class="supp"><p class="details"><div><b>Gene sets are <span style="text-decoration: underline" class="underline">not</span> pathways. (A)</b> shows a small part of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from KEGG. This <span class="highlight" style="background-color:">pathway</span> shows the location of various genes or gene products (inside the cell, outside of it, or in the membrane), what gene interacts with what other gene(s), the type of each interaction (activation, repression, phosphorylation, etc.), the direction of the signal propagation, and potentially many other things (e.g., complex formation, etc.). <b>(B)</b> presents the same part of the same <span class="highlight" style="background-color:">pathway</span> as a gene set (no interactions). The gene set has lost all the structure and the additional information captured by the original <span class="highlight" style="background-color:">pathway</span>. This comparison shows how much important knowledge existent in <span class="highlight" style="background-color:">pathway</span> database is ignored when pathways are treated as simple gene sets.</div></p></div><div class="aux"><div class="resc">Cristina Mitrea, et al. Front Physiol. 2013;4:278.</div><p class="links"><a class="dblinks" href="/pubmed/24133454" ref="ordinalpos=3417&amp;ncbi_uid=5390264&amp;link_uid=24133454">Citation</a><a class="dblinks" href="/pmc/articles/PMC3794382/" ref="ordinalpos=3417&amp;ncbi_uid=5390264&amp;link_uid=PMC3794382">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3418.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1570443/figure/f10/" ref="ordinalpos=3418&amp;ncbi_uid=846120&amp;link_uid=PMC1570443" href="/pmc/articles/PMC1570443/"><img src="/pmc/articles/instance/1570443/bin/7601317f10.gif" src-large="/pmc/articles/instance/1570443/bin/7601317f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1570443/" ref="ordinalpos=3418&amp;ncbi_uid=846120&amp;link_uid=PMC1570443" image-link="/pmc/articles/PMC1570443/figure/f10/" class="imagepopup">Figure 10.  From: Secretion of L-glutamate from osteoclasts through transcytosis. </a></p><div class="supp"><p class="details"><div>Proposed <span style="font-variant: small-caps" class="small-caps">L</span>-glutamate <span class="highlight" style="background-color:">signaling</span> during bone resorption. (left) Under the normal physiological conditions, a proportion of VGLUT1-containing small clear vesicles may fuse with the transcytotic vesicle. Upon stimulation with KCl or ATP, <span style="font-variant: small-caps" class="small-caps">L</span>-glutamate and bone degradation products are secreted through transcytosis. Then, the released <span style="font-variant: small-caps" class="small-caps">L</span>-glutamate acts as a negative feedback regulator through mGluR8-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, keeping osteoclasts in the suppressed state. (right) In osteoclasts of VGLUT1<sup>−/−</sup> mice, <span style="font-variant: small-caps" class="small-caps">L</span>-glutamate-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is impaired. This may induce the desuppressive state of osteoclasts, causing stimulated bone resorption followed by osteoporosis. N, nucleus.</div></p></div><div class="aux"><div class="resc">Riyo Morimoto, et al. EMBO J. 2006 September 20;25(18):4175-4186.</div><p class="links"><a class="dblinks" href="/pubmed/16957773" ref="ordinalpos=3418&amp;ncbi_uid=846120&amp;link_uid=16957773">Citation</a><a class="dblinks" href="/pmc/articles/PMC1570443/" ref="ordinalpos=3418&amp;ncbi_uid=846120&amp;link_uid=PMC1570443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3419.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3006436/figure/F3/" ref="ordinalpos=3419&amp;ncbi_uid=2646084&amp;link_uid=PMC3006436" href="/pmc/articles/PMC3006436/"><img src="/pmc/articles/instance/3006436/bin/nihms231136f3.gif" src-large="/pmc/articles/instance/3006436/bin/nihms231136f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3006436/" ref="ordinalpos=3419&amp;ncbi_uid=2646084&amp;link_uid=PMC3006436" image-link="/pmc/articles/PMC3006436/figure/F3/" class="imagepopup">Figure 3.  From: Mapping Key Features of Transcriptional Regulatory Circuitry in Embryonic Stem Cells. </a></p><div class="supp"><p class="details"><div>Key themes from studies of <span class="highlight" style="background-color:">signaling</span> pathways in ES cells. (<i>A</i>) <span class="highlight" style="background-color:">Signaling</span> pathways can directly connect to key transcription factors in the core interconnected autoregulatory loop through terminal components of their signal transduction <span class="highlight" style="background-color:">pathway</span>. (<i>Colored ovals</i>) Proteins; (<i>orange rectangles</i>) genes; (<i>squiggly lines</i>) their resulting transcripts. An arrow from a protein to a gene indicates direct regulation of the gene by the protein. (<i>Red Y shape</i> and <i>long squiggly arrow</i>) Wnt signal transduction <span class="highlight" style="background-color:">pathway</span>. (<i>B</i>) The terminal component of <span class="highlight" style="background-color:">signaling</span> pathways such as the Wnt <span class="highlight" style="background-color:">pathway</span> can form a feedforward loop by regulating both the key regulators and their target genes. Proteins, genes, and regulation are depicted as in <i>A</i>.</div></p></div><div class="aux"><div class="resc">M.F. Cole, et al. Cold Spring Harb Symp Quant Biol. ;73:183-193.</div><p class="links"><a class="dblinks" href="/pubmed/19022761" ref="ordinalpos=3419&amp;ncbi_uid=2646084&amp;link_uid=19022761">Citation</a><a class="dblinks" href="/pmc/articles/PMC3006436/" ref="ordinalpos=3419&amp;ncbi_uid=2646084&amp;link_uid=PMC3006436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3420.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3553295/figure/F3/" ref="ordinalpos=3420&amp;ncbi_uid=5762356&amp;link_uid=PMC3553295" href="/pmc/articles/PMC3553295/"><img src="/pmc/articles/instance/3553295/bin/nihms416112f3.gif" src-large="/pmc/articles/instance/3553295/bin/nihms416112f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3553295/" ref="ordinalpos=3420&amp;ncbi_uid=5762356&amp;link_uid=PMC3553295" image-link="/pmc/articles/PMC3553295/figure/F3/" class="imagepopup">Figure 3. Gαi and Gαs heterotrimeric G-protein <span class="highlight" style="background-color:">signaling</span> schematic.  From: Effects of Cocaine and Withdrawal on the Mouse Nucleus Accumbens Transcriptome. </a></p><div class="supp"><p class="details"><div>Model based on Panther <span class="highlight" style="background-color:">Pathway</span> diagram (http://www.pantherdb.org). Venn diagram depicts color key for <span class="highlight" style="background-color:">signaling</span> components. Genes labeled yellow showed a sustained increase in Coc and W/D (cluster i), red genes increased in W/D only (cluster ii), and blue genes had decreased expression in both Coc and W/D (cluster v); overlap colors indicate contribution of &gt; 1 cluster to <span class="highlight" style="background-color:">signaling</span> group. For example, GPCRs, which appear in all three clusters, are shown in magenta. Genes from Clusters <i>i, ii</i>, and <i>v</i> were used for this analysis. Abbreviations: Coc, Cocaine; W/D, Withdrawal.</div></p></div><div class="aux"><div class="resc">Jodi E. Eipper-Mains, et al. Genes Brain Behav. ;12(1):21-33.</div><p class="links"><a class="dblinks" href="/pubmed/23094851" ref="ordinalpos=3420&amp;ncbi_uid=5762356&amp;link_uid=23094851">Citation</a><a class="dblinks" href="/pmc/articles/PMC3553295/" ref="ordinalpos=3420&amp;ncbi_uid=5762356&amp;link_uid=PMC3553295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3421.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3504253/figure/fig2/" ref="ordinalpos=3421&amp;ncbi_uid=4419220&amp;link_uid=PMC3504253" href="/pmc/articles/PMC3504253/"><img src="/pmc/articles/instance/3504253/bin/LRT2011-329572.002.gif" src-large="/pmc/articles/instance/3504253/bin/LRT2011-329572.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3504253/" ref="ordinalpos=3421&amp;ncbi_uid=4419220&amp;link_uid=PMC3504253" image-link="/pmc/articles/PMC3504253/figure/fig2/" class="imagepopup">Figure 2.  From: Understanding and Targeting the Wnt/β-Catenin <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Chronic Leukemia. </a></p><div class="supp"><p class="details"><div>The activated Wnt <span class="highlight" style="background-color:">pathway</span>. Wnt: wingless-int; POL: ribonucleic acid polymerase; Dvl: disheveled.</div></p></div><div class="aux"><div class="resc">S. Thanendrarajan, et al. Leuk Res Treatment. 2011;2011:329572.</div><p class="links"><a class="dblinks" href="/pubmed/23213540" ref="ordinalpos=3421&amp;ncbi_uid=4419220&amp;link_uid=23213540">Citation</a><a class="dblinks" href="/pmc/articles/PMC3504253/" ref="ordinalpos=3421&amp;ncbi_uid=4419220&amp;link_uid=PMC3504253">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3422.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4000987/figure/fig5/" ref="ordinalpos=3422&amp;ncbi_uid=6124109&amp;link_uid=PMC4000987" href="/pmc/articles/PMC4000987/"><img src="/pmc/articles/instance/4000987/bin/PPAR2014-189085.005.gif" src-large="/pmc/articles/instance/4000987/bin/PPAR2014-189085.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4000987/" ref="ordinalpos=3422&amp;ncbi_uid=6124109&amp;link_uid=PMC4000987" image-link="/pmc/articles/PMC4000987/figure/fig5/" class="imagepopup">Figure 5.  From: Radix Astragali Improves Dysregulated Triglyceride Metabolism and Attenuates Macrophage Infiltration in Adipose Tissue in High-Fat Diet-Induced Obese Male Rats through Activating mTORC1-PPARγ <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> plays an important role in the upregulation of PPAR<i>γ</i> expression by RA. (a) Differentiated 3T3-L1 cells were cultured with 0, 1, 2, and 4 mg/mL RA for 48 hours. Cell extracts were then subjected to western blot analysis with antibodies specific for total and phosphorylated p70S6K (Thr389), PPAR<i>γ</i>, and <i>β</i>-actin. (b) Differentiated 3T3-L1 cells were cultured with rapamycin (0 or 30 nM) and RA (0 or 2 mg/mL) for 2 or 48 hours. Cell extracts were then subjected to western blot analysis with antibodies specific for total and phosphorylated p70S6K (Thr389), PPAR<i>γ</i>, TNF<i>α</i>, and <i>β</i>-actin.</div></p></div><div class="aux"><div class="resc">Yang Long, et al. PPAR Res. 2014;2014:189085.</div><p class="links"><a class="dblinks" href="/pubmed/24817881" ref="ordinalpos=3422&amp;ncbi_uid=6124109&amp;link_uid=24817881">Citation</a><a class="dblinks" href="/pmc/articles/PMC4000987/" ref="ordinalpos=3422&amp;ncbi_uid=6124109&amp;link_uid=PMC4000987">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3423.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3279748/figure/F2/" ref="ordinalpos=3423&amp;ncbi_uid=3820612&amp;link_uid=PMC3279748" href="/pmc/articles/PMC3279748/"><img src="/pmc/articles/instance/3279748/bin/fnmol-05-00013-g0002.gif" src-large="/pmc/articles/instance/3279748/bin/fnmol-05-00013-g0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3279748/" ref="ordinalpos=3423&amp;ncbi_uid=3820612&amp;link_uid=PMC3279748" image-link="/pmc/articles/PMC3279748/figure/F2/" class="imagepopup">Figure 2.  From: A pivotal role of GSK-3 in synaptic plasticity. </a></p><div class="supp"><p class="details"><div><b>The role of GSK-3 in NMDAR-LTD. (A)</b> <span class="highlight" style="background-color:">Signaling</span> pathways involved in the induction of NMDAR-LTD and the regulation of GSK-3β. <b>(B)</b> <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> by which NMDAR-LTP can inhibit NMDAR-LTD. <b>(C)</b> Potential downstream effectors of GSK-3β in NMDAR-LTD. (See text for details).</div></p></div><div class="aux"><div class="resc">Clarrisa A. Bradley, et al. Front Mol Neurosci. 2012;5:13.</div><p class="links"><a class="dblinks" href="/pubmed/22363262" ref="ordinalpos=3423&amp;ncbi_uid=3820612&amp;link_uid=22363262">Citation</a><a class="dblinks" href="/pmc/articles/PMC3279748/" ref="ordinalpos=3423&amp;ncbi_uid=3820612&amp;link_uid=PMC3279748">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3424.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3900764/figure/F1/" ref="ordinalpos=3424&amp;ncbi_uid=5749718&amp;link_uid=PMC3900764" href="/pmc/articles/PMC3900764/"><img src="/pmc/articles/instance/3900764/bin/fncom-08-00003-g0001.gif" src-large="/pmc/articles/instance/3900764/bin/fncom-08-00003-g0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3900764/" ref="ordinalpos=3424&amp;ncbi_uid=5749718&amp;link_uid=PMC3900764" image-link="/pmc/articles/PMC3900764/figure/F1/" class="imagepopup">Figure 1.  From: Data and model tango to aid the understanding of astrocyte-neuron <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>The “tango” between experimental data and mathematical model is necessary for a faster and deeper understanding of the role of astrocyte-neuron <span class="highlight" style="background-color:">signaling</span> in the brain</b>. Experimental data are important for <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> quantification using mathematical models. The experimental data in turn can gain from the model by testing a hypothesis <i>in silico</i> or designing new experiments to test the hypothesis <i>in vivo</i> or <i>in vitro</i>. It is a combination of both the approaches that could lead us to a wholesome understanding of the human brain, i.e., the holy grail of neuroscience.</div></p></div><div class="aux"><div class="resc">Shivendra Tewari, et al. Front Comput Neurosci. 2014;8:3.</div><p class="links"><a class="dblinks" href="/pubmed/24478686" ref="ordinalpos=3424&amp;ncbi_uid=5749718&amp;link_uid=24478686">Citation</a><a class="dblinks" href="/pmc/articles/PMC3900764/" ref="ordinalpos=3424&amp;ncbi_uid=5749718&amp;link_uid=PMC3900764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3425.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3637970/figure/F1/" ref="ordinalpos=3425&amp;ncbi_uid=4834416&amp;link_uid=PMC3637970" href="/pmc/articles/PMC3637970/"><img src="/pmc/articles/instance/3637970/bin/nihms443292f1.gif" src-large="/pmc/articles/instance/3637970/bin/nihms443292f1.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3637970/" ref="ordinalpos=3425&amp;ncbi_uid=4834416&amp;link_uid=PMC3637970" image-link="/pmc/articles/PMC3637970/figure/F1/" class="imagepopup">FIGURE 1.  From: TGF-β <span class="highlight" style="background-color:">Signaling</span> and the Role of Inhibitory Smads in Non-small Cell Lung Cancer. </a></p><div class="supp"><p class="details"><div>Schematic diagram of transforming growth factor-<i>β</i> (TGF-<i>β</i>) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. TGF-<i>β</i> <span class="highlight" style="background-color:">signaling</span> is initiated by the binding of TGF-<i>β</i> to its receptors, transforming growth factor-<i>β</i> receptor I (TGF-<i>β</i> RI) and transforming growth factor-<i>β</i> receptor II (TGF-<i>β</i> RII), and receptor tyrosine kinase activation which then phosphorylates Smad2/3. Activated Smad2/3 regulates gene expression of Smad4 and other transcription factors (TF). Feedback regulation is mediated by Smad6/7, which interferes the binding of Smad2/3 to TGF-<i>β</i> receptors and inhibits transcription. Both Smad6 and Smad7 are in turn induced by TGF-<i>β</i> and regulated by Smad ubiquitin regulatory factors (Smurfs). Synthetic inhibitors inactivate the TGF-<i>β</i> <span class="highlight" style="background-color:">pathway</span> by inhibition of receptor enzymatic activity.</div></p></div><div class="aux"><div class="resc">Hyo-Sung Jeon, et al. J Thorac Oncol. 2010 April;5(4):417-419.</div><p class="links"><a class="dblinks" href="/pubmed/20107423" ref="ordinalpos=3425&amp;ncbi_uid=4834416&amp;link_uid=20107423">Citation</a><a class="dblinks" href="/pmc/articles/PMC3637970/" ref="ordinalpos=3425&amp;ncbi_uid=4834416&amp;link_uid=PMC3637970">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3426.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3765040/figure/F4/" ref="ordinalpos=3426&amp;ncbi_uid=5278463&amp;link_uid=PMC3765040" href="/pmc/articles/PMC3765040/"><img src="/pmc/articles/instance/3765040/bin/nihms497399f4.gif" src-large="/pmc/articles/instance/3765040/bin/nihms497399f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3765040/" ref="ordinalpos=3426&amp;ncbi_uid=5278463&amp;link_uid=PMC3765040" image-link="/pmc/articles/PMC3765040/figure/F4/" class="imagepopup">Fig. 4.  From: A "Trimer of Dimers"--Based Model for the Chemotactic Signal Transduction Network in Bacterial Chemotaxis. </a></p><div class="supp"><p class="details"><div>Sensitivity in signal transduction for two structures of receptor complexes—“trimer of dimers” and “dimer.” (<b>A</b>) Sensitivities for <span class="highlight" style="background-color:">signaling</span> complexes consisting of “trimer of receptor dimers” in ligand binding <i>S</i>(<i>O</i>|<i>L</i>), activity regulation <i>S</i>(<i>A</i>|<i>O</i>), phosphorylation <i>S</i>(<i>T<sup>p</sup></i>|<i>A</i>), phosphoryl transfer <i>S</i>(<i>Y<sub>p</sub></i>|<i>T<sup>p</sup></i>) and the overall <span class="highlight" style="background-color:">pathway</span> <i>S</i>(<i>Y<sub>p</sub></i>|<i>L</i>). (<b>B</b>) Sensitivities for <span class="highlight" style="background-color:">signaling</span> complexes consisting of “receptor dimers” in ligand binding <i>S</i>(<i>O</i>|<i>L</i>), activity regulation <i>S</i>(<i>A</i>|<i>O</i>), phosphorylation <i>S</i>(<i>T<sup>p</sup></i>|<i>A</i>), phosphoryl transfer <i>S</i>(<i>Y<sub>p</sub></i>|<i>T<sup>p</sup></i>) and the overall <span class="highlight" style="background-color:">pathway</span> <i>S</i>(<i>Y<sub>p</sub></i>|<i>L</i>). (<b>C</b>) and (<b>D</b>) Comparison of <i>S</i>(<i>O</i>|<i>L</i>) and <i>S</i>(<i>A</i>|<i>O</i>) between two structures of receptor complexes (Color figure online)</div></p></div><div class="aux"><div class="resc">Xiangrong Xin, et al. Bull Math Biol. 2012 October;74(10):2339-2382.</div><p class="links"><a class="dblinks" href="/pubmed/22864951" ref="ordinalpos=3426&amp;ncbi_uid=5278463&amp;link_uid=22864951">Citation</a><a class="dblinks" href="/pmc/articles/PMC3765040/" ref="ordinalpos=3426&amp;ncbi_uid=5278463&amp;link_uid=PMC3765040">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3427.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3193440/figure/fig2/" ref="ordinalpos=3427&amp;ncbi_uid=3783122&amp;link_uid=PMC3193440" href="/pmc/articles/PMC3193440/"><img src="/pmc/articles/instance/3193440/bin/cr201113f2.gif" src-large="/pmc/articles/instance/3193440/bin/cr201113f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3193440/" ref="ordinalpos=3427&amp;ncbi_uid=3783122&amp;link_uid=PMC3193440" image-link="/pmc/articles/PMC3193440/figure/fig2/" class="imagepopup">Figure 2.  From: NF-?B in immunobiology. </a></p><div class="supp"><p class="details"><div>NF-κB in lymphoid organogenesis. NF-κB regulates key components of a positive feedback loop between hematopoietic and stromal cells. LTα<sub>1</sub>β<sub>2</sub>-expressing hematopoietic cells induce production of VCAM-1 through the canonical NF-κB <span class="highlight" style="background-color:">pathway</span> and chemokines through the non-canonical <span class="highlight" style="background-color:">pathway</span> in LTβR-expressing stromal cells. Stromal expression of chemokines induces the upregulation of integrins (α<sub>4</sub>β<sub>1</sub>) on hematopoietic cells resulting in increased recruitment of LTα<sub>1</sub>β<sub>2</sub>-expressing cells, and increased <span class="highlight" style="background-color:">signaling</span> through stromal LTβR. RANKL <span class="highlight" style="background-color:">signaling</span> through RANK on the hematopoietic cell leads to activation of NF-κB, and further upregulation of LTα<sub>1</sub>β<sub>2</sub>.</div></p></div><div class="aux"><div class="resc">Matthew S Hayden, et al. Cell Res. 2011 February;21(2):223-244.</div><p class="links"><a class="dblinks" href="/pubmed/21243012" ref="ordinalpos=3427&amp;ncbi_uid=3783122&amp;link_uid=21243012">Citation</a><a class="dblinks" href="/pmc/articles/PMC3193440/" ref="ordinalpos=3427&amp;ncbi_uid=3783122&amp;link_uid=PMC3193440">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3428.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2805009/figure/F1/" ref="ordinalpos=3428&amp;ncbi_uid=2190580&amp;link_uid=PMC2805009" href="/pmc/articles/PMC2805009/"><img src="/pmc/articles/instance/2805009/bin/nihms163580f1.gif" src-large="/pmc/articles/instance/2805009/bin/nihms163580f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2805009/" ref="ordinalpos=3428&amp;ncbi_uid=2190580&amp;link_uid=PMC2805009" image-link="/pmc/articles/PMC2805009/figure/F1/" class="imagepopup">Figure 1.  From: TGF?1 and Treg cells: alliance for tolerance. </a></p><div class="supp"><p class="details"><div>Cross-talk between TCR and TGFβ <span class="highlight" style="background-color:">signaling</span> networks. TCR activation on recognition of MHC–Ag complexes on APCs leads to the phosphorylation of ZAP70, PLCγ and Lck. PLCγ hydrolyses PIP<sub>2</sub> to generate DAG and IP<sub>3</sub>. DAG binds to and activates PKC, whereas IP<sub>3</sub> binds to its receptor on the endoplasmic reticulum (ER) and releases Ca<sup>2+</sup> into cytoplasm. Activated PKC activates NF-κB by degrading I-κB through phosphorylation. Increased cytosolic Ca<sup>2+</sup> leads to the activation of calcineurin (CN), which, in turn, activates NF-AT through dephosphorylation. Active NF-κB and NF-AT translocate to the nucleus and induce the gene transcription that is required for the T-cell response. Co-stimulatory signals through CD28 activate the JNK/p38 MAPK <span class="highlight" style="background-color:">pathway</span>, which activates AP-1 (FOS/JUN). CD28 co-stimulatory signals inhibit <i>Cblb</i>, which is a negative regulator of TCR signals. <span class="highlight" style="background-color:">Signaling</span> through CD28 activates AKT, which, in turn, inhibits SMAD2/3 phosphorylation. Co-stimulatory <span class="highlight" style="background-color:">signaling</span> through CTLA-4, which is a negative regulator of TCR <span class="highlight" style="background-color:">signaling</span>, induces <i>Cblb</i>. TGFβ-receptor <span class="highlight" style="background-color:">signaling</span> activates SMAD2/3 through phosphorylation. Activated SMAD2/3 binds to SMAD4 and the complex translocates to the nucleus. FOXP3 inhibits SMAD7, which is an inhibitory SMAD that inhibits TGFβ <span class="highlight" style="background-color:">signaling</span>. CBL-B also mediates TGFβ <span class="highlight" style="background-color:">signaling</span> in <i>Foxp3</i> induction. CD25 is the α subunit of the high-affinity receptor for the IL-2 complex and IL-2 is essential for TGFβ1-mediated induction of FOXP3. FKBP12, which is associated with the TGFβ receptor, is released on ligand binding. FKBP12 binds to the IP<sub>3</sub> receptor and modulates Ca<sup>2+</sup> release from ER.</div></p></div><div class="aux"><div class="resc">Ramireddy Bommireddy, et al. Trends Mol Med. ;13(11):492-501.</div><p class="links"><a class="dblinks" href="/pubmed/17977791" ref="ordinalpos=3428&amp;ncbi_uid=2190580&amp;link_uid=17977791">Citation</a><a class="dblinks" href="/pmc/articles/PMC2805009/" ref="ordinalpos=3428&amp;ncbi_uid=2190580&amp;link_uid=PMC2805009">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3429.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3612391/figure/F3/" ref="ordinalpos=3429&amp;ncbi_uid=5862353&amp;link_uid=PMC3612391" href="/pmc/articles/PMC3612391/"><img src="/pmc/articles/instance/3612391/bin/nihms-431589-f0003.gif" src-large="/pmc/articles/instance/3612391/bin/nihms-431589-f0003.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3612391/" ref="ordinalpos=3429&amp;ncbi_uid=5862353&amp;link_uid=PMC3612391" image-link="/pmc/articles/PMC3612391/figure/F3/" class="imagepopup">Fig. 3. Diverse functions of Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> in wiring the nervous system.  From: The role and mechanism-of-action of Sema3E and Plexin-D1 in vascular and neural development. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> promotes or inhibits axonal growth depending on its association with other transmembrane proteins. In cortical neurons, Sema3E acts as a classical repulsive cue and inhibits neuronal growth solely through the Plexin-D1 receptor (Left) [21]. However, Sema3E plays as a positive regulator in the subiculo-mammillary neurons where Plexin-D1 forms complex with co-receptors, neuropilin-1 and VEGF receptor (Right) [21, 22]. <b>(B)</b> Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> determines different type of synapse specificity and formation depending on ligand receptor localization on pre- vs. post-synaptic neurons. Two different cases of synaptic recognition regulation by Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> have been reported. (B-i) Postsynaptic target releases Sema3E to control synapse specificity in spinal cord motor neuron circuitry [23]. In the spinal cord, incoming cutaneous maximus (CM) sensory neurons express Plexin-D1 and target CM motor neurons express repulsive cue, Sema3E, thus there is no monosynaptic CM sensory-motor connections. On the other hand, absence of Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> produces abnormal CM sensory-motor circuit. (B-ii) Incoming presynaptic neurons express Sema3E to control synapse specificity in <span class="highlight" style="background-color:">pathway</span> specific thalamostriatal circuitry [20]. In contrast to sensory-motor circuit in the spinal cord, Sema3E is expressed in incoming thalamic neurons in thalamostriatal projections whereas Plexin-D1 is expressed specifically in Drd1-positive direct <span class="highlight" style="background-color:">pathway</span> MSNs. Lack of Sema3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> causes ectopic thalamostriatal synapse in direct <span class="highlight" style="background-color:">pathway</span> MSNs.</div></p></div><div class="aux"><div class="resc">Won-Jong Oh, et al. Semin Cell Dev Biol. 2013 March;24(3):156-162.</div><p class="links"><a class="dblinks" href="/pubmed/23270617" ref="ordinalpos=3429&amp;ncbi_uid=5862353&amp;link_uid=23270617">Citation</a><a class="dblinks" href="/pmc/articles/PMC3612391/" ref="ordinalpos=3429&amp;ncbi_uid=5862353&amp;link_uid=PMC3612391">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3430.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3821580/figure/f2-ijms-14-19681/" ref="ordinalpos=3430&amp;ncbi_uid=5468623&amp;link_uid=PMC3821580" href="/pmc/articles/PMC3821580/"><img src="/pmc/articles/instance/3821580/bin/ijms-14-19681f2.gif" src-large="/pmc/articles/instance/3821580/bin/ijms-14-19681f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3821580/" ref="ordinalpos=3430&amp;ncbi_uid=5468623&amp;link_uid=PMC3821580" image-link="/pmc/articles/PMC3821580/figure/f2-ijms-14-19681/" class="imagepopup">Figure 2.  From: Non-Coding RNAs in Muscle Dystrophies. </a></p><div class="supp"><p class="details"><div>Schematic representation of the functional/physical relationship between Dystrophin-Syntrophins-nNOS <span class="highlight" style="background-color:">pathway</span> and miRNAs involved in such <span class="highlight" style="background-color:">signaling</span>, both in a WT and a DMD context.</div></p></div><div class="aux"><div class="resc">Daniela Erriquez, et al. Int J Mol Sci. 2013 October;14(10):19681-19704.</div><p class="links"><a class="dblinks" href="/pubmed/24084719" ref="ordinalpos=3430&amp;ncbi_uid=5468623&amp;link_uid=24084719">Citation</a><a class="dblinks" href="/pmc/articles/PMC3821580/" ref="ordinalpos=3430&amp;ncbi_uid=5468623&amp;link_uid=PMC3821580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3431.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4030437/figure/F2/" ref="ordinalpos=3431&amp;ncbi_uid=6160099&amp;link_uid=PMC4030437" href="/pmc/articles/PMC4030437/"><img src="/pmc/articles/instance/4030437/bin/nihms580707f2.gif" src-large="/pmc/articles/instance/4030437/bin/nihms580707f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4030437/" ref="ordinalpos=3431&amp;ncbi_uid=6160099&amp;link_uid=PMC4030437" image-link="/pmc/articles/PMC4030437/figure/F2/" class="imagepopup">Figure 2.  From: Role of p90RSK in regulating the Crabtree effect: implications for cancer. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Linking High Glucose to Respiratory Inhibition.</div></p></div><div class="aux"><div class="resc">Emily K. Redman, et al. Biochem Soc Trans. ;41(1):124-126.</div><p class="links"><a class="dblinks" href="/pubmed/23356270" ref="ordinalpos=3431&amp;ncbi_uid=6160099&amp;link_uid=23356270">Citation</a><a class="dblinks" href="/pmc/articles/PMC4030437/" ref="ordinalpos=3431&amp;ncbi_uid=6160099&amp;link_uid=PMC4030437">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3432.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3693006/figure/ijo-06-03-280-g004/" ref="ordinalpos=3432&amp;ncbi_uid=5058215&amp;link_uid=PMC3693006" href="/pmc/articles/PMC3693006/"><img src="/pmc/articles/instance/3693006/bin/ijo-06-03-280-g004.gif" src-large="/pmc/articles/instance/3693006/bin/ijo-06-03-280-g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3693006/" ref="ordinalpos=3432&amp;ncbi_uid=5058215&amp;link_uid=PMC3693006" image-link="/pmc/articles/PMC3693006/figure/ijo-06-03-280-g004/" class="imagepopup">Figure 4. Regulation of the LPS-activated inflammatory <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by molecular hydrogen..  From: Molecular hydrogen regulates the expression of miR-9, miR-21 and miR-199 in LPS-activated retinal microglia cells. </a></p><div class="aux"><div class="resc">Guo-Dan Liu, et al. Int J Ophthalmol. 2013;6(3):280-285.</div><p class="links"><a class="dblinks" href="/pubmed/23826519" ref="ordinalpos=3432&amp;ncbi_uid=5058215&amp;link_uid=23826519">Citation</a><a class="dblinks" href="/pmc/articles/PMC3693006/" ref="ordinalpos=3432&amp;ncbi_uid=5058215&amp;link_uid=PMC3693006">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3433.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3289286/figure/F55/" ref="ordinalpos=3433&amp;ncbi_uid=3829378&amp;link_uid=PMC3289286" href="/pmc/articles/PMC3289286/"><img src="/pmc/articles/instance/3289286/bin/nihms348123f55.gif" src-large="/pmc/articles/instance/3289286/bin/nihms348123f55.jpg" alt="Scheme 53" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3289286/" ref="ordinalpos=3433&amp;ncbi_uid=3829378&amp;link_uid=PMC3289286" image-link="/pmc/articles/PMC3289286/figure/F55/" class="imagepopup">Scheme 53.  From: Diamine Ligands in Copper-Catalyzed Reactions. </a></p><div class="supp"><p class="details"><div>IRBM - Merck Research Laboratories synthesis of inhibitors of the hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">David S. Surry, et al. Chem Sci. ;1(1):13-31.</div><p class="links"><a class="dblinks" href="/pubmed/22384310" ref="ordinalpos=3433&amp;ncbi_uid=3829378&amp;link_uid=22384310">Citation</a><a class="dblinks" href="/pmc/articles/PMC3289286/" ref="ordinalpos=3433&amp;ncbi_uid=3829378&amp;link_uid=PMC3289286">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3434.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3012583/figure/F5/" ref="ordinalpos=3434&amp;ncbi_uid=2699695&amp;link_uid=PMC3012583" href="/pmc/articles/PMC3012583/"><img src="/pmc/articles/instance/3012583/bin/WJG-16-6111-g005.gif" src-large="/pmc/articles/instance/3012583/bin/WJG-16-6111-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3012583/" ref="ordinalpos=3434&amp;ncbi_uid=2699695&amp;link_uid=PMC3012583" image-link="/pmc/articles/PMC3012583/figure/F5/" class="imagepopup">Figure 5.  From: Impaired PI3K/Akt signal <span class="highlight" style="background-color:">pathway</span> and hepatocellular injury in high-fat fed rats. </a></p><div class="supp"><p class="details"><div>Simulation diagram of the relationship between insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and hepatocyte apoptosis. PI3K: Phosphatidylinositol 3-kinase; Akt: Protein kinase B; BAD: Bcl2-associated agonist of cell death; MEK: Mitogen-activated protein kinase kinase; clAP-1: Cell inhibitors of apoptosis protein-1; Cyt C: Cytochrome c; APAF: Apoptotic peptidase activating factor.</div></p></div><div class="aux"><div class="resc">Ji-Wu Han, et al. World J Gastroenterol. 2010 December 28;16(48):6111-6118.</div><p class="links"><a class="dblinks" href="/pubmed/21182226" ref="ordinalpos=3434&amp;ncbi_uid=2699695&amp;link_uid=21182226">Citation</a><a class="dblinks" href="/pmc/articles/PMC3012583/" ref="ordinalpos=3434&amp;ncbi_uid=2699695&amp;link_uid=PMC3012583">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3435.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3117038/figure/F1/" ref="ordinalpos=3435&amp;ncbi_uid=3367440&amp;link_uid=PMC3117038" href="/pmc/articles/PMC3117038/"><img src="/pmc/articles/instance/3117038/bin/cc1009_1343_fig001.gif" src-large="/pmc/articles/instance/3117038/bin/cc1009_1343_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3117038/" ref="ordinalpos=3435&amp;ncbi_uid=3367440&amp;link_uid=PMC3117038" image-link="/pmc/articles/PMC3117038/figure/F1/" class="imagepopup">Figure 1.  From: Roles of protein arginine methylation in DNA damage <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div>The roles of CARM1 and coactivator methylation in DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. DNA damage activates ATM kinase and promotes the activity of tumor suppressors p53 and BRCA1. CARM1 methylates coactivator p300 and histones and induces coactivator complex formation with BRCA1 and other BRCT domain proteins for expression of p21, Gadd45 and other CDKIs. This leads to cell cycle arrest and DNA damage repair in response to genotoxic stress. CARM1, p300 and BRCA1 are not required for expression of genes involved in the apoptosis branch of the DNA damage response <span class="highlight" style="background-color:">pathway</span>. CARM1 may be an essential factor inducing cell cycle arrest and cell survival rather than apoptosis in the DNA damage <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Young-Ho Lee, et al. Cell Cycle. 2011 May 1;10(9):1343-1344.</div><p class="links"><a class="dblinks" href="/pubmed/21445011" ref="ordinalpos=3435&amp;ncbi_uid=3367440&amp;link_uid=21445011">Citation</a><a class="dblinks" href="/pmc/articles/PMC3117038/" ref="ordinalpos=3435&amp;ncbi_uid=3367440&amp;link_uid=PMC3117038">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3436.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3928099/figure/pone-0088034-g006/" ref="ordinalpos=3436&amp;ncbi_uid=5831803&amp;link_uid=PMC3928099" href="/pmc/articles/PMC3928099/"><img src="/pmc/articles/instance/3928099/bin/pone.0088034.g006.gif" src-large="/pmc/articles/instance/3928099/bin/pone.0088034.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3928099/" ref="ordinalpos=3436&amp;ncbi_uid=5831803&amp;link_uid=PMC3928099" image-link="/pmc/articles/PMC3928099/figure/pone-0088034-g006/" class="imagepopup">Figure 6. The PI3K/AKT/GSK3β/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is involved in ER-α36-mediated mitogenic estrogen <span class="highlight" style="background-color:">signaling</span> of ER-positive breast cancer stem/progenitor cells..  From: ER-?36-Mediated Rapid Estrogen <span class="highlight" style="background-color:">Signaling</span> Positively Regulates ER-Positive Breast Cancer Stem/Progenitor Cells. </a></p><div class="supp"><p class="details"><div>(A). Western blot analysis of the cell lysates from tumorspheres derived from MCF7 and T47D cells treated with ethanol (V); 0.1 nM of E2β; the AKT inhibitor IV (10 µM), IAkt; and E2β+IAkt, using indicated antibodies. (B). The effects of different inhibitors of the PI3K/AKT/GSK3β <span class="highlight" style="background-color:">pathway</span> on estrogen-stimulated growth of ER-positive breast cancer stem/progenitor cells. Tumorspheres of MCF7 and T47D cells were treated with vehicle, E2β alone or E2β together with the PI3K inhibitor LY294002 (10 µM), the GSK-3β inhibitor IX (10 µM), the AKT inhibitor IV (10 µM). After seven days, cell numbers from dissociated tumorspheres were determined. The columns represent the means of three experiments; bars, SE.</div></p></div><div class="aux"><div class="resc">Hao Deng, et al. PLoS One. 2014;9(2):e88034.</div><p class="links"><a class="dblinks" href="/pubmed/24558373" ref="ordinalpos=3436&amp;ncbi_uid=5831803&amp;link_uid=24558373">Citation</a><a class="dblinks" href="/pmc/articles/PMC3928099/" ref="ordinalpos=3436&amp;ncbi_uid=5831803&amp;link_uid=PMC3928099">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3437.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1304358/figure/fig5/" ref="ordinalpos=3437&amp;ncbi_uid=662992&amp;link_uid=PMC1304358" href="/pmc/articles/PMC1304358/"><img src="/pmc/articles/instance/1304358/bin/biophysj00029884F05_LW.gif" src-large="/pmc/articles/instance/1304358/bin/biophysj00029884F05_LW.jpg" alt="FIGURE 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1304358/" ref="ordinalpos=3437&amp;ncbi_uid=662992&amp;link_uid=PMC1304358" image-link="/pmc/articles/PMC1304358/figure/fig5/" class="imagepopup">FIGURE 5.  From: The JAK-STAT <span class="highlight" style="background-color:">Signaling</span> Network in the Human B-Cell: An Extreme <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Analysis. </a></p><div class="supp"><p class="details"><div>Reaction participation in JAK-STAT <span class="highlight" style="background-color:">signaling</span> network. The percentage of the extreme <span class="highlight" style="background-color:">signaling</span> pathways in which the corresponding reaction participates is indicated on the <i>y</i> axis. The inset table only lists the reaction participation values greater than 20% (abbreviations in supplementary material). The inset plot is the reaction participation plotted on a log-log scale.</div></p></div><div class="aux"><div class="resc">Jason A. Papin, et al. Biophys J. 2004 July;87(1):37-46.</div><p class="links"><a class="dblinks" href="/pubmed/15240442" ref="ordinalpos=3437&amp;ncbi_uid=662992&amp;link_uid=15240442">Citation</a><a class="dblinks" href="/pmc/articles/PMC1304358/" ref="ordinalpos=3437&amp;ncbi_uid=662992&amp;link_uid=PMC1304358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3438.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2699148/figure/fig10/" ref="ordinalpos=3438&amp;ncbi_uid=1860073&amp;link_uid=PMC2699148" href="/pmc/articles/PMC2699148/"><img src="/pmc/articles/instance/2699148/bin/JCB_200807186_RGB_Fig10.gif" src-large="/pmc/articles/instance/2699148/bin/JCB_200807186_RGB_Fig10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2699148/" ref="ordinalpos=3438&amp;ncbi_uid=1860073&amp;link_uid=PMC2699148" image-link="/pmc/articles/PMC2699148/figure/fig10/" class="imagepopup">Figure 10. From: A clathrin-dependent <span class="highlight" style="background-color:">pathway</span> leads to KRas <span class="highlight" style="background-color:">signaling</span> on late endosomes en route to lysosomes. </a></p><div class="supp"><p class="details"><div><b>Model of the endosomal KRas traffic and <span class="highlight" style="background-color:">signaling</span>.</b> EGF binding triggers endocytosis of the EGFR and downstream <span class="highlight" style="background-color:">signaling</span> molecules to the early endosomal compartment. There, Ras <span class="highlight" style="background-color:">signaling</span> modules settle to elicit signal outputs. HRas is ubiquitinated and is recycled back to the PM, whereas KRas is sorted en route to LEs/MVBs. Once there, KRas can be GTP loaded, thus being able to recruit effectors such as Raf1, which initiates the MAPK <span class="highlight" style="background-color:">signaling</span> cascade on this compartment. Finally, KRas down-regulation/degradation may occur in the late endosomal/lysosomal compartment. CCV, clathrin-coated vesicle.</div></p></div><div class="aux"><div class="resc">Albert Lu, et al. J Cell Biol. 2009 March 23;184(6):863-879.</div><p class="links"><a class="dblinks" href="/pubmed/19289794" ref="ordinalpos=3438&amp;ncbi_uid=1860073&amp;link_uid=19289794">Citation</a><a class="dblinks" href="/pmc/articles/PMC2699148/" ref="ordinalpos=3438&amp;ncbi_uid=1860073&amp;link_uid=PMC2699148">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3439.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC338282/figure/fig8/" ref="ordinalpos=3439&amp;ncbi_uid=439830&amp;link_uid=PMC338282" href="/pmc/articles/PMC338282/"><img src="/pmc/articles/instance/338282/bin/73198-06f8_L1OT_rev1.gif" src-large="/pmc/articles/instance/338282/bin/73198-06f8_L1OT_rev1.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC338282/" ref="ordinalpos=3439&amp;ncbi_uid=439830&amp;link_uid=PMC338282" image-link="/pmc/articles/PMC338282/figure/fig8/" class="imagepopup">Figure 8. From: Constitutive activation of MEK1 in chondrocytes causes Stat1-independent achondroplasia-like dwarfism and rescues the Fgfr3-deficient mouse phenotype. </a></p><div class="supp"><p class="details"><div>Model for downstream <span class="highlight" style="background-color:">signaling</span> pathways that mediate the inhibitory effects of FGFR3 <span class="highlight" style="background-color:">signaling</span> on bone growth. The MAPK <span class="highlight" style="background-color:">pathway</span> mediates inhibition of hypertrophic chondrocyte differentiation, whereas Stat1 mediates inhibition of chondrocyte proliferation.</div></p></div><div class="aux"><div class="resc">Shunichi Murakami, et al. Genes Dev. 2004 February 1;18(3):290-305.</div><p class="links"><a class="dblinks" href="/pubmed/14871928" ref="ordinalpos=3439&amp;ncbi_uid=439830&amp;link_uid=14871928">Citation</a><a class="dblinks" href="/pmc/articles/PMC338282/" ref="ordinalpos=3439&amp;ncbi_uid=439830&amp;link_uid=PMC338282">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3440.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2702845/figure/F7/" ref="ordinalpos=3440&amp;ncbi_uid=1913803&amp;link_uid=PMC2702845" href="/pmc/articles/PMC2702845/"><img src="/pmc/articles/instance/2702845/bin/nihms107615f7.gif" src-large="/pmc/articles/instance/2702845/bin/nihms107615f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2702845/" ref="ordinalpos=3440&amp;ncbi_uid=1913803&amp;link_uid=PMC2702845" image-link="/pmc/articles/PMC2702845/figure/F7/" class="imagepopup">Fig. 7.  From: Mechanism of purinergic activation of endothelial nitric oxide synthase in endothelial cells. </a></p><div class="supp"><p class="details"><div>Proposed <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of the extracellular nucleotide-induced phosphorylation of eNOS in EC.</div></p></div><div class="aux"><div class="resc">Cleide Gonçalves da Silva, et al. Circulation. ;119(6):871-879.</div><p class="links"><a class="dblinks" href="/pubmed/19188511" ref="ordinalpos=3440&amp;ncbi_uid=1913803&amp;link_uid=19188511">Citation</a><a class="dblinks" href="/pmc/articles/PMC2702845/" ref="ordinalpos=3440&amp;ncbi_uid=1913803&amp;link_uid=PMC2702845">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3441.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3440220/figure/fig2/" ref="ordinalpos=3441&amp;ncbi_uid=5251154&amp;link_uid=PMC3440220" href="/pmc/articles/PMC3440220/"><img src="/pmc/articles/instance/3440220/bin/PP_200527_wc_f2.gif" src-large="/pmc/articles/instance/3440220/bin/PP_200527_wc_f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3440220/" ref="ordinalpos=3441&amp;ncbi_uid=5251154&amp;link_uid=PMC3440220" image-link="/pmc/articles/PMC3440220/figure/fig2/" class="imagepopup">Figure 2. From: Ubiquitin-Mediated Control of Plant Hormone Signaling1. </a></p><div class="supp"><p class="details"><div>E3 ligases involved in ABA <span class="highlight" style="background-color:">signaling</span>. The schematic of known UPS components in ABA <span class="highlight" style="background-color:">signaling</span> illustrates that there is regulation of a single substrate (ABI5) by multiple E3 ligases and a number of different classes of E3s act at multiple levels in the ABA <span class="highlight" style="background-color:">pathway</span>, including biosynthesis and downstream gene regulation. RING-type E3 ligases are shown as yellow pentagons, U-box-type E3 ligases as dark orange pentagons, CRL E3 ligases as orange pentagons, and CRL3-based E3 ligases as pink pentagons. AFP is depicted is a green square, and COP1 is a gray triangle. Known (or unknown) ligase substrates are blue pie wedges.</div></p></div><div class="aux"><div class="resc">Dior R. Kelley, et al. Plant Physiol. 2012 September;160(1):47-55.</div><p class="links"><a class="dblinks" href="/pubmed/22723083" ref="ordinalpos=3441&amp;ncbi_uid=5251154&amp;link_uid=22723083">Citation</a><a class="dblinks" href="/pmc/articles/PMC3440220/" ref="ordinalpos=3441&amp;ncbi_uid=5251154&amp;link_uid=PMC3440220">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3442.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4055569/figure/pcbi-1003666-g002/" ref="ordinalpos=3442&amp;ncbi_uid=6262929&amp;link_uid=PMC4055569" href="/pmc/articles/PMC4055569/"><img src="/pmc/articles/instance/4055569/bin/pcbi.1003666.g002.gif" src-large="/pmc/articles/instance/4055569/bin/pcbi.1003666.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4055569/" ref="ordinalpos=3442&amp;ncbi_uid=6262929&amp;link_uid=PMC4055569" image-link="/pmc/articles/PMC4055569/figure/pcbi-1003666-g002/" class="imagepopup">Figure 2. The KEGG p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> entry showing the <span class="highlight" style="background-color:">pathway</span> map and some reference citations..  From: Augmenting Microarray Data with Literature-Based Knowledge to Enhance Gene Regulatory Network Inference. </a></p><div class="supp"><p class="details"><div>The PMIDs for each reference citation were used to identify MeSH terms with PubMed (see section <i>Predication network generation</i>). http://www.genome.jp/kegg-bin/show_<span class="highlight" style="background-color:">pathway</span>?hsa04115.</div></p></div><div class="aux"><div class="resc">Guocai Chen, et al. PLoS Comput Biol. 2014 June;10(6):e1003666.</div><p class="links"><a class="dblinks" href="/pubmed/24921649" ref="ordinalpos=3442&amp;ncbi_uid=6262929&amp;link_uid=24921649">Citation</a><a class="dblinks" href="/pmc/articles/PMC4055569/" ref="ordinalpos=3442&amp;ncbi_uid=6262929&amp;link_uid=PMC4055569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3443.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657919/figure/F3/" ref="ordinalpos=3443&amp;ncbi_uid=1706775&amp;link_uid=PMC2657919" href="/pmc/articles/PMC2657919/"><img src="/pmc/articles/instance/2657919/bin/nihms93245f3.gif" src-large="/pmc/articles/instance/2657919/bin/nihms93245f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657919/" ref="ordinalpos=3443&amp;ncbi_uid=1706775&amp;link_uid=PMC2657919" image-link="/pmc/articles/PMC2657919/figure/F3/" class="imagepopup">Figure 3.  From: Acquisition of Epithelial-Mesenchymal Transition phenotype of gemcitabine-resistant pancreatic cancer cells is linked with activation of Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Notch <span class="highlight" style="background-color:">pathway</span> is up-regualted in GR cells.<br />A: Real-time RT-PCR was used to quantify Notch <span class="highlight" style="background-color:">pathway</span> genes at the mRNA levels in GS and GR cells. *, P&lt;0.05, **, P&lt;0.01 compared with GS.<br />B: Western blot analysis showing the expression of Notch <span class="highlight" style="background-color:">pathway</span> related proteins.<br />C: Immunofluorescence staining for the expression and cellular localization for Notch-2, Notch-4 and Jagged-1.</div></p></div><div class="aux"><div class="resc">Zhiwei Wang, et al. Cancer Res. ;69(6):2400-2407.</div><p class="links"><a class="dblinks" href="/pubmed/19276344" ref="ordinalpos=3443&amp;ncbi_uid=1706775&amp;link_uid=19276344">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657919/" ref="ordinalpos=3443&amp;ncbi_uid=1706775&amp;link_uid=PMC2657919">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3444.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2970559/figure/pone-0015384-g007/" ref="ordinalpos=3444&amp;ncbi_uid=2570523&amp;link_uid=PMC2970559" href="/pmc/articles/PMC2970559/"><img src="/pmc/articles/instance/2970559/bin/pone.0015384.g007.gif" src-large="/pmc/articles/instance/2970559/bin/pone.0015384.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2970559/" ref="ordinalpos=3444&amp;ncbi_uid=2570523&amp;link_uid=PMC2970559" image-link="/pmc/articles/PMC2970559/figure/pone-0015384-g007/" class="imagepopup">Figure 7. Proposed mechanism of HGF-mediated suppression..  From: Hepatocyte Growth Factor Modulates Interleukin-6 Production in Bone Marrow Derived Macrophages: Implications for Inflammatory Mediated Diseases. </a></p><div class="supp"><p class="details"><div>The canonical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of LPS-TLR engagement leads to NFκB dependent pro-inflammatory cytokine production through the interaction of CBP with NFκB. However, TLR <span class="highlight" style="background-color:">signaling</span> has also been shown to weakly activate alternative <span class="highlight" style="background-color:">signaling</span> through PI3K, resulting in phosphorylation and inactivation of GSK3β, subsequent sequestration of CBP from NFκB to phospho-CREB, and resultant anti-inflammatory (IL-10) production[7]. Our results show that the presence of HGF enhances the IL-10 <span class="highlight" style="background-color:">pathway</span>. We postulate that in order to resolve inflammation, the generation of IL-6 by LPS-TLR <span class="highlight" style="background-color:">signaling</span> leads to the production of HGF, ultimately leading to the inhibition of inflammation. Hence, we propose HGF acts as an internal rheostat for resolving acute phase inflammatory responses.</div></p></div><div class="aux"><div class="resc">Gina M. Coudriet, et al. PLoS One. 2010;5(11):e15384.</div><p class="links"><a class="dblinks" href="/pubmed/21072211" ref="ordinalpos=3444&amp;ncbi_uid=2570523&amp;link_uid=21072211">Citation</a><a class="dblinks" href="/pmc/articles/PMC2970559/" ref="ordinalpos=3444&amp;ncbi_uid=2570523&amp;link_uid=PMC2970559">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3445.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1681468/figure/f2/" ref="ordinalpos=3445&amp;ncbi_uid=809201&amp;link_uid=PMC1681468" href="/pmc/articles/PMC1681468/"><img src="/pmc/articles/instance/1681468/bin/msb4100014-f2.gif" src-large="/pmc/articles/instance/1681468/bin/msb4100014-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1681468/" ref="ordinalpos=3445&amp;ncbi_uid=809201&amp;link_uid=PMC1681468" image-link="/pmc/articles/PMC1681468/figure/f2/" class="imagepopup">Figure 2.  From: A comprehensive <span class="highlight" style="background-color:">pathway</span> map of epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The bow-tie architecture of the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. A simplified diagram was created based on the EGFR <span class="highlight" style="background-color:">signaling</span> map in Figure 1. Arrows in this figure represent an informal notation of 'flow of reaction'. Various ligands bind to diverse receptor heterodimers, which then converge into a handful of molecules building a conserved core. Activities of these molecules play important roles in controlling diverse responses. Notable interactions are color-coded: red, positive feedback loop; blue, negative feedback loop; purple, inhibitory feed-forward path; green, crosstalk from GPCR cascade to EGFR cascade via calcium release. This image is also available as high resolution PDF (see Supplementary PDF 2) or Scalable Vector Graphic (SVG).</div></p></div><div class="aux"><div class="resc">Kanae Oda, et al. Mol Syst Biol. 2005;1:2005.0010-2005.0010.</div><p class="links"><a class="dblinks" href="/pubmed/16729045" ref="ordinalpos=3445&amp;ncbi_uid=809201&amp;link_uid=16729045">Citation</a><a class="dblinks" href="/pmc/articles/PMC1681468/" ref="ordinalpos=3445&amp;ncbi_uid=809201&amp;link_uid=PMC1681468">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3446.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4011599/figure/F2/" ref="ordinalpos=3446&amp;ncbi_uid=6111466&amp;link_uid=PMC4011599" href="/pmc/articles/PMC4011599/"><img src="/pmc/articles/instance/4011599/bin/oncotarget-05-978-g002.gif" src-large="/pmc/articles/instance/4011599/bin/oncotarget-05-978-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4011599/" ref="ordinalpos=3446&amp;ncbi_uid=6111466&amp;link_uid=PMC4011599" image-link="/pmc/articles/PMC4011599/figure/F2/" class="imagepopup">Figure 2. Resuming scheme of the Wnt <span class="highlight" style="background-color:">pathway</span> components and carcinogenetic stages analyzed in this study.  From: WNT-<span class="highlight" style="background-color:">pathway</span> components as predictive markers useful for diagnosis, prevention and therapy in inflammatory bowel disease and sporadic colorectal cancer. </a></p><div class="supp"><p class="details"><div>For each molecular components, the function in Wnt <span class="highlight" style="background-color:">signaling</span> (left column), the expression pattern in normal mucosa in absence of Wnt <span class="highlight" style="background-color:">signaling</span> (second column) and the modification indicative of Wnt <span class="highlight" style="background-color:">signaling</span> activation (third column), are reported.</div></p></div><div class="aux"><div class="resc">Annalucia Serafino, et al. Oncotarget. 2014 April;5(4):978-992.</div><p class="links"><a class="dblinks" href="/pubmed/24657851" ref="ordinalpos=3446&amp;ncbi_uid=6111466&amp;link_uid=24657851">Citation</a><a class="dblinks" href="/pmc/articles/PMC4011599/" ref="ordinalpos=3446&amp;ncbi_uid=6111466&amp;link_uid=PMC4011599">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3447.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3458565/figure/gks627-F2/" ref="ordinalpos=3447&amp;ncbi_uid=4256790&amp;link_uid=PMC3458565" href="/pmc/articles/PMC3458565/"><img src="/pmc/articles/instance/3458565/bin/gks627f2.gif" src-large="/pmc/articles/instance/3458565/bin/gks627f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3458565/" ref="ordinalpos=3447&amp;ncbi_uid=4256790&amp;link_uid=PMC3458565" image-link="/pmc/articles/PMC3458565/figure/gks627-F2/" class="imagepopup">Figure 2. From: Involvement of microRNA families in cancer. </a></p><div class="supp"><p class="details"><div>Involvement of miRs in <span class="highlight" style="background-color:">signaling</span> pathways. In the heatmap, we counted the number of cancer genes in a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that were targeted by a given family of miRs. Ward clustering such a matrix, we found a large cluster of families that strongly interacted with cancer genes in <span class="highlight" style="background-color:">signaling</span> pathways and significantly pooled a large fraction of cancer miR families. Cancer miRs collected from the literature were indicated in red in their corresponding families.</div></p></div><div class="aux"><div class="resc">Stefan Wuchty, et al. Nucleic Acids Res. 2012 September;40(17):8219-8226.</div><p class="links"><a class="dblinks" href="/pubmed/22743268" ref="ordinalpos=3447&amp;ncbi_uid=4256790&amp;link_uid=22743268">Citation</a><a class="dblinks" href="/pmc/articles/PMC3458565/" ref="ordinalpos=3447&amp;ncbi_uid=4256790&amp;link_uid=PMC3458565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3448.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2919366/figure/F4/" ref="ordinalpos=3448&amp;ncbi_uid=2406163&amp;link_uid=PMC2919366" href="/pmc/articles/PMC2919366/"><img src="/pmc/articles/instance/2919366/bin/nihms109428f4.gif" src-large="/pmc/articles/instance/2919366/bin/nihms109428f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2919366/" ref="ordinalpos=3448&amp;ncbi_uid=2406163&amp;link_uid=PMC2919366" image-link="/pmc/articles/PMC2919366/figure/F4/" class="imagepopup">Figure 4.  From: Clinical Manifestations, Etiology, and Pathogenesis of the Hyper IgE Syndromes. </a></p><div class="supp"><p class="details"><div>STAT3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Multiple cytokines, including Il-6, IL-10, IL-21, IL-22, and IL-23 activate STAT3 phosphorylation. Dimerized STAT3 enters the nucleus to affect transcription of various genes. STAT3 can also dimerize with other STATs. Normal STAT3 <span class="highlight" style="background-color:">signaling</span> leads to the upregulation and down regulation of the cytokines shown in the right lower corner. IL-10 and SOCS (suppressors of cytokine <span class="highlight" style="background-color:">signaling</span>) can downregulate STAT3 activity.</div></p></div><div class="aux"><div class="resc">Alexandra F. Freeman, et al. Pediatr Res. ;65(5 Pt 2):32R-37R.</div><p class="links"><a class="dblinks" href="/pubmed/19190525" ref="ordinalpos=3448&amp;ncbi_uid=2406163&amp;link_uid=19190525">Citation</a><a class="dblinks" href="/pmc/articles/PMC2919366/" ref="ordinalpos=3448&amp;ncbi_uid=2406163&amp;link_uid=PMC2919366">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3449.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2906687/figure/F2/" ref="ordinalpos=3449&amp;ncbi_uid=2557646&amp;link_uid=PMC2906687" href="/pmc/articles/PMC2906687/"><img src="/pmc/articles/instance/2906687/bin/nihms-190496-f0002.gif" src-large="/pmc/articles/instance/2906687/bin/nihms-190496-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2906687/" ref="ordinalpos=3449&amp;ncbi_uid=2557646&amp;link_uid=PMC2906687" image-link="/pmc/articles/PMC2906687/figure/F2/" class="imagepopup">Figure 2.  From: Role of Adenosine <span class="highlight" style="background-color:">Signaling</span> in Penile Erection and Erectile Disorders. </a></p><div class="supp"><p class="details"><div>Working model of adenosine mediated intracellular <span class="highlight" style="background-color:">signaling</span> in corpus cavernosum. Adenosine induces synthesis of both, cGMP and cAMP, in corpus cavernosum. cAMP induction is mediated through A<sub>2B</sub> receptor on smooth muscle cells. Induction of cGMP by adenosine is most possibly mediated by activation of nitric oxide (NO) synthesis in endothelial cells through A<sub>2B</sub> receptor <span class="highlight" style="background-color:">signaling</span>. NO upon diffusing into smooth muscle cells may induce cGMP. Both, cAMP and cGMP, activate their downstream kinases, PKA and PKG, respectively, which in turn phosphorylate their downstream targets resulting in a decrease in intracellular Ca<sup>+2</sup> concentration that leads to smooth muscle relaxation. PKG also phosphorylates RhoA and inhibits RhoA-Rho kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, thereby contributing to smooth muscle relaxation.</div></p></div><div class="aux"><div class="resc">Prasad V. Phatarpekar, et al. J Sex Med. ;7(11):3553-3564.</div><p class="links"><a class="dblinks" href="/pubmed/19889148" ref="ordinalpos=3449&amp;ncbi_uid=2557646&amp;link_uid=19889148">Citation</a><a class="dblinks" href="/pmc/articles/PMC2906687/" ref="ordinalpos=3449&amp;ncbi_uid=2557646&amp;link_uid=PMC2906687">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3450.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418676/figure/F1/" ref="ordinalpos=3450&amp;ncbi_uid=4547880&amp;link_uid=PMC3418676" href="/pmc/articles/PMC3418676/"><img src="/pmc/articles/instance/3418676/bin/nihms397212f1.gif" src-large="/pmc/articles/instance/3418676/bin/nihms397212f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418676/" ref="ordinalpos=3450&amp;ncbi_uid=4547880&amp;link_uid=PMC3418676" image-link="/pmc/articles/PMC3418676/figure/F1/" class="imagepopup">Fig. 1. Fat <span class="highlight" style="background-color:">signaling</span> and Dachs polarity in the wing disc.  From: Propagation of Dachsous-Fat Planar Cell Polarity. </a></p><div class="supp"><p class="details"><div>A) Simplified schematic of the Fat <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Fat is regulated by Dachsous and Four-jointed. Fat regulates Hippo <span class="highlight" style="background-color:">signaling</span> (red arrows) through Dachs, and regulates PCP <span class="highlight" style="background-color:">signaling</span> (green arrows) partly through Dachs and partly independently of Dachs. B) Schematic of wing disc with the Ds gradient indicated in magenta, and the direction of Dachs polarization indicated by green arrows. The A-P and D-V compartment boundaries are indicated by thin black lines, and Wg expression is indicated in red. C) Dachs:Cit polarization in a wing disc with wild-type Ds and Fj expression, stained for Wg (red), <i>hh-Gal4</i> (yellow, revealed by Dcr2 staining from a <i>UAS-Dcr2</i> transgene), and E-cadherin (E-cad, blue, outlines all cells). The polarity of Dachs localization is indicated by small white arrows pointing in the direction of Dachs:Cit membrane localization. Large Red arrow points to the D-V Wg stripe. D) Ds protein staining in a wild-type wing disc. E) Fj expression in a wild-type wing disc, revealed by a <i>fj-lacZ</i> transgene.</div></p></div><div class="aux"><div class="resc">Abhijit A. Ambegaonkar, et al. Curr Biol. ;22(14):1302-1308.</div><p class="links"><a class="dblinks" href="/pubmed/22727698" ref="ordinalpos=3450&amp;ncbi_uid=4547880&amp;link_uid=22727698">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418676/" ref="ordinalpos=3450&amp;ncbi_uid=4547880&amp;link_uid=PMC3418676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3451.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2991117/figure/F6/" ref="ordinalpos=3451&amp;ncbi_uid=2588314&amp;link_uid=PMC2991117" href="/pmc/articles/PMC2991117/"><img src="/pmc/articles/instance/2991117/bin/nihms242216f6.gif" src-large="/pmc/articles/instance/2991117/bin/nihms242216f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2991117/" ref="ordinalpos=3451&amp;ncbi_uid=2588314&amp;link_uid=PMC2991117" image-link="/pmc/articles/PMC2991117/figure/F6/" class="imagepopup">Figure 6. Caspase-3 affects <span class="highlight" style="background-color:">signaling</span> via Ras-Raf-MEK-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in primitive hematopoietic cells in a cytokine specific manner.  From: Hematopoietic stem cell responsiveness to exogenous signals limited by Caspase-3. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Stimulation of primary bone marrow cells with either 1ng/ml SCF or IL-3 revealed an augmented signal transduction, while stimulation with 4 ng/ml TPO resulted in decreased signal intensity via the ERK-MAPK <span class="highlight" style="background-color:">pathway</span> in absence of Caspase-3. FACS plots demonstrated are representative from three independent experiments and are gated on LK<sup>+</sup>S<sup>+</sup> cells.<br /><b>(B)</b> Stimulation with 10 ng/ml IL-6, SDF, or Ang1 did not reveal measurable phosphorylation of ERK1/2 in LK<sup>+</sup>S<sup>+</sup> cells.<br /><b>(C)</b> Differential activation of MEK upon stimulation with SCF in Caspase-3 KO LK<sup>+</sup>S<sup>+</sup> cells indicate an alteration in ERK <span class="highlight" style="background-color:">signaling</span> upstream of MEK.<br /><b>(D)</b> No difference between the genotypes was observed in the kinetic of AKT- or Stat3-phosphorylation using the same conditions as for ERK detection</div></p></div><div class="aux"><div class="resc">Viktor Janzen, et al. Cell Stem Cell. ;2(6):584-594.</div><p class="links"><a class="dblinks" href="/pubmed/18522851" ref="ordinalpos=3451&amp;ncbi_uid=2588314&amp;link_uid=18522851">Citation</a><a class="dblinks" href="/pmc/articles/PMC2991117/" ref="ordinalpos=3451&amp;ncbi_uid=2588314&amp;link_uid=PMC2991117">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3452.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2757549/figure/fig6/" ref="ordinalpos=3452&amp;ncbi_uid=2494066&amp;link_uid=PMC2757549" href="/pmc/articles/PMC2757549/"><img src="/pmc/articles/instance/2757549/bin/carcinbgp202f06_lw.gif" src-large="/pmc/articles/instance/2757549/bin/carcinbgp202f06_lw.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2757549/" ref="ordinalpos=3452&amp;ncbi_uid=2494066&amp;link_uid=PMC2757549" image-link="/pmc/articles/PMC2757549/figure/fig6/" class="imagepopup">Fig. 6. From: OSU-A9, a potent indole-3-carbinol derivative, suppresses breast tumor growth by targeting the Akt-NF-?B <span class="highlight" style="background-color:">pathway</span> and stress response <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>Diagram depicting the inhibitory and activating effects of OSU-A9 on the Akt–NF-κB <span class="highlight" style="background-color:">signaling</span> axis and stress response <span class="highlight" style="background-color:">signaling</span>, respectively. The ability of OSU-A9 to affect these two <span class="highlight" style="background-color:">signaling</span> networks and thus target different facets of breast malignancy underlies its high potency in suppressing tumor growth.</div></p></div><div class="aux"><div class="resc">Jing-Ru Weng, et al. Carcinogenesis. 2009 October;30(10):1702-1709.</div><p class="links"><a class="dblinks" href="/pubmed/19706645" ref="ordinalpos=3452&amp;ncbi_uid=2494066&amp;link_uid=19706645">Citation</a><a class="dblinks" href="/pmc/articles/PMC2757549/" ref="ordinalpos=3452&amp;ncbi_uid=2494066&amp;link_uid=PMC2757549">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3453.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3121168/figure/F3/" ref="ordinalpos=3453&amp;ncbi_uid=3037916&amp;link_uid=PMC3121168" href="/pmc/articles/PMC3121168/"><img src="/pmc/articles/instance/3121168/bin/nihms-302889-f0004.gif" src-large="/pmc/articles/instance/3121168/bin/nihms-302889-f0004.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3121168/figure/F3/" ref="ordinalpos=3453&amp;ncbi_uid=3037916&amp;link_uid=PMC3121168" href="/pmc/articles/PMC3121168/"><img src="/pmc/articles/instance/3121168/bin/nihms-302889-f0005.gif" src-large="/pmc/articles/instance/3121168/bin/nihms-302889-f0005.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3121168/" ref="ordinalpos=3453&amp;ncbi_uid=3037916&amp;link_uid=PMC3121168" image-link="/pmc/articles/PMC3121168/figure/F3/" class="imagepopup">Figure 3. Localization of Hh <span class="highlight" style="background-color:">pathway</span> complexes in Drosophila and mammals.  From: The Primary Cilium: A <span class="highlight" style="background-color:">Signaling</span> Center During Vertebrate Development. </a></p><div class="supp"><p class="details"><div><b>a</b> | Modulation of protein complex structure and localization in <i>Drosophila</i> by Hh. In the absence of ligand, Ptch prevents translocation of Smo to the plasma membrane. A microtubule-associated complex including Cos2, Fu, Sufu and Ci recruits kinases including PKA, CK1, GSK3β that promote the processing of Ci into its repressor form (CiR)<sup>130–133</sup>. Sufu may also associate with full length Ci to prevent its translocation to the nucleus<sup>130</sup>. Upon activation of the <span class="highlight" style="background-color:">pathway</span>, Smo moves to the plasma membrane, and the Fu-Cos2 complex associates with the C-terminal tail of Smo resulting in the release of Ci. <span class="highlight" style="background-color:">Pathway</span> activation also inactivates the negative regulator<sup>130–133</sup>. <b>b</b> | In vertebrate Hh <span class="highlight" style="background-color:">signaling</span>, signal transduction takes place within cilia, but the behavior of protein complexes may parallel that in <i>Drosophila</i>. In the absence of ligand, Ptch localizes to the cilium, and is thought to block the entry of Smo into cilia<sup>35</sup>. Kif7 (the Cos2 homologue) localizes to the base of the cilium<sup>55</sup> where it may form a complex Gli proteins and other <span class="highlight" style="background-color:">pathway</span> components. Kif7 at the cilium base prevents Gli enrichment within the cilium and promotes processing of GliR. Upon activation of the <span class="highlight" style="background-color:">pathway</span>, Smo moves to the ciliary membrane and Kif7 translocates into the cilium thereby promoting Gli2 accumulation at the cilium tip<sup>40, 55</sup>. Kif7 at the cilia tip may also block the function of Sufu. Activated Gli is transported out of the cilium by Dynein and IFT particles.</div></p></div><div class="aux"><div class="resc">Sarah C. Goetz, et al. Nat Rev Genet. ;11(5):331-344.</div><p class="links"><a class="dblinks" href="/pubmed/20395968" ref="ordinalpos=3453&amp;ncbi_uid=3037916&amp;link_uid=20395968">Citation</a><a class="dblinks" href="/pmc/articles/PMC3121168/" ref="ordinalpos=3453&amp;ncbi_uid=3037916&amp;link_uid=PMC3121168">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3454.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3232251/figure/f1-0961735/" ref="ordinalpos=3454&amp;ncbi_uid=3708044&amp;link_uid=PMC3232251" href="/pmc/articles/PMC3232251/"><img src="/pmc/articles/instance/3232251/bin/961735.fig1.gif" src-large="/pmc/articles/instance/3232251/bin/961735.fig1.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3232251/" ref="ordinalpos=3454&amp;ncbi_uid=3708044&amp;link_uid=PMC3232251" image-link="/pmc/articles/PMC3232251/figure/f1-0961735/" class="imagepopup">Figure 1. From: The Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in hematopoiesis and hematologic malignancies. </a></p><div class="supp"><p class="details"><div>Notch1 <span class="highlight" style="background-color:">signaling</span> influences generation of both definitive myeloid cells and lymphoid cells.</div></p></div><div class="aux"><div class="resc">Ralf Schwanbeck, et al. Haematologica. 2011 December;96(12):1735-1737.</div><p class="links"><a class="dblinks" href="/pubmed/22147769" ref="ordinalpos=3454&amp;ncbi_uid=3708044&amp;link_uid=22147769">Citation</a><a class="dblinks" href="/pmc/articles/PMC3232251/" ref="ordinalpos=3454&amp;ncbi_uid=3708044&amp;link_uid=PMC3232251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3455.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3626841/figure/F4/" ref="ordinalpos=3455&amp;ncbi_uid=4825825&amp;link_uid=PMC3626841" href="/pmc/articles/PMC3626841/"><img src="/pmc/articles/instance/3626841/bin/1752-0509-7-13-4.gif" src-large="/pmc/articles/instance/3626841/bin/1752-0509-7-13-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3626841/" ref="ordinalpos=3455&amp;ncbi_uid=4825825&amp;link_uid=PMC3626841" image-link="/pmc/articles/PMC3626841/figure/F4/" class="imagepopup">Figure 4.  From: Model composition through model reduction: a combined model of CD95 and NF-?B <span class="highlight" style="background-color:">signaling</span> pathways. </a></p><div class="supp"><p class="details"><div><b>The original model by Neumann </b><b><i>et al. </i></b><b>and results of the model reduction. A.</b> The original model decomposed into modules according to three steps of the CD95 <span class="highlight" style="background-color:">signaling</span> pathways: activation of caspase-8, pro-apoptotic <span class="highlight" style="background-color:">pathway</span> resulting in caspase-3 activation and anti-apoptotic <span class="highlight" style="background-color:">pathway</span> regulated by NF-κB. The species retained during the model reduction are colored; reactions are represented by solid lines. <b>B.</b> The modular view of the model. Activation of caspase-8 and p43-FLIP (product of cFLIPL cleavage) occurs at the DISC complex and triggers simultaneous processes of cell death and survival.</div></p></div><div class="aux"><div class="resc">Elena Kutumova, et al. BMC Syst Biol. 2013;7:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/23409788" ref="ordinalpos=3455&amp;ncbi_uid=4825825&amp;link_uid=23409788">Citation</a><a class="dblinks" href="/pmc/articles/PMC3626841/" ref="ordinalpos=3455&amp;ncbi_uid=4825825&amp;link_uid=PMC3626841">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3456.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4122365/figure/pone-0103292-g002/" ref="ordinalpos=3456&amp;ncbi_uid=6488460&amp;link_uid=PMC4122365" href="/pmc/articles/PMC4122365/"><img src="/pmc/articles/instance/4122365/bin/pone.0103292.g002.gif" src-large="/pmc/articles/instance/4122365/bin/pone.0103292.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4122365/" ref="ordinalpos=3456&amp;ncbi_uid=6488460&amp;link_uid=PMC4122365" image-link="/pmc/articles/PMC4122365/figure/pone-0103292-g002/" class="imagepopup">Figure 2. Functional analysis of post-eclosion HR differential expression..  From: Wnt <span class="highlight" style="background-color:">Pathway</span> Activation Increases Hypoxia Tolerance during Development. </a></p><div class="supp"><p class="details"><div>(A) The top-scoring KEGG (square) and GO-BP (circle) annotations for up- and down-regulated genes are shown. Down-regulated genes are colored green and are highly enriched for DNA replication, cell cycle and DNA repair. Up-regulated genes are colored red and relate primarily to development and metabolism. In both cases, more intensive color indicates greater fold-enrichment [ranges: (−1.36)–(−4.61); 1.57–4.84] and size reflects the number of genes identified (4–195). (B) DAVID [11], [12]-identified KEGG [14], [15] and Panther [16] <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> enrichment of genes differentially expressed in post-eclosion AF. The number of differentially expressed <span class="highlight" style="background-color:">pathway</span> genes is plotted and the total number of <span class="highlight" style="background-color:">pathway</span> genes is indicated below each <span class="highlight" style="background-color:">pathway</span> name (HH = Hedgehog, Jk-ST = Jak-STAT, PhosIn = Phosphatidylinositol, TGFB = TGF-beta, Cadh = Cadherin, Ins/IGF = Insulin/IGF).</div></p></div><div class="aux"><div class="resc">Merril Gersten, et al. PLoS One. 2014;9(8):e103292.</div><p class="links"><a class="dblinks" href="/pubmed/25093834" ref="ordinalpos=3456&amp;ncbi_uid=6488460&amp;link_uid=25093834">Citation</a><a class="dblinks" href="/pmc/articles/PMC4122365/" ref="ordinalpos=3456&amp;ncbi_uid=6488460&amp;link_uid=PMC4122365">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3457.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC151924/figure/F7/" ref="ordinalpos=3457&amp;ncbi_uid=259893&amp;link_uid=PMC151924" href="/pmc/articles/PMC151924/"><img src="/pmc/articles/instance/151924/bin/JCI0316491.f7.gif" src-large="/pmc/articles/instance/151924/bin/JCI0316491.f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC151924/" ref="ordinalpos=3457&amp;ncbi_uid=259893&amp;link_uid=PMC151924" image-link="/pmc/articles/PMC151924/figure/F7/" class="imagepopup">Figure 7.  From: The suppressor of cytokine <span class="highlight" style="background-color:">signaling</span>-1 (SOCS1) is a novel therapeutic target for enterovirus-induced cardiac injury. </a></p><div class="supp"><p class="details"><div>JAK-STAT antiviral defense in the cardiac myocyte and SOCS inhibition to limit early virus-induced cardiac injury. (<b>a</b>) Virus infection of the heart stimulates cytokine-receptor <span class="highlight" style="background-color:">signaling</span> through the JAK-STAT <span class="highlight" style="background-color:">pathway</span>. Activation of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> induces antiviral target gene transcription stimulating the innate viral defense in the cardiac myocyte. (<b>b</b>) While activation of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> has an important role in antiviral defense, phosphorylated STAT also induces SOCS expression that attenuates the innate antiviral defense by inhibiting JAK <span class="highlight" style="background-color:">signaling</span>. As in SOCS1-transgenic mice, increased expression of SOCS in cardiac myocytes results in robust virus replication and cardiac injury. (<b>c</b>) The strategies aimed at inhibition of SOCS potentiate the innate antiviral actions of cytokines that utilize the JAK-STAT <span class="highlight" style="background-color:">pathway</span>, resulting in the prevention of virus-mediated myocardial injury. P, phosphorylated.</div></p></div><div class="aux"><div class="resc">Hideo Yasukawa, et al. J Clin Invest. 2003 February 15;111(4):469-478.</div><p class="links"><a class="dblinks" href="/pubmed/12588885" ref="ordinalpos=3457&amp;ncbi_uid=259893&amp;link_uid=12588885">Citation</a><a class="dblinks" href="/pmc/articles/PMC151924/" ref="ordinalpos=3457&amp;ncbi_uid=259893&amp;link_uid=PMC151924">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3458.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2944891/figure/pone-0012968-g004/" ref="ordinalpos=3458&amp;ncbi_uid=2486304&amp;link_uid=PMC2944891" href="/pmc/articles/PMC2944891/"><img src="/pmc/articles/instance/2944891/bin/pone.0012968.g004.gif" src-large="/pmc/articles/instance/2944891/bin/pone.0012968.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2944891/" ref="ordinalpos=3458&amp;ncbi_uid=2486304&amp;link_uid=PMC2944891" image-link="/pmc/articles/PMC2944891/figure/pone-0012968-g004/" class="imagepopup">Figure 4. bFGF <span class="highlight" style="background-color:">signaling</span> in chicken PGCs..  From: Basic Fibroblast Growth Factor Activates MEK/ERK Cell <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> and Stimulates the Proliferation of Chicken Primordial Germ Cells. </a></p><div class="supp"><p class="details"><div>(A) Activation of MAPK and PI3K/AKT <span class="highlight" style="background-color:">signaling</span> pathways by bFGF (10 ng·ml<sup>−1</sup>). NT: no treatment (B) Dose-dependent activation of MEK/ERK <span class="highlight" style="background-color:">signaling</span> by bFGF. NT: no treatment (C) Temporal effect of bFGF (10 ng·ml<sup>−1</sup>) on activation of MEK/ERK <span class="highlight" style="background-color:">signaling</span>. NT: no treatment (D) Inhibition of MEK/ERK <span class="highlight" style="background-color:">signaling</span> by a specific inhibitor of FGFRs (PD173074). NT: no treatment (E) Inhibition of ERK activation by PD0325901 in cultured PGCs (PD173074 was used as a control). (F) Cell recovery 4 days after PD0325901 treatment (mean ± SEM, <i>n</i> = 4). bFGF (10 ng·ml<sup>−1</sup>) was treated in all groups. Superscripts indicate significant differences between treatment groups (<i>p</i>&lt;0.05). Statistical analyses were conducted with ANOVA using SAS software. (G–H) Morphology of PGCs 4 days after treatment with PD0325901 (0.4 µM) (bar  = 100 µm). bFGF (10 ng·ml<sup>−1</sup>) was treated in all groups.</div></p></div><div class="aux"><div class="resc">Jin Won Choi, et al. PLoS One. 2010;5(9):e12968.</div><p class="links"><a class="dblinks" href="/pubmed/20886037" ref="ordinalpos=3458&amp;ncbi_uid=2486304&amp;link_uid=20886037">Citation</a><a class="dblinks" href="/pmc/articles/PMC2944891/" ref="ordinalpos=3458&amp;ncbi_uid=2486304&amp;link_uid=PMC2944891">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3459.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3165197/figure/fig01/" ref="ordinalpos=3459&amp;ncbi_uid=3263902&amp;link_uid=PMC3165197" href="/pmc/articles/PMC3165197/"><img src="/pmc/articles/instance/3180111/bin/ajcr0001-0085-f1.gif" src-large="/pmc/articles/instance/3180111/bin/ajcr0001-0085-f1.jpg" alt="Fiugre 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3165197/" ref="ordinalpos=3459&amp;ncbi_uid=3263902&amp;link_uid=PMC3165197" image-link="/pmc/articles/PMC3165197/figure/fig01/" class="imagepopup">Fiugre 1.  From: Expression profiling and <span class="highlight" style="background-color:">pathway</span> analysis of Kr?ppel-like factor 4 in mouse embryonic fibroblasts. </a></p><div class="supp"><p class="details"><div>Gene set enrichment analysis (GSEA) for the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(A)</b> Shown are enrichment scores with a ranked list metric. The arrow indicates the enrichment score of 0.45. <b>(B)</b> Heat map of the results of microarray analysis of 32 core enrichment genes in the JAK-STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in wild type (<i>Klf4</i><sup>+/+</sup>) and <i>Klf4</i>-null (<i>Klf4</i><sup>-/-</sup>) MEFs in triplicate. Red color represents up-regulated and blue color, down-regulated, genes. Significantly enriched data sets are selected according to GSEA default settings, i.e., <i>P</i> &lt; 0.001 and a false discovery rate (FDR) &lt; 0.</div></p></div><div class="aux"><div class="resc">Engda G Hagos, et al. Am J Cancer Res. 2011;1(1):85-97.</div><p class="links"><a class="dblinks" href="/pubmed/21892412" ref="ordinalpos=3459&amp;ncbi_uid=3263902&amp;link_uid=21892412">Citation</a><a class="dblinks" href="/pmc/articles/PMC3165197/" ref="ordinalpos=3459&amp;ncbi_uid=3263902&amp;link_uid=PMC3165197">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3460.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4372177/figure/Fig5/" ref="ordinalpos=3460&amp;ncbi_uid=7320881&amp;link_uid=PMC4372177" href="/pmc/articles/PMC4372177/"><img src="/pmc/articles/instance/4372177/bin/13148_2015_48_Fig5_HTML.gif" src-large="/pmc/articles/instance/4372177/bin/13148_2015_48_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4372177/" ref="ordinalpos=3460&amp;ncbi_uid=7320881&amp;link_uid=PMC4372177" image-link="/pmc/articles/PMC4372177/figure/Fig5/" class="imagepopup">Figure 5.  From: An epigenetic map of age-associated autosomal loci in northern European families at high risk for the metabolic syndrome. </a></p><div class="supp"><p class="details"><div> <b>Top Kegg pathways enriched with aDMCs identified by the FatiGO analysis. (A)</b> The MODY <span class="highlight" style="background-color:">pathway</span> with identified genes of aDMCs and age-associated DNA methylation sites. Genes found with age-associated sites are shown with a blue background, while genes within an aDMC are shown with a green background. <b>(B)</b> The Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> with identified aDMCs and age-associated DNA methylation sites. Genes found with age-associated sites are shown with a blue background, while genes within an aDMC are shown with a green background. <b>(C)</b> Type 1 diabetes <span class="highlight" style="background-color:">pathway</span> with identified aDMCs and age-associated DNA methylation sites.</div></p></div><div class="aux"><div class="resc">Omar Ali, et al. Clin Epigenetics. 2015;7(1):12.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=3460&amp;ncbi_uid=7320881&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4372177/" ref="ordinalpos=3460&amp;ncbi_uid=7320881&amp;link_uid=PMC4372177">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3461.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2842969/figure/fig5/" ref="ordinalpos=3461&amp;ncbi_uid=2271264&amp;link_uid=PMC2842969" href="/pmc/articles/PMC2842969/"><img src="/pmc/articles/instance/2842969/bin/MI2009-698769.005.gif" src-large="/pmc/articles/instance/2842969/bin/MI2009-698769.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2842969/" ref="ordinalpos=3461&amp;ncbi_uid=2271264&amp;link_uid=PMC2842969" image-link="/pmc/articles/PMC2842969/figure/fig5/" class="imagepopup">Figure 5.  From: The Discovery of Novel Experimental Therapies for Inflammatory Arthritis. </a></p><div class="supp"><p class="details"><div>The Proteasome <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. Normally produced (left) or misfolded (right) proteins that are destined for 28S proteasome-mediated degradation are targeted for degradation either by phosphorylation (P) or after tagging with ubiquitin (Ub), via ubiquitin ligase [133]. In this <span class="highlight" style="background-color:">pathway</span>, a polyubiquitin protein chain becomes bound to the 28S proteasome providing a mechanism required for protein degradation. Nonphosphorylated or non-Ub-tagged proteins (i.e., “activated proteins”) can bypass the proteasome <span class="highlight" style="background-color:">pathway</span> and be transported to the nucleus.</div></p></div><div class="aux"><div class="resc">Charles J. Malemud. Mediators Inflamm. 2009;2009:698769.</div><p class="links"><a class="dblinks" href="/pubmed/20339519" ref="ordinalpos=3461&amp;ncbi_uid=2271264&amp;link_uid=20339519">Citation</a><a class="dblinks" href="/pmc/articles/PMC2842969/" ref="ordinalpos=3461&amp;ncbi_uid=2271264&amp;link_uid=PMC2842969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3462.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2575825/figure/f6/" ref="ordinalpos=3462&amp;ncbi_uid=1528645&amp;link_uid=PMC2575825" href="/pmc/articles/PMC2575825/"><img src="/pmc/articles/instance/2575825/bin/zh00100857340006.gif" src-large="/pmc/articles/instance/2575825/bin/zh00100857340006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2575825/" ref="ordinalpos=3462&amp;ncbi_uid=1528645&amp;link_uid=PMC2575825" image-link="/pmc/articles/PMC2575825/figure/f6/" class="imagepopup">Fig. 6. From: Radical-free biology of oxidative stress. </a></p><div class="supp"><p class="details"><div>Kinetic limitation in a thiol-disulfide electron transfer <span class="highlight" style="background-color:">pathway</span>. Measurement of the fractional reduction of proteins in the <span class="highlight" style="background-color:">pathway</span> from NADPH through TrxR1, Trx-1, and redox factor-1 (Ref-1) to NF-κB p50 subunit under control conditions showed that the <span class="highlight" style="background-color:">pathway</span> is kinetically limited in both the cytoplasmic and nuclear compartments. In the nuclear compartment, the characteristics suggested that electron transfer was blocked between Trx-1 and Ref-1. Under conditions where NADPH supply was limited by removal of Glc and Gln from the culture media, the cytoplasmic <span class="highlight" style="background-color:">pathway</span> was selectively oxidized while the nuclear <span class="highlight" style="background-color:">pathway</span> showed a loss of the kinetic limitation. The results show that metabolic conditions determine sites of rate limitation, supporting the concept that rate control in redox pathways is important in cell <span class="highlight" style="background-color:">signaling</span> and regulation. Scheme is based upon data of Go et al. (57).</div></p></div><div class="aux"><div class="resc">Dean P. Jones. Am J Physiol Cell Physiol. 2008 October;295(4):C849-C868.</div><p class="links"><a class="dblinks" href="/pubmed/18684987" ref="ordinalpos=3462&amp;ncbi_uid=1528645&amp;link_uid=18684987">Citation</a><a class="dblinks" href="/pmc/articles/PMC2575825/" ref="ordinalpos=3462&amp;ncbi_uid=1528645&amp;link_uid=PMC2575825">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3463.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3873150/figure/F2/" ref="ordinalpos=3463&amp;ncbi_uid=6335915&amp;link_uid=PMC3873150" href="/pmc/articles/PMC3873150/"><img src="/pmc/articles/instance/3873150/bin/nihms-537955-f0002.gif" src-large="/pmc/articles/instance/3873150/bin/nihms-537955-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3873150/" ref="ordinalpos=3463&amp;ncbi_uid=6335915&amp;link_uid=PMC3873150" image-link="/pmc/articles/PMC3873150/figure/F2/" class="imagepopup">Figure 2. Tripartite synapse and phospholipase C/inositol 1,4,5-triphosphate-mediated Ca2+ release <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in astrocytes.  From: In vivo astrocytic Ca2+ <span class="highlight" style="background-color:">signaling</span> in health and brain disorders. </a></p><div class="supp"><p class="details"><div><b>(A)</b> PLC/IP<sub>3</sub>-mediated Ca<sup>2+</sup> release <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in astrocytes. Following upstimulation of GPCRs, IP<sub>3</sub> is liberated following the activation of PLC astrocytes. Astrocytes then release Ca<sup>2+</sup> from internal stores through the opening of IP<sub>3</sub>R ion channels. <b>(B)</b> The astrocytic process together with pre- and post-synaptic regions of neurons constitutes a synapse – that is, a tripartite synapse. The local microdomains of astrocytic processes can be excited by neuronal activity through glutamate or other neurotransmitters released from the synaptic cleft by activating GPCRs. Ca<sup>2+</sup> release from internal stores can mediate glutamate or other chemical transmitter release, a process called ‘gliotransmission’, which signals back to the neuron by activating neuronal receptors through the tripartite synapse.<br />DAG: Diacyl glycerol; ER: Endoplasmic reticulum; GPCR: G-protein coupled receptor; IP<sub>3</sub>: Inositol 1,4,5-trisphosphate; IP<sub>3</sub>R: Inositol 1,4,5-trisphosphate receptor; PIP<sub>2</sub>: Phosphatidylinositol 4,5-bisphosphate; PLC: Phospholipase C.</div></p></div><div class="aux"><div class="resc">Shinghua Ding. Future Neurol. ;8(5):DOI 10.2217/fnl.13.38.</div><p class="links"><a class="dblinks" href="/pubmed/24376372" ref="ordinalpos=3463&amp;ncbi_uid=6335915&amp;link_uid=24376372">Citation</a><a class="dblinks" href="/pmc/articles/PMC3873150/" ref="ordinalpos=3463&amp;ncbi_uid=6335915&amp;link_uid=PMC3873150">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3464.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2570596/figure/F1/" ref="ordinalpos=3464&amp;ncbi_uid=1621020&amp;link_uid=PMC2570596" href="/pmc/articles/PMC2570596/"><img src="/pmc/articles/instance/2570596/bin/neo1011_1204_fig001.gif" src-large="/pmc/articles/instance/2570596/bin/neo1011_1204_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2570596/" ref="ordinalpos=3464&amp;ncbi_uid=1621020&amp;link_uid=PMC2570596" image-link="/pmc/articles/PMC2570596/figure/F1/" class="imagepopup">Figure 1.  From: The Neurofibromatosis 2 Tumor Suppressor Gene Product, Merlin, Regulates Human Meningioma Cell Growth by <span class="highlight" style="background-color:">Signaling</span> through YAP. </a></p><div class="supp"><p class="details"><div>Schematic of the mammalian Hippo <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The Hippo <span class="highlight" style="background-color:">pathway</span> is an evolutionary conserved cellular <span class="highlight" style="background-color:">pathway</span> that coordinately regulates cell proliferation and apoptosis. Merlin has been proposed to interact with unknown membrane proteins and transduce a signal that stimulates the phosphorylation of LATS1/2 by the serine/threonine kinases MST1/2 that interact with hWW45. LATS1/2 inhibits the transcriptional coactivator YAP resulting in suppressed expression of downstream target genes such as cyclins that are involved in cell growth and proliferation.</div></p></div><div class="aux"><div class="resc">Katherine Striedinger, et al. Neoplasia. 2008 November;10(11):1204-1212.</div><p class="links"><a class="dblinks" href="/pubmed/18953429" ref="ordinalpos=3464&amp;ncbi_uid=1621020&amp;link_uid=18953429">Citation</a><a class="dblinks" href="/pmc/articles/PMC2570596/" ref="ordinalpos=3464&amp;ncbi_uid=1621020&amp;link_uid=PMC2570596">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3465.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2598776/figure/F2/" ref="ordinalpos=3465&amp;ncbi_uid=1557200&amp;link_uid=PMC2598776" href="/pmc/articles/PMC2598776/"><img src="/pmc/articles/instance/2598776/bin/nihms64791f2.gif" src-large="/pmc/articles/instance/2598776/bin/nihms64791f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2598776/" ref="ordinalpos=3465&amp;ncbi_uid=1557200&amp;link_uid=PMC2598776" image-link="/pmc/articles/PMC2598776/figure/F2/" class="imagepopup">Figure 2. Activation of the TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in IECs of V-TLR4 mice.  From: TLR <span class="highlight" style="background-color:">signaling</span> in small intestinal epithelium promotes B cell recruitment and IgA production in lamina propria. </a></p><div class="supp"><p class="details"><div><b>A)</b> DNA binding ELISA analysis of p65 in IECs isolated from the small intestine of WT and V-TLR4 mice (n=10/group, *p&lt;0.05). <b>B–D)</b>. Relative mRNA expression of TNFα (B), iNOS (C), and A20 (D) in IECs isolated from small intestine of WT and V-TLR4 mice (n=3/group; * p&lt;0.05, ** p&lt;0.01).</div></p></div><div class="aux"><div class="resc">Limin Shang, et al. Gastroenterology. ;135(2):529-538.</div><p class="links"><a class="dblinks" href="/pubmed/18522803" ref="ordinalpos=3465&amp;ncbi_uid=1557200&amp;link_uid=18522803">Citation</a><a class="dblinks" href="/pmc/articles/PMC2598776/" ref="ordinalpos=3465&amp;ncbi_uid=1557200&amp;link_uid=PMC2598776">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3466.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634124/figure/F3/" ref="ordinalpos=3466&amp;ncbi_uid=1708708&amp;link_uid=PMC2634124" href="/pmc/articles/PMC2634124/"><img src="/pmc/articles/instance/2634124/bin/psb0105_0231_fig003.gif" src-large="/pmc/articles/instance/2634124/bin/psb0105_0231_fig003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634124/" ref="ordinalpos=3466&amp;ncbi_uid=1708708&amp;link_uid=PMC2634124" image-link="/pmc/articles/PMC2634124/figure/F3/" class="imagepopup">Figure 3.  From: How Ethylene Works in the Reproductive Organs of Higher Plants. </a></p><div class="supp"><p class="details"><div>ET <span class="highlight" style="background-color:">signaling</span> and degradation of EIN3 by 26 proteasome. In air, the MAPK-cascade is repressed, the ET downstream components EIN2, EIN5 and EIN6 inhibited and EIN3 ubiquinated and degradated by 26S proteasome. In the presence of ET, the EtRs are inactivated, CTR1 is not active and the repression on the downstream <span class="highlight" style="background-color:">pathway</span> do not exist; thus, the ET signal is transmitted to EIN3, which accumulates in the nucleus and activates gene expression. Ub, ubiquitin.</div></p></div><div class="aux"><div class="resc">Francisco De la Torre, et al. Plant Signal Behav. 2006 Sep-Oct;1(5):231-242.</div><p class="links"><a class="dblinks" href="/pubmed/19516984" ref="ordinalpos=3466&amp;ncbi_uid=1708708&amp;link_uid=19516984">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634124/" ref="ordinalpos=3466&amp;ncbi_uid=1708708&amp;link_uid=PMC2634124">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3467.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3606104/figure/fig3/" ref="ordinalpos=3467&amp;ncbi_uid=5985553&amp;link_uid=PMC3606104" href="/pmc/articles/PMC3606104/"><img src="/pmc/articles/instance/3606104/bin/1297fig3.gif" src-large="/pmc/articles/instance/3606104/bin/1297fig3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3606104/" ref="ordinalpos=3467&amp;ncbi_uid=5985553&amp;link_uid=PMC3606104" image-link="/pmc/articles/PMC3606104/figure/fig3/" class="imagepopup">Figure 3.  From: Genetic Networks Inducing Invasive Growth in Saccharomyces cerevisiae Identified Through Systematic Genome-Wide Overexpression. </a></p><div class="supp"><p class="details"><div>Identification of genetic networks that regulate the induction of filamentous growth. (A) The overexpression screen data set was analyzed for enrichment of pathways annotated in the Kyoto Encyclopedia of Genes and Genomes (KEGG). Enriched pathways are shown (cell cycle sce04111, meiosis sce04113, and MAPK <span class="highlight" style="background-color:">signaling</span> sce04011) along with genes identified in the screen that are annotated as belonging to the respective <span class="highlight" style="background-color:">pathway</span>. (B) Network connectivity map was built for the MAPK <span class="highlight" style="background-color:">signaling</span>, meiosis, and cell cycle networks from interactions annotated in KEGG. In-house programs were used to parse the <span class="highlight" style="background-color:">pathway</span> maps into nodes and edges for visualization using Cytoscape. The numbered circles indicate a subset of the genes identified in the overexpression screen that belong to the respective networks. Blowups of central portions of the cell cycle, meiosis, and MAPK <span class="highlight" style="background-color:">signaling</span> networks are provided, with key network components indicated in larger circles. Components of a given <span class="highlight" style="background-color:">pathway</span> in the map are indicated with a dashed line to highlight the overlap between networks. Two key connections between the MAPK <span class="highlight" style="background-color:">signaling</span> modules and the cell cycle/meiosis networks involving the Kss1p and Hog1p <span class="highlight" style="background-color:">signaling</span> cascades are shown in blue within the inset box.</div></p></div><div class="aux"><div class="resc">Christian A. Shively, et al. Genetics. 2013 April;193(4):1297-1310.</div><p class="links"><a class="dblinks" href="/pubmed/23410832" ref="ordinalpos=3467&amp;ncbi_uid=5985553&amp;link_uid=23410832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3606104/" ref="ordinalpos=3467&amp;ncbi_uid=5985553&amp;link_uid=PMC3606104">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3468.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3418731/figure/F3/" ref="ordinalpos=3468&amp;ncbi_uid=4661844&amp;link_uid=PMC3418731" href="/pmc/articles/PMC3418731/"><img src="/pmc/articles/instance/3418731/bin/zii9990997740003.gif" src-large="/pmc/articles/instance/3418731/bin/zii9990997740003.jpg" alt="Fig 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3418731/" ref="ordinalpos=3468&amp;ncbi_uid=4661844&amp;link_uid=PMC3418731" image-link="/pmc/articles/PMC3418731/figure/F3/" class="imagepopup">Fig 3.  From: Mycobacterium avium subsp. paratuberculosis Inhibits Gamma Interferon-Induced <span class="highlight" style="background-color:">Signaling</span> in Bovine Monocytes: Insights into the Cellular Mechanisms of Johne's Disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> within the JAK-STAT <span class="highlight" style="background-color:">pathway</span> in bovine monocytes and M. avium subsp. paratuberculosis-infected bovine monocytes in response to IFN stimulation. Protein members of the JAK-STAT <span class="highlight" style="background-color:">pathway</span> are color coded with respect to fold change in differential phosphorylation. Red indicates increased phosphorylation, green indicates decreased phosphorylation, and white indicates no change. (A) Differential phosphorylation of JAK-STAT intermediates following IFN-γ in bovine monocytes. (B) Relative degrees of phosphorylation of M. avium subsp. paratuberculosis-infected versus uninfected bovine monocytes following IFN-γ stimulation. Diagrams were produced using the cytoscape visualization option of InnateDB.</div></p></div><div class="aux"><div class="resc">Ryan J. Arsenault, et al. Infect Immun. 2012 September;80(9):3039-3048.</div><p class="links"><a class="dblinks" href="/pubmed/22689821" ref="ordinalpos=3468&amp;ncbi_uid=4661844&amp;link_uid=22689821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3418731/" ref="ordinalpos=3468&amp;ncbi_uid=4661844&amp;link_uid=PMC3418731">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3469.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2761108/figure/fig8/" ref="ordinalpos=3469&amp;ncbi_uid=2540408&amp;link_uid=PMC2761108" href="/pmc/articles/PMC2761108/"><img src="/pmc/articles/instance/2761108/bin/DEV038133F8.gif" src-large="/pmc/articles/instance/2761108/bin/DEV038133F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2761108/" ref="ordinalpos=3469&amp;ncbi_uid=2540408&amp;link_uid=PMC2761108" image-link="/pmc/articles/PMC2761108/figure/fig8/" class="imagepopup">Fig. 8. From: BMP <span class="highlight" style="background-color:">signaling</span> regulates sympathetic nervous system development through  Smad4-dependent and -independent pathways. </a></p><div class="supp"><p class="details"><div><b>The roles of BMP <span class="highlight" style="background-color:">signaling</span> pathways during SNS development.</b> Both  Smad4-dependent and -independent pathways are involved in SNS development. The  Smad4-independent <span class="highlight" style="background-color:">pathway</span> activates the paired homeobox factor Phox2b, which  regulates the survival of precursors and the expressions of bHLH factor Hand2  and zinc finger factor Gata3. The Smad4-dependent <span class="highlight" style="background-color:">pathway</span> is not required for  <i>Ascl1</i> gene activation but is required for continued expression. Ascl1  regulates proliferation of neuroblasts but not their noradrenergic  differentiation. Both pathways converge to regulate noradrenergic  differentiation of sympathetic neurons.</div></p></div><div class="aux"><div class="resc">Yuka Morikawa, et al. Development. 2009 November 1;136(21):3575-3584.</div><p class="links"><a class="dblinks" href="/pubmed/19793887" ref="ordinalpos=3469&amp;ncbi_uid=2540408&amp;link_uid=19793887">Citation</a><a class="dblinks" href="/pmc/articles/PMC2761108/" ref="ordinalpos=3469&amp;ncbi_uid=2540408&amp;link_uid=PMC2761108">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3470.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3576376/figure/pone-0056714-g004/" ref="ordinalpos=3470&amp;ncbi_uid=4625157&amp;link_uid=PMC3576376" href="/pmc/articles/PMC3576376/"><img src="/pmc/articles/instance/3576376/bin/pone.0056714.g004.gif" src-large="/pmc/articles/instance/3576376/bin/pone.0056714.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3576376/" ref="ordinalpos=3470&amp;ncbi_uid=4625157&amp;link_uid=PMC3576376" image-link="/pmc/articles/PMC3576376/figure/pone-0056714-g004/" class="imagepopup">Figure 4. PCA <span class="highlight" style="background-color:">pathway</span> analysis of FLT3 ITD AML samples and healthy BMMb compared to FLT3 WT AML..  From: Functional <span class="highlight" style="background-color:">Pathway</span> Analysis Using SCNP of FLT3 Receptor <span class="highlight" style="background-color:">Pathway</span> Deregulation in AML Provides Prognostic Information Independent from Mutational Status. </a></p><div class="supp"><p class="details"><div>PCA analysis of FLT3L-induced <span class="highlight" style="background-color:">signaling</span> (PC 1) and AraC/Daunorubicin-induced apoptosis measured by cPARP (PC 2) in healthy BMMb (blue dots), <i>FLT3</i> ITD (red dots) and <i>FLT3</i> WT (green dots). Donors with low <i>FLT3</i> ITD mutational load (&lt;40%) are indicated by arrows.</div></p></div><div class="aux"><div class="resc">Alessandra Cesano, et al. PLoS One. 2013;8(2):e56714.</div><p class="links"><a class="dblinks" href="/pubmed/23431389" ref="ordinalpos=3470&amp;ncbi_uid=4625157&amp;link_uid=23431389">Citation</a><a class="dblinks" href="/pmc/articles/PMC3576376/" ref="ordinalpos=3470&amp;ncbi_uid=4625157&amp;link_uid=PMC3576376">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3471.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3145766/figure/pone-0022856-g007/" ref="ordinalpos=3471&amp;ncbi_uid=3152859&amp;link_uid=PMC3145766" href="/pmc/articles/PMC3145766/"><img src="/pmc/articles/instance/3145766/bin/pone.0022856.g007.gif" src-large="/pmc/articles/instance/3145766/bin/pone.0022856.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3145766/" ref="ordinalpos=3471&amp;ncbi_uid=3152859&amp;link_uid=PMC3145766" image-link="/pmc/articles/PMC3145766/figure/pone-0022856-g007/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Pathway</span> analysis of enriched transcripts in hESC (light blue) and differentiated cells (dark blue)..  From: Comprehensive Gene Expression Analysis of Human Embryonic Stem Cells during Differentiation into Neural Cells. </a></p><div class="supp"><p class="details"><div>Enriched transcripts searched against KEGG <span class="highlight" style="background-color:">pathway</span> database for finding the members involved in the <span class="highlight" style="background-color:">signaling</span> pathways, mostly transcribed members in each stage (up to 60%) in undifferentiated or neural differentiated cells compared and depicted in the figure. Common active pathways in both stages have not been shown.</div></p></div><div class="aux"><div class="resc">Ali Fathi, et al. PLoS One. 2011;6(7):e22856.</div><p class="links"><a class="dblinks" href="/pubmed/21829537" ref="ordinalpos=3471&amp;ncbi_uid=3152859&amp;link_uid=21829537">Citation</a><a class="dblinks" href="/pmc/articles/PMC3145766/" ref="ordinalpos=3471&amp;ncbi_uid=3152859&amp;link_uid=PMC3145766">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3472.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3621013/figure/F5/" ref="ordinalpos=3472&amp;ncbi_uid=6093149&amp;link_uid=PMC3621013" href="/pmc/articles/PMC3621013/"><img src="/pmc/articles/instance/3621013/bin/nihms437757f5.gif" src-large="/pmc/articles/instance/3621013/bin/nihms437757f5.jpg" alt="Fig. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3621013/" ref="ordinalpos=3472&amp;ncbi_uid=6093149&amp;link_uid=PMC3621013" image-link="/pmc/articles/PMC3621013/figure/F5/" class="imagepopup">Fig. 5. Networks of <span class="highlight" style="background-color:">signaling</span> pathways that regulate AS in T cells.  From: Control of Alternative Splicing in Immune Responses: Many Regulators, Many Predictions, Much Still to Learn. </a></p><div class="supp"><p class="details"><div>Summary of several known kinases (blue/purple diamonds) and RNA binding factors (orange/red ovals) that have been shown to be regulated in response to antigen (Ag) stimulation or Fas Ligand (FasL) <span class="highlight" style="background-color:">signaling</span> in T cells, and the genes (text) that they regulate. Solid arrows and lines indicate direct activities, dotted arrows/lines indicate known relationships that are likely indirect. ‘other’ indicates the fact that there are likely many other <span class="highlight" style="background-color:">signaling</span> pathways that are also involved in mediating antigen-induced changes in AS that are yet to be discovered. Question marks are meant to emphasize that although each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> so far has only been linked to regulation of 1–2 genes, each branch likely regulates a program of co-regulated genes. Cross-coordination between branches is also anticipated, as observed for CD45. See text for detailed description of each <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Nicole M. Martinez, et al. Immunol Rev. 2013 May;253(1):216-236.</div><p class="links"><a class="dblinks" href="/pubmed/23550649" ref="ordinalpos=3472&amp;ncbi_uid=6093149&amp;link_uid=23550649">Citation</a><a class="dblinks" href="/pmc/articles/PMC3621013/" ref="ordinalpos=3472&amp;ncbi_uid=6093149&amp;link_uid=PMC3621013">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3473.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2909662/figure/F1/" ref="ordinalpos=3473&amp;ncbi_uid=3055700&amp;link_uid=PMC2909662" href="/pmc/articles/PMC2909662/"><img src="/pmc/articles/instance/2909662/bin/nihms219568f1.gif" src-large="/pmc/articles/instance/2909662/bin/nihms219568f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2909662/" ref="ordinalpos=3473&amp;ncbi_uid=3055700&amp;link_uid=PMC2909662" image-link="/pmc/articles/PMC2909662/figure/F1/" class="imagepopup">Figure 1.  From: Toll-like Receptor 4 in CNS Pathologies. </a></p><div class="supp"><p class="details"><div>The <span class="highlight" style="background-color:">signaling</span> pathways of TLR4. MyD88-dependant <span class="highlight" style="background-color:">signaling</span> is common to most TLRs including TLRs 1,2,4 (shown) and 5,6,7,8,9,11 (not shown). TRIF-dependant <span class="highlight" style="background-color:">signaling</span> results from TLR4 and TLR3 activation only. Note that TLR3 resides in endosomal vesicles, shown here responding to engulfed foreign RNA. The pentagon shapes denotes proteins that interact through a TIR domain. The TRAM adaptor (lime green) is exclusive to TLR4 and coordinates the TRIF response through TLR4’s TIR domain. TRIF-dependant <span class="highlight" style="background-color:">signaling</span> primarily results in IFN-β production (red adaptors), but the TRIF <span class="highlight" style="background-color:">pathway</span> also induces “late stage” NF-κB activation through RIP 1 (white) and TRAF 6 (seafoam green). The MyD88-dependant cascade initiates “early stage” NF-κB activation through the IKKs (IKKs α,β,λ) and/or the MAPK <span class="highlight" style="background-color:">pathway</span>, leading to proinflammatory cytokine expression and subsequent amplification through additional immune pathways.</div></p></div><div class="aux"><div class="resc">Madison M. Buchanan, et al. J Neurochem. ;114(1):13-27.</div><p class="links"><a class="dblinks" href="/pubmed/20402965" ref="ordinalpos=3473&amp;ncbi_uid=3055700&amp;link_uid=20402965">Citation</a><a class="dblinks" href="/pmc/articles/PMC2909662/" ref="ordinalpos=3473&amp;ncbi_uid=3055700&amp;link_uid=PMC2909662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3474.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3808354/figure/pone-0078284-g002/" ref="ordinalpos=3474&amp;ncbi_uid=5477770&amp;link_uid=PMC3808354" href="/pmc/articles/PMC3808354/"><img src="/pmc/articles/instance/3808354/bin/pone.0078284.g002.gif" src-large="/pmc/articles/instance/3808354/bin/pone.0078284.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3808354/" ref="ordinalpos=3474&amp;ncbi_uid=5477770&amp;link_uid=PMC3808354" image-link="/pmc/articles/PMC3808354/figure/pone-0078284-g002/" class="imagepopup">Figure 2. Uric acid activates the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in pancreatic β-cells..  From: Hyperuricemia Causes Pancreatic ?-Cell Death and Dysfunction through NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Treatment of Min6 cells with 5 mg/dL uric acid for 24 h upregulated NF-κB transcriptional activity, as detected by a luciferase reporter assay. The uric acid-induced increase in NF-κB transcriptional activity was reversed by benzbromarone. (<b>B</b>) IκBα phosphorylation, detected by Western blotting, increased in a time-dependent manner in Min6 cells stimulated with 5 mg/dL uric acid at 60 mins. (<b>C</b>) Benzbromarone (50 µmol/L) reversed the uric acid-induced increase in IκB phosphorylation. (<b>D</b>) Total p65 protein levels in Min6 cells, detected by Western blotting, were unaffected by treatment with different concentrations of uric acid. **<i>P</i>&lt;0.01.</div></p></div><div class="aux"><div class="resc">Lu Jia, et al. PLoS One. 2013;8(10):e78284.</div><p class="links"><a class="dblinks" href="/pubmed/24205181" ref="ordinalpos=3474&amp;ncbi_uid=5477770&amp;link_uid=24205181">Citation</a><a class="dblinks" href="/pmc/articles/PMC3808354/" ref="ordinalpos=3474&amp;ncbi_uid=5477770&amp;link_uid=PMC3808354">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3475.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3109272/figure/F3/" ref="ordinalpos=3475&amp;ncbi_uid=3371214&amp;link_uid=PMC3109272" href="/pmc/articles/PMC3109272/"><img src="/pmc/articles/instance/3109272/bin/nihms293133f3.gif" src-large="/pmc/articles/instance/3109272/bin/nihms293133f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3109272/" ref="ordinalpos=3475&amp;ncbi_uid=3371214&amp;link_uid=PMC3109272" image-link="/pmc/articles/PMC3109272/figure/F3/" class="imagepopup">Figure 3. P. syringae infection inhibits ABA <span class="highlight" style="background-color:">signaling</span> through the EDS1/PAD4 <span class="highlight" style="background-color:">pathway</span>.  From: Chemical genetics reveals negative regulation of abscisic acid <span class="highlight" style="background-color:">signaling</span> by a plant immune response <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Infections by <i>Pseudomonas syringae</i> pv <i>tomato (Pst)</i> DC3000, <i>Pst(avrRps4)</i> and <i>Pst</i>(COR-) inhibit ABA-induced <i>RD29B</i> reporter gene expression. <b>(B)</b> ABA-induced stomatal closing is inhibited by <i>Pst</i>DC3000 and <i>Pst(avrRps4)</i> infection in an <i>EDS1/PAD4/SGT1b</i>-dependent manner but independently of <i>NPR1 and EDS16</i>. Infections by <i>Pst</i>(COR-) also inhibit ABA-induced stomatal closing. Symbols * and • represent p&lt; 0.025 and p&gt; 0.2, respectively (n=3 experiments, 30 stomata per experiment and condition, 2-tail T-test). Error bars mean ± s.e.m. (n=3). ABA was applied at 10 µM (A–B).</div></p></div><div class="aux"><div class="resc">Tae-Houn Kim, et al. Curr Biol. ;21(11):990-997.</div><p class="links"><a class="dblinks" href="/pubmed/21620700" ref="ordinalpos=3475&amp;ncbi_uid=3371214&amp;link_uid=21620700">Citation</a><a class="dblinks" href="/pmc/articles/PMC3109272/" ref="ordinalpos=3475&amp;ncbi_uid=3371214&amp;link_uid=PMC3109272">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3476.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2997918/figure/F4/" ref="ordinalpos=3476&amp;ncbi_uid=3591522&amp;link_uid=PMC2997918" href="/pmc/articles/PMC2997918/"><img src="/pmc/articles/instance/2997918/bin/nihms240641f4.gif" src-large="/pmc/articles/instance/2997918/bin/nihms240641f4.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2997918/" ref="ordinalpos=3476&amp;ncbi_uid=3591522&amp;link_uid=PMC2997918" image-link="/pmc/articles/PMC2997918/figure/F4/" class="imagepopup">Fig 4. sAPP stimulates the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during gliogenesis.  From: Involvement of notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in amyloid precursor protein induced glial differentiation. </a></p><div class="supp"><p class="details"><div>(A) Treatment of sAPP (100ng/ml) promoted GFAP expression as well as NICD generation. (B) and (C) sAPP induced generation of NICD in a time- and dose- dependent manner as indicated above. Generation of NICD was measured by Western blot analysis using anti-active Notch Ab. (D) sAPP induced NICD generation was suppressed by neutralizing antibody (22C11). One-factor ANOVA followed by post hoc analysis (Student-Newman-Keuls) was used to analyze the differences between experimental groups and control groups (*: p&lt;0.05, **: p&lt;0.01).</div></p></div><div class="aux"><div class="resc">Young-Don Kwak, et al. Eur J Pharmacol. ;650(1):18-27.</div><p class="links"><a class="dblinks" href="/pubmed/20883690" ref="ordinalpos=3476&amp;ncbi_uid=3591522&amp;link_uid=20883690">Citation</a><a class="dblinks" href="/pmc/articles/PMC2997918/" ref="ordinalpos=3476&amp;ncbi_uid=3591522&amp;link_uid=PMC2997918">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3477.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2928642/figure/F4/" ref="ordinalpos=3477&amp;ncbi_uid=3130119&amp;link_uid=PMC2928642" href="/pmc/articles/PMC2928642/"><img src="/pmc/articles/instance/2928642/bin/zh00071063140004.gif" src-large="/pmc/articles/instance/2928642/bin/zh00071063140004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2928642/" ref="ordinalpos=3477&amp;ncbi_uid=3130119&amp;link_uid=PMC2928642" image-link="/pmc/articles/PMC2928642/figure/F4/" class="imagepopup">Fig. 4. From: Phosphatidic acid mediates activation of mTORC1 through the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>LPA and PA act through the same <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to activate mTORC1. Cells were incubated in medium containing 0.5% serum without leucine for 2 h before addition of LPA and/or PA. <i>A</i>: ERK1/2 phosphorylation on Thr<sup>202</sup>/Tyr<sup>204</sup> was measured by Western blot analysis using an antibody specific for the phosphorylated form of the protein. <i>B</i>: S6K1 phosphorylation was assessed as described in Fig. 3 legend. Values are means ± SE of 3 experiments; in each experiment, 3 samples per condition were independently analyzed. Representative blots are shown. *<i>P</i> &lt; 0.0005 vs. cells incubated in medium containing 0.5% serum without leucine. †<i>P</i> &lt; 0.02; #<i>P</i> &lt; 0.04 vs. PA alone.</div></p></div><div class="aux"><div class="resc">Jeremiah N. Winter, et al. Am J Physiol Cell Physiol. 2010 August;299(2):C335-C344.</div><p class="links"><a class="dblinks" href="/pubmed/20427710" ref="ordinalpos=3477&amp;ncbi_uid=3130119&amp;link_uid=20427710">Citation</a><a class="dblinks" href="/pmc/articles/PMC2928642/" ref="ordinalpos=3477&amp;ncbi_uid=3130119&amp;link_uid=PMC2928642">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3478.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2857923/figure/F5/" ref="ordinalpos=3478&amp;ncbi_uid=2858888&amp;link_uid=PMC2857923" href="/pmc/articles/PMC2857923/"><img src="/pmc/articles/instance/2857923/bin/nihms169178f5.gif" src-large="/pmc/articles/instance/2857923/bin/nihms169178f5.jpg" alt="Fig 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2857923/" ref="ordinalpos=3478&amp;ncbi_uid=2858888&amp;link_uid=PMC2857923" image-link="/pmc/articles/PMC2857923/figure/F5/" class="imagepopup">Fig 5.  From: The Effect of Membrane Domains on the G protein - Phospholipase C? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The PLCβ – G protein <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing how the binding of an extracellular to a transmembrane GPCR will activate a G protein heterotrimer whose G α or G βγ subunits can bind to and activate PLC β. Activated PLC β will hydrolyze PI(4,5)P<sub>2</sub> releasing Ins(1,4,5)P<sub>3</sub> into the cytosol where it diffuses to the endoplasmic reticulum (ER) causing the release of calcium ions into the cytosol. The lipid portion from PIP(4,5)P<sub>2</sub> hydrolysis, diacylglyercol, activates protein kinase C (PKC).</div></p></div><div class="aux"><div class="resc">Urszula Golebiewska, et al. Crit Rev Biochem Mol Biol. ;45(2):97-105.</div><p class="links"><a class="dblinks" href="/pubmed/20128735" ref="ordinalpos=3478&amp;ncbi_uid=2858888&amp;link_uid=20128735">Citation</a><a class="dblinks" href="/pmc/articles/PMC2857923/" ref="ordinalpos=3478&amp;ncbi_uid=2858888&amp;link_uid=PMC2857923">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3479.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4350144/figure/f6/" ref="ordinalpos=3479&amp;ncbi_uid=7288988&amp;link_uid=PMC4350144" href="/pmc/articles/PMC4350144/"><img src="/pmc/articles/instance/4350144/bin/fig-6.gif" src-large="/pmc/articles/instance/4350144/bin/fig-6.jpg" alt="&lt;b&gt;FIG. 6.&lt;/b&gt;" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4350144/" ref="ordinalpos=3479&amp;ncbi_uid=7288988&amp;link_uid=PMC4350144" image-link="/pmc/articles/PMC4350144/figure/f6/" class="imagepopup">FIG. 6..  From: Retinoic Acid Modulates Interferon-γ Production by Hepatic Natural Killer T Cells via Phosphatase 2A and the Extracellular Signal-Regulated Kinase <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Model of modulating effect of RA on IFN-γ production by NKT cells via the PP2A-ERK <span class="highlight" style="background-color:">pathway</span>. The activity of MEKs (also known as MAPKKs) in the NKT cells was induced by the <span class="highlight" style="background-color:">signaling</span> cascade triggered via TCR and CD28, which promotes ERK phosphorylation and <i>IFN-γ</i> expression (<i>solid arrow</i>). RA increased the PP2A activity to de-phosphorylate p-ERK and, subsequently, reduced the <i>IFN-γ</i> expression (<i>dot arrow</i>).</div></p></div><div class="aux"><div class="resc">Heng-Kwei Chang, et al. J Interferon Cytokine Res. 2015 March 1;35(3):200-212.</div><p class="links"><a class="dblinks" href="/pubmed/25343668" ref="ordinalpos=3479&amp;ncbi_uid=7288988&amp;link_uid=25343668">Citation</a><a class="dblinks" href="/pmc/articles/PMC4350144/" ref="ordinalpos=3479&amp;ncbi_uid=7288988&amp;link_uid=PMC4350144">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3480.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2766922/figure/F2/" ref="ordinalpos=3480&amp;ncbi_uid=2129945&amp;link_uid=PMC2766922" href="/pmc/articles/PMC2766922/"><img src="/pmc/articles/instance/2766922/bin/nihms-136169-f0002.gif" src-large="/pmc/articles/instance/2766922/bin/nihms-136169-f0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2766922/" ref="ordinalpos=3480&amp;ncbi_uid=2129945&amp;link_uid=PMC2766922" image-link="/pmc/articles/PMC2766922/figure/F2/" class="imagepopup">Fig. 2.  From: Regulating the mucosal immune system: the contrasting roles of LIGHT, HVEM, and their various partners. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways downstream of HVEM, LIGHT, and their binding partners are shown. Hypothetical pathways are shown with a <i>question mark</i>. <span class="highlight" style="background-color:">Signaling</span> downstream of the LTβR activates the alternative NF-κB <span class="highlight" style="background-color:">pathway</span> that acts through the NF-κB inducing kinase (NIK)</div></p></div><div class="aux"><div class="resc">Marcos W. Steinberg, et al. Semin Immunopathol. ;31(2):207-221.</div><p class="links"><a class="dblinks" href="/pubmed/19495760" ref="ordinalpos=3480&amp;ncbi_uid=2129945&amp;link_uid=19495760">Citation</a><a class="dblinks" href="/pmc/articles/PMC2766922/" ref="ordinalpos=3480&amp;ncbi_uid=2129945&amp;link_uid=PMC2766922">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3481.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3296768/figure/pone-0033011-g005/" ref="ordinalpos=3481&amp;ncbi_uid=3855987&amp;link_uid=PMC3296768" href="/pmc/articles/PMC3296768/"><img src="/pmc/articles/instance/3296768/bin/pone.0033011.g005.gif" src-large="/pmc/articles/instance/3296768/bin/pone.0033011.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3296768/" ref="ordinalpos=3481&amp;ncbi_uid=3855987&amp;link_uid=PMC3296768" image-link="/pmc/articles/PMC3296768/figure/pone-0033011-g005/" class="imagepopup">Figure 5. IPA network analysis..  From: Targeting Bone Remodeling by Isoflavone and 3,3?-Diindolylmethane in the Context of Prostate Cancer Bone Metastasis. </a></p><div class="supp"><p class="details"><div>BR-DIM regulated the molecules in RANKL <span class="highlight" style="background-color:">signaling</span> (A), prostate cancer <span class="highlight" style="background-color:">signaling</span> (B), and signal transduction networks (C, D) analyzed by microarray and Ingenuity <span class="highlight" style="background-color:">Pathway</span> analysis in C4-2B cells. Red indicates up-regulated molecules while green specifies down-regulated molecules. The figures were automatically created by Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis software.</div></p></div><div class="aux"><div class="resc">Yiwei Li, et al. PLoS One. 2012;7(3):e33011.</div><p class="links"><a class="dblinks" href="/pubmed/22412975" ref="ordinalpos=3481&amp;ncbi_uid=3855987&amp;link_uid=22412975">Citation</a><a class="dblinks" href="/pmc/articles/PMC3296768/" ref="ordinalpos=3481&amp;ncbi_uid=3855987&amp;link_uid=PMC3296768">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3482.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3816537/figure/F3/" ref="ordinalpos=3482&amp;ncbi_uid=5455736&amp;link_uid=PMC3816537" href="/pmc/articles/PMC3816537/"><img src="/pmc/articles/instance/3816537/bin/nihms520763f3.gif" src-large="/pmc/articles/instance/3816537/bin/nihms520763f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3816537/" ref="ordinalpos=3482&amp;ncbi_uid=5455736&amp;link_uid=PMC3816537" image-link="/pmc/articles/PMC3816537/figure/F3/" class="imagepopup">Figure 3.  From: Integrative Survival-Based Molecular Profiling of Human Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> based analysis of high ranking composite score genes reveals key <span class="highlight" style="background-color:">signaling</span> pathways associated with PDAC clinical progression. The diagram depicts putative interactions of highly ranked poor (red) and good (green) prognosis-associated genes from the composite signature in relation to SRC <span class="highlight" style="background-color:">signaling</span> or the PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span>. Genes not highlighted in green or red are implied.</div></p></div><div class="aux"><div class="resc">Timothy R. Donahue, et al. Clin Cancer Res. 2012 March 1;18(5):10.1158/1078-0432.CCR-11-1539.</div><p class="links"><a class="dblinks" href="/pubmed/22261810" ref="ordinalpos=3482&amp;ncbi_uid=5455736&amp;link_uid=22261810">Citation</a><a class="dblinks" href="/pmc/articles/PMC3816537/" ref="ordinalpos=3482&amp;ncbi_uid=5455736&amp;link_uid=PMC3816537">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3483.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679155/figure/F1/" ref="ordinalpos=3483&amp;ncbi_uid=1837312&amp;link_uid=PMC2679155" href="/pmc/articles/PMC2679155/"><img src="/pmc/articles/instance/2679155/bin/nihms103414f1.gif" src-large="/pmc/articles/instance/2679155/bin/nihms103414f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679155/" ref="ordinalpos=3483&amp;ncbi_uid=1837312&amp;link_uid=PMC2679155" image-link="/pmc/articles/PMC2679155/figure/F1/" class="imagepopup">Figure 1.  From: Integrins mediate adhesion of medulloblastoma cells to tenascin and activate pathways associated with survival and proliferation. </a></p><div class="supp"><p class="details"><div>Integrin-RTK <span class="highlight" style="background-color:">signaling</span>. A coordinate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> from both integrins and receptor tyrosine kinases mediates cell survival and proliferation. Integrin-linked kinase (ILK) binds to the integrin <i>β</i>-subunit and is key to the assembly of a multiprotein complex; this multiprotein complex activates Akt. RTKs are linked via PINCH and Nck2 and initiate the Ras–MAPK <span class="highlight" style="background-color:">pathway</span>. (Modified from Cordes <i>et al</i>38 with permission of the authors.)</div></p></div><div class="aux"><div class="resc">Paul Fiorilli, et al. Lab Invest. ;88(11):1143-1156.</div><p class="links"><a class="dblinks" href="/pubmed/18794852" ref="ordinalpos=3483&amp;ncbi_uid=1837312&amp;link_uid=18794852">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679155/" ref="ordinalpos=3483&amp;ncbi_uid=1837312&amp;link_uid=PMC2679155">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3484.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC84690/figure/F9/" ref="ordinalpos=3484&amp;ncbi_uid=180197&amp;link_uid=PMC84690" href="/pmc/articles/PMC84690/"><img src="/pmc/articles/instance/84690/bin/mb1090489009.gif" src-large="/pmc/articles/instance/84690/bin/mb1090489009.jpg" alt="FIG. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC84690/" ref="ordinalpos=3484&amp;ncbi_uid=180197&amp;link_uid=PMC84690" image-link="/pmc/articles/PMC84690/figure/F9/" class="imagepopup">FIG. 9.  From: Cas Mediates Transcriptional Activation of the Serum Response Element by Src. </a></p><div class="supp"><p class="details"><div>Model for Cas-mediated activation of the SRE. Kinase-active Src directly binds to Cas. This association leads to the coupling of Grb2 with Shc and Shp-2 by yet undetermined means. The induced association of Grb2 with Shc and/or Shp-2 may signal to the Ras/Mek/Erk <span class="highlight" style="background-color:">pathway</span>. Activation of the Erk MAPK <span class="highlight" style="background-color:">pathway</span> can then lead to SRE-dependent transcriptional activation. Other Src-dependent <span class="highlight" style="background-color:">signaling</span> pathways that are not shown may also result in activation of Ras and <span class="highlight" style="background-color:">signaling</span> to SREs.</div></p></div><div class="aux"><div class="resc">Yaron Hakak, et al. Mol Cell Biol. 1999 October;19(10):6953-6962.</div><p class="links"><a class="dblinks" href="/pubmed/10490632" ref="ordinalpos=3484&amp;ncbi_uid=180197&amp;link_uid=10490632">Citation</a><a class="dblinks" href="/pmc/articles/PMC84690/" ref="ordinalpos=3484&amp;ncbi_uid=180197&amp;link_uid=PMC84690">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3485.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2757945/figure/F9/" ref="ordinalpos=3485&amp;ncbi_uid=2033155&amp;link_uid=PMC2757945" href="/pmc/articles/PMC2757945/"><img src="/pmc/articles/instance/2757945/bin/nihms77489f9.gif" src-large="/pmc/articles/instance/2757945/bin/nihms77489f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2757945/" ref="ordinalpos=3485&amp;ncbi_uid=2033155&amp;link_uid=PMC2757945" image-link="/pmc/articles/PMC2757945/figure/F9/" class="imagepopup">Fig. 9.  From: An FGF-WNT gene regulatory network controls lung mesenchyme development. </a></p><div class="supp"><p class="details"><div>Molecular pathways by which WNT-β-Catenin and FGF regulate lung mesenchymal development. (1) At early stages of lung development (E10–E12), FGF9 is produced by lung mesothelium and epithelium and signals to mesenchymal FGFR1c and FGFR2c to maintain mesenchymal proliferation (Colvin et al., 2001; White et al., 2006). In the absence of <i>Fgf9</i>, expression of the mesenchymal WNT-β-Catenin <span class="highlight" style="background-color:">pathway</span> ligand, <i>Wnt2a</i>, and target gene, <i>Lef1</i>, was down-regulated. When FGF9 was overexpressed in airway epithelium, <i>Wnt2a</i>, and the β-Catenin target gene, <i>Lef1</i>, were both up-regulated. This suggests a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in which mesenchymal FGFR activation regulates <i>Wnt2a</i> expression (2) and mesenchymal WNT-β-Catenin <span class="highlight" style="background-color:">signaling</span>. (3) Inactivation of mesenchymal β-Catenin resulted in decreased expression of <i>Lef1</i> mRNA and protein, decreased expression of CyclinD1 and decreased mesenchymal proliferation, suggesting that mesenchymal WNT <span class="highlight" style="background-color:">signaling</span> could mediate, in part, FGF regulated mesenchymal proliferation. (4) Inactivation of mesenchymal β-Catenin also resulted in decreased expression of FGFR1 and FGFR2 in lung mesenchyme, suggesting that feedback through FGFR <span class="highlight" style="background-color:">signaling</span> may contribute to the ability of mesenchymal WNT <span class="highlight" style="background-color:">signaling</span> to regulate proliferation (5) and <i>Wnt2a</i> expression (2). (6) <i>Wnt7b</i> is expressed in airway epithelium and likely signals to mesenchyme to regulate WNT-β-Catenin <span class="highlight" style="background-color:">signaling</span>. Red stippled area represents the <i>Wnt2a</i> expression domain in the sub-mesothelial mesenchymal compartment.</div></p></div><div class="aux"><div class="resc">Yongjun Yin, et al. Dev Biol. ;319(2):426-436.</div><p class="links"><a class="dblinks" href="/pubmed/18533146" ref="ordinalpos=3485&amp;ncbi_uid=2033155&amp;link_uid=18533146">Citation</a><a class="dblinks" href="/pmc/articles/PMC2757945/" ref="ordinalpos=3485&amp;ncbi_uid=2033155&amp;link_uid=PMC2757945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3486.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2843003/figure/F3/" ref="ordinalpos=3486&amp;ncbi_uid=2834695&amp;link_uid=PMC2843003" href="/pmc/articles/PMC2843003/"><img src="/pmc/articles/instance/2843003/bin/nihms-167001-f0003.gif" src-large="/pmc/articles/instance/2843003/bin/nihms-167001-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2843003/" ref="ordinalpos=3486&amp;ncbi_uid=2834695&amp;link_uid=PMC2843003" image-link="/pmc/articles/PMC2843003/figure/F3/" class="imagepopup">Figure 3. Epicardial-Myocardial <span class="highlight" style="background-color:">Signaling</span> Pathways in Coronary Vascular Development.  From: Epicardial-Myocardial <span class="highlight" style="background-color:">Signaling</span> Directing Coronary Vasculogenesis. </a></p><div class="supp"><p class="details"><div>Shown above is a schematic of the cross talk between the epicardium (blue) and myocardium (red) during the early stages of coronary development. An FGF-Shh-VEGF/Ang2 dependent <span class="highlight" style="background-color:">pathway</span> is important for the generation of coronary endothelial progenitors (right), while β-catenin, PDGF, TGFβ, and Thymosin β4 are important for the development of coronary vascular smooth muscle (left).</div></p></div><div class="aux"><div class="resc">Harold E. Olivey, et al. Circ Res. ;106(5):818-832.</div><p class="links"><a class="dblinks" href="/pubmed/20299672" ref="ordinalpos=3486&amp;ncbi_uid=2834695&amp;link_uid=20299672">Citation</a><a class="dblinks" href="/pmc/articles/PMC2843003/" ref="ordinalpos=3486&amp;ncbi_uid=2834695&amp;link_uid=PMC2843003">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3487.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85950/figure/F5/" ref="ordinalpos=3487&amp;ncbi_uid=20386&amp;link_uid=PMC85950" href="/pmc/articles/PMC85950/"><img src="/pmc/articles/instance/85950/bin/mb1402005005.gif" src-large="/pmc/articles/instance/85950/bin/mb1402005005.jpg" alt="FIG. 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85950/" ref="ordinalpos=3487&amp;ncbi_uid=20386&amp;link_uid=PMC85950" image-link="/pmc/articles/PMC85950/figure/F5/" class="imagepopup">FIG. 5.  From: Transcriptional Regulation of the TATA-Binding Protein by Ras Cellular <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>hTBP promoter activity is not regulated by the PI-3 kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Primary rat hepatocytes were transiently transfected with the following DNAs: A, 5.5 μg of p−4500/+66hTBP-luc together with 1.7 μg of RasV12 or RasV12-C40; B, 5.5 μg of c-<i>fos</i> promoter-luc together with 1.7 μg of RasV12 or RasV12-C40. Wortmannin (0.5 μM) was added where indicated to cells 14 h prior to harvesting of the transfected cells.</div></p></div><div class="aux"><div class="resc">Sandra A. S. Johnson, et al. Mol Cell Biol. 2000 July;20(14):5000-5009.</div><p class="links"><a class="dblinks" href="/pubmed/10866657" ref="ordinalpos=3487&amp;ncbi_uid=20386&amp;link_uid=10866657">Citation</a><a class="dblinks" href="/pmc/articles/PMC85950/" ref="ordinalpos=3487&amp;ncbi_uid=20386&amp;link_uid=PMC85950">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3488.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3638350/figure/F5/" ref="ordinalpos=3488&amp;ncbi_uid=4857434&amp;link_uid=PMC3638350" href="/pmc/articles/PMC3638350/"><img src="/pmc/articles/instance/3638350/bin/neo1505_0472_fig005.gif" src-large="/pmc/articles/instance/3638350/bin/neo1505_0472_fig005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3638350/" ref="ordinalpos=3488&amp;ncbi_uid=4857434&amp;link_uid=PMC3638350" image-link="/pmc/articles/PMC3638350/figure/F5/" class="imagepopup">Figure 5.  From: Restoring KLF5 in Esophageal Squamous Cell Cancer Cells Activates the JNK <span class="highlight" style="background-color:">Pathway</span> Leading to Apoptosis and Reduced Cell Survival. </a></p><div class="supp"><p class="details"><div>A model for the effects of KLF5 on cell survival and apoptosis in ESCC. When KLF5 is restored in ESCC cells, KLF5 trans-activates <i>ASK1</i> and <i>MKK4</i> to activate the JNK <span class="highlight" style="background-color:">pathway</span>. Activated JNK <span class="highlight" style="background-color:">signaling</span> then phosphorylates 14-3-3 proteins, leading to BAX release and translocation to the mitochondria, to promote apoptosis. In addition, KLF5 can directly transactivate <i>BAX</i> to increase BAX levels.</div></p></div><div class="aux"><div class="resc">Rohinton S Tarapore, et al. Neoplasia. 2013 May;15(5):472-480.</div><p class="links"><a class="dblinks" href="/pubmed/23633919" ref="ordinalpos=3488&amp;ncbi_uid=4857434&amp;link_uid=23633919">Citation</a><a class="dblinks" href="/pmc/articles/PMC3638350/" ref="ordinalpos=3488&amp;ncbi_uid=4857434&amp;link_uid=PMC3638350">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3489.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3218018/figure/pone-0027710-g006/" ref="ordinalpos=3489&amp;ncbi_uid=3666585&amp;link_uid=PMC3218018" href="/pmc/articles/PMC3218018/"><img src="/pmc/articles/instance/3218018/bin/pone.0027710.g006.gif" src-large="/pmc/articles/instance/3218018/bin/pone.0027710.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3218018/" ref="ordinalpos=3489&amp;ncbi_uid=3666585&amp;link_uid=PMC3218018" image-link="/pmc/articles/PMC3218018/figure/pone-0027710-g006/" class="imagepopup">Figure 6. NDRG2 affects the TGF-β1/Smad <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the ratio of MMP2 to TIMP2 in rats..  From: NDRG2 Ameliorates Hepatic Fibrosis by Inhibiting the TGF-?1/Smad <span class="highlight" style="background-color:">Pathway</span> and Altering the MMP2/TIMP2 Ratio in Rats. </a></p><div class="supp"><p class="details"><div>(A) Real-time PCR was used to detect the expression of Smad2, Smad3 and Smad7 in rats with PBS or adenovirus treatment. (B) Western blotting was used to detect the expression of Smad2, phospho-Smad2, Smad3, phospho-Smad3 and Smad7 in rats with PBS or adenovirus treatment. (C) Real-time PCR was used to detect the expression of MMP2 and TIMP2 in rats following PBS or adenovirus treatment. β-actin was used as a loading control. “normal” means rats without treatment. *P&lt;0.05 compared with the PBS or AdLacZ group.</div></p></div><div class="aux"><div class="resc">Jiandong Yang, et al. PLoS One. 2011;6(11):e27710.</div><p class="links"><a class="dblinks" href="/pubmed/22110735" ref="ordinalpos=3489&amp;ncbi_uid=3666585&amp;link_uid=22110735">Citation</a><a class="dblinks" href="/pmc/articles/PMC3218018/" ref="ordinalpos=3489&amp;ncbi_uid=3666585&amp;link_uid=PMC3218018">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3490.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3757429/figure/f3-cancers-03-02501/" ref="ordinalpos=3490&amp;ncbi_uid=5272910&amp;link_uid=PMC3757429" href="/pmc/articles/PMC3757429/"><img src="/pmc/articles/instance/3757429/bin/cancers-03-02501f3.gif" src-large="/pmc/articles/instance/3757429/bin/cancers-03-02501f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3757429/" ref="ordinalpos=3490&amp;ncbi_uid=5272910&amp;link_uid=PMC3757429" image-link="/pmc/articles/PMC3757429/figure/f3-cancers-03-02501/" class="imagepopup">Figure 3. From: Activation of PDGFr-? <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> after Imatinib and Radioimmunotherapy Treatment in Experimental Pancreatic Cancer. </a></p><div class="supp"><p class="details"><div>Activation of human PDGFr-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in response to imatininb and radiation injury. Expression of human PDGFr-β. Western immunoblotting analyses with mouse monoclonal anti-human PDGFr-β monoclonal antibodies ab10847 to determine expression of human PDGFr-β in response to various treatments. The bands were detected with ImmunoPure goat anti-mouse IgG (H + L), peroxidase conjugated antibodies 31430. The lower panel is the same membrane probed with rabbit anti-β-actin mAb 13E5. The bar graph represents protein band intensity normalized to the protein load as measured by the intensity of β-actin band using ImageJ.</div></p></div><div class="aux"><div class="resc">Michio Abe, et al. Cancers (Basel). 2011 June;3(2):2501-2515.</div><p class="links"><a class="dblinks" href="/pubmed/24212821" ref="ordinalpos=3490&amp;ncbi_uid=5272910&amp;link_uid=24212821">Citation</a><a class="dblinks" href="/pmc/articles/PMC3757429/" ref="ordinalpos=3490&amp;ncbi_uid=5272910&amp;link_uid=PMC3757429">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3491.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3595218/figure/pbio-1001505-g007/" ref="ordinalpos=3491&amp;ncbi_uid=4768597&amp;link_uid=PMC3595218" href="/pmc/articles/PMC3595218/"><img src="/pmc/articles/instance/3595218/bin/pbio.1001505.g007.gif" src-large="/pmc/articles/instance/3595218/bin/pbio.1001505.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3595218/" ref="ordinalpos=3491&amp;ncbi_uid=4768597&amp;link_uid=PMC3595218" image-link="/pmc/articles/PMC3595218/figure/pbio-1001505-g007/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Signaling</span> properties of lipoprotein-associated Shh and Shh-N* in Shh-LIGHT2 cells..  From: Secretion and <span class="highlight" style="background-color:">Signaling</span> Activities of Lipoprotein-Associated Hedgehog and Non-Sterol-Modified Hedgehog in Flies and Mammals. </a></p><div class="supp"><p class="details"><div>(A, B) Concentration-dependent <span class="highlight" style="background-color:">signaling</span> activity of (A) lipoprotein-associated Shh and (B) Shh-N*. Lipoprotein concentration in (A) was kept constant, and only the fraction carrying Shh increased. Shh and Shh-N* levels used for <span class="highlight" style="background-color:">signaling</span> assays were assessed by WB. (C,D) Shh <span class="highlight" style="background-color:">pathway</span> activity in cells stimulated by Shh-N* in the absence or presence of lipoproteins, or cells stimulated with lipoprotein-associated Shh. Lipoproteins, where added, were kept at a constant level. (C) Mammalian lipoproteins, (D) <i>Drosophila</i> Lpp. (E) Synergistic <span class="highlight" style="background-color:">signaling</span> activity of Shh-N* and lipoprotein-associated Shh. Shh-N* and lipoprotein-associated Shh were applied to cells alone or in combination. Predicted additive values represent the combined activity of lipoprotein-associated Shh and Shh-N* in the presence of lipoproteins, minus the basal assay activity measured in unstimulated cells. Note that the same batch of samples was used for assays shown in (A) and (B). For (A–E), error bars indicate ± SD (<i>n</i> = 3; **<i>p</i>&lt;0.005; ***<i>p</i>&lt;0.0005) of one representative experiment. Experiments were repeated at least twice.</div></p></div><div class="aux"><div class="resc">Wilhelm Palm, et al. PLoS Biol. 2013 March;11(3):e1001505.</div><p class="links"><a class="dblinks" href="/pubmed/23554573" ref="ordinalpos=3491&amp;ncbi_uid=4768597&amp;link_uid=23554573">Citation</a><a class="dblinks" href="/pmc/articles/PMC3595218/" ref="ordinalpos=3491&amp;ncbi_uid=4768597&amp;link_uid=PMC3595218">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3492.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3737589/figure/F1/" ref="ordinalpos=3492&amp;ncbi_uid=5176352&amp;link_uid=PMC3737589" href="/pmc/articles/PMC3737589/"><img src="/pmc/articles/instance/3737589/bin/nihms493872f1.gif" src-large="/pmc/articles/instance/3737589/bin/nihms493872f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3737589/" ref="ordinalpos=3492&amp;ncbi_uid=5176352&amp;link_uid=PMC3737589" image-link="/pmc/articles/PMC3737589/figure/F1/" class="imagepopup">Figure 1.  From: Quantitative modeling of bacterial chemotaxis: Signal amplification and accurate adaptation. </a></p><div class="supp"><p class="details"><div>Illustration of the <i>E. coli</i> chemotaxis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Yuhai Tu. Annu Rev Biophys. ;42:337-359.</div><p class="links"><a class="dblinks" href="/pubmed/23451887" ref="ordinalpos=3492&amp;ncbi_uid=5176352&amp;link_uid=23451887">Citation</a><a class="dblinks" href="/pmc/articles/PMC3737589/" ref="ordinalpos=3492&amp;ncbi_uid=5176352&amp;link_uid=PMC3737589">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3493.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3224649/figure/F7/" ref="ordinalpos=3493&amp;ncbi_uid=3678193&amp;link_uid=PMC3224649" href="/pmc/articles/PMC3224649/"><img src="/pmc/articles/instance/3224649/bin/1471-2105-11-43-7.gif" src-large="/pmc/articles/instance/3224649/bin/1471-2105-11-43-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3224649/" ref="ordinalpos=3493&amp;ncbi_uid=3678193&amp;link_uid=PMC3224649" image-link="/pmc/articles/PMC3224649/figure/F7/" class="imagepopup">Figure 7.  From: Discovering local patterns of co - evolution: computational aspects and biological examples. </a></p><div class="supp"><p class="details"><div><b>The different steps in generating the small fungi dN/dS dataset and the mammalian <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> dataset</b>.</div></p></div><div class="aux"><div class="resc">Tamir Tuller, et al. BMC Bioinformatics. 2010;11:43-43.</div><p class="links"><a class="dblinks" href="/pubmed/20096103" ref="ordinalpos=3493&amp;ncbi_uid=3678193&amp;link_uid=20096103">Citation</a><a class="dblinks" href="/pmc/articles/PMC3224649/" ref="ordinalpos=3493&amp;ncbi_uid=3678193&amp;link_uid=PMC3224649">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3494.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2491337/figure/F9/" ref="ordinalpos=3494&amp;ncbi_uid=1508032&amp;link_uid=PMC2491337" href="/pmc/articles/PMC2491337/"><img src="/pmc/articles/instance/2491337/bin/nihms-55900-f0009.gif" src-large="/pmc/articles/instance/2491337/bin/nihms-55900-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2491337/" ref="ordinalpos=3494&amp;ncbi_uid=1508032&amp;link_uid=PMC2491337" image-link="/pmc/articles/PMC2491337/figure/F9/" class="imagepopup">Figure 9.  From: Chemokine Receptor CCR2b Inhibition Provides Renal Protection in Angiotensin II-Salt Hypertension. </a></p><div class="supp"><p class="details"><div>Schematic diagram of the proposed inflammatory cytokines <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in ANG/HS hypertension.</div></p></div><div class="aux"><div class="resc">Ahmed A. Elmarakby, et al. Hypertension. ;50(6):1069-1076.</div><p class="links"><a class="dblinks" href="/pubmed/17938380" ref="ordinalpos=3494&amp;ncbi_uid=1508032&amp;link_uid=17938380">Citation</a><a class="dblinks" href="/pmc/articles/PMC2491337/" ref="ordinalpos=3494&amp;ncbi_uid=1508032&amp;link_uid=PMC2491337">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3495.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1482501/figure/F1/" ref="ordinalpos=3495&amp;ncbi_uid=725133&amp;link_uid=PMC1482501" href="/pmc/articles/PMC1482501/"><img src="/pmc/articles/instance/1482501/bin/zpq0200621890001.gif" src-large="/pmc/articles/instance/1482501/bin/zpq0200621890001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1482501/" ref="ordinalpos=3495&amp;ncbi_uid=725133&amp;link_uid=PMC1482501" image-link="/pmc/articles/PMC1482501/figure/F1/" class="imagepopup">Fig. 1. From: Stochastic models for regulatory networks of the genetic toggle switch. </a></p><div class="supp"><p class="details"><div>The genetic toggle switch interfaced with the SOS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Tianhai Tian, et al. Proc Natl Acad Sci U S A. 2006 May 30;103(22):8372-8377.</div><p class="links"><a class="dblinks" href="/pubmed/16714385" ref="ordinalpos=3495&amp;ncbi_uid=725133&amp;link_uid=16714385">Citation</a><a class="dblinks" href="/pmc/articles/PMC1482501/" ref="ordinalpos=3495&amp;ncbi_uid=725133&amp;link_uid=PMC1482501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3496.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2724233/figure/f1/" ref="ordinalpos=3496&amp;ncbi_uid=1911232&amp;link_uid=PMC2724233" href="/pmc/articles/PMC2724233/"><img src="/pmc/articles/instance/2724233/bin/zh60080968690001.gif" src-large="/pmc/articles/instance/2724233/bin/zh60080968690001.jpg" alt="Fig. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2724233/" ref="ordinalpos=3496&amp;ncbi_uid=1911232&amp;link_uid=PMC2724233" image-link="/pmc/articles/PMC2724233/figure/f1/" class="imagepopup">Fig. 1. From: A map and new directions for the (pro)renin receptor in the brain: focus on "A role of the (pro)renin receptor in neuronal cell differentiation". </a></p><div class="supp"><p class="details"><div>ANG II-dependent and independent pathways for renin <span class="highlight" style="background-color:">signaling</span>. Both renin and prorenin can bind the dimerized (pro)renin receptor [(P)RR], leading to the activation of different <span class="highlight" style="background-color:">signaling</span> pathways. Binding of prorenin to the receptor results in either direct activation of downstream MAP kinases <span class="highlight" style="background-color:">signaling</span>, or in the nonproteolytic activation of prorenin, facilitating the activation of ERK <span class="highlight" style="background-color:">signaling</span> cascades that can also be triggered by renin binding. Activation of these <span class="highlight" style="background-color:">signaling</span> cascades results in actin polymerization and the expression for profibrotic genes, leading to TGF-β, PAI-1, collagen-I, and fibronection synthesis. Binding of renin to the (P)RR also activates an ANG II-dependent <span class="highlight" style="background-color:">pathway</span> leading to the formation of ANG II and the activation of the AT<sub>1</sub> receptor. Activation of both pathways ultimately leads to end-organ damage. Abbreviations: TGF-β, transforming growth factor β; PAI-1, plasminogen-activator inhibitor-1.</div></p></div><div class="aux"><div class="resc">Eric Lazartigues. Am J Physiol Regul Integr Comp Physiol. 2009 August;297(2):R248-R249.</div><p class="links"><a class="dblinks" href="/pubmed/19494175" ref="ordinalpos=3496&amp;ncbi_uid=1911232&amp;link_uid=19494175">Citation</a><a class="dblinks" href="/pmc/articles/PMC2724233/" ref="ordinalpos=3496&amp;ncbi_uid=1911232&amp;link_uid=PMC2724233">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3497.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3660562/figure/pone-0064470-g003/" ref="ordinalpos=3497&amp;ncbi_uid=4921905&amp;link_uid=PMC3660562" href="/pmc/articles/PMC3660562/"><img src="/pmc/articles/instance/3660562/bin/pone.0064470.g003.gif" src-large="/pmc/articles/instance/3660562/bin/pone.0064470.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3660562/" ref="ordinalpos=3497&amp;ncbi_uid=4921905&amp;link_uid=PMC3660562" image-link="/pmc/articles/PMC3660562/figure/pone-0064470-g003/" class="imagepopup">Figure 3. Epistatic analysis of ptc6 and mutations affecting the TOR <span class="highlight" style="background-color:">pathway</span>..  From: Ptc6 Is Required for Proper Rapamycin-Induced Down-Regulation of the Genes Coding for Ribosomal and rRNA Processing Proteins in S. cerevisiae. </a></p><div class="supp"><p class="details"><div>A) A simplified model of <span class="highlight" style="background-color:">signaling</span> through the TOR <span class="highlight" style="background-color:">pathway</span>, focused on the regulation of NCR genes and displaying the proposed role for Ptc1 [19]. B) Growth in the presence of the indicated concentrations of rapamycin of diverse mutants in genes involved in the TOR <span class="highlight" style="background-color:">pathway</span> in combination with deletion of <i>PTC6</i>. Relative growth is represented as a percentage respect the growth of each strain in YPD without rapamycin. Six experiments were performed and the mean ± SEM is represented.</div></p></div><div class="aux"><div class="resc">Asier González, et al. PLoS One. 2013;8(5):e64470.</div><p class="links"><a class="dblinks" href="/pubmed/23704987" ref="ordinalpos=3497&amp;ncbi_uid=4921905&amp;link_uid=23704987">Citation</a><a class="dblinks" href="/pmc/articles/PMC3660562/" ref="ordinalpos=3497&amp;ncbi_uid=4921905&amp;link_uid=PMC3660562">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3498.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3165056/figure/F3/" ref="ordinalpos=3498&amp;ncbi_uid=3196011&amp;link_uid=PMC3165056" href="/pmc/articles/PMC3165056/"><img src="/pmc/articles/instance/3165056/bin/nihms132499f3.gif" src-large="/pmc/articles/instance/3165056/bin/nihms132499f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3165056/" ref="ordinalpos=3498&amp;ncbi_uid=3196011&amp;link_uid=PMC3165056" image-link="/pmc/articles/PMC3165056/figure/F3/" class="imagepopup">Fig. 3. The RIG-I and MDA-5 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Functions of the cytoplasmic RNA sensors RIG-I and MDA-5: Key regulators of innate immunity. </a></p><div class="supp"><p class="details"><div>Short dsRNA’s or 5’-triphosphate ssRNA’s activate RIG-I which associates with mitochondrial IPS-1 to trigger FADD/RIP1 (innateosome) controlled activation of the type I interferon genes. Longer RNA’s generated from picornaviruses trigger MDA5-mediated activation of type I IFN. The RIG-I/IPS-1 <span class="highlight" style="background-color:">pathway</span> also requires the assistance of STING to activate type I IFN, which resides in the endoplasmic reticulum (ER). STING may regulate the translocon composed of the TRAP complex and Sec61, which in turn may regulate exocyst-mediated stimulation of the TBK-1 <span class="highlight" style="background-color:">pathway</span>. STING also appears pivotal in regulating DNA-mediated triggering of type I IFN production.</div></p></div><div class="aux"><div class="resc">Paola M Barral, et al. Pharmacol Ther. ;124(2):219-234.</div><p class="links"><a class="dblinks" href="/pubmed/19615405" ref="ordinalpos=3498&amp;ncbi_uid=3196011&amp;link_uid=19615405">Citation</a><a class="dblinks" href="/pmc/articles/PMC3165056/" ref="ordinalpos=3498&amp;ncbi_uid=3196011&amp;link_uid=PMC3165056">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3499.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1705971/figure/F2/" ref="ordinalpos=3499&amp;ncbi_uid=848115&amp;link_uid=PMC1705971" href="/pmc/articles/PMC1705971/"><img src="/pmc/articles/instance/1705971/bin/nihms-14550-0002.gif" src-large="/pmc/articles/instance/1705971/bin/nihms-14550-0002.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1705971/" ref="ordinalpos=3499&amp;ncbi_uid=848115&amp;link_uid=PMC1705971" image-link="/pmc/articles/PMC1705971/figure/F2/" class="imagepopup">Fig. 2.  From: The Ras/cAMP-dependent Protein Kinase <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Regulates an Early Step of the Autophagy Process in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div><b>Inactivation of the Ras/PKA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was sufficient to induce an autophagy response.</b> <i>A</i>, expression of the dominant negative <i>RAS2<sup>ala22</sup></i> allele resulted in a growth arrest after transfer to the galactose medium. Wild-type cells (TN125) carrying either a control vector, pRS416 (<i>circles</i>), or the <i>GAL1-RAS2<sup>ala22</sup></i> plasmid, pPHY2128 (<i>triangles</i>), were grown to mid-log phase in YM-glucose medium and transferred to a YM medium that contained 5% galactose and 2% raffinose. The culture density was assessed by spectrophoto-metric measurement of the OD<sub>600</sub> at the indicated times after the transfer to galactose-containing growth medium. <i>B</i>, inactivation of the Ras/PKA <span class="highlight" style="background-color:">pathway</span> resulted in an induction of autophagy activity. Autophagy levels were assessed with the alkaline phosphatase-based assay as described under “Experimental Procedures.”</div></p></div><div class="aux"><div class="resc">Yelena V. Budovskaya, et al. J Biol Chem. ;279(20):20663-20671.</div><p class="links"><a class="dblinks" href="/pubmed/15016820" ref="ordinalpos=3499&amp;ncbi_uid=848115&amp;link_uid=15016820">Citation</a><a class="dblinks" href="/pmc/articles/PMC1705971/" ref="ordinalpos=3499&amp;ncbi_uid=848115&amp;link_uid=PMC1705971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>3500.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2694616/figure/F4/" ref="ordinalpos=3500&amp;ncbi_uid=1936756&amp;link_uid=PMC2694616" href="/pmc/articles/PMC2694616/"><img src="/pmc/articles/instance/2694616/bin/nihms114667f4.gif" src-large="/pmc/articles/instance/2694616/bin/nihms114667f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2694616/" ref="ordinalpos=3500&amp;ncbi_uid=1936756&amp;link_uid=PMC2694616" image-link="/pmc/articles/PMC2694616/figure/F4/" class="imagepopup">Figure 4. Modules of the Rb/E2F <span class="highlight" style="background-color:">pathway</span>.  From: A Genomic Strategy to Elucidate Modules of Oncogenic <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>A. The Rb/E2F <span class="highlight" style="background-color:">pathway</span> regulates genes and processes in S phase and mitosis.<br />B. Three signatures derived from the Rb/E2F <span class="highlight" style="background-color:">pathway</span> are significantly associated with specific phases of the cell cycle. In this figure, the signatures are produced from the E2F <span class="highlight" style="background-color:">pathway</span>. The scatter plots show the relationship between the scores of signatures 3, 4, 6 and drugs that target events in S phase and mitosis of the cell cycle. The GI50 and red regression lines are defined as in Figure 2B. Here, signature 3 is significantly associated with three drugs that target aspects of S phase (Supplemental Table S3), and not associated with drugs that target mitosis. Furthermore, high scores for signature 3 is associated with mutations in p16, an inhibitor of S phase activities. Conversely, signatures 4 and 6 are associated with mitosis but not S phase.</div></p></div><div class="aux"><div class="resc">Jeffrey T. Chang, et al. Mol Cell. ;34(1):104-114.</div><p class="links"><a class="dblinks" href="/pubmed/19362539" ref="ordinalpos=3500&amp;ncbi_uid=1936756&amp;link_uid=19362539">Citation</a><a class="dblinks" href="/pmc/articles/PMC2694616/" ref="ordinalpos=3500&amp;ncbi_uid=1936756&amp;link_uid=PMC2694616">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="34" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="35" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="36" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:42:30-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A202D543120810000000000471BF0;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A202D543120810000000000471BF0" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>