
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="49" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A110554314361000000000021EB69" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="48" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="49" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="50" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="49" /></div>
            <div><h2 class="result_count">Results: 4801 to 4900 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4801.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1415320/figure/fig6/" ref="ordinalpos=4801&amp;ncbi_uid=718530&amp;link_uid=PMC1415320" href="/pmc/articles/PMC1415320/"><img src="/pmc/articles/instance/1415320/bin/zmk0040676050006.gif" src-large="/pmc/articles/instance/1415320/bin/zmk0040676050006.jpg" alt="Figure 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1415320/" ref="ordinalpos=4801&amp;ncbi_uid=718530&amp;link_uid=PMC1415320" image-link="/pmc/articles/PMC1415320/figure/fig6/" class="imagepopup">Figure 6. From: Cooperation between Snail and LEF-1 Transcription Factors Is Essential for TGF-?1-induced Epithelial-Mesenchymal Transition. </a></p><div class="supp"><p class="details"><div>A schematic representation of the proposed TGF-β1 <span class="highlight" style="background-color:">signaling</span> mechanism that promotes EMT. Early <span class="highlight" style="background-color:">signaling</span> through Ras-Raf-MEK-ERK-AP-1 causes up-regulation in Snail expression. Snail, a well-known repressor of junction proteins, then inhibits expression of claudin-1, claudin-2, and occludin. Snail also provides partial loss of E-cadherin, thus decreasing the level of substrate for β-catenin. Further stabilization of cytoplasmic β-catenin is achieved through PI3K <span class="highlight" style="background-color:">signaling</span>. Molecules downstream of PI3K such as AKT phosphorylate and deactivate p-GSK-3β, which is responsible for degrading both β-catenin and Snail through the ubiquitin proteosome <span class="highlight" style="background-color:">pathway</span>. Smad <span class="highlight" style="background-color:">signaling</span>, controlled through endocytosis of TGF-β receptor complex, promotes transcription of LEF-1, an alternative substrate for β-catenin. On forming β-catenin/LEF-1 complexes, these molecules will then act to transcribe genes that will induce EMT.</div></p></div><div class="aux"><div class="resc">Damian Medici, et al. Mol Biol Cell. 2006 April;17(4):1871-1879.</div><p class="links"><a class="dblinks" href="/pubmed/16467384" ref="ordinalpos=4801&amp;ncbi_uid=718530&amp;link_uid=16467384">Citation</a><a class="dblinks" href="/pmc/articles/PMC1415320/" ref="ordinalpos=4801&amp;ncbi_uid=718530&amp;link_uid=PMC1415320">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4802.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4122293/figure/F8/" ref="ordinalpos=4802&amp;ncbi_uid=6463136&amp;link_uid=PMC4122293" href="/pmc/articles/PMC4122293/"><img src="/pmc/articles/instance/4122293/bin/nihms93683f8.gif" src-large="/pmc/articles/instance/4122293/bin/nihms93683f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4122293/" ref="ordinalpos=4802&amp;ncbi_uid=6463136&amp;link_uid=PMC4122293" image-link="/pmc/articles/PMC4122293/figure/F8/" class="imagepopup">Figure 8. Schematic diagram of endothelin-1 synthesis signal transduction pathways and autocrine effects.  From: Fibronectin Stimulates Endothelin-1 Synthesis in Rat Hepatic Myofibroblasts via a Src/ERK Regulated <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>On the left, fibronectin promotes ET-1 production by activating Src, which, in turn, activates a Shc/Grb2/Sos complex by direct phosphorylation of Shc; phosphorylated Shc then activates the ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which leads to increased preproendothelin-1 expression. Increased endogenous ET-1 activity contributes to enhanced contraction and smooth muscle α actin expression in activated stellate cells. The proposed <span class="highlight" style="background-color:">pathway</span> to cellular contraction is based on previous literature in smooth muscle cells 27. Abbreviations: PKC = protein kinase C, MLC = myosin light chain, MLCK = myosin light chain kinase.</div></p></div><div class="aux"><div class="resc">Shuxin Zhan, et al. Gastroenterology. ;136(7):2345-55.e1-4.</div><p class="links"><a class="dblinks" href="/pubmed/19505428" ref="ordinalpos=4802&amp;ncbi_uid=6463136&amp;link_uid=19505428">Citation</a><a class="dblinks" href="/pmc/articles/PMC4122293/" ref="ordinalpos=4802&amp;ncbi_uid=6463136&amp;link_uid=PMC4122293">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4803.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2797269/figure/F7/" ref="ordinalpos=4803&amp;ncbi_uid=2572493&amp;link_uid=PMC2797269" href="/pmc/articles/PMC2797269/"><img src="/pmc/articles/instance/2797269/bin/zbc0500995790007.gif" src-large="/pmc/articles/instance/2797269/bin/zbc0500995790007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2797269/" ref="ordinalpos=4803&amp;ncbi_uid=2572493&amp;link_uid=PMC2797269" image-link="/pmc/articles/PMC2797269/figure/F7/" class="imagepopup">FIGURE 7. From: Oxidative Stress Inhibits Insulin-like Growth Factor-I Induction of Chondrocyte Proteoglycan Synthesis through Differential Regulation of Phosphatidylinositol 3-Kinase-Akt and MEK-ERK MAPK <span class="highlight" style="background-color:">Signaling</span> Pathways. </a></p><div class="supp"><p class="details"><div><b>Hypothetical model for the effect of oxidative stress on IGF-I regulation of chondrocyte proteoglycan (aggrecan) and collagen II production.</b> In this model, the balance of Akt and ERK1/2 activation determines the final effect of IGF-I on proteoglycan production and collagen expression. In normal chondrocytes, IGF-I activates the IGF-I receptor (<i>IGF-IR</i>), which induces IRS-1 activation through tyrosine phosphorylation and subsequently activates the PI 3-kinase-Akt <span class="highlight" style="background-color:">pathway</span>, which leads downstream to increased aggrecan synthesis and collagen II expression. Increased activation of MEK-ERK <span class="highlight" style="background-color:">signaling</span>, stimulated by oxidative stress and increased ROS levels present in OA cartilage, inhibits aggrecan and collagen gene transcription. Oxidative stress also activates MEK-ERK <span class="highlight" style="background-color:">signaling</span>, which, through inducing IRS-1 serine phosphorylation, negatively regulates IGF-I activation of PI 3-kinase-Akt <span class="highlight" style="background-color:">signaling</span> and proteoglycan synthesis.</div></p></div><div class="aux"><div class="resc">Weihong Yin, et al. J Biol Chem. 2009 November 13;284(46):31972-31981.</div><p class="links"><a class="dblinks" href="/pubmed/19762915" ref="ordinalpos=4803&amp;ncbi_uid=2572493&amp;link_uid=19762915">Citation</a><a class="dblinks" href="/pmc/articles/PMC2797269/" ref="ordinalpos=4803&amp;ncbi_uid=2572493&amp;link_uid=PMC2797269">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4804.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2778150/figure/sc1/" ref="ordinalpos=4804&amp;ncbi_uid=2514802&amp;link_uid=PMC2778150" href="/pmc/articles/PMC2778150/"><img src="/pmc/articles/instance/2778150/bin/AJRCCM1808731S1.gif" src-large="/pmc/articles/instance/2778150/bin/AJRCCM1808731S1.jpg" alt="Scheme 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2778150/" ref="ordinalpos=4804&amp;ncbi_uid=2514802&amp;link_uid=PMC2778150" image-link="/pmc/articles/PMC2778150/figure/sc1/" class="imagepopup">Scheme 1. From: Simvastatin Inhibits Airway Hyperreactivity. </a></p><div class="supp"><p class="details"><div>The mevalonate (MA) <span class="highlight" style="background-color:">pathway</span>: sterol and nonsterol products. Simvastatin and other statins directly inhibit the rate-limiting enzyme hydroxymethylglutaryl (HMG)-CoA reductase to prevent conversion of HMG-CoA to mevalonate. The MA <span class="highlight" style="background-color:">pathway</span> leads to sterol and nonsterol products, that is, cholesterol and the isoprenoids farnesyl pyrophosphate (FPP) and geranylgeranyl pyrophosphate (GGPP), respectively. Isoprenylation describes the formation of covalent bonds between FPP and Ras, and/or between GGPP and RhoA via enzymes farnesyltransferase and geranylgeranyltransferase, respectively. This then allows inactive cytosolic small G proteins Ras and RhoA to attach to the cell membrane as the active form, to transduce <span class="highlight" style="background-color:">signaling</span> cascades. Adapted from Takeda and colleagues (38). FTase = farnesyltransferase; GGTase = geranylgeranyltransferase.</div></p></div><div class="aux"><div class="resc">Amir A. Zeki, et al. Am J Respir Crit Care Med. 2009 October 15;180(8):731-740.</div><p class="links"><a class="dblinks" href="/pubmed/19608720" ref="ordinalpos=4804&amp;ncbi_uid=2514802&amp;link_uid=19608720">Citation</a><a class="dblinks" href="/pmc/articles/PMC2778150/" ref="ordinalpos=4804&amp;ncbi_uid=2514802&amp;link_uid=PMC2778150">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4805.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3753855/figure/F4/" ref="ordinalpos=4805&amp;ncbi_uid=5867473&amp;link_uid=PMC3753855" href="/pmc/articles/PMC3753855/"><img src="/pmc/articles/instance/3753855/bin/zmb9991000960004.gif" src-large="/pmc/articles/instance/3753855/bin/zmb9991000960004.jpg" alt="Fig 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3753855/" ref="ordinalpos=4805&amp;ncbi_uid=5867473&amp;link_uid=PMC3753855" image-link="/pmc/articles/PMC3753855/figure/F4/" class="imagepopup">Fig 4.  From: CTGF Mediates Smad-Dependent Transforming Growth Factor ? <span class="highlight" style="background-color:">Signaling</span> To Regulate Mesenchymal Cell Proliferation during Palate Development. </a></p><div class="supp"><p class="details"><div>CTGF regulates its own expression through the TGF-β <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) qPCR analysis of <i>Ctgf</i> expression in primary MEPM cells from control and <i>Wnt1-Cre</i>; <i>Tgfbr2</i><sup><i>fl/fl</i></sup> (<i>Tgfbr2</i> CKO) embryos treated with CTGF recombinant protein compared with untreated cells. (B) qPCR analysis of <i>PAI-1</i> expression in primary MEPM cells from control and <i>Tgfbr2</i> CKO embryos treated with CTGF compared with untreated cells. (C) Immunoblotting analysis of phosphorylated SMAD2 (P-SMAD2) and SMAD3 (P-SMAD3) in control MEPM treated with CTGF compared with untreated control MEPM cells. *, <i>P</i> &lt; 0.05. CKO, conditional knockout.</div></p></div><div class="aux"><div class="resc">Carolina Parada, et al. Mol Cell Biol. 2013 September;33(17):3482-3493.</div><p class="links"><a class="dblinks" href="/pubmed/23816882" ref="ordinalpos=4805&amp;ncbi_uid=5867473&amp;link_uid=23816882">Citation</a><a class="dblinks" href="/pmc/articles/PMC3753855/" ref="ordinalpos=4805&amp;ncbi_uid=5867473&amp;link_uid=PMC3753855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4806.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1150820/figure/fig4/" ref="ordinalpos=4806&amp;ncbi_uid=502587&amp;link_uid=PMC1150820" href="/pmc/articles/PMC1150820/"><img src="/pmc/articles/instance/1150820/bin/zpq0210583550004.gif" src-large="/pmc/articles/instance/1150820/bin/zpq0210583550004.jpg" alt="Fig. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1150820/" ref="ordinalpos=4806&amp;ncbi_uid=502587&amp;link_uid=PMC1150820" image-link="/pmc/articles/PMC1150820/figure/fig4/" class="imagepopup">Fig. 4. From: Uterine Wnt/?-catenin <span class="highlight" style="background-color:">signaling</span> is required for implantation. </a></p><div class="supp"><p class="details"><div>Activation of the Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the uterus is required for proper blastocyst implantation. (<i>A</i>) Uterine horn on day 6 in which blastocysts along with BSA had been transferred on day 4. (<i>B</i>) Uterine horn on day 6 in which blastocysts along with sFRP1 had been transferred on day 4. (<i>C</i>) Uterine horn on day 6 in which blastocysts along with sFRP2 had been transferred on day 4. (<i>D</i>) Uterus on day 6 in which sFRP2-conditioned media along with embryoid bodies expressing sFRP2 had been injected in the right uterine horn on the morning of day 4. The left horn was not injected and served as a control. (<i>E</i>) Embryos dissected from a uterine horn that had been injected with sFRP2-conditioned media along with embryoid bodies expressing sFRP2 (<i>Right</i>) and embryos from the same uterus from the control uterine horn (<i>Left</i>).</div></p></div><div class="aux"><div class="resc">Othman A. Mohamed, et al. Proc Natl Acad Sci U S A. 2005 June 14;102(24):8579-8584.</div><p class="links"><a class="dblinks" href="/pubmed/15930138" ref="ordinalpos=4806&amp;ncbi_uid=502587&amp;link_uid=15930138">Citation</a><a class="dblinks" href="/pmc/articles/PMC1150820/" ref="ordinalpos=4806&amp;ncbi_uid=502587&amp;link_uid=PMC1150820">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4807.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3267860/figure/F6/" ref="ordinalpos=4807&amp;ncbi_uid=4064475&amp;link_uid=PMC3267860" href="/pmc/articles/PMC3267860/"><img src="/pmc/articles/instance/3267860/bin/nihms337420f6.gif" src-large="/pmc/articles/instance/3267860/bin/nihms337420f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3267860/" ref="ordinalpos=4807&amp;ncbi_uid=4064475&amp;link_uid=PMC3267860" image-link="/pmc/articles/PMC3267860/figure/F6/" class="imagepopup">Figure 6. Postnatal deletion of plexin-D1 perturbs spine density and mEPSC frequency in direct but not indirect <span class="highlight" style="background-color:">pathway</span> MSNs.  From: Semaphorin 3E-Plexin-D1 <span class="highlight" style="background-color:">signaling</span> controls <span class="highlight" style="background-color:">pathway</span>-specific synapse formation in the striatum. </a></p><div class="supp"><p class="details"><div>(<b>a</b>) AAV encoding Cre-mCherry was injected into the striatum of P4–5 <i>Drd2</i>-GFP;<i>plexin-D1</i><sup>f/f</sup> mice or their littermate controls (<i>Drd2</i>-GFP;<i>plexin-D1</i><sup>f/+</sup>). (<b>b</b>) Brain slices were cut 2–3 weeks after virus injection and mCherry fluorescence was used to confirm Cre expression (<i>right</i>). L: lateral, A: anterior. i.c.: internal capsule. (<b>c</b>) Image of a brain slice from a <i>Drd2</i>-GFP;<i>plexin-D1</i><sup>f/f</sup> mouse 2 weeks after AAV virus injection demonstrating sparse Cre expression (red) in both GFP positive (green) and negative neurons. (<b>d</b>) mEPSCs recorded from Cre+ direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs from <i>plexin-D1</i><sup>f/f</sup> mice and <i>plexin-D1</i><sup>f/+</sup> control mice. (<b>e–f</b>) Summary of changes in mEPSC amplitude (<i>left</i>) and frequency (<i>right</i>) in direct (<b>e</b>) and indirect (<b>f</b>) <span class="highlight" style="background-color:">pathway</span> MSNs. (*<i>P</i>&lt;0.05; Mann-Whitney). (<b>g</b>) Images of dendritic segments of Cre-expressing direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs in mice with one (<i>top</i>) or two (<i>bottom</i>) copies of the conditional <i>plexin-D1</i> allele. The Cre+ direct <span class="highlight" style="background-color:">pathway</span> MSNs (red in <b>c</b>) in <i>plexin-D1</i><sup>f/f</sup> animals show decreased spine density compared to those in virus-injected littermate controls (Cre+; <i>plexin-D1</i><sup>f/+</sup>). In contrast, there is no detectable difference in spine density between Cre+ indirect <span class="highlight" style="background-color:">pathway</span> MSNs (yellow in <b>c</b>) in <i>plexin-D1</i><sup>f/f</sup> and <i>plexin-D1</i><sup>f/+</sup> littermate control mice. (<b>h</b>) Summary of changes in direct and indirect <span class="highlight" style="background-color:">pathway</span> MSNs showing a significant difference in spine density between Cre+ <i>plexin-D1</i><sup>f/f</sup> direct <span class="highlight" style="background-color:">pathway</span> MSNs and Cre+ <i>plexin-D1</i><sup>f/+</sup> heterozygous controls (*<i>P</i>&lt;0.05; Mann-Whitney). Error bars denote s.e.m.</div></p></div><div class="aux"><div class="resc">Jun B. Ding, et al. Nat Neurosci. ;15(2):215-223.</div><p class="links"><a class="dblinks" href="/pubmed/22179111" ref="ordinalpos=4807&amp;ncbi_uid=4064475&amp;link_uid=22179111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3267860/" ref="ordinalpos=4807&amp;ncbi_uid=4064475&amp;link_uid=PMC3267860">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4808.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2725709/figure/f7/" ref="ordinalpos=4808&amp;ncbi_uid=1928612&amp;link_uid=PMC2725709" href="/pmc/articles/PMC2725709/"><img src="/pmc/articles/instance/2725709/bin/zmb0170982320007.gif" src-large="/pmc/articles/instance/2725709/bin/zmb0170982320007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2725709/" ref="ordinalpos=4808&amp;ncbi_uid=1928612&amp;link_uid=PMC2725709" image-link="/pmc/articles/PMC2725709/figure/f7/" class="imagepopup">FIG. 7. From: Ubiquitin Ligase Cbl-b Is a Negative Regulator for Insulin-Like Growth Factor 1 <span class="highlight" style="background-color:">Signaling</span> during Muscle Atrophy Caused by Unloading . </a></p><div class="supp"><p class="details"><div>Mechanistic model of unloading-mediated muscle atrophy. Unloading induces ubiquitin ligase Cbl-b in myocytes. Cbl-b stimulates ubiquitination and the degradation of IRS-1, an important intermediate in IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, resulting in IGF-1 resistance in myocytes during unloading. IGF-1 resistance induces impaired protein synthesis and enhances protein degradation in muscle, leading to muscle atrophy. Cbl-b and PI3K may interact with phosphotyrosine<sup>608</sup> of IRS-1. The inhibition of Cbl-b and IRS-1 interaction by oligopeptides may restore this impairment of IGF-1 <span class="highlight" style="background-color:">signaling</span> (see Discussion for more details). GSK3, glycogen synthase kinase 3; mTOR, mammalian target of rapamycin; S6K, p70 S6 kinase; Ub, ubiquitin.</div></p></div><div class="aux"><div class="resc">Reiko Nakao, et al. Mol Cell Biol. 2009 September;29(17):4798-4811.</div><p class="links"><a class="dblinks" href="/pubmed/19546233" ref="ordinalpos=4808&amp;ncbi_uid=1928612&amp;link_uid=19546233">Citation</a><a class="dblinks" href="/pmc/articles/PMC2725709/" ref="ordinalpos=4808&amp;ncbi_uid=1928612&amp;link_uid=PMC2725709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4809.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4280021/figure/F7/" ref="ordinalpos=4809&amp;ncbi_uid=7004483&amp;link_uid=PMC4280021" href="/pmc/articles/PMC4280021/"><img src="/pmc/articles/instance/4280021/bin/nihms65239f7.gif" src-large="/pmc/articles/instance/4280021/bin/nihms65239f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4280021/" ref="ordinalpos=4809&amp;ncbi_uid=7004483&amp;link_uid=PMC4280021" image-link="/pmc/articles/PMC4280021/figure/F7/" class="imagepopup">Figure 7. Schematic illustrating the <span class="highlight" style="background-color:">signaling</span> pathways mediating of ET-1 effect on AR expression.  From: Endothelin-1 enhances the expression of the androgen receptor via activation of the c-Myc <span class="highlight" style="background-color:">pathway</span> in prostate cancer cells. </a></p><div class="supp"><p class="details"><div>After androgen withdrawal, ET-1 enhanced c-Myc expression, via activation of Src and PI-3 K <span class="highlight" style="background-color:">signaling</span>, leading to increased level of AR transcription.</div></p></div><div class="aux"><div class="resc">June G Lee, et al. Mol Carcinog. ;48(2):141-149.</div><p class="links"><a class="dblinks" href="/pubmed/18623111" ref="ordinalpos=4809&amp;ncbi_uid=7004483&amp;link_uid=18623111">Citation</a><a class="dblinks" href="/pmc/articles/PMC4280021/" ref="ordinalpos=4809&amp;ncbi_uid=7004483&amp;link_uid=PMC4280021">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4810.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3224699/figure/F2/" ref="ordinalpos=4810&amp;ncbi_uid=3678343&amp;link_uid=PMC3224699" href="/pmc/articles/PMC3224699/"><img src="/pmc/articles/instance/3224699/bin/1471-2202-10-97-2.gif" src-large="/pmc/articles/instance/3224699/bin/1471-2202-10-97-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3224699/" ref="ordinalpos=4810&amp;ncbi_uid=3678343&amp;link_uid=PMC3224699" image-link="/pmc/articles/PMC3224699/figure/F2/" class="imagepopup">Figure 2.  From: Notch <span class="highlight" style="background-color:">signaling</span> is required for maintaining stem-cell features of neuroprogenitor cells derived from human embryonic stem cells. </a></p><div class="supp"><p class="details"><div><b>Notch <span class="highlight" style="background-color:">signaling</span> is active in neuroectodermal spheres derived from human embryonic stem cells</b>. (<b>A</b>) Transcriptional expression of Notch <span class="highlight" style="background-color:">signaling</span> members in neuroectodermal spheres (NESs). All components functionally related with Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> were up-regulated in the NES samples. <i>NOTCH4 </i>was not expressed in all samples (data not shown). <i>HEY1 </i>and <i>HEY2 </i>were not preferentially expressed in the NESs, because they are regulated by Notch <span class="highlight" style="background-color:">signaling</span> but are associated with vascular development [30]. <i>GAPDH</i>, loading control. (<b>B</b>) Western-blot analyses of Notch <span class="highlight" style="background-color:">signaling</span> molecules in the NESs. All proteins of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> examined were expressed in hESCs, EBs and NESs but all were most abundant in the NES protein extracts. NICD, NOTCH Intracellular Domain (or cleaved Notch1). α-TUBULIN, loading control. (<b>C</b>) Immunostaining of the rosettes for the Notch ligands (JAG1 and DLL1) and receptor (NOTCH1). Rosettes structures derived from hESC-derived NESs were visualized using the rosette marker N-cadherin (N-CAD) and shown to locate to the inner rims of the rosettes. Both JAG1 (<b>a </b>and <b>c</b>) and DLL1 (<b>b </b>and <b>d</b>) transmembrane ligands are localized in the regions where N-cadherin signals exist. Anti-NOTCH1 antibody, which recognizes both NICD and C-terminal cytoplasmic domain of membrane-bound NOTCH1 receptor simultaneously, generates a rather diffused signal (<b>c </b>and <b>d</b>); nevertheless, its location at the luminal side of rosette is evident. Scale bars, 100 μm.</div></p></div><div class="aux"><div class="resc">Sun-Mi Woo, et al. BMC Neurosci. 2009;10:97-97.</div><p class="links"><a class="dblinks" href="/pubmed/19682396" ref="ordinalpos=4810&amp;ncbi_uid=3678343&amp;link_uid=19682396">Citation</a><a class="dblinks" href="/pmc/articles/PMC3224699/" ref="ordinalpos=4810&amp;ncbi_uid=3678343&amp;link_uid=PMC3224699">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4811.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4270583/figure/fig03/" ref="ordinalpos=4811&amp;ncbi_uid=7003095&amp;link_uid=PMC4270583" href="/pmc/articles/PMC4270583/"><img src="/pmc/articles/instance/4270583/bin/ijcep0007-7962-f3.gif" src-large="/pmc/articles/instance/4270583/bin/ijcep0007-7962-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4270583/" ref="ordinalpos=4811&amp;ncbi_uid=7003095&amp;link_uid=PMC4270583" image-link="/pmc/articles/PMC4270583/figure/fig03/" class="imagepopup">Figure 3.  From: RVLM-IML <span class="highlight" style="background-color:">pathway</span> may implicate controlling peripheral airways by melanocortinergic-sympathetic <span class="highlight" style="background-color:">signaling</span>: a transneuronal labeling study using pseudorabies virus. </a></p><div class="supp"><p class="details"><div>Summary diagram showed that the sympathetic <span class="highlight" style="background-color:">pathway</span> between the RVLM and the tracheal wall. Injection of PRV-614 into the tracheal wall resulted in retrograde infection of neurons in the DMV by parasympathetic <span class="highlight" style="background-color:">pathway</span> (shown in black line), and IML and RVLM by sympathetic <span class="highlight" style="background-color:">pathway</span> (shown in red line). PRV-614/MC4R-GFP dual-labeled neurons were detected in the IML, RVLM and DMV. DMV, dorsal motor nucleus of the vagus nerve; IML, intermediolateral column; MC4R, melanocortin-4 receptor; RVLM, rostroventrolateral reticular nucleus. Some drawings were taken from HB Xiang (Brain 2013).</div></p></div><div class="aux"><div class="resc">Cheng-Jin Yue, et al. Int J Clin Exp Pathol. 2014;7(11):7962-7966.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4811&amp;ncbi_uid=7003095&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4270583/" ref="ordinalpos=4811&amp;ncbi_uid=7003095&amp;link_uid=PMC4270583">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4812.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394621/figure/F7/" ref="ordinalpos=4812&amp;ncbi_uid=7414315&amp;link_uid=PMC4394621" href="/pmc/articles/PMC4394621/"><img src="/pmc/articles/instance/4394621/bin/nihms-677118-f0007.gif" src-large="/pmc/articles/instance/4394621/bin/nihms-677118-f0007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394621/" ref="ordinalpos=4812&amp;ncbi_uid=7414315&amp;link_uid=PMC4394621" image-link="/pmc/articles/PMC4394621/figure/F7/" class="imagepopup">Figure 7.  From: Combined treatment with EGFR inhibitors and arsenite upregulated apoptosis in human EGFR-positive melanomas: a role of suppression of the PI3K-AKT <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Kinase targets and inhibitors. A schematic illustration of the principal <span class="highlight" style="background-color:">signaling</span> pathways that are affected by the inhibitors, which are described in this paper. Protein tyrosine kinase activity of EGFR is inhibited with AG1478 or PD153035. Downstream events of EGFR-mediated <span class="highlight" style="background-color:">signaling</span> include a formation of multisubunit scaffold complex with the activation of the Ras-Raf-MEK-ERK <span class="highlight" style="background-color:">pathway</span>. PD98059 and U0126 are specific inhibitors of MEK1/2. The PI3K-AKT <span class="highlight" style="background-color:">pathway</span> that is initiated at the active EGFR can be suppressed with pharmacological inhibitor LY294002 or with the endogenous inhibitor PTEN. The MEK-ERK and PI3K-AKT pathways activate the specific sets of transcription factors, which are involved in the antiapoptotic and cell-cycle gene expression. AKT also regulates general protein translation via mTOR kinase, which is sensitive to rapamycin. The TNFR-mediated <span class="highlight" style="background-color:">signaling</span> induces via adaptor proteins IKK activity followed by phosphorylation and proteasome-dependent degradation of IκB, and release and nuclear translocation of NF-<i>κ</i>B, which regulates the expression of many different genes, including antiapoptotic genes. Simultaneously, the TNFR-mediated <span class="highlight" style="background-color:">signaling</span> may activate caspase-8/caspase-3 proapoptotic <span class="highlight" style="background-color:">pathway</span>; at normal conditions, however, this <span class="highlight" style="background-color:">pathway</span> is suppressed by the NF-<i>κ</i>B-dependent antiapoptotic proteins. Arsenite upregulates the MAPK pathways, but eventually suppresses IKK-NF-<i>κ</i>B activation affecting a balance between cell survival and apoptosis</div></p></div><div class="aux"><div class="resc">Vladimir N Ivanov, et al. Oncogene. ;24(4):616-626.</div><p class="links"><a class="dblinks" href="/pubmed/15580309" ref="ordinalpos=4812&amp;ncbi_uid=7414315&amp;link_uid=15580309">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394621/" ref="ordinalpos=4812&amp;ncbi_uid=7414315&amp;link_uid=PMC4394621">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4813.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2899694/figure/F6/" ref="ordinalpos=4813&amp;ncbi_uid=2322536&amp;link_uid=PMC2899694" href="/pmc/articles/PMC2899694/"><img src="/pmc/articles/instance/2899694/bin/nihms213803f6.gif" src-large="/pmc/articles/instance/2899694/bin/nihms213803f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2899694/" ref="ordinalpos=4813&amp;ncbi_uid=2322536&amp;link_uid=PMC2899694" image-link="/pmc/articles/PMC2899694/figure/F6/" class="imagepopup">Figure 6.  From: THE TWO-COMPONENT <span class="highlight" style="background-color:">SIGNALING</span> <span class="highlight" style="background-color:">PATHWAY</span> OF BACTERIAL CHEMOTAXIS: A Molecular View of Signal Transduction by Receptors, Kinases, and Adaptation Enzymes. </a></p><div class="supp"><p class="details"><div>A typical receptor-kinase <span class="highlight" style="background-color:">signaling</span> complex illustrated by the aspartate receptor. The transmembrane receptor provides the architectural framework of the super-molecular <span class="highlight" style="background-color:">signaling</span> complex (Borkovich et al 1989, Ninfa et al 1991, Gegner et al 1992, Schuster et al 1993, Wu et al 1996). Most of the chemosensory <span class="highlight" style="background-color:">pathway</span> components are associated with this complex, either stably or transiently. The kinetically stable core ternary complex is composed of the dimeric receptor (illustrated as a collection of helices), the coupling protein CheW, and the dimeric histidine kinase CheA. Other components are believed to be in rapid equilibrium between bound and soluble forms, including periplasmic binding proteins, the methyltransferase CheR, the methylesterase CheB, and the motor response regulator CheY.</div></p></div><div class="aux"><div class="resc">Joseph J. Falke, et al. Annu Rev Cell Dev Biol. ;13:457-512.</div><p class="links"><a class="dblinks" href="/pubmed/9442881" ref="ordinalpos=4813&amp;ncbi_uid=2322536&amp;link_uid=9442881">Citation</a><a class="dblinks" href="/pmc/articles/PMC2899694/" ref="ordinalpos=4813&amp;ncbi_uid=2322536&amp;link_uid=PMC2899694">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4814.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2761106/figure/fig8/" ref="ordinalpos=4814&amp;ncbi_uid=2540395&amp;link_uid=PMC2761106" href="/pmc/articles/PMC2761106/"><img src="/pmc/articles/instance/2761106/bin/DEV036335F8.gif" src-large="/pmc/articles/instance/2761106/bin/DEV036335F8.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2761106/" ref="ordinalpos=4814&amp;ncbi_uid=2540395&amp;link_uid=PMC2761106" image-link="/pmc/articles/PMC2761106/figure/fig8/" class="imagepopup">Fig. 8. From: BMP7 promotes proliferation of nephron progenitor cells via a  JNK-dependent mechanism. </a></p><div class="supp"><p class="details"><div><b>Model of BMP7 <span class="highlight" style="background-color:">signaling</span> in the NZ.</b> BMP7 preferentially functions  through the JNK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote proliferation in the nephron  progenitor compartment. The SMAD machinery is silent at the transcriptional  level in cap mesenchyme, but becomes increasingly activated as cells progress  to the renal vesicle stage and beyond.</div></p></div><div class="aux"><div class="resc">Ulrika Blank, et al. Development. 2009 November 1;136(21):3557-3566.</div><p class="links"><a class="dblinks" href="/pubmed/19793891" ref="ordinalpos=4814&amp;ncbi_uid=2540395&amp;link_uid=19793891">Citation</a><a class="dblinks" href="/pmc/articles/PMC2761106/" ref="ordinalpos=4814&amp;ncbi_uid=2540395&amp;link_uid=PMC2761106">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4815.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC137067/figure/F1/" ref="ordinalpos=4815&amp;ncbi_uid=330689&amp;link_uid=PMC137067" href="/pmc/articles/PMC137067/"><img src="/pmc/articles/instance/137067/bin/1475-4924-1-7-1.gif" src-large="/pmc/articles/instance/137067/bin/1475-4924-1-7-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC137067/" ref="ordinalpos=4815&amp;ncbi_uid=330689&amp;link_uid=PMC137067" image-link="/pmc/articles/PMC137067/figure/F1/" class="imagepopup">Figure 1.  From: Agonizing Hedgehog. </a></p><div class="supp"><p class="details"><div>The Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(a)</b> In the absence of Hedgehog ligands, Patched protein inhibits the activity of Smoothened, which resembles G-protein-coupled receptors. <b>(b)</b> On activation by a ligand of the Hedgehog family, the inhibition of Smoothened by Patched is relieved and Smoothened is freed to trigger an intracellular <span class="highlight" style="background-color:">signaling</span> cascade; this ultimately leads to transcriptional regulation by transcription factors of the Gli family.</div></p></div><div class="aux"><div class="resc">Jonathan B Weitzman. J Biol. 2002;1(2):7-7.</div><p class="links"><a class="dblinks" href="/pubmed/12437770" ref="ordinalpos=4815&amp;ncbi_uid=330689&amp;link_uid=12437770">Citation</a><a class="dblinks" href="/pmc/articles/PMC137067/" ref="ordinalpos=4815&amp;ncbi_uid=330689&amp;link_uid=PMC137067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4816.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1995089/figure/F8/" ref="ordinalpos=4816&amp;ncbi_uid=1062477&amp;link_uid=PMC1995089" href="/pmc/articles/PMC1995089/"><img src="/pmc/articles/instance/1995089/bin/nihms-28375-f0008.gif" src-large="/pmc/articles/instance/1995089/bin/nihms-28375-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1995089/" ref="ordinalpos=4816&amp;ncbi_uid=1062477&amp;link_uid=PMC1995089" image-link="/pmc/articles/PMC1995089/figure/F8/" class="imagepopup">Fig. 8.  From: Drosophila dopamine synthesis <span class="highlight" style="background-color:">pathway</span> genes regulate tracheal morphogenesis. </a></p><div class="supp"><p class="details"><div>A model for the action of DA synthesis <span class="highlight" style="background-color:">pathway</span> genes. In wild-type cells (middle), the surface level of Btl/FGFR is modulated by endocytosis via the action of Shi/dynamin, while the stability of Shi/dynamin is regulated by DA, presumably through a DA-receptor mediated <span class="highlight" style="background-color:">signaling</span> event. In <i>Pu</i> or <i>ple</i> mutants (right), or when DA synthesis is blocked by 3-IT, Shi/dynamin level is reduced, resulting in over-accumulation of Btl/FGFR and ectopic migration. In <i>Catsup</i> mutants, or in the presence of exogenously added DA, the level of Shi/dynamin is increased, resulting in down-regulation of surface Btl/FGFR and reduced migration.</div></p></div><div class="aux"><div class="resc">Anita Hsouna, et al. Dev Biol. ;308(1):30-43.</div><p class="links"><a class="dblinks" href="/pubmed/17585895" ref="ordinalpos=4816&amp;ncbi_uid=1062477&amp;link_uid=17585895">Citation</a><a class="dblinks" href="/pmc/articles/PMC1995089/" ref="ordinalpos=4816&amp;ncbi_uid=1062477&amp;link_uid=PMC1995089">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4817.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4116528/figure/F8/" ref="ordinalpos=4817&amp;ncbi_uid=6458030&amp;link_uid=PMC4116528" href="/pmc/articles/PMC4116528/"><img src="/pmc/articles/instance/4116528/bin/oncotarget-05-3880-g008.gif" src-large="/pmc/articles/instance/4116528/bin/oncotarget-05-3880-g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4116528/" ref="ordinalpos=4817&amp;ncbi_uid=6458030&amp;link_uid=PMC4116528" image-link="/pmc/articles/PMC4116528/figure/F8/" class="imagepopup">Figure 8. The Smad1/Akt/GSK3β <span class="highlight" style="background-color:">pathway</span> is consistently activated in CL1-5 cells but is downregulated in Snail-silenced CL1-5 cells.  From: Snail regulates Nanog status during the epithelial-mesenchymal transition via the Smad1/Akt/GSK3? <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in non-small-cell lung cancer. </a></p><div class="supp"><p class="details"><div>(A) CL1-5 cells, which endogenously express Snail, exhibited greater activation of the Smad1/Akt/GSK3β <span class="highlight" style="background-color:">pathway</span> and upregulated Nanog expression in comparison to CL1-0 cells. Both SB431542 and LDN193189 decreased activation of the Smad1/Akt/GSK3β <span class="highlight" style="background-color:">pathway</span> and downregulated Nanog expression in CL1-5 cells. Unlike Akt and GSK3β phosphorylation, which was completely inhibited by LY294002, Smad1 phosphorylation did not respond to LY294002. (B/C) Scramble and Smad1 siRNA were expressed in CL1-5 cells for 40 h. The Smad1 siRNA fully suppressed the Smad1/Akt/GSK3β <span class="highlight" style="background-color:">pathway</span> as well as Nanog expression. In addition, the expression of mesenchymal markers decreased, but the expression of epithelial markers increased in Smad-silenced CL1-5 cells as compared to CL1-0 cells. (D) The endogenously expressed Snail was silenced in CL1-5 cells. An increase in E-cadherin and a decrease in vimentin expression were found in Snail-silenced CL1-5 cells. (E) In addition, a reduction in the number of migrating cells was observed in Snail-silenced CL1-5 cells as compared to the control cells (transfected with Scramble siRNA); ***<i>p</i> &lt; 0.001 indicates statistical significance as compared to the control. (F) Either a decrease in the level of activation of the Smad1/Akt/GSK3β <span class="highlight" style="background-color:">pathway</span> or a reduction in Nanog expression was found in Snail-silenced CL1-5 cells.</div></p></div><div class="aux"><div class="resc">Chen-Wei Liu, et al. Oncotarget. 2014 June;5(11):3880-3894.</div><p class="links"><a class="dblinks" href="/pubmed/25003810" ref="ordinalpos=4817&amp;ncbi_uid=6458030&amp;link_uid=25003810">Citation</a><a class="dblinks" href="/pmc/articles/PMC4116528/" ref="ordinalpos=4817&amp;ncbi_uid=6458030&amp;link_uid=PMC4116528">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4818.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3196058/figure/F2/" ref="ordinalpos=4818&amp;ncbi_uid=3515521&amp;link_uid=PMC3196058" href="/pmc/articles/PMC3196058/"><img src="/pmc/articles/instance/3196058/bin/nihms322202f2.gif" src-large="/pmc/articles/instance/3196058/bin/nihms322202f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3196058/" ref="ordinalpos=4818&amp;ncbi_uid=3515521&amp;link_uid=PMC3196058" image-link="/pmc/articles/PMC3196058/figure/F2/" class="imagepopup">Figure 2. β-catenin in Wnt <span class="highlight" style="background-color:">signaling</span> and the cadherin complex.  From: The Dynamic Role of Beta-Catenin in Synaptic Plasticity. </a></p><div class="supp"><p class="details"><div>In the resting state, β-catenin is phosphorylated by GSK-3β and rapidly degraded by the proteasome <span class="highlight" style="background-color:">pathway</span>. Upon activation of Wnt <span class="highlight" style="background-color:">signaling</span>, β-catenin is stabilized through the inhibition of GSK3β and translocates to the nucleus to regulate the expression of Wnt target genes. β-catenin also associates with the cytoplasmic domain of cadherin and directly links to the actin cytoskeleton through α-catenin. Note that the cadherin adhesion complex is a dynamic structure that also includes other intermediate proteins (such as EPLIN, Abe K &amp; Takeichi M., 2008.</div></p></div><div class="aux"><div class="resc">Kimberly A. Maguschak, et al. Neuropharmacology. ;62(1):78-88.</div><p class="links"><a class="dblinks" href="/pubmed/21903109" ref="ordinalpos=4818&amp;ncbi_uid=3515521&amp;link_uid=21903109">Citation</a><a class="dblinks" href="/pmc/articles/PMC3196058/" ref="ordinalpos=4818&amp;ncbi_uid=3515521&amp;link_uid=PMC3196058">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4819.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3430210/figure/f1-ijms-13-07886/" ref="ordinalpos=4819&amp;ncbi_uid=4210423&amp;link_uid=PMC3430210" href="/pmc/articles/PMC3430210/"><img src="/pmc/articles/instance/3430210/bin/ijms-13-07886f1.gif" src-large="/pmc/articles/instance/3430210/bin/ijms-13-07886f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3430210/" ref="ordinalpos=4819&amp;ncbi_uid=4210423&amp;link_uid=PMC3430210" image-link="/pmc/articles/PMC3430210/figure/f1-ijms-13-07886/" class="imagepopup">Figure 1.  From: Molecular Targets of TRAIL-Sensitizing Agents in Colorectal Cancer. </a></p><div class="supp"><p class="details"><div>TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and mechanisms of TRAIL resistance. Binding of TRAIL to DR4 or DR5 leads to activation of caspase-8 through FADD. Activated caspase-8 can directly activate effector caspase-3, -6, and -7 via the extrinsic <span class="highlight" style="background-color:">pathway</span> or cleave Bid and activate the intrinsic mitochondria-mediated <span class="highlight" style="background-color:">pathway</span>. Initiation of the latter <span class="highlight" style="background-color:">pathway</span> leads to activation of effector caspases through caspase-9. Activation of caspase-3, -6, and -7 leads to apoptosis. TRAIL resistance can occur at several levels. At the outer membrane, TRAIL sensitivity can be reduced by DcR1 and/or DcR2 over-expression as well as by defective expression/function of DR4 and/or DR5 (1). At the DISC, down-regulation, degradation or inactivation of the initiator caspase-8 and/or increased expression of c-FLIP can lead to TRAIL resistance (2). At the mitochondria, defects in pro-apoptotic molecules (e.g., Bid, Bax, Bak, PUMA, Bim) and/or up-regulation of proteins able to prevent cytochrome c and Smac/DIABLO release (e.g., Bcl-2, Bcl-xL and Mcl-1) also interfere with TRAIL <span class="highlight" style="background-color:">signaling</span> (3). Finally, members of the IAP family prevent the activation of both caspase-9 and caspase-3, -6 and -7 (4). →indicates activation, ⊥ indicates inhibition.</div></p></div><div class="aux"><div class="resc">Carmine Stolfi, et al. Int J Mol Sci. 2012;13(7):7886-7901.</div><p class="links"><a class="dblinks" href="/pubmed/22942679" ref="ordinalpos=4819&amp;ncbi_uid=4210423&amp;link_uid=22942679">Citation</a><a class="dblinks" href="/pmc/articles/PMC3430210/" ref="ordinalpos=4819&amp;ncbi_uid=4210423&amp;link_uid=PMC3430210">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4820.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2865574/figure/F3/" ref="ordinalpos=4820&amp;ncbi_uid=2946066&amp;link_uid=PMC2865574" href="/pmc/articles/PMC2865574/"><img src="/pmc/articles/instance/2865574/bin/nihms168601f3.gif" src-large="/pmc/articles/instance/2865574/bin/nihms168601f3.jpg" alt="FIGURE 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2865574/" ref="ordinalpos=4820&amp;ncbi_uid=2946066&amp;link_uid=PMC2865574" image-link="/pmc/articles/PMC2865574/figure/F3/" class="imagepopup">FIGURE 3.  From: PERSPECTIVE: SMALL MOLECULES AND STEM CELLS; POTENCY AND LINEAGE COMMITMENT: THE NEW QUEST FOR THE FOUNTAIN OF YOUTH. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> is a complex <span class="highlight" style="background-color:">pathway</span>, believed to be involved in the regulation of divergent processes, including the maintenance of pluripotency and commitment to differentiation. To resolve these divergent responses to activation of Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, we have developed a model in which β-catenin/CBP-mediated transcription is critical for the maintenance of potency, whereas a switch to β-catenin/p300-mediated transcription is the first critical step to initiate differentiation. Hence the balance between CBP and p300-mediated β-catenin transcription regulates the balance between maintenance of potency and the initiation of commitment to differentiate. The interaction of β-catenin with either CBP or p300 can be modulated by two small molecules, respectively ICG-001 and IQ-1, identified in our laboratory. As a consequence, these small molecules regulate Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> outcomes.</div></p></div><div class="aux"><div class="resc">Agnès I. Lukaszewicz, et al. J Med Chem. ;53(9):3439-3453.</div><p class="links"><a class="dblinks" href="/pubmed/20047330" ref="ordinalpos=4820&amp;ncbi_uid=2946066&amp;link_uid=20047330">Citation</a><a class="dblinks" href="/pmc/articles/PMC2865574/" ref="ordinalpos=4820&amp;ncbi_uid=2946066&amp;link_uid=PMC2865574">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4821.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4010805/figure/F7/" ref="ordinalpos=4821&amp;ncbi_uid=6111881&amp;link_uid=PMC4010805" href="/pmc/articles/PMC4010805/"><img src="/pmc/articles/instance/4010805/bin/zh00081474780007.gif" src-large="/pmc/articles/instance/4010805/bin/zh00081474780007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4010805/" ref="ordinalpos=4821&amp;ncbi_uid=6111881&amp;link_uid=PMC4010805" image-link="/pmc/articles/PMC4010805/figure/F7/" class="imagepopup">Fig. 7. From: Role for high-glucose-induced protein O-GlcNAcylation in stimulating cardiac fibroblast collagen synthesis. </a></p><div class="supp"><p class="details"><div><i>O</i>-GlcNAcase effects on the TGF-β1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>A</i>: Western blots of TGF-β1 and <span class="highlight" style="background-color:">signaling</span> molecules SMAD 2/3 and SMAD 7 proteins in CF cultured in NG (5.5 mM dextrose), HG (25 mM dextrose), HG + GCA (Adv-GCA), or HG + SR- (Adv-SR-). <i>B</i>: Graphical representation of data (<i>n</i> = 3; *<i>P</i> &lt; 0.05; **<i>P</i> &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Hugo Aguilar, et al. Am J Physiol Cell Physiol. 2014 May 1;306(9):C794-C804.</div><p class="links"><a class="dblinks" href="/pubmed/24553187" ref="ordinalpos=4821&amp;ncbi_uid=6111881&amp;link_uid=24553187">Citation</a><a class="dblinks" href="/pmc/articles/PMC4010805/" ref="ordinalpos=4821&amp;ncbi_uid=6111881&amp;link_uid=PMC4010805">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4822.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3131208/figure/f2-viruses-03-00714/" ref="ordinalpos=4822&amp;ncbi_uid=3280184&amp;link_uid=PMC3131208" href="/pmc/articles/PMC3131208/"><img src="/pmc/articles/instance/3185777/bin/viruses-03-00714f2.gif" src-large="/pmc/articles/instance/3185777/bin/viruses-03-00714f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3131208/" ref="ordinalpos=4822&amp;ncbi_uid=3280184&amp;link_uid=PMC3131208" image-link="/pmc/articles/PMC3131208/figure/f2-viruses-03-00714/" class="imagepopup">Figure 2. From: Human T-Cell Lymphotropic Virus: A Model of NF-?B-Associated Tumorigenesis. </a></p><div class="supp"><p class="details"><div>Cellular targets of the Tax viral oncoprotein. NF-κB, PI3K/AKT and SRF are well-known for their roles in various cellular functions, particularly cell survival and cell proliferation. The JAK/STAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is activated indirectly through Tax-dependent cytokine induction, while all other <span class="highlight" style="background-color:">signaling</span> molecules/<span class="highlight" style="background-color:">signaling</span> pathways are directly regulated by Tax. SRF: serum response factor.</div></p></div><div class="aux"><div class="resc">Zhaoxia Qu, et al. Viruses. 2011 June;3(6):714-749.</div><p class="links"><a class="dblinks" href="/pubmed/21743832" ref="ordinalpos=4822&amp;ncbi_uid=3280184&amp;link_uid=21743832">Citation</a><a class="dblinks" href="/pmc/articles/PMC3131208/" ref="ordinalpos=4822&amp;ncbi_uid=3280184&amp;link_uid=PMC3131208">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4823.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2949268/figure/F3/" ref="ordinalpos=4823&amp;ncbi_uid=2502676&amp;link_uid=PMC2949268" href="/pmc/articles/PMC2949268/"><img src="/pmc/articles/instance/2949268/bin/nihms135390f3.gif" src-large="/pmc/articles/instance/2949268/bin/nihms135390f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2949268/" ref="ordinalpos=4823&amp;ncbi_uid=2502676&amp;link_uid=PMC2949268" image-link="/pmc/articles/PMC2949268/figure/F3/" class="imagepopup">Figure 3.  From: The C. elegans germ line: a model for stem cell biology. </a></p><div class="supp"><p class="details"><div>Diagrams of relationships of selected proposed interactions between <span class="highlight" style="background-color:">signaling</span> pathways that influence proliferation and differentiation. A: simple relationship between GLP-1 activity and germ cell fate. B: Additional interactions downstream of the GLP-1/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Protein families/orthologs are indicated in gray. See text for further explanation.</div></p></div><div class="aux"><div class="resc">E. Jane Albert Hubbard. Dev Dyn. ;236(12):3343-3357.</div><p class="links"><a class="dblinks" href="/pubmed/17948315" ref="ordinalpos=4823&amp;ncbi_uid=2502676&amp;link_uid=17948315">Citation</a><a class="dblinks" href="/pmc/articles/PMC2949268/" ref="ordinalpos=4823&amp;ncbi_uid=2502676&amp;link_uid=PMC2949268">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4824.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2637179/figure/F2/" ref="ordinalpos=4824&amp;ncbi_uid=1742344&amp;link_uid=PMC2637179" href="/pmc/articles/PMC2637179/"><img src="/pmc/articles/instance/2637179/bin/nihms85692f2.gif" src-large="/pmc/articles/instance/2637179/bin/nihms85692f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2637179/" ref="ordinalpos=4824&amp;ncbi_uid=1742344&amp;link_uid=PMC2637179" image-link="/pmc/articles/PMC2637179/figure/F2/" class="imagepopup">Figure 2.  From: The cullin7 E3 ubiquitin ligase: A novel player in growth control. </a></p><div class="supp"><p class="details"><div>Role of IRS-1 in insulin and IGF-1 <span class="highlight" style="background-color:">signaling</span>. Upon ligand binding to the receptor, IRS-1 is recruited to the receptor and phosphorylated on tyrosine residues, which serve as docking sites for adaptor proteins of the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> or Ras-Erk MAPK <span class="highlight" style="background-color:">pathway</span>. Akt <span class="highlight" style="background-color:">signaling</span> is restrained by a negative feedback loop via mTOR and its effector serine/threonine kinase S6K. Phosphorylation of multiple serine residues on IRS-1 by mTOR/S6K may create a phosphodegron required for CUL7 E3 ligase-mediated ubiquitination and proteasomal degradation.</div></p></div><div class="aux"><div class="resc">Antonio Sarikas, et al. Cell Cycle. ;7(20):3154-3161.</div><p class="links"><a class="dblinks" href="/pubmed/18927510" ref="ordinalpos=4824&amp;ncbi_uid=1742344&amp;link_uid=18927510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2637179/" ref="ordinalpos=4824&amp;ncbi_uid=1742344&amp;link_uid=PMC2637179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4825.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3119709/figure/F1/" ref="ordinalpos=4825&amp;ncbi_uid=3512504&amp;link_uid=PMC3119709" href="/pmc/articles/PMC3119709/"><img src="/pmc/articles/instance/3119709/bin/nihms-303668-f0001.gif" src-large="/pmc/articles/instance/3119709/bin/nihms-303668-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3119709/" ref="ordinalpos=4825&amp;ncbi_uid=3512504&amp;link_uid=PMC3119709" image-link="/pmc/articles/PMC3119709/figure/F1/" class="imagepopup">Fig. 1. Host <span class="highlight" style="background-color:">signaling</span> manipulation by clonal types of T. gondii.  From: Macrophages as a Battleground for Toxoplasma Pathogenesis. </a></p><div class="supp"><p class="details"><div>ROP proteins are delivered into the cell during and after parasite entry and differentially activate host <span class="highlight" style="background-color:">signaling</span> pathways. Type I strains use ROP16 to phosphorylate and activate STAT3 and STAT6 leading to activation of an AAM-type gene expression profile (left). By contrast, the Type II ROP16 has a negligible effect on STAT3 and STAT6. Instead, GRA15 activates NF-κB to drive a CAM-type response, while the Type I GRA15 has a negligible effect on NF-κB activation in Type I strains where ROP16 activates STAT3 and STAT6. The parasite entry <span class="highlight" style="background-color:">pathway</span> and structure modified from Boothroyd and Dubremetz (2008).</div></p></div><div class="aux"><div class="resc">Peter J. Murray. Cell Host Microbe. ;9(6):445-447.</div><p class="links"><a class="dblinks" href="/pubmed/21669391" ref="ordinalpos=4825&amp;ncbi_uid=3512504&amp;link_uid=21669391">Citation</a><a class="dblinks" href="/pmc/articles/PMC3119709/" ref="ordinalpos=4825&amp;ncbi_uid=3512504&amp;link_uid=PMC3119709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4826.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3548020/figure/F1/" ref="ordinalpos=4826&amp;ncbi_uid=5654304&amp;link_uid=PMC3548020" href="/pmc/articles/PMC3548020/"><img src="/pmc/articles/instance/3548020/bin/nihms406370f1.gif" src-large="/pmc/articles/instance/3548020/bin/nihms406370f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3548020/" ref="ordinalpos=4826&amp;ncbi_uid=5654304&amp;link_uid=PMC3548020" image-link="/pmc/articles/PMC3548020/figure/F1/" class="imagepopup">Figure 1.  From: AT2 RECEPTORS: BENEFICIAL COUNTER-REGULATORY ROLE IN CARDIOVASCULAR AND RENAL FUNCTION. </a></p><div class="supp"><p class="details"><div>Schematic representation of the major intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of angiotensin type-2 receptors (AT<sub>2</sub>Rs). MAP = mitogen-activated protein; ERK = extracellular-regulated kinase; AT<sub>1</sub>R = angiotensin type-1 receptor.</div></p></div><div class="aux"><div class="resc">Shetal H. Padia, et al. Pflugers Arch. ;465(1):99-110.</div><p class="links"><a class="dblinks" href="/pubmed/22949090" ref="ordinalpos=4826&amp;ncbi_uid=5654304&amp;link_uid=22949090">Citation</a><a class="dblinks" href="/pmc/articles/PMC3548020/" ref="ordinalpos=4826&amp;ncbi_uid=5654304&amp;link_uid=PMC3548020">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4827.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2999341/figure/F12/" ref="ordinalpos=4827&amp;ncbi_uid=2622854&amp;link_uid=PMC2999341" href="/pmc/articles/PMC2999341/"><img src="/pmc/articles/instance/2999341/bin/1471-2164-11-S3-S11-12.gif" src-large="/pmc/articles/instance/2999341/bin/1471-2164-11-S3-S11-12.jpg" alt="Figure 12" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2999341/" ref="ordinalpos=4827&amp;ncbi_uid=2622854&amp;link_uid=PMC2999341" image-link="/pmc/articles/PMC2999341/figure/F12/" class="imagepopup">Figure 12.  From: Robust inference of the context specific structure and temporal dynamics of gene regulatory network. </a></p><div class="supp"><p class="details"><div><b>Regulatory PathRNet</b> Only nodes that are close to P38 MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> or "S-7" are kept (distance&lt;3). The color of the nodes indicates cluster attribute.</div></p></div><div class="aux"><div class="resc">Jia Meng, et al. BMC Genomics. 2010;11(Suppl 3):S11-S11.</div><p class="links"><a class="dblinks" href="/pubmed/21143778" ref="ordinalpos=4827&amp;ncbi_uid=2622854&amp;link_uid=21143778">Citation</a><a class="dblinks" href="/pmc/articles/PMC2999341/" ref="ordinalpos=4827&amp;ncbi_uid=2622854&amp;link_uid=PMC2999341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4828.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4065969/figure/F2/" ref="ordinalpos=4828&amp;ncbi_uid=6309559&amp;link_uid=PMC4065969" href="/pmc/articles/PMC4065969/"><img src="/pmc/articles/instance/4065969/bin/WJG-13-497-g002.gif" src-large="/pmc/articles/instance/4065969/bin/WJG-13-497-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4065969/" ref="ordinalpos=4828&amp;ncbi_uid=6309559&amp;link_uid=PMC4065969" image-link="/pmc/articles/PMC4065969/figure/F2/" class="imagepopup">Figure 2.  From: Impact of tiny miRNAs on cancers. </a></p><div class="supp"><p class="details"><div>The positive loop between <i>mir-16</i> and LIN-12. Exogenous signal activates LIN-12/Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which promotes the transcription of <i>mir-61</i>. Overexpression of <i>mir-61</i> inhibits the activity of VAV-1 (purple) that reduces the activity of LIN-12 (pink), where LIN-12, mir-61, and VAV-1 form a feedback loop that helps maximize LIN-12 activity.</div></p></div><div class="aux"><div class="resc">Wei Liu, et al. World J Gastroenterol. 2007 January 28;13(4):497-502.</div><p class="links"><a class="dblinks" href="/pubmed/17278213" ref="ordinalpos=4828&amp;ncbi_uid=6309559&amp;link_uid=17278213">Citation</a><a class="dblinks" href="/pmc/articles/PMC4065969/" ref="ordinalpos=4828&amp;ncbi_uid=6309559&amp;link_uid=PMC4065969">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4829.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1413813/figure/F8/" ref="ordinalpos=4829&amp;ncbi_uid=711100&amp;link_uid=PMC1413813" href="/pmc/articles/PMC1413813/"><img src="/pmc/articles/instance/1413813/bin/zpq0060611520008.gif" src-large="/pmc/articles/instance/1413813/bin/zpq0060611520008.jpg" alt="Fig. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1413813/" ref="ordinalpos=4829&amp;ncbi_uid=711100&amp;link_uid=PMC1413813" image-link="/pmc/articles/PMC1413813/figure/F8/" class="imagepopup">Fig. 8. From: Synergistic roles of platelet-derived growth factor-BB and interleukin-1? in phenotypic modulation of human aortic smooth muscle cells. </a></p><div class="supp"><p class="details"><div>Schematic representation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulating the expressions of SMα-actin, SM-MHC, and calponin in SMCs on polymerized collagen in response to PDGF-BB/IL-1β costimulation.</div></p></div><div class="aux"><div class="resc">Cheng-Nan Chen, et al. Proc Natl Acad Sci U S A. 2006 February 21;103(8):2665-2670.</div><p class="links"><a class="dblinks" href="/pubmed/16477012" ref="ordinalpos=4829&amp;ncbi_uid=711100&amp;link_uid=16477012">Citation</a><a class="dblinks" href="/pmc/articles/PMC1413813/" ref="ordinalpos=4829&amp;ncbi_uid=711100&amp;link_uid=PMC1413813">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4830.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2568057/figure/f10/" ref="ordinalpos=4830&amp;ncbi_uid=1602605&amp;link_uid=PMC2568057" href="/pmc/articles/PMC2568057/"><img src="/pmc/articles/instance/2568057/bin/zek0100832060010.gif" src-large="/pmc/articles/instance/2568057/bin/zek0100832060010.jpg" alt="FIG. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2568057/" ref="ordinalpos=4830&amp;ncbi_uid=1602605&amp;link_uid=PMC2568057" image-link="/pmc/articles/PMC2568057/figure/f10/" class="imagepopup">FIG. 10. From: PKC1 Is Essential for Protection against both Oxidative and Nitrosative Stresses, Cell Integrity, and Normal Manifestation of Virulence Factors in the Pathogenic Fungus Cryptococcus neoformans  . </a></p><div class="supp"><p class="details"><div>Model for <i>PKC1</i> in signal transduction of various stresses. Solid arrows represent functions in which <span class="highlight" style="background-color:">signaling</span> via the <i>PKC1</i> <span class="highlight" style="background-color:">pathway</span> and its components is required. Dotted arrows denote functions in which <i>PKC1</i> is important but <span class="highlight" style="background-color:">pathway</span> components are dispensable. Dashed arrows indicate that a specific gene and function are linked.</div></p></div><div class="aux"><div class="resc">Kimberly J. Gerik, et al. Eukaryot Cell. 2008 October;7(10):1685-1698.</div><p class="links"><a class="dblinks" href="/pubmed/18689526" ref="ordinalpos=4830&amp;ncbi_uid=1602605&amp;link_uid=18689526">Citation</a><a class="dblinks" href="/pmc/articles/PMC2568057/" ref="ordinalpos=4830&amp;ncbi_uid=1602605&amp;link_uid=PMC2568057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4831.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4401766/figure/pcbi.1004169.g001/" ref="ordinalpos=4831&amp;ncbi_uid=7446285&amp;link_uid=PMC4401766" href="/pmc/articles/PMC4401766/"><img src="/pmc/articles/instance/4401766/bin/pcbi.1004169.g001.gif" src-large="/pmc/articles/instance/4401766/bin/pcbi.1004169.g001.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4401766/" ref="ordinalpos=4831&amp;ncbi_uid=7446285&amp;link_uid=PMC4401766" image-link="/pmc/articles/PMC4401766/figure/pcbi.1004169.g001/" class="imagepopup">Fig 1. Insulin <span class="highlight" style="background-color:">signaling</span> in glioma..  From: Simulation Predicts IGFBP2-HIF1α Interaction Drives Glioblastoma Growth. </a></p><div class="supp"><p class="details"><div>(A) Microscope image of glioma spheroids grown in vitro. Inset illustrates detailed intracellular insulin <span class="highlight" style="background-color:">signaling</span>. HIF1α = hypoxia-inducible factor 1 α, IGFBP2 = Insulin like growth factor binding protein 2, IGFI = Insulin like growth factor 1, IGFIR = Insulin like growth factor receptor 1, VHL = von Hippel-Lindau complex, PHDs = prolyl hydroxylase domain proteins and Ub = Ubiquitinated. (B) Schematic of the simplified insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> used in the computational model.</div></p></div><div class="aux"><div class="resc">Ka Wai Lin, et al. PLoS Comput Biol. 2015 April;11(4):e1004169.</div><p class="links"><a class="dblinks" href="/pubmed/25884993" ref="ordinalpos=4831&amp;ncbi_uid=7446285&amp;link_uid=25884993">Citation</a><a class="dblinks" href="/pmc/articles/PMC4401766/" ref="ordinalpos=4831&amp;ncbi_uid=7446285&amp;link_uid=PMC4401766">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4832.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2696359/figure/F3/" ref="ordinalpos=4832&amp;ncbi_uid=1823170&amp;link_uid=PMC2696359" href="/pmc/articles/PMC2696359/"><img src="/pmc/articles/instance/2696359/bin/nihms54315f3.gif" src-large="/pmc/articles/instance/2696359/bin/nihms54315f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2696359/" ref="ordinalpos=4832&amp;ncbi_uid=1823170&amp;link_uid=PMC2696359" image-link="/pmc/articles/PMC2696359/figure/F3/" class="imagepopup">Figure 3. Diagram Showing How <span class="highlight" style="background-color:">Signaling</span> Amplitudes or Synergism can Affect Gene Induction.  From: TNF-Induced MAP Kinase Activation Oscillates in Time. </a></p><div class="supp"><p class="details"><div>(A) Schematic of how the amplitude in a signal can give rise to distinct gene expression profiles. Specifically at high signal amplitude, <span class="highlight" style="background-color:">signaling</span> molecules with a low affinity for say Ca<sup>2+</sup>, can become activated and result in a distinct genetic output. (B) Schematic of how two signal transduction pathways can coordinate to lead to the expression of select genes, with one <span class="highlight" style="background-color:">pathway</span> mediating chromatin modification (for example, p38) and the other <span class="highlight" style="background-color:">pathway</span> initiating transcription (for example, p65).</div></p></div><div class="aux"><div class="resc">Jameel Iqbal, et al. Biochem Biophys Res Commun. ;371(4):906-911.</div><p class="links"><a class="dblinks" href="/pubmed/18384751" ref="ordinalpos=4832&amp;ncbi_uid=1823170&amp;link_uid=18384751">Citation</a><a class="dblinks" href="/pmc/articles/PMC2696359/" ref="ordinalpos=4832&amp;ncbi_uid=1823170&amp;link_uid=PMC2696359">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4833.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4153512/figure/F3/" ref="ordinalpos=4833&amp;ncbi_uid=6596901&amp;link_uid=PMC4153512" href="/pmc/articles/PMC4153512/"><img src="/pmc/articles/instance/4153512/bin/NRR-9-474-g003.gif" src-large="/pmc/articles/instance/4153512/bin/NRR-9-474-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4153512/" ref="ordinalpos=4833&amp;ncbi_uid=6596901&amp;link_uid=PMC4153512" image-link="/pmc/articles/PMC4153512/figure/F3/" class="imagepopup">Figure 3.  From: Geniposide, the component of the Chinese herbal formula Tongluojiunao, protects amyloid-? peptide (1-42-mediated death of hippocampal neurons via the non-classical estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>The phosphatidylinositol 3-kinase (PI3K) and mitogen-activated protein kinase (MAPK) <span class="highlight" style="background-color:">signaling</span> pathways are partially attributed to the neuroprotective effect of <i>Tongluojiunao</i> (TLJN), including geniposide.<br />(A) The reduced release of LDH by TLJN is partially blocked by the PI3K inhibitor, LY294002, under Aβ<sub>1–42</sub>-exposed conditions. (B) The reduced release of LDH by geniposide (Gp) is completely inhibited by LY294002. (C) The neuroprotective effect of TLJN is dependent on the MAPK signal-ing <span class="highlight" style="background-color:">pathway</span>. (D) Gp protects cells <i>via</i> the MAPK estrogen <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Results are presented as a ratio of control cells (CTRL). All data are expressed as mean ± SD (with each experiment in at least triplicate) and were analyzed either by the one-way analysis of variance or the indepen-dent samples <i>t</i>-test. *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01, <i>vs.</i> Aβ<sub>1–42</sub> group; #<i>P</i> &lt; 0.05, ##<i>P</i> &lt; 0.01, <i>vs.</i> Aβ<sub>1–42</sub> + TLJN group; +<i>P</i> &lt; 0.01, <i>vs.</i> Aβ<sub>1–42</sub> + E<sub>2</sub> group; &amp;<i>P</i> &lt; 0.01, <i>vs.</i> Aβ<sub>1–42</sub> + Gp group. E<sub>2</sub>: 17-β estradiol; U0126: MAPK inhibitor; LDH: lactate dehydrogenase.</div></p></div><div class="aux"><div class="resc">Jiao Li, et al. Neural Regen Res. 2014 March 1;9(5):474-480.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4833&amp;ncbi_uid=6596901&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4153512/" ref="ordinalpos=4833&amp;ncbi_uid=6596901&amp;link_uid=PMC4153512">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4834.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2172896/figure/fig10/" ref="ordinalpos=4834&amp;ncbi_uid=1130646&amp;link_uid=PMC2172896" href="/pmc/articles/PMC2172896/"><img src="/pmc/articles/instance/2172896/bin/200210149f10.gif" src-large="/pmc/articles/instance/2172896/bin/200210149f10.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2172896/" ref="ordinalpos=4834&amp;ncbi_uid=1130646&amp;link_uid=PMC2172896" image-link="/pmc/articles/PMC2172896/figure/fig10/" class="imagepopup">Figure 10. From: Control of axon elongation via an SDF-1?/Rho/mDia <span class="highlight" style="background-color:">pathway</span> in cultured cerebellar granule neurons. </a></p><div class="supp"><p class="details"><div><b>Proposed model for the role of mDia in SDF-1</b>α<b>– dependent axon outgrowth in cultured cerebellar granule neurons.</b> Usually Rho and Rac antagonistically controls axon outgrowth. During the very early stages in the primary culture, ROCK action predominates to favor suppression of precocious outgrowth of axons from the cerebellar granule cells. However, as these neurons express a high amount of mDia1 gradual decline in ROCK activity facilitates expression of Rho-dependent mDia1 activity and a subsequent recruitment of a <span class="highlight" style="background-color:">signaling</span> complex, which in concert with the Rac-dependent <span class="highlight" style="background-color:">signaling</span> cascade may help the transition from inhibition to stimulation of axon outgrowth and elongation. Thus, the SDF-1α/Rho/mDia1 <span class="highlight" style="background-color:">pathway</span> may play a critical role in defining and modulating the balance between the Rho- and Rac-based <span class="highlight" style="background-color:">signaling</span> pathways during axon outgrowth.</div></p></div><div class="aux"><div class="resc">Yoshiki Arakawa, et al. J Cell Biol. 2003 April 28;161(2):381-391.</div><p class="links"><a class="dblinks" href="/pubmed/12707308" ref="ordinalpos=4834&amp;ncbi_uid=1130646&amp;link_uid=12707308">Citation</a><a class="dblinks" href="/pmc/articles/PMC2172896/" ref="ordinalpos=4834&amp;ncbi_uid=1130646&amp;link_uid=PMC2172896">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4835.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3848273/figure/fig5/" ref="ordinalpos=4835&amp;ncbi_uid=5594367&amp;link_uid=PMC3848273" href="/pmc/articles/PMC3848273/"><img src="/pmc/articles/instance/3848273/bin/OXIMED2013-680545.005.gif" src-large="/pmc/articles/instance/3848273/bin/OXIMED2013-680545.005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3848273/" ref="ordinalpos=4835&amp;ncbi_uid=5594367&amp;link_uid=PMC3848273" image-link="/pmc/articles/PMC3848273/figure/fig5/" class="imagepopup">Figure 5.  From: Tetrahydroxystilbene Glucoside Attenuates Neuroinflammation through the Inhibition of Microglia Activation. </a></p><div class="supp"><p class="details"><div>TSG suppressed LPS-induced NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. Western blot assay was performed to analyze NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation (a). The ratio of densitometry values of phosphorylated p65 (p-p65) and IKK (p-IKK) compared with total p65 and IKK, respectively, was analyzed and normalized to each control cultures (b). Results were expressed as a percentage of the control cultures and were the mean ± SEM from three independent experiments performed in triplicate. <sup>#</sup><i>P</i> &lt; 0.05 compared with the control cultures; *<i>P</i> &lt; 0.05 compared with LPS-treated cultures.</div></p></div><div class="aux"><div class="resc">Feng Zhang, et al. Oxid Med Cell Longev. 2013;2013:680545.</div><p class="links"><a class="dblinks" href="/pubmed/24349614" ref="ordinalpos=4835&amp;ncbi_uid=5594367&amp;link_uid=24349614">Citation</a><a class="dblinks" href="/pmc/articles/PMC3848273/" ref="ordinalpos=4835&amp;ncbi_uid=5594367&amp;link_uid=PMC3848273">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4836.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3377785/figure/F1/" ref="ordinalpos=4836&amp;ncbi_uid=4986884&amp;link_uid=PMC3377785" href="/pmc/articles/PMC3377785/"><img src="/pmc/articles/instance/3377785/bin/nihms352294f1.gif" src-large="/pmc/articles/instance/3377785/bin/nihms352294f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3377785/" ref="ordinalpos=4836&amp;ncbi_uid=4986884&amp;link_uid=PMC3377785" image-link="/pmc/articles/PMC3377785/figure/F1/" class="imagepopup">Figure 1.  From: Dishonorable Discharge: The Oncogenic Roles of Cleaved E-cadherin Fragments. </a></p><div class="supp"><p class="details"><div>Generation of E-cadherin fragments and their functions. A, cleavage of the full-length E-cadherin by α-secretase, γ-secretase, and caspase-3 generates the sE-cad and E-cad/CTF1, E-cad/CTF2, and E-cad/CTF3 fragments, respectively. B, oncogenic functions of the sE-cad and E-cad/CTF2 fragments. Generation of sE-cad by E-cadherin cleavage reduces the amount of full-length E-cadherin on the plasma membrane (1), disrupts existing adherens junctions (2), activates the expression of MMPs to augment ectodomain shedding (3), and activates EGFR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> by different mechanisms (4). Intracellular cleavage of E-cadherin activates Wnt/β-catenin <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> (5). CD, cytoplasmic domain; TM, transmembrane domain.</div></p></div><div class="aux"><div class="resc">Justin M. David, et al. Cancer Res. ;72(12):2917-2923.</div><p class="links"><a class="dblinks" href="/pubmed/22659456" ref="ordinalpos=4836&amp;ncbi_uid=4986884&amp;link_uid=22659456">Citation</a><a class="dblinks" href="/pmc/articles/PMC3377785/" ref="ordinalpos=4836&amp;ncbi_uid=4986884&amp;link_uid=PMC3377785">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4837.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2786169/figure/F4/" ref="ordinalpos=4837&amp;ncbi_uid=2190464&amp;link_uid=PMC2786169" href="/pmc/articles/PMC2786169/"><img src="/pmc/articles/instance/2786169/bin/nihms-159644-f0004.gif" src-large="/pmc/articles/instance/2786169/bin/nihms-159644-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2786169/" ref="ordinalpos=4837&amp;ncbi_uid=2190464&amp;link_uid=PMC2786169" image-link="/pmc/articles/PMC2786169/figure/F4/" class="imagepopup">Figure 4.  From: Exploring pathways from gene coexpression to network dynamics. </a></p><div class="supp"><p class="details"><div>The topology of leukemia-related BCR-ABL <span class="highlight" style="background-color:">pathway</span>. The arrows represent the directions of the causal relationships among genes. BCR are ABL are linked to the cytoplasm as a part of a large <span class="highlight" style="background-color:">signaling</span> complex with a variety of cellular substrates, related to the development of leukemia. The drug Gleevec is a selective BCR-ABL inhibitor in this <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Huai Li, et al. Methods Mol Biol. ;541:249-267.</div><p class="links"><a class="dblinks" href="/pubmed/19381544" ref="ordinalpos=4837&amp;ncbi_uid=2190464&amp;link_uid=19381544">Citation</a><a class="dblinks" href="/pmc/articles/PMC2786169/" ref="ordinalpos=4837&amp;ncbi_uid=2190464&amp;link_uid=PMC2786169">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4838.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3630382/figure/F10/" ref="ordinalpos=4838&amp;ncbi_uid=4822326&amp;link_uid=PMC3630382" href="/pmc/articles/PMC3630382/"><img src="/pmc/articles/instance/3630382/bin/gni-11-24-g010.gif" src-large="/pmc/articles/instance/3630382/bin/gni-11-24-g010.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3630382/" ref="ordinalpos=4838&amp;ncbi_uid=4822326&amp;link_uid=PMC3630382" image-link="/pmc/articles/PMC3630382/figure/F10/" class="imagepopup">Fig. 10.  From: Bioinformatics Interpretation of Exome Sequencing: Blood Cancer. </a></p><div class="supp"><p class="details"><div>"Cell Death and Survival, Cell Cycle, Cell-to-Cell <span class="highlight" style="background-color:">Signaling</span> and Interaction, Cellular Movement" Network. Data were analyzed through the use of Ingenity <span class="highlight" style="background-color:">Pathway</span> Analysis (IPA) (Ingenuity® Systems, http://www.ingenuity.com) [8].</div></p></div><div class="aux"><div class="resc">Jiwoong Kim, et al. Genomics Inform. 2013 March; 2013 June;11(1):24-33.</div><p class="links"><a class="dblinks" href="/pubmed/23613679" ref="ordinalpos=4838&amp;ncbi_uid=4822326&amp;link_uid=23613679">Citation</a><a class="dblinks" href="/pmc/articles/PMC3630382/" ref="ordinalpos=4838&amp;ncbi_uid=4822326&amp;link_uid=PMC3630382">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4839.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3603375/figure/fig4/" ref="ordinalpos=4839&amp;ncbi_uid=4740996&amp;link_uid=PMC3603375" href="/pmc/articles/PMC3603375/"><img src="/pmc/articles/instance/3603375/bin/ISRN.ONCOLOGY2013-924971.004.gif" src-large="/pmc/articles/instance/3603375/bin/ISRN.ONCOLOGY2013-924971.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3603375/" ref="ordinalpos=4839&amp;ncbi_uid=4740996&amp;link_uid=PMC3603375" image-link="/pmc/articles/PMC3603375/figure/fig4/" class="imagepopup">Figure 4.  From: Gene Expression Profile Analysis of T1 and T2 Breast Cancer Reveals Different Activation Pathways. </a></p><div class="supp"><p class="details"><div>Neurotrophin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is among the pathways downregulated in clinical T1 and upregulated in clinical T2. Red stars imply genes that are downregulated in clinical T1 and upregulated in T2.</div></p></div><div class="aux"><div class="resc">Margit L. H. Riis, et al. ISRN Oncol. 2013;2013:924971.</div><p class="links"><a class="dblinks" href="/pubmed/23533813" ref="ordinalpos=4839&amp;ncbi_uid=4740996&amp;link_uid=23533813">Citation</a><a class="dblinks" href="/pmc/articles/PMC3603375/" ref="ordinalpos=4839&amp;ncbi_uid=4740996&amp;link_uid=PMC3603375">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4840.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3229229/figure/f3-marinedrugs-09-02176/" ref="ordinalpos=4840&amp;ncbi_uid=3709055&amp;link_uid=PMC3229229" href="/pmc/articles/PMC3229229/"><img src="/pmc/articles/instance/3229229/bin/marinedrugs-09-02176f3.gif" src-large="/pmc/articles/instance/3229229/bin/marinedrugs-09-02176f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3229229/" ref="ordinalpos=4840&amp;ncbi_uid=3709055&amp;link_uid=PMC3229229" image-link="/pmc/articles/PMC3229229/figure/f3-marinedrugs-09-02176/" class="imagepopup">Figure 3.  From: Pectenotoxin-2 from Marine Sponges: A Potential Anti-Cancer Agent--A Review. </a></p><div class="supp"><p class="details"><div>Putative apoptosis mechanism induced by PTX-2 treatment. PTX-2 induces apoptosis of cancer cells via increasing ROS generation, inactivation of the NF-kB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and inhibition of telomerase activity.</div></p></div><div class="aux"><div class="resc">Gi-Young Kim, et al. Mar Drugs. 2011;9(11):2176-2187.</div><p class="links"><a class="dblinks" href="/pubmed/22163180" ref="ordinalpos=4840&amp;ncbi_uid=3709055&amp;link_uid=22163180">Citation</a><a class="dblinks" href="/pmc/articles/PMC3229229/" ref="ordinalpos=4840&amp;ncbi_uid=3709055&amp;link_uid=PMC3229229">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4841.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2745588/figure/F3/" ref="ordinalpos=4841&amp;ncbi_uid=2038990&amp;link_uid=PMC2745588" href="/pmc/articles/PMC2745588/"><img src="/pmc/articles/instance/2745588/bin/1471-2105-10-S8-S6-3.gif" src-large="/pmc/articles/instance/2745588/bin/1471-2105-10-S8-S6-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2745588/" ref="ordinalpos=4841&amp;ncbi_uid=2038990&amp;link_uid=PMC2745588" image-link="/pmc/articles/PMC2745588/figure/F3/" class="imagepopup">Figure 3.  From: From SNPs to pathways: integration of functional effect of sequence variations on models of cell signalling pathways. </a></p><div class="supp"><p class="details"><div><b>Cytoscape visualization of the "<span class="highlight" style="background-color:">Signaling</span> by EGFR" <span class="highlight" style="background-color:">pathway</span> in BioPAX format</b>. The nodes are coloured according to the kind of information they are annotated with (see colour mapping Figure 4).</div></p></div><div class="aux"><div class="resc">Anna Bauer-Mehren, et al. BMC Bioinformatics. 2009;10(Suppl 8):S6-S6.</div><p class="links"><a class="dblinks" href="/pubmed/19758470" ref="ordinalpos=4841&amp;ncbi_uid=2038990&amp;link_uid=19758470">Citation</a><a class="dblinks" href="/pmc/articles/PMC2745588/" ref="ordinalpos=4841&amp;ncbi_uid=2038990&amp;link_uid=PMC2745588">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4842.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC535921/figure/fig10/" ref="ordinalpos=4842&amp;ncbi_uid=421446&amp;link_uid=PMC535921" href="/pmc/articles/PMC535921/"><img src="/pmc/articles/instance/535921/bin/89262-11f10_1t.gif" src-large="/pmc/articles/instance/535921/bin/89262-11f10_1t.jpg" alt="Figure 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC535921/" ref="ordinalpos=4842&amp;ncbi_uid=421446&amp;link_uid=PMC535921" image-link="/pmc/articles/PMC535921/figure/fig10/" class="imagepopup">Figure 10. From: The induction of sexual development and virulence in the smut fungus Ustilago maydis depends on Crk1, a novel MAPK protein. </a></p><div class="supp"><p class="details"><div>Proposed model for <span class="highlight" style="background-color:">signaling</span> processes involving Crk1 and the MAPK <span class="highlight" style="background-color:">pathway</span>. See text for a detailed discussion.</div></p></div><div class="aux"><div class="resc">Elia Garrido, et al. Genes Dev. 2004 December 15;18(24):3117-3130.</div><p class="links"><a class="dblinks" href="/pubmed/15601825" ref="ordinalpos=4842&amp;ncbi_uid=421446&amp;link_uid=15601825">Citation</a><a class="dblinks" href="/pmc/articles/PMC535921/" ref="ordinalpos=4842&amp;ncbi_uid=421446&amp;link_uid=PMC535921">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4843.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4236726/figure/F2/" ref="ordinalpos=4843&amp;ncbi_uid=6854131&amp;link_uid=PMC4236726" href="/pmc/articles/PMC4236726/"><img src="/pmc/articles/instance/4236726/bin/s12918-014-0088-0-2.gif" src-large="/pmc/articles/instance/4236726/bin/s12918-014-0088-0-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4236726/" ref="ordinalpos=4843&amp;ncbi_uid=6854131&amp;link_uid=PMC4236726" image-link="/pmc/articles/PMC4236726/figure/F2/" class="imagepopup">Figure 2.  From: Network features suggest new hepatocellular carcinoma treatment strategies. </a></p><div class="supp"><p class="details"><div><b>Targeting the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A)</b> The MAPK <span class="highlight" style="background-color:">pathway</span> communicates with 90 other pathways from the cancer type A network, including crosstalk with Focal Adhesion. <b>(B)</b> MAPK and Focal Adhesion crosstalk is basically the outcome of the sub-network of all correlated genes included in at least one of these two pathways. <b>(C)</b> Although all of these genes function as part of the MAPK <span class="highlight" style="background-color:">pathway</span>, they may have different effects on MAPK connectivity, since each gene correlates with different genes and pathways. Measuring the impact of each gene by the number of the resulting significant edges could give us a way to predict the gene’s involvement and intensity related to MAPK connectivity. For instance, FGFR3 is included in 5 pathways and on average is connected with 15% genes per <span class="highlight" style="background-color:">pathway</span> edge. The deletion of FGFR3 changes the number of pathways that crosstalk with the MAPK <span class="highlight" style="background-color:">pathway</span>, from the initial 90 crosstalks to 87 crosstalks.</div></p></div><div class="aux"><div class="resc">Orit Lavi, et al. BMC Syst Biol. 2014;8:88-88.</div><p class="links"><a class="dblinks" href="/pubmed/25070212" ref="ordinalpos=4843&amp;ncbi_uid=6854131&amp;link_uid=25070212">Citation</a><a class="dblinks" href="/pmc/articles/PMC4236726/" ref="ordinalpos=4843&amp;ncbi_uid=6854131&amp;link_uid=PMC4236726">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4844.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2788953/figure/F4/" ref="ordinalpos=4844&amp;ncbi_uid=2217901&amp;link_uid=PMC2788953" href="/pmc/articles/PMC2788953/"><img src="/pmc/articles/instance/2788953/bin/nihms-100386-f0004.gif" src-large="/pmc/articles/instance/2788953/bin/nihms-100386-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2788953/" ref="ordinalpos=4844&amp;ncbi_uid=2217901&amp;link_uid=PMC2788953" image-link="/pmc/articles/PMC2788953/figure/F4/" class="imagepopup">Fig. 4. Autophagy is a downstream effector of TLR <span class="highlight" style="background-color:">signaling</span>.  From: Autophagy and Pattern Recognition Receptors in Innate Immunity. </a></p><div class="supp"><p class="details"><div>LPS induces autophagy in macrophages by activating the TLR4/TRIF <span class="highlight" style="background-color:">pathway</span>. Zymosan induces autophagy and additionally may induce LC3 translocation to the phagosome (directly or mediated by rapid fusion with LC3-positive autophagic organelles). TLR3 (or RIG-I/MDA5) may activate autophagy upon recognition of double-stranded RNA (dsRNA). TLR7 (and TLR8, not shown) induce autophagy. In a viral infection, viral RNA is recognized by TLR7 (endocytic ssRNA) or by RIG-I (short dsRNA or 3′-triphosphate ssRNA). In macrophages, TLR7 ligands induce autophagy, activating the TLR7/MyD88 <span class="highlight" style="background-color:">pathway</span>. In most cells, but not in pDCs, the viral ligands detected by RIG-I induce the IPS-1 <span class="highlight" style="background-color:">pathway</span> (IPS-1 is associated with mitochondria). As a potential negative feedback, Atg5–Atg12 conjugate inhibits this <span class="highlight" style="background-color:">pathway</span>. In pDCs, constitutive autophagy delivers TLR7 ligands from the cytosol to the compartments containing TLR7.</div></p></div><div class="aux"><div class="resc">Monica Delgado, et al. Immunol Rev. ;227(1):189-202.</div><p class="links"><a class="dblinks" href="/pubmed/19120485" ref="ordinalpos=4844&amp;ncbi_uid=2217901&amp;link_uid=19120485">Citation</a><a class="dblinks" href="/pmc/articles/PMC2788953/" ref="ordinalpos=4844&amp;ncbi_uid=2217901&amp;link_uid=PMC2788953">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4845.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4072676/figure/pone-0100702-g009/" ref="ordinalpos=4845&amp;ncbi_uid=6316048&amp;link_uid=PMC4072676" href="/pmc/articles/PMC4072676/"><img src="/pmc/articles/instance/4072676/bin/pone.0100702.g009.gif" src-large="/pmc/articles/instance/4072676/bin/pone.0100702.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4072676/" ref="ordinalpos=4845&amp;ncbi_uid=6316048&amp;link_uid=PMC4072676" image-link="/pmc/articles/PMC4072676/figure/pone-0100702-g009/" class="imagepopup">Figure 9. The transcription feedback caused by Wnt <span class="highlight" style="background-color:">signaling</span>..  From: Computational Modeling of the Interplay between Cadherin-Mediated Cell Adhesion and Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>The figure shows <b>a)</b> E-N switch of cadherins, and <b>b)</b> oscillation of destruction complex cycle. We have further revealed a decayed oscillation during cadherin switch <b>c)</b>.</div></p></div><div class="aux"><div class="resc">Jiawen Chen, et al. PLoS One. 2014;9(6):e100702.</div><p class="links"><a class="dblinks" href="/pubmed/24967587" ref="ordinalpos=4845&amp;ncbi_uid=6316048&amp;link_uid=24967587">Citation</a><a class="dblinks" href="/pmc/articles/PMC4072676/" ref="ordinalpos=4845&amp;ncbi_uid=6316048&amp;link_uid=PMC4072676">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4846.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3225948/figure/A004507F7/" ref="ordinalpos=4846&amp;ncbi_uid=5532998&amp;link_uid=PMC3225948" href="/pmc/articles/PMC3225948/"><img src="/pmc/articles/instance/3225948/bin/cshperspect-PRT-004507_F7.gif" src-large="/pmc/articles/instance/3225948/bin/cshperspect-PRT-004507_F7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3225948/" ref="ordinalpos=4846&amp;ncbi_uid=5532998&amp;link_uid=PMC3225948" image-link="/pmc/articles/PMC3225948/figure/A004507F7/" class="imagepopup">Figure 7. From: Chemical and Biological Approaches for Adapting Proteostasis to Ameliorate Protein Misfolding and Aggregation Diseases-Progress and Prognosis. </a></p><div class="supp"><p class="details"><div>The three arms of the unfolded protein response stress-responsive <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Proteostasis network capacity in the endoplasmic reticulum (ER) is matched to the level of newly synthesized proteins passing through the secretory <span class="highlight" style="background-color:">pathway</span> by the activation of intracellular <span class="highlight" style="background-color:">signaling</span> pathways collectively referred to as the unfolded protein response. The unfolded protein response responds to the accumulation of misfolded proteins within the lumen of the endoplasmic reticulum. Accumulation of unfolded proteins activates <span class="highlight" style="background-color:">signaling</span> pathways in the cytosol via the <i>trans</i>-membrane stress sensor proteins IRE1, ATF6, and PERK. Activation of the unfolded protein response results in translational attenuation of protein synthesis and transcriptional activation of genes regulated by the transcription factors XBP1s, ATF4, and ATF6 resulting from the three <span class="highlight" style="background-color:">signaling</span> arms of the unfolded protein response. (Figure adapted from Wiseman et al. [2010] and reprinted, with permission, from Elsevier © 2010.)</div></p></div><div class="aux"><div class="resc">Susan L. Lindquist, et al. Cold Spring Harb Perspect Biol. 2011 December;3(12):a004507.</div><p class="links"><a class="dblinks" href="/pubmed/21900404" ref="ordinalpos=4846&amp;ncbi_uid=5532998&amp;link_uid=21900404">Citation</a><a class="dblinks" href="/pmc/articles/PMC3225948/" ref="ordinalpos=4846&amp;ncbi_uid=5532998&amp;link_uid=PMC3225948">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4847.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3956514/figure/F4/" ref="ordinalpos=4847&amp;ncbi_uid=5940676&amp;link_uid=PMC3956514" href="/pmc/articles/PMC3956514/"><img src="/pmc/articles/instance/3956514/bin/viru-5-357-g4.gif" src-large="/pmc/articles/instance/3956514/bin/viru-5-357-g4.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3956514/" ref="ordinalpos=4847&amp;ncbi_uid=5940676&amp;link_uid=PMC3956514" image-link="/pmc/articles/PMC3956514/figure/F4/" class="imagepopup">Figure. From: Impact of the UPR on the virulence of the plant fungal pathogen A. brassicicola. </a></p><div class="supp"><p class="details"><div><b>Figure 4.</b> Model illustration of fungal <span class="highlight" style="background-color:">signaling</span> pathways regulated by camalexin. Stress responses mediated by UPR and MAPK <span class="highlight" style="background-color:">signaling</span> cascades are coordinated to buffer cells against the toxicity of camalexin. Parallel studies with <i>A. brassicicola</i> knockout mutants showed that camalexin induced AbHog1-dependent phosphorylation of AbSlt2 MAP kinase. 43  The UPR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is independently activated by production of spliced <i>AbHacA</i> mRNA (<i>AbHacA<sup>i</sup></i>). 8  This compensatory response may lead to cell wall strengthening and membrane remodeling.</div></p></div><div class="aux"><div class="resc">Thomas Guillemette, et al. Virulence. 2014 February 15;5(2):357-364.</div><p class="links"><a class="dblinks" href="/pubmed/24189567" ref="ordinalpos=4847&amp;ncbi_uid=5940676&amp;link_uid=24189567">Citation</a><a class="dblinks" href="/pmc/articles/PMC3956514/" ref="ordinalpos=4847&amp;ncbi_uid=5940676&amp;link_uid=PMC3956514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4848.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3877019/figure/pone-0083345-g004/" ref="ordinalpos=4848&amp;ncbi_uid=5674051&amp;link_uid=PMC3877019" href="/pmc/articles/PMC3877019/"><img src="/pmc/articles/instance/3877019/bin/pone.0083345.g004.gif" src-large="/pmc/articles/instance/3877019/bin/pone.0083345.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3877019/" ref="ordinalpos=4848&amp;ncbi_uid=5674051&amp;link_uid=PMC3877019" image-link="/pmc/articles/PMC3877019/figure/pone-0083345-g004/" class="imagepopup">Figure 4. Clathrin-mediated endocytosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> identified by the Ingenuity software in the “Mouse High CPE expression sub-dataset”..  From: Gene Expression and Functional Annotation of the Human and Mouse Choroid Plexus Epithelium. </a></p><div class="supp"><p class="details"><div>This is one of the canonical pathways that contained statistically significantly more genes than expected by chance in the group of genes that are highly expressed in the mouse CPE. Gray fields indicate their presence in the “Mouse High CPE expression sub-dataset (&gt;90<sup>th</sup> P)”; uncolored genes are added by the Ingenuity software to show the entire standard <span class="highlight" style="background-color:">pathway</span>. Solid lines between molecules indicate direct physical relationships between molecules (such as regulating and interacting protein domains). Abbreviations of gene names are according to standard abbreviations used in GenBank. As shown, the majority of genes of this <span class="highlight" style="background-color:">pathway</span> are highly expressed in the mouse CPE.</div></p></div><div class="aux"><div class="resc">Sarah F. Janssen, et al. PLoS One. 2013;8(12):e83345.</div><p class="links"><a class="dblinks" href="/pubmed/24391755" ref="ordinalpos=4848&amp;ncbi_uid=5674051&amp;link_uid=24391755">Citation</a><a class="dblinks" href="/pmc/articles/PMC3877019/" ref="ordinalpos=4848&amp;ncbi_uid=5674051&amp;link_uid=PMC3877019">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4849.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2830483/figure/pone-0009494-g007/" ref="ordinalpos=4849&amp;ncbi_uid=2308878&amp;link_uid=PMC2830483" href="/pmc/articles/PMC2830483/"><img src="/pmc/articles/instance/2830483/bin/pone.0009494.g007.gif" src-large="/pmc/articles/instance/2830483/bin/pone.0009494.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2830483/" ref="ordinalpos=4849&amp;ncbi_uid=2308878&amp;link_uid=PMC2830483" image-link="/pmc/articles/PMC2830483/figure/pone-0009494-g007/" class="imagepopup">Figure 7. Schematic illustrating relationship between WWP-1 and DAF-2 insulin-like signal network..  From: WWP-1 Is a Novel Modulator of the DAF-2 Insulin-Like <span class="highlight" style="background-color:">Signaling</span> Network Involved in Pore-Forming Toxin Cellular Defenses in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>In the canonical DAF-2 insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in <i>C. elegans</i>, DAF-2 regulates DAF-16 through the activation of PI3K, composed of AGE-1 and AAP-1 for the catalytic subunit and the regulatory subunit respectively. PI3K activates the activity of PDK-1, which eventually inhibits the nuclear translocation and transcriptional activity of DAF-16. Here we demonstrate that the attenuated DAF-2 signal can also regulate an additional DAF-16-independent signal arm that diverges form PDK-1 to WWP-1 to defense against Cry5B PFT.</div></p></div><div class="aux"><div class="resc">Chang-Shi Chen, et al. PLoS One. 2010;5(3):e9494.</div><p class="links"><a class="dblinks" href="/pubmed/20209166" ref="ordinalpos=4849&amp;ncbi_uid=2308878&amp;link_uid=20209166">Citation</a><a class="dblinks" href="/pmc/articles/PMC2830483/" ref="ordinalpos=4849&amp;ncbi_uid=2308878&amp;link_uid=PMC2830483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4850.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1431720/figure/F2/" ref="ordinalpos=4850&amp;ncbi_uid=750788&amp;link_uid=PMC1431720" href="/pmc/articles/PMC1431720/"><img src="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-2.gif" src-large="/pmc/articles/instance/1431720/bin/gb-2006-7-2-r11-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1431720/" ref="ordinalpos=4850&amp;ncbi_uid=750788&amp;link_uid=PMC1431720" image-link="/pmc/articles/PMC1431720/figure/F2/" class="imagepopup">Figure 2.  From: Identification of <span class="highlight" style="background-color:">signaling</span> components required for the prediction of cytokine release in RAW 264.7 macrophages. </a></p><div class="supp"><p class="details"><div>Two-way hierarchical clustering of the RAW 264.7 macrophage. <b>(a) </b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> responses and <b>(b) </b>cytokine release after single ligand stimulations. Average linkage clustering was performed using un-centered Pearson's correlation metrics on log-transformed and variance-normalized data. Data are averages over the different time points and across repeated experiments. Red = positive change; green = negative change.</div></p></div><div class="aux"><div class="resc">Sylvain Pradervand, et al. Genome Biol. 2006;7(2):R11-R11.</div><p class="links"><a class="dblinks" href="/pubmed/16507166" ref="ordinalpos=4850&amp;ncbi_uid=750788&amp;link_uid=16507166">Citation</a><a class="dblinks" href="/pmc/articles/PMC1431720/" ref="ordinalpos=4850&amp;ncbi_uid=750788&amp;link_uid=PMC1431720">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4851.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3361614/figure/F6/" ref="ordinalpos=4851&amp;ncbi_uid=4922058&amp;link_uid=PMC3361614" href="/pmc/articles/PMC3361614/"><img src="/pmc/articles/instance/3361614/bin/nihms367829f6.gif" src-large="/pmc/articles/instance/3361614/bin/nihms367829f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3361614/" ref="ordinalpos=4851&amp;ncbi_uid=4922058&amp;link_uid=PMC3361614" image-link="/pmc/articles/PMC3361614/figure/F6/" class="imagepopup">Figure 6. Model of multi-component <span class="highlight" style="background-color:">signaling</span> network for NO-induced biofilm formation as a protection mechanism against NO.  From: Nitric oxide modulates bacterial biofilm formation through a multi-component cyclic-di-GMP <span class="highlight" style="background-color:">signaling</span> network. </a></p><div class="supp"><p class="details"><div>(A) The complex multi-component <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is initiated by NO binding to the sensory H-NOX protein (HnoX), which then inhibits HnoK autophosphorylation. Phosphotransfer establishes a branching of the network to three response regulators. HnoB and HnoD form a feed-forward loop. Phosphorylation controls PDE activity of HnoB, which can be fine-tuned by allosteric control from HnoD. NO-controlled repression of the PDE activity leads to an increase in c-di-GMP levels, which serves as a <span class="highlight" style="background-color:">signaling</span> cue for cellular attachment into biofilms. (B) The NO signal switches the bacterial motility pattern from planktonic growth to increased attachment onto surfaces. The thick layers of cells provide a protective barrier against diffusion of reactive and damaging NO and may protect cells in the lower layers.</div></p></div><div class="aux"><div class="resc">Lars Plate, et al. Mol Cell. ;46(4):449-460.</div><p class="links"><a class="dblinks" href="/pubmed/22542454" ref="ordinalpos=4851&amp;ncbi_uid=4922058&amp;link_uid=22542454">Citation</a><a class="dblinks" href="/pmc/articles/PMC3361614/" ref="ordinalpos=4851&amp;ncbi_uid=4922058&amp;link_uid=PMC3361614">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4852.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3667790/figure/pone-0064088-g007/" ref="ordinalpos=4852&amp;ncbi_uid=4966021&amp;link_uid=PMC3667790" href="/pmc/articles/PMC3667790/"><img src="/pmc/articles/instance/3667790/bin/pone.0064088.g007.gif" src-large="/pmc/articles/instance/3667790/bin/pone.0064088.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3667790/" ref="ordinalpos=4852&amp;ncbi_uid=4966021&amp;link_uid=PMC3667790" image-link="/pmc/articles/PMC3667790/figure/pone-0064088-g007/" class="imagepopup">Figure 7. Hypothetical model illustrating the DSG3-plakoglobin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> modulating cell carcinogenesis in head and neck cancer..  From: DSG3 Facilitates Cancer Cell Growth and Invasion through the DSG3-Plakoglobin-TCF/LEF-Myc/Cyclin D1/MMP <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>(A) In cancer cells, DSG3 is overexpressed and captures more molecules of plakoglobins in the membrane. Loss of the antagonist function of plakoglobin results in more molecules of β-catenin entering the nucleus, which subsequently activates the expression of TCF/LEF downstream target genes c-myc, cyclin D1, and MMP-7 resulting in malignant cells. (B) Silencing DSG3 by shRNA treatment reverses the effects of DSG3, resulting in translocation of plakoglobin to the nucleus thus rendering the negative regulatory effect of plakoglobin on TCF/LEF transcription. (Pg: plakoglobin; B: β-catenin).</div></p></div><div class="aux"><div class="resc">Yin-Ju Chen, et al. PLoS One. 2013;8(5):e64088.</div><p class="links"><a class="dblinks" href="/pubmed/23737966" ref="ordinalpos=4852&amp;ncbi_uid=4966021&amp;link_uid=23737966">Citation</a><a class="dblinks" href="/pmc/articles/PMC3667790/" ref="ordinalpos=4852&amp;ncbi_uid=4966021&amp;link_uid=PMC3667790">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4853.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2553141/figure/fig2/" ref="ordinalpos=4853&amp;ncbi_uid=1509445&amp;link_uid=PMC2553141" href="/pmc/articles/PMC2553141/"><img src="/pmc/articles/instance/2553141/bin/BIO.135483.wc.f2.gif" src-large="/pmc/articles/instance/2553141/bin/BIO.135483.wc.f2.jpg" alt="FIGURE 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2553141/" ref="ordinalpos=4853&amp;ncbi_uid=1509445&amp;link_uid=PMC2553141" image-link="/pmc/articles/PMC2553141/figure/fig2/" class="imagepopup">FIGURE 2.  From: Monte Carlo Simulation of Cell Death <span class="highlight" style="background-color:">Signaling</span> Predicts Large Cell-to-Cell Stochastic Fluctuations through the Type 2 <span class="highlight" style="background-color:">Pathway</span> of Apoptosis. </a></p><div class="supp"><p class="details"><div>Time course of caspase-3 activation is shown for (<i>left</i>) the type 1 <span class="highlight" style="background-color:">pathway</span> and (<i>right</i>) the type 2 <span class="highlight" style="background-color:">pathway</span>. The solid black curves represent the average over many (∼100) cells. Different colors correspond to representative individual cells.</div></p></div><div class="aux"><div class="resc">Subhadip Raychaudhuri, et al. Biophys J. 2008 October 15;95(8):3559-3562.</div><p class="links"><a class="dblinks" href="/pubmed/18641073" ref="ordinalpos=4853&amp;ncbi_uid=1509445&amp;link_uid=18641073">Citation</a><a class="dblinks" href="/pmc/articles/PMC2553141/" ref="ordinalpos=4853&amp;ncbi_uid=1509445&amp;link_uid=PMC2553141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4854.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4331762/figure/fig02/" ref="ordinalpos=4854&amp;ncbi_uid=7192278&amp;link_uid=PMC4331762" href="/pmc/articles/PMC4331762/"><img src="/pmc/articles/instance/4331762/bin/acel0013-0850-f2.gif" src-large="/pmc/articles/instance/4331762/bin/acel0013-0850-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4331762/" ref="ordinalpos=4854&amp;ncbi_uid=7192278&amp;link_uid=PMC4331762" image-link="/pmc/articles/PMC4331762/figure/fig02/" class="imagepopup">Figure 2.  From: Hormone replacement therapy enhances IGF-1 <span class="highlight" style="background-color:">signaling</span> in skeletal muscle by diminishing miR-182 and miR-223 expressions: a study on postmenopausal monozygotic twin pairs. </a></p><div class="supp"><p class="details"><div>Putative common targets of the identified estrogen-responsive miRNAs and their putative role in the insulin/IGF1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Venn diagram showing the common target genes of the identified miRNAs. The complete lists of all putative common targets are reported in Table S1. (B) Putative role of the target genes in insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (C) Gene expression of potential common miR-targets with significant role in the insulin/IGF-1 <span class="highlight" style="background-color:">pathway</span> in skeletal muscle of postmenopausal women. Gene expression was measured by qPCR. (D) Western blot analyses of target proteins normalized against GAPDH. Protein levels were analyzed by densitometry and normalized against GAPDH expression. (E) Representative Western blot images from three twin pairs always presenting first the nonuser followed with her HRT-using co-twin. <i>P</i> values are from paired-samples <i>t</i>-test. *<i>P</i> ≤ 0.05.</div></p></div><div class="aux"><div class="resc">Fabiola Olivieri, et al. Aging Cell. 2014 October;13(5):850-861.</div><p class="links"><a class="dblinks" href="/pubmed/25040542" ref="ordinalpos=4854&amp;ncbi_uid=7192278&amp;link_uid=25040542">Citation</a><a class="dblinks" href="/pmc/articles/PMC4331762/" ref="ordinalpos=4854&amp;ncbi_uid=7192278&amp;link_uid=PMC4331762">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4855.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3546138/figure/F5/" ref="ordinalpos=4855&amp;ncbi_uid=5651213&amp;link_uid=PMC3546138" href="/pmc/articles/PMC3546138/"><img src="/pmc/articles/instance/3546138/bin/nihms418921f5.gif" src-large="/pmc/articles/instance/3546138/bin/nihms418921f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3546138/" ref="ordinalpos=4855&amp;ncbi_uid=5651213&amp;link_uid=PMC3546138" image-link="/pmc/articles/PMC3546138/figure/F5/" class="imagepopup">Figure 5. Silencing of SC4MOL suppresses EGFR <span class="highlight" style="background-color:">signaling</span> and sensitizes A431 xenografts to cetuximab.  From: Targeting C4-demethylating genes in the cholesterol <span class="highlight" style="background-color:">pathway</span> sensitizes cancer cells to EGFR inhibitors via increased EGFR degradation. </a></p><div class="supp"><p class="details"><div>(<i>A</i>) Silencing SC4MOL in SCC61 cells suppresses phosphorylation of EGFR downstream effectors following EGF stimulation for indicated time intervals. (<i>B</i>) Effects of a single dose of cetuximab on A431 xenografts at 72 hours. Numbers below bands are tubulin-normalized densities relative to vehicle-treated shControl grafts. (<i>C</i>) Summary of results quantified from (<i>B</i>); *, p&lt;0.05. <i>D</i> Tumor volumes of xenografts of shRNA-modified A431 cells treated with cetuximab (closed symbols) or vehicle (open symbols); <i>p</i>-value (1) 2.2*10<sup>−16</sup>. (<i>E</i>) SCC61 cells depleted of SC4MOL are sensitized to erlotinib-induced suppression of AKT-mTOR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. Graphs represent averaged results of 3 experiments; bars, SEM.</div></p></div><div class="aux"><div class="resc">Anna Sukhanova, et al. Cancer Discov. ;3(1):96-111.</div><p class="links"><a class="dblinks" href="/pubmed/23125191" ref="ordinalpos=4855&amp;ncbi_uid=5651213&amp;link_uid=23125191">Citation</a><a class="dblinks" href="/pmc/articles/PMC3546138/" ref="ordinalpos=4855&amp;ncbi_uid=5651213&amp;link_uid=PMC3546138">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4856.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3561798/figure/F6/" ref="ordinalpos=4856&amp;ncbi_uid=4591884&amp;link_uid=PMC3561798" href="/pmc/articles/PMC3561798/"><img src="/pmc/articles/instance/3561798/bin/JCI65642.f6.gif" src-large="/pmc/articles/instance/3561798/bin/JCI65642.f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3561798/" ref="ordinalpos=4856&amp;ncbi_uid=4591884&amp;link_uid=PMC3561798" image-link="/pmc/articles/PMC3561798/figure/F6/" class="imagepopup">Figure 6. Model of the molecular mechanism of coal tar therapy in AD..  From: Coal tar induces AHR-dependent skin barrier repair in atopic dermatitis. </a></p><div class="supp"><p class="details"><div>Th2 cytokines IL-4 and IL-13 activate the STAT6 <span class="highlight" style="background-color:">signaling</span> cascade, leading to downregulated expression of epidermal differentiation proteins, induction of the eosinophilic chemoattractant CCL26, and histopathological features such as spongiosis and apoptosis. Under normal conditions, PTPN1 dephosphorylates STAT6, resulting in a regulatory feedback loop. However, oxidative stress caused by Th2 cytokines inactivates PTPN1, leading to sustained STAT6 <span class="highlight" style="background-color:">signaling</span>. Coal tar–mediated activation of the AHR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leads to enhanced epidermal differentiation and possible improvement of epidermal barrier function, thereby attenuating allergen exposure and reducing inflammatory cues. Importantly, coal tar activates the AHR/NRF2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, enabling detoxification of reactive oxygen species. This may prevent the oxidative inactivation of PTPN1 and lead to decreased STAT6 <span class="highlight" style="background-color:">signaling</span>, normalization of skin barrier protein expression, and downregulation of CCL26 expression.</div></p></div><div class="aux"><div class="resc">Ellen H. van den Bogaard, et al. J Clin Invest. 2013 February 1;123(2):917-927.</div><p class="links"><a class="dblinks" href="/pubmed/23348739" ref="ordinalpos=4856&amp;ncbi_uid=4591884&amp;link_uid=23348739">Citation</a><a class="dblinks" href="/pmc/articles/PMC3561798/" ref="ordinalpos=4856&amp;ncbi_uid=4591884&amp;link_uid=PMC3561798">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4857.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3213765/figure/F20/" ref="ordinalpos=4857&amp;ncbi_uid=3353811&amp;link_uid=PMC3213765" href="/pmc/articles/PMC3213765/"><img src="/pmc/articles/instance/3213765/bin/nihms-334040-f0020.gif" src-large="/pmc/articles/instance/3213765/bin/nihms-334040-f0020.jpg" alt="Figure 8.20" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3213765/" ref="ordinalpos=4857&amp;ncbi_uid=3353811&amp;link_uid=PMC3213765" image-link="/pmc/articles/PMC3213765/figure/F20/" class="imagepopup">Figure 8.20.  From: Cerebellum: Development and Medulloblastoma. </a></p><div class="supp"><p class="details"><div>Shh and BMP <span class="highlight" style="background-color:">signaling</span>. Both <span class="highlight" style="background-color:">signaling</span> pathways antagonize each other to regulate proliferation of GCPs by regulating the levels of the bHLH transcription factor Atoh1/Math1. Whereas Shh induces proliferation, BMP inhibits proliferation and induces the irreversible differentiation of GCPs. BMP induces cell growth arrest by the rapid posttranscriptional downregulation of Atoh1/Math1 protein by the proteasome. In medulloblastomas, Atoh1/Math1 levels are high whereas the genes regulating the BMP <span class="highlight" style="background-color:">pathway</span> are downregulated.</div></p></div><div class="aux"><div class="resc">Martine F. Roussel, et al. Curr Top Dev Biol. ;94:235-282.</div><p class="links"><a class="dblinks" href="/pubmed/21295689" ref="ordinalpos=4857&amp;ncbi_uid=3353811&amp;link_uid=21295689">Citation</a><a class="dblinks" href="/pmc/articles/PMC3213765/" ref="ordinalpos=4857&amp;ncbi_uid=3353811&amp;link_uid=PMC3213765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4858.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2774718/figure/f8/" ref="ordinalpos=4858&amp;ncbi_uid=2465100&amp;link_uid=PMC2774718" href="/pmc/articles/PMC2774718/"><img src="/pmc/articles/instance/2932543/bin/fig-8.gif" src-large="/pmc/articles/instance/2932543/bin/fig-8.jpg" alt="FIG. 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2774718/" ref="ordinalpos=4858&amp;ncbi_uid=2465100&amp;link_uid=PMC2774718" image-link="/pmc/articles/PMC2774718/figure/f8/" class="imagepopup">FIG. 8. From: Molecular Mechanisms and Clinical Implications of Reversible Protein S-Glutathionylation. </a></p><div class="supp"><p class="details"><div><b>S-glutathionylation as a regulator of the JNK and MAPK <span class="highlight" style="background-color:">signaling</span> pathways in cancer.</b> This figure depicts several sites in the JNK <span class="highlight" style="background-color:">pathway</span> and in other <span class="highlight" style="background-color:">signaling</span> pathways in cancer cells where reversible S-glutathionylation may serve as a potential regulatory mechanism (<i>e.g</i>., Ras-SSG, MAPK/ERK kinase kinase 1 (MEKK1)-SSG, c-Jun-SSG, glutathione S-transferase (GST<sub>pi</sub>-SSG).</div></p></div><div class="aux"><div class="resc">John J. Mieyal, et al. Antioxid Redox Signal. 2008 November;10(11):1941-1988.</div><p class="links"><a class="dblinks" href="/pubmed/18774901" ref="ordinalpos=4858&amp;ncbi_uid=2465100&amp;link_uid=18774901">Citation</a><a class="dblinks" href="/pmc/articles/PMC2774718/" ref="ordinalpos=4858&amp;ncbi_uid=2465100&amp;link_uid=PMC2774718">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4859.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775773/figure/pone-0074322-g004/" ref="ordinalpos=4859&amp;ncbi_uid=5330273&amp;link_uid=PMC3775773" href="/pmc/articles/PMC3775773/"><img src="/pmc/articles/instance/3775773/bin/pone.0074322.g004.gif" src-large="/pmc/articles/instance/3775773/bin/pone.0074322.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775773/" ref="ordinalpos=4859&amp;ncbi_uid=5330273&amp;link_uid=PMC3775773" image-link="/pmc/articles/PMC3775773/figure/pone-0074322-g004/" class="imagepopup">Figure 4. NsPEF inactivates cancer cells metastasis and invasion by inhibiting Wnt/β-Catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to down-regulate expressions of VEGF and MMPs family proteins in vitro..  From: Nanosecond Pulsed Electric Field Inhibits Cancer Growth Followed by Alteration in Expressions of NF-?B and Wnt/?-Catenin <span class="highlight" style="background-color:">Signaling</span> Molecules. </a></p><div class="supp"><p class="details"><div>(A) Migration ability of cancer cells exposed to nsPEF was tested by trans-well assay. The migrated cells exposed to nsPEF were stained purple by 0.1% crystal violet solution, observed under light microscope for 40 or 400 magnifications, and counted for statistical analysis. Original magnification, 40× &amp; 400×. ***p&lt;0.001. (B) Invasion ability of cancer cells exposed to nsPEF was detected using matrigel invasion assay. After nsPEF treatment, the cells that possessed invasion ability penetrated through the matrigel, were stained purple by 0.1% crystal violet solution, and were counted for statistical analysis. Original magnification, 40× &amp; 400×. ***p&lt;0.001. (C) Protein expressions of Wnt/β-Catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> including hDPR1, β-Catenin and c-Myc in cancer cells after exposure to nsPEF with different intensities were detected by Western-blot assay. (D) Protein expressions of MMPs family and VEGF in cancer cells after exposure to nsPEF with different intensities were detected by Western-blot assay. Relative intensity of each protein was analyzed by Photoshop CS4 software. *p&lt;0.05, **p&lt;0.01, ***p&lt;0.001.</div></p></div><div class="aux"><div class="resc">Zhigang Ren, et al. PLoS One. 2013;8(9):e74322.</div><p class="links"><a class="dblinks" href="/pubmed/24069295" ref="ordinalpos=4859&amp;ncbi_uid=5330273&amp;link_uid=24069295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775773/" ref="ordinalpos=4859&amp;ncbi_uid=5330273&amp;link_uid=PMC3775773">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4860.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3656973/figure/F7/" ref="ordinalpos=4860&amp;ncbi_uid=6076991&amp;link_uid=PMC3656973" href="/pmc/articles/PMC3656973/"><img src="/pmc/articles/instance/3656973/bin/nihms442961f7.gif" src-large="/pmc/articles/instance/3656973/bin/nihms442961f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3656973/" ref="ordinalpos=4860&amp;ncbi_uid=6076991&amp;link_uid=PMC3656973" image-link="/pmc/articles/PMC3656973/figure/F7/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Signaling</span> schematic illustrating the effects of JSI-124 treatment in GBM cells.  From: Activation of the NF-?B <span class="highlight" style="background-color:">pathway</span> by the STAT3 inhibitor JSI-124 in human glioblastoma cells. </a></p><div class="supp"><p class="details"><div>JSI-124 treatment activates the NF-κB <span class="highlight" style="background-color:">pathway</span> by phosphorylation (P) of NF-κB p65 protein, subsequent nuclear localization and IκBα degradation (Ub, ubiquitin). Once in the nucleus, phosphorylated NF-κB dimers bind to the promoters of <i>IL-6</i>, <i>IL-8</i> and <i>SOCS3</i> and begin gene transcription. IL-6 and IL-8 mRNA are translated into protein and secreted into the medium. SOCS3 mRNA is also translated into protein and acts as a negative regulator of JAK/STAT3, preventing STAT3 activation.</div></p></div><div class="aux"><div class="resc">Braden C. McFarland, et al. Mol Cancer Res. 2013 May;11(5):494-505.</div><p class="links"><a class="dblinks" href="/pubmed/23386688" ref="ordinalpos=4860&amp;ncbi_uid=6076991&amp;link_uid=23386688">Citation</a><a class="dblinks" href="/pmc/articles/PMC3656973/" ref="ordinalpos=4860&amp;ncbi_uid=6076991&amp;link_uid=PMC3656973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4861.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3402309/figure/F5/" ref="ordinalpos=4861&amp;ncbi_uid=5140632&amp;link_uid=PMC3402309" href="/pmc/articles/PMC3402309/"><img src="/pmc/articles/instance/3402309/bin/1959fig5.gif" src-large="/pmc/articles/instance/3402309/bin/1959fig5.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3402309/" ref="ordinalpos=4861&amp;ncbi_uid=5140632&amp;link_uid=PMC3402309" image-link="/pmc/articles/PMC3402309/figure/F5/" class="imagepopup">FIG. 5. From: Nesfatin-1 Action in the Brain Increases Insulin Sensitivity Through Akt/AMPK/TORC2 <span class="highlight" style="background-color:">Pathway</span> in Diet-Induced Insulin Resistance. </a></p><div class="supp"><p class="details"><div>Effect of ICV nesfatin-1 on liver insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Four treatment groups were studied: NCA, NCN, HFA, and HFN. Central nesfatin-1 administration markedly increased liver InsR (<i>A</i>), IRS-1 (<i>B</i>), AMPK (<i>C</i>), and Akt (<i>D</i>) phosphorylation in both SD- and HFD-fed rats. Ratio of phosphorylated vs. total protein was calculated and is shown as means ± SE (<i>n</i> = 5). *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01 vs. ICV ACF-infused rats (NCA), #<i>P</i> &lt; 0.05, ##<i>P</i> &lt; 0.01 vs. HFD-fed rats (HFA).</div></p></div><div class="aux"><div class="resc">Mengliu Yang, et al. Diabetes. 2012 August;61(8):1959-1968.</div><p class="links"><a class="dblinks" href="/pubmed/22688332" ref="ordinalpos=4861&amp;ncbi_uid=5140632&amp;link_uid=22688332">Citation</a><a class="dblinks" href="/pmc/articles/PMC3402309/" ref="ordinalpos=4861&amp;ncbi_uid=5140632&amp;link_uid=PMC3402309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4862.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3484431/figure/F9/" ref="ordinalpos=4862&amp;ncbi_uid=4490686&amp;link_uid=PMC3484431" href="/pmc/articles/PMC3484431/"><img src="/pmc/articles/instance/3484431/bin/JCI59348.f9.gif" src-large="/pmc/articles/instance/3484431/bin/JCI59348.f9.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3484431/" ref="ordinalpos=4862&amp;ncbi_uid=4490686&amp;link_uid=PMC3484431" image-link="/pmc/articles/PMC3484431/figure/F9/" class="imagepopup">Figure 9. Summary diagram shows how recombinant PN1 added in Matrigel (a) can influence Hh proteins in both autocrine and paracrine scenarios..  From: Protease nexin 1 inhibits hedgehog <span class="highlight" style="background-color:">signaling</span> in prostate adenocarcinoma. </a></p><div class="supp"><p class="details"><div>PN1 disrupts reduces SHH expression, leading to reduction of downstream Hh effectors and tumor growth. Treatment with Hh inhibitor (b) GDC-0449 blocks another avenue of Hh <span class="highlight" style="background-color:">signaling</span> leading to inhibition of SMO. The combined effect slows the growth of prostate metastatic cells, potentially through changes in angiogenesis. The PN1 arm of the <span class="highlight" style="background-color:">pathway</span> can be inhibited by MMP9 activity.</div></p></div><div class="aux"><div class="resc">Chad M. McKee, et al. J Clin Invest. 2012 November 1;122(11):4025-4036.</div><p class="links"><a class="dblinks" href="/pubmed/23041623" ref="ordinalpos=4862&amp;ncbi_uid=4490686&amp;link_uid=23041623">Citation</a><a class="dblinks" href="/pmc/articles/PMC3484431/" ref="ordinalpos=4862&amp;ncbi_uid=4490686&amp;link_uid=PMC3484431">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4863.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2859609/figure/F1/" ref="ordinalpos=4863&amp;ncbi_uid=2621751&amp;link_uid=PMC2859609" href="/pmc/articles/PMC2859609/"><img src="/pmc/articles/instance/2859609/bin/1128-2460.27.025.fig1.gif" src-large="/pmc/articles/instance/2859609/bin/1128-2460.27.025.fig1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2859609/" ref="ordinalpos=4863&amp;ncbi_uid=2621751&amp;link_uid=PMC2859609" image-link="/pmc/articles/PMC2859609/figure/F1/" class="imagepopup">Figure 1.  From: Dok-7/MuSK <span class="highlight" style="background-color:">signaling</span> and a congenital myasthenic syndrome. </a></p><div class="supp"><p class="details"><div>A greatly simplified model of the Dok-7/MuSK <span class="highlight" style="background-color:">pathway</span> for postsynaptic specialization of the mammalian NMJ. Dok-7 can activate MuSK and induce Rapsyn-dependent AChR clustering even in the absence of neural Agrin; however MuSK requires both Dok-7 and Agrin to form functional NMJs in vivo. Many of the molecular mechanisms underlying these <span class="highlight" style="background-color:">signaling</span> events have yet to be determined. AChR, acetylcholine receptor; PH, pleckstrin homology; PTB, phosphotyrosine binding; Y, tyrosine residue in each SH2 target motif of Dok-7.</div></p></div><div class="aux"><div class="resc">Y Yamanashi, et al. Acta Myol. 2008 July;27(1):25-29.</div><p class="links"><a class="dblinks" href="/pubmed/19108574" ref="ordinalpos=4863&amp;ncbi_uid=2621751&amp;link_uid=19108574">Citation</a><a class="dblinks" href="/pmc/articles/PMC2859609/" ref="ordinalpos=4863&amp;ncbi_uid=2621751&amp;link_uid=PMC2859609">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4864.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1409881/figure/f1-15/" ref="ordinalpos=4864&amp;ncbi_uid=704181&amp;link_uid=PMC1409881" href="/pmc/articles/PMC1409881/"><img src="/pmc/articles/instance/1409881/bin/15FF1.gif" src-large="/pmc/articles/instance/1409881/bin/15FF1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1409881/" ref="ordinalpos=4864&amp;ncbi_uid=704181&amp;link_uid=PMC1409881" image-link="/pmc/articles/PMC1409881/figure/f1-15/" class="imagepopup">Figure. From: Transforming Growth Factor-? Promotes Pro-fibrotic Behavior by Serosal Fibroblasts via PKC and ERK1/2 Mitogen Activated Protein Kinase Cell <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>FIGURE 1.</b> TGF-β cell <span class="highlight" style="background-color:">signaling</span> via Smad proteins and ERK 1/2 MAP kinase pathways. The ERK 1/2 <span class="highlight" style="background-color:">pathway</span> is activated by phosphorylation in a PKC-dependent or -independent manner and signals to the nucleus where gene transcription is regulated. Go6850 is a broad spectrum PKC inhibitor, PD98059 inhibits MEK 1, while Uo126 inhibits both MEK 1 and 2 subtypes.</div></p></div><div class="aux"><div class="resc">Jurgen J. W. Mulsow, et al. Ann Surg. 2005 December;242(6):880-889.</div><p class="links"><a class="dblinks" href="/pubmed/16327498" ref="ordinalpos=4864&amp;ncbi_uid=704181&amp;link_uid=16327498">Citation</a><a class="dblinks" href="/pmc/articles/PMC1409881/" ref="ordinalpos=4864&amp;ncbi_uid=704181&amp;link_uid=PMC1409881">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4865.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4160345/figure/fig7/" ref="ordinalpos=4865&amp;ncbi_uid=6610137&amp;link_uid=PMC4160345" href="/pmc/articles/PMC4160345/"><img src="/pmc/articles/instance/4160345/bin/gr7.gif" src-large="/pmc/articles/instance/4160345/bin/gr7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4160345/" ref="ordinalpos=4865&amp;ncbi_uid=6610137&amp;link_uid=PMC4160345" image-link="/pmc/articles/PMC4160345/figure/fig7/" class="imagepopup">Figure 7.  From: Differential Apicobasal VEGF <span class="highlight" style="background-color:">Signaling</span> at Vascular Blood-Neural Barriers. </a></p><div class="supp"><p class="details"><div>Model of VEGF Action on MVECs of Blood-Neural Barriers<br />The majority of VEGFR1 and VEGFR2 is found on the apical (luminal) and basal (abluminal) side, respectively, of brain or retinal MVECs. Based on computer simulation (Mac Gabhann and Popel, 2007), such a receptor distribution is predicted to result primarily in VEGFR1 and VEGFR2 homodimers at the apical and basal surface, respectively. Residual apical VEGFR2 or basal VEGFR1 would not be found in homodimers but rather in heterodimers with the more abundant receptor. <span class="highlight" style="background-color:">Signaling</span> from VEGFR1, mainly triggered by circulating VEGF, activates a PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, which plays a role in endothelial cytoprotection. <span class="highlight" style="background-color:">Signaling</span> from VEGFR2, triggered by tissue-borne VEGF, activates p38 and mediates paracellular permeability, hallmarked by macromolecular flux, electrical conductance, and a loss of Cldn5.</div></p></div><div class="aux"><div class="resc">Natalie Hudson, et al. Dev Cell. 2014 September 8;30(5):541-552.</div><p class="links"><a class="dblinks" href="/pubmed/25175707" ref="ordinalpos=4865&amp;ncbi_uid=6610137&amp;link_uid=25175707">Citation</a><a class="dblinks" href="/pmc/articles/PMC4160345/" ref="ordinalpos=4865&amp;ncbi_uid=6610137&amp;link_uid=PMC4160345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4866.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2911501/figure/F2/" ref="ordinalpos=4866&amp;ncbi_uid=3124106&amp;link_uid=PMC2911501" href="/pmc/articles/PMC2911501/"><img src="/pmc/articles/instance/2911501/bin/nihms194310f2.gif" src-large="/pmc/articles/instance/2911501/bin/nihms194310f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2911501/" ref="ordinalpos=4866&amp;ncbi_uid=3124106&amp;link_uid=PMC2911501" image-link="/pmc/articles/PMC2911501/figure/F2/" class="imagepopup">Figure 2.  From: Neuronal intrinsic barriers for axon regeneration in the adult CNS. </a></p><div class="supp"><p class="details"><div>gp130-dependent cytokines promote axon regeneration and SOCS3 act as a critical negative regulator of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Injury-induced cytokines act on their receptor complexes with a shared component gp130 and activate the JAK-STAT cascade in both PNS and CNS neurons. Phosphorylated STAT-3 is translocated to the nucleus and initiate gene expression for axon regeneration. However, in the adult CNS, the activation of this <span class="highlight" style="background-color:">pathway</span> leads to the up-regulation of SOCS3 which will inhibit JAK2 and STAT3 and in turn inhibit this <span class="highlight" style="background-color:">pathway</span>. Thus, in SOCS3 deleted RGCs, both endogenous and exogenous cytokines such as CNTF promote significant axon regeneration [ 50**].</div></p></div><div class="aux"><div class="resc">Fang Sun, et al. Curr Opin Neurobiol. ;20(4):510-518.</div><p class="links"><a class="dblinks" href="/pubmed/20418094" ref="ordinalpos=4866&amp;ncbi_uid=3124106&amp;link_uid=20418094">Citation</a><a class="dblinks" href="/pmc/articles/PMC2911501/" ref="ordinalpos=4866&amp;ncbi_uid=3124106&amp;link_uid=PMC2911501">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4867.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3409471/figure/F2/" ref="ordinalpos=4867&amp;ncbi_uid=5141898&amp;link_uid=PMC3409471" href="/pmc/articles/PMC3409471/"><img src="/pmc/articles/instance/3409471/bin/nihms390398f2.gif" src-large="/pmc/articles/instance/3409471/bin/nihms390398f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3409471/" ref="ordinalpos=4867&amp;ncbi_uid=5141898&amp;link_uid=PMC3409471" image-link="/pmc/articles/PMC3409471/figure/F2/" class="imagepopup">Figure 2. <span class="highlight" style="background-color:">Signaling</span> Pathways Altered in Hepatic Cancer Stem Cells.  From: Clinical Implications of Cancer Stem Cell Biology in Hepatocellular Carcinoma. </a></p><div class="supp"><p class="details"><div>Wnt/β-catenin, AKT, TGF-β and IL-6/STAT3 <span class="highlight" style="background-color:">signaling</span> pathways are deregulated in hepatic CSCs. Activation of the Wnt <span class="highlight" style="background-color:">pathway</span> results in β-catenin accumulation in the cytosol and translocation into the nucleus, where β-catenin transcriptional activates the expression of its targets, such as cyclinD1, EpCAM, and miR-181. AKT is activated by two phosphorylation sites Thr308 and Ser473. Phosphorylation of Thr308 is promoted by PI3K and suppressed by PTEN. Activated AKT protect hepatic CSCs from apoptosis through the induction of BCL-2 and ABCG2 under the pressure of chemotherapy. Activated AKT could also interact with HIF-1α to induce VEGF and PDGF-BB expression for regulating the homeostasis and drug resistance of hepatic CSCs. Impaired TGF TGF-β <span class="highlight" style="background-color:">signaling</span> together with activation of IL-6/STAT3 <span class="highlight" style="background-color:">pathway</span> is important to regulate the differentiation and chemoresistance of EpCAM<sup>+</sup> hepatic CSCs. Studies have shown that these pathways are interconnected, which may further promote stemness and carcinogenesis in HCC.</div></p></div><div class="aux"><div class="resc">Junfang Ji, et al. Semin Oncol. ;39(4):461-472.</div><p class="links"><a class="dblinks" href="/pubmed/22846863" ref="ordinalpos=4867&amp;ncbi_uid=5141898&amp;link_uid=22846863">Citation</a><a class="dblinks" href="/pmc/articles/PMC3409471/" ref="ordinalpos=4867&amp;ncbi_uid=5141898&amp;link_uid=PMC3409471">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4868.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2790385/figure/F4/" ref="ordinalpos=4868&amp;ncbi_uid=2230383&amp;link_uid=PMC2790385" href="/pmc/articles/PMC2790385/"><img src="/pmc/articles/instance/2790385/bin/nihms-113822-f0004.gif" src-large="/pmc/articles/instance/2790385/bin/nihms-113822-f0004.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC2790385/figure/F4/" ref="ordinalpos=4868&amp;ncbi_uid=2230383&amp;link_uid=PMC2790385" href="/pmc/articles/PMC2790385/"><img src="/pmc/articles/instance/2790385/bin/nihms-113822-f0005.gif" src-large="/pmc/articles/instance/2790385/bin/nihms-113822-f0005.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2790385/" ref="ordinalpos=4868&amp;ncbi_uid=2230383&amp;link_uid=PMC2790385" image-link="/pmc/articles/PMC2790385/figure/F4/" class="imagepopup">Fig. 4.  From: Calcineurin/NFAT <span class="highlight" style="background-color:">Signaling</span> Is Required for Neuregulin-Regulated Schwann Cell Differentiation. </a></p><div class="supp"><p class="details"><div>Synergistic effects of NFATc4 and Sox10 on myelin gene expression. (<b>A</b>) NFATc4 and Sox10 synergistically activate the <i>Krox20</i> MSE and <i>P</i><sub><i>0</i></sub> promoter. DRG cocultures were transfected with luciferase reporter constructs containing the <i>Krox20</i> MSE, <i>P</i><sub><i>0</i></sub> promoter or <i>Krox20</i> promoter. Expression constructs encoding constitutively active calcineurin A (CnA*), NFATc4, and Sox family members were co-transfected. Luciferase activities were measured. Fold activation is relative to reporter baseline activities (<i>n</i> = 6, error bars, +SEM, ***<i>P</i> &lt; 0.001; <i>t</i> test). (<b>B</b>) Model of myelination process regulated by calcineurin/NFAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Solid arrows, calcineurin/NFAT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; dotted arrows, pathways defined by previous literature.</div></p></div><div class="aux"><div class="resc">Shih-Chu Kao, et al. Science. ;323(5914):651-654.</div><p class="links"><a class="dblinks" href="/pubmed/19179536" ref="ordinalpos=4868&amp;ncbi_uid=2230383&amp;link_uid=19179536">Citation</a><a class="dblinks" href="/pmc/articles/PMC2790385/" ref="ordinalpos=4868&amp;ncbi_uid=2230383&amp;link_uid=PMC2790385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4869.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4202127/figure/F3/" ref="ordinalpos=4869&amamp;ncbi_uid=6744336&amp;link_uid=PMC4202127" href="/pmc/articles/PMC4202127/"><img src="/pmc/articles/instance/4202127/bin/oncotarget-05-7342-g003.gif" src-large="/pmc/articles/instance/4202127/bin/oncotarget-05-7342-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4202127/" ref="ordinalpos=4869&amp;ncbi_uid=6744336&amp;link_uid=PMC4202127" image-link="/pmc/articles/PMC4202127/figure/F3/" class="imagepopup">Figure 3. Decreased pERK <span class="highlight" style="background-color:">signaling</span> in G-CIMP+ glioblastomas.  From: Epigenetic suppression of EGFR <span class="highlight" style="background-color:">signaling</span> in G-CIMP+ glioblastomas. </a></p><div class="supp"><p class="details"><div>(A) G-CIMP status of an independent collection of 25 glioblastoma specimens. These specimens were transcriptomally profiled and classified into G-CIMP+ and G-CIMP- using the PAM classifier described above. 6 G-CIMP+ and 19 G-CIMP- glioblastomas were identified. (B) Representative image of Chromatogram profile after direct DNA sequencing showing <i>IDH1</i> mutations in G-CIMP+ glioblastoma samples. All specimens predicted by PAM to be G-CIMP+ harbored the <i>IDH1-R132H</i> mutation. (C) Decreased pERK level in G-CIMP+ glioblastoma specimens. Ras/RTK <span class="highlight" style="background-color:">signaling</span> was analyzed by Western blotting using an antibody against pERK, an established biomarker for Ras/RTK <span class="highlight" style="background-color:">pathway</span> activity. (D) Levels of pERK, ERK and actin in (C) were analyzed using Image J software (Bethesda, MD). The level of pERK was normalized (see Methods) and plotted as a function of G-CIMP status.</div></p></div><div class="aux"><div class="resc">Jie Li, et al. Oncotarget. 2014 September;5(17):7342-7356.</div><p class="links"><a class="dblinks" href="/pubmed/25277177" ref="ordinalpos=4869&amp;ncbi_uid=6744336&amp;link_uid=25277177">Citation</a><a class="dblinks" href="/pmc/articles/PMC4202127/" ref="ordinalpos=4869&amp;ncbi_uid=6744336&amp;link_uid=PMC4202127">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4870.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3417062/figure/F4/" ref="ordinalpos=4870&amp;ncbi_uid=4172722&amp;link_uid=PMC3417062" href="/pmc/articles/PMC3417062/"><img src="/pmc/articles/instance/3417062/bin/nihms-396135-f0004.gif" src-large="/pmc/articles/instance/3417062/bin/nihms-396135-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3417062/" ref="ordinalpos=4870&amp;ncbi_uid=4172722&amp;link_uid=PMC3417062" image-link="/pmc/articles/PMC3417062/figure/F4/" class="imagepopup">Fig. 4.  From: PI3K/AKT <span class="highlight" style="background-color:">pathway</span> alterations are associated with clinically aggressive and histologically anaplastic subsets of pilocytic astrocytoma. </a></p><div class="supp"><p class="details"><div>AKT <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> is increased in histologically anaplastic PA (<i>APA</i>) when compared with conventional and clinically aggressive PA. Tissue microarray sections stained with immunohistochemistry using total protein and phospho-specific antibodies (<b>a</b>). Semiquantification results corresponding to the immunohistochemistry tests are illustrated on the right (<b>b</b>) (<i>white bars</i> conventional PA; <i>gray bars</i> clinically aggressive PA; <i>black bars</i> APA(B)(Wilcoxon rank sum, <i>error bars</i> SEM)</div></p></div><div class="aux"><div class="resc">Erika F. Rodriguez, et al. Acta Neuropathol. ;121(3):407-420.</div><p class="links"><a class="dblinks" href="/pubmed/21113787" ref="ordinalpos=4870&amp;ncbi_uid=4172722&amp;link_uid=21113787">Citation</a><a class="dblinks" href="/pmc/articles/PMC3417062/" ref="ordinalpos=4870&amp;ncbi_uid=4172722&amp;link_uid=PMC3417062">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4871.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1609106/figure/F2/" ref="ordinalpos=4871&amp;ncbi_uid=883204&amp;link_uid=PMC1609106" href="/pmc/articles/PMC1609106/"><img src="/pmc/articles/instance/1609106/bin/1471-213X-6-45-2.gif" src-large="/pmc/articles/instance/1609106/bin/1471-213X-6-45-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1609106/" ref="ordinalpos=4871&amp;ncbi_uid=883204&amp;link_uid=PMC1609106" image-link="/pmc/articles/PMC1609106/figure/F2/" class="imagepopup">Figure 2.  From: Activated AKT/PKB <span class="highlight" style="background-color:">signaling</span> in C. elegans uncouples temporally distinct outputs of DAF-2/insulin-like <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div><b>RNAi against components of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> reverts <i>age-1(mg109) </i>dauer arrest suppression by <i>akt-1(mg247) </i>and <i>pdk-1(mg261) </i>mutant alleles</b>. Animals were fed bacteria containing either control vector, L4440 or vectors producing gene-specific dsRNA. Numbers on the side of the bars represent the total number of animals examined. The bars show the percentage of animals that developed into L4 larvae or adults (grey) and those that arrested as dauers (black). RNAi experiments were diluted with control L4440 RNAi to match the mixed <i>akt-1 </i>and <i>akt-2 </i>RNAi dose. <i>akt-1, akt-2 </i>and <i>pdk-1 </i>RNAi experiments were conducted on wildtype, <i>age-1(mg109), age-1(mg109);akt-1(mg247) </i>and <i>age-1(mg109);pdk-1(mg261) </i>animals</div></p></div><div class="aux"><div class="resc">Minaxi S Gami, et al. BMC Dev Biol. 2006;6:45-45.</div><p class="links"><a class="dblinks" href="/pubmed/17020605" ref="ordinalpos=4871&amp;ncbi_uid=883204&amp;link_uid=17020605">Citation</a><a class="dblinks" href="/pmc/articles/PMC1609106/" ref="ordinalpos=4871&amp;ncbi_uid=883204&amp;link_uid=PMC1609106">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4872.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3197668/figure/pone-0026514-g003/" ref="ordinalpos=4872&amp;ncbi_uid=3301844&amp;link_uid=PMC3197668" href="/pmc/articles/PMC3197668/"><img src="/pmc/articles/instance/3197668/bin/pone.0026514.g003.gif" src-large="/pmc/articles/instance/3197668/bin/pone.0026514.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3197668/" ref="ordinalpos=4872&amp;ncbi_uid=3301844&amp;link_uid=PMC3197668" image-link="/pmc/articles/PMC3197668/figure/pone-0026514-g003/" class="imagepopup">Figure 3. Snail and Slug expression in MCF-7 cells decreases the expression of differentiation markers, and increases the expression of TGF-beta <span class="highlight" style="background-color:">pathway</span> genes..  From: The Transcription Factors Snail and Slug Activate the Transforming Growth Factor-Beta <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Breast Cancer. </a></p><div class="supp"><p class="details"><div>Heat maps depicting (A) differentiated cell markers (C) TGF-b <span class="highlight" style="background-color:">pathway</span> probes following Snail and Slug expression. (B, D and E) RT-PCR of cDNA with real-time quantitation following normalization to 18 s, MCF-7 Day 0 and control adenovirus. The data represents the average of three independent biological replicates.</div></p></div><div class="aux"><div class="resc">Archana Dhasarathy, et al. PLoS One. 2011;6(10):e26514.</div><p class="links"><a class="dblinks" href="/pubmed/22028892" ref="ordinalpos=4872&amp;ncbi_uid=3301844&amp;link_uid=22028892">Citation</a><a class="dblinks" href="/pmc/articles/PMC3197668/" ref="ordinalpos=4872&amp;ncbi_uid=3301844&amp;link_uid=PMC3197668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4873.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2815016/figure/F4/" ref="ordinalpos=4873&amp;ncbi_uid=2182643&amp;link_uid=PMC2815016" href="/pmc/articles/PMC2815016/"><img src="/pmc/articles/instance/2815016/bin/nihms140466f4.gif" src-large="/pmc/articles/instance/2815016/bin/nihms140466f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2815016/" ref="ordinalpos=4873&amp;ncbi_uid=2182643&amp;link_uid=PMC2815016" image-link="/pmc/articles/PMC2815016/figure/F4/" class="imagepopup">Fig. 4.  From: Molecular Mechanisms involved in Farnesol-Induced Apoptosis. </a></p><div class="supp"><p class="details"><div>Activation of the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by farnesol is related to ER stress. Upon activation of the UPR, activated IRE1α binds TRAF2 and recruits IKK, which subsequently phosphorylates IκB thereby promoting its degradation, while activation of the PERK-eIF2α <span class="highlight" style="background-color:">pathway</span> leads to attenuation of the rate of mRNA translation, including a reduction in IκBα synthesis. These events act synergistally to reduce IκBα and lead to the exposure of the nuclear localization signal of NF-κB, its translocation to the nucleus, and activation of NF-κB target genes. Activation of the MEK1/2-ERK1/2-MSK1 <span class="highlight" style="background-color:">pathway</span> by farnesol promotes the phosphorylation of p65/relA (Ser276) thereby enhancing NF-κB transcriptional activity.</div></p></div><div class="aux"><div class="resc">Joung Hyuck Joo, et al. Cancer Lett. ;287(2):123.</div><p class="links"><a class="dblinks" href="/pubmed/19520495" ref="ordinalpos=4873&amp;ncbi_uid=2182643&amp;link_uid=19520495">Citation</a><a class="dblinks" href="/pmc/articles/PMC2815016/" ref="ordinalpos=4873&amp;ncbi_uid=2182643&amp;link_uid=PMC2815016">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4874.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4404310/figure/F1/" ref="ordinalpos=4874&amp;ncbi_uid=7433472&amp;link_uid=PMC4404310" href="/pmc/articles/PMC4404310/"><img src="/pmc/articles/instance/4404310/bin/nihms326998f1.gif" src-large="/pmc/articles/instance/4404310/bin/nihms326998f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4404310/" ref="ordinalpos=4874&amp;ncbi_uid=7433472&amp;link_uid=PMC4404310" image-link="/pmc/articles/PMC4404310/figure/F1/" class="imagepopup">Figure 1.  From: Understanding the Biology of CRLF2-Overexpressing Acute Lymphoblastic Leukemia. </a></p><div class="supp"><p class="details"><div>Proposed schema of aberrant signal transduction mediated by the TSLPR heterodimer in <i>CRLF2</i>-overexpressing ALL. JAK/STAT, PI3K, and possibly MAPK <span class="highlight" style="background-color:">signaling</span> networks are hyperactive. Targeting hyperactive <span class="highlight" style="background-color:">signaling</span> nodes with STIs (such as JAK, PI3K <span class="highlight" style="background-color:">pathway</span>, and MEK inhibitors) is a rational therapeutic strategy for preclinical and clinical testing.</div></p></div><div class="aux"><div class="resc">Sarah K. Tasian, et al. Crit Rev Oncog. ;16(0):13-24.</div><p class="links"><a class="dblinks" href="/pubmed/22150304" ref="ordinalpos=4874&amp;ncbi_uid=7433472&amp;link_uid=22150304">Citation</a><a class="dblinks" href="/pmc/articles/PMC4404310/" ref="ordinalpos=4874&amp;ncbi_uid=7433472&amp;link_uid=PMC4404310">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4875.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4371607/figure/F6/" ref="ordinalpos=4875&amp;ncbi_uid=7321330&amp;link_uid=PMC4371607" href="/pmc/articles/PMC4371607/"><img src="/pmc/articles/instance/4371607/bin/nihms536202f6.gif" src-large="/pmc/articles/instance/4371607/bin/nihms536202f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4371607/" ref="ordinalpos=4875&amp;ncbi_uid=7321330&amp;link_uid=PMC4371607" image-link="/pmc/articles/PMC4371607/figure/F6/" class="imagepopup">Figure 6.  From: Unbiased Screening of Marine Sponge Extracts for Anti-Inflammatory Agents Combined with Chemical Genomics Identifies Girolline as an Inhibitor of Protein Synthesis. </a></p><div class="supp"><p class="details"><div>The inhibitory activity of girolline on TLR <span class="highlight" style="background-color:">signaling</span>. Dose-dependent inhibition profiles were examined following stimulation of a variety of TLRs, as well as IL-1R and TNFR, using HEK-TLR2 (a), HEK-TLR3 (b) HEK-TLR4 (c) and HEK-TLR7 (d) reporter cells. A simplified TLR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was illustrated in (e).</div></p></div><div class="aux"><div class="resc">Shan-Yu Fung, et al. ACS Chem Biol. ;9(1):247-257.</div><p class="links"><a class="dblinks" href="/pubmed/24117378" ref="ordinalpos=4875&amp;ncbi_uid=7321330&amp;link_uid=24117378">Citation</a><a class="dblinks" href="/pmc/articles/PMC4371607/" ref="ordinalpos=4875&amp;ncbi_uid=7321330&amp;link_uid=PMC4371607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4876.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4061041/figure/pone-0099983-g006/" ref="ordinalpos=4876&amp;ncbi_uid=6276407&amp;link_uid=PMC4061041" href="/pmc/articles/PMC4061041/"><img src="/pmc/articles/instance/4061041/bin/pone.0099983.g006.gif" src-large="/pmc/articles/instance/4061041/bin/pone.0099983.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4061041/" ref="ordinalpos=4876&amp;ncbi_uid=6276407&amp;link_uid=PMC4061041" image-link="/pmc/articles/PMC4061041/figure/pone-0099983-g006/" class="imagepopup">Figure 6. A model of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that mediates sCLU induction during senescence showing the role of sCLU in senescence compared to IGF-1/IGF-1R <span class="highlight" style="background-color:">signaling</span> required to prevent apoptosis..  From: ATM Regulates Insulin-Like Growth Factor 1-Secretory Clusterin (IGF-1-sCLU) Expression that Protects Cells against Senescence. </a></p><div class="supp"><p class="details"><div>AAI and AG1024 are small molecule chemical inhibitors that inhibit ATM and ATR, and IGF-1 tyrosine kinase receptor (IGF-1R), respectively.</div></p></div><div class="aux"><div class="resc">Xiuquan Luo, et al. PLoS One. 2014;9(6):e99983.</div><p class="links"><a class="dblinks" href="/pubmed/24937130" ref="ordinalpos=4876&amp;ncbi_uid=6276407&amp;link_uid=24937130">Citation</a><a class="dblinks" href="/pmc/articles/PMC4061041/" ref="ordinalpos=4876&amp;ncbi_uid=6276407&amp;link_uid=PMC4061041">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4877.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3003974/figure/fig3/" ref="ordinalpos=4877&amp;ncbi_uid=2645392&amp;link_uid=PMC3003974" href="/pmc/articles/PMC3003974/"><img src="/pmc/articles/instance/3003974/bin/IJN2011-809378.003.gif" src-large="/pmc/articles/instance/3003974/bin/IJN2011-809378.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3003974/" ref="ordinalpos=4877&amp;ncbi_uid=2645392&amp;link_uid=PMC3003974" image-link="/pmc/articles/PMC3003974/figure/fig3/" class="imagepopup">Figure 3.  From: Integrative Bioinformatics Analysis of Proteins Associated with the Cardiorenal Syndrome. </a></p><div class="supp"><p class="details"><div>Protein interaction network of highly connected proteins associated with the cardiorenal syndrome. Nodes represent genes (gene symbols); edges indicate functional associations. Highlighted nodes represent proteins that are specific for either the VEGF <span class="highlight" style="background-color:">signaling</span>, the leukocyte transendothelial migration, or the endothelin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Irmgard Mühlberger, et al. Int J Nephrol. 2011;2011:809378.</div><p class="links"><a class="dblinks" href="/pubmed/21188212" ref="ordinalpos=4877&amp;ncbi_uid=2645392&amp;link_uid=21188212">Citation</a><a class="dblinks" href="/pmc/articles/PMC3003974/" ref="ordinalpos=4877&amp;ncbi_uid=2645392&amp;link_uid=PMC3003974">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4878.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4016251/figure/pone-0097114-g003/" ref="ordinalpos=4878&amp;ncbi_uid=6131909&amp;link_uid=PMC4016251" href="/pmc/articles/PMC4016251/"><img src="/pmc/articles/instance/4016251/bin/pone.0097114.g003.gif" src-large="/pmc/articles/instance/4016251/bin/pone.0097114.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4016251/" ref="ordinalpos=4878&amp;ncbi_uid=6131909&amp;link_uid=PMC4016251" image-link="/pmc/articles/PMC4016251/figure/pone-0097114-g003/" class="imagepopup">Figure 3. Involvement of the PTEN/PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and hTERT in miR-21 mediated effects..  From: MicroRNA-21 Regulates hTERT via PTEN in Hypertrophic Scar Fibroblasts. </a></p><div class="supp"><p class="details"><div>A, Protein expression after transfection of HSFBs. B, PTEN and hTERT mRNA expression after transfection of HSFBs. C, Indirect immunofluorescence of PTEN (green) and hTERT (red) after transfection of HSFBs.</div></p></div><div class="aux"><div class="resc">Hua-Yu Zhu, et al. PLoS One. 2014;9(5):e97114.</div><p class="links"><a class="dblinks" href="/pubmed/24817011" ref="ordinalpos=4878&amp;ncbi_uid=6131909&amp;link_uid=24817011">Citation</a><a class="dblinks" href="/pmc/articles/PMC4016251/" ref="ordinalpos=4878&amp;ncbi_uid=6131909&amp;link_uid=PMC4016251">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4879.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3697781/figure/bat047-F2/" ref="ordinalpos=4879&amp;ncbi_uid=5050674&amp;link_uid=PMC3697781" href="/pmc/articles/PMC3697781/"><img src="/pmc/articles/instance/3697781/bin/bat047f2p.gif" src-large="/pmc/articles/instance/3697781/bin/bat047f2p.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3697781/" ref="ordinalpos=4879&amp;ncbi_uid=5050674&amp;link_uid=PMC3697781" image-link="/pmc/articles/PMC3697781/figure/bat047-F2/" class="imagepopup">Figure 2. From: CHD@ZJU: a knowledgebase providing network-based research platform on coronary heart disease. </a></p><div class="supp"><p class="details"><div>Query interface in CHD@ZJU. (<b>A</b>) Gene-based query interface; (<b>B</b>) <span class="highlight" style="background-color:">pathway</span>-based query interface; (<b>C</b>) network interactions of PPAR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (homo sapiens). Interaction data are from KEGG database.</div></p></div><div class="aux"><div class="resc">Leihong Wu, et al. Database (Oxford). 2013;2013:bat047.</div><p class="links"><a class="dblinks" href="/pubmed/23818526" ref="ordinalpos=4879&amp;ncbi_uid=5050674&amp;link_uid=23818526">Citation</a><a class="dblinks" href="/pmc/articles/PMC3697781/" ref="ordinalpos=4879&amp;ncbi_uid=5050674&amp;link_uid=PMC3697781">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4880.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3904226/figure/F3/" ref="ordinalpos=4880&amp;ncbi_uid=5746650&amp;link_uid=PMC3904226" href="/pmc/articles/PMC3904226/"><img src="/pmc/articles/instance/3904226/bin/nihms546365f3.gif" src-large="/pmc/articles/instance/3904226/bin/nihms546365f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3904226/" ref="ordinalpos=4880&amp;ncbi_uid=5746650&amp;link_uid=PMC3904226" image-link="/pmc/articles/PMC3904226/figure/F3/" class="imagepopup">Figure 3. Synergistic effects of aPL Abs and TLR7 on pDC maturation.  From: Dendritic cells: An important link between antiphospholipid antibodies, endothelial dysfunction, and atherosclerosis in autoimmune and non-autoimmune diseases. </a></p><div class="supp"><p class="details"><div>aPL Abs work synergistically with TLR7 to promote pDC maturation and pro-inflammatory function by activating the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Anna Broder, et al. Clin Immunol. ;146(3):197-206.</div><p class="links"><a class="dblinks" href="/pubmed/23376063" ref="ordinalpos=4880&amp;ncbi_uid=5746650&amp;link_uid=23376063">Citation</a><a class="dblinks" href="/pmc/articles/PMC3904226/" ref="ordinalpos=4880&amp;ncbi_uid=5746650&amp;link_uid=PMC3904226">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4881.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2912031/figure/F4/" ref="ordinalpos=4881&amp;ncbi_uid=2985996&amp;link_uid=PMC2912031" href="/pmc/articles/PMC2912031/"><img src="/pmc/articles/instance/2912031/bin/mvq043f4.gif" src-large="/pmc/articles/instance/2912031/bin/mvq043f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2912031/" ref="ordinalpos=4881&amp;ncbi_uid=2985996&amp;link_uid=PMC2912031" image-link="/pmc/articles/PMC2912031/figure/F4/" class="imagepopup">Fig. 4.  From: Cellular and molecular basis for the regulation of inflammation by TGF-?. </a></p><div class="supp"><p class="details"><div><b>Reciprocal inhibition mechanism by TGF-β and IFN-γ.</b> IFN-γ receptor activates STAT1, then STAT1-target genes such as Smad7 and unidentified molecules inhibit TGF-β <span class="highlight" style="background-color:">signaling</span>. TGF-β and Smads also inhibits the IFN-γ/STAT1 <span class="highlight" style="background-color:">pathway</span> by several ways. See detail in the text.</div></p></div><div class="aux"><div class="resc">Akihiko Yoshimura, et al. J Biochem. 2010 June;147(6):781-792.</div><p class="links"><a class="dblinks" href="/pubmed/20410014" ref="ordinalpos=4881&amp;ncbi_uid=2985996&amp;link_uid=20410014">Citation</a><a class="dblinks" href="/pmc/articles/PMC2912031/" ref="ordinalpos=4881&amp;ncbi_uid=2985996&amp;link_uid=PMC2912031">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4882.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3508514/figure/F3/" ref="ordinalpos=4882&amp;ncbi_uid=4393066&amp;link_uid=PMC3508514" href="/pmc/articles/PMC3508514/"><img src="/pmc/articles/instance/3508514/bin/nihms421465f3.gif" src-large="/pmc/articles/instance/3508514/bin/nihms421465f3.jpg" alt="Fig. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3508514/" ref="ordinalpos=4882&amp;ncbi_uid=4393066&amp;link_uid=PMC3508514" image-link="/pmc/articles/PMC3508514/figure/F3/" class="imagepopup">Fig. 3.  From: Cooperativity of adaptive and innate immunity: implications for cancer therapy. </a></p><div class="supp"><p class="details"><div>The alternative STAT-independent <span class="highlight" style="background-color:">pathway</span> of cytokine receptor cross-talk. The schematic illustration depicts the alternative <span class="highlight" style="background-color:">pathway</span> of STAT-independent cytokine-mediated regulation of the expression of activating or inhibitory heterologous receptors. Cyto-kine receptors may directly communicate with other cellular receptor signalosomes within plasma membrane microdomains to initiate common downstream <span class="highlight" style="background-color:">signaling</span> pathways</div></p></div><div class="aux"><div class="resc">Anil Shanker, et al. Cancer Immunol Immunother. ;60(8):1061-1074.</div><p class="links"><a class="dblinks" href="/pubmed/21656157" ref="ordinalpos=4882&amp;ncbi_uid=4393066&amp;link_uid=21656157">Citation</a><a class="dblinks" href="/pmc/articles/PMC3508514/" ref="ordinalpos=4882&amp;ncbi_uid=4393066&amp;link_uid=PMC3508514">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4883.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2712453/figure/F5/" ref="ordinalpos=4883&amp;ncbi_uid=1925239&amp;link_uid=PMC2712453" href="/pmc/articles/PMC2712453/"><img src="/pmc/articles/instance/2712453/bin/1752-0509-3-67-5.gif" src-large="/pmc/articles/instance/2712453/bin/1752-0509-3-67-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2712453/" ref="ordinalpos=4883&amp;ncbi_uid=1925239&amp;link_uid=PMC2712453" image-link="/pmc/articles/PMC2712453/figure/F5/" class="imagepopup">Figure 5.  From: BowTieBuilder: modeling signal transduction pathways. </a></p><div class="supp"><p class="details"><div><b>Erb-associated <span class="highlight" style="background-color:">signaling</span> pathways</b>. Erb signal transduction <span class="highlight" style="background-color:">pathway</span> inferred with BowTieBuilder. Membrane-bound proteins are depicted at the top, TFs are depicted at the bottom, and the inferred proteins are depicted in between. The core proteins are embedded in a gray box, in which their bow tie score is provided. For the inferred Erb <span class="highlight" style="background-color:">pathway</span>, however, no core element could be determined. The recall and precision values are given in Table 1.</div></p></div><div class="aux"><div class="resc">Jochen Supper, et al. BMC Syst Biol. 2009;3:67-67.</div><p class="links"><a class="dblinks" href="/pubmed/19566957" ref="ordinalpos=4883&amp;ncbi_uid=1925239&amp;link_uid=19566957">Citation</a><a class="dblinks" href="/pmc/articles/PMC2712453/" ref="ordinalpos=4883&amp;ncbi_uid=1925239&amp;link_uid=PMC2712453">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4884.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679328/figure/F5/" ref="ordinalpos=4884&amp;ncbi_uid=1840320&amp;link_uid=PMC2679328" href="/pmc/articles/PMC2679328/"><img src="/pmc/articles/instance/2679328/bin/emm-41-94-g005.gif" src-large="/pmc/articles/instance/2679328/bin/emm-41-94-g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679328/" ref="ordinalpos=4884&amp;ncbi_uid=1840320&amp;link_uid=PMC2679328" image-link="/pmc/articles/PMC2679328/figure/F5/" class="imagepopup">Figure 5.  From: STAT3 is involved in phosphatidic acid-induced Bcl-2 expression in HeLa cells. </a></p><div class="supp"><p class="details"><div>A proposed model for the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of PA-induced Bcl-2 expression. We diagramed a mechanism of up-regulation of Bcl-2 expression induced by PA. PA can be converted to LPA by PLA<sub>2</sub>, but does not pass through the DAG <span class="highlight" style="background-color:">pathway</span> by PAP. Subsequently, LPA acts as an important signal molecule to upregulate Bcl-2 expression. PA leads to activation of downstream kinases, ERK1/2, which are responsible for the phosphorylation of STAT3 (Ser<sup>727</sup>).</div></p></div><div class="aux"><div class="resc">Hye-Jin Choi, et al. Exp Mol Med. 2009 February 28;41(2):94-101.</div><p class="links"><a class="dblinks" href="/pubmed/19287190" ref="ordinalpos=4884&amp;ncbi_uid=1840320&amp;link_uid=19287190">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679328/" ref="ordinalpos=4884&amp;ncbi_uid=1840320&amp;link_uid=PMC2679328">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4885.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3547545/figure/f7-ebo-9-2013-001/" ref="ordinalpos=4885&amp;ncbi_uid=4564320&amp;link_uid=PMC3547545" href="/pmc/articles/PMC3547545/"><img src="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f7.gif" src-large="/pmc/articles/instance/3547545/bin/ebo-9-2013-001f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3547545/" ref="ordinalpos=4885&amp;ncbi_uid=4564320&amp;link_uid=PMC3547545" image-link="/pmc/articles/PMC3547545/figure/f7-ebo-9-2013-001/" class="imagepopup">Figure 7.  From: The Molecular Evolutionary Patterns of the Insulin/FOXO <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Correlation between the position of the elements in the IIF <span class="highlight" style="background-color:">pathway</span> and the d<sub>N</sub>/d<sub>S</sub> among different species. (<b>A</b>) for human, (<b>B</b>) for mouse and (<b>C</b>) for pufferfish and continuous lines represent regression lines.</div></p></div><div class="aux"><div class="resc">Minghui Wang, et al. Evol Bioinform Online. 2013;9:1-16.</div><p class="links"><a class="dblinks" href="/pubmed/23362368" ref="ordinalpos=4885&amp;ncbi_uid=4564320&amp;link_uid=23362368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3547545/" ref="ordinalpos=4885&amp;ncbi_uid=4564320&amp;link_uid=PMC3547545">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4886.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4342894/figure/Fig5/" ref="ordinalpos=4886&amp;ncbi_uid=7219616&amp;link_uid=PMC4342894" href="/pmc/articles/PMC4342894/"><img src="/pmc/articles/instance/4342894/bin/13058_2015_515_Fig5_HTML.gif" src-large="/pmc/articles/instance/4342894/bin/13058_2015_515_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4342894/" ref="ordinalpos=4886&amp;ncbi_uid=7219616&amp;link_uid=PMC4342894" image-link="/pmc/articles/PMC4342894/figure/Fig5/" class="imagepopup">Figure 5.  From: MicroRNA-125b upregulation confers aromatase inhibitor resistance and is a novel marker of poor prognosis in breast cancer. </a></p><div class="supp"><p class="details"><div> <b>The PI3K/AKT/mTOR <span class="highlight" style="background-color:">pathway</span> is targeted by the microRNAs deregulated in the aromatase inhibitor-resistant cells. (A)</b> Overview of the AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> illustrating the results of our analysis. Genes in yellow rectangles represent the genes targeted by microRNAs (miRs) miR-125b, miR-205, miR-424 and miR-30a. <b>(B)</b> Western blot analysis of the AKT <span class="highlight" style="background-color:">pathway</span> in MCF-7aro and Res-Let cells. <b>(C)</b> Western blot analysis of the AKT <span class="highlight" style="background-color:">pathway</span> in MCF-7aro cells transfected with the mimic of miR-125b-5p or miR-205-5p, the inhibitor of miR-424-3p, or their respective negative controls. <b>(D)</b> Western blot analysis of the AKT <span class="highlight" style="background-color:">pathway</span> activation status in Res-Let cells transfected with the inhibitor of miR-125b-5p or miR-205-5p, the mimic of miR-424-3p, or their respective negative controls. <b>(B)</b>, <b>(C)</b> and <b>(D)</b> show representative blots from three independent experiments and cell lysates. The relative expression levels of the corresponding ratio of phosphorylated to total proteins were quantified, and these data are shown below the blots.</div></p></div><div class="aux"><div class="resc">Paul Vilquin, et al. Breast Cancer Res. 2015;17(1):13.</div><p class="links"><a class="dblinks" href="/pubmed/25633049" ref="ordinalpos=4886&amp;ncbi_uid=7219616&amp;link_uid=25633049">Citation</a><a class="dblinks" href="/pmc/articles/PMC4342894/" ref="ordinalpos=4886&amp;ncbi_uid=7219616&amp;link_uid=PMC4342894">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4887.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC85165/figure/F3/" ref="ordinalpos=4887&amp;ncbi_uid=6546&amp;link_uid=PMC85165" href="/pmc/articles/PMC85165/"><img src="/pmc/articles/instance/85165/bin/mb0200939003.gif" src-large="/pmc/articles/instance/85165/bin/mb0200939003.jpg" alt="FIG. 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC85165/" ref="ordinalpos=4887&amp;ncbi_uid=6546&amp;link_uid=PMC85165" image-link="/pmc/articles/PMC85165/figure/F3/" class="imagepopup">FIG. 3.  From: Cyclin D1 Is Required for Transformation by Activated Neu and Is Induced through an E2F-Dependent <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effects of inhibitors on NeuT induction of cyclin D1. To examine the intracellular <span class="highlight" style="background-color:">signaling</span> pathways involved in Neu induction of the cyclin D1 promoter, the −1745CD1LUC reporter was introduced into MCF7 cells with the NeuT expression vector. (A) Cotransfection experiments were conducted using increasing amounts of dominant negative expression vector, and the inhibition of Neu-induced promoter activation is shown as percent activity. Comparison was made between the effect of the dominant negative mutants for N17Ras, N17Rac, or N19Rho and equal amounts of empty expression vector cassette. (B) The chemical inhibitors of the MEK/ERK <span class="highlight" style="background-color:">pathway</span> (PD098059) (<i>n</i> = 8) and the p38 <span class="highlight" style="background-color:">pathway</span> (SB203580) (<i>n</i> = 8) were added to the culture medium and compared with equal volumes of DMSO vehicle. The expression vector encoding the inhibitor of JNK <span class="highlight" style="background-color:">signaling</span> (JIP-1) (<i>n</i> = 4) for each concentration of plasmid is shown compared with equal amounts of empty expression vector cassette. Inhibition is significant at <i>P</i> &lt; 0.05.</div></p></div><div class="aux"><div class="resc">Richard J. Lee, et al. Mol Cell Biol. 2000 January;20(2):672-683.</div><p class="links"><a class="dblinks" href="/pubmed/10611246" ref="ordinalpos=4887&amp;ncbi_uid=6546&amp;link_uid=10611246">Citation</a><a class="dblinks" href="/pmc/articles/PMC85165/" ref="ordinalpos=4887&amp;ncbi_uid=6546&amp;link_uid=PMC85165">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4888.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3489927/figure/F6/" ref="ordinalpos=4888&amp;ncbi_uid=4346409&amp;link_uid=PMC3489927" href="/pmc/articles/PMC3489927/"><img src="/pmc/articles/instance/3489927/bin/nihms371721f6.gif" src-large="/pmc/articles/instance/3489927/bin/nihms371721f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3489927/" ref="ordinalpos=4888&amp;ncbi_uid=4346409&amp;link_uid=PMC3489927" image-link="/pmc/articles/PMC3489927/figure/F6/" class="imagepopup">Figure 6. Inhibition of c-Kit <span class="highlight" style="background-color:">signaling</span>.  From: Characterization of c-Kit expression and activation in KSHV-infected endothelial cells. </a></p><div class="supp"><p class="details"><div>To confirm the activation of c-Kit <span class="highlight" style="background-color:">signaling</span> by KSHV, a series of kinase inhibitors were utilized. The diagram (A) depicts the c-Kit kinase-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and where each inhibitor acts (modified from (Lennartsson et al., 2005a)). A p-Erk 1/2 immunoblot was performed on KSHV-infected cells pre-incubated with the indicated kinase inhibitors prior to 10 minutes of SCF stimulation (B, top blot). The blot was stripped and reprobed with antibodies to Erk 1/2 to measure total protein levels (B, bottom blot). The concentration (μM) of inhibitor is indicated above the blots.</div></p></div><div class="aux"><div class="resc">Janet L. Douglas, et al. Virology. 2009 August 1;390(2):174-185.</div><p class="links"><a class="dblinks" href="/pubmed/19501868" ref="ordinalpos=4888&amp;ncbi_uid=4346409&amp;link_uid=19501868">Citation</a><a class="dblinks" href="/pmc/articles/PMC3489927/" ref="ordinalpos=4888&amp;ncbi_uid=4346409&amp;link_uid=PMC3489927">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4889.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2668721/figure/F1/" ref="ordinalpos=4889&amp;ncbi_uid=1856783&amp;link_uid=PMC2668721" href="/pmc/articles/PMC2668721/"><img src="/pmc/articles/instance/2668721/bin/nihms95970f1.gif" src-large="/pmc/articles/instance/2668721/bin/nihms95970f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2668721/" ref="ordinalpos=4889&amp;ncbi_uid=1856783&amp;link_uid=PMC2668721" image-link="/pmc/articles/PMC2668721/figure/F1/" class="imagepopup">Figure 1. The Rho GTPase – Rho kinase <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in vascular smooth muscle.  From: The pericyte: Cellular regulator of microvascular blood flow. </a></p><div class="supp"><p class="details"><div>The Rho GTPase mechanochemical <span class="highlight" style="background-color:">signaling</span> apparatus links transmembrane and intracellular <span class="highlight" style="background-color:">signaling</span>-associated Rho GTPase with functional alterations in the actin cytoskeleton. Rho kinase, a principal downstream effector of Rho GTPase, mediates linkage to the actin cytoskeleton via four principle pathways discussed in the text: myosin light chain phosphatase (MLCP), ADF/cofilin, ERM proteins, and p38 MAP kinase.</div></p></div><div class="aux"><div class="resc">Matthew E. Kutcher, et al. Microvasc Res. ;77(3):235-246.</div><p class="links"><a class="dblinks" href="/pubmed/19323975" ref="ordinalpos=4889&amp;ncbi_uid=1856783&amp;link_uid=19323975">Citation</a><a class="dblinks" href="/pmc/articles/PMC2668721/" ref="ordinalpos=4889&amp;ncbi_uid=1856783&amp;link_uid=PMC2668721">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4890.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3441384/figure/F4/" ref="ordinalpos=4890&amp;ncbi_uid=4233774&amp;link_uid=PMC3441384" href="/pmc/articles/PMC3441384/"><img src="/pmc/articles/instance/3441384/bin/1743-8977-9-20-4.gif" src-large="/pmc/articles/instance/3441384/bin/1743-8977-9-20-4.jpg" alt="Figure 4 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3441384/" ref="ordinalpos=4890&amp;ncbi_uid=4233774&amp;link_uid=PMC3441384" image-link="/pmc/articles/PMC3441384/figure/F4/" class="imagepopup">Figure 4 .  From: Autophagy and lysosomal dysfunction as emerging mechanisms of nanomaterial toxicity. </a></p><div class="supp"><p class="details"><div><b>Nanoparticle-induced autophagy. A)</b> Recent evidence supporting ubiquitination of nanomaterials directly, or indirectly through colocalization with protein aggregates, suggests that cells may select nanomaterials for autophagy through a p62-LC3 II <span class="highlight" style="background-color:">pathway</span> similar to invading pathogens (see text). <b>B)</b> Data also supports nanomaterial-induced alteration of autophagy <span class="highlight" style="background-color:">signaling</span> pathways, including: <b>1)</b> induction of oxidative stress-dependent <span class="highlight" style="background-color:">signaling</span> (e.g., ER stress, mitochondrial damage), <b>2)</b> suppression of Akt-mTOR <span class="highlight" style="background-color:">signaling</span>, and <b>3)</b> alteration of autophagy related gene/protein expression.</div></p></div><div class="aux"><div class="resc">Stephan T Stern, et al. Part Fibre Toxicol. 2012;9:20-20.</div><p class="links"><a class="dblinks" href="/pubmed/22697169" ref="ordinalpos=4890&amp;ncbi_uid=4233774&amp;link_uid=22697169">Citation</a><a class="dblinks" href="/pmc/articles/PMC3441384/" ref="ordinalpos=4890&amp;ncbi_uid=4233774&amp;link_uid=PMC3441384">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4891.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2739145/figure/fig10/" ref="ordinalpos=4891&amp;ncbi_uid=1963435&amp;link_uid=PMC2739145" href="/pmc/articles/PMC2739145/"><img src="/pmc/articles/instance/2739145/bin/DEV040451F10.gif" src-large="/pmc/articles/instance/2739145/bin/DEV040451F10.jpg" alt="Fig. 10." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2739145/" ref="ordinalpos=4891&amp;ncbi_uid=1963435&amp;link_uid=PMC2739145" image-link="/pmc/articles/PMC2739145/figure/fig10/" class="imagepopup">Fig. 10. From: A directional Wnt/?-catenin-Sox2-proneural <span class="highlight" style="background-color:">pathway</span> regulates the  transition from proliferation to differentiation in the Xenopus  retina. </a></p><div class="supp"><p class="details"><div><b>A model for the role of the Wnt-Sox2 <span class="highlight" style="background-color:">pathway</span> in the transition from a  progenitor to a differentiated cell.</b> (<b>A</b>) Wnt <span class="highlight" style="background-color:">signaling</span> activation  in a neuroepithelial cell activates Sox2 and the proneural genes, but through  Sox2 and Notch it blocks proneural activity and neuronal differentiation. Wnt  also independently maintains proliferation, and this leads to progenitor  maintenance and expansion. (<b>B</b>) The build-up of Sox2 switches off Wnt,  inhibiting proliferation (B1), and then the accumulation of proneural activity  switches off Sox2, relieving the inhibition of neuronal differentiation and  leading to neurogenesis (B2). (<b>C</b>) Alternatively, if Sox2 levels remain  high it will limit proneural activity and neuronal differentiation will be  blocked, but Wnt <span class="highlight" style="background-color:">signaling</span> will also be inhibited, leading to cell cycle exit  and glial differentiation.</div></p></div><div class="aux"><div class="resc">Michalis Agathocleous, et al. Development. 2009 October 1;136(19):3289-3299.</div><p class="links"><a class="dblinks" href="/pubmed/19736324" ref="ordinalpos=4891&amp;ncbi_uid=1963435&amp;link_uid=19736324">Citation</a><a class="dblinks" href="/pmc/articles/PMC2739145/" ref="ordinalpos=4891&amp;ncbi_uid=1963435&amp;link_uid=PMC2739145">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4892.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2856666/figure/F4/" ref="ordinalpos=4892&amp;ncbi_uid=2263384&amp;link_uid=PMC2856666" href="/pmc/articles/PMC2856666/"><img src="/pmc/articles/instance/2856666/bin/nihms103715f4.gif" src-large="/pmc/articles/instance/2856666/bin/nihms103715f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2856666/" ref="ordinalpos=4892&amp;ncbi_uid=2263384&amp;link_uid=PMC2856666" image-link="/pmc/articles/PMC2856666/figure/F4/" class="imagepopup">Figure 4. Mapping genetic interaction data onto known <span class="highlight" style="background-color:">signaling</span> pathways.  From: Functional Organization of the S. cerevisiae Phosphorylation Network. </a></p><div class="supp"><p class="details"><div>a) General schematic of the Sln1 branch of the HOG <span class="highlight" style="background-color:">pathway</span>. Protein kinases and phosphatases are denoted as green and red, respectively, while green and red arrows correspond to phosphorylation and dephosphorylation actions, respectively. b) The negative <span class="highlight" style="background-color:">pathway</span> regulators <i>YPD1</i>, <i>PTC1</i>, and <i>NBP2</i>, show strong positive genetic interactions with the <span class="highlight" style="background-color:">pathway</span> activators <i>SSK1</i>, <i>SSK2</i>, <i>SSK22</i>, <i>PBS2</i>, and <i>HOG1</i>. In contrast, genes coding for the protein phosphatases (<i>PTC1</i>, <i>PTC2</i>, <i>PTC3</i>, <i>PTP2</i>, and <i>PTP3</i>) acting on <i>HOG1</i> show primarily negative genetic interactions among themselves. c) Scatter plot of the correlation coefficients of <i>pbs2</i>Δ and <i>hog1</i>Δ with all genetic profiles in the <span class="highlight" style="background-color:">signaling</span> E-MAP. d) Schematic illustration of the inositol polyphosphate <span class="highlight" style="background-color:">pathway</span>. e) A subset of genetic interactions for <i>PLC1</i>, <i>KCS1</i>, <i>IPK1</i>, and <i>VIP1</i>. In b) and e), negative and positive genetic interactions are indicated by blue or yellow squares, respectively. Gray squares represent genetic interactions not tested.</div></p></div><div class="aux"><div class="resc">Dorothea Fiedler, et al. Cell. ;136(5):952-963.</div><p class="links"><a class="dblinks" href="/pubmed/19269370" ref="ordinalpos=4892&amp;ncbi_uid=2263384&amp;link_uid=19269370">Citation</a><a class="dblinks" href="/pmc/articles/PMC2856666/" ref="ordinalpos=4892&amp;ncbi_uid=2263384&amp;link_uid=PMC2856666">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4893.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4367558/figure/f1/" ref="ordinalpos=4893&amp;ncbi_uid=7319910&amp;link_uid=PMC4367558" href="/pmc/articles/PMC4367558/"><img src="/pmc/articles/instance/4367558/bin/tlcr-01-03-194-f1.gif" src-large="/pmc/articles/instance/4367558/bin/tlcr-01-03-194-f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4367558/" ref="ordinalpos=4893&amp;ncbi_uid=7319910&amp;link_uid=PMC4367558" image-link="/pmc/articles/PMC4367558/figure/f1/" class="imagepopup">Figure 1.  From: MET genetic lesions in non-small-cell lung cancer: pharmacological and clinical implications. </a></p><div class="supp"><p class="details"><div>The MET oncogene and its <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Panel A. schematic structure of the MET receptor; Panel B. MET activation signalling cascade. Details are discussed in the text</div></p></div><div class="aux"><div class="resc">Michele Zorzetto, et al. Transl Lung Cancer Res. 2012 September;1(3):194-207.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4893&amp;ncbi_uid=7319910&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4367558/" ref="ordinalpos=4893&amp;ncbi_uid=7319910&amp;link_uid=PMC4367558">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4894.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4245781/figure/Fig1/" ref="ordinalpos=4894&amp;ncbi_uid=6885979&amp;link_uid=PMC4245781" href="/pmc/articles/PMC4245781/"><img src="/pmc/articles/instance/4245781/bin/13048_2014_82_Fig1_HTML.gif" src-large="/pmc/articles/instance/4245781/bin/13048_2014_82_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4245781/" ref="ordinalpos=4894&amp;ncbi_uid=6885979&amp;link_uid=PMC4245781" image-link="/pmc/articles/PMC4245781/figure/Fig1/" class="imagepopup">Figure 1.  From: Role of nerve growth factor and its TRKA receptor in normal ovarian and epithelial ovarian cancer angiogenesis. </a></p><div class="supp"><p class="details"><div> <b><span class="highlight" style="background-color:">Signaling</span> pathways of TRKA activation by NGF.</b> The NGF-dependent activation <span class="highlight" style="background-color:">pathway</span> of TRKA is involved in cellular proliferation, survival, differentiation and angiogenesis in EOC.</div></p></div><div class="aux"><div class="resc">Carolina Vera, et al. J Ovarian Res. 2014;7(1):82.</div><p class="links"><a class="dblinks" href="/pubmed/25296882" ref="ordinalpos=4894&amp;ncbi_uid=6885979&amp;link_uid=25296882">Citation</a><a class="dblinks" href="/pmc/articles/PMC4245781/" ref="ordinalpos=4894&amp;ncbi_uid=6885979&amp;link_uid=PMC4245781">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4895.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3403928/figure/F4/" ref="ordinalpos=4895&amp;ncbi_uid=4143540&amp;link_uid=PMC3403928" href="/pmc/articles/PMC3403928/"><img src="/pmc/articles/instance/3403928/bin/1687-4153-2012-4-4.gif" src-large="/pmc/articles/instance/3403928/bin/1687-4153-2012-4-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3403928/" ref="ordinalpos=4895&amp;ncbi_uid=4143540&amp;link_uid=PMC3403928" image-link="/pmc/articles/PMC3403928/figure/F4/" class="imagepopup">Figure 4.  From: A visual analytics approach for models of heterogeneous cell populations. </a></p><div class="supp"><p class="details"><div><b>Caspase cascade</b>. Illustration of proapoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> [35]. Normal arrows refer to conversion reactions, dashed arrows indicate enzymatic activity, and thick arrows illustrate inputs and outputs of the system.</div></p></div><div class="aux"><div class="resc">Jan Hasenauer, et al. EURASIP J Bioinform Syst Biol. 2012;2012(1):4-4.</div><p class="links"><a class="dblinks" href="/pubmed/22651376" ref="ordinalpos=4895&amp;ncbi_uid=4143540&amp;link_uid=22651376">Citation</a><a class="dblinks" href="/pmc/articles/PMC3403928/" ref="ordinalpos=4895&amp;ncbi_uid=4143540&amp;link_uid=PMC3403928">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4896.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3223722/figure/F1/" ref="ordinalpos=4896&amp;ncbi_uid=3771515&amp;link_uid=PMC3223722" href="/pmc/articles/PMC3223722/"><img src="/pmc/articles/instance/3223722/bin/fnmol-04-00047-g001.gif" src-large="/pmc/articles/instance/3223722/bin/fnmol-04-00047-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3223722/" ref="ordinalpos=4896&amp;ncbi_uid=3771515&amp;link_uid=PMC3223722" image-link="/pmc/articles/PMC3223722/figure/F1/" class="imagepopup">Figure 1.  From: Emerging Roles of Glycogen Synthase Kinase 3 in the Treatment of Brain Tumors. </a></p><div class="supp"><p class="details"><div><b>Glycogen synthase kinase 3 modulates the function of key <span class="highlight" style="background-color:">signaling</span> proteins in the wnt <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Caroline N. Mills, et al. Front Mol Neurosci. 2011;4:47.</div><p class="links"><a class="dblinks" href="/pubmed/22275880" ref="ordinalpos=4896&amp;ncbi_uid=3771515&amp;link_uid=22275880">Citation</a><a class="dblinks" href="/pmc/articles/PMC3223722/" ref="ordinalpos=4896&amp;ncbi_uid=3771515&amp;link_uid=PMC3223722">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4897.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2966480/figure/f5/" ref="ordinalpos=4897&amp;ncbi_uid=3583072&amp;link_uid=PMC2966480" href="/pmc/articles/PMC2966480/"><img src="/pmc/articles/instance/2966480/bin/fig-5.gif" src-large="/pmc/articles/instance/2966480/bin/fig-5.jpg" alt="FIG. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2966480/" ref="ordinalpos=4897&amp;ncbi_uid=3583072&amp;link_uid=PMC2966480" image-link="/pmc/articles/PMC2966480/figure/f5/" class="imagepopup">FIG. 5. From: When NRF2 Talks, Who's Listening?. </a></p><div class="supp"><p class="details"><div><b>Repression of NRF2 <span class="highlight" style="background-color:">signaling</span> by members of the NF-κB <span class="highlight" style="background-color:">pathway</span>.</b> p65 represses NRF2-dependent transcription at the ARE when phosphorylated on S276 by either competition for coactivator binding proteins or by recruiting HDAC to the ARE, which can deacetylate histone H4 and MafK.</div></p></div><div class="aux"><div class="resc">Nobunao Wakabayashi, et al. Antioxid Redox Signal. 2010 December 1;13(11):1649-1663.</div><p class="links"><a class="dblinks" href="/pubmed/20367496" ref="ordinalpos=4897&amp;ncbi_uid=3583072&amp;link_uid=20367496">Citation</a><a class="dblinks" href="/pmc/articles/PMC2966480/" ref="ordinalpos=4897&amp;ncbi_uid=3583072&amp;link_uid=PMC2966480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4898.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1601945/figure/F1/" ref="ordinalpos=4898&amp;ncbi_uid=867262&amp;link_uid=PMC1601945" href="/pmc/articles/PMC1601945/"><img src="/pmc/articles/instance/1601945/bin/neo0808_0689_fig001.gif" src-large="/pmc/articles/instance/1601945/bin/neo0808_0689_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1601945/" ref="ordinalpos=4898&amp;ncbi_uid=867262&amp;link_uid=PMC1601945" image-link="/pmc/articles/PMC1601945/figure/F1/" class="imagepopup">Figure 1.  From: Novel RING E3 Ubiquitin Ligases in Breast Cancer. </a></p><div class="supp"><p class="details"><div>Model of RNF11 interactions in the TGFβ receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. RNF11 acts against the degradation of the TGFβ receptor, enhancing its tumor-suppressor effects on breast cancer.</div></p></div><div class="aux"><div class="resc">Angelika Burger, et al. Neoplasia. 2006 August;8(8):689-695.</div><p class="links"><a class="dblinks" href="/pubmed/16925951" ref="ordinalpos=4898&amp;ncbi_uid=867262&amp;link_uid=16925951">Citation</a><a class="dblinks" href="/pmc/articles/PMC1601945/" ref="ordinalpos=4898&amp;ncbi_uid=867262&amp;link_uid=PMC1601945">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4899.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC517537/figure/f7/" ref="ordinalpos=4899&amp;ncbi_uid=505354&amp;link_uid=PMC517537" href="/pmc/articles/PMC517537/"><img src="/pmc/articles/instance/517537/bin/zii0100442890007.gif" src-large="/pmc/articles/instance/517537/bin/zii0100442890007.jpg" alt="FIG. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC517537/" ref="ordinalpos=4899&amp;ncbi_uid=505354&amp;link_uid=PMC517537" image-link="/pmc/articles/PMC517537/figure/f7/" class="imagepopup">FIG. 7. From: The Phosphatidylinositol 3-Kinase/Protein Kinase B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Is Activated by Lipoteichoic Acid and Plays a Role in Kupffer Cell Production of Interleukin-6 (IL-6) and IL-10 . </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> kinases involved in regulation of TNF-α, IL-10, and IL-6 in Kupffer cells. A TLR signal induced by LPS has been shown to activate p42/p44 MAPK (ERK 1/2), p38 MAPK, PI3-K/PKB, and the tyrosine kinases Src and JAK-2 in macrophages (2, 15, 54). These <span class="highlight" style="background-color:">signaling</span> kinases are also important for Kupffer cell responses to LTA. Each box contains <span class="highlight" style="background-color:">signaling</span> kinases clearly involved in regulating the release of the cytokine shown below. The figure is based on the effects of specific <span class="highlight" style="background-color:">signaling</span> kinase inhibitors obtained in this report. Tyr-K, tyrosine class kinases; Ser/Thr-K, serine-threonine class kinases.</div></p></div><div class="aux"><div class="resc">Maria K. Dahle, et al. Infect Immun. 2004 October;72(10):5704-5711.</div><p class="links"><a class="dblinks" href="/pubmed/15385469" ref="ordinalpos=4899&amp;ncbi_uid=505354&amp;link_uid=15385469">Citation</a><a class="dblinks" href="/pmc/articles/PMC517537/" ref="ordinalpos=4899&amp;ncbi_uid=505354&amp;link_uid=PMC517537">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4900.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3533470/figure/F7/" ref="ordinalpos=4900&amp;ncbi_uid=5660610&amp;link_uid=PMC3533470" href="/pmc/articles/PMC3533470/"><img src="/pmc/articles/instance/3533470/bin/mol.112.081760f7.gif" src-large="/pmc/articles/instance/3533470/bin/mol.112.081760f7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3533470/" ref="ordinalpos=4900&amp;ncbi_uid=5660610&amp;link_uid=PMC3533470" image-link="/pmc/articles/PMC3533470/figure/F7/" class="imagepopup">Fig. 7. From: A New Site and Mechanism of Action for the Widely Used Adenylate Cyclase Inhibitor SQ22,536. </a></p><div class="supp"><p class="details"><div>Schematic representation of the role of SQ22,536 in cAMP <span class="highlight" style="background-color:">signaling</span>. In NS-1 cells, cAMP activates at least three sensors (depicted in orange): PKA, Epac, and the neuritogenic cAMP sensor (NCS). Green arrows represent pharmacological activators, red arrows represent inhibitors, and black arrows represent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> linkages. Dashed lines indicate additional <span class="highlight" style="background-color:">signaling</span> components that have not been determined. Cat, catalytic subunit; Reg, regulatory subunit.</div></p></div><div class="aux"><div class="resc">Andrew C. Emery, et al. Mol Pharmacol. 2013 January;83(1):95-105.</div><p class="links"><a class="dblinks" href="/pubmed/23053667" ref="ordinalpos=4900&amp;ncbi_uid=5660610&amp;link_uid=23053667">Citation</a><a class="dblinks" href="/pmc/articles/PMC3533470/" ref="ordinalpos=4900&amp;ncbi_uid=5660610&amp;link_uid=PMC3533470">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="48" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="49" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="50" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:51:17-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A110554314361000000000021EB69;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A110554314361000000000021EB69" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>