
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="44" /><meta name="ncbi_resultcount" content="40695" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="F4FB2E81543134F1000000000001BB6D" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=2" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="43" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="44" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="45" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="44" /></div>
            <div><h2 class="result_count">Results: 4301 to 4400 of 40695</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40695" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>4301.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884164/figure/F10/" ref="ordinalpos=4301&amp;ncbi_uid=2298051&amp;link_uid=PMC2884164" href="/pmc/articles/PMC2884164/"><img src="/pmc/articles/instance/2884164/bin/1755-8794-3-13-10.gif" src-large="/pmc/articles/instance/2884164/bin/1755-8794-3-13-10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884164/" ref="ordinalpos=4301&amp;ncbi_uid=2298051&amp;link_uid=PMC2884164" image-link="/pmc/articles/PMC2884164/figure/F10/" class="imagepopup">Figure 10.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from human pituitary adenoma proteomics data. </a></p><div class="supp"><p class="details"><div><b>Significant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from nitroproteomic dataset</b>. Significant <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> networks that are involved in human pituitary adenoma nitroproteomic data. (A) Network 10 is derived from adenoma nitroproteomic data and function in cancer, cell cycle, reproductive-system disease. A gray node denotes an identified nitroprotein or protein that interact with nitroproteins in our study [4]. (B) Network 11 is derived from control nitroproteomic data and function in gene expression, cellular development, connective tissue development and function. A gray node denotes an identified nitroprotein in our studies [5,6]. An orange solid edge denotes a direct relationship between two nodes (molecules: proteins; genes). An orange unsolid edge denotes an indirect relationship between two nodes (molecules: proteins; genes). The various shapes of nodes denote the different functions (Additional file 1, Tables S3 and S4). A curved line means intracellular translocation; a curved arrow means extracellular translocation.</div></p></div><div class="aux"><div class="resc">Xianquan Zhan, et al. BMC Med Genomics. 2010;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20426862" ref="ordinalpos=4301&amp;ncbi_uid=2298051&amp;link_uid=20426862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884164/" ref="ordinalpos=4301&amp;ncbi_uid=2298051&amp;link_uid=PMC2884164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4302.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3954160/figure/f1/" ref="ordinalpos=4302&amp;ncbi_uid=6002641&amp;link_uid=PMC3954160" href="/pmc/articles/PMC3954160/"><img src="/pmc/articles/instance/3954160/bin/f1000research-3-3698-g0000.gif" src-large="/pmc/articles/instance/3954160/bin/f1000research-3-3698-g0000.jpg" alt="Figure 1. " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3954160/" ref="ordinalpos=4302&amp;ncbi_uid=6002641&amp;link_uid=PMC3954160" image-link="/pmc/articles/PMC3954160/figure/f1/" class="imagepopup">Figure 1.  From:  KEGGViewer, a BioJS component to visualize KEGG Pathways. </a></p><div class="supp"><p class="details"><div>( <b>a</b>) <i>KEGGViewer</i> rendering of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Pathway</span> components can be manually repositioned. Genes and pathways are represented as green and blue boxes respectively while purple dots represent chemical compounds. Relationships represent reactions e.g. activation, inhibtion or phosphorilation. ( <b>b</b>) Zoomed view of the insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>Condition 1</i> is selected in the control panel (top right) and the expression range is set to consider expression levels between -0.43 and 0.43 to be non differentially expressed. Genes <i>PPP1CA</i> and <i>PYGB</i> in red are upregulated while <i>GYS1</i> in blue is downregulated. <i>GSK3B</i> and <i>CALML6</i> in green are non differentially expressed genes. The purple dot <i>C00369</i> represents Starch.</div></p></div><div class="aux"><div class="resc">Jose M. Villaveces, et al. F1000Res. 2014;3:43.</div><p class="links"><a class="dblinks" href="/pubmed/24715980" ref="ordinalpos=4302&amp;ncbi_uid=6002641&amp;link_uid=24715980">Citation</a><a class="dblinks" href="/pmc/articles/PMC3954160/" ref="ordinalpos=4302&amp;ncbi_uid=6002641&amp;link_uid=PMC3954160">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4303.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3885266/figure/fig1/" ref="ordinalpos=4303&amp;ncbi_uid=5714387&amp;link_uid=PMC3885266" href="/pmc/articles/PMC3885266/"><img src="/pmc/articles/instance/3885266/bin/TSWJ2013-178708.001.gif" src-large="/pmc/articles/instance/3885266/bin/TSWJ2013-178708.001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3885266/" ref="ordinalpos=4303&amp;ncbi_uid=5714387&amp;link_uid=PMC3885266" image-link="/pmc/articles/PMC3885266/figure/fig1/" class="imagepopup">Figure 1.  From: Blockage of Notch <span class="highlight" style="background-color:">Signaling</span> Inhibits the Migration and Proliferation of Retinal Pigment Epithelial Cells. </a></p><div class="supp"><p class="details"><div>The members of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> components were differentially expressed in RPE cells. The relative expression levels of Notch <span class="highlight" style="background-color:">signaling</span> components in RPE cells were determined by real-time RT-PCR analysis. The detected components included five Notch ligands (JAG1, JAG2, DLL1, DLL3, and DLL4) and three MAML transcriptional coactivators (MAML1, MAML2, and MAML3). Error bars represent at least two independent experiments.</div></p></div><div class="aux"><div class="resc">Weiwei Liu, et al. ScientificWorldJournal. 2013;2013:178708.</div><p class="links"><a class="dblinks" href="/pubmed/24453806" ref="ordinalpos=4303&amp;ncbi_uid=5714387&amp;link_uid=24453806">Citation</a><a class="dblinks" href="/pmc/articles/PMC3885266/" ref="ordinalpos=4303&amp;ncbi_uid=5714387&amp;link_uid=PMC3885266">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4304.</span></div><div class="rslt"><div class="rprt_img imggrp"><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3472634/figure/F3/" ref="ordinalpos=4304&amp;ncbi_uid=4299636&amp;link_uid=PMC3472634" href="/pmc/articles/PMC3472634/"><img src="/pmc/articles/instance/3472634/bin/nihms137917f3a.gif" src-large="/pmc/articles/instance/3472634/bin/nihms137917f3a.jpg" alt="" /></a><a class="figpopup imagepopup" image-link="/pmc/articles/PMC3472634/figure/F3/" ref="ordinalpos=4304&amp;ncbi_uid=4299636&amp;link_uid=PMC3472634" href="/pmc/articles/PMC3472634/"><img src="/pmc/articles/instance/3472634/bin/nihms137917f3b.gif" src-large="/pmc/articles/instance/3472634/bin/nihms137917f3b.jpg" alt="" /></a></div><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3472634/" ref="ordinalpos=4304&amp;ncbi_uid=4299636&amp;link_uid=PMC3472634" image-link="/pmc/articles/PMC3472634/figure/F3/" class="imagepopup">Figure 3.  From: Dithiolethione Compounds Inhibit Akt <span class="highlight" style="background-color:">Signaling</span> in Human Breast and Lung Cancer Cells by Increasing PP2A Activity. </a></p><div class="supp"><p class="details"><div>ACS-1 inhibits EGF activation of Akt and mTOR in MB231 and A549 cells. Serum starved cells were treated with ACS-1 for 2 hours and stimulated with EGF. (A) Western blot analysis of EGF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in response to EGF and/or ACS-1. (B) Graphical representation of Akt activation in MB231 cells; ratios of P-Akt-(ser 473):total Akt are presented as percent of EGF control (mean  SD). (C) Western blot analysis of ACS-1 effects on EGF <span class="highlight" style="background-color:">signaling</span> in A549 cells. (D) Graphical representation of Akt activation in A549 cells; ratios of P-Akt-(ser 473):total Akt are presented as percent of EGF control (mean  SD). (E) Representative western blot analysis of P-mTOR-(ser 2448) in MB231 cells treated with ACS-1 and stimulated with EGF.</div></p></div><div class="aux"><div class="resc">Christopher H. Switzer, et al. Oncogene. 2009 October 29;28(43):3837-3846.</div><p class="links"><a class="dblinks" href="/pubmed/19701246" ref="ordinalpos=4304&amp;ncbi_uid=4299636&amp;link_uid=19701246">Citation</a><a class="dblinks" href="/pmc/articles/PMC3472634/" ref="ordinalpos=4304&amp;ncbi_uid=4299636&amp;link_uid=PMC3472634">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4305.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4149495/figure/pone-0105792-g001/" ref="ordinalpos=4305&amp;ncbi_uid=6555383&amp;link_uid=PMC4149495" href="/pmc/articles/PMC4149495/"><img src="/pmc/articles/instance/4149495/bin/pone.0105792.g001.gif" src-large="/pmc/articles/instance/4149495/bin/pone.0105792.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4149495/" ref="ordinalpos=4305&amp;ncbi_uid=6555383&amp;link_uid=PMC4149495" image-link="/pmc/articles/PMC4149495/figure/pone-0105792-g001/" class="imagepopup">Figure 1. Schematic representation of a network of PI3K, mTOR and MEK complex <span class="highlight" style="background-color:">signaling</span>..  From: Relationships between <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Usage and Sensitivity to a <span class="highlight" style="background-color:">Pathway</span> Inhibitor: Examination of Trametinib Responses in Cultured Breast Cancer Lines. </a></p><div class="supp"><p class="details"><div>Blue arrows and black lines represent activating and inhibitory connections, respectively.</div></p></div><div class="aux"><div class="resc">Euphemia Y. Leung, et al. PLoS One. 2014;9(8):e105792.</div><p class="links"><a class="dblinks" href="/pubmed/25170609" ref="ordinalpos=4305&amp;ncbi_uid=6555383&amp;link_uid=25170609">Citation</a><a class="dblinks" href="/pmc/articles/PMC4149495/" ref="ordinalpos=4305&amp;ncbi_uid=6555383&amp;link_uid=PMC4149495">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4306.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g002/" ref="ordinalpos=4306&amp;ncbi_uid=4834035&amp;link_uid=PMC3633842" href="/pmc/articles/PMC3633842/"><img src="/pmc/articles/instance/3633842/bin/pone.0062845.g002.gif" src-large="/pmc/articles/instance/3633842/bin/pone.0062845.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3633842/" ref="ordinalpos=4306&amp;ncbi_uid=4834035&amp;link_uid=PMC3633842" image-link="/pmc/articles/PMC3633842/figure/pone-0062845-g002/" class="imagepopup">Figure 2. Activation of IRF by cytosolic pattern recognition receptor <span class="highlight" style="background-color:">pathway</span>..  From: Decreased Pattern Recognition Receptor <span class="highlight" style="background-color:">Signaling</span>, Interferon-Signature, and Bactericidal/Permeability-Increasing Protein Gene Expression in Cord Blood of Term Low Birth Weight Human Newborns. </a></p><div class="supp"><p class="details"><div>The green nodes in this canonical <span class="highlight" style="background-color:">pathway</span> indicate the down-regulated genes in LBW newborns. The uncolored nodes represent the genes integrated by IPA from its knowledge base. The interferon stimulated genes in this <span class="highlight" style="background-color:">pathway</span> are marked as red diamond ().</div></p></div><div class="aux"><div class="resc">Vikas Vikram Singh, et al. PLoS One. 2013;8(4):e62845.</div><p class="links"><a class="dblinks" href="/pubmed/23626859" ref="ordinalpos=4306&amp;ncbi_uid=4834035&amp;link_uid=23626859">Citation</a><a class="dblinks" href="/pmc/articles/PMC3633842/" ref="ordinalpos=4306&amp;ncbi_uid=4834035&amp;link_uid=PMC3633842">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4307.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3570029/figure/F6/" ref="ordinalpos=4307&amp;ncbi_uid=4621795&amp;link_uid=PMC3570029" href="/pmc/articles/PMC3570029/"><img src="/pmc/articles/instance/3570029/bin/nihms435969f6.gif" src-large="/pmc/articles/instance/3570029/bin/nihms435969f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3570029/" ref="ordinalpos=4307&amp;ncbi_uid=4621795&amp;link_uid=PMC3570029" image-link="/pmc/articles/PMC3570029/figure/F6/" class="imagepopup">Fig. 6.  From: Role of VPO1, a newly identified heme-containing peroxidase, in ox-LDL induced endothelial cell apoptosis. </a></p><div class="supp"><p class="details"><div>The proposed <span class="highlight" style="background-color:">pathway</span> of VPO1 mediation of ox-LDL-induced endothelial cell apoptosis. Ox-LDL up-regulates the expression of NADPH oxidase subunit gp91<sup>phox</sup> and consequently increases intracellular ROS production and VPO1 expression as well as HOCl production. There is a positive feedback loop between VPO1/HOCl and the NADPH oxidase/ROS <span class="highlight" style="background-color:">pathway</span>, which in turn activates the p38 MAPK/caspase-3-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to mediate ox-LDL-induced endothelial cell apoptosis.</div></p></div><div class="aux"><div class="resc">Yong-Ping Bai, et al. Free Radic Biol Med. ;51(8):1492-1500.</div><p class="links"><a class="dblinks" href="/pubmed/21820048" ref="ordinalpos=4307&amp;ncbi_uid=4621795&amp;link_uid=21820048">Citation</a><a class="dblinks" href="/pmc/articles/PMC3570029/" ref="ordinalpos=4307&amp;ncbi_uid=4621795&amp;link_uid=PMC3570029">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4308.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3190381/figure/F7/" ref="ordinalpos=4308&amp;ncbi_uid=3509918&amp;link_uid=PMC3190381" href="/pmc/articles/PMC3190381/"><img src="/pmc/articles/instance/3190381/bin/DEV071464F7.gif" src-large="/pmc/articles/instance/3190381/bin/DEV071464F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3190381/" ref="ordinalpos=4308&amp;ncbi_uid=3509918&amp;link_uid=PMC3190381" image-link="/pmc/articles/PMC3190381/figure/F7/" class="imagepopup">Fig. 7. From: JAK/Stat <span class="highlight" style="background-color:">signaling</span> regulates heart precursor diversification in Drosophila. </a></p><div class="supp"><p class="details"><div><b>JAK/Stat <span class="highlight" style="background-color:">signaling</span> regulates phase 3 <i>tin</i> expression.</b> During St10, the JAK/Stat <span class="highlight" style="background-color:">pathway</span> is active in one region of each mesoderm segment (dark blue) and largely inactive in the opposing region (light blue). The Tin-responsive Stat92E mesoderm enhancer contains an SCBS (see Fig. S4 and Table S1 in the supplementary material) and directs segmental expression at St10 (Liu et al., 2009). Upd and Upd2 activate the JAK/Stat <span class="highlight" style="background-color:">pathway</span> and induce phosphorylation of maternally contributed Stat92E (M-Stat). Tin and M-Stat co-activate zygotic expression of <i>Stat92E</i>, which induces segmental expression of the E(spl)-C gene <i>HLHm5</i>. HLHm5 then represses <i>tin</i> to establish the phase 3 expression pattern in the dorsal mesoderm. <i>tin</i> expression is sustained in the region of the segment where JAK/Stat <span class="highlight" style="background-color:">signaling</span> is not active. Images on the right show Phase-2 and Phase-3 Tin expression. Embryos are oriented with anterior to the left.</div></p></div><div class="aux"><div class="resc">Aaron N. Johnson, et al. Development. 2011 November 1;138(21):4627-4638.</div><p class="links"><a class="dblinks" href="/pubmed/21965617" ref="ordinalpos=4308&amp;ncbi_uid=3509918&amp;link_uid=21965617">Citation</a><a class="dblinks" href="/pmc/articles/PMC3190381/" ref="ordinalpos=4308&amp;ncbi_uid=3509918&amp;link_uid=PMC3190381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4309.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3775871/figure/F5/" ref="ordinalpos=4309&amp;ncbi_uid=6451353&amp;link_uid=PMC3775871" href="/pmc/articles/PMC3775871/"><img src="/pmc/articles/instance/3775871/bin/nihms468897f5.gif" src-large="/pmc/articles/instance/3775871/bin/nihms468897f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3775871/" ref="ordinalpos=4309&amp;ncbi_uid=6451353&amp;link_uid=PMC3775871" image-link="/pmc/articles/PMC3775871/figure/F5/" class="imagepopup">Figure 5. The regulation of SOX9 by NF-B <span class="highlight" style="background-color:">signaling</span> is dependent on DNA methylation.  From: Epigenetic Regulation of SOX9 by the NF-?B <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Pancreatic Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div>(A, B) The enrichment of SOX9 promoter is dramatically enhanced by treatment of DNA demethylating agents 5AZA or Zebularine using either ChIP (A) or EMSA (B) assays.<br />(C) A proposed model for the function of NF-B <span class="highlight" style="background-color:">signaling</span> and SOX9 in pancreatic cancer cells. In non-invasive cells, the SOX9 promoter is methylated and SOX9 expression is silenced. In the invasive population, or CSCs, in which SOX9 promoter is demethylated, NF-B complex binds to SOX9 promoter and activates SOX9 expression, contributing to the invasiveness of pancreatic CSCs.</div></p></div><div class="aux"><div class="resc">Lei Sun, et al. Stem Cells. 2013 August;31(8):1454-1466.</div><p class="links"><a class="dblinks" href="/pubmed/23592398" ref="ordinalpos=4309&amp;ncbi_uid=6451353&amp;link_uid=23592398">Citation</a><a class="dblinks" href="/pmc/articles/PMC3775871/" ref="ordinalpos=4309&amp;ncbi_uid=6451353&amp;link_uid=PMC3775871">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4310.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4082426/figure/F6/" ref="ordinalpos=4310&amp;ncbi_uid=6342938&amp;link_uid=PMC4082426" href="/pmc/articles/PMC4082426/"><img src="/pmc/articles/instance/4082426/bin/1479-5876-12-167-6.gif" src-large="/pmc/articles/instance/4082426/bin/1479-5876-12-167-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4082426/" ref="ordinalpos=4310&amp;ncbi_uid=6342938&amp;link_uid=PMC4082426" image-link="/pmc/articles/PMC4082426/figure/F6/" class="imagepopup">Figure 6.  From: Sequential combination of decitabine and idarubicin synergistically enhances anti-leukemia effect followed by demethylating Wnt <span class="highlight" style="background-color:">pathway</span> inhibitor promoters and downregulating Wnt <span class="highlight" style="background-color:">pathway</span> nuclear target. </a></p><div class="supp"><p class="details"><div><b>Effect of sequential combination of DAC and IDA in inhibiting Wnt/-catenin <span class="highlight" style="background-color:">pathway</span> in AML cells in vitro and tumor cells from a subcutaneous AML mouse model. A</b>, The mRNA expression of inhibitors and downstream genes of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was measured by RT-PCR in control cultures, DAC treated cultures, IDA treated cultures and DAC followed by IDA treated cultures in the U937, HEL and SKM-1 cell lines. Expression was normalized in comparison with control cultures. The meanSD of four different experiments is shown. Statistical differences were calculated by one-way ANOVA. *<i>p</i>&lt;0.05, **<i>p</i>&lt;0.01, ***<i>p</i>&lt;0.001. <b>B</b>, Western blot analyze the protein expression of inhibitors and downstream genes of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in four cultures of AML cell lines and tumor cells from AML mouse model. <b>C</b>, Immunohistochemistry for -catenin in tumor tissues of subcutaneous AML mouse model. -catenin expression was more frequently observed in group treated with DAC combining with IDA than treated with DAC or IDA alone.</div></p></div><div class="aux"><div class="resc">Kongfei Li, et al. J Transl Med. 2014;12:167-167.</div><p class="links"><a class="dblinks" href="/pubmed/24923330" ref="ordinalpos=4310&amp;ncbi_uid=6342938&amp;link_uid=24923330">Citation</a><a class="dblinks" href="/pmc/articles/PMC4082426/" ref="ordinalpos=4310&amp;ncbi_uid=6342938&amp;link_uid=PMC4082426">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4311.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3591436/figure/pone-0053370-g006/" ref="ordinalpos=4311&amp;ncbi_uid=4713572&amp;link_uid=PMC3591436" href="/pmc/articles/PMC3591436/"><img src="/pmc/articles/instance/3591436/bin/pone.0053370.g006.gif" src-large="/pmc/articles/instance/3591436/bin/pone.0053370.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3591436/" ref="ordinalpos=4311&amp;ncbi_uid=4713572&amp;link_uid=PMC3591436" image-link="/pmc/articles/PMC3591436/figure/pone-0053370-g006/" class="imagepopup">Figure 6. Possible mechanisms of matrix metalloproteinase (MMP)-9 leading to claudin-5 degradation via the NF-B <span class="highlight" style="background-color:">pathway</span>..  From: Matrix Metalloproteinase-9 Leads to Claudin-5 Degradation via the NF-?B <span class="highlight" style="background-color:">Pathway</span> in BALB/c Mice with Eosinophilic Meningoencephalitis Caused by Angiostrongylus cantonensis. </a></p><div class="supp"><p class="details"><div>The activation of IB- and NF-B could upregulate MMP-9 production in <i>A. cantonensis</i>-induced leukocytes. Blocking IB- and NF-B <span class="highlight" style="background-color:">signaling</span> by MG132 could reduce MMP-9 production. Further, blocking MMP-9 activity by GM6001 could reduce claudin-5 degradation and blood-CSF barrier permeability during angiostrongyliasis meningoencephalitis. Therefore, we suggest that MMP-9 could cause claudin-5 degradation and promote leukocyte infiltration into CSF by the paracellular route during <i>A. cantonensis</i> infection in mouse choroid plexus.</div></p></div><div class="aux"><div class="resc">Ping-Sung Chiu, et al. PLoS One. 2013;8(3):e53370.</div><p class="links"><a class="dblinks" href="/pubmed/23505411" ref="ordinalpos=4311&amp;ncbi_uid=4713572&amp;link_uid=23505411">Citation</a><a class="dblinks" href="/pmc/articles/PMC3591436/" ref="ordinalpos=4311&amp;ncbi_uid=4713572&amp;link_uid=PMC3591436">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4312.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3169508/figure/F2/" ref="ordinalpos=4312&amp;ncbi_uid=3212034&amp;link_uid=PMC3169508" href="/pmc/articles/PMC3169508/"><img src="/pmc/articles/instance/3169508/bin/1752-0509-5-127-2.gif" src-large="/pmc/articles/instance/3169508/bin/1752-0509-5-127-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3169508/" ref="ordinalpos=4312&amp;ncbi_uid=3212034&amp;link_uid=PMC3169508" image-link="/pmc/articles/PMC3169508/figure/F2/" class="imagepopup">Figure 2.  From: Exploring <span class="highlight" style="background-color:">pathway</span> interactions in insulin resistant mouse liver. </a></p><div class="supp"><p class="details"><div><b>An indirect interaction between the Axon Guidance and Insulin <span class="highlight" style="background-color:">Signaling</span> pathways in the network for the comparison between HF and LF diet at t = 0</b>. Left: Network representation of the identified path between the two pathways, consisting of three proteins <i>Gsk3b</i>, <i>Sgk3 </i>and <i>Tsc1</i>. Right: The location of these proteins in the KEGG <span class="highlight" style="background-color:">pathway</span> diagrams. The newly found indirect interactions have been added in red.</div></p></div><div class="aux"><div class="resc">Thomas Kelder, et al. BMC Syst Biol. 2011;5:127-127.</div><p class="links"><a class="dblinks" href="/pubmed/21843341" ref="ordinalpos=4312&amp;ncbi_uid=3212034&amp;link_uid=21843341">Citation</a><a class="dblinks" href="/pmc/articles/PMC3169508/" ref="ordinalpos=4312&amp;ncbi_uid=3212034&amp;link_uid=PMC3169508">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4313.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3710316/figure/F1/" ref="ordinalpos=4313&amp;ncbi_uid=5316063&amp;link_uid=PMC3710316" href="/pmc/articles/PMC3710316/"><img src="/pmc/articles/instance/3710316/bin/jkst-2-e23203-g1.gif" src-large="/pmc/articles/instance/3710316/bin/jkst-2-e23203-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3710316/" ref="ordinalpos=4313&amp;ncbi_uid=5316063&amp;link_uid=PMC3710316" image-link="/pmc/articles/PMC3710316/figure/F1/" class="imagepopup">Figure. From: Why should we care about fly tumors?. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> The combined action of EGFR and JAK-STAT <span class="highlight" style="background-color:">signaling</span> results in tumor progression. (<b>A</b>) Epithelial growth in tissues with normal apico-basal cell polarity is controlled by the balanced activity of EGFR, JAK-STAT and other <span class="highlight" style="background-color:">signaling</span> pathways. Negative regulators such as <i>Socs36E</i> control <span class="highlight" style="background-color:">pathway</span> activity by modulating <span class="highlight" style="background-color:">signaling</span> output. (<b>B</b>) When EGFR is over activated (shown in bold), the epithelium proliferates excessively without necessarily causing metastasis. (<b>C</b>) Over proliferation and metastasis are promoted by the combined misregulation of EGFR and high JAK-STAT expression when the <span class="highlight" style="background-color:">signaling</span> balance is broken by the downregulation of <i>Socs36E</i> by <i>bantam</i> miRNA expression or by JNK mediated JAK-STAT activation.</div></p></div><div class="aux"><div class="resc">James Castelli-Gair Hombra, et al. JAKSTAT. 2013 April 1;2(2):e23203.</div><p class="links"><a class="dblinks" href="/pubmed/24058803" ref="ordinalpos=4313&amp;ncbi_uid=5316063&amp;link_uid=24058803">Citation</a><a class="dblinks" href="/pmc/articles/PMC3710316/" ref="ordinalpos=4313&amp;ncbi_uid=5316063&amp;link_uid=PMC3710316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4314.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2975271/figure/F9/" ref="ordinalpos=4314&amp;ncbi_uid=2567364&amp;link_uid=PMC2975271" href="/pmc/articles/PMC2975271/"><img src="/pmc/articles/instance/2975271/bin/nihms-242741-f0009.gif" src-large="/pmc/articles/instance/2975271/bin/nihms-242741-f0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2975271/" ref="ordinalpos=4314&amp;ncbi_uid=2567364&amp;link_uid=PMC2975271" image-link="/pmc/articles/PMC2975271/figure/F9/" class="imagepopup">Figure 9.  From: The p53 target Plk2 interacts with TSC proteins impacting mTOR <span class="highlight" style="background-color:">signaling</span>, tumor growth and chemosensitivity under hypoxic conditions. </a></p><div class="supp"><p class="details"><div>Schematic model for regulation of mTOR <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> through interaction of the p53 target Plk2 with TSC proteins. Our model proposes that Plk2 and TSC1 interact to facilitate TSC1/TSC2 restraint of mTOR. Decreased mTOR activity leads to decreased phosphorylation of p70S6K and consequent decrease in protein translation, cell growth and proliferation. These events are regulated by hypoxia and DNA damage.</div></p></div><div class="aux"><div class="resc">Elizabeth M. Matthew, et al. Cell Cycle. ;8(24):4168-4175.</div><p class="links"><a class="dblinks" href="/pubmed/20054236" ref="ordinalpos=4314&amp;ncbi_uid=2567364&amp;link_uid=20054236">Citation</a><a class="dblinks" href="/pmc/articles/PMC2975271/" ref="ordinalpos=4314&amp;ncbi_uid=2567364&amp;link_uid=PMC2975271">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4315.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC133668/figure/f9/" ref="ordinalpos=4315&amp;ncbi_uid=262137&amp;link_uid=PMC133668" href="/pmc/articles/PMC133668/"><img src="/pmc/articles/instance/133668/bin/mb0720973009.gif" src-large="/pmc/articles/instance/133668/bin/mb0720973009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC133668/" ref="ordinalpos=4315&amp;ncbi_uid=262137&amp;link_uid=PMC133668" image-link="/pmc/articles/PMC133668/figure/f9/" class="imagepopup">FIG. 9. From: Convergence of Multiple <span class="highlight" style="background-color:">Signaling</span> Cascades at Glycogen Synthase Kinase 3: Edg Receptor-Mediated Phosphorylation and Inactivation by Lysophosphatidic Acid through a Protein Kinase C-Dependent Intracellular <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Convergence of multiple intracellular <span class="highlight" style="background-color:">signaling</span> pathways at GSK-3.</div></p></div><div class="aux"><div class="resc">Xianjun Fang, et al. Mol Cell Biol. 2002 April;22(7):2099-2110.</div><p class="links"><a class="dblinks" href="/pubmed/11884598" ref="ordinalpos=4315&amp;ncbi_uid=262137&amp;link_uid=11884598">Citation</a><a class="dblinks" href="/pmc/articles/PMC133668/" ref="ordinalpos=4315&amp;ncbi_uid=262137&amp;link_uid=PMC133668">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4316.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3105527/figure/F1/" ref="ordinalpos=4316&amp;ncbi_uid=2998605&amp;link_uid=PMC3105527" href="/pmc/articles/PMC3105527/"><img src="/pmc/articles/instance/3105527/bin/nihms295049f1.gif" src-large="/pmc/articles/instance/3105527/bin/nihms295049f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3105527/" ref="ordinalpos=4316&amp;ncbi_uid=2998605&amp;link_uid=PMC3105527" image-link="/pmc/articles/PMC3105527/figure/F1/" class="imagepopup">Figure 1.  From: Computational Modeling of Mammalian <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>Mathematical approaches to model a <span class="highlight" style="background-color:">signaling</span> network.<br />a. A hypothetical <span class="highlight" style="background-color:">signaling</span> network, which transfers information from cytosolic enzymes to transcription factors to regulate gene expression.<br />b. The ordinary differential equation represents the phosphorylation and dephosphorylation of the Y protein. The partial differential equation models the effects of molecular diffusion and biochemical reactions with spatial dependence.<br />c. Measurements of the <span class="highlight" style="background-color:">signaling</span> network and phenotypic output (possibly measured through flow cytometry) are analyzed together to form a reduced space partial least squares regression model.<br />d. Network-based approaches. Network component analysis determines the linear weight of transcription factors on gene regulation. Extreme <span class="highlight" style="background-color:">pathway</span> analysis gives the minimal set of pathways that characterize the functional <span class="highlight" style="background-color:">signaling</span> in a network.</div></p></div><div class="aux"><div class="resc">Jacob J Hughey, et al. Wiley Interdiscip Rev Syst Biol Med. ;2(2):194-209.</div><p class="links"><a class="dblinks" href="/pubmed/20836022" ref="ordinalpos=4316&amp;ncbi_uid=2998605&amp;link_uid=20836022">Citation</a><a class="dblinks" href="/pmc/articles/PMC3105527/" ref="ordinalpos=4316&amp;ncbi_uid=2998605&amp;link_uid=PMC3105527">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4317.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3236571/figure/fig07/" ref="ordinalpos=4317&amp;ncbi_uid=3731664&amp;link_uid=PMC3236571" href="/pmc/articles/PMC3236571/"><img src="/pmc/articles/instance/3236571/bin/ajcr0002-0045-f7.gif" src-large="/pmc/articles/instance/3236571/bin/ajcr0002-0045-f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3236571/" ref="ordinalpos=4317&amp;ncbi_uid=3731664&amp;link_uid=PMC3236571" image-link="/pmc/articles/PMC3236571/figure/fig07/" class="imagepopup">Figure 7.  From: TRAIL receptor <span class="highlight" style="background-color:">signaling</span> and therapeutic option in bone tumors: the trap of the bone microenvironment. </a></p><div class="supp"><p class="details"><div>Resistance mechanisms in the TRAIL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Numerous resistance mechanisms developed by cancer cells have been described at different levels of the intrinsic and extrinsic TRAIL <span class="highlight" style="background-color:">signaling</span> pathways. DcRs: Decoy Receptors, DD: Death Domain, DED: Death Effector Domains, FADD: Fas-Associated protein with Death Domain, c-FLIP: cellular FLICE-inhibitory Protein, Bid: BH3-interacting domain death agonist, BCL-Xl: BCL-extra large, MCL1: Myeloid cell leukemia sequence 1, IAPs: inhibitors of apoptosis.</div></p></div><div class="aux"><div class="resc">Galle Picarda, et al. Am J Cancer Res. 2012;2(1):45-64.</div><p class="links"><a class="dblinks" href="/pubmed/22206045" ref="ordinalpos=4317&amp;ncbi_uid=3731664&amp;link_uid=22206045">Citation</a><a class="dblinks" href="/pmc/articles/PMC3236571/" ref="ordinalpos=4317&amp;ncbi_uid=3731664&amp;link_uid=PMC3236571">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4318.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4114108/figure/F3/" ref="ordinalpos=4318&amp;ncbi_uid=6427113&amp;link_uid=PMC4114108" href="/pmc/articles/PMC4114108/"><img src="/pmc/articles/instance/4114108/bin/1477-7819-12-207-3.gif" src-large="/pmc/articles/instance/4114108/bin/1477-7819-12-207-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4114108/" ref="ordinalpos=4318&amp;ncbi_uid=6427113&amp;link_uid=PMC4114108" image-link="/pmc/articles/PMC4114108/figure/F3/" class="imagepopup">Figure 3.  From: Norcantharidin induces growth inhibition and apoptosis of glioma cells by blocking the Raf/MEK/ERK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>NCTD inhibits Raf/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in U87 (A) and C6 (B) cells.</b> Cells were treated with the indicated concentrations of NCTD for 24 h. After treatment, whole cell protein extracts were prepared, and equal amounts of total protein were resolved on SDS-PAGE gels. Western blotting analysis was performed using specific antibodies against the indicated proteins.</div></p></div><div class="aux"><div class="resc">Jie Zheng, et al. World J Surg Oncol. 2014;12:207-207.</div><p class="links"><a class="dblinks" href="/pubmed/25022352" ref="ordinalpos=4318&amp;ncbi_uid=6427113&amp;link_uid=25022352">Citation</a><a class="dblinks" href="/pmc/articles/PMC4114108/" ref="ordinalpos=4318&amp;ncbi_uid=6427113&amp;link_uid=PMC4114108">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4319.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2879143/figure/fig03/" ref="ordinalpos=4319&amp;ncbi_uid=2513023&amp;link_uid=PMC2879143" href="/pmc/articles/PMC2879143/"><img src="/pmc/articles/instance/2952290/bin/pmic0010-1316-f3.gif" src-large="/pmc/articles/instance/2952290/bin/pmic0010-1316-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2879143/" ref="ordinalpos=4319&amp;ncbi_uid=2513023&amp;link_uid=PMC2879143" image-link="/pmc/articles/PMC2879143/figure/fig03/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Pathway</span> analysis of dilated cardiomyopathy using global proteomic profiling and enrichment maps. </a></p><div class="supp"><p class="details"><div>Activation of apoptotic <span class="highlight" style="background-color:">signaling</span> <i>via</i> caspase 3 and gelsolin. Consecutive zoom-ins of the Enrichment Map gene set cluster representing terms related to cellular apoptosis. Individual protein nodes represented in the <span class="highlight" style="background-color:">pathway</span> network are shown for the caspase neighborhood. Proteins are colored according to the expression ratio of condition <i>versus</i> control at the early (inner circle) and mid-stages (outer circle) of disease.</div></p></div><div class="aux"><div class="resc">Ruth Isserlin, et al. Proteomics. 2010 March;10(6):1316-1327.</div><p class="links"><a class="dblinks" href="/pubmed/20127684" ref="ordinalpos=4319&amp;ncbi_uid=2513023&amp;link_uid=20127684">Citation</a><a class="dblinks" href="/pmc/articles/PMC2879143/" ref="ordinalpos=4319&amp;ncbi_uid=2513023&amp;link_uid=PMC2879143">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4320.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1190364/figure/f4/" ref="ordinalpos=4320&amp;ncbi_uid=591751&amp;link_uid=PMC1190364" href="/pmc/articles/PMC1190364/"><img src="/pmc/articles/instance/1190364/bin/zmb0150552230004.gif" src-large="/pmc/articles/instance/1190364/bin/zmb0150552230004.jpg" alt="FIG. 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1190364/" ref="ordinalpos=4320&amp;ncbi_uid=591751&amp;link_uid=PMC1190364" image-link="/pmc/articles/PMC1190364/figure/f4/" class="imagepopup">FIG. 4. From: Components of the ESCRT <span class="highlight" style="background-color:">Pathway</span>, DFG16, and YGR122w Are Required for Rim101 To Act as a Corepressor with Nrg1 at the Negative Regulatory Element of the DIT1 Gene of Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Rim101-dependent, 3NRE22D-mediated repression requires the RIM <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) The wild-type (WT) strain or the indicated <i>rim</i> deletion strains from the yeast deletion collection, as denoted on the left-hand side of the panel, were transformed with pLGn+3NRE22D, and -galactosidase expression was monitored by a colony overlay assay. (B) Expression of Rim101(1-531)0.3HA restores NRE22D-mediated repression to a <i>rim13</i> strain. Wild-type, <i>rim101</i>, <i>RIM101</i>(<i>1</i>-<i>531</i>).<i>3HA-HIS3MX6</i>, <i>rim13</i>, and <i>rim13</i> <i>RIM101</i>(<i>1</i>-<i>531</i>)<i>0.3HA-HIS3MX6</i> strains, as denoted on the right-hand side of the panel, were transformed with pLG312n (no insert) or pLGn+3NRE22D, and -galactosidase expression was monitored by a colony overlay assay.</div></p></div><div class="aux"><div class="resc">Karen Rothfels, et al. Mol Cell Biol. 2005 August;25(15):6772-6788.</div><p class="links"><a class="dblinks" href="/pubmed/16024810" ref="ordinalpos=4320&amp;ncbi_uid=591751&amp;link_uid=16024810">Citation</a><a class="dblinks" href="/pmc/articles/PMC1190364/" ref="ordinalpos=4320&amp;ncbi_uid=591751&amp;link_uid=PMC1190364">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4321.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3361581/figure/F1/" ref="ordinalpos=4321&amp;ncbi_uid=4835471&amp;link_uid=PMC3361581" href="/pmc/articles/PMC3361581/"><img src="/pmc/articles/instance/3361581/bin/nihms357034f1.gif" src-large="/pmc/articles/instance/3361581/bin/nihms357034f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3361581/" ref="ordinalpos=4321&amp;ncbi_uid=4835471&amp;link_uid=PMC3361581" image-link="/pmc/articles/PMC3361581/figure/F1/" class="imagepopup">Figure 1.  From: Differential age- and disease-related effects on the expression of genes related to the arachidonic acid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in schizophrenia. </a></p><div class="supp"><p class="details"><div>Schematic of depiction of arachidonic acid <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> highlighting genes measured in this study. <i>PTGS1</i>, prostaglandin-endoperoxide synthase 1; <i>PTGS2</i>, prostaglandin-endoperoxide synthase 2; <i>PLA2</i>, phospholipase A2; <i>CYPx</i>, cytochrome P450 family member; <i>ALOX5</i>, arachidonate 5-lipoxygenase; <i>ALOX12</i>, arachidonate 12-lipoxygenase; <i>ALOX15</i>, arachidonate 15-lipoxygenase; <i>TBXA2R</i>, thromboxane A2 (TXA<sub>2</sub>) receptor; <i>PTGIR</i>, prostaglandin I2 (PGI<sub>2</sub>) receptor; <i>PTGDR</i>, prostaglandin D2 (PGD<sub>2</sub>) receptor; <i>PTGFR</i>, prostaglandin F2 (PGF<sub>2</sub>) receptor; <i>PTGER1-4</i>, prostaglandin E2 (PGE<sub>2</sub>) receptors 14.</div></p></div><div class="aux"><div class="resc">Bin Tang, et al. Psychiatry Res. ;196(2-3):201-206.</div><p class="links"><a class="dblinks" href="/pubmed/22397921" ref="ordinalpos=4321&amp;ncbi_uid=4835471&amp;link_uid=22397921">Citation</a><a class="dblinks" href="/pmc/articles/PMC3361581/" ref="ordinalpos=4321&amp;ncbi_uid=4835471&amp;link_uid=PMC3361581">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4322.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2192628/figure/F3/" ref="ordinalpos=4322&amp;ncbi_uid=1192988&amp;link_uid=PMC2192628" href="/pmc/articles/PMC2192628/"><img src="/pmc/articles/instance/2192628/bin/nihms35772f3.gif" src-large="/pmc/articles/instance/2192628/bin/nihms35772f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2192628/" ref="ordinalpos=4322&amp;ncbi_uid=1192988&amp;link_uid=PMC2192628" image-link="/pmc/articles/PMC2192628/figure/F3/" class="imagepopup">Figure 3. RCAS-Noggin blocks the functionality of the BMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: <span class="highlight" style="background-color:">Signaling</span> by Bone Morphogenetic Proteins directs formation of an ectodermal <span class="highlight" style="background-color:">signaling</span> center that regulates craniofacial development. </a></p><div class="supp"><p class="details"><div>Safranin O/Fast Green staining of a coronal section of day 13 chick embryo (~HH39) infected at ~HH22 in the right side of the mandibular primordial illustrates the absence of bone and cartilage (*) on this side. The lower jaw skeleton is outlined on the other side of the face. prominence. mc, mandibular cartilage; sa, sarangular bone; sp, splenial bone; dt, dentary bone. Scale bar: 1mm.</div></p></div><div class="aux"><div class="resc">Silvia Foppiano, et al. Dev Biol. ;312(1):103-114.</div><p class="links"><a class="dblinks" href="/pubmed/18028903" ref="ordinalpos=4322&amp;ncbi_uid=1192988&amp;link_uid=18028903">Citation</a><a class="dblinks" href="/pmc/articles/PMC2192628/" ref="ordinalpos=4322&amp;ncbi_uid=1192988&amp;link_uid=PMC2192628">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4323.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4350433/figure/fig08/" ref="ordinalpos=4323&amp;ncbi_uid=7273686&amp;link_uid=PMC4350433" href="/pmc/articles/PMC4350433/"><img src="/pmc/articles/instance/4350433/bin/tpj0077-0119-f8.gif" src-large="/pmc/articles/instance/4350433/bin/tpj0077-0119-f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4350433/" ref="ordinalpos=4323&amp;ncbi_uid=7273686&amp;link_uid=PMC4350433" image-link="/pmc/articles/PMC4350433/figure/fig08/" class="imagepopup">Figure 8.  From: AtRH57, a DEAD-box RNA helicase, is involved in feedback inhibition of glucose-mediated abscisic acid accumulation during seedling development and additively affects pre-ribosomal RNA processing with high glucose. </a></p><div class="supp"><p class="details"><div>A schematic representation of glucose (Glc) <span class="highlight" style="background-color:">signaling</span> wherein AtRH57 acts as a crucial player in the feedback inhibition of ABA accumulation within the sugar-mediated ABA <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> during Arabidopsis seed germination and early seedling growth. <i>HXK1</i>, hexokinase1; ABA, abscisic acid; ABA biosynthesis genes: <i>ABA2</i>,<i>ABA3</i>, and <i>AAO3</i>; ABA <span class="highlight" style="background-color:">signaling</span> genes: <i>ABI3</i>,<i>ABI4</i>, and <i>ABI5</i>.</div></p></div><div class="aux"><div class="resc">Yi-Feng Hsu, et al. Plant J. 2014 January;77(1):119-135.</div><p class="links"><a class="dblinks" href="/pubmed/24176057" ref="ordinalpos=4323&amp;ncbi_uid=7273686&amp;link_uid=24176057">Citation</a><a class="dblinks" href="/pmc/articles/PMC4350433/" ref="ordinalpos=4323&amp;ncbi_uid=7273686&amp;link_uid=PMC4350433">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4324.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3870656/figure/fig2/" ref="ordinalpos=4324&amp;ncbi_uid=5641368&amp;link_uid=PMC3870656" href="/pmc/articles/PMC3870656/"><img src="/pmc/articles/instance/3870656/bin/ISRN.INFLAMMATION2013-512103.002.gif" src-large="/pmc/articles/instance/3870656/bin/ISRN.INFLAMMATION2013-512103.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3870656/" ref="ordinalpos=4324&amp;ncbi_uid=5641368&amp;link_uid=PMC3870656" image-link="/pmc/articles/PMC3870656/figure/fig2/" class="imagepopup">Figure 2.  From: The Enigmatic Cytokine Oncostatin M and Roles in Disease. </a></p><div class="supp"><p class="details"><div>OSM receptors and <span class="highlight" style="background-color:">signaling</span>. A schematic representation of signal transduction initiated through the OSMR/gp130 receptor complex. While several gp130-utilizing cytokines activate JAK/STAT and MAPK <span class="highlight" style="background-color:">signaling</span> pathways in connective tissue cells, we observe that OSM uniquely activates additional <span class="highlight" style="background-color:">signaling</span> intermediates including STAT5, STAT6, the PI3K/Akt <span class="highlight" style="background-color:">pathway</span>, and the novel PKC isoform PKC delta (PKCd) in fibroblasts. OSM induces IL-4Ralpha and IL-1R (modified from Fritz [39]).</div></p></div><div class="aux"><div class="resc">Carl D. Richards. ISRN Inflamm. 2013;2013:512103.</div><p class="links"><a class="dblinks" href="/pubmed/24381786" ref="ordinalpos=4324&amp;ncbi_uid=5641368&amp;link_uid=24381786">Citation</a><a class="dblinks" href="/pmc/articles/PMC3870656/" ref="ordinalpos=4324&amp;ncbi_uid=5641368&amp;link_uid=PMC3870656">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4325.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3610211/figure/F1/" ref="ordinalpos=4325&amp;ncbi_uid=4746683&amp;link_uid=PMC3610211" href="/pmc/articles/PMC3610211/"><img src="/pmc/articles/instance/3610211/bin/2040-2392-4-5-1.gif" src-large="/pmc/articles/instance/3610211/bin/2040-2392-4-5-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3610211/" ref="ordinalpos=4325&amp;ncbi_uid=4746683&amp;link_uid=PMC3610211" image-link="/pmc/articles/PMC3610211/figure/F1/" class="imagepopup">Figure 1.  From: Lack of association of rare functional variants in TSC1/TSC2 genes with autism spectrum disorder. </a></p><div class="supp"><p class="details"><div><b>A model of TSC/mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> at the neuronal synapse.</b> Tuberin (TSC2) and hamartin (TSC1) act as central players in mTORC1 <span class="highlight" style="background-color:">signaling</span> to regulate neuronal/synaptic activity. Neurotrophic (NT) factors such as BDNF, as well as synaptic activity initiate mTORC1 <span class="highlight" style="background-color:">signaling</span> via receptors (Rc) including neurotrophic Rc and glutamate receptors including NMDA Rc and mGluR Rc. The mTOR candidate genes resequenced in the study are shown in red. Ub, ubiquitin; P, phosphorylation.</div></p></div><div class="aux"><div class="resc">Samira Bahl, et al. Mol Autism. 2013;4:5-5.</div><p class="links"><a class="dblinks" href="/pubmed/23514105" ref="ordinalpos=4325&amp;ncbi_uid=4746683&amp;link_uid=23514105">Citation</a><a class="dblinks" href="/pmc/articles/PMC3610211/" ref="ordinalpos=4325&amp;ncbi_uid=4746683&amp;link_uid=PMC3610211">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4326.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4037577/figure/fig3/" ref="ordinalpos=4326&amp;ncbi_uid=6197723&amp;link_uid=PMC4037577" href="/pmc/articles/PMC4037577/"><img src="/pmc/articles/instance/4037577/bin/MI2014-978463.003.gif" src-large="/pmc/articles/instance/4037577/bin/MI2014-978463.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4037577/" ref="ordinalpos=4326&amp;ncbi_uid=6197723&amp;link_uid=PMC4037577" image-link="/pmc/articles/PMC4037577/figure/fig3/" class="imagepopup">Figure 3.  From: Obesity, Inflammation, and Lung Injury (OILI): The Good. </a></p><div class="supp"><p class="details"><div>The anti-inflammatory mechanism of omentin. Omentin activates AMPK, which further blocks E-selection and reduces endothelial inflammation. AMPK also activates eNOS, which has vasodilation effect and blocks JNK <span class="highlight" style="background-color:">signaling</span>. JNK activates inflammation through TNF<i></i> mediated COX2 effect. Moreover, omentin inhibits NF-<i></i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and thus inhibits inflammation. Under obese state, the production of omentin is lower which is associated with worse proinflammation and possible lung injury.</div></p></div><div class="aux"><div class="resc">Cheryl Wang. Mediators Inflamm. 2014;2014:978463.</div><p class="links"><a class="dblinks" href="/pubmed/24899788" ref="ordinalpos=4326&amp;ncbi_uid=6197723&amp;link_uid=24899788">Citation</a><a class="dblinks" href="/pmc/articles/PMC4037577/" ref="ordinalpos=4326&amp;ncbi_uid=6197723&amp;link_uid=PMC4037577">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4327.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3814570/figure/ppat-1003661-g001/" ref="ordinalpos=4327&amp;ncbi_uid=5476960&amp;link_uid=PMC3814570" href="/pmc/articles/PMC3814570/"><img src="/pmc/articles/instance/3814570/bin/ppat.1003661.g001.gif" src-large="/pmc/articles/instance/3814570/bin/ppat.1003661.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3814570/" ref="ordinalpos=4327&amp;ncbi_uid=5476960&amp;link_uid=PMC3814570" image-link="/pmc/articles/PMC3814570/figure/ppat-1003661-g001/" class="imagepopup">Figure 1. Environmental cues sensed by C. albicans..  From: Sensing of the Microbial Neighborhood by Candida albicans. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) Schematic representation of how <i>C. albicans</i> morphology and biofilm formation is regulated by quorum sensing and <span class="highlight" style="background-color:">signaling</span> with other microbial species. (<b>B</b>) Farnesol and 3-oxo-C(12)-homoserine lactone both act on the Ras1 <span class="highlight" style="background-color:">pathway</span> to inhibit the yeast-to-hyphal transition by inhibiting Cyr1 and cAMP <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Emily M. Mallick, et al. PLoS Pathog. 2013 October;9(10):e1003661.</div><p class="links"><a class="dblinks" href="/pubmed/24204254" ref="ordinalpos=4327&amp;ncbi_uid=5476960&amp;link_uid=24204254">Citation</a><a class="dblinks" href="/pmc/articles/PMC3814570/" ref="ordinalpos=4327&amp;ncbi_uid=5476960&amp;link_uid=PMC3814570">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4328.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3501582/figure/F2/" ref="ordinalpos=4328&amp;ncbi_uid=5496605&amp;link_uid=PMC3501582" href="/pmc/articles/PMC3501582/"><img src="/pmc/articles/instance/3501582/bin/nihms409189f2.gif" src-large="/pmc/articles/instance/3501582/bin/nihms409189f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3501582/" ref="ordinalpos=4328&amp;ncbi_uid=5496605&amp;link_uid=PMC3501582" image-link="/pmc/articles/PMC3501582/figure/F2/" class="imagepopup">Fig. 2.  From: Protective actions of vitamin D in UVB induced skin cancer. </a></p><div class="supp"><p class="details"><div>The hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the absence of Shh, Ptch 1 suppresses <span class="highlight" style="background-color:">signaling</span> by smoothened (Smo). Binding of Shh to Ptch 1 relieves this inhibition. Activation of Smo leads to the activation and translocation of transcription factors of the Gli family into the nucleus, with subsequent changes in gene expression.</div></p></div><div class="aux"><div class="resc">Daniel D. Bikle. Photochem Photobiol Sci. ;11(12):10.1039/c2pp25251a.</div><p class="links"><a class="dblinks" href="/pubmed/22990497" ref="ordinalpos=4328&amp;ncbi_uid=5496605&amp;link_uid=22990497">Citation</a><a class="dblinks" href="/pmc/articles/PMC3501582/" ref="ordinalpos=4328&amp;ncbi_uid=5496605&amp;link_uid=PMC3501582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4329.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3125348/figure/F2/" ref="ordinalpos=4329&amp;ncbi_uid=3044646&amp;link_uid=PMC3125348" href="/pmc/articles/PMC3125348/"><img src="/pmc/articles/instance/3125348/bin/1471-2229-11-88-2.gif" src-large="/pmc/articles/instance/3125348/bin/1471-2229-11-88-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3125348/" ref="ordinalpos=4329&amp;ncbi_uid=3044646&amp;link_uid=PMC3125348" image-link="/pmc/articles/PMC3125348/figure/F2/" class="imagepopup">Figure 2.  From: Prospecting for Genes involved in transcriptional regulation of plant defenses, a bioinformatics approach. </a></p><div class="supp"><p class="details"><div><b>Visual representation of the JA/SA/ET biosynthesis and <span class="highlight" style="background-color:">signaling</span> pathways</b>. Dark green boxes, MAPK kinases leading from flagellin to defense genes; red boxes, genes within the SA biosynthesis <span class="highlight" style="background-color:">pathway</span>; purple boxes, MAPK kinases leading to repression of SA and induction of JA defense genes; yellow boxes, genes involved in JA biosynthesis; light blue boxes, genes involved in ET biosynthesis; pink boxes, genes involved in ET <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Marcel C van Verk, et al. BMC Plant Biol. 2011;11:88-88.</div><p class="links"><a class="dblinks" href="/pubmed/21595873" ref="ordinalpos=4329&amp;ncbi_uid=3044646&amp;link_uid=21595873">Citation</a><a class="dblinks" href="/pmc/articles/PMC3125348/" ref="ordinalpos=4329&amp;ncbi_uid=3044646&amp;link_uid=PMC3125348">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4330.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3005603/figure/F5/" ref="ordinalpos=4330&amp;ncbi_uid=3603646&amp;link_uid=PMC3005603" href="/pmc/articles/PMC3005603/"><img src="/pmc/articles/instance/3005603/bin/DEV051680F5.gif" src-large="/pmc/articles/instance/3005603/bin/DEV051680F5.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3005603/" ref="ordinalpos=4330&amp;ncbi_uid=3603646&amp;link_uid=PMC3005603" image-link="/pmc/articles/PMC3005603/figure/F5/" class="imagepopup">Fig. 5. From: FGF10 controls the patterning of the tracheal cartilage rings via Shh. </a></p><div class="supp"><p class="details"><div><b>Inactivation of <i>Fgf10</i> does not affect the <i>Bmp4/Sox9</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> nor -catenin activation in the epithelium.</b> (<b>A</b>,<b>B</b>) Whole-mount in situ hybridization for <i>Sox9</i> in E13.5 wild-type (A) and <i>Fgf10<sup>/</sup></i> (B) tracheas. (<b>C</b>,<b>D</b>) Whole-mount in situ hybridization for <i>Bmp4</i> in E13.5 wild-type (C) and <i>Fgf10<sup>/</sup></i> (D) tracheas. (<b>E</b>,<b>F</b>) -Catenin activation in wild-type (E) and <i>Fgf10<sup>/</sup></i> (F) tracheas using the <i>Topgal</i> reported line, a <i>lacZ</i> reporter for -catenin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Frdric G. Sala, et al. Development. 2011 January 15;138(2):273-282.</div><p class="links"><a class="dblinks" href="/pubmed/21148187" ref="ordinalpos=4330&amp;ncbi_uid=3603646&amp;link_uid=21148187">Citation</a><a class="dblinks" href="/pmc/articles/PMC3005603/" ref="ordinalpos=4330&amp;ncbi_uid=3603646&amp;link_uid=PMC3005603">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4331.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3494627/figure/F3/" ref="ordinalpos=4331&amp;ncbi_uid=4381609&amp;link_uid=PMC3494627" href="/pmc/articles/PMC3494627/"><img src="/pmc/articles/instance/3494627/bin/onci-1-1146-g3.gif" src-large="/pmc/articles/instance/3494627/bin/onci-1-1146-g3.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3494627/" ref="ordinalpos=4331&amp;ncbi_uid=4381609&amp;link_uid=PMC3494627" image-link="/pmc/articles/PMC3494627/figure/F3/" class="imagepopup">Figure. From: A model for cancer-suppressive inflammation. </a></p><div class="supp"><p class="details"><div><b>Figure 3.</b> IL-1RI and TLRs share a common intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Mature IL-1 and IL-1 mediate their functions by binding to the same IL-1 receptor I (IL-1RI) which is present on the surface of most cells (1a). Pathogen-associated molecular patterns (PAMP) and damage/danger-associated molecular patterns (DAMP) bind to Toll-like receptors (TLRs) (1b). IL-1RI and TLRs share a common intracellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (2), which leads to activation of NFB and hence to the production of pro-inflammatory cytokines, such as IL-1, IL-1, IL-6 and TNF (3). Therefore, IL-1 and IL-1 function within positive feedback loops in inflammation. Because of this common <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, IL-1 and IL-1 operate as natural adjuvants, mimicking the detection of microbial products by TLRs. MyD88, Myeloid Differentiation primary response gene 88; TRAF6, TNF receptor associated factor 6.</div></p></div><div class="aux"><div class="resc">Ole Audun Werner Haabeth, et al. Oncoimmunology. 2012 October 1;1(7):1146-1155.</div><p class="links"><a class="dblinks" href="/pubmed/23170261" ref="ordinalpos=4331&amp;ncbi_uid=4381609&amp;link_uid=23170261">Citation</a><a class="dblinks" href="/pmc/articles/PMC3494627/" ref="ordinalpos=4331&amp;ncbi_uid=4381609&amp;link_uid=PMC3494627">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4332.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3274382/figure/F1/" ref="ordinalpos=4332&amp;ncbi_uid=4414352&amp;link_uid=PMC3274382" href="/pmc/articles/PMC3274382/"><img src="/pmc/articles/instance/3274382/bin/nihms344963f1.gif" src-large="/pmc/articles/instance/3274382/bin/nihms344963f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3274382/" ref="ordinalpos=4332&amp;ncbi_uid=4414352&amp;link_uid=PMC3274382" image-link="/pmc/articles/PMC3274382/figure/F1/" class="imagepopup">Figure 1.  From: Interferon-stimulated genes and their antiviral effector functions. </a></p><div class="supp"><p class="details"><div>Diverse roles for ISGs in the IFN antiviral <span class="highlight" style="background-color:">pathway</span>. Incoming viruses are sensed by pattern reecognition receptors (PRR), leading to activation of interferon regulatory factors (IRFs) and transcriptional induction of IFNs. Antiviral IFNs signal through the JAK/STAT <span class="highlight" style="background-color:">pathway</span> to induce ISG production. ISGs can also be directly induced by some IRFs in an IFN-independent <span class="highlight" style="background-color:">pathway</span> (thin blue arrow). Some ISGs function to block virus replication (thick red bars), while others have the ability to promote or enhance replication of certain viruses (green arrow). A subset of ISGs are themselves components of the IFN <span class="highlight" style="background-color:">pathway</span> or promote its <span class="highlight" style="background-color:">signaling</span> (red dotted arrows). IFN also induces several negative regulators which can target PRR, IRFs, or JAK/STAT to dampen the response (thin red bars).</div></p></div><div class="aux"><div class="resc">John W. Schoggins, et al. Curr Opin Virol. ;1(6):519-525.</div><p class="links"><a class="dblinks" href="/pubmed/22328912" ref="ordinalpos=4332&amp;ncbi_uid=4414352&amp;link_uid=22328912">Citation</a><a class="dblinks" href="/pmc/articles/PMC3274382/" ref="ordinalpos=4332&amp;ncbi_uid=4414352&amp;link_uid=PMC3274382">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4333.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2725943/figure/F7/" ref="ordinalpos=4333&amp;ncbi_uid=1931317&amp;link_uid=PMC2725943" href="/pmc/articles/PMC2725943/"><img src="/pmc/articles/instance/2725943/bin/1910fig7.gif" src-large="/pmc/articles/instance/2725943/bin/1910fig7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2725943/" ref="ordinalpos=4333&amp;ncbi_uid=1931317&amp;link_uid=PMC2725943" image-link="/pmc/articles/PMC2725943/figure/F7/" class="imagepopup">Figure 7. From: Cilium-independent regulation of Gli protein function by Sufu in Hedgehog <span class="highlight" style="background-color:">signaling</span> is evolutionarily conserved. </a></p><div class="supp"><p class="details"><div>A model of mammalian Hh <span class="highlight" style="background-color:">signaling</span>. Sufu plays a pivotal role in controlling Gli protein levels. Sufu protects full-length Gli2 and Gli3 proteins from Spop-mediated ubiquitination and complete degradation by the proteasome. In this way, Sufu functions as an adaptor to preserve a pool of Gli2 and Gli3 that can be readily converted into Gli activators and repressors. This aspect of Hh <span class="highlight" style="background-color:">signaling</span> is evolutionarily conserved and independent of the primary cilium. In contrast, the primary cilium is required for generating Gli repressors via limited proteolysis in the absence of Hh <span class="highlight" style="background-color:">signaling</span> and converting full-length Gli proteins into activators through unknown mechanisms upon Hh <span class="highlight" style="background-color:">pathway</span> activation. These events occurs downstream from Smo, which translocates to the primary cilium when Hh ligand binds to Ptch1 and removes it from the cilium. How other Hh <span class="highlight" style="background-color:">pathway</span> components or ciliary proteins particulate in cilium-dependent and cilium-independent activity needs to be investigated further.</div></p></div><div class="aux"><div class="resc">Miao-Hsueh Chen, et al. Genes Dev. 2009 August 15;23(16):1910-1928.</div><p class="links"><a class="dblinks" href="/pubmed/19684112" ref="ordinalpos=4333&amp;ncbi_uid=1931317&amp;link_uid=19684112">Citation</a><a class="dblinks" href="/pmc/articles/PMC2725943/" ref="ordinalpos=4333&amp;ncbi_uid=1931317&amp;link_uid=PMC2725943">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4334.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2996331/figure/pgen-1001231-g001/" ref="ordinalpos=4334&amp;ncbi_uid=2637905&amp;link_uid=PMC2996331" href="/pmc/articles/PMC2996331/"><img src="/pmc/articles/instance/2996331/bin/pgen.1001231.g001.gif" src-large="/pmc/articles/instance/2996331/bin/pgen.1001231.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2996331/" ref="ordinalpos=4334&amp;ncbi_uid=2637905&amp;link_uid=PMC2996331" image-link="/pmc/articles/PMC2996331/figure/pgen-1001231-g001/" class="imagepopup">Figure 1. Outline of the experimental design..  From: Identification of Y-Box Binding Protein 1 As a Core Regulator of MEK/ERK <span class="highlight" style="background-color:">Pathway</span>-Dependent Gene Signatures in Colorectal Cancer Cells. </a></p><div class="supp"><p class="details"><div>Outline of the experimental design for exploring the effects of individual elements of the RTK/RAS/MEK/ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> on the global transcription pattern in colon carcinoma cells. (A) Components of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and their modulation by inhibitors. Arrows indicate activation, ---o, inhibition. (B) Flow chart of wet lab and computational analysis.</div></p></div><div class="aux"><div class="resc">Karsten Jrchott, et al. PLoS Genet. 2010 December;6(12):e1001231.</div><p class="links"><a class="dblinks" href="/pubmed/21170361" ref="ordinalpos=4334&amp;ncbi_uid=2637905&amp;link_uid=21170361">Citation</a><a class="dblinks" href="/pmc/articles/PMC2996331/" ref="ordinalpos=4334&amp;ncbi_uid=2637905&amp;link_uid=PMC2996331">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4335.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584651/figure/f6-ijo-40-04-1259/" ref="ordinalpos=4335&amp;ncbi_uid=4685633&amp;link_uid=PMC3584651" href="/pmc/articles/PMC3584651/"><img src="/pmc/articles/instance/3584651/bin/IJO-40-04-1259-g05.gif" src-large="/pmc/articles/instance/3584651/bin/IJO-40-04-1259-g05.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584651/" ref="ordinalpos=4335&amp;ncbi_uid=4685633&amp;link_uid=PMC3584651" image-link="/pmc/articles/PMC3584651/figure/f6-ijo-40-04-1259/" class="imagepopup">Figure 6.  From: NVP-BKM120, a novel PI3K inhibitor, shows synergism with a STAT3 inhibitor in human gastric cancer cells harboring KRAS mutations. </a></p><div class="supp"><p class="details"><div>Schematic representation of RAS/PI3K/mTOR/STAT3 <span class="highlight" style="background-color:">signaling</span>. Constitutively active RAS can regulate the STAT3 <span class="highlight" style="background-color:">pathway</span> in dependent of the canonical PI3K/AKT <span class="highlight" style="background-color:">pathway</span>. In addition, RAS can directly activate mTOR via RSK-mediated phosphorylation of Raptor. mTOR as a kinase fully activates STAT3, which is essential for RAS oncogenic transformation. In RAS mutant cancer cells, therefore, despite suppression of PI3K by NVP-BKM120, the RAS <span class="highlight" style="background-color:">signaling</span> still enables to maintain oncogenic transformation through activation of the mTOR/STAT3 <span class="highlight" style="background-color:">signaling</span>. Combination of NVP-BKM120 and AG490 may result in a synergistic effect on gastric cancer harboring KRAS mutation.</div></p></div><div class="aux"><div class="resc">EUNJU PARK, et al. Int J Oncol. 2012 April;40(4):1259-1266.</div><p class="links"><a class="dblinks" href="/pubmed/22159814" ref="ordinalpos=4335&amp;ncbi_uid=4685633&amp;link_uid=22159814">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584651/" ref="ordinalpos=4335&amp;ncbi_uid=4685633&amp;link_uid=PMC3584651">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4336.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2988492/figure/F4/" ref="ordinalpos=4336&amp;ncbi_uid=3354698&amp;link_uid=PMC2988492" href="/pmc/articles/PMC2988492/"><img src="/pmc/articles/instance/2988492/bin/nihms245484f4.gif" src-large="/pmc/articles/instance/2988492/bin/nihms245484f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2988492/" ref="ordinalpos=4336&amp;ncbi_uid=3354698&amp;link_uid=PMC2988492" image-link="/pmc/articles/PMC2988492/figure/F4/" class="imagepopup">Fig. 4.  From: Serine496 of ?2 Subunit of L-type Ca2+ Channel Participates in Molecular Crosstalk between Activation of (Na++K+)-ATPase and the Channel. </a></p><div class="supp"><p class="details"><div>Potential elements of PKG-independent <span class="highlight" style="background-color:">signaling</span> mechanism underlying NKA-LTCC crosstalk. Binding of activator SSA412 to NKA not only enhances native catalytic activity of the enzyme, but also triggers signal to activate Src and Erk1/2. These <span class="highlight" style="background-color:">signaling</span> proteins would then relay the signals to phosphorylate both LTCC <sub>2</sub> Ser<sup>496</sup> and <sub>1</sub> Ser<sup>1928</sup>. Black arrows indicate the direction of activation of NKA-induced <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The dotted lines show undiscovered proteins that may participate in the activation of NKA-induced signal transduction <span class="highlight" style="background-color:">pathway</span>. Phosphorylation of <sub>1</sub> Ser<sup>1928</sup> may lead to up-regulation of LTCC activity. Phosphorylation of <sub>2</sub> Ser<sup>496</sup> may be linked to down-regulation of the channel.</div></p></div><div class="aux"><div class="resc">Kai Y. Xu, et al. Biochem Biophys Res Commun. ;402(2):319-323.</div><p class="links"><a class="dblinks" href="/pubmed/20937253" ref="ordinalpos=4336&amp;ncbi_uid=3354698&amp;link_uid=20937253">Citation</a><a class="dblinks" href="/pmc/articles/PMC2988492/" ref="ordinalpos=4336&amp;ncbi_uid=3354698&amp;link_uid=PMC2988492">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4337.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3473206/figure/F7/" ref="ordinalpos=4337&amp;ncbi_uid=5434298&amp;link_uid=PMC3473206" href="/pmc/articles/PMC3473206/"><img src="/pmc/articles/instance/3473206/bin/zee0111267520007.gif" src-large="/pmc/articles/instance/3473206/bin/zee0111267520007.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3473206/" ref="ordinalpos=4337&amp;ncbi_uid=5434298&amp;link_uid=PMC3473206" image-link="/pmc/articles/PMC3473206/figure/F7/" class="imagepopup">Fig. 7. From: Dual Role for Glucocorticoids in Cardiomyocyte Hypertrophy and Apoptosis. </a></p><div class="supp"><p class="details"><div>Microarray analysis of glucocorticoid-regulated genes in H9C2 cardiomyocytes. Global gene expression analysis was performed on RNA isolated from H9C2 cardiomyocytes treated with vehicle or 100 n<span style="font-variant: small-caps" class="small-caps">m</span> Dex for 6, 24, or 48 h in the presence or absence of serum. Genes significantly regulated by Dex were determined by ANOVA (<i>P</i> &lt; 0.01) and analyzed using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Analysis. A, The number of genes induced and repressed by Dex at each time point in cells cultured with or without serum. B, Venn diagram of Dex-regulated genes at 6-, 24-, and 48-h time points in cardiomyocytes cultured in the presence or absence of serum. C, The cardiac hypertrophy <span class="highlight" style="background-color:">signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> was significantly altered in serum-supplemented H9C2 cells treated with glucocorticoids. Shown are the 58 genes in this <span class="highlight" style="background-color:">pathway</span> that were regulated by Dex at the 48-h time point. D, The apoptosis <span class="highlight" style="background-color:">signaling</span> canonical <span class="highlight" style="background-color:">pathway</span> was significantly altered in serum-deprived H9C2 cells treated with glucocorticoids. Shown are the 15 genes in this <span class="highlight" style="background-color:">pathway</span> that were regulated by Dex at the 48-h time point. <i>Red</i> indicates increased expression, and <i>green</i> indicates decreased expression.</div></p></div><div class="aux"><div class="resc">Rongqin Ren, et al. Endocrinology. 2012 November;153(11):5346-5360.</div><p class="links"><a class="dblinks" href="/pubmed/22989630" ref="ordinalpos=4337&amp;ncbi_uid=5434298&amp;link_uid=22989630">Citation</a><a class="dblinks" href="/pmc/articles/PMC3473206/" ref="ordinalpos=4337&amp;ncbi_uid=5434298&amp;link_uid=PMC3473206">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4338.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2586592/figure/F4/" ref="ordinalpos=4338&amp;ncbi_uid=1634084&amp;link_uid=PMC2586592" href="/pmc/articles/PMC2586592/"><img src="/pmc/articles/instance/2586592/bin/nihms47660f4.gif" src-large="/pmc/articles/instance/2586592/bin/nihms47660f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2586592/" ref="ordinalpos=4338&amp;ncbi_uid=1634084&amp;link_uid=PMC2586592" image-link="/pmc/articles/PMC2586592/figure/F4/" class="imagepopup">Fig. 4.  From: ?E-catenin is not a significant regulator of ?-catenin <span class="highlight" style="background-color:">signaling</span> in the developing mammalian brain. </a></p><div class="supp"><p class="details"><div>Transition from E12.5 to E13.5 results in extensive hyperplasia without significant changes in expression of endogenous transcriptional targets of -catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Hyperplasia in E13.5 <i>E-catenin<sup>/</sup></i> brains. Total cells were isolated from E12.5 and E13.5 wild-type (WT) and <i>E-catenin</i><sup>/</sup> brains and counted using Coulter Counter. Data represent means  SD. n=3 to 5. Asterisk indicates statistically significant difference with P&lt;0.0001. (B) qPCR analysis of -catenin <span class="highlight" style="background-color:">pathway</span> transcripts <i>c-myc, Cyclin D1, Axin2</i> in E12.5 and E13.5 heterozygous and <i>E-catenin<sup>/</sup></i> brains. The levels of expression are shown in arbitrary units with mean of levels in heterozygous embryos adjusted to one. Data represent means  SD. n=4.</div></p></div><div class="aux"><div class="resc">Wen-Hui Lien, et al. J Cell Sci. ;121(Pt 9):1357-1362.</div><p class="links"><a class="dblinks" href="/pubmed/18397997" ref="ordinalpos=4338&amp;ncbi_uid=1634084&amp;link_uid=18397997">Citation</a><a class="dblinks" href="/pmc/articles/PMC2586592/" ref="ordinalpos=4338&amp;ncbi_uid=1634084&amp;link_uid=PMC2586592">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4339.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3899118/figure/pone-0085902-g005/" ref="ordinalpos=4339&amp;ncbi_uid=5740034&amp;link_uid=PMC3899118" href="/pmc/articles/PMC3899118/"><img src="/pmc/articles/instance/3899118/bin/pone.0085902.g005.gif" src-large="/pmc/articles/instance/3899118/bin/pone.0085902.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3899118/" ref="ordinalpos=4339&amp;ncbi_uid=5740034&amp;link_uid=PMC3899118" image-link="/pmc/articles/PMC3899118/figure/pone-0085902-g005/" class="imagepopup">Figure 5. Intergenic Network..  From: Genome Wide Expression Analysis Suggests Perturbation of Vascular Homeostasis during High Altitude Pulmonary Edema. </a></p><div class="supp"><p class="details"><div>The figure represents an integrated network of multiple pathways interacting through common gene products. Specific sub-networks (gene associations within individual pathways) constituting the integrated network are highlighted in Supplementary Figure S3 (A-R). The major sub-network pathways include regulation of actin cytoskeleton, calcium <span class="highlight" style="background-color:">signaling</span>, MAPK <span class="highlight" style="background-color:">pathway</span>, immune cell <span class="highlight" style="background-color:">signaling</span>, cell-cell communication, VEGF <span class="highlight" style="background-color:">signaling</span> amongst others. The network shown was extracted from the list of differentially expressed genes utilizing <span class="highlight" style="background-color:">Pathway</span> Miner. Relationships between the genes (nodes connected by edges) suggest co-occurrence in a biological <span class="highlight" style="background-color:">pathway</span>. Weight of a specific edge is a relative representation of the number of pathways in which the associating nodes (gene products) co-occur in the KEGG <span class="highlight" style="background-color:">pathway</span> resource. The nodes are labeled by gene names and colored based on the respective fold changes for specific genes in the original data set.</div></p></div><div class="aux"><div class="resc">Manish Sharma, et al. PLoS One. 2014;9(1):e85902.</div><p class="links"><a class="dblinks" href="/pubmed/24465776" ref="ordinalpos=4339&amp;ncbi_uid=5740034&amp;link_uid=24465776">Citation</a><a class="dblinks" href="/pmc/articles/PMC3899118/" ref="ordinalpos=4339&amp;ncbi_uid=5740034&amp;link_uid=PMC3899118">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4340.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4196580/figure/F9/" ref="ordinalpos=4340&amp;ncbi_uid=6767316&amp;link_uid=PMC4196580" href="/pmc/articles/PMC4196580/"><img src="/pmc/articles/instance/4196580/bin/fgene-05-00348-g0009.gif" src-large="/pmc/articles/instance/4196580/bin/fgene-05-00348-g0009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4196580/" ref="ordinalpos=4340&amp;ncbi_uid=6767316&amp;link_uid=PMC4196580" image-link="/pmc/articles/PMC4196580/figure/F9/" class="imagepopup">Figure 9.  From: Characterization of the rainbow trout spleen transcriptome and identification of immune-related genes. </a></p><div class="supp"><p class="details"><div><b>Chemokine <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> showing the annotated and non-annotated proteins</b>. Proteins appearing in our transcriptome are represented in green color and absent proteins in white color.</div></p></div><div class="aux"><div class="resc">Ali Ali, et al. Front Genet. 2014;5:348.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4340&amp;ncbi_uid=6767316&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4196580/" ref="ordinalpos=4340&amp;ncbi_uid=6767316&amp;link_uid=PMC4196580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4341.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691184/figure/pone-0066676-g005/" ref="ordinalpos=4341&amp;ncbi_uid=5061665&amp;link_uid=PMC3691184" href="/pmc/articles/PMC3691184/"><img src="/pmc/articles/instance/3691184/bin/pone.0066676.g005.gif" src-large="/pmc/articles/instance/3691184/bin/pone.0066676.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691184/" ref="ordinalpos=4341&amp;ncbi_uid=5061665&amp;link_uid=PMC3691184" image-link="/pmc/articles/PMC3691184/figure/pone-0066676-g005/" class="imagepopup">Figure 5. miRNA-gene network of the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in chordomas..  From: Integrated miRNA-mRNA Analysis Revealing the Potential Roles of miRNAs in Chordomas. </a></p><div class="supp"><p class="details"><div>Blue box nodes represent downregulated mRNAs, pink box nodes represent upregulated mRNAs, and blue cycle nodes represent downregulated miRNAs.</div></p></div><div class="aux"><div class="resc">Cheng Long, et al. PLoS One. 2013;8(6):e66676.</div><p class="links"><a class="dblinks" href="/pubmed/23826111" ref="ordinalpos=4341&amp;ncbi_uid=5061665&amp;link_uid=23826111">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691184/" ref="ordinalpos=4341&amp;ncbi_uid=5061665&amp;link_uid=PMC3691184">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4342.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3584057/figure/pone-0056667-g008/" ref="ordinalpos=4342&amp;ncbi_uid=4690670&amp;link_uid=PMC3584057" href="/pmc/articles/PMC3584057/"><img src="/pmc/articles/instance/3584057/bin/pone.0056667.g008.gif" src-large="/pmc/articles/instance/3584057/bin/pone.0056667.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3584057/" ref="ordinalpos=4342&amp;ncbi_uid=4690670&amp;link_uid=PMC3584057" image-link="/pmc/articles/PMC3584057/figure/pone-0056667-g008/" class="imagepopup">Figure 8. Expression of apoptotic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream proteins in 786-O after ER downregulation and in A498 after ER overexpression..  From: Estrogen Inhibits Renal Cell Carcinoma Cell Progression through Estrogen Receptor-? Activation. </a></p><div class="aux"><div class="resc">Cheng-Ping Yu, et al. PLoS One. 2013;8(2):e56667.</div><p class="links"><a class="dblinks" href="/pubmed/23460808" ref="ordinalpos=4342&amp;ncbi_uid=4690670&amp;link_uid=23460808">Citation</a><a class="dblinks" href="/pmc/articles/PMC3584057/" ref="ordinalpos=4342&amp;ncbi_uid=4690670&amp;link_uid=PMC3584057">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4343.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3152991/figure/fig6/" ref="ordinalpos=4343&amp;ncbi_uid=3158807&amp;link_uid=PMC3152991" href="/pmc/articles/PMC3152991/"><img src="/pmc/articles/instance/3152991/bin/ECAM2011-624375.006.gif" src-large="/pmc/articles/instance/3152991/bin/ECAM2011-624375.006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3152991/" ref="ordinalpos=4343&amp;ncbi_uid=3158807&amp;link_uid=PMC3152991" image-link="/pmc/articles/PMC3152991/figure/fig6/" class="imagepopup">Figure 6.  From: Insulin-Mimetic Action of Rhoifolin and Cosmosiin Isolated from Citrus grandis (L.) Osbeck Leaves: Enhanced Adiponectin Secretion and Insulin Receptor Phosphorylation in 3T3-L1 Cells. </a></p><div class="supp"><p class="details"><div>Insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in adipocytes: showing potential sites of rhoifolin and cosmosiin interactions.</div></p></div><div class="aux"><div class="resc">Yerra Koteswara Rao, et al. Evid Based Complement Alternat Med. 2011;2011:624375.</div><p class="links"><a class="dblinks" href="/pubmed/20008903" ref="ordinalpos=4343&amp;ncbi_uid=3158807&amp;link_uid=20008903">Citation</a><a class="dblinks" href="/pmc/articles/PMC3152991/" ref="ordinalpos=4343&amp;ncbi_uid=3158807&amp;link_uid=PMC3152991">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4344.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3663533/figure/F8/" ref="ordinalpos=4344&amp;ncbi_uid=6165330&amp;link_uid=PMC3663533" href="/pmc/articles/PMC3663533/"><img src="/pmc/articles/instance/3663533/bin/zbc0261350910008.gif" src-large="/pmc/articles/instance/3663533/bin/zbc0261350910008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3663533/" ref="ordinalpos=4344&amp;ncbi_uid=6165330&amp;link_uid=PMC3663533" image-link="/pmc/articles/PMC3663533/figure/F8/" class="imagepopup">FIGURE 8. From: Lgr4/Gpr48 Negatively Regulates TLR2/4-associated Pattern Recognition and Innate Immunity by Targeting CD14 Expression. </a></p><div class="supp"><p class="details"><div><b>Model for the regulation of TLR <span class="highlight" style="background-color:">signaling</span> by Lgr4.</b> Model of Lgr4-mediated <span class="highlight" style="background-color:">signaling</span> cascades in TLR2/4-mediated immune response. Lgr4 can activate the cAMP-PKA-CREB <span class="highlight" style="background-color:">pathway</span> to negatively regulate CD14 expression through CREB binding to the CD14 promoter. Down-regulation of CD14 reduced the production of inflammatory cytokines in TLR2/4-mediated innate immune response.</div></p></div><div class="aux"><div class="resc">Bing Du, et al. J Biol Chem. 2013 May 24;288(21):15131-15141.</div><p class="links"><a class="dblinks" href="/pubmed/23589304" ref="ordinalpos=4344&amp;ncbi_uid=6165330&amp;link_uid=23589304">Citation</a><a class="dblinks" href="/pmc/articles/PMC3663533/" ref="ordinalpos=4344&amp;ncbi_uid=6165330&amp;link_uid=PMC3663533">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4345.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC514582/figure/F10/" ref="ordinalpos=4345&amp;ncbi_uid=479054&amp;link_uid=PMC514582" href="/pmc/articles/PMC514582/"><img src="/pmc/articles/instance/514582/bin/JCI0421004.f10.gif" src-large="/pmc/articles/instance/514582/bin/JCI0421004.f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC514582/" ref="ordinalpos=4345&amp;ncbi_uid=479054&amp;link_uid=PMC514582" image-link="/pmc/articles/PMC514582/figure/F10/" class="imagepopup">Figure 10.  From: Nef stimulates proliferation of glomerular podocytes through activation of Src-dependent Stat3 and MAPK1,2 pathways. </a></p><div class="supp"><p class="details"><div>Schematic of Nef <span class="highlight" style="background-color:">signaling</span> pathways and their relationship with Nefs biologic effects in podocytes. Nef activates both Stat3 and MAPK1,2 pathways via an Src-dependent <span class="highlight" style="background-color:">pathway</span>. Src activation is essential for Nef-induced podocyte proliferation and dedifferentiation. Stat3 and MAPK1,2 mediate these Nef effects in an additive manner. Other downstream <span class="highlight" style="background-color:">signaling</span> of Nef-Src interaction may also be important.</div></p></div><div class="aux"><div class="resc">John Cijiang He, et al. J Clin Invest. 2004 September 1;114(5):643-651.</div><p class="links"><a class="dblinks" href="/pubmed/15343382" ref="ordinalpos=4345&amp;ncbi_uid=479054&amp;link_uid=15343382">Citation</a><a class="dblinks" href="/pmc/articles/PMC514582/" ref="ordinalpos=4345&amp;ncbi_uid=479054&amp;link_uid=PMC514582">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4346.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2366852/figure/F7/" ref="ordinalpos=4346&amp;ncbi_uid=1451716&amp;link_uid=PMC2366852" href="/pmc/articles/PMC2366852/"><img src="/pmc/articles/instance/2366852/bin/zmk0050885120007.gif" src-large="/pmc/articles/instance/2366852/bin/zmk0050885120007.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2366852/" ref="ordinalpos=4346&amp;ncbi_uid=1451716&amp;link_uid=PMC2366852" image-link="/pmc/articles/PMC2366852/figure/F7/" class="imagepopup">Figure 7. From: Neuronal Death by Oxidative Stress Involves Activation of FOXO3 through a Two-Arm <span class="highlight" style="background-color:">Pathway</span> That Activates Stress Kinases and Attenuates Insulin-like Growth Factor I <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Inhibition of IGF-I <span class="highlight" style="background-color:">signaling</span> is involved in neuronal death after H<sub>2</sub>O<sub>2.</sub> A competitive interplay between AKT and JNK in regulating FOXO3 activity dictates the neuronal response to an abrupt rise in ROS elicited by H<sub>2</sub>O<sub>2</sub>. Cytotoxic levels of H<sub>2</sub>O<sub>2</sub> sequentially activate two independent pathways. A rapid route (1) includes activation of p38 MAPK to inhibit IGF-I <span class="highlight" style="background-color:">signaling</span> by interfering IGF-I receptor/IRS-1 interactions through phosphorylation of serine residues in IRS-1. This leads to abrogation of AKT inhibition of FOXO. A delayed route (2) recruits JNK2 to activate FOXO. Activation of the p38 MAPK <span class="highlight" style="background-color:">pathway</span> is essential because JNK is unable to activate FOXO if AKT is active. Conversely, once JNK activates FOXO, it becomes refractory to inactivation by AKT. The different inhibitory drugs and stimulatory and inhibitory DNA constructs used in this study are indicated.</div></p></div><div class="aux"><div class="resc">David Dvila, et al. Mol Biol Cell. 2008 May;19(5):2014-2025.</div><p class="links"><a class="dblinks" href="/pubmed/18287535" ref="ordinalpos=4346&amp;ncbi_uid=1451716&amp;link_uid=18287535">Citation</a><a class="dblinks" href="/pmc/articles/PMC2366852/" ref="ordinalpos=4346&amp;ncbi_uid=1451716&amp;link_uid=PMC2366852">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4347.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4067322/figure/pone-0100018-g011/" ref="ordinalpos=4347&amp;ncbi_uid=6291213&amp;link_uid=PMC4067322" href="/pmc/articles/PMC4067322/"><img src="/pmc/articles/instance/4067322/bin/pone.0100018.g011.gif" src-large="/pmc/articles/instance/4067322/bin/pone.0100018.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4067322/" ref="ordinalpos=4347&amp;ncbi_uid=6291213&amp;link_uid=PMC4067322" image-link="/pmc/articles/PMC4067322/figure/pone-0100018-g011/" class="imagepopup">Figure 11. Percentage of nucleotide and amino acid identity of the Chiloscyllium griseum Toll-like receptor 3 (shTLR3) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> mediators with sequences of other species..  From: Transcriptome Profiling Reveals Higher Vertebrate Orthologous of Intra-Cytoplasmic Pattern Recognition Receptors in Grey Bamboo Shark. </a></p><div class="supp"><p class="details"><div>The full-length coding sequence of the <i>C. griseum</i> TRAF3 (1728 bp), TBK1 (1529 bp), IRF3 (1374 bp), IRF7 (1515 bp) and Mx (partial 353 bp) sequences was aligned with the other sequences representing these <span class="highlight" style="background-color:">signaling</span> molecules types from different lower and higher order species using Clustal W in the Megalign Module of Lasergene (V 7.1.0). <b><i>Note</i></b><i>: These <span class="highlight" style="background-color:">signaling</span> molecules revealed extreme diversity and the identity revealed close relationship with the higher vertebrate TLRs rather than the teleost. The TBK1 molecule was divergent among the <span class="highlight" style="background-color:">signaling</span> mediators</i>.</div></p></div><div class="aux"><div class="resc">Tirumurugaan Krishnaswamy Gopalan, et al. PLoS One. 2014;9(6):e100018.</div><p class="links"><a class="dblinks" href="/pubmed/24956167" ref="ordinalpos=4347&amp;ncbi_uid=6291213&amp;link_uid=24956167">Citation</a><a class="dblinks" href="/pmc/articles/PMC4067322/" ref="ordinalpos=4347&amp;ncbi_uid=6291213&amp;link_uid=PMC4067322">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4348.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3683624/figure/F2/" ref="ordinalpos=4348&amp;ncbi_uid=4998506&amp;link_uid=PMC3683624" href="/pmc/articles/PMC3683624/"><img src="/pmc/articles/instance/3683624/bin/nihms436539f2.gif" src-large="/pmc/articles/instance/3683624/bin/nihms436539f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3683624/" ref="ordinalpos=4348&amp;ncbi_uid=4998506&amp;link_uid=PMC3683624" image-link="/pmc/articles/PMC3683624/figure/F2/" class="imagepopup">Figure 2. Genomic alterations affect core <span class="highlight" style="background-color:">signaling</span> pathways in SHH medulloblastoma.  From: Subgroup specific structural variation across 1,000 medulloblastoma genomes. </a></p><div class="supp"><p class="details"><div><b>a,</b> GISTIC2 significance plot of amplifications (red) and deletions (blue) observed in SHH. The number of genes mapping to each significant region are included in brackets and regions enriched in SHH are shaded red. <b>b, c,</b> Recurrent amplifications of <i>PPM1D</i> (<b>b</b>) and <i>PIK3C2B</i>/<i>MDM4</i> (<b>c</b>) are restricted to SHH. <b>d,</b> FISH validation of <i>MDM4</i> amplification. <b>e,</b> SHH <span class="highlight" style="background-color:">signaling</span>, TP53 <span class="highlight" style="background-color:">signaling</span>, and RTK/PI3K <span class="highlight" style="background-color:">signaling</span> represent the core pathways genomically targeted in SHH. P-values indicate the prevalence with which the respective <span class="highlight" style="background-color:">pathway</span> is targeted in SHH <i>vs.</i> non-SHH cases (Fishers exact test). Frequencies of focal and broad (parentheses) SCNAs are listed. <b>f,</b> Mutual exclusivity analysis of focal SCNAs in SHH. <b>g,</b> Clinical implications of SCNAs affecting <i>MYCN</i>, <i>GLI2, or PTCH1</i> in SHH (log-rank tests).</div></p></div><div class="aux"><div class="resc">Paul A Northcott, et al. Nature. ;488(7409):49-56.</div><p class="links"><a class="dblinks" href="/pubmed/22832581" ref="ordinalpos=4348&amp;ncbi_uid=4998506&amp;link_uid=22832581">Citation</a><a class="dblinks" href="/pmc/articles/PMC3683624/" ref="ordinalpos=4348&amp;ncbi_uid=4998506&amp;link_uid=PMC3683624">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4349.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2769146/figure/f3-ijms-10-04187/" ref="ordinalpos=4349&amp;ncbi_uid=2146529&amp;link_uid=PMC2769146" href="/pmc/articles/PMC2769146/"><img src="/pmc/articles/instance/2769146/bin/ijms-10-04187f3.gif" src-large="/pmc/articles/instance/2769146/bin/ijms-10-04187f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2769146/" ref="ordinalpos=4349&amp;ncbi_uid=2146529&amp;link_uid=PMC2769146" image-link="/pmc/articles/PMC2769146/figure/f3-ijms-10-04187/" class="imagepopup">Figure 3. From: Functional Analysis of Familial Asp67Glu and Thr1051Ser BRCA1 Mutations in Breast/Ovarian Carcinogenesis. </a></p><div class="supp"><p class="details"><div>The role of <i>BRCA1</i> missense mutation, Asp67Glu and Thr1051Serine, to estrogen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Wt.<i>BRCA1</i> expression vector and mutant <i>BRCA1</i> expression vectors were transfected into DU145 cell line for 48 hrs. Cells were stimulated by estrogen and assay for pERE-Elb-Luc reporter activity. The luciferase activity of <i>BRCA1</i> mutants were different from activity of the wild-type (<i>P</i> &lt; 0.05). These missense mutations failed to promote the estrogen receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for breast/ovarian carcinogenesis.</div></p></div><div class="aux"><div class="resc">Malinee Pongsavee, et al. Int J Mol Sci. 2009 September;10(9):4187-4197.</div><p class="links"><a class="dblinks" href="/pubmed/19865540" ref="ordinalpos=4349&amp;ncbi_uid=2146529&amp;link_uid=19865540">Citation</a><a class="dblinks" href="/pmc/articles/PMC2769146/" ref="ordinalpos=4349&amp;ncbi_uid=2146529&amp;link_uid=PMC2769146">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4350.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3813786/figure/f6-ol-06-05-1447/" ref="ordinalpos=4350&amp;ncbi_uid=5448165&amp;link_uid=PMC3813786" href="/pmc/articles/PMC3813786/"><img src="/pmc/articles/instance/3813786/bin/OL-06-05-1447-g05.gif" src-large="/pmc/articles/instance/3813786/bin/OL-06-05-1447-g05.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3813786/" ref="ordinalpos=4350&amp;ncbi_uid=5448165&amp;link_uid=PMC3813786" image-link="/pmc/articles/PMC3813786/figure/f6-ol-06-05-1447/" class="imagepopup">Figure 6.  From: Differential effects of miR-34c-3p and miR-34c-5p on the proliferation, apoptosis and invasion of glioma cells. </a></p><div class="supp"><p class="details"><div>Impact of miR-34c-3p and miR-34c-5p mimics on a component of the Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Data are from one of three representative experiments.</div></p></div><div class="aux"><div class="resc">ZHENDONG WU, et al. Oncol Lett. 2013 November;6(5):1447-1452.</div><p class="links"><a class="dblinks" href="/pubmed/24179539" ref="ordinalpos=4350&amp;ncbi_uid=5448165&amp;link_uid=24179539">Citation</a><a class="dblinks" href="/pmc/articles/PMC3813786/" ref="ordinalpos=4350&amp;ncbi_uid=5448165&amp;link_uid=PMC3813786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4351.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3538621/figure/F2/" ref="ordinalpos=4351&amp;ncbi_uid=4522814&amp;link_uid=PMC3538621" href="/pmc/articles/PMC3538621/"><img src="/pmc/articles/instance/3538621/bin/1743-422X-9-126-2.gif" src-large="/pmc/articles/instance/3538621/bin/1743-422X-9-126-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3538621/" ref="ordinalpos=4351&amp;ncbi_uid=4522814&amp;link_uid=PMC3538621" image-link="/pmc/articles/PMC3538621/figure/F2/" class="imagepopup">Figure 2.  From: Interaction of Hepatitis C virus proteins with pattern recognition receptors. </a></p><div class="supp"><p class="details"><div>HCV NS3/4A blocks two important adapter proteins, TRIF and MAVS (Cardif) of the TLR3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Muhammad Imran, et al. Virol J. 2012;9:126-126.</div><p class="links"><a class="dblinks" href="/pubmed/22726246" ref="ordinalpos=4351&amp;ncbi_uid=4522814&amp;link_uid=22726246">Citation</a><a class="dblinks" href="/pmc/articles/PMC3538621/" ref="ordinalpos=4351&amp;ncbi_uid=4522814&amp;link_uid=PMC3538621">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4352.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3371373/figure/F1/" ref="ordinalpos=4352&amp;ncbi_uid=4765203&amp;link_uid=PMC3371373" href="/pmc/articles/PMC3371373/"><img src="/pmc/articles/instance/3371373/bin/nihms360508f1.gif" src-large="/pmc/articles/instance/3371373/bin/nihms360508f1.jpg" alt="Fig 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3371373/" ref="ordinalpos=4352&amp;ncbi_uid=4765203&amp;link_uid=PMC3371373" image-link="/pmc/articles/PMC3371373/figure/F1/" class="imagepopup">Fig 1. Potential serotonin receptor-mediated <span class="highlight" style="background-color:">signaling</span> in AD.  From: Neurotransmitter receptors and cognitive dysfunction in Alzheimer's disease and Parkinson's disease. </a></p><div class="supp"><p class="details"><div>Stimulation of serotonin receptors potentially activates the secondary messenger, cAMP, and subsequently stimulates the PKA-mediated <span class="highlight" style="background-color:">pathway</span> in AD.</div></p></div><div class="aux"><div class="resc">Yunqi Xu, et al. Prog Neurobiol. ;97(1):1-13.</div><p class="links"><a class="dblinks" href="/pubmed/22387368" ref="ordinalpos=4352&amp;ncbi_uid=4765203&amp;link_uid=22387368">Citation</a><a class="dblinks" href="/pmc/articles/PMC3371373/" ref="ordinalpos=4352&amp;ncbi_uid=4765203&amp;link_uid=PMC3371373">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4353.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2683667/figure/F4/" ref="ordinalpos=4353&amp;ncbi_uid=1796299&amp;link_uid=PMC2683667" href="/pmc/articles/PMC2683667/"><img src="/pmc/articles/instance/2683667/bin/nihms101693f4.gif" src-large="/pmc/articles/instance/2683667/bin/nihms101693f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2683667/" ref="ordinalpos=4353&amp;ncbi_uid=1796299&amp;link_uid=PMC2683667" image-link="/pmc/articles/PMC2683667/figure/F4/" class="imagepopup">Figure 4.  From: Using high-throughput genomics to study hepatitis C: what determines the outcome of infection?. </a></p><div class="supp"><p class="details"><div>Induction of IFN-<span class="highlight" style="background-color:">signaling</span> pathways in HCV-infected uPA-SCID mice (A) and patients (B). <span class="highlight" style="background-color:">Pathway</span> Builder (Protein Lounge) and the <span class="highlight" style="background-color:">Pathway</span> feature of Resolver were used to visualize the expression of genes from the IFN <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The expression profiles of individual genes are shown in boxes and each bar within the box represents one experiment. Color schemes are as indicated in Figure 1. The reference for each microarray experiment was a pool of normal, uninfected human liver tissue.</div></p></div><div class="aux"><div class="resc">Kathie-Anne Walters, et al. Antiviral Res. ;81(3):198-208.</div><p class="links"><a class="dblinks" href="/pubmed/19135090" ref="ordinalpos=4353&amp;ncbi_uid=1796299&amp;link_uid=19135090">Citation</a><a class="dblinks" href="/pmc/articles/PMC2683667/" ref="ordinalpos=4353&amp;ncbi_uid=1796299&amp;link_uid=PMC2683667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4354.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3124358/figure/fig2/" ref="ordinalpos=4354&amp;ncbi_uid=3465177&amp;link_uid=PMC3124358" href="/pmc/articles/PMC3124358/"><img src="/pmc/articles/instance/3124358/bin/gr2.gif" src-large="/pmc/articles/instance/3124358/bin/gr2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3124358/" ref="ordinalpos=4354&amp;ncbi_uid=3465177&amp;link_uid=PMC3124358" image-link="/pmc/articles/PMC3124358/figure/fig2/" class="imagepopup">Figure 2.  From: Use of Gene Expression and <span class="highlight" style="background-color:">Pathway</span> Signatures to Characterize the Complexity of Human Melanoma. </a></p><div class="supp"><p class="details"><div>The status of oncogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activities analyzed in the benign nevi, primary melanoma, and metastatic melanoma samples. Within the heat map, columns represent samples; and rows, the oncogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> analyzed. <span class="highlight" style="background-color:">Pathway</span> activities on a low to high continuum are represented by a blue to red continuum, respectively. The color bar below the heatmap defines the sample in each column. The samples are ordered according to the defined subgroups, as shown in Figure 1 (the array tree above the heat map defines the subgroups of melanoma samples exhibiting similar gene expression patterns). CAT indicates catenin; EGFR, epidermal growth factor receptor; ER, estrogen receptor; HER, 2 human epidermal growth factor receptor 2; PI3k, phosphatidylinositol 3-kinase; PR, progesterone receptor.</div></p></div><div class="aux"><div class="resc">Jennifer A. Freedman, et al. Am J Pathol. 2011 June;178(6):2513-2522.</div><p class="links"><a class="dblinks" href="/pubmed/21641377" ref="ordinalpos=4354&amp;ncbi_uid=3465177&amp;link_uid=21641377">Citation</a><a class="dblinks" href="/pmc/articles/PMC3124358/" ref="ordinalpos=4354&amp;ncbi_uid=3465177&amp;link_uid=PMC3124358">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4355.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3723114/figure/F1/" ref="ordinalpos=4355&amp;ncbi_uid=5126601&amp;link_uid=PMC3723114" href="/pmc/articles/PMC3723114/"><img src="/pmc/articles/instance/3723114/bin/nihms-485166-f0001.gif" src-large="/pmc/articles/instance/3723114/bin/nihms-485166-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3723114/" ref="ordinalpos=4355&amp;ncbi_uid=5126601&amp;link_uid=PMC3723114" image-link="/pmc/articles/PMC3723114/figure/F1/" class="imagepopup">Figure 1.  From: Noncanonical TGF-? <span class="highlight" style="background-color:">Signaling</span> During Mammary Tumorigenesis. </a></p><div class="supp"><p class="details"><div>Schematic depicting the canonical and noncanonical TGF- <span class="highlight" style="background-color:">signaling</span> systems activated during mammary tumorigenesis. Transmembrane <span class="highlight" style="background-color:">signaling</span> by TGF- is stimulated upon its binding and activation of the Ser/Thr protein kinase receptors, TR-I and TR-II. The physical interaction of TGF- with either TR-III or TR-II facilitates the recruitment and transphosphorylation of TR-I, resulting in its activation and subsequent phosphorylation of the receptor-activated Smads, Smad2 and Smad3. Once activated, Smad2/3 form heterocomplexes with Smad4 and translocate to the nucleus to regulate the expression of TGF--responsive genes in concert with an ever expanding list of transcriptional coactivators and repressors. This branch of the bifurcated TGF- <span class="highlight" style="background-color:">signaling</span> system represents the canonical or Smad2/3-dependent TGF- <span class="highlight" style="background-color:">pathway</span>, which is the predominant <span class="highlight" style="background-color:">pathway</span> coupled to cytostasis and activated by TGF- in normal MECs (<i>left panel</i>). Alternatively, TGF- also activates a variety of noncanonical or Smad2/3-independent effectors, including Par6, NF-B, ILK, FAK, Src, Rho-family GTPases, MAP kinases, and the PI3K:AKT:mTOR <span class="highlight" style="background-color:">signaling</span> axis (<i>right panel</i>). During EMT and mammary tumorigenesis, the balance between canonical and noncanonical TGF- <span class="highlight" style="background-color:">signaling</span> systems becomes distorted and favors coupling to noncanonical effector systems, an event that manifests the TGF- Paradox and the initiation of EMT, invasion, and metastasis. These events are further exacerbated by elevated deposition and eventual cross-linking of ECM molecules within tumor microenvironments (LOX, <i>right panel</i>), an event that promotes tumor rigidity and the activation of mechanotransduction pathways operant in amplifying noncanonical TGF- <span class="highlight" style="background-color:">signaling</span> and its oncogenic activities in mammary tumors. See text for additional details.</div></p></div><div class="aux"><div class="resc">Jenny G. Parvani, et al. J Mammary Gland Biol Neoplasia. ;16(2):127-146.</div><p class="links"><a class="dblinks" href="/pubmed/21448580" ref="ordinalpos=4355&amp;ncbi_uid=5126601&amp;link_uid=21448580">Citation</a><a class="dblinks" href="/pmc/articles/PMC3723114/" ref="ordinalpos=4355&amp;ncbi_uid=5126601&amp;link_uid=PMC3723114">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4356.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4316787/figure/F3/" ref="ordinalpos=4356&amp;ncbi_uid=7192252&amp;link_uid=PMC4316787" href="/pmc/articles/PMC4316787/"><img src="/pmc/articles/instance/4316787/bin/fimmu-06-00023-g003.gif" src-large="/pmc/articles/instance/4316787/bin/fimmu-06-00023-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4316787/" ref="ordinalpos=4356&amp;ncbi_uid=7192252&amp;link_uid=PMC4316787" image-link="/pmc/articles/PMC4316787/figure/F3/" class="imagepopup">Figure 3.  From: Analysis of the Bovine Monocyte-Derived Macrophage Response to Mycobacterium avium Subspecies Paratuberculosis Infection Using RNA-seq. </a></p><div class="supp"><p class="details"><div><b>The top-ranked enriched canonical <span class="highlight" style="background-color:">pathway</span> identified using IPA at 2hpi  the <i>IL-10 <span class="highlight" style="background-color:">signaling</span></i> <span class="highlight" style="background-color:">pathway</span></b>. Red shading indicates increased expression in MAP-infected MDM relative to the non-infected control MDM. Green shading indicates decreased expression in MAP-infected MDM relative to the non-infected control MDM. White and gray shading indicates non-expression and non-differential expression, respectively.</div></p></div><div class="aux"><div class="resc">Maura E. Casey, et al. Front Immunol. 2015;6:23.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4356&amp;ncbi_uid=7192252&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4316787/" ref="ordinalpos=4356&amp;ncbi_uid=7192252&amp;link_uid=PMC4316787">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4357.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3909037/figure/F2/" ref="ordinalpos=4357&amp;ncbi_uid=5786782&amp;link_uid=PMC3909037" href="/pmc/articles/PMC3909037/"><img src="/pmc/articles/instance/3909037/bin/psb-8-e24686-g2.gif" src-large="/pmc/articles/instance/3909037/bin/psb-8-e24686-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3909037/" ref="ordinalpos=4357&amp;ncbi_uid=5786782&amp;link_uid=PMC3909037" image-link="/pmc/articles/PMC3909037/figure/F2/" class="imagepopup">Figure. From: Mechanisms of <span class="highlight" style="background-color:">signaling</span> crosstalk between brassinosteroids and gibberellins. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> A model for gibberellin (GA) <span class="highlight" style="background-color:">signaling</span> in <i>Arabidopsis</i>. In the absence of GAs, DELLA proteins accumulate in cell nucleus and inhibit GA responses. In the presence of GA signal, the GA receptor GID1 binds the hormone and undergoes a conformational change suitable for DELLAs binding. The resulting GA-GID1-DELLA complex triggers degradation of DELLAs via the 26S proteasome <span class="highlight" style="background-color:">pathway</span>, resulting in release of DELLA-repressed plant growth.</div></p></div><div class="aux"><div class="resc">Qian-Feng Li, et al. Plant Signal Behav. 2013 July 1;8(7):e24686.</div><p class="links"><a class="dblinks" href="/pubmed/23603943" ref="ordinalpos=4357&amp;ncbi_uid=5786782&amp;link_uid=23603943">Citation</a><a class="dblinks" href="/pmc/articles/PMC3909037/" ref="ordinalpos=4357&amp;ncbi_uid=5786782&amp;link_uid=PMC3909037">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4358.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3149764/figure/F3/" ref="ordinalpos=4358&amp;ncbi_uid=3519917&amp;link_uid=PMC3149764" href="/pmc/articles/PMC3149764/"><img src="/pmc/articles/instance/3149764/bin/nihms295516f3.gif" src-large="/pmc/articles/instance/3149764/bin/nihms295516f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3149764/" ref="ordinalpos=4358&amp;ncbi_uid=3519917&amp;link_uid=PMC3149764" image-link="/pmc/articles/PMC3149764/figure/F3/" class="imagepopup">Figure 3. The JAK/STAT microenvironment in adipose tissue.  From: Emerging roles of JAK-STAT <span class="highlight" style="background-color:">signaling</span> pathways in adipocytes. </a></p><div class="supp"><p class="details"><div>Adipose tissue is largely comprised of adipocytes but also contains preadipocytes, endothelial cells, and infiltrating immune cells, including NK cells, T cells, and macrophages. Adipocytes are highly responsive to many hormones and growth factors that utilize the JAK/STAT <span class="highlight" style="background-color:">pathway</span>. The receptors for these ligands are indicated in the diagram. Infiltrating immune cells also produce cytokines which likely act in a paracrine fashion to induce JAK/STAT <span class="highlight" style="background-color:">signaling</span> in adipocytes. Adipocytes also have important endocrine properties and four JAK/STAT activating hormones have been shown to be produced from adipocytes.</div></p></div><div class="aux"><div class="resc">Allison J. Richard, et al. Trends Endocrinol Metab. ;22(8):325-332.</div><p class="links"><a class="dblinks" href="/pubmed/21561789" ref="ordinalpos=4358&amp;ncbi_uid=3519917&amp;link_uid=21561789">Citation</a><a class="dblinks" href="/pmc/articles/PMC3149764/" ref="ordinalpos=4358&amp;ncbi_uid=3519917&amp;link_uid=PMC3149764">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4359.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2872667/figure/pone-0010692-g002/" ref="ordinalpos=4359&amp;ncbi_uid=2373085&amp;link_uid=PMC2872667" href="/pmc/articles/PMC2872667/"><img src="/pmc/articles/instance/2872667/bin/pone.0010692.g002.gif" src-large="/pmc/articles/instance/2872667/bin/pone.0010692.g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2872667/" ref="ordinalpos=4359&amp;ncbi_uid=2373085&amp;link_uid=PMC2872667" image-link="/pmc/articles/PMC2872667/figure/pone-0010692-g002/" class="imagepopup">Figure 2. BMP <span class="highlight" style="background-color:">signaling</span> in trabecular bone..  From: Skeletal Site-Related Variation in Human Trabecular Bone Transcriptome and <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>In this interaction network model, constructed by <span class="highlight" style="background-color:">Pathway</span> Architect analysis software of differential expressed genes between iliac crest and the lumbar spine, up-regulated transcripts are indicated red whilst the down-regulated are shown in green. Arrows link interacting genes and positive (+) and negative () associations are marked respectively. Green boxes denote regulation, blue boxes binding and orange circles indicating phosphorylation.</div></p></div><div class="aux"><div class="resc">Satya S. Varanasi, et al. PLoS One. 2010;5(5):e10692.</div><p class="links"><a class="dblinks" href="/pubmed/20502692" ref="ordinalpos=4359&amp;ncbi_uid=2373085&amp;link_uid=20502692">Citation</a><a class="dblinks" href="/pmc/articles/PMC2872667/" ref="ordinalpos=4359&amp;ncbi_uid=2373085&amp;link_uid=PMC2872667">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4360.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1894903/figure/F2/" ref="ordinalpos=4360&amp;ncbi_uid=1004032&amp;link_uid=PMC1894903" href="/pmc/articles/PMC1894903/"><img src="/pmc/articles/instance/1894903/bin/nihms-17435-0002.gif" src-large="/pmc/articles/instance/1894903/bin/nihms-17435-0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1894903/" ref="ordinalpos=4360&amp;ncbi_uid=1004032&amp;link_uid=PMC1894903" image-link="/pmc/articles/PMC1894903/figure/F2/" class="imagepopup">Figure 2.  From: Trophoblast Adhesion of the Peri-implantation Mouse Blastocyst is Regulated by Integrin <span class="highlight" style="background-color:">Signaling</span> That Targets Phospholipase C. </a></p><div class="supp"><p class="details"><div>Dependence of FN-induced intracellular Ca<sup>2+</sup> <span class="highlight" style="background-color:">signaling</span> on the PLC-Ins(1,4,5)P<sub>3</sub> <span class="highlight" style="background-color:">pathway</span>. Blastocysts were treated with vehicle (Control), 10 M U73343, 10 M U73122 or 1 M Xestospongin C and then exposed to FN-120. Intracellular Ca<sup>2+</sup> levels (A) were monitored and reported as in Fig. 1 and FN-binding activities (B) were normalized to the Control group and reported as mean  SEM. *, <i>P</i> &lt; 0.05, compared to Control.</div></p></div><div class="aux"><div class="resc">Jun Wang, et al. Dev Biol. ;302(1):143-153.</div><p class="links"><a class="dblinks" href="/pubmed/17027741" ref="ordinalpos=4360&amp;ncbi_uid=1004032&amp;link_uid=17027741">Citation</a><a class="dblinks" href="/pmc/articles/PMC1894903/" ref="ordinalpos=4360&amp;ncbi_uid=1004032&amp;link_uid=PMC1894903">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4361.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1112039/figure/f1/" ref="ordinalpos=4361&amp;ncbi_uid=432450&amp;link_uid=PMC1112039" href="/pmc/articles/PMC1112039/"><img src="/pmc/articles/instance/1112039/bin/zjb0110547680001.gif" src-large="/pmc/articles/instance/1112039/bin/zjb0110547680001.jpg" alt="FIG. 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1112039/" ref="ordinalpos=4361&amp;ncbi_uid=432450&amp;link_uid=PMC1112039" image-link="/pmc/articles/PMC1112039/figure/f1/" class="imagepopup">FIG. 1. From: Layers of <span class="highlight" style="background-color:">Signaling</span> in a Bacterium-Host Association. </a></p><div class="supp"><p class="details"><div>A comparison of the <i>V. harveyi</i> and <i>V. fischeri</i> quorum-sensing systems. Parts of the <span class="highlight" style="background-color:">pathway</span> in <i>V. fischeri</i> are modeled after those described for <i>V. harveyi</i> and are described further in the text. The colors reflect proteins that are homologous between the two species. The acyl-HSL synthases are depicted as ovals. Although the <i>V. fischeri</i> AinS and LuxS proteins are homologous to LuxM and LuxS of <i>V. harveyi</i>, the acyl-HSL <span class="highlight" style="background-color:">signaling</span> molecules (represented by small circles) produced by the different bacteria are distinct. It is not known whether sRNAs are involved in bioluminescence control in <i>V. fischeri</i>. OM, outer membrane; IM, inner membrane.</div></p></div><div class="aux"><div class="resc">Karen L. Visick. J Bacteriol. 2005 June;187(11):3603-3606.</div><p class="links"><a class="dblinks" href="/pubmed/15901681" ref="ordinalpos=4361&amp;ncbi_uid=432450&amp;link_uid=15901681">Citation</a><a class="dblinks" href="/pmc/articles/PMC1112039/" ref="ordinalpos=4361&amp;ncbi_uid=432450&amp;link_uid=PMC1112039">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4362.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1828067/figure/F7/" ref="ordinalpos=4362&amp;ncbi_uid=996177&amp;link_uid=PMC1828067" href="/pmc/articles/PMC1828067/"><img src="/pmc/articles/instance/1828067/bin/1476-4598-6-18-7.gif" src-large="/pmc/articles/instance/1828067/bin/1476-4598-6-18-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1828067/" ref="ordinalpos=4362&amp;ncbi_uid=996177&amp;link_uid=PMC1828067" image-link="/pmc/articles/PMC1828067/figure/F7/" class="imagepopup">Figure 7.  From: Mechanisms of osteopontin and CD44 as metastatic principles in prostate cancer cells. </a></p><div class="supp"><p class="details"><div><b>A schematic representation of osteopontin/v3 <span class="highlight" style="background-color:">signaling</span> in the regulation of prostate cancer cell migration. </b>PC3 cells bind osteopontin through integrin v3 in an RGD-dependent manner. Integrin v3 increases Rho GTPase activity through down stream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that involves RANKL [62] or tyrosine phosphorylation of several <span class="highlight" style="background-color:">signaling</span> molecules such as Src, PYK2, FAK, and p130<sup>Cas </sup>associated with integrin [10,64]. Rho activation increases CD44 surface expression [10,32] that resulted in the activation of MMP-9. The interaction of CD44 with MMP-9 on the cell surface may have a role in the degradation of ECM to facilitate cell migration. Bisphosphonates (BPs) inhibit CD44 surface interaction with MMP-9 either partially or completely. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and the target molecules that exhibit the inhibitory effects of BP treatment are indicated by dotted arrows and 'X'. A decrease (indicated by inverted red arrow) in CD44 surface expression, MMP-9 interaction with CD44 on the cell surface, MMP-9 secretion, and cell migration was observed in cells treated with BP. PC3/OPN cells treated with BPs reproduce the changes observed in PC3/SiRNA and PC3/OPN (RGA) The possible reason for these effects may be due to reduced v3 -<span class="highlight" style="background-color:">signaling</span> in PC3/OPN (RGA) and PC3/SiRNA cells or targeting of v3-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by BPs.</div></p></div><div class="aux"><div class="resc">Bhavik Desai, et al. Mol Cancer. 2007;6:18-18.</div><p class="links"><a class="dblinks" href="/pubmed/17343740" ref="ordinalpos=4362&amp;ncbi_uid=996177&amp;link_uid=17343740">Citation</a><a class="dblinks" href="/pmc/articles/PMC1828067/" ref="ordinalpos=4362&amp;ncbi_uid=996177&amp;link_uid=PMC1828067">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4363.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3513309/figure/pone-0050221-g003/" ref="ordinalpos=4363&amp;ncbi_uid=4428464&amp;link_uid=PMC3513309" href="/pmc/articles/PMC3513309/"><img src="/pmc/articles/instance/3513309/bin/pone.0050221.g003.gif" src-large="/pmc/articles/instance/3513309/bin/pone.0050221.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3513309/" ref="ordinalpos=4363&amp;ncbi_uid=4428464&amp;link_uid=PMC3513309" image-link="/pmc/articles/PMC3513309/figure/pone-0050221-g003/" class="imagepopup">Figure 3. Separate comparison of gene expression post-treatment versus gene expression pre-treatment for responding and non-responding patients. Ingenuity <span class="highlight" style="background-color:">pathway</span> analysis suggests differences in regulation of T-cell (Figure 3A) and B-cell (Figure 3B) receptor <span class="highlight" style="background-color:">signaling</span>..  From: Gene Expression Profile of Peripheral Blood Lymphocytes from Renal Cell Carcinoma Patients Treated with IL-2, Interferon-? and Dendritic Cell Vaccine. </a></p><div class="supp"><p class="details"><div>Most of the molecules in the B-cell as well as in the T-cell receptor <span class="highlight" style="background-color:">pathway</span> are down-regulated for non-responding patients upon immunotherapy (POST &lt;PRE), whereas gene expression of the same molecules is up-regulated for responding patients (POST &gt;PRE). Red: up-regulation, Green: down-regulation.</div></p></div><div class="aux"><div class="resc">Benita Wolf, et al. PLoS One. 2012;7(12):e50221.</div><p class="links"><a class="dblinks" href="/pubmed/23226513" ref="ordinalpos=4363&amp;ncbi_uid=4428464&amp;link_uid=23226513">Citation</a><a class="dblinks" href="/pmc/articles/PMC3513309/" ref="ordinalpos=4363&amp;ncbi_uid=4428464&amp;link_uid=PMC3513309">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4364.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2954045/figure/F1/" ref="ordinalpos=4364&amp;ncbi_uid=2515459&amp;link_uid=PMC2954045" href="/pmc/articles/PMC2954045/"><img src="/pmc/articles/instance/2954045/bin/aging-02-527-g001.gif" src-large="/pmc/articles/instance/2954045/bin/aging-02-527-g001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2954045/" ref="ordinalpos=4364&amp;ncbi_uid=2515459&amp;link_uid=PMC2954045" image-link="/pmc/articles/PMC2954045/figure/F1/" class="imagepopup">Figure 1.The FXR/SHP <span class="highlight" style="background-color:">pathway</span> controlling miR-34a and SIRT1 expression..  From: Controlling SIRT1 expression by microRNAs in health and metabolic disease. </a></p><div class="supp"><p class="details"><div>  Under normal                                                 conditions, activation of FXR <span class="highlight" style="background-color:">signaling</span> induces the metabolic repressor SHP                                                 in liver.  SHP is then recruited to the miR-34a promoter and inhibits                                                 binding of the key activator p53 to the DNA, resulting in decreased miR-34a                                                 expression.  Inhibition of miR-34a results in increased hepatic SIRT1 levels.                                                  In contrast, under pathophysiological conditions such as fatty livers of                                                 obese mice, the dysregulated FXR/SHP <span class="highlight" style="background-color:">pathway</span> due to highly elevated FXR                                                 acetylation no longer inhibits transcription of miR-34a.  The dysregulated                                                 FXR/SHP <span class="highlight" style="background-color:">pathway</span>, along with acetylation of p53 due to cellular stress under                                                 metabolic disease states, result in elevated miR-34a expression, which                                                 contributes to decreased SIRT1 levels.                                                                  </div></p></div><div class="aux"><div class="resc">Jiyoung Lee, et al. Aging (Albany NY). 2010 August;2(8):527-534.</div><p class="links"><a class="dblinks" href="/pubmed/20689156" ref="ordinalpos=4364&amp;ncbi_uid=2515459&amp;link_uid=20689156">Citation</a><a class="dblinks" href="/pmc/articles/PMC2954045/" ref="ordinalpos=4364&amp;ncbi_uid=2515459&amp;link_uid=PMC2954045">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4365.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3670240/figure/F1/" ref="ordinalpos=4365&amp;ncbi_uid=5316802&amp;link_uid=PMC3670240" href="/pmc/articles/PMC3670240/"><img src="/pmc/articles/instance/3670240/bin/jkst-1-168-g1.gif" src-large="/pmc/articles/instance/3670240/bin/jkst-1-168-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3670240/" ref="ordinalpos=4365&amp;ncbi_uid=5316802&amp;link_uid=PMC3670240" image-link="/pmc/articles/PMC3670240/figure/F1/" class="imagepopup">Figure. From: Ubiquitin-mediated regulation of JAK-STAT <span class="highlight" style="background-color:">signaling</span> in embryonic stem cells. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> Regulation of the LIF-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse ESCs. LIF causes heterodimerization of LIFR and gp130. LIF causes phosphorylation and activation of JAK, which is tethered to the intracellular region of gp130. Activated JAK phosphorylates gp130 and its tyrosine phosphorylations causes interaction with STAT3. Then STAT3 is also tyrosine-phosphorylated by JAK and is homodimerized via its SH2 domains. The dimerized STAT3 translocates to the nucleus and activates target gene transcription. JAK-STAT3 activates Klf4, which maintains Oct3/4 expression via Nanog. JAK activates PI3K, which acts as an activator for AKT. AKT then upregulates Tbx3 as another pluripotency gene. The JAK-STAT <span class="highlight" style="background-color:">pathway</span> is negatively regulated by several inhibitory systems: dephosphorylation by tyrosine phosphatases including SHP and physical inhibition or ubiquitin-mediated degradation of JAK by SOCS. JAK phosphorylates SHP and then SHP interacts with the Grb2/SOS complex to activate the MAPK <span class="highlight" style="background-color:">pathway</span>, leading to inhibition of Tbx3 and Nanog. TRIM8 likely negatively regulates Hsp90-mediated translocation of STAT3 into the nucleus of ESCs. SLIM is an E3 ubiquitin ligase that negatively regulates the JAK-STAT signal <span class="highlight" style="background-color:">pathway</span>, but SLIM is expressed at a low level in mouse ESCs. P, phosphorylation.</div></p></div><div class="aux"><div class="resc">Shigetsugu Hatakeyama. JAKSTAT. 2012 July 1;1(3):168-175.</div><p class="links"><a class="dblinks" href="/pubmed/24058766" ref="ordinalpos=4365&amp;ncbi_uid=5316802&amp;link_uid=24058766">Citation</a><a class="dblinks" href="/pmc/articles/PMC3670240/" ref="ordinalpos=4365&amp;ncbi_uid=5316802&amp;link_uid=PMC3670240">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4366.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3406918/figure/fig3/" ref="ordinalpos=4366&amp;ncbi_uid=4158874&amp;link_uid=PMC3406918" href="/pmc/articles/PMC3406918/"><img src="/pmc/articles/instance/3406918/bin/PC_098640_wc_f3.gif" src-large="/pmc/articles/instance/3406918/bin/PC_098640_wc_f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3406918/" ref="ordinalpos=4366&amp;ncbi_uid=4158874&amp;link_uid=PMC3406918" image-link="/pmc/articles/PMC3406918/figure/fig3/" class="imagepopup">Figure 3. From: Ethylene <span class="highlight" style="background-color:">Signaling</span> Negatively Regulates Freezing Tolerance by Repressing Expression of CBF and Type-A ARR Genes in Arabidopsis[W][OA]. </a></p><div class="supp"><p class="details"><div>Ethylene Production and Expression of Genes in the Ethylene <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> under Cold Stress.<br /><b>(A)</b> Ethylene production in wild-type Col and the <i>eto1</i> mutant under cold stress. Two-week-old seedlings grown in 15-mL gas chromatography vials were grown at 22C and treated at 4C or kept at 22C. The ethylene levels in the headspace were measured at various time points. FW, fresh weight.<br /><b>(B)</b> Expression of genes in the ethylene <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> under cold stress. Two-week-old wild-type Col seedlings grown at 22C were treated at 4C for the indicated periods of time. The transcription levels were determined by qRT-PCR. The data represent the mean values of three replicates  <span style="font-variant: small-caps" class="small-caps">sd</span>.</div></p></div><div class="aux"><div class="resc">Yiting Shi, et al. Plant Cell. 2012 June;24(6):2578-2595.</div><p class="links"><a class="dblinks" href="/pubmed/22706288" ref="ordinalpos=4366&amp;ncbi_uid=4158874&amp;link_uid=22706288">Citation</a><a class="dblinks" href="/pmc/articles/PMC3406918/" ref="ordinalpos=4366&amp;ncbi_uid=4158874&amp;link_uid=PMC3406918">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4367.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3539662/figure/F1/" ref="ordinalpos=4367&amp;ncbi_uid=4524477&amp;link_uid=PMC3539662" href="/pmc/articles/PMC3539662/"><img src="/pmc/articles/instance/3539662/bin/fimmu-03-00401-g001.gif" src-large="/pmc/articles/instance/3539662/bin/fimmu-03-00401-g001.jpg" alt="FIGURE 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3539662/" ref="ordinalpos=4367&amp;ncbi_uid=4524477&amp;link_uid=PMC3539662" image-link="/pmc/articles/PMC3539662/figure/F1/" class="imagepopup">FIGURE 1.  From: AKTivation of PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> by KSHV. </a></p><div class="supp"><p class="details"><div><b>Kaposi sarcoma-associated herpesvirus K1 activates PI3K/AKT/mTOR <span class="highlight" style="background-color:">signaling</span> thereby activating protein synthesis and survival pathways, while inhibiting apoptotic pathways</b>. Orange circles denote phosphorylation.</div></p></div><div class="aux"><div class="resc">Aadra P. Bhatt, et al. Front Immunol. 2012;3:401.</div><p class="links"><a class="dblinks" href="/pubmed/23316192" ref="ordinalpos=4367&amp;ncbi_uid=4524477&amp;link_uid=23316192">Citation</a><a class="dblinks" href="/pmc/articles/PMC3539662/" ref="ordinalpos=4367&amp;ncbi_uid=4524477&amp;link_uid=PMC3539662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4368.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3583223/figure/F4/" ref="ordinalpos=4368&amp;ncbi_uid=5656385&amp;link_uid=PMC3583223" href="/pmc/articles/PMC3583223/"><img src="/pmc/articles/instance/3583223/bin/nihms-427572-f0004.gif" src-large="/pmc/articles/instance/3583223/bin/nihms-427572-f0004.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3583223/" ref="ordinalpos=4368&amp;ncbi_uid=5656385&amp;link_uid=PMC3583223" image-link="/pmc/articles/PMC3583223/figure/F4/" class="imagepopup">Fig. 4. Clinical relevance of the miR-30c <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: MicroRNA-30c Targets Cytoskeleton Genes Involved in Breast Cancer Cell Invasion. </a></p><div class="supp"><p class="details"><div><b>a</b>Scatter plots between VIM single gene expression and the expression of miR-30c in GSE22220 and GSE22216 (n=210). Pearson correlation was estimated in each comparison and plots were drawn using the scatterplot function found in the Car package (http://cran.r-project.org).<br /><b>b</b>Box plots of TWF1 expression compared between nodal metastasis positive (n=30) versus nodal-metastasis-negative (n=13) clinical breast tumors (UC set). A students t-test was used to calculate p values.<br /><b>c</b><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> scheme of miR30-TWF1(VIM)-IL-11-pSTAT3 regulation of breast cancer metastasis</div></p></div><div class="aux"><div class="resc">Jessica Bockhorn, et al. Breast Cancer Res Treat. ;137(2):373-382.</div><p class="links"><a class="dblinks" href="/pubmed/23224145" ref="ordinalpos=4368&amp;ncbi_uid=5656385&amp;link_uid=23224145">Citation</a><a class="dblinks" href="/pmc/articles/PMC3583223/" ref="ordinalpos=4368&amp;ncbi_uid=5656385&amp;link_uid=PMC3583223">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4369.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC416421/figure/f9/" ref="ordinalpos=4369&amp;ncbi_uid=571018&amp;link_uid=PMC416421" href="/pmc/articles/PMC416421/"><img src="/pmc/articles/instance/416421/bin/zmb0110440860009.gif" src-large="/pmc/articles/instance/416421/bin/zmb0110440860009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC416421/" ref="ordinalpos=4369&amp;ncbi_uid=571018&amp;link_uid=PMC416421" image-link="/pmc/articles/PMC416421/figure/f9/" class="imagepopup">FIG. 9. From: Wnt Proteins Induce Dishevelled Phosphorylation via an LRP5/6- Independent Mechanism, Irrespective of Their Ability To Stabilize ?-Catenin. </a></p><div class="supp"><p class="details"><div>Model illustrating Dvl phosphorylation as a common response to Wnt proteins that activate distinct <span class="highlight" style="background-color:">signaling</span> pathways. Wnt <span class="highlight" style="background-color:">signaling</span> via the canonical -catenin <span class="highlight" style="background-color:">pathway</span> requires both Fzd and LRP5/6, while noncanonical Wnt <span class="highlight" style="background-color:">signaling</span> requires Fzd and possibly additional receptors as yet uncharacterized. For simplicity, we have indicated that signals leading to phosphorylation of Dvl are derived from Fzd alone. We propose that Dvl phosphorylation is a common response to both the Wnt1 class and Wnt5a class of Wnt proteins and that it potentiates Wnt <span class="highlight" style="background-color:">signaling</span> via either the canonical or noncanonical pathways. Which of these pathways dominates in a given circumstance may depend on the involvement of Wnt coreceptor components in addition to Fzd and possibly on the subcellular distribution of Dvl. The reported association between different subcellular locations of Dvl protein and different <span class="highlight" style="background-color:">signaling</span> pathways suggests that this might be a key determinant (17).</div></p></div><div class="aux"><div class="resc">Jos M. Gonzlez-Sancho, et al. Mol Cell Biol. 2004 June;24(11):4757-4768.</div><p class="links"><a class="dblinks" href="/pubmed/15143170" ref="ordinalpos=4369&amp;ncbi_uid=571018&amp;link_uid=15143170">Citation</a><a class="dblinks" href="/pmc/articles/PMC416421/" ref="ordinalpos=4369&amp;ncbi_uid=571018&amp;link_uid=PMC416421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4370.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC167103/figure/fig4/" ref="ordinalpos=4370&amp;ncbi_uid=386391&amp;link_uid=PMC167103" href="/pmc/articles/PMC167103/"><img src="/pmc/articles/instance/167103/bin/pp0737980004.gif" src-large="/pmc/articles/instance/167103/bin/pp0737980004.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC167103/" ref="ordinalpos=4370&amp;ncbi_uid=386391&amp;link_uid=PMC167103" image-link="/pmc/articles/PMC167103/figure/fig4/" class="imagepopup">Figure 4. From: Viviparous1 Alters Global Gene Expression Patterns through Regulation of  Abscisic Acid  <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Feed-forward regulation of ABA <span class="highlight" style="background-color:">signaling</span> mediated VP1 and ABI3. A,  Feed-forward regulation of ABA <span class="highlight" style="background-color:">signaling</span> mediated by VP1 is mediated by the  <span class="highlight" style="background-color:">pathway</span> proposed by Finkelstein and colleagues  (Finkelstein and Lynch, 2000;  Soderman et al., 2000). In  addition, ABA induces two negative regulators of ABA <span class="highlight" style="background-color:">signaling</span>, <i>ABI1</i>  and <i>ABI2</i>, enabling feedback regulation of ABA <span class="highlight" style="background-color:">signaling</span>  (Leung et al., 1997;  Gosti et al., 1999;  Merlot et al., 2001). These  <span class="highlight" style="background-color:">signaling</span> pathways are shown with dotted lines. VP1 inhibits ABA induction of  the negative regulators, potentiating further enhancement of ABA sensitivity  (shown with bold line). B, Northern analysis of <i>ABI1</i>, <i>ABI2</i>,  and <i>ABI5</i> gene expression in maturing siliques of wild type  (<i>Col</i>) and <i>abi3-6</i>. Total RNA samples were isolated from the  late stage of developing siliques. Fifteen micrograms of the total RNA was  hybridized with the indicated probes.</div></p></div><div class="aux"><div class="resc">Masaharu Suzuki, et al. Plant Physiol. 2003 July;132(3):1664-1677.</div><p class="links"><a class="dblinks" href="/pubmed/12857845" ref="ordinalpos=4370&amp;ncbi_uid=386391&amp;link_uid=12857845">Citation</a><a class="dblinks" href="/pmc/articles/PMC167103/" ref="ordinalpos=4370&amp;ncbi_uid=386391&amp;link_uid=PMC167103">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4371.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3604760/figure/fig1/" ref="ordinalpos=4371&amp;ncbi_uid=4729924&amp;link_uid=PMC3604760" href="/pmc/articles/PMC3604760/"><img src="/pmc/articles/instance/3604760/bin/mbo0021314630001.gif" src-large="/pmc/articles/instance/3604760/bin/mbo0021314630001.jpg" alt="FIG 1 " /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3604760/" ref="ordinalpos=4371&amp;ncbi_uid=4729924&amp;link_uid=PMC3604760" image-link="/pmc/articles/PMC3604760/figure/fig1/" class="imagepopup">FIG 1 .  From: Integration of the Second Messenger c-di-GMP into the Chemotactic <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Cartoon summarizing the behaviors controlled by chemotaxis pathways in <i>A. brasilense</i>. The black and gray lines represent effects that occur under different environmental conditions represented at the top of the figure. The arrows represent interactions within chemotaxis proteins. Proteins of the Che1 <span class="highlight" style="background-color:">pathway</span> and proteins from another <span class="highlight" style="background-color:">pathway</span> are represented. Dashed lines represent putative interactions.</div></p></div><div class="aux"><div class="resc">Matthew H. Russell, et al. mBio. 2013 Mar-Apr;4(2):e00001-13.</div><p class="links"><a class="dblinks" href="/pubmed/23512960" ref="ordinalpos=4371&amp;ncbi_uid=4729924&amp;link_uid=23512960">Citation</a><a class="dblinks" href="/pmc/articles/PMC3604760/" ref="ordinalpos=4371&amp;ncbi_uid=4729924&amp;link_uid=PMC3604760">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4372.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4392064/figure/f0010/" ref="ordinalpos=4372&amp;ncbi_uid=7415048&amp;link_uid=PMC4392064" href="/pmc/articles/PMC4392064/"><img src="/pmc/articles/instance/4392064/bin/gr2.gif" src-large="/pmc/articles/instance/4392064/bin/gr2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4392064/" ref="ordinalpos=4372&amp;ncbi_uid=7415048&amp;link_uid=PMC4392064" image-link="/pmc/articles/PMC4392064/figure/f0010/" class="imagepopup">Fig. 2.  From: Novel Cosic resonance (standing wave) solutions for components of the JAKSTAT cellular <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: A convergence of spectral density profiles. </a></p><div class="supp"><p class="details"><div>Stylized shapes and descriptions of the proteins composing the JAKSTAT <span class="highlight" style="background-color:">pathway</span> (on top) and the distribution of Spectral Power Densities predicted by the Cosic Method for each protein between interleukin 2 (on the receptor) and CASP-9 at the end of the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Lukasz M. Karbowski, et al. FEBS Open Bio. 2015;5:245-250.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4372&amp;ncbi_uid=7415048&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4392064/" ref="ordinalpos=4372&amp;ncbi_uid=7415048&amp;link_uid=PMC4392064">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4373.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3441385/figure/F4/" ref="ordinalpos=4373&amp;ncbi_uid=4233401&amp;link_uid=PMC3441385" href="/pmc/articles/PMC3441385/"><img src="/pmc/articles/instance/3441385/bin/1743-422X-9-165-4.gif" src-large="/pmc/articles/instance/3441385/bin/1743-422X-9-165-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3441385/" ref="ordinalpos=4373&amp;ncbi_uid=4233401&amp;link_uid=PMC3441385" image-link="/pmc/articles/PMC3441385/figure/F4/" class="imagepopup">Figure 4.  From: Induction of interleukin-10 is dependent on p38 mitogen-activated protein kinase <span class="highlight" style="background-color:">pathway</span> in macrophages infected with porcine reproductive and respiratory syndrome virus. </a></p><div class="supp"><p class="details"><div><b>PRRSV infection activated p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in BMDMs.</b> Whole-cell extracts of BMDMs infected with (<b>a</b>) PRRSV at an MOI of 1 or (<b>b</b>) equal amounts of UV-inactivated virus were prepared at indicated time points and analyzed by immunoblot for the phosphorylation status of p38 (p-p38) and ERK1/2 (p-ERK1/2). The blot was also probed for tubulin as loading control.</div></p></div><div class="aux"><div class="resc">Jun Hou, et al. Virol J. 2012;9:165-165.</div><p class="links"><a class="dblinks" href="/pubmed/22909062" ref="ordinalpos=4373&amp;ncbi_uid=4233401&amp;link_uid=22909062">Citation</a><a class="dblinks" href="/pmc/articles/PMC3441385/" ref="ordinalpos=4373&amp;ncbi_uid=4233401&amp;link_uid=PMC3441385">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4374.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2871569/figure/fig7/" ref="ordinalpos=4374&amp;ncbi_uid=2366774&amp;link_uid=PMC2871569" href="/pmc/articles/PMC2871569/"><img src="/pmc/articles/instance/2871569/bin/JBB2010-705431.007.gif" src-large="/pmc/articles/instance/2871569/bin/JBB2010-705431.007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2871569/" ref="ordinalpos=4374&amp;ncbi_uid=2366774&amp;link_uid=PMC2871569" image-link="/pmc/articles/PMC2871569/figure/fig7/" class="imagepopup">Figure 7.  From: Endoplasmic Reticulum Stress-Mediated Apoptosis Involved in Indirect Recognition <span class="highlight" style="background-color:">Pathway</span> Blockade Induces Long-Term Heart Allograft Survival. </a></p><div class="supp"><p class="details"><div>T cells isolated on day 5 after translation from the spleens of C3H recipients bearing B6 heart allografts treated with PBS or recombinant lentivirus transduced DCs were used for analysis of T cell apoptosis <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> molecules by RT-PCR. All data are representative of three separate experiments.</div></p></div><div class="aux"><div class="resc">Jianbin Xiang, et al. J Biomed Biotechnol. 2010;2010:705431.</div><p class="links"><a class="dblinks" href="/pubmed/20490280" ref="ordinalpos=4374&amp;ncbi_uid=2366774&amp;link_uid=20490280">Citation</a><a class="dblinks" href="/pmc/articles/PMC2871569/" ref="ordinalpos=4374&amp;ncbi_uid=2366774&amp;link_uid=PMC2871569">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4375.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3742833/figure/F3/" ref="ordinalpos=4375&amp;ncbi_uid=5197058&amp;link_uid=PMC3742833" href="/pmc/articles/PMC3742833/"><img src="/pmc/articles/instance/3742833/bin/oncotarget-04-729-g003.gif" src-large="/pmc/articles/instance/3742833/bin/oncotarget-04-729-g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3742833/" ref="ordinalpos=4375&amp;ncbi_uid=5197058&amp;link_uid=PMC3742833" image-link="/pmc/articles/PMC3742833/figure/F3/" class="imagepopup">Figure 3. FAM83B expression confers resistance to therapeutic inhibition of the PI3K/AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: FAM83B-mediated activation of PI3K/AKT and MAPK <span class="highlight" style="background-color:">signaling</span> cooperates to promote epithelial cell transformation and resistance to targeted therapies. </a></p><div class="supp"><p class="details"><div>(a) HME1 cells expressing GFP or FAM83B were plated in the presence and absence of a PI3K inhibitor LY294002 (1, 5, and 10 M) and cell number quantified 5 days later. (b) HME1 cells expressing GFP or FAM83B were plated in the presence and absence of an AKT inhibitor MK2206 (1, 5, and 10 M) and cell number quantified 5 days later.</div></p></div><div class="aux"><div class="resc">Rocky Cipriano, et al. Oncotarget. 2013 May;4(5):729-738.</div><p class="links"><a class="dblinks" href="/pubmed/23676467" ref="ordinalpos=4375&amp;ncbi_uid=5197058&amp;link_uid=23676467">Citation</a><a class="dblinks" href="/pmc/articles/PMC3742833/" ref="ordinalpos=4375&amp;ncbi_uid=5197058&amp;link_uid=PMC3742833">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4376.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3655695/figure/F3/" ref="ordinalpos=4376&amp;ncbi_uid=6076755&amp;link_uid=PMC3655695" href="/pmc/articles/PMC3655695/"><img src="/pmc/articles/instance/3655695/bin/nihms-459439-f0003.gif" src-large="/pmc/articles/instance/3655695/bin/nihms-459439-f0003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3655695/" ref="ordinalpos=4376&amp;ncbi_uid=6076755&amp;link_uid=PMC3655695" image-link="/pmc/articles/PMC3655695/figure/F3/" class="imagepopup">Figure 3. VA neuroprotection of -Syn-overexpression in DA neurons is dependent on the ERK-MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: Valproic Acid Ameliorates C. elegans Dopaminergic Neurodegeneration with Implications for ERK-MAPK <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Cell-specific RNAi depletion of <i>mek-2</i> negates the dopaminergic neuroprotection provided by VA at all concentrations of examined compared to empty vector (EV) control, where neuroprotection is still observed at 1, 2, and 3 mM VA. <b>(B)</b> Similar to A, <i>mpk-1</i> depletion reverses neuroprotection of VA in dopaminergic neurons at 3 mM VA compared to EV control. P &lt; 0.05; Students t-test.</div></p></div><div class="aux"><div class="resc">Bwarenaba B. Kautu, et al. Neurosci Lett. 2013 April 29;541:116-119.</div><p class="links"><a class="dblinks" href="/pubmed/23485787" ref="ordinalpos=4376&amp;ncbi_uid=6076755&amp;link_uid=23485787">Citation</a><a class="dblinks" href="/pmc/articles/PMC3655695/" ref="ordinalpos=4376&amp;ncbi_uid=6076755&amp;link_uid=PMC3655695">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4377.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634812/figure/F5/" ref="ordinalpos=4377&amp;ncbi_uid=1715417&amp;link_uid=PMC2634812" href="/pmc/articles/PMC2634812/"><img src="/pmc/articles/instance/2634812/bin/nihms70868f5.gif" src-large="/pmc/articles/instance/2634812/bin/nihms70868f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634812/" ref="ordinalpos=4377&amp;ncbi_uid=1715417&amp;link_uid=PMC2634812" image-link="/pmc/articles/PMC2634812/figure/F5/" class="imagepopup">Figure 5.  From: Competition and collaboration: GATA-3, PU.1, and Notch <span class="highlight" style="background-color:">signaling</span> in early T-cell fate determination. </a></p><div class="supp"><p class="details"><div>Schematic of counterbalancing and dose-dependent regulatory forces that act on T-cell precursors through the ETP and DN2 stages. Regulators named in red denote forces diverting cells from the T-cell <span class="highlight" style="background-color:">pathway</span>. Of these, only EBF and Pax5 are not normally active in the thymus during these stages. Notch <span class="highlight" style="background-color:">signaling</span> acts as a constraint on all of these alternatives. This dynamic equilibrium is normally resolved at the DN3 stage, in part by the downregulation of PU.1, but almost certainly through other regulatory changes as well [163].</div></p></div><div class="aux"><div class="resc">Ellen V. Rothenberg, et al. Semin Immunol. ;20(4):236-246.</div><p class="links"><a class="dblinks" href="/pubmed/18768329" ref="ordinalpos=4377&amp;ncbi_uid=1715417&amp;link_uid=18768329">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634812/" ref="ordinalpos=4377&amp;ncbi_uid=1715417&amp;link_uid=PMC2634812">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4378.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2323231/figure/F3/" ref="ordinalpos=4378&amp;ncbi_uid=1339698&amp;link_uid=PMC2323231" href="/pmc/articles/PMC2323231/"><img src="/pmc/articles/instance/2323231/bin/gb-2007-8-7-r142-3.gif" src-large="/pmc/articles/instance/2323231/bin/gb-2007-8-7-r142-3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2323231/" ref="ordinalpos=4378&amp;ncbi_uid=1339698&amp;link_uid=PMC2323231" image-link="/pmc/articles/PMC2323231/figure/F3/" class="imagepopup">Figure 3.  From: siRNA screen of the human <span class="highlight" style="background-color:">signaling</span> proteome identifies the PtdIns(3,4,5)P3-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as a primary regulator of transferrin uptake. </a></p><div class="supp"><p class="details"><div>The PtdIns(3,4,5)P3-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> controls transferrin uptake. <b>(a) </b>Schematic representation of the PtdIns(3,4,5)P3-mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <b>(b) </b>Quantification of the effects of different siRNAs targeting the PtdIns(3,4,5)P3-mTOR <span class="highlight" style="background-color:">pathway</span> and rapamycin (RAPA) on transferrin uptake. Means  standard error of the mean (SEM; for d-siRNAs, <i>n </i>= 6; for synthetic siRNAs, <i>n </i>= 4; for rapamycin, <i>n </i>= 17). <b>(c-d) </b>Regulation of S6 protein phosphorylation by the PtdIns(3,4,5)P3-mTOR targeting d-siRNAs. Immunofluorescent images compared the effect of mTOR and GL3 luciferase (CTR) knock-downs on phosphorylated S6 protein staining (red). Nuclei are stained with Hoechst (blue). Scale bars, 10 m. Quantification of the staining-associated average fluorescence intensity (d). Means  SEM (<i>n </i>= 6). <b>(e) </b>Correlation between the effect of d-siRNAs on phosphorylation of S6 protein and transferrin uptake. Means  standard deviation (<i>n </i>= 2 experiments). <b>(f) </b>Inhibition of mTOR <span class="highlight" style="background-color:">signaling</span> by rapamycin (20 nM, 24 h) preferentially affects transferrin uptake when compared to DiI-labeled LDL or Alexa Fluor 488-labeled EGF. Average vesicular fluorescence intensities were measured. Mean  SEM (<i>n </i>= 6 replicates).</div></p></div><div class="aux"><div class="resc">Thierry Galvez, et al. Genome Biol. 2007;8(7):R142-R142.</div><p class="links"><a class="dblinks" href="/pubmed/17640392" ref="ordinalpos=4378&amp;ncbi_uid=1339698&amp;link_uid=17640392">Citation</a><a class="dblinks" href="/pmc/articles/PMC2323231/" ref="ordinalpos=4378&amp;ncbi_uid=1339698&amp;link_uid=PMC2323231">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4379.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316535/figure/F6/" ref="ordinalpos=4379&amp;ncbi_uid=398168&amp;link_uid=PMC316535" href="/pmc/articles/PMC316535/"><img src="/pmc/articles/instance/316535/bin/gad.8f6.gif" src-large="/pmc/articles/instance/316535/bin/gad.8f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316535/" ref="ordinalpos=4379&amp;ncbi_uid=398168&amp;link_uid=PMC316535" image-link="/pmc/articles/PMC316535/figure/F6/" class="imagepopup">Figure 6.  From: The Caenorhabditis elegans SH2 domain-containing protein tyrosine phosphatase PTP-2 participates in signal transduction during oogenesis  and vulval development. </a></p><div class="supp"><p class="details"><div>Genetic models for <i>ptp-2</i> <span class="highlight" style="background-color:">signaling</span> during (<i>A</i>) oogenesis and (<i>B</i>) vulval development. <i>C. elegans</i> genes are in italics, and their mammalian homologs are listed to the right. During oogenesis, PTP-2 likely functions to regulate positively <span class="highlight" style="background-color:">signaling</span> downstream from a yet uncharacterized PTK or R-PTK. PTP-2 may lead directly to the activation of the Ras, MEK, and MAPK cascade, or to the activation of an uncharacterized <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is required for proper oogenesis. During vulval development, PTP-2 appears to be acting as a positive regulator of LET-23 <span class="highlight" style="background-color:">signaling</span>, functioning downstream of or in parallel to LET-60 Ras, and in a <span class="highlight" style="background-color:">pathway</span> that cooperates with SEM-5. All genes known to participate in the regulation of vulval induction are not shown, and this model does not intend to imply that <i>ptp-2</i> does not interact with these genes, but rather that such putative relationships are yet to be defined.</div></p></div><div class="aux"><div class="resc">Michael J. Gutch, et al. Genes Dev. 1998 February 15;12(4):571-585.</div><p class="links"><a class="dblinks" href="/pubmed/9472025" ref="ordinalpos=4379&amp;ncbi_uid=398168&amp;link_uid=9472025">Citation</a><a class="dblinks" href="/pmc/articles/PMC316535/" ref="ordinalpos=4379&amp;ncbi_uid=398168&amp;link_uid=PMC316535">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4380.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3204345/figure/fig5/" ref="ordinalpos=4380&amp;ncbi_uid=3314084&amp;link_uid=PMC3204345" href="/pmc/articles/PMC3204345/"><img src="/pmc/articles/instance/3204345/bin/carcinbgr172f05_4c.gif" src-large="/pmc/articles/instance/3204345/bin/carcinbgr172f05_4c.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3204345/" ref="ordinalpos=4380&amp;ncbi_uid=3314084&amp;link_uid=PMC3204345" image-link="/pmc/articles/PMC3204345/figure/fig5/" class="imagepopup">Fig. 5. From: Smad phosphoisoform <span class="highlight" style="background-color:">signaling</span> specificity: the right place at the right time. </a></p><div class="supp"><p class="details"><div>Cell type-specific temporal dynamics of R-Smad phosphoisoforms. Although linker phosphorylation is transient after mitogen treatment of normal epithelial cells (<b>A</b>), mitogen-inducible phosphorylation generally persists in various mesenchymal cells (<b>B</b>). Moreover, constitutive linker phosphorylation is found in almost all types of carcinomas and Ras-transformed cells (<b>C</b>). Because mitogenic pSmad3L <span class="highlight" style="background-color:">signaling</span> is followed by the cytostatic pSmad3C <span class="highlight" style="background-color:">signaling</span> in normal epithelial homeostasis, pSmad2L/C and pSmad3L/C rarely exist in normal epithelial cells (A). Resembling observations in mesenchymal cells (B), carcinomas acquire an invasive phenotype via the pSmad2L/C <span class="highlight" style="background-color:">pathway</span> created by a combination of TGF- signal with intracellular Ras signal (C).</div></p></div><div class="aux"><div class="resc">Koichi Matsuzaki. Carcinogenesis. 2011 November;32(11):1578-1588.</div><p class="links"><a class="dblinks" href="/pubmed/21798854" ref="ordinalpos=4380&amp;ncbi_uid=3314084&amp;link_uid=21798854">Citation</a><a class="dblinks" href="/pmc/articles/PMC3204345/" ref="ordinalpos=4380&amp;ncbi_uid=3314084&amp;link_uid=PMC3204345">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4381.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2820783/figure/F3/" ref="ordinalpos=4381&amp;ncbi_uid=2774751&amp;link_uid=PMC2820783" href="/pmc/articles/PMC2820783/"><img src="/pmc/articles/instance/2820783/bin/zbc0101006530003.gif" src-large="/pmc/articles/instance/2820783/bin/zbc0101006530003.jpg" alt="FIGURE 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2820783/" ref="ordinalpos=4381&amp;ncbi_uid=2774751&amp;link_uid=PMC2820783" image-link="/pmc/articles/PMC2820783/figure/F3/" class="imagepopup">FIGURE 3. From: Cell <span class="highlight" style="background-color:">Signaling</span>, Internalization, and Nuclear Localization of the Angiotensin Converting Enzyme in Smooth Muscle and Endothelial Cells. </a></p><div class="supp"><p class="details"><div><b>ACE enhances the activation of PDGFR- <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> Analysis of the activation of <span class="highlight" style="background-color:">signaling</span> enzyme was done as described in the legend to Fig. 2. <i>A</i>, Western blot of total SMC extract, at different times after the addition of PDGF to cells that were not preincubated (<i>Control</i>) or were preincubated with 10 n<span style="font-variant: small-caps" class="small-caps">m</span> ACE for 120 min (<i>ACE-treated</i>). Graphs show the densitometry plots of the Western blot bands corresponding to activation of SHP2 (<i>B</i>) or AKT (<i>C</i>) by PDGF in control and ACE-treated cells. <i>D</i>, a longer exposure of the band corresponding to pPDGFR- (185 kDa) shown in <i>A. E</i>, densitometry plot of the Western blot bands corresponding to activation of PDGFR-.</div></p></div><div class="aux"><div class="resc">Hctor A. Lucero, et al. J Biol Chem. 2010 February 19;285(8):5555-5568.</div><p class="links"><a class="dblinks" href="/pubmed/20022959" ref="ordinalpos=4381&amp;ncbi_uid=2774751&amp;link_uid=20022959">Citation</a><a class="dblinks" href="/pmc/articles/PMC2820783/" ref="ordinalpos=4381&amp;ncbi_uid=2774751&amp;link_uid=PMC2820783">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4382.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2711543/figure/F1/" ref="ordinalpos=4382&amp;ncbi_uid=1909193&amp;link_uid=PMC2711543" href="/pmc/articles/PMC2711543/"><img src="/pmc/articles/instance/2711543/bin/nihms110404f1.gif" src-large="/pmc/articles/instance/2711543/bin/nihms110404f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2711543/" ref="ordinalpos=4382&amp;ncbi_uid=1909193&amp;link_uid=PMC2711543" image-link="/pmc/articles/PMC2711543/figure/F1/" class="imagepopup">Figure 1. Caspase-8 is essential for EGF-induced activation of the Erk 1/2 <span class="highlight" style="background-color:">pathway</span>.  From: Critical Role for Caspase-8 in EGF <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A) Immunoblot analysis of a time course of EGF (100 ng/mL)-induced Erk and Akt <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 (lanes 14) or same re-constituted with wild type protein (lanes 58). B) Immunoblot analysis of EGF (100 ng/mL, 3 min)-induced Erk <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 and reconstituted with empty vector (E.V. lanes 1 and 2), or with wild type protein (Casp8, lanes 3 and 4), or an inactivating point mutant of caspase-8 (Casp8C360A, lanes 5 and 6). Schematic depiction of caspase-8 showing domains and residues of interest in this study (below). C) Immunoblot analysis of PDGF (50 ng/mL, 10 min)-induced Erk and Akt <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 re-constituted with empty vector, wild type protein or an inactivating point mutant of caspase-8 (lanes 13, respectively). D) Immunoblot analysis of EGF-induced Erk <span class="highlight" style="background-color:">pathway</span> activation in NB7 cells lacking caspase-8 and re-constituted with empty vector, Casp8C360A or with the form of caspase-8 containing DEDs alone (lanes 13, respectively).</div></p></div><div class="aux"><div class="resc">Darren Finlay, et al. Cancer Res. ;69(12):5023-5029.</div><p class="links"><a class="dblinks" href="/pubmed/19470771" ref="ordinalpos=4382&amp;ncbi_uid=1909193&amp;link_uid=19470771">Citation</a><a class="dblinks" href="/pmc/articles/PMC2711543/" ref="ordinalpos=4382&amp;ncbi_uid=1909193&amp;link_uid=PMC2711543">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4383.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1142443/figure/fig2/" ref="ordinalpos=4383&amp;ncbi_uid=491519&amp;link_uid=PMC1142443" href="/pmc/articles/PMC1142443/"><img src="/pmc/articles/instance/1142443/bin/zmk0060571890002.gif" src-large="/pmc/articles/instance/1142443/bin/zmk0060571890002.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1142443/" ref="ordinalpos=4383&amp;ncbi_uid=491519&amp;link_uid=PMC1142443" image-link="/pmc/articles/PMC1142443/figure/fig2/" class="imagepopup">Figure 2. From: The Conserved Mec1/Rad53 Nuclear Checkpoint <span class="highlight" style="background-color:">Pathway</span> Regulates Mitochondrial DNA Copy Number in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>Activity of the Mec1/Rad53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulates mtDNA copy number. Plotted in the same manner as in Figure 1 is the relative mtDNA copy number of the strains indicated. (a) Strains analyzed were <i>sml1</i> null (<i>sml1</i>), <i>pif1</i> null (<i>pif1</i>), <i>pif1</i> null and <i>sml1</i> null (<i>pif1</i> <i>sml1</i>), wild type with the plasmid pBAD70 that overexpresses RNR1 (wt + pBAD70/<i>RNR1</i>), and wild type with empty vector control for pBAD70 comparison (wt + pRS304). (b) Strains analyzed were wild-type DBY2006 (wt), <i>rrm3</i> null (<i>rrm3</i>), <i>sml1</i> null (<i>sml1</i>), <i>rrm3</i> and <i>sml1</i> null (<i>rrm3</i> <i>sml1</i>), and <i>rad53</i> and <i>sml1</i> null (<i>rad53</i> <i>sml1</i>).</div></p></div><div class="aux"><div class="resc">Sean D. Taylor, et al. Mol Biol Cell. 2005 June;16(6):3010-3018.</div><p class="links"><a class="dblinks" href="/pubmed/15829566" ref="ordinalpos=4383&amp;ncbi_uid=491519&amp;link_uid=15829566">Citation</a><a class="dblinks" href="/pmc/articles/PMC1142443/" ref="ordinalpos=4383&amp;ncbi_uid=491519&amp;link_uid=PMC1142443">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4384.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2575163/figure/F8/" ref="ordinalpos=4384&amp;ncbi_uid=1518717&amp;link_uid=PMC2575163" href="/pmc/articles/PMC2575163/"><img src="/pmc/articles/instance/2575163/bin/zmk0110887560008.gif" src-large="/pmc/articles/instance/2575163/bin/zmk0110887560008.jpg" alt="Figure 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2575163/" ref="ordinalpos=4384&amp;ncbi_uid=1518717&amp;link_uid=PMC2575163" image-link="/pmc/articles/PMC2575163/figure/F8/" class="imagepopup">Figure 8. From: Antagonistic Regulation of Cell-Matrix Adhesion by FosB and ?FosB/?2?FosB Encoded by Alternatively Spliced Forms of fosB Transcripts. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> for proceeding from FosB or FosB/2FosB toward thrombospondin 1/TGF-1, which regulates cell-matrix adhesion. Thrombospondin 1/TGF-1 <span class="highlight" style="background-color:">signaling</span>, which may be negatively regulated by Jun/Fos complexes (active AP-1), enhances cell-matrix adhesion and up-regulates genes for basement membrane proteins through <i>fosB</i> gene expression. FosB and 2FosB antagonize not only FosB but also other Fos family members (Fos) constituting the major AP-1 complexes with the three Jun proteins (Jun), and they may up-regulate the TGF-1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> independent of intrinsic AP-1 activity. See details in the text and Supplemental Figure S2. Red represents up-regulated genes and blue indicates down-regulated genes.</div></p></div><div class="aux"><div class="resc">Yoshinori N. Ohnishi, et al. Mol Biol Cell. 2008 November;19(11):4717-4729.</div><p class="links"><a class="dblinks" href="/pubmed/18753407" ref="ordinalpos=4384&amp;ncbi_uid=1518717&amp;link_uid=18753407">Citation</a><a class="dblinks" href="/pmc/articles/PMC2575163/" ref="ordinalpos=4384&amp;ncbi_uid=1518717&amp;link_uid=PMC2575163">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4385.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2684941/figure/F1/" ref="ordinalpos=4385&amp;ncbi_uid=1813431&amp;link_uid=PMC2684941" href="/pmc/articles/PMC2684941/"><img src="/pmc/articles/instance/2684941/bin/nihms108550f1.gif" src-large="/pmc/articles/instance/2684941/bin/nihms108550f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2684941/" ref="ordinalpos=4385&amp;ncbi_uid=1813431&amp;link_uid=PMC2684941" image-link="/pmc/articles/PMC2684941/figure/F1/" class="imagepopup">Figure 1. Intracellular Phospholipases participate in Dual Pathways of Myocardial Lipid Signal Transduction.  From: Calcium Independent Phospholipases in the Heart: Mediators of Cellular <span class="highlight" style="background-color:">Signaling</span>, Bioenergetics and Ischemia-induced Electrophysiologic Dysfunction. </a></p><div class="supp"><p class="details"><div>Cytosolic phospholipases A<sub>2</sub> (cPLA<sub>2</sub>) and calcium-independent phospholipases A<sub>2</sub> (iPLA<sub>2</sub>) hydrolyze both diacyl and plasmalogen phospholipids to form free fatty acids and lysophospholipids. Thus, these intracellular phospholipases mediate dual pathways of cellular activation through the release of arachidonic acid and the concomitant production of lysolipids from membrane phospholipids. Remarkably, iPLA<sub>2</sub> activity on diacyl phospholipids with arachidonic acid at the <i>sn</i>-2 position generates 2-arachidonoyl lysophospholipids that are unique branch point metabolites in myocardial <span class="highlight" style="background-color:">signaling</span> since they can be further processed through an additional bifurcating <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to generate either eicosanoids or endocannabinoids for cellular <span class="highlight" style="background-color:">signaling</span> functions.</div></p></div><div class="aux"><div class="resc">Ari Cedars, et al. J Cardiovasc Pharmacol. ;53(4):277-289.</div><p class="links"><a class="dblinks" href="/pubmed/19390346" ref="ordinalpos=4385&amp;ncbi_uid=1813431&amp;link_uid=19390346">Citation</a><a class="dblinks" href="/pmc/articles/PMC2684941/" ref="ordinalpos=4385&amp;ncbi_uid=1813431&amp;link_uid=PMC2684941">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4386.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4394565/figure/Fig5/" ref="ordinalpos=4386&amp;ncbi_uid=7414341&amp;link_uid=PMC4394565" href="/pmc/articles/PMC4394565/"><img src="/pmc/articles/instance/4394565/bin/12865_2015_80_Fig5_HTML.gif" src-large="/pmc/articles/instance/4394565/bin/12865_2015_80_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4394565/" ref="ordinalpos=4386&amp;ncbi_uid=7414341&amp;link_uid=PMC4394565" image-link="/pmc/articles/PMC4394565/figure/Fig5/" class="imagepopup">Figure 5.  From: Wnt/-Catenin <span class="highlight" style="background-color:">signaling</span> reduces Bacillus Calmette-Guerin-induced macrophage necrosis through a ROS -mediated PARP/AIF-dependent <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>Activation of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> increases GSH concentration in RAW264.7 cells.</b> RAW264.7 cells were treated with BCG, Wnt3a, DKK1, H<sub>2</sub>O<sub>2</sub> or their combination as indicated, they were then used for determination of intracellular GSH levels by an ELISA for GSH. <b>(A)</b> Impact of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> on GSH production of RAW264.7 cells infected with BCG at MOI of 10 for 6 h. An activation of Wnt <span class="highlight" style="background-color:">signaling</span> by addition of Wnt3a-CM could induce GSH generation; in contrast, overexpression of Wnt inhibitor DKK1 reduced GSH levels. <b>(B)</b> Impact of Wnt/-catenin <span class="highlight" style="background-color:">signaling</span> on GSH production of RAW264.7 cells exposed to oxidative stress H<sub>2</sub>O<sub>2</sub> (500 mol/L) or LPS (100 ng/mL) stimulation for 6 h. The Wnt <span class="highlight" style="background-color:">signaling</span> could increase intracellular reduced GSH concentration in cells stressed by H<sub>2</sub>O<sub>2</sub> and LPS. Compared to a control-CM treated cells, *: p&lt;0.05, **: p&lt;0.01. Control-CM treated cells compared to its corresponding Wnt3a-CM treated cells, #: p&lt;0.05; ##: p&lt;0.01. Data represented the meanSD from three independent triplicate experiments (N=9).</div></p></div><div class="aux"><div class="resc">Xiaoling Wu, et al. BMC Immunol. 2015;16:16.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4386&amp;ncbi_uid=7414341&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4394565/" ref="ordinalpos=4386&amp;ncbi_uid=7414341&amp;link_uid=PMC4394565">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4387.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4227122/figure/F7/" ref="ordinalpos=4387&amp;ncbi_uid=6823105&amp;link_uid=PMC4227122" href="/pmc/articles/PMC4227122/"><img src="/pmc/articles/instance/4227122/bin/1423-0127-21-61-7.gif" src-large="/pmc/articles/instance/4227122/bin/1423-0127-21-61-7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4227122/" ref="ordinalpos=4387&amp;ncbi_uid=6823105&amp;link_uid=PMC4227122" image-link="/pmc/articles/PMC4227122/figure/F7/" class="imagepopup">Figure 7.  From: Activation of neutral-sphingomyelinase, MAPKs, and p75 NTR-mediating caffeic acid phenethyl ester-induced apoptosis in C6 glioma cells. </a></p><div class="supp"><p class="details"><div><b>Schematic presentation of the <span class="highlight" style="background-color:">signaling</span> pathways involved in CAPE-induced apoptotic <span class="highlight" style="background-color:">pathway</span> in C6 glioma cells.</b> The effect of CAPE on the MAPKs <span class="highlight" style="background-color:">signaling</span> triggering activation of NGF/p75NTR/JNK and p53 dependent apoptosis, which induces the N-SMase/ceramide up-regulation and increases an apoptosis cascade.</div></p></div><div class="aux"><div class="resc">Tsui-Hwa Tseng, et al. J Biomed Sci. 2014;21(1):61-61.</div><p class="links"><a class="dblinks" href="/pubmed/24997497" ref="ordinalpos=4387&amp;ncbi_uid=6823105&amp;link_uid=24997497">Citation</a><a class="dblinks" href="/pmc/articles/PMC4227122/" ref="ordinalpos=4387&amp;ncbi_uid=6823105&amp;link_uid=PMC4227122">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4388.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC316759/figure/F5/" ref="ordinalpos=4388&amp;ncbi_uid=404314&amp;link_uid=PMC316759" href="/pmc/articles/PMC316759/"><img src="/pmc/articles/instance/316759/bin/gad.10f5.gif" src-large="/pmc/articles/instance/316759/bin/gad.10f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC316759/" ref="ordinalpos=4388&amp;ncbi_uid=404314&amp;link_uid=PMC316759" image-link="/pmc/articles/PMC316759/figure/F5/" class="imagepopup">Figure 5.  From: A PDK1 homolog is necessary and sufficient to transduce AGE-1 PI3 kinase signals that regulate diapause in Caenorhabditis elegans. </a></p><div class="supp"><p class="details"><div>Model for regulation of reproductive growth and metabolism by the DAF-2 insulin receptor-like <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. On DAF-2 activation, AGE-1 generates PtdIns-3,4-P2 and PtdIns-3,4,5-P3 that are required for PDK-1 and AKT-1/AKT-2 activation. Other parallel pathways from DAF-2 are also activated and all <span class="highlight" style="background-color:">signaling</span> converges on DAF-16. See text for details.</div></p></div><div class="aux"><div class="resc">Suzanne Paradis, et al. Genes Dev. 1999 June 1;13(11):1438-1452.</div><p class="links"><a class="dblinks" href="/pubmed/10364160" ref="ordinalpos=4388&amp;ncbi_uid=404314&amp;link_uid=10364160">Citation</a><a class="dblinks" href="/pmc/articles/PMC316759/" ref="ordinalpos=4388&amp;ncbi_uid=404314&amp;link_uid=PMC316759">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4389.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2964935/figure/F5/" ref="ordinalpos=4389&amp;ncbi_uid=2535211&amp;link_uid=PMC2964935" href="/pmc/articles/PMC2964935/"><img src="/pmc/articles/instance/2964935/bin/nihms34081f5.gif" src-large="/pmc/articles/instance/2964935/bin/nihms34081f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2964935/" ref="ordinalpos=4389&amp;ncbi_uid=2535211&amp;link_uid=PMC2964935" image-link="/pmc/articles/PMC2964935/figure/F5/" class="imagepopup">Figure 5. The PAR <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>: Inputs and Outputs.  From: The PAR Proteins: Fundamental Players in Animal Cell Polarization. </a></p><div class="supp"><p class="details"><div>This schematic shows links between the PAR-3/PAR-6/aPKC polarity proteins and a variety of other <span class="highlight" style="background-color:">signaling</span> networks. The components of this scheme are drawn from different model systems, and it is unlikely that the details are common to all organisms or polarization processes. It is therefore somewhat speculative, but does indicate a possible mechanism by which cytoskeletal organization could trigger localized changes in phosphoinositide metabolism, which in turn localizes PAR-6/aPKC through CDC43-GTP.</div></p></div><div class="aux"><div class="resc">Bob Goldstein, et al. Dev Cell. ;13(5):609-622.</div><p class="links"><a class="dblinks" href="/pubmed/17981131" ref="ordinalpos=4389&amp;ncbi_uid=2535211&amp;link_uid=17981131">Citation</a><a class="dblinks" href="/pmc/articles/PMC2964935/" ref="ordinalpos=4389&amp;ncbi_uid=2535211&amp;link_uid=PMC2964935">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4390.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3617288/figure/F8/" ref="ordinalpos=4390&amp;ncbi_uid=5989887&amp;link_uid=PMC3617288" href="/pmc/articles/PMC3617288/"><img src="/pmc/articles/instance/3617288/bin/zbc0171344680008.gif" src-large="/pmc/articles/instance/3617288/bin/zbc0171344680008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3617288/" ref="ordinalpos=4390&amp;ncbi_uid=5989887&amp;link_uid=PMC3617288" image-link="/pmc/articles/PMC3617288/figure/F8/" class="imagepopup">FIGURE 8. From: A New <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> (JAK-Fes-phospholipase D) That Is Enhanced in Highly Proliferative Breast Cancer Cells. </a></p><div class="supp"><p class="details"><div><b>Model for the new PLD2-Fes-Jak3 <span class="highlight" style="background-color:">pathway</span>.</b> In MCF10A cells (<i>left</i>), both the protein expression levels and the enzymatic activity of the three <span class="highlight" style="background-color:">signaling</span> molecules (PLD2, JAK and Fes) are maintained at a low level. A complex inter-regulation equilibrium composed of activations and inhibitions keep PLD and the synthesis of PA in balance and at a basal level. In MDA-MB-231 cells (<i>right</i>), the situation is very different; both the protein expression levels and the activity of PLD2, JAK and Fes are upregulated. The kinases activate PLD that then synthesizes higher amounts of PA. Furthermore, this PA feeds back positively on Fes that, upon binding to the phospholipid, is further activated. <i>PC</i>, phosphatidylcholine; <i>HKD</i>, phospholipase D signature motif.</div></p></div><div class="aux"><div class="resc">Qing Ye, et al. J Biol Chem. 2013 April 5;288(14):9881-9891.</div><p class="links"><a class="dblinks" href="/pubmed/23404507" ref="ordinalpos=4390&amp;ncbi_uid=5989887&amp;link_uid=23404507">Citation</a><a class="dblinks" href="/pmc/articles/PMC3617288/" ref="ordinalpos=4390&amp;ncbi_uid=5989887&amp;link_uid=PMC3617288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4391.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2673765/figure/fig7/" ref="ordinalpos=4391&amp;ncbi_uid=1759938&amp;link_uid=PMC2673765" href="/pmc/articles/PMC2673765/"><img src="/pmc/articles/instance/2673765/bin/DEV034157F7.gif" src-large="/pmc/articles/instance/2673765/bin/DEV034157F7.jpg" alt="Fig. 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2673765/" ref="ordinalpos=4391&amp;ncbi_uid=1759938&amp;link_uid=PMC2673765" image-link="/pmc/articles/PMC2673765/figure/fig7/" class="imagepopup">Fig. 7. From: sonic hedgehog is required in pulmonary endoderm for atrial  septation. </a></p><div class="supp"><p class="details"><div><b>Expanded activation of Hedgehog <span class="highlight" style="background-color:">signaling</span> causes aberrant atrial septum  development.</b> (<b>A-H</b>) Constitutive activation of the Hh <span class="highlight" style="background-color:">pathway</span> using  the Cre-activated smoothened-M2 allele (<i>R26-smoM2</i>) in cells that  normally receive Hh signal has no effect on atrial septation, whereas  activating the Hh <span class="highlight" style="background-color:">pathway</span> in a broader domain results in a hypercellular  atrial septum. Whole-mount (A,E) and cross-section (B-D,F-G) histology. (A,B)  <i>Gli1-CreER<sup>T2</sup></i> expression domain demonstrated with the  <i>R26R</i> reporter. (C,D) Constitutive activation of the Hh <span class="highlight" style="background-color:">pathway</span> in the  <i>Gli1-CreER<sup>T2</sup></i> expression domain has no effect on the size  or overall structure of the atrial septum compared with littermate controls  (arrows). (E,F) <i>Nkx2-5<sup>Cre</sup></i> expression domain demonstrated  with the <i>R26R</i> reporter. Inset in F depicts  <i>Nkx2-5<sup>Cre</sup></i> expression in pulmonary endoderm. (G,H)  Constitutive activation of the Hh <span class="highlight" style="background-color:">pathway</span> in the broader  <i>Nkx2-5<sup>Cre</sup></i> domain results in an enlarged atrial septum when  compared with wild-type littermates (arrows).</div></p></div><div class="aux"><div class="resc">Andrew D. Hoffmann, et al. Development. 2009 May 15;136(10):1761-1770.</div><p class="links"><a class="dblinks" href="/pubmed/19369393" ref="ordinalpos=4391&amp;ncbi_uid=1759938&amp;link_uid=19369393">Citation</a><a class="dblinks" href="/pmc/articles/PMC2673765/" ref="ordinalpos=4391&amp;ncbi_uid=1759938&amp;link_uid=PMC2673765">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4392.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3399452/figure/fig2/" ref="ordinalpos=4392&amp;ncbi_uid=4136162&amp;link_uid=PMC3399452" href="/pmc/articles/PMC3399452/"><img src="/pmc/articles/instance/3399452/bin/IJCB2012-595976.002.gif" src-large="/pmc/articles/instance/3399452/bin/IJCB2012-595976.002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3399452/" ref="ordinalpos=4392&amp;ncbi_uid=4136162&amp;link_uid=PMC3399452" image-link="/pmc/articles/PMC3399452/figure/fig2/" class="imagepopup">Figure 2.  From: Targeting Metabolism and Autophagy in the Context of Haematologic Malignancies. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> pathways regulating autophagy and their inhibitors. Autophagy is a catabolic process that results in the autophagosomic-lysosomal degradation of bulk cytoplasmic contents. The kinase mTOR is a critical regulator of autophagy induction, with activated mTOR (PI3K/Akt) suppressing autophagy. AMPK-<span class="highlight" style="background-color:">signaling</span> negatively regulates mTOR <span class="highlight" style="background-color:">signaling</span> therefore promoting autophagy. ROS stress is an important inducer and regulator of autophagy generated by reduced oxidative phosphorylation and increased glycolysis. Nuclear 53 induces autophagy through transcriptional regulation of multiple genes whereas cytoplasmic p53 inhibits autophagy by blocking the mTOR <span class="highlight" style="background-color:">pathway</span>. Autophagy inhibitors chloroquine (HCQ/CQ) and metformin are under clinical investigation. Many chemotherapy/radiation therapies induce autophagy through ROS, inhibition of the mTOR <span class="highlight" style="background-color:">pathway</span> or nuclear p53 whereas PI3K/mTor inhibitors or rapamycin analogues specifically block the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to autophagy. </div></p></div><div class="aux"><div class="resc">Versha Banerji, et al. Int J Cell Biol. 2012;2012:595976.</div><p class="links"><a class="dblinks" href="/pubmed/22829831" ref="ordinalpos=4392&amp;ncbi_uid=4136162&amp;link_uid=22829831">Citation</a><a class="dblinks" href="/pmc/articles/PMC3399452/" ref="ordinalpos=4392&amp;ncbi_uid=4136162&amp;link_uid=PMC3399452">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4393.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3395658/figure/F7/" ref="ordinalpos=4393&amp;ncbi_uid=4254724&amp;link_uid=PMC3395658" href="/pmc/articles/PMC3395658/"><img src="/pmc/articles/instance/3395658/bin/2692fig7.gif" src-large="/pmc/articles/instance/3395658/bin/2692fig7.jpg" alt="FIGURE 7:" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3395658/" ref="ordinalpos=4393&amp;ncbi_uid=4254724&amp;link_uid=PMC3395658" image-link="/pmc/articles/PMC3395658/figure/F7/" class="imagepopup">FIGURE 7:.  From: The transcription factor Flo8 mediates CO2 sensing in the human fungal pathogen Candida albicans. </a></p><div class="supp"><p class="details"><div>Models of CO<sub>2</sub> mediated morphogenesis in <i>C. albicans</i>. The unidentified <span class="highlight" style="background-color:">pathway</span> represents the major <span class="highlight" style="background-color:">pathway</span> involved in CO<sub>2</sub>-induced responses. The cAMP <span class="highlight" style="background-color:">signaling</span> and the unidentified <span class="highlight" style="background-color:">pathway</span> converge on the key regulator Flo8 in the regulation of white-to-opaque switching, whereas the two pathways converge on the Flo8Efg1 complex in the regulation of filamentous growth. HSG, hyphal-specific genes; OSG, opaque-specific genes. (A) A model of CO<sub>2</sub>-induced filamentation. (B) A model of CO<sub>2</sub>-induced white-to-opaque switching. Wor1 contains a conserved potential PKA phosphorylation site and has been proven to function downstream of the cAMP <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Han Du, et al. Mol Biol Cell. 2012 July 15;23(14):2692-2701.</div><p class="links"><a class="dblinks" href="/pubmed/22621896" ref="ordinalpos=4393&amp;ncbi_uid=4254724&amp;link_uid=22621896">Citation</a><a class="dblinks" href="/pmc/articles/PMC3395658/" ref="ordinalpos=4393&amp;ncbi_uid=4254724&amp;link_uid=PMC3395658">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4394.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4325931/figure/F6/" ref="ordinalpos=4394&amp;ncbi_uid=7218744&amp;link_uid=PMC4325931" href="/pmc/articles/PMC4325931/"><img src="/pmc/articles/instance/4325931/bin/fgene-06-00014-g0006.gif" src-large="/pmc/articles/instance/4325931/bin/fgene-06-00014-g0006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4325931/" ref="ordinalpos=4394&amp;ncbi_uid=7218744&amp;link_uid=PMC4325931" image-link="/pmc/articles/PMC4325931/figure/F6/" class="imagepopup">Figure 6.  From: Toxicogenomics profiling of bone marrow from rats treated with topotecan in combination with oxaliplatin: a mechanistic strategy to inform combination toxicity. </a></p><div class="supp"><p class="details"><div><b>Overlay of gene expression on the p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>. The rat p53 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is overlaid with the average of the log base 2 ratio values from the replicate samples. Red denotes induction, green repression and yellow no change. Gray indicates that the gene was not mapped. The legend illustrates the segmentation of the genes according to the data from a given treatment.</div></p></div><div class="aux"><div class="resc">Myrtle Davis, et al. Front Genet. 2015;6:14.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=4394&amp;ncbi_uid=7218744&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4325931/" ref="ordinalpos=4394&amp;ncbi_uid=7218744&amp;link_uid=PMC4325931">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4395.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3405863/figure/A005975F4/" ref="ordinalpos=4395&amp;ncbi_uid=6439056&amp;link_uid=PMC3405863" href="/pmc/articles/PMC3405863/"><img src="/pmc/articles/instance/3405863/bin/cshperspect-SIG-005975_F4.gif" src-large="/pmc/articles/instance/3405863/bin/cshperspect-SIG-005975_F4.jpg" alt="Figure 4." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3405863/" ref="ordinalpos=4395&amp;ncbi_uid=6439056&amp;link_uid=PMC3405863" image-link="/pmc/articles/PMC3405863/figure/A005975F4/" class="imagepopup">Figure 4. From: <span class="highlight" style="background-color:">Signaling</span> Mechanisms Controlling Cell Fate and Embryonic Patterning. </a></p><div class="supp"><p class="details"><div>The EGFR <span class="highlight" style="background-color:">pathway</span> in <i>Drosophila.</i> Three membrane-anchored ligandsSpitz (SPI), Gurken (GRK), and Keren (KRN)are retained in the ER, and are processed following trafficking by the chaperone protein Star, which is dedicated to these molecules, to the Rab4/14 compartment in the secretory <span class="highlight" style="background-color:">pathway</span>. In this compartment, the ligands encounter Rhomboid proteins, seven-transmembrane-span intramembrane serine proteases, which cleave the ligand precursors within the transmembrane domain, to release the active, secreted form. Rhomboids also reside in the ER cleave and inactivate Star, thus attenuating the level of ligand precursor that is trafficked to the Rab4/14 compartment. Within the receiving cells, the ligands encounter the EGF receptor, which on dimerization triggers the canonical SOS/Ras/Raf/MEK/MAPK <span class="highlight" style="background-color:">pathway</span>. The cardinal transcriptional output of the <span class="highlight" style="background-color:">pathway</span> is mediated by the ETS protein Pointed (PNT). In addition, the ETS protein YAN provides a constitutive repressor, which competes for Pointed binding sites, and can be removed from the nucleus and degraded upon phosphorylation by ERK. Several negative regulators keep the <span class="highlight" style="background-color:">pathway</span> in check. Especially important is a group of inducible repressive elements, which constitute a negative-feedback loop. Argos is a secreted molecule, which sequesters the ligand SPI, whereas Sprouty and Kekkon1 attenuate <span class="highlight" style="background-color:">signaling</span> within the receiving cell.</div></p></div><div class="aux"><div class="resc">Norbert Perrimon, et al. Cold Spring Harb Perspect Biol. 2012 August 1;4(8):a005975.</div><p class="links"><a class="dblinks" href="/pubmed/22855721" ref="ordinalpos=4395&amp;ncbi_uid=6439056&amp;link_uid=22855721">Citation</a><a class="dblinks" href="/pmc/articles/PMC3405863/" ref="ordinalpos=4395&amp;ncbi_uid=6439056&amp;link_uid=PMC3405863">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4396.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3678929/figure/f1-rmmj_2-2-e0043/" ref="ordinalpos=4396&amp;ncbi_uid=5153874&amp;link_uid=PMC3678929" href="/pmc/articles/PMC3678929/"><img src="/pmc/articles/instance/3678929/bin/rmmj_2-2-e0043f1.gif" src-large="/pmc/articles/instance/3678929/bin/rmmj_2-2-e0043f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3678929/" ref="ordinalpos=4396&amp;ncbi_uid=5153874&amp;link_uid=PMC3678929" image-link="/pmc/articles/PMC3678929/figure/f1-rmmj_2-2-e0043/" class="imagepopup">Figure 1.  From: The Role of Insulin and Insulin-like Growth Factors in the Increased Risk of Cancer in Diabetes. </a></p><div class="supp"><p class="details"><div>The insulin receptor (IR) with its two subtypes IR-A and IR-B, the insulin growth factor 1 receptor (IGF-1R) and the hybrid receptors (IGF-1R/IR-A and IGF-1R/IR-B). <i>Structurally,</i> IR and the IGF-1R have two extracellular -subunits and two transmembrane -subunits that are joined to each other by disulfide bonds. <i>Affinity,</i> insulin binds with high affinity to IR-A or IR-B but has low affinity for IGF-1R, while insulin has no binding to the hybrid receptors. IGF-1 binds to the IGF-1R and to the hybrid receptors IGF-1R/IR-A or IGF-1R/IR-B. IGF-2 binds to IR-A, IGF-1R, or to IGF-1R/IR-A hybrid receptor. <i><span class="highlight" style="background-color:">Signaling</span>,</i> ligand binding to insulin receptor-A or to IGF-1 receptor mediates the mitogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, while ligand binding to insulin receptor-B activates metabolic <span class="highlight" style="background-color:">signaling</span>. Binding to the hybrid receptors, leading to mitogenic or metabolic <span class="highlight" style="background-color:">signaling</span>, is determined by the IR isoform that formed the hybrid receptors.</div></p></div><div class="aux"><div class="resc">Derek LeRoith, et al. Rambam Maimonides Med J. 2011 April;2(2):e0043.</div><p class="links"><a class="dblinks" href="/pubmed/23908801" ref="ordinalpos=4396&amp;ncbi_uid=5153874&amp;link_uid=23908801">Citation</a><a class="dblinks" href="/pmc/articles/PMC3678929/" ref="ordinalpos=4396&amp;ncbi_uid=5153874&amp;link_uid=PMC3678929">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4397.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2940477/figure/F8/" ref="ordinalpos=4397&amp;ncbi_uid=3192843&amp;link_uid=PMC2940477" href="/pmc/articles/PMC2940477/"><img src="/pmc/articles/instance/2940477/bin/zmg0091047000008.gif" src-large="/pmc/articles/instance/2940477/bin/zmg0091047000008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2940477/" ref="ordinalpos=4397&amp;ncbi_uid=3192843&amp;link_uid=PMC2940477" image-link="/pmc/articles/PMC2940477/figure/F8/" class="imagepopup">Figure 8.  From: Luteinizing Hormone Receptor-Stimulated Progesterone Production by Preovulatory Granulosa Cells Requires Protein Kinase A-Dependent Activation/Dephosphorylation of the Actin Dynamizing Protein Cofilin. </a></p><div class="supp"><p class="details"><div>Model of LHR <span class="highlight" style="background-color:">signaling</span> to cofilin. The <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> initiated by hCG binding to the LHR leading to cofilin dephosphorylation and resulting rearrangement of the actin cytoskeleton is depicted. <i>Bold lines</i> refer to consequences of LHR signals that promote the dephosphorylation of cofilin. AC, Adenylyl cyclase.</div></p></div><div class="aux"><div class="resc">Amelia B. Karlsson, et al. Mol Endocrinol. 2010 September;24(9):1765-1781.</div><p class="links"><a class="dblinks" href="/pubmed/20610540" ref="ordinalpos=4397&amp;ncbi_uid=3192843&amp;link_uid=20610540">Citation</a><a class="dblinks" href="/pmc/articles/PMC2940477/" ref="ordinalpos=4397&amp;ncbi_uid=3192843&amp;link_uid=PMC2940477">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4398.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3407530/figure/F5/" ref="ordinalpos=4398&amp;ncbi_uid=4143740&amp;link_uid=PMC3407530" href="/pmc/articles/PMC3407530/"><img src="/pmc/articles/instance/3407530/bin/1471-2407-12-203-5.gif" src-large="/pmc/articles/instance/3407530/bin/1471-2407-12-203-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3407530/" ref="ordinalpos=4398&amp;ncbi_uid=4143740&amp;link_uid=PMC3407530" image-link="/pmc/articles/PMC3407530/figure/F5/" class="imagepopup">Figure 5.  From: NF-?B2 mutation targets survival, proliferation and differentiation pathways in the pathogenesis of plasma cell tumors. </a></p><div class="supp"><p class="details"><div>A simplified model for p80HT targeting the IL-10-Stat3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to promote plasma cell expansion and tumor development.</div></p></div><div class="aux"><div class="resc">Brian A McCarthy, et al. BMC Cancer. 2012;12:203-203.</div><p class="links"><a class="dblinks" href="/pubmed/22642622" ref="ordinalpos=4398&amp;ncbi_uid=4143740&amp;link_uid=22642622">Citation</a><a class="dblinks" href="/pmc/articles/PMC3407530/" ref="ordinalpos=4398&amp;ncbi_uid=4143740&amp;link_uid=PMC3407530">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4399.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3881186/figure/F4/" ref="ordinalpos=4399&amp;ncbi_uid=5682199&amp;link_uid=PMC3881186" href="/pmc/articles/PMC3881186/"><img src="/pmc/articles/instance/3881186/bin/nihms262033f4.gif" src-large="/pmc/articles/instance/3881186/bin/nihms262033f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3881186/" ref="ordinalpos=4399&amp;ncbi_uid=5682199&amp;link_uid=PMC3881186" image-link="/pmc/articles/PMC3881186/figure/F4/" class="imagepopup">Figure 4.  From: Innate Immune DNA Sensing Pathways: STING, AIMII and the Regulation of Interferon Production and Inflammatory Responses. </a></p><div class="supp"><p class="details"><div>AIM2-dependent <span class="highlight" style="background-color:">signaling</span> and the inflammatory responses: Cytosolic DNA binds to the HIN-200 domains of AIM2 and activates the ASC/Caspase1 inflammasome <span class="highlight" style="background-color:">pathway</span> to produce IL-1.</div></p></div><div class="aux"><div class="resc">Glen N. Barber. Curr Opin Immunol. ;23(1):10.1016/j.coi.2010.12.015.</div><p class="links"><a class="dblinks" href="/pubmed/21239155" ref="ordinalpos=4399&amp;ncbi_uid=5682199&amp;link_uid=21239155">Citation</a><a class="dblinks" href="/pmc/articles/PMC3881186/" ref="ordinalpos=4399&amp;ncbi_uid=5682199&amp;link_uid=PMC3881186">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>4400.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2697172/figure/F6/" ref="ordinalpos=4400&amp;ncbi_uid=1831252&amp;link_uid=PMC2697172" href="/pmc/articles/PMC2697172/"><img src="/pmc/articles/instance/2697172/bin/1471-2164-10-220-6.gif" src-large="/pmc/articles/instance/2697172/bin/1471-2164-10-220-6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2697172/" ref="ordinalpos=4400&amp;ncbi_uid=1831252&amp;link_uid=PMC2697172" image-link="/pmc/articles/PMC2697172/figure/F6/" class="imagepopup">Figure 6.  From: A network-based integrative approach to prioritize reliable hits from multiple genome-wide RNAi screens in Drosophila. </a></p><div class="supp"><p class="details"><div><b>The reproducibility rate for OT-related and OT-unrelated hits (a), for low-ranked and high-ranked OT-related hits by NePhe score (b) and for low-ranked and high-ranked OT-unrelated hits by NePhe scores (c) for Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b>.</div></p></div><div class="aux"><div class="resc">Li Wang, et al. BMC Genomics. 2009;10:220-220.</div><p class="links"><a class="dblinks" href="/pubmed/19435510" ref="ordinalpos=4400&amp;ncbi_uid=1831252&amp;link_uid=19435510">Citation</a><a class="dblinks" href="/pmc/articles/PMC2697172/" ref="ordinalpos=4400&amp;ncbi_uid=1831252&amp;link_uid=PMC2697172">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="43" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="44" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="45" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=554271aeb24da7902e7742e5">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA ex...</a>
          <div class="ralinkpop offscreen_noflow">Systems biology of myasthenia gravis, integration of aberrant lncRNA and mRNA expression changes.<div class="brieflinkpopdesc">BMC Med Genomics. 2015 Mar 18 ;8(1):13. doi: 10.1186/s12920-015-0087-z.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=55427181437716036a500845">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray ...</a>
          <div class="ralinkpop offscreen_noflow">Revealing the Underlying Mechanism of Diabetic Nephropathy Viewed by Microarray Analysis.<div class="brieflinkpopdesc">Exp Clin Endocrinol Diabetes. 2015 Apr 28. [Epub ahead of print]</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=55427179437716036a5006e8">cytoscape <span class="number">(449)</span></a>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=554263a2b24da7902e74e9db">ACADL  [Bos taurus]</a>
          <div class="ralinkpop offscreen_noflow">ACADL  [Bos taurus]<div class="brieflinkpopdesc">Gene ID:614508</div></div>
          <div class="tertiary">Gene</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=554263a1d0eb51c2c24cf90c">614508[uid] AND (alive[property]) <span class="number">(1)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-05-01T01:46:56-04:00&amp;Host=ptpmc201&amp;Data=+NCBI_Phid:+F4FB2E81543134F1000000000001BB6D;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=F4FB2E81543134F1000000000001BB6D" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="2" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc201 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>