
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head xmlns:xi="http://www.w3.org/2001/XInclude"><meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <!-- meta -->
    <meta name="robots" content="index,nofollow,noarchive" />
<meta name="ncbi_app" content="entrez" /><meta name="ncbi_db" content="pmc" /><meta name="ncbi_term" content="signaling pathway" /><meta name="ncbi_report" content="imagesdocsum" /><meta name="ncbi_format" content="html" /><meta name="ncbi_pagesize" content="100" /><meta name="ncbi_sortorder" content="default" /><meta name="ncbi_pageno" content="6" /><meta name="ncbi_resultcount" content="40693" /><meta name="ncbi_op" content="page_change" /><meta name="ncbi_pdid" content="imagesdocsum" /><meta name="ncbi_sessionid" content="CE8A7326511A03C1_0067SID" /><meta name="ncbi_filter" content="pmc" /><meta name="ncbi_stat" content="false" /><meta name="ncbi_hitstat" content="false" />

    
    <!-- title -->
    <title>signaling pathway - PMC - NCBI</title>
    
    <!-- Common JS and CSS -->
    
		<script type="text/javascript">
		    var ncbi_startTime = new Date();
		</script>
		<script type="text/javascript" src="//static.pubmed.gov/core/jig/1.13.1/js/jig.min.js"></script>
            
            <link type="text/css" href="/corehtml/pmc/css/figpopup.css" rel="stylesheet" />
            <script type="text/javascript" src="/corehtml/pmc/js/figpopup.min.js"></script>
            
              
    
    <link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3881636/3579733.css" xml:base="http://127.0.0.1/sites/static/header_footer/" /><link xmlns="http://www.w3.org/1999/xhtml" type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/css/3529739.css" media="print" xml:base="http://127.0.0.1/sites/static/header_footer/" />    
<link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico" /><meta name="ncbi_phid" content="8A1A728253FE83810000000000F28929" /><script type="text/javascript"><!--
var ScriptUrl = 'http://www.ncbi.nlm.nih.gov/pmc';
var ScriptPath = '/portal/';
var objHierarchy = {"name":"EntrezSystem2","type":"Layout","realname":"EntrezSystem2",
"children":[{"name":"EntrezSystem2.PEntrez","type":"Cluster","realname":"EntrezSystem2.PEntrez",
"children":[{"name":"EntrezSystem2.PEntrez.DbConnector","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.DbConnector","shortname":"DbConnector"},
{"name":"EntrezSystem2.PEntrez.ParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.ParamContainer","shortname":"ParamContainer"},
{"name":"EntrezSystem2.PEntrez.MyNcbi","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.MyNcbi","shortname":"MyNcbi"},
{"name":"EntrezSystem2.PEntrez.UserPreferenceUrlParamContainer","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.UserPreferenceUrlParamContainer","shortname":"UserPreferenceUrlParamContainer"},
{"name":"EntrezSystem2.PEntrez.GridProperty","type":"Portlet","realname":"EntrezSystem2.PEntrez.PEntrez.GridProperty","shortname":"GridProperty"},
{"name":"EntrezSystem2.PEntrez.PMC","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NoPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NoPortlet","shortname":"NoPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_PageController","shortname":"Pmc_PageController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_SearchBar","shortname":"Pmc_SearchBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_BotRequest","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_BotRequest","shortname":"Entrez_BotRequest"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_LimitsTab","shortname":"Pmc_LimitsTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Facets","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Facets","shortname":"Entrez_Facets"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Clipboard","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Clipboard","shortname":"Entrez_Clipboard"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_StaticParts","shortname":"Pmc_StaticParts"},
{"name":"EntrezSystem2.PEntrez.PMC.Entrez_Messages","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.Entrez_Messages","shortname":"Entrez_Messages"},
{"name":"EntrezSystem2.PEntrez.PMC.NcbiJSCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NcbiJSCheck","shortname":"NcbiJSCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIBreadcrumbs","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIBreadcrumbs","shortname":"NCBIBreadcrumbs"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIHelpDesk","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIHelpDesk","shortname":"NCBIHelpDesk"},
{"name":"EntrezSystem2.PEntrez.PMC.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Entrez_Database.NCBIFooter_dynamic.NCBIApplog_NoScript_Ping","shortname":"NCBIApplog_NoScript_Ping"}]},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel","type":"Cluster","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel",
"children":[{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.blankToolPanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.blankToolPanel","shortname":"blankToolPanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController","shortname":"Pmc_ResultsSearchController"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_FilterTab","shortname":"Pmc_FilterTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Pager","shortname":"Entrez_Pager"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar","shortname":"Pmc_DisplayBar"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HelpFormAttributes","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HelpFormAttributes","shortname":"HelpFormAttributes"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Collections","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Entrez_Collections","shortname":"Entrez_Collections"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SpellCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SpellCheck","shortname":"SpellCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.SearchEngineReferralCheck","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.SearchEngineReferralCheck","shortname":"SearchEngineReferralCheck"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.MultiSensorPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.MultiSensorPortlet","shortname":"MultiSensorPortlet"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.KnowledgePanel","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.KnowledgePanel","shortname":"KnowledgePanel"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.HistoryDisplay","shortname":"HistoryDisplay"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Discovery_SearchDetails","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_ResultsPanel.Discovery_SearchDetails","shortname":"Discovery_SearchDetails"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab","shortname":"EmailTab"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Images_RVDocSum","shortname":"Images_RVDocSum"},
{"name":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","type":"Portlet","realname":"EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmptyPortlet","shortname":"EmptyPortlet"}]}]}]}]};
--></script><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3808861/3917732/3974050/3751656/3395415/3326508/3871380/4033160/4032807/3577051/3858295/3331221/12930/4033350/4024054/3861632/63120/3395586.css" /><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/css/3501913/1303451/3529739.css" media="print" /><script type="text/javascript">

var ObjectLinks=[{i:0, ename: "p$ExL", esid:"*", sname: "p$ExL", ssid:"*", dname:"p$el", dsid:"0",m:"CopyValue",p:[],f: function(src, dst) {fn_CopyValue(src, dst);}}]


var ActiveNames = {"p$ExL":1, "EntrezSystem2.PEntrez.DbConnector.Cmd":0, "EntrezSystem2.PEntrez.DbConnector.Db":0, "EntrezSystem2.PEntrez.DbConnector.IdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastDb":0, "EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult":0, "EntrezSystem2.PEntrez.DbConnector.LastQueryKey":0, "EntrezSystem2.PEntrez.DbConnector.LastTabCmd":0, "EntrezSystem2.PEntrez.DbConnector.LinkName":0, "EntrezSystem2.PEntrez.DbConnector.LinkReadableName":0, "EntrezSystem2.PEntrez.DbConnector.LinkSrcDb":0, "EntrezSystem2.PEntrez.DbConnector.QueryKey":0, "EntrezSystem2.PEntrez.DbConnector.TabCmd":0, "EntrezSystem2.PEntrez.DbConnector.Term":0, "EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount":0, "EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery":0};
</script></head>
    <body>
    	<div class="grid">
			<div class="col twelve_col nomargin shadow">
				<form enctype="application/x-www-form-urlencoded" name="EntrezForm" method="post" onsubmit="return false;" action="/pmc" id="EntrezForm">
					<div xmlns:xi="http://www.w3.org/2001/XInclude">
    <!-- no javascript message -->
    
    
    <div xmlns="http://www.w3.org/1999/xhtml" class="universal_header" id="universal_header" xml:base="http://127.0.0.1/sites/static/header_footer/"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list" id="navcontent"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt"><a class="ui-ncbimenu-link-first" href="/" role="banner" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4054678/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo" /></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources">Resources</a><ul id="resources-submenu"><li><a href="/guide/all/">All Resources</a></li><li><a href="#chemicals-bioassays">Chemicals &amp; Bioassays</a><ul class="actual-resources-submenu chemicals-bioassays"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/pcassay" title=" Contributed bioactivity data for PubChem Substances ">PubChem BioAssay</a></li><li><a href="/pccompound" title=" Unique, standardized chemical structures ">PubChem Compound</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="/pcsubstance" title=" Provides information on the biological activities of small molecules. ">PubChem Substance</a></li><li><a href="/guide/chemicals-bioassays/">All Chemicals &amp; Bioassays Resources...</a></li></ul></li><li><a href="#dna-rna">DNA &amp; RNA</a><ul class="actual-resources-submenu dna-rna"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/genbank/" title=" DNA and RNA sequences submitted to NCBI ">GenBank</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/dna-rna/">All DNA &amp; RNA Resources...</a></li></ul></li><li><a href="#data-software">Data &amp; Software</a><ul class="actual-resources-submenu data-software"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/WebSub/?tool=genbank" title=" Web tool for simple submissions of nucleotide sequences ">GenBank: BankIt</a></li><li><a href="/projects/Sequin/" title=" Software tool for complex submissions of nucleotide sequences ">GenBank: Sequin</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="http://pubchem.ncbi.nlm.nih.gov/search/search.cgi" title=" Searches PubChem Compound by chemical structure ">PubChem Structure Search</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/data-software/">All Data &amp; Software Resources...</a></li></ul></li><li><a href="#domains-structures">Domains &amp; Structures</a><ul class="actual-resources-submenu domains-structures"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/Structure/CN3D/cn3d.shtml" title=" Program for viewing 3D structures and alignments ">Cn3D</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/structure/" title=" Database of 3D macromolecular structures derived from PDB ">Structure (Molecular Modeling Database)</a></li><li><a href="/Structure/VAST/vast.html" title=" Tool that generates 3D structural alignments ">Vector Alignment Search Tool (VAST)</a></li><li><a href="/guide/domains-structures/">All Domains &amp; Structures Resources...</a></li></ul></li><li><a href="#genes-expression">Genes &amp; Expression</a><ul class="actual-resources-submenu genes-expression"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/gene" title=" Database of gene loci and associated transcripts and proteins ">Gene</a></li><li><a href="/geo/" title=" Gene Expression Omnibus (GEO) Database ">Gene Expression Omnibus (GEO) Database </a></li><li><a href="/gds" title=" Curated collections of gene expression data sets ">Gene Expression Omnibus (GEO) Datasets</a></li><li><a href="/geoprofiles/" title=" Database of gene expression Profiles for individual sequences ">Gene Expression Omnibus (GEO) Profiles</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/unigene" title=" Gene-centered clusters of transcripts and ESTs ">UniGene</a></li><li><a href="/guide/genes-expression/">All Genes &amp; Expression Resources...</a></li></ul></li><li><a href="#genetics-medicine">Genetics &amp; Medicine</a><ul class="actual-resources-submenu genetics-medicine"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/gtr/" title=" Detailed information about genetic tests and labs ">Genetic Testing Registry</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/omim" title=" Database of human genes and genetic disorders ">Online Mendelian Inheritance in Man (OMIM)</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/refseq/rsg/" title=" RefSeqGene reference standards for genomic sequences of well-characterized genes ">RefSeqGene</a></li><li><a href="/guide/genetics-medicine/">All Genetics &amp; Medicine Resources...</a></li></ul></li><li><a href="#genomes-maps">Genomes &amp; Maps</a><ul class="actual-resources-submenu genomes-maps"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/genbank/tbl2asn2/" title=" Command-line program for submitting sequence data ">GenBank: tbl2asn</a></li><li><a href="/sites/genome" title=" Curated genomic DNA sequences ">Genome</a></li><li><a href="/bioproject/" title=" Central status pages for genome sequencing projects ">Genome Project</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/mapview/" title=" Tool for searching, viewing and downloading genomic maps and related data ">Map Viewer</a></li><li><a href="/nuccore" title=" Nucleotide sequence data in GenBank and RefSeq ">Nucleotide Database</a></li><li><a href="/popset" title=" Sets of nucleotide sequences representing a population ">PopSet</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/Traces/sra/sra.cgi?" title=" Database of sequences reads from next generation sequencing technologies ">Sequence Read Archive (SRA)</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/Traces/trace.cgi" title=" DNA chromatograms, base calls, and quality estimates for single-pass reads. ">Trace Archive</a></li><li><a href="/guide/genomes-maps/">All Genomes &amp; Maps Resources...</a></li></ul></li><li><a href="#homology">Homology</a><ul class="actual-resources-submenu homology"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/homologene" title=" Sets of homologous eukaryotic genes ">HomoloGene</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/guide/homology/">All Homology Resources...</a></li></ul></li><li><a href="#literature">Literature</a><ul class="actual-resources-submenu literature"><li><a href="/books" title=" Online biomedical books ">Bookshelf</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/nlmcatalog/journals" title=" Information on journals cited by NCBI records ">Journals in NCBI Databases</a></li><li><a href="/mesh" title=" Controlled vocabulary for MEDLINE ">MeSH Database</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/pubmed" title=" Biomedical literature abstracts ">PubMed</a></li><li><a href="/pmc/" title=" Free, full-text articles ">PubMed Central (PMC)</a></li><li><a href="/pubmed/clinical " title=" Directory of PubMed queries about clinical topics ">PubMed Clinical Queries</a></li><li><a href="/pubmedhealth/" title=" Reviews of clinical effectiveness research ">PubMed Health</a></li><li><a href="/guide/literature/">All Literature Resources...</a></li></ul></li><li><a href="#proteins">Proteins</a><ul class="actual-resources-submenu proteins"><li><a href="/biosystems/" title=" Database that groups biomedical literature, small molecules, and sequence data in terms of biological relationships. ">BioSystems</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/cdd" title=" Database of conserved protein domains represented by multiple sequence alignments ">Conserved Domain Database (CDD)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/books/NBK25501/ " title=" URL application programming interface for searching, linking and downloading data within NCBI's Entrez system ">E-Utilities</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/proteinclusters" title=" Clusters of related proteins from prokaryotes and organelles ">Protein Clusters</a></li><li><a href="/protein" title=" Protein sequence data ">Protein Database</a></li><li><a href="/refseq/" title=" Curated nucleotide and protein sequences produced by NCBI ">Reference Sequence (RefSeq)</a></li><li><a href="/guide/proteins/">All Proteins Resources...</a></li></ul></li><li><a href="#sequence-analysis">Sequence Analysis</a><ul class="actual-resources-submenu sequence-analysis"><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi" title=" Tool for finding local regions of sequence similarity ">BLAST (Basic Local Alignment Search Tool)</a></li><li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" title=" BLAST programs for a local computer ">BLAST (Stand-alone)</a></li><li><a href="/sutils/blink.cgi?mode=query" title=" Pre-computed reports showing proteins with similar sequences ">BLAST Link (BLink)</a></li><li><a href="/Structure/cdd/wrpsb.cgi" title=" Locates conserved domains within a protein sequence ">Conserved Domain Search Service (CD Search)</a></li><li><a href="/sutils/protmap.cgi" title=" Maps proteins from a COG/VOG back to its genome ">Genome ProtMap</a></li><li><a href="/tools/gbench/" title=" Stand-alone application for analyzing sequence data ">Genome Workbench</a></li><li><a href="/genomes/FLU/" title=" Data from the NIAID Influenza Genome Sequencing Project ">Influenza Virus</a></li><li><a href="/tools/primer-blast/" title=" Designs PCR primers and analyzes them for target specificity using BLAST. ">Primer-BLAST</a></li><li><a href="/sutils/static/prosplign/prosplign.html" title=" Tool for aligning proteins to genomic DNA ">ProSplign</a></li><li><a href="/sutils/splign/" title=" Tool for aligning transcripts to genomic DNA ">Splign</a></li><li><a href="/guide/sequence-analysis/">All Sequence Analysis Resources...</a></li></ul></li><li><a href="#taxonomy">Taxonomy</a><ul class="actual-resources-submenu taxonomy"><li><a href="/taxonomy" title=" The NCBI taxonomy tree in Entrez ">Taxonomy</a></li><li><a href="/Taxonomy/Browser/wwwtax.cgi?mode=Root" title=" Displays and links Entrez data to each taxonomic node ">Taxonomy Browser</a></li><li><a href="/Taxonomy/CommonTree/wwwcmt.cgi" title=" Creates custom taxonomic trees ">Taxonomy Common Tree</a></li><li><a href="/guide/taxonomy/">All Taxonomy Resources...</a></li></ul></li><li><a href="#training-tutorials">Training &amp; Tutorials</a><ul class="actual-resources-submenu training-tutorials"><li><a href="/education/" title=" Central page providing links to tutorials and training materials ">NCBI Education Page</a></li><li><a href="/books/NBK143764/" title=" Introductory articles about NCBI resources ">NCBI Handbook</a></li><li><a href="/books/NBK3831/" title=" Online, searchable help documentation ">NCBI Help Manual</a></li><li><a href="/books/NBK1969/" title=" NCBI's monthly newsletter ">NCBI News</a></li><li><a href="/guide/training-tutorials/">All Training &amp; Tutorials Resources...</a></li></ul></li><li><a href="#variation">Variation</a><ul class="actual-resources-submenu variation"><li><a href="/dbvar" title=" A database of genomic structural variation ">Database of Genomic Structural Variation (dbVar)</a></li><li><a href="/gap" title=" Genotype/Phenotype association studies ">Database of Genotypes and Phenotypes (dbGaP)</a></li><li><a href="/snp" title=" Database of nucleotide sequence polymorphisms ">Database of Single Nucleotide Polymorphisms (dbSNP)</a></li><li><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" title=" The SNP database tools page provides links to the general submission guidelines and to the submission handle request. ">SNP Submission Tool</a></li><li><a href="/guide/variation/">All Variation Resources...</a></li></ul></li></ul></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto">How To</a><ul id="howto-submenu"><li><a href="/guide/all/#howtos">All How To</a></li><li><a href="/guide/chemicals-bioassays/#howtos">Chemicals &amp; Bioassays</a></li><li><a href="/guide/dna-rna/#howtos">DNA &amp; RNA</a></li><li><a href="/guide/data-software/#howtos">Data &amp; Software</a></li><li><a href="/guide/domains-structures/#howtos">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/#howtos">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/#howtos">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/#howtos">Genomes &amp; Maps</a></li><li><a href="/guide/homology/#howtos">Homology</a></li><li><a href="/guide/literature/#howtos">Literature</a></li><li><a href="/guide/proteins/#howtos">Proteins</a></li><li><a href="/guide/sequence-analysis/#howtos">Sequence Analysis</a></li><li><a href="/guide/taxonomy/#howtos">Taxonomy</a></li><li><a href="/guide/training-tutorials/#howtos">Training &amp; Tutorials</a></li><li><a href="/guide/variation/#howtos">Variation</a></li></ul></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi"><span id="myncbiusername" style="display:none"><a href="/account/settings/" id="mnu" title="Edit account settings"></a></span><a accesskey="2" href="/myncbi/" id="myncbi" style="display:none">My NCBI</a><a href="/account/" id="sign_in">Sign in to NCBI</a><a href="/account/signout/" id="sign_out" style="display:none">Sign Out</a></div></div>
    
    <div class="header">
    <!-- logo -->
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="http://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br />
    <a href="http://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <!-- SearchBar -->
    <div class="search"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="clone">Clone</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="epigenomics">Epigenomics</option><option value="nucest">EST</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="nucgss">GSS</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="medgen" data-ac_dict="medgen_disease_name">MedGen</option><option value="mesh" data-ac_dict="mesh_suggestions">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed" data-ac_dict="pm_related_queries_2">PubMed</option><option value="pubmedhealth">PubMed Health</option><option value="snp">SNP</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbook">ToolKitBook</option><option value="unigene">UniGene</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><input type="text" name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="signaling pathway" class="jig-ncbiclearbutton jig-ncbiautocomplete" config="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'yes',afs:'yes'" /></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div><ul class="searchlinks inline_list"><set></set><li><a name="SaveSearch" title="Click to save search strategy in My NCBI" href="/sites/myncbi/searches/save?db=pmc&amp;qk=1" id="SaveSearch">Save search</a></li><li><a href="/pmc/journals/">Journal List</a></li><li><a sid="1" href="/pmc/limits?term=signaling pathway">Limits</a></li><li><a href="/pmc/advanced">Advanced</a></li><li class="help"><a id="help" target="_blank" name="help" href="/books/NBK3825">Help</a></li></ul></div>
</div>   
    <input name="EntrezSystem2.PEntrez.PMC.Pmc_PageController.PreviousPageName" sid="1" type="hidden" value="results" />
    <div class="hidden-email">
        <div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailReport" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailFormat" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailCount" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailStart" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSort" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.Email" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailSubject" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailText" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.EmailQueryKey" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.EmailTab.QueryDescription" sid="1" type="hidden" /></div><!-- hidden inputs -->    
    </div>    
    <div id="maincontent" class="col nine_col">
    <div class="content">
        <div>
    
    
</div>
        <div class="results_settings two_settings"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set" data-jigconfig="triggerPosition : 'bottom center',destPosition : 'top center',destSelector : '#display_settings_menu',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="2" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="3" href="#" sourceContent="display_settings_menu" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="1" value="5" id="ps5" /><label for="ps5">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="2" value="10" id="ps10" /><label for="ps10">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="3" value="20" id="ps20" /><label for="ps20">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="4" value="50" id="ps50" /><label for="ps50">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize" sid="5" value="100" id="ps100" checked="true" /><label for="ps100">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="1" class="button_apply ncbipopper-close-button">Apply</button></div><div><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.FileFormat" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPresentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Presentation" sid="1" type="hidden" value="imagesdocsum" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.PageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastPageSize" sid="1" type="hidden" value="100" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Format" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.LastFormat" sid="1" type="hidden" value="" /></div></div>
        <div class="title_and_pager">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="1" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="2" page="5" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno" type="text" class="num" sid="1" value="6" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="3" page="7" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="4" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.CurrPage" sid="1" type="hidden" value="6" /></div>
            <div><h2 class="result_count">Results: 501 to 600 of 40693</h2><span id="result_sel" class="nowrap"></span><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.ResultCount" sid="1" type="hidden" id="resultcount" value="40693" /><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_ResultsSearchController.RunLastQuery" sid="1" type="hidden" /></div>
        </div>
        <div id="messagearea" class="empty">
            
            
            
        </div>
        <div><div class="rprt"><div class="rprtnum nohighlight"><span>501.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3075259/figure/pone-0018636-g004/" ref="ordinalpos=501&amp;ncbi_uid=2909419&amp;link_uid=PMC3075259" href="/pmc/articles/PMC3075259/"><img src="/pmc/articles/instance/3075259/bin/pone.0018636.g004.gif" src-large="/pmc/articles/instance/3075259/bin/pone.0018636.g004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3075259/" ref="ordinalpos=501&amp;ncbi_uid=2909419&amp;link_uid=PMC3075259" image-link="/pmc/articles/PMC3075259/figure/pone-0018636-g004/" class="imagepopup">Figure 4. <span class="highlight" style="background-color:">Pathway</span> diagram showing the hypothetical positions of the GBM survival associated genes MTR and CCNB1 in the glioblastoma p53 <span class="highlight" style="background-color:">pathway</span> [3]..  From: Data Integration Workflow for Search of Disease Driving Genes and Genetic Variants. </a></p><div class="supp"><p class="details"><div>KEGG annotates <i>CCNB1</i> to belong to the <i>p53</i> <span class="highlight" style="background-color:">pathway</span> together with <i>CCNB2</i>, <i>CCNB3</i> and <i>CDC2</i> [6]. In the KEGG <i>p53</i> <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, these are regulated by the genes <i>SFN</i>, <i>GADD45G</i>, <i>GADD45A</i> and <i>GADD45B</i>, which constitute a subset of the <i>p53</i> target genes. In the KEGG glioma <span class="highlight" style="background-color:">pathway</span>, G2 arrest impairment is annotated as one result of the altered <i>p53</i> <span class="highlight" style="background-color:">pathway</span> activation [6].</div></p></div><div class="aux"><div class="resc">Sirkku Karinen, et al. PLoS One. 2011;6(4):e18636.</div><p class="links"><a class="dblinks" href="/pubmed/21533266" ref="ordinalpos=501&amp;ncbi_uid=2909419&amp;link_uid=21533266">Citation</a><a class="dblinks" href="/pmc/articles/PMC3075259/" ref="ordinalpos=501&amp;ncbi_uid=2909419&amp;link_uid=PMC3075259">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>502.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3831179/figure/F14/" ref="ordinalpos=502&amp;ncbi_uid=6329965&amp;link_uid=PMC3831179" href="/pmc/articles/PMC3831179/"><img src="/pmc/articles/instance/3831179/bin/nihms-440651-f0014.gif" src-large="/pmc/articles/instance/3831179/bin/nihms-440651-f0014.jpg" alt="Figure 14" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3831179/" ref="ordinalpos=502&amp;ncbi_uid=6329965&amp;link_uid=PMC3831179" image-link="/pmc/articles/PMC3831179/figure/F14/" class="imagepopup">Figure 14.  From: Integrating In Silico Resources to Map a <span class="highlight" style="background-color:">Signaling</span> Network. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> databases. <b>(A)</b> EGFR <span class="highlight" style="background-color:">pathway</span> map from BioCarta (http://www.biocarta.com/pathfiles/h_egfPathway.asp; permission to use this image was granted by BioCarta, LLC). <b>(B)</b> Overlap of <span class="highlight" style="background-color:">pathway</span> entities for the EGFR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> among <span class="highlight" style="background-color:">pathway</span> datasets retrieved from Netpath, BioCarta, Protein Lounge, STKE and Cell Designer (CellDes). 371 unique interactors were collectively identified using all five <span class="highlight" style="background-color:">pathway</span> databases. Out of these 371 interactors, only 103, 46, 32, 15 or 10 interactors were mentioned in at least two, three, four or all five databases, respectively. For investigators who are interested in only ‘High-Confidence’ datasets (Figure 10), the 10 interactors identified in all five databases mentioned in this example, would be of particular interest.</div></p></div><div class="aux"><div class="resc">Hanqing Liu, et al. Methods Mol Biol. ;1101:10.1007/978-1-62703-721-1_11.</div><p class="links"><a class="dblinks" href="/pubmed/24233784" ref="ordinalpos=502&amp;ncbi_uid=6329965&amp;link_uid=24233784">Citation</a><a class="dblinks" href="/pmc/articles/PMC3831179/" ref="ordinalpos=502&amp;ncbi_uid=6329965&amp;link_uid=PMC3831179">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>503.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2674579/figure/F1/" ref="ordinalpos=503&amp;ncbi_uid=1772555&amp;link_uid=PMC2674579" href="/pmc/articles/PMC2674579/"><img src="/pmc/articles/instance/2674579/bin/nihms-108483-f0001.gif" src-large="/pmc/articles/instance/2674579/bin/nihms-108483-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2674579/" ref="ordinalpos=503&amp;ncbi_uid=1772555&amp;link_uid=PMC2674579" image-link="/pmc/articles/PMC2674579/figure/F1/" class="imagepopup">Figure 1.  From: PERK-dependent regulation of MDA-7/IL-24-induced autophagy in primary human glioma cells. </a></p><div class="supp"><p class="details"><div>GST-MDA-7 causes PERK-dependent vacuolization and JNK <span class="highlight" style="background-color:">pathway</span> activation in transformed cells that leads to cell death. Exposure to GST-MDA-7 causes a PERK-dependent activation of the JNK <span class="highlight" style="background-color:">pathway</span> and a PERK-dependent inactivation of the ERK1/2 <span class="highlight" style="background-color:">pathway</span>. Enhanced JNK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> plays a key role in the activation of pro-caspase 2, pro-caspase 4 and cathepsin B. Activation of cathepsin B promotes the cleavage/activation of the BH3 domain protein BID. Elevated JNK <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span> also activates BAX. Inactivation of the ERK1/2 <span class="highlight" style="background-color:">pathway</span> facilitates activation of BAD and BIM. Thus GST-MDA-7, via PERK <span class="highlight" style="background-color:">signaling</span>, promotes increased activity of at least 4 BH3 domain proteins which all act to cause mitochondrial dysfunction and ultimately promote cell killing.</div></p></div><div class="aux"><div class="resc">Margaret A. Park, et al. Autophagy. ;4(4):513-515.</div><p class="links"><a class="dblinks" href="/pubmed/18299661" ref="ordinalpos=503&amp;ncbi_uid=1772555&amp;link_uid=18299661">Citation</a><a class="dblinks" href="/pmc/articles/PMC2674579/" ref="ordinalpos=503&amp;ncbi_uid=1772555&amp;link_uid=PMC2674579">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>504.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3880197/figure/F2/" ref="ordinalpos=504&amp;ncbi_uid=5650896&amp;link_uid=PMC3880197" href="/pmc/articles/PMC3880197/"><img src="/pmc/articles/instance/3880197/bin/nihms540200f2.gif" src-large="/pmc/articles/instance/3880197/bin/nihms540200f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3880197/" ref="ordinalpos=504&amp;ncbi_uid=5650896&amp;link_uid=PMC3880197" image-link="/pmc/articles/PMC3880197/figure/F2/" class="imagepopup">Figure 2. ROS-induced cellular <span class="highlight" style="background-color:">signaling</span>.  From: Reactive oxygen species in cancer. </a></p><div class="supp"><p class="details"><div>Reactive oxygen species in cells can be generated by growth factor <span class="highlight" style="background-color:">signaling</span> through activation of the NADPH oxidase NOX1 or through the mitochondria. These ROS then can induce cellular <span class="highlight" style="background-color:">signaling</span> cascades by reversible oxidation of phosphatases such as PTEN or PTP in their active site cysteins or by direct oxidation of kinases such as Src. This leads to the activation of several <span class="highlight" style="background-color:">signaling</span> cascades such as a Src/PKD1-dependent NF-κB activation mechanism, the MAPK (Erk1/2, p38 and JNK) <span class="highlight" style="background-color:">signaling</span> cascades, as well as the PI3K/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Other mechanisms, by which ROS induce cellular <span class="highlight" style="background-color:">signaling</span> is through activation of redox-regulated transcription factors such as AP-1 or FOXO.</div></p></div><div class="aux"><div class="resc">Geou-Yarh Liou, et al. Free Radic Res. ;44(5):10.3109/10715761003667554.</div><p class="links"><a class="dblinks" href="/pubmed/20370557" ref="ordinalpos=504&amp;ncbi_uid=5650896&amp;link_uid=20370557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3880197/" ref="ordinalpos=504&amp;ncbi_uid=5650896&amp;link_uid=PMC3880197">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>505.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3206202/figure/F8/" ref="ordinalpos=505&amp;ncbi_uid=3459437&amp;link_uid=PMC3206202" href="/pmc/articles/PMC3206202/"><img src="/pmc/articles/instance/3206202/bin/nihms-321219-f0008.gif" src-large="/pmc/articles/instance/3206202/bin/nihms-321219-f0008.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3206202/" ref="ordinalpos=505&amp;ncbi_uid=3459437&amp;link_uid=PMC3206202" image-link="/pmc/articles/PMC3206202/figure/F8/" class="imagepopup">Fig. 8.  From: Expression Profiling Reveals Differential Gene Induction Underlying Specific and Non-Specific Memory for Pheromones in Mice. </a></p><div class="supp"><p class="details"><div>Differential expression of canonical <span class="highlight" style="background-color:">pathway</span> molecules during BIPM and MIPM.<br />Significant canonical pathways as determined by using Ingenuity <span class="highlight" style="background-color:">Pathway</span> Knowledge Base functional analysis tools are displayed along the x-axis. The y-axis displays the significance score (negative log of p-value calculated using right-tailed Fisher Exact Test). The yellow threshold line that appears in the bar chart represents a p-value of 0.05. The significance threshold of pathways was set to 1.3 (derived by −log<sub>10</sub> [P-value], whereas P≤0.05). Canonical pathways below the threshold line are not statistically significant. Beyond the threshold, the taller bars represent participation of higher number of genes in the <span class="highlight" style="background-color:">pathway</span> compared to shorter bars. (A) Genes induced during MIPM displaying higher statistical significance value (low p-value) or stronger association to the canonical pathways (Huntington’s disease <span class="highlight" style="background-color:">signaling</span>, 14-3-3 mediated <span class="highlight" style="background-color:">signaling</span>, GABA receptor <span class="highlight" style="background-color:">signaling</span>, CXCR4 <span class="highlight" style="background-color:">signaling</span> and axon guidance <span class="highlight" style="background-color:">signaling</span>) compared to genes induced during BIPM. (B) Bar chart showing the significant association of mating induced genes to the canonical pathways like neuregulin <span class="highlight" style="background-color:">signaling</span>, synaptic long-term potentiation, cAMP-mediated <span class="highlight" style="background-color:">signaling</span>, G-protein coupled receptor <span class="highlight" style="background-color:">signaling</span>, α-adrenergic <span class="highlight" style="background-color:">signaling</span>, and oxidative phosphorylation. The genes in these pathways are expressed only in during MIPM. (C) Graph showing the significant association of genes induced during BIPM with the canonical pathways like neurotrophin.TRK <span class="highlight" style="background-color:">signaling</span>, protein ubiquitination <span class="highlight" style="background-color:">pathway</span>, regulation of actin-based motiliy by Rho and PI3K/AKT <span class="highlight" style="background-color:">signaling</span>. Association with these pathways among genes induced during MIPM is either absent or statistically not significant.</div></p></div><div class="aux"><div class="resc">Sudarshan C. Upadhya, et al. Neurochem Int. ;59(6):787-803.</div><p class="links"><a class="dblinks" href="/pubmed/21884744" ref="ordinalpos=505&amp;ncbi_uid=3459437&amp;link_uid=21884744">Citation</a><a class="dblinks" href="/pmc/articles/PMC3206202/" ref="ordinalpos=505&amp;ncbi_uid=3459437&amp;link_uid=PMC3206202">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>506.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392049/figure/f3-72-joint_summit_t2012/" ref="ordinalpos=506&amp;ncbi_uid=4107414&amp;link_uid=PMC3392049" href="/pmc/articles/PMC3392049/"><img src="/pmc/articles/instance/3392049/bin/72-joint_summit_t2012f3.gif" src-large="/pmc/articles/instance/3392049/bin/72-joint_summit_t2012f3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392049/" ref="ordinalpos=506&amp;ncbi_uid=4107414&amp;link_uid=PMC3392049" image-link="/pmc/articles/PMC3392049/figure/f3-72-joint_summit_t2012/" class="imagepopup">Figure 3. From: Integrating Genome and Functional Genomics Data to Reveal Perturbed <span class="highlight" style="background-color:">Signaling</span> Pathways in Ovarian Cancers. </a></p><div class="supp"><p class="details"><div><b>An example <span class="highlight" style="background-color:">pathway</span></b>. The shown ingenuity <span class="highlight" style="background-color:">pathway</span> (<i>Prostate Cancer <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span></i>) overlaps significantly with the mutation-prone subnetwork corresponding to the expression module annotated by GO:0008285. The overlapping genes are shown as shaded nodes.</div></p></div><div class="aux"><div class="resc">Songjian Lu, et al. AMIA Jt Summits Transl Sci Proc. 2012;2012:72-78.</div><p class="links"><a class="dblinks" href="/pubmed/22779056" ref="ordinalpos=506&amp;ncbi_uid=4107414&amp;link_uid=22779056">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392049/" ref="ordinalpos=506&amp;ncbi_uid=4107414&amp;link_uid=PMC3392049">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>507.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2424295/figure/f2/" ref="ordinalpos=507&amp;ncbi_uid=1487582&amp;link_uid=PMC2424295" href="/pmc/articles/PMC2424295/"><img src="/pmc/articles/instance/2424295/bin/msb200830-f2.gif" src-large="/pmc/articles/instance/2424295/bin/msb200830-f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2424295/" ref="ordinalpos=507&amp;ncbi_uid=1487582&amp;link_uid=PMC2424295" image-link="/pmc/articles/PMC2424295/figure/f2/" class="imagepopup">Figure 2.  From: Understanding NF-?B <span class="highlight" style="background-color:">signaling</span> via mathematical modeling. </a></p><div class="supp"><p class="details"><div>Feedback loops in NF-κB <span class="highlight" style="background-color:">signaling</span>. IKK may be activated by the TNFα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> as well as the MyD88-dependent arm of the LPS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. IKK leads to NF-κB activity, which is regulated by a negative feedback loop involving IκB (described in detail in Box 1), as depicted in the lower center. TNFα-induced NF-κB activity also leads to A20 expression, and subsequent decrease in IKK activation. Also, the Trif-dependent arm of the LPS-<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activates the transcription factor interferon regulatory factor-3 (IRF3), leading to TNFα expression and subsequent autocrine <span class="highlight" style="background-color:">signaling</span>. Thus, A20 and TNF form feedback loops that regulate NF-κB activity.</div></p></div><div class="aux"><div class="resc">Raymond Cheong, et al. Mol Syst Biol. 2008;4:192-192.</div><p class="links"><a class="dblinks" href="/pubmed/18463616" ref="ordinalpos=507&amp;ncbi_uid=1487582&amp;link_uid=18463616">Citation</a><a class="dblinks" href="/pmc/articles/PMC2424295/" ref="ordinalpos=507&amp;ncbi_uid=1487582&amp;link_uid=PMC2424295">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>508.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2763556/figure/F7/" ref="ordinalpos=508&amp;ncbi_uid=2431734&amp;link_uid=PMC2763556" href="/pmc/articles/PMC2763556/"><img src="/pmc/articles/instance/2763556/bin/nihms124335f7.gif" src-large="/pmc/articles/instance/2763556/bin/nihms124335f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2763556/" ref="ordinalpos=508&amp;ncbi_uid=2431734&amp;link_uid=PMC2763556" image-link="/pmc/articles/PMC2763556/figure/F7/" class="imagepopup">Figure 7. Fas <span class="highlight" style="background-color:">signaling</span> in metastatic colon cancer.  From: Human and mouse colon cancer utilizes CD95 <span class="highlight" style="background-color:">signaling</span> for local growth and metastatic spread to liver. </a></p><div class="supp"><p class="details"><div>A) Fas Ag type II apoptotic <span class="highlight" style="background-color:">signaling</span> cascade. The non-apoptotic <span class="highlight" style="background-color:">pathway</span> is not fully defined (arrows, unknown portions of the <span class="highlight" style="background-color:">pathway</span>). B) Our proposed model of the interaction between apoptotic resistant colon cancer cells and hepatocytes.</div></p></div><div class="aux"><div class="resc">Hanchen Li, et al. Gastroenterology. ;137(3):934-944.e4.</div><p class="links"><a class="dblinks" href="/pubmed/19524576" ref="ordinalpos=508&amp;ncbi_uid=2431734&amp;link_uid=19524576">Citation</a><a class="dblinks" href="/pmc/articles/PMC2763556/" ref="ordinalpos=508&amp;ncbi_uid=2431734&amp;link_uid=PMC2763556">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>509.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4310480/figure/F2/" ref="ordinalpos=509&amp;ncbi_uid=7111476&amp;link_uid=PMC4310480" href="/pmc/articles/PMC4310480/"><img src="/pmc/articles/instance/4310480/bin/nihms-655734-f0002.gif" src-large="/pmc/articles/instance/4310480/bin/nihms-655734-f0002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4310480/" ref="ordinalpos=509&amp;ncbi_uid=7111476&amp;link_uid=PMC4310480" image-link="/pmc/articles/PMC4310480/figure/F2/" class="imagepopup">Figure 2.  From: The controversial role of the Hedgehog <span class="highlight" style="background-color:">pathway</span> in normal and malignant hematopoiesis. </a></p><div class="supp"><p class="details"><div>Mechanism of activation of the Hh <span class="highlight" style="background-color:">pathway</span> in cancer. (<b>a</b>) Hh <span class="highlight" style="background-color:">pathway</span> is activated in a ligand-independent manner through lost-of-function mutations in the negative regulators of the <span class="highlight" style="background-color:">pathway</span> such as Ptch (indicated with the asterisk) or SuFu, or by gain-of-function mutations in the positive regulators like Smo. This has been found in BCC, MB and rhabdomyosarcoma. (<b>b</b>) Autocrine activation of the Hh <span class="highlight" style="background-color:">pathway</span>. The Hh ligand is produced by the tumorigenic cell, tumor or stem cell, and activates the Hh <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in a cell-autonomous manner. This has been described in small cell lung cancer, pancreatic, colon, prostate, breast cancers, glioblastoma, melanomas, CML and B-cell lymphoma. (<b>c</b>) Paracrine activation of the Hh <span class="highlight" style="background-color:">pathway</span> occurs when the Hh ligand is produced by the tumor cells and the <span class="highlight" style="background-color:">pathway</span> is activated in the neighboring stroma cells which in turn produce factors that stimulate the growth of the tumor cells. This activation has been reported for pancreatic and colon cancer. (<b>d</b>) Inverse paracrine activation of the Hh <span class="highlight" style="background-color:">pathway</span> in which the stroma cells produce the Hh ligand and the tumor cells respond has been shown in non-Hodgkin lymphoma and multiple myeloma.</div></p></div><div class="aux"><div class="resc">BG Mar, et al. Leukemia. ;25(11):1665-1673.</div><p class="links"><a class="dblinks" href="/pubmed/21660044" ref="ordinalpos=509&amp;ncbi_uid=7111476&amp;link_uid=21660044">Citation</a><a class="dblinks" href="/pmc/articles/PMC4310480/" ref="ordinalpos=509&amp;ncbi_uid=7111476&amp;link_uid=PMC4310480">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>510.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3443855/figure/F7/" ref="ordinalpos=510&amp;ncbi_uid=4240546&amp;link_uid=PMC3443855" href="/pmc/articles/PMC3443855/"><img src="/pmc/articles/instance/3443855/bin/nihms392256f7.gif" src-large="/pmc/articles/instance/3443855/bin/nihms392256f7.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3443855/" ref="ordinalpos=510&amp;ncbi_uid=4240546&amp;link_uid=PMC3443855" image-link="/pmc/articles/PMC3443855/figure/F7/" class="imagepopup">Figure 7.  From: Nodal <span class="highlight" style="background-color:">Signaling</span> via an Autocrine <span class="highlight" style="background-color:">Pathway</span> Promotes Proliferation of Mouse Spermatogonial Stem/Progenitor Cells through Smad2/3 and Oct-4 Activation. </a></p><div class="supp"><p class="details"><div>The schematic diagrams show the function of autocrine Nodal in regulating the fate of mouse spermatogonial stem/progenitor cells (left panel) and the intracellular cascades of Nodal <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in spermatogonial stem/progenitor cells (right panel). Letters “P” indicate “phosphorylate”.</div></p></div><div class="aux"><div class="resc">Zuping He, et al. Stem Cells. 2009 October;27(10):2580-2590.</div><p class="links"><a class="dblinks" href="/pubmed/19688838" ref="ordinalpos=510&amp;ncbi_uid=4240546&amp;link_uid=19688838">Citation</a><a class="dblinks" href="/pmc/articles/PMC3443855/" ref="ordinalpos=510&amp;ncbi_uid=4240546&amp;link_uid=PMC3443855">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>511.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4110525/figure/F5/" ref="ordinalpos=511&amp;ncbi_uid=6474942&amp;link_uid=PMC4110525" href="/pmc/articles/PMC4110525/"><img src="/pmc/articles/instance/4110525/bin/1476-4598-13-139-5.gif" src-large="/pmc/articles/instance/4110525/bin/1476-4598-13-139-5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4110525/" ref="ordinalpos=511&amp;ncbi_uid=6474942&amp;link_uid=PMC4110525" image-link="/pmc/articles/PMC4110525/figure/F5/" class="imagepopup">Figure 5.  From: Targeting Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> positively affects the sensitivity of osteosarcoma to cisplatin by regulating the expression and/or activity of Caspase family. </a></p><div class="supp"><p class="details"><div><b>Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulated cisplatin sensitivity of MG63 and Saos-2 via changing the apoptosis effect.</b> After overexpression of NICD-1, cisplatin could induce more apoptosis in MG63 and Saos2 cells than their control group <b>(A and B)</b>. When Notch1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> was inhibited by DAPT, cisplatin induced fewer apoptosis cells in both osteosarcoma cell lines <b>(C and D)</b>. (*P &lt; 0.05; **P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Lei Wang, et al. Mol Cancer. 2014;13:139-139.</div><p class="links"><a class="dblinks" href="/pubmed/24894297" ref="ordinalpos=511&amp;ncbi_uid=6474942&amp;link_uid=24894297">Citation</a><a class="dblinks" href="/pmc/articles/PMC4110525/" ref="ordinalpos=511&amp;ncbi_uid=6474942&amp;link_uid=PMC4110525">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>512.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3010508/figure/F8/" ref="ordinalpos=512&amp;ncbi_uid=3607417&amp;link_uid=PMC3010508" href="/pmc/articles/PMC3010508/"><img src="/pmc/articles/instance/3010508/bin/nihms-200185-f0009.gif" src-large="/pmc/articles/instance/3010508/bin/nihms-200185-f0009.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3010508/" ref="ordinalpos=512&amp;ncbi_uid=3607417&amp;link_uid=PMC3010508" image-link="/pmc/articles/PMC3010508/figure/F8/" class="imagepopup">Figure 8. Proposed epigenetic mechanisms via which curcumin regulates NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to decreased expression of pro-inflammatory genes.  From: Epigenetic regulation of high glucose-induced proinflammatory cytokine productionin monocytes by curcumin. </a></p><div class="supp"><p class="details"><div>High glucose activates the NF-κB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to pro-inflammatory gene expression. Curcumin treatment of HG-induced cells activates the HDACs activity, especially HDAC2, and suppresses HAT activity, especially p300, leading to deacetylation of p65 NF-κB, subsequently suppressing proinflammatory cytokine release.</div></p></div><div class="aux"><div class="resc">Jung-Mi Yun, et al. J Nutr Biochem. ;22(5):450-458.</div><p class="links"><a class="dblinks" href="/pubmed/20655188" ref="ordinalpos=512&amp;ncbi_uid=3607417&amp;link_uid=20655188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3010508/" ref="ordinalpos=512&amp;ncbi_uid=3607417&amp;link_uid=PMC3010508">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>513.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4052696/figure/fig3/" ref="ordinalpos=513&amp;ncbi_uid=6271086&amp;link_uid=PMC4052696" href="/pmc/articles/PMC4052696/"><img src="/pmc/articles/instance/4052696/bin/BMRI2014-351095.003.gif" src-large="/pmc/articles/instance/4052696/bin/BMRI2014-351095.003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4052696/" ref="ordinalpos=513&amp;ncbi_uid=6271086&amp;link_uid=PMC4052696" image-link="/pmc/articles/PMC4052696/figure/fig3/" class="imagepopup">Figure 3.  From: <span class="highlight" style="background-color:">Pathway</span> Bridge Based Multiobjective Optimization Approach for Lurking <span class="highlight" style="background-color:">Pathway</span> Prediction. </a></p><div class="supp"><p class="details"><div>20 <span class="highlight" style="background-color:">signaling</span> pathways derived from KEGG [11] were analyzed for motifs distribution. Results in Figure 3 showed that proteins on looping motifs are mainly from cancer and correlated signal pathways, in other words, motifs with loop structure are enriched in 14 types of carcinomatosis and related <span class="highlight" style="background-color:">signaling</span> pathways, such as cell cycle <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and immune system <span class="highlight" style="background-color:">signaling</span> pathways.</div></p></div><div class="aux"><div class="resc">Rengjing Zhang, et al. Biomed Res Int. 2014;2014:351095.</div><p class="links"><a class="dblinks" href="/pubmed/24949437" ref="ordinalpos=513&amp;ncbi_uid=6271086&amp;link_uid=24949437">Citation</a><a class="dblinks" href="/pmc/articles/PMC4052696/" ref="ordinalpos=513&amp;ncbi_uid=6271086&amp;link_uid=PMC4052696">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>514.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4210253/figure/f2-biomed-24-3-329/" ref="ordinalpos=514&amp;ncbi_uid=6765939&amp;link_uid=PMC4210253" href="/pmc/articles/PMC4210253/"><img src="/pmc/articles/instance/4210253/bin/biomed-24-3-329f2.gif" src-large="/pmc/articles/instance/4210253/bin/biomed-24-3-329f2.jpg" alt="Figure 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4210253/" ref="ordinalpos=514&amp;ncbi_uid=6765939&amp;link_uid=PMC4210253" image-link="/pmc/articles/PMC4210253/figure/f2-biomed-24-3-329/" class="imagepopup">Figure 2. From: The many faces of estrogen <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>The representation of different mechanisms of estrogen <span class="highlight" style="background-color:">signaling</span>.<br />(I.) Direct genomic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, considered the classical mechanism of estrogen <span class="highlight" style="background-color:">signaling</span>, promotes target gene expression by binding the E2-ER complex directly to the ERE. (II.) In the case of indirect genomic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, E2-activated ERs bind DNA through protein-protein interactions with other classes of transcription factors at their respective response elements. (III.) Non-genomic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> starts with the binding of E2 to the ERs located at the plasma membrane resulting in the activation of various protein-kinase cascades. These can eventually lead to changes in gene expression due to phosphorylation of transcription factors. (IV). Ligand-independent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> causes ER activation and target gene transcription through phosphorylation of ERs or their associated coregulators.<br />E2 – 17β-estradiol; ER – estrogen receptor; ERE – estrogen response element; P – phosphate group; TF – transcription factor; TF RE – transcription factor response element.</div></p></div><div class="aux"><div class="resc">Peter Vrtačnik, et al. Biochem Med (Zagreb). 2014 October;24(3):329-342.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=514&amp;ncbi_uid=6765939&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4210253/" ref="ordinalpos=514&amp;ncbi_uid=6765939&amp;link_uid=PMC4210253">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>515.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g007/" ref="ordinalpos=515&amp;ncbi_uid=5583241&amp;link_uid=PMC3855681" href="/pmc/articles/PMC3855681/"><img src="/pmc/articles/instance/3855681/bin/pone.0080918.g007.gif" src-large="/pmc/articles/instance/3855681/bin/pone.0080918.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855681/" ref="ordinalpos=515&amp;ncbi_uid=5583241&amp;link_uid=PMC3855681" image-link="/pmc/articles/PMC3855681/figure/pone-0080918-g007/" class="imagepopup">Figure 7. Graph shows optimization of two conflicting objective functions in an integrated <span class="highlight" style="background-color:">pathway</span> system of SAg expression regulatory <span class="highlight" style="background-color:">pathway</span> of S. aureus and TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> of an infected host (H. sapiens)..  From: A Model of an Integrated Immune System <span class="highlight" style="background-color:">Pathway</span> in Homo sapiens and Its Interaction with Superantigen Producing Expression Regulatory <span class="highlight" style="background-color:">Pathway</span> in Staphylococcus aureus: Comparing Behavior of Pathogen Perturbed and Unperturbed <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Here, <img src="/pmc/articles/instance/3855681/bin/pone.0080918.e041.jpg" class="inline-graphic" style="border-style: none;" /> is maximized, while <img src="/pmc/articles/instance/3855681/bin/pone.0080918.e042.jpg" class="inline-graphic" style="border-style: none;" /> is minimized.</div></p></div><div class="aux"><div class="resc">Namrata Tomar, et al. PLoS One. 2013;8(12):e80918.</div><p class="links"><a class="dblinks" href="/pubmed/24324645" ref="ordinalpos=515&amp;ncbi_uid=5583241&amp;link_uid=24324645">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855681/" ref="ordinalpos=515&amp;ncbi_uid=5583241&amp;link_uid=PMC3855681">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>516.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3639942/figure/pone-0062042-g007/" ref="ordinalpos=516&amp;ncbi_uid=4871804&amp;link_uid=PMC3639942" href="/pmc/articles/PMC3639942/"><img src="/pmc/articles/instance/3639942/bin/pone.0062042.g007.gif" src-large="/pmc/articles/instance/3639942/bin/pone.0062042.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3639942/" ref="ordinalpos=516&amp;ncbi_uid=4871804&amp;link_uid=PMC3639942" image-link="/pmc/articles/PMC3639942/figure/pone-0062042-g007/" class="imagepopup">Figure 7. <span class="highlight" style="background-color:">Pathway</span> analysis and gene expression subtype analysis..  From: A Fourteen Gene GBM Prognostic Signature Identifies Association of Immune Response <span class="highlight" style="background-color:">Pathway</span> and Mesenchymal Subtype with High Risk Group. </a></p><div class="supp"><p class="details"><div><b>A</b>) KEGG <span class="highlight" style="background-color:">pathway</span> enrichment analysis. Enrichment of cytokine-cytokine receptor interaction and chemokine <span class="highlight" style="background-color:">signaling</span> pathways leads to activation of various pro-survival pathways like Jak-STAT <span class="highlight" style="background-color:">pathway</span>, MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and NFkB <span class="highlight" style="background-color:">pathway</span> with the resultant pro-tumorigenic environment. <b>B</b>) TCGA patients (n = 108) were divided into gene expression sub types – proneural, neural, mesenchymal and classical among low risk and high risk groups. Mann Whitney test was carried out to find out significance of distribution of a given expression subtype within a risk group.</div></p></div><div class="aux"><div class="resc">Arivazhagan Arimappamagan, et al. PLoS One. 2013;8(4):e62042.</div><p class="links"><a class="dblinks" href="/pubmed/23646114" ref="ordinalpos=516&amp;ncbi_uid=4871804&amp;link_uid=23646114">Citation</a><a class="dblinks" href="/pmc/articles/PMC3639942/" ref="ordinalpos=516&amp;ncbi_uid=4871804&amp;link_uid=PMC3639942">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>517.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3636084/figure/pgen-1003482-g006/" ref="ordinalpos=517&amp;ncbi_uid=4870008&amp;link_uid=PMC3636084" href="/pmc/articles/PMC3636084/"><img src="/pmc/articles/instance/3636084/bin/pgen.1003482.g006.gif" src-large="/pmc/articles/instance/3636084/bin/pgen.1003482.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3636084/" ref="ordinalpos=517&amp;ncbi_uid=4870008&amp;link_uid=PMC3636084" image-link="/pmc/articles/PMC3636084/figure/pgen-1003482-g006/" class="imagepopup">Figure 6. Hypothesized overview of ovulation quota control by the various Fecundity mutations..  From: Genome-Wide Association Studies Identify Two Novel BMP15 Mutations Responsible for an Atypical Hyperprolificacy Phenotype in Sheep. </a></p><div class="supp"><p class="details"><div>(A) The mono ovulation quota is tightly controlled by the integrative action of <i>i)</i> BMP15 homodimer through BMPR1B, BMPR2 and SMAD1/5/8, <i>ii)</i> BMP15/GDF9 heterodimer through SMAD2/3 and <i>iii)</i> GDF9 homodimer through TBR1, BMPR2 and SMAD2/3. (B) Increased of ovulation rate due to <i>FecX</i> mutations. This drawing represents an overview of hyperprolificacy dependent of BMP15 variants. Heterozygous or homozygous <i>FecX</i> mutations leading to an increased OR affect either the <span class="highlight" style="background-color:">signaling</span> pathways of BMP15 homodimer, BMP15/GDF9 heterodimer or both whereas the GDF9 homodimer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> remains stable. (C) Increased of ovulation rate due to <i>FecG</i> mutations. This drawing represents an overview of hyperprolificacy dependent of GDF9 variants. Heterozygous or homozygous <i>FecG</i> mutations leading to an increased OR impair either the <span class="highlight" style="background-color:">signaling</span> pathways of GDF9 homodimer, BMP15/GDF9 heterodimer or both whereas the BMP15 homodimer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> remains stable. (D) Dose sensitive effect of each <i>FecX</i> and <i>FecG</i> mutations on the BMP15, GDF9 homodimers and BMP15/GDF9 heterodimer <span class="highlight" style="background-color:">signaling</span> pathways. Based on the various <i>in vitro</i> tests performed, we assigned a dose (n = 0, 1 or 2) to each <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in an attempt to explain the prolificacy phenotype observed for all the <i>FecX</i> and <i>FecG</i> mutations. When the ovulation rate is normal ( = 1), i.e. the WT situation, 2 doses of BMP15 homodimer (blue), GDF9 homodimer (purple) and BMP15/GDF9 heterodimer (pink) were considered. As example, in the case of the <i>FecX<sup>Gr</sup></i> mutation where the BMP15 homodimer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> seemed clearly affected at homozygous and heterozygous status (as shown on Figure 5), we assumed that the BMP15/GDF9 heterodimer <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> remained totally active.</div></p></div><div class="aux"><div class="resc">Julie Demars, et al. PLoS Genet. 2013 April;9(4):e1003482.</div><p class="links"><a class="dblinks" href="/pubmed/23637641" ref="ordinalpos=517&amp;ncbi_uid=4870008&amp;link_uid=23637641">Citation</a><a class="dblinks" href="/pmc/articles/PMC3636084/" ref="ordinalpos=517&amp;ncbi_uid=4870008&amp;link_uid=PMC3636084">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>518.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2895421/figure/F2/" ref="ordinalpos=518&amp;ncbi_uid=2554004&amp;link_uid=PMC2895421" href="/pmc/articles/PMC2895421/"><img src="/pmc/articles/instance/2895421/bin/nihms-212355-f0002.gif" src-large="/pmc/articles/instance/2895421/bin/nihms-212355-f0002.jpg" alt="Fig 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2895421/" ref="ordinalpos=518&amp;ncbi_uid=2554004&amp;link_uid=PMC2895421" image-link="/pmc/articles/PMC2895421/figure/F2/" class="imagepopup">Fig 2.  From: Convergence of Hormones, Inflammation, and Energy-Related Factors: A Novel <span class="highlight" style="background-color:">Pathway</span> of Cancer Etiology. </a></p><div class="supp"><p class="details"><div>Convergent <span class="highlight" style="background-color:">signaling</span> pathways where inflammation and metabolic <span class="highlight" style="background-color:">signaling</span> intersect along the CHIEF <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Martha L. Slattery, et al. Cancer Prev Res (Phila). ;2(11):922-930.</div><p class="links"><a class="dblinks" href="/pubmed/19892662" ref="ordinalpos=518&amp;ncbi_uid=2554004&amp;link_uid=19892662">Citation</a><a class="dblinks" href="/pmc/articles/PMC2895421/" ref="ordinalpos=518&amp;ncbi_uid=2554004&amp;link_uid=PMC2895421">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>519.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3478512/figure/Fig4/" ref="ordinalpos=519&amp;ncbi_uid=4326420&amp;link_uid=PMC3478512" href="/pmc/articles/PMC3478512/"><img src="/pmc/articles/instance/3478512/bin/18_2012_991_Fig4_HTML.gif" src-large="/pmc/articles/instance/3478512/bin/18_2012_991_Fig4_HTML.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3478512/" ref="ordinalpos=519&amp;ncbi_uid=4326420&amp;link_uid=PMC3478512" image-link="/pmc/articles/PMC3478512/figure/Fig4/" class="imagepopup">Fig. 4.  From: Current perspectives of the <span class="highlight" style="background-color:">signaling</span> pathways directing neural crest induction. </a></p><div class="supp"><p class="details"><div>Temporal and spatial participation of <span class="highlight" style="background-color:">signaling</span> pathways involved in chick neural crest induction. <b>a</b> Timeline of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation and requirement in early NC development. <i>Closed arrows/lines</i> indicate activation and requirement in NC tissues. <i>Dotted lines</i> indicate activation in NC tissues, but requirement is unknown. Smad1/5/8 <span class="highlight" style="background-color:">signaling</span> is active in the entire epiblast at blastula stages, but is inactivated by gastrulation [75]. <span class="highlight" style="background-color:">Signaling</span> becomes active with the expression of NPB markers, and remains active through to migration. Erk <span class="highlight" style="background-color:">signaling</span> is also active in most of the epiblast at blastula stages, and is required for neural induction until gastrulation. A requirement for FGF/Erk <span class="highlight" style="background-color:">signaling</span> in NC induction was only demonstrated during gastrulation. Erk <span class="highlight" style="background-color:">signaling</span> remains active in the NPB and NC tissues through to migration, but is no longer required for NC development (<i>gray line</i>). Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span> is thought to be necessary for all stages of early NC development. A requirement for Notch/Delta <span class="highlight" style="background-color:">signaling</span> was demonstrated at mid-neurula stages. <b>b</b> Spatial activation of BMP, FGF, Wnt, and Notch <span class="highlight" style="background-color:">signaling</span> during chick gastrulation and neurulation. BMP/Smad activation based on [76]. FGF/Erk activation based on [76, 128]. Wnt/β-catenin activation inferred from expression of agonists and antagonists, and functional requirements for Wnt <span class="highlight" style="background-color:">signaling</span>. Notch/Delta activation is based on expression of molecules and functional requirements [154]. <b>c</b> Spatial expression of relevant <span class="highlight" style="background-color:">signaling</span> molecules and requirements for chick neural crest induction during gastrulation. Diagram corresponds to section at <i>dotted line</i> HH 3+ gastrula in <b>b</b>. Functional studies have demonstrated a requirement for FGF/Erk and Wnt/β-catenin <span class="highlight" style="background-color:">signaling</span>, but the participation of specific <span class="highlight" style="background-color:">signaling</span> molecules has not been challenged. The spatial expression of some potential <span class="highlight" style="background-color:">signaling</span> molecules is presented. Multiple FGF and Wnt agonists and BMP and Wnt antagonists are expressed in the node/primitive streak, but it is unclear whether these molecules can diffuse the distance to influence the prospective NC tissue (<i>dotted arrow</i>). <b>d</b> Spatial expression and participation of <span class="highlight" style="background-color:">signaling</span> molecules and pathways in the maintenance of NC progenitors during chick neurulation. Diagram corresponds to section at<i> dotted line</i> in HH 6 neurula in <b>b</b>. Smad1/5/8 and β-catenin are likely activated by Bmp4, Bmp7, Wnt1, and Wnt3a expressed in the neural folds and adjacent NNE. Wnt6 in the NNE has also been implicated in NC development, but may act through the non-canonical Rho/JNK <span class="highlight" style="background-color:">pathway</span>. Notch <span class="highlight" style="background-color:">signaling</span> likely participates indirectly by regulating Bmp4 expression. Spatial expression of other potential <span class="highlight" style="background-color:">signaling</span> molecules is presented. Expression data gathered from references in the text and from http://geisha.arizona.edu/geisha</div></p></div><div class="aux"><div class="resc">Timothy J. Stuhlmiller, et al. Cell Mol Life Sci. 2012 November;69(22):3715-3737.</div><p class="links"><a class="dblinks" href="/pubmed/22547091" ref="ordinalpos=519&amp;ncbi_uid=4326420&amp;link_uid=22547091">Citation</a><a class="dblinks" href="/pmc/articles/PMC3478512/" ref="ordinalpos=519&amp;ncbi_uid=4326420&amp;link_uid=PMC3478512">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>520.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3247287/figure/pone-0029733-g008/" ref="ordinalpos=520&amp;ncbi_uid=3738987&amp;link_uid=PMC3247287" href="/pmc/articles/PMC3247287/"><img src="/pmc/articles/instance/3247287/bin/pone.0029733.g008.gif" src-large="/pmc/articles/instance/3247287/bin/pone.0029733.g008.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3247287/" ref="ordinalpos=520&amp;ncbi_uid=3738987&amp;link_uid=PMC3247287" image-link="/pmc/articles/PMC3247287/figure/pone-0029733-g008/" class="imagepopup">Figure 8. <span class="highlight" style="background-color:">Signaling</span> pathways of INSL3 investigated in human osteoblasts..  From: Profiling Insulin Like Factor 3 (INSL3) <span class="highlight" style="background-color:">Signaling</span> in Human Osteoblasts. </a></p><div class="supp"><p class="details"><div>Continuous lines identify the major <span class="highlight" style="background-color:">pathway</span> stimulated by INSL3 and culminating in MEK/ERK phosphorylation via AC/cAMP/PKA. Dotted lines show excluded pathways in INSL3 <span class="highlight" style="background-color:">signaling</span> from our findings. See text for details. P: phosphorylation, U: Ubiquitination.</div></p></div><div class="aux"><div class="resc">Alberto Ferlin, et al. PLoS One. 2011;6(12):e29733.</div><p class="links"><a class="dblinks" href="/pubmed/22216350" ref="ordinalpos=520&amp;ncbi_uid=3738987&amp;link_uid=22216350">Citation</a><a class="dblinks" href="/pmc/articles/PMC3247287/" ref="ordinalpos=520&amp;ncbi_uid=3738987&amp;link_uid=PMC3247287">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>521.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3177754/figure/F4/" ref="ordinalpos=521&amp;ncbi_uid=3475811&amp;link_uid=PMC3177754" href="/pmc/articles/PMC3177754/"><img src="/pmc/articles/instance/3177754/bin/nihms-299486-f0004.gif" src-large="/pmc/articles/instance/3177754/bin/nihms-299486-f0004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3177754/" ref="ordinalpos=521&amp;ncbi_uid=3475811&amp;link_uid=PMC3177754" image-link="/pmc/articles/PMC3177754/figure/F4/" class="imagepopup">Figure 4. RLR-NLR interactions in <span class="highlight" style="background-color:">signaling</span> against pathogens.  From: RIG-I Like Receptors and Their <span class="highlight" style="background-color:">Signaling</span> Crosstalk in the Regulation of Antiviral Immunity. </a></p><div class="supp"><p class="details"><div>The NOD2 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for recognition of bacterial and viral PAMPs is shown on the left. NOD2 is triggered by bacterial MDP to activate NF-κB while ssRNA from RSV leads to activation of IRF-3 <span class="highlight" style="background-color:">signaling</span>. The RIG-I Inflammasome is depicted on the right. Both RIG-I and MDA5 can interact with CARD9/BCL10 to activate NF-κB. This leads to induction of pro-IL-1β. RIG-I but not MDA5 can directly associate with ASC bifurcating NLR mediated activation of Caspase-1. This leads to cleavage of IL-1β and secretion from the cell.</div></p></div><div class="aux"><div class="resc">Hilario J Ramos, et al. Curr Opin Virol. ;1(3):167-176.</div><p class="links"><a class="dblinks" href="/pubmed/21949557" ref="ordinalpos=521&amp;ncbi_uid=3475811&amp;link_uid=21949557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3177754/" ref="ordinalpos=521&amp;ncbi_uid=3475811&amp;link_uid=PMC3177754">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>522.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3035636/figure/F2/" ref="ordinalpos=522&amp;ncbi_uid=3763739&amp;link_uid=PMC3035636" href="/pmc/articles/PMC3035636/"><img src="/pmc/articles/instance/3248130/bin/oncotarget-01-552-g002.gif" src-large="/pmc/articles/instance/3248130/bin/oncotarget-01-552-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3035636/" ref="ordinalpos=522&amp;ncbi_uid=3763739&amp;link_uid=PMC3035636" image-link="/pmc/articles/PMC3035636/figure/F2/" class="imagepopup">Figure 2. Canonical Wnt <span class="highlight" style="background-color:">signaling</span> and dysregulation in cancer.  From: Drug Discovery Approaches to Target Wnt <span class="highlight" style="background-color:">Signaling</span> in Cancer Stem Cells. </a></p><div class="supp"><p class="details"><div>The Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is comprised of extracellular, cytoplasmic and nuclear <span class="highlight" style="background-color:">signaling</span> events that are amenable to therapeutic intervention. Dysregulation at these stages are common in numerous cancers, captured in the white boxes. Upon entering the nucleus and interacting with TCF/LEF and various co-activators, β-catenin drives transcription of programs critical for CSCs, tumor cells and EMT.</div></p></div><div class="aux"><div class="resc">Joshua C. Curtin, et al. Oncotarget. 2010 November;1(7):552-566.</div><p class="links"><a class="dblinks" href="/pubmed/21317451" ref="ordinalpos=522&amp;ncbi_uid=3763739&amp;link_uid=21317451">Citation</a><a class="dblinks" href="/pmc/articles/PMC3035636/" ref="ordinalpos=522&amp;ncbi_uid=3763739&amp;link_uid=PMC3035636">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>523.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3691204/figure/pone-0066685-g001/" ref="ordinalpos=523&amp;ncbi_uid=5061955&amp;link_uid=PMC3691204" href="/pmc/articles/PMC3691204/"><img src="/pmc/articles/instance/3691204/bin/pone.0066685.g001.gif" src-large="/pmc/articles/instance/3691204/bin/pone.0066685.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3691204/" ref="ordinalpos=523&amp;ncbi_uid=5061955&amp;link_uid=PMC3691204" image-link="/pmc/articles/PMC3691204/figure/pone-0066685-g001/" class="imagepopup">Figure 1. Hedgehog <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Mutational Analysis of Hedgehog <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Genes in Human Malignant Mesothelioma. </a></p><div class="supp"><p class="details"><div>HH ligands (SHH, DHH or IHH) bind to the PTCH receptors (PTCH1 and PTCH2) and relieve the inhibition of SMO. HHIP, a negative inhibitor of HH <span class="highlight" style="background-color:">signaling</span>, can compete with PTCH receptors to bind HH ligand, resulting in the attenuation of HH <span class="highlight" style="background-color:">signaling</span>. SMO then transduces signals through the cytoplasmic SUFU-GLI complex, resulting in the activation and nuclear translocation of the downstream GLI transcription factors (GLI1-3). STK36 further promotes activation and nuclear accumulation of GLI by antagonizing SUFU. KIF7 regulates GLI-mediated transcription both positively and negatively through physical interaction with GLI and regulating the stability and degradation of GLI proteins.</div></p></div><div class="aux"><div class="resc">Chuan Bian Lim, et al. PLoS One. 2013;8(6):e66685.</div><p class="links"><a class="dblinks" href="/pubmed/23826113" ref="ordinalpos=523&amp;ncbi_uid=5061955&amp;link_uid=23826113">Citation</a><a class="dblinks" href="/pmc/articles/PMC3691204/" ref="ordinalpos=523&amp;ncbi_uid=5061955&amp;link_uid=PMC3691204">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>524.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2515128/figure/F7/" ref="ordinalpos=524&amp;ncbi_uid=1473438&amp;link_uid=PMC2515128" href="/pmc/articles/PMC2515128/"><img src="/pmc/articles/instance/2515128/bin/zh80170823190007.gif" src-large="/pmc/articles/instance/2515128/bin/zh80170823190007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2515128/" ref="ordinalpos=524&amp;ncbi_uid=1473438&amp;link_uid=PMC2515128" image-link="/pmc/articles/PMC2515128/figure/F7/" class="imagepopup">Figure 7.  From: Src family tyrosine kinase Lyn mediates VWF/GPIb-IX-induced platelet activation via the cGMP <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>The role of Lyn in VWF-induced activation of MAPK and a schematic of Lyn-dependent GPIb-IX <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to platelet adhesion and aggregation.</b> (A) Wild-type or Lyn<sup>−/−</sup> platelets were stimulated with botrocetin (1.2 μg/mL) in the absence or presence of 10 μg/mL of VWF for 8 minutes in a platelet aggregometer. Platelets were solubilized and immunoblotted with anti-phospho-p38 (Thr180/Tyr182) antibody (p-p38) or anti-p38 polyclonal antibody to indicate loading levels (p38; top panels), and anti–phospho-Erk1/2 (Thr202/Tyr204) antibody (p-ERK) or corresponding anti-Erk2 antibody to indicate loading levels (ERK; bottom panels). (B) VWF binding to GPIb–IX activates Lyn, which stimulates the PI3K-Akt-NO-cGMP-PKG-MAPK–dependent early GPIb-IX <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> leading to integrin activation and stable platelet adhesion to VWF under shear stress. This <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> together with integrin outside-in <span class="highlight" style="background-color:">signaling</span> causes TXA<sub>2</sub> synthesis and granule secretion of agonists, such as ADP, leading to amplification of integrin activation, increased stable platelet adhesion, and platelet aggregation.</div></p></div><div class="aux"><div class="resc">Hong Yin, et al. Blood. 2008 August 15;112(4):1139-1146.</div><p class="links"><a class="dblinks" href="/pubmed/18550847" ref="ordinalpos=524&amp;ncbi_uid=1473438&amp;link_uid=18550847">Citation</a><a class="dblinks" href="/pmc/articles/PMC2515128/" ref="ordinalpos=524&amp;ncbi_uid=1473438&amp;link_uid=PMC2515128">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>525.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3640709/figure/F1/" ref="ordinalpos=525&amp;ncbi_uid=6212340&amp;link_uid=PMC3640709" href="/pmc/articles/PMC3640709/"><img src="/pmc/articles/instance/3640709/bin/nihms-458472-f0001.gif" src-large="/pmc/articles/instance/3640709/bin/nihms-458472-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3640709/" ref="ordinalpos=525&amp;ncbi_uid=6212340&amp;link_uid=PMC3640709" image-link="/pmc/articles/PMC3640709/figure/F1/" class="imagepopup">Figure 1. Schematic outlining the MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in BY1404.  From: Frizzled receptors signal through G proteins. </a></p><div class="supp"><p class="details"><div>The single endogenous GPCR, Ste2 is not expressed in the modified <i>S. cerevisiae</i> strain. A chimeric yeast Gpa1-mammalian Gα<sub>s</sub> transplant is expressed in place of endogenous Gpa1. <span class="highlight" style="background-color:">Signaling</span> through the endogenous MAPK <span class="highlight" style="background-color:">pathway</span> initiates transcriptional activation of the Fus1-lacZ reporter gene.</div></p></div><div class="aux"><div class="resc">Andrea S. Nichols, et al. Cell Signal. 2013 June;25(6):1468-1475.</div><p class="links"><a class="dblinks" href="/pubmed/23524329" ref="ordinalpos=525&amp;ncbi_uid=6212340&amp;link_uid=23524329">Citation</a><a class="dblinks" href="/pmc/articles/PMC3640709/" ref="ordinalpos=525&amp;ncbi_uid=6212340&amp;link_uid=PMC3640709">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>526.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3411054/figure/F7/" ref="ordinalpos=526&amp;ncbi_uid=5161751&amp;link_uid=PMC3411054" href="/pmc/articles/PMC3411054/"><img src="/pmc/articles/instance/3411054/bin/zbc0331217470007.gif" src-large="/pmc/articles/instance/3411054/bin/zbc0331217470007.jpg" alt="FIGURE 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3411054/" ref="ordinalpos=526&amp;ncbi_uid=5161751&amp;link_uid=PMC3411054" image-link="/pmc/articles/PMC3411054/figure/F7/" class="imagepopup">FIGURE 7. From: Targeting the Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span> in Keratocystic Odontogenic Tumor. </a></p><div class="supp"><p class="details"><div><b>Treatment of the KCOT-1 cells with cyclopamine down-regulated components of the NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</b> KCOT-1 cells treated with cyclopamine (10, 20, 25 μ<span style="font-variant: small-caps" class="small-caps">m</span>) or tomatidine (5, 10 μ<span style="font-variant: small-caps" class="small-caps">m</span>) for 48 h. <i>A</i>, protein expression of the NOTCH <span class="highlight" style="background-color:">pathway</span> with and without cyclopamine treatment. <i>B</i>, densitometric analysis of the calculated reduction of the protein levels of the NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. <i>C</i>, gene expression of the NOTCH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in KCOT-1 cells. <i>Error bars</i> indicate S.D. <i>D</i>, <i>top</i> and <i>bottom</i>, immunohistochemical detection of NOTCH-2, NOTCH-3, and JAG-2 in KCOT-1 cells (<i>top</i>) and tumor tissue (<i>bottom</i>).</div></p></div><div class="aux"><div class="resc">Changchun Ren, et al. J Biol Chem. 2012 August 3;287(32):27117-27125.</div><p class="links"><a class="dblinks" href="/pubmed/22679015" ref="ordinalpos=526&amp;ncbi_uid=5161751&amp;link_uid=22679015">Citation</a><a class="dblinks" href="/pmc/articles/PMC3411054/" ref="ordinalpos=526&amp;ncbi_uid=5161751&amp;link_uid=PMC3411054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>527.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2663984/figure/F5/" ref="ordinalpos=527&amp;ncbi_uid=1789893&amp;link_uid=PMC2663984" href="/pmc/articles/PMC2663984/"><img src="/pmc/articles/instance/2663984/bin/zpq9990972180005.gif" src-large="/pmc/articles/instance/2663984/bin/zpq9990972180005.jpg" alt="Fig. 5." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2663984/" ref="ordinalpos=527&amp;ncbi_uid=1789893&amp;link_uid=PMC2663984" image-link="/pmc/articles/PMC2663984/figure/F5/" class="imagepopup">Fig. 5. From: Identification of targets of the Wnt <span class="highlight" style="background-color:">pathway</span> destruction complex in addition to ?-catenin. </a></p><div class="supp"><p class="details"><div>A model for the expanded canonical Wnt/GSK3 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Binding of Wnt proteins to receptors triggers canonical (β-catenin and destruction complex dependent) or noncanonical (β-catenin and the destruction complex independent) Wnt <span class="highlight" style="background-color:">signaling</span>. In the canonical Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, the stability of β-catenin is controlled by destruction complex composed by GSK3, APC, and Axin, etc. In addition to β-catenin, the degradation of many different proteins in various functional classes is regulated by the destruction complex and Wnt receptor <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Nam-Gyun Kim, et al. Proc Natl Acad Sci U S A. 2009 March 31;106(13):5165-5170.</div><p class="links"><a class="dblinks" href="/pubmed/19289839" ref="ordinalpos=527&amp;ncbi_uid=1789893&amp;link_uid=19289839">Citation</a><a class="dblinks" href="/pmc/articles/PMC2663984/" ref="ordinalpos=527&amp;ncbi_uid=1789893&amp;link_uid=PMC2663984">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>528.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3504487/figure/F9/" ref="ordinalpos=528&amp;ncbi_uid=4835283&amp;link_uid=PMC3504487" href="/pmc/articles/PMC3504487/"><img src="/pmc/articles/instance/3504487/bin/nihms418878f9.gif" src-large="/pmc/articles/instance/3504487/bin/nihms418878f9.jpg" alt="Fig. 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3504487/" ref="ordinalpos=528&amp;ncbi_uid=4835283&amp;link_uid=PMC3504487" image-link="/pmc/articles/PMC3504487/figure/F9/" class="imagepopup">Fig. 9. Elevation of activity impairs Erk1/2-STAT3 <span class="highlight" style="background-color:">signaling</span> to control p53 expression.  From: Loss of STAT3 <span class="highlight" style="background-color:">signaling</span> during elevated activity causes vulnerability in hippocampal neurons. </a></p><div class="supp"><p class="details"><div>In mature neurons, Erk1/2 activates STAT3, and STAT3 binds and regulates p53 transcription, leading to resistance to the loss of neurotrophin-Akt survival <span class="highlight" style="background-color:">signaling</span>. This alternative survival <span class="highlight" style="background-color:">pathway</span> is impaired when neurons are challenged by elevated activity.</div></p></div><div class="aux"><div class="resc">Sachiko Murase, et al. J Neurosci. ;32(44):15511-15520.</div><p class="links"><a class="dblinks" href="/pubmed/23115188" ref="ordinalpos=528&amp;ncbi_uid=4835283&amp;link_uid=23115188">Citation</a><a class="dblinks" href="/pmc/articles/PMC3504487/" ref="ordinalpos=528&amp;ncbi_uid=4835283&amp;link_uid=PMC3504487">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>529.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2826402/figure/pone-0009370-g003/" ref="ordinalpos=529&amp;ncbi_uid=2278427&amp;link_uid=PMC2826402" href="/pmc/articles/PMC2826402/"><img src="/pmc/articles/instance/2826402/bin/pone.0009370.g003.gif" src-large="/pmc/articles/instance/2826402/bin/pone.0009370.g003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2826402/" ref="ordinalpos=529&amp;ncbi_uid=2278427&amp;link_uid=PMC2826402" image-link="/pmc/articles/PMC2826402/figure/pone-0009370-g003/" class="imagepopup">Figure 3. Lentiviruses that inhibit or activate the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Lentiviral Vectors to Probe and Manipulate the Wnt <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>To measure Wnt <span class="highlight" style="background-color:">signaling</span> activity in HEK293T and HEK293A, cells were transfected with the reporter construct pMegaTOPFlash prior to Wnt3A addition. A) HEK293T cells were infected with the hEF1α-dnTcf4//SV40-mCherry (“EdTC”) virus, and mCherry-expressing cells were selected by fluorescence-activated cell sorting. Induction of a Wnt reporter by purified Wnt3A protein (500 ng/ml final) was impaired in EdTC-infected cells compared to mock-infected cells (top). All selected cells were expressing mCherry (bottom). B) Mouse LS/L or HEK293A cells were infected with the hEF1α-dnTcf4//SV40-Puro<sup>R</sup> (“EdTP”) virus and selected with 1 µg/ml puromycin. Puromycin selection led to enrichment of LS/L cells with an impaired response to Wnt3A-conditioned medium (left, compare “+” with “−” puromycin). Induction of a Wnt reporter by purified Wnt3A protein (500 ng/ml final) was also impaired in EdTP-infected HEK293A cells compared to mock-infected cells (right). C) HEK293T cells were infected with the hEF1α-β-catenin<sup>4A</sup>//SV40-mCherry (“EβC”) virus at different multiplicities of infection (1, 0.3 and 0.1 relative concentration of virus per cell). Increasing the number of infected cells (as seen by the amount of mCherry-positive cells in a confluent field, right) led to increasing activation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in absence of any exogenous Wnt source (left). D) Mouse LS/L and HEK293A cells were infected with the hEF1α-β-catenin<sup>4A</sup>//SV40-Puro<sup>R</sup> (“EβP”) virus and selected with 1 µg/ml puromycin. Selection with puromycin led to the enrichment of cells with constitutive activation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (left, compare “−” with “Puro”). Infection of HEK293A with the EβP virus similarly led to ligand-independent activation of the Wnt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (right). In panels A and B, activity is relative to vehicle-treated cells. In panels C and D, activity is relative to mock-infected cells.</div></p></div><div class="aux"><div class="resc">Christophe Fuerer, et al. PLoS One. 2010;5(2):e9370.</div><p class="links"><a class="dblinks" href="/pubmed/20186325" ref="ordinalpos=529&amp;ncbi_uid=2278427&amp;link_uid=20186325">Citation</a><a class="dblinks" href="/pmc/articles/PMC2826402/" ref="ordinalpos=529&amp;ncbi_uid=2278427&amp;link_uid=PMC2826402">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>530.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4346139/figure/F2/" ref="ordinalpos=530&amp;ncbi_uid=7223937&amp;link_uid=PMC4346139" href="/pmc/articles/PMC4346139/"><img src="/pmc/articles/instance/4346139/bin/nihms663656f2.gif" src-large="/pmc/articles/instance/4346139/bin/nihms663656f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4346139/" ref="ordinalpos=530&amp;ncbi_uid=7223937&amp;link_uid=PMC4346139" image-link="/pmc/articles/PMC4346139/figure/F2/" class="imagepopup">Figure 2.  From: Lipid <span class="highlight" style="background-color:">Signaling</span> in Tumorigenesis. </a></p><div class="supp"><p class="details"><div>Schematic of PI3-K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Renyan Liu, et al. Mol Cell Pharmacol. ;6(1):1-9.</div><p class="links"><a class="dblinks" href="/pubmed/0" ref="ordinalpos=530&amp;ncbi_uid=7223937&amp;link_uid=0">Citation</a><a class="dblinks" href="/pmc/articles/PMC4346139/" ref="ordinalpos=530&amp;ncbi_uid=7223937&amp;link_uid=PMC4346139">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>531.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3464787/figure/F4/" ref="ordinalpos=531&amp;ncbi_uid=4277277&amp;link_uid=PMC3464787" href="/pmc/articles/PMC3464787/"><img src="/pmc/articles/instance/3464787/bin/1479-5876-10-113-4.gif" src-large="/pmc/articles/instance/3464787/bin/1479-5876-10-113-4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3464787/" ref="ordinalpos=531&amp;ncbi_uid=4277277&amp;link_uid=PMC3464787" image-link="/pmc/articles/PMC3464787/figure/F4/" class="imagepopup">Figure 4.  From: Racial differences in B cell receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation. </a></p><div class="supp"><p class="details"><div><b>Correlations between IgD+ B cell percentages and BCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation.</b> The heatmap shows the magnitude of the Pearson correlation coefficients (r) between the IgD+ B cell percentage and the Fold for each <span class="highlight" style="background-color:">signaling</span> node at each anti-IgD modulation time point.</div></p></div><div class="aux"><div class="resc">Diane M Longo, et al. J Transl Med. 2012;10:113-113.</div><p class="links"><a class="dblinks" href="/pubmed/22672557" ref="ordinalpos=531&amp;ncbi_uid=4277277&amp;link_uid=22672557">Citation</a><a class="dblinks" href="/pmc/articles/PMC3464787/" ref="ordinalpos=531&amp;ncbi_uid=4277277&amp;link_uid=PMC3464787">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>532.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4026686/figure/F2/" ref="ordinalpos=532&amp;ncbi_uid=6165611&amp;link_uid=PMC4026686" href="/pmc/articles/PMC4026686/"><img src="/pmc/articles/instance/4026686/bin/fendo-05-00071-g002.gif" src-large="/pmc/articles/instance/4026686/bin/fendo-05-00071-g002.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4026686/" ref="ordinalpos=532&amp;ncbi_uid=6165611&amp;link_uid=PMC4026686" image-link="/pmc/articles/PMC4026686/figure/F2/" class="imagepopup">Figure 2.  From: Diversity and Bias through Receptor-Receptor Interactions in GPCR Heteroreceptor Complexes. Focus on Examples from Dopamine D2 Receptor Heteromerization. </a></p><div class="supp"><p class="details"><div><b>Illustration of the biased agonist action of the hallucinogenic 5-HT2A agonists LSD and DOI at the 5-HT2A protomer of the 5-HT2A–D2LR heterodimer, which leads to enhanced Gi/o <span class="highlight" style="background-color:">signaling</span> over the D2R protomer producing an enhanced inhibition of the AC–PKA–CREB <span class="highlight" style="background-color:">pathway</span></b>. In contrast, the standard 5-HT2A agonist TCB2 did not produce a modulation of the D2R agonist-induced inhibition of the AC–PKA–CREB <span class="highlight" style="background-color:">pathway</span> and the endogenous ligand 5-HT significantly reduced the <span class="highlight" style="background-color:">signaling</span> of this <span class="highlight" style="background-color:">pathway</span>. A biased 5-HT2A agonism with DOI was also observed in the facilitatory D2LR–5-HT2A receptor–receptor interaction of the heteromer, which enhances 5-HT2A <span class="highlight" style="background-color:">signaling</span> over Gq/11–PLC when using 5-HT and TCB2 as 5-HT2A agonists. Thus, when using DOI as a 5-HT2A agonist, a D2R agonist instead diminished 5-HT2A protomer <span class="highlight" style="background-color:">signaling</span> over the Gq/11–PLC <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Kjell Fuxe, et al. Front Endocrinol (Lausanne). 2014;5:71.</div><p class="links"><a class="dblinks" href="/pubmed/24860548" ref="ordinalpos=532&amp;ncbi_uid=6165611&amp;link_uid=24860548">Citation</a><a class="dblinks" href="/pmc/articles/PMC4026686/" ref="ordinalpos=532&amp;ncbi_uid=6165611&amp;link_uid=PMC4026686">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>533.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1594900/figure/f3/" ref="ordinalpos=533&amp;ncbi_uid=855785&amp;link_uid=PMC1594900" href="/pmc/articles/PMC1594900/"><img src="/pmc/articles/instance/1594900/bin/zii0100661800003.gif" src-large="/pmc/articles/instance/1594900/bin/zii0100661800003.jpg" alt="FIG. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1594900/" ref="ordinalpos=533&amp;ncbi_uid=855785&amp;link_uid=PMC1594900" image-link="/pmc/articles/PMC1594900/figure/f3/" class="imagepopup">FIG. 3. From: Induction of Neutrophil Chemotaxis by the Quorum-Sensing Molecule N-(3-Oxododecanoyl)-l-Homoserine Lactone . </a></p><div class="supp"><p class="details"><div>Schematic diagram of the most probable 3OC12-HSL <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Because inhibition of tyrosine kinases (by Gefinitib), of phospholipase C (PLC) (by U73122), of protein kinase C (PKC) (by Staurosporin), and of mitogen-activated protein kinase (MEK) (by SB203580) inhibited the chemotaxis induced by 3OC12-HSL, we propose that the putative 3OC12-HSL receptor binds to a tyrosine kinase and induces the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> indicated on the left. This <span class="highlight" style="background-color:">pathway</span> differs from that for IL-8 because the IL-8 receptor is associated with G protein (the signal can be inhibited by pertussis toxin) (<span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> shown on the right). Triggering of the IL-8 receptor also leads to activation of phospholipase C and phosphatidylinositol-3-kinase (PI3K). The latter presumably activates mitogen-activated protein kinase and thus could represent an alternative or additional <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for IL-8, and this explains why preincubation of PMN with 3OC12-HSL reduced the chemotaxis toward IL-8 and not vice versa. On the right, the well-established <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> for the IL-8-receptor (IL-8-R) is shown; activation of adenylate cyclase (AC) leads to upregulation of cyclic AMP (cAMP) and eventually upregulation of protein kinase A (PKA). DAG, diacylglycerol, Erk, extracellular signal-regulated kinase.</div></p></div><div class="aux"><div class="resc">Sabine Zimmermann, et al. Infect Immun. 2006 October;74(10):5687-5692.</div><p class="links"><a class="dblinks" href="/pubmed/16988244" ref="ordinalpos=533&amp;ncbi_uid=855785&amp;link_uid=16988244">Citation</a><a class="dblinks" href="/pmc/articles/PMC1594900/" ref="ordinalpos=533&amp;ncbi_uid=855785&amp;link_uid=PMC1594900">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>534.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3420937/figure/pgen.1002881-g006/" ref="ordinalpos=534&amp;ncbi_uid=4187560&amp;link_uid=PMC3420937" href="/pmc/articles/PMC3420937/"><img src="/pmc/articles/instance/3420937/bin/pgen.1002881.g006.gif" src-large="/pmc/articles/instance/3420937/bin/pgen.1002881.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3420937/" ref="ordinalpos=534&amp;ncbi_uid=4187560&amp;link_uid=PMC3420937" image-link="/pmc/articles/PMC3420937/figure/pgen.1002881-g006/" class="imagepopup">Figure 6. Multiple modes of crosstalk between the RAS/MAPK and insulin pathways during vulval development..  From: PTEN Negatively Regulates MAPK <span class="highlight" style="background-color:">Signaling</span> during Caenorhabditis elegans Vulval Development. </a></p><div class="supp"><p class="details"><div>Activation of LET-60 RAS in the VPCs causes phosphorylated MPK-1 to enter the nucleus where it phosphorylates transcription factors such as LIN-1 ETS, which represses 1° fate-specific gene expression. The insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> interacts with the RAS/MAPK <span class="highlight" style="background-color:">pathway</span> at several levels. Firstly, DAF-2 InsR activation enhances RAS/MAPK <span class="highlight" style="background-color:">signaling</span> by stimulating the PI3K AGE-1, which positively regulates RAS/MAPK <span class="highlight" style="background-color:">signaling</span> via increased PIP<sub>3</sub> production. In addition, DAF-2 <span class="highlight" style="background-color:">signaling</span> activates the RAS/MAPK <span class="highlight" style="background-color:">signaling</span> independently of AGE-1, possibly by recruiting SEM-5 to the plasma membrane. DAF-18 PTEN inhibits RAS/MAPK <span class="highlight" style="background-color:">signaling</span> in two distinct manners; by dephosphorylating PIP<sub>3</sub> and by negatively regulating MAPK activation.</div></p></div><div class="aux"><div class="resc">Itay Nakdimon, et al. PLoS Genet. 2012 August;8(8):e1002881.</div><p class="links"><a class="dblinks" href="/pubmed/22916028" ref="ordinalpos=534&amp;ncbi_uid=4187560&amp;link_uid=22916028">Citation</a><a class="dblinks" href="/pmc/articles/PMC3420937/" ref="ordinalpos=534&amp;ncbi_uid=4187560&amp;link_uid=PMC3420937">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>535.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4245135/figure/pone-0113419-g005/" ref="ordinalpos=535&amp;ncbi_uid=6926476&amp;link_uid=PMC4245135" href="/pmc/articles/PMC4245135/"><img src="/pmc/articles/instance/4245135/bin/pone.0113419.g005.gif" src-large="/pmc/articles/instance/4245135/bin/pone.0113419.g005.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4245135/" ref="ordinalpos=535&amp;ncbi_uid=6926476&amp;link_uid=PMC4245135" image-link="/pmc/articles/PMC4245135/figure/pone-0113419-g005/" class="imagepopup">Figure 5. <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> downstream of inflammatory mediator (TNF-α) <span class="highlight" style="background-color:">signaling</span> in human DPSC..  From: Chronic Inflammation and Angiogenic <span class="highlight" style="background-color:">Signaling</span> Axis Impairs Differentiation of Dental-Pulp Stem Cells. </a></p><div class="supp"><p class="details"><div>Prolonged exposure to TNF-α induce NF-κB <span class="highlight" style="background-color:">signaling</span> axis, which in turn induce an enhanced antiapoptotic gene expression and decreased telomere length. The resultant chromosomal instability leads to an increase in the proliferation of DPSC, with an impaired differentiation potential.</div></p></div><div class="aux"><div class="resc">Michael Boyle, et al. PLoS One. 2014;9(11):e113419.</div><p class="links"><a class="dblinks" href="/pubmed/25427002" ref="ordinalpos=535&amp;ncbi_uid=6926476&amp;link_uid=25427002">Citation</a><a class="dblinks" href="/pmc/articles/PMC4245135/" ref="ordinalpos=535&amp;ncbi_uid=6926476&amp;link_uid=PMC4245135">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>536.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3107509/figure/F1/" ref="ordinalpos=536&amp;ncbi_uid=3370236&amp;link_uid=PMC3107509" href="/pmc/articles/PMC3107509/"><img src="/pmc/articles/instance/3107509/bin/nihms298362f1.gif" src-large="/pmc/articles/instance/3107509/bin/nihms298362f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3107509/" ref="ordinalpos=536&amp;ncbi_uid=3370236&amp;link_uid=PMC3107509" image-link="/pmc/articles/PMC3107509/figure/F1/" class="imagepopup">Figure 1.  From: Wnt5a as an Effector of TGF? in Mammary Development and Cancer. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> pathways. Wnt <span class="highlight" style="background-color:">signaling</span> can be broadly divided into two categories: canonical, β-catenin dependent <span class="highlight" style="background-color:">signaling</span> and non-canonical, β-catenin independent <span class="highlight" style="background-color:">signaling</span>. The most well characterized of the non-canonical signals are the Planar Cell Polarity (PCP) <span class="highlight" style="background-color:">pathway</span> and Wnt/Ca + 2 <span class="highlight" style="background-color:">pathway</span>. PCP and Wnt/Ca <span class="highlight" style="background-color:">signaling</span> have been shown to regulate cell migration</div></p></div><div class="aux"><div class="resc">Rosa Serra, et al. J Mammary Gland Biol Neoplasia. ;16(2):157-167.</div><p class="links"><a class="dblinks" href="/pubmed/21416313" ref="ordinalpos=536&amp;ncbi_uid=3370236&amp;link_uid=21416313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3107509/" ref="ordinalpos=536&amp;ncbi_uid=3370236&amp;link_uid=PMC3107509">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>537.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2819435/figure/F1/" ref="ordinalpos=537&amp;ncbi_uid=2218094&amp;link_uid=PMC2819435" href="/pmc/articles/PMC2819435/"><img src="/pmc/articles/instance/2819435/bin/psb0412_1117_fig001.gif" src-large="/pmc/articles/instance/2819435/bin/psb0412_1117_fig001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2819435/" ref="ordinalpos=537&amp;ncbi_uid=2218094&amp;link_uid=PMC2819435" image-link="/pmc/articles/PMC2819435/figure/F1/" class="imagepopup">Figure 1.  From: Mechanisms of brassinosteroids interacting with multiple hormones. </a></p><div class="supp"><p class="details"><div>A proposed network of brassinosteroids interacting with multiple plant hormones. BR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is primarily inhibited by ABA <span class="highlight" style="background-color:">signaling</span> through certain unknown components, and BR may also regulate the ABA <span class="highlight" style="background-color:">signaling</span> outputs. BRs regulate auxin <span class="highlight" style="background-color:">signaling</span> via BIN2, while, reciprocally, BR <span class="highlight" style="background-color:">signaling</span> may also be regulated by auxin. BR affects the biosynthesis of ethylene and, in all probability, that of JA. Also, BR co-regulates physiological processes (black boxes), such as solute flow, with GA through primary <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> or the level of phytohormones.</div></p></div><div class="aux"><div class="resc">Shanshan Zhang, et al. Plant Signal Behav. 2009 December;4(12):1117-1120.</div><p class="links"><a class="dblinks" href="/pubmed/20514225" ref="ordinalpos=537&amp;ncbi_uid=2218094&amp;link_uid=20514225">Citation</a><a class="dblinks" href="/pmc/articles/PMC2819435/" ref="ordinalpos=537&amp;ncbi_uid=2218094&amp;link_uid=PMC2819435">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>538.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2952341/figure/F1/" ref="ordinalpos=538&amp;ncbi_uid=2510073&amp;link_uid=PMC2952341" href="/pmc/articles/PMC2952341/"><img src="/pmc/articles/instance/2952341/bin/nihms-232190-f0001.gif" src-large="/pmc/articles/instance/2952341/bin/nihms-232190-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2952341/" ref="ordinalpos=538&amp;ncbi_uid=2510073&amp;link_uid=PMC2952341" image-link="/pmc/articles/PMC2952341/figure/F1/" class="imagepopup">Figure 1. Canonical and non-canonical Notch <span class="highlight" style="background-color:">signaling</span> in mammals.  From: Alteration of Notch <span class="highlight" style="background-color:">signaling</span> in skeletal development and disease. </a></p><div class="supp"><p class="details"><div>In the canonical <span class="highlight" style="background-color:">pathway</span>, binding between transmembrane Notch receptors and its ligands triggers ADAM10- and Presenilin-mediated proteolytic cleavage. This liberates the membrane-bound Notch intracellular domain (NICD) to translocate into the nucleus, where NICD forms a transcriptional complex with nuclear factor RBPJ and the co-activator MAML to regulate transcription of downstream genes, such as <i>HES1</i> and <i>HEY1</i>, the two classic targets of Notch. The non-canonical <span class="highlight" style="background-color:">pathway</span> or RBPJ-independent Notch <span class="highlight" style="background-color:">signaling</span> occurs in Drosophila and mammals. Here, Notch can activate integrin via the membrane associated small GTPase R-Ras independent of binding RBPJ. In addition, NICD can interact with IKKα in the NF-kB <span class="highlight" style="background-color:">pathway</span> or LEF1 in the Wnt <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Jianning Tao, et al. Ann N Y Acad Sci. ;1192:257-268.</div><p class="links"><a class="dblinks" href="/pubmed/20392245" ref="ordinalpos=538&amp;ncbi_uid=2510073&amp;link_uid=20392245">Citation</a><a class="dblinks" href="/pmc/articles/PMC2952341/" ref="ordinalpos=538&amp;ncbi_uid=2510073&amp;link_uid=PMC2952341">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>539.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2810629/figure/F1/" ref="ordinalpos=539&amp;ncbi_uid=2492670&amp;link_uid=PMC2810629" href="/pmc/articles/PMC2810629/"><img src="/pmc/articles/instance/2810629/bin/nihms-134009-f0001.gif" src-large="/pmc/articles/instance/2810629/bin/nihms-134009-f0001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2810629/" ref="ordinalpos=539&amp;ncbi_uid=2492670&amp;link_uid=PMC2810629" image-link="/pmc/articles/PMC2810629/figure/F1/" class="imagepopup">Figure 1. Schematic Representation of Canonical and Non-canonical TGFβ <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>.  From: The Tale of Transforming Growth Factor-? (TGF?) <span class="highlight" style="background-color:">Signaling</span>: A Soign? Enigma. </a></p><div class="supp"><p class="details"><div>Binding of TGFβ to its cognate receptor initiates the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In the canonical <span class="highlight" style="background-color:">pathway</span>, activated type I receptors phosphorylates R-Smads, which subsequently form a complex with the Co-Smad, Smad4. The resulting R-Smads/Co-Smad complex translocates to the nucleus and interacts with distinct transcription factors to turn on or off transcription of many TGFβ-responsive genes that regulate cell proliferation and differentiation. Additionally, TGFβ activates different non-Smad pathways, including PI3K, Ras, Par6, Jnk/p38/MAPK pathways, which cumulatively regulate TGFβ-mediated functions.</div></p></div><div class="aux"><div class="resc">Arindam Chaudhury, et al. IUBMB Life. ;61(10):929-939.</div><p class="links"><a class="dblinks" href="/pubmed/19787707" ref="ordinalpos=539&amp;ncbi_uid=2492670&amp;link_uid=19787707">Citation</a><a class="dblinks" href="/pmc/articles/PMC2810629/" ref="ordinalpos=539&amp;ncbi_uid=2492670&amp;link_uid=PMC2810629">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>540.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4164656/figure/pone-0107678-g007/" ref="ordinalpos=540&amp;ncbi_uid=6628535&amp;link_uid=PMC4164656" href="/pmc/articles/PMC4164656/"><img src="/pmc/articles/instance/4164656/bin/pone.0107678.g007.gif" src-large="/pmc/articles/instance/4164656/bin/pone.0107678.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4164656/" ref="ordinalpos=540&amp;ncbi_uid=6628535&amp;link_uid=PMC4164656" image-link="/pmc/articles/PMC4164656/figure/pone-0107678-g007/" class="imagepopup">Figure 7. Modulation of plant development and acclimation to salinity are interconnected responses influenced by miR393 regulation node..  From: MiR393 Regulation of Auxin <span class="highlight" style="background-color:">Signaling</span> and Redox-Related Components during Acclimation to Salinity in Arabidopsis. </a></p><div class="supp"><p class="details"><div>These two processes are strongly regulated by reciprocal interaction between ROS and auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. NaCl-mediated stress would conduct to miR393 induction that targets TIR1-AFB2 auxin receptors leading to stabilization of Aux/IAA repressors and down- regulation of auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. In turn, auxin modulation influences on ROS-associated metabolism. Salt exposition leading to SIMR would be under negative regulation of the auxin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Depending on time and intensity of salt stress, early ROS-auxin crosstalk may exert impact on SIMR and tolerance to salinity. Others hormones and/or miRNAs may also regulate auxin-dependent <span class="highlight" style="background-color:">pathway</span> and physiological responses during acclimation to salinity.</div></p></div><div class="aux"><div class="resc">María José Iglesias, et al. PLoS One. 2014;9(9):e107678.</div><p class="links"><a class="dblinks" href="/pubmed/25222737" ref="ordinalpos=540&amp;ncbi_uid=6628535&amp;link_uid=25222737">Citation</a><a class="dblinks" href="/pmc/articles/PMC4164656/" ref="ordinalpos=540&amp;ncbi_uid=6628535&amp;link_uid=PMC4164656">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>541.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3741172/figure/pone-0072297-g006/" ref="ordinalpos=541&amp;ncbi_uid=5200789&amp;link_uid=PMC3741172" href="/pmc/articles/PMC3741172/"><img src="/pmc/articles/instance/3741172/bin/pone.0072297.g006.gif" src-large="/pmc/articles/instance/3741172/bin/pone.0072297.g006.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3741172/" ref="ordinalpos=541&amp;ncbi_uid=5200789&amp;link_uid=PMC3741172" image-link="/pmc/articles/PMC3741172/figure/pone-0072297-g006/" class="imagepopup">Figure 6. Schematic model of the impaired Reelin <span class="highlight" style="background-color:">signaling</span> in AD..  From: Beta-Amyloid Impairs Reelin <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>β-Amyloid triggers changes in Reelin glycosylation impairing its ability to form active covalent homodimers. Accordingly, altered Reelin complexes display a reduced efficiency to bind the ApoER2 receptor, fail to induce efficient tyrosine phosphorylation of Dab1, and to transduce the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that modulates tau phosphorylation by inhibition of GSK3β. Soluble Aβ can also contribute to the impaired ability of Reelin to activate its <span class="highlight" style="background-color:">signaling</span> cascade. An imbalance in 14-3-3 levels is also triggered by the inefficient Reelin <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Inmaculada Cuchillo-Ibáñez, et al. PLoS One. 2013;8(8):e72297.</div><p class="links"><a class="dblinks" href="/pubmed/23951306" ref="ordinalpos=541&amp;ncbi_uid=5200789&amp;link_uid=23951306">Citation</a><a class="dblinks" href="/pmc/articles/PMC3741172/" ref="ordinalpos=541&amp;ncbi_uid=5200789&amp;link_uid=PMC3741172">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>542.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2171390/figure/fig7/" ref="ordinalpos=542&amp;ncbi_uid=1110133&amp;link_uid=PMC2171390" href="/pmc/articles/PMC2171390/"><img src="/pmc/articles/instance/2171390/bin/200501111f7.gif" src-large="/pmc/articles/instance/2171390/bin/200501111f7.jpg" alt="Figure 7." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2171390/" ref="ordinalpos=542&amp;ncbi_uid=1110133&amp;link_uid=PMC2171390" image-link="/pmc/articles/PMC2171390/figure/fig7/" class="imagepopup">Figure 7. From: Fc?RI-mediated mast cell degranulation requires calcium-independent microtubule-dependent translocation of granules to the plasma membrane. </a></p><div class="supp"><p class="details"><div><b>Calcium-independent microtubule-dependent <span class="highlight" style="background-color:">pathway</span> is required for FcɛRI-mediated degranulation in mast cells.</b> The FcɛRI-mediated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> can be dissected into calcium-dependent and calcium-independent pathways. The calcium-independent <span class="highlight" style="background-color:">pathway</span> is critical for microtubule-dependent translocation of granules to the plasma membrane, and this <span class="highlight" style="background-color:">pathway</span> is regulated by Fyn/Gab2/RhoA <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Keigo Nishida, et al. J Cell Biol. 2005 July 4;170(1):115-126.</div><p class="links"><a class="dblinks" href="/pubmed/15998803" ref="ordinalpos=542&amp;ncbi_uid=1110133&amp;link_uid=15998803">Citation</a><a class="dblinks" href="/pmc/articles/PMC2171390/" ref="ordinalpos=542&amp;ncbi_uid=1110133&amp;link_uid=PMC2171390">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>543.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3825003/figure/Fig10/" ref="ordinalpos=543&amp;ncbi_uid=5517783&amp;link_uid=PMC3825003" href="/pmc/articles/PMC3825003/"><img src="/pmc/articles/instance/3825003/bin/11357_2013_9520_Fig10_HTML.gif" src-large="/pmc/articles/instance/3825003/bin/11357_2013_9520_Fig10_HTML.jpg" alt="Fig. 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3825003/" ref="ordinalpos=543&amp;ncbi_uid=5517783&amp;link_uid=PMC3825003" image-link="/pmc/articles/PMC3825003/figure/Fig10/" class="imagepopup">Fig. 10.  From: SIRT3 overexpression antagonizes high glucose accelerated cellular senescence in human diploid fibroblasts via the SIRT3-FOXO1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>Schematic model for a mechanism of SIRT3 overexpression antagonizes high glucose accelerated cellular senescence in human diploid fibroblasts via the SIRT3–FOXO1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></div></p></div><div class="aux"><div class="resc">Bin Zhang, et al. Age (Dordr). 2013 December;35(6):2237-2253.</div><p class="links"><a class="dblinks" href="/pubmed/23494737" ref="ordinalpos=543&amp;ncbi_uid=5517783&amp;link_uid=23494737">Citation</a><a class="dblinks" href="/pmc/articles/PMC3825003/" ref="ordinalpos=543&amp;ncbi_uid=5517783&amp;link_uid=PMC3825003">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>544.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2757173/figure/F6/" ref="ordinalpos=544&amp;ncbi_uid=2460310&amp;link_uid=PMC2757173" href="/pmc/articles/PMC2757173/"><img src="/pmc/articles/instance/2757173/bin/zbc0400987980006.gif" src-large="/pmc/articles/instance/2757173/bin/zbc0400987980006.jpg" alt="FIGURE 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2757173/" ref="ordinalpos=544&amp;ncbi_uid=2460310&amp;link_uid=PMC2757173" image-link="/pmc/articles/PMC2757173/figure/F6/" class="imagepopup">FIGURE 6. From: miR-331-3p Regulates ERBB-2 Expression and Androgen Receptor <span class="highlight" style="background-color:">Signaling</span> in Prostate Cancer. </a></p><div class="supp"><p class="details"><div><b>miR-331-3p blocks AR <span class="highlight" style="background-color:">signaling</span> via inhibition of ERBB-2 expression and AKT activity in prostate cancer cells.</b> AR antagonists such as bicalutamide bind to the AR and prevent its activation and expression of AR target genes, such as <i>PSA</i>. Nevertheless, AR <span class="highlight" style="background-color:">signaling</span> may persist in prostate cancer cells despite AR blockade, in part via increased expression of the ERBB-2 receptor tyrosine kinase and subsequent activation of the PI3K/AKT <span class="highlight" style="background-color:">pathway</span>, which causes AR phosphorylation and promotes expression of AR target genes. miR-331-3p directly targets the <i>ERBB-2</i> mRNA 3′-UTR to regulate ERBB-2 protein expression, thereby reducing PI3K/AKT <span class="highlight" style="background-color:">signaling</span> and AR <span class="highlight" style="background-color:">signaling</span>. The combination of an AR antagonist (bicalutamide) and miR-331-3p effectively blocks AR <span class="highlight" style="background-color:">signaling</span> (PSA expression and PSA promoter activity) in LNCaP prostate cancer cells. (+) indicates activation step of <span class="highlight" style="background-color:">pathway</span> and (−) indicates inhibition of <span class="highlight" style="background-color:">pathway</span> component.</div></p></div><div class="aux"><div class="resc">Michael R. Epis, et al. J Biol Chem. 2009 September 11;284(37):24696-24704.</div><p class="links"><a class="dblinks" href="/pubmed/19584056" ref="ordinalpos=544&amp;ncbi_uid=2460310&amp;link_uid=19584056">Citation</a><a class="dblinks" href="/pmc/articles/PMC2757173/" ref="ordinalpos=544&amp;ncbi_uid=2460310&amp;link_uid=PMC2757173">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>545.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3368902/figure/pone-0038604-g001/" ref="ordinalpos=545&amp;ncbi_uid=4062127&amp;link_uid=PMC3368902" href="/pmc/articles/PMC3368902/"><img src="/pmc/articles/instance/3368902/bin/pone.0038604.g001.gif" src-large="/pmc/articles/instance/3368902/bin/pone.0038604.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3368902/" ref="ordinalpos=545&amp;ncbi_uid=4062127&amp;link_uid=PMC3368902" image-link="/pmc/articles/PMC3368902/figure/pone-0038604-g001/" class="imagepopup">Figure 1. Schematic representation of experimentally validated actions of mammalian miRNAs on the mammalian hepatic insulin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Postprandial Regulation of Hepatic MicroRNAs Predicted to Target the Insulin <span class="highlight" style="background-color:">Pathway</span> in Rainbow Trout. </a></p><div class="supp"><p class="details"><div>Encircled + or – symbols indicate a stimulatory and inhibitory role on the insulin <span class="highlight" style="background-color:">pathway</span> in mammalian model systems. See text for explanations.</div></p></div><div class="aux"><div class="resc">Jan A. Mennigen, et al. PLoS One. 2012;7(6):e38604.</div><p class="links"><a class="dblinks" href="/pubmed/22701681" ref="ordinalpos=545&amp;ncbi_uid=4062127&amp;link_uid=22701681">Citation</a><a class="dblinks" href="/pmc/articles/PMC3368902/" ref="ordinalpos=545&amp;ncbi_uid=4062127&amp;link_uid=PMC3368902">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>546.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3172793/figure/F3/" ref="ordinalpos=546&amp;ncbi_uid=3569328&amp;link_uid=PMC3172793" href="/pmc/articles/PMC3172793/"><img src="/pmc/articles/instance/3172793/bin/zh89991178110003.gif" src-large="/pmc/articles/instance/3172793/bin/zh89991178110003.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3172793/" ref="ordinalpos=546&amp;ncbi_uid=3569328&amp;link_uid=PMC3172793" image-link="/pmc/articles/PMC3172793/figure/F3/" class="imagepopup">Figure 3.  From: Innate signals overcome acquired TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> regulation and govern the fate of human CD161hi CD8?+ semi-invariant T cells. </a></p><div class="supp"><p class="details"><div><b>The TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is regulated at multiple levels in CD161<sup>hi</sup> CD62L<sup>+</sup> and CD62L<sup>−</sup> cells.</b> (A) A transcriptional <span class="highlight" style="background-color:">pathway</span> analysis of the TCR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in CD161<sup>hi</sup> and CD161<sup>lo</sup> CD62L<sup>+</sup> subsets from 6 healthy donors was created using Ingenuity Systems software (IPA 7). Green and red symbols represent significantly down-regulated or up-regulated genes, respectively, in CD161<sup>hi</sup> CD62L<sup>+</sup> cells compared with their CD161<sup>lo</sup> counterparts. White, gray, and orange symbols are not differentially expressed between subsets. (B) Volcano plots showing differentially expressed transcriptional profiles of CD161<sup>hi</sup> CD62L<sup>+</sup> (left) and CD161<sup>hi</sup> CD62L<sup>−</sup> (right) subsets compared with their CD161<sup>lo</sup> counterparts are shown from 6 healthy donors. Genes/probes are color-coded as indicated in the legend.</div></p></div><div class="aux"><div class="resc">Cameron J. Turtle, et al. Blood. 2011 September 8;118(10):2752-2762.</div><p class="links"><a class="dblinks" href="/pubmed/21791427" ref="ordinalpos=546&amp;ncbi_uid=3569328&amp;link_uid=21791427">Citation</a><a class="dblinks" href="/pmc/articles/PMC3172793/" ref="ordinalpos=546&amp;ncbi_uid=3569328&amp;link_uid=PMC3172793">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>547.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2099483/figure/F1/" ref="ordinalpos=547&amp;ncbi_uid=1119841&amp;link_uid=PMC2099483" href="/pmc/articles/PMC2099483/"><img src="/pmc/articles/instance/2099483/bin/1471-2105-8-S7-S15-1.gif" src-large="/pmc/articles/instance/2099483/bin/1471-2105-8-S7-S15-1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2099483/" ref="ordinalpos=547&amp;ncbi_uid=1119841&amp;link_uid=PMC2099483" image-link="/pmc/articles/PMC2099483/figure/F1/" class="imagepopup">Figure 1.  From: Analysis of circadian pattern reveals tissue-specific alternative transcription in leptin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>Circadian oscillation in leptin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in mouse brown fat</b>. This is a fragment of the KEGG <span class="highlight" style="background-color:">pathway</span> with superimposed plots of circadian expression profiles for the essential components of the <span class="highlight" style="background-color:">pathway</span>. Every gene involved in leptin <span class="highlight" style="background-color:">signaling</span> demonstrates explicit circadian pattern in baseline oscillation. All except one oscillate in the same phase. Leptin itself is actively expressed in brown adipose tissue and its intensity oscillates with a phase shift, slightly behind leptin receptor (LEPR) and other components of leptin <span class="highlight" style="background-color:">signaling</span>. Remarkably, suppressor of cytokine signal (SOCS3) has two alternative transcripts oscillating in counter-phase.</div></p></div><div class="aux"><div class="resc">Andrey A Ptitsyn, et al. BMC Bioinformatics. 2007;8(Suppl 7):S15-S15.</div><p class="links"><a class="dblinks" href="/pubmed/18047714" ref="ordinalpos=547&amp;ncbi_uid=1119841&amp;link_uid=18047714">Citation</a><a class="dblinks" href="/pmc/articles/PMC2099483/" ref="ordinalpos=547&amp;ncbi_uid=1119841&amp;link_uid=PMC2099483">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>548.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2994580/figure/fig3-1947601910382901/" ref="ordinalpos=548&amp;ncbi_uid=3024626&amp;link_uid=PMC2994580" href="/pmc/articles/PMC2994580/"><img src="/pmc/articles/instance/3092232/bin/10.1177_1947601910382901-fig3.gif" src-large="/pmc/articles/instance/3092232/bin/10.1177_1947601910382901-fig3.jpg" alt="Figure 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2994580/" ref="ordinalpos=548&amp;ncbi_uid=3024626&amp;link_uid=PMC2994580" image-link="/pmc/articles/PMC2994580/figure/fig3-1947601910382901/" class="imagepopup">Figure 3. From: Ubiquitination-Dependent Regulation of <span class="highlight" style="background-color:">Signaling</span> Receptors in Cancer. </a></p><div class="supp"><p class="details"><div>Mechanisms underlying the regulation of <span class="highlight" style="background-color:">signaling</span> pathways. Cellular responses to a specific ligand (e.g., ligand A) result in changes of the gene expression program. Some of the products of ligand A–inducible genes might be capable of inhibiting the <span class="highlight" style="background-color:">signaling</span> cascade via negative feedback (exemplified by cytokine-mediated induction of tyrosine phosphatases of JAK inhibitors and denoted by a blue solid arrow). However, a much more rapid regulation of <span class="highlight" style="background-color:">signaling</span> can occur due to the branching of <span class="highlight" style="background-color:">signaling</span> pathways. In this example, “stem” <span class="highlight" style="background-color:">signaling</span> induced by ligand A includes activation of receptor A1 followed by activation of a <span class="highlight" style="background-color:">signaling</span> element A2. Subsequent “forward” <span class="highlight" style="background-color:">signaling</span> involves activation of A3 and other elements that lead to transcription changes. Concurrently, eliminative <span class="highlight" style="background-color:">signaling</span> (involving A2-dependent activation of a factor X and effect of X on receptor downregulation denoted by a solid red arrow) results in rapid downregulation and degradation of A1, thereby restricting the duration and magnitude of cellular responses to ligand A. This <span class="highlight" style="background-color:">pathway</span> could be employed by an unrelated <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> via “cross”-<span class="highlight" style="background-color:">signaling</span> initiated by ligand B, which can lower the sensitivity of the cell to ligand A via activation of factor X.</div></p></div><div class="aux"><div class="resc">Wei-Chun HuangFu, et al. Genes Cancer. 2010 July;1(7):725-734.</div><p class="links"><a class="dblinks" href="/pubmed/21127735" ref="ordinalpos=548&amp;ncbi_uid=3024626&amp;link_uid=21127735">Citation</a><a class="dblinks" href="/pmc/articles/PMC2994580/" ref="ordinalpos=548&amp;ncbi_uid=3024626&amp;link_uid=PMC2994580">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>549.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g010/" ref="ordinalpos=549&amp;ncbi_uid=2242913&amp;link_uid=PMC2854141" href="/pmc/articles/PMC2854141/"><img src="/pmc/articles/instance/2854141/bin/pone.0010153.g010.gif" src-large="/pmc/articles/instance/2854141/bin/pone.0010153.g010.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2854141/" ref="ordinalpos=549&amp;ncbi_uid=2242913&amp;link_uid=PMC2854141" image-link="/pmc/articles/PMC2854141/figure/pone-0010153-g010/" class="imagepopup">Figure 10. Calcium <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>..  From: Uncovering Molecular Biomarkers That Correlate Cognitive Decline with the Changes of Hippocampus' Gene Expression Profiles in Alzheimer's Disease. </a></p><div class="supp"><p class="details"><div>The upper graph presents the stacked normalized expression values of all the probes involved in the Calcium <span class="highlight" style="background-color:">signaling</span> with an upregulation trend. The lower graph analyses the genes involved in the <span class="highlight" style="background-color:">pathway</span> with a downregulation tendency. In the supplementary material (File S3 sheet ‘Calcium signalling <span class="highlight" style="background-color:">pathway</span>’), the reader will find all the individual gene expression values, normalised and not normalised.</div></p></div><div class="aux"><div class="resc">Martín Gómez Ravetti, et al. PLoS One. 2010;5(4):e10153.</div><p class="links"><a class="dblinks" href="/pubmed/20405009" ref="ordinalpos=549&amp;ncbi_uid=2242913&amp;link_uid=20405009">Citation</a><a class="dblinks" href="/pmc/articles/PMC2854141/" ref="ordinalpos=549&amp;ncbi_uid=2242913&amp;link_uid=PMC2854141">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>550.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3036164/figure/F8/" ref="ordinalpos=550&amp;ncbi_uid=3650293&amp;link_uid=PMC3036164" href="/pmc/articles/PMC3036164/"><img src="/pmc/articles/instance/3036164/bin/nihms259428f8.gif" src-large="/pmc/articles/instance/3036164/bin/nihms259428f8.jpg" alt="Fig. 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3036164/" ref="ordinalpos=550&amp;ncbi_uid=3650293&amp;link_uid=PMC3036164" image-link="/pmc/articles/PMC3036164/figure/F8/" class="imagepopup">Fig. 8. Model demonstrating how Alcama mediates Edn1 <span class="highlight" style="background-color:">signaling</span> and NC differentiation.  From: Alcama mediates Edn1 <span class="highlight" style="background-color:">signaling</span> during zebrafish cartilage morphogenesis. </a></p><div class="supp"><p class="details"><div>Previous data has supported the model that Edn1 peptide binds to its G-protein coupled receptor on NC to turn on transcription of NC genes. We propose a parallel <span class="highlight" style="background-color:">pathway</span> by which Edn1 <span class="highlight" style="background-color:">signaling</span> stabilizes Alcama protein in endodermal cells. Alcama, in turn, binds to Nadl1.1 in NC and regulates NC differentiation.</div></p></div><div class="aux"><div class="resc">Priya Choudhry, et al. Dev Biol. ;349(2):483-493.</div><p class="links"><a class="dblinks" href="/pubmed/21073867" ref="ordinalpos=550&amp;ncbi_uid=3650293&amp;link_uid=21073867">Citation</a><a class="dblinks" href="/pmc/articles/PMC3036164/" ref="ordinalpos=550&amp;ncbi_uid=3650293&amp;link_uid=PMC3036164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>551.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2721971/figure/fig03/" ref="ordinalpos=551&amp;ncbi_uid=2054540&amp;link_uid=PMC2721971" href="/pmc/articles/PMC2721971/"><img src="/pmc/articles/instance/2721971/bin/tra0010-0982-f3.gif" src-large="/pmc/articles/instance/2721971/bin/tra0010-0982-f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2721971/" ref="ordinalpos=551&amp;ncbi_uid=2054540&amp;link_uid=PMC2721971" image-link="/pmc/articles/PMC2721971/figure/fig03/" class="imagepopup">Figure 3. The EGFR-GEP100-Arf6-AMAP1 <span class="highlight" style="background-color:">pathway</span> specific to cancer invasion and metastasis.  From: The EGFR-GEP100-Arf6-AMAP1 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> Specific to Breast Cancer Invasion and Metastasis†. </a></p><div class="supp"><p class="details"><div>Besides well-known EGFR <span class="highlight" style="background-color:">signaling</span> pathways that link to the cell invasion and migration via activation of Ser/Thr kinases, EGFR can evoke a novel <span class="highlight" style="background-color:">pathway</span> that leads to the cancer invasion and metastasis, in which phosphorylated Tyr1068 or Tyr1086 binds directly to the PH domain of GEP100. Through this binding, GEP100 activates Arf6 and its downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> employing AMAP1 as its effector. For functioning of this <span class="highlight" style="background-color:">pathway</span>, it is prerequisite that both Arf6 and AMAP1 are highly overexpressed, and that Tyr1068 or Tyr1086 is highly phosphorylated. This EGFR-GEP100-Arf6-AMAP1 <span class="highlight" style="background-color:">pathway</span> plays pivotal roles not only in the formation and function of invadopodia, but also in perturbation of the E-cadherin-based cell–cell adhesion. Arf6 can be ubiquitinated by Fbx8, a component of the SCF complex. Loss of Fbx8 expression is frequently observed in different breast cancer cells, and contributes to their invasiveness. Monoubiquitination of AMAP1 by Cbl, via AMAP1's binding to CIN85, is also necessary for the invasion.</div></p></div><div class="aux"><div class="resc">Hisataka Sabe, et al. Traffic. 2009 August;10(8):982-993.</div><p class="links"><a class="dblinks" href="/pubmed/19416474" ref="ordinalpos=551&amp;ncbi_uid=2054540&amp;link_uid=19416474">Citation</a><a class="dblinks" href="/pmc/articles/PMC2721971/" ref="ordinalpos=551&amp;ncbi_uid=2054540&amp;link_uid=PMC2721971">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>552.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2888641/figure/F3/" ref="ordinalpos=552&amp;ncbi_uid=2982414&amp;link_uid=PMC2888641" href="/pmc/articles/PMC2888641/"><img src="/pmc/articles/instance/2888641/bin/nihms196795f3.gif" src-large="/pmc/articles/instance/2888641/bin/nihms196795f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2888641/" ref="ordinalpos=552&amp;ncbi_uid=2982414&amp;link_uid=PMC2888641" image-link="/pmc/articles/PMC2888641/figure/F3/" class="imagepopup">Figure 3. Hedgehog <span class="highlight" style="background-color:">Signaling</span> in Multiple Myeloma.  From: Targeting Hedgehog - a Cancer Stem Cell <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Regulation of MM CSC by the HH <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The inhibition of HH <span class="highlight" style="background-color:">signaling</span> (A) inhibits MM CSC displaying the side population phenotype by (B) inducing terminal plasma cells differentiation of MM CSC as indicated by the expression of CD138. (C) Multiple modes of <span class="highlight" style="background-color:">signaling</span> appear to be active in MM. Experimental data suggest that differentiated plasma cells can produce the ligand necessary for CSC survival and proliferation. Blocking <span class="highlight" style="background-color:">signaling</span> leads to CSC differentiation. Normal bone marrow stromal cells can also produce ligand and signal to myeloma cells to support their growth and survival. A possible role for tumor-to-stoma paracrine <span class="highlight" style="background-color:">signaling</span> may also take occur.</div></p></div><div class="aux"><div class="resc">Akil Merchant, et al. Clin Cancer Res. ;16(12):3130-3140.</div><p class="links"><a class="dblinks" href="/pubmed/20530699" ref="ordinalpos=552&amp;ncbi_uid=2982414&amp;link_uid=20530699">Citation</a><a class="dblinks" href="/pmc/articles/PMC2888641/" ref="ordinalpos=552&amp;ncbi_uid=2982414&amp;link_uid=PMC2888641">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>553.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3825669/figure/f6-grsb-7-2013-139/" ref="ordinalpos=553&amp;ncbi_uid=5496678&amp;link_uid=PMC3825669" href="/pmc/articles/PMC3825669/"><img src="/pmc/articles/instance/3825669/bin/grsb-7-2013-139f6.gif" src-large="/pmc/articles/instance/3825669/bin/grsb-7-2013-139f6.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3825669/" ref="ordinalpos=553&amp;ncbi_uid=5496678&amp;link_uid=PMC3825669" image-link="/pmc/articles/PMC3825669/figure/f6-grsb-7-2013-139/" class="imagepopup">Figure 6.  From: <span class="highlight" style="background-color:">Pathway</span> Analysis of ChIP-Seq-Based NRF1 Target Genes Suggests a Logical Hypothesis of their Involvement in the Pathogenesis of Neurodegenerative Diseases. </a></p><div class="supp"><p class="details"><div>IPA “regulation of eIF4 and p70S6K <span class="highlight" style="background-color:">signaling</span>” <span class="highlight" style="background-color:">pathway</span> relevant to NRF target genes. Entrez Gene IDs of 2,470 NRF1 target genes were imported into the Core Analysis tool of IPA. It extracted the “Regulation of eIF4 and p70S6K <span class="highlight" style="background-color:">signaling</span>” <span class="highlight" style="background-color:">pathway</span> as the second rank significant <span class="highlight" style="background-color:">pathway</span> as listed in Table 3. NRF1 target genes are highlighted by red.</div></p></div><div class="aux"><div class="resc">Jun-ichi Satoh, et al. Gene Regul Syst Bio. 2013;7:139-152.</div><p class="links"><a class="dblinks" href="/pubmed/24250222" ref="ordinalpos=553&amp;ncbi_uid=5496678&amp;link_uid=24250222">Citation</a><a class="dblinks" href="/pmc/articles/PMC3825669/" ref="ordinalpos=553&amp;ncbi_uid=5496678&amp;link_uid=PMC3825669">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>554.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3218933/figure/F2/" ref="ordinalpos=554&amp;ncbi_uid=3517579&amp;link_uid=PMC3218933" href="/pmc/articles/PMC3218933/"><img src="/pmc/articles/instance/3218933/bin/bcr2876-2.gif" src-large="/pmc/articles/instance/3218933/bin/bcr2876-2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3218933/" ref="ordinalpos=554&amp;ncbi_uid=3517579&amp;link_uid=PMC3218933" image-link="/pmc/articles/PMC3218933/figure/F2/" class="imagepopup">Figure 2.  From: Breast cancer growth and metastasis: interplay between cancer stem cells, embryonic <span class="highlight" style="background-color:">signaling</span> pathways and epithelial-to-mesenchymal transition. </a></p><div class="supp"><p class="details"><div><b>Cross-talk among embryonic <span class="highlight" style="background-color:">signaling</span> pathways and experimental inhibitors</b>. Potential cross-talk among embryonic <span class="highlight" style="background-color:">signaling</span> pathways (transforming growth factor (TGF)-β, Hedgehog (Hh), Notch, Wnt, and ErbB) is shown. The expansive potential for <span class="highlight" style="background-color:">signaling</span> cross-talk suggests that <span class="highlight" style="background-color:">signaling</span> pathways do not function in isolation, but instead are parts of a complex <span class="highlight" style="background-color:">signaling</span> network. Cross-talk can lead to both enhancing and inhibitory interactions between pathways. The actions of experimental <span class="highlight" style="background-color:">pathway</span> inhibitors are shown in yellow. Positive interactions are shown in green, while negative interactions are shown in red.</div></p></div><div class="aux"><div class="resc">Naoko Takebe, et al. Breast Cancer Res. 2011;13(3):211-211.</div><p class="links"><a class="dblinks" href="/pubmed/21672282" ref="ordinalpos=554&amp;ncbi_uid=3517579&amp;link_uid=21672282">Citation</a><a class="dblinks" href="/pmc/articles/PMC3218933/" ref="ordinalpos=554&amp;ncbi_uid=3517579&amp;link_uid=PMC3218933">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>555.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2838407/figure/fig6/" ref="ordinalpos=555&amp;ncbi_uid=2236967&amp;link_uid=PMC2838407" href="/pmc/articles/PMC2838407/"><img src="/pmc/articles/instance/2838407/bin/tx-2009-003193_0002.gif" src-large="/pmc/articles/instance/2838407/bin/tx-2009-003193_0002.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2838407/" ref="ordinalpos=555&amp;ncbi_uid=2236967&amp;link_uid=PMC2838407" image-link="/pmc/articles/PMC2838407/figure/fig6/" class="imagepopup">Figure 6.  From: Reactive Oxygen Species-Activated Akt/ASK1/p38 <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> in Nickel Compound-Induced Apoptosis in BEAS 2B Cells. </a></p><div class="supp"><p class="details"><div>Schematic model for the role of ROS/Akt/ASK1/p38 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in nickel-induced apoptosis. Nickel-generated ROS stimulate the phosphorylation of Akt, which in turn activates the ASK1 by phosphorylating ASK1 on Thr838, then followed by p38 activation, leading to an apoptotic effect.</div></p></div><div class="aux"><div class="resc">Jingju Pan, et al. Chem Res Toxicol. 2010 March 15;23(3):568-577.</div><p class="links"><a class="dblinks" href="/pubmed/20112989" ref="ordinalpos=555&amp;ncbi_uid=2236967&amp;link_uid=20112989">Citation</a><a class="dblinks" href="/pmc/articles/PMC2838407/" ref="ordinalpos=555&amp;ncbi_uid=2236967&amp;link_uid=PMC2838407">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>556.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3021862/figure/fig4/" ref="ordinalpos=556&amp;ncbi_uid=2719130&amp;link_uid=PMC3021862" href="/pmc/articles/PMC3021862/"><img src="/pmc/articles/instance/3021862/bin/JO2011-946936.004.gif" src-large="/pmc/articles/instance/3021862/bin/JO2011-946936.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3021862/" ref="ordinalpos=556&amp;ncbi_uid=2719130&amp;link_uid=PMC3021862" image-link="/pmc/articles/PMC3021862/figure/fig4/" class="imagepopup">Figure 4.  From: Survivin Selectively Modulates Genes Deregulated in Human Leukemia Stem Cells. </a></p><div class="supp"><p class="details"><div>Genes regulated by Survivin in ITD-Flt3<sup>+</sup>KSL cells and deregulated in LSC are mapped on functional <span class="highlight" style="background-color:">signaling</span> network associated with epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Epidermal growth factor receptor <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (EGFR) is significantly enriched in the shared genes associated with LSC and Survivin <span class="highlight" style="background-color:">signaling</span> by <span class="highlight" style="background-color:">pathway</span> enrichment analysis in KEGG and Gene Ontology database. Our analysis shows that 13 molecules shared by LSC and Survivin <span class="highlight" style="background-color:">signaling</span> are mapped on a functional <span class="highlight" style="background-color:">signaling</span> network associated with EGFR. The network was created based on KEGG database using Cytoscape software. Green circle and red circle represents downregulation and upregulation by Survivin gene deletion, respectively.</div></p></div><div class="aux"><div class="resc">Seiji Fukuda, et al. J Oncol. 2011;2011:946936.</div><p class="links"><a class="dblinks" href="/pubmed/21253548" ref="ordinalpos=556&amp;ncbi_uid=2719130&amp;link_uid=21253548">Citation</a><a class="dblinks" href="/pmc/articles/PMC3021862/" ref="ordinalpos=556&amp;ncbi_uid=2719130&amp;link_uid=PMC3021862">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>557.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3867757/figure/f1-jmbe-14-197/" ref="ordinalpos=557&amp;ncbi_uid=5615598&amp;link_uid=PMC3867757" href="/pmc/articles/PMC3867757/"><img src="/pmc/articles/instance/3867757/bin/jmbe-14-197f1.gif" src-large="/pmc/articles/instance/3867757/bin/jmbe-14-197f1.jpg" alt="FIGURE 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3867757/" ref="ordinalpos=557&amp;ncbi_uid=5615598&amp;link_uid=PMC3867757" image-link="/pmc/articles/PMC3867757/figure/f1-jmbe-14-197/" class="imagepopup">FIGURE 1. From: What's Downstream? A Set of Classroom Exercises to Help Students Understand Recessive Epistasis . </a></p><div class="supp"><p class="details"><div>A hypothetical yeast enzymatic <span class="highlight" style="background-color:">pathway</span> regulated by a hypothetical <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. (A) Enzymatic <span class="highlight" style="background-color:">pathway</span> alone (see text for context). A, B, and C are small molecules. Heavy arrows represent enzymatic reactions, and light arrows represent expression of the genes that encode the corresponding enzymes in the <span class="highlight" style="background-color:">pathway</span>. This <span class="highlight" style="background-color:">pathway</span> is controlled by the presence or absence of sucralose. (B) The same enzymatic <span class="highlight" style="background-color:">pathway</span> (lower line), intersecting with a <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (blue line) that is controlled by the presence or absence of neotame. In the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, R, S, and T are proteins with the functions shown, encoded by Genes 3, 4, and 5 respectively. Light arrows indicate gene expression, but here, heavy arrows represent regulatory interactions: pointed arrows indicate activation, and the blunt arrow indicates inhibition. For example, in the presence of both sucralose and neotame, if the receptor R is activated by neotame binding, it activates the protein S. If S is active, it inactivates the transcription factor T, which is otherwise active and is required to activate transcription of Gene 2.</div></p></div><div class="aux"><div class="resc">Jennifer K. Knight, et al. J Microbiol Biol Educ. 2013;14(2):197-205.</div><p class="links"><a class="dblinks" href="/pubmed/24358383" ref="ordinalpos=557&amp;ncbi_uid=5615598&amp;link_uid=24358383">Citation</a><a class="dblinks" href="/pmc/articles/PMC3867757/" ref="ordinalpos=557&amp;ncbi_uid=5615598&amp;link_uid=PMC3867757">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>558.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3530847/figure/F10/" ref="ordinalpos=558&amp;ncbi_uid=4467471&amp;link_uid=PMC3530847" href="/pmc/articles/PMC3530847/"><img src="/pmc/articles/instance/3530847/bin/nihms427843f10.gif" src-large="/pmc/articles/instance/3530847/bin/nihms427843f10.jpg" alt="Figure 10" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3530847/" ref="ordinalpos=558&amp;ncbi_uid=4467471&amp;link_uid=PMC3530847" image-link="/pmc/articles/PMC3530847/figure/F10/" class="imagepopup">Figure 10.  From: <span class="highlight" style="background-color:">PATHWAY</span> CROSS-TALK BETWEEN RAS/RAF AND PI3K IN PROMOTION OF M-CSF-INDUCED MEK/ERK-MEDIATED OSTEOCLAST SURVIVAL. </a></p><div class="supp"><p class="details"><div>Model of <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Elizabeth W. Bradley, et al. J Cell Biochem. ;104(4):1439-1451.</div><p class="links"><a class="dblinks" href="/pubmed/18275061" ref="ordinalpos=558&amp;ncbi_uid=4467471&amp;link_uid=18275061">Citation</a><a class="dblinks" href="/pmc/articles/PMC3530847/" ref="ordinalpos=558&amp;ncbi_uid=4467471&amp;link_uid=PMC3530847">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>559.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2935335/figure/f2/" ref="ordinalpos=559&amp;ncbi_uid=3059576&amp;link_uid=PMC2935335" href="/pmc/articles/PMC2935335/"><img src="/pmc/articles/instance/2935335/bin/fig-2.gif" src-large="/pmc/articles/instance/2935335/bin/fig-2.jpg" alt="FIG. 2." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2935335/" ref="ordinalpos=559&amp;ncbi_uid=3059576&amp;link_uid=PMC2935335" image-link="/pmc/articles/PMC2935335/figure/f2/" class="imagepopup">FIG. 2. From: Genetic Alterations in the Phosphatidylinositol-3 Kinase/Akt <span class="highlight" style="background-color:">Pathway</span> in Thyroid Cancer. </a></p><div class="supp"><p class="details"><div>Diagrammatic illustration of a self-enhancement mechanism of the activation of PI3K/Akt <span class="highlight" style="background-color:">pathway</span> through PTEN methylation in thyroid cancer. In this model, activating genetic alterations, including gene copy gain or mutations in the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> activate the <span class="highlight" style="background-color:">pathway</span>. This in turn leads to <i>PTEN</i> methylation, resulting in silencing of the <i>PTEN</i> gene. Loss of PTEN removes the inhibitory pressure of the <span class="highlight" style="background-color:">pathway</span>, forming a malicious self-activation cycle for the PI3K/Akt <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. This is seen most commonly in aggressive thyroid cancers, such as anaplastic thyroid cancer (ATC), as discussed in the text.</div></p></div><div class="aux"><div class="resc">Mingzhao Xing. Thyroid. 2010 July;20(7):697-706.</div><p class="links"><a class="dblinks" href="/pubmed/20578891" ref="ordinalpos=559&amp;ncbi_uid=3059576&amp;link_uid=20578891">Citation</a><a class="dblinks" href="/pmc/articles/PMC2935335/" ref="ordinalpos=559&amp;ncbi_uid=3059576&amp;link_uid=PMC2935335">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>560.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3247940/figure/F8/" ref="ordinalpos=560&amp;ncbi_uid=4469980&amp;link_uid=PMC3247940" href="/pmc/articles/PMC3247940/"><img src="/pmc/articles/instance/3247940/bin/zbc0521190740008.gif" src-large="/pmc/articles/instance/3247940/bin/zbc0521190740008.jpg" alt="FIGURE 8." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3247940/" ref="ordinalpos=560&amp;ncbi_uid=4469980&amp;link_uid=PMC3247940" image-link="/pmc/articles/PMC3247940/figure/F8/" class="imagepopup">FIGURE 8. From: HIV-1 Glycoprotein 41 Ectodomain Induces Activation of the CD74 Protein-mediated Extracellular Signal-regulated Kinase/Mitogen-activated Protein Kinase <span class="highlight" style="background-color:">Pathway</span> to Enhance Viral Infection. </a></p><div class="supp"><p class="details"><div><b>Model of enhancement of HIV-1 infection as a result of HIV-1 gp41 binding to CD74 and subsequent activation of the ERK/MAPK signal <span class="highlight" style="background-color:">pathway</span>.</b> <i>a</i>, gp41, which is exposed during membrane fusion, binds CD74 and activates the ERK/MAPK <span class="highlight" style="background-color:">pathway</span>. <i>b</i>, gp41 existing on defective virions directly binds CD74 and activates the ERK/MAPK <span class="highlight" style="background-color:">pathway</span>. <i>c</i>, degraded gp41 fragments containing loop sequence (like peptide 6358) bind CD74 to activate the ERK/MAPK <span class="highlight" style="background-color:">pathway</span> and enhance HIV-1 infection. The ERK/MAPK downstream <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> has been described elsewhere (46, 51).</div></p></div><div class="aux"><div class="resc">Chang Zhou, et al. J Biol Chem. 2011 December 30;286(52):44869-44877.</div><p class="links"><a class="dblinks" href="/pubmed/22039051" ref="ordinalpos=560&amp;ncbi_uid=4469980&amp;link_uid=22039051">Citation</a><a class="dblinks" href="/pmc/articles/PMC3247940/" ref="ordinalpos=560&amp;ncbi_uid=4469980&amp;link_uid=PMC3247940">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>561.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC86054/figure/F1/" ref="ordinalpos=561&amp;ncbi_uid=23081&amp;link_uid=PMC86054" href="/pmc/articles/PMC86054/"><img src="/pmc/articles/instance/86054/bin/mb1502098001.gif" src-large="/pmc/articles/instance/86054/bin/mb1502098001.jpg" alt="FIG. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC86054/" ref="ordinalpos=561&amp;ncbi_uid=23081&amp;link_uid=PMC86054" image-link="/pmc/articles/PMC86054/figure/F1/" class="imagepopup">FIG. 1.  From: Activation of the Kss1 Invasive-Filamentous Growth <span class="highlight" style="background-color:">Pathway</span> Induces Ty1 Transcription and Retrotransposition in Saccharomyces cerevisiae. </a></p><div class="supp"><p class="details"><div>MAPK pathways regulating invasive-filamentous and mating developments. The invasive-filamentous <span class="highlight" style="background-color:">pathway</span> is depicted on the left, and the mating <span class="highlight" style="background-color:">pathway</span> is shown on the right. Ste5 binds Ste11, Ste7, and Fus3 proteins and is required for pheromone-induced <span class="highlight" style="background-color:">signaling</span> (41). FRE, filamentation- and invasion-responsive element; PRE, pheromone-responsive element; Pbox, Mcm1 binding site. In the invasive-filamentous <span class="highlight" style="background-color:">pathway</span>, Ste12 binds to FRE cooperatively with Tec1. Ste12 acts in the mating <span class="highlight" style="background-color:">pathway</span> by binding to DNA as a homomultimer or as a heteromultimer with Mcm1.</div></p></div><div class="aux"><div class="resc">Antonin Morillon, et al. Mol Cell Biol. 2000 August;20(15):5766-5776.</div><p class="links"><a class="dblinks" href="/pubmed/10891512" ref="ordinalpos=561&amp;ncbi_uid=23081&amp;link_uid=10891512">Citation</a><a class="dblinks" href="/pmc/articles/PMC86054/" ref="ordinalpos=561&amp;ncbi_uid=23081&amp;link_uid=PMC86054">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>562.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3723343/figure/F3/" ref="ordinalpos=562&amp;ncbi_uid=5126284&amp;link_uid=PMC3723343" href="/pmc/articles/PMC3723343/"><img src="/pmc/articles/instance/3723343/bin/nihms357082f3.gif" src-large="/pmc/articles/instance/3723343/bin/nihms357082f3.jpg" alt="Figure 3" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3723343/" ref="ordinalpos=562&amp;ncbi_uid=5126284&amp;link_uid=PMC3723343" image-link="/pmc/articles/PMC3723343/figure/F3/" class="imagepopup">Figure 3. Cripto binding to cell surface GRP78 promotes stemness and the tumorigenic phenotype.  From: Cripto/GRP78 modulation of the TGF-? <span class="highlight" style="background-color:">pathway</span> in development and oncogenesis. </a></p><div class="supp"><p class="details"><div>(<b>A</b>) The cell surface Cripto/GRP78 complex interaction required for Cripto regulation of the Smad2/3 <span class="highlight" style="background-color:">pathway</span> both as a Nodal co-receptor and antagonist of activin and TGF-β <span class="highlight" style="background-color:">signaling</span>. Soluble Cripto also binds cell surface GRP78 to cause activation of PI3K and MAPK <span class="highlight" style="background-color:">signaling</span> via activation of Src. The effects of Cripto/GRP78 <span class="highlight" style="background-color:">signaling</span> on Src/MAPK/PI3K and Smad2/3 pathways are likely to function synergistically to promote stemness, plasticity and tumorigenesis. (<b>B</b>) The N-20 antibody blocks Cripto binding to cell surface GRP78 and inhibits Cripto-dependent <span class="highlight" style="background-color:">signaling</span> via MAPK, PI3K and Smad2/3 pathways. By blocking Cripto, this GRP78 antibody blocks Src/MAPK/PI3K <span class="highlight" style="background-color:">pathway</span> activation and enhances activin and TGF-β <span class="highlight" style="background-color:">signaling</span> thereby promoting cytostasis and tumor suppression.</div></p></div><div class="aux"><div class="resc">Peter C. Gray, et al. FEBS Lett. ;586(14):1836-1845.</div><p class="links"><a class="dblinks" href="/pubmed/22306319" ref="ordinalpos=562&amp;ncbi_uid=5126284&amp;link_uid=22306319">Citation</a><a class="dblinks" href="/pmc/articles/PMC3723343/" ref="ordinalpos=562&amp;ncbi_uid=5126284&amp;link_uid=PMC3723343">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>563.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3654583/figure/F1/" ref="ordinalpos=563&amp;ncbi_uid=4955693&amp;link_uid=PMC3654583" href="/pmc/articles/PMC3654583/"><img src="/pmc/articles/instance/3654583/bin/onci-2-e23566-g1.gif" src-large="/pmc/articles/instance/3654583/bin/onci-2-e23566-g1.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3654583/" ref="ordinalpos=563&amp;ncbi_uid=4955693&amp;link_uid=PMC3654583" image-link="/pmc/articles/PMC3654583/figure/F1/" class="imagepopup">Figure. From: The RIG-I/MAVS <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in cancer cell-selective apoptosis. </a></p><div class="supp"><p class="details"><div><b>Figure 1.</b> <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> for HVJ-E-mediated anti-tumor effects. Upon fusion with the plasma membrane, the hemagglutinating virus of Japan envelope (HVJ-E) introduces fragments of the viral RNA genome into the cytoplasm, which are recognized by retinoic acid-inducible gene I (RIG-I). The RIG-I/RNA complex associates with the mitochondrial antiviral <span class="highlight" style="background-color:">signaling</span> (MAVS) protein, which in turn promotes the activation of several transcription factors. In cells from the immune system, the RIG-I/MAVS <span class="highlight" style="background-color:">pathway</span> stimulate antitumor immunity via the production of some cytokines such as interferon (IFN)α, IFNβ and CXCL10. In cancer cells, this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> induces apoptosis upon the activation of pro-apoptotic factors, such as tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) and NOXA.</div></p></div><div class="aux"><div class="resc">Yasufumi Kaneda. Oncoimmunology. 2013 April 1;2(4):e23566.</div><p class="links"><a class="dblinks" href="/pubmed/23734313" ref="ordinalpos=563&amp;ncbi_uid=4955693&amp;link_uid=23734313">Citation</a><a class="dblinks" href="/pmc/articles/PMC3654583/" ref="ordinalpos=563&amp;ncbi_uid=4955693&amp;link_uid=PMC3654583">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>564.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3446243/figure/F2/" ref="ordinalpos=564&amp;ncbi_uid=4242478&amp;link_uid=PMC3446243" href="/pmc/articles/PMC3446243/"><img src="/pmc/articles/instance/3446243/bin/nihms388815f2.gif" src-large="/pmc/articles/instance/3446243/bin/nihms388815f2.jpg" alt="Fig. 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3446243/" ref="ordinalpos=564&amp;ncbi_uid=4242478&amp;link_uid=PMC3446243" image-link="/pmc/articles/PMC3446243/figure/F2/" class="imagepopup">Fig. 2.  From: Nuclear Factor ?B <span class="highlight" style="background-color:">Signaling</span> in Opioid Functions and Receptor Gene Expression. </a></p><div class="supp"><p class="details"><div>Proposed model for the involvement of NF-κB <span class="highlight" style="background-color:">signaling</span> in opioid receptor-expressing immune and neuronal cells. Arrows indicate activation of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>; the “⊣” signs indicate inhibition of the <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Both immune and neuronal cells express opioid receptors. Modulation of NF-κB <span class="highlight" style="background-color:">signaling</span> leads to the regulation of a variety of genes, including the opioid receptor genes and proinflammatory genes. Nitration or nitrosylation of NF-κB may be one of the feedback mechanisms of controlling homeostasis of NF-κB <span class="highlight" style="background-color:">signaling</span> <i>in vivo</i>. Deregulation of NF-κB <span class="highlight" style="background-color:">signaling</span> could result in opioid-mediated immunosuppression, cell survival, morphine tolerance, and physical dependence. MEKK, mitogen-activated protein kinase kinase; IκB, inhibitor of κB protein; p65, NF-κB/p65 subunit; p50, NF-κB/p50 subunit; iNO, inducible NO.</div></p></div><div class="aux"><div class="resc">Yulong L. Chen, et al. J Neuroimmune Pharmacol. 2006 September;1(3):270-279.</div><p class="links"><a class="dblinks" href="/pubmed/18040804" ref="ordinalpos=564&amp;ncbi_uid=4242478&amp;link_uid=18040804">Citation</a><a class="dblinks" href="/pmc/articles/PMC3446243/" ref="ordinalpos=564&amp;ncbi_uid=4242478&amp;link_uid=PMC3446243">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>565.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3920843/figure/fig4/" ref="ordinalpos=565&amp;ncbi_uid=5945339&amp;link_uid=PMC3920843" href="/pmc/articles/PMC3920843/"><img src="/pmc/articles/instance/3920843/bin/SCIENTIFICA2014-793815.004.gif" src-large="/pmc/articles/instance/3920843/bin/SCIENTIFICA2014-793815.004.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3920843/" ref="ordinalpos=565&amp;ncbi_uid=5945339&amp;link_uid=PMC3920843" image-link="/pmc/articles/PMC3920843/figure/fig4/" class="imagepopup">Figure 4.  From: Influenza Viral Manipulation of Sphingolipid Metabolism and <span class="highlight" style="background-color:">Signaling</span> to Modulate Host Defense System. </a></p><div class="supp"><p class="details"><div>NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its regulation by influenza virus. (a) The canonical <span class="highlight" style="background-color:">pathway</span> involves activation of the IKK complex, IKK-mediated I<i>κ</i>B<i>α</i> phosphorylation, and subsequent degradation, resulting in nuclear translocation of the heterodimer RelA(p65)/p50. (b) The noncanonical <span class="highlight" style="background-color:">pathway</span> is dependent on phosphorylation-induced p100 processing. This leads to activation/translocation of the RelB/p52 complex. (c) Influenza virus activates the NF-<i>κ</i>B <span class="highlight" style="background-color:">pathway</span> to facilitate the amplification of vRNAs and production of progeny viruses. Newly synthesized influenza viral proteins antagonize the activation of NF-<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> and inhibit the function of antiviral proteins or immunoregulatory molecules induced by the <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Madhuvanthi Vijayan, et al. Scientifica (Cairo). 2014;2014:793815.</div><p class="links"><a class="dblinks" href="/pubmed/24672735" ref="ordinalpos=565&amp;ncbi_uid=5945339&amp;link_uid=24672735">Citation</a><a class="dblinks" href="/pmc/articles/PMC3920843/" ref="ordinalpos=565&amp;ncbi_uid=5945339&amp;link_uid=PMC3920843">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>566.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3392173/figure/F1/" ref="ordinalpos=566&amp;ncbi_uid=4528833&amp;link_uid=PMC3392173" href="/pmc/articles/PMC3392173/"><img src="/pmc/articles/instance/3392173/bin/nihms-341562-f0001.gif" src-large="/pmc/articles/instance/3392173/bin/nihms-341562-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3392173/" ref="ordinalpos=566&amp;ncbi_uid=4528833&amp;link_uid=PMC3392173" image-link="/pmc/articles/PMC3392173/figure/F1/" class="imagepopup">Fig. 1.  From: Update on Wnt <span class="highlight" style="background-color:">signaling</span> in bone cell biology and bone disease. </a></p><div class="supp"><p class="details"><div>Wnt <span class="highlight" style="background-color:">signaling</span> pathways. The “canonical” Wnt–β-catenin <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is illustrated in the center of the diagram. Secreted and intracellular inhibitors of β-catenin are shown on the right side. Wnt <span class="highlight" style="background-color:">signaling</span> pathways that do not involve β-catenin are summarized on the left side.</div></p></div><div class="aux"><div class="resc">David G. Monroe, et al. Gene. ;492(1):1-18.</div><p class="links"><a class="dblinks" href="/pubmed/22079544" ref="ordinalpos=566&amp;ncbi_uid=4528833&amp;link_uid=22079544">Citation</a><a class="dblinks" href="/pmc/articles/PMC3392173/" ref="ordinalpos=566&amp;ncbi_uid=4528833&amp;link_uid=PMC3392173">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>567.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4020352/figure/F1/" ref="ordinalpos=567&amp;ncbi_uid=6132098&amp;link_uid=PMC4020352" href="/pmc/articles/PMC4020352/"><img src="/pmc/articles/instance/4020352/bin/nihms575521f1.gif" src-large="/pmc/articles/instance/4020352/bin/nihms575521f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4020352/" ref="ordinalpos=567&amp;ncbi_uid=6132098&amp;link_uid=PMC4020352" image-link="/pmc/articles/PMC4020352/figure/F1/" class="imagepopup">Figure 1. Activation of the autocrine CCN2–Smad1 <span class="highlight" style="background-color:">signaling</span> in scleroderma fibroblasts: a hypothetical model.  From: Noncanonical transforming growth factor ? <span class="highlight" style="background-color:">signaling</span> in scleroderma fibrosis. </a></p><div class="supp"><p class="details"><div>Activation of the Smad1 <span class="highlight" style="background-color:">pathway</span> requires cooperation between two ligands, transforming growth factor β (TGF-β) and CCN2, <span class="highlight" style="background-color:">signaling</span> through their cognate receptors. Elevated levels of endoglin [17] promote TGF-β <span class="highlight" style="background-color:">signaling</span> through a receptor complex also consisting of ALK5, ALK1, and TGF-β RII, which leads to induced phosphorylation of Smad1 [7,8]. Concurrent activation of ERK1/2 is required for Smad1 phosphorylation, although the specific role of ERK1/2 in this process has not yet been elucidated [8]. Binding of CCN2 to the α<sub>v</sub>β<sub>3</sub> integrin receptor activates ERK1/2 in a Src-dependent manner. As CCN2 has been shown to bind TGF-β and enhance TGF-β receptor binding and <span class="highlight" style="background-color:">signaling</span> [20], activation of Smad1/Erk1/2 <span class="highlight" style="background-color:">signaling</span> can occur in a spatially and temporally coordinated manner. Smad1 and Erk1/2 activate CCN2 gene expression leading to elevated CCN2 production, which then contributes to sustained activation of this <span class="highlight" style="background-color:">pathway</span>. Several components of this <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> are expressed at elevated levels in scleroderma (SSc) fibroblasts, including ALK5, endoglin, α<sub>v</sub>β<sub>3</sub> integrin, Erk1/2, Smad1, and CCN2.</div></p></div><div class="aux"><div class="resc">Maria Trojanowska. Curr Opin Rheumatol. 2009 November;21(6):623-629.</div><p class="links"><a class="dblinks" href="/pubmed/19713852" ref="ordinalpos=567&amp;ncbi_uid=6132098&amp;link_uid=19713852">Citation</a><a class="dblinks" href="/pmc/articles/PMC4020352/" ref="ordinalpos=567&amp;ncbi_uid=6132098&amp;link_uid=PMC4020352">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>568.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1347046/figure/f9/" ref="ordinalpos=568&amp;ncbi_uid=741040&amp;link_uid=PMC1347046" href="/pmc/articles/PMC1347046/"><img src="/pmc/articles/instance/1347046/bin/zmb0030656850009.gif" src-large="/pmc/articles/instance/1347046/bin/zmb0030656850009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1347046/" ref="ordinalpos=568&amp;ncbi_uid=741040&amp;link_uid=PMC1347046" image-link="/pmc/articles/PMC1347046/figure/f9/" class="imagepopup">FIG. 9. From: The RA Domain of Ste50 Adaptor Protein Is Required for Delivery of Ste11 to the Plasma Membrane in the Filamentous Growth <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span> of the Yeast Saccharomyces cerevisiae . </a></p><div class="supp"><p class="details"><div>Schematic depiction of the proposed function of the Cdc42-Ste50-RA domain and Ste11-Ste50-SAM domain interactions in filamentous growth and HOG <span class="highlight" style="background-color:">pathway</span> <span class="highlight" style="background-color:">signaling</span>. See text for further explanation.</div></p></div><div class="aux"><div class="resc">Dagmar M. Truckses, et al. Mol Cell Biol. 2006 February;26(3):912-928.</div><p class="links"><a class="dblinks" href="/pubmed/16428446" ref="ordinalpos=568&amp;ncbi_uid=741040&amp;link_uid=16428446">Citation</a><a class="dblinks" href="/pmc/articles/PMC1347046/" ref="ordinalpos=568&amp;ncbi_uid=741040&amp;link_uid=PMC1347046">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>569.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2679364/figure/F2/" ref="ordinalpos=569&amp;ncbi_uid=1840971&amp;link_uid=PMC2679364" href="/pmc/articles/PMC2679364/"><img src="/pmc/articles/instance/2679364/bin/nihms85735f2.gif" src-large="/pmc/articles/instance/2679364/bin/nihms85735f2.jpg" alt="Figure 2" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2679364/" ref="ordinalpos=569&amp;ncbi_uid=1840971&amp;link_uid=PMC2679364" image-link="/pmc/articles/PMC2679364/figure/F2/" class="imagepopup">Figure 2. Schematic illustration of the RAS-RAF-MEK-ERK (MAPK) <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.  From: OVARIAN CANCER. </a></p><div class="supp"><p class="details"><div>This cell <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> is important for the cellular response to a variety of growth and differentiation factors. Aberration of this <span class="highlight" style="background-color:">pathway</span> in ovarian SBTs and low-grade serous carcinomas is mainly due to activating mutations of <i>KRAS</i> and <i>BRAF</i>, which result in constitutive activation of MAPK-mediated <span class="highlight" style="background-color:">signaling</span> in these tumors. Activated MAPK <span class="highlight" style="background-color:">signaling</span> alters expression of downstream target genes, including up-regulation of cyclin D1. Several MEK small compound inhibitors such as CI-1040 have been generated. These inhibitors effectively suppress MAPK activation and hold promise for treatment of patients with advanced stage low-grade serous carcinomas.</div></p></div><div class="aux"><div class="resc">Kathleen R. Cho, et al. Annu Rev Pathol. ;4:287-313.</div><p class="links"><a class="dblinks" href="/pubmed/18842102" ref="ordinalpos=569&amp;ncbi_uid=1840971&amp;link_uid=18842102">Citation</a><a class="dblinks" href="/pmc/articles/PMC2679364/" ref="ordinalpos=569&amp;ncbi_uid=1840971&amp;link_uid=PMC2679364">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>570.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3040109/figure/F1/" ref="ordinalpos=570&amp;ncbi_uid=2770889&amp;link_uid=PMC3040109" href="/pmc/articles/PMC3040109/"><img src="/pmc/articles/instance/3040109/bin/nihms268825f1.gif" src-large="/pmc/articles/instance/3040109/bin/nihms268825f1.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3040109/" ref="ordinalpos=570&amp;ncbi_uid=2770889&amp;link_uid=PMC3040109" image-link="/pmc/articles/PMC3040109/figure/F1/" class="imagepopup">Fig. 1.  From: T Cell Receptor <span class="highlight" style="background-color:">Signaling</span> Kinetics Takes the Stage. </a></p><div class="supp"><p class="details"><div>The kinetics of <span class="highlight" style="background-color:">signaling</span> downstream of the TCR influences the balance between the long and the short pathways of granule delivery. Engagement of the TCR in more effective CTLs results in rapid and robust proximal TCR <span class="highlight" style="background-color:">signaling</span>, as opposed to the slower proximal <span class="highlight" style="background-color:">signaling</span> that is induced in less effective CTLs. In contrast to weak <span class="highlight" style="background-color:">signaling</span>, strong <span class="highlight" style="background-color:">signaling</span> is initiated by a greater number of activated <span class="highlight" style="background-color:">signaling</span> proteins per microcluster and leads to rapid downstream <span class="highlight" style="background-color:">signaling</span> kinetics, as determined by measurement of the rate of intracellular Ca<sup>2+</sup> mobilization (10). Increasing the concentration of intracellular Ca<sup>2+</sup> results in the activation of downstream <span class="highlight" style="background-color:">signaling</span> that regulates the velocity of granule movement to the MTOC. In contrast, translocation of the MTOC is independent of Ca<sup>2+</sup> and is mediated by DAG-dependent <span class="highlight" style="background-color:">signaling</span> (21, 34). This <span class="highlight" style="background-color:">signaling</span> dichotomy accounts for the mechanism that controls the two pathways of granule delivery to the secretory domain. A faster <span class="highlight" style="background-color:">signaling</span> kinetics stimulates the swift movement of granules toward the MTOC, and granules become concentrated near the MTOC before its polarization. The MTOC then directly delivers granules to the secretory domain, which is the short path of delivery. Slower <span class="highlight" style="background-color:">signaling</span> kinetics results in delayed recruitment of granules, which move along microtubules to the periphery of the synapse and then loop through the pSMAC to fuse at the outer edge of the secretory domain; this is the longer <span class="highlight" style="background-color:">pathway</span>. Both pathways can operate in a single cell, and the balance between them is regulated by the kinetics of TCR-mediated <span class="highlight" style="background-color:">signaling</span>. (See http://stke.sciencemag.org/cgi/content/full/3/153/pe50/DC1 for Animations 1 and 2.)</div></p></div><div class="aux"><div class="resc">Yuri Sykulev. Sci Signal. ;3(153):pe50-pe50.</div><p class="links"><a class="dblinks" href="/pubmed/21177491" ref="ordinalpos=570&amp;ncbi_uid=2770889&amp;link_uid=21177491">Citation</a><a class="dblinks" href="/pmc/articles/PMC3040109/" ref="ordinalpos=570&amp;ncbi_uid=2770889&amp;link_uid=PMC3040109">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>571.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3579607/figure/F1/" ref="ordinalpos=571&amp;ncbi_uid=5755833&amp;link_uid=PMC3579607" href="/pmc/articles/PMC3579607/"><img src="/pmc/articles/instance/3579607/bin/onc0011311900001.gif" src-large="/pmc/articles/instance/3579607/bin/onc0011311900001.jpg" alt="Figure 1." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3579607/" ref="ordinalpos=571&amp;ncbi_uid=5755833&amp;link_uid=PMC3579607" image-link="/pmc/articles/PMC3579607/figure/F1/" class="imagepopup">Figure 1. From: The Potential Role of Aerobic Exercise to Modulate Cardiotoxicity of Molecularly Targeted Cancer Therapeutics. </a></p><div class="supp"><p class="details"><div>Mechanisms underlying HER2-directed therapy cardiotoxicity. Inhibition of ErbB receptors with HER2-directed therapies impacts numerous <span class="highlight" style="background-color:">signaling</span> pathways resulting in suppression of myofilament protein synthesis via the PI3K-Akt <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> A), suppression of protein hypertrophy via the MAPK <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> B), suppression of cell survival via Src/Fak <span class="highlight" style="background-color:">pathway</span> (<span class="highlight" style="background-color:">pathway</span> C), suppression of myofilament protein synthesis and upregulation of protein degradation via TGF-β1 and C/EBPβ <span class="highlight" style="background-color:">signaling</span> (<span class="highlight" style="background-color:">pathway</span> D), and alterations in cardiac energy metabolism via downregulation of AMPK (<span class="highlight" style="background-color:">pathway</span> E).<br />Abbreviations: AMPK, AMP-activated protein kinase; C/EBPβ, CCAAT/enhancer binding protein; Fak, focal adhesion kinase; MAPK, mitogen-activated protein kinase; Nrg1, Neuregulin-1β; PI3K, phosphatidylinositol 3-kinase; Smad, small mother against decapentaplegic; TGFβ, transforming growth factor β.</div></p></div><div class="aux"><div class="resc">Jessica M. Scott, et al. Oncologist. 2013 February;18(2):221-231.</div><p class="links"><a class="dblinks" href="/pubmed/23335619" ref="ordinalpos=571&amp;ncbi_uid=5755833&amp;link_uid=23335619">Citation</a><a class="dblinks" href="/pmc/articles/PMC3579607/" ref="ordinalpos=571&amp;ncbi_uid=5755833&amp;link_uid=PMC3579607">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>572.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3816662/figure/pharmaceuticals-05-01236-f001/" ref="ordinalpos=572&amp;ncbi_uid=5486604&amp;link_uid=PMC3816662" href="/pmc/articles/PMC3816662/"><img src="/pmc/articles/instance/3816662/bin/pharmaceuticals-05-01236-g001.gif" src-large="/pmc/articles/instance/3816662/bin/pharmaceuticals-05-01236-g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3816662/" ref="ordinalpos=572&amp;ncbi_uid=5486604&amp;link_uid=PMC3816662" image-link="/pmc/articles/PMC3816662/figure/pharmaceuticals-05-01236-f001/" class="imagepopup">Figure 1.  From: The Biological Role of PI3K <span class="highlight" style="background-color:">Pathway</span> in Lung Cancer. </a></p><div class="supp"><p class="details"><div>The PI3K/Akt/mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Evangelos G. Sarris, et al. Pharmaceuticals (Basel). 2012 November;5(11):1236-1264.</div><p class="links"><a class="dblinks" href="/pubmed/24281308" ref="ordinalpos=572&amp;ncbi_uid=5486604&amp;link_uid=24281308">Citation</a><a class="dblinks" href="/pmc/articles/PMC3816662/" ref="ordinalpos=572&amp;ncbi_uid=5486604&amp;link_uid=PMC3816662">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>573.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2543107/figure/pcbi-1000197-g001/" ref="ordinalpos=573&amp;ncbi_uid=1479000&amp;link_uid=PMC2543107" href="/pmc/articles/PMC2543107/"><img src="/pmc/articles/instance/2543107/bin/pcbi.1000197.g001.gif" src-large="/pmc/articles/instance/2543107/bin/pcbi.1000197.g001.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2543107/" ref="ordinalpos=573&amp;ncbi_uid=1479000&amp;link_uid=PMC2543107" image-link="/pmc/articles/PMC2543107/figure/pcbi-1000197-g001/" class="imagepopup">Figure 1. Dose response in <span class="highlight" style="background-color:">signaling</span> pathways..  From: Dose-to-Duration Encoding and <span class="highlight" style="background-color:">Signaling</span> beyond Saturation in Intracellular <span class="highlight" style="background-color:">Signaling</span> Networks. </a></p><div class="supp"><p class="details"><div>(A) A linear <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> (L: ligand, R: receptor, A–C: <span class="highlight" style="background-color:">signaling</span> molecules, red dots: phosphate groups). (B) Dose response curves for a linear <span class="highlight" style="background-color:">pathway</span>. Saturation of a downstream <span class="highlight" style="background-color:">pathway</span> component leads to a shift of the physiological response curve (purple) to the left of the receptor occupancy curve (dashed black). Sensitivity at low stimulus doses is increased but the dynamic range (gray area) is reduced. (C) Dose response curves for a nonlinear <span class="highlight" style="background-color:">pathway</span>. Feedback or feed forward regulation can extend the dynamic range of the system (purple curve) and produce responses with an EC<sub>50</sub> value displaced to the right of the receptor K<sub>d</sub>.</div></p></div><div class="aux"><div class="resc">Marcelo Behar, et al. PLoS Comput Biol. 2008 October;4(10):e1000197.</div><p class="links"><a class="dblinks" href="/pubmed/18846202" ref="ordinalpos=573&amp;ncbi_uid=1479000&amp;link_uid=18846202">Citation</a><a class="dblinks" href="/pmc/articles/PMC2543107/" ref="ordinalpos=573&amp;ncbi_uid=1479000&amp;link_uid=PMC2543107">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>574.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4273316/figure/Fig6/" ref="ordinalpos=574&amp;ncbi_uid=6989566&amp;link_uid=PMC4273316" href="/pmc/articles/PMC4273316/"><img src="/pmc/articles/instance/4273316/bin/12967_2014_330_Fig6_HTML.gif" src-large="/pmc/articles/instance/4273316/bin/12967_2014_330_Fig6_HTML.jpg" alt="Figure 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4273316/" ref="ordinalpos=574&amp;ncbi_uid=6989566&amp;link_uid=PMC4273316" image-link="/pmc/articles/PMC4273316/figure/Fig6/" class="imagepopup">Figure 6.  From: Exogenous IFN-beta regulates the RANKL-c-Fos-IFN-beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in the collagen antibody-induced arthritis model. </a></p><div class="supp"><p class="details"><div> <b>Changes in the RANKL-RANK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> after exogenous IFN-β treatment in the CAIA model mice.</b> The levels of RANKL <b>(A)</b>, TRAF6 <b>(B)</b>, c-Fos <b>(C),</b> and NFATc-1 <b>(D)</b> in the joints of mice in the IFN-β intervention and non-intervention groups. *: <i>P</i> &lt;0.05.</div></p></div><div class="aux"><div class="resc">Rong Zhao, et al. J Transl Med. 2014;12(1):330.</div><p class="links"><a class="dblinks" href="/pubmed/25491303" ref="ordinalpos=574&amp;ncbi_uid=6989566&amp;link_uid=25491303">Citation</a><a class="dblinks" href="/pmc/articles/PMC4273316/" ref="ordinalpos=574&amp;ncbi_uid=6989566&amp;link_uid=PMC4273316">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>575.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4047857/figure/fig5/" ref="ordinalpos=575&amp;ncbi_uid=6241507&amp;link_uid=PMC4047857" href="/pmc/articles/PMC4047857/"><img src="/pmc/articles/instance/4047857/bin/cddis2014222f5.gif" src-large="/pmc/articles/instance/4047857/bin/cddis2014222f5.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4047857/" ref="ordinalpos=575&amp;ncbi_uid=6241507&amp;link_uid=PMC4047857" image-link="/pmc/articles/PMC4047857/figure/fig5/" class="imagepopup">Figure 5.  From: SPAG5 upregulation predicts poor prognosis in cervical cancer patients and alters sensitivity to taxol treatment via the mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>SPAG5 regulation influenced mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation in cervical cancer cell lines under taxol treatment. Taxol treatment after SPAG5 downregulation in (<b>a</b>) HeLa and (<b>b</b>) SiHa cervical cancer cell lines revealing differing drug sensitivities under different concentrations due to (<b>c</b>) apoptosis. (<b>d</b>) Western blot analysis of mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> activation</div></p></div><div class="aux"><div class="resc">L-J Yuan, et al. Cell Death Dis. 2014 May;5(5):e1247.</div><p class="links"><a class="dblinks" href="/pubmed/24853425" ref="ordinalpos=575&amp;ncbi_uid=6241507&amp;link_uid=24853425">Citation</a><a class="dblinks" href="/pmc/articles/PMC4047857/" ref="ordinalpos=575&amp;ncbi_uid=6241507&amp;link_uid=PMC4047857">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>576.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3573097/figure/pgen-1003253-g007/" ref="ordinalpos=576&amp;ncbi_uid=4674652&amp;link_uid=PMC3573097" href="/pmc/articles/PMC3573097/"><img src="/pmc/articles/instance/3573097/bin/pgen.1003253.g007.gif" src-large="/pmc/articles/instance/3573097/bin/pgen.1003253.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3573097/" ref="ordinalpos=576&amp;ncbi_uid=4674652&amp;link_uid=PMC3573097" image-link="/pmc/articles/PMC3573097/figure/pgen-1003253-g007/" class="imagepopup">Figure 7. RIOK1 and RIOK2 are required for EGFR- and PI3K-mediated tumorigenesis..  From: A Kinome-Wide RNAi Screen in Drosophila Glia Reveals That the RIO Kinases Mediate Cell Proliferation and Survival through TORC2-Akt <span class="highlight" style="background-color:">Signaling</span> in Glioblastoma. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Pathway</span> diagram placing RIOK1 and RIOK2 in relation to Akt downstream of EGFR and PI3K <span class="highlight" style="background-color:">signaling</span> in GBM. RIOK2 mediates <span class="highlight" style="background-color:">signaling</span> both upstream and downstream of Akt via stimulation of TORC2 (left). Loss of RIOK1 or RIOK2 reduces Akt <span class="highlight" style="background-color:">signaling</span> downstream of oncogenic EGFR and PI3K <span class="highlight" style="background-color:">signaling</span>, and induces the p53-dependent ribosomal stress checkpoint via RpL11 (right).</div></p></div><div class="aux"><div class="resc">Renee D. Read, et al. PLoS Genet. 2013 February;9(2):e1003253.</div><p class="links"><a class="dblinks" href="/pubmed/23459592" ref="ordinalpos=576&amp;ncbi_uid=4674652&amp;link_uid=23459592">Citation</a><a class="dblinks" href="/pmc/articles/PMC3573097/" ref="ordinalpos=576&amp;ncbi_uid=4674652&amp;link_uid=PMC3573097">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>577.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3855069/figure/F1/" ref="ordinalpos=577&amp;ncbi_uid=6502996&amp;link_uid=PMC3855069" href="/pmc/articles/PMC3855069/"><img src="/pmc/articles/instance/3855069/bin/nihms512984f1.gif" src-large="/pmc/articles/instance/3855069/bin/nihms512984f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3855069/" ref="ordinalpos=577&amp;ncbi_uid=6502996&amp;link_uid=PMC3855069" image-link="/pmc/articles/PMC3855069/figure/F1/" class="imagepopup">Figure 1. IL-11 <span class="highlight" style="background-color:">signaling</span> promotes gastrointestinal tumorigenesis.  From: IL-11: a prominent pro-tumorigenic member of IL-6 family. </a></p><div class="supp"><p class="details"><div><b>A)</b>Putoczki et al. study demonstrates the dominant importance of IL-11-gp130 <span class="highlight" style="background-color:">signaling</span> in models of CAC, CRC and GC. When models with the wild type gp130 are used, both IL-6 and IL-11 are required for tumor development. When models with the gp130<sup>Y757F</sup> mutant are used (indicated with a ‘red star’), the tumorigenesis is solely dependent on IL-11. IL-11-gp130 <span class="highlight" style="background-color:">signaling</span> is required for enhanced cell proliferation and resistance to apoptosis.<br /><b>B)</b>Although STAT3 <span class="highlight" style="background-color:">pathway</span> is important to mediate effects of IL-11 <span class="highlight" style="background-color:">signaling</span>, gp130 is known to also regulate the mTORC1 <span class="highlight" style="background-color:">pathway</span> and the Ras/ERK <span class="highlight" style="background-color:">pathway</span>. The contribution of the mTORC1 <span class="highlight" style="background-color:">pathway</span> in GC and in CAC has been previously demonstrated, IL-6 and IL-11 mostly activate an overlapping set of genes in cancer, with only a few of them being more specific for IL-6 or for IL-11.</div></p></div><div class="aux"><div class="resc">Sergei I. Grivennikov. Cancer Cell. ;24(2):10.1016/j.ccr.2013.07.018.</div><p class="links"><a class="dblinks" href="/pubmed/23948295" ref="ordinalpos=577&amp;ncbi_uid=6502996&amp;link_uid=23948295">Citation</a><a class="dblinks" href="/pmc/articles/PMC3855069/" ref="ordinalpos=577&amp;ncbi_uid=6502996&amp;link_uid=PMC3855069">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>578.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3858325/figure/F4/" ref="ordinalpos=578&amp;ncbi_uid=6331648&amp;link_uid=PMC3858325" href="/pmc/articles/PMC3858325/"><img src="/pmc/articles/instance/3858325/bin/nihms473916f4.gif" src-large="/pmc/articles/instance/3858325/bin/nihms473916f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3858325/" ref="ordinalpos=578&amp;ncbi_uid=6331648&amp;link_uid=PMC3858325" image-link="/pmc/articles/PMC3858325/figure/F4/" class="imagepopup">Fig. 4.  From: Integrative genomic characterization of oral squamous cell carcinomaidentifies frequent somatic drivers. </a></p><div class="supp"><p class="details"><div>Driver pathways and key genes in OSCC. A) <span class="highlight" style="background-color:">Pathway</span> alteration in each sample is indicated by a black box. Sample IDs are shown at the top and <span class="highlight" style="background-color:">pathway</span> names at the left. Selected clinical variables and clusters from Fig. 3 are indicated with colored boxes. B) Notch <span class="highlight" style="background-color:">pathway</span>. C) Mitogenic <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. D) Cell cycle <span class="highlight" style="background-color:">pathway</span>. E) TP53 <span class="highlight" style="background-color:">pathway</span>. <span class="highlight" style="background-color:">Pathway</span> diagrams indicate the genes included in each <span class="highlight" style="background-color:">pathway</span>. Copy gains are indicated with red, copy losses with blue, mutations with black and methylation with purple. Percentages shown indicate the frequency of that event.</div></p></div><div class="aux"><div class="resc">Curtis R. Pickering, et al. Cancer Discov. ;3(7):10.1158/2159-8290.CD-12-0537.</div><p class="links"><a class="dblinks" href="/pubmed/23619168" ref="ordinalpos=578&amp;ncbi_uid=6331648&amp;link_uid=23619168">Citation</a><a class="dblinks" href="/pmc/articles/PMC3858325/" ref="ordinalpos=578&amp;ncbi_uid=6331648&amp;link_uid=PMC3858325">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>579.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g007/" ref="ordinalpos=579&amp;ncbi_uid=1718054&amp;link_uid=PMC2634968" href="/pmc/articles/PMC2634968/"><img src="/pmc/articles/instance/2634968/bin/pcbi.1000292.g007.gif" src-large="/pmc/articles/instance/2634968/bin/pcbi.1000292.g007.jpg" alt="Figure 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2634968/" ref="ordinalpos=579&amp;ncbi_uid=1718054&amp;link_uid=PMC2634968" image-link="/pmc/articles/PMC2634968/figure/pcbi-1000292-g007/" class="imagepopup">Figure 7. A simplified depiction of the MyD88 DIOS <span class="highlight" style="background-color:">pathway</span>..  From: Identification of Potential <span class="highlight" style="background-color:">Pathway</span> Mediation Targets in Toll-like Receptor <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The two critical network reactions <i>MyD88 dimerization</i> and <i>TRAF6/IRAK1 ubiquitination</i> are highlighted in red. Formation of the MyD88 homodimer favors recruitment of IRAK1 into a complex with TRAF6 [65]. The MyD88 dimer then dissociated from this complex to be either degraded or reused. The second critical network reaction, <i>TRAF6/IRAK1 ubiquitination</i>, occurred via the ubiquitin-conjugating enzymes Ubc13 and Uev1A, and was necessary for activation of NF-kappa-B and AP-1 through canonical IKK phosphorylation. Either of the two critical network reactions could completely abrogate the flux through the MyD88 <span class="highlight" style="background-color:">pathway</span> even though the TIR- or TIRAP-dependent TLR <span class="highlight" style="background-color:">signaling</span> was almost always active.</div></p></div><div class="aux"><div class="resc">Fan Li, et al. PLoS Comput Biol. 2009 February;5(2):e1000292.</div><p class="links"><a class="dblinks" href="/pubmed/19229310" ref="ordinalpos=579&amp;ncbi_uid=1718054&amp;link_uid=19229310">Citation</a><a class="dblinks" href="/pmc/articles/PMC2634968/" ref="ordinalpos=579&amp;ncbi_uid=1718054&amp;link_uid=PMC2634968">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>580.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2801866/figure/F4/" ref="ordinalpos=580&amp;ncbi_uid=2160271&amp;link_uid=PMC2801866" href="/pmc/articles/PMC2801866/"><img src="/pmc/articles/instance/2801866/bin/nihms162154f4.gif" src-large="/pmc/articles/instance/2801866/bin/nihms162154f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2801866/" ref="ordinalpos=580&amp;ncbi_uid=2160271&amp;link_uid=PMC2801866" image-link="/pmc/articles/PMC2801866/figure/F4/" class="imagepopup">Figure 4.  From: Molecular Regulation of Tumor Angiogenesis and Perfusion via Redox <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>The NO/cGMP <span class="highlight" style="background-color:">pathway</span> is a <span class="highlight" style="background-color:">signaling</span> node for pro- and anti-angiogenic <span class="highlight" style="background-color:">signaling</span>. In addition to VEGF, angiopoietin-1, estrogens, and insulin can activate eNOS to increase NO synthesis in endothelial cells. TSP1 <span class="highlight" style="background-color:">signaling</span> via CD36 and CD47 inhibits downstream elements of this <span class="highlight" style="background-color:">pathway</span>. Oxidized LDL (oxLDL) is another known ligand of CD36 that could inhibit angiogenesis via an overlapping <span class="highlight" style="background-color:">pathway</span>. In addition, oxLDL <span class="highlight" style="background-color:">signaling</span> inhibits protein kinase Cα, which phosphorylates eNOS at Thr<sup>495</sup> and activates NADPH oxidase (NOX), and the resulting superoxide depletes NO levels. The angiogenesis inhibitors endostatin and vasoinhibins activate the phosphatase PP2A, which dephosphorylates and thereby inactivates eNOS.</div></p></div><div class="aux"><div class="resc">Thomas W. Miller, et al. Chem Rev. ;109(7):3099-3124.</div><p class="links"><a class="dblinks" href="/pubmed/19374334" ref="ordinalpos=580&amp;ncbi_uid=2160271&amp;link_uid=19374334">Citation</a><a class="dblinks" href="/pmc/articles/PMC2801866/" ref="ordinalpos=580&amp;ncbi_uid=2160271&amp;link_uid=PMC2801866">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>581.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3938851/figure/Fig1/" ref="ordinalpos=581&amp;ncbi_uid=5893178&amp;link_uid=PMC3938851" href="/pmc/articles/PMC3938851/"><img src="/pmc/articles/instance/3973063/bin/13238_2013_2_Fig1_HTML.gif" src-large="/pmc/articles/instance/3973063/bin/13238_2013_2_Fig1_HTML.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3938851/" ref="ordinalpos=581&amp;ncbi_uid=5893178&amp;link_uid=PMC3938851" image-link="/pmc/articles/PMC3938851/figure/Fig1/" class="imagepopup">Figure 1.  From: Targeting tissue-specific metabolic <span class="highlight" style="background-color:">signaling</span> pathways in aging: the promise and limitations. </a></p><div class="supp"><p class="details"><div>Insulin/IGF-1 <span class="highlight" style="background-color:">signaling</span> and regulation. The binding of insulin or IGF-1 to the membrane receptors leads to the activation of the PI 3-kinase/Akt <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and subsequent downstream events such as phosphorylation of FOXO1 and activation of the mTORC1 <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, which regulate many important cellular events such as mRNA translation, protein synthesis, cell cycle progression, metabolism, and aging</div></p></div><div class="aux"><div class="resc">Fang Hu, et al. Protein Cell. 2014;5(1):21-35.</div><p class="links"><a class="dblinks" href="/pubmed/24474199" ref="ordinalpos=581&amp;ncbi_uid=5893178&amp;link_uid=24474199">Citation</a><a class="dblinks" href="/pmc/articles/PMC3938851/" ref="ordinalpos=581&amp;ncbi_uid=5893178&amp;link_uid=PMC3938851">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>582.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2657416/figure/F6/" ref="ordinalpos=582&amp;ncbi_uid=1699662&amp;link_uid=PMC2657416" href="/pmc/articles/PMC2657416/"><img src="/pmc/articles/instance/2657416/bin/zpq9990970160006.gif" src-large="/pmc/articles/instance/2657416/bin/zpq9990970160006.jpg" alt="Fig. 6." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2657416/" ref="ordinalpos=582&amp;ncbi_uid=1699662&amp;link_uid=PMC2657416" image-link="/pmc/articles/PMC2657416/figure/F6/" class="imagepopup">Fig. 6. From: The primary <span class="highlight" style="background-color:">signaling</span> outputs of brassinosteroids are regulated by abscisic acid <span class="highlight" style="background-color:">signaling</span>. </a></p><div class="supp"><p class="details"><div>A proposed model for the effect of ABA on BR <span class="highlight" style="background-color:">signaling</span> outputs. For BR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>, BR binding to BRI1 activates BRI1 complex via releasing plasma membrane association of BKI1, association with BAK1, and phosphorylation of BSKs. Activated BRI1 complex likely inhibits BIN2 kinase through yet an unknown mechanism, leading to the dephosphorylation of BES1, which actively regulates BR-responsive gene expression. ABA could regulates unknown components downstream of BRI1, but upstream of BIN2 in the BR <span class="highlight" style="background-color:">signaling</span>, which then lead to the activation of BIN2 through an undiscovered mechanism. Gain-of-function mutation of ABI1 and ABI2 in the ABA early <span class="highlight" style="background-color:">signaling</span> can inhibit the effect of ABA on BR <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">Shanshan Zhang, et al. Proc Natl Acad Sci U S A. 2009 March 17;106(11):4543-4548.</div><p class="links"><a class="dblinks" href="/pubmed/19240210" ref="ordinalpos=582&amp;ncbi_uid=1699662&amp;link_uid=19240210">Citation</a><a class="dblinks" href="/pmc/articles/PMC2657416/" ref="ordinalpos=582&amp;ncbi_uid=1699662&amp;link_uid=PMC2657416">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>583.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3889250/figure/Fig1/" ref="ordinalpos=583&amp;ncbi_uid=5705481&amp;link_uid=PMC3889250" href="/pmc/articles/PMC3889250/"><img src="/pmc/articles/instance/3889250/bin/12079_2013_200_Fig1_HTML.gif" src-large="/pmc/articles/instance/3889250/bin/12079_2013_200_Fig1_HTML.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3889250/" ref="ordinalpos=583&amp;ncbi_uid=5705481&amp;link_uid=PMC3889250" image-link="/pmc/articles/PMC3889250/figure/Fig1/" class="imagepopup">Fig. 1.  From: A network map of BDNF/TRKB and BDNF/p75NTR <span class="highlight" style="background-color:">signaling</span> system. </a></p><div class="supp"><p class="details"><div>Schematic representation of the BDNF <span class="highlight" style="background-color:">signaling</span> network. The BDNF <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> map contains 129 molecules for which the PPIs, PTMs, translocation and activation/inhibition reactions are experimentally proven in BDNF <span class="highlight" style="background-color:">signaling</span>. The major <span class="highlight" style="background-color:">signaling</span> modules activated by BDNF include PI3K/AKT, RAS/ERK, PLC/PKC, AMPK/ACC and JAK/STAT pathways. These <span class="highlight" style="background-color:">signaling</span> events in neurons lead to various context-specific processes such as growth, cell proliferation, differentiation, maintenance of synaptic plasticity, microtubule assembly, fat metabolism, protection, survival, calcification, production of nitric oxide and apoptosis. This <span class="highlight" style="background-color:">pathway</span> map is made publicly available at (http://www.wikipathways.org/index.php/<span class="highlight" style="background-color:">Pathway</span>:WP2380). A high confidence version of this map is available at http://www.netpath.org/netslim/BDNF_<span class="highlight" style="background-color:">pathway</span>.html </div></p></div><div class="aux"><div class="resc">Varot K. Sandhya, et al. J Cell Commun Signal. 2013 December;7(4):301-307.</div><p class="links"><a class="dblinks" href="/pubmed/23606317" ref="ordinalpos=583&amp;ncbi_uid=5705481&amp;link_uid=23606317">Citation</a><a class="dblinks" href="/pmc/articles/PMC3889250/" ref="ordinalpos=583&amp;ncbi_uid=5705481&amp;link_uid=PMC3889250">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>584.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2838237/figure/F1/" ref="ordinalpos=584&amp;ncbi_uid=2236152&amp;link_uid=PMC2838237" href="/pmc/articles/PMC2838237/"><img src="/pmc/articles/instance/2838237/bin/nihms-174338-f0001.gif" src-large="/pmc/articles/instance/2838237/bin/nihms-174338-f0001.jpg" alt="Fig. 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2838237/" ref="ordinalpos=584&amp;ncbi_uid=2236152&amp;link_uid=PMC2838237" image-link="/pmc/articles/PMC2838237/figure/F1/" class="imagepopup">Fig. 1.  From: THERAPEUTIC TARGETING OF "DARPP-32": A KEY <span class="highlight" style="background-color:">SIGNALING</span> MOLECULE IN THE DOPIMINERGIC <span class="highlight" style="background-color:">PATHWAY</span> FOR THE TREATMENT OF OPIATE ADDICTION. </a></p><div class="supp"><p class="details"><div>Schematic of a <span class="highlight" style="background-color:">signaling</span> cascade activated by opiates. Opiates via the <i>μ</i>-opioid receptor activates the dopiminergic <span class="highlight" style="background-color:">pathway</span> leading to the increased DARPP-32 phosphorylation at Thr34 and increased PP-1 inhibition which modulates ERK activation and consequently the expression of the downstream transcriptional molecule CREB.</div></p></div><div class="aux"><div class="resc">Supriya D. Mahajan, et al. Int Rev Neurobiol. ;88:199-222.</div><p class="links"><a class="dblinks" href="/pubmed/19897079" ref="ordinalpos=584&amp;ncbi_uid=2236152&amp;link_uid=19897079">Citation</a><a class="dblinks" href="/pmc/articles/PMC2838237/" ref="ordinalpos=584&amp;ncbi_uid=2236152&amp;link_uid=PMC2838237">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>585.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2174844/figure/F8/" ref="ordinalpos=585&amp;ncbi_uid=1160380&amp;link_uid=PMC2174844" href="/pmc/articles/PMC2174844/"><img src="/pmc/articles/instance/2174844/bin/JCB9910150.f8.gif" src-large="/pmc/articles/instance/2174844/bin/JCB9910150.f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2174844/" ref="ordinalpos=585&amp;ncbi_uid=1160380&amp;link_uid=PMC2174844" image-link="/pmc/articles/PMC2174844/figure/F8/" class="imagepopup">Figure 8.  From: Matrix Survival <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>Model of the FN-dependent matrix survival <span class="highlight" style="background-color:">pathway</span> activated in primary RSF when serum is absent. FN supports the organization of a pro-survival <span class="highlight" style="background-color:">signaling</span> complex based on a FAK/Cas scaffold located at focal adhesion sites. Paxillin (Pax) is thought to participate in the binding of FAK to focal adhesion sites (Tachibana et al. 1995). Cas is recruited to the survival complex via a FAK PR-1–Cas SH3 interaction. Matrix survival <span class="highlight" style="background-color:">signaling</span> also requires an intact Cas SD. FN-FAK–associated survival signals activate Ras, and are propagated further via a Rac1/Pak1/MKK4/JNK <span class="highlight" style="background-color:">signaling</span> module, with pJNK detected in focal contacts as well as in the nucleus. This <span class="highlight" style="background-color:">pathway</span> is distinct from the serum-dependent survival <span class="highlight" style="background-color:">pathway</span> that is activated when matrix is withdrawn. The latter <span class="highlight" style="background-color:">pathway</span> requires FAK, but it also requires PI3-kinase, whereas the FN-FAK <span class="highlight" style="background-color:">pathway</span> described herein does not.</div></p></div><div class="aux"><div class="resc">Eduardo A.C. Almeida, et al. J Cell Biol. 2000 May 1;149(3):741-754.</div><p class="links"><a class="dblinks" href="/pubmed/10791986" ref="ordinalpos=585&amp;ncbi_uid=1160380&amp;link_uid=10791986">Citation</a><a class="dblinks" href="/pmc/articles/PMC2174844/" ref="ordinalpos=585&amp;ncbi_uid=1160380&amp;link_uid=PMC2174844">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>586.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3201973/figure/pone-0026766-g009/" ref="ordinalpos=586&amp;ncbi_uid=3319970&amp;link_uid=PMC3201973" href="/pmc/articles/PMC3201973/"><img src="/pmc/articles/instance/3201973/bin/pone.0026766.g009.gif" src-large="/pmc/articles/instance/3201973/bin/pone.0026766.g009.jpg" alt="Figure 9" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3201973/" ref="ordinalpos=586&amp;ncbi_uid=3319970&amp;link_uid=PMC3201973" image-link="/pmc/articles/PMC3201973/figure/pone-0026766-g009/" class="imagepopup">Figure 9. PIC treatment inhibits the EGF-Shh-mediated Ras-ERK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> in primary cortical neurospheres..  From: Toll-like Receptor 3 Regulates Neural Stem Cell Proliferation by Modulating the Sonic Hedgehog <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Shh binds to the cell surface receptor Ptch, relieves the inhibition of Smo and triggers Shh <span class="highlight" style="background-color:">signaling</span>. Activation of the Shh <span class="highlight" style="background-color:">pathway</span> results in increased levels of Gli1 expression and activation of the downstream Ras-MEK-ERK <span class="highlight" style="background-color:">pathway</span>, causing cell proliferation. In WT neurospheres cultured in the presence of EGF, PIC signals via TLR3 to inhibit Gli1 expression and interfere with MEK and ERK phosphorylation. Inhibition of MEK-ERK activation results in the decreased proliferation and induced Fas expression and apoptosis observed after PIC treatment of primary cortical neurospheres.</div></p></div><div class="aux"><div class="resc">Kavitha Yaddanapudi, et al. PLoS One. 2011;6(10):e26766.</div><p class="links"><a class="dblinks" href="/pubmed/22046349" ref="ordinalpos=586&amp;ncbi_uid=3319970&amp;link_uid=22046349">Citation</a><a class="dblinks" href="/pmc/articles/PMC3201973/" ref="ordinalpos=586&amp;ncbi_uid=3319970&amp;link_uid=PMC3201973">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>587.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3507288/figure/f03/" ref="ordinalpos=587&amp;ncbi_uid=4424640&amp;link_uid=PMC3507288" href="/pmc/articles/PMC3507288/"><img src="/pmc/articles/instance/3507288/bin/bio-01-03-261-f03.gif" src-large="/pmc/articles/instance/3507288/bin/bio-01-03-261-f03.jpg" alt="Fig. 3." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3507288/" ref="ordinalpos=587&amp;ncbi_uid=4424640&amp;link_uid=PMC3507288" image-link="/pmc/articles/PMC3507288/figure/f03/" class="imagepopup">Fig. 3.HLH-29 and the IP3 <span class="highlight" style="background-color:">signaling</span>..  From: HLH-29 regulates ovulation in C. elegans by targeting genes in the inositol triphosphate <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>(A)</b> Transcriptional activity of genes in the IP<sub>3</sub> <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> are affected in <i>hlh-29(tm284)</i> animals. Bars represent the relative fold-change in mRNA levels as detected by RT-qPCR when compared to expression in wild-type animals, error bars represent standard error of the mean. Fold expression range considered to be the same as wild-type expression is indicated by light gray shading, centered around the value of 1.0. P values are indicated to the right of each bar. <b>(B)</b> Components of the LIN-3/LET-23 activated IP<sub>3</sub> <span class="highlight" style="background-color:">signaling</span> transduction <span class="highlight" style="background-color:">pathway</span> are transcriptionally regulated by HLH-29.</div></p></div><div class="aux"><div class="resc">Ana White, et al. Biol Open. 2012 March 15;1(3):261-268.</div><p class="links"><a class="dblinks" href="/pubmed/23213416" ref="ordinalpos=587&amp;ncbi_uid=4424640&amp;link_uid=23213416">Citation</a><a class="dblinks" href="/pmc/articles/PMC3507288/" ref="ordinalpos=587&amp;ncbi_uid=4424640&amp;link_uid=PMC3507288">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>588.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3608394/figure/F6/" ref="ordinalpos=588&amp;ncbi_uid=4743260&amp;link_uid=PMC3608394" href="/pmc/articles/PMC3608394/"><img src="/pmc/articles/instance/3608394/bin/nihms441874f6.gif" src-large="/pmc/articles/instance/3608394/bin/nihms441874f6.jpg" alt="Fig. 6" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3608394/" ref="ordinalpos=588&amp;ncbi_uid=4743260&amp;link_uid=PMC3608394" image-link="/pmc/articles/PMC3608394/figure/F6/" class="imagepopup">Fig. 6.  From: INTEGRATIVE NETWORK ANALYSIS TO IDENTIFY ABERRANT <span class="highlight" style="background-color:">PATHWAY</span> NETWORKS IN OVARIAN CANCER. </a></p><div class="supp"><p class="details"><div>Enriched pathways and GO functional annotations are: (a) ErbB <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>: p=9.10e-13; Signal transduction p=2.33e-07. (b) Notch <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> p=2.86e-15. Figure legends are same as the ones in Fig. 5.</div></p></div><div class="aux"><div class="resc">LI CHEN, et al. Pac Symp Biocomput. 2012: :31-42.</div><p class="links"><a class="dblinks" href="/pubmed/22174260" ref="ordinalpos=588&amp;ncbi_uid=4743260&amp;link_uid=22174260">Citation</a><a class="dblinks" href="/pmc/articles/PMC3608394/" ref="ordinalpos=588&amp;ncbi_uid=4743260&amp;link_uid=PMC3608394">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>589.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3608394/figure/F7/" ref="ordinalpos=589&amp;ncbi_uid=4743259&amp;link_uid=PMC3608394" href="/pmc/articles/PMC3608394/"><img src="/pmc/articles/instance/3608394/bin/nihms441874f7.gif" src-large="/pmc/articles/instance/3608394/bin/nihms441874f7.jpg" alt="Fig. 7" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3608394/" ref="ordinalpos=589&amp;ncbi_uid=4743259&amp;link_uid=PMC3608394" image-link="/pmc/articles/PMC3608394/figure/F7/" class="imagepopup">Fig. 7.  From: INTEGRATIVE NETWORK ANALYSIS TO IDENTIFY ABERRANT <span class="highlight" style="background-color:">PATHWAY</span> NETWORKS IN OVARIAN CANCER. </a></p><div class="supp"><p class="details"><div>Enriched pathways and GO functional annotations are: (a) NF<i>κ</i>B <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> p=7.03e-05. (b) TGF beta <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> p=1.88e-06. Figure legends are same as the ones in Fig. 5.</div></p></div><div class="aux"><div class="resc">LI CHEN, et al. Pac Symp Biocomput. 2012: :31-42.</div><p class="links"><a class="dblinks" href="/pubmed/22174260" ref="ordinalpos=589&amp;ncbi_uid=4743259&amp;link_uid=22174260">Citation</a><a class="dblinks" href="/pmc/articles/PMC3608394/" ref="ordinalpos=589&amp;ncbi_uid=4743259&amp;link_uid=PMC3608394">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>590.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2713381/figure/F8/" ref="ordinalpos=590&amp;ncbi_uid=2438550&amp;link_uid=PMC2713381" href="/pmc/articles/PMC2713381/"><img src="/pmc/articles/instance/2713381/bin/nihms94941f8.gif" src-large="/pmc/articles/instance/2713381/bin/nihms94941f8.jpg" alt="Figure 8" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2713381/" ref="ordinalpos=590&amp;ncbi_uid=2438550&amp;link_uid=PMC2713381" image-link="/pmc/articles/PMC2713381/figure/F8/" class="imagepopup">Figure 8. MMP-9 activation of trophic <span class="highlight" style="background-color:">signaling</span> in Schwann cells (a schematic diagram).  From: MMP-9 controls Schwann cell proliferation and phenotypic remodeling via IGF-1 and ErbB receptor-mediated activation of MEK/ERK <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div>MMP-9 expression is induced by proinflammatory stimuli (see Fig. 1). MMP-9 stimulates MAPK p44/42 (or ERK1/2) <span class="highlight" style="background-color:">signaling</span> in Schwann cells via activation IGF-1, ErbB4, and PDGF tyrosine kinase receptor and Ras/Raf/MEK <span class="highlight" style="background-color:">pathway</span> (shown in black), but not PI3K/PDK/MEK <span class="highlight" style="background-color:">pathway</span> (shown in grey), as determined using the specified pharmacologic inhibitors (see Fig. 6).</div></p></div><div class="aux"><div class="resc">Sharmila Chattopadhyay, et al. Glia. ;57(12):1316-1325.</div><p class="links"><a class="dblinks" href="/pubmed/19229995" ref="ordinalpos=590&amp;ncbi_uid=2438550&amp;link_uid=19229995">Citation</a><a class="dblinks" href="/pmc/articles/PMC2713381/" ref="ordinalpos=590&amp;ncbi_uid=2438550&amp;link_uid=PMC2713381">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>591.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC1317640/figure/f9/" ref="ordinalpos=591&amp;ncbi_uid=710556&amp;link_uid=PMC1317640" href="/pmc/articles/PMC1317640/"><img src="/pmc/articles/instance/1317640/bin/zmb0010656280009.gif" src-large="/pmc/articles/instance/1317640/bin/zmb0010656280009.jpg" alt="FIG. 9." /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC1317640/" ref="ordinalpos=591&amp;ncbi_uid=710556&amp;link_uid=PMC1317640" image-link="/pmc/articles/PMC1317640/figure/f9/" class="imagepopup">FIG. 9. From: Transactivation of Platelet-Derived Growth Factor Receptor ? by the GTPase-Deficient Activated Mutant of G?12. </a></p><div class="supp"><p class="details"><div>Schematic representation of Gα<sub>12</sub>-mediated activation of PDGFRα <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. Activated Gα<sub>12</sub> stimulates the expression of PDGFRα via a Rho-dependent <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span>. The increased expression of PDGF-A in these cells stimulates PDGFRα, thus forming an autocrine <span class="highlight" style="background-color:">signaling</span> loop. The Gα<sub>12</sub>-stimulated PDGFRα engages the downstream PI3K-AKT <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> to inhibit FKHRL1-mediated transcription of apoptotic and antiproliferative genes to promote neoplastic cell growth.</div></p></div><div class="aux"><div class="resc">Rashmi N. Kumar, et al. Mol Cell Biol. 2006 January;26(1):50-62.</div><p class="links"><a class="dblinks" href="/pubmed/16354679" ref="ordinalpos=591&amp;ncbi_uid=710556&amp;link_uid=16354679">Citation</a><a class="dblinks" href="/pmc/articles/PMC1317640/" ref="ordinalpos=591&amp;ncbi_uid=710556&amp;link_uid=PMC1317640">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>592.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2645475/figure/F4/" ref="ordinalpos=592&amp;ncbi_uid=1675272&amp;link_uid=PMC2645475" href="/pmc/articles/PMC2645475/"><img src="/pmc/articles/instance/2645475/bin/nihms72752f4.gif" src-large="/pmc/articles/instance/2645475/bin/nihms72752f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2645475/" ref="ordinalpos=592&amp;ncbi_uid=1675272&amp;link_uid=PMC2645475" image-link="/pmc/articles/PMC2645475/figure/F4/" class="imagepopup">Figure 4. Potential mechanism for enhancement TGF-β <span class="highlight" style="background-color:">signaling</span> in the presence of receptor mutations.  From: Transforming growth factor-beta <span class="highlight" style="background-color:">signaling</span> in thoracic aortic aneurysm development: a paradox in pathogenesis. </a></p><div class="supp"><p class="details"><div>The endocytosis and subsequent recycling or degradation of TGF-β receptors is an important mechanism for regulating TGF-β <span class="highlight" style="background-color:">signaling</span>. There are two independent endocytic pathways, regulated by accessory protein interactions that exist to regulate TGF-β <span class="highlight" style="background-color:">signaling</span>: <b>A</b>.) the clathrin-mediated <span class="highlight" style="background-color:">pathway</span> leading to receptor recycling, and <b>B</b>.) the caveolin-mediated <span class="highlight" style="background-color:">pathway</span> leading to proteasomal degradation of TGF-β receptors. The TGF-βRII mutations identified in several aneurysm syndromes may contribute to the enhanced TGF-β <span class="highlight" style="background-color:">signaling</span> by facilitating interactions with SARA, or by diminishing association with Smad7, favoring receptor recycling and resulting in prolonged activation of TGF-β signals.</div></p></div><div class="aux"><div class="resc">Jeffrey A. Jones, et al. J Vasc Res. ;46(2):119-137.</div><p class="links"><a class="dblinks" href="/pubmed/18765947" ref="ordinalpos=592&amp;ncbi_uid=1675272&amp;link_uid=18765947">Citation</a><a class="dblinks" href="/pmc/articles/PMC2645475/" ref="ordinalpos=592&amp;ncbi_uid=1675272&amp;link_uid=PMC2645475">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>593.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2884164/figure/F13/" ref="ordinalpos=593&amp;ncbi_uid=2298044&amp;link_uid=PMC2884164" href="/pmc/articles/PMC2884164/"><img src="/pmc/articles/instance/2884164/bin/1755-8794-3-13-13.gif" src-large="/pmc/articles/instance/2884164/bin/1755-8794-3-13-13.jpg" alt="Figure 13" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2884164/" ref="ordinalpos=593&amp;ncbi_uid=2298044&amp;link_uid=PMC2884164" image-link="/pmc/articles/PMC2884164/figure/F13/" class="imagepopup">Figure 13.  From: <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">pathway</span> networks mined from human pituitary adenoma proteomics data. </a></p><div class="supp"><p class="details"><div><b>p38 MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> that is involved with pituitary adenoma nitroproteins</b>. A gray label denotes an identified protein. The various shapes of nodes denote the different functions. A duplicated shape means this node contains multiple components. An arrow denotes the <span class="highlight" style="background-color:">pathway</span> direction. A line with a small circle denotes a biological result.</div></p></div><div class="aux"><div class="resc">Xianquan Zhan, et al. BMC Med Genomics. 2010;3:13-13.</div><p class="links"><a class="dblinks" href="/pubmed/20426862" ref="ordinalpos=593&amp;ncbi_uid=2298044&amp;link_uid=20426862">Citation</a><a class="dblinks" href="/pmc/articles/PMC2884164/" ref="ordinalpos=593&amp;ncbi_uid=2298044&amp;link_uid=PMC2884164">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>594.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4200202/figure/Fig5/" ref="ordinalpos=594&amp;ncbi_uid=6731918&amp;link_uid=PMC4200202" href="/pmc/articles/PMC4200202/"><img src="/pmc/articles/instance/4200202/bin/12943_2014_1433_Fig5_HTML.gif" src-large="/pmc/articles/instance/4200202/bin/12943_2014_1433_Fig5_HTML.jpg" alt="Figure 5" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4200202/" ref="ordinalpos=594&amp;ncbi_uid=6731918&amp;link_uid=PMC4200202" image-link="/pmc/articles/PMC4200202/figure/Fig5/" class="imagepopup">Figure 5.  From: miR-193a-3p regulates the multi-drug resistance of bladder cancer by targeting the LOXL4 gene and the Oxidative Stress <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div> <b>The effects of the forced reversal of both miR-193a-3p and LOXL4 levels on the activity of the <span class="highlight" style="background-color:">signaling</span> pathways in 5637 versus H-bc cells. A</b>, The relative activities (in mean ± S.D) of five indicated pathways in 5637 versus H-bc cells. <b>B</b>, The relative <span class="highlight" style="background-color:">pathway</span> activities in the LOXL4 siRNA or miR-193a-3p mimic (3 PM) versus the NC transfected 5637 cells. <b>C</b>, The relative <span class="highlight" style="background-color:">pathway</span> activities in the miR-193a-3p antagomiR (3PA) versus the NC transfected H-bc cells. <b>D</b>, The level of the OS <span class="highlight" style="background-color:">pathway</span> related mRNAs in 5637 and H-bc cells. <b>E</b>, The level of the OS <span class="highlight" style="background-color:">pathway</span> related mRNAs in the miR-193a-3p mimic or siRNA versus the NC transfected 5637 cell. <b>F</b>, The level of the OS <span class="highlight" style="background-color:">pathway</span> related mRNA s in the miR-193a-3p antagomiR versus the NC transfected H-bc cells. (*, p &lt;0.05; **,P &lt; 0.01).</div></p></div><div class="aux"><div class="resc">Hui Deng, et al. Mol Cancer. 2014;13(1):234.</div><p class="links"><a class="dblinks" href="/pubmed/25311867" ref="ordinalpos=594&amp;ncbi_uid=6731918&amp;link_uid=25311867">Citation</a><a class="dblinks" href="/pmc/articles/PMC4200202/" ref="ordinalpos=594&amp;ncbi_uid=6731918&amp;link_uid=PMC4200202">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>595.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3676498/figure/F2/" ref="ordinalpos=595&amp;ncbi_uid=5255171&amp;link_uid=PMC3676498" href="/pmc/articles/PMC3676498/"><img src="/pmc/articles/instance/3676498/bin/psb-8-e23316-g2.gif" src-large="/pmc/articles/instance/3676498/bin/psb-8-e23316-g2.jpg" alt="" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3676498/" ref="ordinalpos=595&amp;ncbi_uid=5255171&amp;link_uid=PMC3676498" image-link="/pmc/articles/PMC3676498/figure/F2/" class="imagepopup">Figure. From: Sucrose <span class="highlight" style="background-color:">signaling</span> in plants. </a></p><div class="supp"><p class="details"><div><b>Figure 2.</b> A schematic diagram of the Suc <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> and its interaction with <span class="highlight" style="background-color:">signaling</span> by light and cold. Components of the intracellular Suc <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> include the still unknown sensor, cytosolic intermediates, transcription factors that are target of Suc and the SURE response element found in many promoters exhibiting Suc regulation. Suc mimics the effect of high light and cold on a number of metabolic and developmental responses. Horizontal arrows indicate well-known (PhyA and ABA) and putative components of light and low temperature <span class="highlight" style="background-color:">signaling</span> pathways with may cross-talk with Suc.</div></p></div><div class="aux"><div class="resc">Jorge A. Tognetti, et al. Plant Signal Behav. 2013 March 1;8(3):e23316.</div><p class="links"><a class="dblinks" href="/pubmed/23333971" ref="ordinalpos=595&amp;ncbi_uid=5255171&amp;link_uid=23333971">Citation</a><a class="dblinks" href="/pmc/articles/PMC3676498/" ref="ordinalpos=595&amp;ncbi_uid=5255171&amp;link_uid=PMC3676498">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>596.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2237899/figure/pone-0001639-g011/" ref="ordinalpos=596&amp;ncbi_uid=1265551&amp;link_uid=PMC2237899" href="/pmc/articles/PMC2237899/"><img src="/pmc/articles/instance/2237899/bin/pone.0001639.g011.gif" src-large="/pmc/articles/instance/2237899/bin/pone.0001639.g011.jpg" alt="Figure 11" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2237899/" ref="ordinalpos=596&amp;ncbi_uid=1265551&amp;link_uid=PMC2237899" image-link="/pmc/articles/PMC2237899/figure/pone-0001639-g011/" class="imagepopup">Figure 11. Summary diagram of cytokine <span class="highlight" style="background-color:">signaling</span> pathways controlling body weight..  From: Loss of Cytokine-STAT5 <span class="highlight" style="background-color:">Signaling</span> in the CNS and Pituitary Gland Alters Energy Balance and Leads to Obesity. </a></p><div class="supp"><p class="details"><div>While leptin preferentially signals through STAT3, GM-CSF utilizes STAT5. However, since the GM-CSF-null mice develop milder symptoms than the STAT5-mutant mice described in this study, we propose that some of the STAT5 <span class="highlight" style="background-color:">signaling</span> is induced by leptin or yet another unknown <span class="highlight" style="background-color:">pathway</span>.</div></p></div><div class="aux"><div class="resc">Ji-Yeon Lee, et al. PLoS ONE. 2008;3(2):e1639.</div><p class="links"><a class="dblinks" href="/pubmed/18286195" ref="ordinalpos=596&amp;ncbi_uid=1265551&amp;link_uid=18286195">Citation</a><a class="dblinks" href="/pmc/articles/PMC2237899/" ref="ordinalpos=596&amp;ncbi_uid=1265551&amp;link_uid=PMC2237899">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>597.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC4093200/figure/f4-ajas-27-5-733-15/" ref="ordinalpos=597&amp;ncbi_uid=6399619&amp;link_uid=PMC4093200" href="/pmc/articles/PMC4093200/"><img src="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f4.gif" src-large="/pmc/articles/instance/4093200/bin/ajas-27-5-733-15f4.jpg" alt="Figure 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC4093200/" ref="ordinalpos=597&amp;ncbi_uid=6399619&amp;link_uid=PMC4093200" image-link="/pmc/articles/PMC4093200/figure/f4-ajas-27-5-733-15/" class="imagepopup">Figure 4.  From: Effect of Glucagon-like Peptide 2 on Tight Junction in Jejunal Epithelium of Weaned Pigs though MAPK <span class="highlight" style="background-color:">Signaling</span> <span class="highlight" style="background-color:">Pathway</span>. </a></p><div class="supp"><p class="details"><div>Effect of GLP-2 (100 nmol/L) on tight junction proteins and <span class="highlight" style="background-color:">signaling</span> molecules expression with MAPK <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> blocked by U0126 (10 μmol/L). GLP-2, glucagon-like peptide 2; MARK, mitogen-activated protein kinase; ZO-1, zonula occludens-1. <sup>a,b,c</sup> Treatments with different letters are different at p&lt;0.05.</div></p></div><div class="aux"><div class="resc">Changsong Yu, et al. Asian-Australas J Anim Sci. 2014 May;27(5):733-742.</div><p class="links"><a class="dblinks" href="/pubmed/25050009" ref="ordinalpos=597&amp;ncbi_uid=6399619&amp;link_uid=25050009">Citation</a><a class="dblinks" href="/pmc/articles/PMC4093200/" ref="ordinalpos=597&amp;ncbi_uid=6399619&amp;link_uid=PMC4093200">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>598.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3160786/figure/GMR066F1/" ref="ordinalpos=598&amp;ncbi_uid=3626316&amp;link_uid=PMC3160786" href="/pmc/articles/PMC3160786/"><img src="/pmc/articles/instance/3160786/bin/gmr06601.gif" src-large="/pmc/articles/instance/3160786/bin/gmr06601.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3160786/" ref="ordinalpos=598&amp;ncbi_uid=3626316&amp;link_uid=PMC3160786" image-link="/pmc/articles/PMC3160786/figure/GMR066F1/" class="imagepopup">Figure 1.  From: Spatial regulation of the mTORC1 system in amino acids sensing <span class="highlight" style="background-color:">pathway</span>. </a></p><div class="supp"><p class="details"><div><b>mTOR <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span></b> The scheme depicts mTOR <span class="highlight" style="background-color:">signaling</span> cascade in the regulation of energy and growth factor-dependent translation and autophagy.</div></p></div><div class="aux"><div class="resc">Tsukasa Suzuki, et al. Acta Biochim Biophys Sin (Shanghai). 2011 September;43(9):671-679.</div><p class="links"><a class="dblinks" href="/pubmed/21785113" ref="ordinalpos=598&amp;ncbi_uid=3626316&amp;link_uid=21785113">Citation</a><a class="dblinks" href="/pmc/articles/PMC3160786/" ref="ordinalpos=598&amp;ncbi_uid=3626316&amp;link_uid=PMC3160786">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>599.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC3202302/figure/F4/" ref="ordinalpos=599&amp;ncbi_uid=3512112&amp;link_uid=PMC3202302" href="/pmc/articles/PMC3202302/"><img src="/pmc/articles/instance/3202302/bin/nihms322868f4.gif" src-large="/pmc/articles/instance/3202302/bin/nihms322868f4.jpg" alt="Fig. 4" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC3202302/" ref="ordinalpos=599&amp;ncbi_uid=3512112&amp;link_uid=PMC3202302" image-link="/pmc/articles/PMC3202302/figure/F4/" class="imagepopup">Fig. 4. <span class="highlight" style="background-color:">Signaling</span> and crosstalk between the BCR and BAFF-R.  From: <span class="highlight" style="background-color:">Signaling</span> by the TNFR superfamily in B-cell biology and disease. </a></p><div class="supp"><p class="details"><div><span class="highlight" style="background-color:">Signaling</span> via the BCR and BAFF-R are both critical for B-cell maturity and homeostasis. Phosphorylation of tyrosine residues in the immunoreceoptor tyrosine activation motifs (ITAMs) of the cytoplasmic tails of BCR-associated Igα and Igβ chains initiates BCR-proximal <span class="highlight" style="background-color:">signaling</span> and the formation of an active signalosome. Phosphorylation of ITAM tyrosines by Src family kinases leads to the recruitment and activation of the kinase Syk and the Tec family kinase BTK, as well as phosphorylation and recruitment of the adaptor BLNK and the lipase PLCγ2. The assembly of an active BCR-proximal <span class="highlight" style="background-color:">signaling</span> complex ultimately leads to activation of MAPKs (not pictured) and canonical NFκB <span class="highlight" style="background-color:">signaling</span> via PKCβ and the CARMA1-MALT1-BCL10 complex which functions to activate IKK1/IKK2. Activation of the IKK1-IKK2-NEMO complex results in phosphorylation of IκBα, marking it for ubiquitination and proteosomal degradation. Activation of canonical NFκB <span class="highlight" style="background-color:">signaling</span> by the BCR leads to induction of transcription of p100, BAFF-R, and cRel (among others), which are critical for BAFF-R <span class="highlight" style="background-color:">signaling</span>. The CD19 coreceptor molecule promotes activation of the PI3K <span class="highlight" style="background-color:">pathway</span> leading to activation of AKT. AKT activation promotes B-cell survival and metabolism through the inhibition of FOXO1, upregulation of MCL-1 and inhibition of BIM, as well as activation of the mTORC1 complex. BAFF-R <span class="highlight" style="background-color:">signaling</span> also activates PI3K (via an unknown mechanism), as well as non-canonical NFκB <span class="highlight" style="background-color:">signaling</span>. Following BAFF engagement of the BAFF-R, TRAF2 and TRAF3 are recruited leading to the release of NIK, which phosphorylates IKK1 leading to p100 processing to p52 and activation of non-canonical NFκB.</div></p></div><div class="aux"><div class="resc">Robert C. Rickert, et al. Immunol Rev. ;244(1):115-133.</div><p class="links"><a class="dblinks" href="/pubmed/22017435" ref="ordinalpos=599&amp;ncbi_uid=3512112&amp;link_uid=22017435">Citation</a><a class="dblinks" href="/pmc/articles/PMC3202302/" ref="ordinalpos=599&amp;ncbi_uid=3512112&amp;link_uid=PMC3202302">Full text</a></p></div></div><div class="clear"></div></div></div><div class="rprt"><div class="rprtnum nohighlight"><span>600.</span></div><div class="rslt"><a class="rprt_img figpopup imagepopup" image-link="/pmc/articles/PMC2838377/figure/F1/" ref="ordinalpos=600&amp;ncbi_uid=2236643&amp;link_uid=PMC2838377" href="/pmc/articles/PMC2838377/"><img src="/pmc/articles/instance/2838377/bin/nihms150685f1.gif" src-large="/pmc/articles/instance/2838377/bin/nihms150685f1.jpg" alt="Figure 1" /></a><div class="rprt_cont"><p class="title"><a href="/pmc/articles/PMC2838377/" ref="ordinalpos=600&amp;ncbi_uid=2236643&amp;link_uid=PMC2838377" image-link="/pmc/articles/PMC2838377/figure/F1/" class="imagepopup">Figure 1.  From: Tissue Factor-Factor VIIa <span class="highlight" style="background-color:">Signaling</span>. </a></p><div class="supp"><p class="details"><div>A schematic representation of TF–VIIa protease-induced <span class="highlight" style="background-color:">signaling</span>. TF–VIIa activates PAR2 and the ternary complex of TF–VIIa–Xa activates both PAR1 and PAR2. Activation of PAR2-specific <span class="highlight" style="background-color:">signaling</span> <span class="highlight" style="background-color:">pathway</span> may lead to phosphorylation of TF cytoplasmic tail. Phosphorylation of TF cytoplasmic domain releases its negative regulatory control of PAR2-mediated <span class="highlight" style="background-color:">signaling</span>.</div></p></div><div class="aux"><div class="resc">L. Vijaya Mohan Rao, et al. Arterioscler Thromb Vasc Biol. ;25(1):47-56.</div><p class="links"><a class="dblinks" href="/pubmed/15569823" ref="ordinalpos=600&amp;ncbi_uid=2236643&amp;link_uid=15569823">Citation</a><a class="dblinks" href="/pmc/articles/PMC2838377/" ref="ordinalpos=600&amp;ncbi_uid=2236643&amp;link_uid=PMC2838377">Full text</a></p></div></div><div class="clear"></div></div></div></div>
        <div id="messagearea_bottom">
            
        </div>
        <div class="title_and_pager bottom">
            <div class="pagination"><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="First page of results" class="active page_link" href="#" sid="6" page="1" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt;&lt; First</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Previous page of results" class="active page_link prev" href="#" sid="7" page="5" accesskey="j" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">&lt; Prev</a><h3 class="page"><label for="pageno2">Page </label><input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.cPage" id="pageno2" type="text" class="num" sid="6" value="6" last="407" /> of 407</h3><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Next page of results" class="active page_link next" href="#" sid="8" page="7" accesskey="k" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Next &gt;</a><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page" title="Last page of results" class="active page_link" href="#" sid="9" page="407" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Entrez_Pager.Page">Last &gt;&gt;</a></div>
        </div>
        <div class="results_settings two_settings bottom"><h4 class="left display_settings content_header jig-ncbipopper" id="display_set2" data-jigconfig="triggerPosition : 'top center',destPosition : 'bottom center',destSelector : '#display_settings_menu2',             hasArrow : false,openEvent : 'click',closeEvent : 'click',isTriggerElementCloseClick: false,addCloseButton : true,              multipleHandlesSelector:'#display_set2 ~ ul.display_settings a', groupName: 'entrez_pg'"><a href="#" class="tgt_dark">Display Settings:</a></h4><ul class="inline_list left display_settings"><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="12" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">PMC Images,</a></li><li><a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display" sid="13" href="#" sourceContent="display_settings_menu2" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.Display">100 per page</a></li></ul><div id="display_settings_menu2" class="disp_settings tabPopper"><fieldset class="items"><legend>Items per page</legend><ul class="column_list"><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="11" value="5" id="ps52" /><label for="ps52">5</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="12" value="10" id="ps102" /><label for="ps102">10</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="13" value="20" id="ps202" /><label for="ps202">20</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="14" value="50" id="ps502" /><label for="ps502">50</label></li><li><input type="radio" name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.sPageSize2" sid="15" value="100" id="ps1002" checked="true" /><label for="ps1002">100</label></li></ul></fieldset><button name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.Pmc_DisplayBar.SetDisplay" sid="2" class="button_apply ncbipopper-close-button">Apply</button></div></div>
    </div>
</div>  
<div class="supplemental col three_col last">
    <h2 class="offscreen_noflow">Supplemental Content</h2>
      
    <div>
    <div class="portlet">
  <div class="portlet_head">
    <div class="portlet_title">
      <h3>Recent activity</h3>
    </div>
    <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter" sid="1" href="#" class="portlet_shutter" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Shutter"></a>
  </div>
  <div class="portlet_content">
    <div id="HTDisplay" class="">
      <input name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.Cmd" sid="1" type="hidden" />
      <div class="action">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory" cmd="ClearHT" href="?cmd=ClearHT&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.ClearHistory">
                                Clear
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOn" cmd="HTOff" href="?cmd=HTOff&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn Off                        
                            </a>
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" sid="2" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle" class="HTOff" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryToggle">
                                Turn On                        
                            </a>
      </div>
      <ul id="activity">
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=553fd41c437716036ac7a6be">signaling pathway <span class="number">(472213)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=553ebcfa437716036a8a1979">cytoscape tool <span class="number">(3429)</span></a>
          <div class="tertiary">PMC</div>
        </li>
        <li class="ra_rcd ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a54aabce7726946b1">Jumonji modulates polycomb activity and self-renewal versus differentiation of s...</a>
          <div class="ralinkpop offscreen_noflow">Jumonji modulates polycomb activity and self-renewal versus differentiation of stem cells.<div class="brieflinkpopdesc">Cell. 2009 Dec 24 ;139(7):1303-14. doi: 10.1016/j.cell.2009.12.003.</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry ralinkpopper two_line">
          <a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=553aaf7a437716036ae3c40d">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activ... <span class="number">(1)</span></a>
          <div class="ralinkpop offscreen_noflow">Jumonji[Title] AND modulates[Title] AND Polycomb[Title] AND activity[Title] AND self-renewal[Title] AND versus[Title] AND differentiation[Title] AND stem[Title] AND cells[Title]<div class="brieflinkpopdesc">Search</div></div>
          <div class="tertiary">PubMed</div>
        </li>
        <li class="ra_qry two_line">
          <a class="htb" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=553a99dd437716036adee376">(stk381) AND "Homo sapiens"[porgn] AND (alive[property]) <span class="number">(0)</span></a>
          <div class="tertiary">Gene</div>
        </li>
      </ul>
      <p class="HTOn">Your browsing activity is empty.</p>
      <p class="HTOff">Activity recording is turned off.</p>
      <p id="turnOn" class="HTOff">
        <a name="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" sid="1" realname="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn" cmd="HTOn" href="?cmd=HTOn&amp;" onclick="return false;" id="EntrezSystem2.PEntrez.PMC.Pmc_ResultsPanel.HistoryDisplay.HistoryOn">Turn recording back on</a>
      </p>
      <a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
    </div>
  </div>
</div>

</div>
</div>
    
    <div id="NCBIFooter_dynamic">
    <div class="breadcrumbs">You are here: 
            <span id="breadcrumb_text"><a href="/guide/">NCBI</a> &gt; <a href="http://www.ncbi.nlm.nih.gov/guide/literature/">Literature</a> &gt; PubMed Central (PMC)</span></div>
    <a id="help-desk-link" class="help_desk jig-ncbihelpwindow" target="_blank" href="/sites/ehelp?&amp;Ncbi_App=entrez&amp;Db=pmc&amp;Page=imagesdocsum&amp;Snapshot=/projects/PMC/PMCEntrez@2.10&amp;Time=2015-04-28T16:08:08-04:00&amp;Host=ptpmc101&amp;Data=+NCBI_Phid:+8A1A728253FE83810000000000F28929;+PageURL:+http://www.ncbi.nlm.nih.gov/pmc;">Write to the Help Desk</a>
    <noscript><img alt="" src="/stat?jsdisabled=true&amp;ncbi_app=entrez&amp;ncbi_db=pmc&amp;ncbi_pdid=imagesdocsum&amp;ncbi_phid=8A1A728253FE83810000000000F28929" /></noscript>
</div>

    <div xmlns="http://www.w3.org/1999/xhtml" class="footer" id="footer" xml:base="http://127.0.0.1/sites/static/header_footer/">
    <!--    <div class="breadcrumbs">You are here: <span id="breadcrumb_text"><a href="#">NCBI</a></span></div>
        <a id="help-desk-link" class="help_desk" target="_blank">Help Desk</a>-->
    <div class="subfooter">
        <h2 class="offscreen_noflow">Simple NCBI Directory</h2>
        <ul class="foot_list" id="getting_started">
            <li><h3>Getting Started</h3></li>
            <!-- removed; hp-250           <li><a href="/guide/all/">Site Map</a></li>-->
            <li><a href="/education/">NCBI Education</a></li>
            <li><a href="/books/NBK3831/">NCBI Help Manual</a></li>
            <li><a href="/books/NBK143764/">NCBI Handbook</a></li>
            <li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li>
        </ul>
        <!-- this section is re-written by an XSLT before output -->
        <ul class="foot_list" id="resources">
            <li><h3>Resources</h3></li>
            <!-- fill this in with categories -->
        <li><a href="/guide/chemicals-bioassays/">Chemicals &amp; Bioassays</a></li><li><a href="/guide/data-software/">Data &amp; Software</a></li><li><a href="/guide/dna-rna/">DNA &amp; RNA</a></li><li><a href="/guide/domains-structures/">Domains &amp; Structures</a></li><li><a href="/guide/genes-expression/">Genes &amp; Expression</a></li><li><a href="/guide/genetics-medicine/">Genetics &amp; Medicine</a></li><li><a href="/guide/genomes-maps/">Genomes &amp; Maps</a></li><li><a href="/guide/homology/">Homology</a></li><li><a href="/guide/literature/">Literature</a></li><li><a href="/guide/proteins/">Proteins</a></li><li><a href="/guide/sequence-analysis/">Sequence Analysis</a></li><li><a href="/guide/taxonomy/">Taxonomy</a></li><li><a href="/guide/training-tutorials/">Training &amp; Tutorials</a></li><li><a href="/guide/variation/">Variation</a></li></ul>
        
        <ul class="foot_list" id="popular">
            <li><h3>Popular</h3></li>
            <li><a href="/pubmed/">PubMed</a></li>
            <li><a href="/books/">Bookshelf</a></li>
            <li><a href="/pmc/">PubMed Central</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></li>            
            <li><a href="/nucleotide/">Nucleotide</a></li>
            <li><a href="/genome/">Genome</a></li>
            <li><a href="/snp/">SNP</a></li>
            <li><a href="/gene/">Gene</a></li>
            <li><a href="/protein/">Protein</a></li>
            <li><a href="http://pubchem.ncbi.nlm.nih.gov/" class="newdomain">PubChem</a></li>
        </ul>
        <ul class="foot_list" id="featured">
            <li><h3>Featured</h3></li>
            <li><a href="/gtr/">Genetic Testing Registry</a></li>
            <li><a href="/pubmedhealth/">PubMed Health</a></li>
            <li><a href="/genbank/">GenBank</a></li>
            <li><a href="/refseq/">Reference Sequences</a></li>
            <li><a href="/geo/">Gene Expression Omnibus</a></li>
            <li><a href="/mapview/">Map Viewer</a></li>
            <li><a href="/genome/guide/human/">Human Genome</a></li>
            <li><a href="/genome/guide/mouse/">Mouse Genome</a></li>
            <li><a href="/genomes/FLU/">Influenza Virus</a></li>
            <li><a href="/tools/primer-blast/">Primer-BLAST</a></li>
            <li><a href="/Traces/sra/">Sequence Read Archive</a></li>
        </ul>
        <ul class="foot_list" id="info">
            <li><h3>NCBI Information</h3></li>
            <li><a href="/About/">About NCBI</a></li>
            <li><a href="/research/">Research at NCBI</a></li>
            <li><a href="/news/">NCBI News</a></li>
            <li><a href="ftp://ftp.ncbi.nlm.nih.gov/">NCBI FTP Site</a></li>
            <li><a class="external" href="https://www.facebook.com/ncbi.nlm">NCBI on Facebook</a></li>
            <li><a class="external" href="https://twitter.com/ncbi">NCBI on Twitter</a></li>
            <li><a class="external" href="http://www.youtube.com/ncbinlm">NCBI on YouTube</a></li>
        </ul>
    </div>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="http://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="http://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="http://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="http://www.dhhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="http://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            
            <a href="/About/disclaimer.html">Copyright</a> | <a href="/About/disclaimer.html#disclaimer">Disclaimer</a> |
            <a href="http://www.nlm.nih.gov/privacy.html" class="newdomain">Privacy</a> | 
            <a href="/guide/browsers/">Browsers</a> | <a href="http://www.nlm.nih.gov/accessibility.html">Accessibility</a> | <a href="/About/glance/contact_info.html">Contact</a>
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="http://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url"><a href="http://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
    
</div>
					<div><input name="EntrezSystem2.PEntrez.DbConnector.Db" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.LastDb" sid="1" type="hidden" value="pmc" /><input name="EntrezSystem2.PEntrez.DbConnector.Term" sid="1" type="hidden" value="signaling pathway" /><input name="EntrezSystem2.PEntrez.DbConnector.LastTabCmd" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastQueryKey" sid="1" type="hidden" value="1" /><input name="EntrezSystem2.PEntrez.DbConnector.IdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LastIdsFromResult" sid="1" type="hidden" value="" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkReadableName" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.LinkSrcDb" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.Cmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.TabCmd" sid="1" type="hidden" /><input name="EntrezSystem2.PEntrez.DbConnector.QueryKey" sid="1" type="hidden" /></div>
				<input type="hidden" name="p$a" id="p$a" /><input type="hidden" name="p$l" id="p$l" value="EntrezSystem2" /><input type="hidden" name="p$st" id="p$st" value="pmc" /><input name="SessionId" id="SessionId" value="CE8A7326511A03C1_0067SID" disabled="disabled" type="hidden" /><input name="Snapshot" id="Snapshot" value="/projects/PMC/PMCEntrez@2.10" disabled="disabled" type="hidden" /></form>
			</div>
		</div>
    

<!-- CE8A7326511A03C1_0067SID /projects/PMC/PMCEntrez@2.10 ptpmc101 v4.1.r465258 Mon, Apr 20 2015 02:46:05 -->


<script type='text/javascript' src='/portal/js/portal.js?v4%2E1%2Er465258+Mon%2C+Apr+20+2015+02%3A46%3A05'></script><script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4035897/js/3812534/4025445/3812535/3781606/3361551/4025474/3815706/3758627/2146917/1053413/3812291/3902119/3908752/3423/2099570/2868104/3397055/4001808.js" snapshot="pmc"></script></body>
</html>