<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1479345/" ref="ordinalpos=1855&amp;ncbi_uid=713499&amp;link_uid=PMC1479345" image-link="/pmc/articles/PMC1479345/figure/F4/" class="imagepopup">Figure 4.  From: How dietary arachidonic- and docosahexaenoic- acid rich oils differentially affect the murine hepatic transcriptome. </a></div><br /><div class="p4l_captionBody">Pathway analysis. Figures 3-7 represent signaling pathways for 58 focus genes selected from Table 3 by Ingenuity Systems software (Redwood City, CA. The following signaling cascades are shown: JUN, TNF, and CDKN2A signaling cascade affecting: DNA replication; recombination and repair; immune response; and cell cycle (Fig. 3); TGFB1 signaling cascade affecting: cell morphology; cancer; and tumor morphology (Fig. 4); CTNB1 signaling cascade affecting: cell signaling; gene expression; and cell cycle (Fig. 5); INS1/hRAS signaling cascade affecting: carbohydrate metabolism; endocrine disorders; and metabolic disease (Fig. 6); and MYC signaling cascade affecting: viral function; gene expression; and cell Cycle (Fig. 7). Differentiating groups (per Table 3) are overlaid onto the signaling diagrams, and abbreviated: FU, fungal; FI, fish oil; CO, combination diet. When CO was the differentiating group, absolute differences between FU and FI are indicated. Intracellular location of focus genes (subscripts) are annotated: C, cytoplasm; E, extracellular; N, nucleus; P, plasma membrane; U, unknown. Major canonical functional/signaling categories associated with genes in the figures identified by the software, are shown in yellow boxes.</div></div>