<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2925131/" ref="ordinalpos=1571&amp;ncbi_uid=3029390&amp;link_uid=PMC2925131" image-link="/pmc/articles/PMC2925131/figure/F5/" class="imagepopup">Fig. 5. <span class="highlight" style="background-color:">Pathway</span> analyses with the Gene Set Enrichment Analysis (GSEA) algorithm.  From: Genome-wide gene expression analysis identifies K-ras as a regulator of alcohol intake. </a></div><br /><div class="p4l_captionBody">To identify pathways regulated by alcohol in a K-ras dependent manner, the expression data set from the comparison of alcohol-treated K-ras+/− vs. Nf1+/− mice was ranked based on the signal-to-noise-ratio between the two genotypes and interrogated with gene sets representative of various molecular pathway. The top plots in each of the panels represent the enrichment score (ES). Asymmetrical distributions observed for ES (higher toward the left-hand side) indicate differential expression of genes in the pathway under study between K-ras+/− and Nf1+/− mice and their positive correlations with the Nf1+/− genotype, which is characterized by increased K-ras function (Costa et al., 2002). In the middle, are the corresponding heat maps and the “gene tags,” i.e., the location of genes from the gene sets representing the pathways under test. Ranked list metrics of gene expression levels are shown in the bottom plots. Differentially expressed pathways included: <b>A</b>) the phosphodiesterases (PDEs) pathway; <b>B</b>) the insulin signaling pathway, which includes the phosphoinositide-3-kinase PI3K pathway; <b>C</b>) the Jak/Stat pathway; <b>D</b>) the nuclear factor kB (NF-κB) pathway; and <b>E</b>) the adipokine signaling pathway. These pathways were not found to be significantly correlated with one of the two genotypes in the ACC of saline treated K-ras+/− vs. Nf1+/−.</div></div>