<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2694616/" ref="ordinalpos=25&amp;ncbi_uid=1936765&amp;link_uid=PMC2694616" image-link="/pmc/articles/PMC2694616/figure/F1/" class="imagepopup">Figure 1. <span class="highlight" style="background-color:">Pathway</span> module gene expression signatures.  From: A Genomic Strategy to Elucidate Modules of Oncogenic <span class="highlight" style="background-color:">Pathway</span> <span class="highlight" style="background-color:">Signaling</span> Networks. </a></div><br /><div class="p4l_captionBody">[Right] The schematic depicts an interconnected signaling network organized as collection of modules that define pathway structure. Perturbations of key genes result in changes to the transcriptional profile of the cell. Expression signatures developed from the pathway module analyses represent activity of units of this structure.[Left] To elucidate pathway modules, we decompose a pathway based on common patterns in the expression of the genes in the pathway. First, we identify a set of core pathway genes and a set of samples with microarray gene expression data evidencing a range of variation in activity in the pathway. We next apply the BFRM software to decompose the pathway into a series of signatures. As part of the modeling process, BFRM expands the pathway genes into a set that more comprehensively describes pathway structure, ultimately yielding a collection of estimated signatures. Finally, we link the signatures with known modules of the pathway. We reason that if a signature is significantly related to activity of a particular module of the pathway, then that signature is a representative of that pathway module.</div></div>