<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2425463/" ref="ordinalpos=616&amp;ncbi_uid=1487959&amp;link_uid=PMC2425463" image-link="/pmc/articles/PMC2425463/figure/F6/" class="imagepopup">Figure 6. From: Enhanced identification and biological validation of differential gene expression via Illumina whole-genome expression arrays through the use of the model-based background correction methodology. </a></div><br /><div class="p4l_captionBody">Comparison of pathway analysis on the data set using the MBCB, RMA or RAW background corrections. (<b>A</b>) IPA canonical pathway analysis depicting the top 9 signaling pathways that changed in AML tissues. The MBCB methodology identified more genes than either methodology in most categories. It also identified the death receptor pathway as the pathway that showed the most differences between the MBCB, RMA and RAW methods. (<b>B</b>) Scheme of death receptor pathway overlaid with the expression ratio of the MBCB data set. (<b>C</b>) Scheme of death receptor pathway overlaid with the expression ratio of the RMA data set. (<b>D</b>) Scheme of the death receptor pathway overlaid with the expression ratios from the RAW data set. Red, over-expressed in AML; green, under-expressed in AML; white, no significant change between AML and normal tissues. The cartoons show activation of Tnfr2 and NF-kappaB signaling in the MBCB data set and the failure of detection of the same pathway by the RMA and RAW data sets.</div></div>