<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3172280/" ref="ordinalpos=1054&amp;ncbi_uid=3225638&amp;link_uid=PMC3172280" image-link="/pmc/articles/PMC3172280/figure/pmed-1001093-g005/" class="imagepopup">Figure 5. Dynamic co-expression modules and their dominant trajectories across the five ocMOF subgroups..  From: Dissecting Inflammatory Complications in Critically Injured Patients by Within-Patient Gene Expression Changes: A Longitudinal Clinical Genomics Study. </a></div><br /><div class="p4l_captionBody">We used DAVID to obtain 54 functionally related gene sets from the 3,663 most significant probesets (10% FDR), which were then clustered into five modules according to the similarity of their dominant trajectories across the ocMOF subgroups. Modules A, B, C, D, and E contain 47, 37, 577, 231, and 364 probesets, respectively. We applied IPA to identify enriched pathways within each module. This IPA analysis shortlisted the following pathways as statistically significant (p-value&lt;0.002, after Bonferroni correction; see Table S5): Oxidative Phosphorylation (Module A); RAN, IL-10 and IL-6 signaling, and the Glycosphingolipid Biosynthesis-Lactoseries Pathway (Module C); Allograft Rejection Signaling, Antigen Presentation Pathway, Cytotoxic T Lymphocyte-mediated Apoptosis of Target Cells, OX40 Signaling Pathway, Nur77 Signaling in T Lymphocytes (Module D); and Protein Ubiquitination Pathway, Hypoxia Signaling, and Cleavage and Polyadenylation of Pre-mRNA in the Cardiovascular System (Module E). Note that Module A contains 47 probesets and one statistically significant pathway, and Module B contains 37 probesets and no statistically significant pathway.</div></div>