<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3931733/" ref="ordinalpos=1736&amp;ncbi_uid=6448182&amp;link_uid=PMC3931733" image-link="/pmc/articles/PMC3931733/figure/F5/" class="imagepopup">Fig. 5. From: Tbx20 acts upstream of Wnt <span class="highlight" style="background-color:">signaling</span> to regulate endocardial cushion formation and valve remodeling during mouse cardiogenesis. </a></div><br /><div class="p4l_captionBody"><b>Tbx20 loss of function in endocardium disrupts Wnt/β-catenin signaling in developing valves.</b> (<b>A</b>) Significant canonical pathways identified from genes differentially expressed in Tbx20 CKO versus control by ingenuity pathway analysis. The y-axis is the -log10 of the P-value for the enrichment of a pathway in the input gene list compared with genes in the whole genome. Red boxes highlight Wnt/β-catenin signaling pathway. The zigzag orange line indicates the enrichment ratio of the percentage of genes in a particular pathway over the percentage of pathway genes in the genome. (<b>B</b>) Significant Gene Ontology (GO) terms identified in the differentially expressed genes. x-axis is the -log10 of the P-value for the enrichment of a GO term in the input gene list compared with genes in the whole genome. (<b>C</b>) Heat map of Wnt signaling pathway genes. Red and green denotes upregulated and downregulated genes, respectively, in the Tbx20 CKO. (<b>D</b>) Relative expression of Wnt pathway genes at E10.5-12.5 determined by RT-qPCR. (<b>E-L</b>) X-Gal staining of TOPGAL indicator mice to detect active Wnt/β-catenin signaling at E11.5 (E-H) and E12.5 (I-L). The mutant tricuspid and mitral valves display fewer X-Gal-stained cells and fainter staining in the positive cells in both endocardium and adjacent mesenchyme (arrows in F,H,J,L versus E,G,I,K). G,H and K,L are higher magnification images of the tricuspid valve region (boxed) from E,F and I,J. (<b>M</b>) Relative expression of lacZ at different stages as determined by RT-qPCR. *P&lt;0.05, versus control. Error bars indicate s.d.</div></div>