<div class="p4l_caption"><div class="p4l_captionTitle"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4122365/" ref="ordinalpos=1978&amp;ncbi_uid=6488461&amp;link_uid=PMC4122365" image-link="/pmc/articles/PMC4122365/figure/pone-0103292-g001/" class="imagepopup">Figure 1. Polymorphisms identified in flies adapted to 4% O2 hypoxia..  From: Wnt <span class="highlight" style="background-color:">Pathway</span> Activation Increases Hypoxia Tolerance during Development. </a></div><br /><div class="p4l_captionBody">(A,B) Chromosome and gene region distributions of SNPs (A) or indels (B) were normalized to the size of (euchromatin) X chromosome or number of total intron loci, respectively. The legends specify the actual number of polymorphisms detected and polymorphisms/kB (euchromatin) chromosome or gene region. Not included are 14 SNPs that map to XHet, YHet or U/Uextra and 2 indels that map to U and Chromosome 4. (C) DAVID [11], [12]-identified KEGG [14], [15] and Panther [16] pathway enrichment of polymorphisms. The number of pathway genes with ≥1 polymorphism is plotted and the total number of pathway genes is indicated below each pathway name (TCA = Citrate, NK = NK cell mediated cytotoxicity, Circad = Circadian, D-V axis = Dorso-ventral axis formation, Cadh = Cadherin, Axon = Axon guidance (semaphorins), Gi/Gs = Gi/Gs mediated signaling, Gq/Go = Gq/Go mediated signaling). (D) The Wnt pathway was adapted from KEGG [14], [15] with additional interactors added. Genes with one or more fixed polymorphisms are indicated with a red star. Genes differentially expressed in post-eclosion AF maintained in 4% O2 are shaded in cyan (up-regulated) or yellow (down-regulated). Wnt pathway genes tested in genetic crosses are diamond-shaped; pathway activators are indicated by a red border and pathway inhibitors by a blue border.</div></div>